<!DOCTYPE html>

<html lang="en">
<head><meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>TemplateNotebook</title><script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
<style type="text/css">
    pre { line-height: 125%; }
td.linenos .normal { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
span.linenos { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
td.linenos .special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
span.linenos.special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
.highlight .hll { background-color: var(--jp-cell-editor-active-background) }
.highlight { background: var(--jp-cell-editor-background); color: var(--jp-mirror-editor-variable-color) }
.highlight .c { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment */
.highlight .err { color: var(--jp-mirror-editor-error-color) } /* Error */
.highlight .k { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword */
.highlight .o { color: var(--jp-mirror-editor-operator-color); font-weight: bold } /* Operator */
.highlight .p { color: var(--jp-mirror-editor-punctuation-color) } /* Punctuation */
.highlight .ch { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Hashbang */
.highlight .cm { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Multiline */
.highlight .cp { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Preproc */
.highlight .cpf { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.PreprocFile */
.highlight .c1 { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Single */
.highlight .cs { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Special */
.highlight .kc { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Constant */
.highlight .kd { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Declaration */
.highlight .kn { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Namespace */
.highlight .kp { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Pseudo */
.highlight .kr { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Reserved */
.highlight .kt { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Type */
.highlight .m { color: var(--jp-mirror-editor-number-color) } /* Literal.Number */
.highlight .s { color: var(--jp-mirror-editor-string-color) } /* Literal.String */
.highlight .ow { color: var(--jp-mirror-editor-operator-color); font-weight: bold } /* Operator.Word */
.highlight .pm { color: var(--jp-mirror-editor-punctuation-color) } /* Punctuation.Marker */
.highlight .w { color: var(--jp-mirror-editor-variable-color) } /* Text.Whitespace */
.highlight .mb { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Bin */
.highlight .mf { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Float */
.highlight .mh { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Hex */
.highlight .mi { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Integer */
.highlight .mo { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Oct */
.highlight .sa { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Affix */
.highlight .sb { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Backtick */
.highlight .sc { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Char */
.highlight .dl { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Delimiter */
.highlight .sd { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Doc */
.highlight .s2 { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Double */
.highlight .se { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Escape */
.highlight .sh { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Heredoc */
.highlight .si { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Interpol */
.highlight .sx { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Other */
.highlight .sr { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Regex */
.highlight .s1 { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Single */
.highlight .ss { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Symbol */
.highlight .il { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Integer.Long */
  </style>
<style type="text/css">
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*
 * Mozilla scrollbar styling
 */

/* use standard opaque scrollbars for most nodes */
[data-jp-theme-scrollbars='true'] {
  scrollbar-color: rgb(var(--jp-scrollbar-thumb-color))
    var(--jp-scrollbar-background-color);
}

/* for code nodes, use a transparent style of scrollbar. These selectors
 * will match lower in the tree, and so will override the above */
[data-jp-theme-scrollbars='true'] .CodeMirror-hscrollbar,
[data-jp-theme-scrollbars='true'] .CodeMirror-vscrollbar {
  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
}

/* tiny scrollbar */

.jp-scrollbar-tiny {
  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
  scrollbar-width: thin;
}

/* tiny scrollbar */

.jp-scrollbar-tiny::-webkit-scrollbar,
.jp-scrollbar-tiny::-webkit-scrollbar-corner {
  background-color: transparent;
  height: 4px;
  width: 4px;
}

.jp-scrollbar-tiny::-webkit-scrollbar-thumb {
  background: rgba(var(--jp-scrollbar-thumb-color), 0.5);
}

.jp-scrollbar-tiny::-webkit-scrollbar-track:horizontal {
  border-left: 0 solid transparent;
  border-right: 0 solid transparent;
}

.jp-scrollbar-tiny::-webkit-scrollbar-track:vertical {
  border-top: 0 solid transparent;
  border-bottom: 0 solid transparent;
}

/*
 * Lumino
 */

.lm-ScrollBar[data-orientation='horizontal'] {
  min-height: 16px;
  max-height: 16px;
  min-width: 45px;
  border-top: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='vertical'] {
  min-width: 16px;
  max-width: 16px;
  min-height: 45px;
  border-left: 1px solid #a0a0a0;
}

.lm-ScrollBar-button {
  background-color: #f0f0f0;
  background-position: center center;
  min-height: 15px;
  max-height: 15px;
  min-width: 15px;
  max-width: 15px;
}

.lm-ScrollBar-button:hover {
  background-color: #dadada;
}

.lm-ScrollBar-button.lm-mod-active {
  background-color: #cdcdcd;
}

.lm-ScrollBar-track {
  background: #f0f0f0;
}

.lm-ScrollBar-thumb {
  background: #cdcdcd;
}

.lm-ScrollBar-thumb:hover {
  background: #bababa;
}

.lm-ScrollBar-thumb.lm-mod-active {
  background: #a0a0a0;
}

.lm-ScrollBar[data-orientation='horizontal'] .lm-ScrollBar-thumb {
  height: 100%;
  min-width: 15px;
  border-left: 1px solid #a0a0a0;
  border-right: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='vertical'] .lm-ScrollBar-thumb {
  width: 100%;
  min-height: 15px;
  border-top: 1px solid #a0a0a0;
  border-bottom: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='horizontal']
  .lm-ScrollBar-button[data-action='decrement'] {
  background-image: var(--jp-icon-caret-left);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='horizontal']
  .lm-ScrollBar-button[data-action='increment'] {
  background-image: var(--jp-icon-caret-right);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='vertical']
  .lm-ScrollBar-button[data-action='decrement'] {
  background-image: var(--jp-icon-caret-up);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='vertical']
  .lm-ScrollBar-button[data-action='increment'] {
  background-image: var(--jp-icon-caret-down);
  background-size: 17px;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-Widget {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
}

.lm-Widget.lm-mod-hidden {
  display: none !important;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.lm-AccordionPanel[data-orientation='horizontal'] > .lm-AccordionPanel-title {
  /* Title is rotated for horizontal accordion panel using CSS */
  display: block;
  transform-origin: top left;
  transform: rotate(-90deg) translate(-100%);
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-CommandPalette {
  display: flex;
  flex-direction: column;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-CommandPalette-search {
  flex: 0 0 auto;
}

.lm-CommandPalette-content {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  min-height: 0;
  overflow: auto;
  list-style-type: none;
}

.lm-CommandPalette-header {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.lm-CommandPalette-item {
  display: flex;
  flex-direction: row;
}

.lm-CommandPalette-itemIcon {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemContent {
  flex: 1 1 auto;
  overflow: hidden;
}

.lm-CommandPalette-itemShortcut {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemLabel {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.lm-close-icon {
  border: 1px solid transparent;
  background-color: transparent;
  position: absolute;
  z-index: 1;
  right: 3%;
  top: 0;
  bottom: 0;
  margin: auto;
  padding: 7px 0;
  display: none;
  vertical-align: middle;
  outline: 0;
  cursor: pointer;
}
.lm-close-icon:after {
  content: 'X';
  display: block;
  width: 15px;
  height: 15px;
  text-align: center;
  color: #000;
  font-weight: normal;
  font-size: 12px;
  cursor: pointer;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-DockPanel {
  z-index: 0;
}

.lm-DockPanel-widget {
  z-index: 0;
}

.lm-DockPanel-tabBar {
  z-index: 1;
}

.lm-DockPanel-handle {
  z-index: 2;
}

.lm-DockPanel-handle.lm-mod-hidden {
  display: none !important;
}

.lm-DockPanel-handle:after {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  content: '';
}

.lm-DockPanel-handle[data-orientation='horizontal'] {
  cursor: ew-resize;
}

.lm-DockPanel-handle[data-orientation='vertical'] {
  cursor: ns-resize;
}

.lm-DockPanel-handle[data-orientation='horizontal']:after {
  left: 50%;
  min-width: 8px;
  transform: translateX(-50%);
}

.lm-DockPanel-handle[data-orientation='vertical']:after {
  top: 50%;
  min-height: 8px;
  transform: translateY(-50%);
}

.lm-DockPanel-overlay {
  z-index: 3;
  box-sizing: border-box;
  pointer-events: none;
}

.lm-DockPanel-overlay.lm-mod-hidden {
  display: none !important;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-Menu {
  z-index: 10000;
  position: absolute;
  white-space: nowrap;
  overflow-x: hidden;
  overflow-y: auto;
  outline: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-Menu-content {
  margin: 0;
  padding: 0;
  display: table;
  list-style-type: none;
}

.lm-Menu-item {
  display: table-row;
}

.lm-Menu-item.lm-mod-hidden,
.lm-Menu-item.lm-mod-collapsed {
  display: none !important;
}

.lm-Menu-itemIcon,
.lm-Menu-itemSubmenuIcon {
  display: table-cell;
  text-align: center;
}

.lm-Menu-itemLabel {
  display: table-cell;
  text-align: left;
}

.lm-Menu-itemShortcut {
  display: table-cell;
  text-align: right;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-MenuBar {
  outline: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-MenuBar-content {
  margin: 0;
  padding: 0;
  display: flex;
  flex-direction: row;
  list-style-type: none;
}

.lm-MenuBar-item {
  box-sizing: border-box;
}

.lm-MenuBar-itemIcon,
.lm-MenuBar-itemLabel {
  display: inline-block;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-ScrollBar {
  display: flex;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-ScrollBar[data-orientation='horizontal'] {
  flex-direction: row;
}

.lm-ScrollBar[data-orientation='vertical'] {
  flex-direction: column;
}

.lm-ScrollBar-button {
  box-sizing: border-box;
  flex: 0 0 auto;
}

.lm-ScrollBar-track {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
  flex: 1 1 auto;
}

.lm-ScrollBar-thumb {
  box-sizing: border-box;
  position: absolute;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-SplitPanel-child {
  z-index: 0;
}

.lm-SplitPanel-handle {
  z-index: 1;
}

.lm-SplitPanel-handle.lm-mod-hidden {
  display: none !important;
}

.lm-SplitPanel-handle:after {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  content: '';
}

.lm-SplitPanel[data-orientation='horizontal'] > .lm-SplitPanel-handle {
  cursor: ew-resize;
}

.lm-SplitPanel[data-orientation='vertical'] > .lm-SplitPanel-handle {
  cursor: ns-resize;
}

.lm-SplitPanel[data-orientation='horizontal'] > .lm-SplitPanel-handle:after {
  left: 50%;
  min-width: 8px;
  transform: translateX(-50%);
}

.lm-SplitPanel[data-orientation='vertical'] > .lm-SplitPanel-handle:after {
  top: 50%;
  min-height: 8px;
  transform: translateY(-50%);
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-TabBar {
  display: flex;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-TabBar[data-orientation='horizontal'] {
  flex-direction: row;
  align-items: flex-end;
}

.lm-TabBar[data-orientation='vertical'] {
  flex-direction: column;
  align-items: flex-end;
}

.lm-TabBar-content {
  margin: 0;
  padding: 0;
  display: flex;
  flex: 1 1 auto;
  list-style-type: none;
}

.lm-TabBar[data-orientation='horizontal'] > .lm-TabBar-content {
  flex-direction: row;
}

.lm-TabBar[data-orientation='vertical'] > .lm-TabBar-content {
  flex-direction: column;
}

.lm-TabBar-tab {
  display: flex;
  flex-direction: row;
  box-sizing: border-box;
  overflow: hidden;
  touch-action: none; /* Disable native Drag/Drop */
}

.lm-TabBar-tabIcon,
.lm-TabBar-tabCloseIcon {
  flex: 0 0 auto;
}

.lm-TabBar-tabLabel {
  flex: 1 1 auto;
  overflow: hidden;
  white-space: nowrap;
}

.lm-TabBar-tabInput {
  user-select: all;
  width: 100%;
  box-sizing: border-box;
}

.lm-TabBar-tab.lm-mod-hidden {
  display: none !important;
}

.lm-TabBar-addButton.lm-mod-hidden {
  display: none !important;
}

.lm-TabBar.lm-mod-dragging .lm-TabBar-tab {
  position: relative;
}

.lm-TabBar.lm-mod-dragging[data-orientation='horizontal'] .lm-TabBar-tab {
  left: 0;
  transition: left 150ms ease;
}

.lm-TabBar.lm-mod-dragging[data-orientation='vertical'] .lm-TabBar-tab {
  top: 0;
  transition: top 150ms ease;
}

.lm-TabBar.lm-mod-dragging .lm-TabBar-tab.lm-mod-dragging {
  transition: none;
}

.lm-TabBar-tabLabel .lm-TabBar-tabInput {
  user-select: all;
  width: 100%;
  box-sizing: border-box;
  background: inherit;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-TabPanel-tabBar {
  z-index: 1;
}

.lm-TabPanel-stackedPanel {
  z-index: 0;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Collapse {
  display: flex;
  flex-direction: column;
  align-items: stretch;
}

.jp-Collapse-header {
  padding: 1px 12px;
  background-color: var(--jp-layout-color1);
  border-bottom: solid var(--jp-border-width) var(--jp-border-color2);
  color: var(--jp-ui-font-color1);
  cursor: pointer;
  display: flex;
  align-items: center;
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  text-transform: uppercase;
  user-select: none;
}

.jp-Collapser-icon {
  height: 16px;
}

.jp-Collapse-header-collapsed .jp-Collapser-icon {
  transform: rotate(-90deg);
  margin: auto 0;
}

.jp-Collapser-title {
  line-height: 25px;
}

.jp-Collapse-contents {
  padding: 0 12px;
  background-color: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  overflow: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/* This file was auto-generated by ensureUiComponents() in @jupyterlab/buildutils */

/**
 * (DEPRECATED) Support for consuming icons as CSS background images
 */

/* Icons urls */

:root {
  --jp-icon-add-above: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGcgY2xpcC1wYXRoPSJ1cmwoI2NsaXAwXzEzN18xOTQ5MikiPgo8cGF0aCBjbGFzcz0ianAtaWNvbjMiIGQ9Ik00Ljc1IDQuOTMwNjZINi42MjVWNi44MDU2NkM2LjYyNSA3LjAxMTkxIDYuNzkzNzUgNy4xODA2NiA3IDcuMTgwNjZDNy4yMDYyNSA3LjE4MDY2IDcuMzc1IDcuMDExOTEgNy4zNzUgNi44MDU2NlY0LjkzMDY2SDkuMjVDOS40NTYyNSA0LjkzMDY2IDkuNjI1IDQuNzYxOTEgOS42MjUgNC41NTU2NkM5LjYyNSA0LjM0OTQxIDkuNDU2MjUgNC4xODA2NiA5LjI1IDQuMTgwNjZINy4zNzVWMi4zMDU2NkM3LjM3NSAyLjA5OTQxIDcuMjA2MjUgMS45MzA2NiA3IDEuOTMwNjZDNi43OTM3NSAxLjkzMDY2IDYuNjI1IDIuMDk5NDEgNi42MjUgMi4zMDU2NlY0LjE4MDY2SDQuNzVDNC41NDM3NSA0LjE4MDY2IDQuMzc1IDQuMzQ5NDEgNC4zNzUgNC41NTU2NkM0LjM3NSA0Ljc2MTkxIDQuNTQzNzUgNC45MzA2NiA0Ljc1IDQuOTMwNjZaIiBmaWxsPSIjNjE2MTYxIiBzdHJva2U9IiM2MTYxNjEiIHN0cm9rZS13aWR0aD0iMC43Ii8+CjwvZz4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTExLjUgOS41VjExLjVMMi41IDExLjVWOS41TDExLjUgOS41Wk0xMiA4QzEyLjU1MjMgOCAxMyA4LjQ0NzcyIDEzIDlWMTJDMTMgMTIuNTUyMyAxMi41NTIzIDEzIDEyIDEzTDIgMTNDMS40NDc3MiAxMyAxIDEyLjU1MjMgMSAxMlY5QzEgOC40NDc3MiAxLjQ0NzcxIDggMiA4TDEyIDhaIiBmaWxsPSIjNjE2MTYxIi8+CjxkZWZzPgo8Y2xpcFBhdGggaWQ9ImNsaXAwXzEzN18xOTQ5MiI+CjxyZWN0IGNsYXNzPSJqcC1pY29uMyIgd2lkdGg9IjYiIGhlaWdodD0iNiIgZmlsbD0id2hpdGUiIHRyYW5zZm9ybT0ibWF0cml4KC0xIDAgMCAxIDEwIDEuNTU1NjYpIi8+CjwvY2xpcFBhdGg+CjwvZGVmcz4KPC9zdmc+Cg==);
  --jp-icon-add-below: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGcgY2xpcC1wYXRoPSJ1cmwoI2NsaXAwXzEzN18xOTQ5OCkiPgo8cGF0aCBjbGFzcz0ianAtaWNvbjMiIGQ9Ik05LjI1IDEwLjA2OTNMNy4zNzUgMTAuMDY5M0w3LjM3NSA4LjE5NDM0QzcuMzc1IDcuOTg4MDkgNy4yMDYyNSA3LjgxOTM0IDcgNy44MTkzNEM2Ljc5Mzc1IDcuODE5MzQgNi42MjUgNy45ODgwOSA2LjYyNSA4LjE5NDM0TDYuNjI1IDEwLjA2OTNMNC43NSAxMC4wNjkzQzQuNTQzNzUgMTAuMDY5MyA0LjM3NSAxMC4yMzgxIDQuMzc1IDEwLjQ0NDNDNC4zNzUgMTAuNjUwNiA0LjU0Mzc1IDEwLjgxOTMgNC43NSAxMC44MTkzTDYuNjI1IDEwLjgxOTNMNi42MjUgMTIuNjk0M0M2LjYyNSAxMi45MDA2IDYuNzkzNzUgMTMuMDY5MyA3IDEzLjA2OTNDNy4yMDYyNSAxMy4wNjkzIDcuMzc1IDEyLjkwMDYgNy4zNzUgMTIuNjk0M0w3LjM3NSAxMC44MTkzTDkuMjUgMTAuODE5M0M5LjQ1NjI1IDEwLjgxOTMgOS42MjUgMTAuNjUwNiA5LjYyNSAxMC40NDQzQzkuNjI1IDEwLjIzODEgOS40NTYyNSAxMC4wNjkzIDkuMjUgMTAuMDY5M1oiIGZpbGw9IiM2MTYxNjEiIHN0cm9rZT0iIzYxNjE2MSIgc3Ryb2tlLXdpZHRoPSIwLjciLz4KPC9nPgo8cGF0aCBjbGFzcz0ianAtaWNvbjMiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMi41IDUuNUwyLjUgMy41TDExLjUgMy41TDExLjUgNS41TDIuNSA1LjVaTTIgN0MxLjQ0NzcyIDcgMSA2LjU1MjI4IDEgNkwxIDNDMSAyLjQ0NzcyIDEuNDQ3NzIgMiAyIDJMMTIgMkMxMi41NTIzIDIgMTMgMi40NDc3MiAxMyAzTDEzIDZDMTMgNi41NTIyOSAxMi41NTIzIDcgMTIgN0wyIDdaIiBmaWxsPSIjNjE2MTYxIi8+CjxkZWZzPgo8Y2xpcFBhdGggaWQ9ImNsaXAwXzEzN18xOTQ5OCI+CjxyZWN0IGNsYXNzPSJqcC1pY29uMyIgd2lkdGg9IjYiIGhlaWdodD0iNiIgZmlsbD0id2hpdGUiIHRyYW5zZm9ybT0ibWF0cml4KDEgMS43NDg0NmUtMDcgMS43NDg0NmUtMDcgLTEgNCAxMy40NDQzKSIvPgo8L2NsaXBQYXRoPgo8L2RlZnM+Cjwvc3ZnPgo=);
  --jp-icon-add: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE5IDEzaC02djZoLTJ2LTZINXYtMmg2VjVoMnY2aDZ2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-bell: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE2IDE2IiB2ZXJzaW9uPSIxLjEiPgogICA8cGF0aCBjbGFzcz0ianAtaWNvbjIganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMzMzMzMzIgogICAgICBkPSJtOCAwLjI5Yy0xLjQgMC0yLjcgMC43My0zLjYgMS44LTEuMiAxLjUtMS40IDMuNC0xLjUgNS4yLTAuMTggMi4yLTAuNDQgNC0yLjMgNS4zbDAuMjggMS4zaDVjMC4wMjYgMC42NiAwLjMyIDEuMSAwLjcxIDEuNSAwLjg0IDAuNjEgMiAwLjYxIDIuOCAwIDAuNTItMC40IDAuNi0xIDAuNzEtMS41aDVsMC4yOC0xLjNjLTEuOS0wLjk3LTIuMi0zLjMtMi4zLTUuMy0wLjEzLTEuOC0wLjI2LTMuNy0xLjUtNS4yLTAuODUtMS0yLjItMS44LTMuNi0xLjh6bTAgMS40YzAuODggMCAxLjkgMC41NSAyLjUgMS4zIDAuODggMS4xIDEuMSAyLjcgMS4yIDQuNCAwLjEzIDEuNyAwLjIzIDMuNiAxLjMgNS4yaC0xMGMxLjEtMS42IDEuMi0zLjQgMS4zLTUuMiAwLjEzLTEuNyAwLjMtMy4zIDEuMi00LjQgMC41OS0wLjcyIDEuNi0xLjMgMi41LTEuM3ptLTAuNzQgMTJoMS41Yy0wLjAwMTUgMC4yOCAwLjAxNSAwLjc5LTAuNzQgMC43OS0wLjczIDAuMDAxNi0wLjcyLTAuNTMtMC43NC0wLjc5eiIgLz4KPC9zdmc+Cg==);
  --jp-icon-bug-dot: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiPgogICAgICAgIDxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMTcuMTkgOEgyMFYxMEgxNy45MUMxNy45NiAxMC4zMyAxOCAxMC42NiAxOCAxMVYxMkgyMFYxNEgxOC41SDE4VjE0LjAyNzVDMTUuNzUgMTQuMjc2MiAxNCAxNi4xODM3IDE0IDE4LjVDMTQgMTkuMjA4IDE0LjE2MzUgMTkuODc3OSAxNC40NTQ5IDIwLjQ3MzlDMTMuNzA2MyAyMC44MTE3IDEyLjg3NTcgMjEgMTIgMjFDOS43OCAyMSA3Ljg1IDE5Ljc5IDYuODEgMThINFYxNkg2LjA5QzYuMDQgMTUuNjcgNiAxNS4zNCA2IDE1VjE0SDRWMTJINlYxMUM2IDEwLjY2IDYuMDQgMTAuMzMgNi4wOSAxMEg0VjhINi44MUM3LjI2IDcuMjIgNy44OCA2LjU1IDguNjIgNi4wNEw3IDQuNDFMOC40MSAzTDEwLjU5IDUuMTdDMTEuMDQgNS4wNiAxMS41MSA1IDEyIDVDMTIuNDkgNSAxMi45NiA1LjA2IDEzLjQyIDUuMTdMMTUuNTkgM0wxNyA0LjQxTDE1LjM3IDYuMDRDMTYuMTIgNi41NSAxNi43NCA3LjIyIDE3LjE5IDhaTTEwIDE2SDE0VjE0SDEwVjE2Wk0xMCAxMkgxNFYxMEgxMFYxMloiIGZpbGw9IiM2MTYxNjEiLz4KICAgICAgICA8cGF0aCBkPSJNMjIgMTguNUMyMiAyMC40MzMgMjAuNDMzIDIyIDE4LjUgMjJDMTYuNTY3IDIyIDE1IDIwLjQzMyAxNSAxOC41QzE1IDE2LjU2NyAxNi41NjcgMTUgMTguNSAxNUMyMC40MzMgMTUgMjIgMTYuNTY3IDIyIDE4LjVaIiBmaWxsPSIjNjE2MTYxIi8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-bug: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik0yMCA4aC0yLjgxYy0uNDUtLjc4LTEuMDctMS40NS0xLjgyLTEuOTZMMTcgNC40MSAxNS41OSAzbC0yLjE3IDIuMTdDMTIuOTYgNS4wNiAxMi40OSA1IDEyIDVjLS40OSAwLS45Ni4wNi0xLjQxLjE3TDguNDEgMyA3IDQuNDFsMS42MiAxLjYzQzcuODggNi41NSA3LjI2IDcuMjIgNi44MSA4SDR2MmgyLjA5Yy0uMDUuMzMtLjA5LjY2LS4wOSAxdjFINHYyaDJ2MWMwIC4zNC4wNC42Ny4wOSAxSDR2MmgyLjgxYzEuMDQgMS43OSAyLjk3IDMgNS4xOSAzczQuMTUtMS4yMSA1LjE5LTNIMjB2LTJoLTIuMDljLjA1LS4zMy4wOS0uNjYuMDktMXYtMWgydi0yaC0ydi0xYzAtLjM0LS4wNC0uNjctLjA5LTFIMjBWOHptLTYgOGgtNHYtMmg0djJ6bTAtNGgtNHYtMmg0djJ6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-build: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE0LjkgMTcuNDVDMTYuMjUgMTcuNDUgMTcuMzUgMTYuMzUgMTcuMzUgMTVDMTcuMzUgMTMuNjUgMTYuMjUgMTIuNTUgMTQuOSAxMi41NUMxMy41NCAxMi41NSAxMi40NSAxMy42NSAxMi40NSAxNUMxMi40NSAxNi4zNSAxMy41NCAxNy40NSAxNC45IDE3LjQ1Wk0yMC4xIDE1LjY4TDIxLjU4IDE2Ljg0QzIxLjcxIDE2Ljk1IDIxLjc1IDE3LjEzIDIxLjY2IDE3LjI5TDIwLjI2IDE5LjcxQzIwLjE3IDE5Ljg2IDIwIDE5LjkyIDE5LjgzIDE5Ljg2TDE4LjA5IDE5LjE2QzE3LjczIDE5LjQ0IDE3LjMzIDE5LjY3IDE2LjkxIDE5Ljg1TDE2LjY0IDIxLjdDMTYuNjIgMjEuODcgMTYuNDcgMjIgMTYuMyAyMkgxMy41QzEzLjMyIDIyIDEzLjE4IDIxLjg3IDEzLjE1IDIxLjdMMTIuODkgMTkuODVDMTIuNDYgMTkuNjcgMTIuMDcgMTkuNDQgMTEuNzEgMTkuMTZMOS45NjAwMiAxOS44NkM5LjgxMDAyIDE5LjkyIDkuNjIwMDIgMTkuODYgOS41NDAwMiAxOS43MUw4LjE0MDAyIDE3LjI5QzguMDUwMDIgMTcuMTMgOC4wOTAwMiAxNi45NSA4LjIyMDAyIDE2Ljg0TDkuNzAwMDIgMTUuNjhMOS42NTAwMSAxNUw5LjcwMDAyIDE0LjMxTDguMjIwMDIgMTMuMTZDOC4wOTAwMiAxMy4wNSA4LjA1MDAyIDEyLjg2IDguMTQwMDIgMTIuNzFMOS41NDAwMiAxMC4yOUM5LjYyMDAyIDEwLjEzIDkuODEwMDIgMTAuMDcgOS45NjAwMiAxMC4xM0wxMS43MSAxMC44NEMxMi4wNyAxMC41NiAxMi40NiAxMC4zMiAxMi44OSAxMC4xNUwxMy4xNSA4LjI4OTk4QzEzLjE4IDguMTI5OTggMTMuMzIgNy45OTk5OCAxMy41IDcuOTk5OThIMTYuM0MxNi40NyA3Ljk5OTk4IDE2LjYyIDguMTI5OTggMTYuNjQgOC4yODk5OEwxNi45MSAxMC4xNUMxNy4zMyAxMC4zMiAxNy43MyAxMC41NiAxOC4wOSAxMC44NEwxOS44MyAxMC4xM0MyMCAxMC4wNyAyMC4xNyAxMC4xMyAyMC4yNiAxMC4yOUwyMS42NiAxMi43MUMyMS43NSAxMi44NiAyMS43MSAxMy4wNSAyMS41OCAxMy4xNkwyMC4xIDE0LjMxTDIwLjE1IDE1TDIwLjEgMTUuNjhaIi8+CiAgICA8cGF0aCBkPSJNNy4zMjk2NiA3LjQ0NDU0QzguMDgzMSA3LjAwOTU0IDguMzM5MzIgNi4wNTMzMiA3LjkwNDMyIDUuMjk5ODhDNy40NjkzMiA0LjU0NjQzIDYuNTA4MSA0LjI4MTU2IDUuNzU0NjYgNC43MTY1NkM1LjM5MTc2IDQuOTI2MDggNS4xMjY5NSA1LjI3MTE4IDUuMDE4NDkgNS42NzU5NEM0LjkxMDA0IDYuMDgwNzEgNC45NjY4MiA2LjUxMTk4IDUuMTc2MzQgNi44NzQ4OEM1LjYxMTM0IDcuNjI4MzIgNi41NzYyMiA3Ljg3OTU0IDcuMzI5NjYgNy40NDQ1NFpNOS42NTcxOCA0Ljc5NTkzTDEwLjg2NzIgNC45NTE3OUMxMC45NjI4IDQuOTc3NDEgMTEuMDQwMiA1LjA3MTMzIDExLjAzODIgNS4xODc5M0wxMS4wMzg4IDYuOTg4OTNDMTEuMDQ1NSA3LjEwMDU0IDEwLjk2MTYgNy4xOTUxOCAxMC44NTUgNy4yMTA1NEw5LjY2MDAxIDcuMzgwODNMOS4yMzkxNSA4LjEzMTg4TDkuNjY5NjEgOS4yNTc0NUM5LjcwNzI5IDkuMzYyNzEgOS42NjkzNCA5LjQ3Njk5IDkuNTc0MDggOS41MzE5OUw4LjAxNTIzIDEwLjQzMkM3LjkxMTMxIDEwLjQ5MiA3Ljc5MzM3IDEwLjQ2NzcgNy43MjEwNSAxMC4zODI0TDYuOTg3NDggOS40MzE4OEw2LjEwOTMxIDkuNDMwODNMNS4zNDcwNCAxMC4zOTA1QzUuMjg5MDkgMTAuNDcwMiA1LjE3MzgzIDEwLjQ5MDUgNS4wNzE4NyAxMC40MzM5TDMuNTEyNDUgOS41MzI5M0MzLjQxMDQ5IDkuNDc2MzMgMy4zNzY0NyA5LjM1NzQxIDMuNDEwNzUgOS4yNTY3OUwzLjg2MzQ3IDguMTQwOTNMMy42MTc0OSA3Ljc3NDg4TDMuNDIzNDcgNy4zNzg4M0wyLjIzMDc1IDcuMjEyOTdDMi4xMjY0NyA3LjE5MjM1IDIuMDQwNDkgNy4xMDM0MiAyLjA0MjQ1IDYuOTg2ODJMMi4wNDE4NyA1LjE4NTgyQzIuMDQzODMgNS4wNjkyMiAyLjExOTA5IDQuOTc5NTggMi4yMTcwNCA0Ljk2OTIyTDMuNDIwNjUgNC43OTM5M0wzLjg2NzQ5IDQuMDI3ODhMMy40MTEwNSAyLjkxNzMxQzMuMzczMzcgMi44MTIwNCAzLjQxMTMxIDIuNjk3NzYgMy41MTUyMyAyLjYzNzc2TDUuMDc0MDggMS43Mzc3NkM1LjE2OTM0IDEuNjgyNzYgNS4yODcyOSAxLjcwNzA0IDUuMzU5NjEgMS43OTIzMUw2LjExOTE1IDIuNzI3ODhMNi45ODAwMSAyLjczODkzTDcuNzI0OTYgMS43ODkyMkM3Ljc5MTU2IDEuNzA0NTggNy45MTU0OCAxLjY3OTIyIDguMDA4NzkgMS43NDA4Mkw5LjU2ODIxIDIuNjQxODJDOS42NzAxNyAyLjY5ODQyIDkuNzEyODUgMi44MTIzNCA5LjY4NzIzIDIuOTA3OTdMOS4yMTcxOCA0LjAzMzgzTDkuNDYzMTYgNC4zOTk4OEw5LjY1NzE4IDQuNzk1OTNaIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down-empty-thin: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iOS45LDEzLjYgMy42LDcuNCA0LjQsNi42IDkuOSwxMi4yIDE1LjQsNi43IDE2LjEsNy40ICIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down-empty: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik01LjIsNS45TDksOS43bDMuOC0zLjhsMS4yLDEuMmwtNC45LDVsLTQuOS01TDUuMiw1Ljl6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik01LjIsNy41TDksMTEuMmwzLjgtMy44SDUuMnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-caret-left: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwYXRoIGQ9Ik0xMC44LDEyLjhMNy4xLDlsMy44LTMuOGwwLDcuNkgxMC44eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-caret-right: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik03LjIsNS4yTDEwLjksOWwtMy44LDMuOFY1LjJINy4yeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-caret-up-empty-thin: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iMTUuNCwxMy4zIDkuOSw3LjcgNC40LDEzLjIgMy42LDEyLjUgOS45LDYuMyAxNi4xLDEyLjYgIi8+Cgk8L2c+Cjwvc3ZnPgo=);
  --jp-icon-caret-up: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwYXRoIGQ9Ik01LjIsMTAuNUw5LDYuOGwzLjgsMy44SDUuMnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-case-sensitive: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0MTQxNDEiPgogICAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiAgPC9nPgogIDxnIGNsYXNzPSJqcC1pY29uLWFjY2VudDIiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTcuNiw4aDAuOWwzLjUsOGgtMS4xTDEwLDE0SDZsLTAuOSwySDRMNy42LDh6IE04LDkuMUw2LjQsMTNoMy4yTDgsOS4xeiIvPgogICAgPHBhdGggZD0iTTE2LjYsOS44Yy0wLjIsMC4xLTAuNCwwLjEtMC43LDAuMWMtMC4yLDAtMC40LTAuMS0wLjYtMC4yYy0wLjEtMC4xLTAuMi0wLjQtMC4yLTAuNyBjLTAuMywwLjMtMC42LDAuNS0wLjksMC43Yy0wLjMsMC4xLTAuNywwLjItMS4xLDAuMmMtMC4zLDAtMC41LDAtMC43LTAuMWMtMC4yLTAuMS0wLjQtMC4yLTAuNi0wLjNjLTAuMi0wLjEtMC4zLTAuMy0wLjQtMC41IGMtMC4xLTAuMi0wLjEtMC40LTAuMS0wLjdjMC0wLjMsMC4xLTAuNiwwLjItMC44YzAuMS0wLjIsMC4zLTAuNCwwLjQtMC41QzEyLDcsMTIuMiw2LjksMTIuNSw2LjhjMC4yLTAuMSwwLjUtMC4xLDAuNy0wLjIgYzAuMy0wLjEsMC41LTAuMSwwLjctMC4xYzAuMiwwLDAuNC0wLjEsMC42LTAuMWMwLjIsMCwwLjMtMC4xLDAuNC0wLjJjMC4xLTAuMSwwLjItMC4yLDAuMi0wLjRjMC0xLTEuMS0xLTEuMy0xIGMtMC40LDAtMS40LDAtMS40LDEuMmgtMC45YzAtMC40LDAuMS0wLjcsMC4yLTFjMC4xLTAuMiwwLjMtMC40LDAuNS0wLjZjMC4yLTAuMiwwLjUtMC4zLDAuOC0wLjNDMTMuMyw0LDEzLjYsNCwxMy45LDQgYzAuMywwLDAuNSwwLDAuOCwwLjFjMC4zLDAsMC41LDAuMSwwLjcsMC4yYzAuMiwwLjEsMC40LDAuMywwLjUsMC41QzE2LDUsMTYsNS4yLDE2LDUuNnYyLjljMCwwLjIsMCwwLjQsMCwwLjUgYzAsMC4xLDAuMSwwLjIsMC4zLDAuMmMwLjEsMCwwLjIsMCwwLjMsMFY5Ljh6IE0xNS4yLDYuOWMtMS4yLDAuNi0zLjEsMC4yLTMuMSwxLjRjMCwxLjQsMy4xLDEsMy4xLTAuNVY2Ljl6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-check: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik05IDE2LjE3TDQuODMgMTJsLTEuNDIgMS40MUw5IDE5IDIxIDdsLTEuNDEtMS40MXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-circle-empty: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyIDJDNi40NyAyIDIgNi40NyAyIDEyczQuNDcgMTAgMTAgMTAgMTAtNC40NyAxMC0xMFMxNy41MyAyIDEyIDJ6bTAgMThjLTQuNDEgMC04LTMuNTktOC04czMuNTktOCA4LTggOCAzLjU5IDggOC0zLjU5IDgtOCA4eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-circle: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPGNpcmNsZSBjeD0iOSIgY3k9IjkiIHI9IjgiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-clear: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8bWFzayBpZD0iZG9udXRIb2xlIj4KICAgIDxyZWN0IHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgZmlsbD0id2hpdGUiIC8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSI4IiBmaWxsPSJibGFjayIvPgogIDwvbWFzaz4KCiAgPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxyZWN0IGhlaWdodD0iMTgiIHdpZHRoPSIyIiB4PSIxMSIgeT0iMyIgdHJhbnNmb3JtPSJyb3RhdGUoMzE1LCAxMiwgMTIpIi8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIxMCIgbWFzaz0idXJsKCNkb251dEhvbGUpIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-close: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1ub25lIGpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIGpwLWljb24zLWhvdmVyIiBmaWxsPSJub25lIj4KICAgIDxjaXJjbGUgY3g9IjEyIiBjeT0iMTIiIHI9IjExIi8+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIGpwLWljb24tYWNjZW50Mi1ob3ZlciIgZmlsbD0iIzYxNjE2MSI+CiAgICA8cGF0aCBkPSJNMTkgNi40MUwxNy41OSA1IDEyIDEwLjU5IDYuNDEgNSA1IDYuNDEgMTAuNTkgMTIgNSAxNy41OSA2LjQxIDE5IDEyIDEzLjQxIDE3LjU5IDE5IDE5IDE3LjU5IDEzLjQxIDEyeiIvPgogIDwvZz4KCiAgPGcgY2xhc3M9ImpwLWljb24tbm9uZSBqcC1pY29uLWJ1c3kiIGZpbGw9Im5vbmUiPgogICAgPGNpcmNsZSBjeD0iMTIiIGN5PSIxMiIgcj0iNyIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-code-check: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBzaGFwZS1yZW5kZXJpbmc9Imdlb21ldHJpY1ByZWNpc2lvbiI+CiAgICA8cGF0aCBkPSJNNi41OSwzLjQxTDIsOEw2LjU5LDEyLjZMOCwxMS4xOEw0LjgyLDhMOCw0LjgyTDYuNTksMy40MU0xMi40MSwzLjQxTDExLDQuODJMMTQuMTgsOEwxMSwxMS4xOEwxMi40MSwxMi42TDE3LDhMMTIuNDEsMy40MU0yMS41OSwxMS41OUwxMy41LDE5LjY4TDkuODMsMTZMOC40MiwxNy40MUwxMy41LDIyLjVMMjMsMTNMMjEuNTksMTEuNTlaIiAvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-code: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyOCAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTExLjQgMTguNkw2LjggMTRMMTEuNCA5LjRMMTAgOEw0IDE0TDEwIDIwTDExLjQgMTguNlpNMTYuNiAxOC42TDIxLjIgMTRMMTYuNiA5LjRMMTggOEwyNCAxNEwxOCAyMEwxNi42IDE4LjZWMTguNloiLz4KCTwvZz4KPC9zdmc+Cg==);
  --jp-icon-collapse-all: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTggMmMxIDAgMTEgMCAxMiAwczIgMSAyIDJjMCAxIDAgMTEgMCAxMnMwIDItMiAyQzIwIDE0IDIwIDQgMjAgNFMxMCA0IDYgNGMwLTIgMS0yIDItMnoiIC8+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTE4IDhjMC0xLTEtMi0yLTJTNSA2IDQgNnMtMiAxLTIgMmMwIDEgMCAxMSAwIDEyczEgMiAyIDJjMSAwIDExIDAgMTIgMHMyLTEgMi0yYzAtMSAwLTExIDAtMTJ6bS0yIDB2MTJINFY4eiIgLz4KICAgICAgICA8cGF0aCBkPSJNNiAxM3YyaDh2LTJ6IiAvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-console: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwMCAyMDAiPgogIDxnIGNsYXNzPSJqcC1jb25zb2xlLWljb24tYmFja2dyb3VuZC1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiMwMjg4RDEiPgogICAgPHBhdGggZD0iTTIwIDE5LjhoMTYwdjE1OS45SDIweiIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtY29uc29sZS1pY29uLWNvbG9yIGpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIiBmaWxsPSIjZmZmIj4KICAgIDxwYXRoIGQ9Ik0xMDUgMTI3LjNoNDB2MTIuOGgtNDB6TTUxLjEgNzdMNzQgOTkuOWwtMjMuMyAyMy4zIDEwLjUgMTAuNSAyMy4zLTIzLjNMOTUgOTkuOSA4NC41IDg5LjQgNjEuNiA2Ni41eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-copy: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTExLjksMUgzLjJDMi40LDEsMS43LDEuNywxLjcsMi41djEwLjJoMS41VjIuNWg4LjdWMXogTTE0LjEsMy45aC04Yy0wLjgsMC0xLjUsMC43LTEuNSwxLjV2MTAuMmMwLDAuOCwwLjcsMS41LDEuNSwxLjVoOCBjMC44LDAsMS41LTAuNywxLjUtMS41VjUuNEMxNS41LDQuNiwxNC45LDMuOSwxNC4xLDMuOXogTTE0LjEsMTUuNWgtOFY1LjRoOFYxNS41eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-copyright: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDI0IDI0IiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCI+CiAgPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik0xMS44OCw5LjE0YzEuMjgsMC4wNiwxLjYxLDEuMTUsMS42MywxLjY2aDEuNzljLTAuMDgtMS45OC0xLjQ5LTMuMTktMy40NS0zLjE5QzkuNjQsNy42MSw4LDksOCwxMi4xNCBjMCwxLjk0LDAuOTMsNC4yNCwzLjg0LDQuMjRjMi4yMiwwLDMuNDEtMS42NSwzLjQ0LTIuOTVoLTEuNzljLTAuMDMsMC41OS0wLjQ1LDEuMzgtMS42MywxLjQ0QzEwLjU1LDE0LjgzLDEwLDEzLjgxLDEwLDEyLjE0IEMxMCw5LjI1LDExLjI4LDkuMTYsMTEuODgsOS4xNHogTTEyLDJDNi40OCwyLDIsNi40OCwyLDEyczQuNDgsMTAsMTAsMTBzMTAtNC40OCwxMC0xMFMxNy41MiwyLDEyLDJ6IE0xMiwyMGMtNC40MSwwLTgtMy41OS04LTggczMuNTktOCw4LThzOCwzLjU5LDgsOFMxNi40MSwyMCwxMiwyMHoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-cut: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkuNjQgNy42NGMuMjMtLjUuMzYtMS4wNS4zNi0xLjY0IDAtMi4yMS0xLjc5LTQtNC00UzIgMy43OSAyIDZzMS43OSA0IDQgNGMuNTkgMCAxLjE0LS4xMyAxLjY0LS4zNkwxMCAxMmwtMi4zNiAyLjM2QzcuMTQgMTQuMTMgNi41OSAxNCA2IDE0Yy0yLjIxIDAtNCAxLjc5LTQgNHMxLjc5IDQgNCA0IDQtMS43OSA0LTRjMC0uNTktLjEzLTEuMTQtLjM2LTEuNjRMMTIgMTRsNyA3aDN2LTFMOS42NCA3LjY0ek02IDhjLTEuMSAwLTItLjg5LTItMnMuOS0yIDItMiAyIC44OSAyIDItLjkgMi0yIDJ6bTAgMTJjLTEuMSAwLTItLjg5LTItMnMuOS0yIDItMiAyIC44OSAyIDItLjkgMi0yIDJ6bTYtNy41Yy0uMjggMC0uNS0uMjItLjUtLjVzLjIyLS41LjUtLjUuNS4yMi41LjUtLjIyLjUtLjUuNXpNMTkgM2wtNiA2IDIgMiA3LTdWM3oiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-delete: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2cHgiIGhlaWdodD0iMTZweCI+CiAgICA8cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIiAvPgogICAgPHBhdGggY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjI2MjYyIiBkPSJNNiAxOWMwIDEuMS45IDIgMiAyaDhjMS4xIDAgMi0uOSAyLTJWN0g2djEyek0xOSA0aC0zLjVsLTEtMWgtNWwtMSAxSDV2MmgxNFY0eiIgLz4KPC9zdmc+Cg==);
  --jp-icon-download: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE5IDloLTRWM0g5djZINWw3IDcgNy03ek01IDE4djJoMTR2LTJINXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-duplicate: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTIuNzk5OTggMC44NzVIOC44OTU4MkM5LjIwMDYxIDAuODc1IDkuNDQ5OTggMS4xMzkxNCA5LjQ0OTk4IDEuNDYxOThDOS40NDk5OCAxLjc4NDgyIDkuMjAwNjEgMi4wNDg5NiA4Ljg5NTgyIDIuMDQ4OTZIMy4zNTQxNUMzLjA0OTM2IDIuMDQ4OTYgMi43OTk5OCAyLjMxMzEgMi43OTk5OCAyLjYzNTk0VjkuNjc5NjlDMi43OTk5OCAxMC4wMDI1IDIuNTUwNjEgMTAuMjY2NyAyLjI0NTgyIDEwLjI2NjdDMS45NDEwMyAxMC4yNjY3IDEuNjkxNjUgMTAuMDAyNSAxLjY5MTY1IDkuNjc5NjlWMi4wNDg5NkMxLjY5MTY1IDEuNDAzMjggMi4xOTA0IDAuODc1IDIuNzk5OTggMC44NzVaTTUuMzY2NjUgMTEuOVY0LjU1SDExLjA4MzNWMTEuOUg1LjM2NjY1Wk00LjE0MTY1IDQuMTQxNjdDNC4xNDE2NSAzLjY5MDYzIDQuNTA3MjggMy4zMjUgNC45NTgzMiAzLjMyNUgxMS40OTE3QzExLjk0MjcgMy4zMjUgMTIuMzA4MyAzLjY5MDYzIDEyLjMwODMgNC4xNDE2N1YxMi4zMDgzQzEyLjMwODMgMTIuNzU5NCAxMS45NDI3IDEzLjEyNSAxMS40OTE3IDEzLjEyNUg0Ljk1ODMyQzQuNTA3MjggMTMuMTI1IDQuMTQxNjUgMTIuNzU5NCA0LjE0MTY1IDEyLjMwODNWNC4xNDE2N1oiIGZpbGw9IiM2MTYxNjEiLz4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBkPSJNOS40MzU3NCA4LjI2NTA3SDguMzY0MzFWOS4zMzY1QzguMzY0MzEgOS40NTQzNSA4LjI2Nzg4IDkuNTUwNzggOC4xNTAwMiA5LjU1MDc4QzguMDMyMTcgOS41NTA3OCA3LjkzNTc0IDkuNDU0MzUgNy45MzU3NCA5LjMzNjVWOC4yNjUwN0g2Ljg2NDMxQzYuNzQ2NDUgOC4yNjUwNyA2LjY1MDAyIDguMTY4NjQgNi42NTAwMiA4LjA1MDc4QzYuNjUwMDIgNy45MzI5MiA2Ljc0NjQ1IDcuODM2NSA2Ljg2NDMxIDcuODM2NUg3LjkzNTc0VjYuNzY1MDdDNy45MzU3NCA2LjY0NzIxIDguMDMyMTcgNi41NTA3OCA4LjE1MDAyIDYuNTUwNzhDOC4yNjc4OCA2LjU1MDc4IDguMzY0MzEgNi42NDcyMSA4LjM2NDMxIDYuNzY1MDdWNy44MzY1SDkuNDM1NzRDOS41NTM2IDcuODM2NSA5LjY1MDAyIDcuOTMyOTIgOS42NTAwMiA4LjA1MDc4QzkuNjUwMDIgOC4xNjg2NCA5LjU1MzYgOC4yNjUwNyA5LjQzNTc0IDguMjY1MDdaIiBmaWxsPSIjNjE2MTYxIiBzdHJva2U9IiM2MTYxNjEiIHN0cm9rZS13aWR0aD0iMC41Ii8+Cjwvc3ZnPgo=);
  --jp-icon-edit: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTMgMTcuMjVWMjFoMy43NUwxNy44MSA5Ljk0bC0zLjc1LTMuNzVMMyAxNy4yNXpNMjAuNzEgNy4wNGMuMzktLjM5LjM5LTEuMDIgMC0xLjQxbC0yLjM0LTIuMzRjLS4zOS0uMzktMS4wMi0uMzktMS40MSAwbC0xLjgzIDEuODMgMy43NSAzLjc1IDEuODMtMS44M3oiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-ellipses: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPGNpcmNsZSBjeD0iNSIgY3k9IjEyIiByPSIyIi8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIyIi8+CiAgICA8Y2lyY2xlIGN4PSIxOSIgY3k9IjEyIiByPSIyIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-error: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj48Y2lyY2xlIGN4PSIxMiIgY3k9IjE5IiByPSIyIi8+PHBhdGggZD0iTTEwIDNoNHYxMmgtNHoiLz48L2c+CjxwYXRoIGZpbGw9Im5vbmUiIGQ9Ik0wIDBoMjR2MjRIMHoiLz4KPC9zdmc+Cg==);
  --jp-icon-expand-all: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTggMmMxIDAgMTEgMCAxMiAwczIgMSAyIDJjMCAxIDAgMTEgMCAxMnMwIDItMiAyQzIwIDE0IDIwIDQgMjAgNFMxMCA0IDYgNGMwLTIgMS0yIDItMnoiIC8+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTE4IDhjMC0xLTEtMi0yLTJTNSA2IDQgNnMtMiAxLTIgMmMwIDEgMCAxMSAwIDEyczEgMiAyIDJjMSAwIDExIDAgMTIgMHMyLTEgMi0yYzAtMSAwLTExIDAtMTJ6bS0yIDB2MTJINFY4eiIgLz4KICAgICAgICA8cGF0aCBkPSJNMTEgMTBIOXYzSDZ2MmgzdjNoMnYtM2gzdi0yaC0zeiIgLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-extension: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwLjUgMTFIMTlWN2MwLTEuMS0uOS0yLTItMmgtNFYzLjVDMTMgMi4xMiAxMS44OCAxIDEwLjUgMVM4IDIuMTIgOCAzLjVWNUg0Yy0xLjEgMC0xLjk5LjktMS45OSAydjMuOEgzLjVjMS40OSAwIDIuNyAxLjIxIDIuNyAyLjdzLTEuMjEgMi43LTIuNyAyLjdIMlYyMGMwIDEuMS45IDIgMiAyaDMuOHYtMS41YzAtMS40OSAxLjIxLTIuNyAyLjctMi43IDEuNDkgMCAyLjcgMS4yMSAyLjcgMi43VjIySDE3YzEuMSAwIDItLjkgMi0ydi00aDEuNWMxLjM4IDAgMi41LTEuMTIgMi41LTIuNVMyMS44OCAxMSAyMC41IDExeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-fast-forward: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTQgMThsOC41LTZMNCA2djEyem05LTEydjEybDguNS02TDEzIDZ6Ii8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-file-upload: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkgMTZoNnYtNmg0bC03LTctNyA3aDR6bS00IDJoMTR2Mkg1eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-file: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkuMyA4LjJsLTUuNS01LjVjLS4zLS4zLS43LS41LTEuMi0uNUgzLjljLS44LjEtMS42LjktMS42IDEuOHYxNC4xYzAgLjkuNyAxLjYgMS42IDEuNmgxNC4yYy45IDAgMS42LS43IDEuNi0xLjZWOS40Yy4xLS41LS4xLS45LS40LTEuMnptLTUuOC0zLjNsMy40IDMuNmgtMy40VjQuOXptMy45IDEyLjdINC43Yy0uMSAwLS4yIDAtLjItLjJWNC43YzAtLjIuMS0uMy4yLS4zaDcuMnY0LjRzMCAuOC4zIDEuMWMuMy4zIDEuMS4zIDEuMS4zaDQuM3Y3LjJzLS4xLjItLjIuMnoiLz4KPC9zdmc+Cg==);
  --jp-icon-filter-dot: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTE0LDEyVjE5Ljg4QzE0LjA0LDIwLjE4IDEzLjk0LDIwLjUgMTMuNzEsMjAuNzFDMTMuMzIsMjEuMSAxMi42OSwyMS4xIDEyLjMsMjAuNzFMMTAuMjksMTguN0MxMC4wNiwxOC40NyA5Ljk2LDE4LjE2IDEwLDE3Ljg3VjEySDkuOTdMNC4yMSw0LjYyQzMuODcsNC4xOSAzLjk1LDMuNTYgNC4zOCwzLjIyQzQuNTcsMy4wOCA0Ljc4LDMgNSwzVjNIMTlWM0MxOS4yMiwzIDE5LjQzLDMuMDggMTkuNjIsMy4yMkMyMC4wNSwzLjU2IDIwLjEzLDQuMTkgMTkuNzksNC42MkwxNC4wMywxMkgxNFoiIC8+CiAgPC9nPgogIDxnIGNsYXNzPSJqcC1pY29uLWRvdCIgZmlsbD0iI0ZGRiI+CiAgICA8Y2lyY2xlIGN4PSIxOCIgY3k9IjE3IiByPSIzIj48L2NpcmNsZT4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-filter-list: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEwIDE4aDR2LTJoLTR2MnpNMyA2djJoMThWNkgzem0zIDdoMTJ2LTJINnYyeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-filter: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTE0LDEyVjE5Ljg4QzE0LjA0LDIwLjE4IDEzLjk0LDIwLjUgMTMuNzEsMjAuNzFDMTMuMzIsMjEuMSAxMi42OSwyMS4xIDEyLjMsMjAuNzFMMTAuMjksMTguN0MxMC4wNiwxOC40NyA5Ljk2LDE4LjE2IDEwLDE3Ljg3VjEySDkuOTdMNC4yMSw0LjYyQzMuODcsNC4xOSAzLjk1LDMuNTYgNC4zOCwzLjIyQzQuNTcsMy4wOCA0Ljc4LDMgNSwzVjNIMTlWM0MxOS4yMiwzIDE5LjQzLDMuMDggMTkuNjIsMy4yMkMyMC4wNSwzLjU2IDIwLjEzLDQuMTkgMTkuNzksNC42MkwxNC4wMywxMkgxNFoiIC8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-folder-favorite: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB3aWR0aD0iMjRweCIgZmlsbD0iIzAwMDAwMCI+CiAgPHBhdGggZD0iTTAgMGgyNHYyNEgwVjB6IiBmaWxsPSJub25lIi8+PHBhdGggY2xhc3M9ImpwLWljb24zIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzYxNjE2MSIgZD0iTTIwIDZoLThsLTItMkg0Yy0xLjEgMC0yIC45LTIgMnYxMmMwIDEuMS45IDIgMiAyaDE2YzEuMSAwIDItLjkgMi0yVjhjMC0xLjEtLjktMi0yLTJ6bS0yLjA2IDExTDE1IDE1LjI4IDEyLjA2IDE3bC43OC0zLjMzLTIuNTktMi4yNCAzLjQxLS4yOUwxNSA4bDEuMzQgMy4xNCAzLjQxLjI5LTIuNTkgMi4yNC43OCAzLjMzeiIvPgo8L3N2Zz4K);
  --jp-icon-folder: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTAgNEg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMThjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY4YzAtMS4xLS45LTItMi0yaC04bC0yLTJ6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-home: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB3aWR0aD0iMjRweCIgZmlsbD0iIzAwMDAwMCI+CiAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPjxwYXRoIGNsYXNzPSJqcC1pY29uMyBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xMCAyMHYtNmg0djZoNXYtOGgzTDEyIDMgMiAxMmgzdjh6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-html5: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUxMiA1MTIiPgogIDxwYXRoIGNsYXNzPSJqcC1pY29uMCBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiMwMDAiIGQ9Ik0xMDguNCAwaDIzdjIyLjhoMjEuMlYwaDIzdjY5aC0yM1Y0NmgtMjF2MjNoLTIzLjJNMjA2IDIzaC0yMC4zVjBoNjMuN3YyM0gyMjl2NDZoLTIzbTUzLjUtNjloMjQuMWwxNC44IDI0LjNMMzEzLjIgMGgyNC4xdjY5aC0yM1YzNC44bC0xNi4xIDI0LjgtMTYuMS0yNC44VjY5aC0yMi42bTg5LjItNjloMjN2NDYuMmgzMi42VjY5aC01NS42Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI2U0NGQyNiIgZD0iTTEwNy42IDQ3MWwtMzMtMzcwLjRoMzYyLjhsLTMzIDM3MC4yTDI1NS43IDUxMiIvPgogIDxwYXRoIGNsYXNzPSJqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNmMTY1MjkiIGQ9Ik0yNTYgNDgwLjVWMTMxaDE0OC4zTDM3NiA0NDciLz4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNlYmViZWIiIGQ9Ik0xNDIgMTc2LjNoMTE0djQ1LjRoLTY0LjJsNC4yIDQ2LjVoNjB2NDUuM0gxNTQuNG0yIDIyLjhIMjAybDMuMiAzNi4zIDUwLjggMTMuNnY0Ny40bC05My4yLTI2Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIiBmaWxsPSIjZmZmIiBkPSJNMzY5LjYgMTc2LjNIMjU1Ljh2NDUuNGgxMDkuNm0tNC4xIDQ2LjVIMjU1Ljh2NDUuNGg1NmwtNS4zIDU5LTUwLjcgMTMuNnY0Ny4ybDkzLTI1LjgiLz4KPC9zdmc+Cg==);
  --jp-icon-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1icmFuZDQganAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNGRkYiIGQ9Ik0yLjIgMi4yaDE3LjV2MTcuNUgyLjJ6Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzNGNTFCNSIgZD0iTTIuMiAyLjJ2MTcuNWgxNy41bC4xLTE3LjVIMi4yem0xMi4xIDIuMmMxLjIgMCAyLjIgMSAyLjIgMi4ycy0xIDIuMi0yLjIgMi4yLTIuMi0xLTIuMi0yLjIgMS0yLjIgMi4yLTIuMnpNNC40IDE3LjZsMy4zLTguOCAzLjMgNi42IDIuMi0zLjIgNC40IDUuNEg0LjR6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-info: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUwLjk3OCA1MC45NzgiPgoJPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KCQk8cGF0aCBkPSJNNDMuNTIsNy40NThDMzguNzExLDIuNjQ4LDMyLjMwNywwLDI1LjQ4OSwwQzE4LjY3LDAsMTIuMjY2LDIuNjQ4LDcuNDU4LDcuNDU4CgkJCWMtOS45NDMsOS45NDEtOS45NDMsMjYuMTE5LDAsMzYuMDYyYzQuODA5LDQuODA5LDExLjIxMiw3LjQ1NiwxOC4wMzEsNy40NThjMCwwLDAuMDAxLDAsMC4wMDIsMAoJCQljNi44MTYsMCwxMy4yMjEtMi42NDgsMTguMDI5LTcuNDU4YzQuODA5LTQuODA5LDcuNDU3LTExLjIxMiw3LjQ1Ny0xOC4wM0M1MC45NzcsMTguNjcsNDguMzI4LDEyLjI2Niw0My41Miw3LjQ1OHoKCQkJIE00Mi4xMDYsNDIuMTA1Yy00LjQzMiw0LjQzMS0xMC4zMzIsNi44NzItMTYuNjE1LDYuODcyaC0wLjAwMmMtNi4yODUtMC4wMDEtMTIuMTg3LTIuNDQxLTE2LjYxNy02Ljg3MgoJCQljLTkuMTYyLTkuMTYzLTkuMTYyLTI0LjA3MSwwLTMzLjIzM0MxMy4zMDMsNC40NCwxOS4yMDQsMiwyNS40ODksMmM2LjI4NCwwLDEyLjE4NiwyLjQ0LDE2LjYxNyw2Ljg3MgoJCQljNC40MzEsNC40MzEsNi44NzEsMTAuMzMyLDYuODcxLDE2LjYxN0M0OC45NzcsMzEuNzcyLDQ2LjUzNiwzNy42NzUsNDIuMTA2LDQyLjEwNXoiLz4KCQk8cGF0aCBkPSJNMjMuNTc4LDMyLjIxOGMtMC4wMjMtMS43MzQsMC4xNDMtMy4wNTksMC40OTYtMy45NzJjMC4zNTMtMC45MTMsMS4xMS0xLjk5NywyLjI3Mi0zLjI1MwoJCQljMC40NjgtMC41MzYsMC45MjMtMS4wNjIsMS4zNjctMS41NzVjMC42MjYtMC43NTMsMS4xMDQtMS40NzgsMS40MzYtMi4xNzVjMC4zMzEtMC43MDcsMC40OTUtMS41NDEsMC40OTUtMi41CgkJCWMwLTEuMDk2LTAuMjYtMi4wODgtMC43NzktMi45NzljLTAuNTY1LTAuODc5LTEuNTAxLTEuMzM2LTIuODA2LTEuMzY5Yy0xLjgwMiwwLjA1Ny0yLjk4NSwwLjY2Ny0zLjU1LDEuODMyCgkJCWMtMC4zMDEsMC41MzUtMC41MDMsMS4xNDEtMC42MDcsMS44MTRjLTAuMTM5LDAuNzA3LTAuMjA3LDEuNDMyLTAuMjA3LDIuMTc0aC0yLjkzN2MtMC4wOTEtMi4yMDgsMC40MDctNC4xMTQsMS40OTMtNS43MTkKCQkJYzEuMDYyLTEuNjQsMi44NTUtMi40ODEsNS4zNzgtMi41MjdjMi4xNiwwLjAyMywzLjg3NCwwLjYwOCw1LjE0MSwxLjc1OGMxLjI3OCwxLjE2LDEuOTI5LDIuNzY0LDEuOTUsNC44MTEKCQkJYzAsMS4xNDItMC4xMzcsMi4xMTEtMC40MSwyLjkxMWMtMC4zMDksMC44NDUtMC43MzEsMS41OTMtMS4yNjgsMi4yNDNjLTAuNDkyLDAuNjUtMS4wNjgsMS4zMTgtMS43MywyLjAwMgoJCQljLTAuNjUsMC42OTctMS4zMTMsMS40NzktMS45ODcsMi4zNDZjLTAuMjM5LDAuMzc3LTAuNDI5LDAuNzc3LTAuNTY1LDEuMTk5Yy0wLjE2LDAuOTU5LTAuMjE3LDEuOTUxLTAuMTcxLDIuOTc5CgkJCUMyNi41ODksMzIuMjE4LDIzLjU3OCwzMi4yMTgsMjMuNTc4LDMyLjIxOHogTTIzLjU3OCwzOC4yMnYtMy40ODRoMy4wNzZ2My40ODRIMjMuNTc4eiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-inspector: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaW5zcGVjdG9yLWljb24tY29sb3IganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMjAgNEg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMThjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY2YzAtMS4xLS45LTItMi0yem0tNSAxNEg0di00aDExdjR6bTAtNUg0VjloMTF2NHptNSA1aC00VjloNHY5eiIvPgo8L3N2Zz4K);
  --jp-icon-json: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtanNvbi1pY29uLWNvbG9yIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI0Y5QTgyNSI+CiAgICA8cGF0aCBkPSJNMjAuMiAxMS44Yy0xLjYgMC0xLjcuNS0xLjcgMSAwIC40LjEuOS4xIDEuMy4xLjUuMS45LjEgMS4zIDAgMS43LTEuNCAyLjMtMy41IDIuM2gtLjl2LTEuOWguNWMxLjEgMCAxLjQgMCAxLjQtLjggMC0uMyAwLS42LS4xLTEgMC0uNC0uMS0uOC0uMS0xLjIgMC0xLjMgMC0xLjggMS4zLTItMS4zLS4yLTEuMy0uNy0xLjMtMiAwLS40LjEtLjguMS0xLjIuMS0uNC4xLS43LjEtMSAwLS44LS40LS43LTEuNC0uOGgtLjVWNC4xaC45YzIuMiAwIDMuNS43IDMuNSAyLjMgMCAuNC0uMS45LS4xIDEuMy0uMS41LS4xLjktLjEgMS4zIDAgLjUuMiAxIDEuNyAxdjEuOHpNMS44IDEwLjFjMS42IDAgMS43LS41IDEuNy0xIDAtLjQtLjEtLjktLjEtMS4zLS4xLS41LS4xLS45LS4xLTEuMyAwLTEuNiAxLjQtMi4zIDMuNS0yLjNoLjl2MS45aC0uNWMtMSAwLTEuNCAwLTEuNC44IDAgLjMgMCAuNi4xIDEgMCAuMi4xLjYuMSAxIDAgMS4zIDAgMS44LTEuMyAyQzYgMTEuMiA2IDExLjcgNiAxM2MwIC40LS4xLjgtLjEgMS4yLS4xLjMtLjEuNy0uMSAxIDAgLjguMy44IDEuNC44aC41djEuOWgtLjljLTIuMSAwLTMuNS0uNi0zLjUtMi4zIDAtLjQuMS0uOS4xLTEuMy4xLS41LjEtLjkuMS0xLjMgMC0uNS0uMi0xLTEuNy0xdi0xLjl6Ii8+CiAgICA8Y2lyY2xlIGN4PSIxMSIgY3k9IjEzLjgiIHI9IjIuMSIvPgogICAgPGNpcmNsZSBjeD0iMTEiIGN5PSI4LjIiIHI9IjIuMSIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-julia: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDMyNSAzMDAiPgogIDxnIGNsYXNzPSJqcC1icmFuZDAganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjY2IzYzMzIj4KICAgIDxwYXRoIGQ9Ik0gMTUwLjg5ODQzOCAyMjUgQyAxNTAuODk4NDM4IDI2Ni40MjE4NzUgMTE3LjMyMDMxMiAzMDAgNzUuODk4NDM4IDMwMCBDIDM0LjQ3NjU2MiAzMDAgMC44OTg0MzggMjY2LjQyMTg3NSAwLjg5ODQzOCAyMjUgQyAwLjg5ODQzOCAxODMuNTc4MTI1IDM0LjQ3NjU2MiAxNTAgNzUuODk4NDM4IDE1MCBDIDExNy4zMjAzMTIgMTUwIDE1MC44OTg0MzggMTgzLjU3ODEyNSAxNTAuODk4NDM4IDIyNSIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzM4OTgyNiI+CiAgICA8cGF0aCBkPSJNIDIzNy41IDc1IEMgMjM3LjUgMTE2LjQyMTg3NSAyMDMuOTIxODc1IDE1MCAxNjIuNSAxNTAgQyAxMjEuMDc4MTI1IDE1MCA4Ny41IDExNi40MjE4NzUgODcuNSA3NSBDIDg3LjUgMzMuNTc4MTI1IDEyMS4wNzgxMjUgMCAxNjIuNSAwIEMgMjAzLjkyMTg3NSAwIDIzNy41IDMzLjU3ODEyNSAyMzcuNSA3NSIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzk1NThiMiI+CiAgICA8cGF0aCBkPSJNIDMyNC4xMDE1NjIgMjI1IEMgMzI0LjEwMTU2MiAyNjYuNDIxODc1IDI5MC41MjM0MzggMzAwIDI0OS4xMDE1NjIgMzAwIEMgMjA3LjY3OTY4OCAzMDAgMTc0LjEwMTU2MiAyNjYuNDIxODc1IDE3NC4xMDE1NjIgMjI1IEMgMTc0LjEwMTU2MiAxODMuNTc4MTI1IDIwNy42Nzk2ODggMTUwIDI0OS4xMDE1NjIgMTUwIEMgMjkwLjUyMzQzOCAxNTAgMzI0LjEwMTU2MiAxODMuNTc4MTI1IDMyNC4xMDE1NjIgMjI1Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-jupyter-favicon: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTUyIiBoZWlnaHQ9IjE2NSIgdmlld0JveD0iMCAwIDE1MiAxNjUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgPGcgY2xhc3M9ImpwLWp1cHl0ZXItaWNvbi1jb2xvciIgZmlsbD0iI0YzNzcyNiI+CiAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjA3ODk0NywgMTEwLjU4MjkyNykiIGQ9Ik03NS45NDIyODQyLDI5LjU4MDQ1NjEgQzQzLjMwMjM5NDcsMjkuNTgwNDU2MSAxNC43OTY3ODMyLDE3LjY1MzQ2MzQgMCwwIEM1LjUxMDgzMjExLDE1Ljg0MDY4MjkgMTUuNzgxNTM4OSwyOS41NjY3NzMyIDI5LjM5MDQ5NDcsMzkuMjc4NDE3MSBDNDIuOTk5Nyw0OC45ODk4NTM3IDU5LjI3MzcsNTQuMjA2NzgwNSA3NS45NjA1Nzg5LDU0LjIwNjc4MDUgQzkyLjY0NzQ1NzksNTQuMjA2NzgwNSAxMDguOTIxNDU4LDQ4Ljk4OTg1MzcgMTIyLjUzMDY2MywzOS4yNzg0MTcxIEMxMzYuMTM5NDUzLDI5LjU2Njc3MzIgMTQ2LjQxMDI4NCwxNS44NDA2ODI5IDE1MS45MjExNTgsMCBDMTM3LjA4Nzg2OCwxNy42NTM0NjM0IDEwOC41ODI1ODksMjkuNTgwNDU2MSA3NS45NDIyODQyLDI5LjU4MDQ1NjEgTDc1Ljk0MjI4NDIsMjkuNTgwNDU2MSBaIiAvPgogICAgPHBhdGggdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMzczNjgsIDAuNzA0ODc4KSIgZD0iTTc1Ljk3ODQ1NzksMjQuNjI2NDA3MyBDMTA4LjYxODc2MywyNC42MjY0MDczIDEzNy4xMjQ0NTgsMzYuNTUzNDQxNSAxNTEuOTIxMTU4LDU0LjIwNjc4MDUgQzE0Ni40MTAyODQsMzguMzY2MjIyIDEzNi4xMzk0NTMsMjQuNjQwMTMxNyAxMjIuNTMwNjYzLDE0LjkyODQ4NzggQzEwOC45MjE0NTgsNS4yMTY4NDM5IDkyLjY0NzQ1NzksMCA3NS45NjA1Nzg5LDAgQzU5LjI3MzcsMCA0Mi45OTk3LDUuMjE2ODQzOSAyOS4zOTA0OTQ3LDE0LjkyODQ4NzggQzE1Ljc4MTUzODksMjQuNjQwMTMxNyA1LjUxMDgzMjExLDM4LjM2NjIyMiAwLDU0LjIwNjc4MDUgQzE0LjgzMzA4MTYsMzYuNTg5OTI5MyA0My4zMzg1Njg0LDI0LjYyNjQwNzMgNzUuOTc4NDU3OSwyNC42MjY0MDczIEw3NS45Nzg0NTc5LDI0LjYyNjQwNzMgWiIgLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-jupyter: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzkiIGhlaWdodD0iNTEiIHZpZXdCb3g9IjAgMCAzOSA1MSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMTYzOCAtMjI4MSkiPgogICAgIDxnIGNsYXNzPSJqcC1qdXB5dGVyLWljb24tY29sb3IiIGZpbGw9IiNGMzc3MjYiPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM5Ljc0IDIzMTEuOTgpIiBkPSJNIDE4LjI2NDYgNy4xMzQxMUMgMTAuNDE0NSA3LjEzNDExIDMuNTU4NzIgNC4yNTc2IDAgMEMgMS4zMjUzOSAzLjgyMDQgMy43OTU1NiA3LjEzMDgxIDcuMDY4NiA5LjQ3MzAzQyAxMC4zNDE3IDExLjgxNTIgMTQuMjU1NyAxMy4wNzM0IDE4LjI2OSAxMy4wNzM0QyAyMi4yODIzIDEzLjA3MzQgMjYuMTk2MyAxMS44MTUyIDI5LjQ2OTQgOS40NzMwM0MgMzIuNzQyNCA3LjEzMDgxIDM1LjIxMjYgMy44MjA0IDM2LjUzOCAwQyAzMi45NzA1IDQuMjU3NiAyNi4xMTQ4IDcuMTM0MTEgMTguMjY0NiA3LjEzNDExWiIvPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM5LjczIDIyODUuNDgpIiBkPSJNIDE4LjI3MzMgNS45MzkzMUMgMjYuMTIzNSA1LjkzOTMxIDMyLjk3OTMgOC44MTU4MyAzNi41MzggMTMuMDczNEMgMzUuMjEyNiA5LjI1MzAzIDMyLjc0MjQgNS45NDI2MiAyOS40Njk0IDMuNjAwNEMgMjYuMTk2MyAxLjI1ODE4IDIyLjI4MjMgMCAxOC4yNjkgMEMgMTQuMjU1NyAwIDEwLjM0MTcgMS4yNTgxOCA3LjA2ODYgMy42MDA0QyAzLjc5NTU2IDUuOTQyNjIgMS4zMjUzOSA5LjI1MzAzIDAgMTMuMDczNEMgMy41Njc0NSA4LjgyNDYzIDEwLjQyMzIgNS45MzkzMSAxOC4yNzMzIDUuOTM5MzFaIi8+CiAgICA8L2c+CiAgICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjY5LjMgMjI4MS4zMSkiIGQ9Ik0gNS44OTM1MyAyLjg0NEMgNS45MTg4OSAzLjQzMTY1IDUuNzcwODUgNC4wMTM2NyA1LjQ2ODE1IDQuNTE2NDVDIDUuMTY1NDUgNS4wMTkyMiA0LjcyMTY4IDUuNDIwMTUgNC4xOTI5OSA1LjY2ODUxQyAzLjY2NDMgNS45MTY4OCAzLjA3NDQ0IDYuMDAxNTEgMi40OTgwNSA1LjkxMTcxQyAxLjkyMTY2IDUuODIxOSAxLjM4NDYzIDUuNTYxNyAwLjk1NDg5OCA1LjE2NDAxQyAwLjUyNTE3IDQuNzY2MzMgMC4yMjIwNTYgNC4yNDkwMyAwLjA4MzkwMzcgMy42Nzc1N0MgLTAuMDU0MjQ4MyAzLjEwNjExIC0wLjAyMTIzIDIuNTA2MTcgMC4xNzg3ODEgMS45NTM2NEMgMC4zNzg3OTMgMS40MDExIDAuNzM2ODA5IDAuOTIwODE3IDEuMjA3NTQgMC41NzM1MzhDIDEuNjc4MjYgMC4yMjYyNTkgMi4yNDA1NSAwLjAyNzU5MTkgMi44MjMyNiAwLjAwMjY3MjI5QyAzLjYwMzg5IC0wLjAzMDcxMTUgNC4zNjU3MyAwLjI0OTc4OSA0Ljk0MTQyIDAuNzgyNTUxQyA1LjUxNzExIDEuMzE1MzEgNS44NTk1NiAyLjA1Njc2IDUuODkzNTMgMi44NDRaIi8+CiAgICAgIDxwYXRoIHRyYW5zZm9ybT0idHJhbnNsYXRlKDE2MzkuOCAyMzIzLjgxKSIgZD0iTSA3LjQyNzg5IDMuNTgzMzhDIDcuNDYwMDggNC4zMjQzIDcuMjczNTUgNS4wNTgxOSA2Ljg5MTkzIDUuNjkyMTNDIDYuNTEwMzEgNi4zMjYwNyA1Ljk1MDc1IDYuODMxNTYgNS4yODQxMSA3LjE0NDZDIDQuNjE3NDcgNy40NTc2MyAzLjg3MzcxIDcuNTY0MTQgMy4xNDcwMiA3LjQ1MDYzQyAyLjQyMDMyIDcuMzM3MTIgMS43NDMzNiA3LjAwODcgMS4yMDE4NCA2LjUwNjk1QyAwLjY2MDMyOCA2LjAwNTIgMC4yNzg2MSA1LjM1MjY4IDAuMTA1MDE3IDQuNjMyMDJDIC0wLjA2ODU3NTcgMy45MTEzNSAtMC4wMjYyMzYxIDMuMTU0OTQgMC4yMjY2NzUgMi40NTg1NkMgMC40Nzk1ODcgMS43NjIxNyAwLjkzMTY5NyAxLjE1NzEzIDEuNTI1NzYgMC43MjAwMzNDIDIuMTE5ODMgMC4yODI5MzUgMi44MjkxNCAwLjAzMzQzOTUgMy41NjM4OSAwLjAwMzEzMzQ0QyA0LjU0NjY3IC0wLjAzNzQwMzMgNS41MDUyOSAwLjMxNjcwNiA2LjIyOTYxIDAuOTg3ODM1QyA2Ljk1MzkzIDEuNjU4OTYgNy4zODQ4NCAyLjU5MjM1IDcuNDI3ODkgMy41ODMzOEwgNy40Mjc4OSAzLjU4MzM4WiIvPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM4LjM2IDIyODYuMDYpIiBkPSJNIDIuMjc0NzEgNC4zOTYyOUMgMS44NDM2MyA0LjQxNTA4IDEuNDE2NzEgNC4zMDQ0NSAxLjA0Nzk5IDQuMDc4NDNDIDAuNjc5MjY4IDMuODUyNCAwLjM4NTMyOCAzLjUyMTE0IDAuMjAzMzcxIDMuMTI2NTZDIDAuMDIxNDEzNiAyLjczMTk4IC0wLjA0MDM3OTggMi4yOTE4MyAwLjAyNTgxMTYgMS44NjE4MUMgMC4wOTIwMDMxIDEuNDMxOCAwLjI4MzIwNCAxLjAzMTI2IDAuNTc1MjEzIDAuNzEwODgzQyAwLjg2NzIyMiAwLjM5MDUxIDEuMjQ2OTEgMC4xNjQ3MDggMS42NjYyMiAwLjA2MjA1OTJDIDIuMDg1NTMgLTAuMDQwNTg5NyAyLjUyNTYxIC0wLjAxNTQ3MTQgMi45MzA3NiAwLjEzNDIzNUMgMy4zMzU5MSAwLjI4Mzk0MSAzLjY4NzkyIDAuNTUxNTA1IDMuOTQyMjIgMC45MDMwNkMgNC4xOTY1MiAxLjI1NDYyIDQuMzQxNjkgMS42NzQzNiA0LjM1OTM1IDIuMTA5MTZDIDQuMzgyOTkgMi42OTEwNyA0LjE3Njc4IDMuMjU4NjkgMy43ODU5NyAzLjY4NzQ2QyAzLjM5NTE2IDQuMTE2MjQgMi44NTE2NiA0LjM3MTE2IDIuMjc0NzEgNC4zOTYyOUwgMi4yNzQ3MSA0LjM5NjI5WiIvPgogICAgPC9nPgogIDwvZz4+Cjwvc3ZnPgo=);
  --jp-icon-jupyterlab-wordmark: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMDAiIHZpZXdCb3g9IjAgMCAxODYwLjggNDc1Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0RTRFNEUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDQ4MC4xMzY0MDEsIDY0LjI3MTQ5MykiPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMDAwMDAsIDU4Ljg3NTU2NikiPgogICAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjA4NzYwMywgMC4xNDAyOTQpIj4KICAgICAgICA8cGF0aCBkPSJNLTQyNi45LDE2OS44YzAsNDguNy0zLjcsNjQuNy0xMy42LDc2LjRjLTEwLjgsMTAtMjUsMTUuNS0zOS43LDE1LjVsMy43LDI5IGMyMi44LDAuMyw0NC44LTcuOSw2MS45LTIzLjFjMTcuOC0xOC41LDI0LTQ0LjEsMjQtODMuM1YwSC00Mjd2MTcwLjFMLTQyNi45LDE2OS44TC00MjYuOSwxNjkuOHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTU1LjA0NTI5NiwgNTYuODM3MTA0KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuNTYyNDUzLCAxLjc5OTg0MikiPgogICAgICAgIDxwYXRoIGQ9Ik0tMzEyLDE0OGMwLDIxLDAsMzkuNSwxLjcsNTUuNGgtMzEuOGwtMi4xLTMzLjNoLTAuOGMtNi43LDExLjYtMTYuNCwyMS4zLTI4LDI3LjkgYy0xMS42LDYuNi0yNC44LDEwLTM4LjIsOS44Yy0zMS40LDAtNjktMTcuNy02OS04OVYwaDM2LjR2MTEyLjdjMCwzOC43LDExLjYsNjQuNyw0NC42LDY0LjdjMTAuMy0wLjIsMjAuNC0zLjUsMjguOS05LjQgYzguNS01LjksMTUuMS0xNC4zLDE4LjktMjMuOWMyLjItNi4xLDMuMy0xMi41LDMuMy0xOC45VjAuMmgzNi40VjE0OEgtMzEyTC0zMTIsMTQ4eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgzOTAuMDEzMzIyLCA1My40Nzk2MzgpIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMS43MDY0NTgsIDAuMjMxNDI1KSI+CiAgICAgICAgPHBhdGggZD0iTS00NzguNiw3MS40YzAtMjYtMC44LTQ3LTEuNy02Ni43aDMyLjdsMS43LDM0LjhoMC44YzcuMS0xMi41LDE3LjUtMjIuOCwzMC4xLTI5LjcgYzEyLjUtNywyNi43LTEwLjMsNDEtOS44YzQ4LjMsMCw4NC43LDQxLjcsODQuNywxMDMuM2MwLDczLjEtNDMuNywxMDkuMi05MSwxMDkuMmMtMTIuMSwwLjUtMjQuMi0yLjItMzUtNy44IGMtMTAuOC01LjYtMTkuOS0xMy45LTI2LjYtMjQuMmgtMC44VjI5MWgtMzZ2LTIyMEwtNDc4LjYsNzEuNEwtNDc4LjYsNzEuNHogTS00NDIuNiwxMjUuNmMwLjEsNS4xLDAuNiwxMC4xLDEuNywxNS4xIGMzLDEyLjMsOS45LDIzLjMsMTkuOCwzMS4xYzkuOSw3LjgsMjIuMSwxMi4xLDM0LjcsMTIuMWMzOC41LDAsNjAuNy0zMS45LDYwLjctNzguNWMwLTQwLjctMjEuMS03NS42LTU5LjUtNzUuNiBjLTEyLjksMC40LTI1LjMsNS4xLTM1LjMsMTMuNGMtOS45LDguMy0xNi45LDE5LjctMTkuNiwzMi40Yy0xLjUsNC45LTIuMywxMC0yLjUsMTUuMVYxMjUuNkwtNDQyLjYsMTI1LjZMLTQ0Mi42LDEyNS42eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSg2MDYuNzQwNzI2LCA1Ni44MzcxMDQpIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC43NTEyMjYsIDEuOTg5Mjk5KSI+CiAgICAgICAgPHBhdGggZD0iTS00NDAuOCwwbDQzLjcsMTIwLjFjNC41LDEzLjQsOS41LDI5LjQsMTIuOCw0MS43aDAuOGMzLjctMTIuMiw3LjktMjcuNywxMi44LTQyLjQgbDM5LjctMTE5LjJoMzguNUwtMzQ2LjksMTQ1Yy0yNiw2OS43LTQzLjcsMTA1LjQtNjguNiwxMjcuMmMtMTIuNSwxMS43LTI3LjksMjAtNDQuNiwyMy45bC05LjEtMzEuMSBjMTEuNy0zLjksMjIuNS0xMC4xLDMxLjgtMTguMWMxMy4yLTExLjEsMjMuNy0yNS4yLDMwLjYtNDEuMmMxLjUtMi44LDIuNS01LjcsMi45LTguOGMtMC4zLTMuMy0xLjItNi42LTIuNS05LjdMLTQ4MC4yLDAuMSBoMzkuN0wtNDQwLjgsMEwtNDQwLjgsMHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoODIyLjc0ODEwNCwgMC4wMDAwMDApIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMS40NjQwNTAsIDAuMzc4OTE0KSI+CiAgICAgICAgPHBhdGggZD0iTS00MTMuNywwdjU4LjNoNTJ2MjguMmgtNTJWMTk2YzAsMjUsNywzOS41LDI3LjMsMzkuNWM3LjEsMC4xLDE0LjItMC43LDIxLjEtMi41IGwxLjcsMjcuN2MtMTAuMywzLjctMjEuMyw1LjQtMzIuMiw1Yy03LjMsMC40LTE0LjYtMC43LTIxLjMtMy40Yy02LjgtMi43LTEyLjktNi44LTE3LjktMTIuMWMtMTAuMy0xMC45LTE0LjEtMjktMTQuMS01Mi45IFY4Ni41aC0zMVY1OC4zaDMxVjkuNkwtNDEzLjcsMEwtNDEzLjcsMHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOTc0LjQzMzI4NiwgNTMuNDc5NjM4KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuOTkwMDM0LCAwLjYxMDMzOSkiPgogICAgICAgIDxwYXRoIGQ9Ik0tNDQ1LjgsMTEzYzAuOCw1MCwzMi4yLDcwLjYsNjguNiw3MC42YzE5LDAuNiwzNy45LTMsNTUuMy0xMC41bDYuMiwyNi40IGMtMjAuOSw4LjktNDMuNSwxMy4xLTY2LjIsMTIuNmMtNjEuNSwwLTk4LjMtNDEuMi05OC4zLTEwMi41Qy00ODAuMiw0OC4yLTQ0NC43LDAtMzg2LjUsMGM2NS4yLDAsODIuNyw1OC4zLDgyLjcsOTUuNyBjLTAuMSw1LjgtMC41LDExLjUtMS4yLDE3LjJoLTE0MC42SC00NDUuOEwtNDQ1LjgsMTEzeiBNLTMzOS4yLDg2LjZjMC40LTIzLjUtOS41LTYwLjEtNTAuNC02MC4xIGMtMzYuOCwwLTUyLjgsMzQuNC01NS43LDYwLjFILTMzOS4yTC0zMzkuMiw4Ni42TC0zMzkuMiw4Ni42eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxMjAxLjk2MTA1OCwgNTMuNDc5NjM4KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuMTc5NjQwLCAwLjcwNTA2OCkiPgogICAgICAgIDxwYXRoIGQ9Ik0tNDc4LjYsNjhjMC0yMy45LTAuNC00NC41LTEuNy02My40aDMxLjhsMS4yLDM5LjloMS43YzkuMS0yNy4zLDMxLTQ0LjUsNTUuMy00NC41IGMzLjUtMC4xLDcsMC40LDEwLjMsMS4ydjM0LjhjLTQuMS0wLjktOC4yLTEuMy0xMi40LTEuMmMtMjUuNiwwLTQzLjcsMTkuNy00OC43LDQ3LjRjLTEsNS43LTEuNiwxMS41LTEuNywxNy4ydjEwOC4zaC0zNlY2OCBMLTQ3OC42LDY4eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbi13YXJuMCIgZmlsbD0iI0YzNzcyNiI+CiAgICA8cGF0aCBkPSJNMTM1Mi4zLDMyNi4yaDM3VjI4aC0zN1YzMjYuMnogTTE2MDQuOCwzMjYuMmMtMi41LTEzLjktMy40LTMxLjEtMy40LTQ4Ljd2LTc2IGMwLTQwLjctMTUuMS04My4xLTc3LjMtODMuMWMtMjUuNiwwLTUwLDcuMS02Ni44LDE4LjFsOC40LDI0LjRjMTQuMy05LjIsMzQtMTUuMSw1My0xNS4xYzQxLjYsMCw0Ni4yLDMwLjIsNDYuMiw0N3Y0LjIgYy03OC42LTAuNC0xMjIuMywyNi41LTEyMi4zLDc1LjZjMCwyOS40LDIxLDU4LjQsNjIuMiw1OC40YzI5LDAsNTAuOS0xNC4zLDYyLjItMzAuMmgxLjNsMi45LDI1LjZIMTYwNC44eiBNMTU2NS43LDI1Ny43IGMwLDMuOC0wLjgsOC0yLjEsMTEuOGMtNS45LDE3LjItMjIuNywzNC00OS4yLDM0Yy0xOC45LDAtMzQuOS0xMS4zLTM0LjktMzUuM2MwLTM5LjUsNDUuOC00Ni42LDg2LjItNDUuOFYyNTcuN3ogTTE2OTguNSwzMjYuMiBsMS43LTMzLjZoMS4zYzE1LjEsMjYuOSwzOC43LDM4LjIsNjguMSwzOC4yYzQ1LjQsMCw5MS4yLTM2LjEsOTEuMi0xMDguOGMwLjQtNjEuNy0zNS4zLTEwMy43LTg1LjctMTAzLjcgYy0zMi44LDAtNTYuMywxNC43LTY5LjMsMzcuNGgtMC44VjI4aC0zNi42djI0NS43YzAsMTguMS0wLjgsMzguNi0xLjcsNTIuNUgxNjk4LjV6IE0xNzA0LjgsMjA4LjJjMC01LjksMS4zLTEwLjksMi4xLTE1LjEgYzcuNi0yOC4xLDMxLjEtNDUuNCw1Ni4zLTQ1LjRjMzkuNSwwLDYwLjUsMzQuOSw2MC41LDc1LjZjMCw0Ni42LTIzLjEsNzguMS02MS44LDc4LjFjLTI2LjksMC00OC4zLTE3LjYtNTUuNS00My4zIGMtMC44LTQuMi0xLjctOC44LTEuNy0xMy40VjIwOC4yeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-kernel: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgZmlsbD0iIzYxNjE2MSIgZD0iTTE1IDlIOXY2aDZWOXptLTIgNGgtMnYtMmgydjJ6bTgtMlY5aC0yVjdjMC0xLjEtLjktMi0yLTJoLTJWM2gtMnYyaC0yVjNIOXYySDdjLTEuMSAwLTIgLjktMiAydjJIM3YyaDJ2MkgzdjJoMnYyYzAgMS4xLjkgMiAyIDJoMnYyaDJ2LTJoMnYyaDJ2LTJoMmMxLjEgMCAyLS45IDItMnYtMmgydi0yaC0ydi0yaDJ6bS00IDZIN1Y3aDEwdjEweiIvPgo8L3N2Zz4K);
  --jp-icon-keyboard: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMjAgNUg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMTdjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY3YzAtMS4xLS45LTItMi0yem0tOSAzaDJ2MmgtMlY4em0wIDNoMnYyaC0ydi0yek04IDhoMnYySDhWOHptMCAzaDJ2Mkg4di0yem0tMSAySDV2LTJoMnYyem0wLTNINVY4aDJ2MnptOSA3SDh2LTJoOHYyem0wLTRoLTJ2LTJoMnYyem0wLTNoLTJWOGgydjJ6bTMgM2gtMnYtMmgydjJ6bTAtM2gtMlY4aDJ2MnoiLz4KPC9zdmc+Cg==);
  --jp-icon-launch: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMzIgMzIiIHdpZHRoPSIzMiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik0yNiwyOEg2YTIuMDAyNywyLjAwMjcsMCwwLDEtMi0yVjZBMi4wMDI3LDIuMDAyNywwLDAsMSw2LDRIMTZWNkg2VjI2SDI2VjE2aDJWMjZBMi4wMDI3LDIuMDAyNywwLDAsMSwyNiwyOFoiLz4KICAgIDxwb2x5Z29uIHBvaW50cz0iMjAgMiAyMCA0IDI2LjU4NiA0IDE4IDEyLjU4NiAxOS40MTQgMTQgMjggNS40MTQgMjggMTIgMzAgMTIgMzAgMiAyMCAyIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-launcher: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkgMTlINVY1aDdWM0g1YTIgMiAwIDAwLTIgMnYxNGEyIDIgMCAwMDIgMmgxNGMxLjEgMCAyLS45IDItMnYtN2gtMnY3ek0xNCAzdjJoMy41OWwtOS44MyA5LjgzIDEuNDEgMS40MUwxOSA2LjQxVjEwaDJWM2gtN3oiLz4KPC9zdmc+Cg==);
  --jp-icon-line-form: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGZpbGw9IndoaXRlIiBkPSJNNS44OCA0LjEyTDEzLjc2IDEybC03Ljg4IDcuODhMOCAyMmwxMC0xMEw4IDJ6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-link: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTMuOSAxMmMwLTEuNzEgMS4zOS0zLjEgMy4xLTMuMWg0VjdIN2MtMi43NiAwLTUgMi4yNC01IDVzMi4yNCA1IDUgNWg0di0xLjlIN2MtMS43MSAwLTMuMS0xLjM5LTMuMS0zLjF6TTggMTNoOHYtMkg4djJ6bTktNmgtNHYxLjloNGMxLjcxIDAgMy4xIDEuMzkgMy4xIDMuMXMtMS4zOSAzLjEtMy4xIDMuMWgtNFYxN2g0YzIuNzYgMCA1LTIuMjQgNS01cy0yLjI0LTUtNS01eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-list: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xOSA1djE0SDVWNWgxNG0xLjEtMkgzLjljLS41IDAtLjkuNC0uOS45djE2LjJjMCAuNC40LjkuOS45aDE2LjJjLjQgMCAuOS0uNS45LS45VjMuOWMwLS41LS41LS45LS45LS45ek0xMSA3aDZ2MmgtNlY3em0wIDRoNnYyaC02di0yem0wIDRoNnYyaC02ek03IDdoMnYySDd6bTAgNGgydjJIN3ptMCA0aDJ2Mkg3eiIvPgo8L3N2Zz4K);
  --jp-icon-markdown: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDAganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjN0IxRkEyIiBkPSJNNSAxNC45aDEybC02LjEgNnptOS40LTYuOGMwLTEuMy0uMS0yLjktLjEtNC41LS40IDEuNC0uOSAyLjktMS4zIDQuM2wtMS4zIDQuM2gtMkw4LjUgNy45Yy0uNC0xLjMtLjctMi45LTEtNC4zLS4xIDEuNi0uMSAzLjItLjIgNC42TDcgMTIuNEg0LjhsLjctMTFoMy4zTDEwIDVjLjQgMS4yLjcgMi43IDEgMy45LjMtMS4yLjctMi42IDEtMy45bDEuMi0zLjdoMy4zbC42IDExaC0yLjRsLS4zLTQuMnoiLz4KPC9zdmc+Cg==);
  --jp-icon-move-down: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBkPSJNMTIuNDcxIDcuNTI4OTlDMTIuNzYzMiA3LjIzNjg0IDEyLjc2MzIgNi43NjMxNiAxMi40NzEgNi40NzEwMVY2LjQ3MTAxQzEyLjE3OSA2LjE3OTA1IDExLjcwNTcgNi4xNzg4NCAxMS40MTM1IDYuNDcwNTRMNy43NSAxMC4xMjc1VjEuNzVDNy43NSAxLjMzNTc5IDcuNDE0MjEgMSA3IDFWMUM2LjU4NTc5IDEgNi4yNSAxLjMzNTc5IDYuMjUgMS43NVYxMC4xMjc1TDIuNTk3MjYgNi40NjgyMkMyLjMwMzM4IDYuMTczODEgMS44MjY0MSA2LjE3MzU5IDEuNTMyMjYgNi40Njc3NFY2LjQ2Nzc0QzEuMjM4MyA2Ljc2MTcgMS4yMzgzIDcuMjM4MyAxLjUzMjI2IDcuNTMyMjZMNi4yOTI4OSAxMi4yOTI5QzYuNjgzNDIgMTIuNjgzNCA3LjMxNjU4IDEyLjY4MzQgNy43MDcxMSAxMi4yOTI5TDEyLjQ3MSA3LjUyODk5WiIgZmlsbD0iIzYxNjE2MSIvPgo8L3N2Zz4K);
  --jp-icon-move-up: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBkPSJNMS41Mjg5OSA2LjQ3MTAxQzEuMjM2ODQgNi43NjMxNiAxLjIzNjg0IDcuMjM2ODQgMS41Mjg5OSA3LjUyODk5VjcuNTI4OTlDMS44MjA5NSA3LjgyMDk1IDIuMjk0MjYgNy44MjExNiAyLjU4NjQ5IDcuNTI5NDZMNi4yNSAzLjg3MjVWMTIuMjVDNi4yNSAxMi42NjQyIDYuNTg1NzkgMTMgNyAxM1YxM0M3LjQxNDIxIDEzIDcuNzUgMTIuNjY0MiA3Ljc1IDEyLjI1VjMuODcyNUwxMS40MDI3IDcuNTMxNzhDMTEuNjk2NiA3LjgyNjE5IDEyLjE3MzYgNy44MjY0MSAxMi40Njc3IDcuNTMyMjZWNy41MzIyNkMxMi43NjE3IDcuMjM4MyAxMi43NjE3IDYuNzYxNyAxMi40Njc3IDYuNDY3NzRMNy43MDcxMSAxLjcwNzExQzcuMzE2NTggMS4zMTY1OCA2LjY4MzQyIDEuMzE2NTggNi4yOTI4OSAxLjcwNzExTDEuNTI4OTkgNi40NzEwMVoiIGZpbGw9IiM2MTYxNjEiLz4KPC9zdmc+Cg==);
  --jp-icon-new-folder: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwIDZoLThsLTItMkg0Yy0xLjExIDAtMS45OS44OS0xLjk5IDJMMiAxOGMwIDEuMTEuODkgMiAyIDJoMTZjMS4xMSAwIDItLjg5IDItMlY4YzAtMS4xMS0uODktMi0yLTJ6bS0xIDhoLTN2M2gtMnYtM2gtM3YtMmgzVjloMnYzaDN2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-not-trusted: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI1IDI1Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMgMykiIGQ9Ik0xLjg2MDk0IDExLjQ0MDlDMC44MjY0NDggOC43NzAyNyAwLjg2Mzc3OSA2LjA1NzY0IDEuMjQ5MDcgNC4xOTkzMkMyLjQ4MjA2IDMuOTMzNDcgNC4wODA2OCAzLjQwMzQ3IDUuNjAxMDIgMi44NDQ5QzcuMjM1NDkgMi4yNDQ0IDguODU2NjYgMS41ODE1IDkuOTg3NiAxLjA5NTM5QzExLjA1OTcgMS41ODM0MSAxMi42MDk0IDIuMjQ0NCAxNC4yMTggMi44NDMzOUMxNS43NTAzIDMuNDEzOTQgMTcuMzk5NSAzLjk1MjU4IDE4Ljc1MzkgNC4yMTM4NUMxOS4xMzY0IDYuMDcxNzcgMTkuMTcwOSA4Ljc3NzIyIDE4LjEzOSAxMS40NDA5QzE3LjAzMDMgMTQuMzAzMiAxNC42NjY4IDE3LjE4NDQgOS45OTk5OSAxOC45MzU0QzUuMzMzMTkgMTcuMTg0NCAyLjk2OTY4IDE0LjMwMzIgMS44NjA5NCAxMS40NDA5WiIvPgogICAgPHBhdGggY2xhc3M9ImpwLWljb24yIiBzdHJva2U9IiMzMzMzMzMiIHN0cm9rZS13aWR0aD0iMiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOS4zMTU5MiA5LjMyMDMxKSIgZD0iTTcuMzY4NDIgMEwwIDcuMzY0NzkiLz4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDkuMzE1OTIgMTYuNjgzNikgc2NhbGUoMSAtMSkiIGQ9Ik03LjM2ODQyIDBMMCA3LjM2NDc5Ii8+Cjwvc3ZnPgo=);
  --jp-icon-notebook: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtbm90ZWJvb2staWNvbi1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNFRjZDMDAiPgogICAgPHBhdGggZD0iTTE4LjcgMy4zdjE1LjRIMy4zVjMuM2gxNS40bTEuNS0xLjVIMS44djE4LjNoMTguM2wuMS0xOC4zeiIvPgogICAgPHBhdGggZD0iTTE2LjUgMTYuNWwtNS40LTQuMy01LjYgNC4zdi0xMWgxMXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-numbering: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyOCAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTQgMTlINlYxOS41SDVWMjAuNUg2VjIxSDRWMjJIN1YxOEg0VjE5Wk01IDEwSDZWNkg0VjdINVYxMFpNNCAxM0g1LjhMNCAxNS4xVjE2SDdWMTVINS4yTDcgMTIuOVYxMkg0VjEzWk05IDdWOUgyM1Y3SDlaTTkgMjFIMjNWMTlIOVYyMVpNOSAxNUgyM1YxM0g5VjE1WiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-offline-bolt: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyIDIuMDJjLTUuNTEgMC05Ljk4IDQuNDctOS45OCA5Ljk4czQuNDcgOS45OCA5Ljk4IDkuOTggOS45OC00LjQ3IDkuOTgtOS45OFMxNy41MSAyLjAyIDEyIDIuMDJ6TTExLjQ4IDIwdi02LjI2SDhMMTMgNHY2LjI2aDMuMzVMMTEuNDggMjB6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-palette: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE4IDEzVjIwSDRWNkg5LjAyQzkuMDcgNS4yOSA5LjI0IDQuNjIgOS41IDRINEMyLjkgNCAyIDQuOSAyIDZWMjBDMiAyMS4xIDIuOSAyMiA0IDIySDE4QzE5LjEgMjIgMjAgMjEuMSAyMCAyMFYxNUwxOCAxM1pNMTkuMyA4Ljg5QzE5Ljc0IDguMTkgMjAgNy4zOCAyMCA2LjVDMjAgNC4wMSAxNy45OSAyIDE1LjUgMkMxMy4wMSAyIDExIDQuMDEgMTEgNi41QzExIDguOTkgMTMuMDEgMTEgMTUuNDkgMTFDMTYuMzcgMTEgMTcuMTkgMTAuNzQgMTcuODggMTAuM0wyMSAxMy40MkwyMi40MiAxMkwxOS4zIDguODlaTTE1LjUgOUMxNC4xMiA5IDEzIDcuODggMTMgNi41QzEzIDUuMTIgMTQuMTIgNCAxNS41IDRDMTYuODggNCAxOCA1LjEyIDE4IDYuNUMxOCA3Ljg4IDE2Ljg4IDkgMTUuNSA5WiIvPgogICAgPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik00IDZIOS4wMTg5NEM5LjAwNjM5IDYuMTY1MDIgOSA2LjMzMTc2IDkgNi41QzkgOC44MTU3NyAxMC4yMTEgMTAuODQ4NyAxMi4wMzQzIDEySDlWMTRIMTZWMTIuOTgxMUMxNi41NzAzIDEyLjkzNzcgMTcuMTIgMTIuODIwNyAxNy42Mzk2IDEyLjYzOTZMMTggMTNWMjBINFY2Wk04IDhINlYxMEg4VjhaTTYgMTJIOFYxNEg2VjEyWk04IDE2SDZWMThIOFYxNlpNOSAxNkgxNlYxOEg5VjE2WiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-paste: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTE5IDJoLTQuMThDMTQuNC44NCAxMy4zIDAgMTIgMGMtMS4zIDAtMi40Ljg0LTIuODIgMkg1Yy0xLjEgMC0yIC45LTIgMnYxNmMwIDEuMS45IDIgMiAyaDE0YzEuMSAwIDItLjkgMi0yVjRjMC0xLjEtLjktMi0yLTJ6bS03IDBjLjU1IDAgMSAuNDUgMSAxcy0uNDUgMS0xIDEtMS0uNDUtMS0xIC40NS0xIDEtMXptNyAxOEg1VjRoMnYzaDEwVjRoMnYxNnoiLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-pdf: url(data:image/svg+xml;base64,PHN2ZwogICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyMiAyMiIgd2lkdGg9IjE2Ij4KICAgIDxwYXRoIHRyYW5zZm9ybT0icm90YXRlKDQ1KSIgY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI0ZGMkEyQSIKICAgICAgIGQ9Im0gMjIuMzQ0MzY5LC0zLjAxNjM2NDIgaCA1LjYzODYwNCB2IDEuNTc5MjQzMyBoIC0zLjU0OTIyNyB2IDEuNTA4NjkyOTkgaCAzLjMzNzU3NiBWIDEuNjUwODE1NCBoIC0zLjMzNzU3NiB2IDMuNDM1MjYxMyBoIC0yLjA4OTM3NyB6IG0gLTcuMTM2NDQ0LDEuNTc5MjQzMyB2IDQuOTQzOTU0MyBoIDAuNzQ4OTIgcSAxLjI4MDc2MSwwIDEuOTUzNzAzLC0wLjYzNDk1MzUgMC42NzgzNjksLTAuNjM0OTUzNSAwLjY3ODM2OSwtMS44NDUxNjQxIDAsLTEuMjA0NzgzNTUgLTAuNjcyOTQyLC0xLjgzNDMxMDExIC0wLjY3Mjk0MiwtMC42Mjk1MjY1OSAtMS45NTkxMywtMC42Mjk1MjY1OSB6IG0gLTIuMDg5Mzc3LC0xLjU3OTI0MzMgaCAyLjIwMzM0MyBxIDEuODQ1MTY0LDAgMi43NDYwMzksMC4yNjU5MjA3IDAuOTA2MzAxLDAuMjYwNDkzNyAxLjU1MjEwOCwwLjg5MDAyMDMgMC41Njk4MywwLjU0ODEyMjMgMC44NDY2MDUsMS4yNjQ0ODAwNiAwLjI3Njc3NCwwLjcxNjM1NzgxIDAuMjc2Nzc0LDEuNjIyNjU4OTQgMCwwLjkxNzE1NTEgLTAuMjc2Nzc0LDEuNjM4OTM5OSAtMC4yNzY3NzUsMC43MTYzNTc4IC0wLjg0NjYwNSwxLjI2NDQ4IC0wLjY1MTIzNCwwLjYyOTUyNjYgLTEuNTYyOTYyLDAuODk1NDQ3MyAtMC45MTE3MjgsMC4yNjA0OTM3IC0yLjczNTE4NSwwLjI2MDQ5MzcgaCAtMi4yMDMzNDMgeiBtIC04LjE0NTg1NjUsMCBoIDMuNDY3ODIzIHEgMS41NDY2ODE2LDAgMi4zNzE1Nzg1LDAuNjg5MjIzIDAuODMwMzI0LDAuNjgzNzk2MSAwLjgzMDMyNCwxLjk1MzcwMzE0IDAsMS4yNzUzMzM5NyAtMC44MzAzMjQsMS45NjQ1NTcwNiBRIDkuOTg3MTk2MSwyLjI3NDkxNSA4LjQ0MDUxNDUsMi4yNzQ5MTUgSCA3LjA2MjA2ODQgViA1LjA4NjA3NjcgSCA0Ljk3MjY5MTUgWiBtIDIuMDg5Mzc2OSwxLjUxNDExOTkgdiAyLjI2MzAzOTQzIGggMS4xNTU5NDEgcSAwLjYwNzgxODgsMCAwLjkzODg2MjksLTAuMjkzMDU1NDcgMC4zMzEwNDQxLC0wLjI5ODQ4MjQxIDAuMzMxMDQ0MSwtMC44NDExNzc3MiAwLC0wLjU0MjY5NTMxIC0wLjMzMTA0NDEsLTAuODM1NzUwNzQgLTAuMzMxMDQ0MSwtMC4yOTMwNTU1IC0wLjkzODg2MjksLTAuMjkzMDU1NSB6IgovPgo8L3N2Zz4K);
  --jp-icon-python: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iLTEwIC0xMCAxMzEuMTYxMzYxNjk0MzM1OTQgMTMyLjM4ODk5OTkzODk2NDg0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMzA2OTk4IiBkPSJNIDU0LjkxODc4NSw5LjE5Mjc0MjFlLTQgQyA1MC4zMzUxMzIsMC4wMjIyMTcyNyA0NS45NTc4NDYsMC40MTMxMzY5NyA0Mi4xMDYyODUsMS4wOTQ2NjkzIDMwLjc2MDA2OSwzLjA5OTE3MzEgMjguNzAwMDM2LDcuMjk0NzcxNCAyOC43MDAwMzUsMTUuMDMyMTY5IHYgMTAuMjE4NzUgaCAyNi44MTI1IHYgMy40MDYyNSBoIC0yNi44MTI1IC0xMC4wNjI1IGMgLTcuNzkyNDU5LDAgLTE0LjYxNTc1ODgsNC42ODM3MTcgLTE2Ljc0OTk5OTgsMTMuNTkzNzUgLTIuNDYxODE5OTgsMTAuMjEyOTY2IC0yLjU3MTAxNTA4LDE2LjU4NjAyMyAwLDI3LjI1IDEuOTA1OTI4Myw3LjkzNzg1MiA2LjQ1NzU0MzIsMTMuNTkzNzQ4IDE0LjI0OTk5OTgsMTMuNTkzNzUgaCA5LjIxODc1IHYgLTEyLjI1IGMgMCwtOC44NDk5MDIgNy42NTcxNDQsLTE2LjY1NjI0OCAxNi43NSwtMTYuNjU2MjUgaCAyNi43ODEyNSBjIDcuNDU0OTUxLDAgMTMuNDA2MjUzLC02LjEzODE2NCAxMy40MDYyNSwtMTMuNjI1IHYgLTI1LjUzMTI1IGMgMCwtNy4yNjYzMzg2IC02LjEyOTk4LC0xMi43MjQ3NzcxIC0xMy40MDYyNSwtMTMuOTM3NDk5NyBDIDY0LjI4MTU0OCwwLjMyNzk0Mzk3IDU5LjUwMjQzOCwtMC4wMjAzNzkwMyA1NC45MTg3ODUsOS4xOTI3NDIxZS00IFogbSAtMTQuNSw4LjIxODc1MDEyNTc5IGMgMi43Njk1NDcsMCA1LjAzMTI1LDIuMjk4NjQ1NiA1LjAzMTI1LDUuMTI0OTk5NiAtMmUtNiwyLjgxNjMzNiAtMi4yNjE3MDMsNS4wOTM3NSAtNS4wMzEyNSw1LjA5Mzc1IC0yLjc3OTQ3NiwtMWUtNiAtNS4wMzEyNSwtMi4yNzc0MTUgLTUuMDMxMjUsLTUuMDkzNzUgLTEwZS03LC0yLjgyNjM1MyAyLjI1MTc3NCwtNS4xMjQ5OTk2IDUuMDMxMjUsLTUuMTI0OTk5NiB6Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI2ZmZDQzYiIgZD0ibSA4NS42Mzc1MzUsMjguNjU3MTY5IHYgMTEuOTA2MjUgYyAwLDkuMjMwNzU1IC03LjgyNTg5NSwxNi45OTk5OTkgLTE2Ljc1LDE3IGggLTI2Ljc4MTI1IGMgLTcuMzM1ODMzLDAgLTEzLjQwNjI0OSw2LjI3ODQ4MyAtMTMuNDA2MjUsMTMuNjI1IHYgMjUuNTMxMjQ3IGMgMCw3LjI2NjM0NCA2LjMxODU4OCwxMS41NDAzMjQgMTMuNDA2MjUsMTMuNjI1MDA0IDguNDg3MzMxLDIuNDk1NjEgMTYuNjI2MjM3LDIuOTQ2NjMgMjYuNzgxMjUsMCA2Ljc1MDE1NSwtMS45NTQzOSAxMy40MDYyNTMsLTUuODg3NjEgMTMuNDA2MjUsLTEzLjYyNTAwNCBWIDg2LjUwMDkxOSBoIC0yNi43ODEyNSB2IC0zLjQwNjI1IGggMjYuNzgxMjUgMTMuNDA2MjU0IGMgNy43OTI0NjEsMCAxMC42OTYyNTEsLTUuNDM1NDA4IDEzLjQwNjI0MSwtMTMuNTkzNzUgMi43OTkzMywtOC4zOTg4ODYgMi42ODAyMiwtMTYuNDc1Nzc2IDAsLTI3LjI1IC0xLjkyNTc4LC03Ljc1NzQ0MSAtNS42MDM4NywtMTMuNTkzNzUgLTEzLjQwNjI0MSwtMTMuNTkzNzUgeiBtIC0xNS4wNjI1LDY0LjY1NjI1IGMgMi43Nzk0NzgsM2UtNiA1LjAzMTI1LDIuMjc3NDE3IDUuMDMxMjUsNS4wOTM3NDcgLTJlLTYsMi44MjYzNTQgLTIuMjUxNzc1LDUuMTI1MDA0IC01LjAzMTI1LDUuMTI1MDA0IC0yLjc2OTU1LDAgLTUuMDMxMjUsLTIuMjk4NjUgLTUuMDMxMjUsLTUuMTI1MDA0IDJlLTYsLTIuODE2MzMgMi4yNjE2OTcsLTUuMDkzNzQ3IDUuMDMxMjUsLTUuMDkzNzQ3IHoiLz4KPC9zdmc+Cg==);
  --jp-icon-r-kernel: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMjE5NkYzIiBkPSJNNC40IDIuNWMxLjItLjEgMi45LS4zIDQuOS0uMyAyLjUgMCA0LjEuNCA1LjIgMS4zIDEgLjcgMS41IDEuOSAxLjUgMy41IDAgMi0xLjQgMy41LTIuOSA0LjEgMS4yLjQgMS43IDEuNiAyLjIgMyAuNiAxLjkgMSAzLjkgMS4zIDQuNmgtMy44Yy0uMy0uNC0uOC0xLjctMS4yLTMuN3MtMS4yLTIuNi0yLjYtMi42aC0uOXY2LjRINC40VjIuNXptMy43IDYuOWgxLjRjMS45IDAgMi45LS45IDIuOS0yLjNzLTEtMi4zLTIuOC0yLjNjLS43IDAtMS4zIDAtMS42LjJ2NC41aC4xdi0uMXoiLz4KPC9zdmc+Cg==);
  --jp-icon-react: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMTUwIDE1MCA1NDEuOSAyOTUuMyI+CiAgPGcgY2xhc3M9ImpwLWljb24tYnJhbmQyIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzYxREFGQiI+CiAgICA8cGF0aCBkPSJNNjY2LjMgMjk2LjVjMC0zMi41LTQwLjctNjMuMy0xMDMuMS04Mi40IDE0LjQtNjMuNiA4LTExNC4yLTIwLjItMTMwLjQtNi41LTMuOC0xNC4xLTUuNi0yMi40LTUuNnYyMi4zYzQuNiAwIDguMy45IDExLjQgMi42IDEzLjYgNy44IDE5LjUgMzcuNSAxNC45IDc1LjctMS4xIDkuNC0yLjkgMTkuMy01LjEgMjkuNC0xOS42LTQuOC00MS04LjUtNjMuNS0xMC45LTEzLjUtMTguNS0yNy41LTM1LjMtNDEuNi01MCAzMi42LTMwLjMgNjMuMi00Ni45IDg0LTQ2LjlWNzhjLTI3LjUgMC02My41IDE5LjYtOTkuOSA1My42LTM2LjQtMzMuOC03Mi40LTUzLjItOTkuOS01My4ydjIyLjNjMjAuNyAwIDUxLjQgMTYuNSA4NCA0Ni42LTE0IDE0LjctMjggMzEuNC00MS4zIDQ5LjktMjIuNiAyLjQtNDQgNi4xLTYzLjYgMTEtMi4zLTEwLTQtMTkuNy01LjItMjktNC43LTM4LjIgMS4xLTY3LjkgMTQuNi03NS44IDMtMS44IDYuOS0yLjYgMTEuNS0yLjZWNzguNWMtOC40IDAtMTYgMS44LTIyLjYgNS42LTI4LjEgMTYuMi0zNC40IDY2LjctMTkuOSAxMzAuMS02Mi4yIDE5LjItMTAyLjcgNDkuOS0xMDIuNyA4Mi4zIDAgMzIuNSA0MC43IDYzLjMgMTAzLjEgODIuNC0xNC40IDYzLjYtOCAxMTQuMiAyMC4yIDEzMC40IDYuNSAzLjggMTQuMSA1LjYgMjIuNSA1LjYgMjcuNSAwIDYzLjUtMTkuNiA5OS45LTUzLjYgMzYuNCAzMy44IDcyLjQgNTMuMiA5OS45IDUzLjIgOC40IDAgMTYtMS44IDIyLjYtNS42IDI4LjEtMTYuMiAzNC40LTY2LjcgMTkuOS0xMzAuMSA2Mi0xOS4xIDEwMi41LTQ5LjkgMTAyLjUtODIuM3ptLTEzMC4yLTY2LjdjLTMuNyAxMi45LTguMyAyNi4yLTEzLjUgMzkuNS00LjEtOC04LjQtMTYtMTMuMS0yNC00LjYtOC05LjUtMTUuOC0xNC40LTIzLjQgMTQuMiAyLjEgMjcuOSA0LjcgNDEgNy45em0tNDUuOCAxMDYuNWMtNy44IDEzLjUtMTUuOCAyNi4zLTI0LjEgMzguMi0xNC45IDEuMy0zMCAyLTQ1LjIgMi0xNS4xIDAtMzAuMi0uNy00NS0xLjktOC4zLTExLjktMTYuNC0yNC42LTI0LjItMzgtNy42LTEzLjEtMTQuNS0yNi40LTIwLjgtMzkuOCA2LjItMTMuNCAxMy4yLTI2LjggMjAuNy0zOS45IDcuOC0xMy41IDE1LjgtMjYuMyAyNC4xLTM4LjIgMTQuOS0xLjMgMzAtMiA0NS4yLTIgMTUuMSAwIDMwLjIuNyA0NSAxLjkgOC4zIDExLjkgMTYuNCAyNC42IDI0LjIgMzggNy42IDEzLjEgMTQuNSAyNi40IDIwLjggMzkuOC02LjMgMTMuNC0xMy4yIDI2LjgtMjAuNyAzOS45em0zMi4zLTEzYzUuNCAxMy40IDEwIDI2LjggMTMuOCAzOS44LTEzLjEgMy4yLTI2LjkgNS45LTQxLjIgOCA0LjktNy43IDkuOC0xNS42IDE0LjQtMjMuNyA0LjYtOCA4LjktMTYuMSAxMy0yNC4xek00MjEuMiA0MzBjLTkuMy05LjYtMTguNi0yMC4zLTI3LjgtMzIgOSAuNCAxOC4yLjcgMjcuNS43IDkuNCAwIDE4LjctLjIgMjcuOC0uNy05IDExLjctMTguMyAyMi40LTI3LjUgMzJ6bS03NC40LTU4LjljLTE0LjItMi4xLTI3LjktNC43LTQxLTcuOSAzLjctMTIuOSA4LjMtMjYuMiAxMy41LTM5LjUgNC4xIDggOC40IDE2IDEzLjEgMjQgNC43IDggOS41IDE1LjggMTQuNCAyMy40ek00MjAuNyAxNjNjOS4zIDkuNiAxOC42IDIwLjMgMjcuOCAzMi05LS40LTE4LjItLjctMjcuNS0uNy05LjQgMC0xOC43LjItMjcuOC43IDktMTEuNyAxOC4zLTIyLjQgMjcuNS0zMnptLTc0IDU4LjljLTQuOSA3LjctOS44IDE1LjYtMTQuNCAyMy43LTQuNiA4LTguOSAxNi0xMyAyNC01LjQtMTMuNC0xMC0yNi44LTEzLjgtMzkuOCAxMy4xLTMuMSAyNi45LTUuOCA0MS4yLTcuOXptLTkwLjUgMTI1LjJjLTM1LjQtMTUuMS01OC4zLTM0LjktNTguMy01MC42IDAtMTUuNyAyMi45LTM1LjYgNTguMy01MC42IDguNi0zLjcgMTgtNyAyNy43LTEwLjEgNS43IDE5LjYgMTMuMiA0MCAyMi41IDYwLjktOS4yIDIwLjgtMTYuNiA0MS4xLTIyLjIgNjAuNi05LjktMy4xLTE5LjMtNi41LTI4LTEwLjJ6TTMxMCA0OTBjLTEzLjYtNy44LTE5LjUtMzcuNS0xNC45LTc1LjcgMS4xLTkuNCAyLjktMTkuMyA1LjEtMjkuNCAxOS42IDQuOCA0MSA4LjUgNjMuNSAxMC45IDEzLjUgMTguNSAyNy41IDM1LjMgNDEuNiA1MC0zMi42IDMwLjMtNjMuMiA0Ni45LTg0IDQ2LjktNC41LS4xLTguMy0xLTExLjMtMi43em0yMzcuMi03Ni4yYzQuNyAzOC4yLTEuMSA2Ny45LTE0LjYgNzUuOC0zIDEuOC02LjkgMi42LTExLjUgMi42LTIwLjcgMC01MS40LTE2LjUtODQtNDYuNiAxNC0xNC43IDI4LTMxLjQgNDEuMy00OS45IDIyLjYtMi40IDQ0LTYuMSA2My42LTExIDIuMyAxMC4xIDQuMSAxOS44IDUuMiAyOS4xem0zOC41LTY2LjdjLTguNiAzLjctMTggNy0yNy43IDEwLjEtNS43LTE5LjYtMTMuMi00MC0yMi41LTYwLjkgOS4yLTIwLjggMTYuNi00MS4xIDIyLjItNjAuNiA5LjkgMy4xIDE5LjMgNi41IDI4LjEgMTAuMiAzNS40IDE1LjEgNTguMyAzNC45IDU4LjMgNTAuNi0uMSAxNS43LTIzIDM1LjYtNTguNCA1MC42ek0zMjAuOCA3OC40eiIvPgogICAgPGNpcmNsZSBjeD0iNDIwLjkiIGN5PSIyOTYuNSIgcj0iNDUuNyIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-redo: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgICA8cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZD0iTTE4LjQgMTAuNkMxNi41NSA4Ljk5IDE0LjE1IDggMTEuNSA4Yy00LjY1IDAtOC41OCAzLjAzLTkuOTYgNy4yMkwzLjkgMTZjMS4wNS0zLjE5IDQuMDUtNS41IDcuNi01LjUgMS45NSAwIDMuNzMuNzIgNS4xMiAxLjg4TDEzIDE2aDlWN2wtMy42IDMuNnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-refresh: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTkgMTMuNWMtMi40OSAwLTQuNS0yLjAxLTQuNS00LjVTNi41MSA0LjUgOSA0LjVjMS4yNCAwIDIuMzYuNTIgMy4xNyAxLjMzTDEwIDhoNVYzbC0xLjc2IDEuNzZDMTIuMTUgMy42OCAxMC42NiAzIDkgMyA1LjY5IDMgMy4wMSA1LjY5IDMuMDEgOVM1LjY5IDE1IDkgMTVjMi45NyAwIDUuNDMtMi4xNiA1LjktNWgtMS41MmMtLjQ2IDItMi4yNCAzLjUtNC4zOCAzLjV6Ii8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-regex: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0MTQxNDEiPgogICAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbi1hY2NlbnQyIiBmaWxsPSIjRkZGIj4KICAgIDxjaXJjbGUgY2xhc3M9InN0MiIgY3g9IjUuNSIgY3k9IjE0LjUiIHI9IjEuNSIvPgogICAgPHJlY3QgeD0iMTIiIHk9IjQiIGNsYXNzPSJzdDIiIHdpZHRoPSIxIiBoZWlnaHQ9IjgiLz4KICAgIDxyZWN0IHg9IjguNSIgeT0iNy41IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjg2NiAtMC41IDAuNSAwLjg2NiAtMi4zMjU1IDcuMzIxOSkiIGNsYXNzPSJzdDIiIHdpZHRoPSI4IiBoZWlnaHQ9IjEiLz4KICAgIDxyZWN0IHg9IjEyIiB5PSI0IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjUgLTAuODY2IDAuODY2IDAuNSAtMC42Nzc5IDE0LjgyNTIpIiBjbGFzcz0ic3QyIiB3aWR0aD0iMSIgaGVpZ2h0PSI4Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-run: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTggNXYxNGwxMS03eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-running: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUxMiA1MTIiPgogIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICA8cGF0aCBkPSJNMjU2IDhDMTE5IDggOCAxMTkgOCAyNTZzMTExIDI0OCAyNDggMjQ4IDI0OC0xMTEgMjQ4LTI0OFMzOTMgOCAyNTYgOHptOTYgMzI4YzAgOC44LTcuMiAxNi0xNiAxNkgxNzZjLTguOCAwLTE2LTcuMi0xNi0xNlYxNzZjMC04LjggNy4yLTE2IDE2LTE2aDE2MGM4LjggMCAxNiA3LjIgMTYgMTZ2MTYweiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-save: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTE3IDNINWMtMS4xMSAwLTIgLjktMiAydjE0YzAgMS4xLjg5IDIgMiAyaDE0YzEuMSAwIDItLjkgMi0yVjdsLTQtNHptLTUgMTZjLTEuNjYgMC0zLTEuMzQtMy0zczEuMzQtMyAzLTMgMyAxLjM0IDMgMy0xLjM0IDMtMyAzem0zLTEwSDVWNWgxMHY0eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-search: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjEsMTAuOWgtMC43bC0wLjItMC4yYzAuOC0wLjksMS4zLTIuMiwxLjMtMy41YzAtMy0yLjQtNS40LTUuNC01LjRTMS44LDQuMiwxLjgsNy4xczIuNCw1LjQsNS40LDUuNCBjMS4zLDAsMi41LTAuNSwzLjUtMS4zbDAuMiwwLjJ2MC43bDQuMSw0LjFsMS4yLTEuMkwxMi4xLDEwLjl6IE03LjEsMTAuOWMtMi4xLDAtMy43LTEuNy0zLjctMy43czEuNy0zLjcsMy43LTMuN3MzLjcsMS43LDMuNywzLjcgUzkuMiwxMC45LDcuMSwxMC45eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-settings: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkuNDMgMTIuOThjLjA0LS4zMi4wNy0uNjQuMDctLjk4cy0uMDMtLjY2LS4wNy0uOThsMi4xMS0xLjY1Yy4xOS0uMTUuMjQtLjQyLjEyLS42NGwtMi0zLjQ2Yy0uMTItLjIyLS4zOS0uMy0uNjEtLjIybC0yLjQ5IDFjLS41Mi0uNC0xLjA4LS43My0xLjY5LS45OGwtLjM4LTIuNjVBLjQ4OC40ODggMCAwMDE0IDJoLTRjLS4yNSAwLS40Ni4xOC0uNDkuNDJsLS4zOCAyLjY1Yy0uNjEuMjUtMS4xNy41OS0xLjY5Ljk4bC0yLjQ5LTFjLS4yMy0uMDktLjQ5IDAtLjYxLjIybC0yIDMuNDZjLS4xMy4yMi0uMDcuNDkuMTIuNjRsMi4xMSAxLjY1Yy0uMDQuMzItLjA3LjY1LS4wNy45OHMuMDMuNjYuMDcuOThsLTIuMTEgMS42NWMtLjE5LjE1LS4yNC40Mi0uMTIuNjRsMiAzLjQ2Yy4xMi4yMi4zOS4zLjYxLjIybDIuNDktMWMuNTIuNCAxLjA4LjczIDEuNjkuOThsLjM4IDIuNjVjLjAzLjI0LjI0LjQyLjQ5LjQyaDRjLjI1IDAgLjQ2LS4xOC40OS0uNDJsLjM4LTIuNjVjLjYxLS4yNSAxLjE3LS41OSAxLjY5LS45OGwyLjQ5IDFjLjIzLjA5LjQ5IDAgLjYxLS4yMmwyLTMuNDZjLjEyLS4yMi4wNy0uNDktLjEyLS42NGwtMi4xMS0xLjY1ek0xMiAxNS41Yy0xLjkzIDAtMy41LTEuNTctMy41LTMuNXMxLjU3LTMuNSAzLjUtMy41IDMuNSAxLjU3IDMuNSAzLjUtMS41NyAzLjUtMy41IDMuNXoiLz4KPC9zdmc+Cg==);
  --jp-icon-share: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTSAxOCAyIEMgMTYuMzU0OTkgMiAxNSAzLjM1NDk5MDQgMTUgNSBDIDE1IDUuMTkwOTUyOSAxNS4wMjE3OTEgNS4zNzcxMjI0IDE1LjA1NjY0MSA1LjU1ODU5MzggTCA3LjkyMTg3NSA5LjcyMDcwMzEgQyA3LjM5ODUzOTkgOS4yNzc4NTM5IDYuNzMyMDc3MSA5IDYgOSBDIDQuMzU0OTkwNCA5IDMgMTAuMzU0OTkgMyAxMiBDIDMgMTMuNjQ1MDEgNC4zNTQ5OTA0IDE1IDYgMTUgQyA2LjczMjA3NzEgMTUgNy4zOTg1Mzk5IDE0LjcyMjE0NiA3LjkyMTg3NSAxNC4yNzkyOTcgTCAxNS4wNTY2NDEgMTguNDM5NDUzIEMgMTUuMDIxNTU1IDE4LjYyMTUxNCAxNSAxOC44MDgzODYgMTUgMTkgQyAxNSAyMC42NDUwMSAxNi4zNTQ5OSAyMiAxOCAyMiBDIDE5LjY0NTAxIDIyIDIxIDIwLjY0NTAxIDIxIDE5IEMgMjEgMTcuMzU0OTkgMTkuNjQ1MDEgMTYgMTggMTYgQyAxNy4yNjc0OCAxNiAxNi42MDE1OTMgMTYuMjc5MzI4IDE2LjA3ODEyNSAxNi43MjI2NTYgTCA4Ljk0MzM1OTQgMTIuNTU4NTk0IEMgOC45NzgyMDk1IDEyLjM3NzEyMiA5IDEyLjE5MDk1MyA5IDEyIEMgOSAxMS44MDkwNDcgOC45NzgyMDk1IDExLjYyMjg3OCA4Ljk0MzM1OTQgMTEuNDQxNDA2IEwgMTYuMDc4MTI1IDcuMjc5Mjk2OSBDIDE2LjYwMTQ2IDcuNzIyMTQ2MSAxNy4yNjc5MjMgOCAxOCA4IEMgMTkuNjQ1MDEgOCAyMSA2LjY0NTAwOTYgMjEgNSBDIDIxIDMuMzU0OTkwNCAxOS42NDUwMSAyIDE4IDIgeiBNIDE4IDQgQyAxOC41NjQxMjkgNCAxOSA0LjQzNTg3MDYgMTkgNSBDIDE5IDUuNTY0MTI5NCAxOC41NjQxMjkgNiAxOCA2IEMgMTcuNDM1ODcxIDYgMTcgNS41NjQxMjk0IDE3IDUgQyAxNyA0LjQzNTg3MDYgMTcuNDM1ODcxIDQgMTggNCB6IE0gNiAxMSBDIDYuNTY0MTI5NCAxMSA3IDExLjQzNTg3MSA3IDEyIEMgNyAxMi41NjQxMjkgNi41NjQxMjk0IDEzIDYgMTMgQyA1LjQzNTg3MDYgMTMgNSAxMi41NjQxMjkgNSAxMiBDIDUgMTEuNDM1ODcxIDUuNDM1ODcwNiAxMSA2IDExIHogTSAxOCAxOCBDIDE4LjU2NDEyOSAxOCAxOSAxOC40MzU4NzEgMTkgMTkgQyAxOSAxOS41NjQxMjkgMTguNTY0MTI5IDIwIDE4IDIwIEMgMTcuNDM1ODcxIDIwIDE3IDE5LjU2NDEyOSAxNyAxOSBDIDE3IDE4LjQzNTg3MSAxNy40MzU4NzEgMTggMTggMTggeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-spreadsheet: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDEganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNENBRjUwIiBkPSJNMi4yIDIuMnYxNy42aDE3LjZWMi4ySDIuMnptMTUuNCA3LjdoLTUuNVY0LjRoNS41djUuNXpNOS45IDQuNHY1LjVINC40VjQuNGg1LjV6bS01LjUgNy43aDUuNXY1LjVINC40di01LjV6bTcuNyA1LjV2LTUuNWg1LjV2NS41aC01LjV6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-stop: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik02IDZoMTJ2MTJINnoiLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-tab: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIxIDNIM2MtMS4xIDAtMiAuOS0yIDJ2MTRjMCAxLjEuOSAyIDIgMmgxOGMxLjEgMCAyLS45IDItMlY1YzAtMS4xLS45LTItMi0yem0wIDE2SDNWNWgxMHY0aDh2MTB6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-table-rows: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik0yMSw4SDNWNGgxOFY4eiBNMjEsMTBIM3Y0aDE4VjEweiBNMjEsMTZIM3Y0aDE4VjE2eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-tag: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjgiIGhlaWdodD0iMjgiIHZpZXdCb3g9IjAgMCA0MyAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTI4LjgzMzIgMTIuMzM0TDMyLjk5OTggMTYuNTAwN0wzNy4xNjY1IDEyLjMzNEgyOC44MzMyWiIvPgoJCTxwYXRoIGQ9Ik0xNi4yMDk1IDIxLjYxMDRDMTUuNjg3MyAyMi4xMjk5IDE0Ljg0NDMgMjIuMTI5OSAxNC4zMjQ4IDIxLjYxMDRMNi45ODI5IDE0LjcyNDVDNi41NzI0IDE0LjMzOTQgNi4wODMxMyAxMy42MDk4IDYuMDQ3ODYgMTMuMDQ4MkM1Ljk1MzQ3IDExLjUyODggNi4wMjAwMiA4LjYxOTQ0IDYuMDY2MjEgNy4wNzY5NUM2LjA4MjgxIDYuNTE0NzcgNi41NTU0OCA2LjA0MzQ3IDcuMTE4MDQgNi4wMzA1NUM5LjA4ODYzIDUuOTg0NzMgMTMuMjYzOCA1LjkzNTc5IDEzLjY1MTggNi4zMjQyNUwyMS43MzY5IDEzLjYzOUMyMi4yNTYgMTQuMTU4NSAyMS43ODUxIDE1LjQ3MjQgMjEuMjYyIDE1Ljk5NDZMMTYuMjA5NSAyMS42MTA0Wk05Ljc3NTg1IDguMjY1QzkuMzM1NTEgNy44MjU2NiA4LjYyMzUxIDcuODI1NjYgOC4xODI4IDguMjY1QzcuNzQzNDYgOC43MDU3MSA3Ljc0MzQ2IDkuNDE3MzMgOC4xODI4IDkuODU2NjdDOC42MjM4MiAxMC4yOTY0IDkuMzM1ODIgMTAuMjk2NCA5Ljc3NTg1IDkuODU2NjdDMTAuMjE1NiA5LjQxNzMzIDEwLjIxNTYgOC43MDUzMyA5Ljc3NTg1IDguMjY1WiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-terminal: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0IiA+CiAgICA8cmVjdCBjbGFzcz0ianAtdGVybWluYWwtaWNvbi1iYWNrZ3JvdW5kLWNvbG9yIGpwLWljb24tc2VsZWN0YWJsZSIgd2lkdGg9IjIwIiBoZWlnaHQ9IjIwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyIDIpIiBmaWxsPSIjMzMzMzMzIi8+CiAgICA8cGF0aCBjbGFzcz0ianAtdGVybWluYWwtaWNvbi1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUtaW52ZXJzZSIgZD0iTTUuMDU2NjQgOC43NjE3MkM1LjA1NjY0IDguNTk3NjYgNS4wMzEyNSA4LjQ1MzEyIDQuOTgwNDcgOC4zMjgxMkM0LjkzMzU5IDguMTk5MjIgNC44NTU0NyA4LjA4MjAzIDQuNzQ2MDkgNy45NzY1NkM0LjY0MDYyIDcuODcxMDkgNC41IDcuNzc1MzkgNC4zMjQyMiA3LjY4OTQ1QzQuMTUyMzQgNy41OTk2MSAzLjk0MzM2IDcuNTExNzIgMy42OTcyNyA3LjQyNTc4QzMuMzAyNzMgNy4yODUxNiAyLjk0MzM2IDcuMTM2NzIgMi42MTkxNCA2Ljk4MDQ3QzIuMjk0OTIgNi44MjQyMiAyLjAxNzU4IDYuNjQyNTggMS43ODcxMSA2LjQzNTU1QzEuNTYwNTUgNi4yMjg1MiAxLjM4NDc3IDUuOTg4MjggMS4yNTk3NyA1LjcxNDg0QzEuMTM0NzcgNS40Mzc1IDEuMDcyMjcgNS4xMDkzOCAxLjA3MjI3IDQuNzMwNDdDMS4wNzIyNyA0LjM5ODQ0IDEuMTI4OTEgNC4wOTU3IDEuMjQyMTkgMy44MjIyN0MxLjM1NTQ3IDMuNTQ0OTIgMS41MTU2MiAzLjMwNDY5IDEuNzIyNjYgMy4xMDE1NkMxLjkyOTY5IDIuODk4NDQgMi4xNzk2OSAyLjczNDM3IDIuNDcyNjYgMi42MDkzOEMyLjc2NTYyIDIuNDg0MzggMy4wOTE4IDIuNDA0MyAzLjQ1MTE3IDIuMzY5MTRWMS4xMDkzOEg0LjM4ODY3VjIuMzgwODZDNC43NDAyMyAyLjQyNzczIDUuMDU2NjQgMi41MjM0NCA1LjMzNzg5IDIuNjY3OTdDNS42MTkxNCAyLjgxMjUgNS44NTc0MiAzLjAwMTk1IDYuMDUyNzMgMy4yMzYzM0M2LjI1MTk1IDMuNDY2OCA2LjQwNDMgMy43NDAyMyA2LjUwOTc3IDQuMDU2NjRDNi42MTkxNCA0LjM2OTE0IDYuNjczODMgNC43MjA3IDYuNjczODMgNS4xMTEzM0g1LjA0NDkyQzUuMDQ0OTIgNC42Mzg2NyA0LjkzNzUgNC4yODEyNSA0LjcyMjY2IDQuMDM5MDZDNC41MDc4MSAzLjc5Mjk3IDQuMjE2OCAzLjY2OTkyIDMuODQ5NjEgMy42Njk5MkMzLjY1MDM5IDMuNjY5OTIgMy40NzY1NiAzLjY5NzI3IDMuMzI4MTIgMy43NTE5NUMzLjE4MzU5IDMuODAyNzMgMy4wNjQ0NSAzLjg3Njk1IDIuOTcwNyAzLjk3NDYxQzIuODc2OTUgNC4wNjgzNiAyLjgwNjY0IDQuMTc5NjkgMi43NTk3NyA0LjMwODU5QzIuNzE2OCA0LjQzNzUgMi42OTUzMSA0LjU3ODEyIDIuNjk1MzEgNC43MzA0N0MyLjY5NTMxIDQuODgyODEgMi43MTY4IDUuMDE5NTMgMi43NTk3NyA1LjE0MDYyQzIuODA2NjQgNS4yNTc4MSAyLjg4MjgxIDUuMzY3MTkgMi45ODgyOCA1LjQ2ODc1QzMuMDk3NjYgNS41NzAzMSAzLjI0MDIzIDUuNjY3OTcgMy40MTYwMiA1Ljc2MTcyQzMuNTkxOCA1Ljg1MTU2IDMuODEwNTUgNS45NDMzNiA0LjA3MjI3IDYuMDM3MTFDNC40NjY4IDYuMTg1NTUgNC44MjQyMiA2LjMzOTg0IDUuMTQ0NTMgNi41QzUuNDY0ODQgNi42NTYyNSA1LjczODI4IDYuODM5ODQgNS45NjQ4NCA3LjA1MDc4QzYuMTk1MzEgNy4yNTc4MSA2LjM3MTA5IDcuNSA2LjQ5MjE5IDcuNzc3MzRDNi42MTcxOSA4LjA1MDc4IDYuNjc5NjkgOC4zNzUgNi42Nzk2OSA4Ljc1QzYuNjc5NjkgOS4wOTM3NSA2LjYyMzA1IDkuNDA0MyA2LjUwOTc3IDkuNjgxNjRDNi4zOTY0OCA5Ljk1NTA4IDYuMjM0MzggMTAuMTkxNCA2LjAyMzQ0IDEwLjM5MDZDNS44MTI1IDEwLjU4OTggNS41NTg1OSAxMC43NSA1LjI2MTcyIDEwLjg3MTFDNC45NjQ4NCAxMC45ODgzIDQuNjMyODEgMTEuMDY0NSA0LjI2NTYyIDExLjA5OTZWMTIuMjQ4SDMuMzMzOThWMTEuMDk5NkMzLjAwMTk1IDExLjA2ODQgMi42Nzk2OSAxMC45OTYxIDIuMzY3MTkgMTAuODgyOEMyLjA1NDY5IDEwLjc2NTYgMS43NzczNCAxMC41OTc3IDEuNTM1MTYgMTAuMzc4OUMxLjI5Njg4IDEwLjE2MDIgMS4xMDU0NyA5Ljg4NDc3IDAuOTYwOTM4IDkuNTUyNzNDMC44MTY0MDYgOS4yMTY4IDAuNzQ0MTQxIDguODE0NDUgMC43NDQxNDEgOC4zNDU3SDIuMzc4OTFDMi4zNzg5MSA4LjYyNjk1IDIuNDE5OTIgOC44NjMyOCAyLjUwMTk1IDkuMDU0NjlDMi41ODM5OCA5LjI0MjE5IDIuNjg5NDUgOS4zOTI1OCAyLjgxODM2IDkuNTA1ODZDMi45NTExNyA5LjYxNTIzIDMuMTAxNTYgOS42OTMzNiAzLjI2OTUzIDkuNzQwMjNDMy40Mzc1IDkuNzg3MTEgMy42MDkzOCA5LjgxMDU1IDMuNzg1MTYgOS44MTA1NUM0LjIwMzEyIDkuODEwNTUgNC41MTk1MyA5LjcxMjg5IDQuNzM0MzggOS41MTc1OEM0Ljk0OTIyIDkuMzIyMjcgNS4wNTY2NCA5LjA3MDMxIDUuMDU2NjQgOC43NjE3MlpNMTMuNDE4IDEyLjI3MTVIOC4wNzQyMlYxMUgxMy40MThWMTIuMjcxNVoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMuOTUyNjQgNikiIGZpbGw9IndoaXRlIi8+Cjwvc3ZnPgo=);
  --jp-icon-text-editor: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtdGV4dC1lZGl0b3ItaWNvbi1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xNSAxNUgzdjJoMTJ2LTJ6bTAtOEgzdjJoMTJWN3pNMyAxM2gxOHYtMkgzdjJ6bTAgOGgxOHYtMkgzdjJ6TTMgM3YyaDE4VjNIM3oiLz4KPC9zdmc+Cg==);
  --jp-icon-toc: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik03LDVIMjFWN0g3VjVNNywxM1YxMUgyMVYxM0g3TTQsNC41QTEuNSwxLjUgMCAwLDEgNS41LDZBMS41LDEuNSAwIDAsMSA0LDcuNUExLjUsMS41IDAgMCwxIDIuNSw2QTEuNSwxLjUgMCAwLDEgNCw0LjVNNCwxMC41QTEuNSwxLjUgMCAwLDEgNS41LDEyQTEuNSwxLjUgMCAwLDEgNCwxMy41QTEuNSwxLjUgMCAwLDEgMi41LDEyQTEuNSwxLjUgMCAwLDEgNCwxMC41TTcsMTlWMTdIMjFWMTlIN000LDE2LjVBMS41LDEuNSAwIDAsMSA1LjUsMThBMS41LDEuNSAwIDAsMSA0LDE5LjVBMS41LDEuNSAwIDAsMSAyLjUsMThBMS41LDEuNSAwIDAsMSA0LDE2LjVaIiAvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-tree-view: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik0yMiAxMVYzaC03djNIOVYzSDJ2OGg3VjhoMnYxMGg0djNoN3YtOGgtN3YzaC0yVjhoMnYzeiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-trusted: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI1Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDIgMykiIGQ9Ik0xLjg2MDk0IDExLjQ0MDlDMC44MjY0NDggOC43NzAyNyAwLjg2Mzc3OSA2LjA1NzY0IDEuMjQ5MDcgNC4xOTkzMkMyLjQ4MjA2IDMuOTMzNDcgNC4wODA2OCAzLjQwMzQ3IDUuNjAxMDIgMi44NDQ5QzcuMjM1NDkgMi4yNDQ0IDguODU2NjYgMS41ODE1IDkuOTg3NiAxLjA5NTM5QzExLjA1OTcgMS41ODM0MSAxMi42MDk0IDIuMjQ0NCAxNC4yMTggMi44NDMzOUMxNS43NTAzIDMuNDEzOTQgMTcuMzk5NSAzLjk1MjU4IDE4Ljc1MzkgNC4yMTM4NUMxOS4xMzY0IDYuMDcxNzcgMTkuMTcwOSA4Ljc3NzIyIDE4LjEzOSAxMS40NDA5QzE3LjAzMDMgMTQuMzAzMiAxNC42NjY4IDE3LjE4NDQgOS45OTk5OSAxOC45MzU0QzUuMzMzMiAxNy4xODQ0IDIuOTY5NjggMTQuMzAzMiAxLjg2MDk0IDExLjQ0MDlaIi8+CiAgICA8cGF0aCBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiMzMzMzMzMiIHN0cm9rZT0iIzMzMzMzMyIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOCA5Ljg2NzE5KSIgZD0iTTIuODYwMTUgNC44NjUzNUwwLjcyNjU0OSAyLjk5OTU5TDAgMy42MzA0NUwyLjg2MDE1IDYuMTMxNTdMOCAwLjYzMDg3Mkw3LjI3ODU3IDBMMi44NjAxNSA0Ljg2NTM1WiIvPgo8L3N2Zz4K);
  --jp-icon-undo: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjUgOGMtMi42NSAwLTUuMDUuOTktNi45IDIuNkwyIDd2OWg5bC0zLjYyLTMuNjJjMS4zOS0xLjE2IDMuMTYtMS44OCA1LjEyLTEuODggMy41NCAwIDYuNTUgMi4zMSA3LjYgNS41bDIuMzctLjc4QzIxLjA4IDExLjAzIDE3LjE1IDggMTIuNSA4eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-user: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE2IDdhNCA0IDAgMTEtOCAwIDQgNCAwIDAxOCAwek0xMiAxNGE3IDcgMCAwMC03IDdoMTRhNyA3IDAgMDAtNy03eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-users: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZlcnNpb249IjEuMSIgdmlld0JveD0iMCAwIDM2IDI0IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgogPGcgY2xhc3M9ImpwLWljb24zIiB0cmFuc2Zvcm09Im1hdHJpeCgxLjczMjcgMCAwIDEuNzMyNyAtMy42MjgyIC4wOTk1NzcpIiBmaWxsPSIjNjE2MTYxIj4KICA8cGF0aCB0cmFuc2Zvcm09Im1hdHJpeCgxLjUsMCwwLDEuNSwwLC02KSIgZD0ibTEyLjE4NiA3LjUwOThjLTEuMDUzNSAwLTEuOTc1NyAwLjU2NjUtMi40Nzg1IDEuNDEwMiAwLjc1MDYxIDAuMzEyNzcgMS4zOTc0IDAuODI2NDggMS44NzMgMS40NzI3aDMuNDg2M2MwLTEuNTkyLTEuMjg4OS0yLjg4MjgtMi44ODA5LTIuODgyOHoiLz4KICA8cGF0aCBkPSJtMjAuNDY1IDIuMzg5NWEyLjE4ODUgMi4xODg1IDAgMCAxLTIuMTg4NCAyLjE4ODUgMi4xODg1IDIuMTg4NSAwIDAgMS0yLjE4ODUtMi4xODg1IDIuMTg4NSAyLjE4ODUgMCAwIDEgMi4xODg1LTIuMTg4NSAyLjE4ODUgMi4xODg1IDAgMCAxIDIuMTg4NCAyLjE4ODV6Ii8+CiAgPHBhdGggdHJhbnNmb3JtPSJtYXRyaXgoMS41LDAsMCwxLjUsMCwtNikiIGQ9Im0zLjU4OTggOC40MjE5Yy0xLjExMjYgMC0yLjAxMzcgMC45MDExMS0yLjAxMzcgMi4wMTM3aDIuODE0NWMwLjI2Nzk3LTAuMzczMDkgMC41OTA3LTAuNzA0MzUgMC45NTg5OC0wLjk3ODUyLTAuMzQ0MzMtMC42MTY4OC0xLjAwMzEtMS4wMzUyLTEuNzU5OC0xLjAzNTJ6Ii8+CiAgPHBhdGggZD0ibTYuOTE1NCA0LjYyM2ExLjUyOTQgMS41Mjk0IDAgMCAxLTEuNTI5NCAxLjUyOTQgMS41Mjk0IDEuNTI5NCAwIDAgMS0xLjUyOTQtMS41Mjk0IDEuNTI5NCAxLjUyOTQgMCAwIDEgMS41Mjk0LTEuNTI5NCAxLjUyOTQgMS41Mjk0IDAgMCAxIDEuNTI5NCAxLjUyOTR6Ii8+CiAgPHBhdGggZD0ibTYuMTM1IDEzLjUzNWMwLTMuMjM5MiAyLjYyNTktNS44NjUgNS44NjUtNS44NjUgMy4yMzkyIDAgNS44NjUgMi42MjU5IDUuODY1IDUuODY1eiIvPgogIDxjaXJjbGUgY3g9IjEyIiBjeT0iMy43Njg1IiByPSIyLjk2ODUiLz4KIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-vega: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbjEganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMjEyMTIxIj4KICAgIDxwYXRoIGQ9Ik0xMC42IDUuNGwyLjItMy4ySDIuMnY3LjNsNC02LjZ6Ii8+CiAgICA8cGF0aCBkPSJNMTUuOCAyLjJsLTQuNCA2LjZMNyA2LjNsLTQuOCA4djUuNWgxNy42VjIuMmgtNHptLTcgMTUuNEg1LjV2LTQuNGgzLjN2NC40em00LjQgMEg5LjhWOS44aDMuNHY3Ljh6bTQuNCAwaC0zLjRWNi41aDMuNHYxMS4xeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-word: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KIDxnIGNsYXNzPSJqcC1pY29uMiIgZmlsbD0iIzQxNDE0MSI+CiAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiA8L2c+CiA8ZyBjbGFzcz0ianAtaWNvbi1hY2NlbnQyIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSguNDMgLjA0MDEpIiBmaWxsPSIjZmZmIj4KICA8cGF0aCBkPSJtNC4xNCA4Ljc2cTAuMDY4Mi0xLjg5IDIuNDItMS44OSAxLjE2IDAgMS42OCAwLjQyIDAuNTY3IDAuNDEgMC41NjcgMS4xNnYzLjQ3cTAgMC40NjIgMC41MTQgMC40NjIgMC4xMDMgMCAwLjItMC4wMjMxdjAuNzE0cS0wLjM5OSAwLjEwMy0wLjY1MSAwLjEwMy0wLjQ1MiAwLTAuNjkzLTAuMjItMC4yMzEtMC4yLTAuMjg0LTAuNjYyLTAuOTU2IDAuODcyLTIgMC44NzItMC45MDMgMC0xLjQ3LTAuNDcyLTAuNTI1LTAuNDcyLTAuNTI1LTEuMjYgMC0wLjI2MiAwLjA0NTItMC40NzIgMC4wNTY3LTAuMjIgMC4xMTYtMC4zNzggMC4wNjgyLTAuMTY4IDAuMjMxLTAuMzA0IDAuMTU4LTAuMTQ3IDAuMjYyLTAuMjQyIDAuMTE2LTAuMDkxNCAwLjM2OC0wLjE2OCAwLjI2Mi0wLjA5MTQgMC4zOTktMC4xMjYgMC4xMzYtMC4wNDUyIDAuNDcyLTAuMTAzIDAuMzM2LTAuMDU3OCAwLjUwNC0wLjA3OTggMC4xNTgtMC4wMjMxIDAuNTY3LTAuMDc5OCAwLjU1Ni0wLjA2ODIgMC43NzctMC4yMjEgMC4yMi0wLjE1MiAwLjIyLTAuNDQxdi0wLjI1MnEwLTAuNDMtMC4zNTctMC42NjItMC4zMzYtMC4yMzEtMC45NzYtMC4yMzEtMC42NjIgMC0wLjk5OCAwLjI2Mi0wLjMzNiAwLjI1Mi0wLjM5OSAwLjc5OHptMS44OSAzLjY4cTAuNzg4IDAgMS4yNi0wLjQxIDAuNTA0LTAuNDIgMC41MDQtMC45MDN2LTEuMDVxLTAuMjg0IDAuMTM2LTAuODYxIDAuMjMxLTAuNTY3IDAuMDkxNC0wLjk4NyAwLjE1OC0wLjQyIDAuMDY4Mi0wLjc2NiAwLjMyNi0wLjMzNiAwLjI1Mi0wLjMzNiAwLjcwNHQwLjMwNCAwLjcwNCAwLjg2MSAwLjI1MnoiIHN0cm9rZS13aWR0aD0iMS4wNSIvPgogIDxwYXRoIGQ9Im0xMCA0LjU2aDAuOTQ1djMuMTVxMC42NTEtMC45NzYgMS44OS0wLjk3NiAxLjE2IDAgMS44OSAwLjg0IDAuNjgyIDAuODQgMC42ODIgMi4zMSAwIDEuNDctMC43MDQgMi40Mi0wLjcwNCAwLjg4Mi0xLjg5IDAuODgyLTEuMjYgMC0xLjg5LTEuMDJ2MC43NjZoLTAuODV6bTIuNjIgMy4wNHEtMC43NDYgMC0xLjE2IDAuNjQtMC40NTIgMC42My0wLjQ1MiAxLjY4IDAgMS4wNSAwLjQ1MiAxLjY4dDEuMTYgMC42M3EwLjc3NyAwIDEuMjYtMC42MyAwLjQ5NC0wLjY0IDAuNDk0LTEuNjggMC0xLjA1LTAuNDcyLTEuNjgtMC40NjItMC42NC0xLjI2LTAuNjR6IiBzdHJva2Utd2lkdGg9IjEuMDUiLz4KICA8cGF0aCBkPSJtMi43MyAxNS44IDEzLjYgMC4wMDgxYzAuMDA2OSAwIDAtMi42IDAtMi42IDAtMC4wMDc4LTEuMTUgMC0xLjE1IDAtMC4wMDY5IDAtMC4wMDgzIDEuNS0wLjAwODMgMS41LTJlLTMgLTAuMDAxNC0xMS4zLTAuMDAxNC0xMS4zLTAuMDAxNGwtMC4wMDU5Mi0xLjVjMC0wLjAwNzgtMS4xNyAwLjAwMTMtMS4xNyAwLjAwMTN6IiBzdHJva2Utd2lkdGg9Ii45NzUiLz4KIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-yaml: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1jb250cmFzdDIganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjRDgxQjYwIj4KICAgIDxwYXRoIGQ9Ik03LjIgMTguNnYtNS40TDMgNS42aDMuM2wxLjQgMy4xYy4zLjkuNiAxLjYgMSAyLjUuMy0uOC42LTEuNiAxLTIuNWwxLjQtMy4xaDMuNGwtNC40IDcuNnY1LjVsLTIuOS0uMXoiLz4KICAgIDxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE3LjYiIGN5PSIxNi41IiByPSIyLjEiLz4KICAgIDxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE3LjYiIGN5PSIxMSIgcj0iMi4xIi8+CiAgPC9nPgo8L3N2Zz4K);
}

/* Icon CSS class declarations */

.jp-AddAboveIcon {
  background-image: var(--jp-icon-add-above);
}

.jp-AddBelowIcon {
  background-image: var(--jp-icon-add-below);
}

.jp-AddIcon {
  background-image: var(--jp-icon-add);
}

.jp-BellIcon {
  background-image: var(--jp-icon-bell);
}

.jp-BugDotIcon {
  background-image: var(--jp-icon-bug-dot);
}

.jp-BugIcon {
  background-image: var(--jp-icon-bug);
}

.jp-BuildIcon {
  background-image: var(--jp-icon-build);
}

.jp-CaretDownEmptyIcon {
  background-image: var(--jp-icon-caret-down-empty);
}

.jp-CaretDownEmptyThinIcon {
  background-image: var(--jp-icon-caret-down-empty-thin);
}

.jp-CaretDownIcon {
  background-image: var(--jp-icon-caret-down);
}

.jp-CaretLeftIcon {
  background-image: var(--jp-icon-caret-left);
}

.jp-CaretRightIcon {
  background-image: var(--jp-icon-caret-right);
}

.jp-CaretUpEmptyThinIcon {
  background-image: var(--jp-icon-caret-up-empty-thin);
}

.jp-CaretUpIcon {
  background-image: var(--jp-icon-caret-up);
}

.jp-CaseSensitiveIcon {
  background-image: var(--jp-icon-case-sensitive);
}

.jp-CheckIcon {
  background-image: var(--jp-icon-check);
}

.jp-CircleEmptyIcon {
  background-image: var(--jp-icon-circle-empty);
}

.jp-CircleIcon {
  background-image: var(--jp-icon-circle);
}

.jp-ClearIcon {
  background-image: var(--jp-icon-clear);
}

.jp-CloseIcon {
  background-image: var(--jp-icon-close);
}

.jp-CodeCheckIcon {
  background-image: var(--jp-icon-code-check);
}

.jp-CodeIcon {
  background-image: var(--jp-icon-code);
}

.jp-CollapseAllIcon {
  background-image: var(--jp-icon-collapse-all);
}

.jp-ConsoleIcon {
  background-image: var(--jp-icon-console);
}

.jp-CopyIcon {
  background-image: var(--jp-icon-copy);
}

.jp-CopyrightIcon {
  background-image: var(--jp-icon-copyright);
}

.jp-CutIcon {
  background-image: var(--jp-icon-cut);
}

.jp-DeleteIcon {
  background-image: var(--jp-icon-delete);
}

.jp-DownloadIcon {
  background-image: var(--jp-icon-download);
}

.jp-DuplicateIcon {
  background-image: var(--jp-icon-duplicate);
}

.jp-EditIcon {
  background-image: var(--jp-icon-edit);
}

.jp-EllipsesIcon {
  background-image: var(--jp-icon-ellipses);
}

.jp-ErrorIcon {
  background-image: var(--jp-icon-error);
}

.jp-ExpandAllIcon {
  background-image: var(--jp-icon-expand-all);
}

.jp-ExtensionIcon {
  background-image: var(--jp-icon-extension);
}

.jp-FastForwardIcon {
  background-image: var(--jp-icon-fast-forward);
}

.jp-FileIcon {
  background-image: var(--jp-icon-file);
}

.jp-FileUploadIcon {
  background-image: var(--jp-icon-file-upload);
}

.jp-FilterDotIcon {
  background-image: var(--jp-icon-filter-dot);
}

.jp-FilterIcon {
  background-image: var(--jp-icon-filter);
}

.jp-FilterListIcon {
  background-image: var(--jp-icon-filter-list);
}

.jp-FolderFavoriteIcon {
  background-image: var(--jp-icon-folder-favorite);
}

.jp-FolderIcon {
  background-image: var(--jp-icon-folder);
}

.jp-HomeIcon {
  background-image: var(--jp-icon-home);
}

.jp-Html5Icon {
  background-image: var(--jp-icon-html5);
}

.jp-ImageIcon {
  background-image: var(--jp-icon-image);
}

.jp-InfoIcon {
  background-image: var(--jp-icon-info);
}

.jp-InspectorIcon {
  background-image: var(--jp-icon-inspector);
}

.jp-JsonIcon {
  background-image: var(--jp-icon-json);
}

.jp-JuliaIcon {
  background-image: var(--jp-icon-julia);
}

.jp-JupyterFaviconIcon {
  background-image: var(--jp-icon-jupyter-favicon);
}

.jp-JupyterIcon {
  background-image: var(--jp-icon-jupyter);
}

.jp-JupyterlabWordmarkIcon {
  background-image: var(--jp-icon-jupyterlab-wordmark);
}

.jp-KernelIcon {
  background-image: var(--jp-icon-kernel);
}

.jp-KeyboardIcon {
  background-image: var(--jp-icon-keyboard);
}

.jp-LaunchIcon {
  background-image: var(--jp-icon-launch);
}

.jp-LauncherIcon {
  background-image: var(--jp-icon-launcher);
}

.jp-LineFormIcon {
  background-image: var(--jp-icon-line-form);
}

.jp-LinkIcon {
  background-image: var(--jp-icon-link);
}

.jp-ListIcon {
  background-image: var(--jp-icon-list);
}

.jp-MarkdownIcon {
  background-image: var(--jp-icon-markdown);
}

.jp-MoveDownIcon {
  background-image: var(--jp-icon-move-down);
}

.jp-MoveUpIcon {
  background-image: var(--jp-icon-move-up);
}

.jp-NewFolderIcon {
  background-image: var(--jp-icon-new-folder);
}

.jp-NotTrustedIcon {
  background-image: var(--jp-icon-not-trusted);
}

.jp-NotebookIcon {
  background-image: var(--jp-icon-notebook);
}

.jp-NumberingIcon {
  background-image: var(--jp-icon-numbering);
}

.jp-OfflineBoltIcon {
  background-image: var(--jp-icon-offline-bolt);
}

.jp-PaletteIcon {
  background-image: var(--jp-icon-palette);
}

.jp-PasteIcon {
  background-image: var(--jp-icon-paste);
}

.jp-PdfIcon {
  background-image: var(--jp-icon-pdf);
}

.jp-PythonIcon {
  background-image: var(--jp-icon-python);
}

.jp-RKernelIcon {
  background-image: var(--jp-icon-r-kernel);
}

.jp-ReactIcon {
  background-image: var(--jp-icon-react);
}

.jp-RedoIcon {
  background-image: var(--jp-icon-redo);
}

.jp-RefreshIcon {
  background-image: var(--jp-icon-refresh);
}

.jp-RegexIcon {
  background-image: var(--jp-icon-regex);
}

.jp-RunIcon {
  background-image: var(--jp-icon-run);
}

.jp-RunningIcon {
  background-image: var(--jp-icon-running);
}

.jp-SaveIcon {
  background-image: var(--jp-icon-save);
}

.jp-SearchIcon {
  background-image: var(--jp-icon-search);
}

.jp-SettingsIcon {
  background-image: var(--jp-icon-settings);
}

.jp-ShareIcon {
  background-image: var(--jp-icon-share);
}

.jp-SpreadsheetIcon {
  background-image: var(--jp-icon-spreadsheet);
}

.jp-StopIcon {
  background-image: var(--jp-icon-stop);
}

.jp-TabIcon {
  background-image: var(--jp-icon-tab);
}

.jp-TableRowsIcon {
  background-image: var(--jp-icon-table-rows);
}

.jp-TagIcon {
  background-image: var(--jp-icon-tag);
}

.jp-TerminalIcon {
  background-image: var(--jp-icon-terminal);
}

.jp-TextEditorIcon {
  background-image: var(--jp-icon-text-editor);
}

.jp-TocIcon {
  background-image: var(--jp-icon-toc);
}

.jp-TreeViewIcon {
  background-image: var(--jp-icon-tree-view);
}

.jp-TrustedIcon {
  background-image: var(--jp-icon-trusted);
}

.jp-UndoIcon {
  background-image: var(--jp-icon-undo);
}

.jp-UserIcon {
  background-image: var(--jp-icon-user);
}

.jp-UsersIcon {
  background-image: var(--jp-icon-users);
}

.jp-VegaIcon {
  background-image: var(--jp-icon-vega);
}

.jp-WordIcon {
  background-image: var(--jp-icon-word);
}

.jp-YamlIcon {
  background-image: var(--jp-icon-yaml);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * (DEPRECATED) Support for consuming icons as CSS background images
 */

.jp-Icon,
.jp-MaterialIcon {
  background-position: center;
  background-repeat: no-repeat;
  background-size: 16px;
  min-width: 16px;
  min-height: 16px;
}

.jp-Icon-cover {
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}

/**
 * (DEPRECATED) Support for specific CSS icon sizes
 */

.jp-Icon-16 {
  background-size: 16px;
  min-width: 16px;
  min-height: 16px;
}

.jp-Icon-18 {
  background-size: 18px;
  min-width: 18px;
  min-height: 18px;
}

.jp-Icon-20 {
  background-size: 20px;
  min-width: 20px;
  min-height: 20px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.lm-TabBar .lm-TabBar-addButton {
  align-items: center;
  display: flex;
  padding: 4px;
  padding-bottom: 5px;
  margin-right: 1px;
  background-color: var(--jp-layout-color2);
}

.lm-TabBar .lm-TabBar-addButton:hover {
  background-color: var(--jp-layout-color1);
}

.lm-DockPanel-tabBar .lm-TabBar-tab {
  width: var(--jp-private-horizontal-tab-width);
}

.lm-DockPanel-tabBar .lm-TabBar-content {
  flex: unset;
}

.lm-DockPanel-tabBar[data-orientation='horizontal'] {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * Support for icons as inline SVG HTMLElements
 */

/* recolor the primary elements of an icon */
.jp-icon0[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon1[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon2[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon3[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon4[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon0[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon1[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon2[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon3[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon4[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/* recolor the accent elements of an icon */
.jp-icon-accent0[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-accent1[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-accent2[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-accent3[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-accent4[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-accent0[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-accent1[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-accent2[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-accent3[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-accent4[stroke] {
  stroke: var(--jp-layout-color4);
}

/* set the color of an icon to transparent */
.jp-icon-none[fill] {
  fill: none;
}

.jp-icon-none[stroke] {
  stroke: none;
}

/* brand icon colors. Same for light and dark */
.jp-icon-brand0[fill] {
  fill: var(--jp-brand-color0);
}

.jp-icon-brand1[fill] {
  fill: var(--jp-brand-color1);
}

.jp-icon-brand2[fill] {
  fill: var(--jp-brand-color2);
}

.jp-icon-brand3[fill] {
  fill: var(--jp-brand-color3);
}

.jp-icon-brand4[fill] {
  fill: var(--jp-brand-color4);
}

.jp-icon-brand0[stroke] {
  stroke: var(--jp-brand-color0);
}

.jp-icon-brand1[stroke] {
  stroke: var(--jp-brand-color1);
}

.jp-icon-brand2[stroke] {
  stroke: var(--jp-brand-color2);
}

.jp-icon-brand3[stroke] {
  stroke: var(--jp-brand-color3);
}

.jp-icon-brand4[stroke] {
  stroke: var(--jp-brand-color4);
}

/* warn icon colors. Same for light and dark */
.jp-icon-warn0[fill] {
  fill: var(--jp-warn-color0);
}

.jp-icon-warn1[fill] {
  fill: var(--jp-warn-color1);
}

.jp-icon-warn2[fill] {
  fill: var(--jp-warn-color2);
}

.jp-icon-warn3[fill] {
  fill: var(--jp-warn-color3);
}

.jp-icon-warn0[stroke] {
  stroke: var(--jp-warn-color0);
}

.jp-icon-warn1[stroke] {
  stroke: var(--jp-warn-color1);
}

.jp-icon-warn2[stroke] {
  stroke: var(--jp-warn-color2);
}

.jp-icon-warn3[stroke] {
  stroke: var(--jp-warn-color3);
}

/* icon colors that contrast well with each other and most backgrounds */
.jp-icon-contrast0[fill] {
  fill: var(--jp-icon-contrast-color0);
}

.jp-icon-contrast1[fill] {
  fill: var(--jp-icon-contrast-color1);
}

.jp-icon-contrast2[fill] {
  fill: var(--jp-icon-contrast-color2);
}

.jp-icon-contrast3[fill] {
  fill: var(--jp-icon-contrast-color3);
}

.jp-icon-contrast0[stroke] {
  stroke: var(--jp-icon-contrast-color0);
}

.jp-icon-contrast1[stroke] {
  stroke: var(--jp-icon-contrast-color1);
}

.jp-icon-contrast2[stroke] {
  stroke: var(--jp-icon-contrast-color2);
}

.jp-icon-contrast3[stroke] {
  stroke: var(--jp-icon-contrast-color3);
}

.jp-icon-dot[fill] {
  fill: var(--jp-warn-color0);
}

.jp-jupyter-icon-color[fill] {
  fill: var(--jp-jupyter-icon-color, var(--jp-warn-color0));
}

.jp-notebook-icon-color[fill] {
  fill: var(--jp-notebook-icon-color, var(--jp-warn-color0));
}

.jp-json-icon-color[fill] {
  fill: var(--jp-json-icon-color, var(--jp-warn-color1));
}

.jp-console-icon-color[fill] {
  fill: var(--jp-console-icon-color, white);
}

.jp-console-icon-background-color[fill] {
  fill: var(--jp-console-icon-background-color, var(--jp-brand-color1));
}

.jp-terminal-icon-color[fill] {
  fill: var(--jp-terminal-icon-color, var(--jp-layout-color2));
}

.jp-terminal-icon-background-color[fill] {
  fill: var(
    --jp-terminal-icon-background-color,
    var(--jp-inverse-layout-color2)
  );
}

.jp-text-editor-icon-color[fill] {
  fill: var(--jp-text-editor-icon-color, var(--jp-inverse-layout-color3));
}

.jp-inspector-icon-color[fill] {
  fill: var(--jp-inspector-icon-color, var(--jp-inverse-layout-color3));
}

/* CSS for icons in selected filebrowser listing items */
.jp-DirListing-item.jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}

.jp-DirListing-item.jp-mod-selected .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}

/* stylelint-disable selector-max-class, selector-max-compound-selectors */

/**
* TODO: come up with non css-hack solution for showing the busy icon on top
*  of the close icon
* CSS for complex behavior of close icon of tabs in the main area tabbar
*/
.lm-DockPanel-tabBar
  .lm-TabBar-tab.lm-mod-closable.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon3[fill] {
  fill: none;
}

.lm-DockPanel-tabBar
  .lm-TabBar-tab.lm-mod-closable.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon-busy[fill] {
  fill: var(--jp-inverse-layout-color3);
}

/* stylelint-enable selector-max-class, selector-max-compound-selectors */

/* CSS for icons in status bar */
#jp-main-statusbar .jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}

#jp-main-statusbar .jp-mod-selected .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}

/* special handling for splash icon CSS. While the theme CSS reloads during
   splash, the splash icon can loose theming. To prevent that, we set a
   default for its color variable */
:root {
  --jp-warn-color0: var(--md-orange-700);
}

/* not sure what to do with this one, used in filebrowser listing */
.jp-DragIcon {
  margin-right: 4px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * Support for alt colors for icons as inline SVG HTMLElements
 */

/* alt recolor the primary elements of an icon */
.jp-icon-alt .jp-icon0[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-alt .jp-icon1[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-alt .jp-icon2[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-alt .jp-icon3[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-alt .jp-icon4[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-alt .jp-icon0[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-alt .jp-icon1[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-alt .jp-icon2[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-alt .jp-icon3[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-alt .jp-icon4[stroke] {
  stroke: var(--jp-layout-color4);
}

/* alt recolor the accent elements of an icon */
.jp-icon-alt .jp-icon-accent0[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon-alt .jp-icon-accent1[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon-alt .jp-icon-accent2[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon-alt .jp-icon-accent3[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon-alt .jp-icon-accent4[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-alt .jp-icon-accent0[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon-alt .jp-icon-accent1[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon-alt .jp-icon-accent2[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon-alt .jp-icon-accent3[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon-alt .jp-icon-accent4[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-icon-hoverShow:not(:hover) .jp-icon-hoverShow-content {
  display: none !important;
}

/**
 * Support for hover colors for icons as inline SVG HTMLElements
 */

/**
 * regular colors
 */

/* recolor the primary elements of an icon */
.jp-icon-hover :hover .jp-icon0-hover[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon-hover :hover .jp-icon1-hover[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon-hover :hover .jp-icon2-hover[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon-hover :hover .jp-icon3-hover[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon-hover :hover .jp-icon4-hover[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-hover :hover .jp-icon0-hover[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon-hover :hover .jp-icon1-hover[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon-hover :hover .jp-icon2-hover[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon-hover :hover .jp-icon3-hover[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon-hover :hover .jp-icon4-hover[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/* recolor the accent elements of an icon */
.jp-icon-hover :hover .jp-icon-accent0-hover[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-hover :hover .jp-icon-accent1-hover[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-hover :hover .jp-icon-accent2-hover[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-hover :hover .jp-icon-accent3-hover[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-hover :hover .jp-icon-accent4-hover[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-hover :hover .jp-icon-accent0-hover[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-hover :hover .jp-icon-accent1-hover[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-hover :hover .jp-icon-accent2-hover[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-hover :hover .jp-icon-accent3-hover[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-hover :hover .jp-icon-accent4-hover[stroke] {
  stroke: var(--jp-layout-color4);
}

/* set the color of an icon to transparent */
.jp-icon-hover :hover .jp-icon-none-hover[fill] {
  fill: none;
}

.jp-icon-hover :hover .jp-icon-none-hover[stroke] {
  stroke: none;
}

/**
 * inverse colors
 */

/* inverse recolor the primary elements of an icon */
.jp-icon-hover.jp-icon-alt :hover .jp-icon0-hover[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon1-hover[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon2-hover[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon3-hover[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon4-hover[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon0-hover[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon1-hover[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon2-hover[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon3-hover[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon4-hover[stroke] {
  stroke: var(--jp-layout-color4);
}

/* inverse recolor the accent elements of an icon */
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent0-hover[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent1-hover[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent2-hover[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent3-hover[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent4-hover[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent0-hover[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent1-hover[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent2-hover[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent3-hover[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent4-hover[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-IFrame {
  width: 100%;
  height: 100%;
}

.jp-IFrame > iframe {
  border: none;
}

/*
When drag events occur, `lm-mod-override-cursor` is added to the body.
Because iframes steal all cursor events, the following two rules are necessary
to suppress pointer events while resize drags are occurring. There may be a
better solution to this problem.
*/
body.lm-mod-override-cursor .jp-IFrame {
  position: relative;
}

body.lm-mod-override-cursor .jp-IFrame::before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-HoverBox {
  position: fixed;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-FormGroup-content fieldset {
  border: none;
  padding: 0;
  min-width: 0;
  width: 100%;
}

/* stylelint-disable selector-max-type */

.jp-FormGroup-content fieldset .jp-inputFieldWrapper input,
.jp-FormGroup-content fieldset .jp-inputFieldWrapper select,
.jp-FormGroup-content fieldset .jp-inputFieldWrapper textarea {
  font-size: var(--jp-content-font-size2);
  border-color: var(--jp-input-border-color);
  border-style: solid;
  border-radius: var(--jp-border-radius);
  border-width: 1px;
  padding: 6px 8px;
  background: none;
  color: var(--jp-ui-font-color0);
  height: inherit;
}

.jp-FormGroup-content fieldset input[type='checkbox'] {
  position: relative;
  top: 2px;
  margin-left: 0;
}

.jp-FormGroup-content button.jp-mod-styled {
  cursor: pointer;
}

.jp-FormGroup-content .checkbox label {
  cursor: pointer;
  font-size: var(--jp-content-font-size1);
}

.jp-FormGroup-content .jp-root > fieldset > legend {
  display: none;
}

.jp-FormGroup-content .jp-root > fieldset > p {
  display: none;
}

/** copy of `input.jp-mod-styled:focus` style */
.jp-FormGroup-content fieldset input:focus,
.jp-FormGroup-content fieldset select:focus {
  -moz-outline-radius: unset;
  outline: var(--jp-border-width) solid var(--md-blue-500);
  outline-offset: -1px;
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-FormGroup-content fieldset input:hover:not(:focus),
.jp-FormGroup-content fieldset select:hover:not(:focus) {
  background-color: var(--jp-border-color2);
}

/* stylelint-enable selector-max-type */

.jp-FormGroup-content .checkbox .field-description {
  /* Disable default description field for checkbox:
   because other widgets do not have description fields,
   we add descriptions to each widget on the field level.
  */
  display: none;
}

.jp-FormGroup-content #root__description {
  display: none;
}

.jp-FormGroup-content .jp-modifiedIndicator {
  width: 5px;
  background-color: var(--jp-brand-color2);
  margin-top: 0;
  margin-left: calc(var(--jp-private-settingeditor-modifier-indent) * -1);
  flex-shrink: 0;
}

.jp-FormGroup-content .jp-modifiedIndicator.jp-errorIndicator {
  background-color: var(--jp-error-color0);
  margin-right: 0.5em;
}

/* RJSF ARRAY style */

.jp-arrayFieldWrapper legend {
  font-size: var(--jp-content-font-size2);
  color: var(--jp-ui-font-color0);
  flex-basis: 100%;
  padding: 4px 0;
  font-weight: var(--jp-content-heading-font-weight);
  border-bottom: 1px solid var(--jp-border-color2);
}

.jp-arrayFieldWrapper .field-description {
  padding: 4px 0;
  white-space: pre-wrap;
}

.jp-arrayFieldWrapper .array-item {
  width: 100%;
  border: 1px solid var(--jp-border-color2);
  border-radius: 4px;
  margin: 4px;
}

.jp-ArrayOperations {
  display: flex;
  margin-left: 8px;
}

.jp-ArrayOperationsButton {
  margin: 2px;
}

.jp-ArrayOperationsButton .jp-icon3[fill] {
  fill: var(--jp-ui-font-color0);
}

button.jp-ArrayOperationsButton.jp-mod-styled:disabled {
  cursor: not-allowed;
  opacity: 0.5;
}

/* RJSF form validation error */

.jp-FormGroup-content .validationErrors {
  color: var(--jp-error-color0);
}

/* Hide panel level error as duplicated the field level error */
.jp-FormGroup-content .panel.errors {
  display: none;
}

/* RJSF normal content (settings-editor) */

.jp-FormGroup-contentNormal {
  display: flex;
  align-items: center;
  flex-wrap: wrap;
}

.jp-FormGroup-contentNormal .jp-FormGroup-contentItem {
  margin-left: 7px;
  color: var(--jp-ui-font-color0);
}

.jp-FormGroup-contentNormal .jp-FormGroup-description {
  flex-basis: 100%;
  padding: 4px 7px;
}

.jp-FormGroup-contentNormal .jp-FormGroup-default {
  flex-basis: 100%;
  padding: 4px 7px;
}

.jp-FormGroup-contentNormal .jp-FormGroup-fieldLabel {
  font-size: var(--jp-content-font-size1);
  font-weight: normal;
  min-width: 120px;
}

.jp-FormGroup-contentNormal fieldset:not(:first-child) {
  margin-left: 7px;
}

.jp-FormGroup-contentNormal .field-array-of-string .array-item {
  /* Display `jp-ArrayOperations` buttons side-by-side with content except
    for small screens where flex-wrap will place them one below the other.
  */
  display: flex;
  align-items: center;
  flex-wrap: wrap;
}

.jp-FormGroup-contentNormal .jp-objectFieldWrapper .form-group {
  padding: 2px 8px 2px var(--jp-private-settingeditor-modifier-indent);
  margin-top: 2px;
}

/* RJSF compact content (metadata-form) */

.jp-FormGroup-content.jp-FormGroup-contentCompact {
  width: 100%;
}

.jp-FormGroup-contentCompact .form-group {
  display: flex;
  padding: 0.5em 0.2em 0.5em 0;
}

.jp-FormGroup-contentCompact
  .jp-FormGroup-compactTitle
  .jp-FormGroup-description {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color2);
}

.jp-FormGroup-contentCompact .jp-FormGroup-fieldLabel {
  padding-bottom: 0.3em;
}

.jp-FormGroup-contentCompact .jp-inputFieldWrapper .form-control {
  width: 100%;
  box-sizing: border-box;
}

.jp-FormGroup-contentCompact .jp-arrayFieldWrapper .jp-FormGroup-compactTitle {
  padding-bottom: 7px;
}

.jp-FormGroup-contentCompact
  .jp-objectFieldWrapper
  .jp-objectFieldWrapper
  .form-group {
  padding: 2px 8px 2px var(--jp-private-settingeditor-modifier-indent);
  margin-top: 2px;
}

.jp-FormGroup-contentCompact ul.error-detail {
  margin-block-start: 0.5em;
  margin-block-end: 0.5em;
  padding-inline-start: 1em;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-SidePanel {
  display: flex;
  flex-direction: column;
  min-width: var(--jp-sidebar-min-width);
  overflow-y: auto;
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);
  font-size: var(--jp-ui-font-size1);
}

.jp-SidePanel-header {
  flex: 0 0 auto;
  display: flex;
  border-bottom: var(--jp-border-width) solid var(--jp-border-color2);
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  letter-spacing: 1px;
  margin: 0;
  padding: 2px;
  text-transform: uppercase;
}

.jp-SidePanel-toolbar {
  flex: 0 0 auto;
}

.jp-SidePanel-content {
  flex: 1 1 auto;
}

.jp-SidePanel-toolbar,
.jp-AccordionPanel-toolbar {
  height: var(--jp-private-toolbar-height);
}

.jp-SidePanel-toolbar.jp-Toolbar-micro {
  display: none;
}

.lm-AccordionPanel .jp-AccordionPanel-title {
  box-sizing: border-box;
  line-height: 25px;
  margin: 0;
  display: flex;
  align-items: center;
  background: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  font-size: var(--jp-ui-font-size0);
}

.jp-AccordionPanel-title {
  cursor: pointer;
  user-select: none;
  -moz-user-select: none;
  -webkit-user-select: none;
  text-transform: uppercase;
}

.lm-AccordionPanel[data-orientation='horizontal'] > .jp-AccordionPanel-title {
  /* Title is rotated for horizontal accordion panel using CSS */
  display: block;
  transform-origin: top left;
  transform: rotate(-90deg) translate(-100%);
}

.jp-AccordionPanel-title .lm-AccordionPanel-titleLabel {
  user-select: none;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}

.jp-AccordionPanel-title .lm-AccordionPanel-titleCollapser {
  transform: rotate(-90deg);
  margin: auto 0;
  height: 16px;
}

.jp-AccordionPanel-title.lm-mod-expanded .lm-AccordionPanel-titleCollapser {
  transform: rotate(0deg);
}

.lm-AccordionPanel .jp-AccordionPanel-toolbar {
  background: none;
  box-shadow: none;
  border: none;
  margin-left: auto;
}

.lm-AccordionPanel .lm-SplitPanel-handle:hover {
  background: var(--jp-layout-color3);
}

.jp-text-truncated {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Spinner {
  position: absolute;
  display: flex;
  justify-content: center;
  align-items: center;
  z-index: 10;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  background: var(--jp-layout-color0);
  outline: none;
}

.jp-SpinnerContent {
  font-size: 10px;
  margin: 50px auto;
  text-indent: -9999em;
  width: 3em;
  height: 3em;
  border-radius: 50%;
  background: var(--jp-brand-color3);
  background: linear-gradient(
    to right,
    #f37626 10%,
    rgba(255, 255, 255, 0) 42%
  );
  position: relative;
  animation: load3 1s infinite linear, fadeIn 1s;
}

.jp-SpinnerContent::before {
  width: 50%;
  height: 50%;
  background: #f37626;
  border-radius: 100% 0 0;
  position: absolute;
  top: 0;
  left: 0;
  content: '';
}

.jp-SpinnerContent::after {
  background: var(--jp-layout-color0);
  width: 75%;
  height: 75%;
  border-radius: 50%;
  content: '';
  margin: auto;
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
}

@keyframes fadeIn {
  0% {
    opacity: 0;
  }

  100% {
    opacity: 1;
  }
}

@keyframes load3 {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

button.jp-mod-styled {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  border: none;
  box-sizing: border-box;
  text-align: center;
  line-height: 32px;
  height: 32px;
  padding: 0 12px;
  letter-spacing: 0.8px;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

input.jp-mod-styled {
  background: var(--jp-input-background);
  height: 28px;
  box-sizing: border-box;
  border: var(--jp-border-width) solid var(--jp-border-color1);
  padding-left: 7px;
  padding-right: 7px;
  font-size: var(--jp-ui-font-size2);
  color: var(--jp-ui-font-color0);
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

input[type='checkbox'].jp-mod-styled {
  appearance: checkbox;
  -webkit-appearance: checkbox;
  -moz-appearance: checkbox;
  height: auto;
}

input.jp-mod-styled:focus {
  border: var(--jp-border-width) solid var(--md-blue-500);
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-select-wrapper {
  display: flex;
  position: relative;
  flex-direction: column;
  padding: 1px;
  background-color: var(--jp-layout-color1);
  box-sizing: border-box;
  margin-bottom: 12px;
}

.jp-select-wrapper:not(.multiple) {
  height: 28px;
}

.jp-select-wrapper.jp-mod-focused select.jp-mod-styled {
  border: var(--jp-border-width) solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
  background-color: var(--jp-input-active-background);
}

select.jp-mod-styled:hover {
  cursor: pointer;
  color: var(--jp-ui-font-color0);
  background-color: var(--jp-input-hover-background);
  box-shadow: inset 0 0 1px rgba(0, 0, 0, 0.5);
}

select.jp-mod-styled {
  flex: 1 1 auto;
  width: 100%;
  font-size: var(--jp-ui-font-size2);
  background: var(--jp-input-background);
  color: var(--jp-ui-font-color0);
  padding: 0 25px 0 8px;
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  border-radius: 0;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

select.jp-mod-styled:not([multiple]) {
  height: 32px;
}

select.jp-mod-styled[multiple] {
  max-height: 200px;
  overflow-y: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-switch {
  display: flex;
  align-items: center;
  padding-left: 4px;
  padding-right: 4px;
  font-size: var(--jp-ui-font-size1);
  background-color: transparent;
  color: var(--jp-ui-font-color1);
  border: none;
  height: 20px;
}

.jp-switch:hover {
  background-color: var(--jp-layout-color2);
}

.jp-switch-label {
  margin-right: 5px;
  font-family: var(--jp-ui-font-family);
}

.jp-switch-track {
  cursor: pointer;
  background-color: var(--jp-switch-color, var(--jp-border-color1));
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 34px;
  height: 16px;
  width: 35px;
  position: relative;
}

.jp-switch-track::before {
  content: '';
  position: absolute;
  height: 10px;
  width: 10px;
  margin: 3px;
  left: 0;
  background-color: var(--jp-ui-inverse-font-color1);
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 50%;
}

.jp-switch[aria-checked='true'] .jp-switch-track {
  background-color: var(--jp-switch-true-position-color, var(--jp-warn-color0));
}

.jp-switch[aria-checked='true'] .jp-switch-track::before {
  /* track width (35) - margins (3 + 3) - thumb width (10) */
  left: 19px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

:root {
  --jp-private-toolbar-height: calc(
    28px + var(--jp-border-width)
  ); /* leave 28px for content */
}

.jp-Toolbar {
  color: var(--jp-ui-font-color1);
  flex: 0 0 auto;
  display: flex;
  flex-direction: row;
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  background: var(--jp-toolbar-background);
  min-height: var(--jp-toolbar-micro-height);
  padding: 2px;
  z-index: 8;
  overflow-x: hidden;
}

/* Toolbar items */

.jp-Toolbar > .jp-Toolbar-item.jp-Toolbar-spacer {
  flex-grow: 1;
  flex-shrink: 1;
}

.jp-Toolbar-item.jp-Toolbar-kernelStatus {
  display: inline-block;
  width: 32px;
  background-repeat: no-repeat;
  background-position: center;
  background-size: 16px;
}

.jp-Toolbar > .jp-Toolbar-item {
  flex: 0 0 auto;
  display: flex;
  padding-left: 1px;
  padding-right: 1px;
  font-size: var(--jp-ui-font-size1);
  line-height: var(--jp-private-toolbar-height);
  height: 100%;
}

/* Toolbar buttons */

/* This is the div we use to wrap the react component into a Widget */
div.jp-ToolbarButton {
  color: transparent;
  border: none;
  box-sizing: border-box;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 0;
  margin: 0;
}

button.jp-ToolbarButtonComponent {
  background: var(--jp-layout-color1);
  border: none;
  box-sizing: border-box;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 0 6px;
  margin: 0;
  height: 24px;
  border-radius: var(--jp-border-radius);
  display: flex;
  align-items: center;
  text-align: center;
  font-size: 14px;
  min-width: unset;
  min-height: unset;
}

button.jp-ToolbarButtonComponent:disabled {
  opacity: 0.4;
}

button.jp-ToolbarButtonComponent > span {
  padding: 0;
  flex: 0 0 auto;
}

button.jp-ToolbarButtonComponent .jp-ToolbarButtonComponent-label {
  font-size: var(--jp-ui-font-size1);
  line-height: 100%;
  padding-left: 2px;
  color: var(--jp-ui-font-color1);
  font-family: var(--jp-ui-font-family);
}

#jp-main-dock-panel[data-mode='single-document']
  .jp-MainAreaWidget
  > .jp-Toolbar.jp-Toolbar-micro {
  padding: 0;
  min-height: 0;
}

#jp-main-dock-panel[data-mode='single-document']
  .jp-MainAreaWidget
  > .jp-Toolbar {
  border: none;
  box-shadow: none;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-WindowedPanel-outer {
  position: relative;
  overflow-y: auto;
}

.jp-WindowedPanel-inner {
  position: relative;
}

.jp-WindowedPanel-window {
  position: absolute;
  left: 0;
  right: 0;
  overflow: visible;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/* Sibling imports */

body {
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
}

/* Disable native link decoration styles everywhere outside of dialog boxes */
a {
  text-decoration: unset;
  color: unset;
}

a:hover {
  text-decoration: unset;
  color: unset;
}

/* Accessibility for links inside dialog box text */
.jp-Dialog-content a {
  text-decoration: revert;
  color: var(--jp-content-link-color);
}

.jp-Dialog-content a:hover {
  text-decoration: revert;
}

/* Styles for ui-components */
.jp-Button {
  color: var(--jp-ui-font-color2);
  border-radius: var(--jp-border-radius);
  padding: 0 12px;
  font-size: var(--jp-ui-font-size1);

  /* Copy from blueprint 3 */
  display: inline-flex;
  flex-direction: row;
  border: none;
  cursor: pointer;
  align-items: center;
  justify-content: center;
  text-align: left;
  vertical-align: middle;
  min-height: 30px;
  min-width: 30px;
}

.jp-Button:disabled {
  cursor: not-allowed;
}

.jp-Button:empty {
  padding: 0 !important;
}

.jp-Button.jp-mod-small {
  min-height: 24px;
  min-width: 24px;
  font-size: 12px;
  padding: 0 7px;
}

/* Use our own theme for hover styles */
.jp-Button.jp-mod-minimal:hover {
  background-color: var(--jp-layout-color2);
}

.jp-Button.jp-mod-minimal {
  background: none;
}

.jp-InputGroup {
  display: block;
  position: relative;
}

.jp-InputGroup input {
  box-sizing: border-box;
  border: none;
  border-radius: 0;
  background-color: transparent;
  color: var(--jp-ui-font-color0);
  box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-border-color);
  padding-bottom: 0;
  padding-top: 0;
  padding-left: 10px;
  padding-right: 28px;
  position: relative;
  width: 100%;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  font-size: 14px;
  font-weight: 400;
  height: 30px;
  line-height: 30px;
  outline: none;
  vertical-align: middle;
}

.jp-InputGroup input:focus {
  box-shadow: inset 0 0 0 var(--jp-border-width)
      var(--jp-input-active-box-shadow-color),
    inset 0 0 0 3px var(--jp-input-active-box-shadow-color);
}

.jp-InputGroup input:disabled {
  cursor: not-allowed;
  resize: block;
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color2);
}

.jp-InputGroup input:disabled ~ span {
  cursor: not-allowed;
  color: var(--jp-ui-font-color2);
}

.jp-InputGroup input::placeholder,
input::placeholder {
  color: var(--jp-ui-font-color2);
}

.jp-InputGroupAction {
  position: absolute;
  bottom: 1px;
  right: 0;
  padding: 6px;
}

.jp-HTMLSelect.jp-DefaultStyle select {
  background-color: initial;
  border: none;
  border-radius: 0;
  box-shadow: none;
  color: var(--jp-ui-font-color0);
  display: block;
  font-size: var(--jp-ui-font-size1);
  font-family: var(--jp-ui-font-family);
  height: 24px;
  line-height: 14px;
  padding: 0 25px 0 10px;
  text-align: left;
  -moz-appearance: none;
  -webkit-appearance: none;
}

.jp-HTMLSelect.jp-DefaultStyle select:disabled {
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color2);
  cursor: not-allowed;
  resize: block;
}

.jp-HTMLSelect.jp-DefaultStyle select:disabled ~ span {
  cursor: not-allowed;
}

/* Use our own theme for hover and option styles */
/* stylelint-disable-next-line selector-max-type */
.jp-HTMLSelect.jp-DefaultStyle select:hover,
.jp-HTMLSelect.jp-DefaultStyle select > option {
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color0);
}

select {
  box-sizing: border-box;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-StatusBar-Widget {
  display: flex;
  align-items: center;
  background: var(--jp-layout-color2);
  min-height: var(--jp-statusbar-height);
  justify-content: space-between;
  padding: 0 10px;
}

.jp-StatusBar-Left {
  display: flex;
  align-items: center;
  flex-direction: row;
}

.jp-StatusBar-Middle {
  display: flex;
  align-items: center;
}

.jp-StatusBar-Right {
  display: flex;
  align-items: center;
  flex-direction: row-reverse;
}

.jp-StatusBar-Item {
  max-height: var(--jp-statusbar-height);
  margin: 0 2px;
  height: var(--jp-statusbar-height);
  white-space: nowrap;
  text-overflow: ellipsis;
  color: var(--jp-ui-font-color1);
  padding: 0 6px;
}

.jp-mod-highlighted:hover {
  background-color: var(--jp-layout-color3);
}

.jp-mod-clicked {
  background-color: var(--jp-brand-color1);
}

.jp-mod-clicked:hover {
  background-color: var(--jp-brand-color0);
}

.jp-mod-clicked .jp-StatusBar-TextItem {
  color: var(--jp-ui-inverse-font-color1);
}

.jp-StatusBar-HoverItem {
  box-shadow: '0px 4px 4px rgba(0, 0, 0, 0.25)';
}

.jp-StatusBar-TextItem {
  font-size: var(--jp-ui-font-size1);
  font-family: var(--jp-ui-font-family);
  line-height: 24px;
  color: var(--jp-ui-font-color1);
}

.jp-StatusBar-GroupItem {
  display: flex;
  align-items: center;
  flex-direction: row;
}

.jp-Statusbar-ProgressCircle svg {
  display: block;
  margin: 0 auto;
  width: 16px;
  height: 24px;
  align-self: normal;
}

.jp-Statusbar-ProgressCircle path {
  fill: var(--jp-inverse-layout-color3);
}

.jp-Statusbar-ProgressBar-progress-bar {
  height: 10px;
  width: 100px;
  border: solid 0.25px var(--jp-brand-color2);
  border-radius: 3px;
  overflow: hidden;
  align-self: center;
}

.jp-Statusbar-ProgressBar-progress-bar > div {
  background-color: var(--jp-brand-color2);
  background-image: linear-gradient(
    -45deg,
    rgba(255, 255, 255, 0.2) 25%,
    transparent 25%,
    transparent 50%,
    rgba(255, 255, 255, 0.2) 50%,
    rgba(255, 255, 255, 0.2) 75%,
    transparent 75%,
    transparent
  );
  background-size: 40px 40px;
  float: left;
  width: 0%;
  height: 100%;
  font-size: 12px;
  line-height: 14px;
  color: #fff;
  text-align: center;
  animation: jp-Statusbar-ExecutionTime-progress-bar 2s linear infinite;
}

.jp-Statusbar-ProgressBar-progress-bar p {
  color: var(--jp-ui-font-color1);
  font-family: var(--jp-ui-font-family);
  font-size: var(--jp-ui-font-size1);
  line-height: 10px;
  width: 100px;
}

@keyframes jp-Statusbar-ExecutionTime-progress-bar {
  0% {
    background-position: 0 0;
  }

  100% {
    background-position: 40px 40px;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-commandpalette-search-height: 28px;
}

/*-----------------------------------------------------------------------------
| Overall styles
|----------------------------------------------------------------------------*/

.lm-CommandPalette {
  padding-bottom: 0;
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);

  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
}

/*-----------------------------------------------------------------------------
| Modal variant
|----------------------------------------------------------------------------*/

.jp-ModalCommandPalette {
  position: absolute;
  z-index: 10000;
  top: 38px;
  left: 30%;
  margin: 0;
  padding: 4px;
  width: 40%;
  box-shadow: var(--jp-elevation-z4);
  border-radius: 4px;
  background: var(--jp-layout-color0);
}

.jp-ModalCommandPalette .lm-CommandPalette {
  max-height: 40vh;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-close-icon::after {
  display: none;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-CommandPalette-header {
  display: none;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-CommandPalette-item {
  margin-left: 4px;
  margin-right: 4px;
}

.jp-ModalCommandPalette
  .lm-CommandPalette
  .lm-CommandPalette-item.lm-mod-disabled {
  display: none;
}

/*-----------------------------------------------------------------------------
| Search
|----------------------------------------------------------------------------*/

.lm-CommandPalette-search {
  padding: 4px;
  background-color: var(--jp-layout-color1);
  z-index: 2;
}

.lm-CommandPalette-wrapper {
  overflow: overlay;
  padding: 0 9px;
  background-color: var(--jp-input-active-background);
  height: 30px;
  box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-border-color);
}

.lm-CommandPalette.lm-mod-focused .lm-CommandPalette-wrapper {
  box-shadow: inset 0 0 0 1px var(--jp-input-active-box-shadow-color),
    inset 0 0 0 3px var(--jp-input-active-box-shadow-color);
}

.jp-SearchIconGroup {
  color: white;
  background-color: var(--jp-brand-color1);
  position: absolute;
  top: 4px;
  right: 4px;
  padding: 5px 5px 1px;
}

.jp-SearchIconGroup svg {
  height: 20px;
  width: 20px;
}

.jp-SearchIconGroup .jp-icon3[fill] {
  fill: var(--jp-layout-color0);
}

.lm-CommandPalette-input {
  background: transparent;
  width: calc(100% - 18px);
  float: left;
  border: none;
  outline: none;
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  line-height: var(--jp-private-commandpalette-search-height);
}

.lm-CommandPalette-input::-webkit-input-placeholder,
.lm-CommandPalette-input::-moz-placeholder,
.lm-CommandPalette-input:-ms-input-placeholder {
  color: var(--jp-ui-font-color2);
  font-size: var(--jp-ui-font-size1);
}

/*-----------------------------------------------------------------------------
| Results
|----------------------------------------------------------------------------*/

.lm-CommandPalette-header:first-child {
  margin-top: 0;
}

.lm-CommandPalette-header {
  border-bottom: solid var(--jp-border-width) var(--jp-border-color2);
  color: var(--jp-ui-font-color1);
  cursor: pointer;
  display: flex;
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  letter-spacing: 1px;
  margin-top: 8px;
  padding: 8px 0 8px 12px;
  text-transform: uppercase;
}

.lm-CommandPalette-header.lm-mod-active {
  background: var(--jp-layout-color2);
}

.lm-CommandPalette-header > mark {
  background-color: transparent;
  font-weight: bold;
  color: var(--jp-ui-font-color1);
}

.lm-CommandPalette-item {
  padding: 4px 12px 4px 4px;
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  font-weight: 400;
  display: flex;
}

.lm-CommandPalette-item.lm-mod-disabled {
  color: var(--jp-ui-font-color2);
}

.lm-CommandPalette-item.lm-mod-active {
  color: var(--jp-ui-inverse-font-color1);
  background: var(--jp-brand-color1);
}

.lm-CommandPalette-item.lm-mod-active .lm-CommandPalette-itemLabel > mark {
  color: var(--jp-ui-inverse-font-color0);
}

.lm-CommandPalette-item.lm-mod-active .jp-icon-selectable[fill] {
  fill: var(--jp-layout-color0);
}

.lm-CommandPalette-item.lm-mod-active:hover:not(.lm-mod-disabled) {
  color: var(--jp-ui-inverse-font-color1);
  background: var(--jp-brand-color1);
}

.lm-CommandPalette-item:hover:not(.lm-mod-active):not(.lm-mod-disabled) {
  background: var(--jp-layout-color2);
}

.lm-CommandPalette-itemContent {
  overflow: hidden;
}

.lm-CommandPalette-itemLabel > mark {
  color: var(--jp-ui-font-color0);
  background-color: transparent;
  font-weight: bold;
}

.lm-CommandPalette-item.lm-mod-disabled mark {
  color: var(--jp-ui-font-color2);
}

.lm-CommandPalette-item .lm-CommandPalette-itemIcon {
  margin: 0 4px 0 0;
  position: relative;
  width: 16px;
  top: 2px;
  flex: 0 0 auto;
}

.lm-CommandPalette-item.lm-mod-disabled .lm-CommandPalette-itemIcon {
  opacity: 0.6;
}

.lm-CommandPalette-item .lm-CommandPalette-itemShortcut {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemCaption {
  display: none;
}

.lm-CommandPalette-content {
  background-color: var(--jp-layout-color1);
}

.lm-CommandPalette-content:empty::after {
  content: 'No results';
  margin: auto;
  margin-top: 20px;
  width: 100px;
  display: block;
  font-size: var(--jp-ui-font-size2);
  font-family: var(--jp-ui-font-family);
  font-weight: lighter;
}

.lm-CommandPalette-emptyMessage {
  text-align: center;
  margin-top: 24px;
  line-height: 1.32;
  padding: 0 8px;
  color: var(--jp-content-font-color3);
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Dialog {
  position: absolute;
  z-index: 10000;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  top: 0;
  left: 0;
  margin: 0;
  padding: 0;
  width: 100%;
  height: 100%;
  background: var(--jp-dialog-background);
}

.jp-Dialog-content {
  display: flex;
  flex-direction: column;
  margin-left: auto;
  margin-right: auto;
  background: var(--jp-layout-color1);
  padding: 24px 24px 12px;
  min-width: 300px;
  min-height: 150px;
  max-width: 1000px;
  max-height: 500px;
  box-sizing: border-box;
  box-shadow: var(--jp-elevation-z20);
  word-wrap: break-word;
  border-radius: var(--jp-border-radius);

  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color1);
  resize: both;
}

.jp-Dialog-content.jp-Dialog-content-small {
  max-width: 500px;
}

.jp-Dialog-button {
  overflow: visible;
}

button.jp-Dialog-button:focus {
  outline: 1px solid var(--jp-brand-color1);
  outline-offset: 4px;
  -moz-outline-radius: 0;
}

button.jp-Dialog-button:focus::-moz-focus-inner {
  border: 0;
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-accept:focus,
button.jp-Dialog-button.jp-mod-styled.jp-mod-warn:focus,
button.jp-Dialog-button.jp-mod-styled.jp-mod-reject:focus {
  outline-offset: 4px;
  -moz-outline-radius: 0;
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-accept:focus {
  outline: 1px solid var(--jp-accept-color-normal, var(--jp-brand-color1));
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-warn:focus {
  outline: 1px solid var(--jp-warn-color-normal, var(--jp-error-color1));
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-reject:focus {
  outline: 1px solid var(--jp-reject-color-normal, var(--md-grey-600));
}

button.jp-Dialog-close-button {
  padding: 0;
  height: 100%;
  min-width: unset;
  min-height: unset;
}

.jp-Dialog-header {
  display: flex;
  justify-content: space-between;
  flex: 0 0 auto;
  padding-bottom: 12px;
  font-size: var(--jp-ui-font-size3);
  font-weight: 400;
  color: var(--jp-ui-font-color1);
}

.jp-Dialog-body {
  display: flex;
  flex-direction: column;
  flex: 1 1 auto;
  font-size: var(--jp-ui-font-size1);
  background: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  overflow: auto;
}

.jp-Dialog-footer {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  align-items: center;
  flex: 0 0 auto;
  margin-left: -12px;
  margin-right: -12px;
  padding: 12px;
}

.jp-Dialog-checkbox {
  padding-right: 5px;
}

.jp-Dialog-checkbox > input:focus-visible {
  outline: 1px solid var(--jp-input-active-border-color);
  outline-offset: 1px;
}

.jp-Dialog-spacer {
  flex: 1 1 auto;
}

.jp-Dialog-title {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.jp-Dialog-body > .jp-select-wrapper {
  width: 100%;
}

.jp-Dialog-body > button {
  padding: 0 16px;
}

.jp-Dialog-body > label {
  line-height: 1.4;
  color: var(--jp-ui-font-color0);
}

.jp-Dialog-button.jp-mod-styled:not(:last-child) {
  margin-right: 12px;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-Input-Boolean-Dialog {
  flex-direction: row-reverse;
  align-items: end;
  width: 100%;
}

.jp-Input-Boolean-Dialog > label {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-MainAreaWidget > :focus {
  outline: none;
}

.jp-MainAreaWidget .jp-MainAreaWidget-error {
  padding: 6px;
}

.jp-MainAreaWidget .jp-MainAreaWidget-error > pre {
  width: auto;
  padding: 10px;
  background: var(--jp-error-color3);
  border: var(--jp-border-width) solid var(--jp-error-color1);
  border-radius: var(--jp-border-radius);
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  white-space: pre-wrap;
  word-wrap: break-word;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/**
 * google-material-color v1.2.6
 * https://github.com/danlevan/google-material-color
 */
:root {
  --md-red-50: #ffebee;
  --md-red-100: #ffcdd2;
  --md-red-200: #ef9a9a;
  --md-red-300: #e57373;
  --md-red-400: #ef5350;
  --md-red-500: #f44336;
  --md-red-600: #e53935;
  --md-red-700: #d32f2f;
  --md-red-800: #c62828;
  --md-red-900: #b71c1c;
  --md-red-A100: #ff8a80;
  --md-red-A200: #ff5252;
  --md-red-A400: #ff1744;
  --md-red-A700: #d50000;
  --md-pink-50: #fce4ec;
  --md-pink-100: #f8bbd0;
  --md-pink-200: #f48fb1;
  --md-pink-300: #f06292;
  --md-pink-400: #ec407a;
  --md-pink-500: #e91e63;
  --md-pink-600: #d81b60;
  --md-pink-700: #c2185b;
  --md-pink-800: #ad1457;
  --md-pink-900: #880e4f;
  --md-pink-A100: #ff80ab;
  --md-pink-A200: #ff4081;
  --md-pink-A400: #f50057;
  --md-pink-A700: #c51162;
  --md-purple-50: #f3e5f5;
  --md-purple-100: #e1bee7;
  --md-purple-200: #ce93d8;
  --md-purple-300: #ba68c8;
  --md-purple-400: #ab47bc;
  --md-purple-500: #9c27b0;
  --md-purple-600: #8e24aa;
  --md-purple-700: #7b1fa2;
  --md-purple-800: #6a1b9a;
  --md-purple-900: #4a148c;
  --md-purple-A100: #ea80fc;
  --md-purple-A200: #e040fb;
  --md-purple-A400: #d500f9;
  --md-purple-A700: #a0f;
  --md-deep-purple-50: #ede7f6;
  --md-deep-purple-100: #d1c4e9;
  --md-deep-purple-200: #b39ddb;
  --md-deep-purple-300: #9575cd;
  --md-deep-purple-400: #7e57c2;
  --md-deep-purple-500: #673ab7;
  --md-deep-purple-600: #5e35b1;
  --md-deep-purple-700: #512da8;
  --md-deep-purple-800: #4527a0;
  --md-deep-purple-900: #311b92;
  --md-deep-purple-A100: #b388ff;
  --md-deep-purple-A200: #7c4dff;
  --md-deep-purple-A400: #651fff;
  --md-deep-purple-A700: #6200ea;
  --md-indigo-50: #e8eaf6;
  --md-indigo-100: #c5cae9;
  --md-indigo-200: #9fa8da;
  --md-indigo-300: #7986cb;
  --md-indigo-400: #5c6bc0;
  --md-indigo-500: #3f51b5;
  --md-indigo-600: #3949ab;
  --md-indigo-700: #303f9f;
  --md-indigo-800: #283593;
  --md-indigo-900: #1a237e;
  --md-indigo-A100: #8c9eff;
  --md-indigo-A200: #536dfe;
  --md-indigo-A400: #3d5afe;
  --md-indigo-A700: #304ffe;
  --md-blue-50: #e3f2fd;
  --md-blue-100: #bbdefb;
  --md-blue-200: #90caf9;
  --md-blue-300: #64b5f6;
  --md-blue-400: #42a5f5;
  --md-blue-500: #2196f3;
  --md-blue-600: #1e88e5;
  --md-blue-700: #1976d2;
  --md-blue-800: #1565c0;
  --md-blue-900: #0d47a1;
  --md-blue-A100: #82b1ff;
  --md-blue-A200: #448aff;
  --md-blue-A400: #2979ff;
  --md-blue-A700: #2962ff;
  --md-light-blue-50: #e1f5fe;
  --md-light-blue-100: #b3e5fc;
  --md-light-blue-200: #81d4fa;
  --md-light-blue-300: #4fc3f7;
  --md-light-blue-400: #29b6f6;
  --md-light-blue-500: #03a9f4;
  --md-light-blue-600: #039be5;
  --md-light-blue-700: #0288d1;
  --md-light-blue-800: #0277bd;
  --md-light-blue-900: #01579b;
  --md-light-blue-A100: #80d8ff;
  --md-light-blue-A200: #40c4ff;
  --md-light-blue-A400: #00b0ff;
  --md-light-blue-A700: #0091ea;
  --md-cyan-50: #e0f7fa;
  --md-cyan-100: #b2ebf2;
  --md-cyan-200: #80deea;
  --md-cyan-300: #4dd0e1;
  --md-cyan-400: #26c6da;
  --md-cyan-500: #00bcd4;
  --md-cyan-600: #00acc1;
  --md-cyan-700: #0097a7;
  --md-cyan-800: #00838f;
  --md-cyan-900: #006064;
  --md-cyan-A100: #84ffff;
  --md-cyan-A200: #18ffff;
  --md-cyan-A400: #00e5ff;
  --md-cyan-A700: #00b8d4;
  --md-teal-50: #e0f2f1;
  --md-teal-100: #b2dfdb;
  --md-teal-200: #80cbc4;
  --md-teal-300: #4db6ac;
  --md-teal-400: #26a69a;
  --md-teal-500: #009688;
  --md-teal-600: #00897b;
  --md-teal-700: #00796b;
  --md-teal-800: #00695c;
  --md-teal-900: #004d40;
  --md-teal-A100: #a7ffeb;
  --md-teal-A200: #64ffda;
  --md-teal-A400: #1de9b6;
  --md-teal-A700: #00bfa5;
  --md-green-50: #e8f5e9;
  --md-green-100: #c8e6c9;
  --md-green-200: #a5d6a7;
  --md-green-300: #81c784;
  --md-green-400: #66bb6a;
  --md-green-500: #4caf50;
  --md-green-600: #43a047;
  --md-green-700: #388e3c;
  --md-green-800: #2e7d32;
  --md-green-900: #1b5e20;
  --md-green-A100: #b9f6ca;
  --md-green-A200: #69f0ae;
  --md-green-A400: #00e676;
  --md-green-A700: #00c853;
  --md-light-green-50: #f1f8e9;
  --md-light-green-100: #dcedc8;
  --md-light-green-200: #c5e1a5;
  --md-light-green-300: #aed581;
  --md-light-green-400: #9ccc65;
  --md-light-green-500: #8bc34a;
  --md-light-green-600: #7cb342;
  --md-light-green-700: #689f38;
  --md-light-green-800: #558b2f;
  --md-light-green-900: #33691e;
  --md-light-green-A100: #ccff90;
  --md-light-green-A200: #b2ff59;
  --md-light-green-A400: #76ff03;
  --md-light-green-A700: #64dd17;
  --md-lime-50: #f9fbe7;
  --md-lime-100: #f0f4c3;
  --md-lime-200: #e6ee9c;
  --md-lime-300: #dce775;
  --md-lime-400: #d4e157;
  --md-lime-500: #cddc39;
  --md-lime-600: #c0ca33;
  --md-lime-700: #afb42b;
  --md-lime-800: #9e9d24;
  --md-lime-900: #827717;
  --md-lime-A100: #f4ff81;
  --md-lime-A200: #eeff41;
  --md-lime-A400: #c6ff00;
  --md-lime-A700: #aeea00;
  --md-yellow-50: #fffde7;
  --md-yellow-100: #fff9c4;
  --md-yellow-200: #fff59d;
  --md-yellow-300: #fff176;
  --md-yellow-400: #ffee58;
  --md-yellow-500: #ffeb3b;
  --md-yellow-600: #fdd835;
  --md-yellow-700: #fbc02d;
  --md-yellow-800: #f9a825;
  --md-yellow-900: #f57f17;
  --md-yellow-A100: #ffff8d;
  --md-yellow-A200: #ff0;
  --md-yellow-A400: #ffea00;
  --md-yellow-A700: #ffd600;
  --md-amber-50: #fff8e1;
  --md-amber-100: #ffecb3;
  --md-amber-200: #ffe082;
  --md-amber-300: #ffd54f;
  --md-amber-400: #ffca28;
  --md-amber-500: #ffc107;
  --md-amber-600: #ffb300;
  --md-amber-700: #ffa000;
  --md-amber-800: #ff8f00;
  --md-amber-900: #ff6f00;
  --md-amber-A100: #ffe57f;
  --md-amber-A200: #ffd740;
  --md-amber-A400: #ffc400;
  --md-amber-A700: #ffab00;
  --md-orange-50: #fff3e0;
  --md-orange-100: #ffe0b2;
  --md-orange-200: #ffcc80;
  --md-orange-300: #ffb74d;
  --md-orange-400: #ffa726;
  --md-orange-500: #ff9800;
  --md-orange-600: #fb8c00;
  --md-orange-700: #f57c00;
  --md-orange-800: #ef6c00;
  --md-orange-900: #e65100;
  --md-orange-A100: #ffd180;
  --md-orange-A200: #ffab40;
  --md-orange-A400: #ff9100;
  --md-orange-A700: #ff6d00;
  --md-deep-orange-50: #fbe9e7;
  --md-deep-orange-100: #ffccbc;
  --md-deep-orange-200: #ffab91;
  --md-deep-orange-300: #ff8a65;
  --md-deep-orange-400: #ff7043;
  --md-deep-orange-500: #ff5722;
  --md-deep-orange-600: #f4511e;
  --md-deep-orange-700: #e64a19;
  --md-deep-orange-800: #d84315;
  --md-deep-orange-900: #bf360c;
  --md-deep-orange-A100: #ff9e80;
  --md-deep-orange-A200: #ff6e40;
  --md-deep-orange-A400: #ff3d00;
  --md-deep-orange-A700: #dd2c00;
  --md-brown-50: #efebe9;
  --md-brown-100: #d7ccc8;
  --md-brown-200: #bcaaa4;
  --md-brown-300: #a1887f;
  --md-brown-400: #8d6e63;
  --md-brown-500: #795548;
  --md-brown-600: #6d4c41;
  --md-brown-700: #5d4037;
  --md-brown-800: #4e342e;
  --md-brown-900: #3e2723;
  --md-grey-50: #fafafa;
  --md-grey-100: #f5f5f5;
  --md-grey-200: #eee;
  --md-grey-300: #e0e0e0;
  --md-grey-400: #bdbdbd;
  --md-grey-500: #9e9e9e;
  --md-grey-600: #757575;
  --md-grey-700: #616161;
  --md-grey-800: #424242;
  --md-grey-900: #212121;
  --md-blue-grey-50: #eceff1;
  --md-blue-grey-100: #cfd8dc;
  --md-blue-grey-200: #b0bec5;
  --md-blue-grey-300: #90a4ae;
  --md-blue-grey-400: #78909c;
  --md-blue-grey-500: #607d8b;
  --md-blue-grey-600: #546e7a;
  --md-blue-grey-700: #455a64;
  --md-blue-grey-800: #37474f;
  --md-blue-grey-900: #263238;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| RenderedText
|----------------------------------------------------------------------------*/

:root {
  /* This is the padding value to fill the gaps between lines containing spans with background color. */
  --jp-private-code-span-padding: calc(
    (var(--jp-code-line-height) - 1) * var(--jp-code-font-size) / 2
  );
}

.jp-RenderedText {
  text-align: left;
  padding-left: var(--jp-code-padding);
  line-height: var(--jp-code-line-height);
  font-family: var(--jp-code-font-family);
}

.jp-RenderedText pre,
.jp-RenderedJavaScript pre,
.jp-RenderedHTMLCommon pre {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-code-font-size);
  border: none;
  margin: 0;
  padding: 0;
}

.jp-RenderedText pre a:link {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

.jp-RenderedText pre a:hover {
  text-decoration: underline;
  color: var(--jp-content-link-color);
}

.jp-RenderedText pre a:visited {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

/* console foregrounds and backgrounds */
.jp-RenderedText pre .ansi-black-fg {
  color: #3e424d;
}

.jp-RenderedText pre .ansi-red-fg {
  color: #e75c58;
}

.jp-RenderedText pre .ansi-green-fg {
  color: #00a250;
}

.jp-RenderedText pre .ansi-yellow-fg {
  color: #ddb62b;
}

.jp-RenderedText pre .ansi-blue-fg {
  color: #208ffb;
}

.jp-RenderedText pre .ansi-magenta-fg {
  color: #d160c4;
}

.jp-RenderedText pre .ansi-cyan-fg {
  color: #60c6c8;
}

.jp-RenderedText pre .ansi-white-fg {
  color: #c5c1b4;
}

.jp-RenderedText pre .ansi-black-bg {
  background-color: #3e424d;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-red-bg {
  background-color: #e75c58;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-green-bg {
  background-color: #00a250;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-yellow-bg {
  background-color: #ddb62b;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-blue-bg {
  background-color: #208ffb;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-magenta-bg {
  background-color: #d160c4;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-cyan-bg {
  background-color: #60c6c8;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-white-bg {
  background-color: #c5c1b4;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-black-intense-fg {
  color: #282c36;
}

.jp-RenderedText pre .ansi-red-intense-fg {
  color: #b22b31;
}

.jp-RenderedText pre .ansi-green-intense-fg {
  color: #007427;
}

.jp-RenderedText pre .ansi-yellow-intense-fg {
  color: #b27d12;
}

.jp-RenderedText pre .ansi-blue-intense-fg {
  color: #0065ca;
}

.jp-RenderedText pre .ansi-magenta-intense-fg {
  color: #a03196;
}

.jp-RenderedText pre .ansi-cyan-intense-fg {
  color: #258f8f;
}

.jp-RenderedText pre .ansi-white-intense-fg {
  color: #a1a6b2;
}

.jp-RenderedText pre .ansi-black-intense-bg {
  background-color: #282c36;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-red-intense-bg {
  background-color: #b22b31;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-green-intense-bg {
  background-color: #007427;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-yellow-intense-bg {
  background-color: #b27d12;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-blue-intense-bg {
  background-color: #0065ca;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-magenta-intense-bg {
  background-color: #a03196;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-cyan-intense-bg {
  background-color: #258f8f;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-white-intense-bg {
  background-color: #a1a6b2;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-default-inverse-fg {
  color: var(--jp-ui-inverse-font-color0);
}

.jp-RenderedText pre .ansi-default-inverse-bg {
  background-color: var(--jp-inverse-layout-color0);
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-bold {
  font-weight: bold;
}

.jp-RenderedText pre .ansi-underline {
  text-decoration: underline;
}

.jp-RenderedText[data-mime-type='application/vnd.jupyter.stderr'] {
  background: var(--jp-rendermime-error-background);
  padding-top: var(--jp-code-padding);
}

/*-----------------------------------------------------------------------------
| RenderedLatex
|----------------------------------------------------------------------------*/

.jp-RenderedLatex {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-content-font-size1);
  line-height: var(--jp-content-line-height);
}

/* Left-justify outputs.*/
.jp-OutputArea-output.jp-RenderedLatex {
  padding: var(--jp-code-padding);
  text-align: left;
}

/*-----------------------------------------------------------------------------
| RenderedHTML
|----------------------------------------------------------------------------*/

.jp-RenderedHTMLCommon {
  color: var(--jp-content-font-color1);
  font-family: var(--jp-content-font-family);
  font-size: var(--jp-content-font-size1);
  line-height: var(--jp-content-line-height);

  /* Give a bit more R padding on Markdown text to keep line lengths reasonable */
  padding-right: 20px;
}

.jp-RenderedHTMLCommon em {
  font-style: italic;
}

.jp-RenderedHTMLCommon strong {
  font-weight: bold;
}

.jp-RenderedHTMLCommon u {
  text-decoration: underline;
}

.jp-RenderedHTMLCommon a:link {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

.jp-RenderedHTMLCommon a:hover {
  text-decoration: underline;
  color: var(--jp-content-link-color);
}

.jp-RenderedHTMLCommon a:visited {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

/* Headings */

.jp-RenderedHTMLCommon h1,
.jp-RenderedHTMLCommon h2,
.jp-RenderedHTMLCommon h3,
.jp-RenderedHTMLCommon h4,
.jp-RenderedHTMLCommon h5,
.jp-RenderedHTMLCommon h6 {
  line-height: var(--jp-content-heading-line-height);
  font-weight: var(--jp-content-heading-font-weight);
  font-style: normal;
  margin: var(--jp-content-heading-margin-top) 0
    var(--jp-content-heading-margin-bottom) 0;
}

.jp-RenderedHTMLCommon h1:first-child,
.jp-RenderedHTMLCommon h2:first-child,
.jp-RenderedHTMLCommon h3:first-child,
.jp-RenderedHTMLCommon h4:first-child,
.jp-RenderedHTMLCommon h5:first-child,
.jp-RenderedHTMLCommon h6:first-child {
  margin-top: calc(0.5 * var(--jp-content-heading-margin-top));
}

.jp-RenderedHTMLCommon h1:last-child,
.jp-RenderedHTMLCommon h2:last-child,
.jp-RenderedHTMLCommon h3:last-child,
.jp-RenderedHTMLCommon h4:last-child,
.jp-RenderedHTMLCommon h5:last-child,
.jp-RenderedHTMLCommon h6:last-child {
  margin-bottom: calc(0.5 * var(--jp-content-heading-margin-bottom));
}

.jp-RenderedHTMLCommon h1 {
  font-size: var(--jp-content-font-size5);
}

.jp-RenderedHTMLCommon h2 {
  font-size: var(--jp-content-font-size4);
}

.jp-RenderedHTMLCommon h3 {
  font-size: var(--jp-content-font-size3);
}

.jp-RenderedHTMLCommon h4 {
  font-size: var(--jp-content-font-size2);
}

.jp-RenderedHTMLCommon h5 {
  font-size: var(--jp-content-font-size1);
}

.jp-RenderedHTMLCommon h6 {
  font-size: var(--jp-content-font-size0);
}

/* Lists */

/* stylelint-disable selector-max-type, selector-max-compound-selectors */

.jp-RenderedHTMLCommon ul:not(.list-inline),
.jp-RenderedHTMLCommon ol:not(.list-inline) {
  padding-left: 2em;
}

.jp-RenderedHTMLCommon ul {
  list-style: disc;
}

.jp-RenderedHTMLCommon ul ul {
  list-style: square;
}

.jp-RenderedHTMLCommon ul ul ul {
  list-style: circle;
}

.jp-RenderedHTMLCommon ol {
  list-style: decimal;
}

.jp-RenderedHTMLCommon ol ol {
  list-style: upper-alpha;
}

.jp-RenderedHTMLCommon ol ol ol {
  list-style: lower-alpha;
}

.jp-RenderedHTMLCommon ol ol ol ol {
  list-style: lower-roman;
}

.jp-RenderedHTMLCommon ol ol ol ol ol {
  list-style: decimal;
}

.jp-RenderedHTMLCommon ol,
.jp-RenderedHTMLCommon ul {
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon ul ul,
.jp-RenderedHTMLCommon ul ol,
.jp-RenderedHTMLCommon ol ul,
.jp-RenderedHTMLCommon ol ol {
  margin-bottom: 0;
}

/* stylelint-enable selector-max-type, selector-max-compound-selectors */

.jp-RenderedHTMLCommon hr {
  color: var(--jp-border-color2);
  background-color: var(--jp-border-color1);
  margin-top: 1em;
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon > pre {
  margin: 1.5em 2em;
}

.jp-RenderedHTMLCommon pre,
.jp-RenderedHTMLCommon code {
  border: 0;
  background-color: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
  font-family: var(--jp-code-font-family);
  font-size: inherit;
  line-height: var(--jp-code-line-height);
  padding: 0;
  white-space: pre-wrap;
}

.jp-RenderedHTMLCommon :not(pre) > code {
  background-color: var(--jp-layout-color2);
  padding: 1px 5px;
}

/* Tables */

.jp-RenderedHTMLCommon table {
  border-collapse: collapse;
  border-spacing: 0;
  border: none;
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  table-layout: fixed;
  margin-left: auto;
  margin-bottom: 1em;
  margin-right: auto;
}

.jp-RenderedHTMLCommon thead {
  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
  vertical-align: bottom;
}

.jp-RenderedHTMLCommon td,
.jp-RenderedHTMLCommon th,
.jp-RenderedHTMLCommon tr {
  vertical-align: middle;
  padding: 0.5em;
  line-height: normal;
  white-space: normal;
  max-width: none;
  border: none;
}

.jp-RenderedMarkdown.jp-RenderedHTMLCommon td,
.jp-RenderedMarkdown.jp-RenderedHTMLCommon th {
  max-width: none;
}

:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon td,
:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon th,
:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon tr {
  text-align: right;
}

.jp-RenderedHTMLCommon th {
  font-weight: bold;
}

.jp-RenderedHTMLCommon tbody tr:nth-child(odd) {
  background: var(--jp-layout-color0);
}

.jp-RenderedHTMLCommon tbody tr:nth-child(even) {
  background: var(--jp-rendermime-table-row-background);
}

.jp-RenderedHTMLCommon tbody tr:hover {
  background: var(--jp-rendermime-table-row-hover-background);
}

.jp-RenderedHTMLCommon p {
  text-align: left;
  margin: 0;
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon img {
  -moz-force-broken-image-icon: 1;
}

/* Restrict to direct children as other images could be nested in other content. */
.jp-RenderedHTMLCommon > img {
  display: block;
  margin-left: 0;
  margin-right: 0;
  margin-bottom: 1em;
}

/* Change color behind transparent images if they need it... */
[data-jp-theme-light='false'] .jp-RenderedImage img.jp-needs-light-background {
  background-color: var(--jp-inverse-layout-color1);
}

[data-jp-theme-light='true'] .jp-RenderedImage img.jp-needs-dark-background {
  background-color: var(--jp-inverse-layout-color1);
}

.jp-RenderedHTMLCommon img,
.jp-RenderedImage img,
.jp-RenderedHTMLCommon svg,
.jp-RenderedSVG svg {
  max-width: 100%;
  height: auto;
}

.jp-RenderedHTMLCommon img.jp-mod-unconfined,
.jp-RenderedImage img.jp-mod-unconfined,
.jp-RenderedHTMLCommon svg.jp-mod-unconfined,
.jp-RenderedSVG svg.jp-mod-unconfined {
  max-width: none;
}

.jp-RenderedHTMLCommon .alert {
  padding: var(--jp-notebook-padding);
  border: var(--jp-border-width) solid transparent;
  border-radius: var(--jp-border-radius);
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon .alert-info {
  color: var(--jp-info-color0);
  background-color: var(--jp-info-color3);
  border-color: var(--jp-info-color2);
}

.jp-RenderedHTMLCommon .alert-info hr {
  border-color: var(--jp-info-color3);
}

.jp-RenderedHTMLCommon .alert-info > p:last-child,
.jp-RenderedHTMLCommon .alert-info > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-warning {
  color: var(--jp-warn-color0);
  background-color: var(--jp-warn-color3);
  border-color: var(--jp-warn-color2);
}

.jp-RenderedHTMLCommon .alert-warning hr {
  border-color: var(--jp-warn-color3);
}

.jp-RenderedHTMLCommon .alert-warning > p:last-child,
.jp-RenderedHTMLCommon .alert-warning > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-success {
  color: var(--jp-success-color0);
  background-color: var(--jp-success-color3);
  border-color: var(--jp-success-color2);
}

.jp-RenderedHTMLCommon .alert-success hr {
  border-color: var(--jp-success-color3);
}

.jp-RenderedHTMLCommon .alert-success > p:last-child,
.jp-RenderedHTMLCommon .alert-success > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-danger {
  color: var(--jp-error-color0);
  background-color: var(--jp-error-color3);
  border-color: var(--jp-error-color2);
}

.jp-RenderedHTMLCommon .alert-danger hr {
  border-color: var(--jp-error-color3);
}

.jp-RenderedHTMLCommon .alert-danger > p:last-child,
.jp-RenderedHTMLCommon .alert-danger > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon blockquote {
  margin: 1em 2em;
  padding: 0 1em;
  border-left: 5px solid var(--jp-border-color2);
}

a.jp-InternalAnchorLink {
  visibility: hidden;
  margin-left: 8px;
  color: var(--md-blue-800);
}

h1:hover .jp-InternalAnchorLink,
h2:hover .jp-InternalAnchorLink,
h3:hover .jp-InternalAnchorLink,
h4:hover .jp-InternalAnchorLink,
h5:hover .jp-InternalAnchorLink,
h6:hover .jp-InternalAnchorLink {
  visibility: visible;
}

.jp-RenderedHTMLCommon kbd {
  background-color: var(--jp-rendermime-table-row-background);
  border: 1px solid var(--jp-border-color0);
  border-bottom-color: var(--jp-border-color2);
  border-radius: 3px;
  box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.25);
  display: inline-block;
  font-size: var(--jp-ui-font-size0);
  line-height: 1em;
  padding: 0.2em 0.5em;
}

/* Most direct children of .jp-RenderedHTMLCommon have a margin-bottom of 1.0.
 * At the bottom of cells this is a bit too much as there is also spacing
 * between cells. Going all the way to 0 gets too tight between markdown and
 * code cells.
 */
.jp-RenderedHTMLCommon > *:last-child {
  margin-bottom: 0.5em;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-cursor-backdrop {
  position: fixed;
  width: 200px;
  height: 200px;
  margin-top: -100px;
  margin-left: -100px;
  will-change: transform;
  z-index: 100;
}

.lm-mod-drag-image {
  will-change: transform;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-lineFormSearch {
  padding: 4px 12px;
  background-color: var(--jp-layout-color2);
  box-shadow: var(--jp-toolbar-box-shadow);
  z-index: 2;
  font-size: var(--jp-ui-font-size1);
}

.jp-lineFormCaption {
  font-size: var(--jp-ui-font-size0);
  line-height: var(--jp-ui-font-size1);
  margin-top: 4px;
  color: var(--jp-ui-font-color0);
}

.jp-baseLineForm {
  border: none;
  border-radius: 0;
  position: absolute;
  background-size: 16px;
  background-repeat: no-repeat;
  background-position: center;
  outline: none;
}

.jp-lineFormButtonContainer {
  top: 4px;
  right: 8px;
  height: 24px;
  padding: 0 12px;
  width: 12px;
}

.jp-lineFormButtonIcon {
  top: 0;
  right: 0;
  background-color: var(--jp-brand-color1);
  height: 100%;
  width: 100%;
  box-sizing: border-box;
  padding: 4px 6px;
}

.jp-lineFormButton {
  top: 0;
  right: 0;
  background-color: transparent;
  height: 100%;
  width: 100%;
  box-sizing: border-box;
}

.jp-lineFormWrapper {
  overflow: hidden;
  padding: 0 8px;
  border: 1px solid var(--jp-border-color0);
  background-color: var(--jp-input-active-background);
  height: 22px;
}

.jp-lineFormWrapperFocusWithin {
  border: var(--jp-border-width) solid var(--md-blue-500);
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-lineFormInput {
  background: transparent;
  width: 200px;
  height: 100%;
  border: none;
  outline: none;
  color: var(--jp-ui-font-color0);
  line-height: 28px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-JSONEditor {
  display: flex;
  flex-direction: column;
  width: 100%;
}

.jp-JSONEditor-host {
  flex: 1 1 auto;
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  border-radius: 0;
  background: var(--jp-layout-color0);
  min-height: 50px;
  padding: 1px;
}

.jp-JSONEditor.jp-mod-error .jp-JSONEditor-host {
  border-color: red;
  outline-color: red;
}

.jp-JSONEditor-header {
  display: flex;
  flex: 1 0 auto;
  padding: 0 0 0 12px;
}

.jp-JSONEditor-header label {
  flex: 0 0 auto;
}

.jp-JSONEditor-commitButton {
  height: 16px;
  width: 16px;
  background-size: 18px;
  background-repeat: no-repeat;
  background-position: center;
}

.jp-JSONEditor-host.jp-mod-focused {
  background-color: var(--jp-input-active-background);
  border: 1px solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
}

.jp-Editor.jp-mod-dropTarget {
  border: var(--jp-border-width) solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/
.jp-DocumentSearch-input {
  border: none;
  outline: none;
  color: var(--jp-ui-font-color0);
  font-size: var(--jp-ui-font-size1);
  background-color: var(--jp-layout-color0);
  font-family: var(--jp-ui-font-family);
  padding: 2px 1px;
  resize: none;
}

.jp-DocumentSearch-overlay {
  position: absolute;
  background-color: var(--jp-toolbar-background);
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  border-left: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  top: 0;
  right: 0;
  z-index: 7;
  min-width: 405px;
  padding: 2px;
  font-size: var(--jp-ui-font-size1);

  --jp-private-document-search-button-height: 20px;
}

.jp-DocumentSearch-overlay button {
  background-color: var(--jp-toolbar-background);
  outline: 0;
}

.jp-DocumentSearch-overlay button:hover {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-overlay button:active {
  background-color: var(--jp-layout-color3);
}

.jp-DocumentSearch-overlay-row {
  display: flex;
  align-items: center;
  margin-bottom: 2px;
}

.jp-DocumentSearch-button-content {
  display: inline-block;
  cursor: pointer;
  box-sizing: border-box;
  width: 100%;
  height: 100%;
}

.jp-DocumentSearch-button-content svg {
  width: 100%;
  height: 100%;
}

.jp-DocumentSearch-input-wrapper {
  border: var(--jp-border-width) solid var(--jp-border-color0);
  display: flex;
  background-color: var(--jp-layout-color0);
  margin: 2px;
}

.jp-DocumentSearch-input-wrapper:focus-within {
  border-color: var(--jp-cell-editor-active-border-color);
}

.jp-DocumentSearch-toggle-wrapper,
.jp-DocumentSearch-button-wrapper {
  all: initial;
  overflow: hidden;
  display: inline-block;
  border: none;
  box-sizing: border-box;
}

.jp-DocumentSearch-toggle-wrapper {
  width: 14px;
  height: 14px;
}

.jp-DocumentSearch-button-wrapper {
  width: var(--jp-private-document-search-button-height);
  height: var(--jp-private-document-search-button-height);
}

.jp-DocumentSearch-toggle-wrapper:focus,
.jp-DocumentSearch-button-wrapper:focus {
  outline: var(--jp-border-width) solid
    var(--jp-cell-editor-active-border-color);
  outline-offset: -1px;
}

.jp-DocumentSearch-toggle-wrapper,
.jp-DocumentSearch-button-wrapper,
.jp-DocumentSearch-button-content:focus {
  outline: none;
}

.jp-DocumentSearch-toggle-placeholder {
  width: 5px;
}

.jp-DocumentSearch-input-button::before {
  display: block;
  padding-top: 100%;
}

.jp-DocumentSearch-input-button-off {
  opacity: var(--jp-search-toggle-off-opacity);
}

.jp-DocumentSearch-input-button-off:hover {
  opacity: var(--jp-search-toggle-hover-opacity);
}

.jp-DocumentSearch-input-button-on {
  opacity: var(--jp-search-toggle-on-opacity);
}

.jp-DocumentSearch-index-counter {
  padding-left: 10px;
  padding-right: 10px;
  user-select: none;
  min-width: 35px;
  display: inline-block;
}

.jp-DocumentSearch-up-down-wrapper {
  display: inline-block;
  padding-right: 2px;
  margin-left: auto;
  white-space: nowrap;
}

.jp-DocumentSearch-spacer {
  margin-left: auto;
}

.jp-DocumentSearch-up-down-wrapper button {
  outline: 0;
  border: none;
  width: var(--jp-private-document-search-button-height);
  height: var(--jp-private-document-search-button-height);
  vertical-align: middle;
  margin: 1px 5px 2px;
}

.jp-DocumentSearch-up-down-button:hover {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-up-down-button:active {
  background-color: var(--jp-layout-color3);
}

.jp-DocumentSearch-filter-button {
  border-radius: var(--jp-border-radius);
}

.jp-DocumentSearch-filter-button:hover {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-filter-button-enabled {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-filter-button-enabled:hover {
  background-color: var(--jp-layout-color3);
}

.jp-DocumentSearch-search-options {
  padding: 0 8px;
  margin-left: 3px;
  width: 100%;
  display: grid;
  justify-content: start;
  grid-template-columns: 1fr 1fr;
  align-items: center;
  justify-items: stretch;
}

.jp-DocumentSearch-search-filter-disabled {
  color: var(--jp-ui-font-color2);
}

.jp-DocumentSearch-search-filter {
  display: flex;
  align-items: center;
  user-select: none;
}

.jp-DocumentSearch-regex-error {
  color: var(--jp-error-color0);
}

.jp-DocumentSearch-replace-button-wrapper {
  overflow: hidden;
  display: inline-block;
  box-sizing: border-box;
  border: var(--jp-border-width) solid var(--jp-border-color0);
  margin: auto 2px;
  padding: 1px 4px;
  height: calc(var(--jp-private-document-search-button-height) + 2px);
}

.jp-DocumentSearch-replace-button-wrapper:focus {
  border: var(--jp-border-width) solid var(--jp-cell-editor-active-border-color);
}

.jp-DocumentSearch-replace-button {
  display: inline-block;
  text-align: center;
  cursor: pointer;
  box-sizing: border-box;
  color: var(--jp-ui-font-color1);

  /* height - 2 * (padding of wrapper) */
  line-height: calc(var(--jp-private-document-search-button-height) - 2px);
  width: 100%;
  height: 100%;
}

.jp-DocumentSearch-replace-button:focus {
  outline: none;
}

.jp-DocumentSearch-replace-wrapper-class {
  margin-left: 14px;
  display: flex;
}

.jp-DocumentSearch-replace-toggle {
  border: none;
  background-color: var(--jp-toolbar-background);
  border-radius: var(--jp-border-radius);
}

.jp-DocumentSearch-replace-toggle:hover {
  background-color: var(--jp-layout-color2);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.cm-editor {
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  font-family: var(--jp-code-font-family);
  border: 0;
  border-radius: 0;
  height: auto;

  /* Changed to auto to autogrow */
}

.cm-editor pre {
  padding: 0 var(--jp-code-padding);
}

.jp-CodeMirrorEditor[data-type='inline'] .cm-dialog {
  background-color: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
}

.jp-CodeMirrorEditor {
  cursor: text;
}

/* When zoomed out 67% and 33% on a screen of 1440 width x 900 height */
@media screen and (min-width: 2138px) and (max-width: 4319px) {
  .jp-CodeMirrorEditor[data-type='inline'] .cm-cursor {
    border-left: var(--jp-code-cursor-width1) solid
      var(--jp-editor-cursor-color);
  }
}

/* When zoomed out less than 33% */
@media screen and (min-width: 4320px) {
  .jp-CodeMirrorEditor[data-type='inline'] .cm-cursor {
    border-left: var(--jp-code-cursor-width2) solid
      var(--jp-editor-cursor-color);
  }
}

.cm-editor.jp-mod-readOnly .cm-cursor {
  display: none;
}

.jp-CollaboratorCursor {
  border-left: 5px solid transparent;
  border-right: 5px solid transparent;
  border-top: none;
  border-bottom: 3px solid;
  background-clip: content-box;
  margin-left: -5px;
  margin-right: -5px;
}

.cm-searching,
.cm-searching span {
  /* `.cm-searching span`: we need to override syntax highlighting */
  background-color: var(--jp-search-unselected-match-background-color);
  color: var(--jp-search-unselected-match-color);
}

.cm-searching::selection,
.cm-searching span::selection {
  background-color: var(--jp-search-unselected-match-background-color);
  color: var(--jp-search-unselected-match-color);
}

.jp-current-match > .cm-searching,
.jp-current-match > .cm-searching span,
.cm-searching > .jp-current-match,
.cm-searching > .jp-current-match span {
  background-color: var(--jp-search-selected-match-background-color);
  color: var(--jp-search-selected-match-color);
}

.jp-current-match > .cm-searching::selection,
.cm-searching > .jp-current-match::selection,
.jp-current-match > .cm-searching span::selection {
  background-color: var(--jp-search-selected-match-background-color);
  color: var(--jp-search-selected-match-color);
}

.cm-trailingspace {
  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAFCAYAAAB4ka1VAAAAsElEQVQIHQGlAFr/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA7+r3zKmT0/+pk9P/7+r3zAAAAAAAAAAABAAAAAAAAAAA6OPzM+/q9wAAAAAA6OPzMwAAAAAAAAAAAgAAAAAAAAAAGR8NiRQaCgAZIA0AGR8NiQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQyoYJ/SY80UAAAAASUVORK5CYII=);
  background-position: center left;
  background-repeat: repeat-x;
}

.jp-CollaboratorCursor-hover {
  position: absolute;
  z-index: 1;
  transform: translateX(-50%);
  color: white;
  border-radius: 3px;
  padding-left: 4px;
  padding-right: 4px;
  padding-top: 1px;
  padding-bottom: 1px;
  text-align: center;
  font-size: var(--jp-ui-font-size1);
  white-space: nowrap;
}

.jp-CodeMirror-ruler {
  border-left: 1px dashed var(--jp-border-color2);
}

/* Styles for shared cursors (remote cursor locations and selected ranges) */
.jp-CodeMirrorEditor .cm-ySelectionCaret {
  position: relative;
  border-left: 1px solid black;
  margin-left: -1px;
  margin-right: -1px;
  box-sizing: border-box;
}

.jp-CodeMirrorEditor .cm-ySelectionCaret > .cm-ySelectionInfo {
  white-space: nowrap;
  position: absolute;
  top: -1.15em;
  padding-bottom: 0.05em;
  left: -1px;
  font-size: 0.95em;
  font-family: var(--jp-ui-font-family);
  font-weight: bold;
  line-height: normal;
  user-select: none;
  color: white;
  padding-left: 2px;
  padding-right: 2px;
  z-index: 101;
  transition: opacity 0.3s ease-in-out;
}

.jp-CodeMirrorEditor .cm-ySelectionInfo {
  transition-delay: 0.7s;
  opacity: 0;
}

.jp-CodeMirrorEditor .cm-ySelectionCaret:hover > .cm-ySelectionInfo {
  opacity: 1;
  transition-delay: 0s;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-MimeDocument {
  outline: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-filebrowser-button-height: 28px;
  --jp-private-filebrowser-button-width: 48px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-FileBrowser .jp-SidePanel-content {
  display: flex;
  flex-direction: column;
}

.jp-FileBrowser-toolbar.jp-Toolbar {
  flex-wrap: wrap;
  row-gap: 12px;
  border-bottom: none;
  height: auto;
  margin: 8px 12px 0;
  box-shadow: none;
  padding: 0;
  justify-content: flex-start;
}

.jp-FileBrowser-Panel {
  flex: 1 1 auto;
  display: flex;
  flex-direction: column;
}

.jp-BreadCrumbs {
  flex: 0 0 auto;
  margin: 8px 12px;
}

.jp-BreadCrumbs-item {
  margin: 0 2px;
  padding: 0 2px;
  border-radius: var(--jp-border-radius);
  cursor: pointer;
}

.jp-BreadCrumbs-item:hover {
  background-color: var(--jp-layout-color2);
}

.jp-BreadCrumbs-item:first-child {
  margin-left: 0;
}

.jp-BreadCrumbs-item.jp-mod-dropTarget {
  background-color: var(--jp-brand-color2);
  opacity: 0.7;
}

/*-----------------------------------------------------------------------------
| Buttons
|----------------------------------------------------------------------------*/

.jp-FileBrowser-toolbar > .jp-Toolbar-item {
  flex: 0 0 auto;
  padding-left: 0;
  padding-right: 2px;
  align-items: center;
  height: unset;
}

.jp-FileBrowser-toolbar > .jp-Toolbar-item .jp-ToolbarButtonComponent {
  width: 40px;
}

/*-----------------------------------------------------------------------------
| Other styles
|----------------------------------------------------------------------------*/

.jp-FileDialog.jp-mod-conflict input {
  color: var(--jp-error-color1);
}

.jp-FileDialog .jp-new-name-title {
  margin-top: 12px;
}

.jp-LastModified-hidden {
  display: none;
}

.jp-FileSize-hidden {
  display: none;
}

.jp-FileBrowser .lm-AccordionPanel > h3:first-child {
  display: none;
}

/*-----------------------------------------------------------------------------
| DirListing
|----------------------------------------------------------------------------*/

.jp-DirListing {
  flex: 1 1 auto;
  display: flex;
  flex-direction: column;
  outline: 0;
}

.jp-DirListing-header {
  flex: 0 0 auto;
  display: flex;
  flex-direction: row;
  align-items: center;
  overflow: hidden;
  border-top: var(--jp-border-width) solid var(--jp-border-color2);
  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
  box-shadow: var(--jp-toolbar-box-shadow);
  z-index: 2;
}

.jp-DirListing-headerItem {
  padding: 4px 12px 2px;
  font-weight: 500;
}

.jp-DirListing-headerItem:hover {
  background: var(--jp-layout-color2);
}

.jp-DirListing-headerItem.jp-id-name {
  flex: 1 0 84px;
}

.jp-DirListing-headerItem.jp-id-modified {
  flex: 0 0 112px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
}

.jp-DirListing-headerItem.jp-id-filesize {
  flex: 0 0 75px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
}

.jp-id-narrow {
  display: none;
  flex: 0 0 5px;
  padding: 4px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
  color: var(--jp-border-color2);
}

.jp-DirListing-narrow .jp-id-narrow {
  display: block;
}

.jp-DirListing-narrow .jp-id-modified,
.jp-DirListing-narrow .jp-DirListing-itemModified {
  display: none;
}

.jp-DirListing-headerItem.jp-mod-selected {
  font-weight: 600;
}

/* increase specificity to override bundled default */
.jp-DirListing-content {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  list-style-type: none;
  overflow: auto;
  background-color: var(--jp-layout-color1);
}

.jp-DirListing-content mark {
  color: var(--jp-ui-font-color0);
  background-color: transparent;
  font-weight: bold;
}

.jp-DirListing-content .jp-DirListing-item.jp-mod-selected mark {
  color: var(--jp-ui-inverse-font-color0);
}

/* Style the directory listing content when a user drops a file to upload */
.jp-DirListing.jp-mod-native-drop .jp-DirListing-content {
  outline: 5px dashed rgba(128, 128, 128, 0.5);
  outline-offset: -10px;
  cursor: copy;
}

.jp-DirListing-item {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding: 4px 12px;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-DirListing-checkboxWrapper {
  /* Increases hit area of checkbox. */
  padding: 4px;
}

.jp-DirListing-header
  .jp-DirListing-checkboxWrapper
  + .jp-DirListing-headerItem {
  padding-left: 4px;
}

.jp-DirListing-content .jp-DirListing-checkboxWrapper {
  position: relative;
  left: -4px;
  margin: -4px 0 -4px -8px;
}

.jp-DirListing-checkboxWrapper.jp-mod-visible {
  visibility: visible;
}

/* For devices that support hovering, hide checkboxes until hovered, selected...
*/
@media (hover: hover) {
  .jp-DirListing-checkboxWrapper {
    visibility: hidden;
  }

  .jp-DirListing-item:hover .jp-DirListing-checkboxWrapper,
  .jp-DirListing-item.jp-mod-selected .jp-DirListing-checkboxWrapper {
    visibility: visible;
  }
}

.jp-DirListing-item[data-is-dot] {
  opacity: 75%;
}

.jp-DirListing-item.jp-mod-selected {
  color: var(--jp-ui-inverse-font-color1);
  background: var(--jp-brand-color1);
}

.jp-DirListing-item.jp-mod-dropTarget {
  background: var(--jp-brand-color3);
}

.jp-DirListing-item:hover:not(.jp-mod-selected) {
  background: var(--jp-layout-color2);
}

.jp-DirListing-itemIcon {
  flex: 0 0 20px;
  margin-right: 4px;
}

.jp-DirListing-itemText {
  flex: 1 0 64px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  user-select: none;
}

.jp-DirListing-itemText:focus {
  outline-width: 2px;
  outline-color: var(--jp-inverse-layout-color1);
  outline-style: solid;
  outline-offset: 1px;
}

.jp-DirListing-item.jp-mod-selected .jp-DirListing-itemText:focus {
  outline-color: var(--jp-layout-color1);
}

.jp-DirListing-itemModified {
  flex: 0 0 125px;
  text-align: right;
}

.jp-DirListing-itemFileSize {
  flex: 0 0 90px;
  text-align: right;
}

.jp-DirListing-editor {
  flex: 1 0 64px;
  outline: none;
  border: none;
  color: var(--jp-ui-font-color1);
  background-color: var(--jp-layout-color1);
}

.jp-DirListing-item.jp-mod-running .jp-DirListing-itemIcon::before {
  color: var(--jp-success-color1);
  content: '\25CF';
  font-size: 8px;
  position: absolute;
  left: -8px;
}

.jp-DirListing-item.jp-mod-running.jp-mod-selected
  .jp-DirListing-itemIcon::before {
  color: var(--jp-ui-inverse-font-color1);
}

.jp-DirListing-item.lm-mod-drag-image,
.jp-DirListing-item.jp-mod-selected.lm-mod-drag-image {
  font-size: var(--jp-ui-font-size1);
  padding-left: 4px;
  margin-left: 4px;
  width: 160px;
  background-color: var(--jp-ui-inverse-font-color2);
  box-shadow: var(--jp-elevation-z2);
  border-radius: 0;
  color: var(--jp-ui-font-color1);
  transform: translateX(-40%) translateY(-58%);
}

.jp-Document {
  min-width: 120px;
  min-height: 120px;
  outline: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Main OutputArea
| OutputArea has a list of Outputs
|----------------------------------------------------------------------------*/

.jp-OutputArea {
  overflow-y: auto;
}

.jp-OutputArea-child {
  display: table;
  table-layout: fixed;
  width: 100%;
  overflow: hidden;
}

.jp-OutputPrompt {
  width: var(--jp-cell-prompt-width);
  color: var(--jp-cell-outprompt-font-color);
  font-family: var(--jp-cell-prompt-font-family);
  padding: var(--jp-code-padding);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
  opacity: var(--jp-cell-prompt-opacity);

  /* Right align prompt text, don't wrap to handle large prompt numbers */
  text-align: right;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;

  /* Disable text selection */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-OutputArea-prompt {
  display: table-cell;
  vertical-align: top;
}

.jp-OutputArea-output {
  display: table-cell;
  width: 100%;
  height: auto;
  overflow: auto;
  user-select: text;
  -moz-user-select: text;
  -webkit-user-select: text;
  -ms-user-select: text;
}

.jp-OutputArea .jp-RenderedText {
  padding-left: 1ch;
}

/**
 * Prompt overlay.
 */

.jp-OutputArea-promptOverlay {
  position: absolute;
  top: 0;
  width: var(--jp-cell-prompt-width);
  height: 100%;
  opacity: 0.5;
}

.jp-OutputArea-promptOverlay:hover {
  background: var(--jp-layout-color2);
  box-shadow: inset 0 0 1px var(--jp-inverse-layout-color0);
  cursor: zoom-out;
}

.jp-mod-outputsScrolled .jp-OutputArea-promptOverlay:hover {
  cursor: zoom-in;
}

/**
 * Isolated output.
 */
.jp-OutputArea-output.jp-mod-isolated {
  width: 100%;
  display: block;
}

/*
When drag events occur, `lm-mod-override-cursor` is added to the body.
Because iframes steal all cursor events, the following two rules are necessary
to suppress pointer events while resize drags are occurring. There may be a
better solution to this problem.
*/
body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated {
  position: relative;
}

body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated::before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: transparent;
}

/* pre */

.jp-OutputArea-output pre {
  border: none;
  margin: 0;
  padding: 0;
  overflow-x: auto;
  overflow-y: auto;
  word-break: break-all;
  word-wrap: break-word;
  white-space: pre-wrap;
}

/* tables */

.jp-OutputArea-output.jp-RenderedHTMLCommon table {
  margin-left: 0;
  margin-right: 0;
}

/* description lists */

.jp-OutputArea-output dl,
.jp-OutputArea-output dt,
.jp-OutputArea-output dd {
  display: block;
}

.jp-OutputArea-output dl {
  width: 100%;
  overflow: hidden;
  padding: 0;
  margin: 0;
}

.jp-OutputArea-output dt {
  font-weight: bold;
  float: left;
  width: 20%;
  padding: 0;
  margin: 0;
}

.jp-OutputArea-output dd {
  float: left;
  width: 80%;
  padding: 0;
  margin: 0;
}

.jp-TrimmedOutputs pre {
  background: var(--jp-layout-color3);
  font-size: calc(var(--jp-code-font-size) * 1.4);
  text-align: center;
  text-transform: uppercase;
}

/* Hide the gutter in case of
 *  - nested output areas (e.g. in the case of output widgets)
 *  - mirrored output areas
 */
.jp-OutputArea .jp-OutputArea .jp-OutputArea-prompt {
  display: none;
}

/* Hide empty lines in the output area, for instance due to cleared widgets */
.jp-OutputArea-prompt:empty {
  padding: 0;
  border: 0;
}

/*-----------------------------------------------------------------------------
| executeResult is added to any Output-result for the display of the object
| returned by a cell
|----------------------------------------------------------------------------*/

.jp-OutputArea-output.jp-OutputArea-executeResult {
  margin-left: 0;
  width: 100%;
}

/* Text output with the Out[] prompt needs a top padding to match the
 * alignment of the Out[] prompt itself.
 */
.jp-OutputArea-executeResult .jp-RenderedText.jp-OutputArea-output {
  padding-top: var(--jp-code-padding);
  border-top: var(--jp-border-width) solid transparent;
}

/*-----------------------------------------------------------------------------
| The Stdin output
|----------------------------------------------------------------------------*/

.jp-Stdin-prompt {
  color: var(--jp-content-font-color0);
  padding-right: var(--jp-code-padding);
  vertical-align: baseline;
  flex: 0 0 auto;
}

.jp-Stdin-input {
  font-family: var(--jp-code-font-family);
  font-size: inherit;
  color: inherit;
  background-color: inherit;
  width: 42%;
  min-width: 200px;

  /* make sure input baseline aligns with prompt */
  vertical-align: baseline;

  /* padding + margin = 0.5em between prompt and cursor */
  padding: 0 0.25em;
  margin: 0 0.25em;
  flex: 0 0 70%;
}

.jp-Stdin-input::placeholder {
  opacity: 0;
}

.jp-Stdin-input:focus {
  box-shadow: none;
}

.jp-Stdin-input:focus::placeholder {
  opacity: 1;
}

/*-----------------------------------------------------------------------------
| Output Area View
|----------------------------------------------------------------------------*/

.jp-LinkedOutputView .jp-OutputArea {
  height: 100%;
  display: block;
}

.jp-LinkedOutputView .jp-OutputArea-output:only-child {
  height: 100%;
}

/*-----------------------------------------------------------------------------
| Printing
|----------------------------------------------------------------------------*/

@media print {
  .jp-OutputArea-child {
    break-inside: avoid-page;
  }
}

/*-----------------------------------------------------------------------------
| Mobile
|----------------------------------------------------------------------------*/
@media only screen and (max-width: 760px) {
  .jp-OutputPrompt {
    display: table-row;
    text-align: left;
  }

  .jp-OutputArea-child .jp-OutputArea-output {
    display: table-row;
    margin-left: var(--jp-notebook-padding);
  }
}

/* Trimmed outputs warning */
.jp-TrimmedOutputs > a {
  margin: 10px;
  text-decoration: none;
  cursor: pointer;
}

.jp-TrimmedOutputs > a:hover {
  text-decoration: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Table of Contents
|----------------------------------------------------------------------------*/

:root {
  --jp-private-toc-active-width: 4px;
}

.jp-TableOfContents {
  display: flex;
  flex-direction: column;
  background: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  height: 100%;
}

.jp-TableOfContents-placeholder {
  text-align: center;
}

.jp-TableOfContents-placeholderContent {
  color: var(--jp-content-font-color2);
  padding: 8px;
}

.jp-TableOfContents-placeholderContent > h3 {
  margin-bottom: var(--jp-content-heading-margin-bottom);
}

.jp-TableOfContents .jp-SidePanel-content {
  overflow-y: auto;
}

.jp-TableOfContents-tree {
  margin: 4px;
}

.jp-TableOfContents ol {
  list-style-type: none;
}

/* stylelint-disable-next-line selector-max-type */
.jp-TableOfContents li > ol {
  /* Align left border with triangle icon center */
  padding-left: 11px;
}

.jp-TableOfContents-content {
  /* left margin for the active heading indicator */
  margin: 0 0 0 var(--jp-private-toc-active-width);
  padding: 0;
  background-color: var(--jp-layout-color1);
}

.jp-tocItem {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-tocItem-heading {
  display: flex;
  cursor: pointer;
}

.jp-tocItem-heading:hover {
  background-color: var(--jp-layout-color2);
}

.jp-tocItem-content {
  display: block;
  padding: 4px 0;
  white-space: nowrap;
  text-overflow: ellipsis;
  overflow-x: hidden;
}

.jp-tocItem-collapser {
  height: 20px;
  margin: 2px 2px 0;
  padding: 0;
  background: none;
  border: none;
  cursor: pointer;
}

.jp-tocItem-collapser:hover {
  background-color: var(--jp-layout-color3);
}

/* Active heading indicator */

.jp-tocItem-heading::before {
  content: ' ';
  background: transparent;
  width: var(--jp-private-toc-active-width);
  height: 24px;
  position: absolute;
  left: 0;
  border-radius: var(--jp-border-radius);
}

.jp-tocItem-heading.jp-tocItem-active::before {
  background-color: var(--jp-brand-color1);
}

.jp-tocItem-heading:hover.jp-tocItem-active::before {
  background: var(--jp-brand-color0);
  opacity: 1;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Collapser {
  flex: 0 0 var(--jp-cell-collapser-width);
  padding: 0;
  margin: 0;
  border: none;
  outline: none;
  background: transparent;
  border-radius: var(--jp-border-radius);
  opacity: 1;
}

.jp-Collapser-child {
  display: block;
  width: 100%;
  box-sizing: border-box;

  /* height: 100% doesn't work because the height of its parent is computed from content */
  position: absolute;
  top: 0;
  bottom: 0;
}

/*-----------------------------------------------------------------------------
| Printing
|----------------------------------------------------------------------------*/

/*
Hiding collapsers in print mode.

Note: input and output wrappers have "display: block" propery in print mode.
*/

@media print {
  .jp-Collapser {
    display: none;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Header/Footer
|----------------------------------------------------------------------------*/

/* Hidden by zero height by default */
.jp-CellHeader,
.jp-CellFooter {
  height: 0;
  width: 100%;
  padding: 0;
  margin: 0;
  border: none;
  outline: none;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Input
|----------------------------------------------------------------------------*/

/* All input areas */
.jp-InputArea {
  display: table;
  table-layout: fixed;
  width: 100%;
  overflow: hidden;
}

.jp-InputArea-editor {
  display: table-cell;
  overflow: hidden;
  vertical-align: top;

  /* This is the non-active, default styling */
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  border-radius: 0;
  background: var(--jp-cell-editor-background);
}

.jp-InputPrompt {
  display: table-cell;
  vertical-align: top;
  width: var(--jp-cell-prompt-width);
  color: var(--jp-cell-inprompt-font-color);
  font-family: var(--jp-cell-prompt-font-family);
  padding: var(--jp-code-padding);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  opacity: var(--jp-cell-prompt-opacity);
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;

  /* Right align prompt text, don't wrap to handle large prompt numbers */
  text-align: right;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;

  /* Disable text selection */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

/*-----------------------------------------------------------------------------
| Mobile
|----------------------------------------------------------------------------*/
@media only screen and (max-width: 760px) {
  .jp-InputArea-editor {
    display: table-row;
    margin-left: var(--jp-notebook-padding);
  }

  .jp-InputPrompt {
    display: table-row;
    text-align: left;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Placeholder
|----------------------------------------------------------------------------*/

.jp-Placeholder {
  display: table;
  table-layout: fixed;
  width: 100%;
}

.jp-Placeholder-prompt {
  display: table-cell;
  box-sizing: border-box;
}

.jp-Placeholder-content {
  display: table-cell;
  padding: 4px 6px;
  border: 1px solid transparent;
  border-radius: 0;
  background: none;
  box-sizing: border-box;
  cursor: pointer;
}

.jp-Placeholder-contentContainer {
  display: flex;
}

.jp-Placeholder-content:hover,
.jp-InputPlaceholder > .jp-Placeholder-content:hover {
  border-color: var(--jp-layout-color3);
}

.jp-Placeholder-content .jp-MoreHorizIcon {
  width: 32px;
  height: 16px;
  border: 1px solid transparent;
  border-radius: var(--jp-border-radius);
}

.jp-Placeholder-content .jp-MoreHorizIcon:hover {
  border: 1px solid var(--jp-border-color1);
  box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.25);
  background-color: var(--jp-layout-color0);
}

.jp-PlaceholderText {
  white-space: nowrap;
  overflow-x: hidden;
  color: var(--jp-inverse-layout-color3);
  font-family: var(--jp-code-font-family);
}

.jp-InputPlaceholder > .jp-Placeholder-content {
  border-color: var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Private CSS variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-cell-scrolling-output-offset: 5px;
}

/*-----------------------------------------------------------------------------
| Cell
|----------------------------------------------------------------------------*/

.jp-Cell {
  padding: var(--jp-cell-padding);
  margin: 0;
  border: none;
  outline: none;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Common input/output
|----------------------------------------------------------------------------*/

.jp-Cell-inputWrapper,
.jp-Cell-outputWrapper {
  display: flex;
  flex-direction: row;
  padding: 0;
  margin: 0;

  /* Added to reveal the box-shadow on the input and output collapsers. */
  overflow: visible;
}

/* Only input/output areas inside cells */
.jp-Cell-inputArea,
.jp-Cell-outputArea {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Collapser
|----------------------------------------------------------------------------*/

/* Make the output collapser disappear when there is not output, but do so
 * in a manner that leaves it in the layout and preserves its width.
 */
.jp-Cell.jp-mod-noOutputs .jp-Cell-outputCollapser {
  border: none !important;
  background: transparent !important;
}

.jp-Cell:not(.jp-mod-noOutputs) .jp-Cell-outputCollapser {
  min-height: var(--jp-cell-collapser-min-height);
}

/*-----------------------------------------------------------------------------
| Output
|----------------------------------------------------------------------------*/

/* Put a space between input and output when there IS output */
.jp-Cell:not(.jp-mod-noOutputs) .jp-Cell-outputWrapper {
  margin-top: 5px;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea {
  overflow-y: auto;
  max-height: 24em;
  margin-left: var(--jp-private-cell-scrolling-output-offset);
  resize: vertical;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea[style*='height'] {
  max-height: unset;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea::after {
  content: ' ';
  box-shadow: inset 0 0 6px 2px rgb(0 0 0 / 30%);
  width: 100%;
  height: 100%;
  position: sticky;
  bottom: 0;
  top: 0;
  margin-top: -50%;
  float: left;
  display: block;
  pointer-events: none;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-child {
  padding-top: 6px;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-prompt {
  width: calc(
    var(--jp-cell-prompt-width) - var(--jp-private-cell-scrolling-output-offset)
  );
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-promptOverlay {
  left: calc(-1 * var(--jp-private-cell-scrolling-output-offset));
}

/*-----------------------------------------------------------------------------
| CodeCell
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| MarkdownCell
|----------------------------------------------------------------------------*/

.jp-MarkdownOutput {
  display: table-cell;
  width: 100%;
  margin-top: 0;
  margin-bottom: 0;
  padding-left: var(--jp-code-padding);
}

.jp-MarkdownOutput.jp-RenderedHTMLCommon {
  overflow: auto;
}

/* collapseHeadingButton (show always if hiddenCellsButton is _not_ shown) */
.jp-collapseHeadingButton {
  display: flex;
  min-height: var(--jp-cell-collapser-min-height);
  font-size: var(--jp-code-font-size);
  position: absolute;
  background-color: transparent;
  background-size: 25px;
  background-repeat: no-repeat;
  background-position-x: center;
  background-position-y: top;
  background-image: var(--jp-icon-caret-down);
  right: 0;
  top: 0;
  bottom: 0;
}

.jp-collapseHeadingButton.jp-mod-collapsed {
  background-image: var(--jp-icon-caret-right);
}

/*
 set the container font size to match that of content
 so that the nested collapse buttons have the right size
*/
.jp-MarkdownCell .jp-InputPrompt {
  font-size: var(--jp-content-font-size1);
}

/*
  Align collapseHeadingButton with cell top header
  The font sizes are identical to the ones in packages/rendermime/style/base.css
*/
.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='1'] {
  font-size: var(--jp-content-font-size5);
  background-position-y: calc(0.3 * var(--jp-content-font-size5));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='2'] {
  font-size: var(--jp-content-font-size4);
  background-position-y: calc(0.3 * var(--jp-content-font-size4));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='3'] {
  font-size: var(--jp-content-font-size3);
  background-position-y: calc(0.3 * var(--jp-content-font-size3));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='4'] {
  font-size: var(--jp-content-font-size2);
  background-position-y: calc(0.3 * var(--jp-content-font-size2));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='5'] {
  font-size: var(--jp-content-font-size1);
  background-position-y: top;
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='6'] {
  font-size: var(--jp-content-font-size0);
  background-position-y: top;
}

/* collapseHeadingButton (show only on (hover,active) if hiddenCellsButton is shown) */
.jp-Notebook.jp-mod-showHiddenCellsButton .jp-collapseHeadingButton {
  display: none;
}

.jp-Notebook.jp-mod-showHiddenCellsButton
  :is(.jp-MarkdownCell:hover, .jp-mod-active)
  .jp-collapseHeadingButton {
  display: flex;
}

/* showHiddenCellsButton (only show if jp-mod-showHiddenCellsButton is set, which
is a consequence of the showHiddenCellsButton option in Notebook Settings)*/
.jp-Notebook.jp-mod-showHiddenCellsButton .jp-showHiddenCellsButton {
  margin-left: calc(var(--jp-cell-prompt-width) + 2 * var(--jp-code-padding));
  margin-top: var(--jp-code-padding);
  border: 1px solid var(--jp-border-color2);
  background-color: var(--jp-border-color3) !important;
  color: var(--jp-content-font-color0) !important;
  display: flex;
}

.jp-Notebook.jp-mod-showHiddenCellsButton .jp-showHiddenCellsButton:hover {
  background-color: var(--jp-border-color2) !important;
}

.jp-showHiddenCellsButton {
  display: none;
}

/*-----------------------------------------------------------------------------
| Printing
|----------------------------------------------------------------------------*/

/*
Using block instead of flex to allow the use of the break-inside CSS property for
cell outputs.
*/

@media print {
  .jp-Cell-inputWrapper,
  .jp-Cell-outputWrapper {
    display: block;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-notebook-toolbar-padding: 2px 5px 2px 2px;
}

/*-----------------------------------------------------------------------------

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-NotebookPanel-toolbar {
  padding: var(--jp-notebook-toolbar-padding);

  /* disable paint containment from lumino 2.0 default strict CSS containment */
  contain: style size !important;
}

.jp-Toolbar-item.jp-Notebook-toolbarCellType .jp-select-wrapper.jp-mod-focused {
  border: none;
  box-shadow: none;
}

.jp-Notebook-toolbarCellTypeDropdown select {
  height: 24px;
  font-size: var(--jp-ui-font-size1);
  line-height: 14px;
  border-radius: 0;
  display: block;
}

.jp-Notebook-toolbarCellTypeDropdown span {
  top: 5px !important;
}

.jp-Toolbar-responsive-popup {
  position: absolute;
  height: fit-content;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: flex-end;
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  background: var(--jp-toolbar-background);
  min-height: var(--jp-toolbar-micro-height);
  padding: var(--jp-notebook-toolbar-padding);
  z-index: 1;
  right: 0;
  top: 0;
}

.jp-Toolbar > .jp-Toolbar-responsive-opener {
  margin-left: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-Notebook-ExecutionIndicator {
  position: relative;
  display: inline-block;
  height: 100%;
  z-index: 9997;
}

.jp-Notebook-ExecutionIndicator-tooltip {
  visibility: hidden;
  height: auto;
  width: max-content;
  width: -moz-max-content;
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color1);
  text-align: justify;
  border-radius: 6px;
  padding: 0 5px;
  position: fixed;
  display: table;
}

.jp-Notebook-ExecutionIndicator-tooltip.up {
  transform: translateX(-50%) translateY(-100%) translateY(-32px);
}

.jp-Notebook-ExecutionIndicator-tooltip.down {
  transform: translateX(calc(-100% + 16px)) translateY(5px);
}

.jp-Notebook-ExecutionIndicator-tooltip.hidden {
  display: none;
}

.jp-Notebook-ExecutionIndicator:hover .jp-Notebook-ExecutionIndicator-tooltip {
  visibility: visible;
}

.jp-Notebook-ExecutionIndicator span {
  font-size: var(--jp-ui-font-size1);
  font-family: var(--jp-ui-font-family);
  color: var(--jp-ui-font-color1);
  line-height: 24px;
  display: block;
}

.jp-Notebook-ExecutionIndicator-progress-bar {
  display: flex;
  justify-content: center;
  height: 100%;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*
 * Execution indicator
 */
.jp-tocItem-content::after {
  content: '';

  /* Must be identical to form a circle */
  width: 12px;
  height: 12px;
  background: none;
  border: none;
  position: absolute;
  right: 0;
}

.jp-tocItem-content[data-running='0']::after {
  border-radius: 50%;
  border: var(--jp-border-width) solid var(--jp-inverse-layout-color3);
  background: none;
}

.jp-tocItem-content[data-running='1']::after {
  border-radius: 50%;
  border: var(--jp-border-width) solid var(--jp-inverse-layout-color3);
  background-color: var(--jp-inverse-layout-color3);
}

.jp-tocItem-content[data-running='0'],
.jp-tocItem-content[data-running='1'] {
  margin-right: 12px;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-Notebook-footer {
  height: 27px;
  margin-left: calc(
    var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) +
      var(--jp-cell-padding)
  );
  width: calc(
    100% -
      (
        var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) +
          var(--jp-cell-padding) + var(--jp-cell-padding)
      )
  );
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  color: var(--jp-ui-font-color3);
  margin-top: 6px;
  background: none;
  cursor: pointer;
}

.jp-Notebook-footer:focus {
  border-color: var(--jp-cell-editor-active-border-color);
}

/* For devices that support hovering, hide footer until hover */
@media (hover: hover) {
  .jp-Notebook-footer {
    opacity: 0;
  }

  .jp-Notebook-footer:focus,
  .jp-Notebook-footer:hover {
    opacity: 1;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Imports
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| CSS variables
|----------------------------------------------------------------------------*/

:root {
  --jp-side-by-side-output-size: 1fr;
  --jp-side-by-side-resized-cell: var(--jp-side-by-side-output-size);
  --jp-private-notebook-dragImage-width: 304px;
  --jp-private-notebook-dragImage-height: 36px;
  --jp-private-notebook-selected-color: var(--md-blue-400);
  --jp-private-notebook-active-color: var(--md-green-400);
}

/*-----------------------------------------------------------------------------
| Notebook
|----------------------------------------------------------------------------*/

/* stylelint-disable selector-max-class */

.jp-NotebookPanel {
  display: block;
  height: 100%;
}

.jp-NotebookPanel.jp-Document {
  min-width: 240px;
  min-height: 120px;
}

.jp-Notebook {
  padding: var(--jp-notebook-padding);
  outline: none;
  overflow: auto;
  background: var(--jp-layout-color0);
}

.jp-Notebook.jp-mod-scrollPastEnd::after {
  display: block;
  content: '';
  min-height: var(--jp-notebook-scroll-padding);
}

.jp-MainAreaWidget-ContainStrict .jp-Notebook * {
  contain: strict;
}

.jp-Notebook .jp-Cell {
  overflow: visible;
}

.jp-Notebook .jp-Cell .jp-InputPrompt {
  cursor: move;
}

/*-----------------------------------------------------------------------------
| Notebook state related styling
|
| The notebook and cells each have states, here are the possibilities:
|
| - Notebook
|   - Command
|   - Edit
| - Cell
|   - None
|   - Active (only one can be active)
|   - Selected (the cells actions are applied to)
|   - Multiselected (when multiple selected, the cursor)
|   - No outputs
|----------------------------------------------------------------------------*/

/* Command or edit modes */

.jp-Notebook .jp-Cell:not(.jp-mod-active) .jp-InputPrompt {
  opacity: var(--jp-cell-prompt-not-active-opacity);
  color: var(--jp-cell-prompt-not-active-font-color);
}

.jp-Notebook .jp-Cell:not(.jp-mod-active) .jp-OutputPrompt {
  opacity: var(--jp-cell-prompt-not-active-opacity);
  color: var(--jp-cell-prompt-not-active-font-color);
}

/* cell is active */
.jp-Notebook .jp-Cell.jp-mod-active .jp-Collapser {
  background: var(--jp-brand-color1);
}

/* cell is dirty */
.jp-Notebook .jp-Cell.jp-mod-dirty .jp-InputPrompt {
  color: var(--jp-warn-color1);
}

.jp-Notebook .jp-Cell.jp-mod-dirty .jp-InputPrompt::before {
  color: var(--jp-warn-color1);
  content: '';
}

.jp-Notebook .jp-Cell.jp-mod-active.jp-mod-dirty .jp-Collapser {
  background: var(--jp-warn-color1);
}

/* collapser is hovered */
.jp-Notebook .jp-Cell .jp-Collapser:hover {
  box-shadow: var(--jp-elevation-z2);
  background: var(--jp-brand-color1);
  opacity: var(--jp-cell-collapser-not-active-hover-opacity);
}

/* cell is active and collapser is hovered */
.jp-Notebook .jp-Cell.jp-mod-active .jp-Collapser:hover {
  background: var(--jp-brand-color0);
  opacity: 1;
}

/* Command mode */

.jp-Notebook.jp-mod-commandMode .jp-Cell.jp-mod-selected {
  background: var(--jp-notebook-multiselected-color);
}

.jp-Notebook.jp-mod-commandMode
  .jp-Cell.jp-mod-active.jp-mod-selected:not(.jp-mod-multiSelected) {
  background: transparent;
}

/* Edit mode */

.jp-Notebook.jp-mod-editMode .jp-Cell.jp-mod-active .jp-InputArea-editor {
  border: var(--jp-border-width) solid var(--jp-cell-editor-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
  background-color: var(--jp-cell-editor-active-background);
}

/*-----------------------------------------------------------------------------
| Notebook drag and drop
|----------------------------------------------------------------------------*/

.jp-Notebook-cell.jp-mod-dropSource {
  opacity: 0.5;
}

.jp-Notebook-cell.jp-mod-dropTarget,
.jp-Notebook.jp-mod-commandMode
  .jp-Notebook-cell.jp-mod-active.jp-mod-selected.jp-mod-dropTarget {
  border-top-color: var(--jp-private-notebook-selected-color);
  border-top-style: solid;
  border-top-width: 2px;
}

.jp-dragImage {
  display: block;
  flex-direction: row;
  width: var(--jp-private-notebook-dragImage-width);
  height: var(--jp-private-notebook-dragImage-height);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background);
  overflow: visible;
}

.jp-dragImage-singlePrompt {
  box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.12);
}

.jp-dragImage .jp-dragImage-content {
  flex: 1 1 auto;
  z-index: 2;
  font-size: var(--jp-code-font-size);
  font-family: var(--jp-code-font-family);
  line-height: var(--jp-code-line-height);
  padding: var(--jp-code-padding);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background-color);
  color: var(--jp-content-font-color3);
  text-align: left;
  margin: 4px 4px 4px 0;
}

.jp-dragImage .jp-dragImage-prompt {
  flex: 0 0 auto;
  min-width: 36px;
  color: var(--jp-cell-inprompt-font-color);
  padding: var(--jp-code-padding);
  padding-left: 12px;
  font-family: var(--jp-cell-prompt-font-family);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  line-height: 1.9;
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
}

.jp-dragImage-multipleBack {
  z-index: -1;
  position: absolute;
  height: 32px;
  width: 300px;
  top: 8px;
  left: 8px;
  background: var(--jp-layout-color2);
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.12);
}

/*-----------------------------------------------------------------------------
| Cell toolbar
|----------------------------------------------------------------------------*/

.jp-NotebookTools {
  display: block;
  min-width: var(--jp-sidebar-min-width);
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);

  /* This is needed so that all font sizing of children done in ems is
    * relative to this base size */
  font-size: var(--jp-ui-font-size1);
  overflow: auto;
}

.jp-ActiveCellTool {
  padding: 12px 0;
  display: flex;
}

.jp-ActiveCellTool-Content {
  flex: 1 1 auto;
}

.jp-ActiveCellTool .jp-ActiveCellTool-CellContent {
  background: var(--jp-cell-editor-background);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  border-radius: 0;
  min-height: 29px;
}

.jp-ActiveCellTool .jp-InputPrompt {
  min-width: calc(var(--jp-cell-prompt-width) * 0.75);
}

.jp-ActiveCellTool-CellContent > pre {
  padding: 5px 4px;
  margin: 0;
  white-space: normal;
}

.jp-MetadataEditorTool {
  flex-direction: column;
  padding: 12px 0;
}

.jp-RankedPanel > :not(:first-child) {
  margin-top: 12px;
}

.jp-KeySelector select.jp-mod-styled {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  border: var(--jp-border-width) solid var(--jp-border-color1);
}

.jp-KeySelector label,
.jp-MetadataEditorTool label,
.jp-NumberSetter label {
  line-height: 1.4;
}

.jp-NotebookTools .jp-select-wrapper {
  margin-top: 4px;
  margin-bottom: 0;
}

.jp-NumberSetter input {
  width: 100%;
  margin-top: 4px;
}

.jp-NotebookTools .jp-Collapse {
  margin-top: 16px;
}

/*-----------------------------------------------------------------------------
| Presentation Mode (.jp-mod-presentationMode)
|----------------------------------------------------------------------------*/

.jp-mod-presentationMode .jp-Notebook {
  --jp-content-font-size1: var(--jp-content-presentation-font-size1);
  --jp-code-font-size: var(--jp-code-presentation-font-size);
}

.jp-mod-presentationMode .jp-Notebook .jp-Cell .jp-InputPrompt,
.jp-mod-presentationMode .jp-Notebook .jp-Cell .jp-OutputPrompt {
  flex: 0 0 110px;
}

/*-----------------------------------------------------------------------------
| Side-by-side Mode (.jp-mod-sideBySide)
|----------------------------------------------------------------------------*/
.jp-mod-sideBySide.jp-Notebook .jp-Notebook-cell {
  margin-top: 3em;
  margin-bottom: 3em;
  margin-left: 5%;
  margin-right: 5%;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell {
  display: grid;
  grid-template-columns: minmax(0, 1fr) min-content minmax(
      0,
      var(--jp-side-by-side-output-size)
    );
  grid-template-rows: auto minmax(0, 1fr) auto;
  grid-template-areas:
    'header header header'
    'input handle output'
    'footer footer footer';
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell.jp-mod-resizedCell {
  grid-template-columns: minmax(0, 1fr) min-content minmax(
      0,
      var(--jp-side-by-side-resized-cell)
    );
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellHeader {
  grid-area: header;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-Cell-inputWrapper {
  grid-area: input;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-Cell-outputWrapper {
  /* overwrite the default margin (no vertical separation needed in side by side move */
  margin-top: 0;
  grid-area: output;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellFooter {
  grid-area: footer;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellResizeHandle {
  grid-area: handle;
  user-select: none;
  display: block;
  height: 100%;
  cursor: ew-resize;
  padding: 0 var(--jp-cell-padding);
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellResizeHandle::after {
  content: '';
  display: block;
  background: var(--jp-border-color2);
  height: 100%;
  width: 5px;
}

.jp-mod-sideBySide.jp-Notebook
  .jp-CodeCell.jp-mod-resizedCell
  .jp-CellResizeHandle::after {
  background: var(--jp-border-color0);
}

.jp-CellResizeHandle {
  display: none;
}

/*-----------------------------------------------------------------------------
| Placeholder
|----------------------------------------------------------------------------*/

.jp-Cell-Placeholder {
  padding-left: 55px;
}

.jp-Cell-Placeholder-wrapper {
  background: #fff;
  border: 1px solid;
  border-color: #e5e6e9 #dfe0e4 #d0d1d5;
  border-radius: 4px;
  -webkit-border-radius: 4px;
  margin: 10px 15px;
}

.jp-Cell-Placeholder-wrapper-inner {
  padding: 15px;
  position: relative;
}

.jp-Cell-Placeholder-wrapper-body {
  background-repeat: repeat;
  background-size: 50% auto;
}

.jp-Cell-Placeholder-wrapper-body div {
  background: #f6f7f8;
  background-image: -webkit-linear-gradient(
    left,
    #f6f7f8 0%,
    #edeef1 20%,
    #f6f7f8 40%,
    #f6f7f8 100%
  );
  background-repeat: no-repeat;
  background-size: 800px 104px;
  height: 104px;
  position: absolute;
  right: 15px;
  left: 15px;
  top: 15px;
}

div.jp-Cell-Placeholder-h1 {
  top: 20px;
  height: 20px;
  left: 15px;
  width: 150px;
}

div.jp-Cell-Placeholder-h2 {
  left: 15px;
  top: 50px;
  height: 10px;
  width: 100px;
}

div.jp-Cell-Placeholder-content-1,
div.jp-Cell-Placeholder-content-2,
div.jp-Cell-Placeholder-content-3 {
  left: 15px;
  right: 15px;
  height: 10px;
}

div.jp-Cell-Placeholder-content-1 {
  top: 100px;
}

div.jp-Cell-Placeholder-content-2 {
  top: 120px;
}

div.jp-Cell-Placeholder-content-3 {
  top: 140px;
}

</style>
<style type="text/css">
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*
The following CSS variables define the main, public API for styling JupyterLab.
These variables should be used by all plugins wherever possible. In other
words, plugins should not define custom colors, sizes, etc unless absolutely
necessary. This enables users to change the visual theme of JupyterLab
by changing these variables.

Many variables appear in an ordered sequence (0,1,2,3). These sequences
are designed to work well together, so for example, `--jp-border-color1` should
be used with `--jp-layout-color1`. The numbers have the following meanings:

* 0: super-primary, reserved for special emphasis
* 1: primary, most important under normal situations
* 2: secondary, next most important under normal situations
* 3: tertiary, next most important under normal situations

Throughout JupyterLab, we are mostly following principles from Google's
Material Design when selecting colors. We are not, however, following
all of MD as it is not optimized for dense, information rich UIs.
*/

:root {
  /* Elevation
   *
   * We style box-shadows using Material Design's idea of elevation. These particular numbers are taken from here:
   *
   * https://github.com/material-components/material-components-web
   * https://material-components-web.appspot.com/elevation.html
   */

  --jp-shadow-base-lightness: 0;
  --jp-shadow-umbra-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.2
  );
  --jp-shadow-penumbra-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.14
  );
  --jp-shadow-ambient-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.12
  );
  --jp-elevation-z0: none;
  --jp-elevation-z1: 0 2px 1px -1px var(--jp-shadow-umbra-color),
    0 1px 1px 0 var(--jp-shadow-penumbra-color),
    0 1px 3px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z2: 0 3px 1px -2px var(--jp-shadow-umbra-color),
    0 2px 2px 0 var(--jp-shadow-penumbra-color),
    0 1px 5px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z4: 0 2px 4px -1px var(--jp-shadow-umbra-color),
    0 4px 5px 0 var(--jp-shadow-penumbra-color),
    0 1px 10px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z6: 0 3px 5px -1px var(--jp-shadow-umbra-color),
    0 6px 10px 0 var(--jp-shadow-penumbra-color),
    0 1px 18px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z8: 0 5px 5px -3px var(--jp-shadow-umbra-color),
    0 8px 10px 1px var(--jp-shadow-penumbra-color),
    0 3px 14px 2px var(--jp-shadow-ambient-color);
  --jp-elevation-z12: 0 7px 8px -4px var(--jp-shadow-umbra-color),
    0 12px 17px 2px var(--jp-shadow-penumbra-color),
    0 5px 22px 4px var(--jp-shadow-ambient-color);
  --jp-elevation-z16: 0 8px 10px -5px var(--jp-shadow-umbra-color),
    0 16px 24px 2px var(--jp-shadow-penumbra-color),
    0 6px 30px 5px var(--jp-shadow-ambient-color);
  --jp-elevation-z20: 0 10px 13px -6px var(--jp-shadow-umbra-color),
    0 20px 31px 3px var(--jp-shadow-penumbra-color),
    0 8px 38px 7px var(--jp-shadow-ambient-color);
  --jp-elevation-z24: 0 11px 15px -7px var(--jp-shadow-umbra-color),
    0 24px 38px 3px var(--jp-shadow-penumbra-color),
    0 9px 46px 8px var(--jp-shadow-ambient-color);

  /* Borders
   *
   * The following variables, specify the visual styling of borders in JupyterLab.
   */

  --jp-border-width: 1px;
  --jp-border-color0: var(--md-grey-400);
  --jp-border-color1: var(--md-grey-400);
  --jp-border-color2: var(--md-grey-300);
  --jp-border-color3: var(--md-grey-200);
  --jp-inverse-border-color: var(--md-grey-600);
  --jp-border-radius: 2px;

  /* UI Fonts
   *
   * The UI font CSS variables are used for the typography all of the JupyterLab
   * user interface elements that are not directly user generated content.
   *
   * The font sizing here is done assuming that the body font size of --jp-ui-font-size1
   * is applied to a parent element. When children elements, such as headings, are sized
   * in em all things will be computed relative to that body size.
   */

  --jp-ui-font-scale-factor: 1.2;
  --jp-ui-font-size0: 0.83333em;
  --jp-ui-font-size1: 13px; /* Base font size */
  --jp-ui-font-size2: 1.2em;
  --jp-ui-font-size3: 1.44em;
  --jp-ui-font-family: system-ui, -apple-system, blinkmacsystemfont, 'Segoe UI',
    helvetica, arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol';

  /*
   * Use these font colors against the corresponding main layout colors.
   * In a light theme, these go from dark to light.
   */

  /* Defaults use Material Design specification */
  --jp-ui-font-color0: rgba(0, 0, 0, 1);
  --jp-ui-font-color1: rgba(0, 0, 0, 0.87);
  --jp-ui-font-color2: rgba(0, 0, 0, 0.54);
  --jp-ui-font-color3: rgba(0, 0, 0, 0.38);

  /*
   * Use these against the brand/accent/warn/error colors.
   * These will typically go from light to darker, in both a dark and light theme.
   */

  --jp-ui-inverse-font-color0: rgba(255, 255, 255, 1);
  --jp-ui-inverse-font-color1: rgba(255, 255, 255, 1);
  --jp-ui-inverse-font-color2: rgba(255, 255, 255, 0.7);
  --jp-ui-inverse-font-color3: rgba(255, 255, 255, 0.5);

  /* Content Fonts
   *
   * Content font variables are used for typography of user generated content.
   *
   * The font sizing here is done assuming that the body font size of --jp-content-font-size1
   * is applied to a parent element. When children elements, such as headings, are sized
   * in em all things will be computed relative to that body size.
   */

  --jp-content-line-height: 1.6;
  --jp-content-font-scale-factor: 1.2;
  --jp-content-font-size0: 0.83333em;
  --jp-content-font-size1: 14px; /* Base font size */
  --jp-content-font-size2: 1.2em;
  --jp-content-font-size3: 1.44em;
  --jp-content-font-size4: 1.728em;
  --jp-content-font-size5: 2.0736em;

  /* This gives a magnification of about 125% in presentation mode over normal. */
  --jp-content-presentation-font-size1: 17px;
  --jp-content-heading-line-height: 1;
  --jp-content-heading-margin-top: 1.2em;
  --jp-content-heading-margin-bottom: 0.8em;
  --jp-content-heading-font-weight: 500;

  /* Defaults use Material Design specification */
  --jp-content-font-color0: rgba(0, 0, 0, 1);
  --jp-content-font-color1: rgba(0, 0, 0, 0.87);
  --jp-content-font-color2: rgba(0, 0, 0, 0.54);
  --jp-content-font-color3: rgba(0, 0, 0, 0.38);
  --jp-content-link-color: var(--md-blue-900);
  --jp-content-font-family: system-ui, -apple-system, blinkmacsystemfont,
    'Segoe UI', helvetica, arial, sans-serif, 'Apple Color Emoji',
    'Segoe UI Emoji', 'Segoe UI Symbol';

  /*
   * Code Fonts
   *
   * Code font variables are used for typography of code and other monospaces content.
   */

  --jp-code-font-size: 13px;
  --jp-code-line-height: 1.3077; /* 17px for 13px base */
  --jp-code-padding: 5px; /* 5px for 13px base, codemirror highlighting needs integer px value */
  --jp-code-font-family-default: menlo, consolas, 'DejaVu Sans Mono', monospace;
  --jp-code-font-family: var(--jp-code-font-family-default);

  /* This gives a magnification of about 125% in presentation mode over normal. */
  --jp-code-presentation-font-size: 16px;

  /* may need to tweak cursor width if you change font size */
  --jp-code-cursor-width0: 1.4px;
  --jp-code-cursor-width1: 2px;
  --jp-code-cursor-width2: 4px;

  /* Layout
   *
   * The following are the main layout colors use in JupyterLab. In a light
   * theme these would go from light to dark.
   */

  --jp-layout-color0: white;
  --jp-layout-color1: white;
  --jp-layout-color2: var(--md-grey-200);
  --jp-layout-color3: var(--md-grey-400);
  --jp-layout-color4: var(--md-grey-600);

  /* Inverse Layout
   *
   * The following are the inverse layout colors use in JupyterLab. In a light
   * theme these would go from dark to light.
   */

  --jp-inverse-layout-color0: #111;
  --jp-inverse-layout-color1: var(--md-grey-900);
  --jp-inverse-layout-color2: var(--md-grey-800);
  --jp-inverse-layout-color3: var(--md-grey-700);
  --jp-inverse-layout-color4: var(--md-grey-600);

  /* Brand/accent */

  --jp-brand-color0: var(--md-blue-900);
  --jp-brand-color1: var(--md-blue-700);
  --jp-brand-color2: var(--md-blue-300);
  --jp-brand-color3: var(--md-blue-100);
  --jp-brand-color4: var(--md-blue-50);
  --jp-accent-color0: var(--md-green-900);
  --jp-accent-color1: var(--md-green-700);
  --jp-accent-color2: var(--md-green-300);
  --jp-accent-color3: var(--md-green-100);

  /* State colors (warn, error, success, info) */

  --jp-warn-color0: var(--md-orange-900);
  --jp-warn-color1: var(--md-orange-700);
  --jp-warn-color2: var(--md-orange-300);
  --jp-warn-color3: var(--md-orange-100);
  --jp-error-color0: var(--md-red-900);
  --jp-error-color1: var(--md-red-700);
  --jp-error-color2: var(--md-red-300);
  --jp-error-color3: var(--md-red-100);
  --jp-success-color0: var(--md-green-900);
  --jp-success-color1: var(--md-green-700);
  --jp-success-color2: var(--md-green-300);
  --jp-success-color3: var(--md-green-100);
  --jp-info-color0: var(--md-cyan-900);
  --jp-info-color1: var(--md-cyan-700);
  --jp-info-color2: var(--md-cyan-300);
  --jp-info-color3: var(--md-cyan-100);

  /* Cell specific styles */

  --jp-cell-padding: 5px;
  --jp-cell-collapser-width: 8px;
  --jp-cell-collapser-min-height: 20px;
  --jp-cell-collapser-not-active-hover-opacity: 0.6;
  --jp-cell-editor-background: var(--md-grey-100);
  --jp-cell-editor-border-color: var(--md-grey-300);
  --jp-cell-editor-box-shadow: inset 0 0 2px var(--md-blue-300);
  --jp-cell-editor-active-background: var(--jp-layout-color0);
  --jp-cell-editor-active-border-color: var(--jp-brand-color1);
  --jp-cell-prompt-width: 64px;
  --jp-cell-prompt-font-family: var(--jp-code-font-family-default);
  --jp-cell-prompt-letter-spacing: 0;
  --jp-cell-prompt-opacity: 1;
  --jp-cell-prompt-not-active-opacity: 0.5;
  --jp-cell-prompt-not-active-font-color: var(--md-grey-700);

  /* A custom blend of MD grey and blue 600
   * See https://meyerweb.com/eric/tools/color-blend/#546E7A:1E88E5:5:hex */
  --jp-cell-inprompt-font-color: #307fc1;

  /* A custom blend of MD grey and orange 600
   * https://meyerweb.com/eric/tools/color-blend/#546E7A:F4511E:5:hex */
  --jp-cell-outprompt-font-color: #bf5b3d;

  /* Notebook specific styles */

  --jp-notebook-padding: 10px;
  --jp-notebook-select-background: var(--jp-layout-color1);
  --jp-notebook-multiselected-color: var(--md-blue-50);

  /* The scroll padding is calculated to fill enough space at the bottom of the
  notebook to show one single-line cell (with appropriate padding) at the top
  when the notebook is scrolled all the way to the bottom. We also subtract one
  pixel so that no scrollbar appears if we have just one single-line cell in the
  notebook. This padding is to enable a 'scroll past end' feature in a notebook.
  */
  --jp-notebook-scroll-padding: calc(
    100% - var(--jp-code-font-size) * var(--jp-code-line-height) -
      var(--jp-code-padding) - var(--jp-cell-padding) - 1px
  );

  /* Rendermime styles */

  --jp-rendermime-error-background: #fdd;
  --jp-rendermime-table-row-background: var(--md-grey-100);
  --jp-rendermime-table-row-hover-background: var(--md-light-blue-50);

  /* Dialog specific styles */

  --jp-dialog-background: rgba(0, 0, 0, 0.25);

  /* Console specific styles */

  --jp-console-padding: 10px;

  /* Toolbar specific styles */

  --jp-toolbar-border-color: var(--jp-border-color1);
  --jp-toolbar-micro-height: 8px;
  --jp-toolbar-background: var(--jp-layout-color1);
  --jp-toolbar-box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.24);
  --jp-toolbar-header-margin: 4px 4px 0 4px;
  --jp-toolbar-active-background: var(--md-grey-300);

  /* Statusbar specific styles */

  --jp-statusbar-height: 24px;

  /* Input field styles */

  --jp-input-box-shadow: inset 0 0 2px var(--md-blue-300);
  --jp-input-active-background: var(--jp-layout-color1);
  --jp-input-hover-background: var(--jp-layout-color1);
  --jp-input-background: var(--md-grey-100);
  --jp-input-border-color: var(--jp-inverse-border-color);
  --jp-input-active-border-color: var(--jp-brand-color1);
  --jp-input-active-box-shadow-color: rgba(19, 124, 189, 0.3);

  /* General editor styles */

  --jp-editor-selected-background: #d9d9d9;
  --jp-editor-selected-focused-background: #d7d4f0;
  --jp-editor-cursor-color: var(--jp-ui-font-color0);

  /* Code mirror specific styles */

  --jp-mirror-editor-keyword-color: #008000;
  --jp-mirror-editor-atom-color: #88f;
  --jp-mirror-editor-number-color: #080;
  --jp-mirror-editor-def-color: #00f;
  --jp-mirror-editor-variable-color: var(--md-grey-900);
  --jp-mirror-editor-variable-2-color: rgb(0, 54, 109);
  --jp-mirror-editor-variable-3-color: #085;
  --jp-mirror-editor-punctuation-color: #05a;
  --jp-mirror-editor-property-color: #05a;
  --jp-mirror-editor-operator-color: #a2f;
  --jp-mirror-editor-comment-color: #408080;
  --jp-mirror-editor-string-color: #ba2121;
  --jp-mirror-editor-string-2-color: #708;
  --jp-mirror-editor-meta-color: #a2f;
  --jp-mirror-editor-qualifier-color: #555;
  --jp-mirror-editor-builtin-color: #008000;
  --jp-mirror-editor-bracket-color: #997;
  --jp-mirror-editor-tag-color: #170;
  --jp-mirror-editor-attribute-color: #00c;
  --jp-mirror-editor-header-color: blue;
  --jp-mirror-editor-quote-color: #090;
  --jp-mirror-editor-link-color: #00c;
  --jp-mirror-editor-error-color: #f00;
  --jp-mirror-editor-hr-color: #999;

  /*
    RTC user specific colors.
    These colors are used for the cursor, username in the editor,
    and the icon of the user.
  */

  --jp-collaborator-color1: #ffad8e;
  --jp-collaborator-color2: #dac83d;
  --jp-collaborator-color3: #72dd76;
  --jp-collaborator-color4: #00e4d0;
  --jp-collaborator-color5: #45d4ff;
  --jp-collaborator-color6: #e2b1ff;
  --jp-collaborator-color7: #ff9de6;

  /* Vega extension styles */

  --jp-vega-background: white;

  /* Sidebar-related styles */

  --jp-sidebar-min-width: 250px;

  /* Search-related styles */

  --jp-search-toggle-off-opacity: 0.5;
  --jp-search-toggle-hover-opacity: 0.8;
  --jp-search-toggle-on-opacity: 1;
  --jp-search-selected-match-background-color: rgb(245, 200, 0);
  --jp-search-selected-match-color: black;
  --jp-search-unselected-match-background-color: var(
    --jp-inverse-layout-color0
  );
  --jp-search-unselected-match-color: var(--jp-ui-inverse-font-color0);

  /* Icon colors that work well with light or dark backgrounds */
  --jp-icon-contrast-color0: var(--md-purple-600);
  --jp-icon-contrast-color1: var(--md-green-600);
  --jp-icon-contrast-color2: var(--md-pink-600);
  --jp-icon-contrast-color3: var(--md-blue-600);

  /* Button colors */
  --jp-accept-color-normal: var(--md-blue-700);
  --jp-accept-color-hover: var(--md-blue-800);
  --jp-accept-color-active: var(--md-blue-900);
  --jp-warn-color-normal: var(--md-red-700);
  --jp-warn-color-hover: var(--md-red-800);
  --jp-warn-color-active: var(--md-red-900);
  --jp-reject-color-normal: var(--md-grey-600);
  --jp-reject-color-hover: var(--md-grey-700);
  --jp-reject-color-active: var(--md-grey-800);

  /* File or activity icons and switch semantic variables */
  --jp-jupyter-icon-color: #f37626;
  --jp-notebook-icon-color: #f37626;
  --jp-json-icon-color: var(--md-orange-700);
  --jp-console-icon-background-color: var(--md-blue-700);
  --jp-console-icon-color: white;
  --jp-terminal-icon-background-color: var(--md-grey-800);
  --jp-terminal-icon-color: var(--md-grey-200);
  --jp-text-editor-icon-color: var(--md-grey-700);
  --jp-inspector-icon-color: var(--md-grey-700);
  --jp-switch-color: var(--md-grey-400);
  --jp-switch-true-position-color: var(--md-orange-900);
}
</style>
<style type="text/css">
/* Force rendering true colors when outputing to pdf */
* {
  -webkit-print-color-adjust: exact;
}

/* Misc */
a.anchor-link {
  display: none;
}

/* Input area styling */
.jp-InputArea {
  overflow: hidden;
}

.jp-InputArea-editor {
  overflow: hidden;
}

.cm-editor.cm-s-jupyter .highlight pre {
/* weird, but --jp-code-padding defined to be 5px but 4px horizontal padding is hardcoded for pre.cm-line */
  padding: var(--jp-code-padding) 4px;
  margin: 0;

  font-family: inherit;
  font-size: inherit;
  line-height: inherit;
  color: inherit;

}

.jp-OutputArea-output pre {
  line-height: inherit;
  font-family: inherit;
}

.jp-RenderedText pre {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-code-font-size);
}

/* Hiding the collapser by default */
.jp-Collapser {
  display: none;
}

@page {
    margin: 0.5in; /* Margin for each printed piece of paper */
}

@media print {
  .jp-Cell-inputWrapper,
  .jp-Cell-outputWrapper {
    display: block;
  }
}
</style>
<!-- Load mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS_CHTML-full,Safe"> </script>
<!-- MathJax configuration -->
<script type="text/x-mathjax-config">
    init_mathjax = function() {
        if (window.MathJax) {
        // MathJax loaded
            MathJax.Hub.Config({
                TeX: {
                    equationNumbers: {
                    autoNumber: "AMS",
                    useLabelIds: true
                    }
                },
                tex2jax: {
                    inlineMath: [ ['$','$'], ["\\(","\\)"] ],
                    displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
                    processEscapes: true,
                    processEnvironments: true
                },
                displayAlign: 'center',
                messageStyle: 'none',
                CommonHTML: {
                    linebreaks: {
                    automatic: true
                    }
                }
            });

            MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
        }
    }
    init_mathjax();
    </script>
<!-- End of mathjax configuration --><script type="module">
  document.addEventListener("DOMContentLoaded", async () => {
    const diagrams = document.querySelectorAll(".jp-Mermaid > pre.mermaid");
    // do not load mermaidjs if not needed
    if (!diagrams.length) {
      return;
    }
    const mermaid = (await import("https://cdnjs.cloudflare.com/ajax/libs/mermaid/10.7.0/mermaid.esm.min.mjs")).default;
    const parser = new DOMParser();

    mermaid.initialize({
      maxTextSize: 100000,
      maxEdges: 100000,
      startOnLoad: false,
      fontFamily: window
        .getComputedStyle(document.body)
        .getPropertyValue("--jp-ui-font-family"),
      theme: document.querySelector("body[data-jp-theme-light='true']")
        ? "default"
        : "dark",
    });

    let _nextMermaidId = 0;

    function makeMermaidImage(svg) {
      const img = document.createElement("img");
      const doc = parser.parseFromString(svg, "image/svg+xml");
      const svgEl = doc.querySelector("svg");
      const { maxWidth } = svgEl?.style || {};
      const firstTitle = doc.querySelector("title");
      const firstDesc = doc.querySelector("desc");

      img.setAttribute("src", `data:image/svg+xml,${encodeURIComponent(svg)}`);
      if (maxWidth) {
        img.width = parseInt(maxWidth);
      }
      if (firstTitle) {
        img.setAttribute("alt", firstTitle.textContent);
      }
      if (firstDesc) {
        const caption = document.createElement("figcaption");
        caption.className = "sr-only";
        caption.textContent = firstDesc.textContent;
        return [img, caption];
      }
      return [img];
    }

    async function makeMermaidError(text) {
      let errorMessage = "";
      try {
        await mermaid.parse(text);
      } catch (err) {
        errorMessage = `${err}`;
      }

      const result = document.createElement("details");
      result.className = 'jp-RenderedMermaid-Details';
      const summary = document.createElement("summary");
      summary.className = 'jp-RenderedMermaid-Summary';
      const pre = document.createElement("pre");
      const code = document.createElement("code");
      code.innerText = text;
      pre.appendChild(code);
      summary.appendChild(pre);
      result.appendChild(summary);

      const warning = document.createElement("pre");
      warning.innerText = errorMessage;
      result.appendChild(warning);
      return [result];
    }

    async function renderOneMarmaid(src) {
      const id = `jp-mermaid-${_nextMermaidId++}`;
      const parent = src.parentNode;
      let raw = src.textContent.trim();
      const el = document.createElement("div");
      el.style.visibility = "hidden";
      document.body.appendChild(el);
      let results = null;
      let output = null;
      try {
        let { svg } = await mermaid.render(id, raw, el);
        svg = cleanMermaidSvg(svg);
        results = makeMermaidImage(svg);
        output = document.createElement("figure");
        results.map(output.appendChild, output);
      } catch (err) {
        parent.classList.add("jp-mod-warning");
        results = await makeMermaidError(raw);
        output = results[0];
      } finally {
        el.remove();
      }
      parent.classList.add("jp-RenderedMermaid");
      parent.appendChild(output);
    }


    /**
     * Post-process to ensure mermaid diagrams contain only valid SVG and XHTML.
     */
    function cleanMermaidSvg(svg) {
      return svg.replace(RE_VOID_ELEMENT, replaceVoidElement);
    }


    /**
     * A regular expression for all void elements, which may include attributes and
     * a slash.
     *
     * @see https://developer.mozilla.org/en-US/docs/Glossary/Void_element
     *
     * Of these, only `<br>` is generated by Mermaid in place of `\n`,
     * but _any_ "malformed" tag will break the SVG rendering entirely.
     */
    const RE_VOID_ELEMENT =
      /<\s*(area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)\s*([^>]*?)\s*>/gi;

    /**
     * Ensure a void element is closed with a slash, preserving any attributes.
     */
    function replaceVoidElement(match, tag, rest) {
      rest = rest.trim();
      if (!rest.endsWith('/')) {
        rest = `${rest} /`;
      }
      return `<${tag} ${rest}>`;
    }

    void Promise.all([...diagrams].map(renderOneMarmaid));
  });
</script>
<style>
  .jp-Mermaid:not(.jp-RenderedMermaid) {
    display: none;
  }

  .jp-RenderedMermaid {
    overflow: auto;
    display: flex;
  }

  .jp-RenderedMermaid.jp-mod-warning {
    width: auto;
    padding: 0.5em;
    margin-top: 0.5em;
    border: var(--jp-border-width) solid var(--jp-warn-color2);
    border-radius: var(--jp-border-radius);
    color: var(--jp-ui-font-color1);
    font-size: var(--jp-ui-font-size1);
    white-space: pre-wrap;
    word-wrap: break-word;
  }

  .jp-RenderedMermaid figure {
    margin: 0;
    overflow: auto;
    max-width: 100%;
  }

  .jp-RenderedMermaid img {
    max-width: 100%;
  }

  .jp-RenderedMermaid-Details > pre {
    margin-top: 1em;
  }

  .jp-RenderedMermaid-Summary {
    color: var(--jp-warn-color2);
  }

  .jp-RenderedMermaid:not(.jp-mod-warning) pre {
    display: none;
  }

  .jp-RenderedMermaid-Summary > pre {
    display: inline-block;
    white-space: normal;
  }
</style>
<!-- End of mermaid configuration --></head>
<body class="jp-Notebook" data-jp-theme-light="true" data-jp-theme-name="JupyterLab Light">
<main><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=e393c336">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[1]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="kn">import</span><span class="w"> </span><span class="nn">prody</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">pr</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">py3Dmol</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">tempfile</span>
<span class="c1"># --- notebook parameters (non-interactive) ---</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">os</span><span class="o">,</span><span class="w"> </span><span class="nn">sys</span><span class="o">,</span><span class="w"> </span><span class="nn">asyncio</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">pathlib</span><span class="w"> </span><span class="kn">import</span> <span class="n">Path</span>

<span class="kn">import</span><span class="w"> </span><span class="nn">pandas</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">pd</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Analysis.PlotUtils</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">scipy.ndimage</span><span class="w"> </span><span class="kn">import</span> <span class="n">binary_dilation</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">matplotlib.patches</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">mpatches</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">IPython.display</span><span class="w"> </span><span class="kn">import</span> <span class="n">display</span><span class="p">,</span> <span class="n">HTML</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDB</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">pairwise2</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio.PDB</span><span class="w"> </span><span class="kn">import</span> <span class="n">Superimposer</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">py3Dmol</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">ipywidgets</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">widgets</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio.PDB</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDBParser</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">pairwise2</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">py3Dmol</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDB</span><span class="p">,</span> <span class="n">Align</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio.PDB</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDBParser</span><span class="p">,</span> <span class="n">PDBIO</span>
<span class="c1"># from Bio.PDB.Polypeptide import protein_letters_3to1</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">io</span><span class="w"> </span><span class="kn">import</span> <span class="n">StringIO</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">numpy</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">np</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">matplotlib.pyplot</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">plt</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">matplotlib.colors</span><span class="w"> </span><span class="kn">import</span> <span class="n">ListedColormap</span><span class="p">,</span> <span class="n">BoundaryNorm</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">matplotlib.patches</span><span class="w"> </span><span class="kn">import</span> <span class="n">Patch</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">config</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">utils.utils</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">utils.protein_utils</span><span class="w"> </span><span class="kn">import</span> <span class="n">extract_protein_sequence</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">utils.protein_plot_utils</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>CONFIG FOLD-SWITCH PIEPLINE WITH USER: orzuk  ENVIRONMENT: Linux
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>PyRosetta not installed; skipping energy/G panels
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=8069b923">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[2]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="c1"># Silence Windows asyncio warning from zmq/tornado</span>
<span class="k">if</span> <span class="n">sys</span><span class="o">.</span><span class="n">platform</span><span class="o">.</span><span class="n">startswith</span><span class="p">(</span><span class="s2">"win"</span><span class="p">):</span>
    <span class="n">asyncio</span><span class="o">.</span><span class="n">set_event_loop_policy</span><span class="p">(</span><span class="n">asyncio</span><span class="o">.</span><span class="n">WindowsSelectorEventLoopPolicy</span><span class="p">())</span>

<span class="c1"># Get pair from environment (set by generate_notebooks.py)</span>
<span class="n">fold_pair</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">getenv</span><span class="p">(</span><span class="s2">"PAIR_ID"</span><span class="p">,</span> <span class="s2">""</span><span class="p">)</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span>

<span class="k">if</span> <span class="ow">not</span> <span class="n">fold_pair</span><span class="p">:</span>
    <span class="c1"># We are likely running interactively in Jupyter UI (not nbconvert).</span>
    <span class="c1"># Fall back to input() ONLY when stdin is available.</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">fold_pair</span> <span class="o">=</span> <span class="nb">input</span><span class="p">(</span><span class="s2">"Enter fold-switch pair (e.g., 3hdfA_3hdeA): "</span><span class="p">)</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span>
    <span class="k">except</span> <span class="ne">Exception</span><span class="p">:</span>
        <span class="k">raise</span> <span class="ne">RuntimeError</span><span class="p">(</span>
            <span class="s2">"PAIR_ID not provided and no stdin available. "</span>
            <span class="s2">"When running via generate_notebooks.py, PAIR_ID is set automatically."</span>
        <span class="p">)</span>

<span class="c1"># Normalize and split into the two PDB+chain ids</span>
<span class="k">if</span> <span class="s2">"_"</span> <span class="ow">not</span> <span class="ow">in</span> <span class="n">fold_pair</span><span class="p">:</span>
    <span class="k">raise</span> <span class="ne">ValueError</span><span class="p">(</span><span class="sa">f</span><span class="s2">"PAIR_ID must look like 'A_B', got: </span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="n">pdb1</span><span class="p">,</span> <span class="n">pdb2</span> <span class="o">=</span> <span class="n">fold_pair</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s2">"_"</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"Using pair:"</span><span class="p">,</span> <span class="n">pdb1</span><span class="p">,</span> <span class="n">pdb2</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>Using pair: 1fzpD 2frhA
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=cf01df3c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[3]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">plot_tool</span> <span class="o">=</span> <span class="n">PlotTool</span><span class="p">(</span><span class="n">folder</span><span class="o">=</span><span class="n">DATA_DIR</span><span class="p">,</span> <span class="n">fold_pair</span><span class="o">=</span><span class="n">fold_pair</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 1997 atoms and 1 coordinate set(s) were parsed in 0.06s.
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 2114 atoms and 1 coordinate set(s) were parsed in 0.05s.
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=80d1165b">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[4]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">VIZ_FOLDER</span>    <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">DATA_DIR</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_cmap_esm/VizCmaps'</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"Fold-pair,"</span><span class="p">,</span> <span class="n">fold_pair</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">chains</span> <span class="o">=</span> <span class="p">(</span><span class="n">pdb1</span><span class="p">[</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="o">.</span><span class="n">upper</span><span class="p">(),</span> <span class="n">pdb2</span><span class="p">[</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="o">.</span><span class="n">upper</span><span class="p">())</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"chains,"</span><span class="p">,</span> <span class="n">chains</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">pdb_path1</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">DATA_DIR</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">pdb1</span><span class="p">[:</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="si">}</span><span class="s2">.pdb"</span>
<span class="n">pdb_path2</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">DATA_DIR</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">pdb2</span><span class="p">[:</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="si">}</span><span class="s2">.pdb"</span>
<span class="c1">#fold_pair_subdir = f"{fold_pair[0]}_{fold_pair[1]}"</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>Fold-pair, 1fzpD_2frhA
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>chains, ('D', 'A')
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=d913aca8">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[5]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">seq_fold1</span> <span class="o">=</span> <span class="n">extract_protein_sequence</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span> <span class="n">chain</span><span class="o">=</span><span class="n">chains</span><span class="p">[</span><span class="mi">0</span><span class="p">],</span> <span class="n">ca_only</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">seq_fold2</span> <span class="o">=</span> <span class="n">extract_protein_sequence</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">,</span> <span class="n">chain</span><span class="o">=</span><span class="n">chains</span><span class="p">[</span><span class="mi">1</span><span class="p">],</span> <span class="n">ca_only</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=7f244dee">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h1 id="Visualizations-and-results">Visualizations and results<a class="anchor-link" href="#Visualizations-and-results"></a></h1>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=0dae88ab">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="Original-Structure-Visualization">Original Structure Visualization<a class="anchor-link" href="#Original-Structure-Visualization"></a></h2>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=e241ff25">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[6]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">pdb_file1</span> <span class="o">=</span> <span class="n">read_pdb_file</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">)</span>
<span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_single_fold</span><span class="p">(</span><span class="n">pdb_file1</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">fold1</span><span class="p">)</span>

<span class="n">pdb_file2</span> <span class="o">=</span> <span class="n">read_pdb_file</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">)</span>
<span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_single_fold</span><span class="p">(</span><span class="n">pdb_file2</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">fold2</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div id="3dmolviewer_17581716205928447" style="position: relative; width: 400px; height: 300px;">
<p id="3dmolwarning_17581716205928447" style="background-color:#ffcccc;color:black">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br/></p>
</div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');
}

var viewer_17581716205928447 = null;
var warn = document.getElementById("3dmolwarning_17581716205928447");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_17581716205928447 = $3Dmol.createViewer(document.getElementById("3dmolviewer_17581716205928447"),{backgroundColor:"white"});
viewer_17581716205928447.zoomTo();
	viewer_17581716205928447.addModel("HEADER    TRANSCRIPTION/DNA                       03-OCT-00   1FZP              \nOBSLTE     17-NOV-21 1FZP                                                       \nTITLE     CRYSTAL STRUCTURES OF SARA: A PLEIOTROPIC REGULATOR OF VIRULENCE GENES\nTITLE    2 IN S. AUREUS                                                         \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: 5'-D(P*AP*TP*AP*TP*AP*TP*A)-3';                            \nCOMPND   3 CHAIN: W, K;                                                         \nCOMPND   4 ENGINEERED: YES;                                                     \nCOMPND   5 MOL_ID: 2;                                                           \nCOMPND   6 MOLECULE: STAPHYLOCOCCAL ACCESSORY REGULATOR A;                      \nCOMPND   7 CHAIN: D, B;                                                         \nCOMPND   8 SYNONYM: SARA;                                                       \nCOMPND   9 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 SYNTHETIC: YES;                                                      \nSOURCE   3 MOL_ID: 2;                                                           \nSOURCE   4 ORGANISM_SCIENTIFIC: STAPHYLOCOCCUS AUREUS;                          \nSOURCE   5 ORGANISM_TAXID: 1280;                                                \nSOURCE   6 STRAIN: DB;                                                          \nSOURCE   7 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \nSOURCE   8 EXPRESSION_SYSTEM_TAXID: 562;                                        \nSOURCE   9 EXPRESSION_SYSTEM_VECTOR_TYPE: PLASMID;                              \nSOURCE  10 EXPRESSION_SYSTEM_PLASMID: PET9                                      \nKEYWDS    PROTEIN-DNA COMPLEX, INDIRECT READOUT, D-DNA, CONFORMATIONAL CHANGES, \nKEYWDS   2 COIL TO HELIX, TRANSCRIPTION-DNA COMPLEX                             \nEXPDTA    X-RAY DIFFRACTION                                                     \nAUTHOR    M.A.SCHUMACHER,B.HURLBURT,R.G.BRENNAN                                 \nREVDAT   5   17-NOV-21 1FZP    1       OBSLTE JRNL   REMARK LINK                \nREVDAT   4   04-APR-18 1FZP    1       REMARK                                   \nREVDAT   3   24-FEB-09 1FZP    1       VERSN                                    \nREVDAT   2   01-APR-03 1FZP    1       JRNL                                     \nREVDAT   1   22-JAN-01 1FZP    0                                                \nJRNL        AUTH   M.A.SCHUMACHER,B.K.HURLBURT,R.G.BRENNAN                      \nJRNL        TITL   CORRECTION: CRYSTAL STRUCTURES OF SARA, A PLEIOTROPIC        \nJRNL        TITL 2 REGULATOR OF VIRULENCE GENES IN S. AUREUS                    \nJRNL        REF    NATURE                        V. 414    85 2001              \nJRNL        REFN                   ISSN 0028-0836                               \nJRNL        DOI    10.1038/35102089                                             \nREMARK   1                                                                      \nREMARK   1 REFERENCE 1                                                          \nREMARK   1  AUTH   M.A.SCHUMACHER,B.K.HURLBURT,R.G.BRENNAN                      \nREMARK   1  TITL   CRYSTAL STRUCTURES OF SARA, A PLEIOTROPIC REGULATOR OF       \nREMARK   1  TITL 2 VIRULENCE GENES IN S. AUREUS.                                \nREMARK   1  REF    NATURE                        V. 409   215 2001              \nREMARK   1  REFN                   ISSN 0028-0836                               \nREMARK   1  PMID   11196648                                                     \nREMARK   1  DOI    10.1038/35051623                                             \nREMARK   2                                                                      \nREMARK   2 RESOLUTION.    2.95 ANGSTROMS.                                       \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : TNT                                                  \nREMARK   3   AUTHORS     : TRONRUD,TEN EYCK,MATTHEWS                            \nREMARK   3                                                                      \nREMARK   3  DATA USED IN REFINEMENT.                                            \nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.95                           \nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 10.00                          \nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 0.000                          \nREMARK   3   COMPLETENESS FOR RANGE        (%) : 89.0                           \nREMARK   3   NUMBER OF REFLECTIONS             : 14850                          \nREMARK   3                                                                      \nREMARK   3  USING DATA ABOVE SIGMA CUTOFF.                                      \nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.220                           \nREMARK   3   R VALUE            (WORKING SET) : 0.220                           \nREMARK   3   FREE R VALUE                     : 0.308                           \nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \nREMARK   3   FREE R VALUE TEST SET COUNT      : 145                             \nREMARK   3                                                                      \nREMARK   3  USING ALL DATA, NO SIGMA CUTOFF.                                    \nREMARK   3   R VALUE   (WORKING + TEST SET, NO CUTOFF) : 0.2300                 \nREMARK   3   R VALUE          (WORKING SET, NO CUTOFF) : NULL                   \nREMARK   3   FREE R VALUE                  (NO CUTOFF) : NULL                   \nREMARK   3   FREE R VALUE TEST SET SIZE (%, NO CUTOFF) : NULL                   \nREMARK   3   FREE R VALUE TEST SET COUNT   (NO CUTOFF) : NULL                   \nREMARK   3   TOTAL NUMBER OF REFLECTIONS   (NO CUTOFF) : 13920                  \nREMARK   3                                                                      \nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \nREMARK   3   PROTEIN ATOMS            : 1707                                    \nREMARK   3   NUCLEIC ACID ATOMS       : 288                                     \nREMARK   3   HETEROGEN ATOMS          : 2                                       \nREMARK   3   SOLVENT ATOMS            : 0                                       \nREMARK   3                                                                      \nREMARK   3  WILSON B VALUE (FROM FCALC, A**2) : 50.000                          \nREMARK   3                                                                      \nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.    RMS    WEIGHT  COUNT           \nREMARK   3   BOND LENGTHS                 (A) : 0.011 ; NULL  ; NULL            \nREMARK   3   BOND ANGLES            (DEGREES) : 1.980 ; NULL  ; NULL            \nREMARK   3   TORSION ANGLES         (DEGREES) : NULL  ; NULL  ; NULL            \nREMARK   3   PSEUDOROTATION ANGLES  (DEGREES) : NULL  ; NULL  ; NULL            \nREMARK   3   TRIGONAL CARBON PLANES       (A) : NULL  ; NULL  ; NULL            \nREMARK   3   GENERAL PLANES               (A) : NULL  ; NULL  ; NULL            \nREMARK   3   ISOTROPIC THERMAL FACTORS (A**2) : NULL  ; NULL  ; NULL            \nREMARK   3   NON-BONDED CONTACTS          (A) : NULL  ; NULL  ; NULL            \nREMARK   3                                                                      \nREMARK   3  INCORRECT CHIRAL-CENTERS (COUNT) : NULL                             \nREMARK   3                                                                      \nREMARK   3  BULK SOLVENT MODELING.                                              \nREMARK   3   METHOD USED : NULL                                                 \nREMARK   3   KSOL        : NULL                                                 \nREMARK   3   BSOL        : NULL                                                 \nREMARK   3                                                                      \nREMARK   3  RESTRAINT LIBRARIES.                                                \nREMARK   3   STEREOCHEMISTRY : ENGH AND HUBER                                   \nREMARK   3   ISOTROPIC THERMAL FACTOR RESTRAINTS : NULL                         \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \nREMARK   4                                                                      \nREMARK   4 1FZP COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK   5                                                                      \nREMARK   5 DEPOSITOR REQUESTED STRUCTURE BE OBSOLETED. ERRATUM WAS PUBLISHED.   \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 13-OCT-00.                  \nREMARK 100 THE DEPOSITION ID IS D_1000012051.                                   \nREMARK 200                                                                      \nREMARK 200 EXPERIMENTAL DETAILS                                                 \nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \nREMARK 200  DATE OF DATA COLLECTION        : 20-DEC-98                          \nREMARK 200  TEMPERATURE           (KELVIN) : 298                                \nREMARK 200  PH                             : 6.5                                \nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \nREMARK 200                                                                      \nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \nREMARK 200  BEAMLINE                       : NULL                               \nREMARK 200  X-RAY GENERATOR MODEL          : RIGAKU RU200                       \nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \nREMARK 200  WAVELENGTH OR RANGE        (A) : 1.5418                             \nREMARK 200  MONOCHROMATOR                  : NULL                               \nREMARK 200  OPTICS                         : NULL                               \nREMARK 200                                                                      \nREMARK 200  DETECTOR TYPE                  : AREA DETECTOR                      \nREMARK 200  DETECTOR MANUFACTURER          : ADSC                               \nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : SDMS                               \nREMARK 200  DATA SCALING SOFTWARE          : SDMS                               \nREMARK 200                                                                      \nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 10951                              \nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.950                              \nREMARK 200  RESOLUTION RANGE LOW       (A) : 15.000                             \nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : 0.000                              \nREMARK 200                                                                      \nREMARK 200 OVERALL.                                                             \nREMARK 200  COMPLETENESS FOR RANGE     (%) : 89.0                               \nREMARK 200  DATA REDUNDANCY                : 2.000                              \nREMARK 200  R MERGE                    (I) : 0.07800                            \nREMARK 200  R SYM                      (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 6.5000                             \nREMARK 200                                                                      \nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 2.95                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 3.20                     \nREMARK 200  COMPLETENESS FOR SHELL     (%) : 80.0                               \nREMARK 200  DATA REDUNDANCY IN SHELL       : 1.20                               \nREMARK 200  R MERGE FOR SHELL          (I) : 0.22000                            \nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \nREMARK 200                                                                      \nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \nREMARK 200 SOFTWARE USED: PHASES                                                \nREMARK 200 STARTING MODEL: NULL                                                 \nREMARK 200                                                                      \nREMARK 200 REMARK: NULL                                                         \nREMARK 280                                                                      \nREMARK 280 CRYSTAL                                                              \nREMARK 280 SOLVENT CONTENT, VS   (%): 54.43                                     \nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.70                     \nREMARK 280                                                                      \nREMARK 280 CRYSTALLIZATION CONDITIONS: PEG 8000, CALCIUM ACETATE, CACODYLATE,   \nREMARK 280  PH 6.5, VAPOR DIFFUSION, HANGING DROP, TEMPERATURE 298K             \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 1 21 1                         \nREMARK 290                                                                      \nREMARK 290      SYMOP   SYMMETRY                                                \nREMARK 290     NNNMMM   OPERATOR                                                \nREMARK 290       1555   X,Y,Z                                                   \nREMARK 290       2555   -X,Y+1/2,-Z                                             \nREMARK 290                                                                      \nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \nREMARK 290           MMM -> TRANSLATION VECTOR                                  \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \nREMARK 290 RELATED MOLECULES.                                                   \nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   2  0.000000  1.000000  0.000000       32.60000            \nREMARK 290   SMTRY3   2  0.000000  0.000000 -1.000000        0.00000            \nREMARK 290                                                                      \nREMARK 290 REMARK: NULL                                                         \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1                                                       \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TETRAMERIC                        \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: W, K, D, B                            \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 465                                                                      \nREMARK 465 MISSING RESIDUES                                                     \nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \nREMARK 465                                                                      \nREMARK 465   M RES C SSSEQI                                                     \nREMARK 465     ILE D    57                                                      \nREMARK 465     ASN D    58                                                      \nREMARK 465     HIS D    59                                                      \nREMARK 465     LEU D    60                                                      \nREMARK 465     ASN D    61                                                      \nREMARK 465     TYR D    62                                                      \nREMARK 465     LYS D    63                                                      \nREMARK 465     GLN D    64                                                      \nREMARK 465     PRO D    65                                                      \nREMARK 465     GLN D    66                                                      \nREMARK 465     ASN D   112                                                      \nREMARK 465     LYS D   113                                                      \nREMARK 465     ARG D   114                                                      \nREMARK 465     ILE D   115                                                      \nREMARK 465     THR D   116                                                      \nREMARK 465     GLU D   117                                                      \nREMARK 465     ALA D   118                                                      \nREMARK 465     ASN D   119                                                      \nREMARK 465     ASN D   120                                                      \nREMARK 465     GLU D   121                                                      \nREMARK 465     ILE D   122                                                      \nREMARK 465     GLU D   123                                                      \nREMARK 465     LEU D   124                                                      \nREMARK 465     ILE B    57                                                      \nREMARK 465     ASN B    58                                                      \nREMARK 465     HIS B    59                                                      \nREMARK 465     LEU B    60                                                      \nREMARK 465     ASN B    61                                                      \nREMARK 465     TYR B    62                                                      \nREMARK 465     LYS B    63                                                      \nREMARK 465     GLN B    64                                                      \nREMARK 465     PRO B    65                                                      \nREMARK 465     GLN B    66                                                      \nREMARK 465     GLU B   117                                                      \nREMARK 465     ALA B   118                                                      \nREMARK 465     ASN B   119                                                      \nREMARK 465     ASN B   120                                                      \nREMARK 465     GLU B   121                                                      \nREMARK 465     ILE B   122                                                      \nREMARK 465     GLU B   123                                                      \nREMARK 465     LEU B   124                                                      \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: CLOSE CONTACTS IN SAME ASYMMETRIC UNIT                     \nREMARK 500                                                                      \nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \nREMARK 500                                                                      \nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \nREMARK 500   CG2  ILE D     3     CG2  ILE D     6              1.54            \nREMARK 500   N1    DA W     3     N6    DA K     7              1.83            \nREMARK 500   N6    DA W     5     N6    DA K     5              1.99            \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \nREMARK 500     DT W   4   C5     DT W   4   C7      0.046                       \nREMARK 500     DT W   8   C5'    DT W   8   C4'     0.083                       \nREMARK 500     DT W   8   C5     DT W   8   C7      0.037                       \nREMARK 500     DT K   4   C5     DT K   4   C7      0.045                       \nREMARK 500     DT K   6   C5'    DT K   6   C4'     0.051                       \nREMARK 500     DT K   8   C5     DT K   8   C7      0.038                       \nREMARK 500     DA K   9   C5'    DA K   9   C4'     0.057                       \nREMARK 500    CYS D   9   CB    CYS D   9   SG      0.150                       \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \nREMARK 500     DA W   3   OP1 -  P   -  OP2 ANGL. DEV. = -16.7 DEGREES          \nREMARK 500     DA W   3   O4' -  C4' -  C3' ANGL. DEV. =   6.0 DEGREES          \nREMARK 500     DA W   3   C2' -  C3' -  O3' ANGL. DEV. =  27.6 DEGREES          \nREMARK 500     DA W   5   O4' -  C1' -  C2' ANGL. DEV. =   3.7 DEGREES          \nREMARK 500     DT W   6   O4' -  C4' -  C3' ANGL. DEV. =  -3.3 DEGREES          \nREMARK 500     DT W   6   C5' -  C4' -  C3' ANGL. DEV. =   8.4 DEGREES          \nREMARK 500     DT W   6   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \nREMARK 500     DT W   6   C3' -  O3' -  P   ANGL. DEV. =   9.8 DEGREES          \nREMARK 500     DA W   7   O4' -  C4' -  C3' ANGL. DEV. =  -5.6 DEGREES          \nREMARK 500     DA W   7   C3' -  C2' -  C1' ANGL. DEV. =  -7.3 DEGREES          \nREMARK 500     DT W   8   O4' -  C4' -  C3' ANGL. DEV. =  -9.6 DEGREES          \nREMARK 500     DT W   8   C3' -  C2' -  C1' ANGL. DEV. =  -6.2 DEGREES          \nREMARK 500     DT W   8   O4' -  C1' -  C2' ANGL. DEV. =   3.3 DEGREES          \nREMARK 500     DA W   9   O5' -  P   -  OP2 ANGL. DEV. =  -8.7 DEGREES          \nREMARK 500     DA K   3   OP1 -  P   -  OP2 ANGL. DEV. =  24.1 DEGREES          \nREMARK 500     DA K   3   O5' -  P   -  OP2 ANGL. DEV. = -16.9 DEGREES          \nREMARK 500     DA K   3   O4' -  C1' -  C2' ANGL. DEV. =   3.9 DEGREES          \nREMARK 500     DA K   5   O4' -  C1' -  C2' ANGL. DEV. =   3.6 DEGREES          \nREMARK 500     DT K   6   O4' -  C4' -  C3' ANGL. DEV. =  -4.7 DEGREES          \nREMARK 500     DT K   6   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \nREMARK 500     DA K   7   O4' -  C4' -  C3' ANGL. DEV. =  -5.2 DEGREES          \nREMARK 500     DA K   7   C3' -  C2' -  C1' ANGL. DEV. =  -7.4 DEGREES          \nREMARK 500     DA K   7   O4' -  C1' -  N9  ANGL. DEV. =   2.4 DEGREES          \nREMARK 500     DT K   8   O4' -  C4' -  C3' ANGL. DEV. =  -5.7 DEGREES          \nREMARK 500     DA K   9   O4' -  C1' -  C2' ANGL. DEV. =   4.1 DEGREES          \nREMARK 500    TYR D  18   N   -  CA  -  CB  ANGL. DEV. = -10.8 DEGREES          \nREMARK 500    ILE D  43   CB  -  CA  -  C   ANGL. DEV. =  13.6 DEGREES          \nREMARK 500    LYS D  54   CB  -  CA  -  C   ANGL. DEV. =  13.5 DEGREES          \nREMARK 500    ARG D  84   NE  -  CZ  -  NH1 ANGL. DEV. =   3.2 DEGREES          \nREMARK 500    LEU B  22   CB  -  CA  -  C   ANGL. DEV. =  13.9 DEGREES          \nREMARK 500    LEU B  40   CA  -  CB  -  CG  ANGL. DEV. =  15.4 DEGREES          \nREMARK 500    TYR B  51   CB  -  CA  -  C   ANGL. DEV. =  12.9 DEGREES          \nREMARK 500    SER B  75   N   -  CA  -  CB  ANGL. DEV. =   9.3 DEGREES          \nREMARK 500    HIS B  87   CB  -  CA  -  C   ANGL. DEV. =  15.5 DEGREES          \nREMARK 500    LEU B  93   CA  -  CB  -  CG  ANGL. DEV. =  15.8 DEGREES          \nREMARK 500    ARG B 110   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \nREMARK 500    ARG B 114   NE  -  CZ  -  NH2 ANGL. DEV. =   4.2 DEGREES          \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    ILE D   3     -178.06    -45.65                                   \nREMARK 500    THR D   4     -114.92   -130.16                                   \nREMARK 500    ASP D   8      163.78    -44.68                                   \nREMARK 500    CYS D   9      136.33    -20.07                                   \nREMARK 500    GLU D  11       -7.44    -54.47                                   \nREMARK 500    LEU D  12      -63.20   -105.61                                   \nREMARK 500    LYS D  21      -84.39    -48.89                                   \nREMARK 500    LEU D  22      -30.42    -31.12                                   \nREMARK 500    PHE D  30     -119.25   -173.96                                   \nREMARK 500    SER D  31      177.89    -30.73                                   \nREMARK 500    ILE D  32      -43.84   -153.41                                   \nREMARK 500    SER D  33       -7.15    -53.85                                   \nREMARK 500    LYS D  54       -6.88    -52.40                                   \nREMARK 500    LYS D  69        0.05    -69.33                                   \nREMARK 500    VAL D  92        5.99    -56.35                                   \nREMARK 500    GLN D 100     -127.42    -51.20                                   \nREMARK 500    ARG D 101      -17.82    -44.99                                   \nREMARK 500    ILE B   3      178.33    -38.43                                   \nREMARK 500    THR B   4      -66.17   -159.76                                   \nREMARK 500    ILE B   6       78.82    -62.67                                   \nREMARK 500    ASN B   7       54.57   -164.26                                   \nREMARK 500    PHE B  10      -89.38    -80.37                                   \nREMARK 500    LEU B  13      -89.43    -49.61                                   \nREMARK 500    PHE B  30     -130.28   -157.46                                   \nREMARK 500    SER B  31     -143.94    -49.49                                   \nREMARK 500    ILE B  32      -17.29   -173.16                                   \nREMARK 500    LEU B  40      -34.99    -36.62                                   \nREMARK 500    THR B  41       11.05    -67.61                                   \nREMARK 500    GLU B  48      -74.82    -67.08                                   \nREMARK 500    LYS B  54        6.84    -57.89                                   \nREMARK 500    ASP B  55      -48.58   -138.52                                   \nREMARK 500    ALA B  70      -76.77    -61.67                                   \nREMARK 500    ARG B  90       24.51   -140.30                                   \nREMARK 500    VAL B  92        3.04    -69.82                                   \nREMARK 500    ILE B  94      -88.33    -49.57                                   \nREMARK 500    SER B 106      -74.50    -51.20                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 620                                                                      \nREMARK 620 METAL COORDINATION                                                   \nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA B 200  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1  DT W   6   OP1                                                    \nREMARK 620 2  DA K   9   OP1 122.5                                              \nREMARK 620 3 GLU B  11   OE2 159.9  52.4                                        \nREMARK 620 N                    1     2                                         \nREMARK 800                                                                      \nREMARK 800 SITE                                                                 \nREMARK 800 SITE_IDENTIFIER: AC1                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA D 200                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC2                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA B 200                  \nREMARK 900                                                                      \nREMARK 900 RELATED ENTRIES                                                      \nREMARK 900 RELATED ID: 1FZN   RELATED DB: PDB                                   \nREMARK 900 CRYSTAL STRUCTURE OF SARA                                            \nDBREF  1FZP D    2   124  UNP    Q53600   SARA_STAAU       1    123             \nDBREF  1FZP B    2   124  UNP    Q53600   SARA_STAAU       1    123             \nDBREF  1FZP W    3     9  PDB    1FZP     1FZP             3      9             \nDBREF  1FZP K    3     9  PDB    1FZP     1FZP             3      9             \nSEQRES   1 W    7   DA  DT  DA  DT  DA  DT  DA                                  \nSEQRES   1 K    7   DA  DT  DA  DT  DA  DT  DA                                  \nSEQRES   1 D  123  ALA ILE THR LYS ILE ASN ASP CYS PHE GLU LEU LEU SER          \nSEQRES   2 D  123  MET VAL THR TYR ALA ASP LYS LEU LYS SER LEU ILE LYS          \nSEQRES   3 D  123  LYS GLU PHE SER ILE SER PHE GLU GLU PHE ALA VAL LEU          \nSEQRES   4 D  123  THR TYR ILE SER GLU ASN LYS GLU LYS GLU TYR TYR LEU          \nSEQRES   5 D  123  LYS ASP ILE ILE ASN HIS LEU ASN TYR LYS GLN PRO GLN          \nSEQRES   6 D  123  VAL VAL LYS ALA VAL LYS ILE LEU SER GLN GLU ASP TYR          \nSEQRES   7 D  123  PHE ASP LYS LYS ARG ASN GLU HIS ASP GLU ARG THR VAL          \nSEQRES   8 D  123  LEU ILE LEU VAL ASN ALA GLN GLN ARG LYS LYS ILE GLU          \nSEQRES   9 D  123  SER LEU LEU SER ARG VAL ASN LYS ARG ILE THR GLU ALA          \nSEQRES  10 D  123  ASN ASN GLU ILE GLU LEU                                      \nSEQRES   1 B  123  ALA ILE THR LYS ILE ASN ASP CYS PHE GLU LEU LEU SER          \nSEQRES   2 B  123  MET VAL THR TYR ALA ASP LYS LEU LYS SER LEU ILE LYS          \nSEQRES   3 B  123  LYS GLU PHE SER ILE SER PHE GLU GLU PHE ALA VAL LEU          \nSEQRES   4 B  123  THR TYR ILE SER GLU ASN LYS GLU LYS GLU TYR TYR LEU          \nSEQRES   5 B  123  LYS ASP ILE ILE ASN HIS LEU ASN TYR LYS GLN PRO GLN          \nSEQRES   6 B  123  VAL VAL LYS ALA VAL LYS ILE LEU SER GLN GLU ASP TYR          \nSEQRES   7 B  123  PHE ASP LYS LYS ARG ASN GLU HIS ASP GLU ARG THR VAL          \nSEQRES   8 B  123  LEU ILE LEU VAL ASN ALA GLN GLN ARG LYS LYS ILE GLU          \nSEQRES   9 B  123  SER LEU LEU SER ARG VAL ASN LYS ARG ILE THR GLU ALA          \nSEQRES  10 B  123  ASN ASN GLU ILE GLU LEU                                      \nHET     CA  D 200       1                                                       \nHET     CA  B 200       1                                                       \nHETNAM      CA CALCIUM ION                                                      \nFORMUL   5   CA    2(CA 2+)                                                     \nHELIX    1   1 CYS D    9  SER D   14  1                                   6    \nHELIX    2   2 SER D   14  PHE D   30  1                                  17    \nHELIX    3   3 ILE D   32  LEU D   40  1                                   9    \nHELIX    4   4 THR D   41  ILE D   43  5                                   3    \nHELIX    5   5 SER D   44  ASN D   46  5                                   3    \nHELIX    6   6 LYS D   47  ASP D   55  1                                   9    \nHELIX    7   7 VAL D   67  ASP D   88  1                                  22    \nHELIX    8   8 THR D   91  VAL D  111  1                                  21    \nHELIX    9   9 PHE B   10  PHE B   30  1                                  21    \nHELIX   10  10 ILE B   32  THR B   41  1                                  10    \nHELIX   11  11 ASN B   46  LYS B   54  1                                   9    \nHELIX   12  12 VAL B   67  GLU B   86  1                                  20    \nHELIX   13  13 HIS B   87  ARG B   90  5                                   4    \nHELIX   14  14 VAL B   92  SER B  106  1                                  15    \nHELIX   15  15 LEU B  107  ILE B  115  1                                   9    \nLINK         OP1  DT W   6                CA    CA B 200     1555   1555  2.72  \nLINK         OP1  DA K   9                CA    CA B 200     1555   1555  2.86  \nLINK         OE2 GLU D  11                CA    CA D 200     1555   1555  1.55  \nLINK         OE2 GLU B  11                CA    CA B 200     1555   1555  2.83  \nCISPEP   1 ASP B    8    CYS B    9          0        -0.12                     \nSITE     1 AC1  3 ARG B  84  GLU D  11   DA W   9                               \nSITE     1 AC2  3 GLU B  11   DA K   9   DT W   6                               \nCRYST1   54.500   65.200   57.800  90.00 118.00  90.00 P 1 21 1      4          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      0.018349  0.000000  0.009756        0.00000                         \nSCALE2      0.000000  0.015337  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  0.019595        0.00000                         \nATOM      1  P    DA W   3       9.641  15.812 -14.279  1.00 60.87           P  \nATOM      2  OP1  DA W   3      11.080  15.668 -13.967  1.00 61.26           O  \nATOM      3  OP2  DA W   3       9.648  16.156 -15.719  1.00 58.83           O  \nATOM      4  O5'  DA W   3       8.881  16.904 -13.366  1.00 52.60           O  \nATOM      5  C5'  DA W   3       8.038  17.881 -14.015  1.00 60.20           C  \nATOM      6  C4'  DA W   3       8.557  19.302 -13.796  1.00 49.01           C  \nATOM      7  O4'  DA W   3       9.729  19.156 -12.966  1.00 48.83           O  \nATOM      8  C3'  DA W   3       8.866  20.000 -15.090  1.00 35.76           C  \nATOM      9  O3'  DA W   3       7.904  21.019 -15.130  1.00 43.27           O  \nATOM     10  C2'  DA W   3      10.365  19.814 -15.172  1.00 39.03           C  \nATOM     11  C1'  DA W   3      10.772  19.770 -13.699  1.00 42.39           C  \nATOM     12  N9   DA W   3      12.002  19.098 -13.307  1.00 42.05           N  \nATOM     13  C8   DA W   3      12.677  18.064 -13.908  1.00 44.95           C  \nATOM     14  N7   DA W   3      13.772  17.704 -13.264  1.00 45.41           N  \nATOM     15  C5   DA W   3      13.810  18.572 -12.185  1.00 45.31           C  \nATOM     16  C6   DA W   3      14.713  18.729 -11.123  1.00 49.24           C  \nATOM     17  N6   DA W   3      15.801  17.954 -10.968  1.00 52.80           N  \nATOM     18  N1   DA W   3      14.449  19.669 -10.224  1.00 48.99           N  \nATOM     19  C2   DA W   3      13.342  20.435 -10.364  1.00 45.51           C  \nATOM     20  N3   DA W   3      12.427  20.395 -11.325  1.00 40.11           N  \nATOM     21  C4   DA W   3      12.727  19.428 -12.204  1.00 41.62           C  \nATOM     22  P    DT W   4       8.232  22.500 -15.583  1.00 34.80           P  \nATOM     23  OP1  DT W   4       6.992  23.161 -16.069  1.00 35.01           O  \nATOM     24  OP2  DT W   4       9.439  22.531 -16.438  1.00 34.65           O  \nATOM     25  O5'  DT W   4       8.718  23.233 -14.233  1.00 25.99           O  \nATOM     26  C5'  DT W   4       7.934  24.338 -13.767  1.00 33.01           C  \nATOM     27  C4'  DT W   4       8.666  25.239 -12.753  1.00 28.60           C  \nATOM     28  O4'  DT W   4       9.670  24.428 -12.079  1.00 29.43           O  \nATOM     29  C3'  DT W   4       9.449  26.446 -13.268  1.00 24.64           C  \nATOM     30  O3'  DT W   4       9.477  27.637 -12.437  1.00 31.85           O  \nATOM     31  C2'  DT W   4      10.872  25.977 -13.443  1.00 27.60           C  \nATOM     32  C1'  DT W   4      10.951  25.062 -12.236  1.00 30.47           C  \nATOM     33  N1   DT W   4      11.975  24.001 -12.309  1.00 33.73           N  \nATOM     34  C2   DT W   4      12.786  23.715 -11.235  1.00 34.62           C  \nATOM     35  O2   DT W   4      12.710  24.309 -10.167  1.00 33.57           O  \nATOM     36  N3   DT W   4      13.709  22.690 -11.443  1.00 37.00           N  \nATOM     37  C4   DT W   4      13.875  21.940 -12.599  1.00 38.58           C  \nATOM     38  O4   DT W   4      14.709  21.034 -12.654  1.00 39.28           O  \nATOM     39  C5   DT W   4      12.987  22.317 -13.664  1.00 38.21           C  \nATOM     40  C7   DT W   4      13.076  21.580 -15.016  1.00 39.44           C  \nATOM     41  C6   DT W   4      12.093  23.308 -13.485  1.00 36.12           C  \nATOM     42  P    DA W   5       8.769  27.750 -11.014  1.00 39.52           P  \nATOM     43  OP1  DA W   5       8.269  26.430 -10.593  1.00 42.95           O  \nATOM     44  OP2  DA W   5       7.766  28.824 -11.033  1.00 37.95           O  \nATOM     45  O5'  DA W   5      10.021  28.223 -10.089  1.00 29.73           O  \nATOM     46  C5'  DA W   5       9.842  28.823  -8.796  1.00 36.11           C  \nATOM     47  C4'  DA W   5      10.922  28.351  -7.804  1.00 28.81           C  \nATOM     48  O4'  DA W   5      11.603  27.240  -8.469  1.00 25.64           O  \nATOM     49  C3'  DA W   5      11.986  29.390  -7.380  1.00 23.74           C  \nATOM     50  O3'  DA W   5      12.315  29.352  -5.995  1.00 30.96           O  \nATOM     51  C2'  DA W   5      13.160  29.117  -8.323  1.00 22.88           C  \nATOM     52  C1'  DA W   5      12.958  27.636  -8.701  1.00 22.63           C  \nATOM     53  N9   DA W   5      13.282  27.398 -10.145  1.00 19.18           N  \nATOM     54  C8   DA W   5      12.441  27.543 -11.197  1.00 18.07           C  \nATOM     55  N7   DA W   5      12.989  27.292 -12.331  1.00 18.87           N  \nATOM     56  C5   DA W   5      14.300  26.965 -12.027  1.00 18.30           C  \nATOM     57  C6   DA W   5      15.422  26.598 -12.825  1.00 15.95           C  \nATOM     58  N6   DA W   5      15.357  26.505 -14.179  1.00 15.73           N  \nATOM     59  N1   DA W   5      16.577  26.346 -12.220  1.00 13.80           N  \nATOM     60  C2   DA W   5      16.636  26.443 -10.857  1.00 13.79           C  \nATOM     61  N3   DA W   5      15.655  26.781 -10.006  1.00 15.62           N  \nATOM     62  C4   DA W   5      14.511  27.031 -10.660  1.00 17.82           C  \nATOM     63  P    DT W   6      12.774  30.665  -5.178  1.00 38.40           P  \nATOM     64  OP1  DT W   6      12.309  30.564  -3.785  1.00 38.43           O  \nATOM     65  OP2  DT W   6      12.371  31.893  -5.903  1.00 40.00           O  \nATOM     66  O5'  DT W   6      14.393  30.512  -5.171  1.00 29.17           O  \nATOM     67  C5'  DT W   6      15.031  29.529  -6.023  1.00 36.03           C  \nATOM     68  C4'  DT W   6      16.514  29.273  -5.674  1.00 29.53           C  \nATOM     69  O4'  DT W   6      17.122  28.747  -6.878  1.00 27.83           O  \nATOM     70  C3'  DT W   6      17.519  30.384  -5.291  1.00 22.49           C  \nATOM     71  O3'  DT W   6      18.707  29.871  -4.582  1.00 28.54           O  \nATOM     72  C2'  DT W   6      17.788  31.031  -6.645  1.00 25.05           C  \nATOM     73  C1'  DT W   6      17.808  29.797  -7.555  1.00 27.11           C  \nATOM     74  N1   DT W   6      17.327  29.926  -8.945  1.00 27.65           N  \nATOM     75  C2   DT W   6      18.248  29.631  -9.898  1.00 28.01           C  \nATOM     76  O2   DT W   6      19.375  29.274  -9.645  1.00 27.37           O  \nATOM     77  N3   DT W   6      17.836  29.736 -11.203  1.00 29.56           N  \nATOM     78  C4   DT W   6      16.579  30.146 -11.631  1.00 31.93           C  \nATOM     79  O4   DT W   6      16.345  30.202 -12.850  1.00 34.79           O  \nATOM     80  C5   DT W   6      15.648  30.483 -10.551  1.00 30.30           C  \nATOM     81  C7   DT W   6      14.236  30.980 -10.869  1.00 30.70           C  \nATOM     82  C6   DT W   6      16.057  30.366  -9.265  1.00 28.44           C  \nATOM     83  P    DA W   7      20.031  30.662  -4.080  1.00 34.92           P  \nATOM     84  OP1  DA W   7      20.535  30.072  -2.816  1.00 33.68           O  \nATOM     85  OP2  DA W   7      19.765  32.117  -4.081  1.00 35.51           O  \nATOM     86  O5'  DA W   7      21.127  30.346  -5.241  1.00 25.19           O  \nATOM     87  C5'  DA W   7      22.478  29.945  -4.865  1.00 31.59           C  \nATOM     88  C4'  DA W   7      22.875  28.646  -5.593  1.00 26.23           C  \nATOM     89  O4'  DA W   7      22.127  28.591  -6.876  1.00 26.40           O  \nATOM     90  C3'  DA W   7      24.275  28.517  -6.135  1.00 22.88           C  \nATOM     91  O3'  DA W   7      24.601  27.171  -6.507  1.00 30.08           O  \nATOM     92  C2'  DA W   7      24.353  29.313  -7.404  1.00 25.56           C  \nATOM     93  C1'  DA W   7      23.032  28.801  -7.965  1.00 27.56           C  \nATOM     94  N9   DA W   7      22.373  29.681  -9.003  1.00 31.34           N  \nATOM     95  C8   DA W   7      21.840  30.927  -8.788  1.00 33.09           C  \nATOM     96  N7   DA W   7      21.342  31.483  -9.840  1.00 33.74           N  \nATOM     97  C5   DA W   7      21.530  30.548 -10.831  1.00 32.48           C  \nATOM     98  C6   DA W   7      21.183  30.556 -12.211  1.00 30.67           C  \nATOM     99  N6   DA W   7      20.593  31.598 -12.821  1.00 29.18           N  \nATOM    100  N1   DA W   7      21.476  29.464 -12.916  1.00 30.78           N  \nATOM    101  C2   DA W   7      22.107  28.440 -12.301  1.00 31.59           C  \nATOM    102  N3   DA W   7      22.513  28.337 -11.032  1.00 31.79           N  \nATOM    103  C4   DA W   7      22.175  29.428 -10.334  1.00 32.05           C  \nATOM    104  P    DT W   8      25.673  26.847  -7.640  1.00 31.38           P  \nATOM    105  OP1  DT W   8      26.738  25.972  -7.097  1.00 32.37           O  \nATOM    106  OP2  DT W   8      26.121  28.098  -8.292  1.00 29.11           O  \nATOM    107  O5'  DT W   8      24.805  26.060  -8.713  1.00 22.13           O  \nATOM    108  C5'  DT W   8      24.427  26.742  -9.932  1.00 28.92           C  \nATOM    109  C4'  DT W   8      25.319  26.273 -11.168  1.00 26.02           C  \nATOM    110  O4'  DT W   8      25.105  27.167 -12.290  1.00 27.10           O  \nATOM    111  C3'  DT W   8      26.895  26.464 -11.140  1.00 22.10           C  \nATOM    112  O3'  DT W   8      27.615  26.128 -12.381  1.00 29.12           O  \nATOM    113  C2'  DT W   8      27.130  27.928 -11.219  1.00 23.97           C  \nATOM    114  C1'  DT W   8      26.071  28.207 -12.279  1.00 26.38           C  \nATOM    115  N1   DT W   8      25.351  29.466 -12.109  1.00 28.67           N  \nATOM    116  C2   DT W   8      24.546  29.847 -13.149  1.00 27.54           C  \nATOM    117  O2   DT W   8      24.398  29.194 -14.178  1.00 25.93           O  \nATOM    118  N3   DT W   8      23.907  31.040 -12.953  1.00 28.37           N  \nATOM    119  C4   DT W   8      23.982  31.872 -11.837  1.00 30.03           C  \nATOM    120  O4   DT W   8      23.339  32.908 -11.817  1.00 31.02           O  \nATOM    121  C5   DT W   8      24.835  31.397 -10.778  1.00 30.15           C  \nATOM    122  C7   DT W   8      25.021  32.176  -9.471  1.00 30.04           C  \nATOM    123  C6   DT W   8      25.479  30.229 -10.955  1.00 29.94           C  \nATOM    124  P    DA W   9      29.112  26.641 -12.750  1.00 53.38           P  \nATOM    125  OP1  DA W   9      30.043  26.282 -11.656  1.00 50.45           O  \nATOM    126  OP2  DA W   9      29.112  27.994 -13.346  1.00 49.37           O  \nATOM    127  O5'  DA W   9      29.379  25.836 -14.095  1.00 45.21           O  \nATOM    128  C5'  DA W   9      28.259  25.069 -14.535  1.00 53.03           C  \nATOM    129  C4'  DA W   9      27.927  25.472 -15.972  1.00 54.85           C  \nATOM    130  O4'  DA W   9      27.014  26.644 -15.916  1.00 59.77           O  \nATOM    131  C3'  DA W   9      29.119  25.875 -16.819  1.00 50.06           C  \nATOM    132  O3'  DA W   9      29.857  24.745 -17.264  1.00 57.89           O  \nATOM    133  C2'  DA W   9      28.515  26.894 -17.778  1.00 53.26           C  \nATOM    134  C1'  DA W   9      27.463  27.584 -16.911  1.00 57.42           C  \nATOM    135  N9   DA W   9      27.939  28.767 -16.197  1.00 55.28           N  \nATOM    136  C8   DA W   9      28.858  28.808 -15.189  1.00 50.94           C  \nATOM    137  N7   DA W   9      29.069  29.998 -14.699  1.00 50.03           N  \nATOM    138  C5   DA W   9      28.228  30.808 -15.439  1.00 50.48           C  \nATOM    139  C6   DA W   9      27.987  32.188 -15.412  1.00 49.49           C  \nATOM    140  N6   DA W   9      28.624  33.027 -14.570  1.00 48.59           N  \nATOM    141  N1   DA W   9      27.091  32.683 -16.289  1.00 51.70           N  \nATOM    142  C2   DA W   9      26.487  31.836 -17.127  1.00 53.03           C  \nATOM    143  N3   DA W   9      26.620  30.518 -17.244  1.00 54.02           N  \nATOM    144  C4   DA W   9      27.520  30.061 -16.366  1.00 53.72           C  \nTER     145       DA W   9                                                      \nATOM    146  P    DA K   3      21.580  36.193 -19.437  1.00 26.48           P  \nATOM    147  OP1  DA K   3      21.455  36.048 -17.969  1.00 23.46           O  \nATOM    148  OP2  DA K   3      22.499  35.998 -20.582  1.00 24.13           O  \nATOM    149  O5'  DA K   3      20.469  35.359 -20.228  1.00 18.50           O  \nATOM    150  C5'  DA K   3      20.768  34.963 -21.571  1.00 26.26           C  \nATOM    151  C4'  DA K   3      20.601  33.458 -21.685  1.00 24.05           C  \nATOM    152  O4'  DA K   3      21.311  32.849 -20.547  1.00 25.74           O  \nATOM    153  C3'  DA K   3      19.177  32.969 -21.676  1.00 20.64           C  \nATOM    154  O3'  DA K   3      18.954  32.075 -22.732  1.00 27.37           O  \nATOM    155  C2'  DA K   3      18.922  32.395 -20.302  1.00 23.53           C  \nATOM    156  C1'  DA K   3      20.329  32.153 -19.754  1.00 26.62           C  \nATOM    157  N9   DA K   3      20.437  32.706 -18.387  1.00 30.34           N  \nATOM    158  C8   DA K   3      19.498  33.418 -17.705  1.00 32.04           C  \nATOM    159  N7   DA K   3      19.876  33.775 -16.503  1.00 33.26           N  \nATOM    160  C5   DA K   3      21.167  33.284 -16.402  1.00 33.71           C  \nATOM    161  C6   DA K   3      22.122  33.326 -15.370  1.00 34.66           C  \nATOM    162  N6   DA K   3      21.892  33.945 -14.189  1.00 36.69           N  \nATOM    163  N1   DA K   3      23.290  32.709 -15.598  1.00 32.42           N  \nATOM    164  C2   DA K   3      23.509  32.105 -16.777  1.00 30.76           C  \nATOM    165  N3   DA K   3      22.707  31.990 -17.824  1.00 30.48           N  \nATOM    166  C4   DA K   3      21.529  32.604 -17.562  1.00 31.72           C  \nATOM    167  P    DT K   4      18.071  30.780 -22.469  1.00 34.80           P  \nATOM    168  OP1  DT K   4      17.740  30.111 -23.743  1.00 35.01           O  \nATOM    169  OP2  DT K   4      17.018  31.073 -21.467  1.00 34.65           O  \nATOM    170  O5'  DT K   4      19.087  29.760 -21.715  1.00 25.99           O  \nATOM    171  C5'  DT K   4      19.502  28.563 -22.431  1.00 33.01           C  \nATOM    172  C4'  DT K   4      20.153  27.484 -21.549  1.00 28.60           C  \nATOM    173  O4'  DT K   4      20.662  28.121 -20.342  1.00 29.43           O  \nATOM    174  C3'  DT K   4      19.249  26.376 -21.035  1.00 24.64           C  \nATOM    175  O3'  DT K   4      19.819  25.073 -20.882  1.00 31.85           O  \nATOM    176  C2'  DT K   4      18.779  26.816 -19.664  1.00 27.60           C  \nATOM    177  C1'  DT K   4      20.059  27.486 -19.202  1.00 30.47           C  \nATOM    178  N1   DT K   4      19.897  28.507 -18.148  1.00 33.73           N  \nATOM    179  C2   DT K   4      20.775  28.553 -17.058  1.00 34.62           C  \nATOM    180  O2   DT K   4      21.705  27.772 -16.904  1.00 33.57           O  \nATOM    181  N3   DT K   4      20.506  29.568 -16.140  1.00 37.00           N  \nATOM    182  C4   DT K   4      19.466  30.522 -16.225  1.00 38.58           C  \nATOM    183  O4   DT K   4      19.330  31.388 -15.359  1.00 39.28           O  \nATOM    184  C5   DT K   4      18.630  30.381 -17.387  1.00 38.21           C  \nATOM    185  C7   DT K   4      17.460  31.357 -17.620  1.00 39.44           C  \nATOM    186  C6   DT K   4      18.870  29.404 -18.282  1.00 36.12           C  \nATOM    187  P    DA K   5      21.341  24.689 -21.206  1.00 39.52           P  \nATOM    188  OP1  DA K   5      22.187  25.895 -21.318  1.00 42.95           O  \nATOM    189  OP2  DA K   5      21.376  23.778 -22.365  1.00 37.95           O  \nATOM    190  O5'  DA K   5      21.766  23.949 -19.836  1.00 29.73           O  \nATOM    191  C5'  DA K   5      22.990  23.180 -19.707  1.00 36.11           C  \nATOM    192  C4'  DA K   5      23.700  23.426 -18.353  1.00 28.81           C  \nATOM    193  O4'  DA K   5      23.078  24.619 -17.778  1.00 25.64           O  \nATOM    194  C3'  DA K   5      23.605  22.291 -17.305  1.00 23.74           C  \nATOM    195  O3'  DA K   5      24.867  22.058 -16.626  1.00 30.96           O  \nATOM    196  C2'  DA K   5      22.438  22.690 -16.417  1.00 22.88           C  \nATOM    197  C1'  DA K   5      22.425  24.229 -16.566  1.00 22.63           C  \nATOM    198  N9   DA K   5      21.011  24.723 -16.608  1.00 19.18           N  \nATOM    199  C8   DA K   5      20.216  24.806 -17.702  1.00 18.07           C  \nATOM    200  N7   DA K   5      19.036  25.247 -17.447  1.00 18.87           N  \nATOM    201  C5   DA K   5      19.028  25.468 -16.080  1.00 18.30           C  \nATOM    202  C6   DA K   5      18.033  25.941 -15.176  1.00 15.95           C  \nATOM    203  N6   DA K   5      16.784  26.288 -15.550  1.00 15.73           N  \nATOM    204  N1   DA K   5      18.339  26.036 -13.887  1.00 13.80           N  \nATOM    205  C2   DA K   5      19.600  25.689 -13.494  1.00 13.79           C  \nATOM    206  N3   DA K   5      20.610  25.224 -14.238  1.00 15.62           N  \nATOM    207  C4   DA K   5      20.255  25.139 -15.529  1.00 17.82           C  \nATOM    208  P    DT K   6      25.292  20.600 -16.081  1.00 38.40           P  \nATOM    209  OP1  DT K   6      26.754  20.390 -16.254  1.00 38.43           O  \nATOM    210  OP2  DT K   6      24.408  19.575 -16.663  1.00 40.00           O  \nATOM    211  O5'  DT K   6      25.014  20.757 -14.477  1.00 29.17           O  \nATOM    212  C5'  DT K   6      23.986  21.674 -14.034  1.00 36.03           C  \nATOM    213  C4'  DT K   6      24.055  21.937 -12.495  1.00 29.53           C  \nATOM    214  O4'  DT K   6      22.841  22.659 -12.178  1.00 27.83           O  \nATOM    215  C3'  DT K   6      23.925  20.740 -11.489  1.00 22.49           C  \nATOM    216  O3'  DT K   6      24.397  21.103 -10.151  1.00 28.54           O  \nATOM    217  C2'  DT K   6      22.490  20.351 -11.664  1.00 25.05           C  \nATOM    218  C1'  DT K   6      21.834  21.732 -11.779  1.00 27.11           C  \nATOM    219  N1   DT K   6      20.624  21.882 -12.612  1.00 27.65           N  \nATOM    220  C2   DT K   6      19.524  22.321 -11.947  1.00 28.01           C  \nATOM    221  O2   DT K   6      19.522  22.586 -10.768  1.00 27.37           O  \nATOM    222  N3   DT K   6      18.380  22.476 -12.690  1.00 29.56           N  \nATOM    223  C4   DT K   6      18.242  22.196 -14.044  1.00 31.93           C  \nATOM    224  O4   DT K   6      17.140  22.377 -14.589  1.00 34.79           O  \nATOM    225  C5   DT K   6      19.458  21.693 -14.657  1.00 30.30           C  \nATOM    226  C7   DT K   6      19.420  21.310 -16.139  1.00 30.70           C  \nATOM    227  C6   DT K   6      20.587  21.552 -13.953  1.00 28.44           C  \nATOM    228  P    DA K   7      24.380  20.157  -8.811  1.00 34.92           P  \nATOM    229  OP1  DA K   7      25.573  20.423  -7.990  1.00 33.68           O  \nATOM    230  OP2  DA K   7      24.169  18.738  -9.187  1.00 35.51           O  \nATOM    231  O5'  DA K   7      23.070  20.680  -8.031  1.00 25.19           O  \nATOM    232  C5'  DA K   7      23.101  20.927  -6.601  1.00 31.59           C  \nATOM    233  C4'  DA K   7      22.528  22.327  -6.300  1.00 26.23           C  \nATOM    234  O4'  DA K   7      21.552  22.671  -7.339  1.00 26.40           O  \nATOM    235  C3'  DA K   7      21.660  22.509  -5.082  1.00 22.88           C  \nATOM    236  O3'  DA K   7      21.462  23.896  -4.761  1.00 30.08           O  \nATOM    237  C2'  DA K   7      20.299  21.963  -5.400  1.00 25.56           C  \nATOM    238  C1'  DA K   7      20.211  22.615  -6.775  1.00 27.56           C  \nATOM    239  N9   DA K   7      19.252  21.962  -7.741  1.00 31.34           N  \nATOM    240  C8   DA K   7      19.386  20.721  -8.309  1.00 33.09           C  \nATOM    241  N7   DA K   7      18.436  20.390  -9.104  1.00 33.74           N  \nATOM    242  C5   DA K   7      17.599  21.487  -9.105  1.00 32.48           C  \nATOM    243  C6   DA K   7      16.383  21.750  -9.796  1.00 30.67           C  \nATOM    244  N6   DA K   7      15.785  20.861 -10.607  1.00 29.18           N  \nATOM    245  N1   DA K   7      15.824  22.945  -9.606  1.00 30.78           N  \nATOM    246  C2   DA K   7      16.414  23.814  -8.756  1.00 31.59           C  \nATOM    247  N3   DA K   7      17.525  23.663  -8.029  1.00 31.79           N  \nATOM    248  C4   DA K   7      18.089  22.472  -8.263  1.00 32.05           C  \nATOM    249  P    DT K   8      20.145  24.385  -3.989  1.00 31.38           P  \nATOM    250  OP1  DT K   8      20.507  25.107  -2.748  1.00 32.37           O  \nATOM    251  OP2  DT K   8      19.213  23.257  -3.821  1.00 29.11           O  \nATOM    252  O5'  DT K   8      19.465  25.391  -5.045  1.00 22.13           O  \nATOM    253  C5'  DT K   8      18.342  24.955  -5.825  1.00 28.92           C  \nATOM    254  C4'  DT K   8      17.068  25.621  -5.337  1.00 26.02           C  \nATOM    255  O4'  DT K   8      15.886  24.930  -5.849  1.00 27.10           O  \nATOM    256  C3'  DT K   8      16.794  25.478  -3.846  1.00 22.10           C  \nATOM    257  O3'  DT K   8      15.930  26.507  -3.370  1.00 29.12           O  \nATOM    258  C2'  DT K   8      16.281  24.061  -3.664  1.00 23.97           C  \nATOM    259  C1'  DT K   8      15.451  23.933  -4.935  1.00 26.38           C  \nATOM    260  N1   DT K   8      15.546  22.663  -5.668  1.00 28.67           N  \nATOM    261  C2   DT K   8      14.711  22.511  -6.743  1.00 27.54           C  \nATOM    262  O2   DT K   8      13.889  23.320  -7.129  1.00 25.93           O  \nATOM    263  N3   DT K   8      14.864  21.324  -7.405  1.00 28.37           N  \nATOM    264  C4   DT K   8      15.758  20.296  -7.112  1.00 30.03           C  \nATOM    265  O4   DT K   8      15.742  19.302  -7.819  1.00 31.02           O  \nATOM    266  C5   DT K   8      16.614  20.535  -5.979  1.00 30.15           C  \nATOM    267  C7   DT K   8      17.669  19.518  -5.526  1.00 30.04           C  \nATOM    268  C6   DT K   8      16.474  21.693  -5.309  1.00 29.94           C  \nATOM    269  P    DA K   9      14.870  26.222  -2.212  1.00 36.70           P  \nATOM    270  OP1  DA K   9      14.914  27.327  -1.214  1.00 39.42           O  \nATOM    271  OP2  DA K   9      14.964  24.819  -1.740  1.00 35.88           O  \nATOM    272  O5'  DA K   9      13.467  26.209  -2.984  1.00 27.24           O  \nATOM    273  C5'  DA K   9      13.294  27.037  -4.138  1.00 34.10           C  \nATOM    274  C4'  DA K   9      11.803  27.095  -4.624  1.00 31.41           C  \nATOM    275  O4'  DA K   9      11.655  25.987  -5.580  1.00 31.10           O  \nATOM    276  C3'  DA K   9      10.693  27.008  -3.570  1.00 29.70           C  \nATOM    277  O3'  DA K   9       9.929  28.204  -3.487  1.00 37.25           O  \nATOM    278  C2'  DA K   9       9.880  25.749  -3.910  1.00 30.63           C  \nATOM    279  C1'  DA K   9      10.580  25.154  -5.156  1.00 30.32           C  \nATOM    280  N9   DA K   9      11.159  23.809  -4.949  1.00 31.53           N  \nATOM    281  C8   DA K   9      11.590  23.242  -3.765  1.00 33.33           C  \nATOM    282  N7   DA K   9      12.103  22.052  -3.882  1.00 34.17           N  \nATOM    283  C5   DA K   9      11.999  21.777  -5.237  1.00 33.36           C  \nATOM    284  C6   DA K   9      12.354  20.653  -6.002  1.00 33.46           C  \nATOM    285  N6   DA K   9      12.917  19.543  -5.463  1.00 35.72           N  \nATOM    286  N1   DA K   9      12.102  20.689  -7.324  1.00 31.61           N  \nATOM    287  C2   DA K   9      11.541  21.778  -7.834  1.00 32.23           C  \nATOM    288  N3   DA K   9      11.154  22.908  -7.221  1.00 33.15           N  \nATOM    289  C4   DA K   9      11.412  22.848  -5.907  1.00 32.62           C  \nTER     290       DA K   9                                                      \nATOM    291  N   ALA D   2      50.958  39.087 -10.179  1.00 12.13           N  \nATOM    292  CA  ALA D   2      49.826  39.020  -9.253  1.00 25.72           C  \nATOM    293  C   ALA D   2      48.587  38.371  -9.881  1.00 26.64           C  \nATOM    294  O   ALA D   2      48.155  38.719 -10.995  1.00 20.61           O  \nATOM    295  CB  ALA D   2      49.542  40.350  -8.568  1.00 29.94           C  \nATOM    296  N   ILE D   3      48.036  37.432  -9.111  1.00 34.84           N  \nATOM    297  CA  ILE D   3      46.885  36.614  -9.466  1.00 35.33           C  \nATOM    298  C   ILE D   3      45.679  37.326 -10.091  1.00 45.94           C  \nATOM    299  O   ILE D   3      45.694  38.543 -10.315  1.00 60.47           O  \nATOM    300  CB  ILE D   3      46.411  35.981  -8.180  1.00 29.83           C  \nATOM    301  CG1 ILE D   3      47.474  35.119  -7.516  1.00 29.01           C  \nATOM    302  CG2 ILE D   3      45.190  35.093  -8.358  1.00 31.21           C  \nATOM    303  CD1 ILE D   3      48.353  35.933  -6.552  1.00 13.95           C  \nATOM    304  N   THR D   4      44.705  36.433 -10.312  1.00 19.10           N  \nATOM    305  CA  THR D   4      43.385  36.687 -10.915  1.00 17.01           C  \nATOM    306  C   THR D   4      42.306  36.079 -10.021  1.00 12.28           C  \nATOM    307  O   THR D   4      42.106  36.505  -8.879  1.00 11.02           O  \nATOM    308  CB  THR D   4      43.339  35.981 -12.274  1.00 38.33           C  \nATOM    309  OG1 THR D   4      44.640  35.498 -12.606  1.00 26.23           O  \nATOM    310  CG2 THR D   4      42.876  36.876 -13.411  1.00 78.24           C  \nATOM    311  N   LYS D   5      41.652  35.089 -10.599  1.00 37.72           N  \nATOM    312  CA  LYS D   5      40.609  34.300  -9.928  1.00 50.23           C  \nATOM    313  C   LYS D   5      41.316  33.085  -9.336  1.00 35.87           C  \nATOM    314  O   LYS D   5      41.199  31.966  -9.834  1.00 21.33           O  \nATOM    315  CB  LYS D   5      39.548  33.869 -10.978  1.00 65.66           C  \nATOM    316  CG  LYS D   5      38.442  32.912 -10.450  1.00 34.40           C  \nATOM    317  CD  LYS D   5      37.078  33.142 -11.176  1.00 66.00           C  \nATOM    318  CE  LYS D   5      36.720  34.605 -11.183  1.00 80.20           C  \nATOM    319  NZ  LYS D   5      35.416  35.145 -11.786  1.00 81.00           N  \nATOM    320  N   ILE D   6      42.065  33.292  -8.255  1.00 34.10           N  \nATOM    321  CA  ILE D   6      42.817  32.167  -7.679  1.00 32.39           C  \nATOM    322  C   ILE D   6      42.141  31.551  -6.423  1.00 54.01           C  \nATOM    323  O   ILE D   6      42.638  30.575  -5.850  1.00 53.35           O  \nATOM    324  CB  ILE D   6      44.258  32.594  -7.340  1.00 39.09           C  \nATOM    325  CG1 ILE D   6      44.428  33.258  -5.983  1.00 43.56           C  \nATOM    326  CG2 ILE D   6      44.836  33.594  -8.349  1.00 47.25           C  \nATOM    327  CD1 ILE D   6      45.905  33.481  -5.619  1.00 35.04           C  \nATOM    328  N   ASN D   7      41.001  32.096  -5.988  1.00 67.67           N  \nATOM    329  CA  ASN D   7      40.268  31.512  -4.811  1.00 68.09           C  \nATOM    330  C   ASN D   7      39.814  30.093  -5.247  1.00 40.38           C  \nATOM    331  O   ASN D   7      39.363  29.886  -6.381  1.00 28.27           O  \nATOM    332  CB  ASN D   7      39.105  32.421  -4.436  1.00 99.39           C  \nATOM    333  CG  ASN D   7      39.584  33.702  -3.727  1.00100.00           C  \nATOM    334  OD1 ASN D   7      40.487  34.382  -4.222  1.00 42.30           O  \nATOM    335  ND2 ASN D   7      39.030  34.078  -2.588  1.00 37.24           N  \nATOM    336  N   ASP D   8      39.920  29.103  -4.341  1.00 42.64           N  \nATOM    337  CA  ASP D   8      39.644  27.689  -4.739  1.00 46.17           C  \nATOM    338  C   ASP D   8      38.395  27.497  -5.590  1.00 39.42           C  \nATOM    339  O   ASP D   8      37.587  28.422  -5.644  1.00 31.26           O  \nATOM    340  CB  ASP D   8      39.508  26.688  -3.626  1.00 45.25           C  \nATOM    341  CG  ASP D   8      40.617  26.733  -2.590  1.00 30.86           C  \nATOM    342  OD1 ASP D   8      41.652  27.471  -2.738  1.00 37.97           O  \nATOM    343  OD2 ASP D   8      40.444  26.053  -1.507  1.00 30.96           O  \nATOM    344  N   CYS D   9      38.295  26.317  -6.246  1.00 49.86           N  \nATOM    345  CA  CYS D   9      37.213  25.897  -7.161  1.00 45.11           C  \nATOM    346  C   CYS D   9      35.983  26.718  -6.928  1.00 33.58           C  \nATOM    347  O   CYS D   9      35.569  26.987  -5.824  1.00 10.28           O  \nATOM    348  CB  CYS D   9      36.912  24.418  -7.197  1.00 41.91           C  \nATOM    349  SG  CYS D   9      35.001  24.217  -7.620  1.00 42.41           S  \nATOM    350  N   PHE D  10      35.370  27.136  -8.027  1.00 45.12           N  \nATOM    351  CA  PHE D  10      34.267  28.068  -7.876  1.00 48.17           C  \nATOM    352  C   PHE D  10      32.876  27.371  -7.623  1.00 55.06           C  \nATOM    353  O   PHE D  10      32.240  27.600  -6.599  1.00 54.26           O  \nATOM    354  CB  PHE D  10      34.303  29.024  -9.074  1.00 48.88           C  \nATOM    355  CG  PHE D  10      35.770  29.190  -9.599  1.00 45.95           C  \nATOM    356  CD1 PHE D  10      36.887  29.167  -8.716  1.00 32.57           C  \nATOM    357  CD2 PHE D  10      36.002  29.346 -10.962  1.00 32.39           C  \nATOM    358  CE1 PHE D  10      38.206  29.233  -9.235  1.00 23.00           C  \nATOM    359  CE2 PHE D  10      37.307  29.385 -11.477  1.00 21.50           C  \nATOM    360  CZ  PHE D  10      38.409  29.324 -10.618  1.00 10.89           C  \nATOM    361  N   GLU D  11      32.329  26.517  -8.474  1.00 55.21           N  \nATOM    362  CA  GLU D  11      30.954  25.890  -8.143  1.00 61.52           C  \nATOM    363  C   GLU D  11      30.979  25.214  -6.773  1.00 63.12           C  \nATOM    364  O   GLU D  11      29.940  24.767  -6.287  1.00 74.91           O  \nATOM    365  CB  GLU D  11      30.597  24.847  -9.190  1.00 67.93           C  \nATOM    366  CG  GLU D  11      31.820  24.303  -9.919  1.00 99.79           C  \nATOM    367  CD  GLU D  11      31.470  23.186 -10.892  1.00 98.76           C  \nATOM    368  OE1 GLU D  11      32.254  22.173 -11.013  1.00 18.00           O  \nATOM    369  OE2 GLU D  11      30.389  23.259 -11.588  1.00100.00           O  \nATOM    370  N   LEU D  12      32.157  25.124  -6.159  1.00 35.34           N  \nATOM    371  CA  LEU D  12      32.293  24.477  -4.854  1.00 27.41           C  \nATOM    372  C   LEU D  12      32.491  25.466  -3.713  1.00 25.60           C  \nATOM    373  O   LEU D  12      31.688  25.521  -2.777  1.00 13.20           O  \nATOM    374  CB  LEU D  12      33.571  23.643  -4.877  1.00 28.10           C  \nATOM    375  CG  LEU D  12      33.939  22.636  -3.796  1.00 28.65           C  \nATOM    376  CD1 LEU D  12      35.300  22.030  -4.097  1.00 24.30           C  \nATOM    377  CD2 LEU D  12      33.797  23.156  -2.351  1.00 22.37           C  \nATOM    378  N   LEU D  13      33.609  26.192  -3.783  1.00 25.07           N  \nATOM    379  CA  LEU D  13      33.998  27.188  -2.787  1.00 19.78           C  \nATOM    380  C   LEU D  13      32.914  28.255  -2.619  1.00 23.97           C  \nATOM    381  O   LEU D  13      32.796  28.881  -1.560  1.00 21.55           O  \nATOM    382  CB  LEU D  13      35.254  27.874  -3.332  1.00 15.15           C  \nATOM    383  CG  LEU D  13      36.463  28.122  -2.434  1.00 20.84           C  \nATOM    384  CD1 LEU D  13      36.091  28.357  -0.972  1.00 12.36           C  \nATOM    385  CD2 LEU D  13      37.550  27.069  -2.688  1.00 18.59           C  \nATOM    386  N   SER D  14      32.130  28.459  -3.677  1.00 19.28           N  \nATOM    387  CA  SER D  14      31.063  29.443  -3.642  1.00 11.02           C  \nATOM    388  C   SER D  14      30.079  29.137  -2.521  1.00 32.80           C  \nATOM    389  O   SER D  14      29.871  29.964  -1.635  1.00 42.74           O  \nATOM    390  CB  SER D  14      30.366  29.544  -4.972  1.00 31.34           C  \nATOM    391  OG  SER D  14      29.783  30.845  -5.080  1.00 19.55           O  \nATOM    392  N   MET D  15      29.484  27.941  -2.556  1.00 24.27           N  \nATOM    393  CA  MET D  15      28.526  27.528  -1.530  1.00 10.32           C  \nATOM    394  C   MET D  15      29.216  27.235  -0.200  1.00 13.61           C  \nATOM    395  O   MET D  15      28.620  27.407   0.869  1.00 11.53           O  \nATOM    396  CB  MET D  15      27.590  26.428  -1.942  1.00 17.84           C  \nATOM    397  CG  MET D  15      26.164  26.903  -1.993  1.00 20.63           C  \nATOM    398  SD  MET D  15      25.690  27.396  -3.628  1.00 33.91           S  \nATOM    399  CE  MET D  15      27.217  28.131  -4.165  1.00 12.75           C  \nATOM    400  N   VAL D  16      30.473  26.791  -0.258  1.00 11.00           N  \nATOM    401  CA  VAL D  16      31.205  26.507   0.974  1.00 11.09           C  \nATOM    402  C   VAL D  16      31.613  27.812   1.662  1.00 25.29           C  \nATOM    403  O   VAL D  16      32.387  27.826   2.630  1.00 28.57           O  \nATOM    404  CB  VAL D  16      32.186  25.244   0.989  1.00 12.28           C  \nATOM    405  CG1 VAL D  16      33.028  25.113   2.273  1.00 11.31           C  \nATOM    406  CG2 VAL D  16      31.430  23.909   0.796  1.00 15.50           C  \nATOM    407  N   THR D  17      31.093  28.914   1.127  1.00 27.53           N  \nATOM    408  CA  THR D  17      31.349  30.250   1.661  1.00 38.36           C  \nATOM    409  C   THR D  17      30.047  30.780   2.240  1.00 29.69           C  \nATOM    410  O   THR D  17      29.887  31.001   3.446  1.00 10.05           O  \nATOM    411  CB  THR D  17      31.713  31.210   0.452  1.00 44.58           C  \nATOM    412  OG1 THR D  17      32.898  30.743  -0.218  1.00 35.14           O  \nATOM    413  CG2 THR D  17      31.897  32.651   0.906  1.00 21.05           C  \nATOM    414  N   TYR D  18      29.114  30.952   1.327  1.00 24.67           N  \nATOM    415  CA  TYR D  18      27.780  31.406   1.623  1.00 14.48           C  \nATOM    416  C   TYR D  18      27.226  30.527   2.736  1.00 21.54           C  \nATOM    417  O   TYR D  18      26.702  31.037   3.720  1.00 27.15           O  \nATOM    418  CB  TYR D  18      27.142  31.269   0.230  1.00 14.73           C  \nATOM    419  CG  TYR D  18      25.789  31.698  -0.035  1.00 21.07           C  \nATOM    420  CD1 TYR D  18      25.260  31.493  -1.267  1.00 21.20           C  \nATOM    421  CD2 TYR D  18      24.993  32.196   0.958  1.00 26.27           C  \nATOM    422  CE1 TYR D  18      23.957  31.842  -1.557  1.00 23.93           C  \nATOM    423  CE2 TYR D  18      23.676  32.544   0.708  1.00 12.85           C  \nATOM    424  CZ  TYR D  18      23.153  32.363  -0.571  1.00 33.77           C  \nATOM    425  OH  TYR D  18      21.844  32.693  -0.897  1.00 57.14           O  \nATOM    426  N   ALA D  19      27.397  29.208   2.600  1.00 29.93           N  \nATOM    427  CA  ALA D  19      26.934  28.267   3.617  1.00 31.37           C  \nATOM    428  C   ALA D  19      27.459  28.672   4.991  1.00 35.21           C  \nATOM    429  O   ALA D  19      26.682  28.777   5.934  1.00 14.53           O  \nATOM    430  CB  ALA D  19      27.304  26.804   3.262  1.00 30.19           C  \nATOM    431  N   ASP D  20      28.766  28.948   5.081  1.00 58.41           N  \nATOM    432  CA  ASP D  20      29.406  29.376   6.333  1.00 63.68           C  \nATOM    433  C   ASP D  20      28.614  30.511   6.967  1.00 60.59           C  \nATOM    434  O   ASP D  20      28.196  30.454   8.128  1.00 56.23           O  \nATOM    435  CB  ASP D  20      30.723  30.131   5.971  1.00 65.36           C  \nATOM    436  CG  ASP D  20      31.971  29.269   6.005  1.00 32.49           C  \nATOM    437  OD1 ASP D  20      32.413  29.007   7.167  1.00 16.75           O  \nATOM    438  OD2 ASP D  20      32.533  28.988   4.896  1.00 13.29           O  \nATOM    439  N   LYS D  21      28.418  31.546   6.160  1.00 45.69           N  \nATOM    440  CA  LYS D  21      27.694  32.735   6.566  1.00 41.29           C  \nATOM    441  C   LYS D  21      26.373  32.374   7.221  1.00 33.49           C  \nATOM    442  O   LYS D  21      26.289  32.297   8.450  1.00 31.83           O  \nATOM    443  CB  LYS D  21      27.338  33.545   5.315  1.00 47.53           C  \nATOM    444  CG  LYS D  21      28.057  34.887   5.178  1.00100.00           C  \nATOM    445  CD  LYS D  21      28.618  35.124   3.782  1.00 85.37           C  \nATOM    446  CE  LYS D  21      29.590  36.292   3.810  1.00 43.77           C  \nATOM    447  NZ  LYS D  21      28.834  37.574   3.788  1.00 42.32           N  \nATOM    448  N   LEU D  22      25.367  32.163   6.363  1.00 31.07           N  \nATOM    449  CA  LEU D  22      23.982  31.828   6.714  1.00 19.89           C  \nATOM    450  C   LEU D  22      23.867  31.021   7.999  1.00 20.94           C  \nATOM    451  O   LEU D  22      22.891  31.147   8.727  1.00 13.97           O  \nATOM    452  CB  LEU D  22      23.255  31.193   5.493  1.00 14.26           C  \nATOM    453  CG  LEU D  22      23.042  32.168   4.273  1.00 18.42           C  \nATOM    454  CD1 LEU D  22      24.299  32.955   3.772  1.00 11.20           C  \nATOM    455  CD2 LEU D  22      22.122  31.639   3.160  1.00 14.21           C  \nATOM    456  N   LYS D  23      24.882  30.222   8.284  1.00 23.55           N  \nATOM    457  CA  LYS D  23      24.875  29.422   9.495  1.00 29.31           C  \nATOM    458  C   LYS D  23      25.108  30.316  10.711  1.00 36.54           C  \nATOM    459  O   LYS D  23      24.464  30.166  11.753  1.00 40.89           O  \nATOM    460  CB  LYS D  23      25.896  28.255   9.398  1.00 28.10           C  \nATOM    461  CG  LYS D  23      25.356  26.879   8.948  1.00 19.58           C  \nATOM    462  CD  LYS D  23      26.251  26.109   7.919  1.00 13.14           C  \nATOM    463  CE  LYS D  23      27.464  25.444   8.507  1.00 11.23           C  \nATOM    464  NZ  LYS D  23      27.574  26.064   9.842  1.00 25.20           N  \nATOM    465  N   SER D  24      26.028  31.262  10.589  1.00 14.01           N  \nATOM    466  CA  SER D  24      26.309  32.138  11.712  1.00 10.32           C  \nATOM    467  C   SER D  24      25.158  33.097  11.965  1.00 20.06           C  \nATOM    468  O   SER D  24      24.823  33.385  13.115  1.00 16.35           O  \nATOM    469  CB  SER D  24      27.667  32.808  11.621  1.00 11.00           C  \nATOM    470  OG  SER D  24      28.202  32.959  12.932  1.00 12.84           O  \nATOM    471  N   LEU D  25      24.548  33.563  10.879  1.00 22.08           N  \nATOM    472  CA  LEU D  25      23.422  34.486  10.956  1.00 13.94           C  \nATOM    473  C   LEU D  25      22.365  33.983  11.933  1.00 33.28           C  \nATOM    474  O   LEU D  25      21.881  34.750  12.763  1.00 22.76           O  \nATOM    475  CB  LEU D  25      22.792  34.570   9.590  1.00 12.80           C  \nATOM    476  CG  LEU D  25      21.961  35.782   9.565  1.00 17.55           C  \nATOM    477  CD1 LEU D  25      22.725  36.750  10.441  1.00 22.42           C  \nATOM    478  CD2 LEU D  25      21.786  36.285   8.118  1.00 11.75           C  \nATOM    479  N   ILE D  26      22.040  32.688  11.824  1.00 43.15           N  \nATOM    480  CA  ILE D  26      21.068  32.012  12.684  1.00 24.08           C  \nATOM    481  C   ILE D  26      21.700  31.770  14.063  1.00 33.06           C  \nATOM    482  O   ILE D  26      21.023  31.692  15.091  1.00 25.60           O  \nATOM    483  CB  ILE D  26      20.703  30.633  11.986  1.00 10.64           C  \nATOM    484  CG1 ILE D  26      20.543  30.870  10.489  1.00 13.59           C  \nATOM    485  CG2 ILE D  26      19.394  30.064  12.497  1.00 12.95           C  \nATOM    486  CD1 ILE D  26      20.408  29.633   9.619  1.00 11.30           C  \nATOM    487  N   LYS D  27      23.017  31.666  14.078  1.00 32.26           N  \nATOM    488  CA  LYS D  27      23.723  31.455  15.319  1.00 33.32           C  \nATOM    489  C   LYS D  27      23.531  32.641  16.254  1.00 57.04           C  \nATOM    490  O   LYS D  27      22.956  32.490  17.326  1.00 72.37           O  \nATOM    491  CB  LYS D  27      25.204  31.365  15.011  1.00 38.49           C  \nATOM    492  CG  LYS D  27      25.649  30.099  14.299  1.00 35.76           C  \nATOM    493  CD  LYS D  27      27.169  29.920  14.372  1.00 21.42           C  \nATOM    494  CE  LYS D  27      27.734  30.361  15.723  1.00 29.46           C  \nATOM    495  NZ  LYS D  27      26.819  30.054  16.874  1.00 21.55           N  \nATOM    496  N   LYS D  28      23.983  33.816  15.824  1.00 46.74           N  \nATOM    497  CA  LYS D  28      23.876  35.027  16.636  1.00 33.16           C  \nATOM    498  C   LYS D  28      22.443  35.541  16.778  1.00 22.54           C  \nATOM    499  O   LYS D  28      21.994  35.801  17.890  1.00 14.05           O  \nATOM    500  CB  LYS D  28      24.866  36.109  16.203  1.00 29.46           C  \nATOM    501  CG  LYS D  28      25.492  36.901  17.350  1.00 87.85           C  \nATOM    502  CD  LYS D  28      25.012  38.336  17.493  1.00 35.26           C  \nATOM    503  CE  LYS D  28      26.122  39.065  18.235  1.00 31.46           C  \nATOM    504  NZ  LYS D  28      26.044  38.862  19.714  1.00 31.10           N  \nATOM    505  N   GLU D  29      21.711  35.703  15.675  1.00 17.63           N  \nATOM    506  CA  GLU D  29      20.345  36.205  15.795  1.00 21.88           C  \nATOM    507  C   GLU D  29      19.485  35.373  16.740  1.00 32.26           C  \nATOM    508  O   GLU D  29      18.502  35.871  17.292  1.00 27.54           O  \nATOM    509  CB  GLU D  29      19.657  36.811  14.589  1.00 28.79           C  \nATOM    510  CG  GLU D  29      18.797  38.025  15.017  1.00 78.50           C  \nATOM    511  CD  GLU D  29      19.617  39.329  15.361  1.00 92.42           C  \nATOM    512  OE1 GLU D  29      20.534  39.675  14.541  1.00 79.59           O  \nATOM    513  OE2 GLU D  29      19.299  40.029  16.399  1.00 11.34           O  \nATOM    514  N   PHE D  30      19.884  34.109  16.933  1.00 25.04           N  \nATOM    515  CA  PHE D  30      19.203  33.169  17.819  1.00 28.94           C  \nATOM    516  C   PHE D  30      19.943  31.865  17.961  1.00 25.26           C  \nATOM    517  O   PHE D  30      21.071  31.839  18.438  1.00 25.98           O  \nATOM    518  CB  PHE D  30      17.717  32.861  17.659  1.00 14.53           C  \nATOM    519  CG  PHE D  30      17.213  32.802  16.240  1.00 26.84           C  \nATOM    520  CD1 PHE D  30      18.068  32.792  15.157  1.00 21.01           C  \nATOM    521  CD2 PHE D  30      15.831  32.721  16.023  1.00 21.64           C  \nATOM    522  CE1 PHE D  30      17.553  32.767  13.873  1.00 15.98           C  \nATOM    523  CE2 PHE D  30      15.306  32.697  14.744  1.00 17.50           C  \nATOM    524  CZ  PHE D  30      16.166  32.714  13.653  1.00 17.29           C  \nATOM    525  N   SER D  31      19.260  30.791  17.553  1.00 36.53           N  \nATOM    526  CA  SER D  31      19.709  29.387  17.616  1.00 40.85           C  \nATOM    527  C   SER D  31      21.212  29.150  17.493  1.00 37.13           C  \nATOM    528  O   SER D  31      22.019  30.066  17.309  1.00 47.40           O  \nATOM    529  CB  SER D  31      18.988  28.491  16.580  1.00 40.85           C  \nATOM    530  OG  SER D  31      19.702  28.348  15.348  1.00 13.80           O  \nATOM    531  N   ILE D  32      21.558  27.875  17.601  1.00 11.14           N  \nATOM    532  CA  ILE D  32      22.933  27.416  17.502  1.00 12.37           C  \nATOM    533  C   ILE D  32      22.938  25.980  17.016  1.00 18.19           C  \nATOM    534  O   ILE D  32      23.693  25.629  16.122  1.00 14.83           O  \nATOM    535  CB  ILE D  32      23.728  27.408  18.852  1.00 13.56           C  \nATOM    536  CG1 ILE D  32      25.058  26.664  18.651  1.00 14.35           C  \nATOM    537  CG2 ILE D  32      22.981  26.571  19.898  1.00 25.91           C  \nATOM    538  CD1 ILE D  32      26.385  27.373  18.861  1.00 11.34           C  \nATOM    539  N   SER D  33      22.044  25.155  17.577  1.00 44.46           N  \nATOM    540  CA  SER D  33      21.918  23.734  17.201  1.00 49.81           C  \nATOM    541  C   SER D  33      21.723  23.545  15.695  1.00 59.75           C  \nATOM    542  O   SER D  33      21.717  22.416  15.182  1.00 26.49           O  \nATOM    543  CB  SER D  33      20.873  22.962  17.995  1.00 32.94           C  \nATOM    544  OG  SER D  33      19.644  23.035  17.321  1.00 10.99           O  \nATOM    545  N   PHE D  34      21.551  24.665  15.007  1.00 77.72           N  \nATOM    546  CA  PHE D  34      21.415  24.708  13.562  1.00 82.79           C  \nATOM    547  C   PHE D  34      22.843  24.729  13.012  1.00 97.00           C  \nATOM    548  O   PHE D  34      23.107  24.297  11.874  1.00 99.54           O  \nATOM    549  CB  PHE D  34      20.656  26.007  13.213  1.00 80.49           C  \nATOM    550  CG  PHE D  34      19.793  25.899  11.991  1.00 71.84           C  \nATOM    551  CD1 PHE D  34      19.806  26.894  11.003  1.00 60.48           C  \nATOM    552  CD2 PHE D  34      19.010  24.795  11.797  1.00 68.54           C  \nATOM    553  CE1 PHE D  34      19.004  26.807   9.874  1.00 47.84           C  \nATOM    554  CE2 PHE D  34      18.220  24.697  10.656  1.00 74.05           C  \nATOM    555  CZ  PHE D  34      18.213  25.709   9.691  1.00 58.94           C  \nATOM    556  N   GLU D  35      23.759  25.175  13.857  1.00 86.84           N  \nATOM    557  CA  GLU D  35      25.171  25.229  13.523  1.00 82.03           C  \nATOM    558  C   GLU D  35      25.685  23.800  13.627  1.00 66.72           C  \nATOM    559  O   GLU D  35      26.728  23.450  13.082  1.00 64.54           O  \nATOM    560  CB  GLU D  35      25.934  26.139  14.507  1.00 83.60           C  \nATOM    561  CG  GLU D  35      27.448  26.205  14.309  1.00 50.06           C  \nATOM    562  CD  GLU D  35      27.856  27.066  13.109  1.00 57.79           C  \nATOM    563  OE1 GLU D  35      26.952  27.420  12.283  1.00 14.50           O  \nATOM    564  OE2 GLU D  35      29.086  27.338  12.987  1.00 15.78           O  \nATOM    565  N   GLU D  36      24.930  22.986  14.341  1.00 59.45           N  \nATOM    566  CA  GLU D  36      25.269  21.585  14.520  1.00 62.06           C  \nATOM    567  C   GLU D  36      24.325  20.678  13.737  1.00 79.73           C  \nATOM    568  O   GLU D  36      24.724  19.602  13.286  1.00 76.12           O  \nATOM    569  CB  GLU D  36      25.284  21.190  15.975  1.00 58.58           C  \nATOM    570  CG  GLU D  36      26.677  20.886  16.456  1.00 61.00           C  \nATOM    571  CD  GLU D  36      27.390  22.103  16.983  1.00 82.93           C  \nATOM    572  OE1 GLU D  36      27.403  23.126  16.256  1.00100.00           O  \nATOM    573  OE2 GLU D  36      27.920  22.047  18.134  1.00 81.24           O  \nATOM    574  N   PHE D  37      23.076  21.119  13.597  1.00 82.84           N  \nATOM    575  CA  PHE D  37      22.070  20.360  12.863  1.00 83.25           C  \nATOM    576  C   PHE D  37      22.618  19.954  11.494  1.00 88.82           C  \nATOM    577  O   PHE D  37      22.232  18.924  10.939  1.00 83.67           O  \nATOM    578  CB  PHE D  37      20.771  21.198  12.752  1.00 82.61           C  \nATOM    579  CG  PHE D  37      19.494  20.424  13.052  1.00 83.56           C  \nATOM    580  CD1 PHE D  37      19.398  19.591  14.167  1.00 83.07           C  \nATOM    581  CD2 PHE D  37      18.367  20.552  12.235  1.00 83.98           C  \nATOM    582  CE1 PHE D  37      18.229  18.886  14.447  1.00 78.36           C  \nATOM    583  CE2 PHE D  37      17.190  19.852  12.516  1.00 79.19           C  \nATOM    584  CZ  PHE D  37      17.120  19.030  13.629  1.00 74.99           C  \nATOM    585  N   ALA D  38      23.538  20.763  10.967  1.00 88.83           N  \nATOM    586  CA  ALA D  38      24.161  20.490   9.673  1.00 84.07           C  \nATOM    587  C   ALA D  38      25.353  19.556   9.844  1.00 76.44           C  \nATOM    588  O   ALA D  38      25.420  18.497   9.225  1.00 79.66           O  \nATOM    589  CB  ALA D  38      24.565  21.771   8.955  1.00 83.02           C  \nATOM    590  N   VAL D  39      26.294  19.959  10.691  1.00 65.88           N  \nATOM    591  CA  VAL D  39      27.466  19.141  10.950  1.00 67.34           C  \nATOM    592  C   VAL D  39      26.998  17.825  11.549  1.00 79.90           C  \nATOM    593  O   VAL D  39      27.676  16.800  11.462  1.00 84.94           O  \nATOM    594  CB  VAL D  39      28.450  19.837  11.923  1.00 79.92           C  \nATOM    595  CG1 VAL D  39      27.743  20.344  13.173  1.00 78.49           C  \nATOM    596  CG2 VAL D  39      29.576  18.899  12.308  1.00 84.03           C  \nATOM    597  N   LEU D  40      25.814  17.864  12.154  1.00 80.83           N  \nATOM    598  CA  LEU D  40      25.250  16.664  12.751  1.00 79.04           C  \nATOM    599  C   LEU D  40      25.071  15.590  11.682  1.00100.00           C  \nATOM    600  O   LEU D  40      25.162  14.394  11.969  1.00100.00           O  \nATOM    601  CB  LEU D  40      23.917  16.931  13.474  1.00 69.50           C  \nATOM    602  CG  LEU D  40      22.633  16.623  12.698  1.00 58.16           C  \nATOM    603  CD1 LEU D  40      22.308  15.140  12.687  1.00 54.42           C  \nATOM    604  CD2 LEU D  40      21.485  17.392  13.281  1.00 51.63           C  \nATOM    605  N   THR D  41      24.805  16.020  10.447  1.00 98.83           N  \nATOM    606  CA  THR D  41      24.627  15.072   9.354  1.00 98.20           C  \nATOM    607  C   THR D  41      25.850  14.168   9.268  1.00 98.99           C  \nATOM    608  O   THR D  41      25.740  12.981   8.949  1.00100.00           O  \nATOM    609  CB  THR D  41      24.298  15.734   7.985  1.00 97.46           C  \nATOM    610  OG1 THR D  41      25.283  16.722   7.667  1.00 71.69           O  \nATOM    611  CG2 THR D  41      22.911  16.361   7.988  1.00 97.15           C  \nATOM    612  N   TYR D  42      27.019  14.733   9.576  1.00 76.01           N  \nATOM    613  CA  TYR D  42      28.254  13.962   9.549  1.00 69.12           C  \nATOM    614  C   TYR D  42      28.637  13.408  10.923  1.00 63.64           C  \nATOM    615  O   TYR D  42      29.557  12.599  11.031  1.00 69.03           O  \nATOM    616  CB  TYR D  42      29.416  14.507   8.687  1.00 69.79           C  \nATOM    617  CG  TYR D  42      30.558  13.498   8.589  1.00 70.99           C  \nATOM    618  CD1 TYR D  42      30.470  12.397   7.759  1.00 69.13           C  \nATOM    619  CD2 TYR D  42      31.651  13.569   9.457  1.00 78.61           C  \nATOM    620  CE1 TYR D  42      31.465  11.433   7.743  1.00 78.99           C  \nATOM    621  CE2 TYR D  42      32.646  12.606   9.460  1.00 80.69           C  \nATOM    622  CZ  TYR D  42      32.552  11.544   8.599  1.00 85.80           C  \nATOM    623  OH  TYR D  42      33.568  10.613   8.597  1.00 67.99           O  \nATOM    624  N   ILE D  43      27.929  13.816  11.975  1.00 60.75           N  \nATOM    625  CA  ILE D  43      28.251  13.318  13.317  1.00 67.90           C  \nATOM    626  C   ILE D  43      27.045  13.044  14.222  1.00 76.61           C  \nATOM    627  O   ILE D  43      25.917  13.409  13.905  1.00 72.74           O  \nATOM    628  CB  ILE D  43      29.795  13.170  13.755  1.00 69.11           C  \nATOM    629  CG1 ILE D  43      30.034  12.904  15.247  1.00 68.62           C  \nATOM    630  CG2 ILE D  43      30.619  14.339  13.248  1.00 68.16           C  \nATOM    631  CD1 ILE D  43      29.901  11.446  15.644  1.00 56.39           C  \nATOM    632  N   SER D  44      27.294  12.375  15.343  1.00 84.03           N  \nATOM    633  CA  SER D  44      26.259  12.038  16.312  1.00 92.70           C  \nATOM    634  C   SER D  44      26.436  12.827  17.608  1.00100.00           C  \nATOM    635  O   SER D  44      25.461  13.155  18.293  1.00100.00           O  \nATOM    636  CB  SER D  44      26.458  10.577  16.702  1.00100.00           C  \nATOM    637  OG  SER D  44      27.716  10.431  17.354  1.00 99.00           O  \nATOM    638  N   GLU D  45      27.689  13.104  17.948  1.00 90.74           N  \nATOM    639  CA  GLU D  45      28.001  13.833  19.171  1.00 87.33           C  \nATOM    640  C   GLU D  45      27.202  15.118  19.257  1.00 86.18           C  \nATOM    641  O   GLU D  45      26.916  15.615  20.347  1.00 86.48           O  \nATOM    642  CB  GLU D  45      29.493  14.066  19.374  1.00 88.38           C  \nATOM    643  CG  GLU D  45      30.057  13.487  20.684  1.00100.00           C  \nATOM    644  CD  GLU D  45      29.181  13.669  21.954  1.00100.00           C  \nATOM    645  OE1 GLU D  45      27.957  13.389  21.924  1.00 74.39           O  \nATOM    646  OE2 GLU D  45      29.744  14.013  23.027  1.00100.00           O  \nATOM    647  N   ASN D  46      26.840  15.641  18.096  1.00 78.61           N  \nATOM    648  CA  ASN D  46      26.060  16.862  18.019  1.00 77.72           C  \nATOM    649  C   ASN D  46      24.602  16.576  18.378  1.00 78.20           C  \nATOM    650  O   ASN D  46      23.757  17.473  18.382  1.00 80.92           O  \nATOM    651  CB  ASN D  46      26.326  17.542  16.679  1.00 79.27           C  \nATOM    652  CG  ASN D  46      27.836  17.441  16.245  1.00 87.16           C  \nATOM    653  OD1 ASN D  46      28.101  17.102  15.090  1.00 99.73           O  \nATOM    654  ND2 ASN D  46      28.810  17.768  17.151  1.00  4.35           N  \nATOM    655  N   LYS D  47      24.325  15.309  18.690  1.00 60.36           N  \nATOM    656  CA  LYS D  47      22.984  14.879  19.056  1.00 52.15           C  \nATOM    657  C   LYS D  47      22.671  15.080  20.495  1.00 32.91           C  \nATOM    658  O   LYS D  47      21.531  15.248  20.898  1.00 14.59           O  \nATOM    659  CB  LYS D  47      22.703  13.439  18.636  1.00 53.94           C  \nATOM    660  CG  LYS D  47      21.677  13.333  17.520  1.00 13.56           C  \nATOM    661  CD  LYS D  47      20.385  12.671  17.978  1.00 10.08           C  \nATOM    662  CE  LYS D  47      19.283  12.743  16.930  1.00 21.30           C  \nATOM    663  NZ  LYS D  47      19.803  12.798  15.558  1.00 21.60           N  \nATOM    664  N   GLU D  48      23.560  15.113  21.438  1.00 38.48           N  \nATOM    665  CA  GLU D  48      22.907  15.339  22.704  1.00 43.15           C  \nATOM    666  C   GLU D  48      22.851  16.837  23.090  1.00 40.20           C  \nATOM    667  O   GLU D  48      21.824  17.324  23.531  1.00 38.57           O  \nATOM    668  CB  GLU D  48      23.414  14.598  23.854  1.00 47.51           C  \nATOM    669  CG  GLU D  48      22.335  14.643  24.962  1.00 66.82           C  \nATOM    670  CD  GLU D  48      22.861  15.405  26.136  1.00 78.73           C  \nATOM    671  OE1 GLU D  48      22.645  14.965  27.315  1.00 96.93           O  \nATOM    672  OE2 GLU D  48      23.556  16.471  25.939  1.00 54.65           O  \nATOM    673  N   LYS D  49      23.910  17.619  22.967  1.00 63.39           N  \nATOM    674  CA  LYS D  49      23.784  19.117  23.291  1.00 65.33           C  \nATOM    675  C   LYS D  49      22.505  19.615  22.645  1.00 65.95           C  \nATOM    676  O   LYS D  49      21.937  20.610  23.078  1.00 68.59           O  \nATOM    677  CB  LYS D  49      24.961  19.839  22.626  1.00 65.61           C  \nATOM    678  CG  LYS D  49      25.707  20.820  23.503  1.00 97.58           C  \nATOM    679  CD  LYS D  49      27.186  20.897  23.107  1.00100.00           C  \nATOM    680  CE  LYS D  49      27.793  22.297  23.193  1.00 63.86           C  \nATOM    681  NZ  LYS D  49      28.804  22.568  22.157  1.00 36.43           N  \nATOM    682  N   GLU D  50      22.074  18.896  21.611  1.00 58.72           N  \nATOM    683  CA  GLU D  50      20.871  19.232  20.863  1.00 56.68           C  \nATOM    684  C   GLU D  50      19.617  19.072  21.709  1.00 57.11           C  \nATOM    685  O   GLU D  50      18.916  20.025  21.963  1.00 45.98           O  \nATOM    686  CB  GLU D  50      20.824  18.239  19.681  1.00 57.24           C  \nATOM    687  CG  GLU D  50      20.466  18.788  18.319  1.00 23.49           C  \nATOM    688  CD  GLU D  50      19.007  18.874  18.127  1.00 42.96           C  \nATOM    689  OE1 GLU D  50      18.476  18.111  17.270  1.00 40.80           O  \nATOM    690  OE2 GLU D  50      18.400  19.690  18.855  1.00 63.54           O  \nATOM    691  N   TYR D  51      19.356  17.842  22.148  1.00 59.94           N  \nATOM    692  CA  TYR D  51      18.169  17.565  22.941  1.00 53.22           C  \nATOM    693  C   TYR D  51      18.307  17.877  24.421  1.00 30.25           C  \nATOM    694  O   TYR D  51      17.330  18.228  25.067  1.00  8.63           O  \nATOM    695  CB  TYR D  51      17.330  16.399  22.474  1.00 59.72           C  \nATOM    696  CG  TYR D  51      16.133  16.826  21.648  1.00 61.96           C  \nATOM    697  CD1 TYR D  51      14.885  17.034  22.257  1.00 46.73           C  \nATOM    698  CD2 TYR D  51      16.228  17.021  20.266  1.00 67.80           C  \nATOM    699  CE1 TYR D  51      13.778  17.383  21.520  1.00 51.61           C  \nATOM    700  CE2 TYR D  51      15.110  17.403  19.531  1.00 62.66           C  \nATOM    701  CZ  TYR D  51      13.901  17.573  20.177  1.00 51.70           C  \nATOM    702  OH  TYR D  51      12.828  17.922  19.428  1.00 38.50           O  \nATOM    703  N   TYR D  52      19.507  17.756  24.973  1.00 50.74           N  \nATOM    704  CA  TYR D  52      19.679  18.067  26.389  1.00 45.85           C  \nATOM    705  C   TYR D  52      19.285  19.525  26.649  1.00 35.16           C  \nATOM    706  O   TYR D  52      19.085  19.942  27.788  1.00 30.71           O  \nATOM    707  CB  TYR D  52      21.096  17.681  26.887  1.00 34.61           C  \nATOM    708  CG  TYR D  52      21.305  17.785  28.390  1.00 38.40           C  \nATOM    709  CD1 TYR D  52      22.583  17.776  28.961  1.00 46.98           C  \nATOM    710  CD2 TYR D  52      20.204  17.868  29.261  1.00 29.74           C  \nATOM    711  CE1 TYR D  52      22.718  17.893  30.341  1.00 66.43           C  \nATOM    712  CE2 TYR D  52      20.351  18.031  30.623  1.00 11.61           C  \nATOM    713  CZ  TYR D  52      21.580  18.026  31.171  1.00 20.02           C  \nATOM    714  OH  TYR D  52      21.664  18.108  32.534  1.00 18.04           O  \nATOM    715  N   LEU D  53      19.133  20.299  25.579  1.00 25.70           N  \nATOM    716  CA  LEU D  53      18.771  21.705  25.742  1.00 27.86           C  \nATOM    717  C   LEU D  53      17.546  22.108  24.943  1.00 40.71           C  \nATOM    718  O   LEU D  53      16.719  22.882  25.421  1.00 51.21           O  \nATOM    719  CB  LEU D  53      19.870  22.640  25.176  1.00 22.56           C  \nATOM    720  CG  LEU D  53      20.639  23.444  26.208  1.00 19.76           C  \nATOM    721  CD1 LEU D  53      19.918  23.289  27.563  1.00  6.19           C  \nATOM    722  CD2 LEU D  53      22.164  23.115  26.233  1.00 12.00           C  \nATOM    723  N   LYS D  54      17.437  21.600  23.725  1.00 28.33           N  \nATOM    724  CA  LYS D  54      16.333  21.950  22.840  1.00 17.50           C  \nATOM    725  C   LYS D  54      14.950  21.765  23.479  1.00 15.77           C  \nATOM    726  O   LYS D  54      13.937  22.180  22.902  1.00 13.83           O  \nATOM    727  CB  LYS D  54      16.674  21.545  21.375  1.00 18.55           C  \nATOM    728  CG  LYS D  54      15.624  21.598  20.310  1.00 28.79           C  \nATOM    729  CD  LYS D  54      16.291  21.526  18.942  1.00 21.20           C  \nATOM    730  CE  LYS D  54      15.290  21.874  17.848  1.00 10.07           C  \nATOM    731  NZ  LYS D  54      15.879  22.443  16.596  1.00 39.38           N  \nATOM    732  N   ASP D  55      14.931  21.180  24.690  1.00 24.97           N  \nATOM    733  CA  ASP D  55      13.712  20.934  25.463  1.00 23.96           C  \nATOM    734  C   ASP D  55      13.935  21.215  26.959  1.00 25.85           C  \nATOM    735  O   ASP D  55      13.200  20.748  27.831  1.00 23.88           O  \nATOM    736  CB  ASP D  55      13.373  19.412  25.275  1.00 34.95           C  \nATOM    737  CG  ASP D  55      11.983  19.008  25.825  1.00 71.34           C  \nATOM    738  OD1 ASP D  55      11.440  19.577  26.809  1.00 63.59           O  \nATOM    739  OD2 ASP D  55      11.426  18.021  25.313  1.00 80.55           O  \nATOM    740  N   ILE D  56      14.969  21.983  27.273  1.00 30.47           N  \nATOM    741  CA  ILE D  56      15.277  22.276  28.661  1.00 34.21           C  \nATOM    742  C   ILE D  56      15.971  23.642  28.828  1.00 57.56           C  \nATOM    743  O   ILE D  56      16.773  23.812  29.745  1.00 75.97           O  \nATOM    744  CB  ILE D  56      16.240  21.103  29.181  1.00 36.27           C  \nATOM    745  CG1 ILE D  56      15.684  19.687  28.856  1.00 48.68           C  \nATOM    746  CG2 ILE D  56      16.570  21.178  30.664  1.00 15.06           C  \nATOM    747  CD1 ILE D  56      16.523  18.820  27.906  1.00 29.03           C  \nATOM    748  N   VAL D  67       1.980  17.944  18.936  1.00 30.03           N  \nATOM    749  CA  VAL D  67       1.134  19.118  18.499  1.00 27.98           C  \nATOM    750  C   VAL D  67       1.701  20.590  18.789  1.00 49.65           C  \nATOM    751  O   VAL D  67       1.937  21.390  17.859  1.00 54.64           O  \nATOM    752  CB  VAL D  67      -0.219  18.964  19.178  1.00 42.40           C  \nATOM    753  CG1 VAL D  67      -0.107  18.901  20.702  1.00 43.63           C  \nATOM    754  CG2 VAL D  67      -1.196  20.082  18.864  1.00 48.36           C  \nATOM    755  N   VAL D  68       1.929  21.001  20.067  1.00 63.40           N  \nATOM    756  CA  VAL D  68       2.411  22.417  20.332  1.00 74.01           C  \nATOM    757  C   VAL D  68       3.910  22.565  20.052  1.00 73.94           C  \nATOM    758  O   VAL D  68       4.349  23.583  19.505  1.00 75.86           O  \nATOM    759  CB  VAL D  68       2.071  22.904  21.735  1.00 85.88           C  \nATOM    760  CG1 VAL D  68       3.040  23.960  22.254  1.00 85.76           C  \nATOM    761  CG2 VAL D  68       0.676  23.575  21.786  1.00 86.27           C  \nATOM    762  N   LYS D  69       4.698  21.551  20.412  1.00 57.07           N  \nATOM    763  CA  LYS D  69       6.123  21.591  20.148  1.00 52.12           C  \nATOM    764  C   LYS D  69       6.361  21.479  18.651  1.00 44.63           C  \nATOM    765  O   LYS D  69       7.488  21.534  18.177  1.00 50.53           O  \nATOM    766  CB  LYS D  69       6.866  20.617  21.055  1.00 62.44           C  \nATOM    767  CG  LYS D  69       7.060  19.180  20.530  1.00 98.36           C  \nATOM    768  CD  LYS D  69       8.182  19.180  19.500  1.00 84.80           C  \nATOM    769  CE  LYS D  69       9.565  19.241  20.179  1.00 67.00           C  \nATOM    770  NZ  LYS D  69      10.265  20.553  19.929  1.00 51.21           N  \nATOM    771  N   ALA D  70       5.248  21.372  17.917  1.00 32.08           N  \nATOM    772  CA  ALA D  70       5.243  21.280  16.469  1.00 39.57           C  \nATOM    773  C   ALA D  70       5.021  22.666  15.865  1.00 66.71           C  \nATOM    774  O   ALA D  70       5.801  23.110  15.020  1.00 83.65           O  \nATOM    775  CB  ALA D  70       4.259  20.248  15.945  1.00 39.81           C  \nATOM    776  N   VAL D  71       3.962  23.356  16.294  1.00 52.98           N  \nATOM    777  CA  VAL D  71       3.711  24.701  15.787  1.00 46.36           C  \nATOM    778  C   VAL D  71       4.823  25.594  16.299  1.00 23.24           C  \nATOM    779  O   VAL D  71       5.311  26.477  15.598  1.00 28.64           O  \nATOM    780  CB  VAL D  71       2.406  25.407  16.271  1.00 51.75           C  \nATOM    781  CG1 VAL D  71       2.654  26.386  17.439  1.00 47.04           C  \nATOM    782  CG2 VAL D  71       1.797  26.190  15.116  1.00 52.06           C  \nATOM    783  N   LYS D  72       5.230  25.333  17.534  1.00 16.34           N  \nATOM    784  CA  LYS D  72       6.300  26.100  18.140  1.00 23.55           C  \nATOM    785  C   LYS D  72       7.539  26.024  17.251  1.00 32.55           C  \nATOM    786  O   LYS D  72       8.210  27.031  17.016  1.00 51.87           O  \nATOM    787  CB  LYS D  72       6.582  25.661  19.609  1.00 26.76           C  \nATOM    788  CG  LYS D  72       7.915  26.141  20.214  1.00 43.56           C  \nATOM    789  CD  LYS D  72       7.961  27.637  20.580  1.00 14.69           C  \nATOM    790  CE  LYS D  72       8.878  27.917  21.789  1.00 59.51           C  \nATOM    791  NZ  LYS D  72       8.722  27.085  23.047  1.00 11.47           N  \nATOM    792  N   ILE D  73       7.827  24.825  16.748  1.00 16.40           N  \nATOM    793  CA  ILE D  73       8.976  24.629  15.878  1.00 13.28           C  \nATOM    794  C   ILE D  73       8.717  25.236  14.515  1.00 32.23           C  \nATOM    795  O   ILE D  73       9.553  25.973  13.992  1.00 57.72           O  \nATOM    796  CB  ILE D  73       9.278  23.173  15.591  1.00 15.24           C  \nATOM    797  CG1 ILE D  73      10.509  22.732  16.337  1.00 16.58           C  \nATOM    798  CG2 ILE D  73       9.525  22.901  14.125  1.00 11.20           C  \nATOM    799  CD1 ILE D  73      10.172  21.765  17.429  1.00 12.04           C  \nATOM    800  N   LEU D  74       7.552  24.937  13.943  1.00 13.96           N  \nATOM    801  CA  LEU D  74       7.230  25.479  12.623  1.00 17.54           C  \nATOM    802  C   LEU D  74       7.260  27.000  12.636  1.00 54.50           C  \nATOM    803  O   LEU D  74       7.792  27.637  11.732  1.00 73.01           O  \nATOM    804  CB  LEU D  74       5.914  25.012  12.036  1.00 20.29           C  \nATOM    805  CG  LEU D  74       5.885  25.179  10.506  1.00 26.09           C  \nATOM    806  CD1 LEU D  74       4.993  26.313  10.048  1.00 19.88           C  \nATOM    807  CD2 LEU D  74       7.302  25.440   9.999  1.00 41.90           C  \nATOM    808  N   SER D  75       6.666  27.592  13.663  1.00 48.69           N  \nATOM    809  CA  SER D  75       6.661  29.038  13.771  1.00 40.46           C  \nATOM    810  C   SER D  75       8.077  29.572  13.891  1.00 48.37           C  \nATOM    811  O   SER D  75       8.464  30.482  13.158  1.00 67.15           O  \nATOM    812  CB  SER D  75       5.933  29.487  15.008  1.00 22.94           C  \nATOM    813  OG  SER D  75       6.297  30.838  15.191  1.00 13.33           O  \nATOM    814  N   GLN D  76       8.851  29.001  14.815  1.00 11.89           N  \nATOM    815  CA  GLN D  76      10.222  29.452  14.999  1.00 15.93           C  \nATOM    816  C   GLN D  76      11.086  29.052  13.811  1.00 18.28           C  \nATOM    817  O   GLN D  76      12.244  29.451  13.706  1.00 18.36           O  \nATOM    818  CB  GLN D  76      10.841  29.081  16.342  1.00 11.30           C  \nATOM    819  CG  GLN D  76      11.973  28.010  16.243  1.00 45.80           C  \nATOM    820  CD  GLN D  76      13.412  28.584  16.139  1.00 55.73           C  \nATOM    821  OE1 GLN D  76      14.426  27.836  15.985  1.00 36.67           O  \nATOM    822  NE2 GLN D  76      13.509  29.910  16.262  1.00 50.21           N  \nATOM    823  N   GLU D  77      10.519  28.253  12.923  1.00 12.96           N  \nATOM    824  CA  GLU D  77      11.246  27.827  11.745  1.00 22.93           C  \nATOM    825  C   GLU D  77      10.933  28.807  10.631  1.00 36.80           C  \nATOM    826  O   GLU D  77      11.606  28.851   9.607  1.00 66.17           O  \nATOM    827  CB  GLU D  77      11.030  26.339  11.424  1.00 29.83           C  \nATOM    828  CG  GLU D  77      12.307  25.552  11.174  1.00 16.16           C  \nATOM    829  CD  GLU D  77      12.525  25.244   9.698  1.00 59.85           C  \nATOM    830  OE1 GLU D  77      11.712  24.476   9.148  1.00100.00           O  \nATOM    831  OE2 GLU D  77      13.448  25.812   9.081  1.00100.00           O  \nATOM    832  N   ASP D  78       9.893  29.603  10.863  1.00 16.24           N  \nATOM    833  CA  ASP D  78       9.473  30.642   9.920  1.00 24.27           C  \nATOM    834  C   ASP D  78      10.198  31.922  10.296  1.00 30.01           C  \nATOM    835  O   ASP D  78      10.537  32.741   9.430  1.00 32.81           O  \nATOM    836  CB  ASP D  78       7.991  30.991  10.038  1.00 31.56           C  \nATOM    837  CG  ASP D  78       7.571  32.155   9.088  1.00 64.99           C  \nATOM    838  OD1 ASP D  78       8.095  33.299   9.134  1.00 17.65           O  \nATOM    839  OD2 ASP D  78       6.700  31.929   8.186  1.00100.00           O  \nATOM    840  N   TYR D  79      10.430  32.079  11.591  1.00 38.89           N  \nATOM    841  CA  TYR D  79      11.132  33.234  12.097  1.00 30.23           C  \nATOM    842  C   TYR D  79      12.456  33.364  11.372  1.00 48.48           C  \nATOM    843  O   TYR D  79      12.903  34.471  11.080  1.00 56.48           O  \nATOM    844  CB  TYR D  79      11.388  33.118  13.601  1.00 27.02           C  \nATOM    845  CG  TYR D  79      10.286  33.721  14.389  1.00 27.13           C  \nATOM    846  CD1 TYR D  79       9.828  33.122  15.519  1.00 24.34           C  \nATOM    847  CD2 TYR D  79       9.661  34.865  13.958  1.00 43.73           C  \nATOM    848  CE1 TYR D  79       8.793  33.687  16.239  1.00 32.44           C  \nATOM    849  CE2 TYR D  79       8.628  35.430  14.663  1.00 50.00           C  \nATOM    850  CZ  TYR D  79       8.197  34.826  15.802  1.00 53.60           C  \nATOM    851  OH  TYR D  79       7.177  35.372  16.522  1.00100.00           O  \nATOM    852  N   PHE D  80      13.082  32.230  11.076  1.00 50.36           N  \nATOM    853  CA  PHE D  80      14.348  32.264  10.366  1.00 58.96           C  \nATOM    854  C   PHE D  80      14.182  32.231   8.847  1.00 52.00           C  \nATOM    855  O   PHE D  80      15.021  32.753   8.113  1.00 46.74           O  \nATOM    856  CB  PHE D  80      15.502  31.433  10.971  1.00 68.16           C  \nATOM    857  CG  PHE D  80      15.128  30.027  11.354  1.00 83.81           C  \nATOM    858  CD1 PHE D  80      15.411  29.536  12.629  1.00 95.99           C  \nATOM    859  CD2 PHE D  80      14.511  29.180  10.441  1.00 94.21           C  \nATOM    860  CE1 PHE D  80      15.068  28.236  12.987  1.00100.00           C  \nATOM    861  CE2 PHE D  80      14.165  27.880  10.801  1.00 99.09           C  \nATOM    862  CZ  PHE D  80      14.450  27.412  12.073  1.00 96.17           C  \nATOM    863  N   ASP D  81      13.096  31.628   8.372  1.00 46.40           N  \nATOM    864  CA  ASP D  81      12.860  31.528   6.930  1.00 44.54           C  \nATOM    865  C   ASP D  81      12.593  32.863   6.229  1.00 35.59           C  \nATOM    866  O   ASP D  81      12.732  32.947   5.006  1.00 21.53           O  \nATOM    867  CB  ASP D  81      12.111  30.280   6.467  1.00 39.56           C  \nATOM    868  CG  ASP D  81      13.026  29.065   6.336  1.00 34.66           C  \nATOM    869  OD1 ASP D  81      13.121  28.291   7.325  1.00 36.47           O  \nATOM    870  OD2 ASP D  81      13.580  28.847   5.218  1.00 10.28           O  \nATOM    871  N   LYS D  82      12.207  33.889   6.998  1.00 30.23           N  \nATOM    872  CA  LYS D  82      11.940  35.222   6.438  1.00 30.52           C  \nATOM    873  C   LYS D  82      13.180  36.093   6.518  1.00 25.21           C  \nATOM    874  O   LYS D  82      13.497  36.862   5.596  1.00 15.58           O  \nATOM    875  CB  LYS D  82      10.809  36.038   7.068  1.00 24.92           C  \nATOM    876  CG  LYS D  82      10.403  37.192   6.119  1.00 11.00           C  \nATOM    877  CD  LYS D  82       9.241  38.005   6.644  1.00 60.52           C  \nATOM    878  CE  LYS D  82       8.069  37.114   7.011  1.00 94.47           C  \nATOM    879  NZ  LYS D  82       7.070  37.839   7.840  1.00100.00           N  \nATOM    880  N   LYS D  83      13.879  35.958   7.634  1.00 24.05           N  \nATOM    881  CA  LYS D  83      15.100  36.705   7.859  1.00 21.00           C  \nATOM    882  C   LYS D  83      16.177  36.417   6.796  1.00 26.34           C  \nATOM    883  O   LYS D  83      17.036  37.270   6.567  1.00 30.91           O  \nATOM    884  CB  LYS D  83      15.495  36.420   9.289  1.00 41.78           C  \nATOM    885  CG  LYS D  83      16.816  36.942   9.818  1.00 13.18           C  \nATOM    886  CD  LYS D  83      16.591  37.106  11.341  1.00 21.30           C  \nATOM    887  CE  LYS D  83      17.820  37.555  12.068  1.00 21.04           C  \nATOM    888  NZ  LYS D  83      18.355  38.757  11.391  1.00 57.65           N  \nATOM    889  N   ARG D  84      16.143  35.228   6.154  1.00 22.97           N  \nATOM    890  CA  ARG D  84      17.134  34.899   5.121  1.00 15.25           C  \nATOM    891  C   ARG D  84      16.640  35.193   3.727  1.00 15.74           C  \nATOM    892  O   ARG D  84      17.406  35.646   2.864  1.00 33.90           O  \nATOM    893  CB  ARG D  84      17.945  33.590   5.227  1.00 32.42           C  \nATOM    894  CG  ARG D  84      18.847  33.302   3.980  1.00 34.50           C  \nATOM    895  CD  ARG D  84      18.049  32.743   2.740  1.00 46.95           C  \nATOM    896  NE  ARG D  84      18.931  32.443   1.608  1.00 59.76           N  \nATOM    897  CZ  ARG D  84      18.763  31.473   0.690  1.00 31.02           C  \nATOM    898  NH1 ARG D  84      17.702  30.642   0.671  1.00 21.00           N  \nATOM    899  NH2 ARG D  84      19.678  31.357  -0.253  1.00 59.72           N  \nATOM    900  N   ASN D  85      15.364  34.944   3.489  1.00 19.01           N  \nATOM    901  CA  ASN D  85      14.803  35.255   2.197  1.00 19.71           C  \nATOM    902  C   ASN D  85      14.908  36.774   2.086  1.00 39.39           C  \nATOM    903  O   ASN D  85      15.146  37.337   1.016  1.00 41.11           O  \nATOM    904  CB  ASN D  85      13.339  34.792   2.184  1.00 21.30           C  \nATOM    905  CG  ASN D  85      12.786  34.522   0.791  1.00 24.46           C  \nATOM    906  OD1 ASN D  85      11.618  34.876   0.482  1.00 18.51           O  \nATOM    907  ND2 ASN D  85      13.522  33.742   0.016  1.00 27.86           N  \nATOM    908  N   GLU D  86      14.757  37.439   3.224  1.00 48.65           N  \nATOM    909  CA  GLU D  86      14.852  38.888   3.242  1.00 61.43           C  \nATOM    910  C   GLU D  86      16.288  39.334   3.017  1.00 85.11           C  \nATOM    911  O   GLU D  86      16.564  40.211   2.198  1.00 99.04           O  \nATOM    912  CB  GLU D  86      14.227  39.544   4.522  1.00 62.56           C  \nATOM    913  CG  GLU D  86      14.973  39.585   5.879  1.00 41.20           C  \nATOM    914  CD  GLU D  86      14.212  40.509   6.895  1.00 56.66           C  \nATOM    915  OE1 GLU D  86      13.123  41.014   6.510  1.00100.00           O  \nATOM    916  OE2 GLU D  86      14.657  40.743   8.059  1.00 71.00           O  \nATOM    917  N   HIS D  87      17.205  38.721   3.755  1.00 77.80           N  \nATOM    918  CA  HIS D  87      18.612  39.072   3.644  1.00 75.23           C  \nATOM    919  C   HIS D  87      19.202  38.748   2.291  1.00 62.76           C  \nATOM    920  O   HIS D  87      20.301  39.201   1.978  1.00 46.19           O  \nATOM    921  CB  HIS D  87      19.517  38.755   4.851  1.00 81.68           C  \nATOM    922  CG  HIS D  87      20.451  37.605   4.649  1.00 93.79           C  \nATOM    923  ND1 HIS D  87      21.821  37.732   4.759  1.00100.00           N  \nATOM    924  CD2 HIS D  87      20.215  36.296   4.394  1.00100.00           C  \nATOM    925  CE1 HIS D  87      22.393  36.554   4.567  1.00100.00           C  \nATOM    926  NE2 HIS D  87      21.439  35.665   4.347  1.00100.00           N  \nATOM    927  N   ASP D  88      18.468  37.968   1.498  1.00 74.06           N  \nATOM    928  CA  ASP D  88      18.927  37.589   0.165  1.00 84.79           C  \nATOM    929  C   ASP D  88      19.067  38.802  -0.755  1.00 83.97           C  \nATOM    930  O   ASP D  88      19.457  38.695  -1.925  1.00 89.61           O  \nATOM    931  CB  ASP D  88      18.498  36.217  -0.368  1.00 99.05           C  \nATOM    932  CG  ASP D  88      19.565  35.138  -0.093  1.00 72.31           C  \nATOM    933  OD1 ASP D  88      20.620  35.438   0.507  1.00 10.32           O  \nATOM    934  OD2 ASP D  88      19.367  33.934  -0.648  1.00 45.42           O  \nATOM    935  N   GLU D  89      18.724  39.959  -0.193  1.00 38.56           N  \nATOM    936  CA  GLU D  89      18.813  41.239  -0.861  1.00 12.33           C  \nATOM    937  C   GLU D  89      19.918  42.051  -0.194  1.00 39.86           C  \nATOM    938  O   GLU D  89      20.476  42.973  -0.788  1.00 55.69           O  \nATOM    939  CB  GLU D  89      17.479  41.929  -0.900  1.00 51.30           C  \nATOM    940  CG  GLU D  89      16.411  41.106  -1.584  1.00 41.57           C  \nATOM    941  CD  GLU D  89      16.759  40.708  -3.026  1.00 92.93           C  \nATOM    942  OE1 GLU D  89      16.022  41.102  -3.965  1.00100.00           O  \nATOM    943  OE2 GLU D  89      17.707  39.908  -3.196  1.00 84.46           O  \nATOM    944  N   ARG D  90      20.232  41.708   1.056  1.00 30.25           N  \nATOM    945  CA  ARG D  90      21.302  42.413   1.753  1.00 36.34           C  \nATOM    946  C   ARG D  90      22.623  41.738   1.479  1.00 63.03           C  \nATOM    947  O   ARG D  90      23.482  42.297   0.797  1.00 65.53           O  \nATOM    948  CB  ARG D  90      21.135  42.785   3.194  1.00 35.54           C  \nATOM    949  CG  ARG D  90      21.098  41.700   4.151  1.00 67.09           C  \nATOM    950  CD  ARG D  90      19.705  41.723   4.502  1.00100.00           C  \nATOM    951  NE  ARG D  90      19.449  41.881   5.910  1.00 78.81           N  \nATOM    952  CZ  ARG D  90      18.319  41.498   6.449  1.00 94.72           C  \nATOM    953  NH1 ARG D  90      17.429  40.911   5.657  1.00100.00           N  \nATOM    954  NH2 ARG D  90      18.098  41.656   7.750  1.00 41.62           N  \nATOM    955  N   THR D  91      22.773  40.509   1.963  1.00 70.24           N  \nATOM    956  CA  THR D  91      24.002  39.776   1.705  1.00 70.69           C  \nATOM    957  C   THR D  91      24.231  39.675   0.197  1.00 63.89           C  \nATOM    958  O   THR D  91      25.338  39.353  -0.252  1.00 57.85           O  \nATOM    959  CB  THR D  91      24.196  38.416   2.496  1.00 88.98           C  \nATOM    960  OG1 THR D  91      24.171  38.661   3.917  1.00 58.89           O  \nATOM    961  CG2 THR D  91      25.527  37.767   2.116  1.00 87.03           C  \nATOM    962  N   VAL D  92      23.184  39.978  -0.590  1.00 57.06           N  \nATOM    963  CA  VAL D  92      23.245  39.967  -2.045  1.00 64.37           C  \nATOM    964  C   VAL D  92      24.352  40.884  -2.595  1.00 72.24           C  \nATOM    965  O   VAL D  92      24.454  41.077  -3.804  1.00 81.89           O  \nATOM    966  CB  VAL D  92      21.895  39.735  -2.826  1.00 70.94           C  \nATOM    967  CG1 VAL D  92      21.685  40.736  -3.936  1.00 72.25           C  \nATOM    968  CG2 VAL D  92      21.808  38.298  -3.349  1.00 70.49           C  \nATOM    969  N   LEU D  93      25.174  41.458  -1.709  1.00 59.00           N  \nATOM    970  CA  LEU D  93      26.294  42.282  -2.133  1.00 54.78           C  \nATOM    971  C   LEU D  93      27.370  41.287  -2.556  1.00 52.68           C  \nATOM    972  O   LEU D  93      28.061  41.465  -3.560  1.00 41.47           O  \nATOM    973  CB  LEU D  93      26.814  43.254  -1.014  1.00 51.89           C  \nATOM    974  CG  LEU D  93      27.240  44.742  -1.334  1.00 41.99           C  \nATOM    975  CD1 LEU D  93      27.257  45.632  -0.074  1.00 37.63           C  \nATOM    976  CD2 LEU D  93      28.576  44.878  -2.029  1.00 33.50           C  \nATOM    977  N   ILE D  94      27.429  40.206  -1.778  1.00 48.93           N  \nATOM    978  CA  ILE D  94      28.343  39.094  -1.978  1.00 50.61           C  \nATOM    979  C   ILE D  94      28.057  38.391  -3.310  1.00 71.23           C  \nATOM    980  O   ILE D  94      28.983  38.107  -4.070  1.00 75.81           O  \nATOM    981  CB  ILE D  94      28.136  38.138  -0.764  1.00 42.93           C  \nATOM    982  CG1 ILE D  94      29.347  37.270  -0.490  1.00 29.27           C  \nATOM    983  CG2 ILE D  94      26.974  37.201  -0.961  1.00 56.80           C  \nATOM    984  CD1 ILE D  94      29.200  36.567   0.853  1.00 41.00           C  \nATOM    985  N   LEU D  95      26.767  38.133  -3.600  1.00 66.41           N  \nATOM    986  CA  LEU D  95      26.358  37.468  -4.842  1.00 54.61           C  \nATOM    987  C   LEU D  95      26.973  38.217  -6.006  1.00 54.05           C  \nATOM    988  O   LEU D  95      27.675  37.655  -6.854  1.00 24.13           O  \nATOM    989  CB  LEU D  95      24.814  37.493  -5.040  1.00 41.98           C  \nATOM    990  CG  LEU D  95      24.255  38.235  -6.285  1.00 15.77           C  \nATOM    991  CD1 LEU D  95      24.176  37.367  -7.493  1.00 11.00           C  \nATOM    992  CD2 LEU D  95      22.957  38.963  -6.015  1.00 12.36           C  \nATOM    993  N   VAL D  96      26.720  39.512  -6.019  1.00 68.59           N  \nATOM    994  CA  VAL D  96      27.281  40.344  -7.042  1.00 69.73           C  \nATOM    995  C   VAL D  96      28.758  40.497  -6.740  1.00 45.97           C  \nATOM    996  O   VAL D  96      29.565  40.484  -7.658  1.00 54.37           O  \nATOM    997  CB  VAL D  96      26.567  41.689  -7.232  1.00 79.13           C  \nATOM    998  CG1 VAL D  96      27.340  42.531  -8.254  1.00 78.42           C  \nATOM    999  CG2 VAL D  96      25.114  41.462  -7.669  1.00 78.44           C  \nATOM   1000  N   ASN D  97      29.113  40.602  -5.452  1.00 18.56           N  \nATOM   1001  CA  ASN D  97      30.521  40.756  -5.072  1.00 29.53           C  \nATOM   1002  C   ASN D  97      31.390  39.629  -5.620  1.00 61.78           C  \nATOM   1003  O   ASN D  97      32.497  39.864  -6.114  1.00 63.43           O  \nATOM   1004  CB  ASN D  97      30.842  41.261  -3.660  1.00 47.00           C  \nATOM   1005  CG  ASN D  97      30.925  42.818  -3.580  1.00100.00           C  \nATOM   1006  OD1 ASN D  97      30.088  43.553  -4.156  1.00 33.33           O  \nATOM   1007  ND2 ASN D  97      32.005  43.314  -2.966  1.00100.00           N  \nATOM   1008  N   ALA D  98      30.873  38.404  -5.544  1.00 72.39           N  \nATOM   1009  CA  ALA D  98      31.582  37.240  -6.063  1.00 67.39           C  \nATOM   1010  C   ALA D  98      31.684  37.393  -7.577  1.00 86.88           C  \nATOM   1011  O   ALA D  98      32.773  37.551  -8.135  1.00 95.57           O  \nATOM   1012  CB  ALA D  98      30.797  35.977  -5.715  1.00 63.82           C  \nATOM   1013  N   GLN D  99      30.512  37.391  -8.210  1.00 76.08           N  \nATOM   1014  CA  GLN D  99      30.335  37.536  -9.647  1.00 63.45           C  \nATOM   1015  C   GLN D  99      30.966  38.828 -10.155  1.00 46.21           C  \nATOM   1016  O   GLN D  99      31.334  38.929 -11.322  1.00 48.78           O  \nATOM   1017  CB  GLN D  99      28.859  37.841  -9.698  1.00 60.88           C  \nATOM   1018  CG  GLN D  99      28.019  37.431 -10.811  1.00 45.79           C  \nATOM   1019  CD  GLN D  99      26.622  37.455 -10.278  1.00 40.33           C  \nATOM   1020  OE1 GLN D  99      25.642  37.253 -10.991  1.00 19.73           O  \nATOM   1021  NE2 GLN D  99      26.531  37.747  -8.962  1.00 21.01           N  \nATOM   1022  N   GLN D 100      31.069  39.811  -9.264  1.00 29.75           N  \nATOM   1023  CA  GLN D 100      31.625  41.125  -9.585  1.00 35.80           C  \nATOM   1024  C   GLN D 100      32.981  41.064 -10.267  1.00 27.57           C  \nATOM   1025  O   GLN D 100      33.161  40.390 -11.285  1.00 30.37           O  \nATOM   1026  CB  GLN D 100      31.708  42.109  -8.390  1.00 41.42           C  \nATOM   1027  CG  GLN D 100      30.460  43.055  -8.298  1.00100.00           C  \nATOM   1028  CD  GLN D 100      30.271  43.770  -6.938  1.00 69.00           C  \nATOM   1029  OE1 GLN D 100      31.206  43.920  -6.135  1.00 21.79           O  \nATOM   1030  NE2 GLN D 100      29.012  44.129  -6.635  1.00 29.22           N  \nATOM   1031  N   ARG D 101      33.926  41.804  -9.688  1.00 10.53           N  \nATOM   1032  CA  ARG D 101      35.288  41.894 -10.205  1.00 22.91           C  \nATOM   1033  C   ARG D 101      35.869  40.541 -10.596  1.00 49.30           C  \nATOM   1034  O   ARG D 101      36.856  40.492 -11.324  1.00 45.72           O  \nATOM   1035  CB  ARG D 101      36.250  42.689  -9.320  1.00 35.86           C  \nATOM   1036  CG  ARG D 101      36.072  44.206  -9.404  1.00 49.59           C  \nATOM   1037  CD  ARG D 101      37.434  44.829  -9.351  1.00 31.30           C  \nATOM   1038  NE  ARG D 101      38.403  43.803  -9.056  1.00 39.84           N  \nATOM   1039  CZ  ARG D 101      39.113  43.741  -7.944  1.00 89.29           C  \nATOM   1040  NH1 ARG D 101      38.986  44.687  -7.035  1.00 78.55           N  \nATOM   1041  NH2 ARG D 101      39.972  42.748  -7.757  1.00100.00           N  \nATOM   1042  N   LYS D 102      35.268  39.442 -10.103  1.00 36.45           N  \nATOM   1043  CA  LYS D 102      35.774  38.120 -10.453  1.00 20.23           C  \nATOM   1044  C   LYS D 102      35.515  37.840 -11.917  1.00 31.82           C  \nATOM   1045  O   LYS D 102      36.442  37.562 -12.677  1.00 37.02           O  \nATOM   1046  CB  LYS D 102      35.759  36.986  -9.458  1.00 31.52           C  \nATOM   1047  CG  LYS D 102      34.608  36.017  -9.700  1.00 36.00           C  \nATOM   1048  CD  LYS D 102      34.804  34.736  -8.886  1.00 37.05           C  \nATOM   1049  CE  LYS D 102      35.190  35.083  -7.439  1.00 36.37           C  \nATOM   1050  NZ  LYS D 102      34.142  35.773  -6.604  1.00 25.94           N  \nATOM   1051  N   LYS D 103      34.261  37.933 -12.333  1.00 31.96           N  \nATOM   1052  CA  LYS D 103      33.983  37.737 -13.742  1.00 30.62           C  \nATOM   1053  C   LYS D 103      34.636  38.935 -14.408  1.00 44.34           C  \nATOM   1054  O   LYS D 103      35.197  38.837 -15.495  1.00 53.70           O  \nATOM   1055  CB  LYS D 103      32.494  37.647 -14.014  1.00 28.04           C  \nATOM   1056  CG  LYS D 103      32.109  36.711 -15.142  1.00 19.30           C  \nATOM   1057  CD  LYS D 103      30.631  36.384 -14.979  1.00 22.11           C  \nATOM   1058  CE  LYS D 103      29.879  36.404 -16.307  1.00 61.54           C  \nATOM   1059  NZ  LYS D 103      28.415  36.724 -16.164  1.00 15.12           N  \nATOM   1060  N   ILE D 104      34.581  40.070 -13.703  1.00 32.38           N  \nATOM   1061  CA  ILE D 104      35.212  41.307 -14.167  1.00 24.87           C  \nATOM   1062  C   ILE D 104      36.696  41.045 -14.283  1.00 16.87           C  \nATOM   1063  O   ILE D 104      37.431  41.707 -15.019  1.00 20.02           O  \nATOM   1064  CB  ILE D 104      34.977  42.457 -13.238  1.00 26.38           C  \nATOM   1065  CG1 ILE D 104      33.916  43.320 -13.870  1.00 37.57           C  \nATOM   1066  CG2 ILE D 104      36.260  43.253 -13.005  1.00 37.03           C  \nATOM   1067  CD1 ILE D 104      33.420  42.677 -15.126  1.00 21.20           C  \nATOM   1068  N   GLU D 105      37.122  40.031 -13.550  1.00 20.01           N  \nATOM   1069  CA  GLU D 105      38.501  39.626 -13.607  1.00 27.33           C  \nATOM   1070  C   GLU D 105      38.664  38.914 -14.931  1.00 30.78           C  \nATOM   1071  O   GLU D 105      39.682  39.052 -15.609  1.00 42.03           O  \nATOM   1072  CB  GLU D 105      38.953  38.755 -12.422  1.00 26.31           C  \nATOM   1073  CG  GLU D 105      40.389  38.246 -12.478  1.00 52.50           C  \nATOM   1074  CD  GLU D 105      41.412  39.331 -12.852  1.00 75.76           C  \nATOM   1075  OE1 GLU D 105      41.228  39.962 -13.923  1.00 41.20           O  \nATOM   1076  OE2 GLU D 105      42.479  39.406 -12.171  1.00 32.66           O  \nATOM   1077  N   SER D 106      37.631  38.193 -15.305  1.00 12.30           N  \nATOM   1078  CA  SER D 106      37.615  37.476 -16.562  1.00 18.13           C  \nATOM   1079  C   SER D 106      37.508  38.457 -17.734  1.00 33.93           C  \nATOM   1080  O   SER D 106      38.264  38.326 -18.696  1.00 41.80           O  \nATOM   1081  CB  SER D 106      36.497  36.536 -16.399  1.00 12.11           C  \nATOM   1082  OG  SER D 106      36.215  36.634 -15.001  1.00 28.62           O  \nATOM   1083  N   LEU D 107      36.601  39.455 -17.658  1.00 34.68           N  \nATOM   1084  CA  LEU D 107      36.475  40.443 -18.743  1.00 27.87           C  \nATOM   1085  C   LEU D 107      37.826  41.075 -18.983  1.00 22.71           C  \nATOM   1086  O   LEU D 107      38.081  41.606 -20.070  1.00 17.30           O  \nATOM   1087  CB  LEU D 107      35.535  41.639 -18.516  1.00 30.23           C  \nATOM   1088  CG  LEU D 107      35.737  42.983 -19.367  1.00 35.79           C  \nATOM   1089  CD1 LEU D 107      36.917  43.924 -19.005  1.00 27.69           C  \nATOM   1090  CD2 LEU D 107      35.459  42.929 -20.876  1.00 14.14           C  \nATOM   1091  N   LEU D 108      38.673  41.021 -17.947  1.00 17.54           N  \nATOM   1092  CA  LEU D 108      40.024  41.554 -18.043  1.00 16.68           C  \nATOM   1093  C   LEU D 108      40.659  40.804 -19.188  1.00 25.85           C  \nATOM   1094  O   LEU D 108      41.491  41.328 -19.933  1.00 28.23           O  \nATOM   1095  CB  LEU D 108      40.831  41.293 -16.757  1.00 16.11           C  \nATOM   1096  CG  LEU D 108      42.354  41.376 -16.848  1.00 16.91           C  \nATOM   1097  CD1 LEU D 108      42.678  42.590 -17.740  1.00 21.76           C  \nATOM   1098  CD2 LEU D 108      43.018  41.498 -15.452  1.00 31.00           C  \nATOM   1099  N   SER D 109      40.214  39.553 -19.311  1.00 18.07           N  \nATOM   1100  CA  SER D 109      40.660  38.668 -20.370  1.00 30.28           C  \nATOM   1101  C   SER D 109      40.018  39.110 -21.679  1.00 48.29           C  \nATOM   1102  O   SER D 109      40.624  38.963 -22.744  1.00 39.94           O  \nATOM   1103  CB  SER D 109      40.390  37.184 -20.076  1.00 62.18           C  \nATOM   1104  OG  SER D 109      40.364  36.952 -18.660  1.00 96.10           O  \nATOM   1105  N   ARG D 110      38.801  39.679 -21.596  1.00 59.58           N  \nATOM   1106  CA  ARG D 110      38.121  40.167 -22.798  1.00 57.10           C  \nATOM   1107  C   ARG D 110      39.045  41.161 -23.468  1.00 55.78           C  \nATOM   1108  O   ARG D 110      39.377  41.014 -24.645  1.00 60.43           O  \nATOM   1109  CB  ARG D 110      36.782  40.859 -22.602  1.00 55.45           C  \nATOM   1110  CG  ARG D 110      36.226  41.414 -23.999  1.00 27.07           C  \nATOM   1111  CD  ARG D 110      34.769  41.987 -23.986  1.00 14.96           C  \nATOM   1112  NE  ARG D 110      34.049  41.795 -25.255  1.00 17.13           N  \nATOM   1113  CZ  ARG D 110      32.974  41.009 -25.406  1.00 57.64           C  \nATOM   1114  NH1 ARG D 110      32.467  40.315 -24.358  1.00 51.20           N  \nATOM   1115  NH2 ARG D 110      32.408  40.911 -26.614  1.00 26.75           N  \nATOM   1116  N   VAL D 111      39.485  42.152 -22.692  1.00 36.68           N  \nATOM   1117  CA  VAL D 111      40.430  43.131 -23.204  1.00 39.12           C  \nATOM   1118  C   VAL D 111      41.772  42.425 -23.357  1.00 37.23           C  \nATOM   1119  O   VAL D 111      42.304  42.271 -24.461  1.00 32.61           O  \nATOM   1120  CB  VAL D 111      40.533  44.414 -22.338  1.00 60.34           C  \nATOM   1121  CG1 VAL D 111      40.093  45.636 -23.141  1.00 59.81           C  \nATOM   1122  CG2 VAL D 111      39.716  44.278 -21.039  1.00 64.68           C  \nTER    1123      VAL D 111                                                      \nATOM   1124  N   ALA B   2       4.032  11.547  16.164  1.00 11.69           N  \nATOM   1125  CA  ALA B   2       5.470  11.718  15.971  1.00 13.61           C  \nATOM   1126  C   ALA B   2       5.795  12.625  14.779  1.00 25.77           C  \nATOM   1127  O   ALA B   2       4.952  12.842  13.899  1.00 22.09           O  \nATOM   1128  CB  ALA B   2       6.183  10.375  15.881  1.00 14.30           C  \nATOM   1129  N   ILE B   3       7.040  13.132  14.763  1.00 28.37           N  \nATOM   1130  CA  ILE B   3       7.558  14.042  13.727  1.00 24.48           C  \nATOM   1131  C   ILE B   3       7.102  13.735  12.305  1.00 46.01           C  \nATOM   1132  O   ILE B   3       6.376  12.775  12.039  1.00 54.62           O  \nATOM   1133  CB  ILE B   3       9.163  14.197  13.638  1.00 29.73           C  \nATOM   1134  CG1 ILE B   3       9.819  13.307  12.596  1.00 51.40           C  \nATOM   1135  CG2 ILE B   3       9.859  13.952  14.947  1.00 52.01           C  \nATOM   1136  CD1 ILE B   3      10.295  14.035  11.351  1.00 58.42           C  \nATOM   1137  N   THR B   4       7.553  14.587  11.401  1.00 48.27           N  \nATOM   1138  CA  THR B   4       7.267  14.490   9.980  1.00 58.04           C  \nATOM   1139  C   THR B   4       8.337  15.301   9.265  1.00 85.48           C  \nATOM   1140  O   THR B   4       9.185  14.758   8.556  1.00 91.74           O  \nATOM   1141  CB  THR B   4       5.894  15.168   9.594  1.00 53.92           C  \nATOM   1142  OG1 THR B   4       4.986  15.146  10.717  1.00 69.37           O  \nATOM   1143  CG2 THR B   4       5.279  14.524   8.317  1.00 61.23           C  \nATOM   1144  N   LYS B   5       8.298  16.613   9.497  1.00 81.67           N  \nATOM   1145  CA  LYS B   5       9.259  17.542   8.915  1.00 71.51           C  \nATOM   1146  C   LYS B   5      10.105  18.241   9.973  1.00 67.04           C  \nATOM   1147  O   LYS B   5      10.866  19.158   9.657  1.00 77.23           O  \nATOM   1148  CB  LYS B   5       8.651  18.485   7.899  1.00 56.51           C  \nATOM   1149  CG  LYS B   5       8.664  17.833   6.547  1.00 14.06           C  \nATOM   1150  CD  LYS B   5      10.090  17.460   6.132  1.00 41.74           C  \nATOM   1151  CE  LYS B   5      10.155  16.833   4.721  1.00 65.16           C  \nATOM   1152  NZ  LYS B   5       9.325  17.455   3.619  1.00 32.65           N  \nATOM   1153  N   ILE B   6       9.970  17.830  11.230  1.00 37.91           N  \nATOM   1154  CA  ILE B   6      10.777  18.466  12.252  1.00 30.14           C  \nATOM   1155  C   ILE B   6      12.231  18.191  11.938  1.00 33.55           C  \nATOM   1156  O   ILE B   6      12.843  17.297  12.524  1.00 41.23           O  \nATOM   1157  CB  ILE B   6      10.342  18.241  13.720  1.00 32.84           C  \nATOM   1158  CG1 ILE B   6      10.692  16.868  14.236  1.00 29.83           C  \nATOM   1159  CG2 ILE B   6       8.921  18.733  13.934  1.00 38.79           C  \nATOM   1160  CD1 ILE B   6      11.898  16.929  15.116  1.00 31.02           C  \nATOM   1161  N   ASN B   7      12.747  18.949  10.989  1.00 42.76           N  \nATOM   1162  CA  ASN B   7      14.118  18.817  10.517  1.00 49.37           C  \nATOM   1163  C   ASN B   7      14.494  20.070   9.725  1.00 54.77           C  \nATOM   1164  O   ASN B   7      14.902  20.004   8.561  1.00 40.95           O  \nATOM   1165  CB  ASN B   7      14.276  17.474   9.711  1.00 48.22           C  \nATOM   1166  CG  ASN B   7      15.262  16.411  10.369  1.00 15.82           C  \nATOM   1167  OD1 ASN B   7      15.235  16.087  11.565  1.00 21.10           O  \nATOM   1168  ND2 ASN B   7      16.108  15.842   9.507  1.00 12.79           N  \nATOM   1169  N   ASP B   8      14.315  21.219  10.387  1.00 45.53           N  \nATOM   1170  CA  ASP B   8      14.619  22.531   9.836  1.00 31.96           C  \nATOM   1171  C   ASP B   8      16.068  22.566   9.370  1.00 43.12           C  \nATOM   1172  O   ASP B   8      16.960  22.176  10.117  1.00 44.60           O  \nATOM   1173  CB  ASP B   8      14.608  23.382  11.092  1.00 30.29           C  \nATOM   1174  CG  ASP B   8      15.269  22.648  12.317  1.00 37.98           C  \nATOM   1175  OD1 ASP B   8      14.815  21.545  12.793  1.00 21.00           O  \nATOM   1176  OD2 ASP B   8      16.206  23.241  12.898  1.00100.00           O  \nATOM   1177  N   CYS B   9      16.330  23.018   8.150  1.00 48.90           N  \nATOM   1178  CA  CYS B   9      15.328  23.490   7.201  1.00 55.42           C  \nATOM   1179  C   CYS B   9      15.646  22.866   5.832  1.00 81.57           C  \nATOM   1180  O   CYS B   9      16.472  21.955   5.770  1.00 94.20           O  \nATOM   1181  CB  CYS B   9      15.434  24.989   7.086  1.00 51.13           C  \nATOM   1182  SG  CYS B   9      16.802  25.280   6.016  1.00 54.79           S  \nATOM   1183  N   PHE B  10      15.004  23.317   4.756  1.00 69.37           N  \nATOM   1184  CA  PHE B  10      15.257  22.707   3.449  1.00 64.62           C  \nATOM   1185  C   PHE B  10      16.541  23.211   2.764  1.00 65.07           C  \nATOM   1186  O   PHE B  10      17.614  22.627   2.937  1.00 74.62           O  \nATOM   1187  CB  PHE B  10      14.003  22.606   2.583  1.00 66.61           C  \nATOM   1188  CG  PHE B  10      12.708  22.735   3.362  1.00 70.07           C  \nATOM   1189  CD1 PHE B  10      12.505  22.043   4.555  1.00 79.03           C  \nATOM   1190  CD2 PHE B  10      11.688  23.556   2.900  1.00 73.09           C  \nATOM   1191  CE1 PHE B  10      11.337  22.183   5.274  1.00 79.46           C  \nATOM   1192  CE2 PHE B  10      10.497  23.683   3.622  1.00 80.18           C  \nATOM   1193  CZ  PHE B  10      10.328  22.995   4.806  1.00 79.90           C  \nATOM   1194  N   GLU B  11      16.420  24.285   1.981  1.00 56.82           N  \nATOM   1195  CA  GLU B  11      17.534  24.881   1.233  1.00 50.86           C  \nATOM   1196  C   GLU B  11      18.839  24.966   2.045  1.00 39.22           C  \nATOM   1197  O   GLU B  11      19.877  24.413   1.660  1.00 12.48           O  \nATOM   1198  CB  GLU B  11      17.124  26.275   0.672  1.00 48.83           C  \nATOM   1199  CG  GLU B  11      15.713  26.713   1.121  1.00 91.67           C  \nATOM   1200  CD  GLU B  11      15.516  28.190   1.495  1.00 84.15           C  \nATOM   1201  OE1 GLU B  11      15.535  28.493   2.716  1.00 74.98           O  \nATOM   1202  OE2 GLU B  11      15.141  29.021   0.622  1.00 28.32           O  \nATOM   1203  N   LEU B  12      18.805  25.651   3.176  1.00 41.37           N  \nATOM   1204  CA  LEU B  12      19.992  25.775   4.012  1.00 28.50           C  \nATOM   1205  C   LEU B  12      20.506  24.393   4.421  1.00 30.46           C  \nATOM   1206  O   LEU B  12      21.708  24.135   4.430  1.00 17.10           O  \nATOM   1207  CB  LEU B  12      19.645  26.641   5.238  1.00 12.06           C  \nATOM   1208  CG  LEU B  12      20.834  27.275   5.939  1.00 14.16           C  \nATOM   1209  CD1 LEU B  12      20.718  27.185   7.448  1.00 11.20           C  \nATOM   1210  CD2 LEU B  12      22.150  26.695   5.476  1.00 17.01           C  \nATOM   1211  N   LEU B  13      19.583  23.490   4.720  1.00 36.24           N  \nATOM   1212  CA  LEU B  13      19.940  22.139   5.130  1.00 45.51           C  \nATOM   1213  C   LEU B  13      20.946  21.397   4.247  1.00 65.55           C  \nATOM   1214  O   LEU B  13      22.150  21.470   4.496  1.00 74.48           O  \nATOM   1215  CB  LEU B  13      19.201  21.437   6.256  1.00 50.32           C  \nATOM   1216  CG  LEU B  13      19.909  21.567   7.607  1.00 68.18           C  \nATOM   1217  CD1 LEU B  13      19.475  20.460   8.580  1.00 72.91           C  \nATOM   1218  CD2 LEU B  13      21.432  21.604   7.403  1.00 76.13           C  \nATOM   1219  N   SER B  14      20.456  20.671   3.228  1.00 48.37           N  \nATOM   1220  CA  SER B  14      21.339  19.900   2.347  1.00 49.15           C  \nATOM   1221  C   SER B  14      22.545  20.679   1.846  1.00 55.71           C  \nATOM   1222  O   SER B  14      23.594  20.082   1.609  1.00 62.47           O  \nATOM   1223  CB  SER B  14      20.734  19.188   1.145  1.00 57.46           C  \nATOM   1224  OG  SER B  14      21.738  19.036   0.101  1.00 29.61           O  \nATOM   1225  N   MET B  15      22.420  21.988   1.667  1.00 46.47           N  \nATOM   1226  CA  MET B  15      23.549  22.781   1.187  1.00 32.59           C  \nATOM   1227  C   MET B  15      24.758  22.575   2.095  1.00 22.20           C  \nATOM   1228  O   MET B  15      25.805  22.080   1.668  1.00 19.28           O  \nATOM   1229  CB  MET B  15      23.173  24.264   1.138  1.00 24.04           C  \nATOM   1230  CG  MET B  15      24.166  25.018   0.310  1.00 19.94           C  \nATOM   1231  SD  MET B  15      24.253  24.075  -1.215  1.00 28.13           S  \nATOM   1232  CE  MET B  15      22.518  23.709  -1.476  1.00 29.92           C  \nATOM   1233  N   VAL B  16      24.587  22.944   3.356  1.00 21.12           N  \nATOM   1234  CA  VAL B  16      25.636  22.797   4.333  1.00 22.20           C  \nATOM   1235  C   VAL B  16      26.166  21.368   4.342  1.00 44.21           C  \nATOM   1236  O   VAL B  16      27.376  21.166   4.259  1.00 64.79           O  \nATOM   1237  CB  VAL B  16      25.038  23.194   5.679  1.00 17.25           C  \nATOM   1238  CG1 VAL B  16      26.123  23.279   6.770  1.00 11.40           C  \nATOM   1239  CG2 VAL B  16      24.264  24.514   5.488  1.00 31.40           C  \nATOM   1240  N   THR B  17      25.253  20.381   4.418  1.00 36.12           N  \nATOM   1241  CA  THR B  17      25.621  18.953   4.430  1.00 21.67           C  \nATOM   1242  C   THR B  17      26.328  18.550   3.136  1.00 17.56           C  \nATOM   1243  O   THR B  17      27.423  17.990   3.145  1.00 20.73           O  \nATOM   1244  CB  THR B  17      24.460  18.023   4.401  1.00 31.20           C  \nATOM   1245  OG1 THR B  17      23.982  17.752   5.734  1.00  3.86           O  \nATOM   1246  CG2 THR B  17      24.982  16.744   3.743  1.00 19.53           C  \nATOM   1247  N   TYR B  18      25.688  18.833   2.016  1.00 10.68           N  \nATOM   1248  CA  TYR B  18      26.275  18.537   0.719  1.00 15.49           C  \nATOM   1249  C   TYR B  18      27.623  19.259   0.627  1.00 36.96           C  \nATOM   1250  O   TYR B  18      28.546  18.803  -0.041  1.00 32.74           O  \nATOM   1251  CB  TYR B  18      25.266  19.014  -0.356  1.00 18.64           C  \nATOM   1252  CG  TYR B  18      25.608  19.142  -1.823  1.00 15.22           C  \nATOM   1253  CD1 TYR B  18      24.588  19.259  -2.757  1.00 10.55           C  \nATOM   1254  CD2 TYR B  18      26.904  19.140  -2.280  1.00 22.62           C  \nATOM   1255  CE1 TYR B  18      24.872  19.415  -4.100  1.00 12.89           C  \nATOM   1256  CE2 TYR B  18      27.200  19.273  -3.625  1.00 28.81           C  \nATOM   1257  CZ  TYR B  18      26.188  19.376  -4.539  1.00 20.60           C  \nATOM   1258  OH  TYR B  18      26.486  19.568  -5.869  1.00 16.39           O  \nATOM   1259  N   ALA B  19      27.749  20.391   1.315  1.00 53.45           N  \nATOM   1260  CA  ALA B  19      29.009  21.138   1.303  1.00 47.87           C  \nATOM   1261  C   ALA B  19      29.986  20.521   2.298  1.00 52.23           C  \nATOM   1262  O   ALA B  19      31.133  20.215   1.960  1.00 58.56           O  \nATOM   1263  CB  ALA B  19      28.785  22.611   1.584  1.00 40.94           C  \nATOM   1264  N   ASP B  20      29.508  20.313   3.525  1.00 26.51           N  \nATOM   1265  CA  ASP B  20      30.309  19.723   4.586  1.00 16.55           C  \nATOM   1266  C   ASP B  20      30.829  18.355   4.162  1.00 35.79           C  \nATOM   1267  O   ASP B  20      31.668  17.770   4.847  1.00 32.95           O  \nATOM   1268  CB  ASP B  20      29.483  19.586   5.869  1.00  5.91           C  \nATOM   1269  CG  ASP B  20      29.309  20.917   6.620  1.00 46.79           C  \nATOM   1270  OD1 ASP B  20      30.047  21.202   7.604  1.00 33.30           O  \nATOM   1271  OD2 ASP B  20      28.411  21.701   6.238  1.00 85.92           O  \nATOM   1272  N   LYS B  21      30.312  17.854   3.036  1.00 46.46           N  \nATOM   1273  CA  LYS B  21      30.684  16.553   2.467  1.00 51.20           C  \nATOM   1274  C   LYS B  21      31.951  16.607   1.589  1.00 63.63           C  \nATOM   1275  O   LYS B  21      32.948  15.934   1.887  1.00 51.28           O  \nATOM   1276  CB  LYS B  21      29.479  15.920   1.732  1.00 34.95           C  \nATOM   1277  CG  LYS B  21      29.772  14.573   1.116  1.00  4.83           C  \nATOM   1278  CD  LYS B  21      30.852  13.840   1.932  1.00 59.30           C  \nATOM   1279  CE  LYS B  21      31.408  12.593   1.225  1.00 92.23           C  \nATOM   1280  NZ  LYS B  21      30.641  11.308   1.405  1.00 58.81           N  \nATOM   1281  N   LEU B  22      31.910  17.380   0.493  1.00 48.28           N  \nATOM   1282  CA  LEU B  22      33.068  17.499  -0.401  1.00 28.89           C  \nATOM   1283  C   LEU B  22      34.216  18.327   0.198  1.00 29.06           C  \nATOM   1284  O   LEU B  22      35.347  18.322  -0.301  1.00 12.54           O  \nATOM   1285  CB  LEU B  22      32.921  17.196  -1.879  1.00 19.50           C  \nATOM   1286  CG  LEU B  22      31.619  17.701  -2.502  1.00 20.81           C  \nATOM   1287  CD1 LEU B  22      31.839  18.906  -3.447  1.00 18.94           C  \nATOM   1288  CD2 LEU B  22      30.684  16.590  -3.050  1.00 11.34           C  \nATOM   1289  N   LYS B  23      33.907  19.032   1.281  1.00 10.78           N  \nATOM   1290  CA  LYS B  23      34.896  19.823   1.981  1.00 21.82           C  \nATOM   1291  C   LYS B  23      35.920  18.868   2.594  1.00 55.40           C  \nATOM   1292  O   LYS B  23      37.137  19.010   2.426  1.00 56.68           O  \nATOM   1293  CB  LYS B  23      34.156  20.413   3.151  1.00 51.20           C  \nATOM   1294  CG  LYS B  23      33.843  21.861   3.005  1.00 41.10           C  \nATOM   1295  CD  LYS B  23      33.300  22.326   4.330  1.00 16.05           C  \nATOM   1296  CE  LYS B  23      34.321  22.055   5.408  1.00 17.83           C  \nATOM   1297  NZ  LYS B  23      35.658  22.502   4.920  1.00 19.08           N  \nATOM   1298  N   SER B  24      35.385  17.872   3.290  1.00 41.56           N  \nATOM   1299  CA  SER B  24      36.160  16.841   3.953  1.00 29.01           C  \nATOM   1300  C   SER B  24      36.718  15.838   2.956  1.00 20.03           C  \nATOM   1301  O   SER B  24      37.859  15.402   3.090  1.00 18.91           O  \nATOM   1302  CB  SER B  24      35.163  16.072   4.779  1.00 30.80           C  \nATOM   1303  OG  SER B  24      34.096  15.676   3.923  1.00 45.21           O  \nATOM   1304  N   LEU B  25      35.903  15.457   1.976  1.00 26.39           N  \nATOM   1305  CA  LEU B  25      36.336  14.484   0.974  1.00 30.43           C  \nATOM   1306  C   LEU B  25      37.662  14.887   0.354  1.00 39.59           C  \nATOM   1307  O   LEU B  25      38.551  14.061   0.175  1.00 45.63           O  \nATOM   1308  CB  LEU B  25      35.404  14.447  -0.210  1.00 31.72           C  \nATOM   1309  CG  LEU B  25      35.697  13.315  -1.188  1.00 29.25           C  \nATOM   1310  CD1 LEU B  25      34.478  12.449  -1.206  1.00 32.85           C  \nATOM   1311  CD2 LEU B  25      35.963  13.829  -2.577  1.00 18.18           C  \nATOM   1312  N   ILE B  26      37.761  16.167   0.008  1.00 30.85           N  \nATOM   1313  CA  ILE B  26      38.954  16.735  -0.587  1.00 19.41           C  \nATOM   1314  C   ILE B  26      40.011  16.915   0.497  1.00 23.63           C  \nATOM   1315  O   ILE B  26      41.208  16.972   0.236  1.00 30.48           O  \nATOM   1316  CB  ILE B  26      38.536  18.111  -1.252  1.00 22.82           C  \nATOM   1317  CG1 ILE B  26      37.973  17.878  -2.679  1.00 31.84           C  \nATOM   1318  CG2 ILE B  26      39.572  19.241  -1.134  1.00 12.24           C  \nATOM   1319  CD1 ILE B  26      36.482  17.574  -2.812  1.00 21.00           C  \nATOM   1320  N   LYS B  27      39.554  17.030   1.737  1.00 23.11           N  \nATOM   1321  CA  LYS B  27      40.450  17.218   2.874  1.00 28.94           C  \nATOM   1322  C   LYS B  27      41.402  16.033   3.075  1.00 27.33           C  \nATOM   1323  O   LYS B  27      42.624  16.193   3.071  1.00 14.93           O  \nATOM   1324  CB  LYS B  27      39.631  17.636   4.118  1.00 35.73           C  \nATOM   1325  CG  LYS B  27      40.410  18.367   5.211  1.00100.00           C  \nATOM   1326  CD  LYS B  27      39.522  19.448   5.849  1.00 86.34           C  \nATOM   1327  CE  LYS B  27      39.732  19.636   7.357  1.00 69.08           C  \nATOM   1328  NZ  LYS B  27      40.623  20.792   7.681  1.00 57.99           N  \nATOM   1329  N   LYS B  28      40.845  14.836   3.232  1.00 35.85           N  \nATOM   1330  CA  LYS B  28      41.643  13.624   3.433  1.00 38.60           C  \nATOM   1331  C   LYS B  28      42.460  13.264   2.200  1.00 33.85           C  \nATOM   1332  O   LYS B  28      43.672  13.063   2.282  1.00 27.90           O  \nATOM   1333  CB  LYS B  28      40.773  12.387   3.665  1.00 39.09           C  \nATOM   1334  CG  LYS B  28      39.387  12.608   4.193  1.00 49.33           C  \nATOM   1335  CD  LYS B  28      39.209  11.567   5.268  1.00 45.58           C  \nATOM   1336  CE  LYS B  28      40.417  11.637   6.199  1.00 31.00           C  \nATOM   1337  NZ  LYS B  28      40.166  11.065   7.569  1.00 27.16           N  \nATOM   1338  N   GLU B  29      41.774  13.182   1.062  1.00 38.79           N  \nATOM   1339  CA  GLU B  29      42.371  12.829  -0.225  1.00 46.16           C  \nATOM   1340  C   GLU B  29      43.520  13.760  -0.647  1.00 29.98           C  \nATOM   1341  O   GLU B  29      44.344  13.382  -1.474  1.00 23.51           O  \nATOM   1342  CB  GLU B  29      41.270  12.614  -1.293  1.00 50.92           C  \nATOM   1343  CG  GLU B  29      41.643  11.850  -2.573  1.00 84.36           C  \nATOM   1344  CD  GLU B  29      41.600  10.321  -2.452  1.00 93.70           C  \nATOM   1345  OE1 GLU B  29      40.493   9.684  -2.514  1.00 49.74           O  \nATOM   1346  OE2 GLU B  29      42.705   9.729  -2.424  1.00 23.88           O  \nATOM   1347  N   PHE B  30      43.579  14.959  -0.077  1.00 21.16           N  \nATOM   1348  CA  PHE B  30      44.652  15.891  -0.420  1.00 26.66           C  \nATOM   1349  C   PHE B  30      44.932  16.962   0.642  1.00 54.33           C  \nATOM   1350  O   PHE B  30      45.125  16.636   1.806  1.00 66.70           O  \nATOM   1351  CB  PHE B  30      44.917  16.238  -1.890  1.00 23.27           C  \nATOM   1352  CG  PHE B  30      43.695  16.176  -2.739  1.00 18.50           C  \nATOM   1353  CD1 PHE B  30      42.462  16.366  -2.187  1.00 24.19           C  \nATOM   1354  CD2 PHE B  30      43.776  15.944  -4.092  1.00 12.06           C  \nATOM   1355  CE1 PHE B  30      41.318  16.307  -2.955  1.00 32.00           C  \nATOM   1356  CE2 PHE B  30      42.623  15.872  -4.883  1.00  3.71           C  \nATOM   1357  CZ  PHE B  30      41.392  16.050  -4.310  1.00 12.34           C  \nATOM   1358  N   SER B  31      44.991  18.220   0.203  1.00 42.94           N  \nATOM   1359  CA  SER B  31      45.282  19.349   1.065  1.00 36.35           C  \nATOM   1360  C   SER B  31      44.438  19.380   2.334  1.00 47.37           C  \nATOM   1361  O   SER B  31      44.162  18.347   2.950  1.00 67.72           O  \nATOM   1362  CB  SER B  31      45.157  20.662   0.303  1.00 25.29           C  \nATOM   1363  OG  SER B  31      43.812  21.100   0.233  1.00 31.26           O  \nATOM   1364  N   ILE B  32      44.083  20.596   2.743  1.00 17.66           N  \nATOM   1365  CA  ILE B  32      43.271  20.836   3.932  1.00 10.87           C  \nATOM   1366  C   ILE B  32      42.898  22.295   4.041  1.00 28.66           C  \nATOM   1367  O   ILE B  32      41.976  22.647   4.747  1.00 28.07           O  \nATOM   1368  CB  ILE B  32      43.906  20.562   5.288  1.00 12.97           C  \nATOM   1369  CG1 ILE B  32      45.371  20.919   5.376  1.00 22.52           C  \nATOM   1370  CG2 ILE B  32      43.512  19.215   5.874  1.00 12.95           C  \nATOM   1371  CD1 ILE B  32      46.039  20.087   6.463  1.00 41.92           C  \nATOM   1372  N   SER B  33      43.617  23.143   3.311  1.00 55.08           N  \nATOM   1373  CA  SER B  33      43.319  24.570   3.339  1.00 62.03           C  \nATOM   1374  C   SER B  33      41.862  24.840   3.006  1.00 65.86           C  \nATOM   1375  O   SER B  33      41.342  25.915   3.280  1.00 50.01           O  \nATOM   1376  CB  SER B  33      44.358  25.441   2.558  1.00 59.19           C  \nATOM   1377  OG  SER B  33      44.425  25.179   1.160  1.00 14.19           O  \nATOM   1378  N   PHE B  34      41.199  23.829   2.450  1.00 70.72           N  \nATOM   1379  CA  PHE B  34      39.799  23.919   2.076  1.00 71.42           C  \nATOM   1380  C   PHE B  34      38.877  24.104   3.275  1.00 29.75           C  \nATOM   1381  O   PHE B  34      37.977  24.928   3.227  1.00 21.00           O  \nATOM   1382  CB  PHE B  34      39.389  22.715   1.284  1.00 86.07           C  \nATOM   1383  CG  PHE B  34      38.472  23.024   0.156  1.00 94.98           C  \nATOM   1384  CD1 PHE B  34      37.213  22.477   0.145  1.00 98.93           C  \nATOM   1385  CD2 PHE B  34      38.876  23.812  -0.900  1.00 98.62           C  \nATOM   1386  CE1 PHE B  34      36.364  22.699  -0.901  1.00 98.72           C  \nATOM   1387  CE2 PHE B  34      38.016  24.053  -1.967  1.00100.00           C  \nATOM   1388  CZ  PHE B  34      36.751  23.500  -1.948  1.00100.00           C  \nATOM   1389  N   GLU B  35      39.076  23.328   4.330  1.00 38.94           N  \nATOM   1390  CA  GLU B  35      38.242  23.475   5.519  1.00 45.78           C  \nATOM   1391  C   GLU B  35      38.540  24.851   6.114  1.00 78.65           C  \nATOM   1392  O   GLU B  35      37.742  25.420   6.868  1.00 92.39           O  \nATOM   1393  CB  GLU B  35      38.640  22.393   6.538  1.00 39.91           C  \nATOM   1394  CG  GLU B  35      37.742  22.190   7.727  1.00  1.00           C  \nATOM   1395  CD  GLU B  35      36.565  21.380   7.302  1.00 63.87           C  \nATOM   1396  OE1 GLU B  35      36.697  20.806   6.192  1.00 30.74           O  \nATOM   1397  OE2 GLU B  35      35.453  21.481   7.895  1.00  1.00           O  \nATOM   1398  N   GLU B  36      39.716  25.371   5.747  1.00 52.15           N  \nATOM   1399  CA  GLU B  36      40.210  26.663   6.202  1.00 23.04           C  \nATOM   1400  C   GLU B  36      39.746  27.788   5.298  1.00 15.64           C  \nATOM   1401  O   GLU B  36      39.060  28.704   5.748  1.00 24.61           O  \nATOM   1402  CB  GLU B  36      41.719  26.583   6.129  1.00 18.39           C  \nATOM   1403  CG  GLU B  36      42.175  25.173   6.468  1.00 38.87           C  \nATOM   1404  CD  GLU B  36      41.511  24.632   7.758  1.00 46.56           C  \nATOM   1405  OE1 GLU B  36      40.882  23.557   7.729  1.00 60.96           O  \nATOM   1406  OE2 GLU B  36      41.563  25.335   8.782  1.00 61.85           O  \nATOM   1407  N   PHE B  37      40.127  27.707   4.027  1.00 13.13           N  \nATOM   1408  CA  PHE B  37      39.758  28.722   3.047  1.00 24.81           C  \nATOM   1409  C   PHE B  37      38.261  29.021   3.092  1.00 52.35           C  \nATOM   1410  O   PHE B  37      37.848  30.177   3.091  1.00 64.45           O  \nATOM   1411  CB  PHE B  37      40.093  28.226   1.627  1.00 21.07           C  \nATOM   1412  CG  PHE B  37      41.290  28.877   1.012  1.00 18.86           C  \nATOM   1413  CD1 PHE B  37      42.404  29.201   1.784  1.00 12.01           C  \nATOM   1414  CD2 PHE B  37      41.303  29.176  -0.350  1.00 17.49           C  \nATOM   1415  CE1 PHE B  37      43.518  29.786   1.203  1.00 11.23           C  \nATOM   1416  CE2 PHE B  37      42.425  29.785  -0.935  1.00 16.40           C  \nATOM   1417  CZ  PHE B  37      43.541  30.088  -0.149  1.00 11.00           C  \nATOM   1418  N   ALA B  38      37.447  27.965   3.114  1.00 36.87           N  \nATOM   1419  CA  ALA B  38      35.997  28.112   3.147  1.00 42.28           C  \nATOM   1420  C   ALA B  38      35.553  29.036   4.273  1.00 68.53           C  \nATOM   1421  O   ALA B  38      34.760  29.955   4.058  1.00 73.66           O  \nATOM   1422  CB  ALA B  38      35.319  26.778   3.255  1.00 45.26           C  \nATOM   1423  N   VAL B  39      36.055  28.789   5.475  1.00 69.27           N  \nATOM   1424  CA  VAL B  39      35.690  29.628   6.609  1.00 66.33           C  \nATOM   1425  C   VAL B  39      36.448  30.949   6.553  1.00 56.37           C  \nATOM   1426  O   VAL B  39      35.940  31.986   6.977  1.00 50.45           O  \nATOM   1427  CB  VAL B  39      35.974  28.954   7.951  1.00 67.19           C  \nATOM   1428  CG1 VAL B  39      37.390  28.385   7.972  1.00 69.55           C  \nATOM   1429  CG2 VAL B  39      35.751  29.965   9.069  1.00 62.43           C  \nATOM   1430  N   LEU B  40      37.663  30.872   6.014  1.00 46.82           N  \nATOM   1431  CA  LEU B  40      38.593  31.995   5.857  1.00 42.03           C  \nATOM   1432  C   LEU B  40      37.902  33.328   5.508  1.00 63.28           C  \nATOM   1433  O   LEU B  40      38.325  34.406   5.954  1.00 51.13           O  \nATOM   1434  CB  LEU B  40      39.613  31.526   4.802  1.00 29.95           C  \nATOM   1435  CG  LEU B  40      40.714  32.295   4.069  1.00 17.32           C  \nATOM   1436  CD1 LEU B  40      40.244  33.159   2.904  1.00 17.11           C  \nATOM   1437  CD2 LEU B  40      41.677  32.979   4.995  1.00 31.00           C  \nATOM   1438  N   THR B  41      36.829  33.260   4.727  1.00 53.82           N  \nATOM   1439  CA  THR B  41      36.105  34.464   4.341  1.00 51.84           C  \nATOM   1440  C   THR B  41      35.415  35.099   5.543  1.00 70.11           C  \nATOM   1441  O   THR B  41      34.597  36.009   5.389  1.00 83.79           O  \nATOM   1442  CB  THR B  41      34.991  34.117   3.382  1.00 54.60           C  \nATOM   1443  OG1 THR B  41      33.720  34.442   3.957  1.00 41.42           O  \nATOM   1444  CG2 THR B  41      35.016  32.628   3.087  1.00 46.59           C  \nATOM   1445  N   TYR B  42      35.750  34.625   6.743  1.00 58.19           N  \nATOM   1446  CA  TYR B  42      35.148  35.135   7.980  1.00 52.18           C  \nATOM   1447  C   TYR B  42      36.172  35.273   9.109  1.00 68.59           C  \nATOM   1448  O   TYR B  42      35.985  36.069  10.033  1.00 74.47           O  \nATOM   1449  CB  TYR B  42      34.028  34.174   8.436  1.00 50.27           C  \nATOM   1450  CG  TYR B  42      33.309  34.547   9.725  1.00 50.51           C  \nATOM   1451  CD1 TYR B  42      31.977  34.988   9.706  1.00 49.78           C  \nATOM   1452  CD2 TYR B  42      33.933  34.415  10.965  1.00 48.93           C  \nATOM   1453  CE1 TYR B  42      31.312  35.339  10.878  1.00 29.13           C  \nATOM   1454  CE2 TYR B  42      33.283  34.757  12.137  1.00 47.62           C  \nATOM   1455  CZ  TYR B  42      31.969  35.217  12.089  1.00 47.61           C  \nATOM   1456  OH  TYR B  42      31.323  35.556  13.257  1.00 55.49           O  \nATOM   1457  N   ILE B  43      37.263  34.513   9.019  1.00 62.35           N  \nATOM   1458  CA  ILE B  43      38.312  34.570  10.031  1.00 56.39           C  \nATOM   1459  C   ILE B  43      39.714  34.684   9.446  1.00 47.07           C  \nATOM   1460  O   ILE B  43      40.014  34.115   8.399  1.00 36.89           O  \nATOM   1461  CB  ILE B  43      38.056  33.856  11.353  1.00 63.08           C  \nATOM   1462  CG1 ILE B  43      37.264  34.841  12.262  1.00 66.49           C  \nATOM   1463  CG2 ILE B  43      39.381  33.360  11.955  1.00 60.01           C  \nATOM   1464  CD1 ILE B  43      36.702  34.300  13.573  1.00 54.39           C  \nATOM   1465  N   SER B  44      40.557  35.453  10.129  1.00 50.11           N  \nATOM   1466  CA  SER B  44      41.929  35.703   9.695  1.00 45.66           C  \nATOM   1467  C   SER B  44      42.958  34.839  10.326  1.00 35.94           C  \nATOM   1468  O   SER B  44      44.138  34.897  10.018  1.00 34.73           O  \nATOM   1469  CB  SER B  44      42.329  37.161   9.940  1.00 41.23           C  \nATOM   1470  OG  SER B  44      42.782  37.323  11.276  1.00 41.65           O  \nATOM   1471  N   GLU B  45      42.694  33.949  11.238  1.00 31.89           N  \nATOM   1472  CA  GLU B  45      43.907  33.293  11.637  1.00 42.64           C  \nATOM   1473  C   GLU B  45      44.098  31.922  10.977  1.00 43.74           C  \nATOM   1474  O   GLU B  45      45.197  31.349  11.000  1.00  1.00           O  \nATOM   1475  CB  GLU B  45      44.116  33.184  13.107  1.00 40.35           C  \nATOM   1476  CG  GLU B  45      45.424  33.856  13.557  1.00 69.62           C  \nATOM   1477  CD  GLU B  45      46.682  33.105  13.090  1.00100.00           C  \nATOM   1478  OE1 GLU B  45      46.580  31.909  12.614  1.00100.00           O  \nATOM   1479  OE2 GLU B  45      47.846  33.661  13.179  1.00100.00           O  \nATOM   1480  N   ASN B  46      43.072  31.380  10.365  1.00 38.12           N  \nATOM   1481  CA  ASN B  46      43.244  30.093   9.670  1.00 27.58           C  \nATOM   1482  C   ASN B  46      44.353  30.285   8.642  1.00 36.95           C  \nATOM   1483  O   ASN B  46      44.992  29.326   8.184  1.00 45.51           O  \nATOM   1484  CB  ASN B  46      41.956  29.669   8.966  1.00 22.10           C  \nATOM   1485  CG  ASN B  46      41.484  30.670   7.913  1.00 33.25           C  \nATOM   1486  OD1 ASN B  46      42.287  31.174   7.131  1.00 26.29           O  \nATOM   1487  ND2 ASN B  46      40.206  30.995   7.848  1.00 48.82           N  \nATOM   1488  N   LYS B  47      44.537  31.549   8.324  1.00 38.55           N  \nATOM   1489  CA  LYS B  47      45.533  31.975   7.373  1.00 40.30           C  \nATOM   1490  C   LYS B  47      46.926  31.747   7.910  1.00 55.85           C  \nATOM   1491  O   LYS B  47      47.907  31.710   7.176  1.00 66.10           O  \nATOM   1492  CB  LYS B  47      45.423  33.480   7.065  1.00 43.27           C  \nATOM   1493  CG  LYS B  47      44.022  34.048   7.270  1.00 66.39           C  \nATOM   1494  CD  LYS B  47      43.844  35.439   6.650  1.00 42.20           C  \nATOM   1495  CE  LYS B  47      43.545  35.389   5.150  1.00 29.09           C  \nATOM   1496  NZ  LYS B  47      44.172  34.241   4.478  1.00 41.03           N  \nATOM   1497  N   GLU B  48      47.086  31.564   9.207  1.00 42.24           N  \nATOM   1498  CA  GLU B  48      48.463  31.437   9.655  1.00 32.88           C  \nATOM   1499  C   GLU B  48      49.124  30.168   9.146  1.00 50.75           C  \nATOM   1500  O   GLU B  48      49.954  30.199   8.253  1.00 55.85           O  \nATOM   1501  CB  GLU B  48      48.667  31.493  11.130  1.00 29.39           C  \nATOM   1502  CG  GLU B  48      50.170  31.796  11.495  1.00 47.31           C  \nATOM   1503  CD  GLU B  48      51.099  32.268  10.321  1.00100.00           C  \nATOM   1504  OE1 GLU B  48      52.392  32.349  10.500  1.00 16.92           O  \nATOM   1505  OE2 GLU B  48      50.611  32.585   9.170  1.00 49.28           O  \nATOM   1506  N   LYS B  49      48.849  28.960   9.654  1.00 38.84           N  \nATOM   1507  CA  LYS B  49      49.627  27.834   9.039  1.00 32.50           C  \nATOM   1508  C   LYS B  49      49.248  27.754   7.540  1.00 17.18           C  \nATOM   1509  O   LYS B  49      49.913  27.076   6.747  1.00 18.41           O  \nATOM   1510  CB  LYS B  49      49.439  26.443   9.717  1.00 41.16           C  \nATOM   1511  CG  LYS B  49      48.020  26.061  10.066  1.00 60.80           C  \nATOM   1512  CD  LYS B  49      47.858  24.579  10.448  1.00 11.29           C  \nATOM   1513  CE  LYS B  49      48.942  23.676   9.819  1.00 13.09           C  \nATOM   1514  NZ  LYS B  49      50.147  23.520  10.656  1.00 11.24           N  \nATOM   1515  N   GLU B  50      48.180  28.481   7.205  1.00 21.05           N  \nATOM   1516  CA  GLU B  50      47.664  28.575   5.816  1.00 34.52           C  \nATOM   1517  C   GLU B  50      48.825  28.893   4.885  1.00 60.39           C  \nATOM   1518  O   GLU B  50      49.189  28.074   4.041  1.00 80.48           O  \nATOM   1519  CB  GLU B  50      46.615  29.707   5.734  1.00 34.77           C  \nATOM   1520  CG  GLU B  50      45.622  29.559   4.568  1.00 31.00           C  \nATOM   1521  CD  GLU B  50      45.868  30.561   3.426  1.00 42.86           C  \nATOM   1522  OE1 GLU B  50      44.923  31.347   3.029  1.00 99.43           O  \nATOM   1523  OE2 GLU B  50      47.026  30.614   2.858  1.00 63.35           O  \nATOM   1524  N   TYR B  51      49.435  30.052   5.071  1.00 43.87           N  \nATOM   1525  CA  TYR B  51      50.565  30.406   4.240  1.00 41.83           C  \nATOM   1526  C   TYR B  51      51.849  29.712   4.693  1.00 58.78           C  \nATOM   1527  O   TYR B  51      52.764  29.485   3.894  1.00 54.32           O  \nATOM   1528  CB  TYR B  51      50.543  31.773   3.607  1.00 37.29           C  \nATOM   1529  CG  TYR B  51      50.478  31.672   2.095  1.00 39.40           C  \nATOM   1530  CD1 TYR B  51      51.460  32.270   1.320  1.00 53.00           C  \nATOM   1531  CD2 TYR B  51      49.503  30.940   1.443  1.00 31.05           C  \nATOM   1532  CE1 TYR B  51      51.456  32.207  -0.058  1.00 46.43           C  \nATOM   1533  CE2 TYR B  51      49.483  30.888   0.045  1.00 26.35           C  \nATOM   1534  CZ  TYR B  51      50.474  31.535  -0.703  1.00 19.21           C  \nATOM   1535  OH  TYR B  51      50.581  31.507  -2.095  1.00 12.22           O  \nATOM   1536  N   TYR B  52      51.890  29.352   5.975  1.00 61.59           N  \nATOM   1537  CA  TYR B  52      53.038  28.659   6.551  1.00 65.21           C  \nATOM   1538  C   TYR B  52      53.190  27.287   5.892  1.00 66.73           C  \nATOM   1539  O   TYR B  52      54.265  26.682   5.901  1.00 62.57           O  \nATOM   1540  CB  TYR B  52      52.769  28.539   8.068  1.00 68.31           C  \nATOM   1541  CG  TYR B  52      53.951  28.211   8.951  1.00 86.64           C  \nATOM   1542  CD1 TYR B  52      53.761  27.531  10.162  1.00 89.62           C  \nATOM   1543  CD2 TYR B  52      55.252  28.552   8.583  1.00 93.11           C  \nATOM   1544  CE1 TYR B  52      54.832  27.212  10.987  1.00 83.07           C  \nATOM   1545  CE2 TYR B  52      56.337  28.223   9.388  1.00 94.43           C  \nATOM   1546  CZ  TYR B  52      56.126  27.552  10.584  1.00 80.30           C  \nATOM   1547  OH  TYR B  52      57.225  27.249  11.356  1.00 32.94           O  \nATOM   1548  N   LEU B  53      52.095  26.811   5.314  1.00 52.81           N  \nATOM   1549  CA  LEU B  53      52.082  25.523   4.651  1.00 42.47           C  \nATOM   1550  C   LEU B  53      52.048  25.660   3.133  1.00 71.11           C  \nATOM   1551  O   LEU B  53      52.768  24.952   2.424  1.00 77.74           O  \nATOM   1552  CB  LEU B  53      50.940  24.651   5.209  1.00 32.46           C  \nATOM   1553  CG  LEU B  53      51.284  23.791   6.452  1.00 30.08           C  \nATOM   1554  CD1 LEU B  53      52.481  22.850   6.211  1.00 12.83           C  \nATOM   1555  CD2 LEU B  53      51.513  24.659   7.690  1.00 44.13           C  \nATOM   1556  N   LYS B  54      51.216  26.575   2.630  1.00 73.40           N  \nATOM   1557  CA  LYS B  54      51.083  26.783   1.182  1.00 74.73           C  \nATOM   1558  C   LYS B  54      52.388  27.144   0.479  1.00 89.10           C  \nATOM   1559  O   LYS B  54      52.412  27.433  -0.720  1.00 75.26           O  \nATOM   1560  CB  LYS B  54      49.955  27.640   0.693  1.00 65.11           C  \nATOM   1561  CG  LYS B  54      49.616  27.230  -0.728  1.00 51.30           C  \nATOM   1562  CD  LYS B  54      48.377  26.382  -0.724  1.00 41.04           C  \nATOM   1563  CE  LYS B  54      47.505  26.907  -1.855  1.00 71.20           C  \nATOM   1564  NZ  LYS B  54      46.042  26.774  -1.631  1.00 65.94           N  \nATOM   1565  N   ASP B  55      53.486  27.089   1.213  1.00 91.09           N  \nATOM   1566  CA  ASP B  55      54.782  27.381   0.626  1.00 83.83           C  \nATOM   1567  C   ASP B  55      55.830  26.401   1.128  1.00 85.16           C  \nATOM   1568  O   ASP B  55      56.591  25.842   0.339  1.00 89.66           O  \nATOM   1569  CB  ASP B  55      55.285  28.778   0.844  1.00 81.95           C  \nATOM   1570  CG  ASP B  55      56.661  28.942   0.244  1.00 54.77           C  \nATOM   1571  OD1 ASP B  55      57.658  28.869   1.008  1.00 56.86           O  \nATOM   1572  OD2 ASP B  55      56.740  28.855  -1.001  1.00  1.00           O  \nATOM   1573  N   ILE B  56      55.873  26.209   2.445  1.00 74.76           N  \nATOM   1574  CA  ILE B  56      56.821  25.287   3.051  1.00 73.33           C  \nATOM   1575  C   ILE B  56      56.126  23.989   3.447  1.00 96.21           C  \nATOM   1576  O   ILE B  56      55.779  23.795   4.615  1.00100.00           O  \nATOM   1577  CB  ILE B  56      57.536  25.825   4.302  1.00 70.45           C  \nATOM   1578  CG1 ILE B  56      57.528  27.356   4.351  1.00 76.56           C  \nATOM   1579  CG2 ILE B  56      58.972  25.306   4.308  1.00 50.20           C  \nATOM   1580  CD1 ILE B  56      58.294  27.950   5.527  1.00 73.04           C  \nATOM   1581  N   VAL B  67      54.108  32.607 -12.332  1.00 23.76           N  \nATOM   1582  CA  VAL B  67      53.937  31.575 -13.375  1.00 41.96           C  \nATOM   1583  C   VAL B  67      54.354  30.154 -12.972  1.00 66.72           C  \nATOM   1584  O   VAL B  67      53.758  29.162 -13.433  1.00 47.36           O  \nATOM   1585  CB  VAL B  67      54.726  31.927 -14.638  1.00 58.26           C  \nATOM   1586  CG1 VAL B  67      55.709  33.074 -14.427  1.00 61.64           C  \nATOM   1587  CG2 VAL B  67      55.548  30.756 -15.176  1.00 62.76           C  \nATOM   1588  N   VAL B  68      55.367  29.966 -12.115  1.00 87.03           N  \nATOM   1589  CA  VAL B  68      55.707  28.570 -11.805  1.00 91.91           C  \nATOM   1590  C   VAL B  68      54.670  28.006 -10.836  1.00 95.96           C  \nATOM   1591  O   VAL B  68      54.021  27.017 -11.144  1.00100.00           O  \nATOM   1592  CB  VAL B  68      57.137  28.345 -11.350  1.00 98.02           C  \nATOM   1593  CG1 VAL B  68      57.440  26.851 -11.110  1.00100.00           C  \nATOM   1594  CG2 VAL B  68      58.159  28.778 -12.423  1.00 98.08           C  \nATOM   1595  N   LYS B  69      54.494  28.642  -9.674  1.00 81.79           N  \nATOM   1596  CA  LYS B  69      53.507  28.145  -8.709  1.00 80.98           C  \nATOM   1597  C   LYS B  69      52.100  28.263  -9.289  1.00 76.87           C  \nATOM   1598  O   LYS B  69      51.149  27.610  -8.800  1.00 71.59           O  \nATOM   1599  CB  LYS B  69      53.596  28.851  -7.362  1.00 81.77           C  \nATOM   1600  CG  LYS B  69      53.553  30.360  -7.455  1.00 59.50           C  \nATOM   1601  CD  LYS B  69      52.875  30.933  -6.237  1.00 26.49           C  \nATOM   1602  CE  LYS B  69      52.759  29.911  -5.113  1.00 31.23           C  \nATOM   1603  NZ  LYS B  69      51.821  30.367  -4.004  1.00 33.29           N  \nATOM   1604  N   ALA B  70      51.969  29.072 -10.323  1.00 56.12           N  \nATOM   1605  CA  ALA B  70      50.688  29.261 -10.984  1.00 54.68           C  \nATOM   1606  C   ALA B  70      50.216  27.935 -11.559  1.00 58.37           C  \nATOM   1607  O   ALA B  70      49.346  27.269 -10.996  1.00 56.05           O  \nATOM   1608  CB  ALA B  70      50.817  30.295 -12.092  1.00 51.72           C  \nATOM   1609  N   VAL B  71      50.809  27.545 -12.677  1.00 49.40           N  \nATOM   1610  CA  VAL B  71      50.441  26.291 -13.312  1.00 43.70           C  \nATOM   1611  C   VAL B  71      50.564  25.106 -12.349  1.00 35.72           C  \nATOM   1612  O   VAL B  71      49.932  24.068 -12.549  1.00 29.34           O  \nATOM   1613  CB  VAL B  71      51.118  26.080 -14.677  1.00 39.90           C  \nATOM   1614  CG1 VAL B  71      50.916  24.644 -15.092  1.00 41.78           C  \nATOM   1615  CG2 VAL B  71      50.512  27.015 -15.714  1.00 34.44           C  \nATOM   1616  N   LYS B  72      51.379  25.258 -11.302  1.00 14.51           N  \nATOM   1617  CA  LYS B  72      51.534  24.195 -10.322  1.00 20.31           C  \nATOM   1618  C   LYS B  72      50.244  24.110  -9.520  1.00 51.41           C  \nATOM   1619  O   LYS B  72      49.689  23.031  -9.298  1.00 64.46           O  \nATOM   1620  CB  LYS B  72      52.742  24.480  -9.417  1.00 26.33           C  \nATOM   1621  CG  LYS B  72      52.670  23.960  -7.976  1.00 54.90           C  \nATOM   1622  CD  LYS B  72      53.232  22.553  -7.825  1.00100.00           C  \nATOM   1623  CE  LYS B  72      53.412  22.150  -6.357  1.00100.00           C  \nATOM   1624  NZ  LYS B  72      54.708  22.541  -5.710  1.00 23.36           N  \nATOM   1625  N   ILE B  73      49.757  25.274  -9.113  1.00 42.17           N  \nATOM   1626  CA  ILE B  73      48.525  25.356  -8.353  1.00 28.04           C  \nATOM   1627  C   ILE B  73      47.341  25.012  -9.233  1.00 18.21           C  \nATOM   1628  O   ILE B  73      46.465  24.261  -8.828  1.00 32.82           O  \nATOM   1629  CB  ILE B  73      48.286  26.794  -7.915  1.00 26.05           C  \nATOM   1630  CG1 ILE B  73      48.666  26.991  -6.447  1.00 26.85           C  \nATOM   1631  CG2 ILE B  73      46.869  27.192  -8.212  1.00 28.17           C  \nATOM   1632  CD1 ILE B  73      50.163  27.114  -6.188  1.00 25.69           C  \nATOM   1633  N   LEU B  74      47.333  25.571 -10.437  1.00 25.78           N  \nATOM   1634  CA  LEU B  74      46.256  25.371 -11.399  1.00 16.84           C  \nATOM   1635  C   LEU B  74      45.977  23.910 -11.705  1.00 31.49           C  \nATOM   1636  O   LEU B  74      44.810  23.494 -11.736  1.00 23.34           O  \nATOM   1637  CB  LEU B  74      46.608  25.982 -12.732  1.00 19.02           C  \nATOM   1638  CG  LEU B  74      45.325  26.240 -13.457  1.00 20.35           C  \nATOM   1639  CD1 LEU B  74      44.906  25.001 -14.258  1.00 25.78           C  \nATOM   1640  CD2 LEU B  74      44.292  26.569 -12.364  1.00 44.90           C  \nATOM   1641  N   SER B  75      47.046  23.148 -11.965  1.00 48.68           N  \nATOM   1642  CA  SER B  75      46.916  21.726 -12.292  1.00 47.19           C  \nATOM   1643  C   SER B  75      46.302  20.891 -11.171  1.00 42.87           C  \nATOM   1644  O   SER B  75      45.581  19.929 -11.442  1.00 44.86           O  \nATOM   1645  CB  SER B  75      48.012  21.030 -13.071  1.00 36.76           C  \nATOM   1646  OG  SER B  75      47.552  19.713 -13.350  1.00 40.75           O  \nATOM   1647  N   GLN B  76      46.585  21.240  -9.920  1.00 26.65           N  \nATOM   1648  CA  GLN B  76      46.000  20.503  -8.809  1.00 18.42           C  \nATOM   1649  C   GLN B  76      44.578  21.001  -8.597  1.00 38.05           C  \nATOM   1650  O   GLN B  76      43.686  20.254  -8.214  1.00 40.37           O  \nATOM   1651  CB  GLN B  76      46.812  20.572  -7.528  1.00 19.11           C  \nATOM   1652  CG  GLN B  76      46.432  21.696  -6.615  1.00 60.29           C  \nATOM   1653  CD  GLN B  76      46.192  21.197  -5.219  1.00 83.09           C  \nATOM   1654  OE1 GLN B  76      46.385  21.927  -4.246  1.00 65.27           O  \nATOM   1655  NE2 GLN B  76      45.743  19.941  -5.114  1.00 74.74           N  \nATOM   1656  N   GLU B  77      44.359  22.286  -8.858  1.00 53.67           N  \nATOM   1657  CA  GLU B  77      43.020  22.842  -8.713  1.00 51.94           C  \nATOM   1658  C   GLU B  77      42.115  22.083  -9.666  1.00 37.31           C  \nATOM   1659  O   GLU B  77      40.901  22.003  -9.484  1.00 31.72           O  \nATOM   1660  CB  GLU B  77      43.002  24.335  -9.000  1.00 56.06           C  \nATOM   1661  CG  GLU B  77      42.719  25.194  -7.775  1.00 69.62           C  \nATOM   1662  CD  GLU B  77      41.508  26.108  -8.031  1.00100.00           C  \nATOM   1663  OE1 GLU B  77      41.640  27.058  -8.846  1.00100.00           O  \nATOM   1664  OE2 GLU B  77      40.423  25.850  -7.441  1.00100.00           O  \nATOM   1665  N   ASP B  78      42.757  21.525 -10.690  1.00 34.70           N  \nATOM   1666  CA  ASP B  78      42.114  20.701 -11.706  1.00 42.84           C  \nATOM   1667  C   ASP B  78      41.969  19.281 -11.148  1.00 47.06           C  \nATOM   1668  O   ASP B  78      40.985  18.577 -11.402  1.00 27.22           O  \nATOM   1669  CB  ASP B  78      43.104  20.626 -12.905  1.00 45.92           C  \nATOM   1670  CG  ASP B  78      42.769  19.529 -13.912  1.00 80.89           C  \nATOM   1671  OD1 ASP B  78      43.718  18.990 -14.554  1.00 72.61           O  \nATOM   1672  OD2 ASP B  78      41.566  19.321 -14.188  1.00100.00           O  \nATOM   1673  N   TYR B  79      42.968  18.881 -10.362  1.00 56.91           N  \nATOM   1674  CA  TYR B  79      43.021  17.564  -9.733  1.00 56.90           C  \nATOM   1675  C   TYR B  79      41.742  17.277  -8.953  1.00 56.84           C  \nATOM   1676  O   TYR B  79      41.098  16.249  -9.172  1.00 57.29           O  \nATOM   1677  CB  TYR B  79      44.181  17.556  -8.733  1.00 51.98           C  \nATOM   1678  CG  TYR B  79      45.203  16.468  -8.734  1.00 36.20           C  \nATOM   1679  CD1 TYR B  79      45.644  15.882  -9.907  1.00 40.68           C  \nATOM   1680  CD2 TYR B  79      45.897  16.184  -7.556  1.00 34.41           C  \nATOM   1681  CE1 TYR B  79      46.649  14.930  -9.875  1.00 39.09           C  \nATOM   1682  CE2 TYR B  79      46.889  15.233  -7.521  1.00 29.01           C  \nATOM   1683  CZ  TYR B  79      47.256  14.623  -8.680  1.00 44.97           C  \nATOM   1684  OH  TYR B  79      48.238  13.671  -8.680  1.00 78.89           O  \nATOM   1685  N   PHE B  80      41.373  18.168  -8.031  1.00 27.97           N  \nATOM   1686  CA  PHE B  80      40.158  17.949  -7.249  1.00 24.42           C  \nATOM   1687  C   PHE B  80      38.946  18.392  -8.033  1.00 46.07           C  \nATOM   1688  O   PHE B  80      37.809  18.082  -7.692  1.00 43.40           O  \nATOM   1689  CB  PHE B  80      40.103  18.538  -5.797  1.00 27.15           C  \nATOM   1690  CG  PHE B  80      40.031  20.092  -5.670  1.00 43.53           C  \nATOM   1691  CD1 PHE B  80      40.289  20.691  -4.415  1.00 35.73           C  \nATOM   1692  CD2 PHE B  80      39.738  20.967  -6.743  1.00 49.76           C  \nATOM   1693  CE1 PHE B  80      40.250  22.057  -4.234  1.00 17.99           C  \nATOM   1694  CE2 PHE B  80      39.695  22.348  -6.561  1.00 35.97           C  \nATOM   1695  CZ  PHE B  80      39.953  22.885  -5.302  1.00 29.74           C  \nATOM   1696  N   ASP B  81      39.206  19.147  -9.078  1.00 62.95           N  \nATOM   1697  CA  ASP B  81      38.141  19.667  -9.907  1.00 65.32           C  \nATOM   1698  C   ASP B  81      37.444  18.555 -10.683  1.00 67.29           C  \nATOM   1699  O   ASP B  81      36.264  18.656 -11.037  1.00 74.30           O  \nATOM   1700  CB  ASP B  81      38.695  20.827 -10.725  1.00 56.08           C  \nATOM   1701  CG  ASP B  81      37.892  22.059 -10.526  1.00  6.13           C  \nATOM   1702  OD1 ASP B  81      37.750  22.452  -9.334  1.00  1.77           O  \nATOM   1703  OD2 ASP B  81      37.161  22.388 -11.462  1.00 15.64           O  \nATOM   1704  N   LYS B  82      38.187  17.492 -10.926  1.00 42.72           N  \nATOM   1705  CA  LYS B  82      37.656  16.359 -11.644  1.00 36.82           C  \nATOM   1706  C   LYS B  82      36.924  15.416 -10.693  1.00 38.22           C  \nATOM   1707  O   LYS B  82      35.862  14.899 -11.027  1.00 53.92           O  \nATOM   1708  CB  LYS B  82      38.809  15.699 -12.367  1.00 34.60           C  \nATOM   1709  CG  LYS B  82      38.443  14.576 -13.285  1.00 33.92           C  \nATOM   1710  CD  LYS B  82      39.674  14.218 -14.103  1.00 38.62           C  \nATOM   1711  CE  LYS B  82      40.224  15.499 -14.731  1.00100.00           C  \nATOM   1712  NZ  LYS B  82      39.129  16.419 -15.165  1.00100.00           N  \nATOM   1713  N   LYS B  83      37.479  15.195  -9.502  1.00 22.49           N  \nATOM   1714  CA  LYS B  83      36.858  14.292  -8.515  1.00 17.71           C  \nATOM   1715  C   LYS B  83      35.590  14.866  -7.880  1.00 38.91           C  \nATOM   1716  O   LYS B  83      35.069  14.296  -6.924  1.00 28.77           O  \nATOM   1717  CB  LYS B  83      37.788  13.932  -7.348  1.00 17.40           C  \nATOM   1718  CG  LYS B  83      38.375  12.533  -7.206  1.00 11.00           C  \nATOM   1719  CD  LYS B  83      38.798  12.531  -5.753  1.00 23.59           C  \nATOM   1720  CE  LYS B  83      39.522  11.305  -5.238  1.00 11.33           C  \nATOM   1721  NZ  LYS B  83      39.203  11.108  -3.824  1.00 21.00           N  \nATOM   1722  N   ARG B  84      35.134  16.018  -8.376  1.00 55.71           N  \nATOM   1723  CA  ARG B  84      33.924  16.678  -7.884  1.00 40.47           C  \nATOM   1724  C   ARG B  84      32.822  16.462  -8.900  1.00 47.68           C  \nATOM   1725  O   ARG B  84      31.715  16.043  -8.545  1.00 53.15           O  \nATOM   1726  CB  ARG B  84      34.135  18.178  -7.679  1.00 22.90           C  \nATOM   1727  CG  ARG B  84      32.954  19.000  -7.120  1.00 16.14           C  \nATOM   1728  CD  ARG B  84      32.055  19.685  -8.194  1.00 51.00           C  \nATOM   1729  NE  ARG B  84      30.689  19.924  -7.665  1.00 80.23           N  \nATOM   1730  CZ  ARG B  84      29.743  20.685  -8.251  1.00 90.06           C  \nATOM   1731  NH1 ARG B  84      29.999  21.325  -9.393  1.00 15.20           N  \nATOM   1732  NH2 ARG B  84      28.546  20.849  -7.672  1.00 65.36           N  \nATOM   1733  N   ASN B  85      33.149  16.736 -10.172  1.00 48.35           N  \nATOM   1734  CA  ASN B  85      32.214  16.556 -11.291  1.00 47.93           C  \nATOM   1735  C   ASN B  85      31.995  15.064 -11.482  1.00 49.93           C  \nATOM   1736  O   ASN B  85      30.937  14.609 -11.923  1.00 35.55           O  \nATOM   1737  CB  ASN B  85      32.844  17.042 -12.613  1.00 41.00           C  \nATOM   1738  CG  ASN B  85      32.654  18.521 -12.793  1.00 77.53           C  \nATOM   1739  OD1 ASN B  85      33.594  19.264 -13.124  1.00100.00           O  \nATOM   1740  ND2 ASN B  85      31.469  18.985 -12.386  1.00 35.18           N  \nATOM   1741  N   GLU B  86      33.025  14.316 -11.109  1.00 48.64           N  \nATOM   1742  CA  GLU B  86      33.024  12.867 -11.196  1.00 45.66           C  \nATOM   1743  C   GLU B  86      32.217  12.286 -10.054  1.00 36.97           C  \nATOM   1744  O   GLU B  86      31.350  11.436 -10.253  1.00 34.21           O  \nATOM   1745  CB  GLU B  86      34.464  12.418 -10.958  1.00 46.46           C  \nATOM   1746  CG  GLU B  86      34.855  11.050 -11.463  1.00 39.06           C  \nATOM   1747  CD  GLU B  86      36.355  10.908 -11.500  1.00 76.63           C  \nATOM   1748  OE1 GLU B  86      36.973  11.574 -12.365  1.00 27.25           O  \nATOM   1749  OE2 GLU B  86      36.909  10.241 -10.592  1.00 77.36           O  \nATOM   1750  N   HIS B  87      32.521  12.752  -8.852  1.00 28.84           N  \nATOM   1751  CA  HIS B  87      31.840  12.265  -7.671  1.00 22.33           C  \nATOM   1752  C   HIS B  87      30.390  12.715  -7.588  1.00 32.18           C  \nATOM   1753  O   HIS B  87      29.596  12.164  -6.823  1.00 16.69           O  \nATOM   1754  CB  HIS B  87      32.760  11.993  -6.487  1.00 20.72           C  \nATOM   1755  CG  HIS B  87      32.761  13.029  -5.441  1.00 30.62           C  \nATOM   1756  ND1 HIS B  87      33.292  12.800  -4.183  1.00 33.08           N  \nATOM   1757  CD2 HIS B  87      32.256  14.280  -5.422  1.00 47.85           C  \nATOM   1758  CE1 HIS B  87      33.088  13.861  -3.432  1.00 35.15           C  \nATOM   1759  NE2 HIS B  87      32.503  14.784  -4.163  1.00 45.66           N  \nATOM   1760  N   ASP B  88      30.056  13.697  -8.419  1.00 45.56           N  \nATOM   1761  CA  ASP B  88      28.710  14.257  -8.492  1.00 46.10           C  \nATOM   1762  C   ASP B  88      27.726  13.307  -9.182  1.00 57.24           C  \nATOM   1763  O   ASP B  88      26.523  13.567  -9.209  1.00 61.73           O  \nATOM   1764  CB  ASP B  88      28.803  15.585  -9.244  1.00 22.20           C  \nATOM   1765  CG  ASP B  88      28.108  16.732  -8.514  1.00 35.26           C  \nATOM   1766  OD1 ASP B  88      27.577  16.586  -7.386  1.00  1.00           O  \nATOM   1767  OD2 ASP B  88      28.112  17.888  -9.163  1.00 19.25           O  \nATOM   1768  N   GLU B  89      28.227  12.217  -9.749  1.00 50.05           N  \nATOM   1769  CA  GLU B  89      27.365  11.252 -10.423  1.00 41.61           C  \nATOM   1770  C   GLU B  89      26.738  10.297  -9.419  1.00 44.71           C  \nATOM   1771  O   GLU B  89      25.823   9.540  -9.746  1.00 43.45           O  \nATOM   1772  CB  GLU B  89      28.191  10.348 -11.345  1.00 39.93           C  \nATOM   1773  CG  GLU B  89      28.237  10.749 -12.793  1.00 36.24           C  \nATOM   1774  CD  GLU B  89      26.872  10.865 -13.392  1.00 74.97           C  \nATOM   1775  OE1 GLU B  89      26.654  10.269 -14.464  1.00 28.48           O  \nATOM   1776  OE2 GLU B  89      26.047  11.635 -12.857  1.00100.00           O  \nATOM   1777  N   ARG B  90      27.241  10.329  -8.193  1.00 40.96           N  \nATOM   1778  CA  ARG B  90      26.742   9.429  -7.172  1.00 44.93           C  \nATOM   1779  C   ARG B  90      26.616  10.092  -5.805  1.00 46.93           C  \nATOM   1780  O   ARG B  90      26.651   9.427  -4.772  1.00 48.71           O  \nATOM   1781  CB  ARG B  90      27.606   8.164  -7.204  1.00 55.48           C  \nATOM   1782  CG  ARG B  90      27.907   7.644  -8.663  1.00 13.98           C  \nATOM   1783  CD  ARG B  90      29.246   6.853  -8.747  1.00 10.60           C  \nATOM   1784  NE  ARG B  90      29.204   5.484  -8.213  1.00 14.75           N  \nATOM   1785  CZ  ARG B  90      29.138   5.168  -6.916  1.00 62.29           C  \nATOM   1786  NH1 ARG B  90      29.079   6.120  -6.002  1.00 57.55           N  \nATOM   1787  NH2 ARG B  90      29.108   3.898  -6.538  1.00 52.34           N  \nATOM   1788  N   THR B  91      26.472  11.411  -5.813  1.00 52.40           N  \nATOM   1789  CA  THR B  91      26.285  12.167  -4.579  1.00 52.19           C  \nATOM   1790  C   THR B  91      24.795  12.459  -4.429  1.00 35.81           C  \nATOM   1791  O   THR B  91      24.344  13.161  -3.522  1.00 22.95           O  \nATOM   1792  CB  THR B  91      27.240  13.347  -4.295  1.00 38.64           C  \nATOM   1793  OG1 THR B  91      27.203  13.603  -2.886  1.00 12.26           O  \nATOM   1794  CG2 THR B  91      26.801  14.607  -5.028  1.00 43.14           C  \nATOM   1795  N   VAL B  92      24.047  11.880  -5.353  1.00 23.63           N  \nATOM   1796  CA  VAL B  92      22.607  12.021  -5.402  1.00 29.28           C  \nATOM   1797  C   VAL B  92      21.926  11.294  -4.252  1.00 58.81           C  \nATOM   1798  O   VAL B  92      20.699  11.272  -4.199  1.00 64.43           O  \nATOM   1799  CB  VAL B  92      22.067  11.299  -6.599  1.00 29.80           C  \nATOM   1800  CG1 VAL B  92      21.959   9.828  -6.236  1.00 35.07           C  \nATOM   1801  CG2 VAL B  92      20.729  11.897  -6.991  1.00 24.89           C  \nATOM   1802  N   LEU B  93      22.705  10.701  -3.350  1.00 56.65           N  \nATOM   1803  CA  LEU B  93      22.135   9.978  -2.207  1.00 42.76           C  \nATOM   1804  C   LEU B  93      21.095  10.832  -1.481  1.00 37.50           C  \nATOM   1805  O   LEU B  93      19.972  10.404  -1.248  1.00 41.78           O  \nATOM   1806  CB  LEU B  93      23.176   9.724  -1.072  1.00 30.21           C  \nATOM   1807  CG  LEU B  93      24.638   9.228  -1.128  1.00 17.33           C  \nATOM   1808  CD1 LEU B  93      25.563  10.178  -1.865  1.00  1.03           C  \nATOM   1809  CD2 LEU B  93      25.173   9.084   0.314  1.00 10.64           C  \nATOM   1810  N   ILE B  94      21.496  12.055  -1.153  1.00 32.03           N  \nATOM   1811  CA  ILE B  94      20.689  13.042  -0.437  1.00 36.08           C  \nATOM   1812  C   ILE B  94      19.288  13.257  -1.002  1.00 47.88           C  \nATOM   1813  O   ILE B  94      18.325  12.634  -0.554  1.00 21.39           O  \nATOM   1814  CB  ILE B  94      21.404  14.461  -0.643  1.00 38.63           C  \nATOM   1815  CG1 ILE B  94      20.566  15.652  -0.018  1.00 47.84           C  \nATOM   1816  CG2 ILE B  94      21.944  14.651  -2.108  1.00 61.00           C  \nATOM   1817  CD1 ILE B  94      19.893  15.353   1.326  1.00 53.08           C  \nATOM   1818  N   LEU B  95      19.199  14.189  -1.964  1.00 64.24           N  \nATOM   1819  CA  LEU B  95      17.963  14.585  -2.621  1.00 61.57           C  \nATOM   1820  C   LEU B  95      17.082  13.393  -2.929  1.00 49.70           C  \nATOM   1821  O   LEU B  95      15.862  13.483  -2.761  1.00 12.33           O  \nATOM   1822  CB  LEU B  95      18.198  15.201  -4.004  1.00 62.81           C  \nATOM   1823  CG  LEU B  95      17.070  14.729  -4.952  1.00 66.87           C  \nATOM   1824  CD1 LEU B  95      16.175  15.845  -5.488  1.00 72.26           C  \nATOM   1825  CD2 LEU B  95      17.512  13.681  -5.974  1.00 42.81           C  \nATOM   1826  N   VAL B  96      17.720  12.307  -3.374  1.00 65.51           N  \nATOM   1827  CA  VAL B  96      17.075  11.047  -3.725  1.00 58.24           C  \nATOM   1828  C   VAL B  96      16.528  10.379  -2.487  1.00 40.11           C  \nATOM   1829  O   VAL B  96      15.326  10.174  -2.363  1.00 27.76           O  \nATOM   1830  CB  VAL B  96      18.167  10.093  -4.197  1.00 55.24           C  \nATOM   1831  CG1 VAL B  96      17.736   8.657  -3.963  1.00 53.70           C  \nATOM   1832  CG2 VAL B  96      18.486  10.366  -5.669  1.00 58.48           C  \nATOM   1833  N   ASN B  97      17.426  10.051  -1.564  1.00 40.28           N  \nATOM   1834  CA  ASN B  97      17.009   9.456  -0.306  1.00 40.44           C  \nATOM   1835  C   ASN B  97      16.022  10.427   0.321  1.00 40.74           C  \nATOM   1836  O   ASN B  97      15.147  10.046   1.083  1.00 52.12           O  \nATOM   1837  CB  ASN B  97      18.221   9.269   0.611  1.00 46.35           C  \nATOM   1838  CG  ASN B  97      17.839   9.058   2.073  1.00 72.48           C  \nATOM   1839  OD1 ASN B  97      18.549   9.532   2.990  1.00 23.30           O  \nATOM   1840  ND2 ASN B  97      16.752   8.301   2.306  1.00 69.70           N  \nATOM   1841  N   ALA B  98      16.191  11.695  -0.043  1.00 32.27           N  \nATOM   1842  CA  ALA B  98      15.318  12.758   0.434  1.00 46.56           C  \nATOM   1843  C   ALA B  98      14.039  12.722  -0.389  1.00 70.07           C  \nATOM   1844  O   ALA B  98      12.944  12.998   0.107  1.00 75.86           O  \nATOM   1845  CB  ALA B  98      15.985  14.111   0.283  1.00 48.67           C  \nATOM   1846  N   GLN B  99      14.197  12.371  -1.668  1.00 59.70           N  \nATOM   1847  CA  GLN B  99      13.076  12.248  -2.591  1.00 58.87           C  \nATOM   1848  C   GLN B  99      12.288  11.061  -2.101  1.00 59.09           C  \nATOM   1849  O   GLN B  99      11.098  11.161  -1.787  1.00 39.12           O  \nATOM   1850  CB  GLN B  99      13.550  11.969  -4.042  1.00 57.35           C  \nATOM   1851  CG  GLN B  99      12.472  12.098  -5.184  1.00  6.74           C  \nATOM   1852  CD  GLN B  99      12.614  10.967  -6.268  1.00 29.51           C  \nATOM   1853  OE1 GLN B  99      11.822  10.873  -7.227  1.00  1.00           O  \nATOM   1854  NE2 GLN B  99      13.627  10.098  -6.081  1.00 41.23           N  \nATOM   1855  N   GLN B 100      12.992   9.952  -1.932  1.00 67.37           N  \nATOM   1856  CA  GLN B 100      12.345   8.766  -1.427  1.00 65.85           C  \nATOM   1857  C   GLN B 100      11.950   9.086   0.001  1.00 83.30           C  \nATOM   1858  O   GLN B 100      11.061   8.485   0.588  1.00 90.63           O  \nATOM   1859  CB  GLN B 100      13.282   7.548  -1.577  1.00 64.64           C  \nATOM   1860  CG  GLN B 100      13.160   6.888  -2.984  1.00 74.33           C  \nATOM   1861  CD  GLN B 100      14.362   7.100  -3.971  1.00 54.79           C  \nATOM   1862  OE1 GLN B 100      14.511   8.186  -4.544  1.00 31.15           O  \nATOM   1863  NE2 GLN B 100      15.234   6.078  -4.109  1.00 12.87           N  \nATOM   1864  N   ARG B 101      12.594  10.108   0.545  1.00 77.90           N  \nATOM   1865  CA  ARG B 101      12.336  10.608   1.905  1.00 70.19           C  \nATOM   1866  C   ARG B 101      10.922  11.177   1.949  1.00 63.58           C  \nATOM   1867  O   ARG B 101      10.049  10.821   2.731  1.00 61.49           O  \nATOM   1868  CB  ARG B 101      13.354  11.636   2.506  1.00 52.09           C  \nATOM   1869  CG  ARG B 101      14.347  11.046   3.581  1.00 14.27           C  \nATOM   1870  CD  ARG B 101      15.809  11.537   3.497  1.00 20.95           C  \nATOM   1871  NE  ARG B 101      16.384  11.795   4.838  1.00 69.66           N  \nATOM   1872  CZ  ARG B 101      17.686  11.896   5.167  1.00 44.91           C  \nATOM   1873  NH1 ARG B 101      18.656  11.738   4.270  1.00 31.00           N  \nATOM   1874  NH2 ARG B 101      18.019  12.139   6.433  1.00 21.20           N  \nATOM   1875  N   LYS B 102      10.631  12.098   1.029  1.00 58.07           N  \nATOM   1876  CA  LYS B 102       9.286  12.634   0.974  1.00 64.39           C  \nATOM   1877  C   LYS B 102       8.336  11.480   0.663  1.00 78.58           C  \nATOM   1878  O   LYS B 102       7.270  11.356   1.274  1.00 63.69           O  \nATOM   1879  CB  LYS B 102       9.181  13.753  -0.036  1.00 65.34           C  \nATOM   1880  CG  LYS B 102       7.722  14.210  -0.226  1.00 11.50           C  \nATOM   1881  CD  LYS B 102       7.268  15.195   0.857  1.00 38.32           C  \nATOM   1882  CE  LYS B 102       7.599  16.685   0.577  1.00 15.18           C  \nATOM   1883  NZ  LYS B 102       6.687  17.281  -0.472  1.00 11.78           N  \nATOM   1884  N   LYS B 103       8.749  10.607  -0.268  1.00 85.73           N  \nATOM   1885  CA  LYS B 103       7.935   9.455  -0.624  1.00 76.22           C  \nATOM   1886  C   LYS B 103       7.670   8.660   0.631  1.00 33.26           C  \nATOM   1887  O   LYS B 103       6.735   7.866   0.699  1.00 21.27           O  \nATOM   1888  CB  LYS B 103       8.641   8.575  -1.646  1.00 80.27           C  \nATOM   1889  CG  LYS B 103       8.147   7.130  -1.623  1.00 24.70           C  \nATOM   1890  CD  LYS B 103       6.995   6.882  -2.595  1.00 21.49           C  \nATOM   1891  CE  LYS B 103       7.475   6.513  -3.999  1.00 20.48           C  \nATOM   1892  NZ  LYS B 103       8.154   7.620  -4.687  1.00 16.86           N  \nATOM   1893  N   ILE B 104       8.493   8.846   1.694  1.00 16.37           N  \nATOM   1894  CA  ILE B 104       8.181   8.054   2.883  1.00 11.88           C  \nATOM   1895  C   ILE B 104       6.973   8.626   3.580  1.00 19.87           C  \nATOM   1896  O   ILE B 104       6.138   7.875   4.071  1.00 11.56           O  \nATOM   1897  CB  ILE B 104       9.278   7.862   3.976  1.00 15.79           C  \nATOM   1898  CG1 ILE B 104       9.118   8.746   5.223  1.00 13.23           C  \nATOM   1899  CG2 ILE B 104      10.699   8.067   3.520  1.00 15.89           C  \nATOM   1900  CD1 ILE B 104       9.953   8.249   6.421  1.00 32.57           C  \nATOM   1901  N   GLU B 105       6.853   9.922   3.621  1.00 17.17           N  \nATOM   1902  CA  GLU B 105       5.694  10.534   4.303  1.00 20.63           C  \nATOM   1903  C   GLU B 105       4.672  11.060   3.288  1.00 53.26           C  \nATOM   1904  O   GLU B 105       3.463  11.051   3.541  1.00 58.36           O  \nATOM   1905  CB  GLU B 105       6.185  11.701   5.190  1.00 18.14           C  \nATOM   1906  CG  GLU B 105       6.481  11.282   6.650  1.00 20.88           C  \nATOM   1907  CD  GLU B 105       7.973  11.386   7.045  1.00 67.66           C  \nATOM   1908  OE1 GLU B 105       8.744  12.251   6.474  1.00 25.55           O  \nATOM   1909  OE2 GLU B 105       8.460  10.602   7.955  1.00 15.49           O  \nATOM   1910  N   SER B 106       5.157  11.529   2.140  1.00 50.09           N  \nATOM   1911  CA  SER B 106       4.293  12.080   1.092  1.00 44.41           C  \nATOM   1912  C   SER B 106       3.140  11.154   0.717  1.00 32.47           C  \nATOM   1913  O   SER B 106       1.994  11.418   1.082  1.00 24.37           O  \nATOM   1914  CB  SER B 106       5.137  12.403  -0.142  1.00 52.70           C  \nATOM   1915  OG  SER B 106       4.287  12.658  -1.246  1.00 68.61           O  \nATOM   1916  N   LEU B 107       3.451  10.080  -0.021  1.00 37.56           N  \nATOM   1917  CA  LEU B 107       2.442   9.098  -0.439  1.00 26.08           C  \nATOM   1918  C   LEU B 107       1.811   8.525   0.812  1.00 22.55           C  \nATOM   1919  O   LEU B 107       0.593   8.576   0.976  1.00  8.28           O  \nATOM   1920  CB  LEU B 107       3.056   7.947  -1.279  1.00 17.74           C  \nATOM   1921  CG  LEU B 107       2.390   7.635  -2.635  1.00 20.98           C  \nATOM   1922  CD1 LEU B 107       1.852   8.918  -3.228  1.00 30.39           C  \nATOM   1923  CD2 LEU B 107       3.361   6.993  -3.662  1.00  1.00           C  \nATOM   1924  N   LEU B 108       2.683   8.043   1.704  1.00 40.19           N  \nATOM   1925  CA  LEU B 108       2.287   7.485   2.988  1.00 55.28           C  \nATOM   1926  C   LEU B 108       1.359   8.390   3.665  1.00 47.72           C  \nATOM   1927  O   LEU B 108       0.502   7.984   4.434  1.00 39.94           O  \nATOM   1928  CB  LEU B 108       3.469   7.200   3.880  1.00 60.93           C  \nATOM   1929  CG  LEU B 108       3.285   7.272   5.390  1.00 67.08           C  \nATOM   1930  CD1 LEU B 108       4.290   6.362   6.134  1.00 78.93           C  \nATOM   1931  CD2 LEU B 108       3.515   8.669   5.956  1.00 32.69           C  \nATOM   1932  N   SER B 109       1.410   9.675   3.454  1.00 36.75           N  \nATOM   1933  CA  SER B 109       0.430  10.355   4.222  1.00 31.33           C  \nATOM   1934  C   SER B 109      -0.908  10.238   3.511  1.00 32.01           C  \nATOM   1935  O   SER B 109      -1.942  10.549   4.092  1.00 13.66           O  \nATOM   1936  CB  SER B 109       0.769  11.740   4.616  1.00 17.50           C  \nATOM   1937  OG  SER B 109       0.662  11.767   6.085  1.00 11.05           O  \nATOM   1938  N   ARG B 110      -0.903   9.802   2.254  1.00 29.34           N  \nATOM   1939  CA  ARG B 110      -2.201   9.448   1.602  1.00 27.35           C  \nATOM   1940  C   ARG B 110      -2.354   8.017   2.096  1.00 39.21           C  \nATOM   1941  O   ARG B 110      -3.455   7.476   2.121  1.00 43.16           O  \nATOM   1942  CB  ARG B 110      -2.138   9.448   0.054  1.00 11.00           C  \nATOM   1943  CG  ARG B 110      -2.287   7.991  -0.523  1.00 12.29           C  \nATOM   1944  CD  ARG B 110      -2.994   7.888  -1.900  1.00 15.29           C  \nATOM   1945  NE  ARG B 110      -2.051   8.026  -3.026  1.00 57.01           N  \nATOM   1946  CZ  ARG B 110      -2.413   8.242  -4.304  1.00 38.83           C  \nATOM   1947  NH1 ARG B 110      -3.703   8.323  -4.657  1.00 11.24           N  \nATOM   1948  NH2 ARG B 110      -1.541   8.400  -5.313  1.00 27.08           N  \nATOM   1949  N   VAL B 111      -1.229   7.398   2.484  1.00 34.06           N  \nATOM   1950  CA  VAL B 111      -1.277   6.025   2.980  1.00 27.17           C  \nATOM   1951  C   VAL B 111      -2.156   5.985   4.210  1.00 37.28           C  \nATOM   1952  O   VAL B 111      -3.259   5.449   4.161  1.00 43.37           O  \nATOM   1953  CB  VAL B 111       0.012   5.166   3.016  1.00 13.01           C  \nATOM   1954  CG1 VAL B 111       0.787   5.226   1.729  1.00 19.66           C  \nATOM   1955  CG2 VAL B 111       0.816   5.247   4.296  1.00 11.00           C  \nATOM   1956  N   ASN B 112      -1.698   6.592   5.302  1.00 40.50           N  \nATOM   1957  CA  ASN B 112      -2.525   6.637   6.500  1.00 46.00           C  \nATOM   1958  C   ASN B 112      -3.749   7.502   6.206  1.00 65.34           C  \nATOM   1959  O   ASN B 112      -4.668   7.589   7.010  1.00 78.80           O  \nATOM   1960  CB  ASN B 112      -1.779   6.999   7.771  1.00 39.17           C  \nATOM   1961  CG  ASN B 112      -0.690   6.016   8.055  1.00 23.49           C  \nATOM   1962  OD1 ASN B 112      -0.033   5.582   7.115  1.00 16.70           O  \nATOM   1963  ND2 ASN B 112      -0.526   5.590   9.315  1.00 11.20           N  \nATOM   1964  N   LYS B 113      -3.744   8.133   5.029  1.00 44.22           N  \nATOM   1965  CA  LYS B 113      -4.852   8.972   4.584  1.00 37.16           C  \nATOM   1966  C   LYS B 113      -5.957   8.110   3.988  1.00 28.68           C  \nATOM   1967  O   LYS B 113      -7.045   8.059   4.545  1.00 19.43           O  \nATOM   1968  CB  LYS B 113      -4.440  10.006   3.511  1.00 40.12           C  \nATOM   1969  CG  LYS B 113      -5.476  10.293   2.393  1.00 99.91           C  \nATOM   1970  CD  LYS B 113      -6.891  10.716   2.847  1.00100.00           C  \nATOM   1971  CE  LYS B 113      -7.985   9.701   2.474  1.00 68.33           C  \nATOM   1972  NZ  LYS B 113      -9.169   9.637   3.382  1.00 29.34           N  \nATOM   1973  N   ARG B 114      -5.660   7.466   2.843  1.00 39.63           N  \nATOM   1974  CA  ARG B 114      -6.599   6.592   2.121  1.00 34.79           C  \nATOM   1975  C   ARG B 114      -7.293   5.699   3.128  1.00 24.60           C  \nATOM   1976  O   ARG B 114      -8.496   5.412   3.001  1.00 14.16           O  \nATOM   1977  CB  ARG B 114      -5.848   5.649   1.122  1.00 11.86           C  \nATOM   1978  CG  ARG B 114      -5.143   6.352  -0.068  1.00 11.23           C  \nATOM   1979  CD  ARG B 114      -5.702   7.734  -0.423  1.00 48.66           C  \nATOM   1980  NE  ARG B 114      -6.763   7.709  -1.455  1.00 59.93           N  \nATOM   1981  CZ  ARG B 114      -7.908   8.396  -1.338  1.00 15.17           C  \nATOM   1982  NH1 ARG B 114      -8.141   9.156  -0.256  1.00 14.55           N  \nATOM   1983  NH2 ARG B 114      -8.892   8.388  -2.245  1.00 21.00           N  \nATOM   1984  N   ILE B 115      -6.522   5.250   4.173  1.00 28.90           N  \nATOM   1985  CA  ILE B 115      -7.142   4.328   5.115  1.00 22.71           C  \nATOM   1986  C   ILE B 115      -8.321   5.023   5.804  1.00 18.89           C  \nATOM   1987  O   ILE B 115      -9.173   4.395   6.423  1.00 21.10           O  \nATOM   1988  CB  ILE B 115      -6.229   3.613   6.100  1.00 25.25           C  \nATOM   1989  CG1 ILE B 115      -6.535   3.916   7.540  1.00 29.78           C  \nATOM   1990  CG2 ILE B 115      -4.768   3.879   5.879  1.00 19.90           C  \nATOM   1991  CD1 ILE B 115      -7.094   2.707   8.294  1.00 63.51           C  \nATOM   1992  N   THR B 116      -8.336   6.345   5.695  1.00 60.62           N  \nATOM   1993  CA  THR B 116      -9.406   7.151   6.282  1.00 75.84           C  \nATOM   1994  C   THR B 116     -10.484   7.457   5.242  1.00 87.66           C  \nATOM   1995  O   THR B 116     -11.589   7.892   5.570  1.00 84.96           O  \nATOM   1996  CB  THR B 116      -8.817   8.441   6.925  1.00 64.79           C  \nATOM   1997  OG1 THR B 116      -8.851   9.560   6.001  1.00 44.54           O  \nATOM   1998  CG2 THR B 116      -7.386   8.108   7.338  1.00 41.44           C  \nTER    1999      THR B 116                                                      \nHETATM 2000 CA    CA D 200      29.179  22.715 -12.389  1.00 71.98          CA  \nHETATM 2001 CA    CA B 200      14.098  29.996  -1.820  1.00 73.34          CA  \nCONECT   64 2001                                                                \nCONECT  270 2001                                                                \nCONECT  369 2000                                                                \nCONECT 1202 2001                                                                \nCONECT 2000  369                                                                \nCONECT 2001   64  270 1202                                                      \nMASTER      395    0    2   15    0    0    2    6 1997    4    6   22          \nEND                                                                             \n","pdb");
	viewer_17581716205928447.setStyle({"model": 0},{"stick": {}});
	viewer_17581716205928447.setStyle({"model": 0},{"cartoon": {"color": "blue"}});
	viewer_17581716205928447.addLabel("1fzpD",{"fontOpacity": 0.3},{"resi": 158});
	viewer_17581716205928447.zoomTo();
	viewer_17581716205928447.zoom(1.2);
	viewer_17581716205928447.setBackgroundColor("white");
viewer_17581716205928447.render();
});
</script>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div id="3dmolviewer_1758171620596065" style="position: relative; width: 400px; height: 300px;">
<p id="3dmolwarning_1758171620596065" style="background-color:#ffcccc;color:black">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br/></p>
</div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');
}

var viewer_1758171620596065 = null;
var warn = document.getElementById("3dmolwarning_1758171620596065");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_1758171620596065 = $3Dmol.createViewer(document.getElementById("3dmolviewer_1758171620596065"),{backgroundColor:"white"});
viewer_1758171620596065.zoomTo();
	viewer_1758171620596065.addModel("HEADER    TRANSCRIPTION                           19-JAN-06   2FRH              \nTITLE     CRYSTAL STRUCTURE OF SARA, A TRANSCRIPTION REGULATOR FROM             \nTITLE    2 STAPHYLOCOCCUS AUREUS                                                \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: STAPHYLOCOCCAL ACCESSORY REGULATOR A;                      \nCOMPND   3 CHAIN: A, B;                                                         \nCOMPND   4 SYNONYM: SARA;                                                       \nCOMPND   5 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 ORGANISM_SCIENTIFIC: STAPHYLOCOCCUS AUREUS;                          \nSOURCE   3 ORGANISM_TAXID: 1280;                                                \nSOURCE   4 GENE: SARA;                                                          \nSOURCE   5 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \nSOURCE   6 EXPRESSION_SYSTEM_TAXID: 562                                         \nKEYWDS    WINGED-HELIX PROTEIN, DIVALENT METAL BINDING, TRANSCRIPTION           \nEXPDTA    X-RAY DIFFRACTION                                                     \nAUTHOR    Y.LIU,A.C.MANNA,S.INGAVALE,A.L.CHEUNG,G.ZHANG                         \nREVDAT   5   14-FEB-24 2FRH    1       REMARK SEQADV LINK                       \nREVDAT   4   16-FEB-10 2FRH    1       DBREF                                    \nREVDAT   3   24-FEB-09 2FRH    1       VERSN                                    \nREVDAT   2   14-MAR-06 2FRH    1       JRNL                                     \nREVDAT   1   31-JAN-06 2FRH    0                                                \nJRNL        AUTH   Y.LIU,A.C.MANNA,C.H.PAN,I.A.KRIKSUNOV,D.J.THIEL,A.L.CHEUNG,  \nJRNL        AUTH 2 G.ZHANG                                                      \nJRNL        TITL   STRUCTURAL AND FUNCTION ANALYSES OF THE GLOBAL REGULATORY    \nJRNL        TITL 2 PROTEIN SARA FROM STAPHYLOCOCCUS AUREUS.                     \nJRNL        REF    PROC.NATL.ACAD.SCI.USA        V. 103  2392 2006              \nJRNL        REFN                   ISSN 0027-8424                               \nJRNL        PMID   16455801                                                     \nJRNL        DOI    10.1073/PNAS.0510439103                                      \nREMARK   2                                                                      \nREMARK   2 RESOLUTION.    2.50 ANGSTROMS.                                       \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : CNS 0.9                                              \nREMARK   3   AUTHORS     : BRUNGER,ADAMS,CLORE,DELANO,GROS,GROSSE-              \nREMARK   3               : KUNSTLEVE,JIANG,KUSZEWSKI,NILGES,PANNU,              \nREMARK   3               : READ,RICE,SIMONSON,WARREN                            \nREMARK   3                                                                      \nREMARK   3  REFINEMENT TARGET : NULL                                            \nREMARK   3                                                                      \nREMARK   3  DATA USED IN REFINEMENT.                                            \nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.50                           \nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 19.50                          \nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 2.000                          \nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : 284287.650                     \nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : 0.0000                         \nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 77.1                           \nREMARK   3   NUMBER OF REFLECTIONS             : 8562                           \nREMARK   3                                                                      \nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \nREMARK   3   CROSS-VALIDATION METHOD          : THROUGHOUT                      \nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \nREMARK   3   R VALUE            (WORKING SET) : 0.266                           \nREMARK   3   FREE R VALUE                     : 0.302                           \nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 5.300                           \nREMARK   3   FREE R VALUE TEST SET COUNT      : 450                             \nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : 0.014                           \nREMARK   3                                                                      \nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \nREMARK   3   TOTAL NUMBER OF BINS USED           : 6                            \nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : 2.50                         \nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : 2.66                         \nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : 22.80                        \nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : 387                          \nREMARK   3   BIN R VALUE           (WORKING SET) : 0.3180                       \nREMARK   3   BIN FREE R VALUE                    : 0.3950                       \nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : 4.70                         \nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : 19                           \nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : 0.091                        \nREMARK   3                                                                      \nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \nREMARK   3   PROTEIN ATOMS            : 2112                                    \nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \nREMARK   3   HETEROGEN ATOMS          : 2                                       \nREMARK   3   SOLVENT ATOMS            : 0                                       \nREMARK   3                                                                      \nREMARK   3  B VALUES.                                                           \nREMARK   3   FROM WILSON PLOT           (A**2) : 37.90                          \nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 77.60                          \nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \nREMARK   3    B11 (A**2) : 3.49000                                              \nREMARK   3    B22 (A**2) : 3.49000                                              \nREMARK   3    B33 (A**2) : -6.97000                                             \nREMARK   3    B12 (A**2) : 10.89000                                             \nREMARK   3    B13 (A**2) : 0.00000                                              \nREMARK   3    B23 (A**2) : 0.00000                                              \nREMARK   3                                                                      \nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \nREMARK   3   ESD FROM LUZZATI PLOT        (A) : 0.37                            \nREMARK   3   ESD FROM SIGMAA              (A) : 0.31                            \nREMARK   3   LOW RESOLUTION CUTOFF        (A) : 5.00                            \nREMARK   3                                                                      \nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : 0.47                            \nREMARK   3   ESD FROM C-V SIGMAA          (A) : 0.23                            \nREMARK   3                                                                      \nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \nREMARK   3   BOND LENGTHS                 (A) : 0.008                           \nREMARK   3   BOND ANGLES            (DEGREES) : 1.200                           \nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : 18.80                           \nREMARK   3   IMPROPER ANGLES        (DEGREES) : 0.710                           \nREMARK   3                                                                      \nREMARK   3  ISOTROPIC THERMAL MODEL : RESTRAINED                                \nREMARK   3                                                                      \nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \nREMARK   3   MAIN-CHAIN BOND              (A**2) : NULL  ; NULL                 \nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \nREMARK   3   SIDE-CHAIN BOND              (A**2) : NULL  ; NULL                 \nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : NULL  ; NULL                 \nREMARK   3                                                                      \nREMARK   3  BULK SOLVENT MODELING.                                              \nREMARK   3   METHOD USED : FLAT MODEL                                           \nREMARK   3   KSOL        : 0.38                                                 \nREMARK   3   BSOL        : 46.24                                                \nREMARK   3                                                                      \nREMARK   3  NCS MODEL : NULL                                                    \nREMARK   3                                                                      \nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \nREMARK   3                                                                      \nREMARK   3  PARAMETER FILE  1  : PROTEIN_REP.PARAM                              \nREMARK   3  PARAMETER FILE  2  : CARBOHYDRATE.PARAM                             \nREMARK   3  PARAMETER FILE  3  : WATER.PARAM                                    \nREMARK   3  PARAMETER FILE  4  : ION.PARAM                                      \nREMARK   3  PARAMETER FILE  5  : NULL                                           \nREMARK   3  TOPOLOGY FILE  1   : PROTEIN.TOP                                    \nREMARK   3  TOPOLOGY FILE  2   : NULL                                           \nREMARK   3  TOPOLOGY FILE  3   : NULL                                           \nREMARK   3  TOPOLOGY FILE  4   : NULL                                           \nREMARK   3  TOPOLOGY FILE  5   : NULL                                           \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \nREMARK   4                                                                      \nREMARK   4 2FRH COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 23-JAN-06.                  \nREMARK 100 THE DEPOSITION ID IS D_1000036223.                                   \nREMARK 200                                                                      \nREMARK 200 EXPERIMENTAL DETAILS                                                 \nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \nREMARK 200  DATE OF DATA COLLECTION        : 01-SEP-01                          \nREMARK 200  TEMPERATURE           (KELVIN) : 103.0                              \nREMARK 200  PH                             : 6.50                               \nREMARK 200  NUMBER OF CRYSTALS USED        : 3                                  \nREMARK 200                                                                      \nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \nREMARK 200  RADIATION SOURCE               : CHESS                              \nREMARK 200  BEAMLINE                       : A1                                 \nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \nREMARK 200  WAVELENGTH OR RANGE        (A) : NULL                               \nREMARK 200  MONOCHROMATOR                  : NULL                               \nREMARK 200  OPTICS                         : NULL                               \nREMARK 200                                                                      \nREMARK 200  DETECTOR TYPE                  : CCD                                \nREMARK 200  DETECTOR MANUFACTURER          : PRINCETON 2K                       \nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : DENZO                              \nREMARK 200  DATA SCALING SOFTWARE          : SCALEPACK                          \nREMARK 200                                                                      \nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 10107                              \nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.500                              \nREMARK 200  RESOLUTION RANGE LOW       (A) : 20.000                             \nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : 0.000                              \nREMARK 200                                                                      \nREMARK 200 OVERALL.                                                             \nREMARK 200  COMPLETENESS FOR RANGE     (%) : 89.7                               \nREMARK 200  DATA REDUNDANCY                : 6.000                              \nREMARK 200  R MERGE                    (I) : 0.07200                            \nREMARK 200  R SYM                      (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 33.0000                            \nREMARK 200                                                                      \nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \nREMARK 200                                                                      \nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MIR                          \nREMARK 200 SOFTWARE USED: SOLVE                                                 \nREMARK 200 STARTING MODEL: NULL                                                 \nREMARK 200                                                                      \nREMARK 200 REMARK: NULL                                                         \nREMARK 280                                                                      \nREMARK 280 CRYSTAL                                                              \nREMARK 280 SOLVENT CONTENT, VS   (%): 49.44                                     \nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.43                     \nREMARK 280                                                                      \nREMARK 280 CRYSTALLIZATION CONDITIONS: PEG 4000, PH 6.5, VAPOR DIFFUSION,       \nREMARK 280  HANGING DROP AT 277K, PH 6.50                                       \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 61 2 2                         \nREMARK 290                                                                      \nREMARK 290      SYMOP   SYMMETRY                                                \nREMARK 290     NNNMMM   OPERATOR                                                \nREMARK 290       1555   X,Y,Z                                                   \nREMARK 290       2555   -Y,X-Y,Z+1/3                                            \nREMARK 290       3555   -X+Y,-X,Z+2/3                                           \nREMARK 290       4555   -X,-Y,Z+1/2                                             \nREMARK 290       5555   Y,-X+Y,Z+5/6                                            \nREMARK 290       6555   X-Y,X,Z+1/6                                             \nREMARK 290       7555   Y,X,-Z+1/3                                              \nREMARK 290       8555   X-Y,-Y,-Z                                               \nREMARK 290       9555   -X,-X+Y,-Z+2/3                                          \nREMARK 290      10555   -Y,-X,-Z+5/6                                            \nREMARK 290      11555   -X+Y,Y,-Z+1/2                                           \nREMARK 290      12555   X,X-Y,-Z+1/6                                            \nREMARK 290                                                                      \nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \nREMARK 290           MMM -> TRANSLATION VECTOR                                  \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \nREMARK 290 RELATED MOLECULES.                                                   \nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       79.09633            \nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000      158.19267            \nREMARK 290   SMTRY1   4 -1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000      118.64450            \nREMARK 290   SMTRY1   5  0.500000  0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   5 -0.866025  0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000      197.74083            \nREMARK 290   SMTRY1   6  0.500000 -0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   6  0.866025  0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       39.54817            \nREMARK 290   SMTRY1   7 -0.500000  0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   7  0.866025  0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000       79.09633            \nREMARK 290   SMTRY1   8  1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   8  0.000000 -1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000        0.00000            \nREMARK 290   SMTRY1   9 -0.500000 -0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   9 -0.866025  0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   9  0.000000  0.000000 -1.000000      158.19267            \nREMARK 290   SMTRY1  10  0.500000 -0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2  10 -0.866025 -0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3  10  0.000000  0.000000 -1.000000      197.74083            \nREMARK 290   SMTRY1  11 -1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2  11  0.000000  1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3  11  0.000000  0.000000 -1.000000      118.64450            \nREMARK 290   SMTRY1  12  0.500000  0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2  12  0.866025 -0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3  12  0.000000  0.000000 -1.000000       39.54817            \nREMARK 290                                                                      \nREMARK 290 REMARK: NULL                                                         \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1                                                       \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \nREMARK 350 SOFTWARE USED: PISA                                                  \nREMARK 350 TOTAL BURIED SURFACE AREA: 3330 ANGSTROM**2                          \nREMARK 350 SURFACE AREA OF THE COMPLEX: 14890 ANGSTROM**2                       \nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -38.0 KCAL/MOL                        \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    HIS A 100      130.63     80.64                                   \nREMARK 500    LYS A 128      -70.29    -46.76                                   \nREMARK 500    PHE A 130      -17.65     58.33                                   \nREMARK 500    SER A 131       84.00     28.82                                   \nREMARK 500    ASN A 146       67.01   -108.34                                   \nREMARK 500    LYS A 163      131.53     69.07                                   \nREMARK 500    ASP A 181      -76.75    -59.17                                   \nREMARK 500    LYS A 182      133.82     68.03                                   \nREMARK 500    ARG A 184     -148.33    -77.71                                   \nREMARK 500    HIS A 187     -115.68   -175.61                                   \nREMARK 500    ASP A 188      -10.75     48.88                                   \nREMARK 500    GLU A 189        4.61     53.53                                   \nREMARK 500    THR A 191       70.67     25.83                                   \nREMARK 500    VAL A 192      147.17     64.23                                   \nREMARK 500    ASN A 197     -105.04    -49.95                                   \nREMARK 500    ALA A 198      -34.37   -145.35                                   \nREMARK 500    ILE A 222      -82.74    -30.86                                   \nREMARK 500    GLU A 223       76.55     33.15                                   \nREMARK 500    HIS B 100      131.13     81.00                                   \nREMARK 500    LYS B 128      -70.06    -48.11                                   \nREMARK 500    PHE B 130      -17.28     58.34                                   \nREMARK 500    SER B 131       83.52     28.50                                   \nREMARK 500    ASN B 146       67.39   -107.99                                   \nREMARK 500    LYS B 163      131.75     68.96                                   \nREMARK 500    ASP B 181      -77.72    -58.87                                   \nREMARK 500    LYS B 182      160.80     61.99                                   \nREMARK 500    LYS B 183        8.98   -169.49                                   \nREMARK 500    ARG B 184     -166.96     54.79                                   \nREMARK 500    ASN B 185       92.68     35.75                                   \nREMARK 500    HIS B 187      160.49     42.71                                   \nREMARK 500    ASP B 188     -167.78     40.00                                   \nREMARK 500    GLU B 189     -149.55     57.22                                   \nREMARK 500    ARG B 190     -152.38    -81.58                                   \nREMARK 500    VAL B 192      159.36     52.01                                   \nREMARK 500    ALA B 198      -15.73    -49.42                                   \nREMARK 500    GLU B 221     -102.48    -74.33                                   \nREMARK 500    ILE B 222      -68.05    -22.56                                   \nREMARK 500    GLU B 223       43.28     31.58                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 620                                                                      \nREMARK 620 METAL COORDINATION                                                   \nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 501  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASN A 107   OD1                                                    \nREMARK 620 2 ASN A 107   OD1  75.7                                              \nREMARK 620 3 ASP A 108   OD1  92.2 142.4                                        \nREMARK 620 4 ASP A 108   OD1 158.4  87.9  92.5                                  \nREMARK 620 5 GLU A 111   OE1  94.4  60.4 157.1  89.4                            \nREMARK 620 6 GLU A 111   OE2  97.3 106.0 110.8 100.7  46.6                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA B 601  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASN B 107   OD1                                                    \nREMARK 620 2 ASN B 107   OD1  78.7                                              \nREMARK 620 3 ASP B 108   OD1  83.2 157.3                                        \nREMARK 620 4 ASP B 108   OD1 146.4  97.8  89.7                                  \nREMARK 620 5 GLU B 111   OE1  59.3  97.1  85.1 152.9                            \nREMARK 620 6 GLU B 111   OE2 104.4 102.5  95.1 108.9  45.4                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 800                                                                      \nREMARK 800 SITE                                                                 \nREMARK 800 SITE_IDENTIFIER: AC1                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 501                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC2                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA B 601                  \nREMARK 900                                                                      \nREMARK 900 RELATED ENTRIES                                                      \nREMARK 900 RELATED ID: 2FNP   RELATED DB: PDB                                   \nREMARK 900 THE SAME PROTEIN IS CRYSTALLIZED UNDER DIFFERENT CONDITION           \nDBREF  2FRH A  102   224  UNP    Q7A1N5   SARA_STAAW       1    123             \nDBREF  2FRH B  102   224  UNP    Q7A1N5   SARA_STAAW       1    123             \nSEQADV 2FRH GLY A   98  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH SER A   99  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH HIS A  100  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH MET A  101  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH GLY B   98  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH SER B   99  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH HIS B  100  UNP  Q7A1N5              INSERTION                      \nSEQADV 2FRH MET B  101  UNP  Q7A1N5              INSERTION                      \nSEQRES   1 A  127  GLY SER HIS MET ALA ILE THR LYS ILE ASN ASP CYS PHE          \nSEQRES   2 A  127  GLU LEU LEU SER MET VAL THR TYR ALA ASP LYS LEU LYS          \nSEQRES   3 A  127  SER LEU ILE LYS LYS GLU PHE SER ILE SER PHE GLU GLU          \nSEQRES   4 A  127  PHE ALA VAL LEU THR TYR ILE SER GLU ASN LYS GLU LYS          \nSEQRES   5 A  127  GLU TYR TYR LEU LYS ASP ILE ILE ASN HIS LEU ASN TYR          \nSEQRES   6 A  127  LYS GLN PRO GLN VAL VAL LYS ALA VAL LYS ILE LEU SER          \nSEQRES   7 A  127  GLN GLU ASP TYR PHE ASP LYS LYS ARG ASN GLU HIS ASP          \nSEQRES   8 A  127  GLU ARG THR VAL LEU ILE LEU VAL ASN ALA GLN GLN ARG          \nSEQRES   9 A  127  LYS LYS ILE GLU SER LEU LEU SER ARG VAL ASN LYS ARG          \nSEQRES  10 A  127  ILE THR GLU ALA ASN ASN GLU ILE GLU LEU                      \nSEQRES   1 B  127  GLY SER HIS MET ALA ILE THR LYS ILE ASN ASP CYS PHE          \nSEQRES   2 B  127  GLU LEU LEU SER MET VAL THR TYR ALA ASP LYS LEU LYS          \nSEQRES   3 B  127  SER LEU ILE LYS LYS GLU PHE SER ILE SER PHE GLU GLU          \nSEQRES   4 B  127  PHE ALA VAL LEU THR TYR ILE SER GLU ASN LYS GLU LYS          \nSEQRES   5 B  127  GLU TYR TYR LEU LYS ASP ILE ILE ASN HIS LEU ASN TYR          \nSEQRES   6 B  127  LYS GLN PRO GLN VAL VAL LYS ALA VAL LYS ILE LEU SER          \nSEQRES   7 B  127  GLN GLU ASP TYR PHE ASP LYS LYS ARG ASN GLU HIS ASP          \nSEQRES   8 B  127  GLU ARG THR VAL LEU ILE LEU VAL ASN ALA GLN GLN ARG          \nSEQRES   9 B  127  LYS LYS ILE GLU SER LEU LEU SER ARG VAL ASN LYS ARG          \nSEQRES  10 B  127  ILE THR GLU ALA ASN ASN GLU ILE GLU LEU                      \nHET     CA  A 501       1                                                       \nHET     CA  B 601       1                                                       \nHETNAM      CA CALCIUM ION                                                      \nFORMUL   3   CA    2(CA 2+)                                                     \nHELIX    1   1 ASP A  108  GLU A  129  1                                  22    \nHELIX    2   2 SER A  133  ASN A  146  1                                  14    \nHELIX    3   3 LEU A  153  LEU A  160  1                                   8    \nHELIX    4   4 TYR A  162  GLU A  177  1                                  16    \nHELIX    5   5 ALA A  198  ILE A  222  1                                  25    \nHELIX    6   6 ASP B  108  GLU B  129  1                                  22    \nHELIX    7   7 SER B  133  ASN B  146  1                                  14    \nHELIX    8   8 TYR B  152  LEU B  160  1                                   9    \nHELIX    9   9 TYR B  162  GLU B  177  1                                  16    \nHELIX   10  10 GLN B  199  GLU B  221  1                                  23    \nSHEET    1   A 2 GLU A 150  TYR A 152  0                                        \nSHEET    2   A 2 LEU A 193  LEU A 195 -1  O  ILE A 194   N  TYR A 151           \nSHEET    1   B 2 GLU B 150  TYR B 151  0                                        \nSHEET    2   B 2 ILE B 194  LEU B 195 -1  O  ILE B 194   N  TYR B 151           \nLINK         OD1 ASN A 107                CA    CA A 501     1555   1555  2.59  \nLINK         OD1 ASN A 107                CA    CA A 501     7555   1555  2.79  \nLINK         OD1 ASP A 108                CA    CA A 501     1555   1555  2.76  \nLINK         OD1 ASP A 108                CA    CA A 501     7555   1555  2.76  \nLINK         OE1 GLU A 111                CA    CA A 501     7555   1555  2.91  \nLINK         OE2 GLU A 111                CA    CA A 501     7555   1555  2.57  \nLINK         OD1 ASN B 107                CA    CA B 601     1555   1555  2.89  \nLINK         OD1 ASN B 107                CA    CA B 601    12555   1555  2.47  \nLINK         OD1 ASP B 108                CA    CA B 601     1555   1555  2.96  \nLINK         OD1 ASP B 108                CA    CA B 601    12555   1555  2.67  \nLINK         OE1 GLU B 111                CA    CA B 601     1555   1555  2.99  \nLINK         OE2 GLU B 111                CA    CA B 601     1555   1555  2.59  \nSITE     1 AC1  4 ASN A 107  ASP A 108  GLU A 111  HIS B 100                    \nSITE     1 AC2  3 ASN B 107  ASP B 108  GLU B 111                               \nCRYST1   65.338   65.338  237.289  90.00  90.00 120.00 P 61 2 2     24          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      0.015305  0.008836  0.000000        0.00000                         \nSCALE2      0.000000  0.017673  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  0.004214        0.00000                         \nATOM      1  N   GLY A  98       7.842   9.976  27.496  1.00 69.73           N  \nATOM      2  CA  GLY A  98       7.974  10.959  26.375  1.00 69.80           C  \nATOM      3  C   GLY A  98       7.656  10.358  25.018  1.00 67.79           C  \nATOM      4  O   GLY A  98       6.855  10.893  24.254  1.00 70.31           O  \nATOM      5  N   SER A  99       8.299   9.240  24.713  1.00 63.38           N  \nATOM      6  CA  SER A  99       8.071   8.554  23.459  1.00 57.39           C  \nATOM      7  C   SER A  99       7.833   7.094  23.783  1.00 53.32           C  \nATOM      8  O   SER A  99       8.488   6.544  24.661  1.00 55.60           O  \nATOM      9  CB  SER A  99       9.291   8.706  22.542  1.00 58.07           C  \nATOM     10  OG  SER A  99      10.472   8.252  23.179  1.00 56.71           O  \nATOM     11  N   HIS A 100       6.890   6.474  23.090  1.00 46.90           N  \nATOM     12  CA  HIS A 100       6.581   5.056  23.291  1.00 43.07           C  \nATOM     13  C   HIS A 100       5.693   4.756  24.478  1.00 40.25           C  \nATOM     14  O   HIS A 100       5.922   5.216  25.585  1.00 41.73           O  \nATOM     15  CB  HIS A 100       7.856   4.225  23.420  1.00 41.97           C  \nATOM     16  CG  HIS A 100       8.588   4.031  22.131  1.00 43.04           C  \nATOM     17  ND1 HIS A 100       9.911   4.381  21.966  1.00 43.90           N  \nATOM     18  CD2 HIS A 100       8.195   3.479  20.958  1.00 43.32           C  \nATOM     19  CE1 HIS A 100      10.301   4.051  20.746  1.00 43.72           C  \nATOM     20  NE2 HIS A 100       9.279   3.500  20.114  1.00 43.11           N  \nATOM     21  N   MET A 101       4.661   3.970  24.226  1.00 36.45           N  \nATOM     22  CA  MET A 101       3.746   3.579  25.270  1.00 32.88           C  \nATOM     23  C   MET A 101       4.508   2.857  26.405  1.00 33.98           C  \nATOM     24  O   MET A 101       5.247   1.894  26.160  1.00 33.92           O  \nATOM     25  CB  MET A 101       2.673   2.656  24.693  1.00 28.83           C  \nATOM     26  CG  MET A 101       1.744   2.129  25.745  1.00 29.39           C  \nATOM     27  SD  MET A 101       0.597   0.884  25.175  1.00 29.97           S  \nATOM     28  CE  MET A 101       1.802  -0.621  25.102  1.00 28.17           C  \nATOM     29  N   ALA A 102       4.329   3.341  27.631  1.00 34.54           N  \nATOM     30  CA  ALA A 102       4.945   2.738  28.803  1.00 32.94           C  \nATOM     31  C   ALA A 102       4.311   1.373  28.997  1.00 34.19           C  \nATOM     32  O   ALA A 102       3.186   1.122  28.552  1.00 34.68           O  \nATOM     33  CB  ALA A 102       4.693   3.598  30.053  1.00 32.20           C  \nATOM     34  N   ILE A 103       5.029   0.481  29.653  1.00 35.59           N  \nATOM     35  CA  ILE A 103       4.498  -0.841  29.918  1.00 37.00           C  \nATOM     36  C   ILE A 103       3.747  -0.780  31.244  1.00 37.50           C  \nATOM     37  O   ILE A 103       4.254  -0.267  32.233  1.00 39.97           O  \nATOM     38  CB  ILE A 103       5.625  -1.887  30.014  1.00 36.83           C  \nATOM     39  CG1 ILE A 103       6.382  -1.948  28.677  1.00 37.04           C  \nATOM     40  CG2 ILE A 103       5.062  -3.239  30.438  1.00 35.99           C  \nATOM     41  CD1 ILE A 103       5.518  -2.126  27.448  1.00 36.80           C  \nATOM     42  N   THR A 104       2.528  -1.291  31.252  1.00 37.68           N  \nATOM     43  CA  THR A 104       1.720  -1.304  32.448  1.00 38.12           C  \nATOM     44  C   THR A 104       2.192  -2.443  33.309  1.00 37.58           C  \nATOM     45  O   THR A 104       2.442  -3.534  32.806  1.00 35.62           O  \nATOM     46  CB  THR A 104       0.244  -1.546  32.127  1.00 38.96           C  \nATOM     47  OG1 THR A 104      -0.195  -0.568  31.183  1.00 44.36           O  \nATOM     48  CG2 THR A 104      -0.602  -1.456  33.385  1.00 37.54           C  \nATOM     49  N   LYS A 105       2.297  -2.170  34.605  1.00 37.46           N  \nATOM     50  CA  LYS A 105       2.716  -3.147  35.585  1.00 36.66           C  \nATOM     51  C   LYS A 105       1.599  -4.139  35.878  1.00 36.13           C  \nATOM     52  O   LYS A 105       0.408  -3.801  35.787  1.00 37.72           O  \nATOM     53  CB  LYS A 105       3.115  -2.442  36.890  1.00 36.39           C  \nATOM     54  CG  LYS A 105       4.432  -1.704  36.790  1.00 40.08           C  \nATOM     55  CD  LYS A 105       4.215  -0.186  36.887  1.00 48.35           C  \nATOM     56  CE  LYS A 105       3.900   0.270  38.329  1.00 52.37           C  \nATOM     57  NZ  LYS A 105       3.442   1.692  38.394  1.00 54.85           N  \nATOM     58  N   ILE A 106       1.988  -5.360  36.230  1.00 33.86           N  \nATOM     59  CA  ILE A 106       1.024  -6.384  36.589  1.00 31.21           C  \nATOM     60  C   ILE A 106       1.127  -6.442  38.106  1.00 30.99           C  \nATOM     61  O   ILE A 106       2.083  -5.921  38.677  1.00 29.91           O  \nATOM     62  CB  ILE A 106       1.357  -7.764  35.894  1.00 30.53           C  \nATOM     63  CG1 ILE A 106       2.794  -8.209  36.172  1.00 28.14           C  \nATOM     64  CG2 ILE A 106       1.169  -7.647  34.401  1.00 28.04           C  \nATOM     65  CD1 ILE A 106       3.002  -8.828  37.511  1.00 27.99           C  \nATOM     66  N   ASN A 107       0.148  -7.030  38.777  1.00 31.36           N  \nATOM     67  CA  ASN A 107       0.218  -7.089  40.233  1.00 31.42           C  \nATOM     68  C   ASN A 107      -0.131  -8.468  40.805  1.00 32.85           C  \nATOM     69  O   ASN A 107      -0.354  -8.602  42.002  1.00 33.71           O  \nATOM     70  CB  ASN A 107      -0.683  -6.025  40.831  1.00 32.17           C  \nATOM     71  CG  ASN A 107      -2.122  -6.415  40.768  1.00 32.53           C  \nATOM     72  OD1 ASN A 107      -2.450  -7.366  40.086  1.00 34.64           O  \nATOM     73  ND2 ASN A 107      -2.997  -5.689  41.475  1.00 30.64           N  \nATOM     74  N   ASP A 108      -0.187  -9.491  39.958  1.00 32.36           N  \nATOM     75  CA  ASP A 108      -0.448 -10.833  40.468  1.00 33.18           C  \nATOM     76  C   ASP A 108       0.079 -11.893  39.520  1.00 31.48           C  \nATOM     77  O   ASP A 108       0.378 -11.598  38.363  1.00 30.62           O  \nATOM     78  CB  ASP A 108      -1.943 -11.034  40.795  1.00 35.12           C  \nATOM     79  CG  ASP A 108      -2.799 -11.153  39.574  1.00 36.82           C  \nATOM     80  OD1 ASP A 108      -3.987 -10.878  39.708  1.00 38.28           O  \nATOM     81  OD2 ASP A 108      -2.312 -11.543  38.505  1.00 37.80           O  \nATOM     82  N   CYS A 109       0.198 -13.122  40.020  1.00 30.53           N  \nATOM     83  CA  CYS A 109       0.763 -14.241  39.256  1.00 28.29           C  \nATOM     84  C   CYS A 109       0.176 -14.530  37.897  1.00 27.83           C  \nATOM     85  O   CYS A 109       0.913 -14.737  36.940  1.00 29.81           O  \nATOM     86  CB  CYS A 109       0.732 -15.513  40.089  1.00 28.48           C  \nATOM     87  SG  CYS A 109       1.564 -15.349  41.683  1.00 29.15           S  \nATOM     88  N   PHE A 110      -1.143 -14.561  37.799  1.00 27.32           N  \nATOM     89  CA  PHE A 110      -1.773 -14.843  36.517  1.00 28.09           C  \nATOM     90  C   PHE A 110      -1.434 -13.782  35.465  1.00 28.13           C  \nATOM     91  O   PHE A 110      -1.270 -14.090  34.284  1.00 27.68           O  \nATOM     92  CB  PHE A 110      -3.294 -14.952  36.679  1.00 30.33           C  \nATOM     93  CG  PHE A 110      -3.983 -15.518  35.465  1.00 34.10           C  \nATOM     94  CD1 PHE A 110      -4.619 -14.682  34.555  1.00 35.04           C  \nATOM     95  CD2 PHE A 110      -3.938 -16.886  35.203  1.00 35.64           C  \nATOM     96  CE1 PHE A 110      -5.198 -15.192  33.401  1.00 35.95           C  \nATOM     97  CE2 PHE A 110      -4.513 -17.410  34.049  1.00 36.75           C  \nATOM     98  CZ  PHE A 110      -5.146 -16.560  33.142  1.00 35.74           C  \nATOM     99  N   GLU A 111      -1.313 -12.532  35.886  1.00 27.87           N  \nATOM    100  CA  GLU A 111      -0.987 -11.478  34.932  1.00 27.92           C  \nATOM    101  C   GLU A 111       0.456 -11.594  34.515  1.00 27.82           C  \nATOM    102  O   GLU A 111       0.799 -11.349  33.362  1.00 30.62           O  \nATOM    103  CB  GLU A 111      -1.278 -10.114  35.540  1.00 26.98           C  \nATOM    104  CG  GLU A 111      -2.646 -10.127  36.097  1.00 27.78           C  \nATOM    105  CD  GLU A 111      -3.096  -8.827  36.634  1.00 27.38           C  \nATOM    106  OE1 GLU A 111      -2.326  -8.144  37.329  1.00 25.53           O  \nATOM    107  OE2 GLU A 111      -4.263  -8.496  36.371  1.00 33.08           O  \nATOM    108  N   LEU A 112       1.297 -11.984  35.460  1.00 26.90           N  \nATOM    109  CA  LEU A 112       2.706 -12.158  35.182  1.00 27.09           C  \nATOM    110  C   LEU A 112       2.832 -13.246  34.113  1.00 28.24           C  \nATOM    111  O   LEU A 112       3.468 -13.061  33.060  1.00 29.56           O  \nATOM    112  CB  LEU A 112       3.433 -12.616  36.442  1.00 26.40           C  \nATOM    113  CG  LEU A 112       4.929 -12.290  36.536  1.00 27.67           C  \nATOM    114  CD1 LEU A 112       5.606 -13.405  37.296  1.00 24.89           C  \nATOM    115  CD2 LEU A 112       5.570 -12.109  35.169  1.00 28.01           C  \nATOM    116  N   LEU A 113       2.211 -14.384  34.393  1.00 28.64           N  \nATOM    117  CA  LEU A 113       2.261 -15.509  33.479  1.00 30.22           C  \nATOM    118  C   LEU A 113       1.823 -15.111  32.073  1.00 30.07           C  \nATOM    119  O   LEU A 113       2.504 -15.441  31.087  1.00 31.28           O  \nATOM    120  CB  LEU A 113       1.396 -16.653  34.012  1.00 33.09           C  \nATOM    121  CG  LEU A 113       1.281 -17.880  33.103  1.00 33.77           C  \nATOM    122  CD1 LEU A 113       2.659 -18.400  32.750  1.00 34.69           C  \nATOM    123  CD2 LEU A 113       0.458 -18.948  33.823  1.00 34.13           C  \nATOM    124  N   SER A 114       0.712 -14.384  31.976  1.00 27.82           N  \nATOM    125  CA  SER A 114       0.226 -13.960  30.666  1.00 27.40           C  \nATOM    126  C   SER A 114       1.228 -13.069  29.992  1.00 28.03           C  \nATOM    127  O   SER A 114       1.514 -13.240  28.794  1.00 30.69           O  \nATOM    128  CB  SER A 114      -1.091 -13.204  30.772  1.00 26.21           C  \nATOM    129  OG  SER A 114      -2.056 -13.984  31.451  1.00 31.97           O  \nATOM    130  N   MET A 115       1.765 -12.108  30.737  1.00 27.88           N  \nATOM    131  CA  MET A 115       2.731 -11.190  30.141  1.00 27.92           C  \nATOM    132  C   MET A 115       3.913 -11.966  29.556  1.00 28.03           C  \nATOM    133  O   MET A 115       4.329 -11.716  28.428  1.00 27.13           O  \nATOM    134  CB  MET A 115       3.221 -10.176  31.170  1.00 29.42           C  \nATOM    135  CG  MET A 115       4.182  -9.172  30.577  1.00 32.13           C  \nATOM    136  SD  MET A 115       4.692  -7.849  31.710  1.00 36.32           S  \nATOM    137  CE  MET A 115       3.446  -6.563  31.322  1.00 31.49           C  \nATOM    138  N   VAL A 116       4.431 -12.921  30.324  1.00 28.59           N  \nATOM    139  CA  VAL A 116       5.565 -13.727  29.880  1.00 28.72           C  \nATOM    140  C   VAL A 116       5.192 -14.567  28.657  1.00 31.19           C  \nATOM    141  O   VAL A 116       5.953 -14.635  27.681  1.00 31.53           O  \nATOM    142  CB  VAL A 116       6.048 -14.648  31.004  1.00 28.09           C  \nATOM    143  CG1 VAL A 116       7.003 -15.672  30.450  1.00 28.30           C  \nATOM    144  CG2 VAL A 116       6.702 -13.808  32.119  1.00 23.75           C  \nATOM    145  N   THR A 117       4.015 -15.190  28.705  1.00 32.42           N  \nATOM    146  CA  THR A 117       3.561 -15.994  27.579  1.00 33.88           C  \nATOM    147  C   THR A 117       3.537 -15.123  26.339  1.00 33.25           C  \nATOM    148  O   THR A 117       4.091 -15.487  25.309  1.00 34.96           O  \nATOM    149  CB  THR A 117       2.154 -16.554  27.818  1.00 34.93           C  \nATOM    150  OG1 THR A 117       2.157 -17.357  29.004  1.00 36.82           O  \nATOM    151  CG2 THR A 117       1.713 -17.407  26.625  1.00 33.69           C  \nATOM    152  N   TYR A 118       2.903 -13.961  26.437  1.00 33.50           N  \nATOM    153  CA  TYR A 118       2.836 -13.053  25.302  1.00 35.17           C  \nATOM    154  C   TYR A 118       4.234 -12.572  24.851  1.00 36.84           C  \nATOM    155  O   TYR A 118       4.519 -12.500  23.654  1.00 38.19           O  \nATOM    156  CB  TYR A 118       1.936 -11.865  25.669  1.00 33.45           C  \nATOM    157  CG  TYR A 118       1.952 -10.714  24.693  1.00 30.33           C  \nATOM    158  CD1 TYR A 118       1.521 -10.869  23.375  1.00 31.08           C  \nATOM    159  CD2 TYR A 118       2.401  -9.471  25.087  1.00 29.66           C  \nATOM    160  CE1 TYR A 118       1.542  -9.794  22.468  1.00 31.13           C  \nATOM    161  CE2 TYR A 118       2.433  -8.385  24.198  1.00 29.99           C  \nATOM    162  CZ  TYR A 118       2.004  -8.551  22.891  1.00 30.53           C  \nATOM    163  OH  TYR A 118       2.059  -7.475  22.019  1.00 31.14           O  \nATOM    164  N   ALA A 119       5.104 -12.249  25.803  1.00 37.94           N  \nATOM    165  CA  ALA A 119       6.440 -11.780  25.441  1.00 39.19           C  \nATOM    166  C   ALA A 119       7.277 -12.891  24.818  1.00 39.54           C  \nATOM    167  O   ALA A 119       8.028 -12.639  23.883  1.00 41.18           O  \nATOM    168  CB  ALA A 119       7.160 -11.198  26.665  1.00 38.29           C  \nATOM    169  N   ASP A 120       7.143 -14.115  25.324  1.00 39.63           N  \nATOM    170  CA  ASP A 120       7.909 -15.228  24.783  1.00 40.95           C  \nATOM    171  C   ASP A 120       7.522 -15.490  23.341  1.00 39.74           C  \nATOM    172  O   ASP A 120       8.381 -15.736  22.494  1.00 39.51           O  \nATOM    173  CB  ASP A 120       7.699 -16.505  25.610  1.00 46.17           C  \nATOM    174  CG  ASP A 120       8.499 -16.511  26.914  1.00 49.81           C  \nATOM    175  OD1 ASP A 120       9.469 -15.733  27.050  1.00 52.48           O  \nATOM    176  OD2 ASP A 120       8.158 -17.313  27.805  1.00 50.63           O  \nATOM    177  N   LYS A 121       6.227 -15.444  23.055  1.00 39.42           N  \nATOM    178  CA  LYS A 121       5.778 -15.670  21.687  1.00 39.14           C  \nATOM    179  C   LYS A 121       6.371 -14.654  20.737  1.00 38.33           C  \nATOM    180  O   LYS A 121       6.894 -15.024  19.694  1.00 40.67           O  \nATOM    181  CB  LYS A 121       4.255 -15.653  21.601  1.00 40.36           C  \nATOM    182  CG  LYS A 121       3.631 -16.863  22.269  1.00 44.95           C  \nATOM    183  CD  LYS A 121       2.114 -16.845  22.186  1.00 50.57           C  \nATOM    184  CE  LYS A 121       1.527 -18.015  22.972  1.00 53.55           C  \nATOM    185  NZ  LYS A 121       0.059 -17.865  23.190  1.00 55.87           N  \nATOM    186  N   LEU A 122       6.318 -13.376  21.086  1.00 38.19           N  \nATOM    187  CA  LEU A 122       6.876 -12.376  20.193  1.00 38.33           C  \nATOM    188  C   LEU A 122       8.382 -12.549  20.056  1.00 38.55           C  \nATOM    189  O   LEU A 122       8.924 -12.328  18.983  1.00 38.33           O  \nATOM    190  CB  LEU A 122       6.539 -10.958  20.673  1.00 37.84           C  \nATOM    191  CG  LEU A 122       5.059 -10.555  20.664  1.00 37.14           C  \nATOM    192  CD1 LEU A 122       4.955  -9.083  21.018  1.00 38.72           C  \nATOM    193  CD2 LEU A 122       4.440 -10.795  19.294  1.00 36.47           C  \nATOM    194  N   LYS A 123       9.057 -12.953  21.127  1.00 40.77           N  \nATOM    195  CA  LYS A 123      10.503 -13.154  21.063  1.00 43.84           C  \nATOM    196  C   LYS A 123      10.877 -14.297  20.127  1.00 45.28           C  \nATOM    197  O   LYS A 123      11.737 -14.140  19.254  1.00 45.01           O  \nATOM    198  CB  LYS A 123      11.102 -13.463  22.441  1.00 43.94           C  \nATOM    199  CG  LYS A 123      11.147 -12.294  23.382  1.00 46.03           C  \nATOM    200  CD  LYS A 123      12.240 -12.474  24.447  1.00 46.71           C  \nATOM    201  CE  LYS A 123      12.054 -13.746  25.250  1.00 46.58           C  \nATOM    202  NZ  LYS A 123      13.093 -13.865  26.307  1.00 47.36           N  \nATOM    203  N   SER A 124      10.237 -15.447  20.313  1.00 47.45           N  \nATOM    204  CA  SER A 124      10.537 -16.614  19.496  1.00 51.18           C  \nATOM    205  C   SER A 124      10.262 -16.355  18.022  1.00 52.87           C  \nATOM    206  O   SER A 124      10.989 -16.849  17.150  1.00 55.23           O  \nATOM    207  CB  SER A 124       9.727 -17.812  19.965  1.00 51.39           C  \nATOM    208  OG  SER A 124       8.366 -17.651  19.615  1.00 54.26           O  \nATOM    209  N   LEU A 125       9.216 -15.579  17.739  1.00 54.19           N  \nATOM    210  CA  LEU A 125       8.879 -15.248  16.358  1.00 53.73           C  \nATOM    211  C   LEU A 125       9.983 -14.405  15.741  1.00 52.72           C  \nATOM    212  O   LEU A 125      10.513 -14.735  14.683  1.00 52.26           O  \nATOM    213  CB  LEU A 125       7.555 -14.490  16.294  1.00 54.76           C  \nATOM    214  CG  LEU A 125       6.321 -15.384  16.184  1.00 56.55           C  \nATOM    215  CD1 LEU A 125       5.056 -14.590  16.487  1.00 57.89           C  \nATOM    216  CD2 LEU A 125       6.268 -15.986  14.779  1.00 57.44           C  \nATOM    217  N   ILE A 126      10.325 -13.311  16.403  1.00 53.08           N  \nATOM    218  CA  ILE A 126      11.372 -12.436  15.901  1.00 55.08           C  \nATOM    219  C   ILE A 126      12.641 -13.242  15.640  1.00 57.42           C  \nATOM    220  O   ILE A 126      13.363 -12.985  14.678  1.00 58.65           O  \nATOM    221  CB  ILE A 126      11.653 -11.298  16.895  1.00 54.25           C  \nATOM    222  CG1 ILE A 126      10.459 -10.339  16.894  1.00 54.46           C  \nATOM    223  CG2 ILE A 126      12.943 -10.592  16.547  1.00 54.05           C  \nATOM    224  CD1 ILE A 126      10.564  -9.209  17.903  1.00 56.43           C  \nATOM    225  N   LYS A 127      12.909 -14.223  16.494  1.00 60.36           N  \nATOM    226  CA  LYS A 127      14.084 -15.057  16.315  1.00 64.17           C  \nATOM    227  C   LYS A 127      13.864 -16.034  15.154  1.00 65.54           C  \nATOM    228  O   LYS A 127      14.668 -16.084  14.230  1.00 65.69           O  \nATOM    229  CB  LYS A 127      14.387 -15.807  17.612  1.00 66.16           C  \nATOM    230  CG  LYS A 127      14.672 -14.873  18.778  1.00 68.72           C  \nATOM    231  CD  LYS A 127      15.115 -15.612  20.037  1.00 70.55           C  \nATOM    232  CE  LYS A 127      15.433 -14.620  21.147  1.00 71.58           C  \nATOM    233  NZ  LYS A 127      16.008 -15.264  22.367  1.00 73.37           N  \nATOM    234  N   LYS A 128      12.776 -16.801  15.206  1.00 66.96           N  \nATOM    235  CA  LYS A 128      12.447 -17.758  14.151  1.00 68.54           C  \nATOM    236  C   LYS A 128      12.611 -17.045  12.812  1.00 67.07           C  \nATOM    237  O   LYS A 128      13.523 -17.334  12.050  1.00 66.93           O  \nATOM    238  CB  LYS A 128      11.009 -18.244  14.333  1.00 71.65           C  \nATOM    239  CG  LYS A 128      10.567 -19.322  13.360  1.00 76.53           C  \nATOM    240  CD  LYS A 128       9.172 -19.826  13.720  1.00 80.78           C  \nATOM    241  CE  LYS A 128       8.698 -20.927  12.774  1.00 82.85           C  \nATOM    242  NZ  LYS A 128       7.345 -21.452  13.140  1.00 86.41           N  \nATOM    243  N   GLU A 129      11.714 -16.113  12.527  1.00 65.60           N  \nATOM    244  CA  GLU A 129      11.807 -15.313  11.315  1.00 64.20           C  \nATOM    245  C   GLU A 129      12.941 -14.359  11.655  1.00 63.85           C  \nATOM    246  O   GLU A 129      13.398 -14.360  12.795  1.00 65.43           O  \nATOM    247  CB  GLU A 129      10.496 -14.558  11.104  1.00 64.39           C  \nATOM    248  CG  GLU A 129       9.344 -15.493  10.755  1.00 69.30           C  \nATOM    249  CD  GLU A 129       7.998 -14.806  10.751  1.00 71.83           C  \nATOM    250  OE1 GLU A 129       7.433 -14.624  11.846  1.00 74.57           O  \nATOM    251  OE2 GLU A 129       7.508 -14.446   9.655  1.00 74.49           O  \nATOM    252  N   PHE A 130      13.420 -13.565  10.705  1.00 63.67           N  \nATOM    253  CA  PHE A 130      14.514 -12.630  11.002  1.00 64.04           C  \nATOM    254  C   PHE A 130      15.784 -13.299  11.525  1.00 65.65           C  \nATOM    255  O   PHE A 130      16.854 -12.692  11.516  1.00 64.92           O  \nATOM    256  CB  PHE A 130      14.066 -11.596  12.038  1.00 61.98           C  \nATOM    257  CG  PHE A 130      12.922 -10.760  11.589  1.00 60.79           C  \nATOM    258  CD1 PHE A 130      13.092  -9.813  10.594  1.00 60.31           C  \nATOM    259  CD2 PHE A 130      11.656 -10.957  12.123  1.00 61.13           C  \nATOM    260  CE1 PHE A 130      12.016  -9.075  10.128  1.00 60.92           C  \nATOM    261  CE2 PHE A 130      10.572 -10.226  11.665  1.00 62.16           C  \nATOM    262  CZ  PHE A 130      10.750  -9.281  10.664  1.00 61.42           C  \nATOM    263  N   SER A 131      15.654 -14.540  11.990  1.00 69.04           N  \nATOM    264  CA  SER A 131      16.773 -15.303  12.536  1.00 71.18           C  \nATOM    265  C   SER A 131      17.810 -14.373  13.144  1.00 71.07           C  \nATOM    266  O   SER A 131      18.797 -14.002  12.500  1.00 70.85           O  \nATOM    267  CB  SER A 131      17.399 -16.201  11.455  1.00 71.33           C  \nATOM    268  OG  SER A 131      17.646 -15.484  10.261  1.00 74.50           O  \nATOM    269  N   ILE A 132      17.558 -13.998  14.395  1.00 70.68           N  \nATOM    270  CA  ILE A 132      18.421 -13.093  15.136  1.00 69.59           C  \nATOM    271  C   ILE A 132      18.135 -13.299  16.624  1.00 68.83           C  \nATOM    272  O   ILE A 132      16.988 -13.521  17.006  1.00 69.99           O  \nATOM    273  CB  ILE A 132      18.118 -11.623  14.737  1.00 70.11           C  \nATOM    274  CG1 ILE A 132      19.079 -10.670  15.438  1.00 71.74           C  \nATOM    275  CG2 ILE A 132      16.679 -11.266  15.099  1.00 68.82           C  \nATOM    276  CD1 ILE A 132      18.798  -9.212  15.131  1.00 72.84           C  \nATOM    277  N   SER A 133      19.169 -13.244  17.461  1.00 67.22           N  \nATOM    278  CA  SER A 133      18.980 -13.420  18.899  1.00 65.76           C  \nATOM    279  C   SER A 133      18.341 -12.160  19.496  1.00 65.62           C  \nATOM    280  O   SER A 133      18.391 -11.079  18.899  1.00 64.46           O  \nATOM    281  CB  SER A 133      20.319 -13.677  19.580  1.00 64.57           C  \nATOM    282  OG  SER A 133      21.109 -12.498  19.571  1.00 62.75           O  \nATOM    283  N   PHE A 134      17.752 -12.291  20.680  1.00 64.82           N  \nATOM    284  CA  PHE A 134      17.123 -11.131  21.279  1.00 64.88           C  \nATOM    285  C   PHE A 134      18.124 -10.056  21.689  1.00 64.23           C  \nATOM    286  O   PHE A 134      17.811  -8.866  21.640  1.00 64.45           O  \nATOM    287  CB  PHE A 134      16.248 -11.524  22.469  1.00 65.87           C  \nATOM    288  CG  PHE A 134      15.040 -10.652  22.615  1.00 66.33           C  \nATOM    289  CD1 PHE A 134      14.055 -10.655  21.629  1.00 66.62           C  \nATOM    290  CD2 PHE A 134      14.922  -9.773  23.683  1.00 65.14           C  \nATOM    291  CE1 PHE A 134      12.972  -9.792  21.698  1.00 66.81           C  \nATOM    292  CE2 PHE A 134      13.843  -8.902  23.766  1.00 66.26           C  \nATOM    293  CZ  PHE A 134      12.862  -8.910  22.767  1.00 67.04           C  \nATOM    294  N   GLU A 135      19.325 -10.464  22.088  1.00 63.98           N  \nATOM    295  CA  GLU A 135      20.340  -9.482  22.456  1.00 63.28           C  \nATOM    296  C   GLU A 135      20.698  -8.666  21.215  1.00 60.02           C  \nATOM    297  O   GLU A 135      20.930  -7.463  21.303  1.00 59.16           O  \nATOM    298  CB  GLU A 135      21.599 -10.154  23.034  1.00 66.31           C  \nATOM    299  CG  GLU A 135      21.720 -11.642  22.756  1.00 69.90           C  \nATOM    300  CD  GLU A 135      20.815 -12.481  23.642  1.00 70.70           C  \nATOM    301  OE1 GLU A 135      21.008 -12.468  24.876  1.00 70.45           O  \nATOM    302  OE2 GLU A 135      19.910 -13.152  23.103  1.00 73.09           O  \nATOM    303  N   GLU A 136      20.734  -9.314  20.056  1.00 57.02           N  \nATOM    304  CA  GLU A 136      21.033  -8.588  18.822  1.00 57.07           C  \nATOM    305  C   GLU A 136      19.897  -7.594  18.578  1.00 54.94           C  \nATOM    306  O   GLU A 136      20.108  -6.480  18.089  1.00 55.21           O  \nATOM    307  CB  GLU A 136      21.119  -9.537  17.626  1.00 59.14           C  \nATOM    308  CG  GLU A 136      22.275 -10.521  17.634  1.00 62.62           C  \nATOM    309  CD  GLU A 136      22.197 -11.499  16.465  1.00 64.31           C  \nATOM    310  OE1 GLU A 136      22.293 -11.055  15.302  1.00 64.27           O  \nATOM    311  OE2 GLU A 136      22.029 -12.711  16.707  1.00 66.36           O  \nATOM    312  N   PHE A 137      18.686  -8.027  18.921  1.00 52.16           N  \nATOM    313  CA  PHE A 137      17.477  -7.227  18.785  1.00 46.80           C  \nATOM    314  C   PHE A 137      17.562  -6.035  19.733  1.00 44.71           C  \nATOM    315  O   PHE A 137      17.320  -4.902  19.330  1.00 42.68           O  \nATOM    316  CB  PHE A 137      16.256  -8.095  19.122  1.00 46.05           C  \nATOM    317  CG  PHE A 137      14.951  -7.355  19.092  1.00 45.18           C  \nATOM    318  CD1 PHE A 137      14.400  -6.937  17.887  1.00 44.79           C  \nATOM    319  CD2 PHE A 137      14.271  -7.077  20.276  1.00 45.65           C  \nATOM    320  CE1 PHE A 137      13.185  -6.251  17.858  1.00 44.16           C  \nATOM    321  CE2 PHE A 137      13.059  -6.395  20.264  1.00 44.75           C  \nATOM    322  CZ  PHE A 137      12.514  -5.982  19.054  1.00 43.66           C  \nATOM    323  N   ALA A 138      17.924  -6.297  20.986  1.00 43.36           N  \nATOM    324  CA  ALA A 138      18.046  -5.247  21.990  1.00 45.79           C  \nATOM    325  C   ALA A 138      19.107  -4.207  21.629  1.00 46.65           C  \nATOM    326  O   ALA A 138      18.918  -3.007  21.864  1.00 47.62           O  \nATOM    327  CB  ALA A 138      18.362  -5.858  23.347  1.00 46.74           C  \nATOM    328  N   VAL A 139      20.225  -4.664  21.065  1.00 45.69           N  \nATOM    329  CA  VAL A 139      21.279  -3.743  20.677  1.00 44.05           C  \nATOM    330  C   VAL A 139      20.749  -2.914  19.520  1.00 43.20           C  \nATOM    331  O   VAL A 139      20.854  -1.682  19.511  1.00 40.51           O  \nATOM    332  CB  VAL A 139      22.570  -4.492  20.237  1.00 43.70           C  \nATOM    333  CG1 VAL A 139      23.528  -3.522  19.540  1.00 42.75           C  \nATOM    334  CG2 VAL A 139      23.251  -5.104  21.452  1.00 43.33           C  \nATOM    335  N   LEU A 140      20.168  -3.600  18.546  1.00 43.05           N  \nATOM    336  CA  LEU A 140      19.619  -2.904  17.402  1.00 44.44           C  \nATOM    337  C   LEU A 140      18.548  -1.873  17.754  1.00 44.82           C  \nATOM    338  O   LEU A 140      18.577  -0.765  17.208  1.00 46.24           O  \nATOM    339  CB  LEU A 140      19.061  -3.897  16.388  1.00 44.15           C  \nATOM    340  CG  LEU A 140      20.080  -4.732  15.620  1.00 44.03           C  \nATOM    341  CD1 LEU A 140      19.380  -5.397  14.440  1.00 42.43           C  \nATOM    342  CD2 LEU A 140      21.225  -3.850  15.132  1.00 44.11           C  \nATOM    343  N   THR A 141      17.614  -2.197  18.657  1.00 43.24           N  \nATOM    344  CA  THR A 141      16.588  -1.203  18.975  1.00 43.49           C  \nATOM    345  C   THR A 141      17.161  -0.094  19.864  1.00 44.43           C  \nATOM    346  O   THR A 141      16.708   1.050  19.820  1.00 43.00           O  \nATOM    347  CB  THR A 141      15.285  -1.830  19.615  1.00 42.01           C  \nATOM    348  OG1 THR A 141      15.331  -1.730  21.036  1.00 43.93           O  \nATOM    349  CG2 THR A 141      15.128  -3.283  19.226  1.00 40.36           C  \nATOM    350  N   TYR A 142      18.171  -0.419  20.660  1.00 47.53           N  \nATOM    351  CA  TYR A 142      18.792   0.598  21.500  1.00 51.48           C  \nATOM    352  C   TYR A 142      19.442   1.631  20.582  1.00 52.69           C  \nATOM    353  O   TYR A 142      19.326   2.840  20.800  1.00 53.30           O  \nATOM    354  CB  TYR A 142      19.873   0.001  22.396  1.00 53.78           C  \nATOM    355  CG  TYR A 142      20.610   1.067  23.161  1.00 57.48           C  \nATOM    356  CD1 TYR A 142      20.128   1.537  24.381  1.00 58.80           C  \nATOM    357  CD2 TYR A 142      21.744   1.681  22.620  1.00 58.80           C  \nATOM    358  CE1 TYR A 142      20.752   2.600  25.047  1.00 59.53           C  \nATOM    359  CE2 TYR A 142      22.373   2.741  23.272  1.00 59.62           C  \nATOM    360  CZ  TYR A 142      21.871   3.198  24.483  1.00 59.54           C  \nATOM    361  OH  TYR A 142      22.474   4.264  25.118  1.00 60.26           O  \nATOM    362  N   ILE A 143      20.137   1.141  19.560  1.00 54.56           N  \nATOM    363  CA  ILE A 143      20.802   2.009  18.598  1.00 56.31           C  \nATOM    364  C   ILE A 143      19.781   2.884  17.886  1.00 58.81           C  \nATOM    365  O   ILE A 143      20.015   4.077  17.680  1.00 59.77           O  \nATOM    366  CB  ILE A 143      21.596   1.179  17.551  1.00 55.14           C  \nATOM    367  CG1 ILE A 143      22.914   0.709  18.162  1.00 54.24           C  \nATOM    368  CG2 ILE A 143      21.854   1.995  16.299  1.00 54.90           C  \nATOM    369  CD1 ILE A 143      23.706  -0.211  17.264  1.00 56.55           C  \nATOM    370  N   SER A 144      18.642   2.293  17.526  1.00 60.71           N  \nATOM    371  CA  SER A 144      17.600   3.038  16.831  1.00 61.95           C  \nATOM    372  C   SER A 144      16.921   4.105  17.693  1.00 62.86           C  \nATOM    373  O   SER A 144      16.466   5.123  17.178  1.00 63.59           O  \nATOM    374  CB  SER A 144      16.544   2.082  16.287  1.00 61.58           C  \nATOM    375  OG  SER A 144      15.671   2.760  15.398  1.00 59.45           O  \nATOM    376  N   GLU A 145      16.856   3.873  19.001  1.00 63.64           N  \nATOM    377  CA  GLU A 145      16.215   4.819  19.908  1.00 65.77           C  \nATOM    378  C   GLU A 145      17.050   6.051  20.172  1.00 67.43           C  \nATOM    379  O   GLU A 145      16.539   7.170  20.183  1.00 67.57           O  \nATOM    380  CB  GLU A 145      15.892   4.143  21.237  1.00 66.24           C  \nATOM    381  CG  GLU A 145      14.468   3.677  21.325  1.00 69.38           C  \nATOM    382  CD  GLU A 145      13.482   4.824  21.293  1.00 70.89           C  \nATOM    383  OE1 GLU A 145      13.390   5.565  22.294  1.00 71.85           O  \nATOM    384  OE2 GLU A 145      12.806   4.987  20.258  1.00 71.51           O  \nATOM    385  N   ASN A 146      18.338   5.830  20.395  1.00 70.15           N  \nATOM    386  CA  ASN A 146      19.275   6.910  20.673  1.00 72.11           C  \nATOM    387  C   ASN A 146      20.179   7.093  19.459  1.00 74.06           C  \nATOM    388  O   ASN A 146      21.374   6.819  19.518  1.00 75.17           O  \nATOM    389  CB  ASN A 146      20.096   6.541  21.903  1.00 70.71           C  \nATOM    390  CG  ASN A 146      19.235   5.969  23.017  1.00 70.75           C  \nATOM    391  OD1 ASN A 146      18.437   6.680  23.632  1.00 69.12           O  \nATOM    392  ND2 ASN A 146      19.381   4.675  23.268  1.00 70.95           N  \nATOM    393  N   LYS A 147      19.594   7.542  18.354  1.00 77.33           N  \nATOM    394  CA  LYS A 147      20.338   7.746  17.119  1.00 79.84           C  \nATOM    395  C   LYS A 147      21.677   8.423  17.391  1.00 81.33           C  \nATOM    396  O   LYS A 147      21.766   9.356  18.189  1.00 81.30           O  \nATOM    397  CB  LYS A 147      19.513   8.587  16.142  1.00 80.69           C  \nATOM    398  CG  LYS A 147      18.191   7.944  15.733  1.00 81.42           C  \nATOM    399  CD  LYS A 147      16.984   8.826  16.080  1.00 82.46           C  \nATOM    400  CE  LYS A 147      16.751   8.943  17.593  1.00 83.00           C  \nATOM    401  NZ  LYS A 147      17.836   9.661  18.327  1.00 80.22           N  \nATOM    402  N   GLU A 148      22.717   7.939  16.722  1.00 82.64           N  \nATOM    403  CA  GLU A 148      24.061   8.480  16.881  1.00 83.18           C  \nATOM    404  C   GLU A 148      24.931   7.955  15.742  1.00 83.16           C  \nATOM    405  O   GLU A 148      24.477   7.161  14.916  1.00 82.57           O  \nATOM    406  CB  GLU A 148      24.654   8.028  18.217  1.00 83.18           C  \nATOM    407  CG  GLU A 148      25.359   9.114  19.009  1.00 83.41           C  \nATOM    408  CD  GLU A 148      24.432   9.828  19.981  1.00 83.90           C  \nATOM    409  OE1 GLU A 148      23.878   9.157  20.878  1.00 85.21           O  \nATOM    410  OE2 GLU A 148      24.257  11.059  19.853  1.00 83.58           O  \nATOM    411  N   LYS A 149      26.181   8.406  15.705  1.00 83.59           N  \nATOM    412  CA  LYS A 149      27.139   7.982  14.689  1.00 84.80           C  \nATOM    413  C   LYS A 149      28.197   7.162  15.402  1.00 86.09           C  \nATOM    414  O   LYS A 149      28.821   6.277  14.822  1.00 86.06           O  \nATOM    415  CB  LYS A 149      27.807   9.197  14.046  1.00 84.36           C  \nATOM    416  CG  LYS A 149      26.917  10.023  13.139  1.00 84.07           C  \nATOM    417  CD  LYS A 149      26.797   9.388  11.768  1.00 84.26           C  \nATOM    418  CE  LYS A 149      26.143  10.350  10.785  1.00 85.30           C  \nATOM    419  NZ  LYS A 149      26.878  11.653  10.722  1.00 86.88           N  \nATOM    420  N   GLU A 150      28.378   7.473  16.679  1.00 88.50           N  \nATOM    421  CA  GLU A 150      29.363   6.812  17.523  1.00 90.25           C  \nATOM    422  C   GLU A 150      28.752   6.352  18.845  1.00 88.69           C  \nATOM    423  O   GLU A 150      28.153   7.145  19.570  1.00 85.09           O  \nATOM    424  CB  GLU A 150      30.525   7.773  17.801  1.00 94.83           C  \nATOM    425  CG  GLU A 150      30.252   9.234  17.410  1.00100.22           C  \nATOM    426  CD  GLU A 150      28.964   9.794  18.013  1.00102.77           C  \nATOM    427  OE1 GLU A 150      28.849   9.852  19.259  1.00103.46           O  \nATOM    428  OE2 GLU A 150      28.063  10.178  17.233  1.00103.47           O  \nATOM    429  N   TYR A 151      28.913   5.069  19.156  1.00 88.60           N  \nATOM    430  CA  TYR A 151      28.383   4.514  20.396  1.00 89.53           C  \nATOM    431  C   TYR A 151      29.464   3.829  21.219  1.00 90.33           C  \nATOM    432  O   TYR A 151      30.339   3.155  20.678  1.00 89.96           O  \nATOM    433  CB  TYR A 151      27.271   3.493  20.120  1.00 89.49           C  \nATOM    434  CG  TYR A 151      26.070   4.032  19.379  1.00 89.54           C  \nATOM    435  CD1 TYR A 151      26.068   4.122  17.988  1.00 91.04           C  \nATOM    436  CD2 TYR A 151      24.925   4.434  20.067  1.00 88.69           C  \nATOM    437  CE1 TYR A 151      24.957   4.594  17.301  1.00 91.17           C  \nATOM    438  CE2 TYR A 151      23.809   4.908  19.388  1.00 89.05           C  \nATOM    439  CZ  TYR A 151      23.832   4.983  18.006  1.00 90.57           C  \nATOM    440  OH  TYR A 151      22.733   5.444  17.322  1.00 91.65           O  \nATOM    441  N   TYR A 152      29.383   4.002  22.535  1.00 91.83           N  \nATOM    442  CA  TYR A 152      30.327   3.394  23.460  1.00 93.18           C  \nATOM    443  C   TYR A 152      29.777   2.044  23.894  1.00 92.08           C  \nATOM    444  O   TYR A 152      28.774   1.980  24.602  1.00 91.91           O  \nATOM    445  CB  TYR A 152      30.512   4.281  24.696  1.00 97.30           C  \nATOM    446  CG  TYR A 152      31.180   5.607  24.422  1.00102.16           C  \nATOM    447  CD1 TYR A 152      30.669   6.487  23.465  1.00104.71           C  \nATOM    448  CD2 TYR A 152      32.327   5.985  25.119  1.00103.83           C  \nATOM    449  CE1 TYR A 152      31.286   7.712  23.207  1.00106.82           C  \nATOM    450  CE2 TYR A 152      32.950   7.206  24.871  1.00106.11           C  \nATOM    451  CZ  TYR A 152      32.426   8.064  23.914  1.00106.90           C  \nATOM    452  OH  TYR A 152      33.041   9.270  23.664  1.00108.41           O  \nATOM    453  N   LEU A 153      30.427   0.966  23.473  1.00 91.02           N  \nATOM    454  CA  LEU A 153      29.975  -0.367  23.849  1.00 90.69           C  \nATOM    455  C   LEU A 153      29.770  -0.437  25.359  1.00 90.09           C  \nATOM    456  O   LEU A 153      29.093  -1.327  25.862  1.00 89.51           O  \nATOM    457  CB  LEU A 153      30.993  -1.423  23.413  1.00 91.87           C  \nATOM    458  CG  LEU A 153      30.658  -2.862  23.816  1.00 93.21           C  \nATOM    459  CD1 LEU A 153      29.352  -3.290  23.172  1.00 93.99           C  \nATOM    460  CD2 LEU A 153      31.780  -3.787  23.392  1.00 93.98           C  \nATOM    461  N   LYS A 154      30.365   0.507  26.077  1.00 90.54           N  \nATOM    462  CA  LYS A 154      30.231   0.561  27.527  1.00 91.44           C  \nATOM    463  C   LYS A 154      28.786   0.881  27.873  1.00 91.49           C  \nATOM    464  O   LYS A 154      28.129   0.131  28.595  1.00 91.11           O  \nATOM    465  CB  LYS A 154      31.141   1.648  28.109  1.00 92.82           C  \nATOM    466  CG  LYS A 154      30.975   1.863  29.610  1.00 93.79           C  \nATOM    467  CD  LYS A 154      31.724   3.104  30.084  1.00 95.28           C  \nATOM    468  CE  LYS A 154      31.631   3.276  31.595  1.00 95.57           C  \nATOM    469  NZ  LYS A 154      30.223   3.380  32.064  1.00 95.84           N  \nATOM    470  N   ASP A 155      28.302   2.003  27.346  1.00 92.36           N  \nATOM    471  CA  ASP A 155      26.935   2.453  27.587  1.00 93.01           C  \nATOM    472  C   ASP A 155      25.911   1.395  27.196  1.00 92.57           C  \nATOM    473  O   ASP A 155      25.054   1.026  27.998  1.00 92.53           O  \nATOM    474  CB  ASP A 155      26.661   3.742  26.811  1.00 93.76           C  \nATOM    475  CG  ASP A 155      27.699   4.818  27.085  1.00 95.05           C  \nATOM    476  OD1 ASP A 155      28.002   5.060  28.273  1.00 95.46           O  \nATOM    477  OD2 ASP A 155      28.206   5.427  26.116  1.00 95.22           O  \nATOM    478  N   ILE A 156      26.007   0.911  25.962  1.00 92.30           N  \nATOM    479  CA  ILE A 156      25.086  -0.105  25.469  1.00 92.10           C  \nATOM    480  C   ILE A 156      24.931  -1.244  26.472  1.00 91.77           C  \nATOM    481  O   ILE A 156      23.813  -1.619  26.830  1.00 92.62           O  \nATOM    482  CB  ILE A 156      25.561  -0.681  24.114  1.00 91.76           C  \nATOM    483  CG1 ILE A 156      25.492   0.410  23.040  1.00 92.04           C  \nATOM    484  CG2 ILE A 156      24.703  -1.878  23.722  1.00 90.71           C  \nATOM    485  CD1 ILE A 156      25.950  -0.037  21.668  1.00 91.70           C  \nATOM    486  N   ILE A 157      26.052  -1.788  26.927  1.00 90.73           N  \nATOM    487  CA  ILE A 157      26.020  -2.876  27.893  1.00 90.22           C  \nATOM    488  C   ILE A 157      25.323  -2.436  29.178  1.00 90.17           C  \nATOM    489  O   ILE A 157      24.619  -3.219  29.814  1.00 89.04           O  \nATOM    490  CB  ILE A 157      27.442  -3.350  28.241  1.00 90.76           C  \nATOM    491  CG1 ILE A 157      28.186  -3.740  26.961  1.00 91.29           C  \nATOM    492  CG2 ILE A 157      27.372  -4.547  29.183  1.00 91.25           C  \nATOM    493  CD1 ILE A 157      29.634  -4.116  27.178  1.00 90.81           C  \nATOM    494  N   ASN A 158      25.523  -1.178  29.552  1.00 91.33           N  \nATOM    495  CA  ASN A 158      24.916  -0.636  30.761  1.00 93.52           C  \nATOM    496  C   ASN A 158      23.404  -0.544  30.642  1.00 93.07           C  \nATOM    497  O   ASN A 158      22.668  -1.015  31.515  1.00 94.10           O  \nATOM    498  CB  ASN A 158      25.476   0.757  31.067  1.00 95.99           C  \nATOM    499  CG  ASN A 158      26.892   0.714  31.614  1.00 98.43           C  \nATOM    500  OD1 ASN A 158      27.152   0.108  32.656  1.00 99.13           O  \nATOM    501  ND2 ASN A 158      27.815   1.364  30.914  1.00100.12           N  \nATOM    502  N   HIS A 159      22.945   0.061  29.553  1.00 91.50           N  \nATOM    503  CA  HIS A 159      21.521   0.244  29.327  1.00 89.98           C  \nATOM    504  C   HIS A 159      20.746  -1.031  29.021  1.00 89.28           C  \nATOM    505  O   HIS A 159      19.519  -1.032  29.041  1.00 89.99           O  \nATOM    506  CB  HIS A 159      21.311   1.287  28.233  1.00 89.50           C  \nATOM    507  CG  HIS A 159      21.798   2.651  28.615  1.00 90.03           C  \nATOM    508  ND1 HIS A 159      21.628   3.759  27.811  1.00 90.94           N  \nATOM    509  CD2 HIS A 159      22.447   3.086  29.721  1.00 89.61           C  \nATOM    510  CE1 HIS A 159      22.150   4.817  28.407  1.00 90.19           C  \nATOM    511  NE2 HIS A 159      22.653   4.435  29.566  1.00 90.10           N  \nATOM    512  N   LEU A 160      21.454  -2.118  28.741  1.00 88.49           N  \nATOM    513  CA  LEU A 160      20.789  -3.386  28.474  1.00 87.40           C  \nATOM    514  C   LEU A 160      20.962  -4.298  29.688  1.00 88.50           C  \nATOM    515  O   LEU A 160      21.773  -4.018  30.572  1.00 89.21           O  \nATOM    516  CB  LEU A 160      21.370  -4.057  27.226  1.00 83.83           C  \nATOM    517  CG  LEU A 160      21.243  -3.305  25.897  1.00 80.91           C  \nATOM    518  CD1 LEU A 160      21.570  -4.258  24.757  1.00 78.53           C  \nATOM    519  CD2 LEU A 160      19.838  -2.754  25.729  1.00 79.96           C  \nATOM    520  N   ASN A 161      20.198  -5.384  29.733  1.00 89.51           N  \nATOM    521  CA  ASN A 161      20.276  -6.323  30.846  1.00 91.09           C  \nATOM    522  C   ASN A 161      20.595  -7.742  30.380  1.00 92.60           C  \nATOM    523  O   ASN A 161      19.882  -8.682  30.733  1.00 93.58           O  \nATOM    524  CB  ASN A 161      18.954  -6.317  31.622  1.00 89.95           C  \nATOM    525  CG  ASN A 161      18.769  -5.057  32.451  1.00 90.14           C  \nATOM    526  OD1 ASN A 161      19.378  -4.902  33.510  1.00 90.24           O  \nATOM    527  ND2 ASN A 161      17.937  -4.143  31.965  1.00 90.25           N  \nATOM    528  N   TYR A 162      21.670  -7.903  29.608  1.00 94.68           N  \nATOM    529  CA  TYR A 162      22.035  -9.226  29.104  1.00 97.60           C  \nATOM    530  C   TYR A 162      23.441  -9.738  29.425  1.00 99.04           C  \nATOM    531  O   TYR A 162      23.833 -10.794  28.929  1.00100.06           O  \nATOM    532  CB  TYR A 162      21.829  -9.286  27.583  1.00 97.58           C  \nATOM    533  CG  TYR A 162      20.381  -9.264  27.147  1.00 98.39           C  \nATOM    534  CD1 TYR A 162      19.699  -8.060  26.978  1.00 98.75           C  \nATOM    535  CD2 TYR A 162      19.681 -10.453  26.936  1.00 98.43           C  \nATOM    536  CE1 TYR A 162      18.351  -8.041  26.613  1.00 99.10           C  \nATOM    537  CE2 TYR A 162      18.338 -10.448  26.574  1.00 98.54           C  \nATOM    538  CZ  TYR A 162      17.678  -9.241  26.415  1.00 99.04           C  \nATOM    539  OH  TYR A 162      16.345  -9.236  26.080  1.00 98.32           O  \nATOM    540  N   LYS A 163      24.197  -9.014  30.249  1.00100.17           N  \nATOM    541  CA  LYS A 163      25.562  -9.432  30.586  1.00100.51           C  \nATOM    542  C   LYS A 163      26.484  -9.303  29.368  1.00 99.20           C  \nATOM    543  O   LYS A 163      26.151  -9.768  28.276  1.00 98.96           O  \nATOM    544  CB  LYS A 163      25.576 -10.884  31.089  1.00102.46           C  \nATOM    545  CG  LYS A 163      25.166 -11.077  32.549  1.00103.63           C  \nATOM    546  CD  LYS A 163      26.191 -10.464  33.502  1.00104.88           C  \nATOM    547  CE  LYS A 163      26.064 -11.020  34.922  1.00105.31           C  \nATOM    548  NZ  LYS A 163      24.741 -10.749  35.550  1.00104.88           N  \nATOM    549  N   GLN A 164      27.645  -8.681  29.569  1.00 96.62           N  \nATOM    550  CA  GLN A 164      28.622  -8.457  28.500  1.00 93.83           C  \nATOM    551  C   GLN A 164      28.828  -9.621  27.531  1.00 91.83           C  \nATOM    552  O   GLN A 164      28.782  -9.436  26.315  1.00 92.16           O  \nATOM    553  CB  GLN A 164      29.975  -8.075  29.094  1.00 94.00           C  \nATOM    554  CG  GLN A 164      29.957  -6.822  29.935  1.00 93.68           C  \nATOM    555  CD  GLN A 164      31.320  -6.509  30.510  1.00 94.28           C  \nATOM    556  OE1 GLN A 164      32.278  -6.282  29.770  1.00 93.29           O  \nATOM    557  NE2 GLN A 164      31.418  -6.503  31.836  1.00 95.48           N  \nATOM    558  N   PRO A 165      29.077 -10.831  28.056  1.00 89.50           N  \nATOM    559  CA  PRO A 165      29.290 -12.004  27.204  1.00 88.47           C  \nATOM    560  C   PRO A 165      28.337 -12.091  26.012  1.00 88.05           C  \nATOM    561  O   PRO A 165      28.777 -12.202  24.870  1.00 87.34           O  \nATOM    562  CB  PRO A 165      29.111 -13.163  28.178  1.00 88.94           C  \nATOM    563  CG  PRO A 165      29.695 -12.607  29.430  1.00 88.73           C  \nATOM    564  CD  PRO A 165      29.108 -11.211  29.480  1.00 88.74           C  \nATOM    565  N   GLN A 166      27.036 -12.034  26.284  1.00 87.95           N  \nATOM    566  CA  GLN A 166      26.022 -12.110  25.234  1.00 88.10           C  \nATOM    567  C   GLN A 166      25.936 -10.828  24.403  1.00 85.87           C  \nATOM    568  O   GLN A 166      25.713 -10.878  23.191  1.00 84.98           O  \nATOM    569  CB  GLN A 166      24.654 -12.413  25.847  1.00 91.33           C  \nATOM    570  CG  GLN A 166      24.601 -13.711  26.637  1.00 96.08           C  \nATOM    571  CD  GLN A 166      23.204 -14.021  27.147  1.00 99.20           C  \nATOM    572  OE1 GLN A 166      22.281 -14.255  26.362  1.00101.11           O  \nATOM    573  NE2 GLN A 166      23.040 -14.018  28.466  1.00100.08           N  \nATOM    574  N   VAL A 167      26.108  -9.685  25.058  1.00 83.51           N  \nATOM    575  CA  VAL A 167      26.055  -8.402  24.372  1.00 82.82           C  \nATOM    576  C   VAL A 167      27.200  -8.293  23.367  1.00 84.06           C  \nATOM    577  O   VAL A 167      26.965  -8.091  22.174  1.00 86.30           O  \nATOM    578  CB  VAL A 167      26.154  -7.232  25.371  1.00 82.19           C  \nATOM    579  CG1 VAL A 167      26.024  -5.905  24.637  1.00 80.91           C  \nATOM    580  CG2 VAL A 167      25.077  -7.366  26.434  1.00 82.33           C  \nATOM    581  N   VAL A 168      28.436  -8.425  23.845  1.00 84.45           N  \nATOM    582  CA  VAL A 168      29.607  -8.341  22.971  1.00 84.47           C  \nATOM    583  C   VAL A 168      29.492  -9.383  21.866  1.00 84.38           C  \nATOM    584  O   VAL A 168      29.900  -9.153  20.727  1.00 84.13           O  \nATOM    585  CB  VAL A 168      30.913  -8.584  23.753  1.00 84.75           C  \nATOM    586  CG1 VAL A 168      32.093  -8.592  22.798  1.00 84.64           C  \nATOM    587  CG2 VAL A 168      31.097  -7.500  24.806  1.00 84.92           C  \nATOM    588  N   LYS A 169      28.929 -10.531  22.226  1.00 84.34           N  \nATOM    589  CA  LYS A 169      28.714 -11.631  21.299  1.00 83.21           C  \nATOM    590  C   LYS A 169      27.768 -11.158  20.198  1.00 82.01           C  \nATOM    591  O   LYS A 169      27.947 -11.488  19.026  1.00 80.96           O  \nATOM    592  CB  LYS A 169      28.102 -12.801  22.063  1.00 85.18           C  \nATOM    593  CG  LYS A 169      27.802 -14.048  21.268  1.00 87.45           C  \nATOM    594  CD  LYS A 169      27.283 -15.120  22.228  1.00 89.77           C  \nATOM    595  CE  LYS A 169      26.972 -16.434  21.536  1.00 90.97           C  \nATOM    596  NZ  LYS A 169      26.564 -17.470  22.526  1.00 92.55           N  \nATOM    597  N   ALA A 170      26.767 -10.368  20.590  1.00 81.02           N  \nATOM    598  CA  ALA A 170      25.780  -9.831  19.657  1.00 79.11           C  \nATOM    599  C   ALA A 170      26.376  -8.712  18.806  1.00 78.92           C  \nATOM    600  O   ALA A 170      26.174  -8.669  17.591  1.00 79.41           O  \nATOM    601  CB  ALA A 170      24.570  -9.317  20.424  1.00 78.15           C  \nATOM    602  N   VAL A 171      27.098  -7.800  19.452  1.00 78.09           N  \nATOM    603  CA  VAL A 171      27.739  -6.691  18.751  1.00 77.10           C  \nATOM    604  C   VAL A 171      28.607  -7.272  17.638  1.00 78.52           C  \nATOM    605  O   VAL A 171      28.544  -6.842  16.483  1.00 78.48           O  \nATOM    606  CB  VAL A 171      28.647  -5.886  19.701  1.00 76.03           C  \nATOM    607  CG1 VAL A 171      29.343  -4.766  18.941  1.00 74.31           C  \nATOM    608  CG2 VAL A 171      27.828  -5.336  20.850  1.00 75.48           C  \nATOM    609  N   LYS A 172      29.410  -8.263  18.014  1.00 80.10           N  \nATOM    610  CA  LYS A 172      30.315  -8.951  17.105  1.00 80.98           C  \nATOM    611  C   LYS A 172      29.592  -9.396  15.834  1.00 79.65           C  \nATOM    612  O   LYS A 172      29.934  -8.970  14.729  1.00 79.95           O  \nATOM    613  CB  LYS A 172      30.917 -10.160  17.827  1.00 82.38           C  \nATOM    614  CG  LYS A 172      32.041 -10.871  17.097  1.00 84.33           C  \nATOM    615  CD  LYS A 172      32.717 -11.858  18.041  1.00 85.67           C  \nATOM    616  CE  LYS A 172      33.882 -12.577  17.381  1.00 87.28           C  \nATOM    617  NZ  LYS A 172      34.612 -13.427  18.356  1.00 87.30           N  \nATOM    618  N   ILE A 173      28.592 -10.254  15.999  1.00 77.61           N  \nATOM    619  CA  ILE A 173      27.820 -10.759  14.868  1.00 74.88           C  \nATOM    620  C   ILE A 173      27.236  -9.612  14.043  1.00 75.65           C  \nATOM    621  O   ILE A 173      27.454  -9.525  12.832  1.00 77.05           O  \nATOM    622  CB  ILE A 173      26.668 -11.657  15.352  1.00 72.48           C  \nATOM    623  CG1 ILE A 173      27.217 -12.759  16.259  1.00 71.17           C  \nATOM    624  CG2 ILE A 173      25.941 -12.261  14.160  1.00 70.76           C  \nATOM    625  CD1 ILE A 173      26.147 -13.563  16.948  1.00 71.10           C  \nATOM    626  N   LEU A 174      26.494  -8.730  14.706  1.00 75.58           N  \nATOM    627  CA  LEU A 174      25.883  -7.590  14.031  1.00 75.63           C  \nATOM    628  C   LEU A 174      26.914  -6.819  13.226  1.00 76.94           C  \nATOM    629  O   LEU A 174      26.623  -6.326  12.136  1.00 76.47           O  \nATOM    630  CB  LEU A 174      25.234  -6.654  15.052  1.00 73.42           C  \nATOM    631  CG  LEU A 174      24.006  -7.186  15.791  1.00 71.53           C  \nATOM    632  CD1 LEU A 174      23.592  -6.200  16.873  1.00 69.66           C  \nATOM    633  CD2 LEU A 174      22.876  -7.423  14.798  1.00 68.71           C  \nATOM    634  N   SER A 175      28.119  -6.718  13.778  1.00 79.26           N  \nATOM    635  CA  SER A 175      29.218  -6.009  13.131  1.00 80.70           C  \nATOM    636  C   SER A 175      29.593  -6.664  11.804  1.00 82.12           C  \nATOM    637  O   SER A 175      29.814  -5.980  10.806  1.00 80.72           O  \nATOM    638  CB  SER A 175      30.432  -5.978  14.056  1.00 79.69           C  \nATOM    639  OG  SER A 175      31.451  -5.172  13.502  1.00 80.67           O  \nATOM    640  N   GLN A 176      29.660  -7.994  11.804  1.00 85.66           N  \nATOM    641  CA  GLN A 176      29.991  -8.753  10.600  1.00 88.99           C  \nATOM    642  C   GLN A 176      28.873  -8.554   9.589  1.00 89.88           C  \nATOM    643  O   GLN A 176      29.119  -8.295   8.412  1.00 89.79           O  \nATOM    644  CB  GLN A 176      30.108 -10.242  10.921  1.00 90.70           C  \nATOM    645  CG  GLN A 176      30.968 -10.568  12.131  1.00 93.20           C  \nATOM    646  CD  GLN A 176      31.014 -12.057  12.410  1.00 94.08           C  \nATOM    647  OE1 GLN A 176      31.518 -12.494  13.447  1.00 94.93           O  \nATOM    648  NE2 GLN A 176      30.489 -12.846  11.480  1.00 94.09           N  \nATOM    649  N   GLU A 177      27.638  -8.688  10.065  1.00 91.84           N  \nATOM    650  CA  GLU A 177      26.458  -8.509   9.225  1.00 93.10           C  \nATOM    651  C   GLU A 177      26.528  -7.112   8.616  1.00 92.99           C  \nATOM    652  O   GLU A 177      25.801  -6.787   7.676  1.00 92.43           O  \nATOM    653  CB  GLU A 177      25.195  -8.652  10.073  1.00 94.47           C  \nATOM    654  CG  GLU A 177      25.076  -9.992  10.775  1.00 96.38           C  \nATOM    655  CD  GLU A 177      24.700 -11.112   9.828  1.00 97.72           C  \nATOM    656  OE1 GLU A 177      25.345 -11.236   8.765  1.00 97.83           O  \nATOM    657  OE2 GLU A 177      23.760 -11.871  10.147  1.00 99.20           O  \nATOM    658  N   ASP A 178      27.412  -6.296   9.179  1.00 92.60           N  \nATOM    659  CA  ASP A 178      27.639  -4.934   8.727  1.00 93.17           C  \nATOM    660  C   ASP A 178      26.465  -4.004   9.004  1.00 92.56           C  \nATOM    661  O   ASP A 178      25.991  -3.296   8.115  1.00 92.07           O  \nATOM    662  CB  ASP A 178      27.987  -4.934   7.233  1.00 95.54           C  \nATOM    663  CG  ASP A 178      28.598  -3.622   6.775  1.00 97.05           C  \nATOM    664  OD1 ASP A 178      29.461  -3.081   7.502  1.00 98.04           O  \nATOM    665  OD2 ASP A 178      28.225  -3.141   5.682  1.00 97.96           O  \nATOM    666  N   TYR A 179      25.997  -4.017  10.249  1.00 91.96           N  \nATOM    667  CA  TYR A 179      24.902  -3.149  10.672  1.00 92.55           C  \nATOM    668  C   TYR A 179      25.562  -1.895  11.222  1.00 94.30           C  \nATOM    669  O   TYR A 179      24.984  -0.810  11.230  1.00 94.63           O  \nATOM    670  CB  TYR A 179      24.060  -3.839  11.749  1.00 90.55           C  \nATOM    671  CG  TYR A 179      23.004  -4.759  11.177  1.00 88.71           C  \nATOM    672  CD1 TYR A 179      21.891  -4.239  10.517  1.00 87.68           C  \nATOM    673  CD2 TYR A 179      23.136  -6.147  11.255  1.00 87.85           C  \nATOM    674  CE1 TYR A 179      20.938  -5.074   9.943  1.00 87.19           C  \nATOM    675  CE2 TYR A 179      22.187  -6.994  10.683  1.00 86.69           C  \nATOM    676  CZ  TYR A 179      21.092  -6.450  10.028  1.00 86.72           C  \nATOM    677  OH  TYR A 179      20.155  -7.275   9.446  1.00 85.04           O  \nATOM    678  N   PHE A 180      26.793  -2.077  11.682  1.00 96.83           N  \nATOM    679  CA  PHE A 180      27.614  -1.004  12.218  1.00 99.56           C  \nATOM    680  C   PHE A 180      29.049  -1.516  12.200  1.00102.69           C  \nATOM    681  O   PHE A 180      29.296  -2.698  12.451  1.00102.81           O  \nATOM    682  CB  PHE A 180      27.187  -0.620  13.644  1.00 97.63           C  \nATOM    683  CG  PHE A 180      26.997  -1.788  14.571  1.00 95.26           C  \nATOM    684  CD1 PHE A 180      25.724  -2.285  14.829  1.00 94.24           C  \nATOM    685  CD2 PHE A 180      28.084  -2.381  15.200  1.00 93.87           C  \nATOM    686  CE1 PHE A 180      25.537  -3.352  15.702  1.00 92.55           C  \nATOM    687  CE2 PHE A 180      27.907  -3.448  16.073  1.00 92.59           C  \nATOM    688  CZ  PHE A 180      26.630  -3.934  16.325  1.00 92.00           C  \nATOM    689  N   ASP A 181      29.990  -0.632  11.890  1.00106.28           N  \nATOM    690  CA  ASP A 181      31.390  -1.026  11.812  1.00109.55           C  \nATOM    691  C   ASP A 181      31.929  -1.600  13.114  1.00111.67           C  \nATOM    692  O   ASP A 181      32.075  -2.816  13.239  1.00113.62           O  \nATOM    693  CB  ASP A 181      32.245   0.155  11.353  1.00108.93           C  \nATOM    694  CG  ASP A 181      31.798   0.703  10.014  1.00109.39           C  \nATOM    695  OD1 ASP A 181      30.711   1.317   9.957  1.00109.92           O  \nATOM    696  OD2 ASP A 181      32.525   0.513   9.018  1.00110.18           O  \nATOM    697  N   LYS A 182      32.217  -0.738  14.083  1.00112.86           N  \nATOM    698  CA  LYS A 182      32.750  -1.197  15.357  1.00114.94           C  \nATOM    699  C   LYS A 182      34.154  -1.740  15.131  1.00117.43           C  \nATOM    700  O   LYS A 182      34.435  -2.417  14.133  1.00118.66           O  \nATOM    701  CB  LYS A 182      31.815  -2.245  15.972  1.00113.75           C  \nATOM    702  CG  LYS A 182      32.337  -2.940  17.217  1.00112.74           C  \nATOM    703  CD  LYS A 182      33.067  -4.228  16.876  1.00110.35           C  \nATOM    704  CE  LYS A 182      33.439  -4.986  18.142  1.00109.32           C  \nATOM    705  NZ  LYS A 182      34.011  -6.332  17.860  1.00108.32           N  \nATOM    706  N   LYS A 183      35.036  -1.303  16.029  1.00120.58           N  \nATOM    707  CA  LYS A 183      36.453  -1.636  16.020  1.00124.47           C  \nATOM    708  C   LYS A 183      36.899  -1.963  17.446  1.00127.39           C  \nATOM    709  O   LYS A 183      36.639  -1.198  18.377  1.00127.42           O  \nATOM    710  CB  LYS A 183      37.258  -0.440  15.506  1.00123.80           C  \nATOM    711  CG  LYS A 183      36.640   0.264  14.307  1.00123.71           C  \nATOM    712  CD  LYS A 183      37.426   1.512  13.915  1.00123.09           C  \nATOM    713  CE  LYS A 183      38.834   1.163  13.447  1.00123.14           C  \nATOM    714  NZ  LYS A 183      39.577   2.358  12.957  1.00122.71           N  \nATOM    715  N   ARG A 184      37.586  -3.092  17.601  1.00130.87           N  \nATOM    716  CA  ARG A 184      38.062  -3.540  18.905  1.00134.01           C  \nATOM    717  C   ARG A 184      39.308  -2.779  19.350  1.00136.31           C  \nATOM    718  O   ARG A 184      39.485  -1.605  19.021  1.00136.51           O  \nATOM    719  CB  ARG A 184      38.378  -5.034  18.851  1.00133.86           C  \nATOM    720  CG  ARG A 184      38.192  -5.756  20.171  1.00134.57           C  \nATOM    721  CD  ARG A 184      36.773  -6.273  20.308  1.00134.27           C  \nATOM    722  NE  ARG A 184      36.558  -6.930  21.592  1.00134.98           N  \nATOM    723  CZ  ARG A 184      35.531  -7.730  21.856  1.00135.26           C  \nATOM    724  NH1 ARG A 184      34.624  -7.974  20.921  1.00134.87           N  \nATOM    725  NH2 ARG A 184      35.416  -8.286  23.053  1.00135.78           N  \nATOM    726  N   ASN A 185      40.168  -3.463  20.102  1.00139.51           N  \nATOM    727  CA  ASN A 185      41.404  -2.870  20.601  1.00142.29           C  \nATOM    728  C   ASN A 185      42.558  -3.870  20.560  1.00143.36           C  \nATOM    729  O   ASN A 185      42.428  -4.992  20.065  1.00143.63           O  \nATOM    730  CB  ASN A 185      41.237  -2.406  22.053  1.00143.29           C  \nATOM    731  CG  ASN A 185      39.872  -1.817  22.333  1.00144.05           C  \nATOM    732  OD1 ASN A 185      39.465  -0.831  21.717  1.00144.47           O  \nATOM    733  ND2 ASN A 185      39.157  -2.420  23.277  1.00144.15           N  \nATOM    734  N   GLU A 186      43.691  -3.432  21.096  1.00144.14           N  \nATOM    735  CA  GLU A 186      44.900  -4.237  21.199  1.00144.29           C  \nATOM    736  C   GLU A 186      45.206  -4.096  22.686  1.00144.56           C  \nATOM    737  O   GLU A 186      45.929  -4.892  23.287  1.00144.27           O  \nATOM    738  CB  GLU A 186      46.023  -3.615  20.368  1.00143.83           C  \nATOM    739  CG  GLU A 186      47.242  -4.499  20.194  1.00143.44           C  \nATOM    740  CD  GLU A 186      46.991  -5.649  19.242  1.00143.45           C  \nATOM    741  OE1 GLU A 186      46.652  -5.384  18.070  1.00143.96           O  \nATOM    742  OE2 GLU A 186      47.133  -6.816  19.661  1.00143.36           O  \nATOM    743  N   HIS A 187      44.615  -3.047  23.251  1.00144.74           N  \nATOM    744  CA  HIS A 187      44.731  -2.688  24.655  1.00144.60           C  \nATOM    745  C   HIS A 187      43.801  -1.489  24.869  1.00145.03           C  \nATOM    746  O   HIS A 187      42.586  -1.619  24.716  1.00144.72           O  \nATOM    747  CB  HIS A 187      46.186  -2.336  25.003  1.00144.05           C  \nATOM    748  CG  HIS A 187      46.769  -1.228  24.180  1.00143.15           C  \nATOM    749  ND1 HIS A 187      46.829  -1.270  22.804  1.00143.03           N  \nATOM    750  CD2 HIS A 187      47.346  -0.058  24.545  1.00142.76           C  \nATOM    751  CE1 HIS A 187      47.419  -0.176  22.357  1.00142.41           C  \nATOM    752  NE2 HIS A 187      47.743   0.576  23.393  1.00142.29           N  \nATOM    753  N   ASP A 188      44.372  -0.337  25.213  1.00145.52           N  \nATOM    754  CA  ASP A 188      43.629   0.908  25.428  1.00145.90           C  \nATOM    755  C   ASP A 188      42.395   0.838  26.331  1.00146.05           C  \nATOM    756  O   ASP A 188      41.845   1.872  26.710  1.00146.27           O  \nATOM    757  CB  ASP A 188      43.234   1.523  24.075  1.00145.69           C  \nATOM    758  CG  ASP A 188      42.321   0.621  23.261  1.00145.55           C  \nATOM    759  OD1 ASP A 188      41.196   0.338  23.722  1.00145.45           O  \nATOM    760  OD2 ASP A 188      42.731   0.195  22.159  1.00144.83           O  \nATOM    761  N   GLU A 189      41.962  -0.372  26.673  1.00145.96           N  \nATOM    762  CA  GLU A 189      40.797  -0.565  27.533  1.00145.68           C  \nATOM    763  C   GLU A 189      39.540   0.155  27.053  1.00145.59           C  \nATOM    764  O   GLU A 189      38.513   0.134  27.734  1.00145.79           O  \nATOM    765  CB  GLU A 189      41.123  -0.127  28.964  1.00145.07           C  \nATOM    766  CG  GLU A 189      41.905  -1.157  29.767  1.00144.98           C  \nATOM    767  CD  GLU A 189      43.033  -1.786  28.973  1.00144.75           C  \nATOM    768  OE1 GLU A 189      42.749  -2.601  28.070  1.00144.44           O  \nATOM    769  OE2 GLU A 189      44.205  -1.461  29.245  1.00144.99           O  \nATOM    770  N   ARG A 190      39.615   0.784  25.883  1.00145.13           N  \nATOM    771  CA  ARG A 190      38.465   1.494  25.334  1.00144.04           C  \nATOM    772  C   ARG A 190      37.384   0.470  25.010  1.00143.54           C  \nATOM    773  O   ARG A 190      36.283   0.822  24.586  1.00143.68           O  \nATOM    774  CB  ARG A 190      38.861   2.254  24.065  1.00143.51           C  \nATOM    775  CG  ARG A 190      37.848   3.301  23.621  1.00142.48           C  \nATOM    776  CD  ARG A 190      37.710   4.401  24.665  1.00142.19           C  \nATOM    777  NE  ARG A 190      36.855   5.496  24.215  1.00142.11           N  \nATOM    778  CZ  ARG A 190      36.622   6.599  24.920  1.00142.02           C  \nATOM    779  NH1 ARG A 190      37.179   6.758  26.113  1.00142.35           N  \nATOM    780  NH2 ARG A 190      35.839   7.548  24.429  1.00141.66           N  \nATOM    781  N   THR A 191      37.724  -0.800  25.220  1.00142.28           N  \nATOM    782  CA  THR A 191      36.831  -1.929  24.976  1.00140.62           C  \nATOM    783  C   THR A 191      35.749  -1.676  23.924  1.00138.75           C  \nATOM    784  O   THR A 191      34.566  -1.549  24.245  1.00139.04           O  \nATOM    785  CB  THR A 191      36.161  -2.398  26.293  1.00141.11           C  \nATOM    786  OG1 THR A 191      37.175  -2.739  27.247  1.00141.21           O  \nATOM    787  CG2 THR A 191      35.284  -3.622  26.048  1.00140.66           C  \nATOM    788  N   VAL A 192      36.175  -1.589  22.667  1.00135.30           N  \nATOM    789  CA  VAL A 192      35.268  -1.393  21.541  1.00130.22           C  \nATOM    790  C   VAL A 192      34.488  -0.081  21.499  1.00126.65           C  \nATOM    791  O   VAL A 192      34.121   0.482  22.529  1.00125.81           O  \nATOM    792  CB  VAL A 192      34.247  -2.547  21.469  1.00130.70           C  \nATOM    793  CG1 VAL A 192      33.393  -2.411  20.225  1.00130.56           C  \nATOM    794  CG2 VAL A 192      34.968  -3.880  21.490  1.00130.52           C  \nATOM    795  N   LEU A 193      34.242   0.387  20.279  1.00122.16           N  \nATOM    796  CA  LEU A 193      33.479   1.603  20.023  1.00118.33           C  \nATOM    797  C   LEU A 193      32.730   1.389  18.713  1.00114.58           C  \nATOM    798  O   LEU A 193      33.327   1.345  17.636  1.00114.18           O  \nATOM    799  CB  LEU A 193      34.398   2.825  19.920  1.00119.90           C  \nATOM    800  CG  LEU A 193      34.940   3.389  21.242  1.00120.90           C  \nATOM    801  CD1 LEU A 193      35.850   4.574  20.955  1.00121.54           C  \nATOM    802  CD2 LEU A 193      33.784   3.811  22.145  1.00120.70           C  \nATOM    803  N   ILE A 194      31.414   1.253  18.826  1.00109.62           N  \nATOM    804  CA  ILE A 194      30.546   1.005  17.685  1.00104.37           C  \nATOM    805  C   ILE A 194      30.249   2.230  16.830  1.00101.80           C  \nATOM    806  O   ILE A 194      29.848   3.280  17.332  1.00100.31           O  \nATOM    807  CB  ILE A 194      29.233   0.384  18.169  1.00104.11           C  \nATOM    808  CG1 ILE A 194      29.535  -0.959  18.839  1.00102.95           C  \nATOM    809  CG2 ILE A 194      28.267   0.229  17.009  1.00103.81           C  \nATOM    810  CD1 ILE A 194      28.371  -1.555  19.581  1.00102.98           C  \nATOM    811  N   LEU A 195      30.442   2.076  15.525  1.00 98.73           N  \nATOM    812  CA  LEU A 195      30.208   3.160  14.584  1.00 97.25           C  \nATOM    813  C   LEU A 195      29.172   2.776  13.540  1.00 97.22           C  \nATOM    814  O   LEU A 195      29.283   1.737  12.890  1.00 97.04           O  \nATOM    815  CB  LEU A 195      31.510   3.539  13.874  1.00 96.34           C  \nATOM    816  CG  LEU A 195      32.699   3.980  14.726  1.00 94.81           C  \nATOM    817  CD1 LEU A 195      33.896   4.188  13.824  1.00 95.47           C  \nATOM    818  CD2 LEU A 195      32.366   5.257  15.474  1.00 94.44           C  \nATOM    819  N   VAL A 196      28.167   3.628  13.388  1.00 97.40           N  \nATOM    820  CA  VAL A 196      27.110   3.407  12.414  1.00 97.85           C  \nATOM    821  C   VAL A 196      27.025   4.653  11.535  1.00 97.64           C  \nATOM    822  O   VAL A 196      26.465   5.676  11.935  1.00 98.17           O  \nATOM    823  CB  VAL A 196      25.753   3.137  13.116  1.00 98.11           C  \nATOM    824  CG1 VAL A 196      25.487   4.199  14.157  1.00 97.59           C  \nATOM    825  CG2 VAL A 196      24.628   3.092  12.089  1.00 98.10           C  \nATOM    826  N   ASN A 197      27.603   4.559  10.341  1.00 96.70           N  \nATOM    827  CA  ASN A 197      27.628   5.673   9.400  1.00 96.92           C  \nATOM    828  C   ASN A 197      26.267   6.323   9.163  1.00 97.49           C  \nATOM    829  O   ASN A 197      25.779   7.070  10.007  1.00 97.87           O  \nATOM    830  CB  ASN A 197      28.239   5.217   8.069  1.00 96.09           C  \nATOM    831  CG  ASN A 197      27.586   3.966   7.519  1.00 95.32           C  \nATOM    832  OD1 ASN A 197      28.035   3.410   6.517  1.00 93.75           O  \nATOM    833  ND2 ASN A 197      26.520   3.517   8.171  1.00 95.32           N  \nATOM    834  N   ALA A 198      25.666   6.051   8.009  1.00 98.19           N  \nATOM    835  CA  ALA A 198      24.363   6.612   7.673  1.00 98.58           C  \nATOM    836  C   ALA A 198      23.576   5.596   6.864  1.00 98.35           C  \nATOM    837  O   ALA A 198      22.354   5.503   6.974  1.00 98.30           O  \nATOM    838  CB  ALA A 198      24.535   7.895   6.874  1.00 98.85           C  \nATOM    839  N   GLN A 199      24.292   4.836   6.047  1.00 98.35           N  \nATOM    840  CA  GLN A 199      23.680   3.813   5.217  1.00 98.78           C  \nATOM    841  C   GLN A 199      23.317   2.626   6.095  1.00 96.83           C  \nATOM    842  O   GLN A 199      22.236   2.052   5.964  1.00 96.21           O  \nATOM    843  CB  GLN A 199      24.659   3.390   4.126  1.00102.48           C  \nATOM    844  CG  GLN A 199      25.125   4.550   3.261  1.00107.67           C  \nATOM    845  CD  GLN A 199      26.352   4.212   2.440  1.00110.83           C  \nATOM    846  OE1 GLN A 199      27.408   3.886   2.987  1.00113.01           O  \nATOM    847  NE2 GLN A 199      26.221   4.290   1.119  1.00112.70           N  \nATOM    848  N   GLN A 200      24.227   2.265   6.994  1.00 94.71           N  \nATOM    849  CA  GLN A 200      23.995   1.156   7.908  1.00 92.84           C  \nATOM    850  C   GLN A 200      22.766   1.473   8.745  1.00 91.50           C  \nATOM    851  O   GLN A 200      21.961   0.591   9.043  1.00 91.55           O  \nATOM    852  CB  GLN A 200      25.204   0.960   8.819  1.00 93.26           C  \nATOM    853  CG  GLN A 200      26.464   0.565   8.084  1.00 94.22           C  \nATOM    854  CD  GLN A 200      27.673   0.532   8.993  1.00 95.07           C  \nATOM    855  OE1 GLN A 200      27.997   1.525   9.649  1.00 94.64           O  \nATOM    856  NE2 GLN A 200      28.351  -0.609   9.036  1.00 95.60           N  \nATOM    857  N   ARG A 201      22.628   2.740   9.121  1.00 89.18           N  \nATOM    858  CA  ARG A 201      21.486   3.180   9.907  1.00 87.40           C  \nATOM    859  C   ARG A 201      20.199   2.858   9.169  1.00 86.59           C  \nATOM    860  O   ARG A 201      19.240   2.362   9.759  1.00 87.10           O  \nATOM    861  CB  ARG A 201      21.559   4.682  10.164  1.00 87.11           C  \nATOM    862  CG  ARG A 201      20.233   5.284  10.592  1.00 88.87           C  \nATOM    863  CD  ARG A 201      20.420   6.553  11.397  1.00 91.76           C  \nATOM    864  NE  ARG A 201      20.466   6.294  12.835  1.00 94.60           N  \nATOM    865  CZ  ARG A 201      21.418   5.601  13.458  1.00 95.74           C  \nATOM    866  NH1 ARG A 201      22.430   5.082  12.774  1.00 96.42           N  \nATOM    867  NH2 ARG A 201      21.356   5.425  14.773  1.00 94.93           N  \nATOM    868  N   LYS A 202      20.184   3.151   7.873  1.00 85.36           N  \nATOM    869  CA  LYS A 202      19.021   2.889   7.039  1.00 83.24           C  \nATOM    870  C   LYS A 202      18.745   1.392   7.008  1.00 80.08           C  \nATOM    871  O   LYS A 202      17.597   0.965   6.919  1.00 77.81           O  \nATOM    872  CB  LYS A 202      19.270   3.402   5.619  1.00 86.85           C  \nATOM    873  CG  LYS A 202      19.524   4.905   5.536  1.00 89.64           C  \nATOM    874  CD  LYS A 202      19.751   5.372   4.095  1.00 91.66           C  \nATOM    875  CE  LYS A 202      21.011   4.755   3.483  1.00 91.65           C  \nATOM    876  NZ  LYS A 202      21.315   5.291   2.121  1.00 91.06           N  \nATOM    877  N   LYS A 203      19.810   0.602   7.077  1.00 76.81           N  \nATOM    878  CA  LYS A 203      19.697  -0.851   7.073  1.00 74.57           C  \nATOM    879  C   LYS A 203      19.045  -1.286   8.384  1.00 73.31           C  \nATOM    880  O   LYS A 203      18.129  -2.114   8.390  1.00 73.62           O  \nATOM    881  CB  LYS A 203      21.088  -1.485   6.937  1.00 74.85           C  \nATOM    882  CG  LYS A 203      21.114  -3.009   6.968  1.00 76.24           C  \nATOM    883  CD  LYS A 203      22.547  -3.527   6.881  1.00 78.17           C  \nATOM    884  CE  LYS A 203      22.635  -5.046   6.971  1.00 78.36           C  \nATOM    885  NZ  LYS A 203      21.961  -5.735   5.837  1.00 78.54           N  \nATOM    886  N   ILE A 204      19.527  -0.719   9.490  1.00 70.47           N  \nATOM    887  CA  ILE A 204      18.997  -1.022  10.816  1.00 67.42           C  \nATOM    888  C   ILE A 204      17.505  -0.683  10.900  1.00 65.55           C  \nATOM    889  O   ILE A 204      16.677  -1.552  11.165  1.00 62.77           O  \nATOM    890  CB  ILE A 204      19.760  -0.232  11.916  1.00 67.88           C  \nATOM    891  CG1 ILE A 204      21.159  -0.825  12.101  1.00 68.27           C  \nATOM    892  CG2 ILE A 204      18.988  -0.268  13.241  1.00 66.97           C  \nATOM    893  CD1 ILE A 204      21.997  -0.103  13.140  1.00 68.14           C  \nATOM    894  N   GLU A 205      17.178   0.585  10.667  1.00 64.74           N  \nATOM    895  CA  GLU A 205      15.802   1.051  10.714  1.00 65.49           C  \nATOM    896  C   GLU A 205      14.880   0.238   9.818  1.00 64.58           C  \nATOM    897  O   GLU A 205      13.767  -0.112  10.210  1.00 65.68           O  \nATOM    898  CB  GLU A 205      15.743   2.523  10.326  1.00 67.57           C  \nATOM    899  CG  GLU A 205      16.583   3.395  11.245  1.00 72.19           C  \nATOM    900  CD  GLU A 205      16.264   4.873  11.119  1.00 74.66           C  \nATOM    901  OE1 GLU A 205      16.338   5.405   9.987  1.00 76.95           O  \nATOM    902  OE2 GLU A 205      15.944   5.498  12.156  1.00 75.17           O  \nATOM    903  N   SER A 206      15.355  -0.063   8.617  1.00 63.15           N  \nATOM    904  CA  SER A 206      14.602  -0.848   7.648  1.00 61.07           C  \nATOM    905  C   SER A 206      14.218  -2.199   8.237  1.00 58.59           C  \nATOM    906  O   SER A 206      13.075  -2.654   8.119  1.00 58.09           O  \nATOM    907  CB  SER A 206      15.456  -1.053   6.394  1.00 61.34           C  \nATOM    908  OG  SER A 206      14.976  -2.129   5.611  1.00 66.37           O  \nATOM    909  N   LEU A 207      15.189  -2.837   8.873  1.00 55.17           N  \nATOM    910  CA  LEU A 207      14.977  -4.138   9.482  1.00 51.73           C  \nATOM    911  C   LEU A 207      14.024  -4.025  10.672  1.00 51.07           C  \nATOM    912  O   LEU A 207      13.070  -4.787  10.789  1.00 49.53           O  \nATOM    913  CB  LEU A 207      16.333  -4.725   9.899  1.00 49.87           C  \nATOM    914  CG  LEU A 207      16.426  -5.998  10.740  1.00 49.30           C  \nATOM    915  CD1 LEU A 207      16.190  -5.646  12.198  1.00 50.54           C  \nATOM    916  CD2 LEU A 207      15.412  -7.036  10.253  1.00 49.39           C  \nATOM    917  N   LEU A 208      14.270  -3.060  11.547  1.00 50.88           N  \nATOM    918  CA  LEU A 208      13.414  -2.882  12.716  1.00 50.04           C  \nATOM    919  C   LEU A 208      11.998  -2.519  12.269  1.00 49.86           C  \nATOM    920  O   LEU A 208      11.024  -2.798  12.957  1.00 48.91           O  \nATOM    921  CB  LEU A 208      13.980  -1.792  13.640  1.00 48.13           C  \nATOM    922  CG  LEU A 208      15.260  -2.114  14.427  1.00 45.35           C  \nATOM    923  CD1 LEU A 208      15.520  -1.011  15.456  1.00 42.05           C  \nATOM    924  CD2 LEU A 208      15.118  -3.461  15.128  1.00 42.90           C  \nATOM    925  N   SER A 209      11.907  -1.903  11.093  1.00 50.52           N  \nATOM    926  CA  SER A 209      10.635  -1.499  10.530  1.00 50.23           C  \nATOM    927  C   SER A 209       9.871  -2.744  10.132  1.00 50.00           C  \nATOM    928  O   SER A 209       8.657  -2.824  10.302  1.00 50.44           O  \nATOM    929  CB  SER A 209      10.868  -0.620   9.315  1.00 51.84           C  \nATOM    930  OG  SER A 209       9.816   0.314   9.164  1.00 57.28           O  \nATOM    931  N   ARG A 210      10.587  -3.723   9.598  1.00 50.45           N  \nATOM    932  CA  ARG A 210       9.963  -4.978   9.200  1.00 50.00           C  \nATOM    933  C   ARG A 210       9.632  -5.813  10.430  1.00 47.38           C  \nATOM    934  O   ARG A 210       8.720  -6.635  10.404  1.00 44.28           O  \nATOM    935  CB  ARG A 210      10.898  -5.755   8.278  1.00 53.70           C  \nATOM    936  CG  ARG A 210      11.100  -5.089   6.918  1.00 56.72           C  \nATOM    937  CD  ARG A 210      12.237  -5.747   6.196  1.00 58.88           C  \nATOM    938  NE  ARG A 210      12.112  -7.199   6.251  1.00 59.33           N  \nATOM    939  CZ  ARG A 210      13.146  -8.024   6.352  1.00 60.11           C  \nATOM    940  NH1 ARG A 210      14.378  -7.526   6.410  1.00 58.93           N  \nATOM    941  NH2 ARG A 210      12.948  -9.338   6.397  1.00 59.23           N  \nATOM    942  N   VAL A 211      10.380  -5.613  11.510  1.00 46.28           N  \nATOM    943  CA  VAL A 211      10.104  -6.362  12.730  1.00 44.96           C  \nATOM    944  C   VAL A 211       8.818  -5.818  13.339  1.00 43.27           C  \nATOM    945  O   VAL A 211       8.027  -6.571  13.907  1.00 41.03           O  \nATOM    946  CB  VAL A 211      11.281  -6.263  13.737  1.00 44.56           C  \nATOM    947  CG1 VAL A 211      10.845  -6.755  15.113  1.00 42.82           C  \nATOM    948  CG2 VAL A 211      12.453  -7.120  13.240  1.00 43.33           C  \nATOM    949  N   ASN A 212       8.603  -4.512  13.185  1.00 43.25           N  \nATOM    950  CA  ASN A 212       7.402  -3.848  13.684  1.00 42.16           C  \nATOM    951  C   ASN A 212       6.138  -4.389  13.011  1.00 41.19           C  \nATOM    952  O   ASN A 212       5.128  -4.597  13.672  1.00 39.03           O  \nATOM    953  CB  ASN A 212       7.497  -2.347  13.452  1.00 42.76           C  \nATOM    954  CG  ASN A 212       8.325  -1.649  14.492  1.00 43.11           C  \nATOM    955  OD1 ASN A 212       8.699  -0.493  14.319  1.00 46.21           O  \nATOM    956  ND2 ASN A 212       8.614  -2.340  15.591  1.00 42.08           N  \nATOM    957  N   LYS A 213       6.199  -4.615  11.698  1.00 40.65           N  \nATOM    958  CA  LYS A 213       5.056  -5.159  10.973  1.00 41.42           C  \nATOM    959  C   LYS A 213       4.778  -6.612  11.366  1.00 43.09           C  \nATOM    960  O   LYS A 213       3.619  -7.034  11.375  1.00 44.01           O  \nATOM    961  CB  LYS A 213       5.263  -5.057   9.457  1.00 41.67           C  \nATOM    962  CG  LYS A 213       5.157  -3.646   8.902  1.00 41.02           C  \nATOM    963  CD  LYS A 213       5.657  -3.598   7.489  1.00 42.57           C  \nATOM    964  CE  LYS A 213       5.960  -2.172   7.032  1.00 44.37           C  \nATOM    965  NZ  LYS A 213       6.481  -2.143   5.624  1.00 47.34           N  \nATOM    966  N   ARG A 214       5.813  -7.382  11.697  1.00 45.26           N  \nATOM    967  CA  ARG A 214       5.568  -8.763  12.114  1.00 47.24           C  \nATOM    968  C   ARG A 214       4.881  -8.794  13.469  1.00 45.55           C  \nATOM    969  O   ARG A 214       4.034  -9.648  13.715  1.00 44.03           O  \nATOM    970  CB  ARG A 214       6.866  -9.584  12.175  1.00 50.21           C  \nATOM    971  CG  ARG A 214       7.040 -10.502  10.962  1.00 55.21           C  \nATOM    972  CD  ARG A 214       6.429 -11.895  11.146  1.00 56.93           C  \nATOM    973  NE  ARG A 214       5.162 -11.878  11.866  1.00 59.51           N  \nATOM    974  CZ  ARG A 214       4.458 -12.966  12.171  1.00 60.64           C  \nATOM    975  NH1 ARG A 214       4.891 -14.164  11.808  1.00 61.59           N  \nATOM    976  NH2 ARG A 214       3.335 -12.860  12.872  1.00 62.12           N  \nATOM    977  N   ILE A 215       5.258  -7.866  14.342  1.00 44.12           N  \nATOM    978  CA  ILE A 215       4.656  -7.768  15.671  1.00 44.64           C  \nATOM    979  C   ILE A 215       3.142  -7.552  15.545  1.00 44.77           C  \nATOM    980  O   ILE A 215       2.353  -8.181  16.255  1.00 45.72           O  \nATOM    981  CB  ILE A 215       5.296  -6.590  16.493  1.00 43.53           C  \nATOM    982  CG1 ILE A 215       6.665  -7.030  17.037  1.00 42.41           C  \nATOM    983  CG2 ILE A 215       4.361  -6.140  17.606  1.00 43.54           C  \nATOM    984  CD1 ILE A 215       7.481  -5.922  17.666  1.00 38.70           C  \nATOM    985  N   THR A 216       2.750  -6.676  14.623  1.00 45.34           N  \nATOM    986  CA  THR A 216       1.347  -6.371  14.391  1.00 45.53           C  \nATOM    987  C   THR A 216       0.580  -7.592  13.885  1.00 45.36           C  \nATOM    988  O   THR A 216      -0.521  -7.899  14.369  1.00 44.31           O  \nATOM    989  CB  THR A 216       1.214  -5.202  13.400  1.00 46.33           C  \nATOM    990  OG1 THR A 216       1.727  -4.014  14.024  1.00 48.54           O  \nATOM    991  CG2 THR A 216      -0.229  -4.976  13.021  1.00 47.26           C  \nATOM    992  N   GLU A 217       1.160  -8.288  12.913  1.00 46.93           N  \nATOM    993  CA  GLU A 217       0.534  -9.497  12.399  1.00 49.54           C  \nATOM    994  C   GLU A 217       0.437 -10.497  13.548  1.00 50.12           C  \nATOM    995  O   GLU A 217      -0.551 -11.222  13.671  1.00 49.63           O  \nATOM    996  CB  GLU A 217       1.363 -10.091  11.263  1.00 52.00           C  \nATOM    997  CG  GLU A 217       1.305 -11.620  11.190  1.00 54.64           C  \nATOM    998  CD  GLU A 217       2.035 -12.192   9.986  1.00 55.92           C  \nATOM    999  OE1 GLU A 217       3.176 -11.748   9.689  1.00 55.58           O  \nATOM   1000  OE2 GLU A 217       1.458 -13.096   9.343  1.00 56.01           O  \nATOM   1001  N   ALA A 218       1.470 -10.525  14.386  1.00 51.12           N  \nATOM   1002  CA  ALA A 218       1.512 -11.421  15.539  1.00 51.99           C  \nATOM   1003  C   ALA A 218       0.434 -11.082  16.559  1.00 54.26           C  \nATOM   1004  O   ALA A 218      -0.165 -11.980  17.140  1.00 55.71           O  \nATOM   1005  CB  ALA A 218       2.880 -11.366  16.205  1.00 52.22           C  \nATOM   1006  N   ASN A 219       0.185  -9.794  16.786  1.00 55.54           N  \nATOM   1007  CA  ASN A 219      -0.844  -9.420  17.746  1.00 57.40           C  \nATOM   1008  C   ASN A 219      -2.239  -9.783  17.246  1.00 61.41           C  \nATOM   1009  O   ASN A 219      -3.197  -9.760  18.012  1.00 63.81           O  \nATOM   1010  CB  ASN A 219      -0.784  -7.927  18.072  1.00 54.06           C  \nATOM   1011  CG  ASN A 219       0.419  -7.561  18.934  1.00 51.43           C  \nATOM   1012  OD1 ASN A 219       0.878  -8.367  19.746  1.00 48.70           O  \nATOM   1013  ND2 ASN A 219       0.921  -6.333  18.774  1.00 49.40           N  \nATOM   1014  N   ASN A 220      -2.359 -10.123  15.965  1.00 65.09           N  \nATOM   1015  CA  ASN A 220      -3.659 -10.495  15.418  1.00 69.21           C  \nATOM   1016  C   ASN A 220      -3.867 -12.009  15.488  1.00 72.26           C  \nATOM   1017  O   ASN A 220      -5.006 -12.484  15.551  1.00 72.52           O  \nATOM   1018  CB  ASN A 220      -3.807  -9.990  13.977  1.00 69.58           C  \nATOM   1019  CG  ASN A 220      -3.603  -8.477  13.862  1.00 72.15           C  \nATOM   1020  OD1 ASN A 220      -4.190  -7.690  14.618  1.00 72.10           O  \nATOM   1021  ND2 ASN A 220      -2.773  -8.065  12.908  1.00 73.81           N  \nATOM   1022  N   GLU A 221      -2.770 -12.763  15.478  1.00 76.04           N  \nATOM   1023  CA  GLU A 221      -2.852 -14.220  15.580  1.00 81.76           C  \nATOM   1024  C   GLU A 221      -2.835 -14.612  17.053  1.00 84.93           C  \nATOM   1025  O   GLU A 221      -3.792 -15.204  17.548  1.00 87.20           O  \nATOM   1026  CB  GLU A 221      -1.678 -14.901  14.861  1.00 82.05           C  \nATOM   1027  CG  GLU A 221      -1.924 -15.228  13.381  1.00 86.56           C  \nATOM   1028  CD  GLU A 221      -2.829 -16.444  13.165  1.00 88.01           C  \nATOM   1029  OE1 GLU A 221      -3.104 -16.783  11.988  1.00 86.31           O  \nATOM   1030  OE2 GLU A 221      -3.259 -17.061  14.168  1.00 89.54           O  \nATOM   1031  N   ILE A 222      -1.746 -14.273  17.745  1.00 88.11           N  \nATOM   1032  CA  ILE A 222      -1.585 -14.583  19.168  1.00 90.27           C  \nATOM   1033  C   ILE A 222      -2.924 -14.599  19.895  1.00 93.03           C  \nATOM   1034  O   ILE A 222      -3.516 -15.660  20.101  1.00 94.48           O  \nATOM   1035  CB  ILE A 222      -0.660 -13.554  19.878  1.00 88.64           C  \nATOM   1036  CG1 ILE A 222       0.786 -13.728  19.410  1.00 87.83           C  \nATOM   1037  CG2 ILE A 222      -0.732 -13.736  21.386  1.00 89.00           C  \nATOM   1038  CD1 ILE A 222       1.750 -12.725  20.021  1.00 86.43           C  \nATOM   1039  N   GLU A 223      -3.390 -13.416  20.282  1.00 95.68           N  \nATOM   1040  CA  GLU A 223      -4.659 -13.272  20.981  1.00 99.57           C  \nATOM   1041  C   GLU A 223      -4.984 -14.457  21.898  1.00101.71           C  \nATOM   1042  O   GLU A 223      -5.807 -15.310  21.563  1.00102.28           O  \nATOM   1043  CB  GLU A 223      -5.777 -13.064  19.959  1.00 99.27           C  \nATOM   1044  CG  GLU A 223      -5.531 -11.874  19.038  1.00100.15           C  \nATOM   1045  CD  GLU A 223      -6.695 -11.592  18.108  1.00101.65           C  \nATOM   1046  OE1 GLU A 223      -6.996 -12.440  17.237  1.00101.89           O  \nATOM   1047  OE2 GLU A 223      -7.311 -10.515  18.254  1.00102.29           O  \nATOM   1048  N   LEU A 224      -4.325 -14.501  23.054  1.00103.33           N  \nATOM   1049  CA  LEU A 224      -4.539 -15.563  24.033  1.00104.31           C  \nATOM   1050  C   LEU A 224      -5.296 -14.977  25.226  1.00105.67           C  \nATOM   1051  O   LEU A 224      -4.669 -14.793  26.292  1.00106.82           O  \nATOM   1052  CB  LEU A 224      -3.195 -16.145  24.500  1.00103.46           C  \nATOM   1053  CG  LEU A 224      -3.175 -17.527  25.176  1.00103.09           C  \nATOM   1054  CD1 LEU A 224      -1.736 -17.912  25.473  1.00102.25           C  \nATOM   1055  CD2 LEU A 224      -3.993 -17.529  26.458  1.00102.18           C  \nATOM   1056  OXT LEU A 224      -6.504 -14.688  25.076  1.00106.85           O  \nTER    1057      LEU A 224                                                      \nATOM   1058  N   GLY B  98     -12.664 -11.152  32.324  1.00 61.24           N  \nATOM   1059  CA  GLY B  98     -13.537 -11.374  33.515  1.00 63.75           C  \nATOM   1060  C   GLY B  98     -12.818 -11.156  34.838  1.00 63.18           C  \nATOM   1061  O   GLY B  98     -13.292 -10.426  35.705  1.00 66.12           O  \nATOM   1062  N   SER B  99     -11.675 -11.807  35.002  1.00 58.95           N  \nATOM   1063  CA  SER B  99     -10.890 -11.661  36.216  1.00 53.06           C  \nATOM   1064  C   SER B  99      -9.458 -11.387  35.788  1.00 51.84           C  \nATOM   1065  O   SER B  99      -8.972 -11.987  34.828  1.00 56.20           O  \nATOM   1066  CB  SER B  99     -10.960 -12.945  37.064  1.00 50.12           C  \nATOM   1067  OG  SER B  99     -10.556 -14.085  36.323  1.00 47.26           O  \nATOM   1068  N   HIS B 100      -8.791 -10.479  36.486  1.00 46.07           N  \nATOM   1069  CA  HIS B 100      -7.395 -10.137  36.189  1.00 41.09           C  \nATOM   1070  C   HIS B 100      -7.218  -9.166  35.043  1.00 39.03           C  \nATOM   1071  O   HIS B 100      -7.781  -9.322  33.981  1.00 39.67           O  \nATOM   1072  CB  HIS B 100      -6.558 -11.383  35.899  1.00 39.39           C  \nATOM   1073  CG  HIS B 100      -6.262 -12.207  37.108  1.00 38.00           C  \nATOM   1074  ND1 HIS B 100      -6.579 -13.546  37.198  1.00 37.45           N  \nATOM   1075  CD2 HIS B 100      -5.650 -11.887  38.268  1.00 37.20           C  \nATOM   1076  CE1 HIS B 100      -6.166 -14.009  38.362  1.00 37.21           C  \nATOM   1077  NE2 HIS B 100      -5.598 -13.023  39.030  1.00 36.23           N  \nATOM   1078  N   MET B 101      -6.418  -8.144  35.292  1.00 37.06           N  \nATOM   1079  CA  MET B 101      -6.134  -7.148  34.293  1.00 33.60           C  \nATOM   1080  C   MET B 101      -5.515  -7.815  33.061  1.00 31.83           C  \nATOM   1081  O   MET B 101      -4.542  -8.570  33.180  1.00 30.35           O  \nATOM   1082  CB  MET B 101      -5.162  -6.108  34.859  1.00 33.04           C  \nATOM   1083  CG  MET B 101      -4.728  -5.108  33.811  1.00 31.86           C  \nATOM   1084  SD  MET B 101      -3.432  -4.010  34.306  1.00 33.35           S  \nATOM   1085  CE  MET B 101      -1.939  -5.151  34.224  1.00 32.77           C  \nATOM   1086  N   ALA B 102      -6.093  -7.538  31.891  1.00 29.69           N  \nATOM   1087  CA  ALA B 102      -5.592  -8.069  30.631  1.00 30.76           C  \nATOM   1088  C   ALA B 102      -4.247  -7.419  30.366  1.00 32.04           C  \nATOM   1089  O   ALA B 102      -3.968  -6.342  30.869  1.00 36.40           O  \nATOM   1090  CB  ALA B 102      -6.550  -7.734  29.487  1.00 26.67           C  \nATOM   1091  N   ILE B 103      -3.411  -8.075  29.579  1.00 33.38           N  \nATOM   1092  CA  ILE B 103      -2.120  -7.512  29.247  1.00 33.44           C  \nATOM   1093  C   ILE B 103      -2.298  -6.668  27.993  1.00 33.27           C  \nATOM   1094  O   ILE B 103      -2.897  -7.106  27.020  1.00 32.92           O  \nATOM   1095  CB  ILE B 103      -1.068  -8.621  28.991  1.00 32.98           C  \nATOM   1096  CG1 ILE B 103      -0.879  -9.463  30.264  1.00 31.41           C  \nATOM   1097  CG2 ILE B 103       0.240  -8.010  28.502  1.00 31.31           C  \nATOM   1098  CD1 ILE B 103      -0.595  -8.666  31.524  1.00 27.59           C  \nATOM   1099  N   THR B 104      -1.786  -5.449  28.029  1.00 35.36           N  \nATOM   1100  CA  THR B 104      -1.888  -4.560  26.896  1.00 37.82           C  \nATOM   1101  C   THR B 104      -0.816  -4.953  25.903  1.00 38.71           C  \nATOM   1102  O   THR B 104       0.320  -5.220  26.279  1.00 38.49           O  \nATOM   1103  CB  THR B 104      -1.637  -3.103  27.301  1.00 40.31           C  \nATOM   1104  OG1 THR B 104      -2.528  -2.741  28.361  1.00 44.00           O  \nATOM   1105  CG2 THR B 104      -1.839  -2.174  26.109  1.00 39.21           C  \nATOM   1106  N   LYS B 105      -1.196  -4.973  24.635  1.00 39.49           N  \nATOM   1107  CA  LYS B 105      -0.301  -5.294  23.545  1.00 41.51           C  \nATOM   1108  C   LYS B 105       0.663  -4.140  23.247  1.00 41.84           C  \nATOM   1109  O   LYS B 105       0.344  -2.964  23.447  1.00 44.61           O  \nATOM   1110  CB  LYS B 105      -1.114  -5.599  22.286  1.00 41.71           C  \nATOM   1111  CG  LYS B 105      -1.842  -6.915  22.358  1.00 50.08           C  \nATOM   1112  CD  LYS B 105      -3.359  -6.720  22.408  1.00 56.77           C  \nATOM   1113  CE  LYS B 105      -3.937  -6.318  21.032  1.00 59.79           C  \nATOM   1114  NZ  LYS B 105      -5.375  -5.887  21.111  1.00 62.22           N  \nATOM   1115  N   ILE B 106       1.850  -4.481  22.770  1.00 39.57           N  \nATOM   1116  CA  ILE B 106       2.821  -3.477  22.402  1.00 36.43           C  \nATOM   1117  C   ILE B 106       2.755  -3.471  20.875  1.00 35.60           C  \nATOM   1118  O   ILE B 106       2.200  -4.394  20.275  1.00 35.47           O  \nATOM   1119  CB  ILE B 106       4.260  -3.828  22.947  1.00 35.74           C  \nATOM   1120  CG1 ILE B 106       4.696  -5.249  22.544  1.00 34.61           C  \nATOM   1121  CG2 ILE B 106       4.265  -3.732  24.470  1.00 31.95           C  \nATOM   1122  CD1 ILE B 106       5.188  -5.393  21.154  1.00 32.37           C  \nATOM   1123  N   ASN B 107       3.286  -2.448  20.227  1.00 33.19           N  \nATOM   1124  CA  ASN B 107       3.209  -2.417  18.775  1.00 33.81           C  \nATOM   1125  C   ASN B 107       4.545  -2.001  18.110  1.00 35.31           C  \nATOM   1126  O   ASN B 107       4.597  -1.685  16.923  1.00 37.56           O  \nATOM   1127  CB  ASN B 107       2.081  -1.477  18.340  1.00 33.19           C  \nATOM   1128  CG  ASN B 107       2.477  -0.040  18.477  1.00 31.76           C  \nATOM   1129  OD1 ASN B 107       3.501   0.228  19.066  1.00 32.02           O  \nATOM   1130  ND2 ASN B 107       1.686   0.885  17.943  1.00 31.21           N  \nATOM   1131  N   ASP B 108       5.627  -1.986  18.877  1.00 34.25           N  \nATOM   1132  CA  ASP B 108       6.918  -1.663  18.280  1.00 33.53           C  \nATOM   1133  C   ASP B 108       8.056  -2.251  19.102  1.00 34.36           C  \nATOM   1134  O   ASP B 108       7.850  -2.669  20.249  1.00 32.78           O  \nATOM   1135  CB  ASP B 108       7.075  -0.152  18.057  1.00 31.84           C  \nATOM   1136  CG  ASP B 108       7.284   0.611  19.328  1.00 31.52           C  \nATOM   1137  OD1 ASP B 108       7.029   1.819  19.337  1.00 30.95           O  \nATOM   1138  OD2 ASP B 108       7.726   0.036  20.312  1.00 31.83           O  \nATOM   1139  N   CYS B 109       9.240  -2.298  18.499  1.00 37.24           N  \nATOM   1140  CA  CYS B 109      10.433  -2.911  19.108  1.00 38.34           C  \nATOM   1141  C   CYS B 109      10.832  -2.431  20.483  1.00 38.76           C  \nATOM   1142  O   CYS B 109      11.111  -3.249  21.367  1.00 39.81           O  \nATOM   1143  CB  CYS B 109      11.640  -2.785  18.175  1.00 37.86           C  \nATOM   1144  SG  CYS B 109      11.379  -3.512  16.546  1.00 38.39           S  \nATOM   1145  N   PHE B 110      10.863  -1.118  20.672  1.00 39.60           N  \nATOM   1146  CA  PHE B 110      11.232  -0.583  21.965  1.00 42.18           C  \nATOM   1147  C   PHE B 110      10.268  -1.014  23.073  1.00 43.08           C  \nATOM   1148  O   PHE B 110      10.686  -1.261  24.213  1.00 44.51           O  \nATOM   1149  CB  PHE B 110      11.319   0.940  21.916  1.00 43.98           C  \nATOM   1150  CG  PHE B 110      11.971   1.536  23.127  1.00 46.68           C  \nATOM   1151  CD1 PHE B 110      11.203   2.096  24.151  1.00 45.50           C  \nATOM   1152  CD2 PHE B 110      13.361   1.498  23.269  1.00 47.64           C  \nATOM   1153  CE1 PHE B 110      11.809   2.611  25.301  1.00 46.11           C  \nATOM   1154  CE2 PHE B 110      13.978   2.010  24.412  1.00 46.74           C  \nATOM   1155  CZ  PHE B 110      13.202   2.568  25.431  1.00 47.04           C  \nATOM   1156  N   GLU B 111       8.984  -1.112  22.759  1.00 42.31           N  \nATOM   1157  CA  GLU B 111       8.022  -1.515  23.772  1.00 40.60           C  \nATOM   1158  C   GLU B 111       8.174  -2.986  24.079  1.00 38.79           C  \nATOM   1159  O   GLU B 111       8.005  -3.404  25.214  1.00 37.94           O  \nATOM   1160  CB  GLU B 111       6.618  -1.200  23.296  1.00 41.40           C  \nATOM   1161  CG  GLU B 111       6.570   0.202  22.836  1.00 42.61           C  \nATOM   1162  CD  GLU B 111       5.214   0.670  22.471  1.00 43.12           C  \nATOM   1163  OE1 GLU B 111       4.458  -0.059  21.799  1.00 43.52           O  \nATOM   1164  OE2 GLU B 111       4.908   1.807  22.851  1.00 46.30           O  \nATOM   1165  N   LEU B 112       8.511  -3.761  23.055  1.00 37.54           N  \nATOM   1166  CA  LEU B 112       8.704  -5.187  23.219  1.00 33.74           C  \nATOM   1167  C   LEU B 112       9.873  -5.364  24.176  1.00 33.32           C  \nATOM   1168  O   LEU B 112       9.764  -6.042  25.192  1.00 34.91           O  \nATOM   1169  CB  LEU B 112       9.056  -5.822  21.884  1.00 33.00           C  \nATOM   1170  CG  LEU B 112       8.723  -7.300  21.692  1.00 32.06           C  \nATOM   1171  CD1 LEU B 112       9.767  -7.918  20.780  1.00 31.53           C  \nATOM   1172  CD2 LEU B 112       8.652  -8.027  23.026  1.00 30.35           C  \nATOM   1173  N   LEU B 113      10.989  -4.721  23.849  1.00 32.31           N  \nATOM   1174  CA  LEU B 113      12.188  -4.820  24.651  1.00 31.20           C  \nATOM   1175  C   LEU B 113      11.906  -4.483  26.119  1.00 32.18           C  \nATOM   1176  O   LEU B 113      12.335  -5.214  27.033  1.00 30.15           O  \nATOM   1177  CB  LEU B 113      13.270  -3.904  24.073  1.00 32.11           C  \nATOM   1178  CG  LEU B 113      14.576  -3.830  24.883  1.00 35.66           C  \nATOM   1179  CD1 LEU B 113      15.142  -5.234  25.080  1.00 36.67           C  \nATOM   1180  CD2 LEU B 113      15.589  -2.932  24.169  1.00 35.74           C  \nATOM   1181  N   SER B 114      11.178  -3.393  26.352  1.00 31.44           N  \nATOM   1182  CA  SER B 114      10.859  -2.999  27.712  1.00 32.02           C  \nATOM   1183  C   SER B 114      10.031  -4.068  28.390  1.00 32.18           C  \nATOM   1184  O   SER B 114      10.295  -4.436  29.536  1.00 35.84           O  \nATOM   1185  CB  SER B 114      10.076  -1.700  27.738  1.00 30.73           C  \nATOM   1186  OG  SER B 114      10.805  -0.679  27.096  1.00 32.00           O  \nATOM   1187  N   MET B 115       9.030  -4.578  27.684  1.00 30.28           N  \nATOM   1188  CA  MET B 115       8.176  -5.594  28.276  1.00 29.84           C  \nATOM   1189  C   MET B 115       9.001  -6.796  28.719  1.00 29.27           C  \nATOM   1190  O   MET B 115       8.864  -7.277  29.839  1.00 29.56           O  \nATOM   1191  CB  MET B 115       7.082  -6.019  27.301  1.00 30.08           C  \nATOM   1192  CG  MET B 115       6.114  -7.005  27.912  1.00 32.99           C  \nATOM   1193  SD  MET B 115       4.729  -7.438  26.864  1.00 38.71           S  \nATOM   1194  CE  MET B 115       3.458  -6.305  27.462  1.00 37.80           C  \nATOM   1195  N   VAL B 116       9.877  -7.257  27.843  1.00 30.62           N  \nATOM   1196  CA  VAL B 116      10.737  -8.391  28.140  1.00 30.76           C  \nATOM   1197  C   VAL B 116      11.674  -8.081  29.315  1.00 33.07           C  \nATOM   1198  O   VAL B 116      11.833  -8.888  30.233  1.00 35.37           O  \nATOM   1199  CB  VAL B 116      11.571  -8.759  26.914  1.00 29.70           C  \nATOM   1200  CG1 VAL B 116      12.637  -9.758  27.311  1.00 29.90           C  \nATOM   1201  CG2 VAL B 116      10.640  -9.325  25.813  1.00 27.28           C  \nATOM   1202  N   THR B 117      12.288  -6.901  29.292  1.00 33.75           N  \nATOM   1203  CA  THR B 117      13.186  -6.522  30.375  1.00 34.17           C  \nATOM   1204  C   THR B 117      12.427  -6.597  31.701  1.00 33.98           C  \nATOM   1205  O   THR B 117      12.909  -7.181  32.677  1.00 34.88           O  \nATOM   1206  CB  THR B 117      13.714  -5.086  30.192  1.00 34.71           C  \nATOM   1207  OG1 THR B 117      14.423  -4.989  28.948  1.00 35.00           O  \nATOM   1208  CG2 THR B 117      14.630  -4.704  31.340  1.00 33.90           C  \nATOM   1209  N   TYR B 118      11.238  -6.007  31.740  1.00 32.20           N  \nATOM   1210  CA  TYR B 118      10.438  -6.017  32.951  1.00 31.83           C  \nATOM   1211  C   TYR B 118       9.995  -7.447  33.330  1.00 31.49           C  \nATOM   1212  O   TYR B 118       9.993  -7.832  34.500  1.00 30.74           O  \nATOM   1213  CB  TYR B 118       9.230  -5.091  32.751  1.00 30.78           C  \nATOM   1214  CG  TYR B 118       8.158  -5.182  33.815  1.00 30.23           C  \nATOM   1215  CD1 TYR B 118       8.423  -4.839  35.141  1.00 30.02           C  \nATOM   1216  CD2 TYR B 118       6.871  -5.635  33.489  1.00 29.87           C  \nATOM   1217  CE1 TYR B 118       7.425  -4.947  36.126  1.00 30.64           C  \nATOM   1218  CE2 TYR B 118       5.866  -5.749  34.451  1.00 29.16           C  \nATOM   1219  CZ  TYR B 118       6.141  -5.406  35.770  1.00 30.09           C  \nATOM   1220  OH  TYR B 118       5.132  -5.516  36.710  1.00 28.17           O  \nATOM   1221  N   ALA B 119       9.633  -8.251  32.343  1.00 31.73           N  \nATOM   1222  CA  ALA B 119       9.195  -9.604  32.653  1.00 33.79           C  \nATOM   1223  C   ALA B 119      10.362 -10.469  33.126  1.00 35.43           C  \nATOM   1224  O   ALA B 119      10.202 -11.291  34.038  1.00 34.66           O  \nATOM   1225  CB  ALA B 119       8.510 -10.244  31.426  1.00 34.26           C  \nATOM   1226  N   ASP B 120      11.531 -10.293  32.511  1.00 38.35           N  \nATOM   1227  CA  ASP B 120      12.697 -11.082  32.899  1.00 42.05           C  \nATOM   1228  C   ASP B 120      13.082 -10.792  34.347  1.00 42.26           C  \nATOM   1229  O   ASP B 120      13.410 -11.708  35.103  1.00 40.84           O  \nATOM   1230  CB  ASP B 120      13.896 -10.802  31.989  1.00 46.28           C  \nATOM   1231  CG  ASP B 120      13.788 -11.487  30.629  1.00 48.74           C  \nATOM   1232  OD1 ASP B 120      13.003 -12.452  30.495  1.00 49.75           O  \nATOM   1233  OD2 ASP B 120      14.506 -11.068  29.694  1.00 50.19           O  \nATOM   1234  N   LYS B 121      13.038  -9.521  34.737  1.00 43.31           N  \nATOM   1235  CA  LYS B 121      13.380  -9.164  36.103  1.00 43.62           C  \nATOM   1236  C   LYS B 121      12.442  -9.838  37.094  1.00 42.71           C  \nATOM   1237  O   LYS B 121      12.893 -10.441  38.065  1.00 45.32           O  \nATOM   1238  CB  LYS B 121      13.362  -7.647  36.295  1.00 44.66           C  \nATOM   1239  CG  LYS B 121      14.497  -6.955  35.573  1.00 49.21           C  \nATOM   1240  CD  LYS B 121      14.484  -5.451  35.773  1.00 51.79           C  \nATOM   1241  CE  LYS B 121      15.577  -4.791  34.932  1.00 53.53           C  \nATOM   1242  NZ  LYS B 121      15.369  -3.315  34.821  1.00 55.92           N  \nATOM   1243  N   LEU B 122      11.138  -9.760  36.864  1.00 40.70           N  \nATOM   1244  CA  LEU B 122      10.229 -10.404  37.788  1.00 38.48           C  \nATOM   1245  C   LEU B 122      10.437 -11.919  37.792  1.00 38.89           C  \nATOM   1246  O   LEU B 122      10.333 -12.554  38.836  1.00 37.55           O  \nATOM   1247  CB  LEU B 122       8.781 -10.060  37.449  1.00 38.06           C  \nATOM   1248  CG  LEU B 122       8.355  -8.591  37.600  1.00 37.89           C  \nATOM   1249  CD1 LEU B 122       6.859  -8.469  37.373  1.00 37.43           C  \nATOM   1250  CD2 LEU B 122       8.696  -8.077  38.998  1.00 38.84           C  \nATOM   1251  N   LYS B 123      10.739 -12.506  36.634  1.00 41.30           N  \nATOM   1252  CA  LYS B 123      10.959 -13.952  36.573  1.00 44.06           C  \nATOM   1253  C   LYS B 123      12.177 -14.385  37.382  1.00 47.10           C  \nATOM   1254  O   LYS B 123      12.095 -15.313  38.187  1.00 48.12           O  \nATOM   1255  CB  LYS B 123      11.156 -14.429  35.137  1.00 43.43           C  \nATOM   1256  CG  LYS B 123       9.917 -14.427  34.298  1.00 45.59           C  \nATOM   1257  CD  LYS B 123      10.015 -15.453  33.163  1.00 47.11           C  \nATOM   1258  CE  LYS B 123      11.198 -15.195  32.267  1.00 47.77           C  \nATOM   1259  NZ  LYS B 123      11.242 -16.167  31.135  1.00 51.36           N  \nATOM   1260  N   SER B 124      13.306 -13.715  37.157  1.00 49.52           N  \nATOM   1261  CA  SER B 124      14.541 -14.051  37.849  1.00 51.17           C  \nATOM   1262  C   SER B 124      14.399 -13.882  39.352  1.00 53.09           C  \nATOM   1263  O   SER B 124      14.975 -14.651  40.119  1.00 55.98           O  \nATOM   1264  CB  SER B 124      15.695 -13.184  37.340  1.00 49.53           C  \nATOM   1265  OG  SER B 124      15.557 -11.849  37.789  1.00 48.98           O  \nATOM   1266  N   LEU B 125      13.637 -12.876  39.774  1.00 53.07           N  \nATOM   1267  CA  LEU B 125      13.429 -12.642  41.204  1.00 52.85           C  \nATOM   1268  C   LEU B 125      12.653 -13.798  41.814  1.00 51.39           C  \nATOM   1269  O   LEU B 125      13.076 -14.398  42.800  1.00 51.30           O  \nATOM   1270  CB  LEU B 125      12.666 -11.339  41.439  1.00 54.40           C  \nATOM   1271  CG  LEU B 125      13.564 -10.106  41.554  1.00 58.89           C  \nATOM   1272  CD1 LEU B 125      12.742  -8.821  41.405  1.00 59.75           C  \nATOM   1273  CD2 LEU B 125      14.286 -10.152  42.914  1.00 58.61           C  \nATOM   1274  N   ILE B 126      11.515 -14.113  41.220  1.00 50.77           N  \nATOM   1275  CA  ILE B 126      10.697 -15.201  41.714  1.00 51.91           C  \nATOM   1276  C   ILE B 126      11.533 -16.468  41.813  1.00 54.08           C  \nATOM   1277  O   ILE B 126      11.363 -17.260  42.738  1.00 54.49           O  \nATOM   1278  CB  ILE B 126       9.479 -15.424  40.792  1.00 51.79           C  \nATOM   1279  CG1 ILE B 126       8.507 -14.252  40.963  1.00 50.75           C  \nATOM   1280  CG2 ILE B 126       8.808 -16.755  41.096  1.00 50.54           C  \nATOM   1281  CD1 ILE B 126       7.316 -14.300  40.036  1.00 52.63           C  \nATOM   1282  N   LYS B 127      12.443 -16.658  40.862  1.00 57.22           N  \nATOM   1283  CA  LYS B 127      13.304 -17.835  40.881  1.00 60.96           C  \nATOM   1284  C   LYS B 127      14.376 -17.696  41.959  1.00 62.04           C  \nATOM   1285  O   LYS B 127      14.520 -18.572  42.803  1.00 62.97           O  \nATOM   1286  CB  LYS B 127      13.942 -18.041  39.511  1.00 62.82           C  \nATOM   1287  CG  LYS B 127      12.909 -18.260  38.414  1.00 66.66           C  \nATOM   1288  CD  LYS B 127      13.539 -18.600  37.072  1.00 68.29           C  \nATOM   1289  CE  LYS B 127      12.456 -18.839  36.018  1.00 70.88           C  \nATOM   1290  NZ  LYS B 127      13.001 -19.323  34.711  1.00 73.58           N  \nATOM   1291  N   LYS B 128      15.115 -16.588  41.935  1.00 63.90           N  \nATOM   1292  CA  LYS B 128      16.159 -16.321  42.924  1.00 65.17           C  \nATOM   1293  C   LYS B 128      15.558 -16.585  44.298  1.00 65.33           C  \nATOM   1294  O   LYS B 128      15.920 -17.547  44.967  1.00 66.74           O  \nATOM   1295  CB  LYS B 128      16.617 -14.869  42.806  1.00 67.98           C  \nATOM   1296  CG  LYS B 128      17.773 -14.479  43.709  1.00 72.48           C  \nATOM   1297  CD  LYS B 128      18.226 -13.052  43.406  1.00 76.62           C  \nATOM   1298  CE  LYS B 128      19.384 -12.622  44.299  1.00 78.99           C  \nATOM   1299  NZ  LYS B 128      19.855 -11.234  43.991  1.00 79.30           N  \nATOM   1300  N   GLU B 129      14.643 -15.724  44.721  1.00 65.75           N  \nATOM   1301  CA  GLU B 129      13.950 -15.914  45.992  1.00 65.77           C  \nATOM   1302  C   GLU B 129      12.979 -17.035  45.656  1.00 64.14           C  \nATOM   1303  O   GLU B 129      12.881 -17.407  44.491  1.00 66.77           O  \nATOM   1304  CB  GLU B 129      13.206 -14.635  46.363  1.00 67.66           C  \nATOM   1305  CG  GLU B 129      14.155 -13.498  46.710  1.00 72.98           C  \nATOM   1306  CD  GLU B 129      13.462 -12.162  46.876  1.00 74.95           C  \nATOM   1307  OE1 GLU B 129      13.187 -11.507  45.852  1.00 75.92           O  \nATOM   1308  OE2 GLU B 129      13.191 -11.767  48.033  1.00 77.31           O  \nATOM   1309  N   PHE B 130      12.277 -17.594  46.630  1.00 61.52           N  \nATOM   1310  CA  PHE B 130      11.329 -18.679  46.331  1.00 61.59           C  \nATOM   1311  C   PHE B 130      11.961 -19.896  45.658  1.00 62.23           C  \nATOM   1312  O   PHE B 130      11.365 -20.970  45.638  1.00 61.41           O  \nATOM   1313  CB  PHE B 130      10.199 -18.180  45.428  1.00 60.56           C  \nATOM   1314  CG  PHE B 130       9.399 -17.077  46.026  1.00 60.04           C  \nATOM   1315  CD1 PHE B 130       8.548 -17.327  47.088  1.00 60.29           C  \nATOM   1316  CD2 PHE B 130       9.535 -15.776  45.564  1.00 59.34           C  \nATOM   1317  CE1 PHE B 130       7.845 -16.294  47.692  1.00 60.83           C  \nATOM   1318  CE2 PHE B 130       8.840 -14.740  46.159  1.00 60.10           C  \nATOM   1319  CZ  PHE B 130       7.992 -14.998  47.227  1.00 61.06           C  \nATOM   1320  N   SER B 131      13.157 -19.717  45.107  1.00 64.52           N  \nATOM   1321  CA  SER B 131      13.879 -20.782  44.421  1.00 66.17           C  \nATOM   1322  C   SER B 131      12.911 -21.789  43.820  1.00 66.12           C  \nATOM   1323  O   SER B 131      12.613 -22.826  44.413  1.00 66.91           O  \nATOM   1324  CB  SER B 131      14.872 -21.467  45.378  1.00 66.10           C  \nATOM   1325  OG  SER B 131      14.272 -21.799  46.613  1.00 68.67           O  \nATOM   1326  N   ILE B 132      12.421 -21.453  42.632  1.00 65.56           N  \nATOM   1327  CA  ILE B 132      11.464 -22.277  41.910  1.00 65.17           C  \nATOM   1328  C   ILE B 132      11.543 -21.881  40.436  1.00 65.51           C  \nATOM   1329  O   ILE B 132      11.742 -20.707  40.124  1.00 66.94           O  \nATOM   1330  CB  ILE B 132      10.036 -22.023  42.447  1.00 65.32           C  \nATOM   1331  CG1 ILE B 132       9.039 -22.938  41.752  1.00 66.40           C  \nATOM   1332  CG2 ILE B 132       9.638 -20.574  42.221  1.00 64.93           C  \nATOM   1333  CD1 ILE B 132       7.607 -22.699  42.198  1.00 67.71           C  \nATOM   1334  N   SER B 133      11.410 -22.849  39.533  1.00 64.79           N  \nATOM   1335  CA  SER B 133      11.467 -22.555  38.101  1.00 63.57           C  \nATOM   1336  C   SER B 133      10.163 -21.894  37.657  1.00 62.11           C  \nATOM   1337  O   SER B 133       9.139 -21.994  38.336  1.00 60.94           O  \nATOM   1338  CB  SER B 133      11.687 -23.836  37.294  1.00 64.17           C  \nATOM   1339  OG  SER B 133      10.537 -24.655  37.328  1.00 64.71           O  \nATOM   1340  N   PHE B 134      10.189 -21.226  36.512  1.00 60.83           N  \nATOM   1341  CA  PHE B 134       8.981 -20.568  36.058  1.00 59.57           C  \nATOM   1342  C   PHE B 134       7.886 -21.544  35.665  1.00 59.03           C  \nATOM   1343  O   PHE B 134       6.695 -21.235  35.827  1.00 58.73           O  \nATOM   1344  CB  PHE B 134       9.268 -19.603  34.907  1.00 59.55           C  \nATOM   1345  CG  PHE B 134       8.376 -18.398  34.924  1.00 58.26           C  \nATOM   1346  CD1 PHE B 134       8.443 -17.494  35.984  1.00 57.76           C  \nATOM   1347  CD2 PHE B 134       7.412 -18.210  33.940  1.00 57.55           C  \nATOM   1348  CE1 PHE B 134       7.565 -16.426  36.071  1.00 56.67           C  \nATOM   1349  CE2 PHE B 134       6.526 -17.149  34.014  1.00 56.49           C  \nATOM   1350  CZ  PHE B 134       6.601 -16.252  35.087  1.00 57.33           C  \nATOM   1351  N   GLU B 135       8.267 -22.716  35.154  1.00 57.29           N  \nATOM   1352  CA  GLU B 135       7.264 -23.710  34.790  1.00 56.76           C  \nATOM   1353  C   GLU B 135       6.560 -24.164  36.061  1.00 55.27           C  \nATOM   1354  O   GLU B 135       5.356 -24.411  36.049  1.00 55.17           O  \nATOM   1355  CB  GLU B 135       7.888 -24.914  34.077  1.00 59.82           C  \nATOM   1356  CG  GLU B 135       9.396 -25.037  34.219  1.00 66.47           C  \nATOM   1357  CD  GLU B 135      10.150 -24.064  33.331  1.00 69.23           C  \nATOM   1358  OE1 GLU B 135      10.033 -24.169  32.088  1.00 70.06           O  \nATOM   1359  OE2 GLU B 135      10.858 -23.195  33.882  1.00 72.74           O  \nATOM   1360  N   GLU B 136       7.301 -24.269  37.161  1.00 53.13           N  \nATOM   1361  CA  GLU B 136       6.690 -24.679  38.421  1.00 53.51           C  \nATOM   1362  C   GLU B 136       5.710 -23.586  38.826  1.00 52.11           C  \nATOM   1363  O   GLU B 136       4.649 -23.856  39.391  1.00 52.47           O  \nATOM   1364  CB  GLU B 136       7.744 -24.835  39.526  1.00 55.76           C  \nATOM   1365  CG  GLU B 136       8.735 -25.976  39.352  1.00 58.84           C  \nATOM   1366  CD  GLU B 136       9.802 -25.970  40.442  1.00 60.34           C  \nATOM   1367  OE1 GLU B 136       9.449 -26.153  41.629  1.00 63.16           O  \nATOM   1368  OE2 GLU B 136      10.993 -25.773  40.121  1.00 60.87           O  \nATOM   1369  N   PHE B 137       6.103 -22.345  38.539  1.00 50.85           N  \nATOM   1370  CA  PHE B 137       5.305 -21.159  38.838  1.00 48.99           C  \nATOM   1371  C   PHE B 137       4.034 -21.192  37.989  1.00 48.09           C  \nATOM   1372  O   PHE B 137       2.938 -20.988  38.495  1.00 48.04           O  \nATOM   1373  CB  PHE B 137       6.126 -19.904  38.517  1.00 49.31           C  \nATOM   1374  CG  PHE B 137       5.382 -18.616  38.710  1.00 50.19           C  \nATOM   1375  CD1 PHE B 137       5.079 -18.155  39.983  1.00 49.13           C  \nATOM   1376  CD2 PHE B 137       4.990 -17.856  37.604  1.00 49.47           C  \nATOM   1377  CE1 PHE B 137       4.393 -16.951  40.156  1.00 48.63           C  \nATOM   1378  CE2 PHE B 137       4.306 -16.654  37.761  1.00 47.40           C  \nATOM   1379  CZ  PHE B 137       4.007 -16.200  39.038  1.00 48.58           C  \nATOM   1380  N   ALA B 138       4.195 -21.459  36.697  1.00 47.54           N  \nATOM   1381  CA  ALA B 138       3.062 -21.528  35.774  1.00 48.15           C  \nATOM   1382  C   ALA B 138       2.065 -22.630  36.156  1.00 49.31           C  \nATOM   1383  O   ALA B 138       0.842 -22.444  36.054  1.00 50.18           O  \nATOM   1384  CB  ALA B 138       3.555 -21.754  34.360  1.00 45.13           C  \nATOM   1385  N   VAL B 139       2.575 -23.776  36.596  1.00 50.04           N  \nATOM   1386  CA  VAL B 139       1.697 -24.864  36.982  1.00 49.65           C  \nATOM   1387  C   VAL B 139       0.969 -24.427  38.234  1.00 51.28           C  \nATOM   1388  O   VAL B 139      -0.254 -24.551  38.341  1.00 51.02           O  \nATOM   1389  CB  VAL B 139       2.490 -26.163  37.261  1.00 48.66           C  \nATOM   1390  CG1 VAL B 139       1.608 -27.187  37.958  1.00 48.02           C  \nATOM   1391  CG2 VAL B 139       3.008 -26.743  35.950  1.00 47.52           C  \nATOM   1392  N   LEU B 140       1.726 -23.902  39.184  1.00 54.51           N  \nATOM   1393  CA  LEU B 140       1.135 -23.450  40.428  1.00 57.56           C  \nATOM   1394  C   LEU B 140       0.063 -22.374  40.249  1.00 59.24           C  \nATOM   1395  O   LEU B 140      -0.989 -22.460  40.887  1.00 57.43           O  \nATOM   1396  CB  LEU B 140       2.221 -22.945  41.379  1.00 58.05           C  \nATOM   1397  CG  LEU B 140       3.124 -24.012  41.996  1.00 57.91           C  \nATOM   1398  CD1 LEU B 140       3.879 -23.399  43.169  1.00 59.58           C  \nATOM   1399  CD2 LEU B 140       2.286 -25.199  42.477  1.00 58.02           C  \nATOM   1400  N   THR B 141       0.298 -21.371  39.396  1.00 61.08           N  \nATOM   1401  CA  THR B 141      -0.720 -20.332  39.234  1.00 61.87           C  \nATOM   1402  C   THR B 141      -1.890 -20.854  38.399  1.00 63.65           C  \nATOM   1403  O   THR B 141      -3.026 -20.413  38.574  1.00 63.26           O  \nATOM   1404  CB  THR B 141      -0.158 -18.989  38.630  1.00 59.88           C  \nATOM   1405  OG1 THR B 141      -0.358 -18.960  37.221  1.00 58.73           O  \nATOM   1406  CG2 THR B 141       1.311 -18.829  38.919  1.00 58.72           C  \nATOM   1407  N   TYR B 142      -1.626 -21.804  37.508  1.00 66.01           N  \nATOM   1408  CA  TYR B 142      -2.711 -22.379  36.718  1.00 68.57           C  \nATOM   1409  C   TYR B 142      -3.658 -23.110  37.672  1.00 67.37           C  \nATOM   1410  O   TYR B 142      -4.882 -23.006  37.568  1.00 66.20           O  \nATOM   1411  CB  TYR B 142      -2.186 -23.383  35.696  1.00 73.00           C  \nATOM   1412  CG  TYR B 142      -3.308 -24.081  34.974  1.00 78.42           C  \nATOM   1413  CD1 TYR B 142      -3.885 -23.521  33.835  1.00 80.98           C  \nATOM   1414  CD2 TYR B 142      -3.859 -25.261  35.482  1.00 81.51           C  \nATOM   1415  CE1 TYR B 142      -4.992 -24.113  33.218  1.00 83.48           C  \nATOM   1416  CE2 TYR B 142      -4.966 -25.862  34.875  1.00 84.46           C  \nATOM   1417  CZ  TYR B 142      -5.529 -25.282  33.745  1.00 84.53           C  \nATOM   1418  OH  TYR B 142      -6.633 -25.865  33.158  1.00 85.96           O  \nATOM   1419  N   ILE B 143      -3.077 -23.860  38.600  1.00 66.82           N  \nATOM   1420  CA  ILE B 143      -3.855 -24.608  39.572  1.00 66.27           C  \nATOM   1421  C   ILE B 143      -4.666 -23.656  40.430  1.00 67.57           C  \nATOM   1422  O   ILE B 143      -5.828 -23.925  40.746  1.00 68.46           O  \nATOM   1423  CB  ILE B 143      -2.934 -25.469  40.481  1.00 64.70           C  \nATOM   1424  CG1 ILE B 143      -2.502 -26.734  39.734  1.00 63.42           C  \nATOM   1425  CG2 ILE B 143      -3.642 -25.828  41.779  1.00 61.32           C  \nATOM   1426  CD1 ILE B 143      -1.504 -27.576  40.497  1.00 63.09           C  \nATOM   1427  N   SER B 144      -4.061 -22.532  40.798  1.00 68.52           N  \nATOM   1428  CA  SER B 144      -4.748 -21.557  41.636  1.00 70.14           C  \nATOM   1429  C   SER B 144      -5.894 -20.830  40.927  1.00 71.77           C  \nATOM   1430  O   SER B 144      -6.871 -20.430  41.564  1.00 70.98           O  \nATOM   1431  CB  SER B 144      -3.753 -20.534  42.178  1.00 67.95           C  \nATOM   1432  OG  SER B 144      -4.363 -19.738  43.182  1.00 67.81           O  \nATOM   1433  N   GLU B 145      -5.776 -20.664  39.615  1.00 73.79           N  \nATOM   1434  CA  GLU B 145      -6.802 -19.979  38.843  1.00 76.78           C  \nATOM   1435  C   GLU B 145      -8.048 -20.817  38.629  1.00 79.98           C  \nATOM   1436  O   GLU B 145      -9.174 -20.327  38.760  1.00 81.10           O  \nATOM   1437  CB  GLU B 145      -6.246 -19.558  37.485  1.00 74.43           C  \nATOM   1438  CG  GLU B 145      -5.797 -18.120  37.455  1.00 72.41           C  \nATOM   1439  CD  GLU B 145      -6.946 -17.161  37.653  1.00 70.69           C  \nATOM   1440  OE1 GLU B 145      -7.768 -17.017  36.724  1.00 67.23           O  \nATOM   1441  OE2 GLU B 145      -7.031 -16.565  38.746  1.00 71.58           O  \nATOM   1442  N   ASN B 146      -7.833 -22.081  38.286  1.00 83.58           N  \nATOM   1443  CA  ASN B 146      -8.922 -23.010  38.033  1.00 86.93           C  \nATOM   1444  C   ASN B 146      -8.988 -24.001  39.189  1.00 90.19           C  \nATOM   1445  O   ASN B 146      -8.705 -25.186  39.019  1.00 92.23           O  \nATOM   1446  CB  ASN B 146      -8.655 -23.736  36.719  1.00 85.24           C  \nATOM   1447  CG  ASN B 146      -8.185 -22.791  35.630  1.00 85.25           C  \nATOM   1448  OD1 ASN B 146      -8.947 -21.953  35.145  1.00 85.55           O  \nATOM   1449  ND2 ASN B 146      -6.918 -22.910  35.250  1.00 84.94           N  \nATOM   1450  N   LYS B 147      -9.352 -23.503  40.366  1.00 92.76           N  \nATOM   1451  CA  LYS B 147      -9.447 -24.335  41.558  1.00 94.53           C  \nATOM   1452  C   LYS B 147     -10.132 -25.663  41.254  1.00 96.96           C  \nATOM   1453  O   LYS B 147     -11.133 -25.707  40.536  1.00 98.99           O  \nATOM   1454  CB  LYS B 147     -10.211 -23.593  42.656  1.00 92.72           C  \nATOM   1455  CG  LYS B 147      -9.547 -22.301  43.102  1.00 92.12           C  \nATOM   1456  CD  LYS B 147     -10.462 -21.089  42.919  1.00 91.74           C  \nATOM   1457  CE  LYS B 147     -10.703 -20.749  41.447  1.00 91.23           C  \nATOM   1458  NZ  LYS B 147     -11.470 -21.790  40.704  1.00 90.72           N  \nATOM   1459  N   GLU B 148      -9.584 -26.742  41.803  1.00 98.47           N  \nATOM   1460  CA  GLU B 148     -10.126 -28.077  41.590  1.00 99.69           C  \nATOM   1461  C   GLU B 148      -9.503 -29.020  42.617  1.00100.14           C  \nATOM   1462  O   GLU B 148      -8.651 -28.616  43.408  1.00100.67           O  \nATOM   1463  CB  GLU B 148      -9.779 -28.569  40.179  1.00100.12           C  \nATOM   1464  CG  GLU B 148     -10.919 -29.231  39.427  1.00102.41           C  \nATOM   1465  CD  GLU B 148     -11.718 -28.242  38.599  1.00103.70           C  \nATOM   1466  OE1 GLU B 148     -11.130 -27.615  37.688  1.00103.43           O  \nATOM   1467  OE2 GLU B 148     -12.931 -28.092  38.857  1.00105.03           O  \nATOM   1468  N   LYS B 149      -9.938 -30.275  42.599  1.00100.47           N  \nATOM   1469  CA  LYS B 149      -9.420 -31.297  43.503  1.00100.57           C  \nATOM   1470  C   LYS B 149      -8.653 -32.295  42.647  1.00101.30           C  \nATOM   1471  O   LYS B 149      -7.720 -32.952  43.107  1.00100.75           O  \nATOM   1472  CB  LYS B 149     -10.568 -32.024  44.198  1.00100.87           C  \nATOM   1473  CG  LYS B 149     -11.320 -31.212  45.229  1.00101.21           C  \nATOM   1474  CD  LYS B 149     -10.579 -31.189  46.547  1.00102.84           C  \nATOM   1475  CE  LYS B 149     -11.457 -30.626  47.652  1.00103.76           C  \nATOM   1476  NZ  LYS B 149     -12.738 -31.373  47.766  1.00103.94           N  \nATOM   1477  N   GLU B 150      -9.066 -32.387  41.388  1.00102.64           N  \nATOM   1478  CA  GLU B 150      -8.471 -33.300  40.424  1.00104.22           C  \nATOM   1479  C   GLU B 150      -8.130 -32.588  39.119  1.00103.19           C  \nATOM   1480  O   GLU B 150      -8.985 -31.939  38.516  1.00102.42           O  \nATOM   1481  CB  GLU B 150      -9.444 -34.449  40.141  1.00107.82           C  \nATOM   1482  CG  GLU B 150     -10.870 -34.226  40.674  1.00111.96           C  \nATOM   1483  CD  GLU B 150     -11.491 -32.908  40.217  1.00114.28           C  \nATOM   1484  OE1 GLU B 150     -11.660 -32.710  38.993  1.00115.48           O  \nATOM   1485  OE2 GLU B 150     -11.812 -32.070  41.090  1.00115.57           O  \nATOM   1486  N   TYR B 151      -6.878 -32.712  38.687  1.00101.99           N  \nATOM   1487  CA  TYR B 151      -6.437 -32.084  37.446  1.00101.29           C  \nATOM   1488  C   TYR B 151      -5.817 -33.093  36.490  1.00101.42           C  \nATOM   1489  O   TYR B 151      -5.092 -33.996  36.905  1.00 99.77           O  \nATOM   1490  CB  TYR B 151      -5.405 -30.981  37.716  1.00100.38           C  \nATOM   1491  CG  TYR B 151      -5.889 -29.844  38.583  1.00 99.29           C  \nATOM   1492  CD1 TYR B 151      -5.868 -29.945  39.974  1.00 98.83           C  \nATOM   1493  CD2 TYR B 151      -6.351 -28.657  38.016  1.00 98.62           C  \nATOM   1494  CE1 TYR B 151      -6.291 -28.892  40.779  1.00 97.55           C  \nATOM   1495  CE2 TYR B 151      -6.776 -27.598  38.813  1.00 98.51           C  \nATOM   1496  CZ  TYR B 151      -6.742 -27.724  40.194  1.00 97.85           C  \nATOM   1497  OH  TYR B 151      -7.153 -26.683  40.993  1.00 97.38           O  \nATOM   1498  N   TYR B 152      -6.104 -32.921  35.205  1.00103.65           N  \nATOM   1499  CA  TYR B 152      -5.568 -33.787  34.165  1.00106.33           C  \nATOM   1500  C   TYR B 152      -4.267 -33.188  33.657  1.00105.26           C  \nATOM   1501  O   TYR B 152      -4.276 -32.137  33.018  1.00104.37           O  \nATOM   1502  CB  TYR B 152      -6.555 -33.895  33.000  1.00110.90           C  \nATOM   1503  CG  TYR B 152      -7.848 -34.603  33.339  1.00115.36           C  \nATOM   1504  CD1 TYR B 152      -8.647 -34.175  34.404  1.00116.47           C  \nATOM   1505  CD2 TYR B 152      -8.277 -35.705  32.592  1.00116.63           C  \nATOM   1506  CE1 TYR B 152      -9.838 -34.827  34.720  1.00118.04           C  \nATOM   1507  CE2 TYR B 152      -9.466 -36.363  32.898  1.00118.22           C  \nATOM   1508  CZ  TYR B 152     -10.241 -35.920  33.963  1.00118.79           C  \nATOM   1509  OH  TYR B 152     -11.416 -36.569  34.272  1.00119.39           O  \nATOM   1510  N   LEU B 153      -3.150 -33.850  33.939  1.00104.46           N  \nATOM   1511  CA  LEU B 153      -1.857 -33.356  33.485  1.00103.78           C  \nATOM   1512  C   LEU B 153      -1.918 -33.046  31.992  1.00102.93           C  \nATOM   1513  O   LEU B 153      -1.077 -32.325  31.463  1.00102.21           O  \nATOM   1514  CB  LEU B 153      -0.760 -34.386  33.757  1.00104.67           C  \nATOM   1515  CG  LEU B 153       0.638 -34.003  33.260  1.00105.17           C  \nATOM   1516  CD1 LEU B 153       1.107 -32.740  33.962  1.00106.26           C  \nATOM   1517  CD2 LEU B 153       1.608 -35.140  33.519  1.00106.21           C  \nATOM   1518  N   LYS B 154      -2.919 -33.600  31.318  1.00102.48           N  \nATOM   1519  CA  LYS B 154      -3.096 -33.363  29.893  1.00103.08           C  \nATOM   1520  C   LYS B 154      -3.452 -31.897  29.680  1.00102.74           C  \nATOM   1521  O   LYS B 154      -2.769 -31.177  28.949  1.00101.72           O  \nATOM   1522  CB  LYS B 154      -4.224 -34.241  29.344  1.00104.55           C  \nATOM   1523  CG  LYS B 154      -4.561 -33.978  27.880  1.00106.48           C  \nATOM   1524  CD  LYS B 154      -5.832 -34.704  27.460  1.00107.22           C  \nATOM   1525  CE  LYS B 154      -6.134 -34.502  25.979  1.00106.87           C  \nATOM   1526  NZ  LYS B 154      -6.297 -33.067  25.620  1.00105.97           N  \nATOM   1527  N   ASP B 155      -4.529 -31.468  30.331  1.00102.51           N  \nATOM   1528  CA  ASP B 155      -5.008 -30.095  30.228  1.00101.68           C  \nATOM   1529  C   ASP B 155      -3.928 -29.087  30.601  1.00100.07           C  \nATOM   1530  O   ASP B 155      -3.628 -28.174  29.829  1.00 99.22           O  \nATOM   1531  CB  ASP B 155      -6.229 -29.897  31.129  1.00103.51           C  \nATOM   1532  CG  ASP B 155      -7.312 -30.925  30.872  1.00105.39           C  \nATOM   1533  OD1 ASP B 155      -7.655 -31.142  29.689  1.00105.61           O  \nATOM   1534  OD2 ASP B 155      -7.822 -31.513  31.850  1.00106.56           O  \nATOM   1535  N   ILE B 156      -3.348 -29.255  31.786  1.00 97.92           N  \nATOM   1536  CA  ILE B 156      -2.300 -28.359  32.254  1.00 96.27           C  \nATOM   1537  C   ILE B 156      -1.254 -28.122  31.171  1.00 96.49           C  \nATOM   1538  O   ILE B 156      -0.923 -26.981  30.859  1.00 97.06           O  \nATOM   1539  CB  ILE B 156      -1.607 -28.922  33.514  1.00 95.22           C  \nATOM   1540  CG1 ILE B 156      -2.602 -28.952  34.678  1.00 94.37           C  \nATOM   1541  CG2 ILE B 156      -0.388 -28.078  33.864  1.00 94.43           C  \nATOM   1542  CD1 ILE B 156      -2.032 -29.492  35.971  1.00 93.55           C  \nATOM   1543  N   ILE B 157      -0.742 -29.201  30.591  1.00 96.79           N  \nATOM   1544  CA  ILE B 157       0.264 -29.086  29.542  1.00 96.97           C  \nATOM   1545  C   ILE B 157      -0.286 -28.307  28.353  1.00 96.82           C  \nATOM   1546  O   ILE B 157       0.437 -27.551  27.705  1.00 95.51           O  \nATOM   1547  CB  ILE B 157       0.722 -30.474  29.054  1.00 97.97           C  \nATOM   1548  CG1 ILE B 157       1.222 -31.300  30.239  1.00 98.62           C  \nATOM   1549  CG2 ILE B 157       1.834 -30.321  28.022  1.00 97.70           C  \nATOM   1550  CD1 ILE B 157       1.590 -32.723  29.887  1.00 99.72           C  \nATOM   1551  N   ASN B 158      -1.571 -28.496  28.073  1.00 97.76           N  \nATOM   1552  CA  ASN B 158      -2.219 -27.813  26.959  1.00 98.85           C  \nATOM   1553  C   ASN B 158      -2.313 -26.309  27.191  1.00 98.03           C  \nATOM   1554  O   ASN B 158      -1.922 -25.506  26.337  1.00 97.23           O  \nATOM   1555  CB  ASN B 158      -3.630 -28.372  26.739  1.00100.78           C  \nATOM   1556  CG  ASN B 158      -3.623 -29.741  26.090  1.00102.39           C  \nATOM   1557  OD1 ASN B 158      -3.111 -29.915  24.983  1.00103.73           O  \nATOM   1558  ND2 ASN B 158      -4.196 -30.722  26.775  1.00102.20           N  \nATOM   1559  N   HIS B 159      -2.829 -25.939  28.358  1.00 96.50           N  \nATOM   1560  CA  HIS B 159      -3.003 -24.541  28.705  1.00 95.45           C  \nATOM   1561  C   HIS B 159      -1.713 -23.773  28.959  1.00 93.80           C  \nATOM   1562  O   HIS B 159      -1.729 -22.546  29.038  1.00 93.49           O  \nATOM   1563  CB  HIS B 159      -3.948 -24.431  29.899  1.00 97.70           C  \nATOM   1564  CG  HIS B 159      -5.336 -24.906  29.600  1.00101.00           C  \nATOM   1565  ND1 HIS B 159      -6.373 -24.808  30.503  1.00102.58           N  \nATOM   1566  CD2 HIS B 159      -5.859 -25.481  28.490  1.00102.13           C  \nATOM   1567  CE1 HIS B 159      -7.474 -25.302  29.962  1.00103.16           C  \nATOM   1568  NE2 HIS B 159      -7.189 -25.717  28.741  1.00103.41           N  \nATOM   1569  N   LEU B 160      -0.598 -24.483  29.085  1.00 92.59           N  \nATOM   1570  CA  LEU B 160       0.683 -23.819  29.297  1.00 92.46           C  \nATOM   1571  C   LEU B 160       1.491 -23.895  28.007  1.00 93.09           C  \nATOM   1572  O   LEU B 160       1.145 -24.651  27.100  1.00 93.69           O  \nATOM   1573  CB  LEU B 160       1.459 -24.476  30.443  1.00 91.80           C  \nATOM   1574  CG  LEU B 160       0.822 -24.458  31.835  1.00 90.45           C  \nATOM   1575  CD1 LEU B 160       1.867 -24.857  32.863  1.00 90.19           C  \nATOM   1576  CD2 LEU B 160       0.281 -23.077  32.155  1.00 89.15           C  \nATOM   1577  N   ASN B 161       2.560 -23.112  27.921  1.00 94.27           N  \nATOM   1578  CA  ASN B 161       3.403 -23.100  26.731  1.00 96.28           C  \nATOM   1579  C   ASN B 161       4.862 -23.432  27.051  1.00 98.46           C  \nATOM   1580  O   ASN B 161       5.768 -22.686  26.676  1.00 99.29           O  \nATOM   1581  CB  ASN B 161       3.320 -21.726  26.055  1.00 94.70           C  \nATOM   1582  CG  ASN B 161       1.996 -21.498  25.351  1.00 93.83           C  \nATOM   1583  OD1 ASN B 161       1.761 -22.021  24.259  1.00 92.18           O  \nATOM   1584  ND2 ASN B 161       1.119 -20.721  25.977  1.00 94.33           N  \nATOM   1585  N   TYR B 162       5.097 -24.556  27.725  1.00100.80           N  \nATOM   1586  CA  TYR B 162       6.462 -24.935  28.089  1.00102.50           C  \nATOM   1587  C   TYR B 162       6.957 -26.310  27.627  1.00103.55           C  \nATOM   1588  O   TYR B 162       8.055 -26.722  28.006  1.00104.20           O  \nATOM   1589  CB  TYR B 162       6.647 -24.836  29.612  1.00102.45           C  \nATOM   1590  CG  TYR B 162       6.649 -23.423  30.152  1.00101.91           C  \nATOM   1591  CD1 TYR B 162       5.459 -22.773  30.472  1.00101.90           C  \nATOM   1592  CD2 TYR B 162       7.844 -22.723  30.311  1.00101.88           C  \nATOM   1593  CE1 TYR B 162       5.460 -21.459  30.935  1.00102.16           C  \nATOM   1594  CE2 TYR B 162       7.857 -21.411  30.770  1.00102.08           C  \nATOM   1595  CZ  TYR B 162       6.664 -20.784  31.078  1.00102.19           C  \nATOM   1596  OH  TYR B 162       6.674 -19.478  31.510  1.00102.60           O  \nATOM   1597  N   LYS B 163       6.173 -27.014  26.813  1.00104.60           N  \nATOM   1598  CA  LYS B 163       6.568 -28.346  26.344  1.00105.58           C  \nATOM   1599  C   LYS B 163       6.547 -29.353  27.501  1.00105.79           C  \nATOM   1600  O   LYS B 163       7.094 -29.090  28.575  1.00106.53           O  \nATOM   1601  CB  LYS B 163       7.971 -28.308  25.721  1.00105.70           C  \nATOM   1602  CG  LYS B 163       8.033 -27.789  24.287  1.00106.72           C  \nATOM   1603  CD  LYS B 163       7.347 -28.748  23.317  1.00107.77           C  \nATOM   1604  CE  LYS B 163       7.782 -28.515  21.868  1.00108.48           C  \nATOM   1605  NZ  LYS B 163       7.451 -27.154  21.358  1.00109.25           N  \nATOM   1606  N   GLN B 164       5.920 -30.505  27.269  1.00105.43           N  \nATOM   1607  CA  GLN B 164       5.795 -31.558  28.280  1.00105.10           C  \nATOM   1608  C   GLN B 164       7.037 -31.815  29.132  1.00103.20           C  \nATOM   1609  O   GLN B 164       6.952 -31.858  30.360  1.00101.51           O  \nATOM   1610  CB  GLN B 164       5.380 -32.871  27.622  1.00107.87           C  \nATOM   1611  CG  GLN B 164       4.059 -32.813  26.889  1.00111.06           C  \nATOM   1612  CD  GLN B 164       3.709 -34.140  26.245  1.00112.71           C  \nATOM   1613  OE1 GLN B 164       3.551 -35.153  26.930  1.00114.49           O  \nATOM   1614  NE2 GLN B 164       3.592 -34.144  24.920  1.00112.48           N  \nATOM   1615  N   PRO B 165       8.203 -32.007  28.491  1.00101.73           N  \nATOM   1616  CA  PRO B 165       9.447 -32.262  29.220  1.00101.51           C  \nATOM   1617  C   PRO B 165       9.626 -31.397  30.468  1.00101.73           C  \nATOM   1618  O   PRO B 165       9.838 -31.916  31.566  1.00101.98           O  \nATOM   1619  CB  PRO B 165      10.514 -31.994  28.167  1.00100.64           C  \nATOM   1620  CG  PRO B 165       9.862 -32.499  26.931  1.00100.31           C  \nATOM   1621  CD  PRO B 165       8.461 -31.935  27.041  1.00100.96           C  \nATOM   1622  N   GLN B 166       9.534 -30.081  30.295  1.00101.10           N  \nATOM   1623  CA  GLN B 166       9.693 -29.144  31.404  1.00 99.54           C  \nATOM   1624  C   GLN B 166       8.487 -29.136  32.343  1.00 98.55           C  \nATOM   1625  O   GLN B 166       8.638 -28.995  33.559  1.00 97.57           O  \nATOM   1626  CB  GLN B 166       9.932 -27.733  30.867  1.00100.07           C  \nATOM   1627  CG  GLN B 166      11.154 -27.606  29.976  1.00101.20           C  \nATOM   1628  CD  GLN B 166      11.409 -26.174  29.544  1.00102.82           C  \nATOM   1629  OE1 GLN B 166      11.704 -25.305  30.368  1.00103.60           O  \nATOM   1630  NE2 GLN B 166      11.291 -25.918  28.246  1.00103.33           N  \nATOM   1631  N   VAL B 167       7.293 -29.285  31.775  1.00 96.97           N  \nATOM   1632  CA  VAL B 167       6.074 -29.299  32.569  1.00 96.10           C  \nATOM   1633  C   VAL B 167       6.057 -30.511  33.500  1.00 95.61           C  \nATOM   1634  O   VAL B 167       5.955 -30.366  34.720  1.00 95.62           O  \nATOM   1635  CB  VAL B 167       4.826 -29.340  31.668  1.00 96.55           C  \nATOM   1636  CG1 VAL B 167       3.565 -29.275  32.521  1.00 96.21           C  \nATOM   1637  CG2 VAL B 167       4.863 -28.188  30.679  1.00 96.63           C  \nATOM   1638  N   VAL B 168       6.157 -31.707  32.924  1.00 94.88           N  \nATOM   1639  CA  VAL B 168       6.161 -32.939  33.716  1.00 93.51           C  \nATOM   1640  C   VAL B 168       7.295 -32.889  34.734  1.00 91.40           C  \nATOM   1641  O   VAL B 168       7.170 -33.378  35.858  1.00 90.00           O  \nATOM   1642  CB  VAL B 168       6.350 -34.180  32.821  1.00 94.32           C  \nATOM   1643  CG1 VAL B 168       6.447 -35.429  33.683  1.00 95.03           C  \nATOM   1644  CG2 VAL B 168       5.186 -34.299  31.846  1.00 94.83           C  \nATOM   1645  N   LYS B 169       8.403 -32.292  34.317  1.00 89.58           N  \nATOM   1646  CA  LYS B 169       9.576 -32.123  35.161  1.00 87.41           C  \nATOM   1647  C   LYS B 169       9.189 -31.258  36.364  1.00 86.84           C  \nATOM   1648  O   LYS B 169       9.628 -31.505  37.490  1.00 85.86           O  \nATOM   1649  CB  LYS B 169      10.671 -31.441  34.344  1.00 86.78           C  \nATOM   1650  CG  LYS B 169      11.979 -31.177  35.051  1.00 87.35           C  \nATOM   1651  CD  LYS B 169      12.958 -30.578  34.045  1.00 90.21           C  \nATOM   1652  CE  LYS B 169      14.325 -30.299  34.645  1.00 91.84           C  \nATOM   1653  NZ  LYS B 169      15.281 -29.803  33.606  1.00 93.15           N  \nATOM   1654  N   ALA B 170       8.354 -30.250  36.112  1.00 86.12           N  \nATOM   1655  CA  ALA B 170       7.891 -29.339  37.153  1.00 84.88           C  \nATOM   1656  C   ALA B 170       6.855 -30.007  38.053  1.00 83.72           C  \nATOM   1657  O   ALA B 170       6.915 -29.888  39.280  1.00 83.48           O  \nATOM   1658  CB  ALA B 170       7.300 -28.083  36.520  1.00 85.19           C  \nATOM   1659  N   VAL B 171       5.901 -30.699  37.437  1.00 82.40           N  \nATOM   1660  CA  VAL B 171       4.860 -31.400  38.179  1.00 81.93           C  \nATOM   1661  C   VAL B 171       5.542 -32.330  39.174  1.00 83.34           C  \nATOM   1662  O   VAL B 171       5.216 -32.344  40.365  1.00 83.30           O  \nATOM   1663  CB  VAL B 171       3.987 -32.253  37.239  1.00 80.91           C  \nATOM   1664  CG1 VAL B 171       2.936 -33.009  38.039  1.00 80.28           C  \nATOM   1665  CG2 VAL B 171       3.334 -31.368  36.197  1.00 81.04           C  \nATOM   1666  N   LYS B 172       6.497 -33.101  38.657  1.00 84.29           N  \nATOM   1667  CA  LYS B 172       7.272 -34.056  39.440  1.00 85.45           C  \nATOM   1668  C   LYS B 172       7.817 -33.420  40.716  1.00 85.10           C  \nATOM   1669  O   LYS B 172       7.482 -33.842  41.823  1.00 84.44           O  \nATOM   1670  CB  LYS B 172       8.425 -34.581  38.580  1.00 87.06           C  \nATOM   1671  CG  LYS B 172       9.203 -35.747  39.168  1.00 89.51           C  \nATOM   1672  CD  LYS B 172      10.113 -36.343  38.102  1.00 90.72           C  \nATOM   1673  CE  LYS B 172      10.878 -37.552  38.610  1.00 90.36           C  \nATOM   1674  NZ  LYS B 172      11.661 -38.189  37.514  1.00 89.74           N  \nATOM   1675  N   ILE B 173       8.655 -32.401  40.555  1.00 84.96           N  \nATOM   1676  CA  ILE B 173       9.245 -31.704  41.690  1.00 84.70           C  \nATOM   1677  C   ILE B 173       8.165 -31.199  42.647  1.00 86.57           C  \nATOM   1678  O   ILE B 173       8.181 -31.512  43.839  1.00 87.53           O  \nATOM   1679  CB  ILE B 173      10.087 -30.504  41.217  1.00 83.26           C  \nATOM   1680  CG1 ILE B 173      11.112 -30.971  40.184  1.00 82.53           C  \nATOM   1681  CG2 ILE B 173      10.786 -29.854  42.401  1.00 81.84           C  \nATOM   1682  CD1 ILE B 173      11.849 -29.840  39.507  1.00 82.75           C  \nATOM   1683  N   LEU B 174       7.228 -30.421  42.120  1.00 88.34           N  \nATOM   1684  CA  LEU B 174       6.149 -29.874  42.931  1.00 90.83           C  \nATOM   1685  C   LEU B 174       5.454 -30.972  43.719  1.00 92.35           C  \nATOM   1686  O   LEU B 174       5.048 -30.766  44.863  1.00 92.87           O  \nATOM   1687  CB  LEU B 174       5.127 -29.161  42.043  1.00 90.96           C  \nATOM   1688  CG  LEU B 174       5.586 -27.882  41.348  1.00 89.85           C  \nATOM   1689  CD1 LEU B 174       4.511 -27.410  40.382  1.00 90.17           C  \nATOM   1690  CD2 LEU B 174       5.888 -26.824  42.398  1.00 89.24           C  \nATOM   1691  N   SER B 175       5.318 -32.136  43.092  1.00 93.58           N  \nATOM   1692  CA  SER B 175       4.672 -33.287  43.715  1.00 94.30           C  \nATOM   1693  C   SER B 175       5.442 -33.750  44.953  1.00 95.09           C  \nATOM   1694  O   SER B 175       4.846 -34.058  45.987  1.00 94.11           O  \nATOM   1695  CB  SER B 175       4.572 -34.432  42.712  1.00 93.37           C  \nATOM   1696  OG  SER B 175       3.832 -35.503  43.259  1.00 94.28           O  \nATOM   1697  N   GLN B 176       6.767 -33.793  44.837  1.00 96.18           N  \nATOM   1698  CA  GLN B 176       7.628 -34.198  45.942  1.00 96.88           C  \nATOM   1699  C   GLN B 176       7.509 -33.159  47.042  1.00 96.25           C  \nATOM   1700  O   GLN B 176       7.363 -33.494  48.217  1.00 96.02           O  \nATOM   1701  CB  GLN B 176       9.085 -34.272  45.489  1.00 98.74           C  \nATOM   1702  CG  GLN B 176       9.313 -35.047  44.206  1.00101.34           C  \nATOM   1703  CD  GLN B 176      10.771 -35.046  43.795  1.00102.60           C  \nATOM   1704  OE1 GLN B 176      11.120 -35.469  42.692  1.00102.32           O  \nATOM   1705  NE2 GLN B 176      11.634 -34.572  44.687  1.00103.42           N  \nATOM   1706  N   GLU B 177       7.585 -31.892  46.646  1.00 96.74           N  \nATOM   1707  CA  GLU B 177       7.470 -30.780  47.581  1.00 98.47           C  \nATOM   1708  C   GLU B 177       6.134 -30.909  48.300  1.00 97.45           C  \nATOM   1709  O   GLU B 177       5.887 -30.252  49.311  1.00 96.91           O  \nATOM   1710  CB  GLU B 177       7.532 -29.455  46.821  1.00101.18           C  \nATOM   1711  CG  GLU B 177       8.811 -29.265  46.015  1.00104.55           C  \nATOM   1712  CD  GLU B 177      10.006 -28.939  46.890  1.00106.74           C  \nATOM   1713  OE1 GLU B 177      10.228 -29.657  47.887  1.00109.59           O  \nATOM   1714  OE2 GLU B 177      10.727 -27.965  46.582  1.00106.61           O  \nATOM   1715  N   ASP B 178       5.279 -31.767  47.751  1.00 96.54           N  \nATOM   1716  CA  ASP B 178       3.960 -32.043  48.300  1.00 96.10           C  \nATOM   1717  C   ASP B 178       2.999 -30.861  48.185  1.00 95.93           C  \nATOM   1718  O   ASP B 178       2.371 -30.455  49.165  1.00 95.19           O  \nATOM   1719  CB  ASP B 178       4.089 -32.491  49.761  1.00 96.67           C  \nATOM   1720  CG  ASP B 178       2.827 -33.153  50.286  1.00 96.09           C  \nATOM   1721  OD1 ASP B 178       2.239 -33.977  49.552  1.00 96.14           O  \nATOM   1722  OD2 ASP B 178       2.432 -32.862  51.436  1.00 95.32           O  \nATOM   1723  N   TYR B 179       2.900 -30.309  46.979  1.00 95.34           N  \nATOM   1724  CA  TYR B 179       1.996 -29.198  46.710  1.00 95.82           C  \nATOM   1725  C   TYR B 179       0.707 -29.833  46.223  1.00 98.40           C  \nATOM   1726  O   TYR B 179      -0.378 -29.275  46.349  1.00 98.94           O  \nATOM   1727  CB  TYR B 179       2.589 -28.273  45.643  1.00 92.06           C  \nATOM   1728  CG  TYR B 179       3.544 -27.247  46.210  1.00 88.34           C  \nATOM   1729  CD1 TYR B 179       3.073 -26.191  46.992  1.00 87.38           C  \nATOM   1730  CD2 TYR B 179       4.919 -27.350  46.003  1.00 86.47           C  \nATOM   1731  CE1 TYR B 179       3.946 -25.265  47.558  1.00 84.94           C  \nATOM   1732  CE2 TYR B 179       5.803 -26.426  46.567  1.00 84.44           C  \nATOM   1733  CZ  TYR B 179       5.306 -25.389  47.345  1.00 84.01           C  \nATOM   1734  OH  TYR B 179       6.163 -24.485  47.927  1.00 81.56           O  \nATOM   1735  N   PHE B 180       0.859 -31.022  45.660  1.00102.56           N  \nATOM   1736  CA  PHE B 180      -0.249 -31.821  45.160  1.00107.16           C  \nATOM   1737  C   PHE B 180       0.274 -33.248  45.031  1.00111.47           C  \nATOM   1738  O   PHE B 180       1.431 -33.462  44.662  1.00113.03           O  \nATOM   1739  CB  PHE B 180      -0.757 -31.300  43.807  1.00104.31           C  \nATOM   1740  CG  PHE B 180       0.325 -31.028  42.804  1.00101.96           C  \nATOM   1741  CD1 PHE B 180       0.796 -29.735  42.605  1.00100.64           C  \nATOM   1742  CD2 PHE B 180       0.863 -32.058  42.046  1.00101.62           C  \nATOM   1743  CE1 PHE B 180       1.786 -29.471  41.662  1.00 99.81           C  \nATOM   1744  CE2 PHE B 180       1.856 -31.801  41.101  1.00101.82           C  \nATOM   1745  CZ  PHE B 180       2.316 -30.504  40.910  1.00100.18           C  \nATOM   1746  N   ASP B 181      -0.570 -34.222  45.352  1.00116.26           N  \nATOM   1747  CA  ASP B 181      -0.163 -35.618  45.291  1.00120.51           C  \nATOM   1748  C   ASP B 181       0.299 -36.062  43.907  1.00122.84           C  \nATOM   1749  O   ASP B 181       1.501 -36.175  43.672  1.00123.84           O  \nATOM   1750  CB  ASP B 181      -1.296 -36.517  45.786  1.00121.49           C  \nATOM   1751  CG  ASP B 181      -1.729 -36.173  47.200  1.00122.30           C  \nATOM   1752  OD1 ASP B 181      -2.335 -35.098  47.393  1.00123.01           O  \nATOM   1753  OD2 ASP B 181      -1.455 -36.972  48.120  1.00122.34           O  \nATOM   1754  N   LYS B 182      -0.637 -36.315  42.993  1.00125.54           N  \nATOM   1755  CA  LYS B 182      -0.272 -36.759  41.642  1.00129.19           C  \nATOM   1756  C   LYS B 182       0.467 -38.105  41.708  1.00132.67           C  \nATOM   1757  O   LYS B 182       1.020 -38.442  42.744  1.00133.02           O  \nATOM   1758  CB  LYS B 182       0.656 -35.720  40.999  1.00127.85           C  \nATOM   1759  CG  LYS B 182       1.288 -36.142  39.679  1.00126.97           C  \nATOM   1760  CD  LYS B 182       2.609 -36.877  39.867  1.00125.18           C  \nATOM   1761  CE  LYS B 182       3.260 -37.154  38.521  1.00124.05           C  \nATOM   1762  NZ  LYS B 182       4.629 -37.715  38.656  1.00122.75           N  \nATOM   1763  N   LYS B 183       0.550 -38.870  40.618  1.00137.42           N  \nATOM   1764  CA  LYS B 183       1.322 -40.118  40.710  1.00141.87           C  \nATOM   1765  C   LYS B 183       1.683 -40.952  39.475  1.00145.48           C  \nATOM   1766  O   LYS B 183       2.197 -42.065  39.620  1.00146.20           O  \nATOM   1767  CB  LYS B 183       0.705 -41.054  41.757  1.00140.10           C  \nATOM   1768  CG  LYS B 183       1.636 -41.305  42.945  1.00138.22           C  \nATOM   1769  CD  LYS B 183       3.031 -41.706  42.469  1.00136.41           C  \nATOM   1770  CE  LYS B 183       4.030 -41.776  43.607  1.00134.58           C  \nATOM   1771  NZ  LYS B 183       5.390 -42.097  43.094  1.00133.87           N  \nATOM   1772  N   ARG B 184       1.440 -40.436  38.274  1.00149.72           N  \nATOM   1773  CA  ARG B 184       1.779 -41.177  37.054  1.00153.31           C  \nATOM   1774  C   ARG B 184       1.153 -42.575  37.020  1.00155.63           C  \nATOM   1775  O   ARG B 184       0.269 -42.895  37.819  1.00156.73           O  \nATOM   1776  CB  ARG B 184       3.301 -41.321  36.931  1.00152.99           C  \nATOM   1777  CG  ARG B 184       4.065 -40.011  36.860  1.00152.77           C  \nATOM   1778  CD  ARG B 184       5.544 -40.233  37.137  1.00152.97           C  \nATOM   1779  NE  ARG B 184       6.168 -41.121  36.161  1.00154.15           N  \nATOM   1780  CZ  ARG B 184       7.413 -41.579  36.259  1.00154.51           C  \nATOM   1781  NH1 ARG B 184       8.167 -41.235  37.293  1.00154.48           N  \nATOM   1782  NH2 ARG B 184       7.906 -42.376  35.320  1.00155.02           N  \nATOM   1783  N   ASN B 185       1.632 -43.394  36.085  1.00157.85           N  \nATOM   1784  CA  ASN B 185       1.173 -44.772  35.901  1.00159.72           C  \nATOM   1785  C   ASN B 185      -0.321 -44.971  36.163  1.00160.66           C  \nATOM   1786  O   ASN B 185      -0.734 -45.214  37.298  1.00161.37           O  \nATOM   1787  CB  ASN B 185       1.982 -45.714  36.803  1.00159.58           C  \nATOM   1788  CG  ASN B 185       1.728 -47.183  36.502  1.00159.34           C  \nATOM   1789  OD1 ASN B 185       1.993 -47.656  35.396  1.00159.30           O  \nATOM   1790  ND2 ASN B 185       1.216 -47.911  37.488  1.00158.52           N  \nATOM   1791  N   GLU B 186      -1.125 -44.872  35.106  1.00161.16           N  \nATOM   1792  CA  GLU B 186      -2.572 -45.048  35.212  1.00161.11           C  \nATOM   1793  C   GLU B 186      -3.132 -45.652  33.921  1.00161.61           C  \nATOM   1794  O   GLU B 186      -4.345 -45.749  33.735  1.00161.38           O  \nATOM   1795  CB  GLU B 186      -3.233 -43.704  35.533  1.00160.61           C  \nATOM   1796  CG  GLU B 186      -2.750 -43.123  36.859  1.00160.37           C  \nATOM   1797  CD  GLU B 186      -3.258 -41.725  37.124  1.00160.39           C  \nATOM   1798  OE1 GLU B 186      -3.049 -40.842  36.267  1.00160.05           O  \nATOM   1799  OE2 GLU B 186      -3.860 -41.506  38.197  1.00161.00           O  \nATOM   1800  N   HIS B 187      -2.211 -46.053  33.047  1.00162.19           N  \nATOM   1801  CA  HIS B 187      -2.490 -46.694  31.760  1.00162.38           C  \nATOM   1802  C   HIS B 187      -3.612 -46.173  30.864  1.00163.26           C  \nATOM   1803  O   HIS B 187      -4.526 -45.477  31.308  1.00163.74           O  \nATOM   1804  CB  HIS B 187      -2.692 -48.197  31.975  1.00161.12           C  \nATOM   1805  CG  HIS B 187      -1.465 -48.906  32.455  1.00159.84           C  \nATOM   1806  ND1 HIS B 187      -0.876 -48.636  33.671  1.00159.27           N  \nATOM   1807  CD2 HIS B 187      -0.708 -49.868  31.875  1.00159.35           C  \nATOM   1808  CE1 HIS B 187       0.190 -49.402  33.821  1.00159.12           C  \nATOM   1809  NE2 HIS B 187       0.314 -50.158  32.745  1.00159.25           N  \nATOM   1810  N   ASP B 188      -3.513 -46.545  29.587  1.00164.08           N  \nATOM   1811  CA  ASP B 188      -4.464 -46.180  28.537  1.00164.37           C  \nATOM   1812  C   ASP B 188      -4.973 -44.742  28.618  1.00164.31           C  \nATOM   1813  O   ASP B 188      -4.424 -43.923  29.356  1.00165.44           O  \nATOM   1814  CB  ASP B 188      -5.649 -47.151  28.537  1.00164.03           C  \nATOM   1815  CG  ASP B 188      -5.992 -47.649  27.143  1.00163.82           C  \nATOM   1816  OD1 ASP B 188      -5.148 -48.342  26.534  1.00163.59           O  \nATOM   1817  OD2 ASP B 188      -7.099 -47.344  26.654  1.00163.45           O  \nATOM   1818  N   GLU B 189      -6.018 -44.440  27.848  1.00163.08           N  \nATOM   1819  CA  GLU B 189      -6.597 -43.097  27.826  1.00161.08           C  \nATOM   1820  C   GLU B 189      -5.486 -42.125  27.440  1.00161.06           C  \nATOM   1821  O   GLU B 189      -4.566 -42.479  26.699  1.00161.77           O  \nATOM   1822  CB  GLU B 189      -7.116 -42.720  29.215  1.00159.20           C  \nATOM   1823  CG  GLU B 189      -7.834 -43.829  29.957  1.00156.39           C  \nATOM   1824  CD  GLU B 189      -7.897 -43.565  31.449  1.00155.13           C  \nATOM   1825  OE1 GLU B 189      -8.463 -42.526  31.848  1.00153.89           O  \nATOM   1826  OE2 GLU B 189      -7.376 -44.393  32.226  1.00154.08           O  \nATOM   1827  N   ARG B 190      -5.582 -40.897  27.942  1.00159.98           N  \nATOM   1828  CA  ARG B 190      -4.557 -39.889  27.696  1.00158.33           C  \nATOM   1829  C   ARG B 190      -3.467 -40.216  28.722  1.00157.00           C  \nATOM   1830  O   ARG B 190      -3.347 -41.373  29.133  1.00157.64           O  \nATOM   1831  CB  ARG B 190      -5.125 -38.483  27.923  1.00157.89           C  \nATOM   1832  CG  ARG B 190      -6.170 -38.035  26.892  1.00157.39           C  \nATOM   1833  CD  ARG B 190      -5.543 -37.725  25.530  1.00156.60           C  \nATOM   1834  NE  ARG B 190      -5.397 -38.899  24.671  1.00155.38           N  \nATOM   1835  CZ  ARG B 190      -6.383 -39.433  23.955  1.00154.01           C  \nATOM   1836  NH1 ARG B 190      -7.597 -38.900  23.988  1.00152.85           N  \nATOM   1837  NH2 ARG B 190      -6.153 -40.500  23.199  1.00153.23           N  \nATOM   1838  N   THR B 191      -2.682 -39.236  29.159  1.00154.50           N  \nATOM   1839  CA  THR B 191      -1.637 -39.548  30.132  1.00151.30           C  \nATOM   1840  C   THR B 191      -1.555 -38.712  31.408  1.00149.09           C  \nATOM   1841  O   THR B 191      -1.688 -37.488  31.389  1.00148.73           O  \nATOM   1842  CB  THR B 191      -0.238 -39.549  29.470  1.00151.30           C  \nATOM   1843  OG1 THR B 191      -0.064 -38.353  28.700  1.00150.61           O  \nATOM   1844  CG2 THR B 191      -0.076 -40.770  28.574  1.00150.87           C  \nATOM   1845  N   VAL B 192      -1.334 -39.420  32.514  1.00145.94           N  \nATOM   1846  CA  VAL B 192      -1.180 -38.855  33.855  1.00142.24           C  \nATOM   1847  C   VAL B 192      -2.268 -37.899  34.348  1.00140.41           C  \nATOM   1848  O   VAL B 192      -3.020 -37.321  33.562  1.00140.50           O  \nATOM   1849  CB  VAL B 192       0.189 -38.146  33.988  1.00142.04           C  \nATOM   1850  CG1 VAL B 192       0.494 -37.864  35.452  1.00140.76           C  \nATOM   1851  CG2 VAL B 192       1.283 -39.006  33.370  1.00141.66           C  \nATOM   1852  N   LEU B 193      -2.331 -37.752  35.671  1.00137.05           N  \nATOM   1853  CA  LEU B 193      -3.286 -36.879  36.350  1.00134.19           C  \nATOM   1854  C   LEU B 193      -2.673 -36.497  37.702  1.00132.80           C  \nATOM   1855  O   LEU B 193      -1.795 -37.203  38.200  1.00132.17           O  \nATOM   1856  CB  LEU B 193      -4.612 -37.614  36.575  1.00132.99           C  \nATOM   1857  CG  LEU B 193      -5.340 -38.191  35.357  1.00131.83           C  \nATOM   1858  CD1 LEU B 193      -6.572 -38.944  35.816  1.00131.61           C  \nATOM   1859  CD2 LEU B 193      -5.730 -37.080  34.404  1.00131.07           C  \nATOM   1860  N   ILE B 194      -3.122 -35.388  38.292  1.00131.30           N  \nATOM   1861  CA  ILE B 194      -2.591 -34.956  39.588  1.00129.17           C  \nATOM   1862  C   ILE B 194      -3.692 -34.640  40.603  1.00126.51           C  \nATOM   1863  O   ILE B 194      -4.788 -34.205  40.243  1.00125.69           O  \nATOM   1864  CB  ILE B 194      -1.671 -33.713  39.452  1.00130.64           C  \nATOM   1865  CG1 ILE B 194      -2.507 -32.449  39.259  1.00131.60           C  \nATOM   1866  CG2 ILE B 194      -0.730 -33.888  38.269  1.00130.46           C  \nATOM   1867  CD1 ILE B 194      -1.683 -31.182  39.246  1.00133.36           C  \nATOM   1868  N   LEU B 195      -3.380 -34.855  41.877  1.00123.38           N  \nATOM   1869  CA  LEU B 195      -4.328 -34.630  42.962  1.00121.58           C  \nATOM   1870  C   LEU B 195      -3.912 -33.485  43.874  1.00120.59           C  \nATOM   1871  O   LEU B 195      -2.805 -33.477  44.407  1.00120.25           O  \nATOM   1872  CB  LEU B 195      -4.469 -35.914  43.787  1.00121.38           C  \nATOM   1873  CG  LEU B 195      -5.275 -35.878  45.087  1.00121.16           C  \nATOM   1874  CD1 LEU B 195      -6.723 -35.501  44.807  1.00121.17           C  \nATOM   1875  CD2 LEU B 195      -5.194 -37.245  45.752  1.00120.38           C  \nATOM   1876  N   VAL B 196      -4.809 -32.523  44.059  1.00119.92           N  \nATOM   1877  CA  VAL B 196      -4.521 -31.380  44.914  1.00118.75           C  \nATOM   1878  C   VAL B 196      -5.305 -31.457  46.213  1.00117.87           C  \nATOM   1879  O   VAL B 196      -6.465 -31.049  46.279  1.00117.35           O  \nATOM   1880  CB  VAL B 196      -4.857 -30.046  44.213  1.00118.87           C  \nATOM   1881  CG1 VAL B 196      -4.514 -28.876  45.129  1.00117.95           C  \nATOM   1882  CG2 VAL B 196      -4.086 -29.938  42.912  1.00118.82           C  \nATOM   1883  N   ASN B 197      -4.666 -31.993  47.244  1.00117.20           N  \nATOM   1884  CA  ASN B 197      -5.302 -32.104  48.544  1.00117.08           C  \nATOM   1885  C   ASN B 197      -5.776 -30.718  48.951  1.00116.52           C  \nATOM   1886  O   ASN B 197      -5.014 -29.753  48.902  1.00116.18           O  \nATOM   1887  CB  ASN B 197      -4.307 -32.646  49.573  1.00117.49           C  \nATOM   1888  CG  ASN B 197      -4.719 -32.346  51.003  1.00117.97           C  \nATOM   1889  OD1 ASN B 197      -5.806 -32.725  51.444  1.00118.90           O  \nATOM   1890  ND2 ASN B 197      -3.848 -31.660  51.735  1.00116.90           N  \nATOM   1891  N   ALA B 198      -7.040 -30.624  49.342  1.00115.77           N  \nATOM   1892  CA  ALA B 198      -7.624 -29.356  49.752  1.00114.34           C  \nATOM   1893  C   ALA B 198      -6.763 -28.611  50.771  1.00113.41           C  \nATOM   1894  O   ALA B 198      -6.961 -27.416  50.992  1.00113.83           O  \nATOM   1895  CB  ALA B 198      -9.019 -29.593  50.319  1.00114.53           C  \nATOM   1896  N   GLN B 199      -5.810 -29.309  51.385  1.00111.76           N  \nATOM   1897  CA  GLN B 199      -4.933 -28.695  52.383  1.00110.03           C  \nATOM   1898  C   GLN B 199      -3.558 -28.340  51.828  1.00108.49           C  \nATOM   1899  O   GLN B 199      -2.820 -27.554  52.428  1.00107.52           O  \nATOM   1900  CB  GLN B 199      -4.780 -29.622  53.582  1.00110.88           C  \nATOM   1901  CG  GLN B 199      -6.080 -29.868  54.307  1.00113.78           C  \nATOM   1902  CD  GLN B 199      -6.028 -31.101  55.175  1.00115.78           C  \nATOM   1903  OE1 GLN B 199      -5.819 -32.211  54.680  1.00116.35           O  \nATOM   1904  NE2 GLN B 199      -6.221 -30.918  56.477  1.00117.57           N  \nATOM   1905  N   GLN B 200      -3.213 -28.935  50.689  1.00106.43           N  \nATOM   1906  CA  GLN B 200      -1.939 -28.657  50.035  1.00103.84           C  \nATOM   1907  C   GLN B 200      -2.126 -27.317  49.338  1.00100.77           C  \nATOM   1908  O   GLN B 200      -1.171 -26.683  48.886  1.00100.26           O  \nATOM   1909  CB  GLN B 200      -1.625 -29.738  48.998  1.00105.81           C  \nATOM   1910  CG  GLN B 200      -1.395 -31.127  49.571  1.00108.07           C  \nATOM   1911  CD  GLN B 200      -1.182 -32.171  48.487  1.00109.85           C  \nATOM   1912  OE1 GLN B 200      -2.075 -32.436  47.680  1.00110.54           O  \nATOM   1913  NE2 GLN B 200       0.007 -32.764  48.460  1.00110.79           N  \nATOM   1914  N   ARG B 201      -3.387 -26.905  49.265  1.00 96.76           N  \nATOM   1915  CA  ARG B 201      -3.786 -25.657  48.639  1.00 92.64           C  \nATOM   1916  C   ARG B 201      -3.444 -24.483  49.543  1.00 90.45           C  \nATOM   1917  O   ARG B 201      -3.363 -23.342  49.090  1.00 89.81           O  \nATOM   1918  CB  ARG B 201      -5.282 -25.702  48.341  1.00 92.19           C  \nATOM   1919  CG  ARG B 201      -5.868 -24.412  47.838  1.00 92.91           C  \nATOM   1920  CD  ARG B 201      -6.948 -24.705  46.822  1.00 93.98           C  \nATOM   1921  NE  ARG B 201      -6.378 -25.008  45.515  1.00 94.87           N  \nATOM   1922  CZ  ARG B 201      -7.053 -25.566  44.518  1.00 96.11           C  \nATOM   1923  NH1 ARG B 201      -8.327 -25.896  44.677  1.00 97.57           N  \nATOM   1924  NH2 ARG B 201      -6.460 -25.777  43.354  1.00 96.15           N  \nATOM   1925  N   LYS B 202      -3.257 -24.765  50.829  1.00 88.38           N  \nATOM   1926  CA  LYS B 202      -2.873 -23.731  51.782  1.00 87.00           C  \nATOM   1927  C   LYS B 202      -1.413 -23.448  51.476  1.00 83.72           C  \nATOM   1928  O   LYS B 202      -0.943 -22.318  51.575  1.00 83.71           O  \nATOM   1929  CB  LYS B 202      -2.980 -24.235  53.224  1.00 89.78           C  \nATOM   1930  CG  LYS B 202      -4.391 -24.379  53.756  1.00 93.64           C  \nATOM   1931  CD  LYS B 202      -4.381 -24.828  55.215  1.00 95.08           C  \nATOM   1932  CE  LYS B 202      -5.797 -24.903  55.777  1.00 96.65           C  \nATOM   1933  NZ  LYS B 202      -5.831 -25.476  57.153  1.00 97.10           N  \nATOM   1934  N   LYS B 203      -0.706 -24.505  51.095  1.00 79.50           N  \nATOM   1935  CA  LYS B 203       0.708 -24.426  50.768  1.00 75.60           C  \nATOM   1936  C   LYS B 203       0.933 -23.639  49.478  1.00 73.12           C  \nATOM   1937  O   LYS B 203       1.896 -22.883  49.360  1.00 73.55           O  \nATOM   1938  CB  LYS B 203       1.269 -25.842  50.619  1.00 74.91           C  \nATOM   1939  CG  LYS B 203       2.780 -25.944  50.626  1.00 73.65           C  \nATOM   1940  CD  LYS B 203       3.200 -27.398  50.462  1.00 73.15           C  \nATOM   1941  CE  LYS B 203       4.682 -27.608  50.742  1.00 73.11           C  \nATOM   1942  NZ  LYS B 203       5.560 -27.229  49.605  1.00 73.96           N  \nATOM   1943  N   ILE B 204       0.036 -23.820  48.516  1.00 68.94           N  \nATOM   1944  CA  ILE B 204       0.148 -23.149  47.233  1.00 65.72           C  \nATOM   1945  C   ILE B 204      -0.271 -21.687  47.254  1.00 66.31           C  \nATOM   1946  O   ILE B 204       0.516 -20.802  46.927  1.00 66.20           O  \nATOM   1947  CB  ILE B 204      -0.677 -23.874  46.159  1.00 63.02           C  \nATOM   1948  CG1 ILE B 204      -0.109 -25.279  45.949  1.00 60.91           C  \nATOM   1949  CG2 ILE B 204      -0.664 -23.066  44.854  1.00 60.98           C  \nATOM   1950  CD1 ILE B 204      -0.804 -26.073  44.878  1.00 60.56           C  \nATOM   1951  N   GLU B 205      -1.510 -21.430  47.641  1.00 66.34           N  \nATOM   1952  CA  GLU B 205      -2.001 -20.065  47.666  1.00 66.42           C  \nATOM   1953  C   GLU B 205      -1.220 -19.169  48.608  1.00 65.13           C  \nATOM   1954  O   GLU B 205      -1.228 -17.950  48.453  1.00 67.17           O  \nATOM   1955  CB  GLU B 205      -3.486 -20.050  48.010  1.00 68.33           C  \nATOM   1956  CG  GLU B 205      -4.287 -20.929  47.074  1.00 73.05           C  \nATOM   1957  CD  GLU B 205      -5.773 -20.712  47.192  1.00 76.50           C  \nATOM   1958  OE1 GLU B 205      -6.264 -20.532  48.331  1.00 78.23           O  \nATOM   1959  OE2 GLU B 205      -6.452 -20.737  46.139  1.00 77.93           O  \nATOM   1960  N   SER B 206      -0.539 -19.768  49.580  1.00 62.88           N  \nATOM   1961  CA  SER B 206       0.259 -18.992  50.520  1.00 60.24           C  \nATOM   1962  C   SER B 206       1.548 -18.624  49.802  1.00 56.48           C  \nATOM   1963  O   SER B 206       2.063 -17.525  49.954  1.00 55.60           O  \nATOM   1964  CB  SER B 206       0.565 -19.817  51.780  1.00 62.54           C  \nATOM   1965  OG  SER B 206       1.386 -19.100  52.693  1.00 65.86           O  \nATOM   1966  N   LEU B 207       2.054 -19.559  49.008  1.00 52.58           N  \nATOM   1967  CA  LEU B 207       3.275 -19.349  48.240  1.00 49.12           C  \nATOM   1968  C   LEU B 207       2.987 -18.261  47.206  1.00 48.01           C  \nATOM   1969  O   LEU B 207       3.779 -17.337  47.013  1.00 47.95           O  \nATOM   1970  CB  LEU B 207       3.691 -20.653  47.548  1.00 48.42           C  \nATOM   1971  CG  LEU B 207       5.023 -20.694  46.789  1.00 47.89           C  \nATOM   1972  CD1 LEU B 207       4.810 -20.337  45.334  1.00 48.27           C  \nATOM   1973  CD2 LEU B 207       6.028 -19.763  47.459  1.00 47.44           C  \nATOM   1974  N   LEU B 208       1.830 -18.370  46.561  1.00 44.59           N  \nATOM   1975  CA  LEU B 208       1.426 -17.392  45.570  1.00 41.97           C  \nATOM   1976  C   LEU B 208       1.091 -16.042  46.216  1.00 41.06           C  \nATOM   1977  O   LEU B 208       1.227 -15.010  45.576  1.00 40.43           O  \nATOM   1978  CB  LEU B 208       0.230 -17.915  44.768  1.00 39.67           C  \nATOM   1979  CG  LEU B 208       0.497 -19.162  43.924  1.00 38.02           C  \nATOM   1980  CD1 LEU B 208      -0.677 -19.386  42.989  1.00 35.39           C  \nATOM   1981  CD2 LEU B 208       1.772 -19.002  43.127  1.00 36.51           C  \nATOM   1982  N   SER B 209       0.657 -16.043  47.473  1.00 42.21           N  \nATOM   1983  CA  SER B 209       0.351 -14.783  48.145  1.00 44.02           C  \nATOM   1984  C   SER B 209       1.645 -14.072  48.462  1.00 44.83           C  \nATOM   1985  O   SER B 209       1.708 -12.837  48.497  1.00 44.32           O  \nATOM   1986  CB  SER B 209      -0.409 -15.007  49.449  1.00 44.91           C  \nATOM   1987  OG  SER B 209      -1.793 -15.101  49.201  1.00 49.32           O  \nATOM   1988  N   ARG B 210       2.681 -14.861  48.708  1.00 46.54           N  \nATOM   1989  CA  ARG B 210       3.971 -14.284  49.006  1.00 47.43           C  \nATOM   1990  C   ARG B 210       4.619 -13.800  47.716  1.00 44.30           C  \nATOM   1991  O   ARG B 210       5.372 -12.836  47.720  1.00 43.26           O  \nATOM   1992  CB  ARG B 210       4.851 -15.302  49.746  1.00 51.80           C  \nATOM   1993  CG  ARG B 210       4.401 -15.494  51.199  1.00 56.81           C  \nATOM   1994  CD  ARG B 210       5.377 -16.301  52.000  1.00 61.06           C  \nATOM   1995  NE  ARG B 210       5.490 -17.664  51.493  1.00 65.76           N  \nATOM   1996  CZ  ARG B 210       6.631 -18.218  51.094  1.00 67.90           C  \nATOM   1997  NH1 ARG B 210       7.770 -17.528  51.142  1.00 68.34           N  \nATOM   1998  NH2 ARG B 210       6.631 -19.463  50.641  1.00 68.63           N  \nATOM   1999  N   VAL B 211       4.314 -14.452  46.600  1.00 41.30           N  \nATOM   2000  CA  VAL B 211       4.889 -13.995  45.351  1.00 41.06           C  \nATOM   2001  C   VAL B 211       4.184 -12.708  44.933  1.00 41.92           C  \nATOM   2002  O   VAL B 211       4.823 -11.802  44.396  1.00 43.94           O  \nATOM   2003  CB  VAL B 211       4.801 -15.064  44.232  1.00 40.59           C  \nATOM   2004  CG1 VAL B 211       5.263 -14.463  42.890  1.00 38.22           C  \nATOM   2005  CG2 VAL B 211       5.695 -16.239  44.588  1.00 38.55           C  \nATOM   2006  N   ASN B 212       2.880 -12.616  45.194  1.00 41.08           N  \nATOM   2007  CA  ASN B 212       2.146 -11.399  44.866  1.00 40.36           C  \nATOM   2008  C   ASN B 212       2.740 -10.222  45.649  1.00 39.42           C  \nATOM   2009  O   ASN B 212       2.865  -9.124  45.112  1.00 41.26           O  \nATOM   2010  CB  ASN B 212       0.653 -11.540  45.195  1.00 41.23           C  \nATOM   2011  CG  ASN B 212      -0.125 -12.337  44.143  1.00 42.29           C  \nATOM   2012  OD1 ASN B 212      -1.242 -12.778  44.410  1.00 45.53           O  \nATOM   2013  ND2 ASN B 212       0.458 -12.529  42.948  1.00 39.07           N  \nATOM   2014  N   LYS B 213       3.108 -10.437  46.909  1.00 38.66           N  \nATOM   2015  CA  LYS B 213       3.702  -9.364  47.696  1.00 40.12           C  \nATOM   2016  C   LYS B 213       5.080  -8.947  47.160  1.00 40.23           C  \nATOM   2017  O   LYS B 213       5.407  -7.755  47.136  1.00 40.24           O  \nATOM   2018  CB  LYS B 213       3.823  -9.757  49.176  1.00 41.17           C  \nATOM   2019  CG  LYS B 213       2.522  -9.681  49.954  1.00 43.42           C  \nATOM   2020  CD  LYS B 213       2.780  -9.678  51.445  1.00 47.05           C  \nATOM   2021  CE  LYS B 213       1.477  -9.745  52.230  1.00 50.53           C  \nATOM   2022  NZ  LYS B 213       1.690  -9.602  53.717  1.00 50.85           N  \nATOM   2023  N   ARG B 214       5.884  -9.913  46.727  1.00 39.94           N  \nATOM   2024  CA  ARG B 214       7.195  -9.580  46.184  1.00 40.18           C  \nATOM   2025  C   ARG B 214       7.067  -8.828  44.865  1.00 39.26           C  \nATOM   2026  O   ARG B 214       7.896  -7.955  44.567  1.00 41.37           O  \nATOM   2027  CB  ARG B 214       8.053 -10.843  45.991  1.00 42.47           C  \nATOM   2028  CG  ARG B 214       8.622 -11.382  47.295  1.00 47.21           C  \nATOM   2029  CD  ARG B 214       9.504 -10.338  47.980  1.00 51.08           C  \nATOM   2030  NE  ARG B 214      10.677 -10.026  47.165  1.00 54.85           N  \nATOM   2031  CZ  ARG B 214      11.124  -8.798  46.926  1.00 55.57           C  \nATOM   2032  NH1 ARG B 214      10.486  -7.752  47.444  1.00 56.51           N  \nATOM   2033  NH2 ARG B 214      12.201  -8.617  46.166  1.00 53.73           N  \nATOM   2034  N   ILE B 215       6.033  -9.164  44.087  1.00 36.79           N  \nATOM   2035  CA  ILE B 215       5.775  -8.512  42.806  1.00 34.18           C  \nATOM   2036  C   ILE B 215       5.466  -7.015  42.988  1.00 34.04           C  \nATOM   2037  O   ILE B 215       5.958  -6.187  42.238  1.00 32.61           O  \nATOM   2038  CB  ILE B 215       4.554  -9.161  42.046  1.00 33.54           C  \nATOM   2039  CG1 ILE B 215       4.907 -10.557  41.529  1.00 32.05           C  \nATOM   2040  CG2 ILE B 215       4.097  -8.259  40.870  1.00 31.80           C  \nATOM   2041  CD1 ILE B 215       3.736 -11.234  40.771  1.00 29.47           C  \nATOM   2042  N   THR B 216       4.651  -6.667  43.979  1.00 34.78           N  \nATOM   2043  CA  THR B 216       4.314  -5.261  44.148  1.00 38.46           C  \nATOM   2044  C   THR B 216       5.503  -4.509  44.725  1.00 38.91           C  \nATOM   2045  O   THR B 216       5.703  -3.322  44.441  1.00 39.86           O  \nATOM   2046  CB  THR B 216       3.037  -5.064  45.015  1.00 37.63           C  \nATOM   2047  OG1 THR B 216       3.197  -5.740  46.264  1.00 40.09           O  \nATOM   2048  CG2 THR B 216       1.808  -5.653  44.282  1.00 36.02           C  \nATOM   2049  N   GLU B 217       6.314  -5.215  45.501  1.00 38.87           N  \nATOM   2050  CA  GLU B 217       7.503  -4.609  46.051  1.00 41.38           C  \nATOM   2051  C   GLU B 217       8.456  -4.354  44.888  1.00 39.94           C  \nATOM   2052  O   GLU B 217       9.022  -3.272  44.771  1.00 41.07           O  \nATOM   2053  CB  GLU B 217       8.124  -5.535  47.101  1.00 45.46           C  \nATOM   2054  CG  GLU B 217       7.394  -5.444  48.452  1.00 54.21           C  \nATOM   2055  CD  GLU B 217       7.678  -6.602  49.386  1.00 57.60           C  \nATOM   2056  OE1 GLU B 217       8.832  -7.068  49.443  1.00 60.88           O  \nATOM   2057  OE2 GLU B 217       6.739  -7.037  50.077  1.00 60.14           O  \nATOM   2058  N   ALA B 218       8.617  -5.342  44.013  1.00 39.55           N  \nATOM   2059  CA  ALA B 218       9.486  -5.184  42.849  1.00 39.39           C  \nATOM   2060  C   ALA B 218       9.009  -4.053  41.950  1.00 40.22           C  \nATOM   2061  O   ALA B 218       9.824  -3.340  41.383  1.00 40.09           O  \nATOM   2062  CB  ALA B 218       9.561  -6.487  42.055  1.00 38.65           C  \nATOM   2063  N   ASN B 219       7.697  -3.877  41.814  1.00 43.58           N  \nATOM   2064  CA  ASN B 219       7.177  -2.793  40.973  1.00 49.37           C  \nATOM   2065  C   ASN B 219       7.597  -1.438  41.563  1.00 54.51           C  \nATOM   2066  O   ASN B 219       7.919  -0.499  40.828  1.00 54.31           O  \nATOM   2067  CB  ASN B 219       5.644  -2.841  40.881  1.00 46.20           C  \nATOM   2068  CG  ASN B 219       5.122  -3.930  39.948  1.00 44.08           C  \nATOM   2069  OD1 ASN B 219       3.990  -4.371  40.102  1.00 45.47           O  \nATOM   2070  ND2 ASN B 219       5.924  -4.353  38.985  1.00 38.99           N  \nATOM   2071  N   ASN B 220       7.580  -1.343  42.894  1.00 61.56           N  \nATOM   2072  CA  ASN B 220       7.962  -0.115  43.582  1.00 68.71           C  \nATOM   2073  C   ASN B 220       9.456   0.087  43.435  1.00 74.13           C  \nATOM   2074  O   ASN B 220       9.940   1.216  43.371  1.00 76.03           O  \nATOM   2075  CB  ASN B 220       7.616  -0.178  45.077  1.00 68.14           C  \nATOM   2076  CG  ASN B 220       6.119  -0.313  45.340  1.00 68.22           C  \nATOM   2077  OD1 ASN B 220       5.295   0.200  44.591  1.00 68.37           O  \nATOM   2078  ND2 ASN B 220       5.770  -0.987  46.426  1.00 67.03           N  \nATOM   2079  N   GLU B 221      10.186  -1.018  43.372  1.00 81.12           N  \nATOM   2080  CA  GLU B 221      11.631  -0.967  43.241  1.00 87.27           C  \nATOM   2081  C   GLU B 221      12.043  -0.568  41.829  1.00 90.35           C  \nATOM   2082  O   GLU B 221      11.968   0.604  41.469  1.00 89.69           O  \nATOM   2083  CB  GLU B 221      12.228  -2.326  43.607  1.00 89.45           C  \nATOM   2084  CG  GLU B 221      13.739  -2.345  43.700  1.00 94.26           C  \nATOM   2085  CD  GLU B 221      14.272  -3.678  44.195  1.00 97.01           C  \nATOM   2086  OE1 GLU B 221      15.514  -3.821  44.286  1.00 98.84           O  \nATOM   2087  OE2 GLU B 221      13.454  -4.581  44.497  1.00 97.48           O  \nATOM   2088  N   ILE B 222      12.462  -1.552  41.036  1.00 96.19           N  \nATOM   2089  CA  ILE B 222      12.910  -1.344  39.656  1.00101.69           C  \nATOM   2090  C   ILE B 222      12.390  -0.074  38.987  1.00105.08           C  \nATOM   2091  O   ILE B 222      13.160   0.846  38.707  1.00106.90           O  \nATOM   2092  CB  ILE B 222      12.549  -2.565  38.757  1.00100.37           C  \nATOM   2093  CG1 ILE B 222      11.029  -2.754  38.676  1.00 98.27           C  \nATOM   2094  CG2 ILE B 222      13.202  -3.821  39.318  1.00101.96           C  \nATOM   2095  CD1 ILE B 222      10.603  -3.896  37.785  1.00 95.25           C  \nATOM   2096  N   GLU B 223      11.086  -0.033  38.730  1.00108.69           N  \nATOM   2097  CA  GLU B 223      10.446   1.111  38.095  1.00111.94           C  \nATOM   2098  C   GLU B 223      11.390   1.817  37.119  1.00113.84           C  \nATOM   2099  O   GLU B 223      11.461   3.047  37.094  1.00114.24           O  \nATOM   2100  CB  GLU B 223       9.967   2.095  39.166  1.00111.97           C  \nATOM   2101  CG  GLU B 223       8.922   3.083  38.681  1.00112.68           C  \nATOM   2102  CD  GLU B 223       8.509   4.067  39.758  1.00113.04           C  \nATOM   2103  OE1 GLU B 223       8.142   3.618  40.866  1.00112.24           O  \nATOM   2104  OE2 GLU B 223       8.550   5.288  39.493  1.00113.98           O  \nATOM   2105  N   LEU B 224      12.118   1.037  36.321  1.00116.21           N  \nATOM   2106  CA  LEU B 224      13.051   1.601  35.347  1.00118.33           C  \nATOM   2107  C   LEU B 224      12.294   2.075  34.107  1.00119.11           C  \nATOM   2108  O   LEU B 224      11.146   1.619  33.913  1.00119.43           O  \nATOM   2109  CB  LEU B 224      14.104   0.554  34.945  1.00118.38           C  \nATOM   2110  CG  LEU B 224      15.195   0.962  33.937  1.00118.07           C  \nATOM   2111  CD1 LEU B 224      16.054   2.083  34.516  1.00117.50           C  \nATOM   2112  CD2 LEU B 224      16.067  -0.244  33.599  1.00116.37           C  \nATOM   2113  OXT LEU B 224      12.859   2.887  33.340  1.00119.76           O  \nTER    2114      LEU B 224                                                      \nHETATM 2115 CA    CA A 501      -4.855  -8.309  40.205  0.50 53.27          CA  \nHETATM 2116 CA    CA B 601       4.399   2.606  20.437  0.50 54.01          CA  \nCONECT   72 2115                                                                \nCONECT   80 2115                                                                \nCONECT 1129 2116                                                                \nCONECT 1137 2116                                                                \nCONECT 1163 2116                                                                \nCONECT 1164 2116                                                                \nCONECT 2115   72   80                                                           \nCONECT 2116 1129 1137 1163 1164                                                 \nMASTER      347    0    2   10    4    0    2    6 2114    2    8   20          \nEND                                                                             \n","pdb");
	viewer_1758171620596065.setStyle({"model": 0},{"stick": {}});
	viewer_1758171620596065.setStyle({"model": 0},{"cartoon": {"color": "blue"}});
	viewer_1758171620596065.addLabel("2frhA",{"fontOpacity": 0.3},{"resi": 158});
	viewer_1758171620596065.zoomTo();
	viewer_1758171620596065.zoom(1.2);
	viewer_1758171620596065.setBackgroundColor("white");
viewer_1758171620596065.render();
});
</script>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=258fa5c0">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[7]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">try</span><span class="p">:</span>
    <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span><span class="n">pdb_path1</span><span class="p">)</span>
<span class="k">except</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s1">'Structural Alignment Plot fail !'</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 1997 atoms and 1 coordinate set(s) were parsed in 0.02s.
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 1997 atoms and 1 coordinate set(s) were parsed in 0.02s.
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>1fzp_TEMP
Blue:1fzp
Red:1fzp
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div id="3dmolviewer_1758171620800664" style="position: relative; width: 800px; height: 600px;">
<p id="3dmolwarning_1758171620800664" style="background-color:#ffcccc;color:black">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br/></p>
</div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');
}

var viewer_1758171620800664 = null;
var warn = document.getElementById("3dmolwarning_1758171620800664");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_1758171620800664 = $3Dmol.createViewer(document.getElementById("3dmolviewer_1758171620800664"),{backgroundColor:"white"});
viewer_1758171620800664.zoomTo();
	viewer_1758171620800664.addModel("HEADER    TRANSCRIPTION/DNA                       03-OCT-00   1FZP              \nOBSLTE     17-NOV-21 1FZP                                                       \nTITLE     CRYSTAL STRUCTURES OF SARA: A PLEIOTROPIC REGULATOR OF VIRULENCE GENES\nTITLE    2 IN S. AUREUS                                                         \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: 5'-D(P*AP*TP*AP*TP*AP*TP*A)-3';                            \nCOMPND   3 CHAIN: W, K;                                                         \nCOMPND   4 ENGINEERED: YES;                                                     \nCOMPND   5 MOL_ID: 2;                                                           \nCOMPND   6 MOLECULE: STAPHYLOCOCCAL ACCESSORY REGULATOR A;                      \nCOMPND   7 CHAIN: D, B;                                                         \nCOMPND   8 SYNONYM: SARA;                                                       \nCOMPND   9 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 SYNTHETIC: YES;                                                      \nSOURCE   3 MOL_ID: 2;                                                           \nSOURCE   4 ORGANISM_SCIENTIFIC: STAPHYLOCOCCUS AUREUS;                          \nSOURCE   5 ORGANISM_TAXID: 1280;                                                \nSOURCE   6 STRAIN: DB;                                                          \nSOURCE   7 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \nSOURCE   8 EXPRESSION_SYSTEM_TAXID: 562;                                        \nSOURCE   9 EXPRESSION_SYSTEM_VECTOR_TYPE: PLASMID;                              \nSOURCE  10 EXPRESSION_SYSTEM_PLASMID: PET9                                      \nKEYWDS    PROTEIN-DNA COMPLEX, INDIRECT READOUT, D-DNA, CONFORMATIONAL CHANGES, \nKEYWDS   2 COIL TO HELIX, TRANSCRIPTION-DNA COMPLEX                             \nEXPDTA    X-RAY DIFFRACTION                                                     \nAUTHOR    M.A.SCHUMACHER,B.HURLBURT,R.G.BRENNAN                                 \nREVDAT   5   17-NOV-21 1FZP    1       OBSLTE JRNL   REMARK LINK                \nREVDAT   4   04-APR-18 1FZP    1       REMARK                                   \nREVDAT   3   24-FEB-09 1FZP    1       VERSN                                    \nREVDAT   2   01-APR-03 1FZP    1       JRNL                                     \nREVDAT   1   22-JAN-01 1FZP    0                                                \nJRNL        AUTH   M.A.SCHUMACHER,B.K.HURLBURT,R.G.BRENNAN                      \nJRNL        TITL   CORRECTION: CRYSTAL STRUCTURES OF SARA, A PLEIOTROPIC        \nJRNL        TITL 2 REGULATOR OF VIRULENCE GENES IN S. AUREUS                    \nJRNL        REF    NATURE                        V. 414    85 2001              \nJRNL        REFN                   ISSN 0028-0836                               \nJRNL        DOI    10.1038/35102089                                             \nREMARK   1                                                                      \nREMARK   1 REFERENCE 1                                                          \nREMARK   1  AUTH   M.A.SCHUMACHER,B.K.HURLBURT,R.G.BRENNAN                      \nREMARK   1  TITL   CRYSTAL STRUCTURES OF SARA, A PLEIOTROPIC REGULATOR OF       \nREMARK   1  TITL 2 VIRULENCE GENES IN S. AUREUS.                                \nREMARK   1  REF    NATURE                        V. 409   215 2001              \nREMARK   1  REFN                   ISSN 0028-0836                               \nREMARK   1  PMID   11196648                                                     \nREMARK   1  DOI    10.1038/35051623                                             \nREMARK   2                                                                      \nREMARK   2 RESOLUTION.    2.95 ANGSTROMS.                                       \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : TNT                                                  \nREMARK   3   AUTHORS     : TRONRUD,TEN EYCK,MATTHEWS                            \nREMARK   3                                                                      \nREMARK   3  DATA USED IN REFINEMENT.                                            \nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.95                           \nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 10.00                          \nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 0.000                          \nREMARK   3   COMPLETENESS FOR RANGE        (%) : 89.0                           \nREMARK   3   NUMBER OF REFLECTIONS             : 14850                          \nREMARK   3                                                                      \nREMARK   3  USING DATA ABOVE SIGMA CUTOFF.                                      \nREMARK   3   CROSS-VALIDATION METHOD          : NULL                            \nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.220                           \nREMARK   3   R VALUE            (WORKING SET) : 0.220                           \nREMARK   3   FREE R VALUE                     : 0.308                           \nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : NULL                            \nREMARK   3   FREE R VALUE TEST SET COUNT      : 145                             \nREMARK   3                                                                      \nREMARK   3  USING ALL DATA, NO SIGMA CUTOFF.                                    \nREMARK   3   R VALUE   (WORKING + TEST SET, NO CUTOFF) : 0.2300                 \nREMARK   3   R VALUE          (WORKING SET, NO CUTOFF) : NULL                   \nREMARK   3   FREE R VALUE                  (NO CUTOFF) : NULL                   \nREMARK   3   FREE R VALUE TEST SET SIZE (%, NO CUTOFF) : NULL                   \nREMARK   3   FREE R VALUE TEST SET COUNT   (NO CUTOFF) : NULL                   \nREMARK   3   TOTAL NUMBER OF REFLECTIONS   (NO CUTOFF) : 13920                  \nREMARK   3                                                                      \nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \nREMARK   3   PROTEIN ATOMS            : 1707                                    \nREMARK   3   NUCLEIC ACID ATOMS       : 288                                     \nREMARK   3   HETEROGEN ATOMS          : 2                                       \nREMARK   3   SOLVENT ATOMS            : 0                                       \nREMARK   3                                                                      \nREMARK   3  WILSON B VALUE (FROM FCALC, A**2) : 50.000                          \nREMARK   3                                                                      \nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.    RMS    WEIGHT  COUNT           \nREMARK   3   BOND LENGTHS                 (A) : 0.011 ; NULL  ; NULL            \nREMARK   3   BOND ANGLES            (DEGREES) : 1.980 ; NULL  ; NULL            \nREMARK   3   TORSION ANGLES         (DEGREES) : NULL  ; NULL  ; NULL            \nREMARK   3   PSEUDOROTATION ANGLES  (DEGREES) : NULL  ; NULL  ; NULL            \nREMARK   3   TRIGONAL CARBON PLANES       (A) : NULL  ; NULL  ; NULL            \nREMARK   3   GENERAL PLANES               (A) : NULL  ; NULL  ; NULL            \nREMARK   3   ISOTROPIC THERMAL FACTORS (A**2) : NULL  ; NULL  ; NULL            \nREMARK   3   NON-BONDED CONTACTS          (A) : NULL  ; NULL  ; NULL            \nREMARK   3                                                                      \nREMARK   3  INCORRECT CHIRAL-CENTERS (COUNT) : NULL                             \nREMARK   3                                                                      \nREMARK   3  BULK SOLVENT MODELING.                                              \nREMARK   3   METHOD USED : NULL                                                 \nREMARK   3   KSOL        : NULL                                                 \nREMARK   3   BSOL        : NULL                                                 \nREMARK   3                                                                      \nREMARK   3  RESTRAINT LIBRARIES.                                                \nREMARK   3   STEREOCHEMISTRY : ENGH AND HUBER                                   \nREMARK   3   ISOTROPIC THERMAL FACTOR RESTRAINTS : NULL                         \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \nREMARK   4                                                                      \nREMARK   4 1FZP COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK   5                                                                      \nREMARK   5 DEPOSITOR REQUESTED STRUCTURE BE OBSOLETED. ERRATUM WAS PUBLISHED.   \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 13-OCT-00.                  \nREMARK 100 THE DEPOSITION ID IS D_1000012051.                                   \nREMARK 200                                                                      \nREMARK 200 EXPERIMENTAL DETAILS                                                 \nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \nREMARK 200  DATE OF DATA COLLECTION        : 20-DEC-98                          \nREMARK 200  TEMPERATURE           (KELVIN) : 298                                \nREMARK 200  PH                             : 6.5                                \nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \nREMARK 200                                                                      \nREMARK 200  SYNCHROTRON              (Y/N) : N                                  \nREMARK 200  RADIATION SOURCE               : ROTATING ANODE                     \nREMARK 200  BEAMLINE                       : NULL                               \nREMARK 200  X-RAY GENERATOR MODEL          : RIGAKU RU200                       \nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \nREMARK 200  WAVELENGTH OR RANGE        (A) : 1.5418                             \nREMARK 200  MONOCHROMATOR                  : NULL                               \nREMARK 200  OPTICS                         : NULL                               \nREMARK 200                                                                      \nREMARK 200  DETECTOR TYPE                  : AREA DETECTOR                      \nREMARK 200  DETECTOR MANUFACTURER          : ADSC                               \nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : SDMS                               \nREMARK 200  DATA SCALING SOFTWARE          : SDMS                               \nREMARK 200                                                                      \nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 10951                              \nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.950                              \nREMARK 200  RESOLUTION RANGE LOW       (A) : 15.000                             \nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : 0.000                              \nREMARK 200                                                                      \nREMARK 200 OVERALL.                                                             \nREMARK 200  COMPLETENESS FOR RANGE     (%) : 89.0                               \nREMARK 200  DATA REDUNDANCY                : 2.000                              \nREMARK 200  R MERGE                    (I) : 0.07800                            \nREMARK 200  R SYM                      (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 6.5000                             \nREMARK 200                                                                      \nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 2.95                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 3.20                     \nREMARK 200  COMPLETENESS FOR SHELL     (%) : 80.0                               \nREMARK 200  DATA REDUNDANCY IN SHELL       : 1.20                               \nREMARK 200  R MERGE FOR SHELL          (I) : 0.22000                            \nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \nREMARK 200                                                                      \nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \nREMARK 200 SOFTWARE USED: PHASES                                                \nREMARK 200 STARTING MODEL: NULL                                                 \nREMARK 200                                                                      \nREMARK 200 REMARK: NULL                                                         \nREMARK 280                                                                      \nREMARK 280 CRYSTAL                                                              \nREMARK 280 SOLVENT CONTENT, VS   (%): 54.43                                     \nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.70                     \nREMARK 280                                                                      \nREMARK 280 CRYSTALLIZATION CONDITIONS: PEG 8000, CALCIUM ACETATE, CACODYLATE,   \nREMARK 280  PH 6.5, VAPOR DIFFUSION, HANGING DROP, TEMPERATURE 298K             \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 1 21 1                         \nREMARK 290                                                                      \nREMARK 290      SYMOP   SYMMETRY                                                \nREMARK 290     NNNMMM   OPERATOR                                                \nREMARK 290       1555   X,Y,Z                                                   \nREMARK 290       2555   -X,Y+1/2,-Z                                             \nREMARK 290                                                                      \nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \nREMARK 290           MMM -> TRANSLATION VECTOR                                  \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \nREMARK 290 RELATED MOLECULES.                                                   \nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   2  0.000000  1.000000  0.000000       32.60000            \nREMARK 290   SMTRY3   2  0.000000  0.000000 -1.000000        0.00000            \nREMARK 290                                                                      \nREMARK 290 REMARK: NULL                                                         \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1                                                       \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TETRAMERIC                        \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: W, K, D, B                            \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 465                                                                      \nREMARK 465 MISSING RESIDUES                                                     \nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \nREMARK 465                                                                      \nREMARK 465   M RES C SSSEQI                                                     \nREMARK 465     ILE D    57                                                      \nREMARK 465     ASN D    58                                                      \nREMARK 465     HIS D    59                                                      \nREMARK 465     LEU D    60                                                      \nREMARK 465     ASN D    61                                                      \nREMARK 465     TYR D    62                                                      \nREMARK 465     LYS D    63                                                      \nREMARK 465     GLN D    64                                                      \nREMARK 465     PRO D    65                                                      \nREMARK 465     GLN D    66                                                      \nREMARK 465     ASN D   112                                                      \nREMARK 465     LYS D   113                                                      \nREMARK 465     ARG D   114                                                      \nREMARK 465     ILE D   115                                                      \nREMARK 465     THR D   116                                                      \nREMARK 465     GLU D   117                                                      \nREMARK 465     ALA D   118                                                      \nREMARK 465     ASN D   119                                                      \nREMARK 465     ASN D   120                                                      \nREMARK 465     GLU D   121                                                      \nREMARK 465     ILE D   122                                                      \nREMARK 465     GLU D   123                                                      \nREMARK 465     LEU D   124                                                      \nREMARK 465     ILE B    57                                                      \nREMARK 465     ASN B    58                                                      \nREMARK 465     HIS B    59                                                      \nREMARK 465     LEU B    60                                                      \nREMARK 465     ASN B    61                                                      \nREMARK 465     TYR B    62                                                      \nREMARK 465     LYS B    63                                                      \nREMARK 465     GLN B    64                                                      \nREMARK 465     PRO B    65                                                      \nREMARK 465     GLN B    66                                                      \nREMARK 465     GLU B   117                                                      \nREMARK 465     ALA B   118                                                      \nREMARK 465     ASN B   119                                                      \nREMARK 465     ASN B   120                                                      \nREMARK 465     GLU B   121                                                      \nREMARK 465     ILE B   122                                                      \nREMARK 465     GLU B   123                                                      \nREMARK 465     LEU B   124                                                      \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: CLOSE CONTACTS IN SAME ASYMMETRIC UNIT                     \nREMARK 500                                                                      \nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \nREMARK 500                                                                      \nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \nREMARK 500   CG2  ILE D     3     CG2  ILE D     6              1.54            \nREMARK 500   N1    DA W     3     N6    DA K     7              1.83            \nREMARK 500   N6    DA W     5     N6    DA K     5              1.99            \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \nREMARK 500     DT W   4   C5     DT W   4   C7      0.046                       \nREMARK 500     DT W   8   C5'    DT W   8   C4'     0.083                       \nREMARK 500     DT W   8   C5     DT W   8   C7      0.037                       \nREMARK 500     DT K   4   C5     DT K   4   C7      0.045                       \nREMARK 500     DT K   6   C5'    DT K   6   C4'     0.051                       \nREMARK 500     DT K   8   C5     DT K   8   C7      0.038                       \nREMARK 500     DA K   9   C5'    DA K   9   C4'     0.057                       \nREMARK 500    CYS D   9   CB    CYS D   9   SG      0.150                       \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \nREMARK 500     DA W   3   OP1 -  P   -  OP2 ANGL. DEV. = -16.7 DEGREES          \nREMARK 500     DA W   3   O4' -  C4' -  C3' ANGL. DEV. =   6.0 DEGREES          \nREMARK 500     DA W   3   C2' -  C3' -  O3' ANGL. DEV. =  27.6 DEGREES          \nREMARK 500     DA W   5   O4' -  C1' -  C2' ANGL. DEV. =   3.7 DEGREES          \nREMARK 500     DT W   6   O4' -  C4' -  C3' ANGL. DEV. =  -3.3 DEGREES          \nREMARK 500     DT W   6   C5' -  C4' -  C3' ANGL. DEV. =   8.4 DEGREES          \nREMARK 500     DT W   6   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \nREMARK 500     DT W   6   C3' -  O3' -  P   ANGL. DEV. =   9.8 DEGREES          \nREMARK 500     DA W   7   O4' -  C4' -  C3' ANGL. DEV. =  -5.6 DEGREES          \nREMARK 500     DA W   7   C3' -  C2' -  C1' ANGL. DEV. =  -7.3 DEGREES          \nREMARK 500     DT W   8   O4' -  C4' -  C3' ANGL. DEV. =  -9.6 DEGREES          \nREMARK 500     DT W   8   C3' -  C2' -  C1' ANGL. DEV. =  -6.2 DEGREES          \nREMARK 500     DT W   8   O4' -  C1' -  C2' ANGL. DEV. =   3.3 DEGREES          \nREMARK 500     DA W   9   O5' -  P   -  OP2 ANGL. DEV. =  -8.7 DEGREES          \nREMARK 500     DA K   3   OP1 -  P   -  OP2 ANGL. DEV. =  24.1 DEGREES          \nREMARK 500     DA K   3   O5' -  P   -  OP2 ANGL. DEV. = -16.9 DEGREES          \nREMARK 500     DA K   3   O4' -  C1' -  C2' ANGL. DEV. =   3.9 DEGREES          \nREMARK 500     DA K   5   O4' -  C1' -  C2' ANGL. DEV. =   3.6 DEGREES          \nREMARK 500     DT K   6   O4' -  C4' -  C3' ANGL. DEV. =  -4.7 DEGREES          \nREMARK 500     DT K   6   O4' -  C1' -  N1  ANGL. DEV. =   2.5 DEGREES          \nREMARK 500     DA K   7   O4' -  C4' -  C3' ANGL. DEV. =  -5.2 DEGREES          \nREMARK 500     DA K   7   C3' -  C2' -  C1' ANGL. DEV. =  -7.4 DEGREES          \nREMARK 500     DA K   7   O4' -  C1' -  N9  ANGL. DEV. =   2.4 DEGREES          \nREMARK 500     DT K   8   O4' -  C4' -  C3' ANGL. DEV. =  -5.7 DEGREES          \nREMARK 500     DA K   9   O4' -  C1' -  C2' ANGL. DEV. =   4.1 DEGREES          \nREMARK 500    TYR D  18   N   -  CA  -  CB  ANGL. DEV. = -10.8 DEGREES          \nREMARK 500    ILE D  43   CB  -  CA  -  C   ANGL. DEV. =  13.6 DEGREES          \nREMARK 500    LYS D  54   CB  -  CA  -  C   ANGL. DEV. =  13.5 DEGREES          \nREMARK 500    ARG D  84   NE  -  CZ  -  NH1 ANGL. DEV. =   3.2 DEGREES          \nREMARK 500    LEU B  22   CB  -  CA  -  C   ANGL. DEV. =  13.9 DEGREES          \nREMARK 500    LEU B  40   CA  -  CB  -  CG  ANGL. DEV. =  15.4 DEGREES          \nREMARK 500    TYR B  51   CB  -  CA  -  C   ANGL. DEV. =  12.9 DEGREES          \nREMARK 500    SER B  75   N   -  CA  -  CB  ANGL. DEV. =   9.3 DEGREES          \nREMARK 500    HIS B  87   CB  -  CA  -  C   ANGL. DEV. =  15.5 DEGREES          \nREMARK 500    LEU B  93   CA  -  CB  -  CG  ANGL. DEV. =  15.8 DEGREES          \nREMARK 500    ARG B 110   NE  -  CZ  -  NH2 ANGL. DEV. =   3.6 DEGREES          \nREMARK 500    ARG B 114   NE  -  CZ  -  NH2 ANGL. DEV. =   4.2 DEGREES          \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    ILE D   3     -178.06    -45.65                                   \nREMARK 500    THR D   4     -114.92   -130.16                                   \nREMARK 500    ASP D   8      163.78    -44.68                                   \nREMARK 500    CYS D   9      136.33    -20.07                                   \nREMARK 500    GLU D  11       -7.44    -54.47                                   \nREMARK 500    LEU D  12      -63.20   -105.61                                   \nREMARK 500    LYS D  21      -84.39    -48.89                                   \nREMARK 500    LEU D  22      -30.42    -31.12                                   \nREMARK 500    PHE D  30     -119.25   -173.96                                   \nREMARK 500    SER D  31      177.89    -30.73                                   \nREMARK 500    ILE D  32      -43.84   -153.41                                   \nREMARK 500    SER D  33       -7.15    -53.85                                   \nREMARK 500    LYS D  54       -6.88    -52.40                                   \nREMARK 500    LYS D  69        0.05    -69.33                                   \nREMARK 500    VAL D  92        5.99    -56.35                                   \nREMARK 500    GLN D 100     -127.42    -51.20                                   \nREMARK 500    ARG D 101      -17.82    -44.99                                   \nREMARK 500    ILE B   3      178.33    -38.43                                   \nREMARK 500    THR B   4      -66.17   -159.76                                   \nREMARK 500    ILE B   6       78.82    -62.67                                   \nREMARK 500    ASN B   7       54.57   -164.26                                   \nREMARK 500    PHE B  10      -89.38    -80.37                                   \nREMARK 500    LEU B  13      -89.43    -49.61                                   \nREMARK 500    PHE B  30     -130.28   -157.46                                   \nREMARK 500    SER B  31     -143.94    -49.49                                   \nREMARK 500    ILE B  32      -17.29   -173.16                                   \nREMARK 500    LEU B  40      -34.99    -36.62                                   \nREMARK 500    THR B  41       11.05    -67.61                                   \nREMARK 500    GLU B  48      -74.82    -67.08                                   \nREMARK 500    LYS B  54        6.84    -57.89                                   \nREMARK 500    ASP B  55      -48.58   -138.52                                   \nREMARK 500    ALA B  70      -76.77    -61.67                                   \nREMARK 500    ARG B  90       24.51   -140.30                                   \nREMARK 500    VAL B  92        3.04    -69.82                                   \nREMARK 500    ILE B  94      -88.33    -49.57                                   \nREMARK 500    SER B 106      -74.50    -51.20                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 620                                                                      \nREMARK 620 METAL COORDINATION                                                   \nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA B 200  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1  DT W   6   OP1                                                    \nREMARK 620 2  DA K   9   OP1 122.5                                              \nREMARK 620 3 GLU B  11   OE2 159.9  52.4                                        \nREMARK 620 N                    1     2                                         \nREMARK 800                                                                      \nREMARK 800 SITE                                                                 \nREMARK 800 SITE_IDENTIFIER: AC1                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA D 200                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC2                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA B 200                  \nREMARK 900                                                                      \nREMARK 900 RELATED ENTRIES                                                      \nREMARK 900 RELATED ID: 1FZN   RELATED DB: PDB                                   \nREMARK 900 CRYSTAL STRUCTURE OF SARA                                            \nDBREF  1FZP D    2   124  UNP    Q53600   SARA_STAAU       1    123             \nDBREF  1FZP B    2   124  UNP    Q53600   SARA_STAAU       1    123             \nDBREF  1FZP W    3     9  PDB    1FZP     1FZP             3      9             \nDBREF  1FZP K    3     9  PDB    1FZP     1FZP             3      9             \nSEQRES   1 W    7   DA  DT  DA  DT  DA  DT  DA                                  \nSEQRES   1 K    7   DA  DT  DA  DT  DA  DT  DA                                  \nSEQRES   1 D  123  ALA ILE THR LYS ILE ASN ASP CYS PHE GLU LEU LEU SER          \nSEQRES   2 D  123  MET VAL THR TYR ALA ASP LYS LEU LYS SER LEU ILE LYS          \nSEQRES   3 D  123  LYS GLU PHE SER ILE SER PHE GLU GLU PHE ALA VAL LEU          \nSEQRES   4 D  123  THR TYR ILE SER GLU ASN LYS GLU LYS GLU TYR TYR LEU          \nSEQRES   5 D  123  LYS ASP ILE ILE ASN HIS LEU ASN TYR LYS GLN PRO GLN          \nSEQRES   6 D  123  VAL VAL LYS ALA VAL LYS ILE LEU SER GLN GLU ASP TYR          \nSEQRES   7 D  123  PHE ASP LYS LYS ARG ASN GLU HIS ASP GLU ARG THR VAL          \nSEQRES   8 D  123  LEU ILE LEU VAL ASN ALA GLN GLN ARG LYS LYS ILE GLU          \nSEQRES   9 D  123  SER LEU LEU SER ARG VAL ASN LYS ARG ILE THR GLU ALA          \nSEQRES  10 D  123  ASN ASN GLU ILE GLU LEU                                      \nSEQRES   1 B  123  ALA ILE THR LYS ILE ASN ASP CYS PHE GLU LEU LEU SER          \nSEQRES   2 B  123  MET VAL THR TYR ALA ASP LYS LEU LYS SER LEU ILE LYS          \nSEQRES   3 B  123  LYS GLU PHE SER ILE SER PHE GLU GLU PHE ALA VAL LEU          \nSEQRES   4 B  123  THR TYR ILE SER GLU ASN LYS GLU LYS GLU TYR TYR LEU          \nSEQRES   5 B  123  LYS ASP ILE ILE ASN HIS LEU ASN TYR LYS GLN PRO GLN          \nSEQRES   6 B  123  VAL VAL LYS ALA VAL LYS ILE LEU SER GLN GLU ASP TYR          \nSEQRES   7 B  123  PHE ASP LYS LYS ARG ASN GLU HIS ASP GLU ARG THR VAL          \nSEQRES   8 B  123  LEU ILE LEU VAL ASN ALA GLN GLN ARG LYS LYS ILE GLU          \nSEQRES   9 B  123  SER LEU LEU SER ARG VAL ASN LYS ARG ILE THR GLU ALA          \nSEQRES  10 B  123  ASN ASN GLU ILE GLU LEU                                      \nHET     CA  D 200       1                                                       \nHET     CA  B 200       1                                                       \nHETNAM      CA CALCIUM ION                                                      \nFORMUL   5   CA    2(CA 2+)                                                     \nHELIX    1   1 CYS D    9  SER D   14  1                                   6    \nHELIX    2   2 SER D   14  PHE D   30  1                                  17    \nHELIX    3   3 ILE D   32  LEU D   40  1                                   9    \nHELIX    4   4 THR D   41  ILE D   43  5                                   3    \nHELIX    5   5 SER D   44  ASN D   46  5                                   3    \nHELIX    6   6 LYS D   47  ASP D   55  1                                   9    \nHELIX    7   7 VAL D   67  ASP D   88  1                                  22    \nHELIX    8   8 THR D   91  VAL D  111  1                                  21    \nHELIX    9   9 PHE B   10  PHE B   30  1                                  21    \nHELIX   10  10 ILE B   32  THR B   41  1                                  10    \nHELIX   11  11 ASN B   46  LYS B   54  1                                   9    \nHELIX   12  12 VAL B   67  GLU B   86  1                                  20    \nHELIX   13  13 HIS B   87  ARG B   90  5                                   4    \nHELIX   14  14 VAL B   92  SER B  106  1                                  15    \nHELIX   15  15 LEU B  107  ILE B  115  1                                   9    \nLINK         OP1  DT W   6                CA    CA B 200     1555   1555  2.72  \nLINK         OP1  DA K   9                CA    CA B 200     1555   1555  2.86  \nLINK         OE2 GLU D  11                CA    CA D 200     1555   1555  1.55  \nLINK         OE2 GLU B  11                CA    CA B 200     1555   1555  2.83  \nCISPEP   1 ASP B    8    CYS B    9          0        -0.12                     \nSITE     1 AC1  3 ARG B  84  GLU D  11   DA W   9                               \nSITE     1 AC2  3 GLU B  11   DA K   9   DT W   6                               \nCRYST1   54.500   65.200   57.800  90.00 118.00  90.00 P 1 21 1      4          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      0.018349  0.000000  0.009756        0.00000                         \nSCALE2      0.000000  0.015337  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  0.019595        0.00000                         \nATOM      1  P    DA W   3       9.641  15.812 -14.279  1.00 60.87           P  \nATOM      2  OP1  DA W   3      11.080  15.668 -13.967  1.00 61.26           O  \nATOM      3  OP2  DA W   3       9.648  16.156 -15.719  1.00 58.83           O  \nATOM      4  O5'  DA W   3       8.881  16.904 -13.366  1.00 52.60           O  \nATOM      5  C5'  DA W   3       8.038  17.881 -14.015  1.00 60.20           C  \nATOM      6  C4'  DA W   3       8.557  19.302 -13.796  1.00 49.01           C  \nATOM      7  O4'  DA W   3       9.729  19.156 -12.966  1.00 48.83           O  \nATOM      8  C3'  DA W   3       8.866  20.000 -15.090  1.00 35.76           C  \nATOM      9  O3'  DA W   3       7.904  21.019 -15.130  1.00 43.27           O  \nATOM     10  C2'  DA W   3      10.365  19.814 -15.172  1.00 39.03           C  \nATOM     11  C1'  DA W   3      10.772  19.770 -13.699  1.00 42.39           C  \nATOM     12  N9   DA W   3      12.002  19.098 -13.307  1.00 42.05           N  \nATOM     13  C8   DA W   3      12.677  18.064 -13.908  1.00 44.95           C  \nATOM     14  N7   DA W   3      13.772  17.704 -13.264  1.00 45.41           N  \nATOM     15  C5   DA W   3      13.810  18.572 -12.185  1.00 45.31           C  \nATOM     16  C6   DA W   3      14.713  18.729 -11.123  1.00 49.24           C  \nATOM     17  N6   DA W   3      15.801  17.954 -10.968  1.00 52.80           N  \nATOM     18  N1   DA W   3      14.449  19.669 -10.224  1.00 48.99           N  \nATOM     19  C2   DA W   3      13.342  20.435 -10.364  1.00 45.51           C  \nATOM     20  N3   DA W   3      12.427  20.395 -11.325  1.00 40.11           N  \nATOM     21  C4   DA W   3      12.727  19.428 -12.204  1.00 41.62           C  \nATOM     22  P    DT W   4       8.232  22.500 -15.583  1.00 34.80           P  \nATOM     23  OP1  DT W   4       6.992  23.161 -16.069  1.00 35.01           O  \nATOM     24  OP2  DT W   4       9.439  22.531 -16.438  1.00 34.65           O  \nATOM     25  O5'  DT W   4       8.718  23.233 -14.233  1.00 25.99           O  \nATOM     26  C5'  DT W   4       7.934  24.338 -13.767  1.00 33.01           C  \nATOM     27  C4'  DT W   4       8.666  25.239 -12.753  1.00 28.60           C  \nATOM     28  O4'  DT W   4       9.670  24.428 -12.079  1.00 29.43           O  \nATOM     29  C3'  DT W   4       9.449  26.446 -13.268  1.00 24.64           C  \nATOM     30  O3'  DT W   4       9.477  27.637 -12.437  1.00 31.85           O  \nATOM     31  C2'  DT W   4      10.872  25.977 -13.443  1.00 27.60           C  \nATOM     32  C1'  DT W   4      10.951  25.062 -12.236  1.00 30.47           C  \nATOM     33  N1   DT W   4      11.975  24.001 -12.309  1.00 33.73           N  \nATOM     34  C2   DT W   4      12.786  23.715 -11.235  1.00 34.62           C  \nATOM     35  O2   DT W   4      12.710  24.309 -10.167  1.00 33.57           O  \nATOM     36  N3   DT W   4      13.709  22.690 -11.443  1.00 37.00           N  \nATOM     37  C4   DT W   4      13.875  21.940 -12.599  1.00 38.58           C  \nATOM     38  O4   DT W   4      14.709  21.034 -12.654  1.00 39.28           O  \nATOM     39  C5   DT W   4      12.987  22.317 -13.664  1.00 38.21           C  \nATOM     40  C7   DT W   4      13.076  21.580 -15.016  1.00 39.44           C  \nATOM     41  C6   DT W   4      12.093  23.308 -13.485  1.00 36.12           C  \nATOM     42  P    DA W   5       8.769  27.750 -11.014  1.00 39.52           P  \nATOM     43  OP1  DA W   5       8.269  26.430 -10.593  1.00 42.95           O  \nATOM     44  OP2  DA W   5       7.766  28.824 -11.033  1.00 37.95           O  \nATOM     45  O5'  DA W   5      10.021  28.223 -10.089  1.00 29.73           O  \nATOM     46  C5'  DA W   5       9.842  28.823  -8.796  1.00 36.11           C  \nATOM     47  C4'  DA W   5      10.922  28.351  -7.804  1.00 28.81           C  \nATOM     48  O4'  DA W   5      11.603  27.240  -8.469  1.00 25.64           O  \nATOM     49  C3'  DA W   5      11.986  29.390  -7.380  1.00 23.74           C  \nATOM     50  O3'  DA W   5      12.315  29.352  -5.995  1.00 30.96           O  \nATOM     51  C2'  DA W   5      13.160  29.117  -8.323  1.00 22.88           C  \nATOM     52  C1'  DA W   5      12.958  27.636  -8.701  1.00 22.63           C  \nATOM     53  N9   DA W   5      13.282  27.398 -10.145  1.00 19.18           N  \nATOM     54  C8   DA W   5      12.441  27.543 -11.197  1.00 18.07           C  \nATOM     55  N7   DA W   5      12.989  27.292 -12.331  1.00 18.87           N  \nATOM     56  C5   DA W   5      14.300  26.965 -12.027  1.00 18.30           C  \nATOM     57  C6   DA W   5      15.422  26.598 -12.825  1.00 15.95           C  \nATOM     58  N6   DA W   5      15.357  26.505 -14.179  1.00 15.73           N  \nATOM     59  N1   DA W   5      16.577  26.346 -12.220  1.00 13.80           N  \nATOM     60  C2   DA W   5      16.636  26.443 -10.857  1.00 13.79           C  \nATOM     61  N3   DA W   5      15.655  26.781 -10.006  1.00 15.62           N  \nATOM     62  C4   DA W   5      14.511  27.031 -10.660  1.00 17.82           C  \nATOM     63  P    DT W   6      12.774  30.665  -5.178  1.00 38.40           P  \nATOM     64  OP1  DT W   6      12.309  30.564  -3.785  1.00 38.43           O  \nATOM     65  OP2  DT W   6      12.371  31.893  -5.903  1.00 40.00           O  \nATOM     66  O5'  DT W   6      14.393  30.512  -5.171  1.00 29.17           O  \nATOM     67  C5'  DT W   6      15.031  29.529  -6.023  1.00 36.03           C  \nATOM     68  C4'  DT W   6      16.514  29.273  -5.674  1.00 29.53           C  \nATOM     69  O4'  DT W   6      17.122  28.747  -6.878  1.00 27.83           O  \nATOM     70  C3'  DT W   6      17.519  30.384  -5.291  1.00 22.49           C  \nATOM     71  O3'  DT W   6      18.707  29.871  -4.582  1.00 28.54           O  \nATOM     72  C2'  DT W   6      17.788  31.031  -6.645  1.00 25.05           C  \nATOM     73  C1'  DT W   6      17.808  29.797  -7.555  1.00 27.11           C  \nATOM     74  N1   DT W   6      17.327  29.926  -8.945  1.00 27.65           N  \nATOM     75  C2   DT W   6      18.248  29.631  -9.898  1.00 28.01           C  \nATOM     76  O2   DT W   6      19.375  29.274  -9.645  1.00 27.37           O  \nATOM     77  N3   DT W   6      17.836  29.736 -11.203  1.00 29.56           N  \nATOM     78  C4   DT W   6      16.579  30.146 -11.631  1.00 31.93           C  \nATOM     79  O4   DT W   6      16.345  30.202 -12.850  1.00 34.79           O  \nATOM     80  C5   DT W   6      15.648  30.483 -10.551  1.00 30.30           C  \nATOM     81  C7   DT W   6      14.236  30.980 -10.869  1.00 30.70           C  \nATOM     82  C6   DT W   6      16.057  30.366  -9.265  1.00 28.44           C  \nATOM     83  P    DA W   7      20.031  30.662  -4.080  1.00 34.92           P  \nATOM     84  OP1  DA W   7      20.535  30.072  -2.816  1.00 33.68           O  \nATOM     85  OP2  DA W   7      19.765  32.117  -4.081  1.00 35.51           O  \nATOM     86  O5'  DA W   7      21.127  30.346  -5.241  1.00 25.19           O  \nATOM     87  C5'  DA W   7      22.478  29.945  -4.865  1.00 31.59           C  \nATOM     88  C4'  DA W   7      22.875  28.646  -5.593  1.00 26.23           C  \nATOM     89  O4'  DA W   7      22.127  28.591  -6.876  1.00 26.40           O  \nATOM     90  C3'  DA W   7      24.275  28.517  -6.135  1.00 22.88           C  \nATOM     91  O3'  DA W   7      24.601  27.171  -6.507  1.00 30.08           O  \nATOM     92  C2'  DA W   7      24.353  29.313  -7.404  1.00 25.56           C  \nATOM     93  C1'  DA W   7      23.032  28.801  -7.965  1.00 27.56           C  \nATOM     94  N9   DA W   7      22.373  29.681  -9.003  1.00 31.34           N  \nATOM     95  C8   DA W   7      21.840  30.927  -8.788  1.00 33.09           C  \nATOM     96  N7   DA W   7      21.342  31.483  -9.840  1.00 33.74           N  \nATOM     97  C5   DA W   7      21.530  30.548 -10.831  1.00 32.48           C  \nATOM     98  C6   DA W   7      21.183  30.556 -12.211  1.00 30.67           C  \nATOM     99  N6   DA W   7      20.593  31.598 -12.821  1.00 29.18           N  \nATOM    100  N1   DA W   7      21.476  29.464 -12.916  1.00 30.78           N  \nATOM    101  C2   DA W   7      22.107  28.440 -12.301  1.00 31.59           C  \nATOM    102  N3   DA W   7      22.513  28.337 -11.032  1.00 31.79           N  \nATOM    103  C4   DA W   7      22.175  29.428 -10.334  1.00 32.05           C  \nATOM    104  P    DT W   8      25.673  26.847  -7.640  1.00 31.38           P  \nATOM    105  OP1  DT W   8      26.738  25.972  -7.097  1.00 32.37           O  \nATOM    106  OP2  DT W   8      26.121  28.098  -8.292  1.00 29.11           O  \nATOM    107  O5'  DT W   8      24.805  26.060  -8.713  1.00 22.13           O  \nATOM    108  C5'  DT W   8      24.427  26.742  -9.932  1.00 28.92           C  \nATOM    109  C4'  DT W   8      25.319  26.273 -11.168  1.00 26.02           C  \nATOM    110  O4'  DT W   8      25.105  27.167 -12.290  1.00 27.10           O  \nATOM    111  C3'  DT W   8      26.895  26.464 -11.140  1.00 22.10           C  \nATOM    112  O3'  DT W   8      27.615  26.128 -12.381  1.00 29.12           O  \nATOM    113  C2'  DT W   8      27.130  27.928 -11.219  1.00 23.97           C  \nATOM    114  C1'  DT W   8      26.071  28.207 -12.279  1.00 26.38           C  \nATOM    115  N1   DT W   8      25.351  29.466 -12.109  1.00 28.67           N  \nATOM    116  C2   DT W   8      24.546  29.847 -13.149  1.00 27.54           C  \nATOM    117  O2   DT W   8      24.398  29.194 -14.178  1.00 25.93           O  \nATOM    118  N3   DT W   8      23.907  31.040 -12.953  1.00 28.37           N  \nATOM    119  C4   DT W   8      23.982  31.872 -11.837  1.00 30.03           C  \nATOM    120  O4   DT W   8      23.339  32.908 -11.817  1.00 31.02           O  \nATOM    121  C5   DT W   8      24.835  31.397 -10.778  1.00 30.15           C  \nATOM    122  C7   DT W   8      25.021  32.176  -9.471  1.00 30.04           C  \nATOM    123  C6   DT W   8      25.479  30.229 -10.955  1.00 29.94           C  \nATOM    124  P    DA W   9      29.112  26.641 -12.750  1.00 53.38           P  \nATOM    125  OP1  DA W   9      30.043  26.282 -11.656  1.00 50.45           O  \nATOM    126  OP2  DA W   9      29.112  27.994 -13.346  1.00 49.37           O  \nATOM    127  O5'  DA W   9      29.379  25.836 -14.095  1.00 45.21           O  \nATOM    128  C5'  DA W   9      28.259  25.069 -14.535  1.00 53.03           C  \nATOM    129  C4'  DA W   9      27.927  25.472 -15.972  1.00 54.85           C  \nATOM    130  O4'  DA W   9      27.014  26.644 -15.916  1.00 59.77           O  \nATOM    131  C3'  DA W   9      29.119  25.875 -16.819  1.00 50.06           C  \nATOM    132  O3'  DA W   9      29.857  24.745 -17.264  1.00 57.89           O  \nATOM    133  C2'  DA W   9      28.515  26.894 -17.778  1.00 53.26           C  \nATOM    134  C1'  DA W   9      27.463  27.584 -16.911  1.00 57.42           C  \nATOM    135  N9   DA W   9      27.939  28.767 -16.197  1.00 55.28           N  \nATOM    136  C8   DA W   9      28.858  28.808 -15.189  1.00 50.94           C  \nATOM    137  N7   DA W   9      29.069  29.998 -14.699  1.00 50.03           N  \nATOM    138  C5   DA W   9      28.228  30.808 -15.439  1.00 50.48           C  \nATOM    139  C6   DA W   9      27.987  32.188 -15.412  1.00 49.49           C  \nATOM    140  N6   DA W   9      28.624  33.027 -14.570  1.00 48.59           N  \nATOM    141  N1   DA W   9      27.091  32.683 -16.289  1.00 51.70           N  \nATOM    142  C2   DA W   9      26.487  31.836 -17.127  1.00 53.03           C  \nATOM    143  N3   DA W   9      26.620  30.518 -17.244  1.00 54.02           N  \nATOM    144  C4   DA W   9      27.520  30.061 -16.366  1.00 53.72           C  \nTER     145       DA W   9                                                      \nATOM    146  P    DA K   3      21.580  36.193 -19.437  1.00 26.48           P  \nATOM    147  OP1  DA K   3      21.455  36.048 -17.969  1.00 23.46           O  \nATOM    148  OP2  DA K   3      22.499  35.998 -20.582  1.00 24.13           O  \nATOM    149  O5'  DA K   3      20.469  35.359 -20.228  1.00 18.50           O  \nATOM    150  C5'  DA K   3      20.768  34.963 -21.571  1.00 26.26           C  \nATOM    151  C4'  DA K   3      20.601  33.458 -21.685  1.00 24.05           C  \nATOM    152  O4'  DA K   3      21.311  32.849 -20.547  1.00 25.74           O  \nATOM    153  C3'  DA K   3      19.177  32.969 -21.676  1.00 20.64           C  \nATOM    154  O3'  DA K   3      18.954  32.075 -22.732  1.00 27.37           O  \nATOM    155  C2'  DA K   3      18.922  32.395 -20.302  1.00 23.53           C  \nATOM    156  C1'  DA K   3      20.329  32.153 -19.754  1.00 26.62           C  \nATOM    157  N9   DA K   3      20.437  32.706 -18.387  1.00 30.34           N  \nATOM    158  C8   DA K   3      19.498  33.418 -17.705  1.00 32.04           C  \nATOM    159  N7   DA K   3      19.876  33.775 -16.503  1.00 33.26           N  \nATOM    160  C5   DA K   3      21.167  33.284 -16.402  1.00 33.71           C  \nATOM    161  C6   DA K   3      22.122  33.326 -15.370  1.00 34.66           C  \nATOM    162  N6   DA K   3      21.892  33.945 -14.189  1.00 36.69           N  \nATOM    163  N1   DA K   3      23.290  32.709 -15.598  1.00 32.42           N  \nATOM    164  C2   DA K   3      23.509  32.105 -16.777  1.00 30.76           C  \nATOM    165  N3   DA K   3      22.707  31.990 -17.824  1.00 30.48           N  \nATOM    166  C4   DA K   3      21.529  32.604 -17.562  1.00 31.72           C  \nATOM    167  P    DT K   4      18.071  30.780 -22.469  1.00 34.80           P  \nATOM    168  OP1  DT K   4      17.740  30.111 -23.743  1.00 35.01           O  \nATOM    169  OP2  DT K   4      17.018  31.073 -21.467  1.00 34.65           O  \nATOM    170  O5'  DT K   4      19.087  29.760 -21.715  1.00 25.99           O  \nATOM    171  C5'  DT K   4      19.502  28.563 -22.431  1.00 33.01           C  \nATOM    172  C4'  DT K   4      20.153  27.484 -21.549  1.00 28.60           C  \nATOM    173  O4'  DT K   4      20.662  28.121 -20.342  1.00 29.43           O  \nATOM    174  C3'  DT K   4      19.249  26.376 -21.035  1.00 24.64           C  \nATOM    175  O3'  DT K   4      19.819  25.073 -20.882  1.00 31.85           O  \nATOM    176  C2'  DT K   4      18.779  26.816 -19.664  1.00 27.60           C  \nATOM    177  C1'  DT K   4      20.059  27.486 -19.202  1.00 30.47           C  \nATOM    178  N1   DT K   4      19.897  28.507 -18.148  1.00 33.73           N  \nATOM    179  C2   DT K   4      20.775  28.553 -17.058  1.00 34.62           C  \nATOM    180  O2   DT K   4      21.705  27.772 -16.904  1.00 33.57           O  \nATOM    181  N3   DT K   4      20.506  29.568 -16.140  1.00 37.00           N  \nATOM    182  C4   DT K   4      19.466  30.522 -16.225  1.00 38.58           C  \nATOM    183  O4   DT K   4      19.330  31.388 -15.359  1.00 39.28           O  \nATOM    184  C5   DT K   4      18.630  30.381 -17.387  1.00 38.21           C  \nATOM    185  C7   DT K   4      17.460  31.357 -17.620  1.00 39.44           C  \nATOM    186  C6   DT K   4      18.870  29.404 -18.282  1.00 36.12           C  \nATOM    187  P    DA K   5      21.341  24.689 -21.206  1.00 39.52           P  \nATOM    188  OP1  DA K   5      22.187  25.895 -21.318  1.00 42.95           O  \nATOM    189  OP2  DA K   5      21.376  23.778 -22.365  1.00 37.95           O  \nATOM    190  O5'  DA K   5      21.766  23.949 -19.836  1.00 29.73           O  \nATOM    191  C5'  DA K   5      22.990  23.180 -19.707  1.00 36.11           C  \nATOM    192  C4'  DA K   5      23.700  23.426 -18.353  1.00 28.81           C  \nATOM    193  O4'  DA K   5      23.078  24.619 -17.778  1.00 25.64           O  \nATOM    194  C3'  DA K   5      23.605  22.291 -17.305  1.00 23.74           C  \nATOM    195  O3'  DA K   5      24.867  22.058 -16.626  1.00 30.96           O  \nATOM    196  C2'  DA K   5      22.438  22.690 -16.417  1.00 22.88           C  \nATOM    197  C1'  DA K   5      22.425  24.229 -16.566  1.00 22.63           C  \nATOM    198  N9   DA K   5      21.011  24.723 -16.608  1.00 19.18           N  \nATOM    199  C8   DA K   5      20.216  24.806 -17.702  1.00 18.07           C  \nATOM    200  N7   DA K   5      19.036  25.247 -17.447  1.00 18.87           N  \nATOM    201  C5   DA K   5      19.028  25.468 -16.080  1.00 18.30           C  \nATOM    202  C6   DA K   5      18.033  25.941 -15.176  1.00 15.95           C  \nATOM    203  N6   DA K   5      16.784  26.288 -15.550  1.00 15.73           N  \nATOM    204  N1   DA K   5      18.339  26.036 -13.887  1.00 13.80           N  \nATOM    205  C2   DA K   5      19.600  25.689 -13.494  1.00 13.79           C  \nATOM    206  N3   DA K   5      20.610  25.224 -14.238  1.00 15.62           N  \nATOM    207  C4   DA K   5      20.255  25.139 -15.529  1.00 17.82           C  \nATOM    208  P    DT K   6      25.292  20.600 -16.081  1.00 38.40           P  \nATOM    209  OP1  DT K   6      26.754  20.390 -16.254  1.00 38.43           O  \nATOM    210  OP2  DT K   6      24.408  19.575 -16.663  1.00 40.00           O  \nATOM    211  O5'  DT K   6      25.014  20.757 -14.477  1.00 29.17           O  \nATOM    212  C5'  DT K   6      23.986  21.674 -14.034  1.00 36.03           C  \nATOM    213  C4'  DT K   6      24.055  21.937 -12.495  1.00 29.53           C  \nATOM    214  O4'  DT K   6      22.841  22.659 -12.178  1.00 27.83           O  \nATOM    215  C3'  DT K   6      23.925  20.740 -11.489  1.00 22.49           C  \nATOM    216  O3'  DT K   6      24.397  21.103 -10.151  1.00 28.54           O  \nATOM    217  C2'  DT K   6      22.490  20.351 -11.664  1.00 25.05           C  \nATOM    218  C1'  DT K   6      21.834  21.732 -11.779  1.00 27.11           C  \nATOM    219  N1   DT K   6      20.624  21.882 -12.612  1.00 27.65           N  \nATOM    220  C2   DT K   6      19.524  22.321 -11.947  1.00 28.01           C  \nATOM    221  O2   DT K   6      19.522  22.586 -10.768  1.00 27.37           O  \nATOM    222  N3   DT K   6      18.380  22.476 -12.690  1.00 29.56           N  \nATOM    223  C4   DT K   6      18.242  22.196 -14.044  1.00 31.93           C  \nATOM    224  O4   DT K   6      17.140  22.377 -14.589  1.00 34.79           O  \nATOM    225  C5   DT K   6      19.458  21.693 -14.657  1.00 30.30           C  \nATOM    226  C7   DT K   6      19.420  21.310 -16.139  1.00 30.70           C  \nATOM    227  C6   DT K   6      20.587  21.552 -13.953  1.00 28.44           C  \nATOM    228  P    DA K   7      24.380  20.157  -8.811  1.00 34.92           P  \nATOM    229  OP1  DA K   7      25.573  20.423  -7.990  1.00 33.68           O  \nATOM    230  OP2  DA K   7      24.169  18.738  -9.187  1.00 35.51           O  \nATOM    231  O5'  DA K   7      23.070  20.680  -8.031  1.00 25.19           O  \nATOM    232  C5'  DA K   7      23.101  20.927  -6.601  1.00 31.59           C  \nATOM    233  C4'  DA K   7      22.528  22.327  -6.300  1.00 26.23           C  \nATOM    234  O4'  DA K   7      21.552  22.671  -7.339  1.00 26.40           O  \nATOM    235  C3'  DA K   7      21.660  22.509  -5.082  1.00 22.88           C  \nATOM    236  O3'  DA K   7      21.462  23.896  -4.761  1.00 30.08           O  \nATOM    237  C2'  DA K   7      20.299  21.963  -5.400  1.00 25.56           C  \nATOM    238  C1'  DA K   7      20.211  22.615  -6.775  1.00 27.56           C  \nATOM    239  N9   DA K   7      19.252  21.962  -7.741  1.00 31.34           N  \nATOM    240  C8   DA K   7      19.386  20.721  -8.309  1.00 33.09           C  \nATOM    241  N7   DA K   7      18.436  20.390  -9.104  1.00 33.74           N  \nATOM    242  C5   DA K   7      17.599  21.487  -9.105  1.00 32.48           C  \nATOM    243  C6   DA K   7      16.383  21.750  -9.796  1.00 30.67           C  \nATOM    244  N6   DA K   7      15.785  20.861 -10.607  1.00 29.18           N  \nATOM    245  N1   DA K   7      15.824  22.945  -9.606  1.00 30.78           N  \nATOM    246  C2   DA K   7      16.414  23.814  -8.756  1.00 31.59           C  \nATOM    247  N3   DA K   7      17.525  23.663  -8.029  1.00 31.79           N  \nATOM    248  C4   DA K   7      18.089  22.472  -8.263  1.00 32.05           C  \nATOM    249  P    DT K   8      20.145  24.385  -3.989  1.00 31.38           P  \nATOM    250  OP1  DT K   8      20.507  25.107  -2.748  1.00 32.37           O  \nATOM    251  OP2  DT K   8      19.213  23.257  -3.821  1.00 29.11           O  \nATOM    252  O5'  DT K   8      19.465  25.391  -5.045  1.00 22.13           O  \nATOM    253  C5'  DT K   8      18.342  24.955  -5.825  1.00 28.92           C  \nATOM    254  C4'  DT K   8      17.068  25.621  -5.337  1.00 26.02           C  \nATOM    255  O4'  DT K   8      15.886  24.930  -5.849  1.00 27.10           O  \nATOM    256  C3'  DT K   8      16.794  25.478  -3.846  1.00 22.10           C  \nATOM    257  O3'  DT K   8      15.930  26.507  -3.370  1.00 29.12           O  \nATOM    258  C2'  DT K   8      16.281  24.061  -3.664  1.00 23.97           C  \nATOM    259  C1'  DT K   8      15.451  23.933  -4.935  1.00 26.38           C  \nATOM    260  N1   DT K   8      15.546  22.663  -5.668  1.00 28.67           N  \nATOM    261  C2   DT K   8      14.711  22.511  -6.743  1.00 27.54           C  \nATOM    262  O2   DT K   8      13.889  23.320  -7.129  1.00 25.93           O  \nATOM    263  N3   DT K   8      14.864  21.324  -7.405  1.00 28.37           N  \nATOM    264  C4   DT K   8      15.758  20.296  -7.112  1.00 30.03           C  \nATOM    265  O4   DT K   8      15.742  19.302  -7.819  1.00 31.02           O  \nATOM    266  C5   DT K   8      16.614  20.535  -5.979  1.00 30.15           C  \nATOM    267  C7   DT K   8      17.669  19.518  -5.526  1.00 30.04           C  \nATOM    268  C6   DT K   8      16.474  21.693  -5.309  1.00 29.94           C  \nATOM    269  P    DA K   9      14.870  26.222  -2.212  1.00 36.70           P  \nATOM    270  OP1  DA K   9      14.914  27.327  -1.214  1.00 39.42           O  \nATOM    271  OP2  DA K   9      14.964  24.819  -1.740  1.00 35.88           O  \nATOM    272  O5'  DA K   9      13.467  26.209  -2.984  1.00 27.24           O  \nATOM    273  C5'  DA K   9      13.294  27.037  -4.138  1.00 34.10           C  \nATOM    274  C4'  DA K   9      11.803  27.095  -4.624  1.00 31.41           C  \nATOM    275  O4'  DA K   9      11.655  25.987  -5.580  1.00 31.10           O  \nATOM    276  C3'  DA K   9      10.693  27.008  -3.570  1.00 29.70           C  \nATOM    277  O3'  DA K   9       9.929  28.204  -3.487  1.00 37.25           O  \nATOM    278  C2'  DA K   9       9.880  25.749  -3.910  1.00 30.63           C  \nATOM    279  C1'  DA K   9      10.580  25.154  -5.156  1.00 30.32           C  \nATOM    280  N9   DA K   9      11.159  23.809  -4.949  1.00 31.53           N  \nATOM    281  C8   DA K   9      11.590  23.242  -3.765  1.00 33.33           C  \nATOM    282  N7   DA K   9      12.103  22.052  -3.882  1.00 34.17           N  \nATOM    283  C5   DA K   9      11.999  21.777  -5.237  1.00 33.36           C  \nATOM    284  C6   DA K   9      12.354  20.653  -6.002  1.00 33.46           C  \nATOM    285  N6   DA K   9      12.917  19.543  -5.463  1.00 35.72           N  \nATOM    286  N1   DA K   9      12.102  20.689  -7.324  1.00 31.61           N  \nATOM    287  C2   DA K   9      11.541  21.778  -7.834  1.00 32.23           C  \nATOM    288  N3   DA K   9      11.154  22.908  -7.221  1.00 33.15           N  \nATOM    289  C4   DA K   9      11.412  22.848  -5.907  1.00 32.62           C  \nTER     290       DA K   9                                                      \nATOM    291  N   ALA D   2      50.958  39.087 -10.179  1.00 12.13           N  \nATOM    292  CA  ALA D   2      49.826  39.020  -9.253  1.00 25.72           C  \nATOM    293  C   ALA D   2      48.587  38.371  -9.881  1.00 26.64           C  \nATOM    294  O   ALA D   2      48.155  38.719 -10.995  1.00 20.61           O  \nATOM    295  CB  ALA D   2      49.542  40.350  -8.568  1.00 29.94           C  \nATOM    296  N   ILE D   3      48.036  37.432  -9.111  1.00 34.84           N  \nATOM    297  CA  ILE D   3      46.885  36.614  -9.466  1.00 35.33           C  \nATOM    298  C   ILE D   3      45.679  37.326 -10.091  1.00 45.94           C  \nATOM    299  O   ILE D   3      45.694  38.543 -10.315  1.00 60.47           O  \nATOM    300  CB  ILE D   3      46.411  35.981  -8.180  1.00 29.83           C  \nATOM    301  CG1 ILE D   3      47.474  35.119  -7.516  1.00 29.01           C  \nATOM    302  CG2 ILE D   3      45.190  35.093  -8.358  1.00 31.21           C  \nATOM    303  CD1 ILE D   3      48.353  35.933  -6.552  1.00 13.95           C  \nATOM    304  N   THR D   4      44.705  36.433 -10.312  1.00 19.10           N  \nATOM    305  CA  THR D   4      43.385  36.687 -10.915  1.00 17.01           C  \nATOM    306  C   THR D   4      42.306  36.079 -10.021  1.00 12.28           C  \nATOM    307  O   THR D   4      42.106  36.505  -8.879  1.00 11.02           O  \nATOM    308  CB  THR D   4      43.339  35.981 -12.274  1.00 38.33           C  \nATOM    309  OG1 THR D   4      44.640  35.498 -12.606  1.00 26.23           O  \nATOM    310  CG2 THR D   4      42.876  36.876 -13.411  1.00 78.24           C  \nATOM    311  N   LYS D   5      41.652  35.089 -10.599  1.00 37.72           N  \nATOM    312  CA  LYS D   5      40.609  34.300  -9.928  1.00 50.23           C  \nATOM    313  C   LYS D   5      41.316  33.085  -9.336  1.00 35.87           C  \nATOM    314  O   LYS D   5      41.199  31.966  -9.834  1.00 21.33           O  \nATOM    315  CB  LYS D   5      39.548  33.869 -10.978  1.00 65.66           C  \nATOM    316  CG  LYS D   5      38.442  32.912 -10.450  1.00 34.40           C  \nATOM    317  CD  LYS D   5      37.078  33.142 -11.176  1.00 66.00           C  \nATOM    318  CE  LYS D   5      36.720  34.605 -11.183  1.00 80.20           C  \nATOM    319  NZ  LYS D   5      35.416  35.145 -11.786  1.00 81.00           N  \nATOM    320  N   ILE D   6      42.065  33.292  -8.255  1.00 34.10           N  \nATOM    321  CA  ILE D   6      42.817  32.167  -7.679  1.00 32.39           C  \nATOM    322  C   ILE D   6      42.141  31.551  -6.423  1.00 54.01           C  \nATOM    323  O   ILE D   6      42.638  30.575  -5.850  1.00 53.35           O  \nATOM    324  CB  ILE D   6      44.258  32.594  -7.340  1.00 39.09           C  \nATOM    325  CG1 ILE D   6      44.428  33.258  -5.983  1.00 43.56           C  \nATOM    326  CG2 ILE D   6      44.836  33.594  -8.349  1.00 47.25           C  \nATOM    327  CD1 ILE D   6      45.905  33.481  -5.619  1.00 35.04           C  \nATOM    328  N   ASN D   7      41.001  32.096  -5.988  1.00 67.67           N  \nATOM    329  CA  ASN D   7      40.268  31.512  -4.811  1.00 68.09           C  \nATOM    330  C   ASN D   7      39.814  30.093  -5.247  1.00 40.38           C  \nATOM    331  O   ASN D   7      39.363  29.886  -6.381  1.00 28.27           O  \nATOM    332  CB  ASN D   7      39.105  32.421  -4.436  1.00 99.39           C  \nATOM    333  CG  ASN D   7      39.584  33.702  -3.727  1.00100.00           C  \nATOM    334  OD1 ASN D   7      40.487  34.382  -4.222  1.00 42.30           O  \nATOM    335  ND2 ASN D   7      39.030  34.078  -2.588  1.00 37.24           N  \nATOM    336  N   ASP D   8      39.920  29.103  -4.341  1.00 42.64           N  \nATOM    337  CA  ASP D   8      39.644  27.689  -4.739  1.00 46.17           C  \nATOM    338  C   ASP D   8      38.395  27.497  -5.590  1.00 39.42           C  \nATOM    339  O   ASP D   8      37.587  28.422  -5.644  1.00 31.26           O  \nATOM    340  CB  ASP D   8      39.508  26.688  -3.626  1.00 45.25           C  \nATOM    341  CG  ASP D   8      40.617  26.733  -2.590  1.00 30.86           C  \nATOM    342  OD1 ASP D   8      41.652  27.471  -2.738  1.00 37.97           O  \nATOM    343  OD2 ASP D   8      40.444  26.053  -1.507  1.00 30.96           O  \nATOM    344  N   CYS D   9      38.295  26.317  -6.246  1.00 49.86           N  \nATOM    345  CA  CYS D   9      37.213  25.897  -7.161  1.00 45.11           C  \nATOM    346  C   CYS D   9      35.983  26.718  -6.928  1.00 33.58           C  \nATOM    347  O   CYS D   9      35.569  26.987  -5.824  1.00 10.28           O  \nATOM    348  CB  CYS D   9      36.912  24.418  -7.197  1.00 41.91           C  \nATOM    349  SG  CYS D   9      35.001  24.217  -7.620  1.00 42.41           S  \nATOM    350  N   PHE D  10      35.370  27.136  -8.027  1.00 45.12           N  \nATOM    351  CA  PHE D  10      34.267  28.068  -7.876  1.00 48.17           C  \nATOM    352  C   PHE D  10      32.876  27.371  -7.623  1.00 55.06           C  \nATOM    353  O   PHE D  10      32.240  27.600  -6.599  1.00 54.26           O  \nATOM    354  CB  PHE D  10      34.303  29.024  -9.074  1.00 48.88           C  \nATOM    355  CG  PHE D  10      35.770  29.190  -9.599  1.00 45.95           C  \nATOM    356  CD1 PHE D  10      36.887  29.167  -8.716  1.00 32.57           C  \nATOM    357  CD2 PHE D  10      36.002  29.346 -10.962  1.00 32.39           C  \nATOM    358  CE1 PHE D  10      38.206  29.233  -9.235  1.00 23.00           C  \nATOM    359  CE2 PHE D  10      37.307  29.385 -11.477  1.00 21.50           C  \nATOM    360  CZ  PHE D  10      38.409  29.324 -10.618  1.00 10.89           C  \nATOM    361  N   GLU D  11      32.329  26.517  -8.474  1.00 55.21           N  \nATOM    362  CA  GLU D  11      30.954  25.890  -8.143  1.00 61.52           C  \nATOM    363  C   GLU D  11      30.979  25.214  -6.773  1.00 63.12           C  \nATOM    364  O   GLU D  11      29.940  24.767  -6.287  1.00 74.91           O  \nATOM    365  CB  GLU D  11      30.597  24.847  -9.190  1.00 67.93           C  \nATOM    366  CG  GLU D  11      31.820  24.303  -9.919  1.00 99.79           C  \nATOM    367  CD  GLU D  11      31.470  23.186 -10.892  1.00 98.76           C  \nATOM    368  OE1 GLU D  11      32.254  22.173 -11.013  1.00 18.00           O  \nATOM    369  OE2 GLU D  11      30.389  23.259 -11.588  1.00100.00           O  \nATOM    370  N   LEU D  12      32.157  25.124  -6.159  1.00 35.34           N  \nATOM    371  CA  LEU D  12      32.293  24.477  -4.854  1.00 27.41           C  \nATOM    372  C   LEU D  12      32.491  25.466  -3.713  1.00 25.60           C  \nATOM    373  O   LEU D  12      31.688  25.521  -2.777  1.00 13.20           O  \nATOM    374  CB  LEU D  12      33.571  23.643  -4.877  1.00 28.10           C  \nATOM    375  CG  LEU D  12      33.939  22.636  -3.796  1.00 28.65           C  \nATOM    376  CD1 LEU D  12      35.300  22.030  -4.097  1.00 24.30           C  \nATOM    377  CD2 LEU D  12      33.797  23.156  -2.351  1.00 22.37           C  \nATOM    378  N   LEU D  13      33.609  26.192  -3.783  1.00 25.07           N  \nATOM    379  CA  LEU D  13      33.998  27.188  -2.787  1.00 19.78           C  \nATOM    380  C   LEU D  13      32.914  28.255  -2.619  1.00 23.97           C  \nATOM    381  O   LEU D  13      32.796  28.881  -1.560  1.00 21.55           O  \nATOM    382  CB  LEU D  13      35.254  27.874  -3.332  1.00 15.15           C  \nATOM    383  CG  LEU D  13      36.463  28.122  -2.434  1.00 20.84           C  \nATOM    384  CD1 LEU D  13      36.091  28.357  -0.972  1.00 12.36           C  \nATOM    385  CD2 LEU D  13      37.550  27.069  -2.688  1.00 18.59           C  \nATOM    386  N   SER D  14      32.130  28.459  -3.677  1.00 19.28           N  \nATOM    387  CA  SER D  14      31.063  29.443  -3.642  1.00 11.02           C  \nATOM    388  C   SER D  14      30.079  29.137  -2.521  1.00 32.80           C  \nATOM    389  O   SER D  14      29.871  29.964  -1.635  1.00 42.74           O  \nATOM    390  CB  SER D  14      30.366  29.544  -4.972  1.00 31.34           C  \nATOM    391  OG  SER D  14      29.783  30.845  -5.080  1.00 19.55           O  \nATOM    392  N   MET D  15      29.484  27.941  -2.556  1.00 24.27           N  \nATOM    393  CA  MET D  15      28.526  27.528  -1.530  1.00 10.32           C  \nATOM    394  C   MET D  15      29.216  27.235  -0.200  1.00 13.61           C  \nATOM    395  O   MET D  15      28.620  27.407   0.869  1.00 11.53           O  \nATOM    396  CB  MET D  15      27.590  26.428  -1.942  1.00 17.84           C  \nATOM    397  CG  MET D  15      26.164  26.903  -1.993  1.00 20.63           C  \nATOM    398  SD  MET D  15      25.690  27.396  -3.628  1.00 33.91           S  \nATOM    399  CE  MET D  15      27.217  28.131  -4.165  1.00 12.75           C  \nATOM    400  N   VAL D  16      30.473  26.791  -0.258  1.00 11.00           N  \nATOM    401  CA  VAL D  16      31.205  26.507   0.974  1.00 11.09           C  \nATOM    402  C   VAL D  16      31.613  27.812   1.662  1.00 25.29           C  \nATOM    403  O   VAL D  16      32.387  27.826   2.630  1.00 28.57           O  \nATOM    404  CB  VAL D  16      32.186  25.244   0.989  1.00 12.28           C  \nATOM    405  CG1 VAL D  16      33.028  25.113   2.273  1.00 11.31           C  \nATOM    406  CG2 VAL D  16      31.430  23.909   0.796  1.00 15.50           C  \nATOM    407  N   THR D  17      31.093  28.914   1.127  1.00 27.53           N  \nATOM    408  CA  THR D  17      31.349  30.250   1.661  1.00 38.36           C  \nATOM    409  C   THR D  17      30.047  30.780   2.240  1.00 29.69           C  \nATOM    410  O   THR D  17      29.887  31.001   3.446  1.00 10.05           O  \nATOM    411  CB  THR D  17      31.713  31.210   0.452  1.00 44.58           C  \nATOM    412  OG1 THR D  17      32.898  30.743  -0.218  1.00 35.14           O  \nATOM    413  CG2 THR D  17      31.897  32.651   0.906  1.00 21.05           C  \nATOM    414  N   TYR D  18      29.114  30.952   1.327  1.00 24.67           N  \nATOM    415  CA  TYR D  18      27.780  31.406   1.623  1.00 14.48           C  \nATOM    416  C   TYR D  18      27.226  30.527   2.736  1.00 21.54           C  \nATOM    417  O   TYR D  18      26.702  31.037   3.720  1.00 27.15           O  \nATOM    418  CB  TYR D  18      27.142  31.269   0.230  1.00 14.73           C  \nATOM    419  CG  TYR D  18      25.789  31.698  -0.035  1.00 21.07           C  \nATOM    420  CD1 TYR D  18      25.260  31.493  -1.267  1.00 21.20           C  \nATOM    421  CD2 TYR D  18      24.993  32.196   0.958  1.00 26.27           C  \nATOM    422  CE1 TYR D  18      23.957  31.842  -1.557  1.00 23.93           C  \nATOM    423  CE2 TYR D  18      23.676  32.544   0.708  1.00 12.85           C  \nATOM    424  CZ  TYR D  18      23.153  32.363  -0.571  1.00 33.77           C  \nATOM    425  OH  TYR D  18      21.844  32.693  -0.897  1.00 57.14           O  \nATOM    426  N   ALA D  19      27.397  29.208   2.600  1.00 29.93           N  \nATOM    427  CA  ALA D  19      26.934  28.267   3.617  1.00 31.37           C  \nATOM    428  C   ALA D  19      27.459  28.672   4.991  1.00 35.21           C  \nATOM    429  O   ALA D  19      26.682  28.777   5.934  1.00 14.53           O  \nATOM    430  CB  ALA D  19      27.304  26.804   3.262  1.00 30.19           C  \nATOM    431  N   ASP D  20      28.766  28.948   5.081  1.00 58.41           N  \nATOM    432  CA  ASP D  20      29.406  29.376   6.333  1.00 63.68           C  \nATOM    433  C   ASP D  20      28.614  30.511   6.967  1.00 60.59           C  \nATOM    434  O   ASP D  20      28.196  30.454   8.128  1.00 56.23           O  \nATOM    435  CB  ASP D  20      30.723  30.131   5.971  1.00 65.36           C  \nATOM    436  CG  ASP D  20      31.971  29.269   6.005  1.00 32.49           C  \nATOM    437  OD1 ASP D  20      32.413  29.007   7.167  1.00 16.75           O  \nATOM    438  OD2 ASP D  20      32.533  28.988   4.896  1.00 13.29           O  \nATOM    439  N   LYS D  21      28.418  31.546   6.160  1.00 45.69           N  \nATOM    440  CA  LYS D  21      27.694  32.735   6.566  1.00 41.29           C  \nATOM    441  C   LYS D  21      26.373  32.374   7.221  1.00 33.49           C  \nATOM    442  O   LYS D  21      26.289  32.297   8.450  1.00 31.83           O  \nATOM    443  CB  LYS D  21      27.338  33.545   5.315  1.00 47.53           C  \nATOM    444  CG  LYS D  21      28.057  34.887   5.178  1.00100.00           C  \nATOM    445  CD  LYS D  21      28.618  35.124   3.782  1.00 85.37           C  \nATOM    446  CE  LYS D  21      29.590  36.292   3.810  1.00 43.77           C  \nATOM    447  NZ  LYS D  21      28.834  37.574   3.788  1.00 42.32           N  \nATOM    448  N   LEU D  22      25.367  32.163   6.363  1.00 31.07           N  \nATOM    449  CA  LEU D  22      23.982  31.828   6.714  1.00 19.89           C  \nATOM    450  C   LEU D  22      23.867  31.021   7.999  1.00 20.94           C  \nATOM    451  O   LEU D  22      22.891  31.147   8.727  1.00 13.97           O  \nATOM    452  CB  LEU D  22      23.255  31.193   5.493  1.00 14.26           C  \nATOM    453  CG  LEU D  22      23.042  32.168   4.273  1.00 18.42           C  \nATOM    454  CD1 LEU D  22      24.299  32.955   3.772  1.00 11.20           C  \nATOM    455  CD2 LEU D  22      22.122  31.639   3.160  1.00 14.21           C  \nATOM    456  N   LYS D  23      24.882  30.222   8.284  1.00 23.55           N  \nATOM    457  CA  LYS D  23      24.875  29.422   9.495  1.00 29.31           C  \nATOM    458  C   LYS D  23      25.108  30.316  10.711  1.00 36.54           C  \nATOM    459  O   LYS D  23      24.464  30.166  11.753  1.00 40.89           O  \nATOM    460  CB  LYS D  23      25.896  28.255   9.398  1.00 28.10           C  \nATOM    461  CG  LYS D  23      25.356  26.879   8.948  1.00 19.58           C  \nATOM    462  CD  LYS D  23      26.251  26.109   7.919  1.00 13.14           C  \nATOM    463  CE  LYS D  23      27.464  25.444   8.507  1.00 11.23           C  \nATOM    464  NZ  LYS D  23      27.574  26.064   9.842  1.00 25.20           N  \nATOM    465  N   SER D  24      26.028  31.262  10.589  1.00 14.01           N  \nATOM    466  CA  SER D  24      26.309  32.138  11.712  1.00 10.32           C  \nATOM    467  C   SER D  24      25.158  33.097  11.965  1.00 20.06           C  \nATOM    468  O   SER D  24      24.823  33.385  13.115  1.00 16.35           O  \nATOM    469  CB  SER D  24      27.667  32.808  11.621  1.00 11.00           C  \nATOM    470  OG  SER D  24      28.202  32.959  12.932  1.00 12.84           O  \nATOM    471  N   LEU D  25      24.548  33.563  10.879  1.00 22.08           N  \nATOM    472  CA  LEU D  25      23.422  34.486  10.956  1.00 13.94           C  \nATOM    473  C   LEU D  25      22.365  33.983  11.933  1.00 33.28           C  \nATOM    474  O   LEU D  25      21.881  34.750  12.763  1.00 22.76           O  \nATOM    475  CB  LEU D  25      22.792  34.570   9.590  1.00 12.80           C  \nATOM    476  CG  LEU D  25      21.961  35.782   9.565  1.00 17.55           C  \nATOM    477  CD1 LEU D  25      22.725  36.750  10.441  1.00 22.42           C  \nATOM    478  CD2 LEU D  25      21.786  36.285   8.118  1.00 11.75           C  \nATOM    479  N   ILE D  26      22.040  32.688  11.824  1.00 43.15           N  \nATOM    480  CA  ILE D  26      21.068  32.012  12.684  1.00 24.08           C  \nATOM    481  C   ILE D  26      21.700  31.770  14.063  1.00 33.06           C  \nATOM    482  O   ILE D  26      21.023  31.692  15.091  1.00 25.60           O  \nATOM    483  CB  ILE D  26      20.703  30.633  11.986  1.00 10.64           C  \nATOM    484  CG1 ILE D  26      20.543  30.870  10.489  1.00 13.59           C  \nATOM    485  CG2 ILE D  26      19.394  30.064  12.497  1.00 12.95           C  \nATOM    486  CD1 ILE D  26      20.408  29.633   9.619  1.00 11.30           C  \nATOM    487  N   LYS D  27      23.017  31.666  14.078  1.00 32.26           N  \nATOM    488  CA  LYS D  27      23.723  31.455  15.319  1.00 33.32           C  \nATOM    489  C   LYS D  27      23.531  32.641  16.254  1.00 57.04           C  \nATOM    490  O   LYS D  27      22.956  32.490  17.326  1.00 72.37           O  \nATOM    491  CB  LYS D  27      25.204  31.365  15.011  1.00 38.49           C  \nATOM    492  CG  LYS D  27      25.649  30.099  14.299  1.00 35.76           C  \nATOM    493  CD  LYS D  27      27.169  29.920  14.372  1.00 21.42           C  \nATOM    494  CE  LYS D  27      27.734  30.361  15.723  1.00 29.46           C  \nATOM    495  NZ  LYS D  27      26.819  30.054  16.874  1.00 21.55           N  \nATOM    496  N   LYS D  28      23.983  33.816  15.824  1.00 46.74           N  \nATOM    497  CA  LYS D  28      23.876  35.027  16.636  1.00 33.16           C  \nATOM    498  C   LYS D  28      22.443  35.541  16.778  1.00 22.54           C  \nATOM    499  O   LYS D  28      21.994  35.801  17.890  1.00 14.05           O  \nATOM    500  CB  LYS D  28      24.866  36.109  16.203  1.00 29.46           C  \nATOM    501  CG  LYS D  28      25.492  36.901  17.350  1.00 87.85           C  \nATOM    502  CD  LYS D  28      25.012  38.336  17.493  1.00 35.26           C  \nATOM    503  CE  LYS D  28      26.122  39.065  18.235  1.00 31.46           C  \nATOM    504  NZ  LYS D  28      26.044  38.862  19.714  1.00 31.10           N  \nATOM    505  N   GLU D  29      21.711  35.703  15.675  1.00 17.63           N  \nATOM    506  CA  GLU D  29      20.345  36.205  15.795  1.00 21.88           C  \nATOM    507  C   GLU D  29      19.485  35.373  16.740  1.00 32.26           C  \nATOM    508  O   GLU D  29      18.502  35.871  17.292  1.00 27.54           O  \nATOM    509  CB  GLU D  29      19.657  36.811  14.589  1.00 28.79           C  \nATOM    510  CG  GLU D  29      18.797  38.025  15.017  1.00 78.50           C  \nATOM    511  CD  GLU D  29      19.617  39.329  15.361  1.00 92.42           C  \nATOM    512  OE1 GLU D  29      20.534  39.675  14.541  1.00 79.59           O  \nATOM    513  OE2 GLU D  29      19.299  40.029  16.399  1.00 11.34           O  \nATOM    514  N   PHE D  30      19.884  34.109  16.933  1.00 25.04           N  \nATOM    515  CA  PHE D  30      19.203  33.169  17.819  1.00 28.94           C  \nATOM    516  C   PHE D  30      19.943  31.865  17.961  1.00 25.26           C  \nATOM    517  O   PHE D  30      21.071  31.839  18.438  1.00 25.98           O  \nATOM    518  CB  PHE D  30      17.717  32.861  17.659  1.00 14.53           C  \nATOM    519  CG  PHE D  30      17.213  32.802  16.240  1.00 26.84           C  \nATOM    520  CD1 PHE D  30      18.068  32.792  15.157  1.00 21.01           C  \nATOM    521  CD2 PHE D  30      15.831  32.721  16.023  1.00 21.64           C  \nATOM    522  CE1 PHE D  30      17.553  32.767  13.873  1.00 15.98           C  \nATOM    523  CE2 PHE D  30      15.306  32.697  14.744  1.00 17.50           C  \nATOM    524  CZ  PHE D  30      16.166  32.714  13.653  1.00 17.29           C  \nATOM    525  N   SER D  31      19.260  30.791  17.553  1.00 36.53           N  \nATOM    526  CA  SER D  31      19.709  29.387  17.616  1.00 40.85           C  \nATOM    527  C   SER D  31      21.212  29.150  17.493  1.00 37.13           C  \nATOM    528  O   SER D  31      22.019  30.066  17.309  1.00 47.40           O  \nATOM    529  CB  SER D  31      18.988  28.491  16.580  1.00 40.85           C  \nATOM    530  OG  SER D  31      19.702  28.348  15.348  1.00 13.80           O  \nATOM    531  N   ILE D  32      21.558  27.875  17.601  1.00 11.14           N  \nATOM    532  CA  ILE D  32      22.933  27.416  17.502  1.00 12.37           C  \nATOM    533  C   ILE D  32      22.938  25.980  17.016  1.00 18.19           C  \nATOM    534  O   ILE D  32      23.693  25.629  16.122  1.00 14.83           O  \nATOM    535  CB  ILE D  32      23.728  27.408  18.852  1.00 13.56           C  \nATOM    536  CG1 ILE D  32      25.058  26.664  18.651  1.00 14.35           C  \nATOM    537  CG2 ILE D  32      22.981  26.571  19.898  1.00 25.91           C  \nATOM    538  CD1 ILE D  32      26.385  27.373  18.861  1.00 11.34           C  \nATOM    539  N   SER D  33      22.044  25.155  17.577  1.00 44.46           N  \nATOM    540  CA  SER D  33      21.918  23.734  17.201  1.00 49.81           C  \nATOM    541  C   SER D  33      21.723  23.545  15.695  1.00 59.75           C  \nATOM    542  O   SER D  33      21.717  22.416  15.182  1.00 26.49           O  \nATOM    543  CB  SER D  33      20.873  22.962  17.995  1.00 32.94           C  \nATOM    544  OG  SER D  33      19.644  23.035  17.321  1.00 10.99           O  \nATOM    545  N   PHE D  34      21.551  24.665  15.007  1.00 77.72           N  \nATOM    546  CA  PHE D  34      21.415  24.708  13.562  1.00 82.79           C  \nATOM    547  C   PHE D  34      22.843  24.729  13.012  1.00 97.00           C  \nATOM    548  O   PHE D  34      23.107  24.297  11.874  1.00 99.54           O  \nATOM    549  CB  PHE D  34      20.656  26.007  13.213  1.00 80.49           C  \nATOM    550  CG  PHE D  34      19.793  25.899  11.991  1.00 71.84           C  \nATOM    551  CD1 PHE D  34      19.806  26.894  11.003  1.00 60.48           C  \nATOM    552  CD2 PHE D  34      19.010  24.795  11.797  1.00 68.54           C  \nATOM    553  CE1 PHE D  34      19.004  26.807   9.874  1.00 47.84           C  \nATOM    554  CE2 PHE D  34      18.220  24.697  10.656  1.00 74.05           C  \nATOM    555  CZ  PHE D  34      18.213  25.709   9.691  1.00 58.94           C  \nATOM    556  N   GLU D  35      23.759  25.175  13.857  1.00 86.84           N  \nATOM    557  CA  GLU D  35      25.171  25.229  13.523  1.00 82.03           C  \nATOM    558  C   GLU D  35      25.685  23.800  13.627  1.00 66.72           C  \nATOM    559  O   GLU D  35      26.728  23.450  13.082  1.00 64.54           O  \nATOM    560  CB  GLU D  35      25.934  26.139  14.507  1.00 83.60           C  \nATOM    561  CG  GLU D  35      27.448  26.205  14.309  1.00 50.06           C  \nATOM    562  CD  GLU D  35      27.856  27.066  13.109  1.00 57.79           C  \nATOM    563  OE1 GLU D  35      26.952  27.420  12.283  1.00 14.50           O  \nATOM    564  OE2 GLU D  35      29.086  27.338  12.987  1.00 15.78           O  \nATOM    565  N   GLU D  36      24.930  22.986  14.341  1.00 59.45           N  \nATOM    566  CA  GLU D  36      25.269  21.585  14.520  1.00 62.06           C  \nATOM    567  C   GLU D  36      24.325  20.678  13.737  1.00 79.73           C  \nATOM    568  O   GLU D  36      24.724  19.602  13.286  1.00 76.12           O  \nATOM    569  CB  GLU D  36      25.284  21.190  15.975  1.00 58.58           C  \nATOM    570  CG  GLU D  36      26.677  20.886  16.456  1.00 61.00           C  \nATOM    571  CD  GLU D  36      27.390  22.103  16.983  1.00 82.93           C  \nATOM    572  OE1 GLU D  36      27.403  23.126  16.256  1.00100.00           O  \nATOM    573  OE2 GLU D  36      27.920  22.047  18.134  1.00 81.24           O  \nATOM    574  N   PHE D  37      23.076  21.119  13.597  1.00 82.84           N  \nATOM    575  CA  PHE D  37      22.070  20.360  12.863  1.00 83.25           C  \nATOM    576  C   PHE D  37      22.618  19.954  11.494  1.00 88.82           C  \nATOM    577  O   PHE D  37      22.232  18.924  10.939  1.00 83.67           O  \nATOM    578  CB  PHE D  37      20.771  21.198  12.752  1.00 82.61           C  \nATOM    579  CG  PHE D  37      19.494  20.424  13.052  1.00 83.56           C  \nATOM    580  CD1 PHE D  37      19.398  19.591  14.167  1.00 83.07           C  \nATOM    581  CD2 PHE D  37      18.367  20.552  12.235  1.00 83.98           C  \nATOM    582  CE1 PHE D  37      18.229  18.886  14.447  1.00 78.36           C  \nATOM    583  CE2 PHE D  37      17.190  19.852  12.516  1.00 79.19           C  \nATOM    584  CZ  PHE D  37      17.120  19.030  13.629  1.00 74.99           C  \nATOM    585  N   ALA D  38      23.538  20.763  10.967  1.00 88.83           N  \nATOM    586  CA  ALA D  38      24.161  20.490   9.673  1.00 84.07           C  \nATOM    587  C   ALA D  38      25.353  19.556   9.844  1.00 76.44           C  \nATOM    588  O   ALA D  38      25.420  18.497   9.225  1.00 79.66           O  \nATOM    589  CB  ALA D  38      24.565  21.771   8.955  1.00 83.02           C  \nATOM    590  N   VAL D  39      26.294  19.959  10.691  1.00 65.88           N  \nATOM    591  CA  VAL D  39      27.466  19.141  10.950  1.00 67.34           C  \nATOM    592  C   VAL D  39      26.998  17.825  11.549  1.00 79.90           C  \nATOM    593  O   VAL D  39      27.676  16.800  11.462  1.00 84.94           O  \nATOM    594  CB  VAL D  39      28.450  19.837  11.923  1.00 79.92           C  \nATOM    595  CG1 VAL D  39      27.743  20.344  13.173  1.00 78.49           C  \nATOM    596  CG2 VAL D  39      29.576  18.899  12.308  1.00 84.03           C  \nATOM    597  N   LEU D  40      25.814  17.864  12.154  1.00 80.83           N  \nATOM    598  CA  LEU D  40      25.250  16.664  12.751  1.00 79.04           C  \nATOM    599  C   LEU D  40      25.071  15.590  11.682  1.00100.00           C  \nATOM    600  O   LEU D  40      25.162  14.394  11.969  1.00100.00           O  \nATOM    601  CB  LEU D  40      23.917  16.931  13.474  1.00 69.50           C  \nATOM    602  CG  LEU D  40      22.633  16.623  12.698  1.00 58.16           C  \nATOM    603  CD1 LEU D  40      22.308  15.140  12.687  1.00 54.42           C  \nATOM    604  CD2 LEU D  40      21.485  17.392  13.281  1.00 51.63           C  \nATOM    605  N   THR D  41      24.805  16.020  10.447  1.00 98.83           N  \nATOM    606  CA  THR D  41      24.627  15.072   9.354  1.00 98.20           C  \nATOM    607  C   THR D  41      25.850  14.168   9.268  1.00 98.99           C  \nATOM    608  O   THR D  41      25.740  12.981   8.949  1.00100.00           O  \nATOM    609  CB  THR D  41      24.298  15.734   7.985  1.00 97.46           C  \nATOM    610  OG1 THR D  41      25.283  16.722   7.667  1.00 71.69           O  \nATOM    611  CG2 THR D  41      22.911  16.361   7.988  1.00 97.15           C  \nATOM    612  N   TYR D  42      27.019  14.733   9.576  1.00 76.01           N  \nATOM    613  CA  TYR D  42      28.254  13.962   9.549  1.00 69.12           C  \nATOM    614  C   TYR D  42      28.637  13.408  10.923  1.00 63.64           C  \nATOM    615  O   TYR D  42      29.557  12.599  11.031  1.00 69.03           O  \nATOM    616  CB  TYR D  42      29.416  14.507   8.687  1.00 69.79           C  \nATOM    617  CG  TYR D  42      30.558  13.498   8.589  1.00 70.99           C  \nATOM    618  CD1 TYR D  42      30.470  12.397   7.759  1.00 69.13           C  \nATOM    619  CD2 TYR D  42      31.651  13.569   9.457  1.00 78.61           C  \nATOM    620  CE1 TYR D  42      31.465  11.433   7.743  1.00 78.99           C  \nATOM    621  CE2 TYR D  42      32.646  12.606   9.460  1.00 80.69           C  \nATOM    622  CZ  TYR D  42      32.552  11.544   8.599  1.00 85.80           C  \nATOM    623  OH  TYR D  42      33.568  10.613   8.597  1.00 67.99           O  \nATOM    624  N   ILE D  43      27.929  13.816  11.975  1.00 60.75           N  \nATOM    625  CA  ILE D  43      28.251  13.318  13.317  1.00 67.90           C  \nATOM    626  C   ILE D  43      27.045  13.044  14.222  1.00 76.61           C  \nATOM    627  O   ILE D  43      25.917  13.409  13.905  1.00 72.74           O  \nATOM    628  CB  ILE D  43      29.795  13.170  13.755  1.00 69.11           C  \nATOM    629  CG1 ILE D  43      30.034  12.904  15.247  1.00 68.62           C  \nATOM    630  CG2 ILE D  43      30.619  14.339  13.248  1.00 68.16           C  \nATOM    631  CD1 ILE D  43      29.901  11.446  15.644  1.00 56.39           C  \nATOM    632  N   SER D  44      27.294  12.375  15.343  1.00 84.03           N  \nATOM    633  CA  SER D  44      26.259  12.038  16.312  1.00 92.70           C  \nATOM    634  C   SER D  44      26.436  12.827  17.608  1.00100.00           C  \nATOM    635  O   SER D  44      25.461  13.155  18.293  1.00100.00           O  \nATOM    636  CB  SER D  44      26.458  10.577  16.702  1.00100.00           C  \nATOM    637  OG  SER D  44      27.716  10.431  17.354  1.00 99.00           O  \nATOM    638  N   GLU D  45      27.689  13.104  17.948  1.00 90.74           N  \nATOM    639  CA  GLU D  45      28.001  13.833  19.171  1.00 87.33           C  \nATOM    640  C   GLU D  45      27.202  15.118  19.257  1.00 86.18           C  \nATOM    641  O   GLU D  45      26.916  15.615  20.347  1.00 86.48           O  \nATOM    642  CB  GLU D  45      29.493  14.066  19.374  1.00 88.38           C  \nATOM    643  CG  GLU D  45      30.057  13.487  20.684  1.00100.00           C  \nATOM    644  CD  GLU D  45      29.181  13.669  21.954  1.00100.00           C  \nATOM    645  OE1 GLU D  45      27.957  13.389  21.924  1.00 74.39           O  \nATOM    646  OE2 GLU D  45      29.744  14.013  23.027  1.00100.00           O  \nATOM    647  N   ASN D  46      26.840  15.641  18.096  1.00 78.61           N  \nATOM    648  CA  ASN D  46      26.060  16.862  18.019  1.00 77.72           C  \nATOM    649  C   ASN D  46      24.602  16.576  18.378  1.00 78.20           C  \nATOM    650  O   ASN D  46      23.757  17.473  18.382  1.00 80.92           O  \nATOM    651  CB  ASN D  46      26.326  17.542  16.679  1.00 79.27           C  \nATOM    652  CG  ASN D  46      27.836  17.441  16.245  1.00 87.16           C  \nATOM    653  OD1 ASN D  46      28.101  17.102  15.090  1.00 99.73           O  \nATOM    654  ND2 ASN D  46      28.810  17.768  17.151  1.00  4.35           N  \nATOM    655  N   LYS D  47      24.325  15.309  18.690  1.00 60.36           N  \nATOM    656  CA  LYS D  47      22.984  14.879  19.056  1.00 52.15           C  \nATOM    657  C   LYS D  47      22.671  15.080  20.495  1.00 32.91           C  \nATOM    658  O   LYS D  47      21.531  15.248  20.898  1.00 14.59           O  \nATOM    659  CB  LYS D  47      22.703  13.439  18.636  1.00 53.94           C  \nATOM    660  CG  LYS D  47      21.677  13.333  17.520  1.00 13.56           C  \nATOM    661  CD  LYS D  47      20.385  12.671  17.978  1.00 10.08           C  \nATOM    662  CE  LYS D  47      19.283  12.743  16.930  1.00 21.30           C  \nATOM    663  NZ  LYS D  47      19.803  12.798  15.558  1.00 21.60           N  \nATOM    664  N   GLU D  48      23.560  15.113  21.438  1.00 38.48           N  \nATOM    665  CA  GLU D  48      22.907  15.339  22.704  1.00 43.15           C  \nATOM    666  C   GLU D  48      22.851  16.837  23.090  1.00 40.20           C  \nATOM    667  O   GLU D  48      21.824  17.324  23.531  1.00 38.57           O  \nATOM    668  CB  GLU D  48      23.414  14.598  23.854  1.00 47.51           C  \nATOM    669  CG  GLU D  48      22.335  14.643  24.962  1.00 66.82           C  \nATOM    670  CD  GLU D  48      22.861  15.405  26.136  1.00 78.73           C  \nATOM    671  OE1 GLU D  48      22.645  14.965  27.315  1.00 96.93           O  \nATOM    672  OE2 GLU D  48      23.556  16.471  25.939  1.00 54.65           O  \nATOM    673  N   LYS D  49      23.910  17.619  22.967  1.00 63.39           N  \nATOM    674  CA  LYS D  49      23.784  19.117  23.291  1.00 65.33           C  \nATOM    675  C   LYS D  49      22.505  19.615  22.645  1.00 65.95           C  \nATOM    676  O   LYS D  49      21.937  20.610  23.078  1.00 68.59           O  \nATOM    677  CB  LYS D  49      24.961  19.839  22.626  1.00 65.61           C  \nATOM    678  CG  LYS D  49      25.707  20.820  23.503  1.00 97.58           C  \nATOM    679  CD  LYS D  49      27.186  20.897  23.107  1.00100.00           C  \nATOM    680  CE  LYS D  49      27.793  22.297  23.193  1.00 63.86           C  \nATOM    681  NZ  LYS D  49      28.804  22.568  22.157  1.00 36.43           N  \nATOM    682  N   GLU D  50      22.074  18.896  21.611  1.00 58.72           N  \nATOM    683  CA  GLU D  50      20.871  19.232  20.863  1.00 56.68           C  \nATOM    684  C   GLU D  50      19.617  19.072  21.709  1.00 57.11           C  \nATOM    685  O   GLU D  50      18.916  20.025  21.963  1.00 45.98           O  \nATOM    686  CB  GLU D  50      20.824  18.239  19.681  1.00 57.24           C  \nATOM    687  CG  GLU D  50      20.466  18.788  18.319  1.00 23.49           C  \nATOM    688  CD  GLU D  50      19.007  18.874  18.127  1.00 42.96           C  \nATOM    689  OE1 GLU D  50      18.476  18.111  17.270  1.00 40.80           O  \nATOM    690  OE2 GLU D  50      18.400  19.690  18.855  1.00 63.54           O  \nATOM    691  N   TYR D  51      19.356  17.842  22.148  1.00 59.94           N  \nATOM    692  CA  TYR D  51      18.169  17.565  22.941  1.00 53.22           C  \nATOM    693  C   TYR D  51      18.307  17.877  24.421  1.00 30.25           C  \nATOM    694  O   TYR D  51      17.330  18.228  25.067  1.00  8.63           O  \nATOM    695  CB  TYR D  51      17.330  16.399  22.474  1.00 59.72           C  \nATOM    696  CG  TYR D  51      16.133  16.826  21.648  1.00 61.96           C  \nATOM    697  CD1 TYR D  51      14.885  17.034  22.257  1.00 46.73           C  \nATOM    698  CD2 TYR D  51      16.228  17.021  20.266  1.00 67.80           C  \nATOM    699  CE1 TYR D  51      13.778  17.383  21.520  1.00 51.61           C  \nATOM    700  CE2 TYR D  51      15.110  17.403  19.531  1.00 62.66           C  \nATOM    701  CZ  TYR D  51      13.901  17.573  20.177  1.00 51.70           C  \nATOM    702  OH  TYR D  51      12.828  17.922  19.428  1.00 38.50           O  \nATOM    703  N   TYR D  52      19.507  17.756  24.973  1.00 50.74           N  \nATOM    704  CA  TYR D  52      19.679  18.067  26.389  1.00 45.85           C  \nATOM    705  C   TYR D  52      19.285  19.525  26.649  1.00 35.16           C  \nATOM    706  O   TYR D  52      19.085  19.942  27.788  1.00 30.71           O  \nATOM    707  CB  TYR D  52      21.096  17.681  26.887  1.00 34.61           C  \nATOM    708  CG  TYR D  52      21.305  17.785  28.390  1.00 38.40           C  \nATOM    709  CD1 TYR D  52      22.583  17.776  28.961  1.00 46.98           C  \nATOM    710  CD2 TYR D  52      20.204  17.868  29.261  1.00 29.74           C  \nATOM    711  CE1 TYR D  52      22.718  17.893  30.341  1.00 66.43           C  \nATOM    712  CE2 TYR D  52      20.351  18.031  30.623  1.00 11.61           C  \nATOM    713  CZ  TYR D  52      21.580  18.026  31.171  1.00 20.02           C  \nATOM    714  OH  TYR D  52      21.664  18.108  32.534  1.00 18.04           O  \nATOM    715  N   LEU D  53      19.133  20.299  25.579  1.00 25.70           N  \nATOM    716  CA  LEU D  53      18.771  21.705  25.742  1.00 27.86           C  \nATOM    717  C   LEU D  53      17.546  22.108  24.943  1.00 40.71           C  \nATOM    718  O   LEU D  53      16.719  22.882  25.421  1.00 51.21           O  \nATOM    719  CB  LEU D  53      19.870  22.640  25.176  1.00 22.56           C  \nATOM    720  CG  LEU D  53      20.639  23.444  26.208  1.00 19.76           C  \nATOM    721  CD1 LEU D  53      19.918  23.289  27.563  1.00  6.19           C  \nATOM    722  CD2 LEU D  53      22.164  23.115  26.233  1.00 12.00           C  \nATOM    723  N   LYS D  54      17.437  21.600  23.725  1.00 28.33           N  \nATOM    724  CA  LYS D  54      16.333  21.950  22.840  1.00 17.50           C  \nATOM    725  C   LYS D  54      14.950  21.765  23.479  1.00 15.77           C  \nATOM    726  O   LYS D  54      13.937  22.180  22.902  1.00 13.83           O  \nATOM    727  CB  LYS D  54      16.674  21.545  21.375  1.00 18.55           C  \nATOM    728  CG  LYS D  54      15.624  21.598  20.310  1.00 28.79           C  \nATOM    729  CD  LYS D  54      16.291  21.526  18.942  1.00 21.20           C  \nATOM    730  CE  LYS D  54      15.290  21.874  17.848  1.00 10.07           C  \nATOM    731  NZ  LYS D  54      15.879  22.443  16.596  1.00 39.38           N  \nATOM    732  N   ASP D  55      14.931  21.180  24.690  1.00 24.97           N  \nATOM    733  CA  ASP D  55      13.712  20.934  25.463  1.00 23.96           C  \nATOM    734  C   ASP D  55      13.935  21.215  26.959  1.00 25.85           C  \nATOM    735  O   ASP D  55      13.200  20.748  27.831  1.00 23.88           O  \nATOM    736  CB  ASP D  55      13.373  19.412  25.275  1.00 34.95           C  \nATOM    737  CG  ASP D  55      11.983  19.008  25.825  1.00 71.34           C  \nATOM    738  OD1 ASP D  55      11.440  19.577  26.809  1.00 63.59           O  \nATOM    739  OD2 ASP D  55      11.426  18.021  25.313  1.00 80.55           O  \nATOM    740  N   ILE D  56      14.969  21.983  27.273  1.00 30.47           N  \nATOM    741  CA  ILE D  56      15.277  22.276  28.661  1.00 34.21           C  \nATOM    742  C   ILE D  56      15.971  23.642  28.828  1.00 57.56           C  \nATOM    743  O   ILE D  56      16.773  23.812  29.745  1.00 75.97           O  \nATOM    744  CB  ILE D  56      16.240  21.103  29.181  1.00 36.27           C  \nATOM    745  CG1 ILE D  56      15.684  19.687  28.856  1.00 48.68           C  \nATOM    746  CG2 ILE D  56      16.570  21.178  30.664  1.00 15.06           C  \nATOM    747  CD1 ILE D  56      16.523  18.820  27.906  1.00 29.03           C  \nATOM    748  N   VAL D  67       1.980  17.944  18.936  1.00 30.03           N  \nATOM    749  CA  VAL D  67       1.134  19.118  18.499  1.00 27.98           C  \nATOM    750  C   VAL D  67       1.701  20.590  18.789  1.00 49.65           C  \nATOM    751  O   VAL D  67       1.937  21.390  17.859  1.00 54.64           O  \nATOM    752  CB  VAL D  67      -0.219  18.964  19.178  1.00 42.40           C  \nATOM    753  CG1 VAL D  67      -0.107  18.901  20.702  1.00 43.63           C  \nATOM    754  CG2 VAL D  67      -1.196  20.082  18.864  1.00 48.36           C  \nATOM    755  N   VAL D  68       1.929  21.001  20.067  1.00 63.40           N  \nATOM    756  CA  VAL D  68       2.411  22.417  20.332  1.00 74.01           C  \nATOM    757  C   VAL D  68       3.910  22.565  20.052  1.00 73.94           C  \nATOM    758  O   VAL D  68       4.349  23.583  19.505  1.00 75.86           O  \nATOM    759  CB  VAL D  68       2.071  22.904  21.735  1.00 85.88           C  \nATOM    760  CG1 VAL D  68       3.040  23.960  22.254  1.00 85.76           C  \nATOM    761  CG2 VAL D  68       0.676  23.575  21.786  1.00 86.27           C  \nATOM    762  N   LYS D  69       4.698  21.551  20.412  1.00 57.07           N  \nATOM    763  CA  LYS D  69       6.123  21.591  20.148  1.00 52.12           C  \nATOM    764  C   LYS D  69       6.361  21.479  18.651  1.00 44.63           C  \nATOM    765  O   LYS D  69       7.488  21.534  18.177  1.00 50.53           O  \nATOM    766  CB  LYS D  69       6.866  20.617  21.055  1.00 62.44           C  \nATOM    767  CG  LYS D  69       7.060  19.180  20.530  1.00 98.36           C  \nATOM    768  CD  LYS D  69       8.182  19.180  19.500  1.00 84.80           C  \nATOM    769  CE  LYS D  69       9.565  19.241  20.179  1.00 67.00           C  \nATOM    770  NZ  LYS D  69      10.265  20.553  19.929  1.00 51.21           N  \nATOM    771  N   ALA D  70       5.248  21.372  17.917  1.00 32.08           N  \nATOM    772  CA  ALA D  70       5.243  21.280  16.469  1.00 39.57           C  \nATOM    773  C   ALA D  70       5.021  22.666  15.865  1.00 66.71           C  \nATOM    774  O   ALA D  70       5.801  23.110  15.020  1.00 83.65           O  \nATOM    775  CB  ALA D  70       4.259  20.248  15.945  1.00 39.81           C  \nATOM    776  N   VAL D  71       3.962  23.356  16.294  1.00 52.98           N  \nATOM    777  CA  VAL D  71       3.711  24.701  15.787  1.00 46.36           C  \nATOM    778  C   VAL D  71       4.823  25.594  16.299  1.00 23.24           C  \nATOM    779  O   VAL D  71       5.311  26.477  15.598  1.00 28.64           O  \nATOM    780  CB  VAL D  71       2.406  25.407  16.271  1.00 51.75           C  \nATOM    781  CG1 VAL D  71       2.654  26.386  17.439  1.00 47.04           C  \nATOM    782  CG2 VAL D  71       1.797  26.190  15.116  1.00 52.06           C  \nATOM    783  N   LYS D  72       5.230  25.333  17.534  1.00 16.34           N  \nATOM    784  CA  LYS D  72       6.300  26.100  18.140  1.00 23.55           C  \nATOM    785  C   LYS D  72       7.539  26.024  17.251  1.00 32.55           C  \nATOM    786  O   LYS D  72       8.210  27.031  17.016  1.00 51.87           O  \nATOM    787  CB  LYS D  72       6.582  25.661  19.609  1.00 26.76           C  \nATOM    788  CG  LYS D  72       7.915  26.141  20.214  1.00 43.56           C  \nATOM    789  CD  LYS D  72       7.961  27.637  20.580  1.00 14.69           C  \nATOM    790  CE  LYS D  72       8.878  27.917  21.789  1.00 59.51           C  \nATOM    791  NZ  LYS D  72       8.722  27.085  23.047  1.00 11.47           N  \nATOM    792  N   ILE D  73       7.827  24.825  16.748  1.00 16.40           N  \nATOM    793  CA  ILE D  73       8.976  24.629  15.878  1.00 13.28           C  \nATOM    794  C   ILE D  73       8.717  25.236  14.515  1.00 32.23           C  \nATOM    795  O   ILE D  73       9.553  25.973  13.992  1.00 57.72           O  \nATOM    796  CB  ILE D  73       9.278  23.173  15.591  1.00 15.24           C  \nATOM    797  CG1 ILE D  73      10.509  22.732  16.337  1.00 16.58           C  \nATOM    798  CG2 ILE D  73       9.525  22.901  14.125  1.00 11.20           C  \nATOM    799  CD1 ILE D  73      10.172  21.765  17.429  1.00 12.04           C  \nATOM    800  N   LEU D  74       7.552  24.937  13.943  1.00 13.96           N  \nATOM    801  CA  LEU D  74       7.230  25.479  12.623  1.00 17.54           C  \nATOM    802  C   LEU D  74       7.260  27.000  12.636  1.00 54.50           C  \nATOM    803  O   LEU D  74       7.792  27.637  11.732  1.00 73.01           O  \nATOM    804  CB  LEU D  74       5.914  25.012  12.036  1.00 20.29           C  \nATOM    805  CG  LEU D  74       5.885  25.179  10.506  1.00 26.09           C  \nATOM    806  CD1 LEU D  74       4.993  26.313  10.048  1.00 19.88           C  \nATOM    807  CD2 LEU D  74       7.302  25.440   9.999  1.00 41.90           C  \nATOM    808  N   SER D  75       6.666  27.592  13.663  1.00 48.69           N  \nATOM    809  CA  SER D  75       6.661  29.038  13.771  1.00 40.46           C  \nATOM    810  C   SER D  75       8.077  29.572  13.891  1.00 48.37           C  \nATOM    811  O   SER D  75       8.464  30.482  13.158  1.00 67.15           O  \nATOM    812  CB  SER D  75       5.933  29.487  15.008  1.00 22.94           C  \nATOM    813  OG  SER D  75       6.297  30.838  15.191  1.00 13.33           O  \nATOM    814  N   GLN D  76       8.851  29.001  14.815  1.00 11.89           N  \nATOM    815  CA  GLN D  76      10.222  29.452  14.999  1.00 15.93           C  \nATOM    816  C   GLN D  76      11.086  29.052  13.811  1.00 18.28           C  \nATOM    817  O   GLN D  76      12.244  29.451  13.706  1.00 18.36           O  \nATOM    818  CB  GLN D  76      10.841  29.081  16.342  1.00 11.30           C  \nATOM    819  CG  GLN D  76      11.973  28.010  16.243  1.00 45.80           C  \nATOM    820  CD  GLN D  76      13.412  28.584  16.139  1.00 55.73           C  \nATOM    821  OE1 GLN D  76      14.426  27.836  15.985  1.00 36.67           O  \nATOM    822  NE2 GLN D  76      13.509  29.910  16.262  1.00 50.21           N  \nATOM    823  N   GLU D  77      10.519  28.253  12.923  1.00 12.96           N  \nATOM    824  CA  GLU D  77      11.246  27.827  11.745  1.00 22.93           C  \nATOM    825  C   GLU D  77      10.933  28.807  10.631  1.00 36.80           C  \nATOM    826  O   GLU D  77      11.606  28.851   9.607  1.00 66.17           O  \nATOM    827  CB  GLU D  77      11.030  26.339  11.424  1.00 29.83           C  \nATOM    828  CG  GLU D  77      12.307  25.552  11.174  1.00 16.16           C  \nATOM    829  CD  GLU D  77      12.525  25.244   9.698  1.00 59.85           C  \nATOM    830  OE1 GLU D  77      11.712  24.476   9.148  1.00100.00           O  \nATOM    831  OE2 GLU D  77      13.448  25.812   9.081  1.00100.00           O  \nATOM    832  N   ASP D  78       9.893  29.603  10.863  1.00 16.24           N  \nATOM    833  CA  ASP D  78       9.473  30.642   9.920  1.00 24.27           C  \nATOM    834  C   ASP D  78      10.198  31.922  10.296  1.00 30.01           C  \nATOM    835  O   ASP D  78      10.537  32.741   9.430  1.00 32.81           O  \nATOM    836  CB  ASP D  78       7.991  30.991  10.038  1.00 31.56           C  \nATOM    837  CG  ASP D  78       7.571  32.155   9.088  1.00 64.99           C  \nATOM    838  OD1 ASP D  78       8.095  33.299   9.134  1.00 17.65           O  \nATOM    839  OD2 ASP D  78       6.700  31.929   8.186  1.00100.00           O  \nATOM    840  N   TYR D  79      10.430  32.079  11.591  1.00 38.89           N  \nATOM    841  CA  TYR D  79      11.132  33.234  12.097  1.00 30.23           C  \nATOM    842  C   TYR D  79      12.456  33.364  11.372  1.00 48.48           C  \nATOM    843  O   TYR D  79      12.903  34.471  11.080  1.00 56.48           O  \nATOM    844  CB  TYR D  79      11.388  33.118  13.601  1.00 27.02           C  \nATOM    845  CG  TYR D  79      10.286  33.721  14.389  1.00 27.13           C  \nATOM    846  CD1 TYR D  79       9.828  33.122  15.519  1.00 24.34           C  \nATOM    847  CD2 TYR D  79       9.661  34.865  13.958  1.00 43.73           C  \nATOM    848  CE1 TYR D  79       8.793  33.687  16.239  1.00 32.44           C  \nATOM    849  CE2 TYR D  79       8.628  35.430  14.663  1.00 50.00           C  \nATOM    850  CZ  TYR D  79       8.197  34.826  15.802  1.00 53.60           C  \nATOM    851  OH  TYR D  79       7.177  35.372  16.522  1.00100.00           O  \nATOM    852  N   PHE D  80      13.082  32.230  11.076  1.00 50.36           N  \nATOM    853  CA  PHE D  80      14.348  32.264  10.366  1.00 58.96           C  \nATOM    854  C   PHE D  80      14.182  32.231   8.847  1.00 52.00           C  \nATOM    855  O   PHE D  80      15.021  32.753   8.113  1.00 46.74           O  \nATOM    856  CB  PHE D  80      15.502  31.433  10.971  1.00 68.16           C  \nATOM    857  CG  PHE D  80      15.128  30.027  11.354  1.00 83.81           C  \nATOM    858  CD1 PHE D  80      15.411  29.536  12.629  1.00 95.99           C  \nATOM    859  CD2 PHE D  80      14.511  29.180  10.441  1.00 94.21           C  \nATOM    860  CE1 PHE D  80      15.068  28.236  12.987  1.00100.00           C  \nATOM    861  CE2 PHE D  80      14.165  27.880  10.801  1.00 99.09           C  \nATOM    862  CZ  PHE D  80      14.450  27.412  12.073  1.00 96.17           C  \nATOM    863  N   ASP D  81      13.096  31.628   8.372  1.00 46.40           N  \nATOM    864  CA  ASP D  81      12.860  31.528   6.930  1.00 44.54           C  \nATOM    865  C   ASP D  81      12.593  32.863   6.229  1.00 35.59           C  \nATOM    866  O   ASP D  81      12.732  32.947   5.006  1.00 21.53           O  \nATOM    867  CB  ASP D  81      12.111  30.280   6.467  1.00 39.56           C  \nATOM    868  CG  ASP D  81      13.026  29.065   6.336  1.00 34.66           C  \nATOM    869  OD1 ASP D  81      13.121  28.291   7.325  1.00 36.47           O  \nATOM    870  OD2 ASP D  81      13.580  28.847   5.218  1.00 10.28           O  \nATOM    871  N   LYS D  82      12.207  33.889   6.998  1.00 30.23           N  \nATOM    872  CA  LYS D  82      11.940  35.222   6.438  1.00 30.52           C  \nATOM    873  C   LYS D  82      13.180  36.093   6.518  1.00 25.21           C  \nATOM    874  O   LYS D  82      13.497  36.862   5.596  1.00 15.58           O  \nATOM    875  CB  LYS D  82      10.809  36.038   7.068  1.00 24.92           C  \nATOM    876  CG  LYS D  82      10.403  37.192   6.119  1.00 11.00           C  \nATOM    877  CD  LYS D  82       9.241  38.005   6.644  1.00 60.52           C  \nATOM    878  CE  LYS D  82       8.069  37.114   7.011  1.00 94.47           C  \nATOM    879  NZ  LYS D  82       7.070  37.839   7.840  1.00100.00           N  \nATOM    880  N   LYS D  83      13.879  35.958   7.634  1.00 24.05           N  \nATOM    881  CA  LYS D  83      15.100  36.705   7.859  1.00 21.00           C  \nATOM    882  C   LYS D  83      16.177  36.417   6.796  1.00 26.34           C  \nATOM    883  O   LYS D  83      17.036  37.270   6.567  1.00 30.91           O  \nATOM    884  CB  LYS D  83      15.495  36.420   9.289  1.00 41.78           C  \nATOM    885  CG  LYS D  83      16.816  36.942   9.818  1.00 13.18           C  \nATOM    886  CD  LYS D  83      16.591  37.106  11.341  1.00 21.30           C  \nATOM    887  CE  LYS D  83      17.820  37.555  12.068  1.00 21.04           C  \nATOM    888  NZ  LYS D  83      18.355  38.757  11.391  1.00 57.65           N  \nATOM    889  N   ARG D  84      16.143  35.228   6.154  1.00 22.97           N  \nATOM    890  CA  ARG D  84      17.134  34.899   5.121  1.00 15.25           C  \nATOM    891  C   ARG D  84      16.640  35.193   3.727  1.00 15.74           C  \nATOM    892  O   ARG D  84      17.406  35.646   2.864  1.00 33.90           O  \nATOM    893  CB  ARG D  84      17.945  33.590   5.227  1.00 32.42           C  \nATOM    894  CG  ARG D  84      18.847  33.302   3.980  1.00 34.50           C  \nATOM    895  CD  ARG D  84      18.049  32.743   2.740  1.00 46.95           C  \nATOM    896  NE  ARG D  84      18.931  32.443   1.608  1.00 59.76           N  \nATOM    897  CZ  ARG D  84      18.763  31.473   0.690  1.00 31.02           C  \nATOM    898  NH1 ARG D  84      17.702  30.642   0.671  1.00 21.00           N  \nATOM    899  NH2 ARG D  84      19.678  31.357  -0.253  1.00 59.72           N  \nATOM    900  N   ASN D  85      15.364  34.944   3.489  1.00 19.01           N  \nATOM    901  CA  ASN D  85      14.803  35.255   2.197  1.00 19.71           C  \nATOM    902  C   ASN D  85      14.908  36.774   2.086  1.00 39.39           C  \nATOM    903  O   ASN D  85      15.146  37.337   1.016  1.00 41.11           O  \nATOM    904  CB  ASN D  85      13.339  34.792   2.184  1.00 21.30           C  \nATOM    905  CG  ASN D  85      12.786  34.522   0.791  1.00 24.46           C  \nATOM    906  OD1 ASN D  85      11.618  34.876   0.482  1.00 18.51           O  \nATOM    907  ND2 ASN D  85      13.522  33.742   0.016  1.00 27.86           N  \nATOM    908  N   GLU D  86      14.757  37.439   3.224  1.00 48.65           N  \nATOM    909  CA  GLU D  86      14.852  38.888   3.242  1.00 61.43           C  \nATOM    910  C   GLU D  86      16.288  39.334   3.017  1.00 85.11           C  \nATOM    911  O   GLU D  86      16.564  40.211   2.198  1.00 99.04           O  \nATOM    912  CB  GLU D  86      14.227  39.544   4.522  1.00 62.56           C  \nATOM    913  CG  GLU D  86      14.973  39.585   5.879  1.00 41.20           C  \nATOM    914  CD  GLU D  86      14.212  40.509   6.895  1.00 56.66           C  \nATOM    915  OE1 GLU D  86      13.123  41.014   6.510  1.00100.00           O  \nATOM    916  OE2 GLU D  86      14.657  40.743   8.059  1.00 71.00           O  \nATOM    917  N   HIS D  87      17.205  38.721   3.755  1.00 77.80           N  \nATOM    918  CA  HIS D  87      18.612  39.072   3.644  1.00 75.23           C  \nATOM    919  C   HIS D  87      19.202  38.748   2.291  1.00 62.76           C  \nATOM    920  O   HIS D  87      20.301  39.201   1.978  1.00 46.19           O  \nATOM    921  CB  HIS D  87      19.517  38.755   4.851  1.00 81.68           C  \nATOM    922  CG  HIS D  87      20.451  37.605   4.649  1.00 93.79           C  \nATOM    923  ND1 HIS D  87      21.821  37.732   4.759  1.00100.00           N  \nATOM    924  CD2 HIS D  87      20.215  36.296   4.394  1.00100.00           C  \nATOM    925  CE1 HIS D  87      22.393  36.554   4.567  1.00100.00           C  \nATOM    926  NE2 HIS D  87      21.439  35.665   4.347  1.00100.00           N  \nATOM    927  N   ASP D  88      18.468  37.968   1.498  1.00 74.06           N  \nATOM    928  CA  ASP D  88      18.927  37.589   0.165  1.00 84.79           C  \nATOM    929  C   ASP D  88      19.067  38.802  -0.755  1.00 83.97           C  \nATOM    930  O   ASP D  88      19.457  38.695  -1.925  1.00 89.61           O  \nATOM    931  CB  ASP D  88      18.498  36.217  -0.368  1.00 99.05           C  \nATOM    932  CG  ASP D  88      19.565  35.138  -0.093  1.00 72.31           C  \nATOM    933  OD1 ASP D  88      20.620  35.438   0.507  1.00 10.32           O  \nATOM    934  OD2 ASP D  88      19.367  33.934  -0.648  1.00 45.42           O  \nATOM    935  N   GLU D  89      18.724  39.959  -0.193  1.00 38.56           N  \nATOM    936  CA  GLU D  89      18.813  41.239  -0.861  1.00 12.33           C  \nATOM    937  C   GLU D  89      19.918  42.051  -0.194  1.00 39.86           C  \nATOM    938  O   GLU D  89      20.476  42.973  -0.788  1.00 55.69           O  \nATOM    939  CB  GLU D  89      17.479  41.929  -0.900  1.00 51.30           C  \nATOM    940  CG  GLU D  89      16.411  41.106  -1.584  1.00 41.57           C  \nATOM    941  CD  GLU D  89      16.759  40.708  -3.026  1.00 92.93           C  \nATOM    942  OE1 GLU D  89      16.022  41.102  -3.965  1.00100.00           O  \nATOM    943  OE2 GLU D  89      17.707  39.908  -3.196  1.00 84.46           O  \nATOM    944  N   ARG D  90      20.232  41.708   1.056  1.00 30.25           N  \nATOM    945  CA  ARG D  90      21.302  42.413   1.753  1.00 36.34           C  \nATOM    946  C   ARG D  90      22.623  41.738   1.479  1.00 63.03           C  \nATOM    947  O   ARG D  90      23.482  42.297   0.797  1.00 65.53           O  \nATOM    948  CB  ARG D  90      21.135  42.785   3.194  1.00 35.54           C  \nATOM    949  CG  ARG D  90      21.098  41.700   4.151  1.00 67.09           C  \nATOM    950  CD  ARG D  90      19.705  41.723   4.502  1.00100.00           C  \nATOM    951  NE  ARG D  90      19.449  41.881   5.910  1.00 78.81           N  \nATOM    952  CZ  ARG D  90      18.319  41.498   6.449  1.00 94.72           C  \nATOM    953  NH1 ARG D  90      17.429  40.911   5.657  1.00100.00           N  \nATOM    954  NH2 ARG D  90      18.098  41.656   7.750  1.00 41.62           N  \nATOM    955  N   THR D  91      22.773  40.509   1.963  1.00 70.24           N  \nATOM    956  CA  THR D  91      24.002  39.776   1.705  1.00 70.69           C  \nATOM    957  C   THR D  91      24.231  39.675   0.197  1.00 63.89           C  \nATOM    958  O   THR D  91      25.338  39.353  -0.252  1.00 57.85           O  \nATOM    959  CB  THR D  91      24.196  38.416   2.496  1.00 88.98           C  \nATOM    960  OG1 THR D  91      24.171  38.661   3.917  1.00 58.89           O  \nATOM    961  CG2 THR D  91      25.527  37.767   2.116  1.00 87.03           C  \nATOM    962  N   VAL D  92      23.184  39.978  -0.590  1.00 57.06           N  \nATOM    963  CA  VAL D  92      23.245  39.967  -2.045  1.00 64.37           C  \nATOM    964  C   VAL D  92      24.352  40.884  -2.595  1.00 72.24           C  \nATOM    965  O   VAL D  92      24.454  41.077  -3.804  1.00 81.89           O  \nATOM    966  CB  VAL D  92      21.895  39.735  -2.826  1.00 70.94           C  \nATOM    967  CG1 VAL D  92      21.685  40.736  -3.936  1.00 72.25           C  \nATOM    968  CG2 VAL D  92      21.808  38.298  -3.349  1.00 70.49           C  \nATOM    969  N   LEU D  93      25.174  41.458  -1.709  1.00 59.00           N  \nATOM    970  CA  LEU D  93      26.294  42.282  -2.133  1.00 54.78           C  \nATOM    971  C   LEU D  93      27.370  41.287  -2.556  1.00 52.68           C  \nATOM    972  O   LEU D  93      28.061  41.465  -3.560  1.00 41.47           O  \nATOM    973  CB  LEU D  93      26.814  43.254  -1.014  1.00 51.89           C  \nATOM    974  CG  LEU D  93      27.240  44.742  -1.334  1.00 41.99           C  \nATOM    975  CD1 LEU D  93      27.257  45.632  -0.074  1.00 37.63           C  \nATOM    976  CD2 LEU D  93      28.576  44.878  -2.029  1.00 33.50           C  \nATOM    977  N   ILE D  94      27.429  40.206  -1.778  1.00 48.93           N  \nATOM    978  CA  ILE D  94      28.343  39.094  -1.978  1.00 50.61           C  \nATOM    979  C   ILE D  94      28.057  38.391  -3.310  1.00 71.23           C  \nATOM    980  O   ILE D  94      28.983  38.107  -4.070  1.00 75.81           O  \nATOM    981  CB  ILE D  94      28.136  38.138  -0.764  1.00 42.93           C  \nATOM    982  CG1 ILE D  94      29.347  37.270  -0.490  1.00 29.27           C  \nATOM    983  CG2 ILE D  94      26.974  37.201  -0.961  1.00 56.80           C  \nATOM    984  CD1 ILE D  94      29.200  36.567   0.853  1.00 41.00           C  \nATOM    985  N   LEU D  95      26.767  38.133  -3.600  1.00 66.41           N  \nATOM    986  CA  LEU D  95      26.358  37.468  -4.842  1.00 54.61           C  \nATOM    987  C   LEU D  95      26.973  38.217  -6.006  1.00 54.05           C  \nATOM    988  O   LEU D  95      27.675  37.655  -6.854  1.00 24.13           O  \nATOM    989  CB  LEU D  95      24.814  37.493  -5.040  1.00 41.98           C  \nATOM    990  CG  LEU D  95      24.255  38.235  -6.285  1.00 15.77           C  \nATOM    991  CD1 LEU D  95      24.176  37.367  -7.493  1.00 11.00           C  \nATOM    992  CD2 LEU D  95      22.957  38.963  -6.015  1.00 12.36           C  \nATOM    993  N   VAL D  96      26.720  39.512  -6.019  1.00 68.59           N  \nATOM    994  CA  VAL D  96      27.281  40.344  -7.042  1.00 69.73           C  \nATOM    995  C   VAL D  96      28.758  40.497  -6.740  1.00 45.97           C  \nATOM    996  O   VAL D  96      29.565  40.484  -7.658  1.00 54.37           O  \nATOM    997  CB  VAL D  96      26.567  41.689  -7.232  1.00 79.13           C  \nATOM    998  CG1 VAL D  96      27.340  42.531  -8.254  1.00 78.42           C  \nATOM    999  CG2 VAL D  96      25.114  41.462  -7.669  1.00 78.44           C  \nATOM   1000  N   ASN D  97      29.113  40.602  -5.452  1.00 18.56           N  \nATOM   1001  CA  ASN D  97      30.521  40.756  -5.072  1.00 29.53           C  \nATOM   1002  C   ASN D  97      31.390  39.629  -5.620  1.00 61.78           C  \nATOM   1003  O   ASN D  97      32.497  39.864  -6.114  1.00 63.43           O  \nATOM   1004  CB  ASN D  97      30.842  41.261  -3.660  1.00 47.00           C  \nATOM   1005  CG  ASN D  97      30.925  42.818  -3.580  1.00100.00           C  \nATOM   1006  OD1 ASN D  97      30.088  43.553  -4.156  1.00 33.33           O  \nATOM   1007  ND2 ASN D  97      32.005  43.314  -2.966  1.00100.00           N  \nATOM   1008  N   ALA D  98      30.873  38.404  -5.544  1.00 72.39           N  \nATOM   1009  CA  ALA D  98      31.582  37.240  -6.063  1.00 67.39           C  \nATOM   1010  C   ALA D  98      31.684  37.393  -7.577  1.00 86.88           C  \nATOM   1011  O   ALA D  98      32.773  37.551  -8.135  1.00 95.57           O  \nATOM   1012  CB  ALA D  98      30.797  35.977  -5.715  1.00 63.82           C  \nATOM   1013  N   GLN D  99      30.512  37.391  -8.210  1.00 76.08           N  \nATOM   1014  CA  GLN D  99      30.335  37.536  -9.647  1.00 63.45           C  \nATOM   1015  C   GLN D  99      30.966  38.828 -10.155  1.00 46.21           C  \nATOM   1016  O   GLN D  99      31.334  38.929 -11.322  1.00 48.78           O  \nATOM   1017  CB  GLN D  99      28.859  37.841  -9.698  1.00 60.88           C  \nATOM   1018  CG  GLN D  99      28.019  37.431 -10.811  1.00 45.79           C  \nATOM   1019  CD  GLN D  99      26.622  37.455 -10.278  1.00 40.33           C  \nATOM   1020  OE1 GLN D  99      25.642  37.253 -10.991  1.00 19.73           O  \nATOM   1021  NE2 GLN D  99      26.531  37.747  -8.962  1.00 21.01           N  \nATOM   1022  N   GLN D 100      31.069  39.811  -9.264  1.00 29.75           N  \nATOM   1023  CA  GLN D 100      31.625  41.125  -9.585  1.00 35.80           C  \nATOM   1024  C   GLN D 100      32.981  41.064 -10.267  1.00 27.57           C  \nATOM   1025  O   GLN D 100      33.161  40.390 -11.285  1.00 30.37           O  \nATOM   1026  CB  GLN D 100      31.708  42.109  -8.390  1.00 41.42           C  \nATOM   1027  CG  GLN D 100      30.460  43.055  -8.298  1.00100.00           C  \nATOM   1028  CD  GLN D 100      30.271  43.770  -6.938  1.00 69.00           C  \nATOM   1029  OE1 GLN D 100      31.206  43.920  -6.135  1.00 21.79           O  \nATOM   1030  NE2 GLN D 100      29.012  44.129  -6.635  1.00 29.22           N  \nATOM   1031  N   ARG D 101      33.926  41.804  -9.688  1.00 10.53           N  \nATOM   1032  CA  ARG D 101      35.288  41.894 -10.205  1.00 22.91           C  \nATOM   1033  C   ARG D 101      35.869  40.541 -10.596  1.00 49.30           C  \nATOM   1034  O   ARG D 101      36.856  40.492 -11.324  1.00 45.72           O  \nATOM   1035  CB  ARG D 101      36.250  42.689  -9.320  1.00 35.86           C  \nATOM   1036  CG  ARG D 101      36.072  44.206  -9.404  1.00 49.59           C  \nATOM   1037  CD  ARG D 101      37.434  44.829  -9.351  1.00 31.30           C  \nATOM   1038  NE  ARG D 101      38.403  43.803  -9.056  1.00 39.84           N  \nATOM   1039  CZ  ARG D 101      39.113  43.741  -7.944  1.00 89.29           C  \nATOM   1040  NH1 ARG D 101      38.986  44.687  -7.035  1.00 78.55           N  \nATOM   1041  NH2 ARG D 101      39.972  42.748  -7.757  1.00100.00           N  \nATOM   1042  N   LYS D 102      35.268  39.442 -10.103  1.00 36.45           N  \nATOM   1043  CA  LYS D 102      35.774  38.120 -10.453  1.00 20.23           C  \nATOM   1044  C   LYS D 102      35.515  37.840 -11.917  1.00 31.82           C  \nATOM   1045  O   LYS D 102      36.442  37.562 -12.677  1.00 37.02           O  \nATOM   1046  CB  LYS D 102      35.759  36.986  -9.458  1.00 31.52           C  \nATOM   1047  CG  LYS D 102      34.608  36.017  -9.700  1.00 36.00           C  \nATOM   1048  CD  LYS D 102      34.804  34.736  -8.886  1.00 37.05           C  \nATOM   1049  CE  LYS D 102      35.190  35.083  -7.439  1.00 36.37           C  \nATOM   1050  NZ  LYS D 102      34.142  35.773  -6.604  1.00 25.94           N  \nATOM   1051  N   LYS D 103      34.261  37.933 -12.333  1.00 31.96           N  \nATOM   1052  CA  LYS D 103      33.983  37.737 -13.742  1.00 30.62           C  \nATOM   1053  C   LYS D 103      34.636  38.935 -14.408  1.00 44.34           C  \nATOM   1054  O   LYS D 103      35.197  38.837 -15.495  1.00 53.70           O  \nATOM   1055  CB  LYS D 103      32.494  37.647 -14.014  1.00 28.04           C  \nATOM   1056  CG  LYS D 103      32.109  36.711 -15.142  1.00 19.30           C  \nATOM   1057  CD  LYS D 103      30.631  36.384 -14.979  1.00 22.11           C  \nATOM   1058  CE  LYS D 103      29.879  36.404 -16.307  1.00 61.54           C  \nATOM   1059  NZ  LYS D 103      28.415  36.724 -16.164  1.00 15.12           N  \nATOM   1060  N   ILE D 104      34.581  40.070 -13.703  1.00 32.38           N  \nATOM   1061  CA  ILE D 104      35.212  41.307 -14.167  1.00 24.87           C  \nATOM   1062  C   ILE D 104      36.696  41.045 -14.283  1.00 16.87           C  \nATOM   1063  O   ILE D 104      37.431  41.707 -15.019  1.00 20.02           O  \nATOM   1064  CB  ILE D 104      34.977  42.457 -13.238  1.00 26.38           C  \nATOM   1065  CG1 ILE D 104      33.916  43.320 -13.870  1.00 37.57           C  \nATOM   1066  CG2 ILE D 104      36.260  43.253 -13.005  1.00 37.03           C  \nATOM   1067  CD1 ILE D 104      33.420  42.677 -15.126  1.00 21.20           C  \nATOM   1068  N   GLU D 105      37.122  40.031 -13.550  1.00 20.01           N  \nATOM   1069  CA  GLU D 105      38.501  39.626 -13.607  1.00 27.33           C  \nATOM   1070  C   GLU D 105      38.664  38.914 -14.931  1.00 30.78           C  \nATOM   1071  O   GLU D 105      39.682  39.052 -15.609  1.00 42.03           O  \nATOM   1072  CB  GLU D 105      38.953  38.755 -12.422  1.00 26.31           C  \nATOM   1073  CG  GLU D 105      40.389  38.246 -12.478  1.00 52.50           C  \nATOM   1074  CD  GLU D 105      41.412  39.331 -12.852  1.00 75.76           C  \nATOM   1075  OE1 GLU D 105      41.228  39.962 -13.923  1.00 41.20           O  \nATOM   1076  OE2 GLU D 105      42.479  39.406 -12.171  1.00 32.66           O  \nATOM   1077  N   SER D 106      37.631  38.193 -15.305  1.00 12.30           N  \nATOM   1078  CA  SER D 106      37.615  37.476 -16.562  1.00 18.13           C  \nATOM   1079  C   SER D 106      37.508  38.457 -17.734  1.00 33.93           C  \nATOM   1080  O   SER D 106      38.264  38.326 -18.696  1.00 41.80           O  \nATOM   1081  CB  SER D 106      36.497  36.536 -16.399  1.00 12.11           C  \nATOM   1082  OG  SER D 106      36.215  36.634 -15.001  1.00 28.62           O  \nATOM   1083  N   LEU D 107      36.601  39.455 -17.658  1.00 34.68           N  \nATOM   1084  CA  LEU D 107      36.475  40.443 -18.743  1.00 27.87           C  \nATOM   1085  C   LEU D 107      37.826  41.075 -18.983  1.00 22.71           C  \nATOM   1086  O   LEU D 107      38.081  41.606 -20.070  1.00 17.30           O  \nATOM   1087  CB  LEU D 107      35.535  41.639 -18.516  1.00 30.23           C  \nATOM   1088  CG  LEU D 107      35.737  42.983 -19.367  1.00 35.79           C  \nATOM   1089  CD1 LEU D 107      36.917  43.924 -19.005  1.00 27.69           C  \nATOM   1090  CD2 LEU D 107      35.459  42.929 -20.876  1.00 14.14           C  \nATOM   1091  N   LEU D 108      38.673  41.021 -17.947  1.00 17.54           N  \nATOM   1092  CA  LEU D 108      40.024  41.554 -18.043  1.00 16.68           C  \nATOM   1093  C   LEU D 108      40.659  40.804 -19.188  1.00 25.85           C  \nATOM   1094  O   LEU D 108      41.491  41.328 -19.933  1.00 28.23           O  \nATOM   1095  CB  LEU D 108      40.831  41.293 -16.757  1.00 16.11           C  \nATOM   1096  CG  LEU D 108      42.354  41.376 -16.848  1.00 16.91           C  \nATOM   1097  CD1 LEU D 108      42.678  42.590 -17.740  1.00 21.76           C  \nATOM   1098  CD2 LEU D 108      43.018  41.498 -15.452  1.00 31.00           C  \nATOM   1099  N   SER D 109      40.214  39.553 -19.311  1.00 18.07           N  \nATOM   1100  CA  SER D 109      40.660  38.668 -20.370  1.00 30.28           C  \nATOM   1101  C   SER D 109      40.018  39.110 -21.679  1.00 48.29           C  \nATOM   1102  O   SER D 109      40.624  38.963 -22.744  1.00 39.94           O  \nATOM   1103  CB  SER D 109      40.390  37.184 -20.076  1.00 62.18           C  \nATOM   1104  OG  SER D 109      40.364  36.952 -18.660  1.00 96.10           O  \nATOM   1105  N   ARG D 110      38.801  39.679 -21.596  1.00 59.58           N  \nATOM   1106  CA  ARG D 110      38.121  40.167 -22.798  1.00 57.10           C  \nATOM   1107  C   ARG D 110      39.045  41.161 -23.468  1.00 55.78           C  \nATOM   1108  O   ARG D 110      39.377  41.014 -24.645  1.00 60.43           O  \nATOM   1109  CB  ARG D 110      36.782  40.859 -22.602  1.00 55.45           C  \nATOM   1110  CG  ARG D 110      36.226  41.414 -23.999  1.00 27.07           C  \nATOM   1111  CD  ARG D 110      34.769  41.987 -23.986  1.00 14.96           C  \nATOM   1112  NE  ARG D 110      34.049  41.795 -25.255  1.00 17.13           N  \nATOM   1113  CZ  ARG D 110      32.974  41.009 -25.406  1.00 57.64           C  \nATOM   1114  NH1 ARG D 110      32.467  40.315 -24.358  1.00 51.20           N  \nATOM   1115  NH2 ARG D 110      32.408  40.911 -26.614  1.00 26.75           N  \nATOM   1116  N   VAL D 111      39.485  42.152 -22.692  1.00 36.68           N  \nATOM   1117  CA  VAL D 111      40.430  43.131 -23.204  1.00 39.12           C  \nATOM   1118  C   VAL D 111      41.772  42.425 -23.357  1.00 37.23           C  \nATOM   1119  O   VAL D 111      42.304  42.271 -24.461  1.00 32.61           O  \nATOM   1120  CB  VAL D 111      40.533  44.414 -22.338  1.00 60.34           C  \nATOM   1121  CG1 VAL D 111      40.093  45.636 -23.141  1.00 59.81           C  \nATOM   1122  CG2 VAL D 111      39.716  44.278 -21.039  1.00 64.68           C  \nTER    1123      VAL D 111                                                      \nATOM   1124  N   ALA B   2       4.032  11.547  16.164  1.00 11.69           N  \nATOM   1125  CA  ALA B   2       5.470  11.718  15.971  1.00 13.61           C  \nATOM   1126  C   ALA B   2       5.795  12.625  14.779  1.00 25.77           C  \nATOM   1127  O   ALA B   2       4.952  12.842  13.899  1.00 22.09           O  \nATOM   1128  CB  ALA B   2       6.183  10.375  15.881  1.00 14.30           C  \nATOM   1129  N   ILE B   3       7.040  13.132  14.763  1.00 28.37           N  \nATOM   1130  CA  ILE B   3       7.558  14.042  13.727  1.00 24.48           C  \nATOM   1131  C   ILE B   3       7.102  13.735  12.305  1.00 46.01           C  \nATOM   1132  O   ILE B   3       6.376  12.775  12.039  1.00 54.62           O  \nATOM   1133  CB  ILE B   3       9.163  14.197  13.638  1.00 29.73           C  \nATOM   1134  CG1 ILE B   3       9.819  13.307  12.596  1.00 51.40           C  \nATOM   1135  CG2 ILE B   3       9.859  13.952  14.947  1.00 52.01           C  \nATOM   1136  CD1 ILE B   3      10.295  14.035  11.351  1.00 58.42           C  \nATOM   1137  N   THR B   4       7.553  14.587  11.401  1.00 48.27           N  \nATOM   1138  CA  THR B   4       7.267  14.490   9.980  1.00 58.04           C  \nATOM   1139  C   THR B   4       8.337  15.301   9.265  1.00 85.48           C  \nATOM   1140  O   THR B   4       9.185  14.758   8.556  1.00 91.74           O  \nATOM   1141  CB  THR B   4       5.894  15.168   9.594  1.00 53.92           C  \nATOM   1142  OG1 THR B   4       4.986  15.146  10.717  1.00 69.37           O  \nATOM   1143  CG2 THR B   4       5.279  14.524   8.317  1.00 61.23           C  \nATOM   1144  N   LYS B   5       8.298  16.613   9.497  1.00 81.67           N  \nATOM   1145  CA  LYS B   5       9.259  17.542   8.915  1.00 71.51           C  \nATOM   1146  C   LYS B   5      10.105  18.241   9.973  1.00 67.04           C  \nATOM   1147  O   LYS B   5      10.866  19.158   9.657  1.00 77.23           O  \nATOM   1148  CB  LYS B   5       8.651  18.485   7.899  1.00 56.51           C  \nATOM   1149  CG  LYS B   5       8.664  17.833   6.547  1.00 14.06           C  \nATOM   1150  CD  LYS B   5      10.090  17.460   6.132  1.00 41.74           C  \nATOM   1151  CE  LYS B   5      10.155  16.833   4.721  1.00 65.16           C  \nATOM   1152  NZ  LYS B   5       9.325  17.455   3.619  1.00 32.65           N  \nATOM   1153  N   ILE B   6       9.970  17.830  11.230  1.00 37.91           N  \nATOM   1154  CA  ILE B   6      10.777  18.466  12.252  1.00 30.14           C  \nATOM   1155  C   ILE B   6      12.231  18.191  11.938  1.00 33.55           C  \nATOM   1156  O   ILE B   6      12.843  17.297  12.524  1.00 41.23           O  \nATOM   1157  CB  ILE B   6      10.342  18.241  13.720  1.00 32.84           C  \nATOM   1158  CG1 ILE B   6      10.692  16.868  14.236  1.00 29.83           C  \nATOM   1159  CG2 ILE B   6       8.921  18.733  13.934  1.00 38.79           C  \nATOM   1160  CD1 ILE B   6      11.898  16.929  15.116  1.00 31.02           C  \nATOM   1161  N   ASN B   7      12.747  18.949  10.989  1.00 42.76           N  \nATOM   1162  CA  ASN B   7      14.118  18.817  10.517  1.00 49.37           C  \nATOM   1163  C   ASN B   7      14.494  20.070   9.725  1.00 54.77           C  \nATOM   1164  O   ASN B   7      14.902  20.004   8.561  1.00 40.95           O  \nATOM   1165  CB  ASN B   7      14.276  17.474   9.711  1.00 48.22           C  \nATOM   1166  CG  ASN B   7      15.262  16.411  10.369  1.00 15.82           C  \nATOM   1167  OD1 ASN B   7      15.235  16.087  11.565  1.00 21.10           O  \nATOM   1168  ND2 ASN B   7      16.108  15.842   9.507  1.00 12.79           N  \nATOM   1169  N   ASP B   8      14.315  21.219  10.387  1.00 45.53           N  \nATOM   1170  CA  ASP B   8      14.619  22.531   9.836  1.00 31.96           C  \nATOM   1171  C   ASP B   8      16.068  22.566   9.370  1.00 43.12           C  \nATOM   1172  O   ASP B   8      16.960  22.176  10.117  1.00 44.60           O  \nATOM   1173  CB  ASP B   8      14.608  23.382  11.092  1.00 30.29           C  \nATOM   1174  CG  ASP B   8      15.269  22.648  12.317  1.00 37.98           C  \nATOM   1175  OD1 ASP B   8      14.815  21.545  12.793  1.00 21.00           O  \nATOM   1176  OD2 ASP B   8      16.206  23.241  12.898  1.00100.00           O  \nATOM   1177  N   CYS B   9      16.330  23.018   8.150  1.00 48.90           N  \nATOM   1178  CA  CYS B   9      15.328  23.490   7.201  1.00 55.42           C  \nATOM   1179  C   CYS B   9      15.646  22.866   5.832  1.00 81.57           C  \nATOM   1180  O   CYS B   9      16.472  21.955   5.770  1.00 94.20           O  \nATOM   1181  CB  CYS B   9      15.434  24.989   7.086  1.00 51.13           C  \nATOM   1182  SG  CYS B   9      16.802  25.280   6.016  1.00 54.79           S  \nATOM   1183  N   PHE B  10      15.004  23.317   4.756  1.00 69.37           N  \nATOM   1184  CA  PHE B  10      15.257  22.707   3.449  1.00 64.62           C  \nATOM   1185  C   PHE B  10      16.541  23.211   2.764  1.00 65.07           C  \nATOM   1186  O   PHE B  10      17.614  22.627   2.937  1.00 74.62           O  \nATOM   1187  CB  PHE B  10      14.003  22.606   2.583  1.00 66.61           C  \nATOM   1188  CG  PHE B  10      12.708  22.735   3.362  1.00 70.07           C  \nATOM   1189  CD1 PHE B  10      12.505  22.043   4.555  1.00 79.03           C  \nATOM   1190  CD2 PHE B  10      11.688  23.556   2.900  1.00 73.09           C  \nATOM   1191  CE1 PHE B  10      11.337  22.183   5.274  1.00 79.46           C  \nATOM   1192  CE2 PHE B  10      10.497  23.683   3.622  1.00 80.18           C  \nATOM   1193  CZ  PHE B  10      10.328  22.995   4.806  1.00 79.90           C  \nATOM   1194  N   GLU B  11      16.420  24.285   1.981  1.00 56.82           N  \nATOM   1195  CA  GLU B  11      17.534  24.881   1.233  1.00 50.86           C  \nATOM   1196  C   GLU B  11      18.839  24.966   2.045  1.00 39.22           C  \nATOM   1197  O   GLU B  11      19.877  24.413   1.660  1.00 12.48           O  \nATOM   1198  CB  GLU B  11      17.124  26.275   0.672  1.00 48.83           C  \nATOM   1199  CG  GLU B  11      15.713  26.713   1.121  1.00 91.67           C  \nATOM   1200  CD  GLU B  11      15.516  28.190   1.495  1.00 84.15           C  \nATOM   1201  OE1 GLU B  11      15.535  28.493   2.716  1.00 74.98           O  \nATOM   1202  OE2 GLU B  11      15.141  29.021   0.622  1.00 28.32           O  \nATOM   1203  N   LEU B  12      18.805  25.651   3.176  1.00 41.37           N  \nATOM   1204  CA  LEU B  12      19.992  25.775   4.012  1.00 28.50           C  \nATOM   1205  C   LEU B  12      20.506  24.393   4.421  1.00 30.46           C  \nATOM   1206  O   LEU B  12      21.708  24.135   4.430  1.00 17.10           O  \nATOM   1207  CB  LEU B  12      19.645  26.641   5.238  1.00 12.06           C  \nATOM   1208  CG  LEU B  12      20.834  27.275   5.939  1.00 14.16           C  \nATOM   1209  CD1 LEU B  12      20.718  27.185   7.448  1.00 11.20           C  \nATOM   1210  CD2 LEU B  12      22.150  26.695   5.476  1.00 17.01           C  \nATOM   1211  N   LEU B  13      19.583  23.490   4.720  1.00 36.24           N  \nATOM   1212  CA  LEU B  13      19.940  22.139   5.130  1.00 45.51           C  \nATOM   1213  C   LEU B  13      20.946  21.397   4.247  1.00 65.55           C  \nATOM   1214  O   LEU B  13      22.150  21.470   4.496  1.00 74.48           O  \nATOM   1215  CB  LEU B  13      19.201  21.437   6.256  1.00 50.32           C  \nATOM   1216  CG  LEU B  13      19.909  21.567   7.607  1.00 68.18           C  \nATOM   1217  CD1 LEU B  13      19.475  20.460   8.580  1.00 72.91           C  \nATOM   1218  CD2 LEU B  13      21.432  21.604   7.403  1.00 76.13           C  \nATOM   1219  N   SER B  14      20.456  20.671   3.228  1.00 48.37           N  \nATOM   1220  CA  SER B  14      21.339  19.900   2.347  1.00 49.15           C  \nATOM   1221  C   SER B  14      22.545  20.679   1.846  1.00 55.71           C  \nATOM   1222  O   SER B  14      23.594  20.082   1.609  1.00 62.47           O  \nATOM   1223  CB  SER B  14      20.734  19.188   1.145  1.00 57.46           C  \nATOM   1224  OG  SER B  14      21.738  19.036   0.101  1.00 29.61           O  \nATOM   1225  N   MET B  15      22.420  21.988   1.667  1.00 46.47           N  \nATOM   1226  CA  MET B  15      23.549  22.781   1.187  1.00 32.59           C  \nATOM   1227  C   MET B  15      24.758  22.575   2.095  1.00 22.20           C  \nATOM   1228  O   MET B  15      25.805  22.080   1.668  1.00 19.28           O  \nATOM   1229  CB  MET B  15      23.173  24.264   1.138  1.00 24.04           C  \nATOM   1230  CG  MET B  15      24.166  25.018   0.310  1.00 19.94           C  \nATOM   1231  SD  MET B  15      24.253  24.075  -1.215  1.00 28.13           S  \nATOM   1232  CE  MET B  15      22.518  23.709  -1.476  1.00 29.92           C  \nATOM   1233  N   VAL B  16      24.587  22.944   3.356  1.00 21.12           N  \nATOM   1234  CA  VAL B  16      25.636  22.797   4.333  1.00 22.20           C  \nATOM   1235  C   VAL B  16      26.166  21.368   4.342  1.00 44.21           C  \nATOM   1236  O   VAL B  16      27.376  21.166   4.259  1.00 64.79           O  \nATOM   1237  CB  VAL B  16      25.038  23.194   5.679  1.00 17.25           C  \nATOM   1238  CG1 VAL B  16      26.123  23.279   6.770  1.00 11.40           C  \nATOM   1239  CG2 VAL B  16      24.264  24.514   5.488  1.00 31.40           C  \nATOM   1240  N   THR B  17      25.253  20.381   4.418  1.00 36.12           N  \nATOM   1241  CA  THR B  17      25.621  18.953   4.430  1.00 21.67           C  \nATOM   1242  C   THR B  17      26.328  18.550   3.136  1.00 17.56           C  \nATOM   1243  O   THR B  17      27.423  17.990   3.145  1.00 20.73           O  \nATOM   1244  CB  THR B  17      24.460  18.023   4.401  1.00 31.20           C  \nATOM   1245  OG1 THR B  17      23.982  17.752   5.734  1.00  3.86           O  \nATOM   1246  CG2 THR B  17      24.982  16.744   3.743  1.00 19.53           C  \nATOM   1247  N   TYR B  18      25.688  18.833   2.016  1.00 10.68           N  \nATOM   1248  CA  TYR B  18      26.275  18.537   0.719  1.00 15.49           C  \nATOM   1249  C   TYR B  18      27.623  19.259   0.627  1.00 36.96           C  \nATOM   1250  O   TYR B  18      28.546  18.803  -0.041  1.00 32.74           O  \nATOM   1251  CB  TYR B  18      25.266  19.014  -0.356  1.00 18.64           C  \nATOM   1252  CG  TYR B  18      25.608  19.142  -1.823  1.00 15.22           C  \nATOM   1253  CD1 TYR B  18      24.588  19.259  -2.757  1.00 10.55           C  \nATOM   1254  CD2 TYR B  18      26.904  19.140  -2.280  1.00 22.62           C  \nATOM   1255  CE1 TYR B  18      24.872  19.415  -4.100  1.00 12.89           C  \nATOM   1256  CE2 TYR B  18      27.200  19.273  -3.625  1.00 28.81           C  \nATOM   1257  CZ  TYR B  18      26.188  19.376  -4.539  1.00 20.60           C  \nATOM   1258  OH  TYR B  18      26.486  19.568  -5.869  1.00 16.39           O  \nATOM   1259  N   ALA B  19      27.749  20.391   1.315  1.00 53.45           N  \nATOM   1260  CA  ALA B  19      29.009  21.138   1.303  1.00 47.87           C  \nATOM   1261  C   ALA B  19      29.986  20.521   2.298  1.00 52.23           C  \nATOM   1262  O   ALA B  19      31.133  20.215   1.960  1.00 58.56           O  \nATOM   1263  CB  ALA B  19      28.785  22.611   1.584  1.00 40.94           C  \nATOM   1264  N   ASP B  20      29.508  20.313   3.525  1.00 26.51           N  \nATOM   1265  CA  ASP B  20      30.309  19.723   4.586  1.00 16.55           C  \nATOM   1266  C   ASP B  20      30.829  18.355   4.162  1.00 35.79           C  \nATOM   1267  O   ASP B  20      31.668  17.770   4.847  1.00 32.95           O  \nATOM   1268  CB  ASP B  20      29.483  19.586   5.869  1.00  5.91           C  \nATOM   1269  CG  ASP B  20      29.309  20.917   6.620  1.00 46.79           C  \nATOM   1270  OD1 ASP B  20      30.047  21.202   7.604  1.00 33.30           O  \nATOM   1271  OD2 ASP B  20      28.411  21.701   6.238  1.00 85.92           O  \nATOM   1272  N   LYS B  21      30.312  17.854   3.036  1.00 46.46           N  \nATOM   1273  CA  LYS B  21      30.684  16.553   2.467  1.00 51.20           C  \nATOM   1274  C   LYS B  21      31.951  16.607   1.589  1.00 63.63           C  \nATOM   1275  O   LYS B  21      32.948  15.934   1.887  1.00 51.28           O  \nATOM   1276  CB  LYS B  21      29.479  15.920   1.732  1.00 34.95           C  \nATOM   1277  CG  LYS B  21      29.772  14.573   1.116  1.00  4.83           C  \nATOM   1278  CD  LYS B  21      30.852  13.840   1.932  1.00 59.30           C  \nATOM   1279  CE  LYS B  21      31.408  12.593   1.225  1.00 92.23           C  \nATOM   1280  NZ  LYS B  21      30.641  11.308   1.405  1.00 58.81           N  \nATOM   1281  N   LEU B  22      31.910  17.380   0.493  1.00 48.28           N  \nATOM   1282  CA  LEU B  22      33.068  17.499  -0.401  1.00 28.89           C  \nATOM   1283  C   LEU B  22      34.216  18.327   0.198  1.00 29.06           C  \nATOM   1284  O   LEU B  22      35.347  18.322  -0.301  1.00 12.54           O  \nATOM   1285  CB  LEU B  22      32.921  17.196  -1.879  1.00 19.50           C  \nATOM   1286  CG  LEU B  22      31.619  17.701  -2.502  1.00 20.81           C  \nATOM   1287  CD1 LEU B  22      31.839  18.906  -3.447  1.00 18.94           C  \nATOM   1288  CD2 LEU B  22      30.684  16.590  -3.050  1.00 11.34           C  \nATOM   1289  N   LYS B  23      33.907  19.032   1.281  1.00 10.78           N  \nATOM   1290  CA  LYS B  23      34.896  19.823   1.981  1.00 21.82           C  \nATOM   1291  C   LYS B  23      35.920  18.868   2.594  1.00 55.40           C  \nATOM   1292  O   LYS B  23      37.137  19.010   2.426  1.00 56.68           O  \nATOM   1293  CB  LYS B  23      34.156  20.413   3.151  1.00 51.20           C  \nATOM   1294  CG  LYS B  23      33.843  21.861   3.005  1.00 41.10           C  \nATOM   1295  CD  LYS B  23      33.300  22.326   4.330  1.00 16.05           C  \nATOM   1296  CE  LYS B  23      34.321  22.055   5.408  1.00 17.83           C  \nATOM   1297  NZ  LYS B  23      35.658  22.502   4.920  1.00 19.08           N  \nATOM   1298  N   SER B  24      35.385  17.872   3.290  1.00 41.56           N  \nATOM   1299  CA  SER B  24      36.160  16.841   3.953  1.00 29.01           C  \nATOM   1300  C   SER B  24      36.718  15.838   2.956  1.00 20.03           C  \nATOM   1301  O   SER B  24      37.859  15.402   3.090  1.00 18.91           O  \nATOM   1302  CB  SER B  24      35.163  16.072   4.779  1.00 30.80           C  \nATOM   1303  OG  SER B  24      34.096  15.676   3.923  1.00 45.21           O  \nATOM   1304  N   LEU B  25      35.903  15.457   1.976  1.00 26.39           N  \nATOM   1305  CA  LEU B  25      36.336  14.484   0.974  1.00 30.43           C  \nATOM   1306  C   LEU B  25      37.662  14.887   0.354  1.00 39.59           C  \nATOM   1307  O   LEU B  25      38.551  14.061   0.175  1.00 45.63           O  \nATOM   1308  CB  LEU B  25      35.404  14.447  -0.210  1.00 31.72           C  \nATOM   1309  CG  LEU B  25      35.697  13.315  -1.188  1.00 29.25           C  \nATOM   1310  CD1 LEU B  25      34.478  12.449  -1.206  1.00 32.85           C  \nATOM   1311  CD2 LEU B  25      35.963  13.829  -2.577  1.00 18.18           C  \nATOM   1312  N   ILE B  26      37.761  16.167   0.008  1.00 30.85           N  \nATOM   1313  CA  ILE B  26      38.954  16.735  -0.587  1.00 19.41           C  \nATOM   1314  C   ILE B  26      40.011  16.915   0.497  1.00 23.63           C  \nATOM   1315  O   ILE B  26      41.208  16.972   0.236  1.00 30.48           O  \nATOM   1316  CB  ILE B  26      38.536  18.111  -1.252  1.00 22.82           C  \nATOM   1317  CG1 ILE B  26      37.973  17.878  -2.679  1.00 31.84           C  \nATOM   1318  CG2 ILE B  26      39.572  19.241  -1.134  1.00 12.24           C  \nATOM   1319  CD1 ILE B  26      36.482  17.574  -2.812  1.00 21.00           C  \nATOM   1320  N   LYS B  27      39.554  17.030   1.737  1.00 23.11           N  \nATOM   1321  CA  LYS B  27      40.450  17.218   2.874  1.00 28.94           C  \nATOM   1322  C   LYS B  27      41.402  16.033   3.075  1.00 27.33           C  \nATOM   1323  O   LYS B  27      42.624  16.193   3.071  1.00 14.93           O  \nATOM   1324  CB  LYS B  27      39.631  17.636   4.118  1.00 35.73           C  \nATOM   1325  CG  LYS B  27      40.410  18.367   5.211  1.00100.00           C  \nATOM   1326  CD  LYS B  27      39.522  19.448   5.849  1.00 86.34           C  \nATOM   1327  CE  LYS B  27      39.732  19.636   7.357  1.00 69.08           C  \nATOM   1328  NZ  LYS B  27      40.623  20.792   7.681  1.00 57.99           N  \nATOM   1329  N   LYS B  28      40.845  14.836   3.232  1.00 35.85           N  \nATOM   1330  CA  LYS B  28      41.643  13.624   3.433  1.00 38.60           C  \nATOM   1331  C   LYS B  28      42.460  13.264   2.200  1.00 33.85           C  \nATOM   1332  O   LYS B  28      43.672  13.063   2.282  1.00 27.90           O  \nATOM   1333  CB  LYS B  28      40.773  12.387   3.665  1.00 39.09           C  \nATOM   1334  CG  LYS B  28      39.387  12.608   4.193  1.00 49.33           C  \nATOM   1335  CD  LYS B  28      39.209  11.567   5.268  1.00 45.58           C  \nATOM   1336  CE  LYS B  28      40.417  11.637   6.199  1.00 31.00           C  \nATOM   1337  NZ  LYS B  28      40.166  11.065   7.569  1.00 27.16           N  \nATOM   1338  N   GLU B  29      41.774  13.182   1.062  1.00 38.79           N  \nATOM   1339  CA  GLU B  29      42.371  12.829  -0.225  1.00 46.16           C  \nATOM   1340  C   GLU B  29      43.520  13.760  -0.647  1.00 29.98           C  \nATOM   1341  O   GLU B  29      44.344  13.382  -1.474  1.00 23.51           O  \nATOM   1342  CB  GLU B  29      41.270  12.614  -1.293  1.00 50.92           C  \nATOM   1343  CG  GLU B  29      41.643  11.850  -2.573  1.00 84.36           C  \nATOM   1344  CD  GLU B  29      41.600  10.321  -2.452  1.00 93.70           C  \nATOM   1345  OE1 GLU B  29      40.493   9.684  -2.514  1.00 49.74           O  \nATOM   1346  OE2 GLU B  29      42.705   9.729  -2.424  1.00 23.88           O  \nATOM   1347  N   PHE B  30      43.579  14.959  -0.077  1.00 21.16           N  \nATOM   1348  CA  PHE B  30      44.652  15.891  -0.420  1.00 26.66           C  \nATOM   1349  C   PHE B  30      44.932  16.962   0.642  1.00 54.33           C  \nATOM   1350  O   PHE B  30      45.125  16.636   1.806  1.00 66.70           O  \nATOM   1351  CB  PHE B  30      44.917  16.238  -1.890  1.00 23.27           C  \nATOM   1352  CG  PHE B  30      43.695  16.176  -2.739  1.00 18.50           C  \nATOM   1353  CD1 PHE B  30      42.462  16.366  -2.187  1.00 24.19           C  \nATOM   1354  CD2 PHE B  30      43.776  15.944  -4.092  1.00 12.06           C  \nATOM   1355  CE1 PHE B  30      41.318  16.307  -2.955  1.00 32.00           C  \nATOM   1356  CE2 PHE B  30      42.623  15.872  -4.883  1.00  3.71           C  \nATOM   1357  CZ  PHE B  30      41.392  16.050  -4.310  1.00 12.34           C  \nATOM   1358  N   SER B  31      44.991  18.220   0.203  1.00 42.94           N  \nATOM   1359  CA  SER B  31      45.282  19.349   1.065  1.00 36.35           C  \nATOM   1360  C   SER B  31      44.438  19.380   2.334  1.00 47.37           C  \nATOM   1361  O   SER B  31      44.162  18.347   2.950  1.00 67.72           O  \nATOM   1362  CB  SER B  31      45.157  20.662   0.303  1.00 25.29           C  \nATOM   1363  OG  SER B  31      43.812  21.100   0.233  1.00 31.26           O  \nATOM   1364  N   ILE B  32      44.083  20.596   2.743  1.00 17.66           N  \nATOM   1365  CA  ILE B  32      43.271  20.836   3.932  1.00 10.87           C  \nATOM   1366  C   ILE B  32      42.898  22.295   4.041  1.00 28.66           C  \nATOM   1367  O   ILE B  32      41.976  22.647   4.747  1.00 28.07           O  \nATOM   1368  CB  ILE B  32      43.906  20.562   5.288  1.00 12.97           C  \nATOM   1369  CG1 ILE B  32      45.371  20.919   5.376  1.00 22.52           C  \nATOM   1370  CG2 ILE B  32      43.512  19.215   5.874  1.00 12.95           C  \nATOM   1371  CD1 ILE B  32      46.039  20.087   6.463  1.00 41.92           C  \nATOM   1372  N   SER B  33      43.617  23.143   3.311  1.00 55.08           N  \nATOM   1373  CA  SER B  33      43.319  24.570   3.339  1.00 62.03           C  \nATOM   1374  C   SER B  33      41.862  24.840   3.006  1.00 65.86           C  \nATOM   1375  O   SER B  33      41.342  25.915   3.280  1.00 50.01           O  \nATOM   1376  CB  SER B  33      44.358  25.441   2.558  1.00 59.19           C  \nATOM   1377  OG  SER B  33      44.425  25.179   1.160  1.00 14.19           O  \nATOM   1378  N   PHE B  34      41.199  23.829   2.450  1.00 70.72           N  \nATOM   1379  CA  PHE B  34      39.799  23.919   2.076  1.00 71.42           C  \nATOM   1380  C   PHE B  34      38.877  24.104   3.275  1.00 29.75           C  \nATOM   1381  O   PHE B  34      37.977  24.928   3.227  1.00 21.00           O  \nATOM   1382  CB  PHE B  34      39.389  22.715   1.284  1.00 86.07           C  \nATOM   1383  CG  PHE B  34      38.472  23.024   0.156  1.00 94.98           C  \nATOM   1384  CD1 PHE B  34      37.213  22.477   0.145  1.00 98.93           C  \nATOM   1385  CD2 PHE B  34      38.876  23.812  -0.900  1.00 98.62           C  \nATOM   1386  CE1 PHE B  34      36.364  22.699  -0.901  1.00 98.72           C  \nATOM   1387  CE2 PHE B  34      38.016  24.053  -1.967  1.00100.00           C  \nATOM   1388  CZ  PHE B  34      36.751  23.500  -1.948  1.00100.00           C  \nATOM   1389  N   GLU B  35      39.076  23.328   4.330  1.00 38.94           N  \nATOM   1390  CA  GLU B  35      38.242  23.475   5.519  1.00 45.78           C  \nATOM   1391  C   GLU B  35      38.540  24.851   6.114  1.00 78.65           C  \nATOM   1392  O   GLU B  35      37.742  25.420   6.868  1.00 92.39           O  \nATOM   1393  CB  GLU B  35      38.640  22.393   6.538  1.00 39.91           C  \nATOM   1394  CG  GLU B  35      37.742  22.190   7.727  1.00  1.00           C  \nATOM   1395  CD  GLU B  35      36.565  21.380   7.302  1.00 63.87           C  \nATOM   1396  OE1 GLU B  35      36.697  20.806   6.192  1.00 30.74           O  \nATOM   1397  OE2 GLU B  35      35.453  21.481   7.895  1.00  1.00           O  \nATOM   1398  N   GLU B  36      39.716  25.371   5.747  1.00 52.15           N  \nATOM   1399  CA  GLU B  36      40.210  26.663   6.202  1.00 23.04           C  \nATOM   1400  C   GLU B  36      39.746  27.788   5.298  1.00 15.64           C  \nATOM   1401  O   GLU B  36      39.060  28.704   5.748  1.00 24.61           O  \nATOM   1402  CB  GLU B  36      41.719  26.583   6.129  1.00 18.39           C  \nATOM   1403  CG  GLU B  36      42.175  25.173   6.468  1.00 38.87           C  \nATOM   1404  CD  GLU B  36      41.511  24.632   7.758  1.00 46.56           C  \nATOM   1405  OE1 GLU B  36      40.882  23.557   7.729  1.00 60.96           O  \nATOM   1406  OE2 GLU B  36      41.563  25.335   8.782  1.00 61.85           O  \nATOM   1407  N   PHE B  37      40.127  27.707   4.027  1.00 13.13           N  \nATOM   1408  CA  PHE B  37      39.758  28.722   3.047  1.00 24.81           C  \nATOM   1409  C   PHE B  37      38.261  29.021   3.092  1.00 52.35           C  \nATOM   1410  O   PHE B  37      37.848  30.177   3.091  1.00 64.45           O  \nATOM   1411  CB  PHE B  37      40.093  28.226   1.627  1.00 21.07           C  \nATOM   1412  CG  PHE B  37      41.290  28.877   1.012  1.00 18.86           C  \nATOM   1413  CD1 PHE B  37      42.404  29.201   1.784  1.00 12.01           C  \nATOM   1414  CD2 PHE B  37      41.303  29.176  -0.350  1.00 17.49           C  \nATOM   1415  CE1 PHE B  37      43.518  29.786   1.203  1.00 11.23           C  \nATOM   1416  CE2 PHE B  37      42.425  29.785  -0.935  1.00 16.40           C  \nATOM   1417  CZ  PHE B  37      43.541  30.088  -0.149  1.00 11.00           C  \nATOM   1418  N   ALA B  38      37.447  27.965   3.114  1.00 36.87           N  \nATOM   1419  CA  ALA B  38      35.997  28.112   3.147  1.00 42.28           C  \nATOM   1420  C   ALA B  38      35.553  29.036   4.273  1.00 68.53           C  \nATOM   1421  O   ALA B  38      34.760  29.955   4.058  1.00 73.66           O  \nATOM   1422  CB  ALA B  38      35.319  26.778   3.255  1.00 45.26           C  \nATOM   1423  N   VAL B  39      36.055  28.789   5.475  1.00 69.27           N  \nATOM   1424  CA  VAL B  39      35.690  29.628   6.609  1.00 66.33           C  \nATOM   1425  C   VAL B  39      36.448  30.949   6.553  1.00 56.37           C  \nATOM   1426  O   VAL B  39      35.940  31.986   6.977  1.00 50.45           O  \nATOM   1427  CB  VAL B  39      35.974  28.954   7.951  1.00 67.19           C  \nATOM   1428  CG1 VAL B  39      37.390  28.385   7.972  1.00 69.55           C  \nATOM   1429  CG2 VAL B  39      35.751  29.965   9.069  1.00 62.43           C  \nATOM   1430  N   LEU B  40      37.663  30.872   6.014  1.00 46.82           N  \nATOM   1431  CA  LEU B  40      38.593  31.995   5.857  1.00 42.03           C  \nATOM   1432  C   LEU B  40      37.902  33.328   5.508  1.00 63.28           C  \nATOM   1433  O   LEU B  40      38.325  34.406   5.954  1.00 51.13           O  \nATOM   1434  CB  LEU B  40      39.613  31.526   4.802  1.00 29.95           C  \nATOM   1435  CG  LEU B  40      40.714  32.295   4.069  1.00 17.32           C  \nATOM   1436  CD1 LEU B  40      40.244  33.159   2.904  1.00 17.11           C  \nATOM   1437  CD2 LEU B  40      41.677  32.979   4.995  1.00 31.00           C  \nATOM   1438  N   THR B  41      36.829  33.260   4.727  1.00 53.82           N  \nATOM   1439  CA  THR B  41      36.105  34.464   4.341  1.00 51.84           C  \nATOM   1440  C   THR B  41      35.415  35.099   5.543  1.00 70.11           C  \nATOM   1441  O   THR B  41      34.597  36.009   5.389  1.00 83.79           O  \nATOM   1442  CB  THR B  41      34.991  34.117   3.382  1.00 54.60           C  \nATOM   1443  OG1 THR B  41      33.720  34.442   3.957  1.00 41.42           O  \nATOM   1444  CG2 THR B  41      35.016  32.628   3.087  1.00 46.59           C  \nATOM   1445  N   TYR B  42      35.750  34.625   6.743  1.00 58.19           N  \nATOM   1446  CA  TYR B  42      35.148  35.135   7.980  1.00 52.18           C  \nATOM   1447  C   TYR B  42      36.172  35.273   9.109  1.00 68.59           C  \nATOM   1448  O   TYR B  42      35.985  36.069  10.033  1.00 74.47           O  \nATOM   1449  CB  TYR B  42      34.028  34.174   8.436  1.00 50.27           C  \nATOM   1450  CG  TYR B  42      33.309  34.547   9.725  1.00 50.51           C  \nATOM   1451  CD1 TYR B  42      31.977  34.988   9.706  1.00 49.78           C  \nATOM   1452  CD2 TYR B  42      33.933  34.415  10.965  1.00 48.93           C  \nATOM   1453  CE1 TYR B  42      31.312  35.339  10.878  1.00 29.13           C  \nATOM   1454  CE2 TYR B  42      33.283  34.757  12.137  1.00 47.62           C  \nATOM   1455  CZ  TYR B  42      31.969  35.217  12.089  1.00 47.61           C  \nATOM   1456  OH  TYR B  42      31.323  35.556  13.257  1.00 55.49           O  \nATOM   1457  N   ILE B  43      37.263  34.513   9.019  1.00 62.35           N  \nATOM   1458  CA  ILE B  43      38.312  34.570  10.031  1.00 56.39           C  \nATOM   1459  C   ILE B  43      39.714  34.684   9.446  1.00 47.07           C  \nATOM   1460  O   ILE B  43      40.014  34.115   8.399  1.00 36.89           O  \nATOM   1461  CB  ILE B  43      38.056  33.856  11.353  1.00 63.08           C  \nATOM   1462  CG1 ILE B  43      37.264  34.841  12.262  1.00 66.49           C  \nATOM   1463  CG2 ILE B  43      39.381  33.360  11.955  1.00 60.01           C  \nATOM   1464  CD1 ILE B  43      36.702  34.300  13.573  1.00 54.39           C  \nATOM   1465  N   SER B  44      40.557  35.453  10.129  1.00 50.11           N  \nATOM   1466  CA  SER B  44      41.929  35.703   9.695  1.00 45.66           C  \nATOM   1467  C   SER B  44      42.958  34.839  10.326  1.00 35.94           C  \nATOM   1468  O   SER B  44      44.138  34.897  10.018  1.00 34.73           O  \nATOM   1469  CB  SER B  44      42.329  37.161   9.940  1.00 41.23           C  \nATOM   1470  OG  SER B  44      42.782  37.323  11.276  1.00 41.65           O  \nATOM   1471  N   GLU B  45      42.694  33.949  11.238  1.00 31.89           N  \nATOM   1472  CA  GLU B  45      43.907  33.293  11.637  1.00 42.64           C  \nATOM   1473  C   GLU B  45      44.098  31.922  10.977  1.00 43.74           C  \nATOM   1474  O   GLU B  45      45.197  31.349  11.000  1.00  1.00           O  \nATOM   1475  CB  GLU B  45      44.116  33.184  13.107  1.00 40.35           C  \nATOM   1476  CG  GLU B  45      45.424  33.856  13.557  1.00 69.62           C  \nATOM   1477  CD  GLU B  45      46.682  33.105  13.090  1.00100.00           C  \nATOM   1478  OE1 GLU B  45      46.580  31.909  12.614  1.00100.00           O  \nATOM   1479  OE2 GLU B  45      47.846  33.661  13.179  1.00100.00           O  \nATOM   1480  N   ASN B  46      43.072  31.380  10.365  1.00 38.12           N  \nATOM   1481  CA  ASN B  46      43.244  30.093   9.670  1.00 27.58           C  \nATOM   1482  C   ASN B  46      44.353  30.285   8.642  1.00 36.95           C  \nATOM   1483  O   ASN B  46      44.992  29.326   8.184  1.00 45.51           O  \nATOM   1484  CB  ASN B  46      41.956  29.669   8.966  1.00 22.10           C  \nATOM   1485  CG  ASN B  46      41.484  30.670   7.913  1.00 33.25           C  \nATOM   1486  OD1 ASN B  46      42.287  31.174   7.131  1.00 26.29           O  \nATOM   1487  ND2 ASN B  46      40.206  30.995   7.848  1.00 48.82           N  \nATOM   1488  N   LYS B  47      44.537  31.549   8.324  1.00 38.55           N  \nATOM   1489  CA  LYS B  47      45.533  31.975   7.373  1.00 40.30           C  \nATOM   1490  C   LYS B  47      46.926  31.747   7.910  1.00 55.85           C  \nATOM   1491  O   LYS B  47      47.907  31.710   7.176  1.00 66.10           O  \nATOM   1492  CB  LYS B  47      45.423  33.480   7.065  1.00 43.27           C  \nATOM   1493  CG  LYS B  47      44.022  34.048   7.270  1.00 66.39           C  \nATOM   1494  CD  LYS B  47      43.844  35.439   6.650  1.00 42.20           C  \nATOM   1495  CE  LYS B  47      43.545  35.389   5.150  1.00 29.09           C  \nATOM   1496  NZ  LYS B  47      44.172  34.241   4.478  1.00 41.03           N  \nATOM   1497  N   GLU B  48      47.086  31.564   9.207  1.00 42.24           N  \nATOM   1498  CA  GLU B  48      48.463  31.437   9.655  1.00 32.88           C  \nATOM   1499  C   GLU B  48      49.124  30.168   9.146  1.00 50.75           C  \nATOM   1500  O   GLU B  48      49.954  30.199   8.253  1.00 55.85           O  \nATOM   1501  CB  GLU B  48      48.667  31.493  11.130  1.00 29.39           C  \nATOM   1502  CG  GLU B  48      50.170  31.796  11.495  1.00 47.31           C  \nATOM   1503  CD  GLU B  48      51.099  32.268  10.321  1.00100.00           C  \nATOM   1504  OE1 GLU B  48      52.392  32.349  10.500  1.00 16.92           O  \nATOM   1505  OE2 GLU B  48      50.611  32.585   9.170  1.00 49.28           O  \nATOM   1506  N   LYS B  49      48.849  28.960   9.654  1.00 38.84           N  \nATOM   1507  CA  LYS B  49      49.627  27.834   9.039  1.00 32.50           C  \nATOM   1508  C   LYS B  49      49.248  27.754   7.540  1.00 17.18           C  \nATOM   1509  O   LYS B  49      49.913  27.076   6.747  1.00 18.41           O  \nATOM   1510  CB  LYS B  49      49.439  26.443   9.717  1.00 41.16           C  \nATOM   1511  CG  LYS B  49      48.020  26.061  10.066  1.00 60.80           C  \nATOM   1512  CD  LYS B  49      47.858  24.579  10.448  1.00 11.29           C  \nATOM   1513  CE  LYS B  49      48.942  23.676   9.819  1.00 13.09           C  \nATOM   1514  NZ  LYS B  49      50.147  23.520  10.656  1.00 11.24           N  \nATOM   1515  N   GLU B  50      48.180  28.481   7.205  1.00 21.05           N  \nATOM   1516  CA  GLU B  50      47.664  28.575   5.816  1.00 34.52           C  \nATOM   1517  C   GLU B  50      48.825  28.893   4.885  1.00 60.39           C  \nATOM   1518  O   GLU B  50      49.189  28.074   4.041  1.00 80.48           O  \nATOM   1519  CB  GLU B  50      46.615  29.707   5.734  1.00 34.77           C  \nATOM   1520  CG  GLU B  50      45.622  29.559   4.568  1.00 31.00           C  \nATOM   1521  CD  GLU B  50      45.868  30.561   3.426  1.00 42.86           C  \nATOM   1522  OE1 GLU B  50      44.923  31.347   3.029  1.00 99.43           O  \nATOM   1523  OE2 GLU B  50      47.026  30.614   2.858  1.00 63.35           O  \nATOM   1524  N   TYR B  51      49.435  30.052   5.071  1.00 43.87           N  \nATOM   1525  CA  TYR B  51      50.565  30.406   4.240  1.00 41.83           C  \nATOM   1526  C   TYR B  51      51.849  29.712   4.693  1.00 58.78           C  \nATOM   1527  O   TYR B  51      52.764  29.485   3.894  1.00 54.32           O  \nATOM   1528  CB  TYR B  51      50.543  31.773   3.607  1.00 37.29           C  \nATOM   1529  CG  TYR B  51      50.478  31.672   2.095  1.00 39.40           C  \nATOM   1530  CD1 TYR B  51      51.460  32.270   1.320  1.00 53.00           C  \nATOM   1531  CD2 TYR B  51      49.503  30.940   1.443  1.00 31.05           C  \nATOM   1532  CE1 TYR B  51      51.456  32.207  -0.058  1.00 46.43           C  \nATOM   1533  CE2 TYR B  51      49.483  30.888   0.045  1.00 26.35           C  \nATOM   1534  CZ  TYR B  51      50.474  31.535  -0.703  1.00 19.21           C  \nATOM   1535  OH  TYR B  51      50.581  31.507  -2.095  1.00 12.22           O  \nATOM   1536  N   TYR B  52      51.890  29.352   5.975  1.00 61.59           N  \nATOM   1537  CA  TYR B  52      53.038  28.659   6.551  1.00 65.21           C  \nATOM   1538  C   TYR B  52      53.190  27.287   5.892  1.00 66.73           C  \nATOM   1539  O   TYR B  52      54.265  26.682   5.901  1.00 62.57           O  \nATOM   1540  CB  TYR B  52      52.769  28.539   8.068  1.00 68.31           C  \nATOM   1541  CG  TYR B  52      53.951  28.211   8.951  1.00 86.64           C  \nATOM   1542  CD1 TYR B  52      53.761  27.531  10.162  1.00 89.62           C  \nATOM   1543  CD2 TYR B  52      55.252  28.552   8.583  1.00 93.11           C  \nATOM   1544  CE1 TYR B  52      54.832  27.212  10.987  1.00 83.07           C  \nATOM   1545  CE2 TYR B  52      56.337  28.223   9.388  1.00 94.43           C  \nATOM   1546  CZ  TYR B  52      56.126  27.552  10.584  1.00 80.30           C  \nATOM   1547  OH  TYR B  52      57.225  27.249  11.356  1.00 32.94           O  \nATOM   1548  N   LEU B  53      52.095  26.811   5.314  1.00 52.81           N  \nATOM   1549  CA  LEU B  53      52.082  25.523   4.651  1.00 42.47           C  \nATOM   1550  C   LEU B  53      52.048  25.660   3.133  1.00 71.11           C  \nATOM   1551  O   LEU B  53      52.768  24.952   2.424  1.00 77.74           O  \nATOM   1552  CB  LEU B  53      50.940  24.651   5.209  1.00 32.46           C  \nATOM   1553  CG  LEU B  53      51.284  23.791   6.452  1.00 30.08           C  \nATOM   1554  CD1 LEU B  53      52.481  22.850   6.211  1.00 12.83           C  \nATOM   1555  CD2 LEU B  53      51.513  24.659   7.690  1.00 44.13           C  \nATOM   1556  N   LYS B  54      51.216  26.575   2.630  1.00 73.40           N  \nATOM   1557  CA  LYS B  54      51.083  26.783   1.182  1.00 74.73           C  \nATOM   1558  C   LYS B  54      52.388  27.144   0.479  1.00 89.10           C  \nATOM   1559  O   LYS B  54      52.412  27.433  -0.720  1.00 75.26           O  \nATOM   1560  CB  LYS B  54      49.955  27.640   0.693  1.00 65.11           C  \nATOM   1561  CG  LYS B  54      49.616  27.230  -0.728  1.00 51.30           C  \nATOM   1562  CD  LYS B  54      48.377  26.382  -0.724  1.00 41.04           C  \nATOM   1563  CE  LYS B  54      47.505  26.907  -1.855  1.00 71.20           C  \nATOM   1564  NZ  LYS B  54      46.042  26.774  -1.631  1.00 65.94           N  \nATOM   1565  N   ASP B  55      53.486  27.089   1.213  1.00 91.09           N  \nATOM   1566  CA  ASP B  55      54.782  27.381   0.626  1.00 83.83           C  \nATOM   1567  C   ASP B  55      55.830  26.401   1.128  1.00 85.16           C  \nATOM   1568  O   ASP B  55      56.591  25.842   0.339  1.00 89.66           O  \nATOM   1569  CB  ASP B  55      55.285  28.778   0.844  1.00 81.95           C  \nATOM   1570  CG  ASP B  55      56.661  28.942   0.244  1.00 54.77           C  \nATOM   1571  OD1 ASP B  55      57.658  28.869   1.008  1.00 56.86           O  \nATOM   1572  OD2 ASP B  55      56.740  28.855  -1.001  1.00  1.00           O  \nATOM   1573  N   ILE B  56      55.873  26.209   2.445  1.00 74.76           N  \nATOM   1574  CA  ILE B  56      56.821  25.287   3.051  1.00 73.33           C  \nATOM   1575  C   ILE B  56      56.126  23.989   3.447  1.00 96.21           C  \nATOM   1576  O   ILE B  56      55.779  23.795   4.615  1.00100.00           O  \nATOM   1577  CB  ILE B  56      57.536  25.825   4.302  1.00 70.45           C  \nATOM   1578  CG1 ILE B  56      57.528  27.356   4.351  1.00 76.56           C  \nATOM   1579  CG2 ILE B  56      58.972  25.306   4.308  1.00 50.20           C  \nATOM   1580  CD1 ILE B  56      58.294  27.950   5.527  1.00 73.04           C  \nATOM   1581  N   VAL B  67      54.108  32.607 -12.332  1.00 23.76           N  \nATOM   1582  CA  VAL B  67      53.937  31.575 -13.375  1.00 41.96           C  \nATOM   1583  C   VAL B  67      54.354  30.154 -12.972  1.00 66.72           C  \nATOM   1584  O   VAL B  67      53.758  29.162 -13.433  1.00 47.36           O  \nATOM   1585  CB  VAL B  67      54.726  31.927 -14.638  1.00 58.26           C  \nATOM   1586  CG1 VAL B  67      55.709  33.074 -14.427  1.00 61.64           C  \nATOM   1587  CG2 VAL B  67      55.548  30.756 -15.176  1.00 62.76           C  \nATOM   1588  N   VAL B  68      55.367  29.966 -12.115  1.00 87.03           N  \nATOM   1589  CA  VAL B  68      55.707  28.570 -11.805  1.00 91.91           C  \nATOM   1590  C   VAL B  68      54.670  28.006 -10.836  1.00 95.96           C  \nATOM   1591  O   VAL B  68      54.021  27.017 -11.144  1.00100.00           O  \nATOM   1592  CB  VAL B  68      57.137  28.345 -11.350  1.00 98.02           C  \nATOM   1593  CG1 VAL B  68      57.440  26.851 -11.110  1.00100.00           C  \nATOM   1594  CG2 VAL B  68      58.159  28.778 -12.423  1.00 98.08           C  \nATOM   1595  N   LYS B  69      54.494  28.642  -9.674  1.00 81.79           N  \nATOM   1596  CA  LYS B  69      53.507  28.145  -8.709  1.00 80.98           C  \nATOM   1597  C   LYS B  69      52.100  28.263  -9.289  1.00 76.87           C  \nATOM   1598  O   LYS B  69      51.149  27.610  -8.800  1.00 71.59           O  \nATOM   1599  CB  LYS B  69      53.596  28.851  -7.362  1.00 81.77           C  \nATOM   1600  CG  LYS B  69      53.553  30.360  -7.455  1.00 59.50           C  \nATOM   1601  CD  LYS B  69      52.875  30.933  -6.237  1.00 26.49           C  \nATOM   1602  CE  LYS B  69      52.759  29.911  -5.113  1.00 31.23           C  \nATOM   1603  NZ  LYS B  69      51.821  30.367  -4.004  1.00 33.29           N  \nATOM   1604  N   ALA B  70      51.969  29.072 -10.323  1.00 56.12           N  \nATOM   1605  CA  ALA B  70      50.688  29.261 -10.984  1.00 54.68           C  \nATOM   1606  C   ALA B  70      50.216  27.935 -11.559  1.00 58.37           C  \nATOM   1607  O   ALA B  70      49.346  27.269 -10.996  1.00 56.05           O  \nATOM   1608  CB  ALA B  70      50.817  30.295 -12.092  1.00 51.72           C  \nATOM   1609  N   VAL B  71      50.809  27.545 -12.677  1.00 49.40           N  \nATOM   1610  CA  VAL B  71      50.441  26.291 -13.312  1.00 43.70           C  \nATOM   1611  C   VAL B  71      50.564  25.106 -12.349  1.00 35.72           C  \nATOM   1612  O   VAL B  71      49.932  24.068 -12.549  1.00 29.34           O  \nATOM   1613  CB  VAL B  71      51.118  26.080 -14.677  1.00 39.90           C  \nATOM   1614  CG1 VAL B  71      50.916  24.644 -15.092  1.00 41.78           C  \nATOM   1615  CG2 VAL B  71      50.512  27.015 -15.714  1.00 34.44           C  \nATOM   1616  N   LYS B  72      51.379  25.258 -11.302  1.00 14.51           N  \nATOM   1617  CA  LYS B  72      51.534  24.195 -10.322  1.00 20.31           C  \nATOM   1618  C   LYS B  72      50.244  24.110  -9.520  1.00 51.41           C  \nATOM   1619  O   LYS B  72      49.689  23.031  -9.298  1.00 64.46           O  \nATOM   1620  CB  LYS B  72      52.742  24.480  -9.417  1.00 26.33           C  \nATOM   1621  CG  LYS B  72      52.670  23.960  -7.976  1.00 54.90           C  \nATOM   1622  CD  LYS B  72      53.232  22.553  -7.825  1.00100.00           C  \nATOM   1623  CE  LYS B  72      53.412  22.150  -6.357  1.00100.00           C  \nATOM   1624  NZ  LYS B  72      54.708  22.541  -5.710  1.00 23.36           N  \nATOM   1625  N   ILE B  73      49.757  25.274  -9.113  1.00 42.17           N  \nATOM   1626  CA  ILE B  73      48.525  25.356  -8.353  1.00 28.04           C  \nATOM   1627  C   ILE B  73      47.341  25.012  -9.233  1.00 18.21           C  \nATOM   1628  O   ILE B  73      46.465  24.261  -8.828  1.00 32.82           O  \nATOM   1629  CB  ILE B  73      48.286  26.794  -7.915  1.00 26.05           C  \nATOM   1630  CG1 ILE B  73      48.666  26.991  -6.447  1.00 26.85           C  \nATOM   1631  CG2 ILE B  73      46.869  27.192  -8.212  1.00 28.17           C  \nATOM   1632  CD1 ILE B  73      50.163  27.114  -6.188  1.00 25.69           C  \nATOM   1633  N   LEU B  74      47.333  25.571 -10.437  1.00 25.78           N  \nATOM   1634  CA  LEU B  74      46.256  25.371 -11.399  1.00 16.84           C  \nATOM   1635  C   LEU B  74      45.977  23.910 -11.705  1.00 31.49           C  \nATOM   1636  O   LEU B  74      44.810  23.494 -11.736  1.00 23.34           O  \nATOM   1637  CB  LEU B  74      46.608  25.982 -12.732  1.00 19.02           C  \nATOM   1638  CG  LEU B  74      45.325  26.240 -13.457  1.00 20.35           C  \nATOM   1639  CD1 LEU B  74      44.906  25.001 -14.258  1.00 25.78           C  \nATOM   1640  CD2 LEU B  74      44.292  26.569 -12.364  1.00 44.90           C  \nATOM   1641  N   SER B  75      47.046  23.148 -11.965  1.00 48.68           N  \nATOM   1642  CA  SER B  75      46.916  21.726 -12.292  1.00 47.19           C  \nATOM   1643  C   SER B  75      46.302  20.891 -11.171  1.00 42.87           C  \nATOM   1644  O   SER B  75      45.581  19.929 -11.442  1.00 44.86           O  \nATOM   1645  CB  SER B  75      48.012  21.030 -13.071  1.00 36.76           C  \nATOM   1646  OG  SER B  75      47.552  19.713 -13.350  1.00 40.75           O  \nATOM   1647  N   GLN B  76      46.585  21.240  -9.920  1.00 26.65           N  \nATOM   1648  CA  GLN B  76      46.000  20.503  -8.809  1.00 18.42           C  \nATOM   1649  C   GLN B  76      44.578  21.001  -8.597  1.00 38.05           C  \nATOM   1650  O   GLN B  76      43.686  20.254  -8.214  1.00 40.37           O  \nATOM   1651  CB  GLN B  76      46.812  20.572  -7.528  1.00 19.11           C  \nATOM   1652  CG  GLN B  76      46.432  21.696  -6.615  1.00 60.29           C  \nATOM   1653  CD  GLN B  76      46.192  21.197  -5.219  1.00 83.09           C  \nATOM   1654  OE1 GLN B  76      46.385  21.927  -4.246  1.00 65.27           O  \nATOM   1655  NE2 GLN B  76      45.743  19.941  -5.114  1.00 74.74           N  \nATOM   1656  N   GLU B  77      44.359  22.286  -8.858  1.00 53.67           N  \nATOM   1657  CA  GLU B  77      43.020  22.842  -8.713  1.00 51.94           C  \nATOM   1658  C   GLU B  77      42.115  22.083  -9.666  1.00 37.31           C  \nATOM   1659  O   GLU B  77      40.901  22.003  -9.484  1.00 31.72           O  \nATOM   1660  CB  GLU B  77      43.002  24.335  -9.000  1.00 56.06           C  \nATOM   1661  CG  GLU B  77      42.719  25.194  -7.775  1.00 69.62           C  \nATOM   1662  CD  GLU B  77      41.508  26.108  -8.031  1.00100.00           C  \nATOM   1663  OE1 GLU B  77      41.640  27.058  -8.846  1.00100.00           O  \nATOM   1664  OE2 GLU B  77      40.423  25.850  -7.441  1.00100.00           O  \nATOM   1665  N   ASP B  78      42.757  21.525 -10.690  1.00 34.70           N  \nATOM   1666  CA  ASP B  78      42.114  20.701 -11.706  1.00 42.84           C  \nATOM   1667  C   ASP B  78      41.969  19.281 -11.148  1.00 47.06           C  \nATOM   1668  O   ASP B  78      40.985  18.577 -11.402  1.00 27.22           O  \nATOM   1669  CB  ASP B  78      43.104  20.626 -12.905  1.00 45.92           C  \nATOM   1670  CG  ASP B  78      42.769  19.529 -13.912  1.00 80.89           C  \nATOM   1671  OD1 ASP B  78      43.718  18.990 -14.554  1.00 72.61           O  \nATOM   1672  OD2 ASP B  78      41.566  19.321 -14.188  1.00100.00           O  \nATOM   1673  N   TYR B  79      42.968  18.881 -10.362  1.00 56.91           N  \nATOM   1674  CA  TYR B  79      43.021  17.564  -9.733  1.00 56.90           C  \nATOM   1675  C   TYR B  79      41.742  17.277  -8.953  1.00 56.84           C  \nATOM   1676  O   TYR B  79      41.098  16.249  -9.172  1.00 57.29           O  \nATOM   1677  CB  TYR B  79      44.181  17.556  -8.733  1.00 51.98           C  \nATOM   1678  CG  TYR B  79      45.203  16.468  -8.734  1.00 36.20           C  \nATOM   1679  CD1 TYR B  79      45.644  15.882  -9.907  1.00 40.68           C  \nATOM   1680  CD2 TYR B  79      45.897  16.184  -7.556  1.00 34.41           C  \nATOM   1681  CE1 TYR B  79      46.649  14.930  -9.875  1.00 39.09           C  \nATOM   1682  CE2 TYR B  79      46.889  15.233  -7.521  1.00 29.01           C  \nATOM   1683  CZ  TYR B  79      47.256  14.623  -8.680  1.00 44.97           C  \nATOM   1684  OH  TYR B  79      48.238  13.671  -8.680  1.00 78.89           O  \nATOM   1685  N   PHE B  80      41.373  18.168  -8.031  1.00 27.97           N  \nATOM   1686  CA  PHE B  80      40.158  17.949  -7.249  1.00 24.42           C  \nATOM   1687  C   PHE B  80      38.946  18.392  -8.033  1.00 46.07           C  \nATOM   1688  O   PHE B  80      37.809  18.082  -7.692  1.00 43.40           O  \nATOM   1689  CB  PHE B  80      40.103  18.538  -5.797  1.00 27.15           C  \nATOM   1690  CG  PHE B  80      40.031  20.092  -5.670  1.00 43.53           C  \nATOM   1691  CD1 PHE B  80      40.289  20.691  -4.415  1.00 35.73           C  \nATOM   1692  CD2 PHE B  80      39.738  20.967  -6.743  1.00 49.76           C  \nATOM   1693  CE1 PHE B  80      40.250  22.057  -4.234  1.00 17.99           C  \nATOM   1694  CE2 PHE B  80      39.695  22.348  -6.561  1.00 35.97           C  \nATOM   1695  CZ  PHE B  80      39.953  22.885  -5.302  1.00 29.74           C  \nATOM   1696  N   ASP B  81      39.206  19.147  -9.078  1.00 62.95           N  \nATOM   1697  CA  ASP B  81      38.141  19.667  -9.907  1.00 65.32           C  \nATOM   1698  C   ASP B  81      37.444  18.555 -10.683  1.00 67.29           C  \nATOM   1699  O   ASP B  81      36.264  18.656 -11.037  1.00 74.30           O  \nATOM   1700  CB  ASP B  81      38.695  20.827 -10.725  1.00 56.08           C  \nATOM   1701  CG  ASP B  81      37.892  22.059 -10.526  1.00  6.13           C  \nATOM   1702  OD1 ASP B  81      37.750  22.452  -9.334  1.00  1.77           O  \nATOM   1703  OD2 ASP B  81      37.161  22.388 -11.462  1.00 15.64           O  \nATOM   1704  N   LYS B  82      38.187  17.492 -10.926  1.00 42.72           N  \nATOM   1705  CA  LYS B  82      37.656  16.359 -11.644  1.00 36.82           C  \nATOM   1706  C   LYS B  82      36.924  15.416 -10.693  1.00 38.22           C  \nATOM   1707  O   LYS B  82      35.862  14.899 -11.027  1.00 53.92           O  \nATOM   1708  CB  LYS B  82      38.809  15.699 -12.367  1.00 34.60           C  \nATOM   1709  CG  LYS B  82      38.443  14.576 -13.285  1.00 33.92           C  \nATOM   1710  CD  LYS B  82      39.674  14.218 -14.103  1.00 38.62           C  \nATOM   1711  CE  LYS B  82      40.224  15.499 -14.731  1.00100.00           C  \nATOM   1712  NZ  LYS B  82      39.129  16.419 -15.165  1.00100.00           N  \nATOM   1713  N   LYS B  83      37.479  15.195  -9.502  1.00 22.49           N  \nATOM   1714  CA  LYS B  83      36.858  14.292  -8.515  1.00 17.71           C  \nATOM   1715  C   LYS B  83      35.590  14.866  -7.880  1.00 38.91           C  \nATOM   1716  O   LYS B  83      35.069  14.296  -6.924  1.00 28.77           O  \nATOM   1717  CB  LYS B  83      37.788  13.932  -7.348  1.00 17.40           C  \nATOM   1718  CG  LYS B  83      38.375  12.533  -7.206  1.00 11.00           C  \nATOM   1719  CD  LYS B  83      38.798  12.531  -5.753  1.00 23.59           C  \nATOM   1720  CE  LYS B  83      39.522  11.305  -5.238  1.00 11.33           C  \nATOM   1721  NZ  LYS B  83      39.203  11.108  -3.824  1.00 21.00           N  \nATOM   1722  N   ARG B  84      35.134  16.018  -8.376  1.00 55.71           N  \nATOM   1723  CA  ARG B  84      33.924  16.678  -7.884  1.00 40.47           C  \nATOM   1724  C   ARG B  84      32.822  16.462  -8.900  1.00 47.68           C  \nATOM   1725  O   ARG B  84      31.715  16.043  -8.545  1.00 53.15           O  \nATOM   1726  CB  ARG B  84      34.135  18.178  -7.679  1.00 22.90           C  \nATOM   1727  CG  ARG B  84      32.954  19.000  -7.120  1.00 16.14           C  \nATOM   1728  CD  ARG B  84      32.055  19.685  -8.194  1.00 51.00           C  \nATOM   1729  NE  ARG B  84      30.689  19.924  -7.665  1.00 80.23           N  \nATOM   1730  CZ  ARG B  84      29.743  20.685  -8.251  1.00 90.06           C  \nATOM   1731  NH1 ARG B  84      29.999  21.325  -9.393  1.00 15.20           N  \nATOM   1732  NH2 ARG B  84      28.546  20.849  -7.672  1.00 65.36           N  \nATOM   1733  N   ASN B  85      33.149  16.736 -10.172  1.00 48.35           N  \nATOM   1734  CA  ASN B  85      32.214  16.556 -11.291  1.00 47.93           C  \nATOM   1735  C   ASN B  85      31.995  15.064 -11.482  1.00 49.93           C  \nATOM   1736  O   ASN B  85      30.937  14.609 -11.923  1.00 35.55           O  \nATOM   1737  CB  ASN B  85      32.844  17.042 -12.613  1.00 41.00           C  \nATOM   1738  CG  ASN B  85      32.654  18.521 -12.793  1.00 77.53           C  \nATOM   1739  OD1 ASN B  85      33.594  19.264 -13.124  1.00100.00           O  \nATOM   1740  ND2 ASN B  85      31.469  18.985 -12.386  1.00 35.18           N  \nATOM   1741  N   GLU B  86      33.025  14.316 -11.109  1.00 48.64           N  \nATOM   1742  CA  GLU B  86      33.024  12.867 -11.196  1.00 45.66           C  \nATOM   1743  C   GLU B  86      32.217  12.286 -10.054  1.00 36.97           C  \nATOM   1744  O   GLU B  86      31.350  11.436 -10.253  1.00 34.21           O  \nATOM   1745  CB  GLU B  86      34.464  12.418 -10.958  1.00 46.46           C  \nATOM   1746  CG  GLU B  86      34.855  11.050 -11.463  1.00 39.06           C  \nATOM   1747  CD  GLU B  86      36.355  10.908 -11.500  1.00 76.63           C  \nATOM   1748  OE1 GLU B  86      36.973  11.574 -12.365  1.00 27.25           O  \nATOM   1749  OE2 GLU B  86      36.909  10.241 -10.592  1.00 77.36           O  \nATOM   1750  N   HIS B  87      32.521  12.752  -8.852  1.00 28.84           N  \nATOM   1751  CA  HIS B  87      31.840  12.265  -7.671  1.00 22.33           C  \nATOM   1752  C   HIS B  87      30.390  12.715  -7.588  1.00 32.18           C  \nATOM   1753  O   HIS B  87      29.596  12.164  -6.823  1.00 16.69           O  \nATOM   1754  CB  HIS B  87      32.760  11.993  -6.487  1.00 20.72           C  \nATOM   1755  CG  HIS B  87      32.761  13.029  -5.441  1.00 30.62           C  \nATOM   1756  ND1 HIS B  87      33.292  12.800  -4.183  1.00 33.08           N  \nATOM   1757  CD2 HIS B  87      32.256  14.280  -5.422  1.00 47.85           C  \nATOM   1758  CE1 HIS B  87      33.088  13.861  -3.432  1.00 35.15           C  \nATOM   1759  NE2 HIS B  87      32.503  14.784  -4.163  1.00 45.66           N  \nATOM   1760  N   ASP B  88      30.056  13.697  -8.419  1.00 45.56           N  \nATOM   1761  CA  ASP B  88      28.710  14.257  -8.492  1.00 46.10           C  \nATOM   1762  C   ASP B  88      27.726  13.307  -9.182  1.00 57.24           C  \nATOM   1763  O   ASP B  88      26.523  13.567  -9.209  1.00 61.73           O  \nATOM   1764  CB  ASP B  88      28.803  15.585  -9.244  1.00 22.20           C  \nATOM   1765  CG  ASP B  88      28.108  16.732  -8.514  1.00 35.26           C  \nATOM   1766  OD1 ASP B  88      27.577  16.586  -7.386  1.00  1.00           O  \nATOM   1767  OD2 ASP B  88      28.112  17.888  -9.163  1.00 19.25           O  \nATOM   1768  N   GLU B  89      28.227  12.217  -9.749  1.00 50.05           N  \nATOM   1769  CA  GLU B  89      27.365  11.252 -10.423  1.00 41.61           C  \nATOM   1770  C   GLU B  89      26.738  10.297  -9.419  1.00 44.71           C  \nATOM   1771  O   GLU B  89      25.823   9.540  -9.746  1.00 43.45           O  \nATOM   1772  CB  GLU B  89      28.191  10.348 -11.345  1.00 39.93           C  \nATOM   1773  CG  GLU B  89      28.237  10.749 -12.793  1.00 36.24           C  \nATOM   1774  CD  GLU B  89      26.872  10.865 -13.392  1.00 74.97           C  \nATOM   1775  OE1 GLU B  89      26.654  10.269 -14.464  1.00 28.48           O  \nATOM   1776  OE2 GLU B  89      26.047  11.635 -12.857  1.00100.00           O  \nATOM   1777  N   ARG B  90      27.241  10.329  -8.193  1.00 40.96           N  \nATOM   1778  CA  ARG B  90      26.742   9.429  -7.172  1.00 44.93           C  \nATOM   1779  C   ARG B  90      26.616  10.092  -5.805  1.00 46.93           C  \nATOM   1780  O   ARG B  90      26.651   9.427  -4.772  1.00 48.71           O  \nATOM   1781  CB  ARG B  90      27.606   8.164  -7.204  1.00 55.48           C  \nATOM   1782  CG  ARG B  90      27.907   7.644  -8.663  1.00 13.98           C  \nATOM   1783  CD  ARG B  90      29.246   6.853  -8.747  1.00 10.60           C  \nATOM   1784  NE  ARG B  90      29.204   5.484  -8.213  1.00 14.75           N  \nATOM   1785  CZ  ARG B  90      29.138   5.168  -6.916  1.00 62.29           C  \nATOM   1786  NH1 ARG B  90      29.079   6.120  -6.002  1.00 57.55           N  \nATOM   1787  NH2 ARG B  90      29.108   3.898  -6.538  1.00 52.34           N  \nATOM   1788  N   THR B  91      26.472  11.411  -5.813  1.00 52.40           N  \nATOM   1789  CA  THR B  91      26.285  12.167  -4.579  1.00 52.19           C  \nATOM   1790  C   THR B  91      24.795  12.459  -4.429  1.00 35.81           C  \nATOM   1791  O   THR B  91      24.344  13.161  -3.522  1.00 22.95           O  \nATOM   1792  CB  THR B  91      27.240  13.347  -4.295  1.00 38.64           C  \nATOM   1793  OG1 THR B  91      27.203  13.603  -2.886  1.00 12.26           O  \nATOM   1794  CG2 THR B  91      26.801  14.607  -5.028  1.00 43.14           C  \nATOM   1795  N   VAL B  92      24.047  11.880  -5.353  1.00 23.63           N  \nATOM   1796  CA  VAL B  92      22.607  12.021  -5.402  1.00 29.28           C  \nATOM   1797  C   VAL B  92      21.926  11.294  -4.252  1.00 58.81           C  \nATOM   1798  O   VAL B  92      20.699  11.272  -4.199  1.00 64.43           O  \nATOM   1799  CB  VAL B  92      22.067  11.299  -6.599  1.00 29.80           C  \nATOM   1800  CG1 VAL B  92      21.959   9.828  -6.236  1.00 35.07           C  \nATOM   1801  CG2 VAL B  92      20.729  11.897  -6.991  1.00 24.89           C  \nATOM   1802  N   LEU B  93      22.705  10.701  -3.350  1.00 56.65           N  \nATOM   1803  CA  LEU B  93      22.135   9.978  -2.207  1.00 42.76           C  \nATOM   1804  C   LEU B  93      21.095  10.832  -1.481  1.00 37.50           C  \nATOM   1805  O   LEU B  93      19.972  10.404  -1.248  1.00 41.78           O  \nATOM   1806  CB  LEU B  93      23.176   9.724  -1.072  1.00 30.21           C  \nATOM   1807  CG  LEU B  93      24.638   9.228  -1.128  1.00 17.33           C  \nATOM   1808  CD1 LEU B  93      25.563  10.178  -1.865  1.00  1.03           C  \nATOM   1809  CD2 LEU B  93      25.173   9.084   0.314  1.00 10.64           C  \nATOM   1810  N   ILE B  94      21.496  12.055  -1.153  1.00 32.03           N  \nATOM   1811  CA  ILE B  94      20.689  13.042  -0.437  1.00 36.08           C  \nATOM   1812  C   ILE B  94      19.288  13.257  -1.002  1.00 47.88           C  \nATOM   1813  O   ILE B  94      18.325  12.634  -0.554  1.00 21.39           O  \nATOM   1814  CB  ILE B  94      21.404  14.461  -0.643  1.00 38.63           C  \nATOM   1815  CG1 ILE B  94      20.566  15.652  -0.018  1.00 47.84           C  \nATOM   1816  CG2 ILE B  94      21.944  14.651  -2.108  1.00 61.00           C  \nATOM   1817  CD1 ILE B  94      19.893  15.353   1.326  1.00 53.08           C  \nATOM   1818  N   LEU B  95      19.199  14.189  -1.964  1.00 64.24           N  \nATOM   1819  CA  LEU B  95      17.963  14.585  -2.621  1.00 61.57           C  \nATOM   1820  C   LEU B  95      17.082  13.393  -2.929  1.00 49.70           C  \nATOM   1821  O   LEU B  95      15.862  13.483  -2.761  1.00 12.33           O  \nATOM   1822  CB  LEU B  95      18.198  15.201  -4.004  1.00 62.81           C  \nATOM   1823  CG  LEU B  95      17.070  14.729  -4.952  1.00 66.87           C  \nATOM   1824  CD1 LEU B  95      16.175  15.845  -5.488  1.00 72.26           C  \nATOM   1825  CD2 LEU B  95      17.512  13.681  -5.974  1.00 42.81           C  \nATOM   1826  N   VAL B  96      17.720  12.307  -3.374  1.00 65.51           N  \nATOM   1827  CA  VAL B  96      17.075  11.047  -3.725  1.00 58.24           C  \nATOM   1828  C   VAL B  96      16.528  10.379  -2.487  1.00 40.11           C  \nATOM   1829  O   VAL B  96      15.326  10.174  -2.363  1.00 27.76           O  \nATOM   1830  CB  VAL B  96      18.167  10.093  -4.197  1.00 55.24           C  \nATOM   1831  CG1 VAL B  96      17.736   8.657  -3.963  1.00 53.70           C  \nATOM   1832  CG2 VAL B  96      18.486  10.366  -5.669  1.00 58.48           C  \nATOM   1833  N   ASN B  97      17.426  10.051  -1.564  1.00 40.28           N  \nATOM   1834  CA  ASN B  97      17.009   9.456  -0.306  1.00 40.44           C  \nATOM   1835  C   ASN B  97      16.022  10.427   0.321  1.00 40.74           C  \nATOM   1836  O   ASN B  97      15.147  10.046   1.083  1.00 52.12           O  \nATOM   1837  CB  ASN B  97      18.221   9.269   0.611  1.00 46.35           C  \nATOM   1838  CG  ASN B  97      17.839   9.058   2.073  1.00 72.48           C  \nATOM   1839  OD1 ASN B  97      18.549   9.532   2.990  1.00 23.30           O  \nATOM   1840  ND2 ASN B  97      16.752   8.301   2.306  1.00 69.70           N  \nATOM   1841  N   ALA B  98      16.191  11.695  -0.043  1.00 32.27           N  \nATOM   1842  CA  ALA B  98      15.318  12.758   0.434  1.00 46.56           C  \nATOM   1843  C   ALA B  98      14.039  12.722  -0.389  1.00 70.07           C  \nATOM   1844  O   ALA B  98      12.944  12.998   0.107  1.00 75.86           O  \nATOM   1845  CB  ALA B  98      15.985  14.111   0.283  1.00 48.67           C  \nATOM   1846  N   GLN B  99      14.197  12.371  -1.668  1.00 59.70           N  \nATOM   1847  CA  GLN B  99      13.076  12.248  -2.591  1.00 58.87           C  \nATOM   1848  C   GLN B  99      12.288  11.061  -2.101  1.00 59.09           C  \nATOM   1849  O   GLN B  99      11.098  11.161  -1.787  1.00 39.12           O  \nATOM   1850  CB  GLN B  99      13.550  11.969  -4.042  1.00 57.35           C  \nATOM   1851  CG  GLN B  99      12.472  12.098  -5.184  1.00  6.74           C  \nATOM   1852  CD  GLN B  99      12.614  10.967  -6.268  1.00 29.51           C  \nATOM   1853  OE1 GLN B  99      11.822  10.873  -7.227  1.00  1.00           O  \nATOM   1854  NE2 GLN B  99      13.627  10.098  -6.081  1.00 41.23           N  \nATOM   1855  N   GLN B 100      12.992   9.952  -1.932  1.00 67.37           N  \nATOM   1856  CA  GLN B 100      12.345   8.766  -1.427  1.00 65.85           C  \nATOM   1857  C   GLN B 100      11.950   9.086   0.001  1.00 83.30           C  \nATOM   1858  O   GLN B 100      11.061   8.485   0.588  1.00 90.63           O  \nATOM   1859  CB  GLN B 100      13.282   7.548  -1.577  1.00 64.64           C  \nATOM   1860  CG  GLN B 100      13.160   6.888  -2.984  1.00 74.33           C  \nATOM   1861  CD  GLN B 100      14.362   7.100  -3.971  1.00 54.79           C  \nATOM   1862  OE1 GLN B 100      14.511   8.186  -4.544  1.00 31.15           O  \nATOM   1863  NE2 GLN B 100      15.234   6.078  -4.109  1.00 12.87           N  \nATOM   1864  N   ARG B 101      12.594  10.108   0.545  1.00 77.90           N  \nATOM   1865  CA  ARG B 101      12.336  10.608   1.905  1.00 70.19           C  \nATOM   1866  C   ARG B 101      10.922  11.177   1.949  1.00 63.58           C  \nATOM   1867  O   ARG B 101      10.049  10.821   2.731  1.00 61.49           O  \nATOM   1868  CB  ARG B 101      13.354  11.636   2.506  1.00 52.09           C  \nATOM   1869  CG  ARG B 101      14.347  11.046   3.581  1.00 14.27           C  \nATOM   1870  CD  ARG B 101      15.809  11.537   3.497  1.00 20.95           C  \nATOM   1871  NE  ARG B 101      16.384  11.795   4.838  1.00 69.66           N  \nATOM   1872  CZ  ARG B 101      17.686  11.896   5.167  1.00 44.91           C  \nATOM   1873  NH1 ARG B 101      18.656  11.738   4.270  1.00 31.00           N  \nATOM   1874  NH2 ARG B 101      18.019  12.139   6.433  1.00 21.20           N  \nATOM   1875  N   LYS B 102      10.631  12.098   1.029  1.00 58.07           N  \nATOM   1876  CA  LYS B 102       9.286  12.634   0.974  1.00 64.39           C  \nATOM   1877  C   LYS B 102       8.336  11.480   0.663  1.00 78.58           C  \nATOM   1878  O   LYS B 102       7.270  11.356   1.274  1.00 63.69           O  \nATOM   1879  CB  LYS B 102       9.181  13.753  -0.036  1.00 65.34           C  \nATOM   1880  CG  LYS B 102       7.722  14.210  -0.226  1.00 11.50           C  \nATOM   1881  CD  LYS B 102       7.268  15.195   0.857  1.00 38.32           C  \nATOM   1882  CE  LYS B 102       7.599  16.685   0.577  1.00 15.18           C  \nATOM   1883  NZ  LYS B 102       6.687  17.281  -0.472  1.00 11.78           N  \nATOM   1884  N   LYS B 103       8.749  10.607  -0.268  1.00 85.73           N  \nATOM   1885  CA  LYS B 103       7.935   9.455  -0.624  1.00 76.22           C  \nATOM   1886  C   LYS B 103       7.670   8.660   0.631  1.00 33.26           C  \nATOM   1887  O   LYS B 103       6.735   7.866   0.699  1.00 21.27           O  \nATOM   1888  CB  LYS B 103       8.641   8.575  -1.646  1.00 80.27           C  \nATOM   1889  CG  LYS B 103       8.147   7.130  -1.623  1.00 24.70           C  \nATOM   1890  CD  LYS B 103       6.995   6.882  -2.595  1.00 21.49           C  \nATOM   1891  CE  LYS B 103       7.475   6.513  -3.999  1.00 20.48           C  \nATOM   1892  NZ  LYS B 103       8.154   7.620  -4.687  1.00 16.86           N  \nATOM   1893  N   ILE B 104       8.493   8.846   1.694  1.00 16.37           N  \nATOM   1894  CA  ILE B 104       8.181   8.054   2.883  1.00 11.88           C  \nATOM   1895  C   ILE B 104       6.973   8.626   3.580  1.00 19.87           C  \nATOM   1896  O   ILE B 104       6.138   7.875   4.071  1.00 11.56           O  \nATOM   1897  CB  ILE B 104       9.278   7.862   3.976  1.00 15.79           C  \nATOM   1898  CG1 ILE B 104       9.118   8.746   5.223  1.00 13.23           C  \nATOM   1899  CG2 ILE B 104      10.699   8.067   3.520  1.00 15.89           C  \nATOM   1900  CD1 ILE B 104       9.953   8.249   6.421  1.00 32.57           C  \nATOM   1901  N   GLU B 105       6.853   9.922   3.621  1.00 17.17           N  \nATOM   1902  CA  GLU B 105       5.694  10.534   4.303  1.00 20.63           C  \nATOM   1903  C   GLU B 105       4.672  11.060   3.288  1.00 53.26           C  \nATOM   1904  O   GLU B 105       3.463  11.051   3.541  1.00 58.36           O  \nATOM   1905  CB  GLU B 105       6.185  11.701   5.190  1.00 18.14           C  \nATOM   1906  CG  GLU B 105       6.481  11.282   6.650  1.00 20.88           C  \nATOM   1907  CD  GLU B 105       7.973  11.386   7.045  1.00 67.66           C  \nATOM   1908  OE1 GLU B 105       8.744  12.251   6.474  1.00 25.55           O  \nATOM   1909  OE2 GLU B 105       8.460  10.602   7.955  1.00 15.49           O  \nATOM   1910  N   SER B 106       5.157  11.529   2.140  1.00 50.09           N  \nATOM   1911  CA  SER B 106       4.293  12.080   1.092  1.00 44.41           C  \nATOM   1912  C   SER B 106       3.140  11.154   0.717  1.00 32.47           C  \nATOM   1913  O   SER B 106       1.994  11.418   1.082  1.00 24.37           O  \nATOM   1914  CB  SER B 106       5.137  12.403  -0.142  1.00 52.70           C  \nATOM   1915  OG  SER B 106       4.287  12.658  -1.246  1.00 68.61           O  \nATOM   1916  N   LEU B 107       3.451  10.080  -0.021  1.00 37.56           N  \nATOM   1917  CA  LEU B 107       2.442   9.098  -0.439  1.00 26.08           C  \nATOM   1918  C   LEU B 107       1.811   8.525   0.812  1.00 22.55           C  \nATOM   1919  O   LEU B 107       0.593   8.576   0.976  1.00  8.28           O  \nATOM   1920  CB  LEU B 107       3.056   7.947  -1.279  1.00 17.74           C  \nATOM   1921  CG  LEU B 107       2.390   7.635  -2.635  1.00 20.98           C  \nATOM   1922  CD1 LEU B 107       1.852   8.918  -3.228  1.00 30.39           C  \nATOM   1923  CD2 LEU B 107       3.361   6.993  -3.662  1.00  1.00           C  \nATOM   1924  N   LEU B 108       2.683   8.043   1.704  1.00 40.19           N  \nATOM   1925  CA  LEU B 108       2.287   7.485   2.988  1.00 55.28           C  \nATOM   1926  C   LEU B 108       1.359   8.390   3.665  1.00 47.72           C  \nATOM   1927  O   LEU B 108       0.502   7.984   4.434  1.00 39.94           O  \nATOM   1928  CB  LEU B 108       3.469   7.200   3.880  1.00 60.93           C  \nATOM   1929  CG  LEU B 108       3.285   7.272   5.390  1.00 67.08           C  \nATOM   1930  CD1 LEU B 108       4.290   6.362   6.134  1.00 78.93           C  \nATOM   1931  CD2 LEU B 108       3.515   8.669   5.956  1.00 32.69           C  \nATOM   1932  N   SER B 109       1.410   9.675   3.454  1.00 36.75           N  \nATOM   1933  CA  SER B 109       0.430  10.355   4.222  1.00 31.33           C  \nATOM   1934  C   SER B 109      -0.908  10.238   3.511  1.00 32.01           C  \nATOM   1935  O   SER B 109      -1.942  10.549   4.092  1.00 13.66           O  \nATOM   1936  CB  SER B 109       0.769  11.740   4.616  1.00 17.50           C  \nATOM   1937  OG  SER B 109       0.662  11.767   6.085  1.00 11.05           O  \nATOM   1938  N   ARG B 110      -0.903   9.802   2.254  1.00 29.34           N  \nATOM   1939  CA  ARG B 110      -2.201   9.448   1.602  1.00 27.35           C  \nATOM   1940  C   ARG B 110      -2.354   8.017   2.096  1.00 39.21           C  \nATOM   1941  O   ARG B 110      -3.455   7.476   2.121  1.00 43.16           O  \nATOM   1942  CB  ARG B 110      -2.138   9.448   0.054  1.00 11.00           C  \nATOM   1943  CG  ARG B 110      -2.287   7.991  -0.523  1.00 12.29           C  \nATOM   1944  CD  ARG B 110      -2.994   7.888  -1.900  1.00 15.29           C  \nATOM   1945  NE  ARG B 110      -2.051   8.026  -3.026  1.00 57.01           N  \nATOM   1946  CZ  ARG B 110      -2.413   8.242  -4.304  1.00 38.83           C  \nATOM   1947  NH1 ARG B 110      -3.703   8.323  -4.657  1.00 11.24           N  \nATOM   1948  NH2 ARG B 110      -1.541   8.400  -5.313  1.00 27.08           N  \nATOM   1949  N   VAL B 111      -1.229   7.398   2.484  1.00 34.06           N  \nATOM   1950  CA  VAL B 111      -1.277   6.025   2.980  1.00 27.17           C  \nATOM   1951  C   VAL B 111      -2.156   5.985   4.210  1.00 37.28           C  \nATOM   1952  O   VAL B 111      -3.259   5.449   4.161  1.00 43.37           O  \nATOM   1953  CB  VAL B 111       0.012   5.166   3.016  1.00 13.01           C  \nATOM   1954  CG1 VAL B 111       0.787   5.226   1.729  1.00 19.66           C  \nATOM   1955  CG2 VAL B 111       0.816   5.247   4.296  1.00 11.00           C  \nATOM   1956  N   ASN B 112      -1.698   6.592   5.302  1.00 40.50           N  \nATOM   1957  CA  ASN B 112      -2.525   6.637   6.500  1.00 46.00           C  \nATOM   1958  C   ASN B 112      -3.749   7.502   6.206  1.00 65.34           C  \nATOM   1959  O   ASN B 112      -4.668   7.589   7.010  1.00 78.80           O  \nATOM   1960  CB  ASN B 112      -1.779   6.999   7.771  1.00 39.17           C  \nATOM   1961  CG  ASN B 112      -0.690   6.016   8.055  1.00 23.49           C  \nATOM   1962  OD1 ASN B 112      -0.033   5.582   7.115  1.00 16.70           O  \nATOM   1963  ND2 ASN B 112      -0.526   5.590   9.315  1.00 11.20           N  \nATOM   1964  N   LYS B 113      -3.744   8.133   5.029  1.00 44.22           N  \nATOM   1965  CA  LYS B 113      -4.852   8.972   4.584  1.00 37.16           C  \nATOM   1966  C   LYS B 113      -5.957   8.110   3.988  1.00 28.68           C  \nATOM   1967  O   LYS B 113      -7.045   8.059   4.545  1.00 19.43           O  \nATOM   1968  CB  LYS B 113      -4.440  10.006   3.511  1.00 40.12           C  \nATOM   1969  CG  LYS B 113      -5.476  10.293   2.393  1.00 99.91           C  \nATOM   1970  CD  LYS B 113      -6.891  10.716   2.847  1.00100.00           C  \nATOM   1971  CE  LYS B 113      -7.985   9.701   2.474  1.00 68.33           C  \nATOM   1972  NZ  LYS B 113      -9.169   9.637   3.382  1.00 29.34           N  \nATOM   1973  N   ARG B 114      -5.660   7.466   2.843  1.00 39.63           N  \nATOM   1974  CA  ARG B 114      -6.599   6.592   2.121  1.00 34.79           C  \nATOM   1975  C   ARG B 114      -7.293   5.699   3.128  1.00 24.60           C  \nATOM   1976  O   ARG B 114      -8.496   5.412   3.001  1.00 14.16           O  \nATOM   1977  CB  ARG B 114      -5.848   5.649   1.122  1.00 11.86           C  \nATOM   1978  CG  ARG B 114      -5.143   6.352  -0.068  1.00 11.23           C  \nATOM   1979  CD  ARG B 114      -5.702   7.734  -0.423  1.00 48.66           C  \nATOM   1980  NE  ARG B 114      -6.763   7.709  -1.455  1.00 59.93           N  \nATOM   1981  CZ  ARG B 114      -7.908   8.396  -1.338  1.00 15.17           C  \nATOM   1982  NH1 ARG B 114      -8.141   9.156  -0.256  1.00 14.55           N  \nATOM   1983  NH2 ARG B 114      -8.892   8.388  -2.245  1.00 21.00           N  \nATOM   1984  N   ILE B 115      -6.522   5.250   4.173  1.00 28.90           N  \nATOM   1985  CA  ILE B 115      -7.142   4.328   5.115  1.00 22.71           C  \nATOM   1986  C   ILE B 115      -8.321   5.023   5.804  1.00 18.89           C  \nATOM   1987  O   ILE B 115      -9.173   4.395   6.423  1.00 21.10           O  \nATOM   1988  CB  ILE B 115      -6.229   3.613   6.100  1.00 25.25           C  \nATOM   1989  CG1 ILE B 115      -6.535   3.916   7.540  1.00 29.78           C  \nATOM   1990  CG2 ILE B 115      -4.768   3.879   5.879  1.00 19.90           C  \nATOM   1991  CD1 ILE B 115      -7.094   2.707   8.294  1.00 63.51           C  \nATOM   1992  N   THR B 116      -8.336   6.345   5.695  1.00 60.62           N  \nATOM   1993  CA  THR B 116      -9.406   7.151   6.282  1.00 75.84           C  \nATOM   1994  C   THR B 116     -10.484   7.457   5.242  1.00 87.66           C  \nATOM   1995  O   THR B 116     -11.589   7.892   5.570  1.00 84.96           O  \nATOM   1996  CB  THR B 116      -8.817   8.441   6.925  1.00 64.79           C  \nATOM   1997  OG1 THR B 116      -8.851   9.560   6.001  1.00 44.54           O  \nATOM   1998  CG2 THR B 116      -7.386   8.108   7.338  1.00 41.44           C  \nTER    1999      THR B 116                                                      \nHETATM 2000 CA    CA D 200      29.179  22.715 -12.389  1.00 71.98          CA  \nHETATM 2001 CA    CA B 200      14.098  29.996  -1.820  1.00 73.34          CA  \nCONECT   64 2001                                                                \nCONECT  270 2001                                                                \nCONECT  369 2000                                                                \nCONECT 1202 2001                                                                \nCONECT 2000  369                                                                \nCONECT 2001   64  270 1202                                                      \nMASTER      395    0    2   15    0    0    2    6 1997    4    6   22          \nEND                                                                             \n","pdb");
	viewer_1758171620800664.setStyle({"model": 0},{"cartoon": {"color": "blue"}});
	viewer_1758171620800664.addModel("REMARK AtomGroup 2frh\nATOM      1  N   GLY A  98       7.842   9.976  27.496  1.00 69.73           N  \nATOM      2  CA  GLY A  98       7.974  10.959  26.375  1.00 69.80           C  \nATOM      3  C   GLY A  98       7.656  10.358  25.018  1.00 67.79           C  \nATOM      4  O   GLY A  98       6.855  10.893  24.254  1.00 70.31           O  \nATOM      5  N   SER A  99       8.299   9.240  24.713  1.00 63.38           N  \nATOM      6  CA  SER A  99       8.071   8.554  23.459  1.00 57.39           C  \nATOM      7  C   SER A  99       7.833   7.094  23.783  1.00 53.32           C  \nATOM      8  O   SER A  99       8.488   6.544  24.661  1.00 55.60           O  \nATOM      9  CB  SER A  99       9.291   8.706  22.542  1.00 58.07           C  \nATOM     10  OG  SER A  99      10.472   8.252  23.179  1.00 56.71           O  \nATOM     11  N   HIS A 100       6.890   6.474  23.090  1.00 46.90           N  \nATOM     12  CA  HIS A 100       6.581   5.056  23.291  1.00 43.07           C  \nATOM     13  C   HIS A 100       5.693   4.756  24.478  1.00 40.25           C  \nATOM     14  O   HIS A 100       5.922   5.216  25.585  1.00 41.73           O  \nATOM     15  CB  HIS A 100       7.856   4.225  23.420  1.00 41.97           C  \nATOM     16  CG  HIS A 100       8.588   4.031  22.131  1.00 43.04           C  \nATOM     17  ND1 HIS A 100       9.911   4.381  21.966  1.00 43.90           N  \nATOM     18  CD2 HIS A 100       8.195   3.479  20.958  1.00 43.32           C  \nATOM     19  CE1 HIS A 100      10.301   4.051  20.746  1.00 43.72           C  \nATOM     20  NE2 HIS A 100       9.279   3.500  20.114  1.00 43.11           N  \nATOM     21  N   MET A 101       4.661   3.970  24.226  1.00 36.45           N  \nATOM     22  CA  MET A 101       3.746   3.579  25.270  1.00 32.88           C  \nATOM     23  C   MET A 101       4.508   2.857  26.405  1.00 33.98           C  \nATOM     24  O   MET A 101       5.247   1.894  26.160  1.00 33.92           O  \nATOM     25  CB  MET A 101       2.673   2.656  24.693  1.00 28.83           C  \nATOM     26  CG  MET A 101       1.744   2.129  25.745  1.00 29.39           C  \nATOM     27  SD  MET A 101       0.597   0.884  25.175  1.00 29.97           S  \nATOM     28  CE  MET A 101       1.802  -0.621  25.102  1.00 28.17           C  \nATOM     29  N   ALA A 102       4.329   3.341  27.631  1.00 34.54           N  \nATOM     30  CA  ALA A 102       4.945   2.738  28.803  1.00 32.94           C  \nATOM     31  C   ALA A 102       4.311   1.373  28.997  1.00 34.19           C  \nATOM     32  O   ALA A 102       3.186   1.122  28.552  1.00 34.68           O  \nATOM     33  CB  ALA A 102       4.693   3.598  30.053  1.00 32.20           C  \nATOM     34  N   ILE A 103       5.029   0.481  29.653  1.00 35.59           N  \nATOM     35  CA  ILE A 103       4.498  -0.841  29.918  1.00 37.00           C  \nATOM     36  C   ILE A 103       3.747  -0.780  31.244  1.00 37.50           C  \nATOM     37  O   ILE A 103       4.254  -0.267  32.233  1.00 39.97           O  \nATOM     38  CB  ILE A 103       5.625  -1.887  30.014  1.00 36.83           C  \nATOM     39  CG1 ILE A 103       6.382  -1.948  28.677  1.00 37.04           C  \nATOM     40  CG2 ILE A 103       5.062  -3.239  30.438  1.00 35.99           C  \nATOM     41  CD1 ILE A 103       5.518  -2.126  27.448  1.00 36.80           C  \nATOM     42  N   THR A 104       2.528  -1.291  31.252  1.00 37.68           N  \nATOM     43  CA  THR A 104       1.720  -1.304  32.448  1.00 38.12           C  \nATOM     44  C   THR A 104       2.192  -2.443  33.309  1.00 37.58           C  \nATOM     45  O   THR A 104       2.442  -3.534  32.806  1.00 35.62           O  \nATOM     46  CB  THR A 104       0.244  -1.546  32.127  1.00 38.96           C  \nATOM     47  OG1 THR A 104      -0.195  -0.568  31.183  1.00 44.36           O  \nATOM     48  CG2 THR A 104      -0.602  -1.456  33.385  1.00 37.54           C  \nATOM     49  N   LYS A 105       2.297  -2.170  34.605  1.00 37.46           N  \nATOM     50  CA  LYS A 105       2.716  -3.147  35.585  1.00 36.66           C  \nATOM     51  C   LYS A 105       1.599  -4.139  35.878  1.00 36.13           C  \nATOM     52  O   LYS A 105       0.408  -3.801  35.787  1.00 37.72           O  \nATOM     53  CB  LYS A 105       3.115  -2.442  36.890  1.00 36.39           C  \nATOM     54  CG  LYS A 105       4.432  -1.704  36.790  1.00 40.08           C  \nATOM     55  CD  LYS A 105       4.215  -0.186  36.887  1.00 48.35           C  \nATOM     56  CE  LYS A 105       3.900   0.270  38.329  1.00 52.37           C  \nATOM     57  NZ  LYS A 105       3.442   1.692  38.394  1.00 54.85           N  \nATOM     58  N   ILE A 106       1.988  -5.360  36.230  1.00 33.86           N  \nATOM     59  CA  ILE A 106       1.024  -6.384  36.589  1.00 31.21           C  \nATOM     60  C   ILE A 106       1.127  -6.442  38.106  1.00 30.99           C  \nATOM     61  O   ILE A 106       2.083  -5.921  38.677  1.00 29.91           O  \nATOM     62  CB  ILE A 106       1.357  -7.764  35.894  1.00 30.53           C  \nATOM     63  CG1 ILE A 106       2.794  -8.209  36.172  1.00 28.14           C  \nATOM     64  CG2 ILE A 106       1.169  -7.647  34.401  1.00 28.04           C  \nATOM     65  CD1 ILE A 106       3.002  -8.828  37.511  1.00 27.99           C  \nATOM     66  N   ASN A 107       0.148  -7.030  38.777  1.00 31.36           N  \nATOM     67  CA  ASN A 107       0.218  -7.089  40.233  1.00 31.42           C  \nATOM     68  C   ASN A 107      -0.131  -8.468  40.805  1.00 32.85           C  \nATOM     69  O   ASN A 107      -0.354  -8.602  42.002  1.00 33.71           O  \nATOM     70  CB  ASN A 107      -0.683  -6.025  40.831  1.00 32.17           C  \nATOM     71  CG  ASN A 107      -2.122  -6.415  40.768  1.00 32.53           C  \nATOM     72  OD1 ASN A 107      -2.450  -7.366  40.086  1.00 34.64           O  \nATOM     73  ND2 ASN A 107      -2.997  -5.689  41.475  1.00 30.64           N  \nATOM     74  N   ASP A 108      -0.187  -9.491  39.958  1.00 32.36           N  \nATOM     75  CA  ASP A 108      -0.448 -10.833  40.468  1.00 33.18           C  \nATOM     76  C   ASP A 108       0.079 -11.893  39.520  1.00 31.48           C  \nATOM     77  O   ASP A 108       0.378 -11.598  38.363  1.00 30.62           O  \nATOM     78  CB  ASP A 108      -1.943 -11.034  40.795  1.00 35.12           C  \nATOM     79  CG  ASP A 108      -2.799 -11.153  39.574  1.00 36.82           C  \nATOM     80  OD1 ASP A 108      -3.987 -10.878  39.708  1.00 38.28           O  \nATOM     81  OD2 ASP A 108      -2.312 -11.543  38.505  1.00 37.80           O  \nATOM     82  N   CYS A 109       0.198 -13.122  40.020  1.00 30.53           N  \nATOM     83  CA  CYS A 109       0.763 -14.241  39.256  1.00 28.29           C  \nATOM     84  C   CYS A 109       0.176 -14.530  37.897  1.00 27.83           C  \nATOM     85  O   CYS A 109       0.913 -14.737  36.940  1.00 29.81           O  \nATOM     86  CB  CYS A 109       0.732 -15.513  40.089  1.00 28.48           C  \nATOM     87  SG  CYS A 109       1.564 -15.349  41.683  1.00 29.15           S  \nATOM     88  N   PHE A 110      -1.143 -14.561  37.799  1.00 27.32           N  \nATOM     89  CA  PHE A 110      -1.773 -14.843  36.517  1.00 28.09           C  \nATOM     90  C   PHE A 110      -1.434 -13.782  35.465  1.00 28.13           C  \nATOM     91  O   PHE A 110      -1.270 -14.090  34.284  1.00 27.68           O  \nATOM     92  CB  PHE A 110      -3.294 -14.952  36.679  1.00 30.33           C  \nATOM     93  CG  PHE A 110      -3.983 -15.518  35.465  1.00 34.10           C  \nATOM     94  CD1 PHE A 110      -4.619 -14.682  34.555  1.00 35.04           C  \nATOM     95  CD2 PHE A 110      -3.938 -16.886  35.203  1.00 35.64           C  \nATOM     96  CE1 PHE A 110      -5.198 -15.192  33.401  1.00 35.95           C  \nATOM     97  CE2 PHE A 110      -4.513 -17.410  34.049  1.00 36.75           C  \nATOM     98  CZ  PHE A 110      -5.146 -16.560  33.142  1.00 35.74           C  \nATOM     99  N   GLU A 111      -1.313 -12.532  35.886  1.00 27.87           N  \nATOM    100  CA  GLU A 111      -0.987 -11.478  34.932  1.00 27.92           C  \nATOM    101  C   GLU A 111       0.456 -11.594  34.515  1.00 27.82           C  \nATOM    102  O   GLU A 111       0.799 -11.349  33.362  1.00 30.62           O  \nATOM    103  CB  GLU A 111      -1.278 -10.114  35.540  1.00 26.98           C  \nATOM    104  CG  GLU A 111      -2.646 -10.127  36.097  1.00 27.78           C  \nATOM    105  CD  GLU A 111      -3.096  -8.827  36.634  1.00 27.38           C  \nATOM    106  OE1 GLU A 111      -2.326  -8.144  37.329  1.00 25.53           O  \nATOM    107  OE2 GLU A 111      -4.263  -8.496  36.371  1.00 33.08           O  \nATOM    108  N   LEU A 112       1.297 -11.984  35.460  1.00 26.90           N  \nATOM    109  CA  LEU A 112       2.706 -12.158  35.182  1.00 27.09           C  \nATOM    110  C   LEU A 112       2.832 -13.246  34.113  1.00 28.24           C  \nATOM    111  O   LEU A 112       3.468 -13.061  33.060  1.00 29.56           O  \nATOM    112  CB  LEU A 112       3.433 -12.616  36.442  1.00 26.40           C  \nATOM    113  CG  LEU A 112       4.929 -12.290  36.536  1.00 27.67           C  \nATOM    114  CD1 LEU A 112       5.606 -13.405  37.296  1.00 24.89           C  \nATOM    115  CD2 LEU A 112       5.570 -12.109  35.169  1.00 28.01           C  \nATOM    116  N   LEU A 113       2.211 -14.384  34.393  1.00 28.64           N  \nATOM    117  CA  LEU A 113       2.261 -15.509  33.479  1.00 30.22           C  \nATOM    118  C   LEU A 113       1.823 -15.111  32.073  1.00 30.07           C  \nATOM    119  O   LEU A 113       2.504 -15.441  31.087  1.00 31.28           O  \nATOM    120  CB  LEU A 113       1.396 -16.653  34.012  1.00 33.09           C  \nATOM    121  CG  LEU A 113       1.281 -17.880  33.103  1.00 33.77           C  \nATOM    122  CD1 LEU A 113       2.659 -18.400  32.750  1.00 34.69           C  \nATOM    123  CD2 LEU A 113       0.458 -18.948  33.823  1.00 34.13           C  \nATOM    124  N   SER A 114       0.712 -14.384  31.976  1.00 27.82           N  \nATOM    125  CA  SER A 114       0.226 -13.960  30.666  1.00 27.40           C  \nATOM    126  C   SER A 114       1.228 -13.069  29.992  1.00 28.03           C  \nATOM    127  O   SER A 114       1.514 -13.240  28.794  1.00 30.69           O  \nATOM    128  CB  SER A 114      -1.091 -13.204  30.772  1.00 26.21           C  \nATOM    129  OG  SER A 114      -2.056 -13.984  31.451  1.00 31.97           O  \nATOM    130  N   MET A 115       1.765 -12.108  30.737  1.00 27.88           N  \nATOM    131  CA  MET A 115       2.731 -11.190  30.141  1.00 27.92           C  \nATOM    132  C   MET A 115       3.913 -11.966  29.556  1.00 28.03           C  \nATOM    133  O   MET A 115       4.329 -11.716  28.428  1.00 27.13           O  \nATOM    134  CB  MET A 115       3.221 -10.176  31.170  1.00 29.42           C  \nATOM    135  CG  MET A 115       4.182  -9.172  30.577  1.00 32.13           C  \nATOM    136  SD  MET A 115       4.692  -7.849  31.710  1.00 36.32           S  \nATOM    137  CE  MET A 115       3.446  -6.563  31.322  1.00 31.49           C  \nATOM    138  N   VAL A 116       4.431 -12.921  30.324  1.00 28.59           N  \nATOM    139  CA  VAL A 116       5.565 -13.727  29.880  1.00 28.72           C  \nATOM    140  C   VAL A 116       5.192 -14.567  28.657  1.00 31.19           C  \nATOM    141  O   VAL A 116       5.953 -14.635  27.681  1.00 31.53           O  \nATOM    142  CB  VAL A 116       6.048 -14.648  31.004  1.00 28.09           C  \nATOM    143  CG1 VAL A 116       7.003 -15.672  30.450  1.00 28.30           C  \nATOM    144  CG2 VAL A 116       6.702 -13.808  32.119  1.00 23.75           C  \nATOM    145  N   THR A 117       4.015 -15.190  28.705  1.00 32.42           N  \nATOM    146  CA  THR A 117       3.561 -15.994  27.579  1.00 33.88           C  \nATOM    147  C   THR A 117       3.537 -15.123  26.339  1.00 33.25           C  \nATOM    148  O   THR A 117       4.091 -15.487  25.309  1.00 34.96           O  \nATOM    149  CB  THR A 117       2.154 -16.554  27.818  1.00 34.93           C  \nATOM    150  OG1 THR A 117       2.157 -17.357  29.004  1.00 36.82           O  \nATOM    151  CG2 THR A 117       1.713 -17.407  26.625  1.00 33.69           C  \nATOM    152  N   TYR A 118       2.903 -13.961  26.437  1.00 33.50           N  \nATOM    153  CA  TYR A 118       2.836 -13.053  25.302  1.00 35.17           C  \nATOM    154  C   TYR A 118       4.234 -12.572  24.851  1.00 36.84           C  \nATOM    155  O   TYR A 118       4.519 -12.500  23.654  1.00 38.19           O  \nATOM    156  CB  TYR A 118       1.936 -11.865  25.669  1.00 33.45           C  \nATOM    157  CG  TYR A 118       1.952 -10.714  24.693  1.00 30.33           C  \nATOM    158  CD1 TYR A 118       1.521 -10.869  23.375  1.00 31.08           C  \nATOM    159  CD2 TYR A 118       2.401  -9.471  25.087  1.00 29.66           C  \nATOM    160  CE1 TYR A 118       1.542  -9.794  22.468  1.00 31.13           C  \nATOM    161  CE2 TYR A 118       2.433  -8.385  24.198  1.00 29.99           C  \nATOM    162  CZ  TYR A 118       2.004  -8.551  22.891  1.00 30.53           C  \nATOM    163  OH  TYR A 118       2.059  -7.475  22.019  1.00 31.14           O  \nATOM    164  N   ALA A 119       5.104 -12.249  25.803  1.00 37.94           N  \nATOM    165  CA  ALA A 119       6.440 -11.780  25.441  1.00 39.19           C  \nATOM    166  C   ALA A 119       7.277 -12.891  24.818  1.00 39.54           C  \nATOM    167  O   ALA A 119       8.028 -12.639  23.883  1.00 41.18           O  \nATOM    168  CB  ALA A 119       7.160 -11.198  26.665  1.00 38.29           C  \nATOM    169  N   ASP A 120       7.143 -14.115  25.324  1.00 39.63           N  \nATOM    170  CA  ASP A 120       7.909 -15.228  24.783  1.00 40.95           C  \nATOM    171  C   ASP A 120       7.522 -15.490  23.341  1.00 39.74           C  \nATOM    172  O   ASP A 120       8.381 -15.736  22.494  1.00 39.51           O  \nATOM    173  CB  ASP A 120       7.699 -16.505  25.610  1.00 46.17           C  \nATOM    174  CG  ASP A 120       8.499 -16.511  26.914  1.00 49.81           C  \nATOM    175  OD1 ASP A 120       9.469 -15.733  27.050  1.00 52.48           O  \nATOM    176  OD2 ASP A 120       8.158 -17.313  27.805  1.00 50.63           O  \nATOM    177  N   LYS A 121       6.227 -15.444  23.055  1.00 39.42           N  \nATOM    178  CA  LYS A 121       5.778 -15.670  21.687  1.00 39.14           C  \nATOM    179  C   LYS A 121       6.371 -14.654  20.737  1.00 38.33           C  \nATOM    180  O   LYS A 121       6.894 -15.024  19.694  1.00 40.67           O  \nATOM    181  CB  LYS A 121       4.255 -15.653  21.601  1.00 40.36           C  \nATOM    182  CG  LYS A 121       3.631 -16.863  22.269  1.00 44.95           C  \nATOM    183  CD  LYS A 121       2.114 -16.845  22.186  1.00 50.57           C  \nATOM    184  CE  LYS A 121       1.527 -18.015  22.972  1.00 53.55           C  \nATOM    185  NZ  LYS A 121       0.059 -17.865  23.190  1.00 55.87           N  \nATOM    186  N   LEU A 122       6.318 -13.376  21.086  1.00 38.19           N  \nATOM    187  CA  LEU A 122       6.876 -12.376  20.193  1.00 38.33           C  \nATOM    188  C   LEU A 122       8.382 -12.549  20.056  1.00 38.55           C  \nATOM    189  O   LEU A 122       8.924 -12.328  18.983  1.00 38.33           O  \nATOM    190  CB  LEU A 122       6.539 -10.958  20.673  1.00 37.84           C  \nATOM    191  CG  LEU A 122       5.059 -10.555  20.664  1.00 37.14           C  \nATOM    192  CD1 LEU A 122       4.955  -9.083  21.018  1.00 38.72           C  \nATOM    193  CD2 LEU A 122       4.440 -10.795  19.294  1.00 36.47           C  \nATOM    194  N   LYS A 123       9.057 -12.953  21.127  1.00 40.77           N  \nATOM    195  CA  LYS A 123      10.503 -13.154  21.063  1.00 43.84           C  \nATOM    196  C   LYS A 123      10.877 -14.297  20.127  1.00 45.28           C  \nATOM    197  O   LYS A 123      11.737 -14.140  19.254  1.00 45.01           O  \nATOM    198  CB  LYS A 123      11.102 -13.463  22.441  1.00 43.94           C  \nATOM    199  CG  LYS A 123      11.147 -12.294  23.382  1.00 46.03           C  \nATOM    200  CD  LYS A 123      12.240 -12.474  24.447  1.00 46.71           C  \nATOM    201  CE  LYS A 123      12.054 -13.746  25.250  1.00 46.58           C  \nATOM    202  NZ  LYS A 123      13.093 -13.865  26.307  1.00 47.36           N  \nATOM    203  N   SER A 124      10.237 -15.447  20.313  1.00 47.45           N  \nATOM    204  CA  SER A 124      10.537 -16.614  19.496  1.00 51.18           C  \nATOM    205  C   SER A 124      10.262 -16.355  18.022  1.00 52.87           C  \nATOM    206  O   SER A 124      10.989 -16.849  17.150  1.00 55.23           O  \nATOM    207  CB  SER A 124       9.727 -17.812  19.965  1.00 51.39           C  \nATOM    208  OG  SER A 124       8.366 -17.651  19.615  1.00 54.26           O  \nATOM    209  N   LEU A 125       9.216 -15.579  17.739  1.00 54.19           N  \nATOM    210  CA  LEU A 125       8.879 -15.248  16.358  1.00 53.73           C  \nATOM    211  C   LEU A 125       9.983 -14.405  15.741  1.00 52.72           C  \nATOM    212  O   LEU A 125      10.513 -14.735  14.683  1.00 52.26           O  \nATOM    213  CB  LEU A 125       7.555 -14.490  16.294  1.00 54.76           C  \nATOM    214  CG  LEU A 125       6.321 -15.384  16.184  1.00 56.55           C  \nATOM    215  CD1 LEU A 125       5.056 -14.590  16.487  1.00 57.89           C  \nATOM    216  CD2 LEU A 125       6.268 -15.986  14.779  1.00 57.44           C  \nATOM    217  N   ILE A 126      10.325 -13.311  16.403  1.00 53.08           N  \nATOM    218  CA  ILE A 126      11.372 -12.436  15.901  1.00 55.08           C  \nATOM    219  C   ILE A 126      12.641 -13.242  15.640  1.00 57.42           C  \nATOM    220  O   ILE A 126      13.363 -12.985  14.678  1.00 58.65           O  \nATOM    221  CB  ILE A 126      11.653 -11.298  16.895  1.00 54.25           C  \nATOM    222  CG1 ILE A 126      10.459 -10.339  16.894  1.00 54.46           C  \nATOM    223  CG2 ILE A 126      12.943 -10.592  16.547  1.00 54.05           C  \nATOM    224  CD1 ILE A 126      10.564  -9.209  17.903  1.00 56.43           C  \nATOM    225  N   LYS A 127      12.909 -14.223  16.494  1.00 60.36           N  \nATOM    226  CA  LYS A 127      14.084 -15.057  16.315  1.00 64.17           C  \nATOM    227  C   LYS A 127      13.864 -16.034  15.154  1.00 65.54           C  \nATOM    228  O   LYS A 127      14.668 -16.084  14.230  1.00 65.69           O  \nATOM    229  CB  LYS A 127      14.387 -15.807  17.612  1.00 66.16           C  \nATOM    230  CG  LYS A 127      14.672 -14.873  18.778  1.00 68.72           C  \nATOM    231  CD  LYS A 127      15.115 -15.612  20.037  1.00 70.55           C  \nATOM    232  CE  LYS A 127      15.433 -14.620  21.147  1.00 71.58           C  \nATOM    233  NZ  LYS A 127      16.008 -15.264  22.367  1.00 73.37           N  \nATOM    234  N   LYS A 128      12.776 -16.801  15.206  1.00 66.96           N  \nATOM    235  CA  LYS A 128      12.447 -17.758  14.151  1.00 68.54           C  \nATOM    236  C   LYS A 128      12.611 -17.045  12.812  1.00 67.07           C  \nATOM    237  O   LYS A 128      13.523 -17.334  12.050  1.00 66.93           O  \nATOM    238  CB  LYS A 128      11.009 -18.244  14.333  1.00 71.65           C  \nATOM    239  CG  LYS A 128      10.567 -19.322  13.360  1.00 76.53           C  \nATOM    240  CD  LYS A 128       9.172 -19.826  13.720  1.00 80.78           C  \nATOM    241  CE  LYS A 128       8.698 -20.927  12.774  1.00 82.85           C  \nATOM    242  NZ  LYS A 128       7.345 -21.452  13.140  1.00 86.41           N  \nATOM    243  N   GLU A 129      11.714 -16.113  12.527  1.00 65.60           N  \nATOM    244  CA  GLU A 129      11.807 -15.313  11.315  1.00 64.20           C  \nATOM    245  C   GLU A 129      12.941 -14.359  11.655  1.00 63.85           C  \nATOM    246  O   GLU A 129      13.398 -14.360  12.795  1.00 65.43           O  \nATOM    247  CB  GLU A 129      10.496 -14.558  11.104  1.00 64.39           C  \nATOM    248  CG  GLU A 129       9.344 -15.493  10.755  1.00 69.30           C  \nATOM    249  CD  GLU A 129       7.998 -14.806  10.751  1.00 71.83           C  \nATOM    250  OE1 GLU A 129       7.433 -14.624  11.846  1.00 74.57           O  \nATOM    251  OE2 GLU A 129       7.508 -14.446   9.655  1.00 74.49           O  \nATOM    252  N   PHE A 130      13.420 -13.565  10.705  1.00 63.67           N  \nATOM    253  CA  PHE A 130      14.514 -12.630  11.002  1.00 64.04           C  \nATOM    254  C   PHE A 130      15.784 -13.299  11.525  1.00 65.65           C  \nATOM    255  O   PHE A 130      16.854 -12.692  11.516  1.00 64.92           O  \nATOM    256  CB  PHE A 130      14.066 -11.596  12.038  1.00 61.98           C  \nATOM    257  CG  PHE A 130      12.922 -10.760  11.589  1.00 60.79           C  \nATOM    258  CD1 PHE A 130      13.092  -9.813  10.594  1.00 60.31           C  \nATOM    259  CD2 PHE A 130      11.656 -10.957  12.123  1.00 61.13           C  \nATOM    260  CE1 PHE A 130      12.016  -9.075  10.128  1.00 60.92           C  \nATOM    261  CE2 PHE A 130      10.572 -10.226  11.665  1.00 62.16           C  \nATOM    262  CZ  PHE A 130      10.750  -9.281  10.664  1.00 61.42           C  \nATOM    263  N   SER A 131      15.654 -14.540  11.990  1.00 69.04           N  \nATOM    264  CA  SER A 131      16.773 -15.303  12.536  1.00 71.18           C  \nATOM    265  C   SER A 131      17.810 -14.373  13.144  1.00 71.07           C  \nATOM    266  O   SER A 131      18.797 -14.002  12.500  1.00 70.85           O  \nATOM    267  CB  SER A 131      17.399 -16.201  11.455  1.00 71.33           C  \nATOM    268  OG  SER A 131      17.646 -15.484  10.261  1.00 74.50           O  \nATOM    269  N   ILE A 132      17.558 -13.998  14.395  1.00 70.68           N  \nATOM    270  CA  ILE A 132      18.421 -13.093  15.136  1.00 69.59           C  \nATOM    271  C   ILE A 132      18.135 -13.299  16.624  1.00 68.83           C  \nATOM    272  O   ILE A 132      16.988 -13.521  17.006  1.00 69.99           O  \nATOM    273  CB  ILE A 132      18.118 -11.623  14.737  1.00 70.11           C  \nATOM    274  CG1 ILE A 132      19.079 -10.670  15.438  1.00 71.74           C  \nATOM    275  CG2 ILE A 132      16.679 -11.266  15.099  1.00 68.82           C  \nATOM    276  CD1 ILE A 132      18.798  -9.212  15.131  1.00 72.84           C  \nATOM    277  N   SER A 133      19.169 -13.244  17.461  1.00 67.22           N  \nATOM    278  CA  SER A 133      18.980 -13.420  18.899  1.00 65.76           C  \nATOM    279  C   SER A 133      18.341 -12.160  19.496  1.00 65.62           C  \nATOM    280  O   SER A 133      18.391 -11.079  18.899  1.00 64.46           O  \nATOM    281  CB  SER A 133      20.319 -13.677  19.580  1.00 64.57           C  \nATOM    282  OG  SER A 133      21.109 -12.498  19.571  1.00 62.75           O  \nATOM    283  N   PHE A 134      17.752 -12.291  20.680  1.00 64.82           N  \nATOM    284  CA  PHE A 134      17.123 -11.131  21.279  1.00 64.88           C  \nATOM    285  C   PHE A 134      18.124 -10.056  21.689  1.00 64.23           C  \nATOM    286  O   PHE A 134      17.811  -8.866  21.640  1.00 64.45           O  \nATOM    287  CB  PHE A 134      16.248 -11.524  22.469  1.00 65.87           C  \nATOM    288  CG  PHE A 134      15.040 -10.652  22.615  1.00 66.33           C  \nATOM    289  CD1 PHE A 134      14.055 -10.655  21.629  1.00 66.62           C  \nATOM    290  CD2 PHE A 134      14.922  -9.773  23.683  1.00 65.14           C  \nATOM    291  CE1 PHE A 134      12.972  -9.792  21.698  1.00 66.81           C  \nATOM    292  CE2 PHE A 134      13.843  -8.902  23.766  1.00 66.26           C  \nATOM    293  CZ  PHE A 134      12.862  -8.910  22.767  1.00 67.04           C  \nATOM    294  N   GLU A 135      19.325 -10.464  22.088  1.00 63.98           N  \nATOM    295  CA  GLU A 135      20.340  -9.482  22.456  1.00 63.28           C  \nATOM    296  C   GLU A 135      20.698  -8.666  21.215  1.00 60.02           C  \nATOM    297  O   GLU A 135      20.930  -7.463  21.303  1.00 59.16           O  \nATOM    298  CB  GLU A 135      21.599 -10.154  23.034  1.00 66.31           C  \nATOM    299  CG  GLU A 135      21.720 -11.642  22.756  1.00 69.90           C  \nATOM    300  CD  GLU A 135      20.815 -12.481  23.642  1.00 70.70           C  \nATOM    301  OE1 GLU A 135      21.008 -12.468  24.876  1.00 70.45           O  \nATOM    302  OE2 GLU A 135      19.910 -13.152  23.103  1.00 73.09           O  \nATOM    303  N   GLU A 136      20.734  -9.314  20.056  1.00 57.02           N  \nATOM    304  CA  GLU A 136      21.033  -8.588  18.822  1.00 57.07           C  \nATOM    305  C   GLU A 136      19.897  -7.594  18.578  1.00 54.94           C  \nATOM    306  O   GLU A 136      20.108  -6.480  18.089  1.00 55.21           O  \nATOM    307  CB  GLU A 136      21.119  -9.537  17.626  1.00 59.14           C  \nATOM    308  CG  GLU A 136      22.275 -10.521  17.634  1.00 62.62           C  \nATOM    309  CD  GLU A 136      22.197 -11.499  16.465  1.00 64.31           C  \nATOM    310  OE1 GLU A 136      22.293 -11.055  15.302  1.00 64.27           O  \nATOM    311  OE2 GLU A 136      22.029 -12.711  16.707  1.00 66.36           O  \nATOM    312  N   PHE A 137      18.686  -8.027  18.921  1.00 52.16           N  \nATOM    313  CA  PHE A 137      17.477  -7.227  18.785  1.00 46.80           C  \nATOM    314  C   PHE A 137      17.562  -6.035  19.733  1.00 44.71           C  \nATOM    315  O   PHE A 137      17.320  -4.902  19.330  1.00 42.68           O  \nATOM    316  CB  PHE A 137      16.256  -8.095  19.122  1.00 46.05           C  \nATOM    317  CG  PHE A 137      14.951  -7.355  19.092  1.00 45.18           C  \nATOM    318  CD1 PHE A 137      14.400  -6.937  17.887  1.00 44.79           C  \nATOM    319  CD2 PHE A 137      14.271  -7.077  20.276  1.00 45.65           C  \nATOM    320  CE1 PHE A 137      13.185  -6.251  17.858  1.00 44.16           C  \nATOM    321  CE2 PHE A 137      13.059  -6.395  20.264  1.00 44.75           C  \nATOM    322  CZ  PHE A 137      12.514  -5.982  19.054  1.00 43.66           C  \nATOM    323  N   ALA A 138      17.924  -6.297  20.986  1.00 43.36           N  \nATOM    324  CA  ALA A 138      18.046  -5.247  21.990  1.00 45.79           C  \nATOM    325  C   ALA A 138      19.107  -4.207  21.629  1.00 46.65           C  \nATOM    326  O   ALA A 138      18.918  -3.007  21.864  1.00 47.62           O  \nATOM    327  CB  ALA A 138      18.362  -5.858  23.347  1.00 46.74           C  \nATOM    328  N   VAL A 139      20.225  -4.664  21.065  1.00 45.69           N  \nATOM    329  CA  VAL A 139      21.279  -3.743  20.677  1.00 44.05           C  \nATOM    330  C   VAL A 139      20.749  -2.914  19.520  1.00 43.20           C  \nATOM    331  O   VAL A 139      20.854  -1.682  19.511  1.00 40.51           O  \nATOM    332  CB  VAL A 139      22.570  -4.492  20.237  1.00 43.70           C  \nATOM    333  CG1 VAL A 139      23.528  -3.522  19.540  1.00 42.75           C  \nATOM    334  CG2 VAL A 139      23.251  -5.104  21.452  1.00 43.33           C  \nATOM    335  N   LEU A 140      20.168  -3.600  18.546  1.00 43.05           N  \nATOM    336  CA  LEU A 140      19.619  -2.904  17.402  1.00 44.44           C  \nATOM    337  C   LEU A 140      18.548  -1.873  17.754  1.00 44.82           C  \nATOM    338  O   LEU A 140      18.577  -0.765  17.208  1.00 46.24           O  \nATOM    339  CB  LEU A 140      19.061  -3.897  16.388  1.00 44.15           C  \nATOM    340  CG  LEU A 140      20.080  -4.732  15.620  1.00 44.03           C  \nATOM    341  CD1 LEU A 140      19.380  -5.397  14.440  1.00 42.43           C  \nATOM    342  CD2 LEU A 140      21.225  -3.850  15.132  1.00 44.11           C  \nATOM    343  N   THR A 141      17.614  -2.197  18.657  1.00 43.24           N  \nATOM    344  CA  THR A 141      16.588  -1.203  18.975  1.00 43.49           C  \nATOM    345  C   THR A 141      17.161  -0.094  19.864  1.00 44.43           C  \nATOM    346  O   THR A 141      16.708   1.050  19.820  1.00 43.00           O  \nATOM    347  CB  THR A 141      15.285  -1.830  19.615  1.00 42.01           C  \nATOM    348  OG1 THR A 141      15.331  -1.730  21.036  1.00 43.93           O  \nATOM    349  CG2 THR A 141      15.128  -3.283  19.226  1.00 40.36           C  \nATOM    350  N   TYR A 142      18.171  -0.419  20.660  1.00 47.53           N  \nATOM    351  CA  TYR A 142      18.792   0.598  21.500  1.00 51.48           C  \nATOM    352  C   TYR A 142      19.442   1.631  20.582  1.00 52.69           C  \nATOM    353  O   TYR A 142      19.326   2.840  20.800  1.00 53.30           O  \nATOM    354  CB  TYR A 142      19.873   0.001  22.396  1.00 53.78           C  \nATOM    355  CG  TYR A 142      20.610   1.067  23.161  1.00 57.48           C  \nATOM    356  CD1 TYR A 142      20.128   1.537  24.381  1.00 58.80           C  \nATOM    357  CD2 TYR A 142      21.744   1.681  22.620  1.00 58.80           C  \nATOM    358  CE1 TYR A 142      20.752   2.600  25.047  1.00 59.53           C  \nATOM    359  CE2 TYR A 142      22.373   2.741  23.272  1.00 59.62           C  \nATOM    360  CZ  TYR A 142      21.871   3.198  24.483  1.00 59.54           C  \nATOM    361  OH  TYR A 142      22.474   4.264  25.118  1.00 60.26           O  \nATOM    362  N   ILE A 143      20.137   1.141  19.560  1.00 54.56           N  \nATOM    363  CA  ILE A 143      20.802   2.009  18.598  1.00 56.31           C  \nATOM    364  C   ILE A 143      19.781   2.884  17.886  1.00 58.81           C  \nATOM    365  O   ILE A 143      20.015   4.077  17.680  1.00 59.77           O  \nATOM    366  CB  ILE A 143      21.596   1.179  17.551  1.00 55.14           C  \nATOM    367  CG1 ILE A 143      22.914   0.709  18.162  1.00 54.24           C  \nATOM    368  CG2 ILE A 143      21.854   1.995  16.299  1.00 54.90           C  \nATOM    369  CD1 ILE A 143      23.706  -0.211  17.264  1.00 56.55           C  \nATOM    370  N   SER A 144      18.642   2.293  17.526  1.00 60.71           N  \nATOM    371  CA  SER A 144      17.600   3.038  16.831  1.00 61.95           C  \nATOM    372  C   SER A 144      16.921   4.105  17.693  1.00 62.86           C  \nATOM    373  O   SER A 144      16.466   5.123  17.178  1.00 63.59           O  \nATOM    374  CB  SER A 144      16.544   2.082  16.287  1.00 61.58           C  \nATOM    375  OG  SER A 144      15.671   2.760  15.398  1.00 59.45           O  \nATOM    376  N   GLU A 145      16.856   3.873  19.001  1.00 63.64           N  \nATOM    377  CA  GLU A 145      16.215   4.819  19.908  1.00 65.77           C  \nATOM    378  C   GLU A 145      17.050   6.051  20.172  1.00 67.43           C  \nATOM    379  O   GLU A 145      16.539   7.170  20.183  1.00 67.57           O  \nATOM    380  CB  GLU A 145      15.892   4.143  21.237  1.00 66.24           C  \nATOM    381  CG  GLU A 145      14.468   3.677  21.325  1.00 69.38           C  \nATOM    382  CD  GLU A 145      13.482   4.824  21.293  1.00 70.89           C  \nATOM    383  OE1 GLU A 145      13.390   5.565  22.294  1.00 71.85           O  \nATOM    384  OE2 GLU A 145      12.806   4.987  20.258  1.00 71.51           O  \nATOM    385  N   ASN A 146      18.338   5.830  20.395  1.00 70.15           N  \nATOM    386  CA  ASN A 146      19.275   6.910  20.673  1.00 72.11           C  \nATOM    387  C   ASN A 146      20.179   7.093  19.459  1.00 74.06           C  \nATOM    388  O   ASN A 146      21.374   6.819  19.518  1.00 75.17           O  \nATOM    389  CB  ASN A 146      20.096   6.541  21.903  1.00 70.71           C  \nATOM    390  CG  ASN A 146      19.235   5.969  23.017  1.00 70.75           C  \nATOM    391  OD1 ASN A 146      18.437   6.680  23.632  1.00 69.12           O  \nATOM    392  ND2 ASN A 146      19.381   4.675  23.268  1.00 70.95           N  \nATOM    393  N   LYS A 147      19.594   7.542  18.354  1.00 77.33           N  \nATOM    394  CA  LYS A 147      20.338   7.746  17.119  1.00 79.84           C  \nATOM    395  C   LYS A 147      21.677   8.423  17.391  1.00 81.33           C  \nATOM    396  O   LYS A 147      21.766   9.356  18.189  1.00 81.30           O  \nATOM    397  CB  LYS A 147      19.513   8.587  16.142  1.00 80.69           C  \nATOM    398  CG  LYS A 147      18.191   7.944  15.733  1.00 81.42           C  \nATOM    399  CD  LYS A 147      16.984   8.826  16.080  1.00 82.46           C  \nATOM    400  CE  LYS A 147      16.751   8.943  17.593  1.00 83.00           C  \nATOM    401  NZ  LYS A 147      17.836   9.661  18.327  1.00 80.22           N  \nATOM    402  N   GLU A 148      22.717   7.939  16.722  1.00 82.64           N  \nATOM    403  CA  GLU A 148      24.061   8.480  16.881  1.00 83.18           C  \nATOM    404  C   GLU A 148      24.931   7.955  15.742  1.00 83.16           C  \nATOM    405  O   GLU A 148      24.477   7.161  14.916  1.00 82.57           O  \nATOM    406  CB  GLU A 148      24.654   8.028  18.217  1.00 83.18           C  \nATOM    407  CG  GLU A 148      25.359   9.114  19.009  1.00 83.41           C  \nATOM    408  CD  GLU A 148      24.432   9.828  19.981  1.00 83.90           C  \nATOM    409  OE1 GLU A 148      23.878   9.157  20.878  1.00 85.21           O  \nATOM    410  OE2 GLU A 148      24.257  11.059  19.853  1.00 83.58           O  \nATOM    411  N   LYS A 149      26.181   8.406  15.705  1.00 83.59           N  \nATOM    412  CA  LYS A 149      27.139   7.982  14.689  1.00 84.80           C  \nATOM    413  C   LYS A 149      28.197   7.162  15.402  1.00 86.09           C  \nATOM    414  O   LYS A 149      28.821   6.277  14.822  1.00 86.06           O  \nATOM    415  CB  LYS A 149      27.807   9.197  14.046  1.00 84.36           C  \nATOM    416  CG  LYS A 149      26.917  10.023  13.139  1.00 84.07           C  \nATOM    417  CD  LYS A 149      26.797   9.388  11.768  1.00 84.26           C  \nATOM    418  CE  LYS A 149      26.143  10.350  10.785  1.00 85.30           C  \nATOM    419  NZ  LYS A 149      26.878  11.653  10.722  1.00 86.88           N  \nATOM    420  N   GLU A 150      28.378   7.473  16.679  1.00 88.50           N  \nATOM    421  CA  GLU A 150      29.363   6.812  17.523  1.00 90.25           C  \nATOM    422  C   GLU A 150      28.752   6.352  18.845  1.00 88.69           C  \nATOM    423  O   GLU A 150      28.153   7.145  19.570  1.00 85.09           O  \nATOM    424  CB  GLU A 150      30.525   7.773  17.801  1.00 94.83           C  \nATOM    425  CG  GLU A 150      30.252   9.234  17.410  1.00100.22           C  \nATOM    426  CD  GLU A 150      28.964   9.794  18.013  1.00102.77           C  \nATOM    427  OE1 GLU A 150      28.849   9.852  19.259  1.00103.46           O  \nATOM    428  OE2 GLU A 150      28.063  10.178  17.233  1.00103.47           O  \nATOM    429  N   TYR A 151      28.913   5.069  19.156  1.00 88.60           N  \nATOM    430  CA  TYR A 151      28.383   4.514  20.396  1.00 89.53           C  \nATOM    431  C   TYR A 151      29.464   3.829  21.219  1.00 90.33           C  \nATOM    432  O   TYR A 151      30.339   3.155  20.678  1.00 89.96           O  \nATOM    433  CB  TYR A 151      27.271   3.493  20.120  1.00 89.49           C  \nATOM    434  CG  TYR A 151      26.070   4.032  19.379  1.00 89.54           C  \nATOM    435  CD1 TYR A 151      26.068   4.122  17.988  1.00 91.04           C  \nATOM    436  CD2 TYR A 151      24.925   4.434  20.067  1.00 88.69           C  \nATOM    437  CE1 TYR A 151      24.957   4.594  17.301  1.00 91.17           C  \nATOM    438  CE2 TYR A 151      23.809   4.908  19.388  1.00 89.05           C  \nATOM    439  CZ  TYR A 151      23.832   4.983  18.006  1.00 90.57           C  \nATOM    440  OH  TYR A 151      22.733   5.444  17.322  1.00 91.65           O  \nATOM    441  N   TYR A 152      29.383   4.002  22.535  1.00 91.83           N  \nATOM    442  CA  TYR A 152      30.327   3.394  23.460  1.00 93.18           C  \nATOM    443  C   TYR A 152      29.777   2.044  23.894  1.00 92.08           C  \nATOM    444  O   TYR A 152      28.774   1.980  24.602  1.00 91.91           O  \nATOM    445  CB  TYR A 152      30.512   4.281  24.696  1.00 97.30           C  \nATOM    446  CG  TYR A 152      31.180   5.607  24.422  1.00102.16           C  \nATOM    447  CD1 TYR A 152      30.669   6.487  23.465  1.00104.71           C  \nATOM    448  CD2 TYR A 152      32.327   5.985  25.119  1.00103.83           C  \nATOM    449  CE1 TYR A 152      31.286   7.712  23.207  1.00106.82           C  \nATOM    450  CE2 TYR A 152      32.950   7.206  24.871  1.00106.11           C  \nATOM    451  CZ  TYR A 152      32.426   8.064  23.914  1.00106.90           C  \nATOM    452  OH  TYR A 152      33.041   9.270  23.664  1.00108.41           O  \nATOM    453  N   LEU A 153      30.427   0.966  23.473  1.00 91.02           N  \nATOM    454  CA  LEU A 153      29.975  -0.367  23.849  1.00 90.69           C  \nATOM    455  C   LEU A 153      29.770  -0.437  25.359  1.00 90.09           C  \nATOM    456  O   LEU A 153      29.093  -1.327  25.862  1.00 89.51           O  \nATOM    457  CB  LEU A 153      30.993  -1.423  23.413  1.00 91.87           C  \nATOM    458  CG  LEU A 153      30.658  -2.862  23.816  1.00 93.21           C  \nATOM    459  CD1 LEU A 153      29.352  -3.290  23.172  1.00 93.99           C  \nATOM    460  CD2 LEU A 153      31.780  -3.787  23.392  1.00 93.98           C  \nATOM    461  N   LYS A 154      30.365   0.507  26.077  1.00 90.54           N  \nATOM    462  CA  LYS A 154      30.231   0.561  27.527  1.00 91.44           C  \nATOM    463  C   LYS A 154      28.786   0.881  27.873  1.00 91.49           C  \nATOM    464  O   LYS A 154      28.129   0.131  28.595  1.00 91.11           O  \nATOM    465  CB  LYS A 154      31.141   1.648  28.109  1.00 92.82           C  \nATOM    466  CG  LYS A 154      30.975   1.863  29.610  1.00 93.79           C  \nATOM    467  CD  LYS A 154      31.724   3.104  30.084  1.00 95.28           C  \nATOM    468  CE  LYS A 154      31.631   3.276  31.595  1.00 95.57           C  \nATOM    469  NZ  LYS A 154      30.223   3.380  32.064  1.00 95.84           N  \nATOM    470  N   ASP A 155      28.302   2.003  27.346  1.00 92.36           N  \nATOM    471  CA  ASP A 155      26.935   2.453  27.587  1.00 93.01           C  \nATOM    472  C   ASP A 155      25.911   1.395  27.196  1.00 92.57           C  \nATOM    473  O   ASP A 155      25.054   1.026  27.998  1.00 92.53           O  \nATOM    474  CB  ASP A 155      26.661   3.742  26.811  1.00 93.76           C  \nATOM    475  CG  ASP A 155      27.699   4.818  27.085  1.00 95.05           C  \nATOM    476  OD1 ASP A 155      28.002   5.060  28.273  1.00 95.46           O  \nATOM    477  OD2 ASP A 155      28.206   5.427  26.116  1.00 95.22           O  \nATOM    478  N   ILE A 156      26.007   0.911  25.962  1.00 92.30           N  \nATOM    479  CA  ILE A 156      25.086  -0.105  25.469  1.00 92.10           C  \nATOM    480  C   ILE A 156      24.931  -1.244  26.472  1.00 91.77           C  \nATOM    481  O   ILE A 156      23.813  -1.619  26.830  1.00 92.62           O  \nATOM    482  CB  ILE A 156      25.561  -0.681  24.114  1.00 91.76           C  \nATOM    483  CG1 ILE A 156      25.492   0.410  23.040  1.00 92.04           C  \nATOM    484  CG2 ILE A 156      24.703  -1.878  23.722  1.00 90.71           C  \nATOM    485  CD1 ILE A 156      25.950  -0.037  21.668  1.00 91.70           C  \nATOM    486  N   ILE A 157      26.052  -1.788  26.927  1.00 90.73           N  \nATOM    487  CA  ILE A 157      26.020  -2.876  27.893  1.00 90.22           C  \nATOM    488  C   ILE A 157      25.323  -2.436  29.178  1.00 90.17           C  \nATOM    489  O   ILE A 157      24.619  -3.219  29.814  1.00 89.04           O  \nATOM    490  CB  ILE A 157      27.442  -3.350  28.241  1.00 90.76           C  \nATOM    491  CG1 ILE A 157      28.186  -3.740  26.961  1.00 91.29           C  \nATOM    492  CG2 ILE A 157      27.372  -4.547  29.183  1.00 91.25           C  \nATOM    493  CD1 ILE A 157      29.634  -4.116  27.178  1.00 90.81           C  \nATOM    494  N   ASN A 158      25.523  -1.178  29.552  1.00 91.33           N  \nATOM    495  CA  ASN A 158      24.916  -0.636  30.761  1.00 93.52           C  \nATOM    496  C   ASN A 158      23.404  -0.544  30.642  1.00 93.07           C  \nATOM    497  O   ASN A 158      22.668  -1.015  31.515  1.00 94.10           O  \nATOM    498  CB  ASN A 158      25.476   0.757  31.067  1.00 95.99           C  \nATOM    499  CG  ASN A 158      26.892   0.714  31.614  1.00 98.43           C  \nATOM    500  OD1 ASN A 158      27.152   0.108  32.656  1.00 99.13           O  \nATOM    501  ND2 ASN A 158      27.815   1.364  30.914  1.00100.12           N  \nATOM    502  N   HIS A 159      22.945   0.061  29.553  1.00 91.50           N  \nATOM    503  CA  HIS A 159      21.521   0.244  29.327  1.00 89.98           C  \nATOM    504  C   HIS A 159      20.746  -1.031  29.021  1.00 89.28           C  \nATOM    505  O   HIS A 159      19.519  -1.032  29.041  1.00 89.99           O  \nATOM    506  CB  HIS A 159      21.311   1.287  28.233  1.00 89.50           C  \nATOM    507  CG  HIS A 159      21.798   2.651  28.615  1.00 90.03           C  \nATOM    508  ND1 HIS A 159      21.628   3.759  27.811  1.00 90.94           N  \nATOM    509  CD2 HIS A 159      22.447   3.086  29.721  1.00 89.61           C  \nATOM    510  CE1 HIS A 159      22.150   4.817  28.407  1.00 90.19           C  \nATOM    511  NE2 HIS A 159      22.653   4.435  29.566  1.00 90.10           N  \nATOM    512  N   LEU A 160      21.454  -2.118  28.741  1.00 88.49           N  \nATOM    513  CA  LEU A 160      20.789  -3.386  28.474  1.00 87.40           C  \nATOM    514  C   LEU A 160      20.962  -4.298  29.688  1.00 88.50           C  \nATOM    515  O   LEU A 160      21.773  -4.018  30.572  1.00 89.21           O  \nATOM    516  CB  LEU A 160      21.370  -4.057  27.226  1.00 83.83           C  \nATOM    517  CG  LEU A 160      21.243  -3.305  25.897  1.00 80.91           C  \nATOM    518  CD1 LEU A 160      21.570  -4.258  24.757  1.00 78.53           C  \nATOM    519  CD2 LEU A 160      19.838  -2.754  25.729  1.00 79.96           C  \nATOM    520  N   ASN A 161      20.198  -5.384  29.733  1.00 89.51           N  \nATOM    521  CA  ASN A 161      20.276  -6.323  30.846  1.00 91.09           C  \nATOM    522  C   ASN A 161      20.595  -7.742  30.380  1.00 92.60           C  \nATOM    523  O   ASN A 161      19.882  -8.682  30.733  1.00 93.58           O  \nATOM    524  CB  ASN A 161      18.954  -6.317  31.622  1.00 89.95           C  \nATOM    525  CG  ASN A 161      18.769  -5.057  32.451  1.00 90.14           C  \nATOM    526  OD1 ASN A 161      19.378  -4.902  33.510  1.00 90.24           O  \nATOM    527  ND2 ASN A 161      17.937  -4.143  31.965  1.00 90.25           N  \nATOM    528  N   TYR A 162      21.670  -7.903  29.608  1.00 94.68           N  \nATOM    529  CA  TYR A 162      22.035  -9.226  29.104  1.00 97.60           C  \nATOM    530  C   TYR A 162      23.441  -9.738  29.425  1.00 99.04           C  \nATOM    531  O   TYR A 162      23.833 -10.794  28.929  1.00100.06           O  \nATOM    532  CB  TYR A 162      21.829  -9.286  27.583  1.00 97.58           C  \nATOM    533  CG  TYR A 162      20.381  -9.264  27.147  1.00 98.39           C  \nATOM    534  CD1 TYR A 162      19.699  -8.060  26.978  1.00 98.75           C  \nATOM    535  CD2 TYR A 162      19.681 -10.453  26.936  1.00 98.43           C  \nATOM    536  CE1 TYR A 162      18.351  -8.041  26.613  1.00 99.10           C  \nATOM    537  CE2 TYR A 162      18.338 -10.448  26.574  1.00 98.54           C  \nATOM    538  CZ  TYR A 162      17.678  -9.241  26.415  1.00 99.04           C  \nATOM    539  OH  TYR A 162      16.345  -9.236  26.080  1.00 98.32           O  \nATOM    540  N   LYS A 163      24.197  -9.014  30.249  1.00100.17           N  \nATOM    541  CA  LYS A 163      25.562  -9.432  30.586  1.00100.51           C  \nATOM    542  C   LYS A 163      26.484  -9.303  29.368  1.00 99.20           C  \nATOM    543  O   LYS A 163      26.151  -9.768  28.276  1.00 98.96           O  \nATOM    544  CB  LYS A 163      25.576 -10.884  31.089  1.00102.46           C  \nATOM    545  CG  LYS A 163      25.166 -11.077  32.549  1.00103.63           C  \nATOM    546  CD  LYS A 163      26.191 -10.464  33.502  1.00104.88           C  \nATOM    547  CE  LYS A 163      26.064 -11.020  34.922  1.00105.31           C  \nATOM    548  NZ  LYS A 163      24.741 -10.749  35.550  1.00104.88           N  \nATOM    549  N   GLN A 164      27.645  -8.681  29.569  1.00 96.62           N  \nATOM    550  CA  GLN A 164      28.622  -8.457  28.500  1.00 93.83           C  \nATOM    551  C   GLN A 164      28.828  -9.621  27.531  1.00 91.83           C  \nATOM    552  O   GLN A 164      28.782  -9.436  26.315  1.00 92.16           O  \nATOM    553  CB  GLN A 164      29.975  -8.075  29.094  1.00 94.00           C  \nATOM    554  CG  GLN A 164      29.957  -6.822  29.935  1.00 93.68           C  \nATOM    555  CD  GLN A 164      31.320  -6.509  30.510  1.00 94.28           C  \nATOM    556  OE1 GLN A 164      32.278  -6.282  29.770  1.00 93.29           O  \nATOM    557  NE2 GLN A 164      31.418  -6.503  31.836  1.00 95.48           N  \nATOM    558  N   PRO A 165      29.077 -10.831  28.056  1.00 89.50           N  \nATOM    559  CA  PRO A 165      29.290 -12.004  27.204  1.00 88.47           C  \nATOM    560  C   PRO A 165      28.337 -12.091  26.012  1.00 88.05           C  \nATOM    561  O   PRO A 165      28.777 -12.202  24.870  1.00 87.34           O  \nATOM    562  CB  PRO A 165      29.111 -13.163  28.178  1.00 88.94           C  \nATOM    563  CG  PRO A 165      29.695 -12.607  29.430  1.00 88.73           C  \nATOM    564  CD  PRO A 165      29.108 -11.211  29.480  1.00 88.74           C  \nATOM    565  N   GLN A 166      27.036 -12.034  26.284  1.00 87.95           N  \nATOM    566  CA  GLN A 166      26.022 -12.110  25.234  1.00 88.10           C  \nATOM    567  C   GLN A 166      25.936 -10.828  24.403  1.00 85.87           C  \nATOM    568  O   GLN A 166      25.713 -10.878  23.191  1.00 84.98           O  \nATOM    569  CB  GLN A 166      24.654 -12.413  25.847  1.00 91.33           C  \nATOM    570  CG  GLN A 166      24.601 -13.711  26.637  1.00 96.08           C  \nATOM    571  CD  GLN A 166      23.204 -14.021  27.147  1.00 99.20           C  \nATOM    572  OE1 GLN A 166      22.281 -14.255  26.362  1.00101.11           O  \nATOM    573  NE2 GLN A 166      23.040 -14.018  28.466  1.00100.08           N  \nATOM    574  N   VAL A 167      26.108  -9.685  25.058  1.00 83.51           N  \nATOM    575  CA  VAL A 167      26.055  -8.402  24.372  1.00 82.82           C  \nATOM    576  C   VAL A 167      27.200  -8.293  23.367  1.00 84.06           C  \nATOM    577  O   VAL A 167      26.965  -8.091  22.174  1.00 86.30           O  \nATOM    578  CB  VAL A 167      26.154  -7.232  25.371  1.00 82.19           C  \nATOM    579  CG1 VAL A 167      26.024  -5.905  24.637  1.00 80.91           C  \nATOM    580  CG2 VAL A 167      25.077  -7.366  26.434  1.00 82.33           C  \nATOM    581  N   VAL A 168      28.436  -8.425  23.845  1.00 84.45           N  \nATOM    582  CA  VAL A 168      29.607  -8.341  22.971  1.00 84.47           C  \nATOM    583  C   VAL A 168      29.492  -9.383  21.866  1.00 84.38           C  \nATOM    584  O   VAL A 168      29.900  -9.153  20.727  1.00 84.13           O  \nATOM    585  CB  VAL A 168      30.913  -8.584  23.753  1.00 84.75           C  \nATOM    586  CG1 VAL A 168      32.093  -8.592  22.798  1.00 84.64           C  \nATOM    587  CG2 VAL A 168      31.097  -7.500  24.806  1.00 84.92           C  \nATOM    588  N   LYS A 169      28.929 -10.531  22.226  1.00 84.34           N  \nATOM    589  CA  LYS A 169      28.714 -11.631  21.299  1.00 83.21           C  \nATOM    590  C   LYS A 169      27.768 -11.158  20.198  1.00 82.01           C  \nATOM    591  O   LYS A 169      27.947 -11.488  19.026  1.00 80.96           O  \nATOM    592  CB  LYS A 169      28.102 -12.801  22.063  1.00 85.18           C  \nATOM    593  CG  LYS A 169      27.802 -14.048  21.268  1.00 87.45           C  \nATOM    594  CD  LYS A 169      27.283 -15.120  22.228  1.00 89.77           C  \nATOM    595  CE  LYS A 169      26.972 -16.434  21.536  1.00 90.97           C  \nATOM    596  NZ  LYS A 169      26.564 -17.470  22.526  1.00 92.55           N  \nATOM    597  N   ALA A 170      26.767 -10.368  20.590  1.00 81.02           N  \nATOM    598  CA  ALA A 170      25.780  -9.831  19.657  1.00 79.11           C  \nATOM    599  C   ALA A 170      26.376  -8.712  18.806  1.00 78.92           C  \nATOM    600  O   ALA A 170      26.174  -8.669  17.591  1.00 79.41           O  \nATOM    601  CB  ALA A 170      24.570  -9.317  20.424  1.00 78.15           C  \nATOM    602  N   VAL A 171      27.098  -7.800  19.452  1.00 78.09           N  \nATOM    603  CA  VAL A 171      27.739  -6.691  18.751  1.00 77.10           C  \nATOM    604  C   VAL A 171      28.607  -7.272  17.638  1.00 78.52           C  \nATOM    605  O   VAL A 171      28.544  -6.842  16.483  1.00 78.48           O  \nATOM    606  CB  VAL A 171      28.647  -5.886  19.701  1.00 76.03           C  \nATOM    607  CG1 VAL A 171      29.343  -4.766  18.941  1.00 74.31           C  \nATOM    608  CG2 VAL A 171      27.828  -5.336  20.850  1.00 75.48           C  \nATOM    609  N   LYS A 172      29.410  -8.263  18.014  1.00 80.10           N  \nATOM    610  CA  LYS A 172      30.315  -8.951  17.105  1.00 80.98           C  \nATOM    611  C   LYS A 172      29.592  -9.396  15.834  1.00 79.65           C  \nATOM    612  O   LYS A 172      29.934  -8.970  14.729  1.00 79.95           O  \nATOM    613  CB  LYS A 172      30.917 -10.160  17.827  1.00 82.38           C  \nATOM    614  CG  LYS A 172      32.041 -10.871  17.097  1.00 84.33           C  \nATOM    615  CD  LYS A 172      32.717 -11.858  18.041  1.00 85.67           C  \nATOM    616  CE  LYS A 172      33.882 -12.577  17.381  1.00 87.28           C  \nATOM    617  NZ  LYS A 172      34.612 -13.427  18.356  1.00 87.30           N  \nATOM    618  N   ILE A 173      28.592 -10.254  15.999  1.00 77.61           N  \nATOM    619  CA  ILE A 173      27.820 -10.759  14.868  1.00 74.88           C  \nATOM    620  C   ILE A 173      27.236  -9.612  14.043  1.00 75.65           C  \nATOM    621  O   ILE A 173      27.454  -9.525  12.832  1.00 77.05           O  \nATOM    622  CB  ILE A 173      26.668 -11.657  15.352  1.00 72.48           C  \nATOM    623  CG1 ILE A 173      27.217 -12.759  16.259  1.00 71.17           C  \nATOM    624  CG2 ILE A 173      25.941 -12.261  14.160  1.00 70.76           C  \nATOM    625  CD1 ILE A 173      26.147 -13.563  16.948  1.00 71.10           C  \nATOM    626  N   LEU A 174      26.494  -8.730  14.706  1.00 75.58           N  \nATOM    627  CA  LEU A 174      25.883  -7.590  14.031  1.00 75.63           C  \nATOM    628  C   LEU A 174      26.914  -6.819  13.226  1.00 76.94           C  \nATOM    629  O   LEU A 174      26.623  -6.326  12.136  1.00 76.47           O  \nATOM    630  CB  LEU A 174      25.234  -6.654  15.052  1.00 73.42           C  \nATOM    631  CG  LEU A 174      24.006  -7.186  15.791  1.00 71.53           C  \nATOM    632  CD1 LEU A 174      23.592  -6.200  16.873  1.00 69.66           C  \nATOM    633  CD2 LEU A 174      22.876  -7.423  14.798  1.00 68.71           C  \nATOM    634  N   SER A 175      28.119  -6.718  13.778  1.00 79.26           N  \nATOM    635  CA  SER A 175      29.218  -6.009  13.131  1.00 80.70           C  \nATOM    636  C   SER A 175      29.593  -6.664  11.804  1.00 82.12           C  \nATOM    637  O   SER A 175      29.814  -5.980  10.806  1.00 80.72           O  \nATOM    638  CB  SER A 175      30.432  -5.978  14.056  1.00 79.69           C  \nATOM    639  OG  SER A 175      31.451  -5.172  13.502  1.00 80.67           O  \nATOM    640  N   GLN A 176      29.660  -7.994  11.804  1.00 85.66           N  \nATOM    641  CA  GLN A 176      29.991  -8.753  10.600  1.00 88.99           C  \nATOM    642  C   GLN A 176      28.873  -8.554   9.589  1.00 89.88           C  \nATOM    643  O   GLN A 176      29.119  -8.295   8.412  1.00 89.79           O  \nATOM    644  CB  GLN A 176      30.108 -10.242  10.921  1.00 90.70           C  \nATOM    645  CG  GLN A 176      30.968 -10.568  12.131  1.00 93.20           C  \nATOM    646  CD  GLN A 176      31.014 -12.057  12.410  1.00 94.08           C  \nATOM    647  OE1 GLN A 176      31.518 -12.494  13.447  1.00 94.93           O  \nATOM    648  NE2 GLN A 176      30.489 -12.846  11.480  1.00 94.09           N  \nATOM    649  N   GLU A 177      27.638  -8.688  10.065  1.00 91.84           N  \nATOM    650  CA  GLU A 177      26.458  -8.509   9.225  1.00 93.10           C  \nATOM    651  C   GLU A 177      26.528  -7.112   8.616  1.00 92.99           C  \nATOM    652  O   GLU A 177      25.801  -6.787   7.676  1.00 92.43           O  \nATOM    653  CB  GLU A 177      25.195  -8.652  10.073  1.00 94.47           C  \nATOM    654  CG  GLU A 177      25.076  -9.992  10.775  1.00 96.38           C  \nATOM    655  CD  GLU A 177      24.700 -11.112   9.828  1.00 97.72           C  \nATOM    656  OE1 GLU A 177      25.345 -11.236   8.765  1.00 97.83           O  \nATOM    657  OE2 GLU A 177      23.760 -11.871  10.147  1.00 99.20           O  \nATOM    658  N   ASP A 178      27.412  -6.296   9.179  1.00 92.60           N  \nATOM    659  CA  ASP A 178      27.639  -4.934   8.727  1.00 93.17           C  \nATOM    660  C   ASP A 178      26.465  -4.004   9.004  1.00 92.56           C  \nATOM    661  O   ASP A 178      25.991  -3.296   8.115  1.00 92.07           O  \nATOM    662  CB  ASP A 178      27.987  -4.934   7.233  1.00 95.54           C  \nATOM    663  CG  ASP A 178      28.598  -3.622   6.775  1.00 97.05           C  \nATOM    664  OD1 ASP A 178      29.461  -3.081   7.502  1.00 98.04           O  \nATOM    665  OD2 ASP A 178      28.225  -3.141   5.682  1.00 97.96           O  \nATOM    666  N   TYR A 179      25.997  -4.017  10.249  1.00 91.96           N  \nATOM    667  CA  TYR A 179      24.902  -3.149  10.672  1.00 92.55           C  \nATOM    668  C   TYR A 179      25.562  -1.895  11.222  1.00 94.30           C  \nATOM    669  O   TYR A 179      24.984  -0.810  11.230  1.00 94.63           O  \nATOM    670  CB  TYR A 179      24.060  -3.839  11.749  1.00 90.55           C  \nATOM    671  CG  TYR A 179      23.004  -4.759  11.177  1.00 88.71           C  \nATOM    672  CD1 TYR A 179      21.891  -4.239  10.517  1.00 87.68           C  \nATOM    673  CD2 TYR A 179      23.136  -6.147  11.255  1.00 87.85           C  \nATOM    674  CE1 TYR A 179      20.938  -5.074   9.943  1.00 87.19           C  \nATOM    675  CE2 TYR A 179      22.187  -6.994  10.683  1.00 86.69           C  \nATOM    676  CZ  TYR A 179      21.092  -6.450  10.028  1.00 86.72           C  \nATOM    677  OH  TYR A 179      20.155  -7.275   9.446  1.00 85.04           O  \nATOM    678  N   PHE A 180      26.793  -2.077  11.682  1.00 96.83           N  \nATOM    679  CA  PHE A 180      27.614  -1.004  12.218  1.00 99.56           C  \nATOM    680  C   PHE A 180      29.049  -1.516  12.200  1.00102.69           C  \nATOM    681  O   PHE A 180      29.296  -2.698  12.451  1.00102.81           O  \nATOM    682  CB  PHE A 180      27.187  -0.620  13.644  1.00 97.63           C  \nATOM    683  CG  PHE A 180      26.997  -1.788  14.571  1.00 95.26           C  \nATOM    684  CD1 PHE A 180      25.724  -2.285  14.829  1.00 94.24           C  \nATOM    685  CD2 PHE A 180      28.084  -2.381  15.200  1.00 93.87           C  \nATOM    686  CE1 PHE A 180      25.537  -3.352  15.702  1.00 92.55           C  \nATOM    687  CE2 PHE A 180      27.907  -3.448  16.073  1.00 92.59           C  \nATOM    688  CZ  PHE A 180      26.630  -3.934  16.325  1.00 92.00           C  \nATOM    689  N   ASP A 181      29.990  -0.632  11.890  1.00106.28           N  \nATOM    690  CA  ASP A 181      31.390  -1.026  11.812  1.00109.55           C  \nATOM    691  C   ASP A 181      31.929  -1.600  13.114  1.00111.67           C  \nATOM    692  O   ASP A 181      32.075  -2.816  13.239  1.00113.62           O  \nATOM    693  CB  ASP A 181      32.245   0.155  11.353  1.00108.93           C  \nATOM    694  CG  ASP A 181      31.798   0.703  10.014  1.00109.39           C  \nATOM    695  OD1 ASP A 181      30.711   1.317   9.957  1.00109.92           O  \nATOM    696  OD2 ASP A 181      32.525   0.513   9.018  1.00110.18           O  \nATOM    697  N   LYS A 182      32.217  -0.738  14.083  1.00112.86           N  \nATOM    698  CA  LYS A 182      32.750  -1.197  15.357  1.00114.94           C  \nATOM    699  C   LYS A 182      34.154  -1.740  15.131  1.00117.43           C  \nATOM    700  O   LYS A 182      34.435  -2.417  14.133  1.00118.66           O  \nATOM    701  CB  LYS A 182      31.815  -2.245  15.972  1.00113.75           C  \nATOM    702  CG  LYS A 182      32.337  -2.940  17.217  1.00112.74           C  \nATOM    703  CD  LYS A 182      33.067  -4.228  16.876  1.00110.35           C  \nATOM    704  CE  LYS A 182      33.439  -4.986  18.142  1.00109.32           C  \nATOM    705  NZ  LYS A 182      34.011  -6.332  17.860  1.00108.32           N  \nATOM    706  N   LYS A 183      35.036  -1.303  16.029  1.00120.58           N  \nATOM    707  CA  LYS A 183      36.453  -1.636  16.020  1.00124.47           C  \nATOM    708  C   LYS A 183      36.899  -1.963  17.446  1.00127.39           C  \nATOM    709  O   LYS A 183      36.639  -1.198  18.377  1.00127.42           O  \nATOM    710  CB  LYS A 183      37.258  -0.440  15.506  1.00123.80           C  \nATOM    711  CG  LYS A 183      36.640   0.264  14.307  1.00123.71           C  \nATOM    712  CD  LYS A 183      37.426   1.512  13.915  1.00123.09           C  \nATOM    713  CE  LYS A 183      38.834   1.163  13.447  1.00123.14           C  \nATOM    714  NZ  LYS A 183      39.577   2.358  12.957  1.00122.71           N  \nATOM    715  N   ARG A 184      37.586  -3.092  17.601  1.00130.87           N  \nATOM    716  CA  ARG A 184      38.062  -3.540  18.905  1.00134.01           C  \nATOM    717  C   ARG A 184      39.308  -2.779  19.350  1.00136.31           C  \nATOM    718  O   ARG A 184      39.485  -1.605  19.021  1.00136.51           O  \nATOM    719  CB  ARG A 184      38.378  -5.034  18.851  1.00133.86           C  \nATOM    720  CG  ARG A 184      38.192  -5.756  20.171  1.00134.57           C  \nATOM    721  CD  ARG A 184      36.773  -6.273  20.308  1.00134.27           C  \nATOM    722  NE  ARG A 184      36.558  -6.930  21.592  1.00134.98           N  \nATOM    723  CZ  ARG A 184      35.531  -7.730  21.856  1.00135.26           C  \nATOM    724  NH1 ARG A 184      34.624  -7.974  20.921  1.00134.87           N  \nATOM    725  NH2 ARG A 184      35.416  -8.286  23.053  1.00135.78           N  \nATOM    726  N   ASN A 185      40.168  -3.463  20.102  1.00139.51           N  \nATOM    727  CA  ASN A 185      41.404  -2.870  20.601  1.00142.29           C  \nATOM    728  C   ASN A 185      42.558  -3.870  20.560  1.00143.36           C  \nATOM    729  O   ASN A 185      42.428  -4.992  20.065  1.00143.63           O  \nATOM    730  CB  ASN A 185      41.237  -2.406  22.053  1.00143.29           C  \nATOM    731  CG  ASN A 185      39.872  -1.817  22.333  1.00144.05           C  \nATOM    732  OD1 ASN A 185      39.465  -0.831  21.717  1.00144.47           O  \nATOM    733  ND2 ASN A 185      39.157  -2.420  23.277  1.00144.15           N  \nATOM    734  N   GLU A 186      43.691  -3.432  21.096  1.00144.14           N  \nATOM    735  CA  GLU A 186      44.900  -4.237  21.199  1.00144.29           C  \nATOM    736  C   GLU A 186      45.206  -4.096  22.686  1.00144.56           C  \nATOM    737  O   GLU A 186      45.929  -4.892  23.287  1.00144.27           O  \nATOM    738  CB  GLU A 186      46.023  -3.615  20.368  1.00143.83           C  \nATOM    739  CG  GLU A 186      47.242  -4.499  20.194  1.00143.44           C  \nATOM    740  CD  GLU A 186      46.991  -5.649  19.242  1.00143.45           C  \nATOM    741  OE1 GLU A 186      46.652  -5.384  18.070  1.00143.96           O  \nATOM    742  OE2 GLU A 186      47.133  -6.816  19.661  1.00143.36           O  \nATOM    743  N   HIS A 187      44.615  -3.047  23.251  1.00144.74           N  \nATOM    744  CA  HIS A 187      44.731  -2.688  24.655  1.00144.60           C  \nATOM    745  C   HIS A 187      43.801  -1.489  24.869  1.00145.03           C  \nATOM    746  O   HIS A 187      42.586  -1.619  24.716  1.00144.72           O  \nATOM    747  CB  HIS A 187      46.186  -2.336  25.003  1.00144.05           C  \nATOM    748  CG  HIS A 187      46.769  -1.228  24.180  1.00143.15           C  \nATOM    749  ND1 HIS A 187      46.829  -1.270  22.804  1.00143.03           N  \nATOM    750  CD2 HIS A 187      47.346  -0.058  24.545  1.00142.76           C  \nATOM    751  CE1 HIS A 187      47.419  -0.176  22.357  1.00142.41           C  \nATOM    752  NE2 HIS A 187      47.743   0.576  23.393  1.00142.29           N  \nATOM    753  N   ASP A 188      44.372  -0.337  25.213  1.00145.52           N  \nATOM    754  CA  ASP A 188      43.629   0.908  25.428  1.00145.90           C  \nATOM    755  C   ASP A 188      42.395   0.838  26.331  1.00146.05           C  \nATOM    756  O   ASP A 188      41.845   1.872  26.710  1.00146.27           O  \nATOM    757  CB  ASP A 188      43.234   1.523  24.075  1.00145.69           C  \nATOM    758  CG  ASP A 188      42.321   0.621  23.261  1.00145.55           C  \nATOM    759  OD1 ASP A 188      41.196   0.338  23.722  1.00145.45           O  \nATOM    760  OD2 ASP A 188      42.731   0.195  22.159  1.00144.83           O  \nATOM    761  N   GLU A 189      41.962  -0.372  26.673  1.00145.96           N  \nATOM    762  CA  GLU A 189      40.797  -0.565  27.533  1.00145.68           C  \nATOM    763  C   GLU A 189      39.540   0.155  27.053  1.00145.59           C  \nATOM    764  O   GLU A 189      38.513   0.134  27.734  1.00145.79           O  \nATOM    765  CB  GLU A 189      41.123  -0.127  28.964  1.00145.07           C  \nATOM    766  CG  GLU A 189      41.905  -1.157  29.767  1.00144.98           C  \nATOM    767  CD  GLU A 189      43.033  -1.786  28.973  1.00144.75           C  \nATOM    768  OE1 GLU A 189      42.749  -2.601  28.070  1.00144.44           O  \nATOM    769  OE2 GLU A 189      44.205  -1.461  29.245  1.00144.99           O  \nATOM    770  N   ARG A 190      39.615   0.784  25.883  1.00145.13           N  \nATOM    771  CA  ARG A 190      38.465   1.494  25.334  1.00144.04           C  \nATOM    772  C   ARG A 190      37.384   0.470  25.010  1.00143.54           C  \nATOM    773  O   ARG A 190      36.283   0.822  24.586  1.00143.68           O  \nATOM    774  CB  ARG A 190      38.861   2.254  24.065  1.00143.51           C  \nATOM    775  CG  ARG A 190      37.848   3.301  23.621  1.00142.48           C  \nATOM    776  CD  ARG A 190      37.710   4.401  24.665  1.00142.19           C  \nATOM    777  NE  ARG A 190      36.855   5.496  24.215  1.00142.11           N  \nATOM    778  CZ  ARG A 190      36.622   6.599  24.920  1.00142.02           C  \nATOM    779  NH1 ARG A 190      37.179   6.758  26.113  1.00142.35           N  \nATOM    780  NH2 ARG A 190      35.839   7.548  24.429  1.00141.66           N  \nATOM    781  N   THR A 191      37.724  -0.800  25.220  1.00142.28           N  \nATOM    782  CA  THR A 191      36.831  -1.929  24.976  1.00140.62           C  \nATOM    783  C   THR A 191      35.749  -1.676  23.924  1.00138.75           C  \nATOM    784  O   THR A 191      34.566  -1.549  24.245  1.00139.04           O  \nATOM    785  CB  THR A 191      36.161  -2.398  26.293  1.00141.11           C  \nATOM    786  OG1 THR A 191      37.175  -2.739  27.247  1.00141.21           O  \nATOM    787  CG2 THR A 191      35.284  -3.622  26.048  1.00140.66           C  \nATOM    788  N   VAL A 192      36.175  -1.589  22.667  1.00135.30           N  \nATOM    789  CA  VAL A 192      35.268  -1.393  21.541  1.00130.22           C  \nATOM    790  C   VAL A 192      34.488  -0.081  21.499  1.00126.65           C  \nATOM    791  O   VAL A 192      34.121   0.482  22.529  1.00125.81           O  \nATOM    792  CB  VAL A 192      34.247  -2.547  21.469  1.00130.70           C  \nATOM    793  CG1 VAL A 192      33.393  -2.411  20.225  1.00130.56           C  \nATOM    794  CG2 VAL A 192      34.968  -3.880  21.490  1.00130.52           C  \nATOM    795  N   LEU A 193      34.242   0.387  20.279  1.00122.16           N  \nATOM    796  CA  LEU A 193      33.479   1.603  20.023  1.00118.33           C  \nATOM    797  C   LEU A 193      32.730   1.389  18.713  1.00114.58           C  \nATOM    798  O   LEU A 193      33.327   1.345  17.636  1.00114.18           O  \nATOM    799  CB  LEU A 193      34.398   2.825  19.920  1.00119.90           C  \nATOM    800  CG  LEU A 193      34.940   3.389  21.242  1.00120.90           C  \nATOM    801  CD1 LEU A 193      35.850   4.574  20.955  1.00121.54           C  \nATOM    802  CD2 LEU A 193      33.784   3.811  22.145  1.00120.70           C  \nATOM    803  N   ILE A 194      31.414   1.253  18.826  1.00109.62           N  \nATOM    804  CA  ILE A 194      30.546   1.005  17.685  1.00104.37           C  \nATOM    805  C   ILE A 194      30.249   2.230  16.830  1.00101.80           C  \nATOM    806  O   ILE A 194      29.848   3.280  17.332  1.00100.31           O  \nATOM    807  CB  ILE A 194      29.233   0.384  18.169  1.00104.11           C  \nATOM    808  CG1 ILE A 194      29.535  -0.959  18.839  1.00102.95           C  \nATOM    809  CG2 ILE A 194      28.267   0.229  17.009  1.00103.81           C  \nATOM    810  CD1 ILE A 194      28.371  -1.555  19.581  1.00102.98           C  \nATOM    811  N   LEU A 195      30.442   2.076  15.525  1.00 98.73           N  \nATOM    812  CA  LEU A 195      30.208   3.160  14.584  1.00 97.25           C  \nATOM    813  C   LEU A 195      29.172   2.776  13.540  1.00 97.22           C  \nATOM    814  O   LEU A 195      29.283   1.737  12.890  1.00 97.04           O  \nATOM    815  CB  LEU A 195      31.510   3.539  13.874  1.00 96.34           C  \nATOM    816  CG  LEU A 195      32.699   3.980  14.726  1.00 94.81           C  \nATOM    817  CD1 LEU A 195      33.896   4.188  13.824  1.00 95.47           C  \nATOM    818  CD2 LEU A 195      32.366   5.257  15.474  1.00 94.44           C  \nATOM    819  N   VAL A 196      28.167   3.628  13.388  1.00 97.40           N  \nATOM    820  CA  VAL A 196      27.110   3.407  12.414  1.00 97.85           C  \nATOM    821  C   VAL A 196      27.025   4.653  11.535  1.00 97.64           C  \nATOM    822  O   VAL A 196      26.465   5.676  11.935  1.00 98.17           O  \nATOM    823  CB  VAL A 196      25.753   3.137  13.116  1.00 98.11           C  \nATOM    824  CG1 VAL A 196      25.487   4.199  14.157  1.00 97.59           C  \nATOM    825  CG2 VAL A 196      24.628   3.092  12.089  1.00 98.10           C  \nATOM    826  N   ASN A 197      27.603   4.559  10.341  1.00 96.70           N  \nATOM    827  CA  ASN A 197      27.628   5.673   9.400  1.00 96.92           C  \nATOM    828  C   ASN A 197      26.267   6.323   9.163  1.00 97.49           C  \nATOM    829  O   ASN A 197      25.779   7.070  10.007  1.00 97.87           O  \nATOM    830  CB  ASN A 197      28.239   5.217   8.069  1.00 96.09           C  \nATOM    831  CG  ASN A 197      27.586   3.966   7.519  1.00 95.32           C  \nATOM    832  OD1 ASN A 197      28.035   3.410   6.517  1.00 93.75           O  \nATOM    833  ND2 ASN A 197      26.520   3.517   8.171  1.00 95.32           N  \nATOM    834  N   ALA A 198      25.666   6.051   8.009  1.00 98.19           N  \nATOM    835  CA  ALA A 198      24.363   6.612   7.673  1.00 98.58           C  \nATOM    836  C   ALA A 198      23.576   5.596   6.864  1.00 98.35           C  \nATOM    837  O   ALA A 198      22.354   5.503   6.974  1.00 98.30           O  \nATOM    838  CB  ALA A 198      24.535   7.895   6.874  1.00 98.85           C  \nATOM    839  N   GLN A 199      24.292   4.836   6.047  1.00 98.35           N  \nATOM    840  CA  GLN A 199      23.680   3.813   5.217  1.00 98.78           C  \nATOM    841  C   GLN A 199      23.317   2.626   6.095  1.00 96.83           C  \nATOM    842  O   GLN A 199      22.236   2.052   5.964  1.00 96.21           O  \nATOM    843  CB  GLN A 199      24.659   3.390   4.126  1.00102.48           C  \nATOM    844  CG  GLN A 199      25.125   4.550   3.261  1.00107.67           C  \nATOM    845  CD  GLN A 199      26.352   4.212   2.440  1.00110.83           C  \nATOM    846  OE1 GLN A 199      27.408   3.886   2.987  1.00113.01           O  \nATOM    847  NE2 GLN A 199      26.221   4.290   1.119  1.00112.70           N  \nATOM    848  N   GLN A 200      24.227   2.265   6.994  1.00 94.71           N  \nATOM    849  CA  GLN A 200      23.995   1.156   7.908  1.00 92.84           C  \nATOM    850  C   GLN A 200      22.766   1.473   8.745  1.00 91.50           C  \nATOM    851  O   GLN A 200      21.961   0.591   9.043  1.00 91.55           O  \nATOM    852  CB  GLN A 200      25.204   0.960   8.819  1.00 93.26           C  \nATOM    853  CG  GLN A 200      26.464   0.565   8.084  1.00 94.22           C  \nATOM    854  CD  GLN A 200      27.673   0.532   8.993  1.00 95.07           C  \nATOM    855  OE1 GLN A 200      27.997   1.525   9.649  1.00 94.64           O  \nATOM    856  NE2 GLN A 200      28.351  -0.609   9.036  1.00 95.60           N  \nATOM    857  N   ARG A 201      22.628   2.740   9.121  1.00 89.18           N  \nATOM    858  CA  ARG A 201      21.486   3.180   9.907  1.00 87.40           C  \nATOM    859  C   ARG A 201      20.199   2.858   9.169  1.00 86.59           C  \nATOM    860  O   ARG A 201      19.240   2.362   9.759  1.00 87.10           O  \nATOM    861  CB  ARG A 201      21.559   4.682  10.164  1.00 87.11           C  \nATOM    862  CG  ARG A 201      20.233   5.284  10.592  1.00 88.87           C  \nATOM    863  CD  ARG A 201      20.420   6.553  11.397  1.00 91.76           C  \nATOM    864  NE  ARG A 201      20.466   6.294  12.835  1.00 94.60           N  \nATOM    865  CZ  ARG A 201      21.418   5.601  13.458  1.00 95.74           C  \nATOM    866  NH1 ARG A 201      22.430   5.082  12.774  1.00 96.42           N  \nATOM    867  NH2 ARG A 201      21.356   5.425  14.773  1.00 94.93           N  \nATOM    868  N   LYS A 202      20.184   3.151   7.873  1.00 85.36           N  \nATOM    869  CA  LYS A 202      19.021   2.889   7.039  1.00 83.24           C  \nATOM    870  C   LYS A 202      18.745   1.392   7.008  1.00 80.08           C  \nATOM    871  O   LYS A 202      17.597   0.965   6.919  1.00 77.81           O  \nATOM    872  CB  LYS A 202      19.270   3.402   5.619  1.00 86.85           C  \nATOM    873  CG  LYS A 202      19.524   4.905   5.536  1.00 89.64           C  \nATOM    874  CD  LYS A 202      19.751   5.372   4.095  1.00 91.66           C  \nATOM    875  CE  LYS A 202      21.011   4.755   3.483  1.00 91.65           C  \nATOM    876  NZ  LYS A 202      21.315   5.291   2.121  1.00 91.06           N  \nATOM    877  N   LYS A 203      19.810   0.602   7.077  1.00 76.81           N  \nATOM    878  CA  LYS A 203      19.697  -0.851   7.073  1.00 74.57           C  \nATOM    879  C   LYS A 203      19.045  -1.286   8.384  1.00 73.31           C  \nATOM    880  O   LYS A 203      18.129  -2.114   8.390  1.00 73.62           O  \nATOM    881  CB  LYS A 203      21.088  -1.485   6.937  1.00 74.85           C  \nATOM    882  CG  LYS A 203      21.114  -3.009   6.968  1.00 76.24           C  \nATOM    883  CD  LYS A 203      22.547  -3.527   6.881  1.00 78.17           C  \nATOM    884  CE  LYS A 203      22.635  -5.046   6.971  1.00 78.36           C  \nATOM    885  NZ  LYS A 203      21.961  -5.735   5.837  1.00 78.54           N  \nATOM    886  N   ILE A 204      19.527  -0.719   9.490  1.00 70.47           N  \nATOM    887  CA  ILE A 204      18.997  -1.022  10.816  1.00 67.42           C  \nATOM    888  C   ILE A 204      17.505  -0.683  10.900  1.00 65.55           C  \nATOM    889  O   ILE A 204      16.677  -1.552  11.165  1.00 62.77           O  \nATOM    890  CB  ILE A 204      19.760  -0.232  11.916  1.00 67.88           C  \nATOM    891  CG1 ILE A 204      21.159  -0.825  12.101  1.00 68.27           C  \nATOM    892  CG2 ILE A 204      18.988  -0.268  13.241  1.00 66.97           C  \nATOM    893  CD1 ILE A 204      21.997  -0.103  13.140  1.00 68.14           C  \nATOM    894  N   GLU A 205      17.178   0.585  10.667  1.00 64.74           N  \nATOM    895  CA  GLU A 205      15.802   1.051  10.714  1.00 65.49           C  \nATOM    896  C   GLU A 205      14.880   0.238   9.818  1.00 64.58           C  \nATOM    897  O   GLU A 205      13.767  -0.112  10.210  1.00 65.68           O  \nATOM    898  CB  GLU A 205      15.743   2.523  10.326  1.00 67.57           C  \nATOM    899  CG  GLU A 205      16.583   3.395  11.245  1.00 72.19           C  \nATOM    900  CD  GLU A 205      16.264   4.873  11.119  1.00 74.66           C  \nATOM    901  OE1 GLU A 205      16.338   5.405   9.987  1.00 76.95           O  \nATOM    902  OE2 GLU A 205      15.944   5.498  12.156  1.00 75.17           O  \nATOM    903  N   SER A 206      15.355  -0.063   8.617  1.00 63.15           N  \nATOM    904  CA  SER A 206      14.602  -0.848   7.648  1.00 61.07           C  \nATOM    905  C   SER A 206      14.218  -2.199   8.237  1.00 58.59           C  \nATOM    906  O   SER A 206      13.075  -2.654   8.119  1.00 58.09           O  \nATOM    907  CB  SER A 206      15.456  -1.053   6.394  1.00 61.34           C  \nATOM    908  OG  SER A 206      14.976  -2.129   5.611  1.00 66.37           O  \nATOM    909  N   LEU A 207      15.189  -2.837   8.873  1.00 55.17           N  \nATOM    910  CA  LEU A 207      14.977  -4.138   9.482  1.00 51.73           C  \nATOM    911  C   LEU A 207      14.024  -4.025  10.672  1.00 51.07           C  \nATOM    912  O   LEU A 207      13.070  -4.787  10.789  1.00 49.53           O  \nATOM    913  CB  LEU A 207      16.333  -4.725   9.899  1.00 49.87           C  \nATOM    914  CG  LEU A 207      16.426  -5.998  10.740  1.00 49.30           C  \nATOM    915  CD1 LEU A 207      16.190  -5.646  12.198  1.00 50.54           C  \nATOM    916  CD2 LEU A 207      15.412  -7.036  10.253  1.00 49.39           C  \nATOM    917  N   LEU A 208      14.270  -3.060  11.547  1.00 50.88           N  \nATOM    918  CA  LEU A 208      13.414  -2.882  12.716  1.00 50.04           C  \nATOM    919  C   LEU A 208      11.998  -2.519  12.269  1.00 49.86           C  \nATOM    920  O   LEU A 208      11.024  -2.798  12.957  1.00 48.91           O  \nATOM    921  CB  LEU A 208      13.980  -1.792  13.640  1.00 48.13           C  \nATOM    922  CG  LEU A 208      15.260  -2.114  14.427  1.00 45.35           C  \nATOM    923  CD1 LEU A 208      15.520  -1.011  15.456  1.00 42.05           C  \nATOM    924  CD2 LEU A 208      15.118  -3.461  15.128  1.00 42.90           C  \nATOM    925  N   SER A 209      11.907  -1.903  11.093  1.00 50.52           N  \nATOM    926  CA  SER A 209      10.635  -1.499  10.530  1.00 50.23           C  \nATOM    927  C   SER A 209       9.871  -2.744  10.132  1.00 50.00           C  \nATOM    928  O   SER A 209       8.657  -2.824  10.302  1.00 50.44           O  \nATOM    929  CB  SER A 209      10.868  -0.620   9.315  1.00 51.84           C  \nATOM    930  OG  SER A 209       9.816   0.314   9.164  1.00 57.28           O  \nATOM    931  N   ARG A 210      10.587  -3.723   9.598  1.00 50.45           N  \nATOM    932  CA  ARG A 210       9.963  -4.978   9.200  1.00 50.00           C  \nATOM    933  C   ARG A 210       9.632  -5.813  10.430  1.00 47.38           C  \nATOM    934  O   ARG A 210       8.720  -6.635  10.404  1.00 44.28           O  \nATOM    935  CB  ARG A 210      10.898  -5.755   8.278  1.00 53.70           C  \nATOM    936  CG  ARG A 210      11.100  -5.089   6.918  1.00 56.72           C  \nATOM    937  CD  ARG A 210      12.237  -5.747   6.196  1.00 58.88           C  \nATOM    938  NE  ARG A 210      12.112  -7.199   6.251  1.00 59.33           N  \nATOM    939  CZ  ARG A 210      13.146  -8.024   6.352  1.00 60.11           C  \nATOM    940  NH1 ARG A 210      14.378  -7.526   6.410  1.00 58.93           N  \nATOM    941  NH2 ARG A 210      12.948  -9.338   6.397  1.00 59.23           N  \nATOM    942  N   VAL A 211      10.380  -5.613  11.510  1.00 46.28           N  \nATOM    943  CA  VAL A 211      10.104  -6.362  12.730  1.00 44.96           C  \nATOM    944  C   VAL A 211       8.818  -5.818  13.339  1.00 43.27           C  \nATOM    945  O   VAL A 211       8.027  -6.571  13.907  1.00 41.03           O  \nATOM    946  CB  VAL A 211      11.281  -6.263  13.737  1.00 44.56           C  \nATOM    947  CG1 VAL A 211      10.845  -6.755  15.113  1.00 42.82           C  \nATOM    948  CG2 VAL A 211      12.453  -7.120  13.240  1.00 43.33           C  \nATOM    949  N   ASN A 212       8.603  -4.512  13.185  1.00 43.25           N  \nATOM    950  CA  ASN A 212       7.402  -3.848  13.684  1.00 42.16           C  \nATOM    951  C   ASN A 212       6.138  -4.389  13.011  1.00 41.19           C  \nATOM    952  O   ASN A 212       5.128  -4.597  13.672  1.00 39.03           O  \nATOM    953  CB  ASN A 212       7.497  -2.347  13.452  1.00 42.76           C  \nATOM    954  CG  ASN A 212       8.325  -1.649  14.492  1.00 43.11           C  \nATOM    955  OD1 ASN A 212       8.699  -0.493  14.319  1.00 46.21           O  \nATOM    956  ND2 ASN A 212       8.614  -2.340  15.591  1.00 42.08           N  \nATOM    957  N   LYS A 213       6.199  -4.615  11.698  1.00 40.65           N  \nATOM    958  CA  LYS A 213       5.056  -5.159  10.973  1.00 41.42           C  \nATOM    959  C   LYS A 213       4.778  -6.612  11.366  1.00 43.09           C  \nATOM    960  O   LYS A 213       3.619  -7.034  11.375  1.00 44.01           O  \nATOM    961  CB  LYS A 213       5.263  -5.057   9.457  1.00 41.67           C  \nATOM    962  CG  LYS A 213       5.157  -3.646   8.902  1.00 41.02           C  \nATOM    963  CD  LYS A 213       5.657  -3.598   7.489  1.00 42.57           C  \nATOM    964  CE  LYS A 213       5.960  -2.172   7.032  1.00 44.37           C  \nATOM    965  NZ  LYS A 213       6.481  -2.143   5.624  1.00 47.34           N  \nATOM    966  N   ARG A 214       5.813  -7.382  11.697  1.00 45.26           N  \nATOM    967  CA  ARG A 214       5.568  -8.763  12.114  1.00 47.24           C  \nATOM    968  C   ARG A 214       4.881  -8.794  13.469  1.00 45.55           C  \nATOM    969  O   ARG A 214       4.034  -9.648  13.715  1.00 44.03           O  \nATOM    970  CB  ARG A 214       6.866  -9.584  12.175  1.00 50.21           C  \nATOM    971  CG  ARG A 214       7.040 -10.502  10.962  1.00 55.21           C  \nATOM    972  CD  ARG A 214       6.429 -11.895  11.146  1.00 56.93           C  \nATOM    973  NE  ARG A 214       5.162 -11.878  11.866  1.00 59.51           N  \nATOM    974  CZ  ARG A 214       4.458 -12.966  12.171  1.00 60.64           C  \nATOM    975  NH1 ARG A 214       4.891 -14.164  11.808  1.00 61.59           N  \nATOM    976  NH2 ARG A 214       3.335 -12.860  12.872  1.00 62.12           N  \nATOM    977  N   ILE A 215       5.258  -7.866  14.342  1.00 44.12           N  \nATOM    978  CA  ILE A 215       4.656  -7.768  15.671  1.00 44.64           C  \nATOM    979  C   ILE A 215       3.142  -7.552  15.545  1.00 44.77           C  \nATOM    980  O   ILE A 215       2.353  -8.181  16.255  1.00 45.72           O  \nATOM    981  CB  ILE A 215       5.296  -6.590  16.493  1.00 43.53           C  \nATOM    982  CG1 ILE A 215       6.665  -7.030  17.037  1.00 42.41           C  \nATOM    983  CG2 ILE A 215       4.361  -6.140  17.606  1.00 43.54           C  \nATOM    984  CD1 ILE A 215       7.481  -5.922  17.666  1.00 38.70           C  \nATOM    985  N   THR A 216       2.750  -6.676  14.623  1.00 45.34           N  \nATOM    986  CA  THR A 216       1.347  -6.371  14.391  1.00 45.53           C  \nATOM    987  C   THR A 216       0.580  -7.592  13.885  1.00 45.36           C  \nATOM    988  O   THR A 216      -0.521  -7.899  14.369  1.00 44.31           O  \nATOM    989  CB  THR A 216       1.214  -5.202  13.400  1.00 46.33           C  \nATOM    990  OG1 THR A 216       1.727  -4.014  14.024  1.00 48.54           O  \nATOM    991  CG2 THR A 216      -0.229  -4.976  13.021  1.00 47.26           C  \nATOM    992  N   GLU A 217       1.160  -8.288  12.913  1.00 46.93           N  \nATOM    993  CA  GLU A 217       0.534  -9.497  12.399  1.00 49.54           C  \nATOM    994  C   GLU A 217       0.437 -10.497  13.548  1.00 50.12           C  \nATOM    995  O   GLU A 217      -0.551 -11.222  13.671  1.00 49.63           O  \nATOM    996  CB  GLU A 217       1.363 -10.091  11.263  1.00 52.00           C  \nATOM    997  CG  GLU A 217       1.305 -11.620  11.190  1.00 54.64           C  \nATOM    998  CD  GLU A 217       2.035 -12.192   9.986  1.00 55.92           C  \nATOM    999  OE1 GLU A 217       3.176 -11.748   9.689  1.00 55.58           O  \nATOM   1000  OE2 GLU A 217       1.458 -13.096   9.343  1.00 56.01           O  \nATOM   1001  N   ALA A 218       1.470 -10.525  14.386  1.00 51.12           N  \nATOM   1002  CA  ALA A 218       1.512 -11.421  15.539  1.00 51.99           C  \nATOM   1003  C   ALA A 218       0.434 -11.082  16.559  1.00 54.26           C  \nATOM   1004  O   ALA A 218      -0.165 -11.980  17.140  1.00 55.71           O  \nATOM   1005  CB  ALA A 218       2.880 -11.366  16.205  1.00 52.22           C  \nATOM   1006  N   ASN A 219       0.185  -9.794  16.786  1.00 55.54           N  \nATOM   1007  CA  ASN A 219      -0.844  -9.420  17.746  1.00 57.40           C  \nATOM   1008  C   ASN A 219      -2.239  -9.783  17.246  1.00 61.41           C  \nATOM   1009  O   ASN A 219      -3.197  -9.760  18.012  1.00 63.81           O  \nATOM   1010  CB  ASN A 219      -0.784  -7.927  18.072  1.00 54.06           C  \nATOM   1011  CG  ASN A 219       0.419  -7.561  18.934  1.00 51.43           C  \nATOM   1012  OD1 ASN A 219       0.878  -8.367  19.746  1.00 48.70           O  \nATOM   1013  ND2 ASN A 219       0.921  -6.333  18.774  1.00 49.40           N  \nATOM   1014  N   ASN A 220      -2.359 -10.123  15.965  1.00 65.09           N  \nATOM   1015  CA  ASN A 220      -3.659 -10.495  15.418  1.00 69.21           C  \nATOM   1016  C   ASN A 220      -3.867 -12.009  15.488  1.00 72.26           C  \nATOM   1017  O   ASN A 220      -5.006 -12.484  15.551  1.00 72.52           O  \nATOM   1018  CB  ASN A 220      -3.807  -9.990  13.977  1.00 69.58           C  \nATOM   1019  CG  ASN A 220      -3.603  -8.477  13.862  1.00 72.15           C  \nATOM   1020  OD1 ASN A 220      -4.190  -7.690  14.618  1.00 72.10           O  \nATOM   1021  ND2 ASN A 220      -2.773  -8.065  12.908  1.00 73.81           N  \nATOM   1022  N   GLU A 221      -2.770 -12.763  15.478  1.00 76.04           N  \nATOM   1023  CA  GLU A 221      -2.852 -14.220  15.580  1.00 81.76           C  \nATOM   1024  C   GLU A 221      -2.835 -14.612  17.053  1.00 84.93           C  \nATOM   1025  O   GLU A 221      -3.792 -15.204  17.548  1.00 87.20           O  \nATOM   1026  CB  GLU A 221      -1.678 -14.901  14.861  1.00 82.05           C  \nATOM   1027  CG  GLU A 221      -1.924 -15.228  13.381  1.00 86.56           C  \nATOM   1028  CD  GLU A 221      -2.829 -16.444  13.165  1.00 88.01           C  \nATOM   1029  OE1 GLU A 221      -3.104 -16.783  11.988  1.00 86.31           O  \nATOM   1030  OE2 GLU A 221      -3.259 -17.061  14.168  1.00 89.54           O  \nATOM   1031  N   ILE A 222      -1.746 -14.273  17.745  1.00 88.11           N  \nATOM   1032  CA  ILE A 222      -1.585 -14.583  19.168  1.00 90.27           C  \nATOM   1033  C   ILE A 222      -2.924 -14.599  19.895  1.00 93.03           C  \nATOM   1034  O   ILE A 222      -3.516 -15.660  20.101  1.00 94.48           O  \nATOM   1035  CB  ILE A 222      -0.660 -13.554  19.878  1.00 88.64           C  \nATOM   1036  CG1 ILE A 222       0.786 -13.728  19.410  1.00 87.83           C  \nATOM   1037  CG2 ILE A 222      -0.732 -13.736  21.386  1.00 89.00           C  \nATOM   1038  CD1 ILE A 222       1.750 -12.725  20.021  1.00 86.43           C  \nATOM   1039  N   GLU A 223      -3.390 -13.416  20.282  1.00 95.68           N  \nATOM   1040  CA  GLU A 223      -4.659 -13.272  20.981  1.00 99.57           C  \nATOM   1041  C   GLU A 223      -4.984 -14.457  21.898  1.00101.71           C  \nATOM   1042  O   GLU A 223      -5.807 -15.310  21.563  1.00102.28           O  \nATOM   1043  CB  GLU A 223      -5.777 -13.064  19.959  1.00 99.27           C  \nATOM   1044  CG  GLU A 223      -5.531 -11.874  19.038  1.00100.15           C  \nATOM   1045  CD  GLU A 223      -6.695 -11.592  18.108  1.00101.65           C  \nATOM   1046  OE1 GLU A 223      -6.996 -12.440  17.237  1.00101.89           O  \nATOM   1047  OE2 GLU A 223      -7.311 -10.515  18.254  1.00102.29           O  \nATOM   1048  N   LEU A 224      -4.325 -14.501  23.054  1.00103.33           N  \nATOM   1049  CA  LEU A 224      -4.539 -15.563  24.033  1.00104.31           C  \nATOM   1050  C   LEU A 224      -5.296 -14.977  25.226  1.00105.67           C  \nATOM   1051  O   LEU A 224      -4.669 -14.793  26.292  1.00106.82           O  \nATOM   1052  CB  LEU A 224      -3.195 -16.145  24.500  1.00103.46           C  \nATOM   1053  CG  LEU A 224      -3.175 -17.527  25.176  1.00103.09           C  \nATOM   1054  CD1 LEU A 224      -1.736 -17.912  25.473  1.00102.25           C  \nATOM   1055  CD2 LEU A 224      -3.993 -17.529  26.458  1.00102.18           C  \nATOM   1056  OXT LEU A 224      -6.504 -14.688  25.076  1.00106.85           O  \nTER    1057      LEU A 224                                                      \nATOM   1058  N   GLY B  98     -12.664 -11.152  32.324  1.00 61.24           N  \nATOM   1059  CA  GLY B  98     -13.537 -11.374  33.515  1.00 63.75           C  \nATOM   1060  C   GLY B  98     -12.818 -11.156  34.838  1.00 63.18           C  \nATOM   1061  O   GLY B  98     -13.292 -10.426  35.705  1.00 66.12           O  \nATOM   1062  N   SER B  99     -11.675 -11.807  35.002  1.00 58.95           N  \nATOM   1063  CA  SER B  99     -10.890 -11.661  36.216  1.00 53.06           C  \nATOM   1064  C   SER B  99      -9.458 -11.387  35.788  1.00 51.84           C  \nATOM   1065  O   SER B  99      -8.972 -11.987  34.828  1.00 56.20           O  \nATOM   1066  CB  SER B  99     -10.960 -12.945  37.064  1.00 50.12           C  \nATOM   1067  OG  SER B  99     -10.556 -14.085  36.323  1.00 47.26           O  \nATOM   1068  N   HIS B 100      -8.791 -10.479  36.486  1.00 46.07           N  \nATOM   1069  CA  HIS B 100      -7.395 -10.137  36.189  1.00 41.09           C  \nATOM   1070  C   HIS B 100      -7.218  -9.166  35.043  1.00 39.03           C  \nATOM   1071  O   HIS B 100      -7.781  -9.322  33.981  1.00 39.67           O  \nATOM   1072  CB  HIS B 100      -6.558 -11.383  35.899  1.00 39.39           C  \nATOM   1073  CG  HIS B 100      -6.262 -12.207  37.108  1.00 38.00           C  \nATOM   1074  ND1 HIS B 100      -6.579 -13.546  37.198  1.00 37.45           N  \nATOM   1075  CD2 HIS B 100      -5.650 -11.887  38.268  1.00 37.20           C  \nATOM   1076  CE1 HIS B 100      -6.166 -14.009  38.362  1.00 37.21           C  \nATOM   1077  NE2 HIS B 100      -5.598 -13.023  39.030  1.00 36.23           N  \nATOM   1078  N   MET B 101      -6.418  -8.144  35.292  1.00 37.06           N  \nATOM   1079  CA  MET B 101      -6.134  -7.148  34.293  1.00 33.60           C  \nATOM   1080  C   MET B 101      -5.515  -7.815  33.061  1.00 31.83           C  \nATOM   1081  O   MET B 101      -4.542  -8.570  33.180  1.00 30.35           O  \nATOM   1082  CB  MET B 101      -5.162  -6.108  34.859  1.00 33.04           C  \nATOM   1083  CG  MET B 101      -4.728  -5.108  33.811  1.00 31.86           C  \nATOM   1084  SD  MET B 101      -3.432  -4.010  34.306  1.00 33.35           S  \nATOM   1085  CE  MET B 101      -1.939  -5.151  34.224  1.00 32.77           C  \nATOM   1086  N   ALA B 102      -6.093  -7.538  31.891  1.00 29.69           N  \nATOM   1087  CA  ALA B 102      -5.592  -8.069  30.631  1.00 30.76           C  \nATOM   1088  C   ALA B 102      -4.247  -7.419  30.366  1.00 32.04           C  \nATOM   1089  O   ALA B 102      -3.968  -6.342  30.869  1.00 36.40           O  \nATOM   1090  CB  ALA B 102      -6.550  -7.734  29.487  1.00 26.67           C  \nATOM   1091  N   ILE B 103      -3.411  -8.075  29.579  1.00 33.38           N  \nATOM   1092  CA  ILE B 103      -2.120  -7.512  29.247  1.00 33.44           C  \nATOM   1093  C   ILE B 103      -2.298  -6.668  27.993  1.00 33.27           C  \nATOM   1094  O   ILE B 103      -2.897  -7.106  27.020  1.00 32.92           O  \nATOM   1095  CB  ILE B 103      -1.068  -8.621  28.991  1.00 32.98           C  \nATOM   1096  CG1 ILE B 103      -0.879  -9.463  30.264  1.00 31.41           C  \nATOM   1097  CG2 ILE B 103       0.240  -8.010  28.502  1.00 31.31           C  \nATOM   1098  CD1 ILE B 103      -0.595  -8.666  31.524  1.00 27.59           C  \nATOM   1099  N   THR B 104      -1.786  -5.449  28.029  1.00 35.36           N  \nATOM   1100  CA  THR B 104      -1.888  -4.560  26.896  1.00 37.82           C  \nATOM   1101  C   THR B 104      -0.816  -4.953  25.903  1.00 38.71           C  \nATOM   1102  O   THR B 104       0.320  -5.220  26.279  1.00 38.49           O  \nATOM   1103  CB  THR B 104      -1.637  -3.103  27.301  1.00 40.31           C  \nATOM   1104  OG1 THR B 104      -2.528  -2.741  28.361  1.00 44.00           O  \nATOM   1105  CG2 THR B 104      -1.839  -2.174  26.109  1.00 39.21           C  \nATOM   1106  N   LYS B 105      -1.196  -4.973  24.635  1.00 39.49           N  \nATOM   1107  CA  LYS B 105      -0.301  -5.294  23.545  1.00 41.51           C  \nATOM   1108  C   LYS B 105       0.663  -4.140  23.247  1.00 41.84           C  \nATOM   1109  O   LYS B 105       0.344  -2.964  23.447  1.00 44.61           O  \nATOM   1110  CB  LYS B 105      -1.114  -5.599  22.286  1.00 41.71           C  \nATOM   1111  CG  LYS B 105      -1.842  -6.915  22.358  1.00 50.08           C  \nATOM   1112  CD  LYS B 105      -3.359  -6.720  22.408  1.00 56.77           C  \nATOM   1113  CE  LYS B 105      -3.937  -6.318  21.032  1.00 59.79           C  \nATOM   1114  NZ  LYS B 105      -5.375  -5.887  21.111  1.00 62.22           N  \nATOM   1115  N   ILE B 106       1.850  -4.481  22.770  1.00 39.57           N  \nATOM   1116  CA  ILE B 106       2.821  -3.477  22.402  1.00 36.43           C  \nATOM   1117  C   ILE B 106       2.755  -3.471  20.875  1.00 35.60           C  \nATOM   1118  O   ILE B 106       2.200  -4.394  20.275  1.00 35.47           O  \nATOM   1119  CB  ILE B 106       4.260  -3.828  22.947  1.00 35.74           C  \nATOM   1120  CG1 ILE B 106       4.696  -5.249  22.544  1.00 34.61           C  \nATOM   1121  CG2 ILE B 106       4.265  -3.732  24.470  1.00 31.95           C  \nATOM   1122  CD1 ILE B 106       5.188  -5.393  21.154  1.00 32.37           C  \nATOM   1123  N   ASN B 107       3.286  -2.448  20.227  1.00 33.19           N  \nATOM   1124  CA  ASN B 107       3.209  -2.417  18.775  1.00 33.81           C  \nATOM   1125  C   ASN B 107       4.545  -2.001  18.110  1.00 35.31           C  \nATOM   1126  O   ASN B 107       4.597  -1.685  16.923  1.00 37.56           O  \nATOM   1127  CB  ASN B 107       2.081  -1.477  18.340  1.00 33.19           C  \nATOM   1128  CG  ASN B 107       2.477  -0.040  18.477  1.00 31.76           C  \nATOM   1129  OD1 ASN B 107       3.501   0.228  19.066  1.00 32.02           O  \nATOM   1130  ND2 ASN B 107       1.686   0.885  17.943  1.00 31.21           N  \nATOM   1131  N   ASP B 108       5.627  -1.986  18.877  1.00 34.25           N  \nATOM   1132  CA  ASP B 108       6.918  -1.663  18.280  1.00 33.53           C  \nATOM   1133  C   ASP B 108       8.056  -2.251  19.102  1.00 34.36           C  \nATOM   1134  O   ASP B 108       7.850  -2.669  20.249  1.00 32.78           O  \nATOM   1135  CB  ASP B 108       7.075  -0.152  18.057  1.00 31.84           C  \nATOM   1136  CG  ASP B 108       7.284   0.611  19.328  1.00 31.52           C  \nATOM   1137  OD1 ASP B 108       7.029   1.819  19.337  1.00 30.95           O  \nATOM   1138  OD2 ASP B 108       7.726   0.036  20.312  1.00 31.83           O  \nATOM   1139  N   CYS B 109       9.240  -2.298  18.499  1.00 37.24           N  \nATOM   1140  CA  CYS B 109      10.433  -2.911  19.108  1.00 38.34           C  \nATOM   1141  C   CYS B 109      10.832  -2.431  20.483  1.00 38.76           C  \nATOM   1142  O   CYS B 109      11.111  -3.249  21.367  1.00 39.81           O  \nATOM   1143  CB  CYS B 109      11.640  -2.785  18.175  1.00 37.86           C  \nATOM   1144  SG  CYS B 109      11.379  -3.512  16.546  1.00 38.39           S  \nATOM   1145  N   PHE B 110      10.863  -1.118  20.672  1.00 39.60           N  \nATOM   1146  CA  PHE B 110      11.232  -0.583  21.965  1.00 42.18           C  \nATOM   1147  C   PHE B 110      10.268  -1.014  23.073  1.00 43.08           C  \nATOM   1148  O   PHE B 110      10.686  -1.261  24.213  1.00 44.51           O  \nATOM   1149  CB  PHE B 110      11.319   0.940  21.916  1.00 43.98           C  \nATOM   1150  CG  PHE B 110      11.971   1.536  23.127  1.00 46.68           C  \nATOM   1151  CD1 PHE B 110      11.203   2.096  24.151  1.00 45.50           C  \nATOM   1152  CD2 PHE B 110      13.361   1.498  23.269  1.00 47.64           C  \nATOM   1153  CE1 PHE B 110      11.809   2.611  25.301  1.00 46.11           C  \nATOM   1154  CE2 PHE B 110      13.978   2.010  24.412  1.00 46.74           C  \nATOM   1155  CZ  PHE B 110      13.202   2.568  25.431  1.00 47.04           C  \nATOM   1156  N   GLU B 111       8.984  -1.112  22.759  1.00 42.31           N  \nATOM   1157  CA  GLU B 111       8.022  -1.515  23.772  1.00 40.60           C  \nATOM   1158  C   GLU B 111       8.174  -2.986  24.079  1.00 38.79           C  \nATOM   1159  O   GLU B 111       8.005  -3.404  25.214  1.00 37.94           O  \nATOM   1160  CB  GLU B 111       6.618  -1.200  23.296  1.00 41.40           C  \nATOM   1161  CG  GLU B 111       6.570   0.202  22.836  1.00 42.61           C  \nATOM   1162  CD  GLU B 111       5.214   0.670  22.471  1.00 43.12           C  \nATOM   1163  OE1 GLU B 111       4.458  -0.059  21.799  1.00 43.52           O  \nATOM   1164  OE2 GLU B 111       4.908   1.807  22.851  1.00 46.30           O  \nATOM   1165  N   LEU B 112       8.511  -3.761  23.055  1.00 37.54           N  \nATOM   1166  CA  LEU B 112       8.704  -5.187  23.219  1.00 33.74           C  \nATOM   1167  C   LEU B 112       9.873  -5.364  24.176  1.00 33.32           C  \nATOM   1168  O   LEU B 112       9.764  -6.042  25.192  1.00 34.91           O  \nATOM   1169  CB  LEU B 112       9.056  -5.822  21.884  1.00 33.00           C  \nATOM   1170  CG  LEU B 112       8.723  -7.300  21.692  1.00 32.06           C  \nATOM   1171  CD1 LEU B 112       9.767  -7.918  20.780  1.00 31.53           C  \nATOM   1172  CD2 LEU B 112       8.652  -8.027  23.026  1.00 30.35           C  \nATOM   1173  N   LEU B 113      10.989  -4.721  23.849  1.00 32.31           N  \nATOM   1174  CA  LEU B 113      12.188  -4.820  24.651  1.00 31.20           C  \nATOM   1175  C   LEU B 113      11.906  -4.483  26.119  1.00 32.18           C  \nATOM   1176  O   LEU B 113      12.335  -5.214  27.033  1.00 30.15           O  \nATOM   1177  CB  LEU B 113      13.270  -3.904  24.073  1.00 32.11           C  \nATOM   1178  CG  LEU B 113      14.576  -3.830  24.883  1.00 35.66           C  \nATOM   1179  CD1 LEU B 113      15.142  -5.234  25.080  1.00 36.67           C  \nATOM   1180  CD2 LEU B 113      15.589  -2.932  24.169  1.00 35.74           C  \nATOM   1181  N   SER B 114      11.178  -3.393  26.352  1.00 31.44           N  \nATOM   1182  CA  SER B 114      10.859  -2.999  27.712  1.00 32.02           C  \nATOM   1183  C   SER B 114      10.031  -4.068  28.390  1.00 32.18           C  \nATOM   1184  O   SER B 114      10.295  -4.436  29.536  1.00 35.84           O  \nATOM   1185  CB  SER B 114      10.076  -1.700  27.738  1.00 30.73           C  \nATOM   1186  OG  SER B 114      10.805  -0.679  27.096  1.00 32.00           O  \nATOM   1187  N   MET B 115       9.030  -4.578  27.684  1.00 30.28           N  \nATOM   1188  CA  MET B 115       8.176  -5.594  28.276  1.00 29.84           C  \nATOM   1189  C   MET B 115       9.001  -6.796  28.719  1.00 29.27           C  \nATOM   1190  O   MET B 115       8.864  -7.277  29.839  1.00 29.56           O  \nATOM   1191  CB  MET B 115       7.082  -6.019  27.301  1.00 30.08           C  \nATOM   1192  CG  MET B 115       6.114  -7.005  27.912  1.00 32.99           C  \nATOM   1193  SD  MET B 115       4.729  -7.438  26.864  1.00 38.71           S  \nATOM   1194  CE  MET B 115       3.458  -6.305  27.462  1.00 37.80           C  \nATOM   1195  N   VAL B 116       9.877  -7.257  27.843  1.00 30.62           N  \nATOM   1196  CA  VAL B 116      10.737  -8.391  28.140  1.00 30.76           C  \nATOM   1197  C   VAL B 116      11.674  -8.081  29.315  1.00 33.07           C  \nATOM   1198  O   VAL B 116      11.833  -8.888  30.233  1.00 35.37           O  \nATOM   1199  CB  VAL B 116      11.571  -8.759  26.914  1.00 29.70           C  \nATOM   1200  CG1 VAL B 116      12.637  -9.758  27.311  1.00 29.90           C  \nATOM   1201  CG2 VAL B 116      10.640  -9.325  25.813  1.00 27.28           C  \nATOM   1202  N   THR B 117      12.288  -6.901  29.292  1.00 33.75           N  \nATOM   1203  CA  THR B 117      13.186  -6.522  30.375  1.00 34.17           C  \nATOM   1204  C   THR B 117      12.427  -6.597  31.701  1.00 33.98           C  \nATOM   1205  O   THR B 117      12.909  -7.181  32.677  1.00 34.88           O  \nATOM   1206  CB  THR B 117      13.714  -5.086  30.192  1.00 34.71           C  \nATOM   1207  OG1 THR B 117      14.423  -4.989  28.948  1.00 35.00           O  \nATOM   1208  CG2 THR B 117      14.630  -4.704  31.340  1.00 33.90           C  \nATOM   1209  N   TYR B 118      11.238  -6.007  31.740  1.00 32.20           N  \nATOM   1210  CA  TYR B 118      10.438  -6.017  32.951  1.00 31.83           C  \nATOM   1211  C   TYR B 118       9.995  -7.447  33.330  1.00 31.49           C  \nATOM   1212  O   TYR B 118       9.993  -7.832  34.500  1.00 30.74           O  \nATOM   1213  CB  TYR B 118       9.230  -5.091  32.751  1.00 30.78           C  \nATOM   1214  CG  TYR B 118       8.158  -5.182  33.815  1.00 30.23           C  \nATOM   1215  CD1 TYR B 118       8.423  -4.839  35.141  1.00 30.02           C  \nATOM   1216  CD2 TYR B 118       6.871  -5.635  33.489  1.00 29.87           C  \nATOM   1217  CE1 TYR B 118       7.425  -4.947  36.126  1.00 30.64           C  \nATOM   1218  CE2 TYR B 118       5.866  -5.749  34.451  1.00 29.16           C  \nATOM   1219  CZ  TYR B 118       6.141  -5.406  35.770  1.00 30.09           C  \nATOM   1220  OH  TYR B 118       5.132  -5.516  36.710  1.00 28.17           O  \nATOM   1221  N   ALA B 119       9.633  -8.251  32.343  1.00 31.73           N  \nATOM   1222  CA  ALA B 119       9.195  -9.604  32.653  1.00 33.79           C  \nATOM   1223  C   ALA B 119      10.362 -10.469  33.126  1.00 35.43           C  \nATOM   1224  O   ALA B 119      10.202 -11.291  34.038  1.00 34.66           O  \nATOM   1225  CB  ALA B 119       8.510 -10.244  31.426  1.00 34.26           C  \nATOM   1226  N   ASP B 120      11.531 -10.293  32.511  1.00 38.35           N  \nATOM   1227  CA  ASP B 120      12.697 -11.082  32.899  1.00 42.05           C  \nATOM   1228  C   ASP B 120      13.082 -10.792  34.347  1.00 42.26           C  \nATOM   1229  O   ASP B 120      13.410 -11.708  35.103  1.00 40.84           O  \nATOM   1230  CB  ASP B 120      13.896 -10.802  31.989  1.00 46.28           C  \nATOM   1231  CG  ASP B 120      13.788 -11.487  30.629  1.00 48.74           C  \nATOM   1232  OD1 ASP B 120      13.003 -12.452  30.495  1.00 49.75           O  \nATOM   1233  OD2 ASP B 120      14.506 -11.068  29.694  1.00 50.19           O  \nATOM   1234  N   LYS B 121      13.038  -9.521  34.737  1.00 43.31           N  \nATOM   1235  CA  LYS B 121      13.380  -9.164  36.103  1.00 43.62           C  \nATOM   1236  C   LYS B 121      12.442  -9.838  37.094  1.00 42.71           C  \nATOM   1237  O   LYS B 121      12.893 -10.441  38.065  1.00 45.32           O  \nATOM   1238  CB  LYS B 121      13.362  -7.647  36.295  1.00 44.66           C  \nATOM   1239  CG  LYS B 121      14.497  -6.955  35.573  1.00 49.21           C  \nATOM   1240  CD  LYS B 121      14.484  -5.451  35.773  1.00 51.79           C  \nATOM   1241  CE  LYS B 121      15.577  -4.791  34.932  1.00 53.53           C  \nATOM   1242  NZ  LYS B 121      15.369  -3.315  34.821  1.00 55.92           N  \nATOM   1243  N   LEU B 122      11.138  -9.760  36.864  1.00 40.70           N  \nATOM   1244  CA  LEU B 122      10.229 -10.404  37.788  1.00 38.48           C  \nATOM   1245  C   LEU B 122      10.437 -11.919  37.792  1.00 38.89           C  \nATOM   1246  O   LEU B 122      10.333 -12.554  38.836  1.00 37.55           O  \nATOM   1247  CB  LEU B 122       8.781 -10.060  37.449  1.00 38.06           C  \nATOM   1248  CG  LEU B 122       8.355  -8.591  37.600  1.00 37.89           C  \nATOM   1249  CD1 LEU B 122       6.859  -8.469  37.373  1.00 37.43           C  \nATOM   1250  CD2 LEU B 122       8.696  -8.077  38.998  1.00 38.84           C  \nATOM   1251  N   LYS B 123      10.739 -12.506  36.634  1.00 41.30           N  \nATOM   1252  CA  LYS B 123      10.959 -13.952  36.573  1.00 44.06           C  \nATOM   1253  C   LYS B 123      12.177 -14.385  37.382  1.00 47.10           C  \nATOM   1254  O   LYS B 123      12.095 -15.313  38.187  1.00 48.12           O  \nATOM   1255  CB  LYS B 123      11.156 -14.429  35.137  1.00 43.43           C  \nATOM   1256  CG  LYS B 123       9.917 -14.427  34.298  1.00 45.59           C  \nATOM   1257  CD  LYS B 123      10.015 -15.453  33.163  1.00 47.11           C  \nATOM   1258  CE  LYS B 123      11.198 -15.195  32.267  1.00 47.77           C  \nATOM   1259  NZ  LYS B 123      11.242 -16.167  31.135  1.00 51.36           N  \nATOM   1260  N   SER B 124      13.306 -13.715  37.157  1.00 49.52           N  \nATOM   1261  CA  SER B 124      14.541 -14.051  37.849  1.00 51.17           C  \nATOM   1262  C   SER B 124      14.399 -13.882  39.352  1.00 53.09           C  \nATOM   1263  O   SER B 124      14.975 -14.651  40.119  1.00 55.98           O  \nATOM   1264  CB  SER B 124      15.695 -13.184  37.340  1.00 49.53           C  \nATOM   1265  OG  SER B 124      15.557 -11.849  37.789  1.00 48.98           O  \nATOM   1266  N   LEU B 125      13.637 -12.876  39.774  1.00 53.07           N  \nATOM   1267  CA  LEU B 125      13.429 -12.642  41.204  1.00 52.85           C  \nATOM   1268  C   LEU B 125      12.653 -13.798  41.814  1.00 51.39           C  \nATOM   1269  O   LEU B 125      13.076 -14.398  42.800  1.00 51.30           O  \nATOM   1270  CB  LEU B 125      12.666 -11.339  41.439  1.00 54.40           C  \nATOM   1271  CG  LEU B 125      13.564 -10.106  41.554  1.00 58.89           C  \nATOM   1272  CD1 LEU B 125      12.742  -8.821  41.405  1.00 59.75           C  \nATOM   1273  CD2 LEU B 125      14.286 -10.152  42.914  1.00 58.61           C  \nATOM   1274  N   ILE B 126      11.515 -14.113  41.220  1.00 50.77           N  \nATOM   1275  CA  ILE B 126      10.697 -15.201  41.714  1.00 51.91           C  \nATOM   1276  C   ILE B 126      11.533 -16.468  41.813  1.00 54.08           C  \nATOM   1277  O   ILE B 126      11.363 -17.260  42.738  1.00 54.49           O  \nATOM   1278  CB  ILE B 126       9.479 -15.424  40.792  1.00 51.79           C  \nATOM   1279  CG1 ILE B 126       8.507 -14.252  40.963  1.00 50.75           C  \nATOM   1280  CG2 ILE B 126       8.808 -16.755  41.096  1.00 50.54           C  \nATOM   1281  CD1 ILE B 126       7.316 -14.300  40.036  1.00 52.63           C  \nATOM   1282  N   LYS B 127      12.443 -16.658  40.862  1.00 57.22           N  \nATOM   1283  CA  LYS B 127      13.304 -17.835  40.881  1.00 60.96           C  \nATOM   1284  C   LYS B 127      14.376 -17.696  41.959  1.00 62.04           C  \nATOM   1285  O   LYS B 127      14.520 -18.572  42.803  1.00 62.97           O  \nATOM   1286  CB  LYS B 127      13.942 -18.041  39.511  1.00 62.82           C  \nATOM   1287  CG  LYS B 127      12.909 -18.260  38.414  1.00 66.66           C  \nATOM   1288  CD  LYS B 127      13.539 -18.600  37.072  1.00 68.29           C  \nATOM   1289  CE  LYS B 127      12.456 -18.839  36.018  1.00 70.88           C  \nATOM   1290  NZ  LYS B 127      13.001 -19.323  34.711  1.00 73.58           N  \nATOM   1291  N   LYS B 128      15.115 -16.588  41.935  1.00 63.90           N  \nATOM   1292  CA  LYS B 128      16.159 -16.321  42.924  1.00 65.17           C  \nATOM   1293  C   LYS B 128      15.558 -16.585  44.298  1.00 65.33           C  \nATOM   1294  O   LYS B 128      15.920 -17.547  44.967  1.00 66.74           O  \nATOM   1295  CB  LYS B 128      16.617 -14.869  42.806  1.00 67.98           C  \nATOM   1296  CG  LYS B 128      17.773 -14.479  43.709  1.00 72.48           C  \nATOM   1297  CD  LYS B 128      18.226 -13.052  43.406  1.00 76.62           C  \nATOM   1298  CE  LYS B 128      19.384 -12.622  44.299  1.00 78.99           C  \nATOM   1299  NZ  LYS B 128      19.855 -11.234  43.991  1.00 79.30           N  \nATOM   1300  N   GLU B 129      14.643 -15.724  44.721  1.00 65.75           N  \nATOM   1301  CA  GLU B 129      13.950 -15.914  45.992  1.00 65.77           C  \nATOM   1302  C   GLU B 129      12.979 -17.035  45.656  1.00 64.14           C  \nATOM   1303  O   GLU B 129      12.881 -17.407  44.491  1.00 66.77           O  \nATOM   1304  CB  GLU B 129      13.206 -14.635  46.363  1.00 67.66           C  \nATOM   1305  CG  GLU B 129      14.155 -13.498  46.710  1.00 72.98           C  \nATOM   1306  CD  GLU B 129      13.462 -12.162  46.876  1.00 74.95           C  \nATOM   1307  OE1 GLU B 129      13.187 -11.507  45.852  1.00 75.92           O  \nATOM   1308  OE2 GLU B 129      13.191 -11.767  48.033  1.00 77.31           O  \nATOM   1309  N   PHE B 130      12.277 -17.594  46.630  1.00 61.52           N  \nATOM   1310  CA  PHE B 130      11.329 -18.679  46.331  1.00 61.59           C  \nATOM   1311  C   PHE B 130      11.961 -19.896  45.658  1.00 62.23           C  \nATOM   1312  O   PHE B 130      11.365 -20.970  45.638  1.00 61.41           O  \nATOM   1313  CB  PHE B 130      10.199 -18.180  45.428  1.00 60.56           C  \nATOM   1314  CG  PHE B 130       9.399 -17.077  46.026  1.00 60.04           C  \nATOM   1315  CD1 PHE B 130       8.548 -17.327  47.088  1.00 60.29           C  \nATOM   1316  CD2 PHE B 130       9.535 -15.776  45.564  1.00 59.34           C  \nATOM   1317  CE1 PHE B 130       7.845 -16.294  47.692  1.00 60.83           C  \nATOM   1318  CE2 PHE B 130       8.840 -14.740  46.159  1.00 60.10           C  \nATOM   1319  CZ  PHE B 130       7.992 -14.998  47.227  1.00 61.06           C  \nATOM   1320  N   SER B 131      13.157 -19.717  45.107  1.00 64.52           N  \nATOM   1321  CA  SER B 131      13.879 -20.782  44.421  1.00 66.17           C  \nATOM   1322  C   SER B 131      12.911 -21.789  43.820  1.00 66.12           C  \nATOM   1323  O   SER B 131      12.613 -22.826  44.413  1.00 66.91           O  \nATOM   1324  CB  SER B 131      14.872 -21.467  45.378  1.00 66.10           C  \nATOM   1325  OG  SER B 131      14.272 -21.799  46.613  1.00 68.67           O  \nATOM   1326  N   ILE B 132      12.421 -21.453  42.632  1.00 65.56           N  \nATOM   1327  CA  ILE B 132      11.464 -22.277  41.910  1.00 65.17           C  \nATOM   1328  C   ILE B 132      11.543 -21.881  40.436  1.00 65.51           C  \nATOM   1329  O   ILE B 132      11.742 -20.707  40.124  1.00 66.94           O  \nATOM   1330  CB  ILE B 132      10.036 -22.023  42.447  1.00 65.32           C  \nATOM   1331  CG1 ILE B 132       9.039 -22.938  41.752  1.00 66.40           C  \nATOM   1332  CG2 ILE B 132       9.638 -20.574  42.221  1.00 64.93           C  \nATOM   1333  CD1 ILE B 132       7.607 -22.699  42.198  1.00 67.71           C  \nATOM   1334  N   SER B 133      11.410 -22.849  39.533  1.00 64.79           N  \nATOM   1335  CA  SER B 133      11.467 -22.555  38.101  1.00 63.57           C  \nATOM   1336  C   SER B 133      10.163 -21.894  37.657  1.00 62.11           C  \nATOM   1337  O   SER B 133       9.139 -21.994  38.336  1.00 60.94           O  \nATOM   1338  CB  SER B 133      11.687 -23.836  37.294  1.00 64.17           C  \nATOM   1339  OG  SER B 133      10.537 -24.655  37.328  1.00 64.71           O  \nATOM   1340  N   PHE B 134      10.189 -21.226  36.512  1.00 60.83           N  \nATOM   1341  CA  PHE B 134       8.981 -20.568  36.058  1.00 59.57           C  \nATOM   1342  C   PHE B 134       7.886 -21.544  35.665  1.00 59.03           C  \nATOM   1343  O   PHE B 134       6.695 -21.235  35.827  1.00 58.73           O  \nATOM   1344  CB  PHE B 134       9.268 -19.603  34.907  1.00 59.55           C  \nATOM   1345  CG  PHE B 134       8.376 -18.398  34.924  1.00 58.26           C  \nATOM   1346  CD1 PHE B 134       8.443 -17.494  35.984  1.00 57.76           C  \nATOM   1347  CD2 PHE B 134       7.412 -18.210  33.940  1.00 57.55           C  \nATOM   1348  CE1 PHE B 134       7.565 -16.426  36.071  1.00 56.67           C  \nATOM   1349  CE2 PHE B 134       6.526 -17.149  34.014  1.00 56.49           C  \nATOM   1350  CZ  PHE B 134       6.601 -16.252  35.087  1.00 57.33           C  \nATOM   1351  N   GLU B 135       8.267 -22.716  35.154  1.00 57.29           N  \nATOM   1352  CA  GLU B 135       7.264 -23.710  34.790  1.00 56.76           C  \nATOM   1353  C   GLU B 135       6.560 -24.164  36.061  1.00 55.27           C  \nATOM   1354  O   GLU B 135       5.356 -24.411  36.049  1.00 55.17           O  \nATOM   1355  CB  GLU B 135       7.888 -24.914  34.077  1.00 59.82           C  \nATOM   1356  CG  GLU B 135       9.396 -25.037  34.219  1.00 66.47           C  \nATOM   1357  CD  GLU B 135      10.150 -24.064  33.331  1.00 69.23           C  \nATOM   1358  OE1 GLU B 135      10.033 -24.169  32.088  1.00 70.06           O  \nATOM   1359  OE2 GLU B 135      10.858 -23.195  33.882  1.00 72.74           O  \nATOM   1360  N   GLU B 136       7.301 -24.269  37.161  1.00 53.13           N  \nATOM   1361  CA  GLU B 136       6.690 -24.679  38.421  1.00 53.51           C  \nATOM   1362  C   GLU B 136       5.710 -23.586  38.826  1.00 52.11           C  \nATOM   1363  O   GLU B 136       4.649 -23.856  39.391  1.00 52.47           O  \nATOM   1364  CB  GLU B 136       7.744 -24.835  39.526  1.00 55.76           C  \nATOM   1365  CG  GLU B 136       8.735 -25.976  39.352  1.00 58.84           C  \nATOM   1366  CD  GLU B 136       9.802 -25.970  40.442  1.00 60.34           C  \nATOM   1367  OE1 GLU B 136       9.449 -26.153  41.629  1.00 63.16           O  \nATOM   1368  OE2 GLU B 136      10.993 -25.773  40.121  1.00 60.87           O  \nATOM   1369  N   PHE B 137       6.103 -22.345  38.539  1.00 50.85           N  \nATOM   1370  CA  PHE B 137       5.305 -21.159  38.838  1.00 48.99           C  \nATOM   1371  C   PHE B 137       4.034 -21.192  37.989  1.00 48.09           C  \nATOM   1372  O   PHE B 137       2.938 -20.988  38.495  1.00 48.04           O  \nATOM   1373  CB  PHE B 137       6.126 -19.904  38.517  1.00 49.31           C  \nATOM   1374  CG  PHE B 137       5.382 -18.616  38.710  1.00 50.19           C  \nATOM   1375  CD1 PHE B 137       5.079 -18.155  39.983  1.00 49.13           C  \nATOM   1376  CD2 PHE B 137       4.990 -17.856  37.604  1.00 49.47           C  \nATOM   1377  CE1 PHE B 137       4.393 -16.951  40.156  1.00 48.63           C  \nATOM   1378  CE2 PHE B 137       4.306 -16.654  37.761  1.00 47.40           C  \nATOM   1379  CZ  PHE B 137       4.007 -16.200  39.038  1.00 48.58           C  \nATOM   1380  N   ALA B 138       4.195 -21.459  36.697  1.00 47.54           N  \nATOM   1381  CA  ALA B 138       3.062 -21.528  35.774  1.00 48.15           C  \nATOM   1382  C   ALA B 138       2.065 -22.630  36.156  1.00 49.31           C  \nATOM   1383  O   ALA B 138       0.842 -22.444  36.054  1.00 50.18           O  \nATOM   1384  CB  ALA B 138       3.555 -21.754  34.360  1.00 45.13           C  \nATOM   1385  N   VAL B 139       2.575 -23.776  36.596  1.00 50.04           N  \nATOM   1386  CA  VAL B 139       1.697 -24.864  36.982  1.00 49.65           C  \nATOM   1387  C   VAL B 139       0.969 -24.427  38.234  1.00 51.28           C  \nATOM   1388  O   VAL B 139      -0.254 -24.551  38.341  1.00 51.02           O  \nATOM   1389  CB  VAL B 139       2.490 -26.163  37.261  1.00 48.66           C  \nATOM   1390  CG1 VAL B 139       1.608 -27.187  37.958  1.00 48.02           C  \nATOM   1391  CG2 VAL B 139       3.008 -26.743  35.950  1.00 47.52           C  \nATOM   1392  N   LEU B 140       1.726 -23.902  39.184  1.00 54.51           N  \nATOM   1393  CA  LEU B 140       1.135 -23.450  40.428  1.00 57.56           C  \nATOM   1394  C   LEU B 140       0.063 -22.374  40.249  1.00 59.24           C  \nATOM   1395  O   LEU B 140      -0.989 -22.460  40.887  1.00 57.43           O  \nATOM   1396  CB  LEU B 140       2.221 -22.945  41.379  1.00 58.05           C  \nATOM   1397  CG  LEU B 140       3.124 -24.012  41.996  1.00 57.91           C  \nATOM   1398  CD1 LEU B 140       3.879 -23.399  43.169  1.00 59.58           C  \nATOM   1399  CD2 LEU B 140       2.286 -25.199  42.477  1.00 58.02           C  \nATOM   1400  N   THR B 141       0.298 -21.371  39.396  1.00 61.08           N  \nATOM   1401  CA  THR B 141      -0.720 -20.332  39.234  1.00 61.87           C  \nATOM   1402  C   THR B 141      -1.890 -20.854  38.399  1.00 63.65           C  \nATOM   1403  O   THR B 141      -3.026 -20.413  38.574  1.00 63.26           O  \nATOM   1404  CB  THR B 141      -0.158 -18.989  38.630  1.00 59.88           C  \nATOM   1405  OG1 THR B 141      -0.358 -18.960  37.221  1.00 58.73           O  \nATOM   1406  CG2 THR B 141       1.311 -18.829  38.919  1.00 58.72           C  \nATOM   1407  N   TYR B 142      -1.626 -21.804  37.508  1.00 66.01           N  \nATOM   1408  CA  TYR B 142      -2.711 -22.379  36.718  1.00 68.57           C  \nATOM   1409  C   TYR B 142      -3.658 -23.110  37.672  1.00 67.37           C  \nATOM   1410  O   TYR B 142      -4.882 -23.006  37.568  1.00 66.20           O  \nATOM   1411  CB  TYR B 142      -2.186 -23.383  35.696  1.00 73.00           C  \nATOM   1412  CG  TYR B 142      -3.308 -24.081  34.974  1.00 78.42           C  \nATOM   1413  CD1 TYR B 142      -3.885 -23.521  33.835  1.00 80.98           C  \nATOM   1414  CD2 TYR B 142      -3.859 -25.261  35.482  1.00 81.51           C  \nATOM   1415  CE1 TYR B 142      -4.992 -24.113  33.218  1.00 83.48           C  \nATOM   1416  CE2 TYR B 142      -4.966 -25.862  34.875  1.00 84.46           C  \nATOM   1417  CZ  TYR B 142      -5.529 -25.282  33.745  1.00 84.53           C  \nATOM   1418  OH  TYR B 142      -6.633 -25.865  33.158  1.00 85.96           O  \nATOM   1419  N   ILE B 143      -3.077 -23.860  38.600  1.00 66.82           N  \nATOM   1420  CA  ILE B 143      -3.855 -24.608  39.572  1.00 66.27           C  \nATOM   1421  C   ILE B 143      -4.666 -23.656  40.430  1.00 67.57           C  \nATOM   1422  O   ILE B 143      -5.828 -23.925  40.746  1.00 68.46           O  \nATOM   1423  CB  ILE B 143      -2.934 -25.469  40.481  1.00 64.70           C  \nATOM   1424  CG1 ILE B 143      -2.502 -26.734  39.734  1.00 63.42           C  \nATOM   1425  CG2 ILE B 143      -3.642 -25.828  41.779  1.00 61.32           C  \nATOM   1426  CD1 ILE B 143      -1.504 -27.576  40.497  1.00 63.09           C  \nATOM   1427  N   SER B 144      -4.061 -22.532  40.798  1.00 68.52           N  \nATOM   1428  CA  SER B 144      -4.748 -21.557  41.636  1.00 70.14           C  \nATOM   1429  C   SER B 144      -5.894 -20.830  40.927  1.00 71.77           C  \nATOM   1430  O   SER B 144      -6.871 -20.430  41.564  1.00 70.98           O  \nATOM   1431  CB  SER B 144      -3.753 -20.534  42.178  1.00 67.95           C  \nATOM   1432  OG  SER B 144      -4.363 -19.738  43.182  1.00 67.81           O  \nATOM   1433  N   GLU B 145      -5.776 -20.664  39.615  1.00 73.79           N  \nATOM   1434  CA  GLU B 145      -6.802 -19.979  38.843  1.00 76.78           C  \nATOM   1435  C   GLU B 145      -8.048 -20.817  38.629  1.00 79.98           C  \nATOM   1436  O   GLU B 145      -9.174 -20.327  38.760  1.00 81.10           O  \nATOM   1437  CB  GLU B 145      -6.246 -19.558  37.485  1.00 74.43           C  \nATOM   1438  CG  GLU B 145      -5.797 -18.120  37.455  1.00 72.41           C  \nATOM   1439  CD  GLU B 145      -6.946 -17.161  37.653  1.00 70.69           C  \nATOM   1440  OE1 GLU B 145      -7.768 -17.017  36.724  1.00 67.23           O  \nATOM   1441  OE2 GLU B 145      -7.031 -16.565  38.746  1.00 71.58           O  \nATOM   1442  N   ASN B 146      -7.833 -22.081  38.286  1.00 83.58           N  \nATOM   1443  CA  ASN B 146      -8.922 -23.010  38.033  1.00 86.93           C  \nATOM   1444  C   ASN B 146      -8.988 -24.001  39.189  1.00 90.19           C  \nATOM   1445  O   ASN B 146      -8.705 -25.186  39.019  1.00 92.23           O  \nATOM   1446  CB  ASN B 146      -8.655 -23.736  36.719  1.00 85.24           C  \nATOM   1447  CG  ASN B 146      -8.185 -22.791  35.630  1.00 85.25           C  \nATOM   1448  OD1 ASN B 146      -8.947 -21.953  35.145  1.00 85.55           O  \nATOM   1449  ND2 ASN B 146      -6.918 -22.910  35.250  1.00 84.94           N  \nATOM   1450  N   LYS B 147      -9.352 -23.503  40.366  1.00 92.76           N  \nATOM   1451  CA  LYS B 147      -9.447 -24.335  41.558  1.00 94.53           C  \nATOM   1452  C   LYS B 147     -10.132 -25.663  41.254  1.00 96.96           C  \nATOM   1453  O   LYS B 147     -11.133 -25.707  40.536  1.00 98.99           O  \nATOM   1454  CB  LYS B 147     -10.211 -23.593  42.656  1.00 92.72           C  \nATOM   1455  CG  LYS B 147      -9.547 -22.301  43.102  1.00 92.12           C  \nATOM   1456  CD  LYS B 147     -10.462 -21.089  42.919  1.00 91.74           C  \nATOM   1457  CE  LYS B 147     -10.703 -20.749  41.447  1.00 91.23           C  \nATOM   1458  NZ  LYS B 147     -11.470 -21.790  40.704  1.00 90.72           N  \nATOM   1459  N   GLU B 148      -9.584 -26.742  41.803  1.00 98.47           N  \nATOM   1460  CA  GLU B 148     -10.126 -28.077  41.590  1.00 99.69           C  \nATOM   1461  C   GLU B 148      -9.503 -29.020  42.617  1.00100.14           C  \nATOM   1462  O   GLU B 148      -8.651 -28.616  43.408  1.00100.67           O  \nATOM   1463  CB  GLU B 148      -9.779 -28.569  40.179  1.00100.12           C  \nATOM   1464  CG  GLU B 148     -10.919 -29.231  39.427  1.00102.41           C  \nATOM   1465  CD  GLU B 148     -11.718 -28.242  38.599  1.00103.70           C  \nATOM   1466  OE1 GLU B 148     -11.130 -27.615  37.688  1.00103.43           O  \nATOM   1467  OE2 GLU B 148     -12.931 -28.092  38.857  1.00105.03           O  \nATOM   1468  N   LYS B 149      -9.938 -30.275  42.599  1.00100.47           N  \nATOM   1469  CA  LYS B 149      -9.420 -31.297  43.503  1.00100.57           C  \nATOM   1470  C   LYS B 149      -8.653 -32.295  42.647  1.00101.30           C  \nATOM   1471  O   LYS B 149      -7.720 -32.952  43.107  1.00100.75           O  \nATOM   1472  CB  LYS B 149     -10.568 -32.024  44.198  1.00100.87           C  \nATOM   1473  CG  LYS B 149     -11.320 -31.212  45.229  1.00101.21           C  \nATOM   1474  CD  LYS B 149     -10.579 -31.189  46.547  1.00102.84           C  \nATOM   1475  CE  LYS B 149     -11.457 -30.626  47.652  1.00103.76           C  \nATOM   1476  NZ  LYS B 149     -12.738 -31.373  47.766  1.00103.94           N  \nATOM   1477  N   GLU B 150      -9.066 -32.387  41.388  1.00102.64           N  \nATOM   1478  CA  GLU B 150      -8.471 -33.300  40.424  1.00104.22           C  \nATOM   1479  C   GLU B 150      -8.130 -32.588  39.119  1.00103.19           C  \nATOM   1480  O   GLU B 150      -8.985 -31.939  38.516  1.00102.42           O  \nATOM   1481  CB  GLU B 150      -9.444 -34.449  40.141  1.00107.82           C  \nATOM   1482  CG  GLU B 150     -10.870 -34.226  40.674  1.00111.96           C  \nATOM   1483  CD  GLU B 150     -11.491 -32.908  40.217  1.00114.28           C  \nATOM   1484  OE1 GLU B 150     -11.660 -32.710  38.993  1.00115.48           O  \nATOM   1485  OE2 GLU B 150     -11.812 -32.070  41.090  1.00115.57           O  \nATOM   1486  N   TYR B 151      -6.878 -32.712  38.687  1.00101.99           N  \nATOM   1487  CA  TYR B 151      -6.437 -32.084  37.446  1.00101.29           C  \nATOM   1488  C   TYR B 151      -5.817 -33.093  36.490  1.00101.42           C  \nATOM   1489  O   TYR B 151      -5.092 -33.996  36.905  1.00 99.77           O  \nATOM   1490  CB  TYR B 151      -5.405 -30.981  37.716  1.00100.38           C  \nATOM   1491  CG  TYR B 151      -5.889 -29.844  38.583  1.00 99.29           C  \nATOM   1492  CD1 TYR B 151      -5.868 -29.945  39.974  1.00 98.83           C  \nATOM   1493  CD2 TYR B 151      -6.351 -28.657  38.016  1.00 98.62           C  \nATOM   1494  CE1 TYR B 151      -6.291 -28.892  40.779  1.00 97.55           C  \nATOM   1495  CE2 TYR B 151      -6.776 -27.598  38.813  1.00 98.51           C  \nATOM   1496  CZ  TYR B 151      -6.742 -27.724  40.194  1.00 97.85           C  \nATOM   1497  OH  TYR B 151      -7.153 -26.683  40.993  1.00 97.38           O  \nATOM   1498  N   TYR B 152      -6.104 -32.921  35.205  1.00103.65           N  \nATOM   1499  CA  TYR B 152      -5.568 -33.787  34.165  1.00106.33           C  \nATOM   1500  C   TYR B 152      -4.267 -33.188  33.657  1.00105.26           C  \nATOM   1501  O   TYR B 152      -4.276 -32.137  33.018  1.00104.37           O  \nATOM   1502  CB  TYR B 152      -6.555 -33.895  33.000  1.00110.90           C  \nATOM   1503  CG  TYR B 152      -7.848 -34.603  33.339  1.00115.36           C  \nATOM   1504  CD1 TYR B 152      -8.647 -34.175  34.404  1.00116.47           C  \nATOM   1505  CD2 TYR B 152      -8.277 -35.705  32.592  1.00116.63           C  \nATOM   1506  CE1 TYR B 152      -9.838 -34.827  34.720  1.00118.04           C  \nATOM   1507  CE2 TYR B 152      -9.466 -36.363  32.898  1.00118.22           C  \nATOM   1508  CZ  TYR B 152     -10.241 -35.920  33.963  1.00118.79           C  \nATOM   1509  OH  TYR B 152     -11.416 -36.569  34.272  1.00119.39           O  \nATOM   1510  N   LEU B 153      -3.150 -33.850  33.939  1.00104.46           N  \nATOM   1511  CA  LEU B 153      -1.857 -33.356  33.485  1.00103.78           C  \nATOM   1512  C   LEU B 153      -1.918 -33.046  31.992  1.00102.93           C  \nATOM   1513  O   LEU B 153      -1.077 -32.325  31.463  1.00102.21           O  \nATOM   1514  CB  LEU B 153      -0.760 -34.386  33.757  1.00104.67           C  \nATOM   1515  CG  LEU B 153       0.638 -34.003  33.260  1.00105.17           C  \nATOM   1516  CD1 LEU B 153       1.107 -32.740  33.962  1.00106.26           C  \nATOM   1517  CD2 LEU B 153       1.608 -35.140  33.519  1.00106.21           C  \nATOM   1518  N   LYS B 154      -2.919 -33.600  31.318  1.00102.48           N  \nATOM   1519  CA  LYS B 154      -3.096 -33.363  29.893  1.00103.08           C  \nATOM   1520  C   LYS B 154      -3.452 -31.897  29.680  1.00102.74           C  \nATOM   1521  O   LYS B 154      -2.769 -31.177  28.949  1.00101.72           O  \nATOM   1522  CB  LYS B 154      -4.224 -34.241  29.344  1.00104.55           C  \nATOM   1523  CG  LYS B 154      -4.561 -33.978  27.880  1.00106.48           C  \nATOM   1524  CD  LYS B 154      -5.832 -34.704  27.460  1.00107.22           C  \nATOM   1525  CE  LYS B 154      -6.134 -34.502  25.979  1.00106.87           C  \nATOM   1526  NZ  LYS B 154      -6.297 -33.067  25.620  1.00105.97           N  \nATOM   1527  N   ASP B 155      -4.529 -31.468  30.331  1.00102.51           N  \nATOM   1528  CA  ASP B 155      -5.008 -30.095  30.228  1.00101.68           C  \nATOM   1529  C   ASP B 155      -3.928 -29.087  30.601  1.00100.07           C  \nATOM   1530  O   ASP B 155      -3.628 -28.174  29.829  1.00 99.22           O  \nATOM   1531  CB  ASP B 155      -6.229 -29.897  31.129  1.00103.51           C  \nATOM   1532  CG  ASP B 155      -7.312 -30.925  30.872  1.00105.39           C  \nATOM   1533  OD1 ASP B 155      -7.655 -31.142  29.689  1.00105.61           O  \nATOM   1534  OD2 ASP B 155      -7.822 -31.513  31.850  1.00106.56           O  \nATOM   1535  N   ILE B 156      -3.348 -29.255  31.786  1.00 97.92           N  \nATOM   1536  CA  ILE B 156      -2.300 -28.359  32.254  1.00 96.27           C  \nATOM   1537  C   ILE B 156      -1.254 -28.122  31.171  1.00 96.49           C  \nATOM   1538  O   ILE B 156      -0.923 -26.981  30.859  1.00 97.06           O  \nATOM   1539  CB  ILE B 156      -1.607 -28.922  33.514  1.00 95.22           C  \nATOM   1540  CG1 ILE B 156      -2.602 -28.952  34.678  1.00 94.37           C  \nATOM   1541  CG2 ILE B 156      -0.388 -28.078  33.864  1.00 94.43           C  \nATOM   1542  CD1 ILE B 156      -2.032 -29.492  35.971  1.00 93.55           C  \nATOM   1543  N   ILE B 157      -0.742 -29.201  30.591  1.00 96.79           N  \nATOM   1544  CA  ILE B 157       0.264 -29.086  29.542  1.00 96.97           C  \nATOM   1545  C   ILE B 157      -0.286 -28.307  28.353  1.00 96.82           C  \nATOM   1546  O   ILE B 157       0.437 -27.551  27.705  1.00 95.51           O  \nATOM   1547  CB  ILE B 157       0.722 -30.474  29.054  1.00 97.97           C  \nATOM   1548  CG1 ILE B 157       1.222 -31.300  30.239  1.00 98.62           C  \nATOM   1549  CG2 ILE B 157       1.834 -30.321  28.022  1.00 97.70           C  \nATOM   1550  CD1 ILE B 157       1.590 -32.723  29.887  1.00 99.72           C  \nATOM   1551  N   ASN B 158      -1.571 -28.496  28.073  1.00 97.76           N  \nATOM   1552  CA  ASN B 158      -2.219 -27.813  26.959  1.00 98.85           C  \nATOM   1553  C   ASN B 158      -2.313 -26.309  27.191  1.00 98.03           C  \nATOM   1554  O   ASN B 158      -1.922 -25.506  26.337  1.00 97.23           O  \nATOM   1555  CB  ASN B 158      -3.630 -28.372  26.739  1.00100.78           C  \nATOM   1556  CG  ASN B 158      -3.623 -29.741  26.090  1.00102.39           C  \nATOM   1557  OD1 ASN B 158      -3.111 -29.915  24.983  1.00103.73           O  \nATOM   1558  ND2 ASN B 158      -4.196 -30.722  26.775  1.00102.20           N  \nATOM   1559  N   HIS B 159      -2.829 -25.939  28.358  1.00 96.50           N  \nATOM   1560  CA  HIS B 159      -3.003 -24.541  28.705  1.00 95.45           C  \nATOM   1561  C   HIS B 159      -1.713 -23.773  28.959  1.00 93.80           C  \nATOM   1562  O   HIS B 159      -1.729 -22.546  29.038  1.00 93.49           O  \nATOM   1563  CB  HIS B 159      -3.948 -24.431  29.899  1.00 97.70           C  \nATOM   1564  CG  HIS B 159      -5.336 -24.906  29.600  1.00101.00           C  \nATOM   1565  ND1 HIS B 159      -6.373 -24.808  30.503  1.00102.58           N  \nATOM   1566  CD2 HIS B 159      -5.859 -25.481  28.490  1.00102.13           C  \nATOM   1567  CE1 HIS B 159      -7.474 -25.302  29.962  1.00103.16           C  \nATOM   1568  NE2 HIS B 159      -7.189 -25.717  28.741  1.00103.41           N  \nATOM   1569  N   LEU B 160      -0.598 -24.483  29.085  1.00 92.59           N  \nATOM   1570  CA  LEU B 160       0.683 -23.819  29.297  1.00 92.46           C  \nATOM   1571  C   LEU B 160       1.491 -23.895  28.007  1.00 93.09           C  \nATOM   1572  O   LEU B 160       1.145 -24.651  27.100  1.00 93.69           O  \nATOM   1573  CB  LEU B 160       1.459 -24.476  30.443  1.00 91.80           C  \nATOM   1574  CG  LEU B 160       0.822 -24.458  31.835  1.00 90.45           C  \nATOM   1575  CD1 LEU B 160       1.867 -24.857  32.863  1.00 90.19           C  \nATOM   1576  CD2 LEU B 160       0.281 -23.077  32.155  1.00 89.15           C  \nATOM   1577  N   ASN B 161       2.560 -23.112  27.921  1.00 94.27           N  \nATOM   1578  CA  ASN B 161       3.403 -23.100  26.731  1.00 96.28           C  \nATOM   1579  C   ASN B 161       4.862 -23.432  27.051  1.00 98.46           C  \nATOM   1580  O   ASN B 161       5.768 -22.686  26.676  1.00 99.29           O  \nATOM   1581  CB  ASN B 161       3.320 -21.726  26.055  1.00 94.70           C  \nATOM   1582  CG  ASN B 161       1.996 -21.498  25.351  1.00 93.83           C  \nATOM   1583  OD1 ASN B 161       1.761 -22.021  24.259  1.00 92.18           O  \nATOM   1584  ND2 ASN B 161       1.119 -20.721  25.977  1.00 94.33           N  \nATOM   1585  N   TYR B 162       5.097 -24.556  27.725  1.00100.80           N  \nATOM   1586  CA  TYR B 162       6.462 -24.935  28.089  1.00102.50           C  \nATOM   1587  C   TYR B 162       6.957 -26.310  27.627  1.00103.55           C  \nATOM   1588  O   TYR B 162       8.055 -26.722  28.006  1.00104.20           O  \nATOM   1589  CB  TYR B 162       6.647 -24.836  29.612  1.00102.45           C  \nATOM   1590  CG  TYR B 162       6.649 -23.423  30.152  1.00101.91           C  \nATOM   1591  CD1 TYR B 162       5.459 -22.773  30.472  1.00101.90           C  \nATOM   1592  CD2 TYR B 162       7.844 -22.723  30.311  1.00101.88           C  \nATOM   1593  CE1 TYR B 162       5.460 -21.459  30.935  1.00102.16           C  \nATOM   1594  CE2 TYR B 162       7.857 -21.411  30.770  1.00102.08           C  \nATOM   1595  CZ  TYR B 162       6.664 -20.784  31.078  1.00102.19           C  \nATOM   1596  OH  TYR B 162       6.674 -19.478  31.510  1.00102.60           O  \nATOM   1597  N   LYS B 163       6.173 -27.014  26.813  1.00104.60           N  \nATOM   1598  CA  LYS B 163       6.568 -28.346  26.344  1.00105.58           C  \nATOM   1599  C   LYS B 163       6.547 -29.353  27.501  1.00105.79           C  \nATOM   1600  O   LYS B 163       7.094 -29.090  28.575  1.00106.53           O  \nATOM   1601  CB  LYS B 163       7.971 -28.308  25.721  1.00105.70           C  \nATOM   1602  CG  LYS B 163       8.033 -27.789  24.287  1.00106.72           C  \nATOM   1603  CD  LYS B 163       7.347 -28.748  23.317  1.00107.77           C  \nATOM   1604  CE  LYS B 163       7.782 -28.515  21.868  1.00108.48           C  \nATOM   1605  NZ  LYS B 163       7.451 -27.154  21.358  1.00109.25           N  \nATOM   1606  N   GLN B 164       5.920 -30.505  27.269  1.00105.43           N  \nATOM   1607  CA  GLN B 164       5.795 -31.558  28.280  1.00105.10           C  \nATOM   1608  C   GLN B 164       7.037 -31.815  29.132  1.00103.20           C  \nATOM   1609  O   GLN B 164       6.952 -31.858  30.360  1.00101.51           O  \nATOM   1610  CB  GLN B 164       5.380 -32.871  27.622  1.00107.87           C  \nATOM   1611  CG  GLN B 164       4.059 -32.813  26.889  1.00111.06           C  \nATOM   1612  CD  GLN B 164       3.709 -34.140  26.245  1.00112.71           C  \nATOM   1613  OE1 GLN B 164       3.551 -35.153  26.930  1.00114.49           O  \nATOM   1614  NE2 GLN B 164       3.592 -34.144  24.920  1.00112.48           N  \nATOM   1615  N   PRO B 165       8.203 -32.007  28.491  1.00101.73           N  \nATOM   1616  CA  PRO B 165       9.447 -32.262  29.220  1.00101.51           C  \nATOM   1617  C   PRO B 165       9.626 -31.397  30.468  1.00101.73           C  \nATOM   1618  O   PRO B 165       9.838 -31.916  31.566  1.00101.98           O  \nATOM   1619  CB  PRO B 165      10.514 -31.994  28.167  1.00100.64           C  \nATOM   1620  CG  PRO B 165       9.862 -32.499  26.931  1.00100.31           C  \nATOM   1621  CD  PRO B 165       8.461 -31.935  27.041  1.00100.96           C  \nATOM   1622  N   GLN B 166       9.534 -30.081  30.295  1.00101.10           N  \nATOM   1623  CA  GLN B 166       9.693 -29.144  31.404  1.00 99.54           C  \nATOM   1624  C   GLN B 166       8.487 -29.136  32.343  1.00 98.55           C  \nATOM   1625  O   GLN B 166       8.638 -28.995  33.559  1.00 97.57           O  \nATOM   1626  CB  GLN B 166       9.932 -27.733  30.867  1.00100.07           C  \nATOM   1627  CG  GLN B 166      11.154 -27.606  29.976  1.00101.20           C  \nATOM   1628  CD  GLN B 166      11.409 -26.174  29.544  1.00102.82           C  \nATOM   1629  OE1 GLN B 166      11.704 -25.305  30.368  1.00103.60           O  \nATOM   1630  NE2 GLN B 166      11.291 -25.918  28.246  1.00103.33           N  \nATOM   1631  N   VAL B 167       7.293 -29.285  31.775  1.00 96.97           N  \nATOM   1632  CA  VAL B 167       6.074 -29.299  32.569  1.00 96.10           C  \nATOM   1633  C   VAL B 167       6.057 -30.511  33.500  1.00 95.61           C  \nATOM   1634  O   VAL B 167       5.955 -30.366  34.720  1.00 95.62           O  \nATOM   1635  CB  VAL B 167       4.826 -29.340  31.668  1.00 96.55           C  \nATOM   1636  CG1 VAL B 167       3.565 -29.275  32.521  1.00 96.21           C  \nATOM   1637  CG2 VAL B 167       4.863 -28.188  30.679  1.00 96.63           C  \nATOM   1638  N   VAL B 168       6.157 -31.707  32.924  1.00 94.88           N  \nATOM   1639  CA  VAL B 168       6.161 -32.939  33.716  1.00 93.51           C  \nATOM   1640  C   VAL B 168       7.295 -32.889  34.734  1.00 91.40           C  \nATOM   1641  O   VAL B 168       7.170 -33.378  35.858  1.00 90.00           O  \nATOM   1642  CB  VAL B 168       6.350 -34.180  32.821  1.00 94.32           C  \nATOM   1643  CG1 VAL B 168       6.447 -35.429  33.683  1.00 95.03           C  \nATOM   1644  CG2 VAL B 168       5.186 -34.299  31.846  1.00 94.83           C  \nATOM   1645  N   LYS B 169       8.403 -32.292  34.317  1.00 89.58           N  \nATOM   1646  CA  LYS B 169       9.576 -32.123  35.161  1.00 87.41           C  \nATOM   1647  C   LYS B 169       9.189 -31.258  36.364  1.00 86.84           C  \nATOM   1648  O   LYS B 169       9.628 -31.505  37.490  1.00 85.86           O  \nATOM   1649  CB  LYS B 169      10.671 -31.441  34.344  1.00 86.78           C  \nATOM   1650  CG  LYS B 169      11.979 -31.177  35.051  1.00 87.35           C  \nATOM   1651  CD  LYS B 169      12.958 -30.578  34.045  1.00 90.21           C  \nATOM   1652  CE  LYS B 169      14.325 -30.299  34.645  1.00 91.84           C  \nATOM   1653  NZ  LYS B 169      15.281 -29.803  33.606  1.00 93.15           N  \nATOM   1654  N   ALA B 170       8.354 -30.250  36.112  1.00 86.12           N  \nATOM   1655  CA  ALA B 170       7.891 -29.339  37.153  1.00 84.88           C  \nATOM   1656  C   ALA B 170       6.855 -30.007  38.053  1.00 83.72           C  \nATOM   1657  O   ALA B 170       6.915 -29.888  39.280  1.00 83.48           O  \nATOM   1658  CB  ALA B 170       7.300 -28.083  36.520  1.00 85.19           C  \nATOM   1659  N   VAL B 171       5.901 -30.699  37.437  1.00 82.40           N  \nATOM   1660  CA  VAL B 171       4.860 -31.400  38.179  1.00 81.93           C  \nATOM   1661  C   VAL B 171       5.542 -32.330  39.174  1.00 83.34           C  \nATOM   1662  O   VAL B 171       5.216 -32.344  40.365  1.00 83.30           O  \nATOM   1663  CB  VAL B 171       3.987 -32.253  37.239  1.00 80.91           C  \nATOM   1664  CG1 VAL B 171       2.936 -33.009  38.039  1.00 80.28           C  \nATOM   1665  CG2 VAL B 171       3.334 -31.368  36.197  1.00 81.04           C  \nATOM   1666  N   LYS B 172       6.497 -33.101  38.657  1.00 84.29           N  \nATOM   1667  CA  LYS B 172       7.272 -34.056  39.440  1.00 85.45           C  \nATOM   1668  C   LYS B 172       7.817 -33.420  40.716  1.00 85.10           C  \nATOM   1669  O   LYS B 172       7.482 -33.842  41.823  1.00 84.44           O  \nATOM   1670  CB  LYS B 172       8.425 -34.581  38.580  1.00 87.06           C  \nATOM   1671  CG  LYS B 172       9.203 -35.747  39.168  1.00 89.51           C  \nATOM   1672  CD  LYS B 172      10.113 -36.343  38.102  1.00 90.72           C  \nATOM   1673  CE  LYS B 172      10.878 -37.552  38.610  1.00 90.36           C  \nATOM   1674  NZ  LYS B 172      11.661 -38.189  37.514  1.00 89.74           N  \nATOM   1675  N   ILE B 173       8.655 -32.401  40.555  1.00 84.96           N  \nATOM   1676  CA  ILE B 173       9.245 -31.704  41.690  1.00 84.70           C  \nATOM   1677  C   ILE B 173       8.165 -31.199  42.647  1.00 86.57           C  \nATOM   1678  O   ILE B 173       8.181 -31.512  43.839  1.00 87.53           O  \nATOM   1679  CB  ILE B 173      10.087 -30.504  41.217  1.00 83.26           C  \nATOM   1680  CG1 ILE B 173      11.112 -30.971  40.184  1.00 82.53           C  \nATOM   1681  CG2 ILE B 173      10.786 -29.854  42.401  1.00 81.84           C  \nATOM   1682  CD1 ILE B 173      11.849 -29.840  39.507  1.00 82.75           C  \nATOM   1683  N   LEU B 174       7.228 -30.421  42.120  1.00 88.34           N  \nATOM   1684  CA  LEU B 174       6.149 -29.874  42.931  1.00 90.83           C  \nATOM   1685  C   LEU B 174       5.454 -30.972  43.719  1.00 92.35           C  \nATOM   1686  O   LEU B 174       5.048 -30.766  44.863  1.00 92.87           O  \nATOM   1687  CB  LEU B 174       5.127 -29.161  42.043  1.00 90.96           C  \nATOM   1688  CG  LEU B 174       5.586 -27.882  41.348  1.00 89.85           C  \nATOM   1689  CD1 LEU B 174       4.511 -27.410  40.382  1.00 90.17           C  \nATOM   1690  CD2 LEU B 174       5.888 -26.824  42.398  1.00 89.24           C  \nATOM   1691  N   SER B 175       5.318 -32.136  43.092  1.00 93.58           N  \nATOM   1692  CA  SER B 175       4.672 -33.287  43.715  1.00 94.30           C  \nATOM   1693  C   SER B 175       5.442 -33.750  44.953  1.00 95.09           C  \nATOM   1694  O   SER B 175       4.846 -34.058  45.987  1.00 94.11           O  \nATOM   1695  CB  SER B 175       4.572 -34.432  42.712  1.00 93.37           C  \nATOM   1696  OG  SER B 175       3.832 -35.503  43.259  1.00 94.28           O  \nATOM   1697  N   GLN B 176       6.767 -33.793  44.837  1.00 96.18           N  \nATOM   1698  CA  GLN B 176       7.628 -34.198  45.942  1.00 96.88           C  \nATOM   1699  C   GLN B 176       7.509 -33.159  47.042  1.00 96.25           C  \nATOM   1700  O   GLN B 176       7.363 -33.494  48.217  1.00 96.02           O  \nATOM   1701  CB  GLN B 176       9.085 -34.272  45.489  1.00 98.74           C  \nATOM   1702  CG  GLN B 176       9.313 -35.047  44.206  1.00101.34           C  \nATOM   1703  CD  GLN B 176      10.771 -35.046  43.795  1.00102.60           C  \nATOM   1704  OE1 GLN B 176      11.120 -35.469  42.692  1.00102.32           O  \nATOM   1705  NE2 GLN B 176      11.634 -34.572  44.687  1.00103.42           N  \nATOM   1706  N   GLU B 177       7.585 -31.892  46.646  1.00 96.74           N  \nATOM   1707  CA  GLU B 177       7.470 -30.780  47.581  1.00 98.47           C  \nATOM   1708  C   GLU B 177       6.134 -30.909  48.300  1.00 97.45           C  \nATOM   1709  O   GLU B 177       5.887 -30.252  49.311  1.00 96.91           O  \nATOM   1710  CB  GLU B 177       7.532 -29.455  46.821  1.00101.18           C  \nATOM   1711  CG  GLU B 177       8.811 -29.265  46.015  1.00104.55           C  \nATOM   1712  CD  GLU B 177      10.006 -28.939  46.890  1.00106.74           C  \nATOM   1713  OE1 GLU B 177      10.228 -29.657  47.887  1.00109.59           O  \nATOM   1714  OE2 GLU B 177      10.727 -27.965  46.582  1.00106.61           O  \nATOM   1715  N   ASP B 178       5.279 -31.767  47.751  1.00 96.54           N  \nATOM   1716  CA  ASP B 178       3.960 -32.043  48.300  1.00 96.10           C  \nATOM   1717  C   ASP B 178       2.999 -30.861  48.185  1.00 95.93           C  \nATOM   1718  O   ASP B 178       2.371 -30.455  49.165  1.00 95.19           O  \nATOM   1719  CB  ASP B 178       4.089 -32.491  49.761  1.00 96.67           C  \nATOM   1720  CG  ASP B 178       2.827 -33.153  50.286  1.00 96.09           C  \nATOM   1721  OD1 ASP B 178       2.239 -33.977  49.552  1.00 96.14           O  \nATOM   1722  OD2 ASP B 178       2.432 -32.862  51.436  1.00 95.32           O  \nATOM   1723  N   TYR B 179       2.900 -30.309  46.979  1.00 95.34           N  \nATOM   1724  CA  TYR B 179       1.996 -29.198  46.710  1.00 95.82           C  \nATOM   1725  C   TYR B 179       0.707 -29.833  46.223  1.00 98.40           C  \nATOM   1726  O   TYR B 179      -0.378 -29.275  46.349  1.00 98.94           O  \nATOM   1727  CB  TYR B 179       2.589 -28.273  45.643  1.00 92.06           C  \nATOM   1728  CG  TYR B 179       3.544 -27.247  46.210  1.00 88.34           C  \nATOM   1729  CD1 TYR B 179       3.073 -26.191  46.992  1.00 87.38           C  \nATOM   1730  CD2 TYR B 179       4.919 -27.350  46.003  1.00 86.47           C  \nATOM   1731  CE1 TYR B 179       3.946 -25.265  47.558  1.00 84.94           C  \nATOM   1732  CE2 TYR B 179       5.803 -26.426  46.567  1.00 84.44           C  \nATOM   1733  CZ  TYR B 179       5.306 -25.389  47.345  1.00 84.01           C  \nATOM   1734  OH  TYR B 179       6.163 -24.485  47.927  1.00 81.56           O  \nATOM   1735  N   PHE B 180       0.859 -31.022  45.660  1.00102.56           N  \nATOM   1736  CA  PHE B 180      -0.249 -31.821  45.160  1.00107.16           C  \nATOM   1737  C   PHE B 180       0.274 -33.248  45.031  1.00111.47           C  \nATOM   1738  O   PHE B 180       1.431 -33.462  44.662  1.00113.03           O  \nATOM   1739  CB  PHE B 180      -0.757 -31.300  43.807  1.00104.31           C  \nATOM   1740  CG  PHE B 180       0.325 -31.028  42.804  1.00101.96           C  \nATOM   1741  CD1 PHE B 180       0.796 -29.735  42.605  1.00100.64           C  \nATOM   1742  CD2 PHE B 180       0.863 -32.058  42.046  1.00101.62           C  \nATOM   1743  CE1 PHE B 180       1.786 -29.471  41.662  1.00 99.81           C  \nATOM   1744  CE2 PHE B 180       1.856 -31.801  41.101  1.00101.82           C  \nATOM   1745  CZ  PHE B 180       2.316 -30.504  40.910  1.00100.18           C  \nATOM   1746  N   ASP B 181      -0.570 -34.222  45.352  1.00116.26           N  \nATOM   1747  CA  ASP B 181      -0.163 -35.618  45.291  1.00120.51           C  \nATOM   1748  C   ASP B 181       0.299 -36.062  43.907  1.00122.84           C  \nATOM   1749  O   ASP B 181       1.501 -36.175  43.672  1.00123.84           O  \nATOM   1750  CB  ASP B 181      -1.296 -36.517  45.786  1.00121.49           C  \nATOM   1751  CG  ASP B 181      -1.729 -36.173  47.200  1.00122.30           C  \nATOM   1752  OD1 ASP B 181      -2.335 -35.098  47.393  1.00123.01           O  \nATOM   1753  OD2 ASP B 181      -1.455 -36.972  48.120  1.00122.34           O  \nATOM   1754  N   LYS B 182      -0.637 -36.315  42.993  1.00125.54           N  \nATOM   1755  CA  LYS B 182      -0.272 -36.759  41.642  1.00129.19           C  \nATOM   1756  C   LYS B 182       0.467 -38.105  41.708  1.00132.67           C  \nATOM   1757  O   LYS B 182       1.020 -38.442  42.744  1.00133.02           O  \nATOM   1758  CB  LYS B 182       0.656 -35.720  40.999  1.00127.85           C  \nATOM   1759  CG  LYS B 182       1.288 -36.142  39.679  1.00126.97           C  \nATOM   1760  CD  LYS B 182       2.609 -36.877  39.867  1.00125.18           C  \nATOM   1761  CE  LYS B 182       3.260 -37.154  38.521  1.00124.05           C  \nATOM   1762  NZ  LYS B 182       4.629 -37.715  38.656  1.00122.75           N  \nATOM   1763  N   LYS B 183       0.550 -38.870  40.618  1.00137.42           N  \nATOM   1764  CA  LYS B 183       1.322 -40.118  40.710  1.00141.87           C  \nATOM   1765  C   LYS B 183       1.683 -40.952  39.475  1.00145.48           C  \nATOM   1766  O   LYS B 183       2.197 -42.065  39.620  1.00146.20           O  \nATOM   1767  CB  LYS B 183       0.705 -41.054  41.757  1.00140.10           C  \nATOM   1768  CG  LYS B 183       1.636 -41.305  42.945  1.00138.22           C  \nATOM   1769  CD  LYS B 183       3.031 -41.706  42.469  1.00136.41           C  \nATOM   1770  CE  LYS B 183       4.030 -41.776  43.607  1.00134.58           C  \nATOM   1771  NZ  LYS B 183       5.390 -42.097  43.094  1.00133.87           N  \nATOM   1772  N   ARG B 184       1.440 -40.436  38.274  1.00149.72           N  \nATOM   1773  CA  ARG B 184       1.779 -41.177  37.054  1.00153.31           C  \nATOM   1774  C   ARG B 184       1.153 -42.575  37.020  1.00155.63           C  \nATOM   1775  O   ARG B 184       0.269 -42.895  37.819  1.00156.73           O  \nATOM   1776  CB  ARG B 184       3.301 -41.321  36.931  1.00152.99           C  \nATOM   1777  CG  ARG B 184       4.065 -40.011  36.860  1.00152.77           C  \nATOM   1778  CD  ARG B 184       5.544 -40.233  37.137  1.00152.97           C  \nATOM   1779  NE  ARG B 184       6.168 -41.121  36.161  1.00154.15           N  \nATOM   1780  CZ  ARG B 184       7.413 -41.579  36.259  1.00154.51           C  \nATOM   1781  NH1 ARG B 184       8.167 -41.235  37.293  1.00154.48           N  \nATOM   1782  NH2 ARG B 184       7.906 -42.376  35.320  1.00155.02           N  \nATOM   1783  N   ASN B 185       1.632 -43.394  36.085  1.00157.85           N  \nATOM   1784  CA  ASN B 185       1.173 -44.772  35.901  1.00159.72           C  \nATOM   1785  C   ASN B 185      -0.321 -44.971  36.163  1.00160.66           C  \nATOM   1786  O   ASN B 185      -0.734 -45.214  37.298  1.00161.37           O  \nATOM   1787  CB  ASN B 185       1.982 -45.714  36.803  1.00159.58           C  \nATOM   1788  CG  ASN B 185       1.728 -47.183  36.502  1.00159.34           C  \nATOM   1789  OD1 ASN B 185       1.993 -47.656  35.396  1.00159.30           O  \nATOM   1790  ND2 ASN B 185       1.216 -47.911  37.488  1.00158.52           N  \nATOM   1791  N   GLU B 186      -1.125 -44.872  35.106  1.00161.16           N  \nATOM   1792  CA  GLU B 186      -2.572 -45.048  35.212  1.00161.11           C  \nATOM   1793  C   GLU B 186      -3.132 -45.652  33.921  1.00161.61           C  \nATOM   1794  O   GLU B 186      -4.345 -45.749  33.735  1.00161.38           O  \nATOM   1795  CB  GLU B 186      -3.233 -43.704  35.533  1.00160.61           C  \nATOM   1796  CG  GLU B 186      -2.750 -43.123  36.859  1.00160.37           C  \nATOM   1797  CD  GLU B 186      -3.258 -41.725  37.124  1.00160.39           C  \nATOM   1798  OE1 GLU B 186      -3.049 -40.842  36.267  1.00160.05           O  \nATOM   1799  OE2 GLU B 186      -3.860 -41.506  38.197  1.00161.00           O  \nATOM   1800  N   HIS B 187      -2.211 -46.053  33.047  1.00162.19           N  \nATOM   1801  CA  HIS B 187      -2.490 -46.694  31.760  1.00162.38           C  \nATOM   1802  C   HIS B 187      -3.612 -46.173  30.864  1.00163.26           C  \nATOM   1803  O   HIS B 187      -4.526 -45.477  31.308  1.00163.74           O  \nATOM   1804  CB  HIS B 187      -2.692 -48.197  31.975  1.00161.12           C  \nATOM   1805  CG  HIS B 187      -1.465 -48.906  32.455  1.00159.84           C  \nATOM   1806  ND1 HIS B 187      -0.876 -48.636  33.671  1.00159.27           N  \nATOM   1807  CD2 HIS B 187      -0.708 -49.868  31.875  1.00159.35           C  \nATOM   1808  CE1 HIS B 187       0.190 -49.402  33.821  1.00159.12           C  \nATOM   1809  NE2 HIS B 187       0.314 -50.158  32.745  1.00159.25           N  \nATOM   1810  N   ASP B 188      -3.513 -46.545  29.587  1.00164.08           N  \nATOM   1811  CA  ASP B 188      -4.464 -46.180  28.537  1.00164.37           C  \nATOM   1812  C   ASP B 188      -4.973 -44.742  28.618  1.00164.31           C  \nATOM   1813  O   ASP B 188      -4.424 -43.923  29.356  1.00165.44           O  \nATOM   1814  CB  ASP B 188      -5.649 -47.151  28.537  1.00164.03           C  \nATOM   1815  CG  ASP B 188      -5.992 -47.649  27.143  1.00163.82           C  \nATOM   1816  OD1 ASP B 188      -5.148 -48.342  26.534  1.00163.59           O  \nATOM   1817  OD2 ASP B 188      -7.099 -47.344  26.654  1.00163.45           O  \nATOM   1818  N   GLU B 189      -6.018 -44.440  27.848  1.00163.08           N  \nATOM   1819  CA  GLU B 189      -6.597 -43.097  27.826  1.00161.08           C  \nATOM   1820  C   GLU B 189      -5.486 -42.125  27.440  1.00161.06           C  \nATOM   1821  O   GLU B 189      -4.566 -42.479  26.699  1.00161.77           O  \nATOM   1822  CB  GLU B 189      -7.116 -42.720  29.215  1.00159.20           C  \nATOM   1823  CG  GLU B 189      -7.834 -43.829  29.957  1.00156.39           C  \nATOM   1824  CD  GLU B 189      -7.897 -43.565  31.449  1.00155.13           C  \nATOM   1825  OE1 GLU B 189      -8.463 -42.526  31.848  1.00153.89           O  \nATOM   1826  OE2 GLU B 189      -7.376 -44.393  32.226  1.00154.08           O  \nATOM   1827  N   ARG B 190      -5.582 -40.897  27.942  1.00159.98           N  \nATOM   1828  CA  ARG B 190      -4.557 -39.889  27.696  1.00158.33           C  \nATOM   1829  C   ARG B 190      -3.467 -40.216  28.722  1.00157.00           C  \nATOM   1830  O   ARG B 190      -3.347 -41.373  29.133  1.00157.64           O  \nATOM   1831  CB  ARG B 190      -5.125 -38.483  27.923  1.00157.89           C  \nATOM   1832  CG  ARG B 190      -6.170 -38.035  26.892  1.00157.39           C  \nATOM   1833  CD  ARG B 190      -5.543 -37.725  25.530  1.00156.60           C  \nATOM   1834  NE  ARG B 190      -5.397 -38.899  24.671  1.00155.38           N  \nATOM   1835  CZ  ARG B 190      -6.383 -39.433  23.955  1.00154.01           C  \nATOM   1836  NH1 ARG B 190      -7.597 -38.900  23.988  1.00152.85           N  \nATOM   1837  NH2 ARG B 190      -6.153 -40.500  23.199  1.00153.23           N  \nATOM   1838  N   THR B 191      -2.682 -39.236  29.159  1.00154.50           N  \nATOM   1839  CA  THR B 191      -1.637 -39.548  30.132  1.00151.30           C  \nATOM   1840  C   THR B 191      -1.555 -38.712  31.408  1.00149.09           C  \nATOM   1841  O   THR B 191      -1.688 -37.488  31.389  1.00148.73           O  \nATOM   1842  CB  THR B 191      -0.238 -39.549  29.470  1.00151.30           C  \nATOM   1843  OG1 THR B 191      -0.064 -38.353  28.700  1.00150.61           O  \nATOM   1844  CG2 THR B 191      -0.076 -40.770  28.574  1.00150.87           C  \nATOM   1845  N   VAL B 192      -1.334 -39.420  32.514  1.00145.94           N  \nATOM   1846  CA  VAL B 192      -1.180 -38.855  33.855  1.00142.24           C  \nATOM   1847  C   VAL B 192      -2.268 -37.899  34.348  1.00140.41           C  \nATOM   1848  O   VAL B 192      -3.020 -37.321  33.562  1.00140.50           O  \nATOM   1849  CB  VAL B 192       0.189 -38.146  33.988  1.00142.04           C  \nATOM   1850  CG1 VAL B 192       0.494 -37.864  35.452  1.00140.76           C  \nATOM   1851  CG2 VAL B 192       1.283 -39.006  33.370  1.00141.66           C  \nATOM   1852  N   LEU B 193      -2.331 -37.752  35.671  1.00137.05           N  \nATOM   1853  CA  LEU B 193      -3.286 -36.879  36.350  1.00134.19           C  \nATOM   1854  C   LEU B 193      -2.673 -36.497  37.702  1.00132.80           C  \nATOM   1855  O   LEU B 193      -1.795 -37.203  38.200  1.00132.17           O  \nATOM   1856  CB  LEU B 193      -4.612 -37.614  36.575  1.00132.99           C  \nATOM   1857  CG  LEU B 193      -5.340 -38.191  35.357  1.00131.83           C  \nATOM   1858  CD1 LEU B 193      -6.572 -38.944  35.816  1.00131.61           C  \nATOM   1859  CD2 LEU B 193      -5.730 -37.080  34.404  1.00131.07           C  \nATOM   1860  N   ILE B 194      -3.122 -35.388  38.292  1.00131.30           N  \nATOM   1861  CA  ILE B 194      -2.591 -34.956  39.588  1.00129.17           C  \nATOM   1862  C   ILE B 194      -3.692 -34.640  40.603  1.00126.51           C  \nATOM   1863  O   ILE B 194      -4.788 -34.205  40.243  1.00125.69           O  \nATOM   1864  CB  ILE B 194      -1.671 -33.713  39.452  1.00130.64           C  \nATOM   1865  CG1 ILE B 194      -2.507 -32.449  39.259  1.00131.60           C  \nATOM   1866  CG2 ILE B 194      -0.730 -33.888  38.269  1.00130.46           C  \nATOM   1867  CD1 ILE B 194      -1.683 -31.182  39.246  1.00133.36           C  \nATOM   1868  N   LEU B 195      -3.380 -34.855  41.877  1.00123.38           N  \nATOM   1869  CA  LEU B 195      -4.328 -34.630  42.962  1.00121.58           C  \nATOM   1870  C   LEU B 195      -3.912 -33.485  43.874  1.00120.59           C  \nATOM   1871  O   LEU B 195      -2.805 -33.477  44.407  1.00120.25           O  \nATOM   1872  CB  LEU B 195      -4.469 -35.914  43.787  1.00121.38           C  \nATOM   1873  CG  LEU B 195      -5.275 -35.878  45.087  1.00121.16           C  \nATOM   1874  CD1 LEU B 195      -6.723 -35.501  44.807  1.00121.17           C  \nATOM   1875  CD2 LEU B 195      -5.194 -37.245  45.752  1.00120.38           C  \nATOM   1876  N   VAL B 196      -4.809 -32.523  44.059  1.00119.92           N  \nATOM   1877  CA  VAL B 196      -4.521 -31.380  44.914  1.00118.75           C  \nATOM   1878  C   VAL B 196      -5.305 -31.457  46.213  1.00117.87           C  \nATOM   1879  O   VAL B 196      -6.465 -31.049  46.279  1.00117.35           O  \nATOM   1880  CB  VAL B 196      -4.857 -30.046  44.213  1.00118.87           C  \nATOM   1881  CG1 VAL B 196      -4.514 -28.876  45.129  1.00117.95           C  \nATOM   1882  CG2 VAL B 196      -4.086 -29.938  42.912  1.00118.82           C  \nATOM   1883  N   ASN B 197      -4.666 -31.993  47.244  1.00117.20           N  \nATOM   1884  CA  ASN B 197      -5.302 -32.104  48.544  1.00117.08           C  \nATOM   1885  C   ASN B 197      -5.776 -30.718  48.951  1.00116.52           C  \nATOM   1886  O   ASN B 197      -5.014 -29.753  48.902  1.00116.18           O  \nATOM   1887  CB  ASN B 197      -4.307 -32.646  49.573  1.00117.49           C  \nATOM   1888  CG  ASN B 197      -4.719 -32.346  51.003  1.00117.97           C  \nATOM   1889  OD1 ASN B 197      -5.806 -32.725  51.444  1.00118.90           O  \nATOM   1890  ND2 ASN B 197      -3.848 -31.660  51.735  1.00116.90           N  \nATOM   1891  N   ALA B 198      -7.040 -30.624  49.342  1.00115.77           N  \nATOM   1892  CA  ALA B 198      -7.624 -29.356  49.752  1.00114.34           C  \nATOM   1893  C   ALA B 198      -6.763 -28.611  50.771  1.00113.41           C  \nATOM   1894  O   ALA B 198      -6.961 -27.416  50.992  1.00113.83           O  \nATOM   1895  CB  ALA B 198      -9.019 -29.593  50.319  1.00114.53           C  \nATOM   1896  N   GLN B 199      -5.810 -29.309  51.385  1.00111.76           N  \nATOM   1897  CA  GLN B 199      -4.933 -28.695  52.383  1.00110.03           C  \nATOM   1898  C   GLN B 199      -3.558 -28.340  51.828  1.00108.49           C  \nATOM   1899  O   GLN B 199      -2.820 -27.554  52.428  1.00107.52           O  \nATOM   1900  CB  GLN B 199      -4.780 -29.622  53.582  1.00110.88           C  \nATOM   1901  CG  GLN B 199      -6.080 -29.868  54.307  1.00113.78           C  \nATOM   1902  CD  GLN B 199      -6.028 -31.101  55.175  1.00115.78           C  \nATOM   1903  OE1 GLN B 199      -5.819 -32.211  54.680  1.00116.35           O  \nATOM   1904  NE2 GLN B 199      -6.221 -30.918  56.477  1.00117.57           N  \nATOM   1905  N   GLN B 200      -3.213 -28.935  50.689  1.00106.43           N  \nATOM   1906  CA  GLN B 200      -1.939 -28.657  50.035  1.00103.84           C  \nATOM   1907  C   GLN B 200      -2.126 -27.317  49.338  1.00100.77           C  \nATOM   1908  O   GLN B 200      -1.171 -26.683  48.886  1.00100.26           O  \nATOM   1909  CB  GLN B 200      -1.625 -29.738  48.998  1.00105.81           C  \nATOM   1910  CG  GLN B 200      -1.395 -31.127  49.571  1.00108.07           C  \nATOM   1911  CD  GLN B 200      -1.182 -32.171  48.487  1.00109.85           C  \nATOM   1912  OE1 GLN B 200      -2.075 -32.436  47.680  1.00110.54           O  \nATOM   1913  NE2 GLN B 200       0.007 -32.764  48.460  1.00110.79           N  \nATOM   1914  N   ARG B 201      -3.387 -26.905  49.265  1.00 96.76           N  \nATOM   1915  CA  ARG B 201      -3.786 -25.657  48.639  1.00 92.64           C  \nATOM   1916  C   ARG B 201      -3.444 -24.483  49.543  1.00 90.45           C  \nATOM   1917  O   ARG B 201      -3.363 -23.342  49.090  1.00 89.81           O  \nATOM   1918  CB  ARG B 201      -5.282 -25.702  48.341  1.00 92.19           C  \nATOM   1919  CG  ARG B 201      -5.868 -24.412  47.838  1.00 92.91           C  \nATOM   1920  CD  ARG B 201      -6.948 -24.705  46.822  1.00 93.98           C  \nATOM   1921  NE  ARG B 201      -6.378 -25.008  45.515  1.00 94.87           N  \nATOM   1922  CZ  ARG B 201      -7.053 -25.566  44.518  1.00 96.11           C  \nATOM   1923  NH1 ARG B 201      -8.327 -25.896  44.677  1.00 97.57           N  \nATOM   1924  NH2 ARG B 201      -6.460 -25.777  43.354  1.00 96.15           N  \nATOM   1925  N   LYS B 202      -3.257 -24.765  50.829  1.00 88.38           N  \nATOM   1926  CA  LYS B 202      -2.873 -23.731  51.782  1.00 87.00           C  \nATOM   1927  C   LYS B 202      -1.413 -23.448  51.476  1.00 83.72           C  \nATOM   1928  O   LYS B 202      -0.943 -22.318  51.575  1.00 83.71           O  \nATOM   1929  CB  LYS B 202      -2.980 -24.235  53.224  1.00 89.78           C  \nATOM   1930  CG  LYS B 202      -4.391 -24.379  53.756  1.00 93.64           C  \nATOM   1931  CD  LYS B 202      -4.381 -24.828  55.215  1.00 95.08           C  \nATOM   1932  CE  LYS B 202      -5.797 -24.903  55.777  1.00 96.65           C  \nATOM   1933  NZ  LYS B 202      -5.831 -25.476  57.153  1.00 97.10           N  \nATOM   1934  N   LYS B 203      -0.706 -24.505  51.095  1.00 79.50           N  \nATOM   1935  CA  LYS B 203       0.708 -24.426  50.768  1.00 75.60           C  \nATOM   1936  C   LYS B 203       0.933 -23.639  49.478  1.00 73.12           C  \nATOM   1937  O   LYS B 203       1.896 -22.883  49.360  1.00 73.55           O  \nATOM   1938  CB  LYS B 203       1.269 -25.842  50.619  1.00 74.91           C  \nATOM   1939  CG  LYS B 203       2.780 -25.944  50.626  1.00 73.65           C  \nATOM   1940  CD  LYS B 203       3.200 -27.398  50.462  1.00 73.15           C  \nATOM   1941  CE  LYS B 203       4.682 -27.608  50.742  1.00 73.11           C  \nATOM   1942  NZ  LYS B 203       5.560 -27.229  49.605  1.00 73.96           N  \nATOM   1943  N   ILE B 204       0.036 -23.820  48.516  1.00 68.94           N  \nATOM   1944  CA  ILE B 204       0.148 -23.149  47.233  1.00 65.72           C  \nATOM   1945  C   ILE B 204      -0.271 -21.687  47.254  1.00 66.31           C  \nATOM   1946  O   ILE B 204       0.516 -20.802  46.927  1.00 66.20           O  \nATOM   1947  CB  ILE B 204      -0.677 -23.874  46.159  1.00 63.02           C  \nATOM   1948  CG1 ILE B 204      -0.109 -25.279  45.949  1.00 60.91           C  \nATOM   1949  CG2 ILE B 204      -0.664 -23.066  44.854  1.00 60.98           C  \nATOM   1950  CD1 ILE B 204      -0.804 -26.073  44.878  1.00 60.56           C  \nATOM   1951  N   GLU B 205      -1.510 -21.430  47.641  1.00 66.34           N  \nATOM   1952  CA  GLU B 205      -2.001 -20.065  47.666  1.00 66.42           C  \nATOM   1953  C   GLU B 205      -1.220 -19.169  48.608  1.00 65.13           C  \nATOM   1954  O   GLU B 205      -1.228 -17.950  48.453  1.00 67.17           O  \nATOM   1955  CB  GLU B 205      -3.486 -20.050  48.010  1.00 68.33           C  \nATOM   1956  CG  GLU B 205      -4.287 -20.929  47.074  1.00 73.05           C  \nATOM   1957  CD  GLU B 205      -5.773 -20.712  47.192  1.00 76.50           C  \nATOM   1958  OE1 GLU B 205      -6.264 -20.532  48.331  1.00 78.23           O  \nATOM   1959  OE2 GLU B 205      -6.452 -20.737  46.139  1.00 77.93           O  \nATOM   1960  N   SER B 206      -0.539 -19.768  49.580  1.00 62.88           N  \nATOM   1961  CA  SER B 206       0.259 -18.992  50.520  1.00 60.24           C  \nATOM   1962  C   SER B 206       1.548 -18.624  49.802  1.00 56.48           C  \nATOM   1963  O   SER B 206       2.063 -17.525  49.954  1.00 55.60           O  \nATOM   1964  CB  SER B 206       0.565 -19.817  51.780  1.00 62.54           C  \nATOM   1965  OG  SER B 206       1.386 -19.100  52.693  1.00 65.86           O  \nATOM   1966  N   LEU B 207       2.054 -19.559  49.008  1.00 52.58           N  \nATOM   1967  CA  LEU B 207       3.275 -19.349  48.240  1.00 49.12           C  \nATOM   1968  C   LEU B 207       2.987 -18.261  47.206  1.00 48.01           C  \nATOM   1969  O   LEU B 207       3.779 -17.337  47.013  1.00 47.95           O  \nATOM   1970  CB  LEU B 207       3.691 -20.653  47.548  1.00 48.42           C  \nATOM   1971  CG  LEU B 207       5.023 -20.694  46.789  1.00 47.89           C  \nATOM   1972  CD1 LEU B 207       4.810 -20.337  45.334  1.00 48.27           C  \nATOM   1973  CD2 LEU B 207       6.028 -19.763  47.459  1.00 47.44           C  \nATOM   1974  N   LEU B 208       1.830 -18.370  46.561  1.00 44.59           N  \nATOM   1975  CA  LEU B 208       1.426 -17.392  45.570  1.00 41.97           C  \nATOM   1976  C   LEU B 208       1.091 -16.042  46.216  1.00 41.06           C  \nATOM   1977  O   LEU B 208       1.227 -15.010  45.576  1.00 40.43           O  \nATOM   1978  CB  LEU B 208       0.230 -17.915  44.768  1.00 39.67           C  \nATOM   1979  CG  LEU B 208       0.497 -19.162  43.924  1.00 38.02           C  \nATOM   1980  CD1 LEU B 208      -0.677 -19.386  42.989  1.00 35.39           C  \nATOM   1981  CD2 LEU B 208       1.772 -19.002  43.127  1.00 36.51           C  \nATOM   1982  N   SER B 209       0.657 -16.043  47.473  1.00 42.21           N  \nATOM   1983  CA  SER B 209       0.351 -14.783  48.145  1.00 44.02           C  \nATOM   1984  C   SER B 209       1.645 -14.072  48.462  1.00 44.83           C  \nATOM   1985  O   SER B 209       1.708 -12.837  48.497  1.00 44.32           O  \nATOM   1986  CB  SER B 209      -0.409 -15.007  49.449  1.00 44.91           C  \nATOM   1987  OG  SER B 209      -1.793 -15.101  49.201  1.00 49.32           O  \nATOM   1988  N   ARG B 210       2.681 -14.861  48.708  1.00 46.54           N  \nATOM   1989  CA  ARG B 210       3.971 -14.284  49.006  1.00 47.43           C  \nATOM   1990  C   ARG B 210       4.619 -13.800  47.716  1.00 44.30           C  \nATOM   1991  O   ARG B 210       5.372 -12.836  47.720  1.00 43.26           O  \nATOM   1992  CB  ARG B 210       4.851 -15.302  49.746  1.00 51.80           C  \nATOM   1993  CG  ARG B 210       4.401 -15.494  51.199  1.00 56.81           C  \nATOM   1994  CD  ARG B 210       5.377 -16.301  52.000  1.00 61.06           C  \nATOM   1995  NE  ARG B 210       5.490 -17.664  51.493  1.00 65.76           N  \nATOM   1996  CZ  ARG B 210       6.631 -18.218  51.094  1.00 67.90           C  \nATOM   1997  NH1 ARG B 210       7.770 -17.528  51.142  1.00 68.34           N  \nATOM   1998  NH2 ARG B 210       6.631 -19.463  50.641  1.00 68.63           N  \nATOM   1999  N   VAL B 211       4.314 -14.452  46.600  1.00 41.30           N  \nATOM   2000  CA  VAL B 211       4.889 -13.995  45.351  1.00 41.06           C  \nATOM   2001  C   VAL B 211       4.184 -12.708  44.933  1.00 41.92           C  \nATOM   2002  O   VAL B 211       4.823 -11.802  44.396  1.00 43.94           O  \nATOM   2003  CB  VAL B 211       4.801 -15.064  44.232  1.00 40.59           C  \nATOM   2004  CG1 VAL B 211       5.263 -14.463  42.890  1.00 38.22           C  \nATOM   2005  CG2 VAL B 211       5.695 -16.239  44.588  1.00 38.55           C  \nATOM   2006  N   ASN B 212       2.880 -12.616  45.194  1.00 41.08           N  \nATOM   2007  CA  ASN B 212       2.146 -11.399  44.866  1.00 40.36           C  \nATOM   2008  C   ASN B 212       2.740 -10.222  45.649  1.00 39.42           C  \nATOM   2009  O   ASN B 212       2.865  -9.124  45.112  1.00 41.26           O  \nATOM   2010  CB  ASN B 212       0.653 -11.540  45.195  1.00 41.23           C  \nATOM   2011  CG  ASN B 212      -0.125 -12.337  44.143  1.00 42.29           C  \nATOM   2012  OD1 ASN B 212      -1.242 -12.778  44.410  1.00 45.53           O  \nATOM   2013  ND2 ASN B 212       0.458 -12.529  42.948  1.00 39.07           N  \nATOM   2014  N   LYS B 213       3.108 -10.437  46.909  1.00 38.66           N  \nATOM   2015  CA  LYS B 213       3.702  -9.364  47.696  1.00 40.12           C  \nATOM   2016  C   LYS B 213       5.080  -8.947  47.160  1.00 40.23           C  \nATOM   2017  O   LYS B 213       5.407  -7.755  47.136  1.00 40.24           O  \nATOM   2018  CB  LYS B 213       3.823  -9.757  49.176  1.00 41.17           C  \nATOM   2019  CG  LYS B 213       2.522  -9.681  49.954  1.00 43.42           C  \nATOM   2020  CD  LYS B 213       2.780  -9.678  51.445  1.00 47.05           C  \nATOM   2021  CE  LYS B 213       1.477  -9.745  52.230  1.00 50.53           C  \nATOM   2022  NZ  LYS B 213       1.690  -9.602  53.717  1.00 50.85           N  \nATOM   2023  N   ARG B 214       5.884  -9.913  46.727  1.00 39.94           N  \nATOM   2024  CA  ARG B 214       7.195  -9.580  46.184  1.00 40.18           C  \nATOM   2025  C   ARG B 214       7.067  -8.828  44.865  1.00 39.26           C  \nATOM   2026  O   ARG B 214       7.896  -7.955  44.567  1.00 41.37           O  \nATOM   2027  CB  ARG B 214       8.053 -10.843  45.991  1.00 42.47           C  \nATOM   2028  CG  ARG B 214       8.622 -11.382  47.295  1.00 47.21           C  \nATOM   2029  CD  ARG B 214       9.504 -10.338  47.980  1.00 51.08           C  \nATOM   2030  NE  ARG B 214      10.677 -10.026  47.165  1.00 54.85           N  \nATOM   2031  CZ  ARG B 214      11.124  -8.798  46.926  1.00 55.57           C  \nATOM   2032  NH1 ARG B 214      10.486  -7.752  47.444  1.00 56.51           N  \nATOM   2033  NH2 ARG B 214      12.201  -8.617  46.166  1.00 53.73           N  \nATOM   2034  N   ILE B 215       6.033  -9.164  44.087  1.00 36.79           N  \nATOM   2035  CA  ILE B 215       5.775  -8.512  42.806  1.00 34.18           C  \nATOM   2036  C   ILE B 215       5.466  -7.015  42.988  1.00 34.04           C  \nATOM   2037  O   ILE B 215       5.958  -6.187  42.238  1.00 32.61           O  \nATOM   2038  CB  ILE B 215       4.554  -9.161  42.046  1.00 33.54           C  \nATOM   2039  CG1 ILE B 215       4.907 -10.557  41.529  1.00 32.05           C  \nATOM   2040  CG2 ILE B 215       4.097  -8.259  40.870  1.00 31.80           C  \nATOM   2041  CD1 ILE B 215       3.736 -11.234  40.771  1.00 29.47           C  \nATOM   2042  N   THR B 216       4.651  -6.667  43.979  1.00 34.78           N  \nATOM   2043  CA  THR B 216       4.314  -5.261  44.148  1.00 38.46           C  \nATOM   2044  C   THR B 216       5.503  -4.509  44.725  1.00 38.91           C  \nATOM   2045  O   THR B 216       5.703  -3.322  44.441  1.00 39.86           O  \nATOM   2046  CB  THR B 216       3.037  -5.064  45.015  1.00 37.63           C  \nATOM   2047  OG1 THR B 216       3.197  -5.740  46.264  1.00 40.09           O  \nATOM   2048  CG2 THR B 216       1.808  -5.653  44.282  1.00 36.02           C  \nATOM   2049  N   GLU B 217       6.314  -5.215  45.501  1.00 38.87           N  \nATOM   2050  CA  GLU B 217       7.503  -4.609  46.051  1.00 41.38           C  \nATOM   2051  C   GLU B 217       8.456  -4.354  44.888  1.00 39.94           C  \nATOM   2052  O   GLU B 217       9.022  -3.272  44.771  1.00 41.07           O  \nATOM   2053  CB  GLU B 217       8.124  -5.535  47.101  1.00 45.46           C  \nATOM   2054  CG  GLU B 217       7.394  -5.444  48.452  1.00 54.21           C  \nATOM   2055  CD  GLU B 217       7.678  -6.602  49.386  1.00 57.60           C  \nATOM   2056  OE1 GLU B 217       8.832  -7.068  49.443  1.00 60.88           O  \nATOM   2057  OE2 GLU B 217       6.739  -7.037  50.077  1.00 60.14           O  \nATOM   2058  N   ALA B 218       8.617  -5.342  44.013  1.00 39.55           N  \nATOM   2059  CA  ALA B 218       9.486  -5.184  42.849  1.00 39.39           C  \nATOM   2060  C   ALA B 218       9.009  -4.053  41.950  1.00 40.22           C  \nATOM   2061  O   ALA B 218       9.824  -3.340  41.383  1.00 40.09           O  \nATOM   2062  CB  ALA B 218       9.561  -6.487  42.055  1.00 38.65           C  \nATOM   2063  N   ASN B 219       7.697  -3.877  41.814  1.00 43.58           N  \nATOM   2064  CA  ASN B 219       7.177  -2.793  40.973  1.00 49.37           C  \nATOM   2065  C   ASN B 219       7.597  -1.438  41.563  1.00 54.51           C  \nATOM   2066  O   ASN B 219       7.919  -0.499  40.828  1.00 54.31           O  \nATOM   2067  CB  ASN B 219       5.644  -2.841  40.881  1.00 46.20           C  \nATOM   2068  CG  ASN B 219       5.122  -3.930  39.948  1.00 44.08           C  \nATOM   2069  OD1 ASN B 219       3.990  -4.371  40.102  1.00 45.47           O  \nATOM   2070  ND2 ASN B 219       5.924  -4.353  38.985  1.00 38.99           N  \nATOM   2071  N   ASN B 220       7.580  -1.343  42.894  1.00 61.56           N  \nATOM   2072  CA  ASN B 220       7.962  -0.115  43.582  1.00 68.71           C  \nATOM   2073  C   ASN B 220       9.456   0.087  43.435  1.00 74.13           C  \nATOM   2074  O   ASN B 220       9.940   1.216  43.371  1.00 76.03           O  \nATOM   2075  CB  ASN B 220       7.616  -0.178  45.077  1.00 68.14           C  \nATOM   2076  CG  ASN B 220       6.119  -0.313  45.340  1.00 68.22           C  \nATOM   2077  OD1 ASN B 220       5.295   0.200  44.591  1.00 68.37           O  \nATOM   2078  ND2 ASN B 220       5.770  -0.987  46.426  1.00 67.03           N  \nATOM   2079  N   GLU B 221      10.186  -1.018  43.372  1.00 81.12           N  \nATOM   2080  CA  GLU B 221      11.631  -0.967  43.241  1.00 87.27           C  \nATOM   2081  C   GLU B 221      12.043  -0.568  41.829  1.00 90.35           C  \nATOM   2082  O   GLU B 221      11.968   0.604  41.469  1.00 89.69           O  \nATOM   2083  CB  GLU B 221      12.228  -2.326  43.607  1.00 89.45           C  \nATOM   2084  CG  GLU B 221      13.739  -2.345  43.700  1.00 94.26           C  \nATOM   2085  CD  GLU B 221      14.272  -3.678  44.195  1.00 97.01           C  \nATOM   2086  OE1 GLU B 221      15.514  -3.821  44.286  1.00 98.84           O  \nATOM   2087  OE2 GLU B 221      13.454  -4.581  44.497  1.00 97.48           O  \nATOM   2088  N   ILE B 222      12.462  -1.552  41.036  1.00 96.19           N  \nATOM   2089  CA  ILE B 222      12.910  -1.344  39.656  1.00101.69           C  \nATOM   2090  C   ILE B 222      12.390  -0.074  38.987  1.00105.08           C  \nATOM   2091  O   ILE B 222      13.160   0.846  38.707  1.00106.90           O  \nATOM   2092  CB  ILE B 222      12.549  -2.565  38.757  1.00100.37           C  \nATOM   2093  CG1 ILE B 222      11.029  -2.754  38.676  1.00 98.27           C  \nATOM   2094  CG2 ILE B 222      13.202  -3.821  39.318  1.00101.96           C  \nATOM   2095  CD1 ILE B 222      10.603  -3.896  37.785  1.00 95.25           C  \nATOM   2096  N   GLU B 223      11.086  -0.033  38.730  1.00108.69           N  \nATOM   2097  CA  GLU B 223      10.446   1.111  38.095  1.00111.94           C  \nATOM   2098  C   GLU B 223      11.390   1.817  37.119  1.00113.84           C  \nATOM   2099  O   GLU B 223      11.461   3.047  37.094  1.00114.24           O  \nATOM   2100  CB  GLU B 223       9.967   2.095  39.166  1.00111.97           C  \nATOM   2101  CG  GLU B 223       8.922   3.083  38.681  1.00112.68           C  \nATOM   2102  CD  GLU B 223       8.509   4.067  39.758  1.00113.04           C  \nATOM   2103  OE1 GLU B 223       8.142   3.618  40.866  1.00112.24           O  \nATOM   2104  OE2 GLU B 223       8.550   5.288  39.493  1.00113.98           O  \nATOM   2105  N   LEU B 224      12.118   1.037  36.321  1.00116.21           N  \nATOM   2106  CA  LEU B 224      13.051   1.601  35.347  1.00118.33           C  \nATOM   2107  C   LEU B 224      12.294   2.075  34.107  1.00119.11           C  \nATOM   2108  O   LEU B 224      11.146   1.619  33.913  1.00119.43           O  \nATOM   2109  CB  LEU B 224      14.104   0.554  34.945  1.00118.38           C  \nATOM   2110  CG  LEU B 224      15.195   0.962  33.937  1.00118.07           C  \nATOM   2111  CD1 LEU B 224      16.054   2.083  34.516  1.00117.50           C  \nATOM   2112  CD2 LEU B 224      16.067  -0.244  33.599  1.00116.37           C  \nATOM   2113  OXT LEU B 224      12.859   2.887  33.340  1.00119.76           O  \nTER    2114      LEU B 224                                                      \nHETATM 2115  CA  CA  A 501      -4.855  -8.309  40.205  0.50 53.27          CA  \nHETATM 2116  CA  CA  B 601       4.399   2.606  20.437  0.50 54.01          CA  \nEND                                                                             \n","pdb");
	viewer_1758171620800664.setStyle({"model": 1},{"cartoon": {"color": "red"}});
	viewer_1758171620800664.zoomTo();
	viewer_1758171620800664.zoom(1.2);
	viewer_1758171620800664.setBackgroundColor("white");
viewer_1758171620800664.render();
});
</script>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=0d6dc04a">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="AlphaFold-predictions">AlphaFold predictions<a class="anchor-link" href="#AlphaFold-predictions"></a></h2>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=c8628fb7">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-AF-prediction-to-fold-1">Best AF prediction to fold 1<a class="anchor-link" href="#Best-AF-prediction-to-fold-1"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=e7c9b423">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[8]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="kn">from</span><span class="w"> </span><span class="nn">pathlib</span><span class="w"> </span><span class="kn">import</span> <span class="n">Path</span>

<span class="n">df_af</span> <span class="o">=</span> <span class="n">load_csv_or_none</span><span class="p">(</span><span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">/Analysis/df_af.csv"</span><span class="p">)</span>
<span class="k">if</span> <span class="n">df_af</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
    <span class="c1"># keep your existing filters</span>
    <span class="n">df_af</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">cluster_num</span> <span class="o">!=</span> <span class="s1">'Query'</span><span class="p">]</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span> <span class="mi">1</span><span class="p">:</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span>
    <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[run] AlphaFold df ready: </span><span class="si">{</span><span class="nb">len</span><span class="p">(</span><span class="n">df_af</span><span class="p">)</span><span class="si">}</span><span class="s2"> rows"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] AlphaFold section (df_af.csv not available)"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[ok] loaded: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1fzpD_2frhA/Analysis/df_af.csv  rows=28
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[run] AlphaFold df ready: 28 rows
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=03ee9989">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[9]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">MAX_AF_1</span><span class="p">,</span> <span class="n">max_af_pdb1</span> <span class="o">=</span> <span class="s1">''</span><span class="p">,</span> <span class="s1">''</span>
<span class="k">if</span> <span class="n">df_af</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_af</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">max_af_pdb1</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&gt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb1'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">pdb_file</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">max_af_pdb1</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/AF_preds/</span><span class="si">{</span><span class="n">max_af_pdb1</span><span class="si">}</span><span class="s1">'</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">MAX_AF_1</span> <span class="o">=</span> <span class="n">max_af_pdb1</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s1">'/'</span><span class="p">)[</span><span class="o">-</span><span class="mi">1</span><span class="p">][</span><span class="mi">11</span><span class="p">:</span><span class="mi">14</span><span class="p">]</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="n">e</span><span class="p">)</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&gt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb1'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">score_pdb1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">max_af_pdb1</span><span class="p">,</span><span class="n">pdb_path1</span><span class="p">,</span><span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 alignment: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">max_af_pdb1</span><span class="p">,</span><span class="n">pdb_path1</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e3</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 fallback visualization: </span><span class="si">{</span><span class="n">e3</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] AF fold1: df_af not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] AF fold1 alignment: 'DataFrame' object has no attribute 'score_pdb1'
[skip] AF fold1 fallback visualization:  is not a valid filename or a valid PDB identifier.
</pre>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=5584ce3b">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-AF-prediction-to-fold-2">Best AF prediction to fold 2<a class="anchor-link" href="#Best-AF-prediction-to-fold-2"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=5473c6cd">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[10]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">MAX_AF_2</span><span class="p">,</span> <span class="n">max_af_pdb2</span> <span class="o">=</span> <span class="s1">''</span><span class="p">,</span> <span class="s1">''</span>
<span class="k">if</span> <span class="n">df_af</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_af</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">max_af_pdb2</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&lt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb2'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">pdb_file</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">max_af_pdb2</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/AF_preds/</span><span class="si">{</span><span class="n">max_af_pdb2</span><span class="si">}</span><span class="s1">'</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">MAX_AF_2</span> <span class="o">=</span> <span class="n">max_af_pdb2</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s1">'/'</span><span class="p">)[</span><span class="o">-</span><span class="mi">1</span><span class="p">][</span><span class="mi">11</span><span class="p">:</span><span class="mi">14</span><span class="p">]</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="n">e</span><span class="p">)</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&lt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb2'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">score_pdb2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">max_af_pdb2</span><span class="p">,</span><span class="n">pdb_path2</span><span class="p">,</span><span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 alignment: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">max_af_pdb2</span><span class="p">,</span><span class="n">pdb_path2</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e3</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 fallback visualization: </span><span class="si">{</span><span class="n">e3</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] AF fold1: df_af not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] AF fold1 alignment: 'DataFrame' object has no attribute 'score_pdb1'
[skip] AF fold1 fallback visualization:  is not a valid filename or a valid PDB identifier.
</pre>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=a817e741">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="EsmFold-Prediction">EsmFold Prediction<a class="anchor-link" href="#EsmFold-Prediction"></a></h2>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=d914376c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[11]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="kn">from</span><span class="w"> </span><span class="nn">pathlib</span><span class="w"> </span><span class="kn">import</span> <span class="n">Path</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">os</span><span class="o">,</span><span class="w"> </span><span class="nn">pandas</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">pd</span>

<span class="c1"># Inputs from the generator (below patch passes these env vars)</span>
<span class="n">ROOT</span> <span class="o">=</span> <span class="n">Path</span><span class="p">(</span><span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s2">"MSACLUSTER_ROOT"</span><span class="p">,</span> <span class="s2">"."</span><span class="p">))</span><span class="o">.</span><span class="n">resolve</span><span class="p">()</span>
<span class="n">PAIR_ID</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s2">"PAIR_ID"</span><span class="p">,</span> <span class="kc">None</span><span class="p">)</span> <span class="ow">or</span> <span class="n">fold_pair</span>  <span class="c1"># keep using fold_pair if already defined</span>

<span class="n">PAIR_ANALYSIS</span> <span class="o">=</span> <span class="n">ROOT</span> <span class="o">/</span> <span class="s2">"Pipeline"</span> <span class="o">/</span> <span class="n">PAIR_ID</span> <span class="o">/</span> <span class="s2">"Analysis"</span>
<span class="n">GLOBAL_RES</span>    <span class="o">=</span> <span class="n">ROOT</span> <span class="o">/</span> <span class="s2">"Pipeline_res"</span>

<span class="k">def</span><span class="w"> </span><span class="nf">_read_csv_or_none</span><span class="p">(</span><span class="n">p</span><span class="p">:</span> <span class="n">Path</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">pd</span><span class="o">.</span><span class="n">read_csv</span><span class="p">(</span><span class="n">p</span><span class="p">)</span> <span class="k">if</span> <span class="n">p</span><span class="o">.</span><span class="n">exists</span><span class="p">()</span> <span class="ow">and</span> <span class="n">p</span><span class="o">.</span><span class="n">stat</span><span class="p">()</span><span class="o">.</span><span class="n">st_size</span> <span class="o">&gt;</span> <span class="mi">1</span> <span class="k">else</span> <span class="kc">None</span>

<span class="c1"># Prefer per-pair CSV; only fall back to global CSV if really needed</span>
<span class="n">df_esmfold_analysis</span> <span class="o">=</span> <span class="n">_read_csv_or_none</span><span class="p">(</span><span class="n">PAIR_ANALYSIS</span> <span class="o">/</span> <span class="s2">"df_esm.csv"</span><span class="p">)</span>
<span class="k">if</span> <span class="n">df_esmfold_analysis</span> <span class="ow">is</span> <span class="kc">None</span><span class="p">:</span>
    <span class="n">df_esmfold_analysis</span> <span class="o">=</span> <span class="n">_read_csv_or_none</span><span class="p">(</span><span class="n">GLOBAL_RES</span> <span class="o">/</span> <span class="s2">"df_esmfold_analysis.csv"</span><span class="p">)</span>

<span class="n">df_esmfold</span> <span class="o">=</span> <span class="kc">None</span>
<span class="k">if</span> <span class="n">df_esmfold_analysis</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
    <span class="c1"># Your original filtering logic</span>
    <span class="n">df_esmfold</span> <span class="o">=</span> <span class="n">df_esmfold_analysis</span><span class="p">[</span><span class="n">df_esmfold_analysis</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s2">"fold_pair"</span><span class="p">,</span> <span class="n">PAIR_ID</span><span class="p">)</span> <span class="o">==</span> <span class="n">PAIR_ID</span><span class="p">]</span>
    <span class="k">if</span> <span class="s2">"fold"</span> <span class="ow">in</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">columns</span><span class="p">:</span>
        <span class="n">df_esmfold</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="p">[</span><span class="n">df_esmfold</span><span class="p">[</span><span class="s2">"fold"</span><span class="p">]</span><span class="o">.</span><span class="n">astype</span><span class="p">(</span><span class="nb">str</span><span class="p">)</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="s2">"ShallowMsa"</span><span class="p">)]</span>
    <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[run] ESMFold df ready: </span><span class="si">{</span><span class="nb">len</span><span class="p">(</span><span class="n">df_esmfold</span><span class="p">)</span><span class="si">}</span><span class="s2"> rows"</span><span class="p">)</span>
    <span class="n">display</span><span class="p">(</span><span class="n">df_esmfold</span><span class="o">.</span><span class="n">head</span><span class="p">())</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESMFold section (no per-pair df_esm.csv and no global df_esmfold_analysis.csv)"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[run] ESMFold df ready: 120 rows
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div>
<style scoped="">
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
<thead>
<tr style="text-align: right;">
<th></th>
<th>fold_pair</th>
<th>model</th>
<th>cluster_num</th>
<th>name</th>
<th>TMscore_fold1</th>
<th>TMscore_fold2</th>
<th>TMdiff</th>
</tr>
</thead>
<tbody>
<tr>
<th>0</th>
<td>1fzpD_2frhA</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_001</td>
<td>0.561660</td>
<td>0.736273</td>
<td>-0.174613</td>
</tr>
<tr>
<th>1</th>
<td>1fzpD_2frhA</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_002</td>
<td>0.477546</td>
<td>0.598404</td>
<td>-0.120858</td>
</tr>
<tr>
<th>2</th>
<td>1fzpD_2frhA</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_003</td>
<td>0.518182</td>
<td>0.723642</td>
<td>-0.205460</td>
</tr>
<tr>
<th>3</th>
<td>1fzpD_2frhA</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_004</td>
<td>0.428388</td>
<td>0.520875</td>
<td>-0.092487</td>
</tr>
<tr>
<th>4</th>
<td>1fzpD_2frhA</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_005</td>
<td>0.491631</td>
<td>0.621712</td>
<td>-0.130081</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=2ad0b9e2">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-EsmFold-prediction-to-fold-1">Best EsmFold prediction to fold 1<a class="anchor-link" href="#Best-EsmFold-prediction-to-fold-1"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=c9be45fc">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[12]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">if</span> <span class="n">df_esmfold</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="c1"># keep your original selection logic here (fixed snippets below are placeholders)</span>
        <span class="n">max_esm_pdb1</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold1'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">fold</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold1'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">TMscore_fold1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb1</span><span class="si">}</span><span class="s1">'</span><span class="p">,</span> <span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold1 align/plot: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb1</span><span class="si">}</span><span class="s1">'</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold1 fallback visualization: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESM fold1: df_esmfold not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] ESM fold1 align/plot: 'DataFrame' object has no attribute 'fold'
[skip] ESM fold1 fallback visualization: name 'max_esm_pdb1' is not defined
</pre>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=1c4279db">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-EsmFold-prediction-to-fold-2">Best EsmFold prediction to fold 2<a class="anchor-link" href="#Best-EsmFold-prediction-to-fold-2"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=3f4fda59">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[13]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">if</span> <span class="n">df_esmfold</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">max_esm_pdb2</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold2'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">fold</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold2'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">TMscore_fold2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb2</span><span class="si">}</span><span class="s1">'</span><span class="p">,</span> <span class="n">score</span><span class="p">)</span>
        <span class="nb">print</span><span class="p">(</span><span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold2 align/plot: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="n">score</span><span class="p">)</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb2</span><span class="si">}</span><span class="s1">'</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold2 fallback visualization: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESM fold2: df_esmfold not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] ESM fold2 align/plot: 'DataFrame' object has no attribute 'fold'
[skip] ESM fold2 fallback visualization: name 'score' is not defined
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=4daa9929">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[14]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">if</span> <span class="n">df_esmfold</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
    <span class="n">df_esmfold</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESM df display (not available)"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=caed9007">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="Contact-maps-Msa-Transformers">Contact maps Msa Transformers<a class="anchor-link" href="#Contact-maps-Msa-Transformers"></a></h2>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=56fbbee9">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[15]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="c1"># Robust load of CMap CSV referenced by CMAP_RES_PATH (if defined)</span>
<span class="n">df_cmap</span> <span class="o">=</span> <span class="kc">None</span>
<span class="k">try</span><span class="p">:</span>
    <span class="n">CMAP_RES_PATH</span>  <span class="c1"># just to see if it exists</span>
    <span class="n">df_cmap</span> <span class="o">=</span> <span class="n">load_csv_or_none</span><span class="p">(</span><span class="n">CMAP_RES_PATH</span><span class="p">)</span>
<span class="k">except</span> <span class="ne">NameError</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] CMAP_RES_PATH is not defined in this notebook"</span><span class="p">)</span>

<span class="n">MAX_RECALL_1</span> <span class="o">=</span> <span class="n">MAX_RECALL_2</span> <span class="o">=</span> <span class="kc">None</span>
<span class="n">MAX_CLUSTER_RECALL_1</span> <span class="o">=</span> <span class="n">MAX_CLUSTER_RECALL_2</span> <span class="o">=</span> <span class="s1">''</span>

<span class="k">if</span> <span class="n">df_cmap</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="n">df_cmap</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">fold_pair</span> <span class="o">==</span> <span class="n">fold_pair</span><span class="p">]</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">MAX_RECALL_1</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold1</span> <span class="o">&gt;</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="p">]</span> \
            <span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'recall_only_fold1'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">MAX_CLUSTER_RECALL_1</span> <span class="o">=</span> <span class="n">MAX_RECALL_1</span><span class="o">.</span><span class="n">File</span><span class="p">[</span><span class="o">-</span><span class="mi">7</span><span class="p">:</span><span class="o">-</span><span class="mi">4</span><span class="p">]</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] MAX_RECALL_1: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>

    <span class="k">try</span><span class="p">:</span>
        <span class="n">MAX_RECALL_2</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold1</span> <span class="o">&lt;</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="p">]</span> \
            <span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'recall_only_fold2'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">MAX_CLUSTER_RECALL_2</span> <span class="o">=</span> <span class="n">MAX_RECALL_2</span><span class="o">.</span><span class="n">File</span><span class="p">[</span><span class="o">-</span><span class="mi">7</span><span class="p">:</span><span class="o">-</span><span class="mi">4</span><span class="p">]</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] MAX_RECALL_2: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] df_cmap not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] missing CSV: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/data/df_cmap_all.csv
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] df_cmap not available
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=d06b9bea">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[16]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">viz_folder</span> <span class="o">=</span> <span class="n">VIZ_FOLDER</span>
<span class="k">if</span> <span class="n">MAX_CLUSTER_RECALL_1</span> <span class="o">!=</span> <span class="s1">''</span><span class="p">:</span>
    <span class="n">file_max_recall_1</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">viz_folder</span><span class="si">}</span><span class="s1">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_CLUSTER_RECALL_1</span><span class="si">}</span><span class="s1">_visualization_map_1_tol_0.npy'</span>
<span class="k">else</span><span class="p">:</span>
    <span class="n">file_max_recall_1</span> <span class="o">=</span> <span class="s1">''</span>

<span class="k">if</span> <span class="n">MAX_CLUSTER_RECALL_2</span> <span class="o">!=</span> <span class="s1">''</span><span class="p">:</span>
    <span class="n">file_max_recall_2</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">viz_folder</span><span class="si">}</span><span class="s1">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_CLUSTER_RECALL_2</span><span class="si">}</span><span class="s1">_visualization_map_2_tol_0.npy'</span>
<span class="k">else</span><span class="p">:</span>
    <span class="n">file_max_recall_2</span> <span class="o">=</span> <span class="s1">''</span>
</pre></div>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=4c68d72c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span> 
</pre></div>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=69a04a8c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[17]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="c1"># build AF npy path explicitly and debug-print</span>
<span class="n">af1_npy</span> <span class="o">=</span> <span class="p">(</span><span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">VIZ_FOLDER</span><span class="si">}</span><span class="s2">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_AF_1</span><span class="si">}</span><span class="s2">_visualization_map_1_tol_0.npy"</span>
    <span class="k">if</span> <span class="n">MAX_AF_1</span> <span class="k">else</span> <span class="s2">""</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"VIZ_FOLDER:"</span><span class="p">,</span> <span class="n">VIZ_FOLDER</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"file_max_recall_1:"</span><span class="p">,</span> <span class="n">file_max_recall_1</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">),</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"af1_npy:"</span><span class="p">,</span> <span class="n">af1_npy</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">)</span> <span class="k">if</span> <span class="n">af1_npy</span> <span class="k">else</span> <span class="kc">None</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"fold_pair:"</span><span class="p">,</span> <span class="n">fold_pair</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"viz_folder:"</span><span class="p">,</span> <span class="n">viz_folder</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"MAX_CLUSTER_RECALL_1:"</span><span class="p">,</span> <span class="n">MAX_CLUSTER_RECALL_1</span><span class="p">,</span> <span class="s2">"MAX_CLUSTER_RECALL_2:"</span><span class="p">,</span> <span class="n">MAX_CLUSTER_RECALL_2</span><span class="p">)</span>


<span class="k">if</span> <span class="n">df_cmap</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">file_max_recall_1</span> <span class="ow">and</span> <span class="n">af1_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">)</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_1</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap2</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">,</span> <span class="n">af1_npy</span><span class="p">,</span>
                       <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 1 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_1</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold1"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">) '</span>
                                   <span class="sa">f</span><span class="s1">'Vs Best AF 1 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">file_max_recall_1</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">):</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">,</span>
                      <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 1 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_1</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold1"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">af1_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_1</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">,</span> <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best AF 1 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">else</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] fold1 viz: missing both maps"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] cmap fold1 viz: df_cmap not available"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>VIZ_FOLDER: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1fzpD_2frhA/output_cmap_esm/VizCmaps
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>file_max_recall_1:  exists? False
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>af1_npy:  exists? None
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>fold_pair: 1fzpD_2frhA
viz_folder: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1fzpD_2frhA/output_cmap_esm/VizCmaps
MAX_CLUSTER_RECALL_1:  MAX_CLUSTER_RECALL_2: 
[skip] cmap fold1 viz: df_cmap not available
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=a4dd4c44">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[18]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">af2_npy</span> <span class="o">=</span> <span class="p">(</span><span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">VIZ_FOLDER</span><span class="si">}</span><span class="s2">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_AF_2</span><span class="si">}</span><span class="s2">_visualization_map_2_tol_0.npy"</span>
    <span class="k">if</span> <span class="n">MAX_AF_2</span> <span class="k">else</span> <span class="s2">""</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"VIZ_FOLDER:"</span><span class="p">,</span> <span class="n">VIZ_FOLDER</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"file_max_recall_2:"</span><span class="p">,</span> <span class="n">file_max_recall_2</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">),</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"af2_npy:"</span><span class="p">,</span> <span class="n">af2_npy</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">)</span> <span class="k">if</span> <span class="n">af2_npy</span> <span class="k">else</span> <span class="kc">None</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="k">if</span> <span class="n">df_cmap</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">file_max_recall_2</span> <span class="ow">and</span> <span class="n">af2_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">)</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_2</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap2</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">,</span> <span class="n">af2_npy</span><span class="p">,</span>
                       <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 2 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_2</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold2"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">) '</span>
                                   <span class="sa">f</span><span class="s1">'Vs Best AF 2 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">file_max_recall_2</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">):</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">,</span>
                      <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 2 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_2</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold2"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">af2_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_2</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">,</span> <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best AF 2 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">else</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] fold2 viz: missing both maps"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] cmap fold2 viz: df_cmap not available"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>VIZ_FOLDER: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1fzpD_2frhA/output_cmap_esm/VizCmaps
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>file_max_recall_2:  exists? False
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>af2_npy:  exists? None
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] cmap fold2 viz: df_cmap not available
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=b0112a5c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span> 
</pre></div>
</div>
</div>
</div>
</div>
</div>
</main>
</body>
</html>
