<!DOCTYPE html>

<html lang="en">
<head><meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>TemplateNotebook</title><script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
<style type="text/css">
    pre { line-height: 125%; }
td.linenos .normal { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
span.linenos { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
td.linenos .special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
span.linenos.special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
.highlight .hll { background-color: var(--jp-cell-editor-active-background) }
.highlight { background: var(--jp-cell-editor-background); color: var(--jp-mirror-editor-variable-color) }
.highlight .c { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment */
.highlight .err { color: var(--jp-mirror-editor-error-color) } /* Error */
.highlight .k { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword */
.highlight .o { color: var(--jp-mirror-editor-operator-color); font-weight: bold } /* Operator */
.highlight .p { color: var(--jp-mirror-editor-punctuation-color) } /* Punctuation */
.highlight .ch { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Hashbang */
.highlight .cm { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Multiline */
.highlight .cp { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Preproc */
.highlight .cpf { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.PreprocFile */
.highlight .c1 { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Single */
.highlight .cs { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Special */
.highlight .kc { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Constant */
.highlight .kd { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Declaration */
.highlight .kn { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Namespace */
.highlight .kp { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Pseudo */
.highlight .kr { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Reserved */
.highlight .kt { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Type */
.highlight .m { color: var(--jp-mirror-editor-number-color) } /* Literal.Number */
.highlight .s { color: var(--jp-mirror-editor-string-color) } /* Literal.String */
.highlight .ow { color: var(--jp-mirror-editor-operator-color); font-weight: bold } /* Operator.Word */
.highlight .pm { color: var(--jp-mirror-editor-punctuation-color) } /* Punctuation.Marker */
.highlight .w { color: var(--jp-mirror-editor-variable-color) } /* Text.Whitespace */
.highlight .mb { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Bin */
.highlight .mf { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Float */
.highlight .mh { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Hex */
.highlight .mi { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Integer */
.highlight .mo { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Oct */
.highlight .sa { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Affix */
.highlight .sb { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Backtick */
.highlight .sc { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Char */
.highlight .dl { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Delimiter */
.highlight .sd { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Doc */
.highlight .s2 { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Double */
.highlight .se { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Escape */
.highlight .sh { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Heredoc */
.highlight .si { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Interpol */
.highlight .sx { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Other */
.highlight .sr { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Regex */
.highlight .s1 { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Single */
.highlight .ss { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Symbol */
.highlight .il { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Integer.Long */
  </style>
<style type="text/css">
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*
 * Mozilla scrollbar styling
 */

/* use standard opaque scrollbars for most nodes */
[data-jp-theme-scrollbars='true'] {
  scrollbar-color: rgb(var(--jp-scrollbar-thumb-color))
    var(--jp-scrollbar-background-color);
}

/* for code nodes, use a transparent style of scrollbar. These selectors
 * will match lower in the tree, and so will override the above */
[data-jp-theme-scrollbars='true'] .CodeMirror-hscrollbar,
[data-jp-theme-scrollbars='true'] .CodeMirror-vscrollbar {
  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
}

/* tiny scrollbar */

.jp-scrollbar-tiny {
  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
  scrollbar-width: thin;
}

/* tiny scrollbar */

.jp-scrollbar-tiny::-webkit-scrollbar,
.jp-scrollbar-tiny::-webkit-scrollbar-corner {
  background-color: transparent;
  height: 4px;
  width: 4px;
}

.jp-scrollbar-tiny::-webkit-scrollbar-thumb {
  background: rgba(var(--jp-scrollbar-thumb-color), 0.5);
}

.jp-scrollbar-tiny::-webkit-scrollbar-track:horizontal {
  border-left: 0 solid transparent;
  border-right: 0 solid transparent;
}

.jp-scrollbar-tiny::-webkit-scrollbar-track:vertical {
  border-top: 0 solid transparent;
  border-bottom: 0 solid transparent;
}

/*
 * Lumino
 */

.lm-ScrollBar[data-orientation='horizontal'] {
  min-height: 16px;
  max-height: 16px;
  min-width: 45px;
  border-top: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='vertical'] {
  min-width: 16px;
  max-width: 16px;
  min-height: 45px;
  border-left: 1px solid #a0a0a0;
}

.lm-ScrollBar-button {
  background-color: #f0f0f0;
  background-position: center center;
  min-height: 15px;
  max-height: 15px;
  min-width: 15px;
  max-width: 15px;
}

.lm-ScrollBar-button:hover {
  background-color: #dadada;
}

.lm-ScrollBar-button.lm-mod-active {
  background-color: #cdcdcd;
}

.lm-ScrollBar-track {
  background: #f0f0f0;
}

.lm-ScrollBar-thumb {
  background: #cdcdcd;
}

.lm-ScrollBar-thumb:hover {
  background: #bababa;
}

.lm-ScrollBar-thumb.lm-mod-active {
  background: #a0a0a0;
}

.lm-ScrollBar[data-orientation='horizontal'] .lm-ScrollBar-thumb {
  height: 100%;
  min-width: 15px;
  border-left: 1px solid #a0a0a0;
  border-right: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='vertical'] .lm-ScrollBar-thumb {
  width: 100%;
  min-height: 15px;
  border-top: 1px solid #a0a0a0;
  border-bottom: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='horizontal']
  .lm-ScrollBar-button[data-action='decrement'] {
  background-image: var(--jp-icon-caret-left);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='horizontal']
  .lm-ScrollBar-button[data-action='increment'] {
  background-image: var(--jp-icon-caret-right);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='vertical']
  .lm-ScrollBar-button[data-action='decrement'] {
  background-image: var(--jp-icon-caret-up);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='vertical']
  .lm-ScrollBar-button[data-action='increment'] {
  background-image: var(--jp-icon-caret-down);
  background-size: 17px;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-Widget {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
}

.lm-Widget.lm-mod-hidden {
  display: none !important;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.lm-AccordionPanel[data-orientation='horizontal'] > .lm-AccordionPanel-title {
  /* Title is rotated for horizontal accordion panel using CSS */
  display: block;
  transform-origin: top left;
  transform: rotate(-90deg) translate(-100%);
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-CommandPalette {
  display: flex;
  flex-direction: column;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-CommandPalette-search {
  flex: 0 0 auto;
}

.lm-CommandPalette-content {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  min-height: 0;
  overflow: auto;
  list-style-type: none;
}

.lm-CommandPalette-header {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.lm-CommandPalette-item {
  display: flex;
  flex-direction: row;
}

.lm-CommandPalette-itemIcon {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemContent {
  flex: 1 1 auto;
  overflow: hidden;
}

.lm-CommandPalette-itemShortcut {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemLabel {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.lm-close-icon {
  border: 1px solid transparent;
  background-color: transparent;
  position: absolute;
  z-index: 1;
  right: 3%;
  top: 0;
  bottom: 0;
  margin: auto;
  padding: 7px 0;
  display: none;
  vertical-align: middle;
  outline: 0;
  cursor: pointer;
}
.lm-close-icon:after {
  content: 'X';
  display: block;
  width: 15px;
  height: 15px;
  text-align: center;
  color: #000;
  font-weight: normal;
  font-size: 12px;
  cursor: pointer;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-DockPanel {
  z-index: 0;
}

.lm-DockPanel-widget {
  z-index: 0;
}

.lm-DockPanel-tabBar {
  z-index: 1;
}

.lm-DockPanel-handle {
  z-index: 2;
}

.lm-DockPanel-handle.lm-mod-hidden {
  display: none !important;
}

.lm-DockPanel-handle:after {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  content: '';
}

.lm-DockPanel-handle[data-orientation='horizontal'] {
  cursor: ew-resize;
}

.lm-DockPanel-handle[data-orientation='vertical'] {
  cursor: ns-resize;
}

.lm-DockPanel-handle[data-orientation='horizontal']:after {
  left: 50%;
  min-width: 8px;
  transform: translateX(-50%);
}

.lm-DockPanel-handle[data-orientation='vertical']:after {
  top: 50%;
  min-height: 8px;
  transform: translateY(-50%);
}

.lm-DockPanel-overlay {
  z-index: 3;
  box-sizing: border-box;
  pointer-events: none;
}

.lm-DockPanel-overlay.lm-mod-hidden {
  display: none !important;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-Menu {
  z-index: 10000;
  position: absolute;
  white-space: nowrap;
  overflow-x: hidden;
  overflow-y: auto;
  outline: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-Menu-content {
  margin: 0;
  padding: 0;
  display: table;
  list-style-type: none;
}

.lm-Menu-item {
  display: table-row;
}

.lm-Menu-item.lm-mod-hidden,
.lm-Menu-item.lm-mod-collapsed {
  display: none !important;
}

.lm-Menu-itemIcon,
.lm-Menu-itemSubmenuIcon {
  display: table-cell;
  text-align: center;
}

.lm-Menu-itemLabel {
  display: table-cell;
  text-align: left;
}

.lm-Menu-itemShortcut {
  display: table-cell;
  text-align: right;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-MenuBar {
  outline: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-MenuBar-content {
  margin: 0;
  padding: 0;
  display: flex;
  flex-direction: row;
  list-style-type: none;
}

.lm-MenuBar-item {
  box-sizing: border-box;
}

.lm-MenuBar-itemIcon,
.lm-MenuBar-itemLabel {
  display: inline-block;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-ScrollBar {
  display: flex;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-ScrollBar[data-orientation='horizontal'] {
  flex-direction: row;
}

.lm-ScrollBar[data-orientation='vertical'] {
  flex-direction: column;
}

.lm-ScrollBar-button {
  box-sizing: border-box;
  flex: 0 0 auto;
}

.lm-ScrollBar-track {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
  flex: 1 1 auto;
}

.lm-ScrollBar-thumb {
  box-sizing: border-box;
  position: absolute;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-SplitPanel-child {
  z-index: 0;
}

.lm-SplitPanel-handle {
  z-index: 1;
}

.lm-SplitPanel-handle.lm-mod-hidden {
  display: none !important;
}

.lm-SplitPanel-handle:after {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  content: '';
}

.lm-SplitPanel[data-orientation='horizontal'] > .lm-SplitPanel-handle {
  cursor: ew-resize;
}

.lm-SplitPanel[data-orientation='vertical'] > .lm-SplitPanel-handle {
  cursor: ns-resize;
}

.lm-SplitPanel[data-orientation='horizontal'] > .lm-SplitPanel-handle:after {
  left: 50%;
  min-width: 8px;
  transform: translateX(-50%);
}

.lm-SplitPanel[data-orientation='vertical'] > .lm-SplitPanel-handle:after {
  top: 50%;
  min-height: 8px;
  transform: translateY(-50%);
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-TabBar {
  display: flex;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.lm-TabBar[data-orientation='horizontal'] {
  flex-direction: row;
  align-items: flex-end;
}

.lm-TabBar[data-orientation='vertical'] {
  flex-direction: column;
  align-items: flex-end;
}

.lm-TabBar-content {
  margin: 0;
  padding: 0;
  display: flex;
  flex: 1 1 auto;
  list-style-type: none;
}

.lm-TabBar[data-orientation='horizontal'] > .lm-TabBar-content {
  flex-direction: row;
}

.lm-TabBar[data-orientation='vertical'] > .lm-TabBar-content {
  flex-direction: column;
}

.lm-TabBar-tab {
  display: flex;
  flex-direction: row;
  box-sizing: border-box;
  overflow: hidden;
  touch-action: none; /* Disable native Drag/Drop */
}

.lm-TabBar-tabIcon,
.lm-TabBar-tabCloseIcon {
  flex: 0 0 auto;
}

.lm-TabBar-tabLabel {
  flex: 1 1 auto;
  overflow: hidden;
  white-space: nowrap;
}

.lm-TabBar-tabInput {
  user-select: all;
  width: 100%;
  box-sizing: border-box;
}

.lm-TabBar-tab.lm-mod-hidden {
  display: none !important;
}

.lm-TabBar-addButton.lm-mod-hidden {
  display: none !important;
}

.lm-TabBar.lm-mod-dragging .lm-TabBar-tab {
  position: relative;
}

.lm-TabBar.lm-mod-dragging[data-orientation='horizontal'] .lm-TabBar-tab {
  left: 0;
  transition: left 150ms ease;
}

.lm-TabBar.lm-mod-dragging[data-orientation='vertical'] .lm-TabBar-tab {
  top: 0;
  transition: top 150ms ease;
}

.lm-TabBar.lm-mod-dragging .lm-TabBar-tab.lm-mod-dragging {
  transition: none;
}

.lm-TabBar-tabLabel .lm-TabBar-tabInput {
  user-select: all;
  width: 100%;
  box-sizing: border-box;
  background: inherit;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-TabPanel-tabBar {
  z-index: 1;
}

.lm-TabPanel-stackedPanel {
  z-index: 0;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Collapse {
  display: flex;
  flex-direction: column;
  align-items: stretch;
}

.jp-Collapse-header {
  padding: 1px 12px;
  background-color: var(--jp-layout-color1);
  border-bottom: solid var(--jp-border-width) var(--jp-border-color2);
  color: var(--jp-ui-font-color1);
  cursor: pointer;
  display: flex;
  align-items: center;
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  text-transform: uppercase;
  user-select: none;
}

.jp-Collapser-icon {
  height: 16px;
}

.jp-Collapse-header-collapsed .jp-Collapser-icon {
  transform: rotate(-90deg);
  margin: auto 0;
}

.jp-Collapser-title {
  line-height: 25px;
}

.jp-Collapse-contents {
  padding: 0 12px;
  background-color: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  overflow: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/* This file was auto-generated by ensureUiComponents() in @jupyterlab/buildutils */

/**
 * (DEPRECATED) Support for consuming icons as CSS background images
 */

/* Icons urls */

:root {
  --jp-icon-add-above: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGcgY2xpcC1wYXRoPSJ1cmwoI2NsaXAwXzEzN18xOTQ5MikiPgo8cGF0aCBjbGFzcz0ianAtaWNvbjMiIGQ9Ik00Ljc1IDQuOTMwNjZINi42MjVWNi44MDU2NkM2LjYyNSA3LjAxMTkxIDYuNzkzNzUgNy4xODA2NiA3IDcuMTgwNjZDNy4yMDYyNSA3LjE4MDY2IDcuMzc1IDcuMDExOTEgNy4zNzUgNi44MDU2NlY0LjkzMDY2SDkuMjVDOS40NTYyNSA0LjkzMDY2IDkuNjI1IDQuNzYxOTEgOS42MjUgNC41NTU2NkM5LjYyNSA0LjM0OTQxIDkuNDU2MjUgNC4xODA2NiA5LjI1IDQuMTgwNjZINy4zNzVWMi4zMDU2NkM3LjM3NSAyLjA5OTQxIDcuMjA2MjUgMS45MzA2NiA3IDEuOTMwNjZDNi43OTM3NSAxLjkzMDY2IDYuNjI1IDIuMDk5NDEgNi42MjUgMi4zMDU2NlY0LjE4MDY2SDQuNzVDNC41NDM3NSA0LjE4MDY2IDQuMzc1IDQuMzQ5NDEgNC4zNzUgNC41NTU2NkM0LjM3NSA0Ljc2MTkxIDQuNTQzNzUgNC45MzA2NiA0Ljc1IDQuOTMwNjZaIiBmaWxsPSIjNjE2MTYxIiBzdHJva2U9IiM2MTYxNjEiIHN0cm9rZS13aWR0aD0iMC43Ii8+CjwvZz4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTExLjUgOS41VjExLjVMMi41IDExLjVWOS41TDExLjUgOS41Wk0xMiA4QzEyLjU1MjMgOCAxMyA4LjQ0NzcyIDEzIDlWMTJDMTMgMTIuNTUyMyAxMi41NTIzIDEzIDEyIDEzTDIgMTNDMS40NDc3MiAxMyAxIDEyLjU1MjMgMSAxMlY5QzEgOC40NDc3MiAxLjQ0NzcxIDggMiA4TDEyIDhaIiBmaWxsPSIjNjE2MTYxIi8+CjxkZWZzPgo8Y2xpcFBhdGggaWQ9ImNsaXAwXzEzN18xOTQ5MiI+CjxyZWN0IGNsYXNzPSJqcC1pY29uMyIgd2lkdGg9IjYiIGhlaWdodD0iNiIgZmlsbD0id2hpdGUiIHRyYW5zZm9ybT0ibWF0cml4KC0xIDAgMCAxIDEwIDEuNTU1NjYpIi8+CjwvY2xpcFBhdGg+CjwvZGVmcz4KPC9zdmc+Cg==);
  --jp-icon-add-below: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPGcgY2xpcC1wYXRoPSJ1cmwoI2NsaXAwXzEzN18xOTQ5OCkiPgo8cGF0aCBjbGFzcz0ianAtaWNvbjMiIGQ9Ik05LjI1IDEwLjA2OTNMNy4zNzUgMTAuMDY5M0w3LjM3NSA4LjE5NDM0QzcuMzc1IDcuOTg4MDkgNy4yMDYyNSA3LjgxOTM0IDcgNy44MTkzNEM2Ljc5Mzc1IDcuODE5MzQgNi42MjUgNy45ODgwOSA2LjYyNSA4LjE5NDM0TDYuNjI1IDEwLjA2OTNMNC43NSAxMC4wNjkzQzQuNTQzNzUgMTAuMDY5MyA0LjM3NSAxMC4yMzgxIDQuMzc1IDEwLjQ0NDNDNC4zNzUgMTAuNjUwNiA0LjU0Mzc1IDEwLjgxOTMgNC43NSAxMC44MTkzTDYuNjI1IDEwLjgxOTNMNi42MjUgMTIuNjk0M0M2LjYyNSAxMi45MDA2IDYuNzkzNzUgMTMuMDY5MyA3IDEzLjA2OTNDNy4yMDYyNSAxMy4wNjkzIDcuMzc1IDEyLjkwMDYgNy4zNzUgMTIuNjk0M0w3LjM3NSAxMC44MTkzTDkuMjUgMTAuODE5M0M5LjQ1NjI1IDEwLjgxOTMgOS42MjUgMTAuNjUwNiA5LjYyNSAxMC40NDQzQzkuNjI1IDEwLjIzODEgOS40NTYyNSAxMC4wNjkzIDkuMjUgMTAuMDY5M1oiIGZpbGw9IiM2MTYxNjEiIHN0cm9rZT0iIzYxNjE2MSIgc3Ryb2tlLXdpZHRoPSIwLjciLz4KPC9nPgo8cGF0aCBjbGFzcz0ianAtaWNvbjMiIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMi41IDUuNUwyLjUgMy41TDExLjUgMy41TDExLjUgNS41TDIuNSA1LjVaTTIgN0MxLjQ0NzcyIDcgMSA2LjU1MjI4IDEgNkwxIDNDMSAyLjQ0NzcyIDEuNDQ3NzIgMiAyIDJMMTIgMkMxMi41NTIzIDIgMTMgMi40NDc3MiAxMyAzTDEzIDZDMTMgNi41NTIyOSAxMi41NTIzIDcgMTIgN0wyIDdaIiBmaWxsPSIjNjE2MTYxIi8+CjxkZWZzPgo8Y2xpcFBhdGggaWQ9ImNsaXAwXzEzN18xOTQ5OCI+CjxyZWN0IGNsYXNzPSJqcC1pY29uMyIgd2lkdGg9IjYiIGhlaWdodD0iNiIgZmlsbD0id2hpdGUiIHRyYW5zZm9ybT0ibWF0cml4KDEgMS43NDg0NmUtMDcgMS43NDg0NmUtMDcgLTEgNCAxMy40NDQzKSIvPgo8L2NsaXBQYXRoPgo8L2RlZnM+Cjwvc3ZnPgo=);
  --jp-icon-add: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE5IDEzaC02djZoLTJ2LTZINXYtMmg2VjVoMnY2aDZ2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-bell: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE2IDE2IiB2ZXJzaW9uPSIxLjEiPgogICA8cGF0aCBjbGFzcz0ianAtaWNvbjIganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMzMzMzMzIgogICAgICBkPSJtOCAwLjI5Yy0xLjQgMC0yLjcgMC43My0zLjYgMS44LTEuMiAxLjUtMS40IDMuNC0xLjUgNS4yLTAuMTggMi4yLTAuNDQgNC0yLjMgNS4zbDAuMjggMS4zaDVjMC4wMjYgMC42NiAwLjMyIDEuMSAwLjcxIDEuNSAwLjg0IDAuNjEgMiAwLjYxIDIuOCAwIDAuNTItMC40IDAuNi0xIDAuNzEtMS41aDVsMC4yOC0xLjNjLTEuOS0wLjk3LTIuMi0zLjMtMi4zLTUuMy0wLjEzLTEuOC0wLjI2LTMuNy0xLjUtNS4yLTAuODUtMS0yLjItMS44LTMuNi0xLjh6bTAgMS40YzAuODggMCAxLjkgMC41NSAyLjUgMS4zIDAuODggMS4xIDEuMSAyLjcgMS4yIDQuNCAwLjEzIDEuNyAwLjIzIDMuNiAxLjMgNS4yaC0xMGMxLjEtMS42IDEuMi0zLjQgMS4zLTUuMiAwLjEzLTEuNyAwLjMtMy4zIDEuMi00LjQgMC41OS0wLjcyIDEuNi0xLjMgMi41LTEuM3ptLTAuNzQgMTJoMS41Yy0wLjAwMTUgMC4yOCAwLjAxNSAwLjc5LTAuNzQgMC43OS0wLjczIDAuMDAxNi0wLjcyLTAuNTMtMC43NC0wLjc5eiIgLz4KPC9zdmc+Cg==);
  --jp-icon-bug-dot: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiPgogICAgICAgIDxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMTcuMTkgOEgyMFYxMEgxNy45MUMxNy45NiAxMC4zMyAxOCAxMC42NiAxOCAxMVYxMkgyMFYxNEgxOC41SDE4VjE0LjAyNzVDMTUuNzUgMTQuMjc2MiAxNCAxNi4xODM3IDE0IDE4LjVDMTQgMTkuMjA4IDE0LjE2MzUgMTkuODc3OSAxNC40NTQ5IDIwLjQ3MzlDMTMuNzA2MyAyMC44MTE3IDEyLjg3NTcgMjEgMTIgMjFDOS43OCAyMSA3Ljg1IDE5Ljc5IDYuODEgMThINFYxNkg2LjA5QzYuMDQgMTUuNjcgNiAxNS4zNCA2IDE1VjE0SDRWMTJINlYxMUM2IDEwLjY2IDYuMDQgMTAuMzMgNi4wOSAxMEg0VjhINi44MUM3LjI2IDcuMjIgNy44OCA2LjU1IDguNjIgNi4wNEw3IDQuNDFMOC40MSAzTDEwLjU5IDUuMTdDMTEuMDQgNS4wNiAxMS41MSA1IDEyIDVDMTIuNDkgNSAxMi45NiA1LjA2IDEzLjQyIDUuMTdMMTUuNTkgM0wxNyA0LjQxTDE1LjM3IDYuMDRDMTYuMTIgNi41NSAxNi43NCA3LjIyIDE3LjE5IDhaTTEwIDE2SDE0VjE0SDEwVjE2Wk0xMCAxMkgxNFYxMEgxMFYxMloiIGZpbGw9IiM2MTYxNjEiLz4KICAgICAgICA8cGF0aCBkPSJNMjIgMTguNUMyMiAyMC40MzMgMjAuNDMzIDIyIDE4LjUgMjJDMTYuNTY3IDIyIDE1IDIwLjQzMyAxNSAxOC41QzE1IDE2LjU2NyAxNi41NjcgMTUgMTguNSAxNUMyMC40MzMgMTUgMjIgMTYuNTY3IDIyIDE4LjVaIiBmaWxsPSIjNjE2MTYxIi8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-bug: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik0yMCA4aC0yLjgxYy0uNDUtLjc4LTEuMDctMS40NS0xLjgyLTEuOTZMMTcgNC40MSAxNS41OSAzbC0yLjE3IDIuMTdDMTIuOTYgNS4wNiAxMi40OSA1IDEyIDVjLS40OSAwLS45Ni4wNi0xLjQxLjE3TDguNDEgMyA3IDQuNDFsMS42MiAxLjYzQzcuODggNi41NSA3LjI2IDcuMjIgNi44MSA4SDR2MmgyLjA5Yy0uMDUuMzMtLjA5LjY2LS4wOSAxdjFINHYyaDJ2MWMwIC4zNC4wNC42Ny4wOSAxSDR2MmgyLjgxYzEuMDQgMS43OSAyLjk3IDMgNS4xOSAzczQuMTUtMS4yMSA1LjE5LTNIMjB2LTJoLTIuMDljLjA1LS4zMy4wOS0uNjYuMDktMXYtMWgydi0yaC0ydi0xYzAtLjM0LS4wNC0uNjctLjA5LTFIMjBWOHptLTYgOGgtNHYtMmg0djJ6bTAtNGgtNHYtMmg0djJ6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-build: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE0LjkgMTcuNDVDMTYuMjUgMTcuNDUgMTcuMzUgMTYuMzUgMTcuMzUgMTVDMTcuMzUgMTMuNjUgMTYuMjUgMTIuNTUgMTQuOSAxMi41NUMxMy41NCAxMi41NSAxMi40NSAxMy42NSAxMi40NSAxNUMxMi40NSAxNi4zNSAxMy41NCAxNy40NSAxNC45IDE3LjQ1Wk0yMC4xIDE1LjY4TDIxLjU4IDE2Ljg0QzIxLjcxIDE2Ljk1IDIxLjc1IDE3LjEzIDIxLjY2IDE3LjI5TDIwLjI2IDE5LjcxQzIwLjE3IDE5Ljg2IDIwIDE5LjkyIDE5LjgzIDE5Ljg2TDE4LjA5IDE5LjE2QzE3LjczIDE5LjQ0IDE3LjMzIDE5LjY3IDE2LjkxIDE5Ljg1TDE2LjY0IDIxLjdDMTYuNjIgMjEuODcgMTYuNDcgMjIgMTYuMyAyMkgxMy41QzEzLjMyIDIyIDEzLjE4IDIxLjg3IDEzLjE1IDIxLjdMMTIuODkgMTkuODVDMTIuNDYgMTkuNjcgMTIuMDcgMTkuNDQgMTEuNzEgMTkuMTZMOS45NjAwMiAxOS44NkM5LjgxMDAyIDE5LjkyIDkuNjIwMDIgMTkuODYgOS41NDAwMiAxOS43MUw4LjE0MDAyIDE3LjI5QzguMDUwMDIgMTcuMTMgOC4wOTAwMiAxNi45NSA4LjIyMDAyIDE2Ljg0TDkuNzAwMDIgMTUuNjhMOS42NTAwMSAxNUw5LjcwMDAyIDE0LjMxTDguMjIwMDIgMTMuMTZDOC4wOTAwMiAxMy4wNSA4LjA1MDAyIDEyLjg2IDguMTQwMDIgMTIuNzFMOS41NDAwMiAxMC4yOUM5LjYyMDAyIDEwLjEzIDkuODEwMDIgMTAuMDcgOS45NjAwMiAxMC4xM0wxMS43MSAxMC44NEMxMi4wNyAxMC41NiAxMi40NiAxMC4zMiAxMi44OSAxMC4xNUwxMy4xNSA4LjI4OTk4QzEzLjE4IDguMTI5OTggMTMuMzIgNy45OTk5OCAxMy41IDcuOTk5OThIMTYuM0MxNi40NyA3Ljk5OTk4IDE2LjYyIDguMTI5OTggMTYuNjQgOC4yODk5OEwxNi45MSAxMC4xNUMxNy4zMyAxMC4zMiAxNy43MyAxMC41NiAxOC4wOSAxMC44NEwxOS44MyAxMC4xM0MyMCAxMC4wNyAyMC4xNyAxMC4xMyAyMC4yNiAxMC4yOUwyMS42NiAxMi43MUMyMS43NSAxMi44NiAyMS43MSAxMy4wNSAyMS41OCAxMy4xNkwyMC4xIDE0LjMxTDIwLjE1IDE1TDIwLjEgMTUuNjhaIi8+CiAgICA8cGF0aCBkPSJNNy4zMjk2NiA3LjQ0NDU0QzguMDgzMSA3LjAwOTU0IDguMzM5MzIgNi4wNTMzMiA3LjkwNDMyIDUuMjk5ODhDNy40NjkzMiA0LjU0NjQzIDYuNTA4MSA0LjI4MTU2IDUuNzU0NjYgNC43MTY1NkM1LjM5MTc2IDQuOTI2MDggNS4xMjY5NSA1LjI3MTE4IDUuMDE4NDkgNS42NzU5NEM0LjkxMDA0IDYuMDgwNzEgNC45NjY4MiA2LjUxMTk4IDUuMTc2MzQgNi44NzQ4OEM1LjYxMTM0IDcuNjI4MzIgNi41NzYyMiA3Ljg3OTU0IDcuMzI5NjYgNy40NDQ1NFpNOS42NTcxOCA0Ljc5NTkzTDEwLjg2NzIgNC45NTE3OUMxMC45NjI4IDQuOTc3NDEgMTEuMDQwMiA1LjA3MTMzIDExLjAzODIgNS4xODc5M0wxMS4wMzg4IDYuOTg4OTNDMTEuMDQ1NSA3LjEwMDU0IDEwLjk2MTYgNy4xOTUxOCAxMC44NTUgNy4yMTA1NEw5LjY2MDAxIDcuMzgwODNMOS4yMzkxNSA4LjEzMTg4TDkuNjY5NjEgOS4yNTc0NUM5LjcwNzI5IDkuMzYyNzEgOS42NjkzNCA5LjQ3Njk5IDkuNTc0MDggOS41MzE5OUw4LjAxNTIzIDEwLjQzMkM3LjkxMTMxIDEwLjQ5MiA3Ljc5MzM3IDEwLjQ2NzcgNy43MjEwNSAxMC4zODI0TDYuOTg3NDggOS40MzE4OEw2LjEwOTMxIDkuNDMwODNMNS4zNDcwNCAxMC4zOTA1QzUuMjg5MDkgMTAuNDcwMiA1LjE3MzgzIDEwLjQ5MDUgNS4wNzE4NyAxMC40MzM5TDMuNTEyNDUgOS41MzI5M0MzLjQxMDQ5IDkuNDc2MzMgMy4zNzY0NyA5LjM1NzQxIDMuNDEwNzUgOS4yNTY3OUwzLjg2MzQ3IDguMTQwOTNMMy42MTc0OSA3Ljc3NDg4TDMuNDIzNDcgNy4zNzg4M0wyLjIzMDc1IDcuMjEyOTdDMi4xMjY0NyA3LjE5MjM1IDIuMDQwNDkgNy4xMDM0MiAyLjA0MjQ1IDYuOTg2ODJMMi4wNDE4NyA1LjE4NTgyQzIuMDQzODMgNS4wNjkyMiAyLjExOTA5IDQuOTc5NTggMi4yMTcwNCA0Ljk2OTIyTDMuNDIwNjUgNC43OTM5M0wzLjg2NzQ5IDQuMDI3ODhMMy40MTEwNSAyLjkxNzMxQzMuMzczMzcgMi44MTIwNCAzLjQxMTMxIDIuNjk3NzYgMy41MTUyMyAyLjYzNzc2TDUuMDc0MDggMS43Mzc3NkM1LjE2OTM0IDEuNjgyNzYgNS4yODcyOSAxLjcwNzA0IDUuMzU5NjEgMS43OTIzMUw2LjExOTE1IDIuNzI3ODhMNi45ODAwMSAyLjczODkzTDcuNzI0OTYgMS43ODkyMkM3Ljc5MTU2IDEuNzA0NTggNy45MTU0OCAxLjY3OTIyIDguMDA4NzkgMS43NDA4Mkw5LjU2ODIxIDIuNjQxODJDOS42NzAxNyAyLjY5ODQyIDkuNzEyODUgMi44MTIzNCA5LjY4NzIzIDIuOTA3OTdMOS4yMTcxOCA0LjAzMzgzTDkuNDYzMTYgNC4zOTk4OEw5LjY1NzE4IDQuNzk1OTNaIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down-empty-thin: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iOS45LDEzLjYgMy42LDcuNCA0LjQsNi42IDkuOSwxMi4yIDE1LjQsNi43IDE2LjEsNy40ICIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down-empty: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik01LjIsNS45TDksOS43bDMuOC0zLjhsMS4yLDEuMmwtNC45LDVsLTQuOS01TDUuMiw1Ljl6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik01LjIsNy41TDksMTEuMmwzLjgtMy44SDUuMnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-caret-left: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwYXRoIGQ9Ik0xMC44LDEyLjhMNy4xLDlsMy44LTMuOGwwLDcuNkgxMC44eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-caret-right: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik03LjIsNS4yTDEwLjksOWwtMy44LDMuOFY1LjJINy4yeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-caret-up-empty-thin: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iMTUuNCwxMy4zIDkuOSw3LjcgNC40LDEzLjIgMy42LDEyLjUgOS45LDYuMyAxNi4xLDEyLjYgIi8+Cgk8L2c+Cjwvc3ZnPgo=);
  --jp-icon-caret-up: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwYXRoIGQ9Ik01LjIsMTAuNUw5LDYuOGwzLjgsMy44SDUuMnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-case-sensitive: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0MTQxNDEiPgogICAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiAgPC9nPgogIDxnIGNsYXNzPSJqcC1pY29uLWFjY2VudDIiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTcuNiw4aDAuOWwzLjUsOGgtMS4xTDEwLDE0SDZsLTAuOSwySDRMNy42LDh6IE04LDkuMUw2LjQsMTNoMy4yTDgsOS4xeiIvPgogICAgPHBhdGggZD0iTTE2LjYsOS44Yy0wLjIsMC4xLTAuNCwwLjEtMC43LDAuMWMtMC4yLDAtMC40LTAuMS0wLjYtMC4yYy0wLjEtMC4xLTAuMi0wLjQtMC4yLTAuNyBjLTAuMywwLjMtMC42LDAuNS0wLjksMC43Yy0wLjMsMC4xLTAuNywwLjItMS4xLDAuMmMtMC4zLDAtMC41LDAtMC43LTAuMWMtMC4yLTAuMS0wLjQtMC4yLTAuNi0wLjNjLTAuMi0wLjEtMC4zLTAuMy0wLjQtMC41IGMtMC4xLTAuMi0wLjEtMC40LTAuMS0wLjdjMC0wLjMsMC4xLTAuNiwwLjItMC44YzAuMS0wLjIsMC4zLTAuNCwwLjQtMC41QzEyLDcsMTIuMiw2LjksMTIuNSw2LjhjMC4yLTAuMSwwLjUtMC4xLDAuNy0wLjIgYzAuMy0wLjEsMC41LTAuMSwwLjctMC4xYzAuMiwwLDAuNC0wLjEsMC42LTAuMWMwLjIsMCwwLjMtMC4xLDAuNC0wLjJjMC4xLTAuMSwwLjItMC4yLDAuMi0wLjRjMC0xLTEuMS0xLTEuMy0xIGMtMC40LDAtMS40LDAtMS40LDEuMmgtMC45YzAtMC40LDAuMS0wLjcsMC4yLTFjMC4xLTAuMiwwLjMtMC40LDAuNS0wLjZjMC4yLTAuMiwwLjUtMC4zLDAuOC0wLjNDMTMuMyw0LDEzLjYsNCwxMy45LDQgYzAuMywwLDAuNSwwLDAuOCwwLjFjMC4zLDAsMC41LDAuMSwwLjcsMC4yYzAuMiwwLjEsMC40LDAuMywwLjUsMC41QzE2LDUsMTYsNS4yLDE2LDUuNnYyLjljMCwwLjIsMCwwLjQsMCwwLjUgYzAsMC4xLDAuMSwwLjIsMC4zLDAuMmMwLjEsMCwwLjIsMCwwLjMsMFY5Ljh6IE0xNS4yLDYuOWMtMS4yLDAuNi0zLjEsMC4yLTMuMSwxLjRjMCwxLjQsMy4xLDEsMy4xLTAuNVY2Ljl6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-check: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik05IDE2LjE3TDQuODMgMTJsLTEuNDIgMS40MUw5IDE5IDIxIDdsLTEuNDEtMS40MXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-circle-empty: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyIDJDNi40NyAyIDIgNi40NyAyIDEyczQuNDcgMTAgMTAgMTAgMTAtNC40NyAxMC0xMFMxNy41MyAyIDEyIDJ6bTAgMThjLTQuNDEgMC04LTMuNTktOC04czMuNTktOCA4LTggOCAzLjU5IDggOC0zLjU5IDgtOCA4eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-circle: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPGNpcmNsZSBjeD0iOSIgY3k9IjkiIHI9IjgiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-clear: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8bWFzayBpZD0iZG9udXRIb2xlIj4KICAgIDxyZWN0IHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgZmlsbD0id2hpdGUiIC8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSI4IiBmaWxsPSJibGFjayIvPgogIDwvbWFzaz4KCiAgPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxyZWN0IGhlaWdodD0iMTgiIHdpZHRoPSIyIiB4PSIxMSIgeT0iMyIgdHJhbnNmb3JtPSJyb3RhdGUoMzE1LCAxMiwgMTIpIi8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIxMCIgbWFzaz0idXJsKCNkb251dEhvbGUpIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-close: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1ub25lIGpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIGpwLWljb24zLWhvdmVyIiBmaWxsPSJub25lIj4KICAgIDxjaXJjbGUgY3g9IjEyIiBjeT0iMTIiIHI9IjExIi8+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIGpwLWljb24tYWNjZW50Mi1ob3ZlciIgZmlsbD0iIzYxNjE2MSI+CiAgICA8cGF0aCBkPSJNMTkgNi40MUwxNy41OSA1IDEyIDEwLjU5IDYuNDEgNSA1IDYuNDEgMTAuNTkgMTIgNSAxNy41OSA2LjQxIDE5IDEyIDEzLjQxIDE3LjU5IDE5IDE5IDE3LjU5IDEzLjQxIDEyeiIvPgogIDwvZz4KCiAgPGcgY2xhc3M9ImpwLWljb24tbm9uZSBqcC1pY29uLWJ1c3kiIGZpbGw9Im5vbmUiPgogICAgPGNpcmNsZSBjeD0iMTIiIGN5PSIxMiIgcj0iNyIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-code-check: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBzaGFwZS1yZW5kZXJpbmc9Imdlb21ldHJpY1ByZWNpc2lvbiI+CiAgICA8cGF0aCBkPSJNNi41OSwzLjQxTDIsOEw2LjU5LDEyLjZMOCwxMS4xOEw0LjgyLDhMOCw0LjgyTDYuNTksMy40MU0xMi40MSwzLjQxTDExLDQuODJMMTQuMTgsOEwxMSwxMS4xOEwxMi40MSwxMi42TDE3LDhMMTIuNDEsMy40MU0yMS41OSwxMS41OUwxMy41LDE5LjY4TDkuODMsMTZMOC40MiwxNy40MUwxMy41LDIyLjVMMjMsMTNMMjEuNTksMTEuNTlaIiAvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-code: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyOCAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTExLjQgMTguNkw2LjggMTRMMTEuNCA5LjRMMTAgOEw0IDE0TDEwIDIwTDExLjQgMTguNlpNMTYuNiAxOC42TDIxLjIgMTRMMTYuNiA5LjRMMTggOEwyNCAxNEwxOCAyMEwxNi42IDE4LjZWMTguNloiLz4KCTwvZz4KPC9zdmc+Cg==);
  --jp-icon-collapse-all: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTggMmMxIDAgMTEgMCAxMiAwczIgMSAyIDJjMCAxIDAgMTEgMCAxMnMwIDItMiAyQzIwIDE0IDIwIDQgMjAgNFMxMCA0IDYgNGMwLTIgMS0yIDItMnoiIC8+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTE4IDhjMC0xLTEtMi0yLTJTNSA2IDQgNnMtMiAxLTIgMmMwIDEgMCAxMSAwIDEyczEgMiAyIDJjMSAwIDExIDAgMTIgMHMyLTEgMi0yYzAtMSAwLTExIDAtMTJ6bS0yIDB2MTJINFY4eiIgLz4KICAgICAgICA8cGF0aCBkPSJNNiAxM3YyaDh2LTJ6IiAvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-console: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwMCAyMDAiPgogIDxnIGNsYXNzPSJqcC1jb25zb2xlLWljb24tYmFja2dyb3VuZC1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiMwMjg4RDEiPgogICAgPHBhdGggZD0iTTIwIDE5LjhoMTYwdjE1OS45SDIweiIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtY29uc29sZS1pY29uLWNvbG9yIGpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIiBmaWxsPSIjZmZmIj4KICAgIDxwYXRoIGQ9Ik0xMDUgMTI3LjNoNDB2MTIuOGgtNDB6TTUxLjEgNzdMNzQgOTkuOWwtMjMuMyAyMy4zIDEwLjUgMTAuNSAyMy4zLTIzLjNMOTUgOTkuOSA4NC41IDg5LjQgNjEuNiA2Ni41eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-copy: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTExLjksMUgzLjJDMi40LDEsMS43LDEuNywxLjcsMi41djEwLjJoMS41VjIuNWg4LjdWMXogTTE0LjEsMy45aC04Yy0wLjgsMC0xLjUsMC43LTEuNSwxLjV2MTAuMmMwLDAuOCwwLjcsMS41LDEuNSwxLjVoOCBjMC44LDAsMS41LTAuNywxLjUtMS41VjUuNEMxNS41LDQuNiwxNC45LDMuOSwxNC4xLDMuOXogTTE0LjEsMTUuNWgtOFY1LjRoOFYxNS41eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-copyright: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGVuYWJsZS1iYWNrZ3JvdW5kPSJuZXcgMCAwIDI0IDI0IiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCI+CiAgPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik0xMS44OCw5LjE0YzEuMjgsMC4wNiwxLjYxLDEuMTUsMS42MywxLjY2aDEuNzljLTAuMDgtMS45OC0xLjQ5LTMuMTktMy40NS0zLjE5QzkuNjQsNy42MSw4LDksOCwxMi4xNCBjMCwxLjk0LDAuOTMsNC4yNCwzLjg0LDQuMjRjMi4yMiwwLDMuNDEtMS42NSwzLjQ0LTIuOTVoLTEuNzljLTAuMDMsMC41OS0wLjQ1LDEuMzgtMS42MywxLjQ0QzEwLjU1LDE0LjgzLDEwLDEzLjgxLDEwLDEyLjE0IEMxMCw5LjI1LDExLjI4LDkuMTYsMTEuODgsOS4xNHogTTEyLDJDNi40OCwyLDIsNi40OCwyLDEyczQuNDgsMTAsMTAsMTBzMTAtNC40OCwxMC0xMFMxNy41MiwyLDEyLDJ6IE0xMiwyMGMtNC40MSwwLTgtMy41OS04LTggczMuNTktOCw4LThzOCwzLjU5LDgsOFMxNi40MSwyMCwxMiwyMHoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-cut: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkuNjQgNy42NGMuMjMtLjUuMzYtMS4wNS4zNi0xLjY0IDAtMi4yMS0xLjc5LTQtNC00UzIgMy43OSAyIDZzMS43OSA0IDQgNGMuNTkgMCAxLjE0LS4xMyAxLjY0LS4zNkwxMCAxMmwtMi4zNiAyLjM2QzcuMTQgMTQuMTMgNi41OSAxNCA2IDE0Yy0yLjIxIDAtNCAxLjc5LTQgNHMxLjc5IDQgNCA0IDQtMS43OSA0LTRjMC0uNTktLjEzLTEuMTQtLjM2LTEuNjRMMTIgMTRsNyA3aDN2LTFMOS42NCA3LjY0ek02IDhjLTEuMSAwLTItLjg5LTItMnMuOS0yIDItMiAyIC44OSAyIDItLjkgMi0yIDJ6bTAgMTJjLTEuMSAwLTItLjg5LTItMnMuOS0yIDItMiAyIC44OSAyIDItLjkgMi0yIDJ6bTYtNy41Yy0uMjggMC0uNS0uMjItLjUtLjVzLjIyLS41LjUtLjUuNS4yMi41LjUtLjIyLjUtLjUuNXpNMTkgM2wtNiA2IDIgMiA3LTdWM3oiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-delete: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2cHgiIGhlaWdodD0iMTZweCI+CiAgICA8cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIiAvPgogICAgPHBhdGggY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjI2MjYyIiBkPSJNNiAxOWMwIDEuMS45IDIgMiAyaDhjMS4xIDAgMi0uOSAyLTJWN0g2djEyek0xOSA0aC0zLjVsLTEtMWgtNWwtMSAxSDV2MmgxNFY0eiIgLz4KPC9zdmc+Cg==);
  --jp-icon-download: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE5IDloLTRWM0g5djZINWw3IDcgNy03ek01IDE4djJoMTR2LTJINXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-duplicate: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTIuNzk5OTggMC44NzVIOC44OTU4MkM5LjIwMDYxIDAuODc1IDkuNDQ5OTggMS4xMzkxNCA5LjQ0OTk4IDEuNDYxOThDOS40NDk5OCAxLjc4NDgyIDkuMjAwNjEgMi4wNDg5NiA4Ljg5NTgyIDIuMDQ4OTZIMy4zNTQxNUMzLjA0OTM2IDIuMDQ4OTYgMi43OTk5OCAyLjMxMzEgMi43OTk5OCAyLjYzNTk0VjkuNjc5NjlDMi43OTk5OCAxMC4wMDI1IDIuNTUwNjEgMTAuMjY2NyAyLjI0NTgyIDEwLjI2NjdDMS45NDEwMyAxMC4yNjY3IDEuNjkxNjUgMTAuMDAyNSAxLjY5MTY1IDkuNjc5NjlWMi4wNDg5NkMxLjY5MTY1IDEuNDAzMjggMi4xOTA0IDAuODc1IDIuNzk5OTggMC44NzVaTTUuMzY2NjUgMTEuOVY0LjU1SDExLjA4MzNWMTEuOUg1LjM2NjY1Wk00LjE0MTY1IDQuMTQxNjdDNC4xNDE2NSAzLjY5MDYzIDQuNTA3MjggMy4zMjUgNC45NTgzMiAzLjMyNUgxMS40OTE3QzExLjk0MjcgMy4zMjUgMTIuMzA4MyAzLjY5MDYzIDEyLjMwODMgNC4xNDE2N1YxMi4zMDgzQzEyLjMwODMgMTIuNzU5NCAxMS45NDI3IDEzLjEyNSAxMS40OTE3IDEzLjEyNUg0Ljk1ODMyQzQuNTA3MjggMTMuMTI1IDQuMTQxNjUgMTIuNzU5NCA0LjE0MTY1IDEyLjMwODNWNC4xNDE2N1oiIGZpbGw9IiM2MTYxNjEiLz4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBkPSJNOS40MzU3NCA4LjI2NTA3SDguMzY0MzFWOS4zMzY1QzguMzY0MzEgOS40NTQzNSA4LjI2Nzg4IDkuNTUwNzggOC4xNTAwMiA5LjU1MDc4QzguMDMyMTcgOS41NTA3OCA3LjkzNTc0IDkuNDU0MzUgNy45MzU3NCA5LjMzNjVWOC4yNjUwN0g2Ljg2NDMxQzYuNzQ2NDUgOC4yNjUwNyA2LjY1MDAyIDguMTY4NjQgNi42NTAwMiA4LjA1MDc4QzYuNjUwMDIgNy45MzI5MiA2Ljc0NjQ1IDcuODM2NSA2Ljg2NDMxIDcuODM2NUg3LjkzNTc0VjYuNzY1MDdDNy45MzU3NCA2LjY0NzIxIDguMDMyMTcgNi41NTA3OCA4LjE1MDAyIDYuNTUwNzhDOC4yNjc4OCA2LjU1MDc4IDguMzY0MzEgNi42NDcyMSA4LjM2NDMxIDYuNzY1MDdWNy44MzY1SDkuNDM1NzRDOS41NTM2IDcuODM2NSA5LjY1MDAyIDcuOTMyOTIgOS42NTAwMiA4LjA1MDc4QzkuNjUwMDIgOC4xNjg2NCA5LjU1MzYgOC4yNjUwNyA5LjQzNTc0IDguMjY1MDdaIiBmaWxsPSIjNjE2MTYxIiBzdHJva2U9IiM2MTYxNjEiIHN0cm9rZS13aWR0aD0iMC41Ii8+Cjwvc3ZnPgo=);
  --jp-icon-edit: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTMgMTcuMjVWMjFoMy43NUwxNy44MSA5Ljk0bC0zLjc1LTMuNzVMMyAxNy4yNXpNMjAuNzEgNy4wNGMuMzktLjM5LjM5LTEuMDIgMC0xLjQxbC0yLjM0LTIuMzRjLS4zOS0uMzktMS4wMi0uMzktMS40MSAwbC0xLjgzIDEuODMgMy43NSAzLjc1IDEuODMtMS44M3oiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-ellipses: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPGNpcmNsZSBjeD0iNSIgY3k9IjEyIiByPSIyIi8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIyIi8+CiAgICA8Y2lyY2xlIGN4PSIxOSIgY3k9IjEyIiByPSIyIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-error: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj48Y2lyY2xlIGN4PSIxMiIgY3k9IjE5IiByPSIyIi8+PHBhdGggZD0iTTEwIDNoNHYxMmgtNHoiLz48L2c+CjxwYXRoIGZpbGw9Im5vbmUiIGQ9Ik0wIDBoMjR2MjRIMHoiLz4KPC9zdmc+Cg==);
  --jp-icon-expand-all: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTggMmMxIDAgMTEgMCAxMiAwczIgMSAyIDJjMCAxIDAgMTEgMCAxMnMwIDItMiAyQzIwIDE0IDIwIDQgMjAgNFMxMCA0IDYgNGMwLTIgMS0yIDItMnoiIC8+CiAgICAgICAgPHBhdGgKICAgICAgICAgICAgZD0iTTE4IDhjMC0xLTEtMi0yLTJTNSA2IDQgNnMtMiAxLTIgMmMwIDEgMCAxMSAwIDEyczEgMiAyIDJjMSAwIDExIDAgMTIgMHMyLTEgMi0yYzAtMSAwLTExIDAtMTJ6bS0yIDB2MTJINFY4eiIgLz4KICAgICAgICA8cGF0aCBkPSJNMTEgMTBIOXYzSDZ2MmgzdjNoMnYtM2gzdi0yaC0zeiIgLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-extension: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwLjUgMTFIMTlWN2MwLTEuMS0uOS0yLTItMmgtNFYzLjVDMTMgMi4xMiAxMS44OCAxIDEwLjUgMVM4IDIuMTIgOCAzLjVWNUg0Yy0xLjEgMC0xLjk5LjktMS45OSAydjMuOEgzLjVjMS40OSAwIDIuNyAxLjIxIDIuNyAyLjdzLTEuMjEgMi43LTIuNyAyLjdIMlYyMGMwIDEuMS45IDIgMiAyaDMuOHYtMS41YzAtMS40OSAxLjIxLTIuNyAyLjctMi43IDEuNDkgMCAyLjcgMS4yMSAyLjcgMi43VjIySDE3YzEuMSAwIDItLjkgMi0ydi00aDEuNWMxLjM4IDAgMi41LTEuMTIgMi41LTIuNVMyMS44OCAxMSAyMC41IDExeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-fast-forward: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTQgMThsOC41LTZMNCA2djEyem05LTEydjEybDguNS02TDEzIDZ6Ii8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-file-upload: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkgMTZoNnYtNmg0bC03LTctNyA3aDR6bS00IDJoMTR2Mkg1eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-file: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkuMyA4LjJsLTUuNS01LjVjLS4zLS4zLS43LS41LTEuMi0uNUgzLjljLS44LjEtMS42LjktMS42IDEuOHYxNC4xYzAgLjkuNyAxLjYgMS42IDEuNmgxNC4yYy45IDAgMS42LS43IDEuNi0xLjZWOS40Yy4xLS41LS4xLS45LS40LTEuMnptLTUuOC0zLjNsMy40IDMuNmgtMy40VjQuOXptMy45IDEyLjdINC43Yy0uMSAwLS4yIDAtLjItLjJWNC43YzAtLjIuMS0uMy4yLS4zaDcuMnY0LjRzMCAuOC4zIDEuMWMuMy4zIDEuMS4zIDEuMS4zaDQuM3Y3LjJzLS4xLjItLjIuMnoiLz4KPC9zdmc+Cg==);
  --jp-icon-filter-dot: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTE0LDEyVjE5Ljg4QzE0LjA0LDIwLjE4IDEzLjk0LDIwLjUgMTMuNzEsMjAuNzFDMTMuMzIsMjEuMSAxMi42OSwyMS4xIDEyLjMsMjAuNzFMMTAuMjksMTguN0MxMC4wNiwxOC40NyA5Ljk2LDE4LjE2IDEwLDE3Ljg3VjEySDkuOTdMNC4yMSw0LjYyQzMuODcsNC4xOSAzLjk1LDMuNTYgNC4zOCwzLjIyQzQuNTcsMy4wOCA0Ljc4LDMgNSwzVjNIMTlWM0MxOS4yMiwzIDE5LjQzLDMuMDggMTkuNjIsMy4yMkMyMC4wNSwzLjU2IDIwLjEzLDQuMTkgMTkuNzksNC42MkwxNC4wMywxMkgxNFoiIC8+CiAgPC9nPgogIDxnIGNsYXNzPSJqcC1pY29uLWRvdCIgZmlsbD0iI0ZGRiI+CiAgICA8Y2lyY2xlIGN4PSIxOCIgY3k9IjE3IiByPSIzIj48L2NpcmNsZT4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-filter-list: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEwIDE4aDR2LTJoLTR2MnpNMyA2djJoMThWNkgzem0zIDdoMTJ2LTJINnYyeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-filter: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTE0LDEyVjE5Ljg4QzE0LjA0LDIwLjE4IDEzLjk0LDIwLjUgMTMuNzEsMjAuNzFDMTMuMzIsMjEuMSAxMi42OSwyMS4xIDEyLjMsMjAuNzFMMTAuMjksMTguN0MxMC4wNiwxOC40NyA5Ljk2LDE4LjE2IDEwLDE3Ljg3VjEySDkuOTdMNC4yMSw0LjYyQzMuODcsNC4xOSAzLjk1LDMuNTYgNC4zOCwzLjIyQzQuNTcsMy4wOCA0Ljc4LDMgNSwzVjNIMTlWM0MxOS4yMiwzIDE5LjQzLDMuMDggMTkuNjIsMy4yMkMyMC4wNSwzLjU2IDIwLjEzLDQuMTkgMTkuNzksNC42MkwxNC4wMywxMkgxNFoiIC8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-folder-favorite: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB3aWR0aD0iMjRweCIgZmlsbD0iIzAwMDAwMCI+CiAgPHBhdGggZD0iTTAgMGgyNHYyNEgwVjB6IiBmaWxsPSJub25lIi8+PHBhdGggY2xhc3M9ImpwLWljb24zIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzYxNjE2MSIgZD0iTTIwIDZoLThsLTItMkg0Yy0xLjEgMC0yIC45LTIgMnYxMmMwIDEuMS45IDIgMiAyaDE2YzEuMSAwIDItLjkgMi0yVjhjMC0xLjEtLjktMi0yLTJ6bS0yLjA2IDExTDE1IDE1LjI4IDEyLjA2IDE3bC43OC0zLjMzLTIuNTktMi4yNCAzLjQxLS4yOUwxNSA4bDEuMzQgMy4xNCAzLjQxLjI5LTIuNTkgMi4yNC43OCAzLjMzeiIvPgo8L3N2Zz4K);
  --jp-icon-folder: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTAgNEg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMThjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY4YzAtMS4xLS45LTItMi0yaC04bC0yLTJ6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-home: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjRweCIgdmlld0JveD0iMCAwIDI0IDI0IiB3aWR0aD0iMjRweCIgZmlsbD0iIzAwMDAwMCI+CiAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPjxwYXRoIGNsYXNzPSJqcC1pY29uMyBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xMCAyMHYtNmg0djZoNXYtOGgzTDEyIDMgMiAxMmgzdjh6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-html5: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUxMiA1MTIiPgogIDxwYXRoIGNsYXNzPSJqcC1pY29uMCBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiMwMDAiIGQ9Ik0xMDguNCAwaDIzdjIyLjhoMjEuMlYwaDIzdjY5aC0yM1Y0NmgtMjF2MjNoLTIzLjJNMjA2IDIzaC0yMC4zVjBoNjMuN3YyM0gyMjl2NDZoLTIzbTUzLjUtNjloMjQuMWwxNC44IDI0LjNMMzEzLjIgMGgyNC4xdjY5aC0yM1YzNC44bC0xNi4xIDI0LjgtMTYuMS0yNC44VjY5aC0yMi42bTg5LjItNjloMjN2NDYuMmgzMi42VjY5aC01NS42Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI2U0NGQyNiIgZD0iTTEwNy42IDQ3MWwtMzMtMzcwLjRoMzYyLjhsLTMzIDM3MC4yTDI1NS43IDUxMiIvPgogIDxwYXRoIGNsYXNzPSJqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNmMTY1MjkiIGQ9Ik0yNTYgNDgwLjVWMTMxaDE0OC4zTDM3NiA0NDciLz4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNlYmViZWIiIGQ9Ik0xNDIgMTc2LjNoMTE0djQ1LjRoLTY0LjJsNC4yIDQ2LjVoNjB2NDUuM0gxNTQuNG0yIDIyLjhIMjAybDMuMiAzNi4zIDUwLjggMTMuNnY0Ny40bC05My4yLTI2Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIiBmaWxsPSIjZmZmIiBkPSJNMzY5LjYgMTc2LjNIMjU1Ljh2NDUuNGgxMDkuNm0tNC4xIDQ2LjVIMjU1Ljh2NDUuNGg1NmwtNS4zIDU5LTUwLjcgMTMuNnY0Ny4ybDkzLTI1LjgiLz4KPC9zdmc+Cg==);
  --jp-icon-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1icmFuZDQganAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNGRkYiIGQ9Ik0yLjIgMi4yaDE3LjV2MTcuNUgyLjJ6Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzNGNTFCNSIgZD0iTTIuMiAyLjJ2MTcuNWgxNy41bC4xLTE3LjVIMi4yem0xMi4xIDIuMmMxLjIgMCAyLjIgMSAyLjIgMi4ycy0xIDIuMi0yLjIgMi4yLTIuMi0xLTIuMi0yLjIgMS0yLjIgMi4yLTIuMnpNNC40IDE3LjZsMy4zLTguOCAzLjMgNi42IDIuMi0zLjIgNC40IDUuNEg0LjR6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-info: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUwLjk3OCA1MC45NzgiPgoJPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KCQk8cGF0aCBkPSJNNDMuNTIsNy40NThDMzguNzExLDIuNjQ4LDMyLjMwNywwLDI1LjQ4OSwwQzE4LjY3LDAsMTIuMjY2LDIuNjQ4LDcuNDU4LDcuNDU4CgkJCWMtOS45NDMsOS45NDEtOS45NDMsMjYuMTE5LDAsMzYuMDYyYzQuODA5LDQuODA5LDExLjIxMiw3LjQ1NiwxOC4wMzEsNy40NThjMCwwLDAuMDAxLDAsMC4wMDIsMAoJCQljNi44MTYsMCwxMy4yMjEtMi42NDgsMTguMDI5LTcuNDU4YzQuODA5LTQuODA5LDcuNDU3LTExLjIxMiw3LjQ1Ny0xOC4wM0M1MC45NzcsMTguNjcsNDguMzI4LDEyLjI2Niw0My41Miw3LjQ1OHoKCQkJIE00Mi4xMDYsNDIuMTA1Yy00LjQzMiw0LjQzMS0xMC4zMzIsNi44NzItMTYuNjE1LDYuODcyaC0wLjAwMmMtNi4yODUtMC4wMDEtMTIuMTg3LTIuNDQxLTE2LjYxNy02Ljg3MgoJCQljLTkuMTYyLTkuMTYzLTkuMTYyLTI0LjA3MSwwLTMzLjIzM0MxMy4zMDMsNC40NCwxOS4yMDQsMiwyNS40ODksMmM2LjI4NCwwLDEyLjE4NiwyLjQ0LDE2LjYxNyw2Ljg3MgoJCQljNC40MzEsNC40MzEsNi44NzEsMTAuMzMyLDYuODcxLDE2LjYxN0M0OC45NzcsMzEuNzcyLDQ2LjUzNiwzNy42NzUsNDIuMTA2LDQyLjEwNXoiLz4KCQk8cGF0aCBkPSJNMjMuNTc4LDMyLjIxOGMtMC4wMjMtMS43MzQsMC4xNDMtMy4wNTksMC40OTYtMy45NzJjMC4zNTMtMC45MTMsMS4xMS0xLjk5NywyLjI3Mi0zLjI1MwoJCQljMC40NjgtMC41MzYsMC45MjMtMS4wNjIsMS4zNjctMS41NzVjMC42MjYtMC43NTMsMS4xMDQtMS40NzgsMS40MzYtMi4xNzVjMC4zMzEtMC43MDcsMC40OTUtMS41NDEsMC40OTUtMi41CgkJCWMwLTEuMDk2LTAuMjYtMi4wODgtMC43NzktMi45NzljLTAuNTY1LTAuODc5LTEuNTAxLTEuMzM2LTIuODA2LTEuMzY5Yy0xLjgwMiwwLjA1Ny0yLjk4NSwwLjY2Ny0zLjU1LDEuODMyCgkJCWMtMC4zMDEsMC41MzUtMC41MDMsMS4xNDEtMC42MDcsMS44MTRjLTAuMTM5LDAuNzA3LTAuMjA3LDEuNDMyLTAuMjA3LDIuMTc0aC0yLjkzN2MtMC4wOTEtMi4yMDgsMC40MDctNC4xMTQsMS40OTMtNS43MTkKCQkJYzEuMDYyLTEuNjQsMi44NTUtMi40ODEsNS4zNzgtMi41MjdjMi4xNiwwLjAyMywzLjg3NCwwLjYwOCw1LjE0MSwxLjc1OGMxLjI3OCwxLjE2LDEuOTI5LDIuNzY0LDEuOTUsNC44MTEKCQkJYzAsMS4xNDItMC4xMzcsMi4xMTEtMC40MSwyLjkxMWMtMC4zMDksMC44NDUtMC43MzEsMS41OTMtMS4yNjgsMi4yNDNjLTAuNDkyLDAuNjUtMS4wNjgsMS4zMTgtMS43MywyLjAwMgoJCQljLTAuNjUsMC42OTctMS4zMTMsMS40NzktMS45ODcsMi4zNDZjLTAuMjM5LDAuMzc3LTAuNDI5LDAuNzc3LTAuNTY1LDEuMTk5Yy0wLjE2LDAuOTU5LTAuMjE3LDEuOTUxLTAuMTcxLDIuOTc5CgkJCUMyNi41ODksMzIuMjE4LDIzLjU3OCwzMi4yMTgsMjMuNTc4LDMyLjIxOHogTTIzLjU3OCwzOC4yMnYtMy40ODRoMy4wNzZ2My40ODRIMjMuNTc4eiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-inspector: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaW5zcGVjdG9yLWljb24tY29sb3IganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMjAgNEg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMThjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY2YzAtMS4xLS45LTItMi0yem0tNSAxNEg0di00aDExdjR6bTAtNUg0VjloMTF2NHptNSA1aC00VjloNHY5eiIvPgo8L3N2Zz4K);
  --jp-icon-json: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtanNvbi1pY29uLWNvbG9yIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI0Y5QTgyNSI+CiAgICA8cGF0aCBkPSJNMjAuMiAxMS44Yy0xLjYgMC0xLjcuNS0xLjcgMSAwIC40LjEuOS4xIDEuMy4xLjUuMS45LjEgMS4zIDAgMS43LTEuNCAyLjMtMy41IDIuM2gtLjl2LTEuOWguNWMxLjEgMCAxLjQgMCAxLjQtLjggMC0uMyAwLS42LS4xLTEgMC0uNC0uMS0uOC0uMS0xLjIgMC0xLjMgMC0xLjggMS4zLTItMS4zLS4yLTEuMy0uNy0xLjMtMiAwLS40LjEtLjguMS0xLjIuMS0uNC4xLS43LjEtMSAwLS44LS40LS43LTEuNC0uOGgtLjVWNC4xaC45YzIuMiAwIDMuNS43IDMuNSAyLjMgMCAuNC0uMS45LS4xIDEuMy0uMS41LS4xLjktLjEgMS4zIDAgLjUuMiAxIDEuNyAxdjEuOHpNMS44IDEwLjFjMS42IDAgMS43LS41IDEuNy0xIDAtLjQtLjEtLjktLjEtMS4zLS4xLS41LS4xLS45LS4xLTEuMyAwLTEuNiAxLjQtMi4zIDMuNS0yLjNoLjl2MS45aC0uNWMtMSAwLTEuNCAwLTEuNC44IDAgLjMgMCAuNi4xIDEgMCAuMi4xLjYuMSAxIDAgMS4zIDAgMS44LTEuMyAyQzYgMTEuMiA2IDExLjcgNiAxM2MwIC40LS4xLjgtLjEgMS4yLS4xLjMtLjEuNy0uMSAxIDAgLjguMy44IDEuNC44aC41djEuOWgtLjljLTIuMSAwLTMuNS0uNi0zLjUtMi4zIDAtLjQuMS0uOS4xLTEuMy4xLS41LjEtLjkuMS0xLjMgMC0uNS0uMi0xLTEuNy0xdi0xLjl6Ii8+CiAgICA8Y2lyY2xlIGN4PSIxMSIgY3k9IjEzLjgiIHI9IjIuMSIvPgogICAgPGNpcmNsZSBjeD0iMTEiIGN5PSI4LjIiIHI9IjIuMSIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-julia: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDMyNSAzMDAiPgogIDxnIGNsYXNzPSJqcC1icmFuZDAganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjY2IzYzMzIj4KICAgIDxwYXRoIGQ9Ik0gMTUwLjg5ODQzOCAyMjUgQyAxNTAuODk4NDM4IDI2Ni40MjE4NzUgMTE3LjMyMDMxMiAzMDAgNzUuODk4NDM4IDMwMCBDIDM0LjQ3NjU2MiAzMDAgMC44OTg0MzggMjY2LjQyMTg3NSAwLjg5ODQzOCAyMjUgQyAwLjg5ODQzOCAxODMuNTc4MTI1IDM0LjQ3NjU2MiAxNTAgNzUuODk4NDM4IDE1MCBDIDExNy4zMjAzMTIgMTUwIDE1MC44OTg0MzggMTgzLjU3ODEyNSAxNTAuODk4NDM4IDIyNSIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzM4OTgyNiI+CiAgICA8cGF0aCBkPSJNIDIzNy41IDc1IEMgMjM3LjUgMTE2LjQyMTg3NSAyMDMuOTIxODc1IDE1MCAxNjIuNSAxNTAgQyAxMjEuMDc4MTI1IDE1MCA4Ny41IDExNi40MjE4NzUgODcuNSA3NSBDIDg3LjUgMzMuNTc4MTI1IDEyMS4wNzgxMjUgMCAxNjIuNSAwIEMgMjAzLjkyMTg3NSAwIDIzNy41IDMzLjU3ODEyNSAyMzcuNSA3NSIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzk1NThiMiI+CiAgICA8cGF0aCBkPSJNIDMyNC4xMDE1NjIgMjI1IEMgMzI0LjEwMTU2MiAyNjYuNDIxODc1IDI5MC41MjM0MzggMzAwIDI0OS4xMDE1NjIgMzAwIEMgMjA3LjY3OTY4OCAzMDAgMTc0LjEwMTU2MiAyNjYuNDIxODc1IDE3NC4xMDE1NjIgMjI1IEMgMTc0LjEwMTU2MiAxODMuNTc4MTI1IDIwNy42Nzk2ODggMTUwIDI0OS4xMDE1NjIgMTUwIEMgMjkwLjUyMzQzOCAxNTAgMzI0LjEwMTU2MiAxODMuNTc4MTI1IDMyNC4xMDE1NjIgMjI1Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-jupyter-favicon: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTUyIiBoZWlnaHQ9IjE2NSIgdmlld0JveD0iMCAwIDE1MiAxNjUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgPGcgY2xhc3M9ImpwLWp1cHl0ZXItaWNvbi1jb2xvciIgZmlsbD0iI0YzNzcyNiI+CiAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjA3ODk0NywgMTEwLjU4MjkyNykiIGQ9Ik03NS45NDIyODQyLDI5LjU4MDQ1NjEgQzQzLjMwMjM5NDcsMjkuNTgwNDU2MSAxNC43OTY3ODMyLDE3LjY1MzQ2MzQgMCwwIEM1LjUxMDgzMjExLDE1Ljg0MDY4MjkgMTUuNzgxNTM4OSwyOS41NjY3NzMyIDI5LjM5MDQ5NDcsMzkuMjc4NDE3MSBDNDIuOTk5Nyw0OC45ODk4NTM3IDU5LjI3MzcsNTQuMjA2NzgwNSA3NS45NjA1Nzg5LDU0LjIwNjc4MDUgQzkyLjY0NzQ1NzksNTQuMjA2NzgwNSAxMDguOTIxNDU4LDQ4Ljk4OTg1MzcgMTIyLjUzMDY2MywzOS4yNzg0MTcxIEMxMzYuMTM5NDUzLDI5LjU2Njc3MzIgMTQ2LjQxMDI4NCwxNS44NDA2ODI5IDE1MS45MjExNTgsMCBDMTM3LjA4Nzg2OCwxNy42NTM0NjM0IDEwOC41ODI1ODksMjkuNTgwNDU2MSA3NS45NDIyODQyLDI5LjU4MDQ1NjEgTDc1Ljk0MjI4NDIsMjkuNTgwNDU2MSBaIiAvPgogICAgPHBhdGggdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMzczNjgsIDAuNzA0ODc4KSIgZD0iTTc1Ljk3ODQ1NzksMjQuNjI2NDA3MyBDMTA4LjYxODc2MywyNC42MjY0MDczIDEzNy4xMjQ0NTgsMzYuNTUzNDQxNSAxNTEuOTIxMTU4LDU0LjIwNjc4MDUgQzE0Ni40MTAyODQsMzguMzY2MjIyIDEzNi4xMzk0NTMsMjQuNjQwMTMxNyAxMjIuNTMwNjYzLDE0LjkyODQ4NzggQzEwOC45MjE0NTgsNS4yMTY4NDM5IDkyLjY0NzQ1NzksMCA3NS45NjA1Nzg5LDAgQzU5LjI3MzcsMCA0Mi45OTk3LDUuMjE2ODQzOSAyOS4zOTA0OTQ3LDE0LjkyODQ4NzggQzE1Ljc4MTUzODksMjQuNjQwMTMxNyA1LjUxMDgzMjExLDM4LjM2NjIyMiAwLDU0LjIwNjc4MDUgQzE0LjgzMzA4MTYsMzYuNTg5OTI5MyA0My4zMzg1Njg0LDI0LjYyNjQwNzMgNzUuOTc4NDU3OSwyNC42MjY0MDczIEw3NS45Nzg0NTc5LDI0LjYyNjQwNzMgWiIgLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-jupyter: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzkiIGhlaWdodD0iNTEiIHZpZXdCb3g9IjAgMCAzOSA1MSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMTYzOCAtMjI4MSkiPgogICAgIDxnIGNsYXNzPSJqcC1qdXB5dGVyLWljb24tY29sb3IiIGZpbGw9IiNGMzc3MjYiPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM5Ljc0IDIzMTEuOTgpIiBkPSJNIDE4LjI2NDYgNy4xMzQxMUMgMTAuNDE0NSA3LjEzNDExIDMuNTU4NzIgNC4yNTc2IDAgMEMgMS4zMjUzOSAzLjgyMDQgMy43OTU1NiA3LjEzMDgxIDcuMDY4NiA5LjQ3MzAzQyAxMC4zNDE3IDExLjgxNTIgMTQuMjU1NyAxMy4wNzM0IDE4LjI2OSAxMy4wNzM0QyAyMi4yODIzIDEzLjA3MzQgMjYuMTk2MyAxMS44MTUyIDI5LjQ2OTQgOS40NzMwM0MgMzIuNzQyNCA3LjEzMDgxIDM1LjIxMjYgMy44MjA0IDM2LjUzOCAwQyAzMi45NzA1IDQuMjU3NiAyNi4xMTQ4IDcuMTM0MTEgMTguMjY0NiA3LjEzNDExWiIvPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM5LjczIDIyODUuNDgpIiBkPSJNIDE4LjI3MzMgNS45MzkzMUMgMjYuMTIzNSA1LjkzOTMxIDMyLjk3OTMgOC44MTU4MyAzNi41MzggMTMuMDczNEMgMzUuMjEyNiA5LjI1MzAzIDMyLjc0MjQgNS45NDI2MiAyOS40Njk0IDMuNjAwNEMgMjYuMTk2MyAxLjI1ODE4IDIyLjI4MjMgMCAxOC4yNjkgMEMgMTQuMjU1NyAwIDEwLjM0MTcgMS4yNTgxOCA3LjA2ODYgMy42MDA0QyAzLjc5NTU2IDUuOTQyNjIgMS4zMjUzOSA5LjI1MzAzIDAgMTMuMDczNEMgMy41Njc0NSA4LjgyNDYzIDEwLjQyMzIgNS45MzkzMSAxOC4yNzMzIDUuOTM5MzFaIi8+CiAgICA8L2c+CiAgICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjY5LjMgMjI4MS4zMSkiIGQ9Ik0gNS44OTM1MyAyLjg0NEMgNS45MTg4OSAzLjQzMTY1IDUuNzcwODUgNC4wMTM2NyA1LjQ2ODE1IDQuNTE2NDVDIDUuMTY1NDUgNS4wMTkyMiA0LjcyMTY4IDUuNDIwMTUgNC4xOTI5OSA1LjY2ODUxQyAzLjY2NDMgNS45MTY4OCAzLjA3NDQ0IDYuMDAxNTEgMi40OTgwNSA1LjkxMTcxQyAxLjkyMTY2IDUuODIxOSAxLjM4NDYzIDUuNTYxNyAwLjk1NDg5OCA1LjE2NDAxQyAwLjUyNTE3IDQuNzY2MzMgMC4yMjIwNTYgNC4yNDkwMyAwLjA4MzkwMzcgMy42Nzc1N0MgLTAuMDU0MjQ4MyAzLjEwNjExIC0wLjAyMTIzIDIuNTA2MTcgMC4xNzg3ODEgMS45NTM2NEMgMC4zNzg3OTMgMS40MDExIDAuNzM2ODA5IDAuOTIwODE3IDEuMjA3NTQgMC41NzM1MzhDIDEuNjc4MjYgMC4yMjYyNTkgMi4yNDA1NSAwLjAyNzU5MTkgMi44MjMyNiAwLjAwMjY3MjI5QyAzLjYwMzg5IC0wLjAzMDcxMTUgNC4zNjU3MyAwLjI0OTc4OSA0Ljk0MTQyIDAuNzgyNTUxQyA1LjUxNzExIDEuMzE1MzEgNS44NTk1NiAyLjA1Njc2IDUuODkzNTMgMi44NDRaIi8+CiAgICAgIDxwYXRoIHRyYW5zZm9ybT0idHJhbnNsYXRlKDE2MzkuOCAyMzIzLjgxKSIgZD0iTSA3LjQyNzg5IDMuNTgzMzhDIDcuNDYwMDggNC4zMjQzIDcuMjczNTUgNS4wNTgxOSA2Ljg5MTkzIDUuNjkyMTNDIDYuNTEwMzEgNi4zMjYwNyA1Ljk1MDc1IDYuODMxNTYgNS4yODQxMSA3LjE0NDZDIDQuNjE3NDcgNy40NTc2MyAzLjg3MzcxIDcuNTY0MTQgMy4xNDcwMiA3LjQ1MDYzQyAyLjQyMDMyIDcuMzM3MTIgMS43NDMzNiA3LjAwODcgMS4yMDE4NCA2LjUwNjk1QyAwLjY2MDMyOCA2LjAwNTIgMC4yNzg2MSA1LjM1MjY4IDAuMTA1MDE3IDQuNjMyMDJDIC0wLjA2ODU3NTcgMy45MTEzNSAtMC4wMjYyMzYxIDMuMTU0OTQgMC4yMjY2NzUgMi40NTg1NkMgMC40Nzk1ODcgMS43NjIxNyAwLjkzMTY5NyAxLjE1NzEzIDEuNTI1NzYgMC43MjAwMzNDIDIuMTE5ODMgMC4yODI5MzUgMi44MjkxNCAwLjAzMzQzOTUgMy41NjM4OSAwLjAwMzEzMzQ0QyA0LjU0NjY3IC0wLjAzNzQwMzMgNS41MDUyOSAwLjMxNjcwNiA2LjIyOTYxIDAuOTg3ODM1QyA2Ljk1MzkzIDEuNjU4OTYgNy4zODQ4NCAyLjU5MjM1IDcuNDI3ODkgMy41ODMzOEwgNy40Mjc4OSAzLjU4MzM4WiIvPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM4LjM2IDIyODYuMDYpIiBkPSJNIDIuMjc0NzEgNC4zOTYyOUMgMS44NDM2MyA0LjQxNTA4IDEuNDE2NzEgNC4zMDQ0NSAxLjA0Nzk5IDQuMDc4NDNDIDAuNjc5MjY4IDMuODUyNCAwLjM4NTMyOCAzLjUyMTE0IDAuMjAzMzcxIDMuMTI2NTZDIDAuMDIxNDEzNiAyLjczMTk4IC0wLjA0MDM3OTggMi4yOTE4MyAwLjAyNTgxMTYgMS44NjE4MUMgMC4wOTIwMDMxIDEuNDMxOCAwLjI4MzIwNCAxLjAzMTI2IDAuNTc1MjEzIDAuNzEwODgzQyAwLjg2NzIyMiAwLjM5MDUxIDEuMjQ2OTEgMC4xNjQ3MDggMS42NjYyMiAwLjA2MjA1OTJDIDIuMDg1NTMgLTAuMDQwNTg5NyAyLjUyNTYxIC0wLjAxNTQ3MTQgMi45MzA3NiAwLjEzNDIzNUMgMy4zMzU5MSAwLjI4Mzk0MSAzLjY4NzkyIDAuNTUxNTA1IDMuOTQyMjIgMC45MDMwNkMgNC4xOTY1MiAxLjI1NDYyIDQuMzQxNjkgMS42NzQzNiA0LjM1OTM1IDIuMTA5MTZDIDQuMzgyOTkgMi42OTEwNyA0LjE3Njc4IDMuMjU4NjkgMy43ODU5NyAzLjY4NzQ2QyAzLjM5NTE2IDQuMTE2MjQgMi44NTE2NiA0LjM3MTE2IDIuMjc0NzEgNC4zOTYyOUwgMi4yNzQ3MSA0LjM5NjI5WiIvPgogICAgPC9nPgogIDwvZz4+Cjwvc3ZnPgo=);
  --jp-icon-jupyterlab-wordmark: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMDAiIHZpZXdCb3g9IjAgMCAxODYwLjggNDc1Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0RTRFNEUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDQ4MC4xMzY0MDEsIDY0LjI3MTQ5MykiPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMDAwMDAsIDU4Ljg3NTU2NikiPgogICAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjA4NzYwMywgMC4xNDAyOTQpIj4KICAgICAgICA8cGF0aCBkPSJNLTQyNi45LDE2OS44YzAsNDguNy0zLjcsNjQuNy0xMy42LDc2LjRjLTEwLjgsMTAtMjUsMTUuNS0zOS43LDE1LjVsMy43LDI5IGMyMi44LDAuMyw0NC44LTcuOSw2MS45LTIzLjFjMTcuOC0xOC41LDI0LTQ0LjEsMjQtODMuM1YwSC00Mjd2MTcwLjFMLTQyNi45LDE2OS44TC00MjYuOSwxNjkuOHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTU1LjA0NTI5NiwgNTYuODM3MTA0KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuNTYyNDUzLCAxLjc5OTg0MikiPgogICAgICAgIDxwYXRoIGQ9Ik0tMzEyLDE0OGMwLDIxLDAsMzkuNSwxLjcsNTUuNGgtMzEuOGwtMi4xLTMzLjNoLTAuOGMtNi43LDExLjYtMTYuNCwyMS4zLTI4LDI3LjkgYy0xMS42LDYuNi0yNC44LDEwLTM4LjIsOS44Yy0zMS40LDAtNjktMTcuNy02OS04OVYwaDM2LjR2MTEyLjdjMCwzOC43LDExLjYsNjQuNyw0NC42LDY0LjdjMTAuMy0wLjIsMjAuNC0zLjUsMjguOS05LjQgYzguNS01LjksMTUuMS0xNC4zLDE4LjktMjMuOWMyLjItNi4xLDMuMy0xMi41LDMuMy0xOC45VjAuMmgzNi40VjE0OEgtMzEyTC0zMTIsMTQ4eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgzOTAuMDEzMzIyLCA1My40Nzk2MzgpIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMS43MDY0NTgsIDAuMjMxNDI1KSI+CiAgICAgICAgPHBhdGggZD0iTS00NzguNiw3MS40YzAtMjYtMC44LTQ3LTEuNy02Ni43aDMyLjdsMS43LDM0LjhoMC44YzcuMS0xMi41LDE3LjUtMjIuOCwzMC4xLTI5LjcgYzEyLjUtNywyNi43LTEwLjMsNDEtOS44YzQ4LjMsMCw4NC43LDQxLjcsODQuNywxMDMuM2MwLDczLjEtNDMuNywxMDkuMi05MSwxMDkuMmMtMTIuMSwwLjUtMjQuMi0yLjItMzUtNy44IGMtMTAuOC01LjYtMTkuOS0xMy45LTI2LjYtMjQuMmgtMC44VjI5MWgtMzZ2LTIyMEwtNDc4LjYsNzEuNEwtNDc4LjYsNzEuNHogTS00NDIuNiwxMjUuNmMwLjEsNS4xLDAuNiwxMC4xLDEuNywxNS4xIGMzLDEyLjMsOS45LDIzLjMsMTkuOCwzMS4xYzkuOSw3LjgsMjIuMSwxMi4xLDM0LjcsMTIuMWMzOC41LDAsNjAuNy0zMS45LDYwLjctNzguNWMwLTQwLjctMjEuMS03NS42LTU5LjUtNzUuNiBjLTEyLjksMC40LTI1LjMsNS4xLTM1LjMsMTMuNGMtOS45LDguMy0xNi45LDE5LjctMTkuNiwzMi40Yy0xLjUsNC45LTIuMywxMC0yLjUsMTUuMVYxMjUuNkwtNDQyLjYsMTI1LjZMLTQ0Mi42LDEyNS42eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSg2MDYuNzQwNzI2LCA1Ni44MzcxMDQpIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC43NTEyMjYsIDEuOTg5Mjk5KSI+CiAgICAgICAgPHBhdGggZD0iTS00NDAuOCwwbDQzLjcsMTIwLjFjNC41LDEzLjQsOS41LDI5LjQsMTIuOCw0MS43aDAuOGMzLjctMTIuMiw3LjktMjcuNywxMi44LTQyLjQgbDM5LjctMTE5LjJoMzguNUwtMzQ2LjksMTQ1Yy0yNiw2OS43LTQzLjcsMTA1LjQtNjguNiwxMjcuMmMtMTIuNSwxMS43LTI3LjksMjAtNDQuNiwyMy45bC05LjEtMzEuMSBjMTEuNy0zLjksMjIuNS0xMC4xLDMxLjgtMTguMWMxMy4yLTExLjEsMjMuNy0yNS4yLDMwLjYtNDEuMmMxLjUtMi44LDIuNS01LjcsMi45LTguOGMtMC4zLTMuMy0xLjItNi42LTIuNS05LjdMLTQ4MC4yLDAuMSBoMzkuN0wtNDQwLjgsMEwtNDQwLjgsMHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoODIyLjc0ODEwNCwgMC4wMDAwMDApIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMS40NjQwNTAsIDAuMzc4OTE0KSI+CiAgICAgICAgPHBhdGggZD0iTS00MTMuNywwdjU4LjNoNTJ2MjguMmgtNTJWMTk2YzAsMjUsNywzOS41LDI3LjMsMzkuNWM3LjEsMC4xLDE0LjItMC43LDIxLjEtMi41IGwxLjcsMjcuN2MtMTAuMywzLjctMjEuMyw1LjQtMzIuMiw1Yy03LjMsMC40LTE0LjYtMC43LTIxLjMtMy40Yy02LjgtMi43LTEyLjktNi44LTE3LjktMTIuMWMtMTAuMy0xMC45LTE0LjEtMjktMTQuMS01Mi45IFY4Ni41aC0zMVY1OC4zaDMxVjkuNkwtNDEzLjcsMEwtNDEzLjcsMHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOTc0LjQzMzI4NiwgNTMuNDc5NjM4KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuOTkwMDM0LCAwLjYxMDMzOSkiPgogICAgICAgIDxwYXRoIGQ9Ik0tNDQ1LjgsMTEzYzAuOCw1MCwzMi4yLDcwLjYsNjguNiw3MC42YzE5LDAuNiwzNy45LTMsNTUuMy0xMC41bDYuMiwyNi40IGMtMjAuOSw4LjktNDMuNSwxMy4xLTY2LjIsMTIuNmMtNjEuNSwwLTk4LjMtNDEuMi05OC4zLTEwMi41Qy00ODAuMiw0OC4yLTQ0NC43LDAtMzg2LjUsMGM2NS4yLDAsODIuNyw1OC4zLDgyLjcsOTUuNyBjLTAuMSw1LjgtMC41LDExLjUtMS4yLDE3LjJoLTE0MC42SC00NDUuOEwtNDQ1LjgsMTEzeiBNLTMzOS4yLDg2LjZjMC40LTIzLjUtOS41LTYwLjEtNTAuNC02MC4xIGMtMzYuOCwwLTUyLjgsMzQuNC01NS43LDYwLjFILTMzOS4yTC0zMzkuMiw4Ni42TC0zMzkuMiw4Ni42eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxMjAxLjk2MTA1OCwgNTMuNDc5NjM4KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuMTc5NjQwLCAwLjcwNTA2OCkiPgogICAgICAgIDxwYXRoIGQ9Ik0tNDc4LjYsNjhjMC0yMy45LTAuNC00NC41LTEuNy02My40aDMxLjhsMS4yLDM5LjloMS43YzkuMS0yNy4zLDMxLTQ0LjUsNTUuMy00NC41IGMzLjUtMC4xLDcsMC40LDEwLjMsMS4ydjM0LjhjLTQuMS0wLjktOC4yLTEuMy0xMi40LTEuMmMtMjUuNiwwLTQzLjcsMTkuNy00OC43LDQ3LjRjLTEsNS43LTEuNiwxMS41LTEuNywxNy4ydjEwOC4zaC0zNlY2OCBMLTQ3OC42LDY4eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbi13YXJuMCIgZmlsbD0iI0YzNzcyNiI+CiAgICA8cGF0aCBkPSJNMTM1Mi4zLDMyNi4yaDM3VjI4aC0zN1YzMjYuMnogTTE2MDQuOCwzMjYuMmMtMi41LTEzLjktMy40LTMxLjEtMy40LTQ4Ljd2LTc2IGMwLTQwLjctMTUuMS04My4xLTc3LjMtODMuMWMtMjUuNiwwLTUwLDcuMS02Ni44LDE4LjFsOC40LDI0LjRjMTQuMy05LjIsMzQtMTUuMSw1My0xNS4xYzQxLjYsMCw0Ni4yLDMwLjIsNDYuMiw0N3Y0LjIgYy03OC42LTAuNC0xMjIuMywyNi41LTEyMi4zLDc1LjZjMCwyOS40LDIxLDU4LjQsNjIuMiw1OC40YzI5LDAsNTAuOS0xNC4zLDYyLjItMzAuMmgxLjNsMi45LDI1LjZIMTYwNC44eiBNMTU2NS43LDI1Ny43IGMwLDMuOC0wLjgsOC0yLjEsMTEuOGMtNS45LDE3LjItMjIuNywzNC00OS4yLDM0Yy0xOC45LDAtMzQuOS0xMS4zLTM0LjktMzUuM2MwLTM5LjUsNDUuOC00Ni42LDg2LjItNDUuOFYyNTcuN3ogTTE2OTguNSwzMjYuMiBsMS43LTMzLjZoMS4zYzE1LjEsMjYuOSwzOC43LDM4LjIsNjguMSwzOC4yYzQ1LjQsMCw5MS4yLTM2LjEsOTEuMi0xMDguOGMwLjQtNjEuNy0zNS4zLTEwMy43LTg1LjctMTAzLjcgYy0zMi44LDAtNTYuMywxNC43LTY5LjMsMzcuNGgtMC44VjI4aC0zNi42djI0NS43YzAsMTguMS0wLjgsMzguNi0xLjcsNTIuNUgxNjk4LjV6IE0xNzA0LjgsMjA4LjJjMC01LjksMS4zLTEwLjksMi4xLTE1LjEgYzcuNi0yOC4xLDMxLjEtNDUuNCw1Ni4zLTQ1LjRjMzkuNSwwLDYwLjUsMzQuOSw2MC41LDc1LjZjMCw0Ni42LTIzLjEsNzguMS02MS44LDc4LjFjLTI2LjksMC00OC4zLTE3LjYtNTUuNS00My4zIGMtMC44LTQuMi0xLjctOC44LTEuNy0xMy40VjIwOC4yeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-kernel: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgZmlsbD0iIzYxNjE2MSIgZD0iTTE1IDlIOXY2aDZWOXptLTIgNGgtMnYtMmgydjJ6bTgtMlY5aC0yVjdjMC0xLjEtLjktMi0yLTJoLTJWM2gtMnYyaC0yVjNIOXYySDdjLTEuMSAwLTIgLjktMiAydjJIM3YyaDJ2MkgzdjJoMnYyYzAgMS4xLjkgMiAyIDJoMnYyaDJ2LTJoMnYyaDJ2LTJoMmMxLjEgMCAyLS45IDItMnYtMmgydi0yaC0ydi0yaDJ6bS00IDZIN1Y3aDEwdjEweiIvPgo8L3N2Zz4K);
  --jp-icon-keyboard: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMjAgNUg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMTdjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY3YzAtMS4xLS45LTItMi0yem0tOSAzaDJ2MmgtMlY4em0wIDNoMnYyaC0ydi0yek04IDhoMnYySDhWOHptMCAzaDJ2Mkg4di0yem0tMSAySDV2LTJoMnYyem0wLTNINVY4aDJ2MnptOSA3SDh2LTJoOHYyem0wLTRoLTJ2LTJoMnYyem0wLTNoLTJWOGgydjJ6bTMgM2gtMnYtMmgydjJ6bTAtM2gtMlY4aDJ2MnoiLz4KPC9zdmc+Cg==);
  --jp-icon-launch: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMzIgMzIiIHdpZHRoPSIzMiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik0yNiwyOEg2YTIuMDAyNywyLjAwMjcsMCwwLDEtMi0yVjZBMi4wMDI3LDIuMDAyNywwLDAsMSw2LDRIMTZWNkg2VjI2SDI2VjE2aDJWMjZBMi4wMDI3LDIuMDAyNywwLDAsMSwyNiwyOFoiLz4KICAgIDxwb2x5Z29uIHBvaW50cz0iMjAgMiAyMCA0IDI2LjU4NiA0IDE4IDEyLjU4NiAxOS40MTQgMTQgMjggNS40MTQgMjggMTIgMzAgMTIgMzAgMiAyMCAyIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-launcher: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkgMTlINVY1aDdWM0g1YTIgMiAwIDAwLTIgMnYxNGEyIDIgMCAwMDIgMmgxNGMxLjEgMCAyLS45IDItMnYtN2gtMnY3ek0xNCAzdjJoMy41OWwtOS44MyA5LjgzIDEuNDEgMS40MUwxOSA2LjQxVjEwaDJWM2gtN3oiLz4KPC9zdmc+Cg==);
  --jp-icon-line-form: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGZpbGw9IndoaXRlIiBkPSJNNS44OCA0LjEyTDEzLjc2IDEybC03Ljg4IDcuODhMOCAyMmwxMC0xMEw4IDJ6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-link: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTMuOSAxMmMwLTEuNzEgMS4zOS0zLjEgMy4xLTMuMWg0VjdIN2MtMi43NiAwLTUgMi4yNC01IDVzMi4yNCA1IDUgNWg0di0xLjlIN2MtMS43MSAwLTMuMS0xLjM5LTMuMS0zLjF6TTggMTNoOHYtMkg4djJ6bTktNmgtNHYxLjloNGMxLjcxIDAgMy4xIDEuMzkgMy4xIDMuMXMtMS4zOSAzLjEtMy4xIDMuMWgtNFYxN2g0YzIuNzYgMCA1LTIuMjQgNS01cy0yLjI0LTUtNS01eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-list: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xOSA1djE0SDVWNWgxNG0xLjEtMkgzLjljLS41IDAtLjkuNC0uOS45djE2LjJjMCAuNC40LjkuOS45aDE2LjJjLjQgMCAuOS0uNS45LS45VjMuOWMwLS41LS41LS45LS45LS45ek0xMSA3aDZ2MmgtNlY3em0wIDRoNnYyaC02di0yem0wIDRoNnYyaC02ek03IDdoMnYySDd6bTAgNGgydjJIN3ptMCA0aDJ2Mkg3eiIvPgo8L3N2Zz4K);
  --jp-icon-markdown: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDAganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjN0IxRkEyIiBkPSJNNSAxNC45aDEybC02LjEgNnptOS40LTYuOGMwLTEuMy0uMS0yLjktLjEtNC41LS40IDEuNC0uOSAyLjktMS4zIDQuM2wtMS4zIDQuM2gtMkw4LjUgNy45Yy0uNC0xLjMtLjctMi45LTEtNC4zLS4xIDEuNi0uMSAzLjItLjIgNC42TDcgMTIuNEg0LjhsLjctMTFoMy4zTDEwIDVjLjQgMS4yLjcgMi43IDEgMy45LjMtMS4yLjctMi42IDEtMy45bDEuMi0zLjdoMy4zbC42IDExaC0yLjRsLS4zLTQuMnoiLz4KPC9zdmc+Cg==);
  --jp-icon-move-down: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBkPSJNMTIuNDcxIDcuNTI4OTlDMTIuNzYzMiA3LjIzNjg0IDEyLjc2MzIgNi43NjMxNiAxMi40NzEgNi40NzEwMVY2LjQ3MTAxQzEyLjE3OSA2LjE3OTA1IDExLjcwNTcgNi4xNzg4NCAxMS40MTM1IDYuNDcwNTRMNy43NSAxMC4xMjc1VjEuNzVDNy43NSAxLjMzNTc5IDcuNDE0MjEgMSA3IDFWMUM2LjU4NTc5IDEgNi4yNSAxLjMzNTc5IDYuMjUgMS43NVYxMC4xMjc1TDIuNTk3MjYgNi40NjgyMkMyLjMwMzM4IDYuMTczODEgMS44MjY0MSA2LjE3MzU5IDEuNTMyMjYgNi40Njc3NFY2LjQ2Nzc0QzEuMjM4MyA2Ljc2MTcgMS4yMzgzIDcuMjM4MyAxLjUzMjI2IDcuNTMyMjZMNi4yOTI4OSAxMi4yOTI5QzYuNjgzNDIgMTIuNjgzNCA3LjMxNjU4IDEyLjY4MzQgNy43MDcxMSAxMi4yOTI5TDEyLjQ3MSA3LjUyODk5WiIgZmlsbD0iIzYxNjE2MSIvPgo8L3N2Zz4K);
  --jp-icon-move-up: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMTQiIHZpZXdCb3g9IjAgMCAxNCAxNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggY2xhc3M9ImpwLWljb24zIiBkPSJNMS41Mjg5OSA2LjQ3MTAxQzEuMjM2ODQgNi43NjMxNiAxLjIzNjg0IDcuMjM2ODQgMS41Mjg5OSA3LjUyODk5VjcuNTI4OTlDMS44MjA5NSA3LjgyMDk1IDIuMjk0MjYgNy44MjExNiAyLjU4NjQ5IDcuNTI5NDZMNi4yNSAzLjg3MjVWMTIuMjVDNi4yNSAxMi42NjQyIDYuNTg1NzkgMTMgNyAxM1YxM0M3LjQxNDIxIDEzIDcuNzUgMTIuNjY0MiA3Ljc1IDEyLjI1VjMuODcyNUwxMS40MDI3IDcuNTMxNzhDMTEuNjk2NiA3LjgyNjE5IDEyLjE3MzYgNy44MjY0MSAxMi40Njc3IDcuNTMyMjZWNy41MzIyNkMxMi43NjE3IDcuMjM4MyAxMi43NjE3IDYuNzYxNyAxMi40Njc3IDYuNDY3NzRMNy43MDcxMSAxLjcwNzExQzcuMzE2NTggMS4zMTY1OCA2LjY4MzQyIDEuMzE2NTggNi4yOTI4OSAxLjcwNzExTDEuNTI4OTkgNi40NzEwMVoiIGZpbGw9IiM2MTYxNjEiLz4KPC9zdmc+Cg==);
  --jp-icon-new-folder: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwIDZoLThsLTItMkg0Yy0xLjExIDAtMS45OS44OS0xLjk5IDJMMiAxOGMwIDEuMTEuODkgMiAyIDJoMTZjMS4xMSAwIDItLjg5IDItMlY4YzAtMS4xMS0uODktMi0yLTJ6bS0xIDhoLTN2M2gtMnYtM2gtM3YtMmgzVjloMnYzaDN2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-not-trusted: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI1IDI1Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMgMykiIGQ9Ik0xLjg2MDk0IDExLjQ0MDlDMC44MjY0NDggOC43NzAyNyAwLjg2Mzc3OSA2LjA1NzY0IDEuMjQ5MDcgNC4xOTkzMkMyLjQ4MjA2IDMuOTMzNDcgNC4wODA2OCAzLjQwMzQ3IDUuNjAxMDIgMi44NDQ5QzcuMjM1NDkgMi4yNDQ0IDguODU2NjYgMS41ODE1IDkuOTg3NiAxLjA5NTM5QzExLjA1OTcgMS41ODM0MSAxMi42MDk0IDIuMjQ0NCAxNC4yMTggMi44NDMzOUMxNS43NTAzIDMuNDEzOTQgMTcuMzk5NSAzLjk1MjU4IDE4Ljc1MzkgNC4yMTM4NUMxOS4xMzY0IDYuMDcxNzcgMTkuMTcwOSA4Ljc3NzIyIDE4LjEzOSAxMS40NDA5QzE3LjAzMDMgMTQuMzAzMiAxNC42NjY4IDE3LjE4NDQgOS45OTk5OSAxOC45MzU0QzUuMzMzMTkgMTcuMTg0NCAyLjk2OTY4IDE0LjMwMzIgMS44NjA5NCAxMS40NDA5WiIvPgogICAgPHBhdGggY2xhc3M9ImpwLWljb24yIiBzdHJva2U9IiMzMzMzMzMiIHN0cm9rZS13aWR0aD0iMiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOS4zMTU5MiA5LjMyMDMxKSIgZD0iTTcuMzY4NDIgMEwwIDcuMzY0NzkiLz4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDkuMzE1OTIgMTYuNjgzNikgc2NhbGUoMSAtMSkiIGQ9Ik03LjM2ODQyIDBMMCA3LjM2NDc5Ii8+Cjwvc3ZnPgo=);
  --jp-icon-notebook: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtbm90ZWJvb2staWNvbi1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNFRjZDMDAiPgogICAgPHBhdGggZD0iTTE4LjcgMy4zdjE1LjRIMy4zVjMuM2gxNS40bTEuNS0xLjVIMS44djE4LjNoMTguM2wuMS0xOC4zeiIvPgogICAgPHBhdGggZD0iTTE2LjUgMTYuNWwtNS40LTQuMy01LjYgNC4zdi0xMWgxMXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-numbering: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyOCAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTQgMTlINlYxOS41SDVWMjAuNUg2VjIxSDRWMjJIN1YxOEg0VjE5Wk01IDEwSDZWNkg0VjdINVYxMFpNNCAxM0g1LjhMNCAxNS4xVjE2SDdWMTVINS4yTDcgMTIuOVYxMkg0VjEzWk05IDdWOUgyM1Y3SDlaTTkgMjFIMjNWMTlIOVYyMVpNOSAxNUgyM1YxM0g5VjE1WiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-offline-bolt: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyIDIuMDJjLTUuNTEgMC05Ljk4IDQuNDctOS45OCA5Ljk4czQuNDcgOS45OCA5Ljk4IDkuOTggOS45OC00LjQ3IDkuOTgtOS45OFMxNy41MSAyLjAyIDEyIDIuMDJ6TTExLjQ4IDIwdi02LjI2SDhMMTMgNHY2LjI2aDMuMzVMMTEuNDggMjB6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-palette: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE4IDEzVjIwSDRWNkg5LjAyQzkuMDcgNS4yOSA5LjI0IDQuNjIgOS41IDRINEMyLjkgNCAyIDQuOSAyIDZWMjBDMiAyMS4xIDIuOSAyMiA0IDIySDE4QzE5LjEgMjIgMjAgMjEuMSAyMCAyMFYxNUwxOCAxM1pNMTkuMyA4Ljg5QzE5Ljc0IDguMTkgMjAgNy4zOCAyMCA2LjVDMjAgNC4wMSAxNy45OSAyIDE1LjUgMkMxMy4wMSAyIDExIDQuMDEgMTEgNi41QzExIDguOTkgMTMuMDEgMTEgMTUuNDkgMTFDMTYuMzcgMTEgMTcuMTkgMTAuNzQgMTcuODggMTAuM0wyMSAxMy40MkwyMi40MiAxMkwxOS4zIDguODlaTTE1LjUgOUMxNC4xMiA5IDEzIDcuODggMTMgNi41QzEzIDUuMTIgMTQuMTIgNCAxNS41IDRDMTYuODggNCAxOCA1LjEyIDE4IDYuNUMxOCA3Ljg4IDE2Ljg4IDkgMTUuNSA5WiIvPgogICAgPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik00IDZIOS4wMTg5NEM5LjAwNjM5IDYuMTY1MDIgOSA2LjMzMTc2IDkgNi41QzkgOC44MTU3NyAxMC4yMTEgMTAuODQ4NyAxMi4wMzQzIDEySDlWMTRIMTZWMTIuOTgxMUMxNi41NzAzIDEyLjkzNzcgMTcuMTIgMTIuODIwNyAxNy42Mzk2IDEyLjYzOTZMMTggMTNWMjBINFY2Wk04IDhINlYxMEg4VjhaTTYgMTJIOFYxNEg2VjEyWk04IDE2SDZWMThIOFYxNlpNOSAxNkgxNlYxOEg5VjE2WiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-paste: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTE5IDJoLTQuMThDMTQuNC44NCAxMy4zIDAgMTIgMGMtMS4zIDAtMi40Ljg0LTIuODIgMkg1Yy0xLjEgMC0yIC45LTIgMnYxNmMwIDEuMS45IDIgMiAyaDE0YzEuMSAwIDItLjkgMi0yVjRjMC0xLjEtLjktMi0yLTJ6bS03IDBjLjU1IDAgMSAuNDUgMSAxcy0uNDUgMS0xIDEtMS0uNDUtMS0xIC40NS0xIDEtMXptNyAxOEg1VjRoMnYzaDEwVjRoMnYxNnoiLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-pdf: url(data:image/svg+xml;base64,PHN2ZwogICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyMiAyMiIgd2lkdGg9IjE2Ij4KICAgIDxwYXRoIHRyYW5zZm9ybT0icm90YXRlKDQ1KSIgY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI0ZGMkEyQSIKICAgICAgIGQ9Im0gMjIuMzQ0MzY5LC0zLjAxNjM2NDIgaCA1LjYzODYwNCB2IDEuNTc5MjQzMyBoIC0zLjU0OTIyNyB2IDEuNTA4NjkyOTkgaCAzLjMzNzU3NiBWIDEuNjUwODE1NCBoIC0zLjMzNzU3NiB2IDMuNDM1MjYxMyBoIC0yLjA4OTM3NyB6IG0gLTcuMTM2NDQ0LDEuNTc5MjQzMyB2IDQuOTQzOTU0MyBoIDAuNzQ4OTIgcSAxLjI4MDc2MSwwIDEuOTUzNzAzLC0wLjYzNDk1MzUgMC42NzgzNjksLTAuNjM0OTUzNSAwLjY3ODM2OSwtMS44NDUxNjQxIDAsLTEuMjA0NzgzNTUgLTAuNjcyOTQyLC0xLjgzNDMxMDExIC0wLjY3Mjk0MiwtMC42Mjk1MjY1OSAtMS45NTkxMywtMC42Mjk1MjY1OSB6IG0gLTIuMDg5Mzc3LC0xLjU3OTI0MzMgaCAyLjIwMzM0MyBxIDEuODQ1MTY0LDAgMi43NDYwMzksMC4yNjU5MjA3IDAuOTA2MzAxLDAuMjYwNDkzNyAxLjU1MjEwOCwwLjg5MDAyMDMgMC41Njk4MywwLjU0ODEyMjMgMC44NDY2MDUsMS4yNjQ0ODAwNiAwLjI3Njc3NCwwLjcxNjM1NzgxIDAuMjc2Nzc0LDEuNjIyNjU4OTQgMCwwLjkxNzE1NTEgLTAuMjc2Nzc0LDEuNjM4OTM5OSAtMC4yNzY3NzUsMC43MTYzNTc4IC0wLjg0NjYwNSwxLjI2NDQ4IC0wLjY1MTIzNCwwLjYyOTUyNjYgLTEuNTYyOTYyLDAuODk1NDQ3MyAtMC45MTE3MjgsMC4yNjA0OTM3IC0yLjczNTE4NSwwLjI2MDQ5MzcgaCAtMi4yMDMzNDMgeiBtIC04LjE0NTg1NjUsMCBoIDMuNDY3ODIzIHEgMS41NDY2ODE2LDAgMi4zNzE1Nzg1LDAuNjg5MjIzIDAuODMwMzI0LDAuNjgzNzk2MSAwLjgzMDMyNCwxLjk1MzcwMzE0IDAsMS4yNzUzMzM5NyAtMC44MzAzMjQsMS45NjQ1NTcwNiBRIDkuOTg3MTk2MSwyLjI3NDkxNSA4LjQ0MDUxNDUsMi4yNzQ5MTUgSCA3LjA2MjA2ODQgViA1LjA4NjA3NjcgSCA0Ljk3MjY5MTUgWiBtIDIuMDg5Mzc2OSwxLjUxNDExOTkgdiAyLjI2MzAzOTQzIGggMS4xNTU5NDEgcSAwLjYwNzgxODgsMCAwLjkzODg2MjksLTAuMjkzMDU1NDcgMC4zMzEwNDQxLC0wLjI5ODQ4MjQxIDAuMzMxMDQ0MSwtMC44NDExNzc3MiAwLC0wLjU0MjY5NTMxIC0wLjMzMTA0NDEsLTAuODM1NzUwNzQgLTAuMzMxMDQ0MSwtMC4yOTMwNTU1IC0wLjkzODg2MjksLTAuMjkzMDU1NSB6IgovPgo8L3N2Zz4K);
  --jp-icon-python: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iLTEwIC0xMCAxMzEuMTYxMzYxNjk0MzM1OTQgMTMyLjM4ODk5OTkzODk2NDg0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMzA2OTk4IiBkPSJNIDU0LjkxODc4NSw5LjE5Mjc0MjFlLTQgQyA1MC4zMzUxMzIsMC4wMjIyMTcyNyA0NS45NTc4NDYsMC40MTMxMzY5NyA0Mi4xMDYyODUsMS4wOTQ2NjkzIDMwLjc2MDA2OSwzLjA5OTE3MzEgMjguNzAwMDM2LDcuMjk0NzcxNCAyOC43MDAwMzUsMTUuMDMyMTY5IHYgMTAuMjE4NzUgaCAyNi44MTI1IHYgMy40MDYyNSBoIC0yNi44MTI1IC0xMC4wNjI1IGMgLTcuNzkyNDU5LDAgLTE0LjYxNTc1ODgsNC42ODM3MTcgLTE2Ljc0OTk5OTgsMTMuNTkzNzUgLTIuNDYxODE5OTgsMTAuMjEyOTY2IC0yLjU3MTAxNTA4LDE2LjU4NjAyMyAwLDI3LjI1IDEuOTA1OTI4Myw3LjkzNzg1MiA2LjQ1NzU0MzIsMTMuNTkzNzQ4IDE0LjI0OTk5OTgsMTMuNTkzNzUgaCA5LjIxODc1IHYgLTEyLjI1IGMgMCwtOC44NDk5MDIgNy42NTcxNDQsLTE2LjY1NjI0OCAxNi43NSwtMTYuNjU2MjUgaCAyNi43ODEyNSBjIDcuNDU0OTUxLDAgMTMuNDA2MjUzLC02LjEzODE2NCAxMy40MDYyNSwtMTMuNjI1IHYgLTI1LjUzMTI1IGMgMCwtNy4yNjYzMzg2IC02LjEyOTk4LC0xMi43MjQ3NzcxIC0xMy40MDYyNSwtMTMuOTM3NDk5NyBDIDY0LjI4MTU0OCwwLjMyNzk0Mzk3IDU5LjUwMjQzOCwtMC4wMjAzNzkwMyA1NC45MTg3ODUsOS4xOTI3NDIxZS00IFogbSAtMTQuNSw4LjIxODc1MDEyNTc5IGMgMi43Njk1NDcsMCA1LjAzMTI1LDIuMjk4NjQ1NiA1LjAzMTI1LDUuMTI0OTk5NiAtMmUtNiwyLjgxNjMzNiAtMi4yNjE3MDMsNS4wOTM3NSAtNS4wMzEyNSw1LjA5Mzc1IC0yLjc3OTQ3NiwtMWUtNiAtNS4wMzEyNSwtMi4yNzc0MTUgLTUuMDMxMjUsLTUuMDkzNzUgLTEwZS03LC0yLjgyNjM1MyAyLjI1MTc3NCwtNS4xMjQ5OTk2IDUuMDMxMjUsLTUuMTI0OTk5NiB6Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI2ZmZDQzYiIgZD0ibSA4NS42Mzc1MzUsMjguNjU3MTY5IHYgMTEuOTA2MjUgYyAwLDkuMjMwNzU1IC03LjgyNTg5NSwxNi45OTk5OTkgLTE2Ljc1LDE3IGggLTI2Ljc4MTI1IGMgLTcuMzM1ODMzLDAgLTEzLjQwNjI0OSw2LjI3ODQ4MyAtMTMuNDA2MjUsMTMuNjI1IHYgMjUuNTMxMjQ3IGMgMCw3LjI2NjM0NCA2LjMxODU4OCwxMS41NDAzMjQgMTMuNDA2MjUsMTMuNjI1MDA0IDguNDg3MzMxLDIuNDk1NjEgMTYuNjI2MjM3LDIuOTQ2NjMgMjYuNzgxMjUsMCA2Ljc1MDE1NSwtMS45NTQzOSAxMy40MDYyNTMsLTUuODg3NjEgMTMuNDA2MjUsLTEzLjYyNTAwNCBWIDg2LjUwMDkxOSBoIC0yNi43ODEyNSB2IC0zLjQwNjI1IGggMjYuNzgxMjUgMTMuNDA2MjU0IGMgNy43OTI0NjEsMCAxMC42OTYyNTEsLTUuNDM1NDA4IDEzLjQwNjI0MSwtMTMuNTkzNzUgMi43OTkzMywtOC4zOTg4ODYgMi42ODAyMiwtMTYuNDc1Nzc2IDAsLTI3LjI1IC0xLjkyNTc4LC03Ljc1NzQ0MSAtNS42MDM4NywtMTMuNTkzNzUgLTEzLjQwNjI0MSwtMTMuNTkzNzUgeiBtIC0xNS4wNjI1LDY0LjY1NjI1IGMgMi43Nzk0NzgsM2UtNiA1LjAzMTI1LDIuMjc3NDE3IDUuMDMxMjUsNS4wOTM3NDcgLTJlLTYsMi44MjYzNTQgLTIuMjUxNzc1LDUuMTI1MDA0IC01LjAzMTI1LDUuMTI1MDA0IC0yLjc2OTU1LDAgLTUuMDMxMjUsLTIuMjk4NjUgLTUuMDMxMjUsLTUuMTI1MDA0IDJlLTYsLTIuODE2MzMgMi4yNjE2OTcsLTUuMDkzNzQ3IDUuMDMxMjUsLTUuMDkzNzQ3IHoiLz4KPC9zdmc+Cg==);
  --jp-icon-r-kernel: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMjE5NkYzIiBkPSJNNC40IDIuNWMxLjItLjEgMi45LS4zIDQuOS0uMyAyLjUgMCA0LjEuNCA1LjIgMS4zIDEgLjcgMS41IDEuOSAxLjUgMy41IDAgMi0xLjQgMy41LTIuOSA0LjEgMS4yLjQgMS43IDEuNiAyLjIgMyAuNiAxLjkgMSAzLjkgMS4zIDQuNmgtMy44Yy0uMy0uNC0uOC0xLjctMS4yLTMuN3MtMS4yLTIuNi0yLjYtMi42aC0uOXY2LjRINC40VjIuNXptMy43IDYuOWgxLjRjMS45IDAgMi45LS45IDIuOS0yLjNzLTEtMi4zLTIuOC0yLjNjLS43IDAtMS4zIDAtMS42LjJ2NC41aC4xdi0uMXoiLz4KPC9zdmc+Cg==);
  --jp-icon-react: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMTUwIDE1MCA1NDEuOSAyOTUuMyI+CiAgPGcgY2xhc3M9ImpwLWljb24tYnJhbmQyIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzYxREFGQiI+CiAgICA8cGF0aCBkPSJNNjY2LjMgMjk2LjVjMC0zMi41LTQwLjctNjMuMy0xMDMuMS04Mi40IDE0LjQtNjMuNiA4LTExNC4yLTIwLjItMTMwLjQtNi41LTMuOC0xNC4xLTUuNi0yMi40LTUuNnYyMi4zYzQuNiAwIDguMy45IDExLjQgMi42IDEzLjYgNy44IDE5LjUgMzcuNSAxNC45IDc1LjctMS4xIDkuNC0yLjkgMTkuMy01LjEgMjkuNC0xOS42LTQuOC00MS04LjUtNjMuNS0xMC45LTEzLjUtMTguNS0yNy41LTM1LjMtNDEuNi01MCAzMi42LTMwLjMgNjMuMi00Ni45IDg0LTQ2LjlWNzhjLTI3LjUgMC02My41IDE5LjYtOTkuOSA1My42LTM2LjQtMzMuOC03Mi40LTUzLjItOTkuOS01My4ydjIyLjNjMjAuNyAwIDUxLjQgMTYuNSA4NCA0Ni42LTE0IDE0LjctMjggMzEuNC00MS4zIDQ5LjktMjIuNiAyLjQtNDQgNi4xLTYzLjYgMTEtMi4zLTEwLTQtMTkuNy01LjItMjktNC43LTM4LjIgMS4xLTY3LjkgMTQuNi03NS44IDMtMS44IDYuOS0yLjYgMTEuNS0yLjZWNzguNWMtOC40IDAtMTYgMS44LTIyLjYgNS42LTI4LjEgMTYuMi0zNC40IDY2LjctMTkuOSAxMzAuMS02Mi4yIDE5LjItMTAyLjcgNDkuOS0xMDIuNyA4Mi4zIDAgMzIuNSA0MC43IDYzLjMgMTAzLjEgODIuNC0xNC40IDYzLjYtOCAxMTQuMiAyMC4yIDEzMC40IDYuNSAzLjggMTQuMSA1LjYgMjIuNSA1LjYgMjcuNSAwIDYzLjUtMTkuNiA5OS45LTUzLjYgMzYuNCAzMy44IDcyLjQgNTMuMiA5OS45IDUzLjIgOC40IDAgMTYtMS44IDIyLjYtNS42IDI4LjEtMTYuMiAzNC40LTY2LjcgMTkuOS0xMzAuMSA2Mi0xOS4xIDEwMi41LTQ5LjkgMTAyLjUtODIuM3ptLTEzMC4yLTY2LjdjLTMuNyAxMi45LTguMyAyNi4yLTEzLjUgMzkuNS00LjEtOC04LjQtMTYtMTMuMS0yNC00LjYtOC05LjUtMTUuOC0xNC40LTIzLjQgMTQuMiAyLjEgMjcuOSA0LjcgNDEgNy45em0tNDUuOCAxMDYuNWMtNy44IDEzLjUtMTUuOCAyNi4zLTI0LjEgMzguMi0xNC45IDEuMy0zMCAyLTQ1LjIgMi0xNS4xIDAtMzAuMi0uNy00NS0xLjktOC4zLTExLjktMTYuNC0yNC42LTI0LjItMzgtNy42LTEzLjEtMTQuNS0yNi40LTIwLjgtMzkuOCA2LjItMTMuNCAxMy4yLTI2LjggMjAuNy0zOS45IDcuOC0xMy41IDE1LjgtMjYuMyAyNC4xLTM4LjIgMTQuOS0xLjMgMzAtMiA0NS4yLTIgMTUuMSAwIDMwLjIuNyA0NSAxLjkgOC4zIDExLjkgMTYuNCAyNC42IDI0LjIgMzggNy42IDEzLjEgMTQuNSAyNi40IDIwLjggMzkuOC02LjMgMTMuNC0xMy4yIDI2LjgtMjAuNyAzOS45em0zMi4zLTEzYzUuNCAxMy40IDEwIDI2LjggMTMuOCAzOS44LTEzLjEgMy4yLTI2LjkgNS45LTQxLjIgOCA0LjktNy43IDkuOC0xNS42IDE0LjQtMjMuNyA0LjYtOCA4LjktMTYuMSAxMy0yNC4xek00MjEuMiA0MzBjLTkuMy05LjYtMTguNi0yMC4zLTI3LjgtMzIgOSAuNCAxOC4yLjcgMjcuNS43IDkuNCAwIDE4LjctLjIgMjcuOC0uNy05IDExLjctMTguMyAyMi40LTI3LjUgMzJ6bS03NC40LTU4LjljLTE0LjItMi4xLTI3LjktNC43LTQxLTcuOSAzLjctMTIuOSA4LjMtMjYuMiAxMy41LTM5LjUgNC4xIDggOC40IDE2IDEzLjEgMjQgNC43IDggOS41IDE1LjggMTQuNCAyMy40ek00MjAuNyAxNjNjOS4zIDkuNiAxOC42IDIwLjMgMjcuOCAzMi05LS40LTE4LjItLjctMjcuNS0uNy05LjQgMC0xOC43LjItMjcuOC43IDktMTEuNyAxOC4zLTIyLjQgMjcuNS0zMnptLTc0IDU4LjljLTQuOSA3LjctOS44IDE1LjYtMTQuNCAyMy43LTQuNiA4LTguOSAxNi0xMyAyNC01LjQtMTMuNC0xMC0yNi44LTEzLjgtMzkuOCAxMy4xLTMuMSAyNi45LTUuOCA0MS4yLTcuOXptLTkwLjUgMTI1LjJjLTM1LjQtMTUuMS01OC4zLTM0LjktNTguMy01MC42IDAtMTUuNyAyMi45LTM1LjYgNTguMy01MC42IDguNi0zLjcgMTgtNyAyNy43LTEwLjEgNS43IDE5LjYgMTMuMiA0MCAyMi41IDYwLjktOS4yIDIwLjgtMTYuNiA0MS4xLTIyLjIgNjAuNi05LjktMy4xLTE5LjMtNi41LTI4LTEwLjJ6TTMxMCA0OTBjLTEzLjYtNy44LTE5LjUtMzcuNS0xNC45LTc1LjcgMS4xLTkuNCAyLjktMTkuMyA1LjEtMjkuNCAxOS42IDQuOCA0MSA4LjUgNjMuNSAxMC45IDEzLjUgMTguNSAyNy41IDM1LjMgNDEuNiA1MC0zMi42IDMwLjMtNjMuMiA0Ni45LTg0IDQ2LjktNC41LS4xLTguMy0xLTExLjMtMi43em0yMzcuMi03Ni4yYzQuNyAzOC4yLTEuMSA2Ny45LTE0LjYgNzUuOC0zIDEuOC02LjkgMi42LTExLjUgMi42LTIwLjcgMC01MS40LTE2LjUtODQtNDYuNiAxNC0xNC43IDI4LTMxLjQgNDEuMy00OS45IDIyLjYtMi40IDQ0LTYuMSA2My42LTExIDIuMyAxMC4xIDQuMSAxOS44IDUuMiAyOS4xem0zOC41LTY2LjdjLTguNiAzLjctMTggNy0yNy43IDEwLjEtNS43LTE5LjYtMTMuMi00MC0yMi41LTYwLjkgOS4yLTIwLjggMTYuNi00MS4xIDIyLjItNjAuNiA5LjkgMy4xIDE5LjMgNi41IDI4LjEgMTAuMiAzNS40IDE1LjEgNTguMyAzNC45IDU4LjMgNTAuNi0uMSAxNS43LTIzIDM1LjYtNTguNCA1MC42ek0zMjAuOCA3OC40eiIvPgogICAgPGNpcmNsZSBjeD0iNDIwLjkiIGN5PSIyOTYuNSIgcj0iNDUuNyIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-redo: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgICA8cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZD0iTTE4LjQgMTAuNkMxNi41NSA4Ljk5IDE0LjE1IDggMTEuNSA4Yy00LjY1IDAtOC41OCAzLjAzLTkuOTYgNy4yMkwzLjkgMTZjMS4wNS0zLjE5IDQuMDUtNS41IDcuNi01LjUgMS45NSAwIDMuNzMuNzIgNS4xMiAxLjg4TDEzIDE2aDlWN2wtMy42IDMuNnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-refresh: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTkgMTMuNWMtMi40OSAwLTQuNS0yLjAxLTQuNS00LjVTNi41MSA0LjUgOSA0LjVjMS4yNCAwIDIuMzYuNTIgMy4xNyAxLjMzTDEwIDhoNVYzbC0xLjc2IDEuNzZDMTIuMTUgMy42OCAxMC42NiAzIDkgMyA1LjY5IDMgMy4wMSA1LjY5IDMuMDEgOVM1LjY5IDE1IDkgMTVjMi45NyAwIDUuNDMtMi4xNiA1LjktNWgtMS41MmMtLjQ2IDItMi4yNCAzLjUtNC4zOCAzLjV6Ii8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-regex: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0MTQxNDEiPgogICAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbi1hY2NlbnQyIiBmaWxsPSIjRkZGIj4KICAgIDxjaXJjbGUgY2xhc3M9InN0MiIgY3g9IjUuNSIgY3k9IjE0LjUiIHI9IjEuNSIvPgogICAgPHJlY3QgeD0iMTIiIHk9IjQiIGNsYXNzPSJzdDIiIHdpZHRoPSIxIiBoZWlnaHQ9IjgiLz4KICAgIDxyZWN0IHg9IjguNSIgeT0iNy41IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjg2NiAtMC41IDAuNSAwLjg2NiAtMi4zMjU1IDcuMzIxOSkiIGNsYXNzPSJzdDIiIHdpZHRoPSI4IiBoZWlnaHQ9IjEiLz4KICAgIDxyZWN0IHg9IjEyIiB5PSI0IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjUgLTAuODY2IDAuODY2IDAuNSAtMC42Nzc5IDE0LjgyNTIpIiBjbGFzcz0ic3QyIiB3aWR0aD0iMSIgaGVpZ2h0PSI4Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-run: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTggNXYxNGwxMS03eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-running: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUxMiA1MTIiPgogIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICA8cGF0aCBkPSJNMjU2IDhDMTE5IDggOCAxMTkgOCAyNTZzMTExIDI0OCAyNDggMjQ4IDI0OC0xMTEgMjQ4LTI0OFMzOTMgOCAyNTYgOHptOTYgMzI4YzAgOC44LTcuMiAxNi0xNiAxNkgxNzZjLTguOCAwLTE2LTcuMi0xNi0xNlYxNzZjMC04LjggNy4yLTE2IDE2LTE2aDE2MGM4LjggMCAxNiA3LjIgMTYgMTZ2MTYweiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-save: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTE3IDNINWMtMS4xMSAwLTIgLjktMiAydjE0YzAgMS4xLjg5IDIgMiAyaDE0YzEuMSAwIDItLjkgMi0yVjdsLTQtNHptLTUgMTZjLTEuNjYgMC0zLTEuMzQtMy0zczEuMzQtMyAzLTMgMyAxLjM0IDMgMy0xLjM0IDMtMyAzem0zLTEwSDVWNWgxMHY0eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-search: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjEsMTAuOWgtMC43bC0wLjItMC4yYzAuOC0wLjksMS4zLTIuMiwxLjMtMy41YzAtMy0yLjQtNS40LTUuNC01LjRTMS44LDQuMiwxLjgsNy4xczIuNCw1LjQsNS40LDUuNCBjMS4zLDAsMi41LTAuNSwzLjUtMS4zbDAuMiwwLjJ2MC43bDQuMSw0LjFsMS4yLTEuMkwxMi4xLDEwLjl6IE03LjEsMTAuOWMtMi4xLDAtMy43LTEuNy0zLjctMy43czEuNy0zLjcsMy43LTMuN3MzLjcsMS43LDMuNywzLjcgUzkuMiwxMC45LDcuMSwxMC45eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-settings: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkuNDMgMTIuOThjLjA0LS4zMi4wNy0uNjQuMDctLjk4cy0uMDMtLjY2LS4wNy0uOThsMi4xMS0xLjY1Yy4xOS0uMTUuMjQtLjQyLjEyLS42NGwtMi0zLjQ2Yy0uMTItLjIyLS4zOS0uMy0uNjEtLjIybC0yLjQ5IDFjLS41Mi0uNC0xLjA4LS43My0xLjY5LS45OGwtLjM4LTIuNjVBLjQ4OC40ODggMCAwMDE0IDJoLTRjLS4yNSAwLS40Ni4xOC0uNDkuNDJsLS4zOCAyLjY1Yy0uNjEuMjUtMS4xNy41OS0xLjY5Ljk4bC0yLjQ5LTFjLS4yMy0uMDktLjQ5IDAtLjYxLjIybC0yIDMuNDZjLS4xMy4yMi0uMDcuNDkuMTIuNjRsMi4xMSAxLjY1Yy0uMDQuMzItLjA3LjY1LS4wNy45OHMuMDMuNjYuMDcuOThsLTIuMTEgMS42NWMtLjE5LjE1LS4yNC40Mi0uMTIuNjRsMiAzLjQ2Yy4xMi4yMi4zOS4zLjYxLjIybDIuNDktMWMuNTIuNCAxLjA4LjczIDEuNjkuOThsLjM4IDIuNjVjLjAzLjI0LjI0LjQyLjQ5LjQyaDRjLjI1IDAgLjQ2LS4xOC40OS0uNDJsLjM4LTIuNjVjLjYxLS4yNSAxLjE3LS41OSAxLjY5LS45OGwyLjQ5IDFjLjIzLjA5LjQ5IDAgLjYxLS4yMmwyLTMuNDZjLjEyLS4yMi4wNy0uNDktLjEyLS42NGwtMi4xMS0xLjY1ek0xMiAxNS41Yy0xLjkzIDAtMy41LTEuNTctMy41LTMuNXMxLjU3LTMuNSAzLjUtMy41IDMuNSAxLjU3IDMuNSAzLjUtMS41NyAzLjUtMy41IDMuNXoiLz4KPC9zdmc+Cg==);
  --jp-icon-share: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTSAxOCAyIEMgMTYuMzU0OTkgMiAxNSAzLjM1NDk5MDQgMTUgNSBDIDE1IDUuMTkwOTUyOSAxNS4wMjE3OTEgNS4zNzcxMjI0IDE1LjA1NjY0MSA1LjU1ODU5MzggTCA3LjkyMTg3NSA5LjcyMDcwMzEgQyA3LjM5ODUzOTkgOS4yNzc4NTM5IDYuNzMyMDc3MSA5IDYgOSBDIDQuMzU0OTkwNCA5IDMgMTAuMzU0OTkgMyAxMiBDIDMgMTMuNjQ1MDEgNC4zNTQ5OTA0IDE1IDYgMTUgQyA2LjczMjA3NzEgMTUgNy4zOTg1Mzk5IDE0LjcyMjE0NiA3LjkyMTg3NSAxNC4yNzkyOTcgTCAxNS4wNTY2NDEgMTguNDM5NDUzIEMgMTUuMDIxNTU1IDE4LjYyMTUxNCAxNSAxOC44MDgzODYgMTUgMTkgQyAxNSAyMC42NDUwMSAxNi4zNTQ5OSAyMiAxOCAyMiBDIDE5LjY0NTAxIDIyIDIxIDIwLjY0NTAxIDIxIDE5IEMgMjEgMTcuMzU0OTkgMTkuNjQ1MDEgMTYgMTggMTYgQyAxNy4yNjc0OCAxNiAxNi42MDE1OTMgMTYuMjc5MzI4IDE2LjA3ODEyNSAxNi43MjI2NTYgTCA4Ljk0MzM1OTQgMTIuNTU4NTk0IEMgOC45NzgyMDk1IDEyLjM3NzEyMiA5IDEyLjE5MDk1MyA5IDEyIEMgOSAxMS44MDkwNDcgOC45NzgyMDk1IDExLjYyMjg3OCA4Ljk0MzM1OTQgMTEuNDQxNDA2IEwgMTYuMDc4MTI1IDcuMjc5Mjk2OSBDIDE2LjYwMTQ2IDcuNzIyMTQ2MSAxNy4yNjc5MjMgOCAxOCA4IEMgMTkuNjQ1MDEgOCAyMSA2LjY0NTAwOTYgMjEgNSBDIDIxIDMuMzU0OTkwNCAxOS42NDUwMSAyIDE4IDIgeiBNIDE4IDQgQyAxOC41NjQxMjkgNCAxOSA0LjQzNTg3MDYgMTkgNSBDIDE5IDUuNTY0MTI5NCAxOC41NjQxMjkgNiAxOCA2IEMgMTcuNDM1ODcxIDYgMTcgNS41NjQxMjk0IDE3IDUgQyAxNyA0LjQzNTg3MDYgMTcuNDM1ODcxIDQgMTggNCB6IE0gNiAxMSBDIDYuNTY0MTI5NCAxMSA3IDExLjQzNTg3MSA3IDEyIEMgNyAxMi41NjQxMjkgNi41NjQxMjk0IDEzIDYgMTMgQyA1LjQzNTg3MDYgMTMgNSAxMi41NjQxMjkgNSAxMiBDIDUgMTEuNDM1ODcxIDUuNDM1ODcwNiAxMSA2IDExIHogTSAxOCAxOCBDIDE4LjU2NDEyOSAxOCAxOSAxOC40MzU4NzEgMTkgMTkgQyAxOSAxOS41NjQxMjkgMTguNTY0MTI5IDIwIDE4IDIwIEMgMTcuNDM1ODcxIDIwIDE3IDE5LjU2NDEyOSAxNyAxOSBDIDE3IDE4LjQzNTg3MSAxNy40MzU4NzEgMTggMTggMTggeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-spreadsheet: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDEganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNENBRjUwIiBkPSJNMi4yIDIuMnYxNy42aDE3LjZWMi4ySDIuMnptMTUuNCA3LjdoLTUuNVY0LjRoNS41djUuNXpNOS45IDQuNHY1LjVINC40VjQuNGg1LjV6bS01LjUgNy43aDUuNXY1LjVINC40di01LjV6bTcuNyA1LjV2LTUuNWg1LjV2NS41aC01LjV6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-stop: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik02IDZoMTJ2MTJINnoiLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-tab: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIxIDNIM2MtMS4xIDAtMiAuOS0yIDJ2MTRjMCAxLjEuOSAyIDIgMmgxOGMxLjEgMCAyLS45IDItMlY1YzAtMS4xLS45LTItMi0yem0wIDE2SDNWNWgxMHY0aDh2MTB6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-table-rows: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik0yMSw4SDNWNGgxOFY4eiBNMjEsMTBIM3Y0aDE4VjEweiBNMjEsMTZIM3Y0aDE4VjE2eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-tag: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjgiIGhlaWdodD0iMjgiIHZpZXdCb3g9IjAgMCA0MyAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTI4LjgzMzIgMTIuMzM0TDMyLjk5OTggMTYuNTAwN0wzNy4xNjY1IDEyLjMzNEgyOC44MzMyWiIvPgoJCTxwYXRoIGQ9Ik0xNi4yMDk1IDIxLjYxMDRDMTUuNjg3MyAyMi4xMjk5IDE0Ljg0NDMgMjIuMTI5OSAxNC4zMjQ4IDIxLjYxMDRMNi45ODI5IDE0LjcyNDVDNi41NzI0IDE0LjMzOTQgNi4wODMxMyAxMy42MDk4IDYuMDQ3ODYgMTMuMDQ4MkM1Ljk1MzQ3IDExLjUyODggNi4wMjAwMiA4LjYxOTQ0IDYuMDY2MjEgNy4wNzY5NUM2LjA4MjgxIDYuNTE0NzcgNi41NTU0OCA2LjA0MzQ3IDcuMTE4MDQgNi4wMzA1NUM5LjA4ODYzIDUuOTg0NzMgMTMuMjYzOCA1LjkzNTc5IDEzLjY1MTggNi4zMjQyNUwyMS43MzY5IDEzLjYzOUMyMi4yNTYgMTQuMTU4NSAyMS43ODUxIDE1LjQ3MjQgMjEuMjYyIDE1Ljk5NDZMMTYuMjA5NSAyMS42MTA0Wk05Ljc3NTg1IDguMjY1QzkuMzM1NTEgNy44MjU2NiA4LjYyMzUxIDcuODI1NjYgOC4xODI4IDguMjY1QzcuNzQzNDYgOC43MDU3MSA3Ljc0MzQ2IDkuNDE3MzMgOC4xODI4IDkuODU2NjdDOC42MjM4MiAxMC4yOTY0IDkuMzM1ODIgMTAuMjk2NCA5Ljc3NTg1IDkuODU2NjdDMTAuMjE1NiA5LjQxNzMzIDEwLjIxNTYgOC43MDUzMyA5Ljc3NTg1IDguMjY1WiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-terminal: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0IiA+CiAgICA8cmVjdCBjbGFzcz0ianAtdGVybWluYWwtaWNvbi1iYWNrZ3JvdW5kLWNvbG9yIGpwLWljb24tc2VsZWN0YWJsZSIgd2lkdGg9IjIwIiBoZWlnaHQ9IjIwIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgyIDIpIiBmaWxsPSIjMzMzMzMzIi8+CiAgICA8cGF0aCBjbGFzcz0ianAtdGVybWluYWwtaWNvbi1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUtaW52ZXJzZSIgZD0iTTUuMDU2NjQgOC43NjE3MkM1LjA1NjY0IDguNTk3NjYgNS4wMzEyNSA4LjQ1MzEyIDQuOTgwNDcgOC4zMjgxMkM0LjkzMzU5IDguMTk5MjIgNC44NTU0NyA4LjA4MjAzIDQuNzQ2MDkgNy45NzY1NkM0LjY0MDYyIDcuODcxMDkgNC41IDcuNzc1MzkgNC4zMjQyMiA3LjY4OTQ1QzQuMTUyMzQgNy41OTk2MSAzLjk0MzM2IDcuNTExNzIgMy42OTcyNyA3LjQyNTc4QzMuMzAyNzMgNy4yODUxNiAyLjk0MzM2IDcuMTM2NzIgMi42MTkxNCA2Ljk4MDQ3QzIuMjk0OTIgNi44MjQyMiAyLjAxNzU4IDYuNjQyNTggMS43ODcxMSA2LjQzNTU1QzEuNTYwNTUgNi4yMjg1MiAxLjM4NDc3IDUuOTg4MjggMS4yNTk3NyA1LjcxNDg0QzEuMTM0NzcgNS40Mzc1IDEuMDcyMjcgNS4xMDkzOCAxLjA3MjI3IDQuNzMwNDdDMS4wNzIyNyA0LjM5ODQ0IDEuMTI4OTEgNC4wOTU3IDEuMjQyMTkgMy44MjIyN0MxLjM1NTQ3IDMuNTQ0OTIgMS41MTU2MiAzLjMwNDY5IDEuNzIyNjYgMy4xMDE1NkMxLjkyOTY5IDIuODk4NDQgMi4xNzk2OSAyLjczNDM3IDIuNDcyNjYgMi42MDkzOEMyLjc2NTYyIDIuNDg0MzggMy4wOTE4IDIuNDA0MyAzLjQ1MTE3IDIuMzY5MTRWMS4xMDkzOEg0LjM4ODY3VjIuMzgwODZDNC43NDAyMyAyLjQyNzczIDUuMDU2NjQgMi41MjM0NCA1LjMzNzg5IDIuNjY3OTdDNS42MTkxNCAyLjgxMjUgNS44NTc0MiAzLjAwMTk1IDYuMDUyNzMgMy4yMzYzM0M2LjI1MTk1IDMuNDY2OCA2LjQwNDMgMy43NDAyMyA2LjUwOTc3IDQuMDU2NjRDNi42MTkxNCA0LjM2OTE0IDYuNjczODMgNC43MjA3IDYuNjczODMgNS4xMTEzM0g1LjA0NDkyQzUuMDQ0OTIgNC42Mzg2NyA0LjkzNzUgNC4yODEyNSA0LjcyMjY2IDQuMDM5MDZDNC41MDc4MSAzLjc5Mjk3IDQuMjE2OCAzLjY2OTkyIDMuODQ5NjEgMy42Njk5MkMzLjY1MDM5IDMuNjY5OTIgMy40NzY1NiAzLjY5NzI3IDMuMzI4MTIgMy43NTE5NUMzLjE4MzU5IDMuODAyNzMgMy4wNjQ0NSAzLjg3Njk1IDIuOTcwNyAzLjk3NDYxQzIuODc2OTUgNC4wNjgzNiAyLjgwNjY0IDQuMTc5NjkgMi43NTk3NyA0LjMwODU5QzIuNzE2OCA0LjQzNzUgMi42OTUzMSA0LjU3ODEyIDIuNjk1MzEgNC43MzA0N0MyLjY5NTMxIDQuODgyODEgMi43MTY4IDUuMDE5NTMgMi43NTk3NyA1LjE0MDYyQzIuODA2NjQgNS4yNTc4MSAyLjg4MjgxIDUuMzY3MTkgMi45ODgyOCA1LjQ2ODc1QzMuMDk3NjYgNS41NzAzMSAzLjI0MDIzIDUuNjY3OTcgMy40MTYwMiA1Ljc2MTcyQzMuNTkxOCA1Ljg1MTU2IDMuODEwNTUgNS45NDMzNiA0LjA3MjI3IDYuMDM3MTFDNC40NjY4IDYuMTg1NTUgNC44MjQyMiA2LjMzOTg0IDUuMTQ0NTMgNi41QzUuNDY0ODQgNi42NTYyNSA1LjczODI4IDYuODM5ODQgNS45NjQ4NCA3LjA1MDc4QzYuMTk1MzEgNy4yNTc4MSA2LjM3MTA5IDcuNSA2LjQ5MjE5IDcuNzc3MzRDNi42MTcxOSA4LjA1MDc4IDYuNjc5NjkgOC4zNzUgNi42Nzk2OSA4Ljc1QzYuNjc5NjkgOS4wOTM3NSA2LjYyMzA1IDkuNDA0MyA2LjUwOTc3IDkuNjgxNjRDNi4zOTY0OCA5Ljk1NTA4IDYuMjM0MzggMTAuMTkxNCA2LjAyMzQ0IDEwLjM5MDZDNS44MTI1IDEwLjU4OTggNS41NTg1OSAxMC43NSA1LjI2MTcyIDEwLjg3MTFDNC45NjQ4NCAxMC45ODgzIDQuNjMyODEgMTEuMDY0NSA0LjI2NTYyIDExLjA5OTZWMTIuMjQ4SDMuMzMzOThWMTEuMDk5NkMzLjAwMTk1IDExLjA2ODQgMi42Nzk2OSAxMC45OTYxIDIuMzY3MTkgMTAuODgyOEMyLjA1NDY5IDEwLjc2NTYgMS43NzczNCAxMC41OTc3IDEuNTM1MTYgMTAuMzc4OUMxLjI5Njg4IDEwLjE2MDIgMS4xMDU0NyA5Ljg4NDc3IDAuOTYwOTM4IDkuNTUyNzNDMC44MTY0MDYgOS4yMTY4IDAuNzQ0MTQxIDguODE0NDUgMC43NDQxNDEgOC4zNDU3SDIuMzc4OTFDMi4zNzg5MSA4LjYyNjk1IDIuNDE5OTIgOC44NjMyOCAyLjUwMTk1IDkuMDU0NjlDMi41ODM5OCA5LjI0MjE5IDIuNjg5NDUgOS4zOTI1OCAyLjgxODM2IDkuNTA1ODZDMi45NTExNyA5LjYxNTIzIDMuMTAxNTYgOS42OTMzNiAzLjI2OTUzIDkuNzQwMjNDMy40Mzc1IDkuNzg3MTEgMy42MDkzOCA5LjgxMDU1IDMuNzg1MTYgOS44MTA1NUM0LjIwMzEyIDkuODEwNTUgNC41MTk1MyA5LjcxMjg5IDQuNzM0MzggOS41MTc1OEM0Ljk0OTIyIDkuMzIyMjcgNS4wNTY2NCA5LjA3MDMxIDUuMDU2NjQgOC43NjE3MlpNMTMuNDE4IDEyLjI3MTVIOC4wNzQyMlYxMUgxMy40MThWMTIuMjcxNVoiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMuOTUyNjQgNikiIGZpbGw9IndoaXRlIi8+Cjwvc3ZnPgo=);
  --jp-icon-text-editor: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtdGV4dC1lZGl0b3ItaWNvbi1jb2xvciBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xNSAxNUgzdjJoMTJ2LTJ6bTAtOEgzdjJoMTJWN3pNMyAxM2gxOHYtMkgzdjJ6bTAgOGgxOHYtMkgzdjJ6TTMgM3YyaDE4VjNIM3oiLz4KPC9zdmc+Cg==);
  --jp-icon-toc: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxwYXRoIGQ9Ik03LDVIMjFWN0g3VjVNNywxM1YxMUgyMVYxM0g3TTQsNC41QTEuNSwxLjUgMCAwLDEgNS41LDZBMS41LDEuNSAwIDAsMSA0LDcuNUExLjUsMS41IDAgMCwxIDIuNSw2QTEuNSwxLjUgMCAwLDEgNCw0LjVNNCwxMC41QTEuNSwxLjUgMCAwLDEgNS41LDEyQTEuNSwxLjUgMCAwLDEgNCwxMy41QTEuNSwxLjUgMCAwLDEgMi41LDEyQTEuNSwxLjUgMCAwLDEgNCwxMC41TTcsMTlWMTdIMjFWMTlIN000LDE2LjVBMS41LDEuNSAwIDAsMSA1LjUsMThBMS41LDEuNSAwIDAsMSA0LDE5LjVBMS41LDEuNSAwIDAsMSAyLjUsMThBMS41LDEuNSAwIDAsMSA0LDE2LjVaIiAvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-tree-view: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik0yMiAxMVYzaC03djNIOVYzSDJ2OGg3VjhoMnYxMGg0djNoN3YtOGgtN3YzaC0yVjhoMnYzeiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-trusted: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI1Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDIgMykiIGQ9Ik0xLjg2MDk0IDExLjQ0MDlDMC44MjY0NDggOC43NzAyNyAwLjg2Mzc3OSA2LjA1NzY0IDEuMjQ5MDcgNC4xOTkzMkMyLjQ4MjA2IDMuOTMzNDcgNC4wODA2OCAzLjQwMzQ3IDUuNjAxMDIgMi44NDQ5QzcuMjM1NDkgMi4yNDQ0IDguODU2NjYgMS41ODE1IDkuOTg3NiAxLjA5NTM5QzExLjA1OTcgMS41ODM0MSAxMi42MDk0IDIuMjQ0NCAxNC4yMTggMi44NDMzOUMxNS43NTAzIDMuNDEzOTQgMTcuMzk5NSAzLjk1MjU4IDE4Ljc1MzkgNC4yMTM4NUMxOS4xMzY0IDYuMDcxNzcgMTkuMTcwOSA4Ljc3NzIyIDE4LjEzOSAxMS40NDA5QzE3LjAzMDMgMTQuMzAzMiAxNC42NjY4IDE3LjE4NDQgOS45OTk5OSAxOC45MzU0QzUuMzMzMiAxNy4xODQ0IDIuOTY5NjggMTQuMzAzMiAxLjg2MDk0IDExLjQ0MDlaIi8+CiAgICA8cGF0aCBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiMzMzMzMzMiIHN0cm9rZT0iIzMzMzMzMyIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOCA5Ljg2NzE5KSIgZD0iTTIuODYwMTUgNC44NjUzNUwwLjcyNjU0OSAyLjk5OTU5TDAgMy42MzA0NUwyLjg2MDE1IDYuMTMxNTdMOCAwLjYzMDg3Mkw3LjI3ODU3IDBMMi44NjAxNSA0Ljg2NTM1WiIvPgo8L3N2Zz4K);
  --jp-icon-undo: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjUgOGMtMi42NSAwLTUuMDUuOTktNi45IDIuNkwyIDd2OWg5bC0zLjYyLTMuNjJjMS4zOS0xLjE2IDMuMTYtMS44OCA1LjEyLTEuODggMy41NCAwIDYuNTUgMi4zMSA3LjYgNS41bDIuMzctLjc4QzIxLjA4IDExLjAzIDE3LjE1IDggMTIuNSA4eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-user: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE2IDdhNCA0IDAgMTEtOCAwIDQgNCAwIDAxOCAwek0xMiAxNGE3IDcgMCAwMC03IDdoMTRhNyA3IDAgMDAtNy03eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-users: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZlcnNpb249IjEuMSIgdmlld0JveD0iMCAwIDM2IDI0IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgogPGcgY2xhc3M9ImpwLWljb24zIiB0cmFuc2Zvcm09Im1hdHJpeCgxLjczMjcgMCAwIDEuNzMyNyAtMy42MjgyIC4wOTk1NzcpIiBmaWxsPSIjNjE2MTYxIj4KICA8cGF0aCB0cmFuc2Zvcm09Im1hdHJpeCgxLjUsMCwwLDEuNSwwLC02KSIgZD0ibTEyLjE4NiA3LjUwOThjLTEuMDUzNSAwLTEuOTc1NyAwLjU2NjUtMi40Nzg1IDEuNDEwMiAwLjc1MDYxIDAuMzEyNzcgMS4zOTc0IDAuODI2NDggMS44NzMgMS40NzI3aDMuNDg2M2MwLTEuNTkyLTEuMjg4OS0yLjg4MjgtMi44ODA5LTIuODgyOHoiLz4KICA8cGF0aCBkPSJtMjAuNDY1IDIuMzg5NWEyLjE4ODUgMi4xODg1IDAgMCAxLTIuMTg4NCAyLjE4ODUgMi4xODg1IDIuMTg4NSAwIDAgMS0yLjE4ODUtMi4xODg1IDIuMTg4NSAyLjE4ODUgMCAwIDEgMi4xODg1LTIuMTg4NSAyLjE4ODUgMi4xODg1IDAgMCAxIDIuMTg4NCAyLjE4ODV6Ii8+CiAgPHBhdGggdHJhbnNmb3JtPSJtYXRyaXgoMS41LDAsMCwxLjUsMCwtNikiIGQ9Im0zLjU4OTggOC40MjE5Yy0xLjExMjYgMC0yLjAxMzcgMC45MDExMS0yLjAxMzcgMi4wMTM3aDIuODE0NWMwLjI2Nzk3LTAuMzczMDkgMC41OTA3LTAuNzA0MzUgMC45NTg5OC0wLjk3ODUyLTAuMzQ0MzMtMC42MTY4OC0xLjAwMzEtMS4wMzUyLTEuNzU5OC0xLjAzNTJ6Ii8+CiAgPHBhdGggZD0ibTYuOTE1NCA0LjYyM2ExLjUyOTQgMS41Mjk0IDAgMCAxLTEuNTI5NCAxLjUyOTQgMS41Mjk0IDEuNTI5NCAwIDAgMS0xLjUyOTQtMS41Mjk0IDEuNTI5NCAxLjUyOTQgMCAwIDEgMS41Mjk0LTEuNTI5NCAxLjUyOTQgMS41Mjk0IDAgMCAxIDEuNTI5NCAxLjUyOTR6Ii8+CiAgPHBhdGggZD0ibTYuMTM1IDEzLjUzNWMwLTMuMjM5MiAyLjYyNTktNS44NjUgNS44NjUtNS44NjUgMy4yMzkyIDAgNS44NjUgMi42MjU5IDUuODY1IDUuODY1eiIvPgogIDxjaXJjbGUgY3g9IjEyIiBjeT0iMy43Njg1IiByPSIyLjk2ODUiLz4KIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-vega: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbjEganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMjEyMTIxIj4KICAgIDxwYXRoIGQ9Ik0xMC42IDUuNGwyLjItMy4ySDIuMnY3LjNsNC02LjZ6Ii8+CiAgICA8cGF0aCBkPSJNMTUuOCAyLjJsLTQuNCA2LjZMNyA2LjNsLTQuOCA4djUuNWgxNy42VjIuMmgtNHptLTcgMTUuNEg1LjV2LTQuNGgzLjN2NC40em00LjQgMEg5LjhWOS44aDMuNHY3Ljh6bTQuNCAwaC0zLjRWNi41aDMuNHYxMS4xeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-word: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KIDxnIGNsYXNzPSJqcC1pY29uMiIgZmlsbD0iIzQxNDE0MSI+CiAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiA8L2c+CiA8ZyBjbGFzcz0ianAtaWNvbi1hY2NlbnQyIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSguNDMgLjA0MDEpIiBmaWxsPSIjZmZmIj4KICA8cGF0aCBkPSJtNC4xNCA4Ljc2cTAuMDY4Mi0xLjg5IDIuNDItMS44OSAxLjE2IDAgMS42OCAwLjQyIDAuNTY3IDAuNDEgMC41NjcgMS4xNnYzLjQ3cTAgMC40NjIgMC41MTQgMC40NjIgMC4xMDMgMCAwLjItMC4wMjMxdjAuNzE0cS0wLjM5OSAwLjEwMy0wLjY1MSAwLjEwMy0wLjQ1MiAwLTAuNjkzLTAuMjItMC4yMzEtMC4yLTAuMjg0LTAuNjYyLTAuOTU2IDAuODcyLTIgMC44NzItMC45MDMgMC0xLjQ3LTAuNDcyLTAuNTI1LTAuNDcyLTAuNTI1LTEuMjYgMC0wLjI2MiAwLjA0NTItMC40NzIgMC4wNTY3LTAuMjIgMC4xMTYtMC4zNzggMC4wNjgyLTAuMTY4IDAuMjMxLTAuMzA0IDAuMTU4LTAuMTQ3IDAuMjYyLTAuMjQyIDAuMTE2LTAuMDkxNCAwLjM2OC0wLjE2OCAwLjI2Mi0wLjA5MTQgMC4zOTktMC4xMjYgMC4xMzYtMC4wNDUyIDAuNDcyLTAuMTAzIDAuMzM2LTAuMDU3OCAwLjUwNC0wLjA3OTggMC4xNTgtMC4wMjMxIDAuNTY3LTAuMDc5OCAwLjU1Ni0wLjA2ODIgMC43NzctMC4yMjEgMC4yMi0wLjE1MiAwLjIyLTAuNDQxdi0wLjI1MnEwLTAuNDMtMC4zNTctMC42NjItMC4zMzYtMC4yMzEtMC45NzYtMC4yMzEtMC42NjIgMC0wLjk5OCAwLjI2Mi0wLjMzNiAwLjI1Mi0wLjM5OSAwLjc5OHptMS44OSAzLjY4cTAuNzg4IDAgMS4yNi0wLjQxIDAuNTA0LTAuNDIgMC41MDQtMC45MDN2LTEuMDVxLTAuMjg0IDAuMTM2LTAuODYxIDAuMjMxLTAuNTY3IDAuMDkxNC0wLjk4NyAwLjE1OC0wLjQyIDAuMDY4Mi0wLjc2NiAwLjMyNi0wLjMzNiAwLjI1Mi0wLjMzNiAwLjcwNHQwLjMwNCAwLjcwNCAwLjg2MSAwLjI1MnoiIHN0cm9rZS13aWR0aD0iMS4wNSIvPgogIDxwYXRoIGQ9Im0xMCA0LjU2aDAuOTQ1djMuMTVxMC42NTEtMC45NzYgMS44OS0wLjk3NiAxLjE2IDAgMS44OSAwLjg0IDAuNjgyIDAuODQgMC42ODIgMi4zMSAwIDEuNDctMC43MDQgMi40Mi0wLjcwNCAwLjg4Mi0xLjg5IDAuODgyLTEuMjYgMC0xLjg5LTEuMDJ2MC43NjZoLTAuODV6bTIuNjIgMy4wNHEtMC43NDYgMC0xLjE2IDAuNjQtMC40NTIgMC42My0wLjQ1MiAxLjY4IDAgMS4wNSAwLjQ1MiAxLjY4dDEuMTYgMC42M3EwLjc3NyAwIDEuMjYtMC42MyAwLjQ5NC0wLjY0IDAuNDk0LTEuNjggMC0xLjA1LTAuNDcyLTEuNjgtMC40NjItMC42NC0xLjI2LTAuNjR6IiBzdHJva2Utd2lkdGg9IjEuMDUiLz4KICA8cGF0aCBkPSJtMi43MyAxNS44IDEzLjYgMC4wMDgxYzAuMDA2OSAwIDAtMi42IDAtMi42IDAtMC4wMDc4LTEuMTUgMC0xLjE1IDAtMC4wMDY5IDAtMC4wMDgzIDEuNS0wLjAwODMgMS41LTJlLTMgLTAuMDAxNC0xMS4zLTAuMDAxNC0xMS4zLTAuMDAxNGwtMC4wMDU5Mi0xLjVjMC0wLjAwNzgtMS4xNyAwLjAwMTMtMS4xNyAwLjAwMTN6IiBzdHJva2Utd2lkdGg9Ii45NzUiLz4KIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-yaml: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1jb250cmFzdDIganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjRDgxQjYwIj4KICAgIDxwYXRoIGQ9Ik03LjIgMTguNnYtNS40TDMgNS42aDMuM2wxLjQgMy4xYy4zLjkuNiAxLjYgMSAyLjUuMy0uOC42LTEuNiAxLTIuNWwxLjQtMy4xaDMuNGwtNC40IDcuNnY1LjVsLTIuOS0uMXoiLz4KICAgIDxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE3LjYiIGN5PSIxNi41IiByPSIyLjEiLz4KICAgIDxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE3LjYiIGN5PSIxMSIgcj0iMi4xIi8+CiAgPC9nPgo8L3N2Zz4K);
}

/* Icon CSS class declarations */

.jp-AddAboveIcon {
  background-image: var(--jp-icon-add-above);
}

.jp-AddBelowIcon {
  background-image: var(--jp-icon-add-below);
}

.jp-AddIcon {
  background-image: var(--jp-icon-add);
}

.jp-BellIcon {
  background-image: var(--jp-icon-bell);
}

.jp-BugDotIcon {
  background-image: var(--jp-icon-bug-dot);
}

.jp-BugIcon {
  background-image: var(--jp-icon-bug);
}

.jp-BuildIcon {
  background-image: var(--jp-icon-build);
}

.jp-CaretDownEmptyIcon {
  background-image: var(--jp-icon-caret-down-empty);
}

.jp-CaretDownEmptyThinIcon {
  background-image: var(--jp-icon-caret-down-empty-thin);
}

.jp-CaretDownIcon {
  background-image: var(--jp-icon-caret-down);
}

.jp-CaretLeftIcon {
  background-image: var(--jp-icon-caret-left);
}

.jp-CaretRightIcon {
  background-image: var(--jp-icon-caret-right);
}

.jp-CaretUpEmptyThinIcon {
  background-image: var(--jp-icon-caret-up-empty-thin);
}

.jp-CaretUpIcon {
  background-image: var(--jp-icon-caret-up);
}

.jp-CaseSensitiveIcon {
  background-image: var(--jp-icon-case-sensitive);
}

.jp-CheckIcon {
  background-image: var(--jp-icon-check);
}

.jp-CircleEmptyIcon {
  background-image: var(--jp-icon-circle-empty);
}

.jp-CircleIcon {
  background-image: var(--jp-icon-circle);
}

.jp-ClearIcon {
  background-image: var(--jp-icon-clear);
}

.jp-CloseIcon {
  background-image: var(--jp-icon-close);
}

.jp-CodeCheckIcon {
  background-image: var(--jp-icon-code-check);
}

.jp-CodeIcon {
  background-image: var(--jp-icon-code);
}

.jp-CollapseAllIcon {
  background-image: var(--jp-icon-collapse-all);
}

.jp-ConsoleIcon {
  background-image: var(--jp-icon-console);
}

.jp-CopyIcon {
  background-image: var(--jp-icon-copy);
}

.jp-CopyrightIcon {
  background-image: var(--jp-icon-copyright);
}

.jp-CutIcon {
  background-image: var(--jp-icon-cut);
}

.jp-DeleteIcon {
  background-image: var(--jp-icon-delete);
}

.jp-DownloadIcon {
  background-image: var(--jp-icon-download);
}

.jp-DuplicateIcon {
  background-image: var(--jp-icon-duplicate);
}

.jp-EditIcon {
  background-image: var(--jp-icon-edit);
}

.jp-EllipsesIcon {
  background-image: var(--jp-icon-ellipses);
}

.jp-ErrorIcon {
  background-image: var(--jp-icon-error);
}

.jp-ExpandAllIcon {
  background-image: var(--jp-icon-expand-all);
}

.jp-ExtensionIcon {
  background-image: var(--jp-icon-extension);
}

.jp-FastForwardIcon {
  background-image: var(--jp-icon-fast-forward);
}

.jp-FileIcon {
  background-image: var(--jp-icon-file);
}

.jp-FileUploadIcon {
  background-image: var(--jp-icon-file-upload);
}

.jp-FilterDotIcon {
  background-image: var(--jp-icon-filter-dot);
}

.jp-FilterIcon {
  background-image: var(--jp-icon-filter);
}

.jp-FilterListIcon {
  background-image: var(--jp-icon-filter-list);
}

.jp-FolderFavoriteIcon {
  background-image: var(--jp-icon-folder-favorite);
}

.jp-FolderIcon {
  background-image: var(--jp-icon-folder);
}

.jp-HomeIcon {
  background-image: var(--jp-icon-home);
}

.jp-Html5Icon {
  background-image: var(--jp-icon-html5);
}

.jp-ImageIcon {
  background-image: var(--jp-icon-image);
}

.jp-InfoIcon {
  background-image: var(--jp-icon-info);
}

.jp-InspectorIcon {
  background-image: var(--jp-icon-inspector);
}

.jp-JsonIcon {
  background-image: var(--jp-icon-json);
}

.jp-JuliaIcon {
  background-image: var(--jp-icon-julia);
}

.jp-JupyterFaviconIcon {
  background-image: var(--jp-icon-jupyter-favicon);
}

.jp-JupyterIcon {
  background-image: var(--jp-icon-jupyter);
}

.jp-JupyterlabWordmarkIcon {
  background-image: var(--jp-icon-jupyterlab-wordmark);
}

.jp-KernelIcon {
  background-image: var(--jp-icon-kernel);
}

.jp-KeyboardIcon {
  background-image: var(--jp-icon-keyboard);
}

.jp-LaunchIcon {
  background-image: var(--jp-icon-launch);
}

.jp-LauncherIcon {
  background-image: var(--jp-icon-launcher);
}

.jp-LineFormIcon {
  background-image: var(--jp-icon-line-form);
}

.jp-LinkIcon {
  background-image: var(--jp-icon-link);
}

.jp-ListIcon {
  background-image: var(--jp-icon-list);
}

.jp-MarkdownIcon {
  background-image: var(--jp-icon-markdown);
}

.jp-MoveDownIcon {
  background-image: var(--jp-icon-move-down);
}

.jp-MoveUpIcon {
  background-image: var(--jp-icon-move-up);
}

.jp-NewFolderIcon {
  background-image: var(--jp-icon-new-folder);
}

.jp-NotTrustedIcon {
  background-image: var(--jp-icon-not-trusted);
}

.jp-NotebookIcon {
  background-image: var(--jp-icon-notebook);
}

.jp-NumberingIcon {
  background-image: var(--jp-icon-numbering);
}

.jp-OfflineBoltIcon {
  background-image: var(--jp-icon-offline-bolt);
}

.jp-PaletteIcon {
  background-image: var(--jp-icon-palette);
}

.jp-PasteIcon {
  background-image: var(--jp-icon-paste);
}

.jp-PdfIcon {
  background-image: var(--jp-icon-pdf);
}

.jp-PythonIcon {
  background-image: var(--jp-icon-python);
}

.jp-RKernelIcon {
  background-image: var(--jp-icon-r-kernel);
}

.jp-ReactIcon {
  background-image: var(--jp-icon-react);
}

.jp-RedoIcon {
  background-image: var(--jp-icon-redo);
}

.jp-RefreshIcon {
  background-image: var(--jp-icon-refresh);
}

.jp-RegexIcon {
  background-image: var(--jp-icon-regex);
}

.jp-RunIcon {
  background-image: var(--jp-icon-run);
}

.jp-RunningIcon {
  background-image: var(--jp-icon-running);
}

.jp-SaveIcon {
  background-image: var(--jp-icon-save);
}

.jp-SearchIcon {
  background-image: var(--jp-icon-search);
}

.jp-SettingsIcon {
  background-image: var(--jp-icon-settings);
}

.jp-ShareIcon {
  background-image: var(--jp-icon-share);
}

.jp-SpreadsheetIcon {
  background-image: var(--jp-icon-spreadsheet);
}

.jp-StopIcon {
  background-image: var(--jp-icon-stop);
}

.jp-TabIcon {
  background-image: var(--jp-icon-tab);
}

.jp-TableRowsIcon {
  background-image: var(--jp-icon-table-rows);
}

.jp-TagIcon {
  background-image: var(--jp-icon-tag);
}

.jp-TerminalIcon {
  background-image: var(--jp-icon-terminal);
}

.jp-TextEditorIcon {
  background-image: var(--jp-icon-text-editor);
}

.jp-TocIcon {
  background-image: var(--jp-icon-toc);
}

.jp-TreeViewIcon {
  background-image: var(--jp-icon-tree-view);
}

.jp-TrustedIcon {
  background-image: var(--jp-icon-trusted);
}

.jp-UndoIcon {
  background-image: var(--jp-icon-undo);
}

.jp-UserIcon {
  background-image: var(--jp-icon-user);
}

.jp-UsersIcon {
  background-image: var(--jp-icon-users);
}

.jp-VegaIcon {
  background-image: var(--jp-icon-vega);
}

.jp-WordIcon {
  background-image: var(--jp-icon-word);
}

.jp-YamlIcon {
  background-image: var(--jp-icon-yaml);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * (DEPRECATED) Support for consuming icons as CSS background images
 */

.jp-Icon,
.jp-MaterialIcon {
  background-position: center;
  background-repeat: no-repeat;
  background-size: 16px;
  min-width: 16px;
  min-height: 16px;
}

.jp-Icon-cover {
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}

/**
 * (DEPRECATED) Support for specific CSS icon sizes
 */

.jp-Icon-16 {
  background-size: 16px;
  min-width: 16px;
  min-height: 16px;
}

.jp-Icon-18 {
  background-size: 18px;
  min-width: 18px;
  min-height: 18px;
}

.jp-Icon-20 {
  background-size: 20px;
  min-width: 20px;
  min-height: 20px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.lm-TabBar .lm-TabBar-addButton {
  align-items: center;
  display: flex;
  padding: 4px;
  padding-bottom: 5px;
  margin-right: 1px;
  background-color: var(--jp-layout-color2);
}

.lm-TabBar .lm-TabBar-addButton:hover {
  background-color: var(--jp-layout-color1);
}

.lm-DockPanel-tabBar .lm-TabBar-tab {
  width: var(--jp-private-horizontal-tab-width);
}

.lm-DockPanel-tabBar .lm-TabBar-content {
  flex: unset;
}

.lm-DockPanel-tabBar[data-orientation='horizontal'] {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * Support for icons as inline SVG HTMLElements
 */

/* recolor the primary elements of an icon */
.jp-icon0[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon1[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon2[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon3[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon4[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon0[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon1[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon2[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon3[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon4[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/* recolor the accent elements of an icon */
.jp-icon-accent0[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-accent1[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-accent2[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-accent3[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-accent4[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-accent0[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-accent1[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-accent2[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-accent3[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-accent4[stroke] {
  stroke: var(--jp-layout-color4);
}

/* set the color of an icon to transparent */
.jp-icon-none[fill] {
  fill: none;
}

.jp-icon-none[stroke] {
  stroke: none;
}

/* brand icon colors. Same for light and dark */
.jp-icon-brand0[fill] {
  fill: var(--jp-brand-color0);
}

.jp-icon-brand1[fill] {
  fill: var(--jp-brand-color1);
}

.jp-icon-brand2[fill] {
  fill: var(--jp-brand-color2);
}

.jp-icon-brand3[fill] {
  fill: var(--jp-brand-color3);
}

.jp-icon-brand4[fill] {
  fill: var(--jp-brand-color4);
}

.jp-icon-brand0[stroke] {
  stroke: var(--jp-brand-color0);
}

.jp-icon-brand1[stroke] {
  stroke: var(--jp-brand-color1);
}

.jp-icon-brand2[stroke] {
  stroke: var(--jp-brand-color2);
}

.jp-icon-brand3[stroke] {
  stroke: var(--jp-brand-color3);
}

.jp-icon-brand4[stroke] {
  stroke: var(--jp-brand-color4);
}

/* warn icon colors. Same for light and dark */
.jp-icon-warn0[fill] {
  fill: var(--jp-warn-color0);
}

.jp-icon-warn1[fill] {
  fill: var(--jp-warn-color1);
}

.jp-icon-warn2[fill] {
  fill: var(--jp-warn-color2);
}

.jp-icon-warn3[fill] {
  fill: var(--jp-warn-color3);
}

.jp-icon-warn0[stroke] {
  stroke: var(--jp-warn-color0);
}

.jp-icon-warn1[stroke] {
  stroke: var(--jp-warn-color1);
}

.jp-icon-warn2[stroke] {
  stroke: var(--jp-warn-color2);
}

.jp-icon-warn3[stroke] {
  stroke: var(--jp-warn-color3);
}

/* icon colors that contrast well with each other and most backgrounds */
.jp-icon-contrast0[fill] {
  fill: var(--jp-icon-contrast-color0);
}

.jp-icon-contrast1[fill] {
  fill: var(--jp-icon-contrast-color1);
}

.jp-icon-contrast2[fill] {
  fill: var(--jp-icon-contrast-color2);
}

.jp-icon-contrast3[fill] {
  fill: var(--jp-icon-contrast-color3);
}

.jp-icon-contrast0[stroke] {
  stroke: var(--jp-icon-contrast-color0);
}

.jp-icon-contrast1[stroke] {
  stroke: var(--jp-icon-contrast-color1);
}

.jp-icon-contrast2[stroke] {
  stroke: var(--jp-icon-contrast-color2);
}

.jp-icon-contrast3[stroke] {
  stroke: var(--jp-icon-contrast-color3);
}

.jp-icon-dot[fill] {
  fill: var(--jp-warn-color0);
}

.jp-jupyter-icon-color[fill] {
  fill: var(--jp-jupyter-icon-color, var(--jp-warn-color0));
}

.jp-notebook-icon-color[fill] {
  fill: var(--jp-notebook-icon-color, var(--jp-warn-color0));
}

.jp-json-icon-color[fill] {
  fill: var(--jp-json-icon-color, var(--jp-warn-color1));
}

.jp-console-icon-color[fill] {
  fill: var(--jp-console-icon-color, white);
}

.jp-console-icon-background-color[fill] {
  fill: var(--jp-console-icon-background-color, var(--jp-brand-color1));
}

.jp-terminal-icon-color[fill] {
  fill: var(--jp-terminal-icon-color, var(--jp-layout-color2));
}

.jp-terminal-icon-background-color[fill] {
  fill: var(
    --jp-terminal-icon-background-color,
    var(--jp-inverse-layout-color2)
  );
}

.jp-text-editor-icon-color[fill] {
  fill: var(--jp-text-editor-icon-color, var(--jp-inverse-layout-color3));
}

.jp-inspector-icon-color[fill] {
  fill: var(--jp-inspector-icon-color, var(--jp-inverse-layout-color3));
}

/* CSS for icons in selected filebrowser listing items */
.jp-DirListing-item.jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}

.jp-DirListing-item.jp-mod-selected .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}

/* stylelint-disable selector-max-class, selector-max-compound-selectors */

/**
* TODO: come up with non css-hack solution for showing the busy icon on top
*  of the close icon
* CSS for complex behavior of close icon of tabs in the main area tabbar
*/
.lm-DockPanel-tabBar
  .lm-TabBar-tab.lm-mod-closable.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon3[fill] {
  fill: none;
}

.lm-DockPanel-tabBar
  .lm-TabBar-tab.lm-mod-closable.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon-busy[fill] {
  fill: var(--jp-inverse-layout-color3);
}

/* stylelint-enable selector-max-class, selector-max-compound-selectors */

/* CSS for icons in status bar */
#jp-main-statusbar .jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}

#jp-main-statusbar .jp-mod-selected .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}

/* special handling for splash icon CSS. While the theme CSS reloads during
   splash, the splash icon can loose theming. To prevent that, we set a
   default for its color variable */
:root {
  --jp-warn-color0: var(--md-orange-700);
}

/* not sure what to do with this one, used in filebrowser listing */
.jp-DragIcon {
  margin-right: 4px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * Support for alt colors for icons as inline SVG HTMLElements
 */

/* alt recolor the primary elements of an icon */
.jp-icon-alt .jp-icon0[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-alt .jp-icon1[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-alt .jp-icon2[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-alt .jp-icon3[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-alt .jp-icon4[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-alt .jp-icon0[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-alt .jp-icon1[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-alt .jp-icon2[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-alt .jp-icon3[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-alt .jp-icon4[stroke] {
  stroke: var(--jp-layout-color4);
}

/* alt recolor the accent elements of an icon */
.jp-icon-alt .jp-icon-accent0[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon-alt .jp-icon-accent1[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon-alt .jp-icon-accent2[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon-alt .jp-icon-accent3[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon-alt .jp-icon-accent4[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-alt .jp-icon-accent0[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon-alt .jp-icon-accent1[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon-alt .jp-icon-accent2[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon-alt .jp-icon-accent3[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon-alt .jp-icon-accent4[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-icon-hoverShow:not(:hover) .jp-icon-hoverShow-content {
  display: none !important;
}

/**
 * Support for hover colors for icons as inline SVG HTMLElements
 */

/**
 * regular colors
 */

/* recolor the primary elements of an icon */
.jp-icon-hover :hover .jp-icon0-hover[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon-hover :hover .jp-icon1-hover[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon-hover :hover .jp-icon2-hover[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon-hover :hover .jp-icon3-hover[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon-hover :hover .jp-icon4-hover[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-hover :hover .jp-icon0-hover[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon-hover :hover .jp-icon1-hover[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon-hover :hover .jp-icon2-hover[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon-hover :hover .jp-icon3-hover[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon-hover :hover .jp-icon4-hover[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/* recolor the accent elements of an icon */
.jp-icon-hover :hover .jp-icon-accent0-hover[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-hover :hover .jp-icon-accent1-hover[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-hover :hover .jp-icon-accent2-hover[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-hover :hover .jp-icon-accent3-hover[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-hover :hover .jp-icon-accent4-hover[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-hover :hover .jp-icon-accent0-hover[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-hover :hover .jp-icon-accent1-hover[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-hover :hover .jp-icon-accent2-hover[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-hover :hover .jp-icon-accent3-hover[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-hover :hover .jp-icon-accent4-hover[stroke] {
  stroke: var(--jp-layout-color4);
}

/* set the color of an icon to transparent */
.jp-icon-hover :hover .jp-icon-none-hover[fill] {
  fill: none;
}

.jp-icon-hover :hover .jp-icon-none-hover[stroke] {
  stroke: none;
}

/**
 * inverse colors
 */

/* inverse recolor the primary elements of an icon */
.jp-icon-hover.jp-icon-alt :hover .jp-icon0-hover[fill] {
  fill: var(--jp-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon1-hover[fill] {
  fill: var(--jp-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon2-hover[fill] {
  fill: var(--jp-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon3-hover[fill] {
  fill: var(--jp-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon4-hover[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon0-hover[stroke] {
  stroke: var(--jp-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon1-hover[stroke] {
  stroke: var(--jp-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon2-hover[stroke] {
  stroke: var(--jp-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon3-hover[stroke] {
  stroke: var(--jp-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon4-hover[stroke] {
  stroke: var(--jp-layout-color4);
}

/* inverse recolor the accent elements of an icon */
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent0-hover[fill] {
  fill: var(--jp-inverse-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent1-hover[fill] {
  fill: var(--jp-inverse-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent2-hover[fill] {
  fill: var(--jp-inverse-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent3-hover[fill] {
  fill: var(--jp-inverse-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent4-hover[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent0-hover[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent1-hover[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent2-hover[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent3-hover[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent4-hover[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-IFrame {
  width: 100%;
  height: 100%;
}

.jp-IFrame > iframe {
  border: none;
}

/*
When drag events occur, `lm-mod-override-cursor` is added to the body.
Because iframes steal all cursor events, the following two rules are necessary
to suppress pointer events while resize drags are occurring. There may be a
better solution to this problem.
*/
body.lm-mod-override-cursor .jp-IFrame {
  position: relative;
}

body.lm-mod-override-cursor .jp-IFrame::before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-HoverBox {
  position: fixed;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-FormGroup-content fieldset {
  border: none;
  padding: 0;
  min-width: 0;
  width: 100%;
}

/* stylelint-disable selector-max-type */

.jp-FormGroup-content fieldset .jp-inputFieldWrapper input,
.jp-FormGroup-content fieldset .jp-inputFieldWrapper select,
.jp-FormGroup-content fieldset .jp-inputFieldWrapper textarea {
  font-size: var(--jp-content-font-size2);
  border-color: var(--jp-input-border-color);
  border-style: solid;
  border-radius: var(--jp-border-radius);
  border-width: 1px;
  padding: 6px 8px;
  background: none;
  color: var(--jp-ui-font-color0);
  height: inherit;
}

.jp-FormGroup-content fieldset input[type='checkbox'] {
  position: relative;
  top: 2px;
  margin-left: 0;
}

.jp-FormGroup-content button.jp-mod-styled {
  cursor: pointer;
}

.jp-FormGroup-content .checkbox label {
  cursor: pointer;
  font-size: var(--jp-content-font-size1);
}

.jp-FormGroup-content .jp-root > fieldset > legend {
  display: none;
}

.jp-FormGroup-content .jp-root > fieldset > p {
  display: none;
}

/** copy of `input.jp-mod-styled:focus` style */
.jp-FormGroup-content fieldset input:focus,
.jp-FormGroup-content fieldset select:focus {
  -moz-outline-radius: unset;
  outline: var(--jp-border-width) solid var(--md-blue-500);
  outline-offset: -1px;
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-FormGroup-content fieldset input:hover:not(:focus),
.jp-FormGroup-content fieldset select:hover:not(:focus) {
  background-color: var(--jp-border-color2);
}

/* stylelint-enable selector-max-type */

.jp-FormGroup-content .checkbox .field-description {
  /* Disable default description field for checkbox:
   because other widgets do not have description fields,
   we add descriptions to each widget on the field level.
  */
  display: none;
}

.jp-FormGroup-content #root__description {
  display: none;
}

.jp-FormGroup-content .jp-modifiedIndicator {
  width: 5px;
  background-color: var(--jp-brand-color2);
  margin-top: 0;
  margin-left: calc(var(--jp-private-settingeditor-modifier-indent) * -1);
  flex-shrink: 0;
}

.jp-FormGroup-content .jp-modifiedIndicator.jp-errorIndicator {
  background-color: var(--jp-error-color0);
  margin-right: 0.5em;
}

/* RJSF ARRAY style */

.jp-arrayFieldWrapper legend {
  font-size: var(--jp-content-font-size2);
  color: var(--jp-ui-font-color0);
  flex-basis: 100%;
  padding: 4px 0;
  font-weight: var(--jp-content-heading-font-weight);
  border-bottom: 1px solid var(--jp-border-color2);
}

.jp-arrayFieldWrapper .field-description {
  padding: 4px 0;
  white-space: pre-wrap;
}

.jp-arrayFieldWrapper .array-item {
  width: 100%;
  border: 1px solid var(--jp-border-color2);
  border-radius: 4px;
  margin: 4px;
}

.jp-ArrayOperations {
  display: flex;
  margin-left: 8px;
}

.jp-ArrayOperationsButton {
  margin: 2px;
}

.jp-ArrayOperationsButton .jp-icon3[fill] {
  fill: var(--jp-ui-font-color0);
}

button.jp-ArrayOperationsButton.jp-mod-styled:disabled {
  cursor: not-allowed;
  opacity: 0.5;
}

/* RJSF form validation error */

.jp-FormGroup-content .validationErrors {
  color: var(--jp-error-color0);
}

/* Hide panel level error as duplicated the field level error */
.jp-FormGroup-content .panel.errors {
  display: none;
}

/* RJSF normal content (settings-editor) */

.jp-FormGroup-contentNormal {
  display: flex;
  align-items: center;
  flex-wrap: wrap;
}

.jp-FormGroup-contentNormal .jp-FormGroup-contentItem {
  margin-left: 7px;
  color: var(--jp-ui-font-color0);
}

.jp-FormGroup-contentNormal .jp-FormGroup-description {
  flex-basis: 100%;
  padding: 4px 7px;
}

.jp-FormGroup-contentNormal .jp-FormGroup-default {
  flex-basis: 100%;
  padding: 4px 7px;
}

.jp-FormGroup-contentNormal .jp-FormGroup-fieldLabel {
  font-size: var(--jp-content-font-size1);
  font-weight: normal;
  min-width: 120px;
}

.jp-FormGroup-contentNormal fieldset:not(:first-child) {
  margin-left: 7px;
}

.jp-FormGroup-contentNormal .field-array-of-string .array-item {
  /* Display `jp-ArrayOperations` buttons side-by-side with content except
    for small screens where flex-wrap will place them one below the other.
  */
  display: flex;
  align-items: center;
  flex-wrap: wrap;
}

.jp-FormGroup-contentNormal .jp-objectFieldWrapper .form-group {
  padding: 2px 8px 2px var(--jp-private-settingeditor-modifier-indent);
  margin-top: 2px;
}

/* RJSF compact content (metadata-form) */

.jp-FormGroup-content.jp-FormGroup-contentCompact {
  width: 100%;
}

.jp-FormGroup-contentCompact .form-group {
  display: flex;
  padding: 0.5em 0.2em 0.5em 0;
}

.jp-FormGroup-contentCompact
  .jp-FormGroup-compactTitle
  .jp-FormGroup-description {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color2);
}

.jp-FormGroup-contentCompact .jp-FormGroup-fieldLabel {
  padding-bottom: 0.3em;
}

.jp-FormGroup-contentCompact .jp-inputFieldWrapper .form-control {
  width: 100%;
  box-sizing: border-box;
}

.jp-FormGroup-contentCompact .jp-arrayFieldWrapper .jp-FormGroup-compactTitle {
  padding-bottom: 7px;
}

.jp-FormGroup-contentCompact
  .jp-objectFieldWrapper
  .jp-objectFieldWrapper
  .form-group {
  padding: 2px 8px 2px var(--jp-private-settingeditor-modifier-indent);
  margin-top: 2px;
}

.jp-FormGroup-contentCompact ul.error-detail {
  margin-block-start: 0.5em;
  margin-block-end: 0.5em;
  padding-inline-start: 1em;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-SidePanel {
  display: flex;
  flex-direction: column;
  min-width: var(--jp-sidebar-min-width);
  overflow-y: auto;
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);
  font-size: var(--jp-ui-font-size1);
}

.jp-SidePanel-header {
  flex: 0 0 auto;
  display: flex;
  border-bottom: var(--jp-border-width) solid var(--jp-border-color2);
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  letter-spacing: 1px;
  margin: 0;
  padding: 2px;
  text-transform: uppercase;
}

.jp-SidePanel-toolbar {
  flex: 0 0 auto;
}

.jp-SidePanel-content {
  flex: 1 1 auto;
}

.jp-SidePanel-toolbar,
.jp-AccordionPanel-toolbar {
  height: var(--jp-private-toolbar-height);
}

.jp-SidePanel-toolbar.jp-Toolbar-micro {
  display: none;
}

.lm-AccordionPanel .jp-AccordionPanel-title {
  box-sizing: border-box;
  line-height: 25px;
  margin: 0;
  display: flex;
  align-items: center;
  background: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  font-size: var(--jp-ui-font-size0);
}

.jp-AccordionPanel-title {
  cursor: pointer;
  user-select: none;
  -moz-user-select: none;
  -webkit-user-select: none;
  text-transform: uppercase;
}

.lm-AccordionPanel[data-orientation='horizontal'] > .jp-AccordionPanel-title {
  /* Title is rotated for horizontal accordion panel using CSS */
  display: block;
  transform-origin: top left;
  transform: rotate(-90deg) translate(-100%);
}

.jp-AccordionPanel-title .lm-AccordionPanel-titleLabel {
  user-select: none;
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}

.jp-AccordionPanel-title .lm-AccordionPanel-titleCollapser {
  transform: rotate(-90deg);
  margin: auto 0;
  height: 16px;
}

.jp-AccordionPanel-title.lm-mod-expanded .lm-AccordionPanel-titleCollapser {
  transform: rotate(0deg);
}

.lm-AccordionPanel .jp-AccordionPanel-toolbar {
  background: none;
  box-shadow: none;
  border: none;
  margin-left: auto;
}

.lm-AccordionPanel .lm-SplitPanel-handle:hover {
  background: var(--jp-layout-color3);
}

.jp-text-truncated {
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Spinner {
  position: absolute;
  display: flex;
  justify-content: center;
  align-items: center;
  z-index: 10;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  background: var(--jp-layout-color0);
  outline: none;
}

.jp-SpinnerContent {
  font-size: 10px;
  margin: 50px auto;
  text-indent: -9999em;
  width: 3em;
  height: 3em;
  border-radius: 50%;
  background: var(--jp-brand-color3);
  background: linear-gradient(
    to right,
    #f37626 10%,
    rgba(255, 255, 255, 0) 42%
  );
  position: relative;
  animation: load3 1s infinite linear, fadeIn 1s;
}

.jp-SpinnerContent::before {
  width: 50%;
  height: 50%;
  background: #f37626;
  border-radius: 100% 0 0;
  position: absolute;
  top: 0;
  left: 0;
  content: '';
}

.jp-SpinnerContent::after {
  background: var(--jp-layout-color0);
  width: 75%;
  height: 75%;
  border-radius: 50%;
  content: '';
  margin: auto;
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
}

@keyframes fadeIn {
  0% {
    opacity: 0;
  }

  100% {
    opacity: 1;
  }
}

@keyframes load3 {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

button.jp-mod-styled {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  border: none;
  box-sizing: border-box;
  text-align: center;
  line-height: 32px;
  height: 32px;
  padding: 0 12px;
  letter-spacing: 0.8px;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

input.jp-mod-styled {
  background: var(--jp-input-background);
  height: 28px;
  box-sizing: border-box;
  border: var(--jp-border-width) solid var(--jp-border-color1);
  padding-left: 7px;
  padding-right: 7px;
  font-size: var(--jp-ui-font-size2);
  color: var(--jp-ui-font-color0);
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

input[type='checkbox'].jp-mod-styled {
  appearance: checkbox;
  -webkit-appearance: checkbox;
  -moz-appearance: checkbox;
  height: auto;
}

input.jp-mod-styled:focus {
  border: var(--jp-border-width) solid var(--md-blue-500);
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-select-wrapper {
  display: flex;
  position: relative;
  flex-direction: column;
  padding: 1px;
  background-color: var(--jp-layout-color1);
  box-sizing: border-box;
  margin-bottom: 12px;
}

.jp-select-wrapper:not(.multiple) {
  height: 28px;
}

.jp-select-wrapper.jp-mod-focused select.jp-mod-styled {
  border: var(--jp-border-width) solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
  background-color: var(--jp-input-active-background);
}

select.jp-mod-styled:hover {
  cursor: pointer;
  color: var(--jp-ui-font-color0);
  background-color: var(--jp-input-hover-background);
  box-shadow: inset 0 0 1px rgba(0, 0, 0, 0.5);
}

select.jp-mod-styled {
  flex: 1 1 auto;
  width: 100%;
  font-size: var(--jp-ui-font-size2);
  background: var(--jp-input-background);
  color: var(--jp-ui-font-color0);
  padding: 0 25px 0 8px;
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  border-radius: 0;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

select.jp-mod-styled:not([multiple]) {
  height: 32px;
}

select.jp-mod-styled[multiple] {
  max-height: 200px;
  overflow-y: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-switch {
  display: flex;
  align-items: center;
  padding-left: 4px;
  padding-right: 4px;
  font-size: var(--jp-ui-font-size1);
  background-color: transparent;
  color: var(--jp-ui-font-color1);
  border: none;
  height: 20px;
}

.jp-switch:hover {
  background-color: var(--jp-layout-color2);
}

.jp-switch-label {
  margin-right: 5px;
  font-family: var(--jp-ui-font-family);
}

.jp-switch-track {
  cursor: pointer;
  background-color: var(--jp-switch-color, var(--jp-border-color1));
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 34px;
  height: 16px;
  width: 35px;
  position: relative;
}

.jp-switch-track::before {
  content: '';
  position: absolute;
  height: 10px;
  width: 10px;
  margin: 3px;
  left: 0;
  background-color: var(--jp-ui-inverse-font-color1);
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 50%;
}

.jp-switch[aria-checked='true'] .jp-switch-track {
  background-color: var(--jp-switch-true-position-color, var(--jp-warn-color0));
}

.jp-switch[aria-checked='true'] .jp-switch-track::before {
  /* track width (35) - margins (3 + 3) - thumb width (10) */
  left: 19px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

:root {
  --jp-private-toolbar-height: calc(
    28px + var(--jp-border-width)
  ); /* leave 28px for content */
}

.jp-Toolbar {
  color: var(--jp-ui-font-color1);
  flex: 0 0 auto;
  display: flex;
  flex-direction: row;
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  background: var(--jp-toolbar-background);
  min-height: var(--jp-toolbar-micro-height);
  padding: 2px;
  z-index: 8;
  overflow-x: hidden;
}

/* Toolbar items */

.jp-Toolbar > .jp-Toolbar-item.jp-Toolbar-spacer {
  flex-grow: 1;
  flex-shrink: 1;
}

.jp-Toolbar-item.jp-Toolbar-kernelStatus {
  display: inline-block;
  width: 32px;
  background-repeat: no-repeat;
  background-position: center;
  background-size: 16px;
}

.jp-Toolbar > .jp-Toolbar-item {
  flex: 0 0 auto;
  display: flex;
  padding-left: 1px;
  padding-right: 1px;
  font-size: var(--jp-ui-font-size1);
  line-height: var(--jp-private-toolbar-height);
  height: 100%;
}

/* Toolbar buttons */

/* This is the div we use to wrap the react component into a Widget */
div.jp-ToolbarButton {
  color: transparent;
  border: none;
  box-sizing: border-box;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 0;
  margin: 0;
}

button.jp-ToolbarButtonComponent {
  background: var(--jp-layout-color1);
  border: none;
  box-sizing: border-box;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 0 6px;
  margin: 0;
  height: 24px;
  border-radius: var(--jp-border-radius);
  display: flex;
  align-items: center;
  text-align: center;
  font-size: 14px;
  min-width: unset;
  min-height: unset;
}

button.jp-ToolbarButtonComponent:disabled {
  opacity: 0.4;
}

button.jp-ToolbarButtonComponent > span {
  padding: 0;
  flex: 0 0 auto;
}

button.jp-ToolbarButtonComponent .jp-ToolbarButtonComponent-label {
  font-size: var(--jp-ui-font-size1);
  line-height: 100%;
  padding-left: 2px;
  color: var(--jp-ui-font-color1);
  font-family: var(--jp-ui-font-family);
}

#jp-main-dock-panel[data-mode='single-document']
  .jp-MainAreaWidget
  > .jp-Toolbar.jp-Toolbar-micro {
  padding: 0;
  min-height: 0;
}

#jp-main-dock-panel[data-mode='single-document']
  .jp-MainAreaWidget
  > .jp-Toolbar {
  border: none;
  box-shadow: none;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-WindowedPanel-outer {
  position: relative;
  overflow-y: auto;
}

.jp-WindowedPanel-inner {
  position: relative;
}

.jp-WindowedPanel-window {
  position: absolute;
  left: 0;
  right: 0;
  overflow: visible;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/* Sibling imports */

body {
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
}

/* Disable native link decoration styles everywhere outside of dialog boxes */
a {
  text-decoration: unset;
  color: unset;
}

a:hover {
  text-decoration: unset;
  color: unset;
}

/* Accessibility for links inside dialog box text */
.jp-Dialog-content a {
  text-decoration: revert;
  color: var(--jp-content-link-color);
}

.jp-Dialog-content a:hover {
  text-decoration: revert;
}

/* Styles for ui-components */
.jp-Button {
  color: var(--jp-ui-font-color2);
  border-radius: var(--jp-border-radius);
  padding: 0 12px;
  font-size: var(--jp-ui-font-size1);

  /* Copy from blueprint 3 */
  display: inline-flex;
  flex-direction: row;
  border: none;
  cursor: pointer;
  align-items: center;
  justify-content: center;
  text-align: left;
  vertical-align: middle;
  min-height: 30px;
  min-width: 30px;
}

.jp-Button:disabled {
  cursor: not-allowed;
}

.jp-Button:empty {
  padding: 0 !important;
}

.jp-Button.jp-mod-small {
  min-height: 24px;
  min-width: 24px;
  font-size: 12px;
  padding: 0 7px;
}

/* Use our own theme for hover styles */
.jp-Button.jp-mod-minimal:hover {
  background-color: var(--jp-layout-color2);
}

.jp-Button.jp-mod-minimal {
  background: none;
}

.jp-InputGroup {
  display: block;
  position: relative;
}

.jp-InputGroup input {
  box-sizing: border-box;
  border: none;
  border-radius: 0;
  background-color: transparent;
  color: var(--jp-ui-font-color0);
  box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-border-color);
  padding-bottom: 0;
  padding-top: 0;
  padding-left: 10px;
  padding-right: 28px;
  position: relative;
  width: 100%;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  font-size: 14px;
  font-weight: 400;
  height: 30px;
  line-height: 30px;
  outline: none;
  vertical-align: middle;
}

.jp-InputGroup input:focus {
  box-shadow: inset 0 0 0 var(--jp-border-width)
      var(--jp-input-active-box-shadow-color),
    inset 0 0 0 3px var(--jp-input-active-box-shadow-color);
}

.jp-InputGroup input:disabled {
  cursor: not-allowed;
  resize: block;
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color2);
}

.jp-InputGroup input:disabled ~ span {
  cursor: not-allowed;
  color: var(--jp-ui-font-color2);
}

.jp-InputGroup input::placeholder,
input::placeholder {
  color: var(--jp-ui-font-color2);
}

.jp-InputGroupAction {
  position: absolute;
  bottom: 1px;
  right: 0;
  padding: 6px;
}

.jp-HTMLSelect.jp-DefaultStyle select {
  background-color: initial;
  border: none;
  border-radius: 0;
  box-shadow: none;
  color: var(--jp-ui-font-color0);
  display: block;
  font-size: var(--jp-ui-font-size1);
  font-family: var(--jp-ui-font-family);
  height: 24px;
  line-height: 14px;
  padding: 0 25px 0 10px;
  text-align: left;
  -moz-appearance: none;
  -webkit-appearance: none;
}

.jp-HTMLSelect.jp-DefaultStyle select:disabled {
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color2);
  cursor: not-allowed;
  resize: block;
}

.jp-HTMLSelect.jp-DefaultStyle select:disabled ~ span {
  cursor: not-allowed;
}

/* Use our own theme for hover and option styles */
/* stylelint-disable-next-line selector-max-type */
.jp-HTMLSelect.jp-DefaultStyle select:hover,
.jp-HTMLSelect.jp-DefaultStyle select > option {
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color0);
}

select {
  box-sizing: border-box;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-StatusBar-Widget {
  display: flex;
  align-items: center;
  background: var(--jp-layout-color2);
  min-height: var(--jp-statusbar-height);
  justify-content: space-between;
  padding: 0 10px;
}

.jp-StatusBar-Left {
  display: flex;
  align-items: center;
  flex-direction: row;
}

.jp-StatusBar-Middle {
  display: flex;
  align-items: center;
}

.jp-StatusBar-Right {
  display: flex;
  align-items: center;
  flex-direction: row-reverse;
}

.jp-StatusBar-Item {
  max-height: var(--jp-statusbar-height);
  margin: 0 2px;
  height: var(--jp-statusbar-height);
  white-space: nowrap;
  text-overflow: ellipsis;
  color: var(--jp-ui-font-color1);
  padding: 0 6px;
}

.jp-mod-highlighted:hover {
  background-color: var(--jp-layout-color3);
}

.jp-mod-clicked {
  background-color: var(--jp-brand-color1);
}

.jp-mod-clicked:hover {
  background-color: var(--jp-brand-color0);
}

.jp-mod-clicked .jp-StatusBar-TextItem {
  color: var(--jp-ui-inverse-font-color1);
}

.jp-StatusBar-HoverItem {
  box-shadow: '0px 4px 4px rgba(0, 0, 0, 0.25)';
}

.jp-StatusBar-TextItem {
  font-size: var(--jp-ui-font-size1);
  font-family: var(--jp-ui-font-family);
  line-height: 24px;
  color: var(--jp-ui-font-color1);
}

.jp-StatusBar-GroupItem {
  display: flex;
  align-items: center;
  flex-direction: row;
}

.jp-Statusbar-ProgressCircle svg {
  display: block;
  margin: 0 auto;
  width: 16px;
  height: 24px;
  align-self: normal;
}

.jp-Statusbar-ProgressCircle path {
  fill: var(--jp-inverse-layout-color3);
}

.jp-Statusbar-ProgressBar-progress-bar {
  height: 10px;
  width: 100px;
  border: solid 0.25px var(--jp-brand-color2);
  border-radius: 3px;
  overflow: hidden;
  align-self: center;
}

.jp-Statusbar-ProgressBar-progress-bar > div {
  background-color: var(--jp-brand-color2);
  background-image: linear-gradient(
    -45deg,
    rgba(255, 255, 255, 0.2) 25%,
    transparent 25%,
    transparent 50%,
    rgba(255, 255, 255, 0.2) 50%,
    rgba(255, 255, 255, 0.2) 75%,
    transparent 75%,
    transparent
  );
  background-size: 40px 40px;
  float: left;
  width: 0%;
  height: 100%;
  font-size: 12px;
  line-height: 14px;
  color: #fff;
  text-align: center;
  animation: jp-Statusbar-ExecutionTime-progress-bar 2s linear infinite;
}

.jp-Statusbar-ProgressBar-progress-bar p {
  color: var(--jp-ui-font-color1);
  font-family: var(--jp-ui-font-family);
  font-size: var(--jp-ui-font-size1);
  line-height: 10px;
  width: 100px;
}

@keyframes jp-Statusbar-ExecutionTime-progress-bar {
  0% {
    background-position: 0 0;
  }

  100% {
    background-position: 40px 40px;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-commandpalette-search-height: 28px;
}

/*-----------------------------------------------------------------------------
| Overall styles
|----------------------------------------------------------------------------*/

.lm-CommandPalette {
  padding-bottom: 0;
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);

  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
}

/*-----------------------------------------------------------------------------
| Modal variant
|----------------------------------------------------------------------------*/

.jp-ModalCommandPalette {
  position: absolute;
  z-index: 10000;
  top: 38px;
  left: 30%;
  margin: 0;
  padding: 4px;
  width: 40%;
  box-shadow: var(--jp-elevation-z4);
  border-radius: 4px;
  background: var(--jp-layout-color0);
}

.jp-ModalCommandPalette .lm-CommandPalette {
  max-height: 40vh;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-close-icon::after {
  display: none;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-CommandPalette-header {
  display: none;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-CommandPalette-item {
  margin-left: 4px;
  margin-right: 4px;
}

.jp-ModalCommandPalette
  .lm-CommandPalette
  .lm-CommandPalette-item.lm-mod-disabled {
  display: none;
}

/*-----------------------------------------------------------------------------
| Search
|----------------------------------------------------------------------------*/

.lm-CommandPalette-search {
  padding: 4px;
  background-color: var(--jp-layout-color1);
  z-index: 2;
}

.lm-CommandPalette-wrapper {
  overflow: overlay;
  padding: 0 9px;
  background-color: var(--jp-input-active-background);
  height: 30px;
  box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-border-color);
}

.lm-CommandPalette.lm-mod-focused .lm-CommandPalette-wrapper {
  box-shadow: inset 0 0 0 1px var(--jp-input-active-box-shadow-color),
    inset 0 0 0 3px var(--jp-input-active-box-shadow-color);
}

.jp-SearchIconGroup {
  color: white;
  background-color: var(--jp-brand-color1);
  position: absolute;
  top: 4px;
  right: 4px;
  padding: 5px 5px 1px;
}

.jp-SearchIconGroup svg {
  height: 20px;
  width: 20px;
}

.jp-SearchIconGroup .jp-icon3[fill] {
  fill: var(--jp-layout-color0);
}

.lm-CommandPalette-input {
  background: transparent;
  width: calc(100% - 18px);
  float: left;
  border: none;
  outline: none;
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  line-height: var(--jp-private-commandpalette-search-height);
}

.lm-CommandPalette-input::-webkit-input-placeholder,
.lm-CommandPalette-input::-moz-placeholder,
.lm-CommandPalette-input:-ms-input-placeholder {
  color: var(--jp-ui-font-color2);
  font-size: var(--jp-ui-font-size1);
}

/*-----------------------------------------------------------------------------
| Results
|----------------------------------------------------------------------------*/

.lm-CommandPalette-header:first-child {
  margin-top: 0;
}

.lm-CommandPalette-header {
  border-bottom: solid var(--jp-border-width) var(--jp-border-color2);
  color: var(--jp-ui-font-color1);
  cursor: pointer;
  display: flex;
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  letter-spacing: 1px;
  margin-top: 8px;
  padding: 8px 0 8px 12px;
  text-transform: uppercase;
}

.lm-CommandPalette-header.lm-mod-active {
  background: var(--jp-layout-color2);
}

.lm-CommandPalette-header > mark {
  background-color: transparent;
  font-weight: bold;
  color: var(--jp-ui-font-color1);
}

.lm-CommandPalette-item {
  padding: 4px 12px 4px 4px;
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  font-weight: 400;
  display: flex;
}

.lm-CommandPalette-item.lm-mod-disabled {
  color: var(--jp-ui-font-color2);
}

.lm-CommandPalette-item.lm-mod-active {
  color: var(--jp-ui-inverse-font-color1);
  background: var(--jp-brand-color1);
}

.lm-CommandPalette-item.lm-mod-active .lm-CommandPalette-itemLabel > mark {
  color: var(--jp-ui-inverse-font-color0);
}

.lm-CommandPalette-item.lm-mod-active .jp-icon-selectable[fill] {
  fill: var(--jp-layout-color0);
}

.lm-CommandPalette-item.lm-mod-active:hover:not(.lm-mod-disabled) {
  color: var(--jp-ui-inverse-font-color1);
  background: var(--jp-brand-color1);
}

.lm-CommandPalette-item:hover:not(.lm-mod-active):not(.lm-mod-disabled) {
  background: var(--jp-layout-color2);
}

.lm-CommandPalette-itemContent {
  overflow: hidden;
}

.lm-CommandPalette-itemLabel > mark {
  color: var(--jp-ui-font-color0);
  background-color: transparent;
  font-weight: bold;
}

.lm-CommandPalette-item.lm-mod-disabled mark {
  color: var(--jp-ui-font-color2);
}

.lm-CommandPalette-item .lm-CommandPalette-itemIcon {
  margin: 0 4px 0 0;
  position: relative;
  width: 16px;
  top: 2px;
  flex: 0 0 auto;
}

.lm-CommandPalette-item.lm-mod-disabled .lm-CommandPalette-itemIcon {
  opacity: 0.6;
}

.lm-CommandPalette-item .lm-CommandPalette-itemShortcut {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemCaption {
  display: none;
}

.lm-CommandPalette-content {
  background-color: var(--jp-layout-color1);
}

.lm-CommandPalette-content:empty::after {
  content: 'No results';
  margin: auto;
  margin-top: 20px;
  width: 100px;
  display: block;
  font-size: var(--jp-ui-font-size2);
  font-family: var(--jp-ui-font-family);
  font-weight: lighter;
}

.lm-CommandPalette-emptyMessage {
  text-align: center;
  margin-top: 24px;
  line-height: 1.32;
  padding: 0 8px;
  color: var(--jp-content-font-color3);
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Dialog {
  position: absolute;
  z-index: 10000;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  top: 0;
  left: 0;
  margin: 0;
  padding: 0;
  width: 100%;
  height: 100%;
  background: var(--jp-dialog-background);
}

.jp-Dialog-content {
  display: flex;
  flex-direction: column;
  margin-left: auto;
  margin-right: auto;
  background: var(--jp-layout-color1);
  padding: 24px 24px 12px;
  min-width: 300px;
  min-height: 150px;
  max-width: 1000px;
  max-height: 500px;
  box-sizing: border-box;
  box-shadow: var(--jp-elevation-z20);
  word-wrap: break-word;
  border-radius: var(--jp-border-radius);

  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color1);
  resize: both;
}

.jp-Dialog-content.jp-Dialog-content-small {
  max-width: 500px;
}

.jp-Dialog-button {
  overflow: visible;
}

button.jp-Dialog-button:focus {
  outline: 1px solid var(--jp-brand-color1);
  outline-offset: 4px;
  -moz-outline-radius: 0;
}

button.jp-Dialog-button:focus::-moz-focus-inner {
  border: 0;
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-accept:focus,
button.jp-Dialog-button.jp-mod-styled.jp-mod-warn:focus,
button.jp-Dialog-button.jp-mod-styled.jp-mod-reject:focus {
  outline-offset: 4px;
  -moz-outline-radius: 0;
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-accept:focus {
  outline: 1px solid var(--jp-accept-color-normal, var(--jp-brand-color1));
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-warn:focus {
  outline: 1px solid var(--jp-warn-color-normal, var(--jp-error-color1));
}

button.jp-Dialog-button.jp-mod-styled.jp-mod-reject:focus {
  outline: 1px solid var(--jp-reject-color-normal, var(--md-grey-600));
}

button.jp-Dialog-close-button {
  padding: 0;
  height: 100%;
  min-width: unset;
  min-height: unset;
}

.jp-Dialog-header {
  display: flex;
  justify-content: space-between;
  flex: 0 0 auto;
  padding-bottom: 12px;
  font-size: var(--jp-ui-font-size3);
  font-weight: 400;
  color: var(--jp-ui-font-color1);
}

.jp-Dialog-body {
  display: flex;
  flex-direction: column;
  flex: 1 1 auto;
  font-size: var(--jp-ui-font-size1);
  background: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  overflow: auto;
}

.jp-Dialog-footer {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  align-items: center;
  flex: 0 0 auto;
  margin-left: -12px;
  margin-right: -12px;
  padding: 12px;
}

.jp-Dialog-checkbox {
  padding-right: 5px;
}

.jp-Dialog-checkbox > input:focus-visible {
  outline: 1px solid var(--jp-input-active-border-color);
  outline-offset: 1px;
}

.jp-Dialog-spacer {
  flex: 1 1 auto;
}

.jp-Dialog-title {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.jp-Dialog-body > .jp-select-wrapper {
  width: 100%;
}

.jp-Dialog-body > button {
  padding: 0 16px;
}

.jp-Dialog-body > label {
  line-height: 1.4;
  color: var(--jp-ui-font-color0);
}

.jp-Dialog-button.jp-mod-styled:not(:last-child) {
  margin-right: 12px;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-Input-Boolean-Dialog {
  flex-direction: row-reverse;
  align-items: end;
  width: 100%;
}

.jp-Input-Boolean-Dialog > label {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-MainAreaWidget > :focus {
  outline: none;
}

.jp-MainAreaWidget .jp-MainAreaWidget-error {
  padding: 6px;
}

.jp-MainAreaWidget .jp-MainAreaWidget-error > pre {
  width: auto;
  padding: 10px;
  background: var(--jp-error-color3);
  border: var(--jp-border-width) solid var(--jp-error-color1);
  border-radius: var(--jp-border-radius);
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  white-space: pre-wrap;
  word-wrap: break-word;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/**
 * google-material-color v1.2.6
 * https://github.com/danlevan/google-material-color
 */
:root {
  --md-red-50: #ffebee;
  --md-red-100: #ffcdd2;
  --md-red-200: #ef9a9a;
  --md-red-300: #e57373;
  --md-red-400: #ef5350;
  --md-red-500: #f44336;
  --md-red-600: #e53935;
  --md-red-700: #d32f2f;
  --md-red-800: #c62828;
  --md-red-900: #b71c1c;
  --md-red-A100: #ff8a80;
  --md-red-A200: #ff5252;
  --md-red-A400: #ff1744;
  --md-red-A700: #d50000;
  --md-pink-50: #fce4ec;
  --md-pink-100: #f8bbd0;
  --md-pink-200: #f48fb1;
  --md-pink-300: #f06292;
  --md-pink-400: #ec407a;
  --md-pink-500: #e91e63;
  --md-pink-600: #d81b60;
  --md-pink-700: #c2185b;
  --md-pink-800: #ad1457;
  --md-pink-900: #880e4f;
  --md-pink-A100: #ff80ab;
  --md-pink-A200: #ff4081;
  --md-pink-A400: #f50057;
  --md-pink-A700: #c51162;
  --md-purple-50: #f3e5f5;
  --md-purple-100: #e1bee7;
  --md-purple-200: #ce93d8;
  --md-purple-300: #ba68c8;
  --md-purple-400: #ab47bc;
  --md-purple-500: #9c27b0;
  --md-purple-600: #8e24aa;
  --md-purple-700: #7b1fa2;
  --md-purple-800: #6a1b9a;
  --md-purple-900: #4a148c;
  --md-purple-A100: #ea80fc;
  --md-purple-A200: #e040fb;
  --md-purple-A400: #d500f9;
  --md-purple-A700: #a0f;
  --md-deep-purple-50: #ede7f6;
  --md-deep-purple-100: #d1c4e9;
  --md-deep-purple-200: #b39ddb;
  --md-deep-purple-300: #9575cd;
  --md-deep-purple-400: #7e57c2;
  --md-deep-purple-500: #673ab7;
  --md-deep-purple-600: #5e35b1;
  --md-deep-purple-700: #512da8;
  --md-deep-purple-800: #4527a0;
  --md-deep-purple-900: #311b92;
  --md-deep-purple-A100: #b388ff;
  --md-deep-purple-A200: #7c4dff;
  --md-deep-purple-A400: #651fff;
  --md-deep-purple-A700: #6200ea;
  --md-indigo-50: #e8eaf6;
  --md-indigo-100: #c5cae9;
  --md-indigo-200: #9fa8da;
  --md-indigo-300: #7986cb;
  --md-indigo-400: #5c6bc0;
  --md-indigo-500: #3f51b5;
  --md-indigo-600: #3949ab;
  --md-indigo-700: #303f9f;
  --md-indigo-800: #283593;
  --md-indigo-900: #1a237e;
  --md-indigo-A100: #8c9eff;
  --md-indigo-A200: #536dfe;
  --md-indigo-A400: #3d5afe;
  --md-indigo-A700: #304ffe;
  --md-blue-50: #e3f2fd;
  --md-blue-100: #bbdefb;
  --md-blue-200: #90caf9;
  --md-blue-300: #64b5f6;
  --md-blue-400: #42a5f5;
  --md-blue-500: #2196f3;
  --md-blue-600: #1e88e5;
  --md-blue-700: #1976d2;
  --md-blue-800: #1565c0;
  --md-blue-900: #0d47a1;
  --md-blue-A100: #82b1ff;
  --md-blue-A200: #448aff;
  --md-blue-A400: #2979ff;
  --md-blue-A700: #2962ff;
  --md-light-blue-50: #e1f5fe;
  --md-light-blue-100: #b3e5fc;
  --md-light-blue-200: #81d4fa;
  --md-light-blue-300: #4fc3f7;
  --md-light-blue-400: #29b6f6;
  --md-light-blue-500: #03a9f4;
  --md-light-blue-600: #039be5;
  --md-light-blue-700: #0288d1;
  --md-light-blue-800: #0277bd;
  --md-light-blue-900: #01579b;
  --md-light-blue-A100: #80d8ff;
  --md-light-blue-A200: #40c4ff;
  --md-light-blue-A400: #00b0ff;
  --md-light-blue-A700: #0091ea;
  --md-cyan-50: #e0f7fa;
  --md-cyan-100: #b2ebf2;
  --md-cyan-200: #80deea;
  --md-cyan-300: #4dd0e1;
  --md-cyan-400: #26c6da;
  --md-cyan-500: #00bcd4;
  --md-cyan-600: #00acc1;
  --md-cyan-700: #0097a7;
  --md-cyan-800: #00838f;
  --md-cyan-900: #006064;
  --md-cyan-A100: #84ffff;
  --md-cyan-A200: #18ffff;
  --md-cyan-A400: #00e5ff;
  --md-cyan-A700: #00b8d4;
  --md-teal-50: #e0f2f1;
  --md-teal-100: #b2dfdb;
  --md-teal-200: #80cbc4;
  --md-teal-300: #4db6ac;
  --md-teal-400: #26a69a;
  --md-teal-500: #009688;
  --md-teal-600: #00897b;
  --md-teal-700: #00796b;
  --md-teal-800: #00695c;
  --md-teal-900: #004d40;
  --md-teal-A100: #a7ffeb;
  --md-teal-A200: #64ffda;
  --md-teal-A400: #1de9b6;
  --md-teal-A700: #00bfa5;
  --md-green-50: #e8f5e9;
  --md-green-100: #c8e6c9;
  --md-green-200: #a5d6a7;
  --md-green-300: #81c784;
  --md-green-400: #66bb6a;
  --md-green-500: #4caf50;
  --md-green-600: #43a047;
  --md-green-700: #388e3c;
  --md-green-800: #2e7d32;
  --md-green-900: #1b5e20;
  --md-green-A100: #b9f6ca;
  --md-green-A200: #69f0ae;
  --md-green-A400: #00e676;
  --md-green-A700: #00c853;
  --md-light-green-50: #f1f8e9;
  --md-light-green-100: #dcedc8;
  --md-light-green-200: #c5e1a5;
  --md-light-green-300: #aed581;
  --md-light-green-400: #9ccc65;
  --md-light-green-500: #8bc34a;
  --md-light-green-600: #7cb342;
  --md-light-green-700: #689f38;
  --md-light-green-800: #558b2f;
  --md-light-green-900: #33691e;
  --md-light-green-A100: #ccff90;
  --md-light-green-A200: #b2ff59;
  --md-light-green-A400: #76ff03;
  --md-light-green-A700: #64dd17;
  --md-lime-50: #f9fbe7;
  --md-lime-100: #f0f4c3;
  --md-lime-200: #e6ee9c;
  --md-lime-300: #dce775;
  --md-lime-400: #d4e157;
  --md-lime-500: #cddc39;
  --md-lime-600: #c0ca33;
  --md-lime-700: #afb42b;
  --md-lime-800: #9e9d24;
  --md-lime-900: #827717;
  --md-lime-A100: #f4ff81;
  --md-lime-A200: #eeff41;
  --md-lime-A400: #c6ff00;
  --md-lime-A700: #aeea00;
  --md-yellow-50: #fffde7;
  --md-yellow-100: #fff9c4;
  --md-yellow-200: #fff59d;
  --md-yellow-300: #fff176;
  --md-yellow-400: #ffee58;
  --md-yellow-500: #ffeb3b;
  --md-yellow-600: #fdd835;
  --md-yellow-700: #fbc02d;
  --md-yellow-800: #f9a825;
  --md-yellow-900: #f57f17;
  --md-yellow-A100: #ffff8d;
  --md-yellow-A200: #ff0;
  --md-yellow-A400: #ffea00;
  --md-yellow-A700: #ffd600;
  --md-amber-50: #fff8e1;
  --md-amber-100: #ffecb3;
  --md-amber-200: #ffe082;
  --md-amber-300: #ffd54f;
  --md-amber-400: #ffca28;
  --md-amber-500: #ffc107;
  --md-amber-600: #ffb300;
  --md-amber-700: #ffa000;
  --md-amber-800: #ff8f00;
  --md-amber-900: #ff6f00;
  --md-amber-A100: #ffe57f;
  --md-amber-A200: #ffd740;
  --md-amber-A400: #ffc400;
  --md-amber-A700: #ffab00;
  --md-orange-50: #fff3e0;
  --md-orange-100: #ffe0b2;
  --md-orange-200: #ffcc80;
  --md-orange-300: #ffb74d;
  --md-orange-400: #ffa726;
  --md-orange-500: #ff9800;
  --md-orange-600: #fb8c00;
  --md-orange-700: #f57c00;
  --md-orange-800: #ef6c00;
  --md-orange-900: #e65100;
  --md-orange-A100: #ffd180;
  --md-orange-A200: #ffab40;
  --md-orange-A400: #ff9100;
  --md-orange-A700: #ff6d00;
  --md-deep-orange-50: #fbe9e7;
  --md-deep-orange-100: #ffccbc;
  --md-deep-orange-200: #ffab91;
  --md-deep-orange-300: #ff8a65;
  --md-deep-orange-400: #ff7043;
  --md-deep-orange-500: #ff5722;
  --md-deep-orange-600: #f4511e;
  --md-deep-orange-700: #e64a19;
  --md-deep-orange-800: #d84315;
  --md-deep-orange-900: #bf360c;
  --md-deep-orange-A100: #ff9e80;
  --md-deep-orange-A200: #ff6e40;
  --md-deep-orange-A400: #ff3d00;
  --md-deep-orange-A700: #dd2c00;
  --md-brown-50: #efebe9;
  --md-brown-100: #d7ccc8;
  --md-brown-200: #bcaaa4;
  --md-brown-300: #a1887f;
  --md-brown-400: #8d6e63;
  --md-brown-500: #795548;
  --md-brown-600: #6d4c41;
  --md-brown-700: #5d4037;
  --md-brown-800: #4e342e;
  --md-brown-900: #3e2723;
  --md-grey-50: #fafafa;
  --md-grey-100: #f5f5f5;
  --md-grey-200: #eee;
  --md-grey-300: #e0e0e0;
  --md-grey-400: #bdbdbd;
  --md-grey-500: #9e9e9e;
  --md-grey-600: #757575;
  --md-grey-700: #616161;
  --md-grey-800: #424242;
  --md-grey-900: #212121;
  --md-blue-grey-50: #eceff1;
  --md-blue-grey-100: #cfd8dc;
  --md-blue-grey-200: #b0bec5;
  --md-blue-grey-300: #90a4ae;
  --md-blue-grey-400: #78909c;
  --md-blue-grey-500: #607d8b;
  --md-blue-grey-600: #546e7a;
  --md-blue-grey-700: #455a64;
  --md-blue-grey-800: #37474f;
  --md-blue-grey-900: #263238;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| RenderedText
|----------------------------------------------------------------------------*/

:root {
  /* This is the padding value to fill the gaps between lines containing spans with background color. */
  --jp-private-code-span-padding: calc(
    (var(--jp-code-line-height) - 1) * var(--jp-code-font-size) / 2
  );
}

.jp-RenderedText {
  text-align: left;
  padding-left: var(--jp-code-padding);
  line-height: var(--jp-code-line-height);
  font-family: var(--jp-code-font-family);
}

.jp-RenderedText pre,
.jp-RenderedJavaScript pre,
.jp-RenderedHTMLCommon pre {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-code-font-size);
  border: none;
  margin: 0;
  padding: 0;
}

.jp-RenderedText pre a:link {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

.jp-RenderedText pre a:hover {
  text-decoration: underline;
  color: var(--jp-content-link-color);
}

.jp-RenderedText pre a:visited {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

/* console foregrounds and backgrounds */
.jp-RenderedText pre .ansi-black-fg {
  color: #3e424d;
}

.jp-RenderedText pre .ansi-red-fg {
  color: #e75c58;
}

.jp-RenderedText pre .ansi-green-fg {
  color: #00a250;
}

.jp-RenderedText pre .ansi-yellow-fg {
  color: #ddb62b;
}

.jp-RenderedText pre .ansi-blue-fg {
  color: #208ffb;
}

.jp-RenderedText pre .ansi-magenta-fg {
  color: #d160c4;
}

.jp-RenderedText pre .ansi-cyan-fg {
  color: #60c6c8;
}

.jp-RenderedText pre .ansi-white-fg {
  color: #c5c1b4;
}

.jp-RenderedText pre .ansi-black-bg {
  background-color: #3e424d;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-red-bg {
  background-color: #e75c58;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-green-bg {
  background-color: #00a250;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-yellow-bg {
  background-color: #ddb62b;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-blue-bg {
  background-color: #208ffb;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-magenta-bg {
  background-color: #d160c4;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-cyan-bg {
  background-color: #60c6c8;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-white-bg {
  background-color: #c5c1b4;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-black-intense-fg {
  color: #282c36;
}

.jp-RenderedText pre .ansi-red-intense-fg {
  color: #b22b31;
}

.jp-RenderedText pre .ansi-green-intense-fg {
  color: #007427;
}

.jp-RenderedText pre .ansi-yellow-intense-fg {
  color: #b27d12;
}

.jp-RenderedText pre .ansi-blue-intense-fg {
  color: #0065ca;
}

.jp-RenderedText pre .ansi-magenta-intense-fg {
  color: #a03196;
}

.jp-RenderedText pre .ansi-cyan-intense-fg {
  color: #258f8f;
}

.jp-RenderedText pre .ansi-white-intense-fg {
  color: #a1a6b2;
}

.jp-RenderedText pre .ansi-black-intense-bg {
  background-color: #282c36;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-red-intense-bg {
  background-color: #b22b31;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-green-intense-bg {
  background-color: #007427;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-yellow-intense-bg {
  background-color: #b27d12;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-blue-intense-bg {
  background-color: #0065ca;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-magenta-intense-bg {
  background-color: #a03196;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-cyan-intense-bg {
  background-color: #258f8f;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-white-intense-bg {
  background-color: #a1a6b2;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-default-inverse-fg {
  color: var(--jp-ui-inverse-font-color0);
}

.jp-RenderedText pre .ansi-default-inverse-bg {
  background-color: var(--jp-inverse-layout-color0);
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-bold {
  font-weight: bold;
}

.jp-RenderedText pre .ansi-underline {
  text-decoration: underline;
}

.jp-RenderedText[data-mime-type='application/vnd.jupyter.stderr'] {
  background: var(--jp-rendermime-error-background);
  padding-top: var(--jp-code-padding);
}

/*-----------------------------------------------------------------------------
| RenderedLatex
|----------------------------------------------------------------------------*/

.jp-RenderedLatex {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-content-font-size1);
  line-height: var(--jp-content-line-height);
}

/* Left-justify outputs.*/
.jp-OutputArea-output.jp-RenderedLatex {
  padding: var(--jp-code-padding);
  text-align: left;
}

/*-----------------------------------------------------------------------------
| RenderedHTML
|----------------------------------------------------------------------------*/

.jp-RenderedHTMLCommon {
  color: var(--jp-content-font-color1);
  font-family: var(--jp-content-font-family);
  font-size: var(--jp-content-font-size1);
  line-height: var(--jp-content-line-height);

  /* Give a bit more R padding on Markdown text to keep line lengths reasonable */
  padding-right: 20px;
}

.jp-RenderedHTMLCommon em {
  font-style: italic;
}

.jp-RenderedHTMLCommon strong {
  font-weight: bold;
}

.jp-RenderedHTMLCommon u {
  text-decoration: underline;
}

.jp-RenderedHTMLCommon a:link {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

.jp-RenderedHTMLCommon a:hover {
  text-decoration: underline;
  color: var(--jp-content-link-color);
}

.jp-RenderedHTMLCommon a:visited {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

/* Headings */

.jp-RenderedHTMLCommon h1,
.jp-RenderedHTMLCommon h2,
.jp-RenderedHTMLCommon h3,
.jp-RenderedHTMLCommon h4,
.jp-RenderedHTMLCommon h5,
.jp-RenderedHTMLCommon h6 {
  line-height: var(--jp-content-heading-line-height);
  font-weight: var(--jp-content-heading-font-weight);
  font-style: normal;
  margin: var(--jp-content-heading-margin-top) 0
    var(--jp-content-heading-margin-bottom) 0;
}

.jp-RenderedHTMLCommon h1:first-child,
.jp-RenderedHTMLCommon h2:first-child,
.jp-RenderedHTMLCommon h3:first-child,
.jp-RenderedHTMLCommon h4:first-child,
.jp-RenderedHTMLCommon h5:first-child,
.jp-RenderedHTMLCommon h6:first-child {
  margin-top: calc(0.5 * var(--jp-content-heading-margin-top));
}

.jp-RenderedHTMLCommon h1:last-child,
.jp-RenderedHTMLCommon h2:last-child,
.jp-RenderedHTMLCommon h3:last-child,
.jp-RenderedHTMLCommon h4:last-child,
.jp-RenderedHTMLCommon h5:last-child,
.jp-RenderedHTMLCommon h6:last-child {
  margin-bottom: calc(0.5 * var(--jp-content-heading-margin-bottom));
}

.jp-RenderedHTMLCommon h1 {
  font-size: var(--jp-content-font-size5);
}

.jp-RenderedHTMLCommon h2 {
  font-size: var(--jp-content-font-size4);
}

.jp-RenderedHTMLCommon h3 {
  font-size: var(--jp-content-font-size3);
}

.jp-RenderedHTMLCommon h4 {
  font-size: var(--jp-content-font-size2);
}

.jp-RenderedHTMLCommon h5 {
  font-size: var(--jp-content-font-size1);
}

.jp-RenderedHTMLCommon h6 {
  font-size: var(--jp-content-font-size0);
}

/* Lists */

/* stylelint-disable selector-max-type, selector-max-compound-selectors */

.jp-RenderedHTMLCommon ul:not(.list-inline),
.jp-RenderedHTMLCommon ol:not(.list-inline) {
  padding-left: 2em;
}

.jp-RenderedHTMLCommon ul {
  list-style: disc;
}

.jp-RenderedHTMLCommon ul ul {
  list-style: square;
}

.jp-RenderedHTMLCommon ul ul ul {
  list-style: circle;
}

.jp-RenderedHTMLCommon ol {
  list-style: decimal;
}

.jp-RenderedHTMLCommon ol ol {
  list-style: upper-alpha;
}

.jp-RenderedHTMLCommon ol ol ol {
  list-style: lower-alpha;
}

.jp-RenderedHTMLCommon ol ol ol ol {
  list-style: lower-roman;
}

.jp-RenderedHTMLCommon ol ol ol ol ol {
  list-style: decimal;
}

.jp-RenderedHTMLCommon ol,
.jp-RenderedHTMLCommon ul {
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon ul ul,
.jp-RenderedHTMLCommon ul ol,
.jp-RenderedHTMLCommon ol ul,
.jp-RenderedHTMLCommon ol ol {
  margin-bottom: 0;
}

/* stylelint-enable selector-max-type, selector-max-compound-selectors */

.jp-RenderedHTMLCommon hr {
  color: var(--jp-border-color2);
  background-color: var(--jp-border-color1);
  margin-top: 1em;
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon > pre {
  margin: 1.5em 2em;
}

.jp-RenderedHTMLCommon pre,
.jp-RenderedHTMLCommon code {
  border: 0;
  background-color: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
  font-family: var(--jp-code-font-family);
  font-size: inherit;
  line-height: var(--jp-code-line-height);
  padding: 0;
  white-space: pre-wrap;
}

.jp-RenderedHTMLCommon :not(pre) > code {
  background-color: var(--jp-layout-color2);
  padding: 1px 5px;
}

/* Tables */

.jp-RenderedHTMLCommon table {
  border-collapse: collapse;
  border-spacing: 0;
  border: none;
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  table-layout: fixed;
  margin-left: auto;
  margin-bottom: 1em;
  margin-right: auto;
}

.jp-RenderedHTMLCommon thead {
  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
  vertical-align: bottom;
}

.jp-RenderedHTMLCommon td,
.jp-RenderedHTMLCommon th,
.jp-RenderedHTMLCommon tr {
  vertical-align: middle;
  padding: 0.5em;
  line-height: normal;
  white-space: normal;
  max-width: none;
  border: none;
}

.jp-RenderedMarkdown.jp-RenderedHTMLCommon td,
.jp-RenderedMarkdown.jp-RenderedHTMLCommon th {
  max-width: none;
}

:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon td,
:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon th,
:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon tr {
  text-align: right;
}

.jp-RenderedHTMLCommon th {
  font-weight: bold;
}

.jp-RenderedHTMLCommon tbody tr:nth-child(odd) {
  background: var(--jp-layout-color0);
}

.jp-RenderedHTMLCommon tbody tr:nth-child(even) {
  background: var(--jp-rendermime-table-row-background);
}

.jp-RenderedHTMLCommon tbody tr:hover {
  background: var(--jp-rendermime-table-row-hover-background);
}

.jp-RenderedHTMLCommon p {
  text-align: left;
  margin: 0;
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon img {
  -moz-force-broken-image-icon: 1;
}

/* Restrict to direct children as other images could be nested in other content. */
.jp-RenderedHTMLCommon > img {
  display: block;
  margin-left: 0;
  margin-right: 0;
  margin-bottom: 1em;
}

/* Change color behind transparent images if they need it... */
[data-jp-theme-light='false'] .jp-RenderedImage img.jp-needs-light-background {
  background-color: var(--jp-inverse-layout-color1);
}

[data-jp-theme-light='true'] .jp-RenderedImage img.jp-needs-dark-background {
  background-color: var(--jp-inverse-layout-color1);
}

.jp-RenderedHTMLCommon img,
.jp-RenderedImage img,
.jp-RenderedHTMLCommon svg,
.jp-RenderedSVG svg {
  max-width: 100%;
  height: auto;
}

.jp-RenderedHTMLCommon img.jp-mod-unconfined,
.jp-RenderedImage img.jp-mod-unconfined,
.jp-RenderedHTMLCommon svg.jp-mod-unconfined,
.jp-RenderedSVG svg.jp-mod-unconfined {
  max-width: none;
}

.jp-RenderedHTMLCommon .alert {
  padding: var(--jp-notebook-padding);
  border: var(--jp-border-width) solid transparent;
  border-radius: var(--jp-border-radius);
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon .alert-info {
  color: var(--jp-info-color0);
  background-color: var(--jp-info-color3);
  border-color: var(--jp-info-color2);
}

.jp-RenderedHTMLCommon .alert-info hr {
  border-color: var(--jp-info-color3);
}

.jp-RenderedHTMLCommon .alert-info > p:last-child,
.jp-RenderedHTMLCommon .alert-info > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-warning {
  color: var(--jp-warn-color0);
  background-color: var(--jp-warn-color3);
  border-color: var(--jp-warn-color2);
}

.jp-RenderedHTMLCommon .alert-warning hr {
  border-color: var(--jp-warn-color3);
}

.jp-RenderedHTMLCommon .alert-warning > p:last-child,
.jp-RenderedHTMLCommon .alert-warning > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-success {
  color: var(--jp-success-color0);
  background-color: var(--jp-success-color3);
  border-color: var(--jp-success-color2);
}

.jp-RenderedHTMLCommon .alert-success hr {
  border-color: var(--jp-success-color3);
}

.jp-RenderedHTMLCommon .alert-success > p:last-child,
.jp-RenderedHTMLCommon .alert-success > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-danger {
  color: var(--jp-error-color0);
  background-color: var(--jp-error-color3);
  border-color: var(--jp-error-color2);
}

.jp-RenderedHTMLCommon .alert-danger hr {
  border-color: var(--jp-error-color3);
}

.jp-RenderedHTMLCommon .alert-danger > p:last-child,
.jp-RenderedHTMLCommon .alert-danger > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon blockquote {
  margin: 1em 2em;
  padding: 0 1em;
  border-left: 5px solid var(--jp-border-color2);
}

a.jp-InternalAnchorLink {
  visibility: hidden;
  margin-left: 8px;
  color: var(--md-blue-800);
}

h1:hover .jp-InternalAnchorLink,
h2:hover .jp-InternalAnchorLink,
h3:hover .jp-InternalAnchorLink,
h4:hover .jp-InternalAnchorLink,
h5:hover .jp-InternalAnchorLink,
h6:hover .jp-InternalAnchorLink {
  visibility: visible;
}

.jp-RenderedHTMLCommon kbd {
  background-color: var(--jp-rendermime-table-row-background);
  border: 1px solid var(--jp-border-color0);
  border-bottom-color: var(--jp-border-color2);
  border-radius: 3px;
  box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.25);
  display: inline-block;
  font-size: var(--jp-ui-font-size0);
  line-height: 1em;
  padding: 0.2em 0.5em;
}

/* Most direct children of .jp-RenderedHTMLCommon have a margin-bottom of 1.0.
 * At the bottom of cells this is a bit too much as there is also spacing
 * between cells. Going all the way to 0 gets too tight between markdown and
 * code cells.
 */
.jp-RenderedHTMLCommon > *:last-child {
  margin-bottom: 0.5em;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

.lm-cursor-backdrop {
  position: fixed;
  width: 200px;
  height: 200px;
  margin-top: -100px;
  margin-left: -100px;
  will-change: transform;
  z-index: 100;
}

.lm-mod-drag-image {
  will-change: transform;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-lineFormSearch {
  padding: 4px 12px;
  background-color: var(--jp-layout-color2);
  box-shadow: var(--jp-toolbar-box-shadow);
  z-index: 2;
  font-size: var(--jp-ui-font-size1);
}

.jp-lineFormCaption {
  font-size: var(--jp-ui-font-size0);
  line-height: var(--jp-ui-font-size1);
  margin-top: 4px;
  color: var(--jp-ui-font-color0);
}

.jp-baseLineForm {
  border: none;
  border-radius: 0;
  position: absolute;
  background-size: 16px;
  background-repeat: no-repeat;
  background-position: center;
  outline: none;
}

.jp-lineFormButtonContainer {
  top: 4px;
  right: 8px;
  height: 24px;
  padding: 0 12px;
  width: 12px;
}

.jp-lineFormButtonIcon {
  top: 0;
  right: 0;
  background-color: var(--jp-brand-color1);
  height: 100%;
  width: 100%;
  box-sizing: border-box;
  padding: 4px 6px;
}

.jp-lineFormButton {
  top: 0;
  right: 0;
  background-color: transparent;
  height: 100%;
  width: 100%;
  box-sizing: border-box;
}

.jp-lineFormWrapper {
  overflow: hidden;
  padding: 0 8px;
  border: 1px solid var(--jp-border-color0);
  background-color: var(--jp-input-active-background);
  height: 22px;
}

.jp-lineFormWrapperFocusWithin {
  border: var(--jp-border-width) solid var(--md-blue-500);
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-lineFormInput {
  background: transparent;
  width: 200px;
  height: 100%;
  border: none;
  outline: none;
  color: var(--jp-ui-font-color0);
  line-height: 28px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-JSONEditor {
  display: flex;
  flex-direction: column;
  width: 100%;
}

.jp-JSONEditor-host {
  flex: 1 1 auto;
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  border-radius: 0;
  background: var(--jp-layout-color0);
  min-height: 50px;
  padding: 1px;
}

.jp-JSONEditor.jp-mod-error .jp-JSONEditor-host {
  border-color: red;
  outline-color: red;
}

.jp-JSONEditor-header {
  display: flex;
  flex: 1 0 auto;
  padding: 0 0 0 12px;
}

.jp-JSONEditor-header label {
  flex: 0 0 auto;
}

.jp-JSONEditor-commitButton {
  height: 16px;
  width: 16px;
  background-size: 18px;
  background-repeat: no-repeat;
  background-position: center;
}

.jp-JSONEditor-host.jp-mod-focused {
  background-color: var(--jp-input-active-background);
  border: 1px solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
}

.jp-Editor.jp-mod-dropTarget {
  border: var(--jp-border-width) solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/
.jp-DocumentSearch-input {
  border: none;
  outline: none;
  color: var(--jp-ui-font-color0);
  font-size: var(--jp-ui-font-size1);
  background-color: var(--jp-layout-color0);
  font-family: var(--jp-ui-font-family);
  padding: 2px 1px;
  resize: none;
}

.jp-DocumentSearch-overlay {
  position: absolute;
  background-color: var(--jp-toolbar-background);
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  border-left: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  top: 0;
  right: 0;
  z-index: 7;
  min-width: 405px;
  padding: 2px;
  font-size: var(--jp-ui-font-size1);

  --jp-private-document-search-button-height: 20px;
}

.jp-DocumentSearch-overlay button {
  background-color: var(--jp-toolbar-background);
  outline: 0;
}

.jp-DocumentSearch-overlay button:hover {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-overlay button:active {
  background-color: var(--jp-layout-color3);
}

.jp-DocumentSearch-overlay-row {
  display: flex;
  align-items: center;
  margin-bottom: 2px;
}

.jp-DocumentSearch-button-content {
  display: inline-block;
  cursor: pointer;
  box-sizing: border-box;
  width: 100%;
  height: 100%;
}

.jp-DocumentSearch-button-content svg {
  width: 100%;
  height: 100%;
}

.jp-DocumentSearch-input-wrapper {
  border: var(--jp-border-width) solid var(--jp-border-color0);
  display: flex;
  background-color: var(--jp-layout-color0);
  margin: 2px;
}

.jp-DocumentSearch-input-wrapper:focus-within {
  border-color: var(--jp-cell-editor-active-border-color);
}

.jp-DocumentSearch-toggle-wrapper,
.jp-DocumentSearch-button-wrapper {
  all: initial;
  overflow: hidden;
  display: inline-block;
  border: none;
  box-sizing: border-box;
}

.jp-DocumentSearch-toggle-wrapper {
  width: 14px;
  height: 14px;
}

.jp-DocumentSearch-button-wrapper {
  width: var(--jp-private-document-search-button-height);
  height: var(--jp-private-document-search-button-height);
}

.jp-DocumentSearch-toggle-wrapper:focus,
.jp-DocumentSearch-button-wrapper:focus {
  outline: var(--jp-border-width) solid
    var(--jp-cell-editor-active-border-color);
  outline-offset: -1px;
}

.jp-DocumentSearch-toggle-wrapper,
.jp-DocumentSearch-button-wrapper,
.jp-DocumentSearch-button-content:focus {
  outline: none;
}

.jp-DocumentSearch-toggle-placeholder {
  width: 5px;
}

.jp-DocumentSearch-input-button::before {
  display: block;
  padding-top: 100%;
}

.jp-DocumentSearch-input-button-off {
  opacity: var(--jp-search-toggle-off-opacity);
}

.jp-DocumentSearch-input-button-off:hover {
  opacity: var(--jp-search-toggle-hover-opacity);
}

.jp-DocumentSearch-input-button-on {
  opacity: var(--jp-search-toggle-on-opacity);
}

.jp-DocumentSearch-index-counter {
  padding-left: 10px;
  padding-right: 10px;
  user-select: none;
  min-width: 35px;
  display: inline-block;
}

.jp-DocumentSearch-up-down-wrapper {
  display: inline-block;
  padding-right: 2px;
  margin-left: auto;
  white-space: nowrap;
}

.jp-DocumentSearch-spacer {
  margin-left: auto;
}

.jp-DocumentSearch-up-down-wrapper button {
  outline: 0;
  border: none;
  width: var(--jp-private-document-search-button-height);
  height: var(--jp-private-document-search-button-height);
  vertical-align: middle;
  margin: 1px 5px 2px;
}

.jp-DocumentSearch-up-down-button:hover {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-up-down-button:active {
  background-color: var(--jp-layout-color3);
}

.jp-DocumentSearch-filter-button {
  border-radius: var(--jp-border-radius);
}

.jp-DocumentSearch-filter-button:hover {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-filter-button-enabled {
  background-color: var(--jp-layout-color2);
}

.jp-DocumentSearch-filter-button-enabled:hover {
  background-color: var(--jp-layout-color3);
}

.jp-DocumentSearch-search-options {
  padding: 0 8px;
  margin-left: 3px;
  width: 100%;
  display: grid;
  justify-content: start;
  grid-template-columns: 1fr 1fr;
  align-items: center;
  justify-items: stretch;
}

.jp-DocumentSearch-search-filter-disabled {
  color: var(--jp-ui-font-color2);
}

.jp-DocumentSearch-search-filter {
  display: flex;
  align-items: center;
  user-select: none;
}

.jp-DocumentSearch-regex-error {
  color: var(--jp-error-color0);
}

.jp-DocumentSearch-replace-button-wrapper {
  overflow: hidden;
  display: inline-block;
  box-sizing: border-box;
  border: var(--jp-border-width) solid var(--jp-border-color0);
  margin: auto 2px;
  padding: 1px 4px;
  height: calc(var(--jp-private-document-search-button-height) + 2px);
}

.jp-DocumentSearch-replace-button-wrapper:focus {
  border: var(--jp-border-width) solid var(--jp-cell-editor-active-border-color);
}

.jp-DocumentSearch-replace-button {
  display: inline-block;
  text-align: center;
  cursor: pointer;
  box-sizing: border-box;
  color: var(--jp-ui-font-color1);

  /* height - 2 * (padding of wrapper) */
  line-height: calc(var(--jp-private-document-search-button-height) - 2px);
  width: 100%;
  height: 100%;
}

.jp-DocumentSearch-replace-button:focus {
  outline: none;
}

.jp-DocumentSearch-replace-wrapper-class {
  margin-left: 14px;
  display: flex;
}

.jp-DocumentSearch-replace-toggle {
  border: none;
  background-color: var(--jp-toolbar-background);
  border-radius: var(--jp-border-radius);
}

.jp-DocumentSearch-replace-toggle:hover {
  background-color: var(--jp-layout-color2);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.cm-editor {
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  font-family: var(--jp-code-font-family);
  border: 0;
  border-radius: 0;
  height: auto;

  /* Changed to auto to autogrow */
}

.cm-editor pre {
  padding: 0 var(--jp-code-padding);
}

.jp-CodeMirrorEditor[data-type='inline'] .cm-dialog {
  background-color: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
}

.jp-CodeMirrorEditor {
  cursor: text;
}

/* When zoomed out 67% and 33% on a screen of 1440 width x 900 height */
@media screen and (min-width: 2138px) and (max-width: 4319px) {
  .jp-CodeMirrorEditor[data-type='inline'] .cm-cursor {
    border-left: var(--jp-code-cursor-width1) solid
      var(--jp-editor-cursor-color);
  }
}

/* When zoomed out less than 33% */
@media screen and (min-width: 4320px) {
  .jp-CodeMirrorEditor[data-type='inline'] .cm-cursor {
    border-left: var(--jp-code-cursor-width2) solid
      var(--jp-editor-cursor-color);
  }
}

.cm-editor.jp-mod-readOnly .cm-cursor {
  display: none;
}

.jp-CollaboratorCursor {
  border-left: 5px solid transparent;
  border-right: 5px solid transparent;
  border-top: none;
  border-bottom: 3px solid;
  background-clip: content-box;
  margin-left: -5px;
  margin-right: -5px;
}

.cm-searching,
.cm-searching span {
  /* `.cm-searching span`: we need to override syntax highlighting */
  background-color: var(--jp-search-unselected-match-background-color);
  color: var(--jp-search-unselected-match-color);
}

.cm-searching::selection,
.cm-searching span::selection {
  background-color: var(--jp-search-unselected-match-background-color);
  color: var(--jp-search-unselected-match-color);
}

.jp-current-match > .cm-searching,
.jp-current-match > .cm-searching span,
.cm-searching > .jp-current-match,
.cm-searching > .jp-current-match span {
  background-color: var(--jp-search-selected-match-background-color);
  color: var(--jp-search-selected-match-color);
}

.jp-current-match > .cm-searching::selection,
.cm-searching > .jp-current-match::selection,
.jp-current-match > .cm-searching span::selection {
  background-color: var(--jp-search-selected-match-background-color);
  color: var(--jp-search-selected-match-color);
}

.cm-trailingspace {
  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAFCAYAAAB4ka1VAAAAsElEQVQIHQGlAFr/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA7+r3zKmT0/+pk9P/7+r3zAAAAAAAAAAABAAAAAAAAAAA6OPzM+/q9wAAAAAA6OPzMwAAAAAAAAAAAgAAAAAAAAAAGR8NiRQaCgAZIA0AGR8NiQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQyoYJ/SY80UAAAAASUVORK5CYII=);
  background-position: center left;
  background-repeat: repeat-x;
}

.jp-CollaboratorCursor-hover {
  position: absolute;
  z-index: 1;
  transform: translateX(-50%);
  color: white;
  border-radius: 3px;
  padding-left: 4px;
  padding-right: 4px;
  padding-top: 1px;
  padding-bottom: 1px;
  text-align: center;
  font-size: var(--jp-ui-font-size1);
  white-space: nowrap;
}

.jp-CodeMirror-ruler {
  border-left: 1px dashed var(--jp-border-color2);
}

/* Styles for shared cursors (remote cursor locations and selected ranges) */
.jp-CodeMirrorEditor .cm-ySelectionCaret {
  position: relative;
  border-left: 1px solid black;
  margin-left: -1px;
  margin-right: -1px;
  box-sizing: border-box;
}

.jp-CodeMirrorEditor .cm-ySelectionCaret > .cm-ySelectionInfo {
  white-space: nowrap;
  position: absolute;
  top: -1.15em;
  padding-bottom: 0.05em;
  left: -1px;
  font-size: 0.95em;
  font-family: var(--jp-ui-font-family);
  font-weight: bold;
  line-height: normal;
  user-select: none;
  color: white;
  padding-left: 2px;
  padding-right: 2px;
  z-index: 101;
  transition: opacity 0.3s ease-in-out;
}

.jp-CodeMirrorEditor .cm-ySelectionInfo {
  transition-delay: 0.7s;
  opacity: 0;
}

.jp-CodeMirrorEditor .cm-ySelectionCaret:hover > .cm-ySelectionInfo {
  opacity: 1;
  transition-delay: 0s;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-MimeDocument {
  outline: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-filebrowser-button-height: 28px;
  --jp-private-filebrowser-button-width: 48px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-FileBrowser .jp-SidePanel-content {
  display: flex;
  flex-direction: column;
}

.jp-FileBrowser-toolbar.jp-Toolbar {
  flex-wrap: wrap;
  row-gap: 12px;
  border-bottom: none;
  height: auto;
  margin: 8px 12px 0;
  box-shadow: none;
  padding: 0;
  justify-content: flex-start;
}

.jp-FileBrowser-Panel {
  flex: 1 1 auto;
  display: flex;
  flex-direction: column;
}

.jp-BreadCrumbs {
  flex: 0 0 auto;
  margin: 8px 12px;
}

.jp-BreadCrumbs-item {
  margin: 0 2px;
  padding: 0 2px;
  border-radius: var(--jp-border-radius);
  cursor: pointer;
}

.jp-BreadCrumbs-item:hover {
  background-color: var(--jp-layout-color2);
}

.jp-BreadCrumbs-item:first-child {
  margin-left: 0;
}

.jp-BreadCrumbs-item.jp-mod-dropTarget {
  background-color: var(--jp-brand-color2);
  opacity: 0.7;
}

/*-----------------------------------------------------------------------------
| Buttons
|----------------------------------------------------------------------------*/

.jp-FileBrowser-toolbar > .jp-Toolbar-item {
  flex: 0 0 auto;
  padding-left: 0;
  padding-right: 2px;
  align-items: center;
  height: unset;
}

.jp-FileBrowser-toolbar > .jp-Toolbar-item .jp-ToolbarButtonComponent {
  width: 40px;
}

/*-----------------------------------------------------------------------------
| Other styles
|----------------------------------------------------------------------------*/

.jp-FileDialog.jp-mod-conflict input {
  color: var(--jp-error-color1);
}

.jp-FileDialog .jp-new-name-title {
  margin-top: 12px;
}

.jp-LastModified-hidden {
  display: none;
}

.jp-FileSize-hidden {
  display: none;
}

.jp-FileBrowser .lm-AccordionPanel > h3:first-child {
  display: none;
}

/*-----------------------------------------------------------------------------
| DirListing
|----------------------------------------------------------------------------*/

.jp-DirListing {
  flex: 1 1 auto;
  display: flex;
  flex-direction: column;
  outline: 0;
}

.jp-DirListing-header {
  flex: 0 0 auto;
  display: flex;
  flex-direction: row;
  align-items: center;
  overflow: hidden;
  border-top: var(--jp-border-width) solid var(--jp-border-color2);
  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
  box-shadow: var(--jp-toolbar-box-shadow);
  z-index: 2;
}

.jp-DirListing-headerItem {
  padding: 4px 12px 2px;
  font-weight: 500;
}

.jp-DirListing-headerItem:hover {
  background: var(--jp-layout-color2);
}

.jp-DirListing-headerItem.jp-id-name {
  flex: 1 0 84px;
}

.jp-DirListing-headerItem.jp-id-modified {
  flex: 0 0 112px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
}

.jp-DirListing-headerItem.jp-id-filesize {
  flex: 0 0 75px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
}

.jp-id-narrow {
  display: none;
  flex: 0 0 5px;
  padding: 4px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
  color: var(--jp-border-color2);
}

.jp-DirListing-narrow .jp-id-narrow {
  display: block;
}

.jp-DirListing-narrow .jp-id-modified,
.jp-DirListing-narrow .jp-DirListing-itemModified {
  display: none;
}

.jp-DirListing-headerItem.jp-mod-selected {
  font-weight: 600;
}

/* increase specificity to override bundled default */
.jp-DirListing-content {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  list-style-type: none;
  overflow: auto;
  background-color: var(--jp-layout-color1);
}

.jp-DirListing-content mark {
  color: var(--jp-ui-font-color0);
  background-color: transparent;
  font-weight: bold;
}

.jp-DirListing-content .jp-DirListing-item.jp-mod-selected mark {
  color: var(--jp-ui-inverse-font-color0);
}

/* Style the directory listing content when a user drops a file to upload */
.jp-DirListing.jp-mod-native-drop .jp-DirListing-content {
  outline: 5px dashed rgba(128, 128, 128, 0.5);
  outline-offset: -10px;
  cursor: copy;
}

.jp-DirListing-item {
  display: flex;
  flex-direction: row;
  align-items: center;
  padding: 4px 12px;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-DirListing-checkboxWrapper {
  /* Increases hit area of checkbox. */
  padding: 4px;
}

.jp-DirListing-header
  .jp-DirListing-checkboxWrapper
  + .jp-DirListing-headerItem {
  padding-left: 4px;
}

.jp-DirListing-content .jp-DirListing-checkboxWrapper {
  position: relative;
  left: -4px;
  margin: -4px 0 -4px -8px;
}

.jp-DirListing-checkboxWrapper.jp-mod-visible {
  visibility: visible;
}

/* For devices that support hovering, hide checkboxes until hovered, selected...
*/
@media (hover: hover) {
  .jp-DirListing-checkboxWrapper {
    visibility: hidden;
  }

  .jp-DirListing-item:hover .jp-DirListing-checkboxWrapper,
  .jp-DirListing-item.jp-mod-selected .jp-DirListing-checkboxWrapper {
    visibility: visible;
  }
}

.jp-DirListing-item[data-is-dot] {
  opacity: 75%;
}

.jp-DirListing-item.jp-mod-selected {
  color: var(--jp-ui-inverse-font-color1);
  background: var(--jp-brand-color1);
}

.jp-DirListing-item.jp-mod-dropTarget {
  background: var(--jp-brand-color3);
}

.jp-DirListing-item:hover:not(.jp-mod-selected) {
  background: var(--jp-layout-color2);
}

.jp-DirListing-itemIcon {
  flex: 0 0 20px;
  margin-right: 4px;
}

.jp-DirListing-itemText {
  flex: 1 0 64px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  user-select: none;
}

.jp-DirListing-itemText:focus {
  outline-width: 2px;
  outline-color: var(--jp-inverse-layout-color1);
  outline-style: solid;
  outline-offset: 1px;
}

.jp-DirListing-item.jp-mod-selected .jp-DirListing-itemText:focus {
  outline-color: var(--jp-layout-color1);
}

.jp-DirListing-itemModified {
  flex: 0 0 125px;
  text-align: right;
}

.jp-DirListing-itemFileSize {
  flex: 0 0 90px;
  text-align: right;
}

.jp-DirListing-editor {
  flex: 1 0 64px;
  outline: none;
  border: none;
  color: var(--jp-ui-font-color1);
  background-color: var(--jp-layout-color1);
}

.jp-DirListing-item.jp-mod-running .jp-DirListing-itemIcon::before {
  color: var(--jp-success-color1);
  content: '\25CF';
  font-size: 8px;
  position: absolute;
  left: -8px;
}

.jp-DirListing-item.jp-mod-running.jp-mod-selected
  .jp-DirListing-itemIcon::before {
  color: var(--jp-ui-inverse-font-color1);
}

.jp-DirListing-item.lm-mod-drag-image,
.jp-DirListing-item.jp-mod-selected.lm-mod-drag-image {
  font-size: var(--jp-ui-font-size1);
  padding-left: 4px;
  margin-left: 4px;
  width: 160px;
  background-color: var(--jp-ui-inverse-font-color2);
  box-shadow: var(--jp-elevation-z2);
  border-radius: 0;
  color: var(--jp-ui-font-color1);
  transform: translateX(-40%) translateY(-58%);
}

.jp-Document {
  min-width: 120px;
  min-height: 120px;
  outline: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Main OutputArea
| OutputArea has a list of Outputs
|----------------------------------------------------------------------------*/

.jp-OutputArea {
  overflow-y: auto;
}

.jp-OutputArea-child {
  display: table;
  table-layout: fixed;
  width: 100%;
  overflow: hidden;
}

.jp-OutputPrompt {
  width: var(--jp-cell-prompt-width);
  color: var(--jp-cell-outprompt-font-color);
  font-family: var(--jp-cell-prompt-font-family);
  padding: var(--jp-code-padding);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
  opacity: var(--jp-cell-prompt-opacity);

  /* Right align prompt text, don't wrap to handle large prompt numbers */
  text-align: right;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;

  /* Disable text selection */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-OutputArea-prompt {
  display: table-cell;
  vertical-align: top;
}

.jp-OutputArea-output {
  display: table-cell;
  width: 100%;
  height: auto;
  overflow: auto;
  user-select: text;
  -moz-user-select: text;
  -webkit-user-select: text;
  -ms-user-select: text;
}

.jp-OutputArea .jp-RenderedText {
  padding-left: 1ch;
}

/**
 * Prompt overlay.
 */

.jp-OutputArea-promptOverlay {
  position: absolute;
  top: 0;
  width: var(--jp-cell-prompt-width);
  height: 100%;
  opacity: 0.5;
}

.jp-OutputArea-promptOverlay:hover {
  background: var(--jp-layout-color2);
  box-shadow: inset 0 0 1px var(--jp-inverse-layout-color0);
  cursor: zoom-out;
}

.jp-mod-outputsScrolled .jp-OutputArea-promptOverlay:hover {
  cursor: zoom-in;
}

/**
 * Isolated output.
 */
.jp-OutputArea-output.jp-mod-isolated {
  width: 100%;
  display: block;
}

/*
When drag events occur, `lm-mod-override-cursor` is added to the body.
Because iframes steal all cursor events, the following two rules are necessary
to suppress pointer events while resize drags are occurring. There may be a
better solution to this problem.
*/
body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated {
  position: relative;
}

body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated::before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: transparent;
}

/* pre */

.jp-OutputArea-output pre {
  border: none;
  margin: 0;
  padding: 0;
  overflow-x: auto;
  overflow-y: auto;
  word-break: break-all;
  word-wrap: break-word;
  white-space: pre-wrap;
}

/* tables */

.jp-OutputArea-output.jp-RenderedHTMLCommon table {
  margin-left: 0;
  margin-right: 0;
}

/* description lists */

.jp-OutputArea-output dl,
.jp-OutputArea-output dt,
.jp-OutputArea-output dd {
  display: block;
}

.jp-OutputArea-output dl {
  width: 100%;
  overflow: hidden;
  padding: 0;
  margin: 0;
}

.jp-OutputArea-output dt {
  font-weight: bold;
  float: left;
  width: 20%;
  padding: 0;
  margin: 0;
}

.jp-OutputArea-output dd {
  float: left;
  width: 80%;
  padding: 0;
  margin: 0;
}

.jp-TrimmedOutputs pre {
  background: var(--jp-layout-color3);
  font-size: calc(var(--jp-code-font-size) * 1.4);
  text-align: center;
  text-transform: uppercase;
}

/* Hide the gutter in case of
 *  - nested output areas (e.g. in the case of output widgets)
 *  - mirrored output areas
 */
.jp-OutputArea .jp-OutputArea .jp-OutputArea-prompt {
  display: none;
}

/* Hide empty lines in the output area, for instance due to cleared widgets */
.jp-OutputArea-prompt:empty {
  padding: 0;
  border: 0;
}

/*-----------------------------------------------------------------------------
| executeResult is added to any Output-result for the display of the object
| returned by a cell
|----------------------------------------------------------------------------*/

.jp-OutputArea-output.jp-OutputArea-executeResult {
  margin-left: 0;
  width: 100%;
}

/* Text output with the Out[] prompt needs a top padding to match the
 * alignment of the Out[] prompt itself.
 */
.jp-OutputArea-executeResult .jp-RenderedText.jp-OutputArea-output {
  padding-top: var(--jp-code-padding);
  border-top: var(--jp-border-width) solid transparent;
}

/*-----------------------------------------------------------------------------
| The Stdin output
|----------------------------------------------------------------------------*/

.jp-Stdin-prompt {
  color: var(--jp-content-font-color0);
  padding-right: var(--jp-code-padding);
  vertical-align: baseline;
  flex: 0 0 auto;
}

.jp-Stdin-input {
  font-family: var(--jp-code-font-family);
  font-size: inherit;
  color: inherit;
  background-color: inherit;
  width: 42%;
  min-width: 200px;

  /* make sure input baseline aligns with prompt */
  vertical-align: baseline;

  /* padding + margin = 0.5em between prompt and cursor */
  padding: 0 0.25em;
  margin: 0 0.25em;
  flex: 0 0 70%;
}

.jp-Stdin-input::placeholder {
  opacity: 0;
}

.jp-Stdin-input:focus {
  box-shadow: none;
}

.jp-Stdin-input:focus::placeholder {
  opacity: 1;
}

/*-----------------------------------------------------------------------------
| Output Area View
|----------------------------------------------------------------------------*/

.jp-LinkedOutputView .jp-OutputArea {
  height: 100%;
  display: block;
}

.jp-LinkedOutputView .jp-OutputArea-output:only-child {
  height: 100%;
}

/*-----------------------------------------------------------------------------
| Printing
|----------------------------------------------------------------------------*/

@media print {
  .jp-OutputArea-child {
    break-inside: avoid-page;
  }
}

/*-----------------------------------------------------------------------------
| Mobile
|----------------------------------------------------------------------------*/
@media only screen and (max-width: 760px) {
  .jp-OutputPrompt {
    display: table-row;
    text-align: left;
  }

  .jp-OutputArea-child .jp-OutputArea-output {
    display: table-row;
    margin-left: var(--jp-notebook-padding);
  }
}

/* Trimmed outputs warning */
.jp-TrimmedOutputs > a {
  margin: 10px;
  text-decoration: none;
  cursor: pointer;
}

.jp-TrimmedOutputs > a:hover {
  text-decoration: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Table of Contents
|----------------------------------------------------------------------------*/

:root {
  --jp-private-toc-active-width: 4px;
}

.jp-TableOfContents {
  display: flex;
  flex-direction: column;
  background: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  height: 100%;
}

.jp-TableOfContents-placeholder {
  text-align: center;
}

.jp-TableOfContents-placeholderContent {
  color: var(--jp-content-font-color2);
  padding: 8px;
}

.jp-TableOfContents-placeholderContent > h3 {
  margin-bottom: var(--jp-content-heading-margin-bottom);
}

.jp-TableOfContents .jp-SidePanel-content {
  overflow-y: auto;
}

.jp-TableOfContents-tree {
  margin: 4px;
}

.jp-TableOfContents ol {
  list-style-type: none;
}

/* stylelint-disable-next-line selector-max-type */
.jp-TableOfContents li > ol {
  /* Align left border with triangle icon center */
  padding-left: 11px;
}

.jp-TableOfContents-content {
  /* left margin for the active heading indicator */
  margin: 0 0 0 var(--jp-private-toc-active-width);
  padding: 0;
  background-color: var(--jp-layout-color1);
}

.jp-tocItem {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-tocItem-heading {
  display: flex;
  cursor: pointer;
}

.jp-tocItem-heading:hover {
  background-color: var(--jp-layout-color2);
}

.jp-tocItem-content {
  display: block;
  padding: 4px 0;
  white-space: nowrap;
  text-overflow: ellipsis;
  overflow-x: hidden;
}

.jp-tocItem-collapser {
  height: 20px;
  margin: 2px 2px 0;
  padding: 0;
  background: none;
  border: none;
  cursor: pointer;
}

.jp-tocItem-collapser:hover {
  background-color: var(--jp-layout-color3);
}

/* Active heading indicator */

.jp-tocItem-heading::before {
  content: ' ';
  background: transparent;
  width: var(--jp-private-toc-active-width);
  height: 24px;
  position: absolute;
  left: 0;
  border-radius: var(--jp-border-radius);
}

.jp-tocItem-heading.jp-tocItem-active::before {
  background-color: var(--jp-brand-color1);
}

.jp-tocItem-heading:hover.jp-tocItem-active::before {
  background: var(--jp-brand-color0);
  opacity: 1;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Collapser {
  flex: 0 0 var(--jp-cell-collapser-width);
  padding: 0;
  margin: 0;
  border: none;
  outline: none;
  background: transparent;
  border-radius: var(--jp-border-radius);
  opacity: 1;
}

.jp-Collapser-child {
  display: block;
  width: 100%;
  box-sizing: border-box;

  /* height: 100% doesn't work because the height of its parent is computed from content */
  position: absolute;
  top: 0;
  bottom: 0;
}

/*-----------------------------------------------------------------------------
| Printing
|----------------------------------------------------------------------------*/

/*
Hiding collapsers in print mode.

Note: input and output wrappers have "display: block" propery in print mode.
*/

@media print {
  .jp-Collapser {
    display: none;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Header/Footer
|----------------------------------------------------------------------------*/

/* Hidden by zero height by default */
.jp-CellHeader,
.jp-CellFooter {
  height: 0;
  width: 100%;
  padding: 0;
  margin: 0;
  border: none;
  outline: none;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Input
|----------------------------------------------------------------------------*/

/* All input areas */
.jp-InputArea {
  display: table;
  table-layout: fixed;
  width: 100%;
  overflow: hidden;
}

.jp-InputArea-editor {
  display: table-cell;
  overflow: hidden;
  vertical-align: top;

  /* This is the non-active, default styling */
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  border-radius: 0;
  background: var(--jp-cell-editor-background);
}

.jp-InputPrompt {
  display: table-cell;
  vertical-align: top;
  width: var(--jp-cell-prompt-width);
  color: var(--jp-cell-inprompt-font-color);
  font-family: var(--jp-cell-prompt-font-family);
  padding: var(--jp-code-padding);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  opacity: var(--jp-cell-prompt-opacity);
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;

  /* Right align prompt text, don't wrap to handle large prompt numbers */
  text-align: right;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;

  /* Disable text selection */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

/*-----------------------------------------------------------------------------
| Mobile
|----------------------------------------------------------------------------*/
@media only screen and (max-width: 760px) {
  .jp-InputArea-editor {
    display: table-row;
    margin-left: var(--jp-notebook-padding);
  }

  .jp-InputPrompt {
    display: table-row;
    text-align: left;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Placeholder
|----------------------------------------------------------------------------*/

.jp-Placeholder {
  display: table;
  table-layout: fixed;
  width: 100%;
}

.jp-Placeholder-prompt {
  display: table-cell;
  box-sizing: border-box;
}

.jp-Placeholder-content {
  display: table-cell;
  padding: 4px 6px;
  border: 1px solid transparent;
  border-radius: 0;
  background: none;
  box-sizing: border-box;
  cursor: pointer;
}

.jp-Placeholder-contentContainer {
  display: flex;
}

.jp-Placeholder-content:hover,
.jp-InputPlaceholder > .jp-Placeholder-content:hover {
  border-color: var(--jp-layout-color3);
}

.jp-Placeholder-content .jp-MoreHorizIcon {
  width: 32px;
  height: 16px;
  border: 1px solid transparent;
  border-radius: var(--jp-border-radius);
}

.jp-Placeholder-content .jp-MoreHorizIcon:hover {
  border: 1px solid var(--jp-border-color1);
  box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.25);
  background-color: var(--jp-layout-color0);
}

.jp-PlaceholderText {
  white-space: nowrap;
  overflow-x: hidden;
  color: var(--jp-inverse-layout-color3);
  font-family: var(--jp-code-font-family);
}

.jp-InputPlaceholder > .jp-Placeholder-content {
  border-color: var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Private CSS variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-cell-scrolling-output-offset: 5px;
}

/*-----------------------------------------------------------------------------
| Cell
|----------------------------------------------------------------------------*/

.jp-Cell {
  padding: var(--jp-cell-padding);
  margin: 0;
  border: none;
  outline: none;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Common input/output
|----------------------------------------------------------------------------*/

.jp-Cell-inputWrapper,
.jp-Cell-outputWrapper {
  display: flex;
  flex-direction: row;
  padding: 0;
  margin: 0;

  /* Added to reveal the box-shadow on the input and output collapsers. */
  overflow: visible;
}

/* Only input/output areas inside cells */
.jp-Cell-inputArea,
.jp-Cell-outputArea {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Collapser
|----------------------------------------------------------------------------*/

/* Make the output collapser disappear when there is not output, but do so
 * in a manner that leaves it in the layout and preserves its width.
 */
.jp-Cell.jp-mod-noOutputs .jp-Cell-outputCollapser {
  border: none !important;
  background: transparent !important;
}

.jp-Cell:not(.jp-mod-noOutputs) .jp-Cell-outputCollapser {
  min-height: var(--jp-cell-collapser-min-height);
}

/*-----------------------------------------------------------------------------
| Output
|----------------------------------------------------------------------------*/

/* Put a space between input and output when there IS output */
.jp-Cell:not(.jp-mod-noOutputs) .jp-Cell-outputWrapper {
  margin-top: 5px;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea {
  overflow-y: auto;
  max-height: 24em;
  margin-left: var(--jp-private-cell-scrolling-output-offset);
  resize: vertical;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea[style*='height'] {
  max-height: unset;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea::after {
  content: ' ';
  box-shadow: inset 0 0 6px 2px rgb(0 0 0 / 30%);
  width: 100%;
  height: 100%;
  position: sticky;
  bottom: 0;
  top: 0;
  margin-top: -50%;
  float: left;
  display: block;
  pointer-events: none;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-child {
  padding-top: 6px;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-prompt {
  width: calc(
    var(--jp-cell-prompt-width) - var(--jp-private-cell-scrolling-output-offset)
  );
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-promptOverlay {
  left: calc(-1 * var(--jp-private-cell-scrolling-output-offset));
}

/*-----------------------------------------------------------------------------
| CodeCell
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| MarkdownCell
|----------------------------------------------------------------------------*/

.jp-MarkdownOutput {
  display: table-cell;
  width: 100%;
  margin-top: 0;
  margin-bottom: 0;
  padding-left: var(--jp-code-padding);
}

.jp-MarkdownOutput.jp-RenderedHTMLCommon {
  overflow: auto;
}

/* collapseHeadingButton (show always if hiddenCellsButton is _not_ shown) */
.jp-collapseHeadingButton {
  display: flex;
  min-height: var(--jp-cell-collapser-min-height);
  font-size: var(--jp-code-font-size);
  position: absolute;
  background-color: transparent;
  background-size: 25px;
  background-repeat: no-repeat;
  background-position-x: center;
  background-position-y: top;
  background-image: var(--jp-icon-caret-down);
  right: 0;
  top: 0;
  bottom: 0;
}

.jp-collapseHeadingButton.jp-mod-collapsed {
  background-image: var(--jp-icon-caret-right);
}

/*
 set the container font size to match that of content
 so that the nested collapse buttons have the right size
*/
.jp-MarkdownCell .jp-InputPrompt {
  font-size: var(--jp-content-font-size1);
}

/*
  Align collapseHeadingButton with cell top header
  The font sizes are identical to the ones in packages/rendermime/style/base.css
*/
.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='1'] {
  font-size: var(--jp-content-font-size5);
  background-position-y: calc(0.3 * var(--jp-content-font-size5));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='2'] {
  font-size: var(--jp-content-font-size4);
  background-position-y: calc(0.3 * var(--jp-content-font-size4));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='3'] {
  font-size: var(--jp-content-font-size3);
  background-position-y: calc(0.3 * var(--jp-content-font-size3));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='4'] {
  font-size: var(--jp-content-font-size2);
  background-position-y: calc(0.3 * var(--jp-content-font-size2));
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='5'] {
  font-size: var(--jp-content-font-size1);
  background-position-y: top;
}

.jp-mod-rendered .jp-collapseHeadingButton[data-heading-level='6'] {
  font-size: var(--jp-content-font-size0);
  background-position-y: top;
}

/* collapseHeadingButton (show only on (hover,active) if hiddenCellsButton is shown) */
.jp-Notebook.jp-mod-showHiddenCellsButton .jp-collapseHeadingButton {
  display: none;
}

.jp-Notebook.jp-mod-showHiddenCellsButton
  :is(.jp-MarkdownCell:hover, .jp-mod-active)
  .jp-collapseHeadingButton {
  display: flex;
}

/* showHiddenCellsButton (only show if jp-mod-showHiddenCellsButton is set, which
is a consequence of the showHiddenCellsButton option in Notebook Settings)*/
.jp-Notebook.jp-mod-showHiddenCellsButton .jp-showHiddenCellsButton {
  margin-left: calc(var(--jp-cell-prompt-width) + 2 * var(--jp-code-padding));
  margin-top: var(--jp-code-padding);
  border: 1px solid var(--jp-border-color2);
  background-color: var(--jp-border-color3) !important;
  color: var(--jp-content-font-color0) !important;
  display: flex;
}

.jp-Notebook.jp-mod-showHiddenCellsButton .jp-showHiddenCellsButton:hover {
  background-color: var(--jp-border-color2) !important;
}

.jp-showHiddenCellsButton {
  display: none;
}

/*-----------------------------------------------------------------------------
| Printing
|----------------------------------------------------------------------------*/

/*
Using block instead of flex to allow the use of the break-inside CSS property for
cell outputs.
*/

@media print {
  .jp-Cell-inputWrapper,
  .jp-Cell-outputWrapper {
    display: block;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-notebook-toolbar-padding: 2px 5px 2px 2px;
}

/*-----------------------------------------------------------------------------

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-NotebookPanel-toolbar {
  padding: var(--jp-notebook-toolbar-padding);

  /* disable paint containment from lumino 2.0 default strict CSS containment */
  contain: style size !important;
}

.jp-Toolbar-item.jp-Notebook-toolbarCellType .jp-select-wrapper.jp-mod-focused {
  border: none;
  box-shadow: none;
}

.jp-Notebook-toolbarCellTypeDropdown select {
  height: 24px;
  font-size: var(--jp-ui-font-size1);
  line-height: 14px;
  border-radius: 0;
  display: block;
}

.jp-Notebook-toolbarCellTypeDropdown span {
  top: 5px !important;
}

.jp-Toolbar-responsive-popup {
  position: absolute;
  height: fit-content;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: flex-end;
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  background: var(--jp-toolbar-background);
  min-height: var(--jp-toolbar-micro-height);
  padding: var(--jp-notebook-toolbar-padding);
  z-index: 1;
  right: 0;
  top: 0;
}

.jp-Toolbar > .jp-Toolbar-responsive-opener {
  margin-left: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-Notebook-ExecutionIndicator {
  position: relative;
  display: inline-block;
  height: 100%;
  z-index: 9997;
}

.jp-Notebook-ExecutionIndicator-tooltip {
  visibility: hidden;
  height: auto;
  width: max-content;
  width: -moz-max-content;
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color1);
  text-align: justify;
  border-radius: 6px;
  padding: 0 5px;
  position: fixed;
  display: table;
}

.jp-Notebook-ExecutionIndicator-tooltip.up {
  transform: translateX(-50%) translateY(-100%) translateY(-32px);
}

.jp-Notebook-ExecutionIndicator-tooltip.down {
  transform: translateX(calc(-100% + 16px)) translateY(5px);
}

.jp-Notebook-ExecutionIndicator-tooltip.hidden {
  display: none;
}

.jp-Notebook-ExecutionIndicator:hover .jp-Notebook-ExecutionIndicator-tooltip {
  visibility: visible;
}

.jp-Notebook-ExecutionIndicator span {
  font-size: var(--jp-ui-font-size1);
  font-family: var(--jp-ui-font-family);
  color: var(--jp-ui-font-color1);
  line-height: 24px;
  display: block;
}

.jp-Notebook-ExecutionIndicator-progress-bar {
  display: flex;
  justify-content: center;
  height: 100%;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

/*
 * Execution indicator
 */
.jp-tocItem-content::after {
  content: '';

  /* Must be identical to form a circle */
  width: 12px;
  height: 12px;
  background: none;
  border: none;
  position: absolute;
  right: 0;
}

.jp-tocItem-content[data-running='0']::after {
  border-radius: 50%;
  border: var(--jp-border-width) solid var(--jp-inverse-layout-color3);
  background: none;
}

.jp-tocItem-content[data-running='1']::after {
  border-radius: 50%;
  border: var(--jp-border-width) solid var(--jp-inverse-layout-color3);
  background-color: var(--jp-inverse-layout-color3);
}

.jp-tocItem-content[data-running='0'],
.jp-tocItem-content[data-running='1'] {
  margin-right: 12px;
}

/*
 * Copyright (c) Jupyter Development Team.
 * Distributed under the terms of the Modified BSD License.
 */

.jp-Notebook-footer {
  height: 27px;
  margin-left: calc(
    var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) +
      var(--jp-cell-padding)
  );
  width: calc(
    100% -
      (
        var(--jp-cell-prompt-width) + var(--jp-cell-collapser-width) +
          var(--jp-cell-padding) + var(--jp-cell-padding)
      )
  );
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  color: var(--jp-ui-font-color3);
  margin-top: 6px;
  background: none;
  cursor: pointer;
}

.jp-Notebook-footer:focus {
  border-color: var(--jp-cell-editor-active-border-color);
}

/* For devices that support hovering, hide footer until hover */
@media (hover: hover) {
  .jp-Notebook-footer {
    opacity: 0;
  }

  .jp-Notebook-footer:focus,
  .jp-Notebook-footer:hover {
    opacity: 1;
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Imports
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| CSS variables
|----------------------------------------------------------------------------*/

:root {
  --jp-side-by-side-output-size: 1fr;
  --jp-side-by-side-resized-cell: var(--jp-side-by-side-output-size);
  --jp-private-notebook-dragImage-width: 304px;
  --jp-private-notebook-dragImage-height: 36px;
  --jp-private-notebook-selected-color: var(--md-blue-400);
  --jp-private-notebook-active-color: var(--md-green-400);
}

/*-----------------------------------------------------------------------------
| Notebook
|----------------------------------------------------------------------------*/

/* stylelint-disable selector-max-class */

.jp-NotebookPanel {
  display: block;
  height: 100%;
}

.jp-NotebookPanel.jp-Document {
  min-width: 240px;
  min-height: 120px;
}

.jp-Notebook {
  padding: var(--jp-notebook-padding);
  outline: none;
  overflow: auto;
  background: var(--jp-layout-color0);
}

.jp-Notebook.jp-mod-scrollPastEnd::after {
  display: block;
  content: '';
  min-height: var(--jp-notebook-scroll-padding);
}

.jp-MainAreaWidget-ContainStrict .jp-Notebook * {
  contain: strict;
}

.jp-Notebook .jp-Cell {
  overflow: visible;
}

.jp-Notebook .jp-Cell .jp-InputPrompt {
  cursor: move;
}

/*-----------------------------------------------------------------------------
| Notebook state related styling
|
| The notebook and cells each have states, here are the possibilities:
|
| - Notebook
|   - Command
|   - Edit
| - Cell
|   - None
|   - Active (only one can be active)
|   - Selected (the cells actions are applied to)
|   - Multiselected (when multiple selected, the cursor)
|   - No outputs
|----------------------------------------------------------------------------*/

/* Command or edit modes */

.jp-Notebook .jp-Cell:not(.jp-mod-active) .jp-InputPrompt {
  opacity: var(--jp-cell-prompt-not-active-opacity);
  color: var(--jp-cell-prompt-not-active-font-color);
}

.jp-Notebook .jp-Cell:not(.jp-mod-active) .jp-OutputPrompt {
  opacity: var(--jp-cell-prompt-not-active-opacity);
  color: var(--jp-cell-prompt-not-active-font-color);
}

/* cell is active */
.jp-Notebook .jp-Cell.jp-mod-active .jp-Collapser {
  background: var(--jp-brand-color1);
}

/* cell is dirty */
.jp-Notebook .jp-Cell.jp-mod-dirty .jp-InputPrompt {
  color: var(--jp-warn-color1);
}

.jp-Notebook .jp-Cell.jp-mod-dirty .jp-InputPrompt::before {
  color: var(--jp-warn-color1);
  content: '';
}

.jp-Notebook .jp-Cell.jp-mod-active.jp-mod-dirty .jp-Collapser {
  background: var(--jp-warn-color1);
}

/* collapser is hovered */
.jp-Notebook .jp-Cell .jp-Collapser:hover {
  box-shadow: var(--jp-elevation-z2);
  background: var(--jp-brand-color1);
  opacity: var(--jp-cell-collapser-not-active-hover-opacity);
}

/* cell is active and collapser is hovered */
.jp-Notebook .jp-Cell.jp-mod-active .jp-Collapser:hover {
  background: var(--jp-brand-color0);
  opacity: 1;
}

/* Command mode */

.jp-Notebook.jp-mod-commandMode .jp-Cell.jp-mod-selected {
  background: var(--jp-notebook-multiselected-color);
}

.jp-Notebook.jp-mod-commandMode
  .jp-Cell.jp-mod-active.jp-mod-selected:not(.jp-mod-multiSelected) {
  background: transparent;
}

/* Edit mode */

.jp-Notebook.jp-mod-editMode .jp-Cell.jp-mod-active .jp-InputArea-editor {
  border: var(--jp-border-width) solid var(--jp-cell-editor-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
  background-color: var(--jp-cell-editor-active-background);
}

/*-----------------------------------------------------------------------------
| Notebook drag and drop
|----------------------------------------------------------------------------*/

.jp-Notebook-cell.jp-mod-dropSource {
  opacity: 0.5;
}

.jp-Notebook-cell.jp-mod-dropTarget,
.jp-Notebook.jp-mod-commandMode
  .jp-Notebook-cell.jp-mod-active.jp-mod-selected.jp-mod-dropTarget {
  border-top-color: var(--jp-private-notebook-selected-color);
  border-top-style: solid;
  border-top-width: 2px;
}

.jp-dragImage {
  display: block;
  flex-direction: row;
  width: var(--jp-private-notebook-dragImage-width);
  height: var(--jp-private-notebook-dragImage-height);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background);
  overflow: visible;
}

.jp-dragImage-singlePrompt {
  box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.12);
}

.jp-dragImage .jp-dragImage-content {
  flex: 1 1 auto;
  z-index: 2;
  font-size: var(--jp-code-font-size);
  font-family: var(--jp-code-font-family);
  line-height: var(--jp-code-line-height);
  padding: var(--jp-code-padding);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background-color);
  color: var(--jp-content-font-color3);
  text-align: left;
  margin: 4px 4px 4px 0;
}

.jp-dragImage .jp-dragImage-prompt {
  flex: 0 0 auto;
  min-width: 36px;
  color: var(--jp-cell-inprompt-font-color);
  padding: var(--jp-code-padding);
  padding-left: 12px;
  font-family: var(--jp-cell-prompt-font-family);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  line-height: 1.9;
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
}

.jp-dragImage-multipleBack {
  z-index: -1;
  position: absolute;
  height: 32px;
  width: 300px;
  top: 8px;
  left: 8px;
  background: var(--jp-layout-color2);
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.12);
}

/*-----------------------------------------------------------------------------
| Cell toolbar
|----------------------------------------------------------------------------*/

.jp-NotebookTools {
  display: block;
  min-width: var(--jp-sidebar-min-width);
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);

  /* This is needed so that all font sizing of children done in ems is
    * relative to this base size */
  font-size: var(--jp-ui-font-size1);
  overflow: auto;
}

.jp-ActiveCellTool {
  padding: 12px 0;
  display: flex;
}

.jp-ActiveCellTool-Content {
  flex: 1 1 auto;
}

.jp-ActiveCellTool .jp-ActiveCellTool-CellContent {
  background: var(--jp-cell-editor-background);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  border-radius: 0;
  min-height: 29px;
}

.jp-ActiveCellTool .jp-InputPrompt {
  min-width: calc(var(--jp-cell-prompt-width) * 0.75);
}

.jp-ActiveCellTool-CellContent > pre {
  padding: 5px 4px;
  margin: 0;
  white-space: normal;
}

.jp-MetadataEditorTool {
  flex-direction: column;
  padding: 12px 0;
}

.jp-RankedPanel > :not(:first-child) {
  margin-top: 12px;
}

.jp-KeySelector select.jp-mod-styled {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  border: var(--jp-border-width) solid var(--jp-border-color1);
}

.jp-KeySelector label,
.jp-MetadataEditorTool label,
.jp-NumberSetter label {
  line-height: 1.4;
}

.jp-NotebookTools .jp-select-wrapper {
  margin-top: 4px;
  margin-bottom: 0;
}

.jp-NumberSetter input {
  width: 100%;
  margin-top: 4px;
}

.jp-NotebookTools .jp-Collapse {
  margin-top: 16px;
}

/*-----------------------------------------------------------------------------
| Presentation Mode (.jp-mod-presentationMode)
|----------------------------------------------------------------------------*/

.jp-mod-presentationMode .jp-Notebook {
  --jp-content-font-size1: var(--jp-content-presentation-font-size1);
  --jp-code-font-size: var(--jp-code-presentation-font-size);
}

.jp-mod-presentationMode .jp-Notebook .jp-Cell .jp-InputPrompt,
.jp-mod-presentationMode .jp-Notebook .jp-Cell .jp-OutputPrompt {
  flex: 0 0 110px;
}

/*-----------------------------------------------------------------------------
| Side-by-side Mode (.jp-mod-sideBySide)
|----------------------------------------------------------------------------*/
.jp-mod-sideBySide.jp-Notebook .jp-Notebook-cell {
  margin-top: 3em;
  margin-bottom: 3em;
  margin-left: 5%;
  margin-right: 5%;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell {
  display: grid;
  grid-template-columns: minmax(0, 1fr) min-content minmax(
      0,
      var(--jp-side-by-side-output-size)
    );
  grid-template-rows: auto minmax(0, 1fr) auto;
  grid-template-areas:
    'header header header'
    'input handle output'
    'footer footer footer';
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell.jp-mod-resizedCell {
  grid-template-columns: minmax(0, 1fr) min-content minmax(
      0,
      var(--jp-side-by-side-resized-cell)
    );
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellHeader {
  grid-area: header;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-Cell-inputWrapper {
  grid-area: input;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-Cell-outputWrapper {
  /* overwrite the default margin (no vertical separation needed in side by side move */
  margin-top: 0;
  grid-area: output;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellFooter {
  grid-area: footer;
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellResizeHandle {
  grid-area: handle;
  user-select: none;
  display: block;
  height: 100%;
  cursor: ew-resize;
  padding: 0 var(--jp-cell-padding);
}

.jp-mod-sideBySide.jp-Notebook .jp-CodeCell .jp-CellResizeHandle::after {
  content: '';
  display: block;
  background: var(--jp-border-color2);
  height: 100%;
  width: 5px;
}

.jp-mod-sideBySide.jp-Notebook
  .jp-CodeCell.jp-mod-resizedCell
  .jp-CellResizeHandle::after {
  background: var(--jp-border-color0);
}

.jp-CellResizeHandle {
  display: none;
}

/*-----------------------------------------------------------------------------
| Placeholder
|----------------------------------------------------------------------------*/

.jp-Cell-Placeholder {
  padding-left: 55px;
}

.jp-Cell-Placeholder-wrapper {
  background: #fff;
  border: 1px solid;
  border-color: #e5e6e9 #dfe0e4 #d0d1d5;
  border-radius: 4px;
  -webkit-border-radius: 4px;
  margin: 10px 15px;
}

.jp-Cell-Placeholder-wrapper-inner {
  padding: 15px;
  position: relative;
}

.jp-Cell-Placeholder-wrapper-body {
  background-repeat: repeat;
  background-size: 50% auto;
}

.jp-Cell-Placeholder-wrapper-body div {
  background: #f6f7f8;
  background-image: -webkit-linear-gradient(
    left,
    #f6f7f8 0%,
    #edeef1 20%,
    #f6f7f8 40%,
    #f6f7f8 100%
  );
  background-repeat: no-repeat;
  background-size: 800px 104px;
  height: 104px;
  position: absolute;
  right: 15px;
  left: 15px;
  top: 15px;
}

div.jp-Cell-Placeholder-h1 {
  top: 20px;
  height: 20px;
  left: 15px;
  width: 150px;
}

div.jp-Cell-Placeholder-h2 {
  left: 15px;
  top: 50px;
  height: 10px;
  width: 100px;
}

div.jp-Cell-Placeholder-content-1,
div.jp-Cell-Placeholder-content-2,
div.jp-Cell-Placeholder-content-3 {
  left: 15px;
  right: 15px;
  height: 10px;
}

div.jp-Cell-Placeholder-content-1 {
  top: 100px;
}

div.jp-Cell-Placeholder-content-2 {
  top: 120px;
}

div.jp-Cell-Placeholder-content-3 {
  top: 140px;
}

</style>
<style type="text/css">
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*
The following CSS variables define the main, public API for styling JupyterLab.
These variables should be used by all plugins wherever possible. In other
words, plugins should not define custom colors, sizes, etc unless absolutely
necessary. This enables users to change the visual theme of JupyterLab
by changing these variables.

Many variables appear in an ordered sequence (0,1,2,3). These sequences
are designed to work well together, so for example, `--jp-border-color1` should
be used with `--jp-layout-color1`. The numbers have the following meanings:

* 0: super-primary, reserved for special emphasis
* 1: primary, most important under normal situations
* 2: secondary, next most important under normal situations
* 3: tertiary, next most important under normal situations

Throughout JupyterLab, we are mostly following principles from Google's
Material Design when selecting colors. We are not, however, following
all of MD as it is not optimized for dense, information rich UIs.
*/

:root {
  /* Elevation
   *
   * We style box-shadows using Material Design's idea of elevation. These particular numbers are taken from here:
   *
   * https://github.com/material-components/material-components-web
   * https://material-components-web.appspot.com/elevation.html
   */

  --jp-shadow-base-lightness: 0;
  --jp-shadow-umbra-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.2
  );
  --jp-shadow-penumbra-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.14
  );
  --jp-shadow-ambient-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.12
  );
  --jp-elevation-z0: none;
  --jp-elevation-z1: 0 2px 1px -1px var(--jp-shadow-umbra-color),
    0 1px 1px 0 var(--jp-shadow-penumbra-color),
    0 1px 3px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z2: 0 3px 1px -2px var(--jp-shadow-umbra-color),
    0 2px 2px 0 var(--jp-shadow-penumbra-color),
    0 1px 5px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z4: 0 2px 4px -1px var(--jp-shadow-umbra-color),
    0 4px 5px 0 var(--jp-shadow-penumbra-color),
    0 1px 10px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z6: 0 3px 5px -1px var(--jp-shadow-umbra-color),
    0 6px 10px 0 var(--jp-shadow-penumbra-color),
    0 1px 18px 0 var(--jp-shadow-ambient-color);
  --jp-elevation-z8: 0 5px 5px -3px var(--jp-shadow-umbra-color),
    0 8px 10px 1px var(--jp-shadow-penumbra-color),
    0 3px 14px 2px var(--jp-shadow-ambient-color);
  --jp-elevation-z12: 0 7px 8px -4px var(--jp-shadow-umbra-color),
    0 12px 17px 2px var(--jp-shadow-penumbra-color),
    0 5px 22px 4px var(--jp-shadow-ambient-color);
  --jp-elevation-z16: 0 8px 10px -5px var(--jp-shadow-umbra-color),
    0 16px 24px 2px var(--jp-shadow-penumbra-color),
    0 6px 30px 5px var(--jp-shadow-ambient-color);
  --jp-elevation-z20: 0 10px 13px -6px var(--jp-shadow-umbra-color),
    0 20px 31px 3px var(--jp-shadow-penumbra-color),
    0 8px 38px 7px var(--jp-shadow-ambient-color);
  --jp-elevation-z24: 0 11px 15px -7px var(--jp-shadow-umbra-color),
    0 24px 38px 3px var(--jp-shadow-penumbra-color),
    0 9px 46px 8px var(--jp-shadow-ambient-color);

  /* Borders
   *
   * The following variables, specify the visual styling of borders in JupyterLab.
   */

  --jp-border-width: 1px;
  --jp-border-color0: var(--md-grey-400);
  --jp-border-color1: var(--md-grey-400);
  --jp-border-color2: var(--md-grey-300);
  --jp-border-color3: var(--md-grey-200);
  --jp-inverse-border-color: var(--md-grey-600);
  --jp-border-radius: 2px;

  /* UI Fonts
   *
   * The UI font CSS variables are used for the typography all of the JupyterLab
   * user interface elements that are not directly user generated content.
   *
   * The font sizing here is done assuming that the body font size of --jp-ui-font-size1
   * is applied to a parent element. When children elements, such as headings, are sized
   * in em all things will be computed relative to that body size.
   */

  --jp-ui-font-scale-factor: 1.2;
  --jp-ui-font-size0: 0.83333em;
  --jp-ui-font-size1: 13px; /* Base font size */
  --jp-ui-font-size2: 1.2em;
  --jp-ui-font-size3: 1.44em;
  --jp-ui-font-family: system-ui, -apple-system, blinkmacsystemfont, 'Segoe UI',
    helvetica, arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol';

  /*
   * Use these font colors against the corresponding main layout colors.
   * In a light theme, these go from dark to light.
   */

  /* Defaults use Material Design specification */
  --jp-ui-font-color0: rgba(0, 0, 0, 1);
  --jp-ui-font-color1: rgba(0, 0, 0, 0.87);
  --jp-ui-font-color2: rgba(0, 0, 0, 0.54);
  --jp-ui-font-color3: rgba(0, 0, 0, 0.38);

  /*
   * Use these against the brand/accent/warn/error colors.
   * These will typically go from light to darker, in both a dark and light theme.
   */

  --jp-ui-inverse-font-color0: rgba(255, 255, 255, 1);
  --jp-ui-inverse-font-color1: rgba(255, 255, 255, 1);
  --jp-ui-inverse-font-color2: rgba(255, 255, 255, 0.7);
  --jp-ui-inverse-font-color3: rgba(255, 255, 255, 0.5);

  /* Content Fonts
   *
   * Content font variables are used for typography of user generated content.
   *
   * The font sizing here is done assuming that the body font size of --jp-content-font-size1
   * is applied to a parent element. When children elements, such as headings, are sized
   * in em all things will be computed relative to that body size.
   */

  --jp-content-line-height: 1.6;
  --jp-content-font-scale-factor: 1.2;
  --jp-content-font-size0: 0.83333em;
  --jp-content-font-size1: 14px; /* Base font size */
  --jp-content-font-size2: 1.2em;
  --jp-content-font-size3: 1.44em;
  --jp-content-font-size4: 1.728em;
  --jp-content-font-size5: 2.0736em;

  /* This gives a magnification of about 125% in presentation mode over normal. */
  --jp-content-presentation-font-size1: 17px;
  --jp-content-heading-line-height: 1;
  --jp-content-heading-margin-top: 1.2em;
  --jp-content-heading-margin-bottom: 0.8em;
  --jp-content-heading-font-weight: 500;

  /* Defaults use Material Design specification */
  --jp-content-font-color0: rgba(0, 0, 0, 1);
  --jp-content-font-color1: rgba(0, 0, 0, 0.87);
  --jp-content-font-color2: rgba(0, 0, 0, 0.54);
  --jp-content-font-color3: rgba(0, 0, 0, 0.38);
  --jp-content-link-color: var(--md-blue-900);
  --jp-content-font-family: system-ui, -apple-system, blinkmacsystemfont,
    'Segoe UI', helvetica, arial, sans-serif, 'Apple Color Emoji',
    'Segoe UI Emoji', 'Segoe UI Symbol';

  /*
   * Code Fonts
   *
   * Code font variables are used for typography of code and other monospaces content.
   */

  --jp-code-font-size: 13px;
  --jp-code-line-height: 1.3077; /* 17px for 13px base */
  --jp-code-padding: 5px; /* 5px for 13px base, codemirror highlighting needs integer px value */
  --jp-code-font-family-default: menlo, consolas, 'DejaVu Sans Mono', monospace;
  --jp-code-font-family: var(--jp-code-font-family-default);

  /* This gives a magnification of about 125% in presentation mode over normal. */
  --jp-code-presentation-font-size: 16px;

  /* may need to tweak cursor width if you change font size */
  --jp-code-cursor-width0: 1.4px;
  --jp-code-cursor-width1: 2px;
  --jp-code-cursor-width2: 4px;

  /* Layout
   *
   * The following are the main layout colors use in JupyterLab. In a light
   * theme these would go from light to dark.
   */

  --jp-layout-color0: white;
  --jp-layout-color1: white;
  --jp-layout-color2: var(--md-grey-200);
  --jp-layout-color3: var(--md-grey-400);
  --jp-layout-color4: var(--md-grey-600);

  /* Inverse Layout
   *
   * The following are the inverse layout colors use in JupyterLab. In a light
   * theme these would go from dark to light.
   */

  --jp-inverse-layout-color0: #111;
  --jp-inverse-layout-color1: var(--md-grey-900);
  --jp-inverse-layout-color2: var(--md-grey-800);
  --jp-inverse-layout-color3: var(--md-grey-700);
  --jp-inverse-layout-color4: var(--md-grey-600);

  /* Brand/accent */

  --jp-brand-color0: var(--md-blue-900);
  --jp-brand-color1: var(--md-blue-700);
  --jp-brand-color2: var(--md-blue-300);
  --jp-brand-color3: var(--md-blue-100);
  --jp-brand-color4: var(--md-blue-50);
  --jp-accent-color0: var(--md-green-900);
  --jp-accent-color1: var(--md-green-700);
  --jp-accent-color2: var(--md-green-300);
  --jp-accent-color3: var(--md-green-100);

  /* State colors (warn, error, success, info) */

  --jp-warn-color0: var(--md-orange-900);
  --jp-warn-color1: var(--md-orange-700);
  --jp-warn-color2: var(--md-orange-300);
  --jp-warn-color3: var(--md-orange-100);
  --jp-error-color0: var(--md-red-900);
  --jp-error-color1: var(--md-red-700);
  --jp-error-color2: var(--md-red-300);
  --jp-error-color3: var(--md-red-100);
  --jp-success-color0: var(--md-green-900);
  --jp-success-color1: var(--md-green-700);
  --jp-success-color2: var(--md-green-300);
  --jp-success-color3: var(--md-green-100);
  --jp-info-color0: var(--md-cyan-900);
  --jp-info-color1: var(--md-cyan-700);
  --jp-info-color2: var(--md-cyan-300);
  --jp-info-color3: var(--md-cyan-100);

  /* Cell specific styles */

  --jp-cell-padding: 5px;
  --jp-cell-collapser-width: 8px;
  --jp-cell-collapser-min-height: 20px;
  --jp-cell-collapser-not-active-hover-opacity: 0.6;
  --jp-cell-editor-background: var(--md-grey-100);
  --jp-cell-editor-border-color: var(--md-grey-300);
  --jp-cell-editor-box-shadow: inset 0 0 2px var(--md-blue-300);
  --jp-cell-editor-active-background: var(--jp-layout-color0);
  --jp-cell-editor-active-border-color: var(--jp-brand-color1);
  --jp-cell-prompt-width: 64px;
  --jp-cell-prompt-font-family: var(--jp-code-font-family-default);
  --jp-cell-prompt-letter-spacing: 0;
  --jp-cell-prompt-opacity: 1;
  --jp-cell-prompt-not-active-opacity: 0.5;
  --jp-cell-prompt-not-active-font-color: var(--md-grey-700);

  /* A custom blend of MD grey and blue 600
   * See https://meyerweb.com/eric/tools/color-blend/#546E7A:1E88E5:5:hex */
  --jp-cell-inprompt-font-color: #307fc1;

  /* A custom blend of MD grey and orange 600
   * https://meyerweb.com/eric/tools/color-blend/#546E7A:F4511E:5:hex */
  --jp-cell-outprompt-font-color: #bf5b3d;

  /* Notebook specific styles */

  --jp-notebook-padding: 10px;
  --jp-notebook-select-background: var(--jp-layout-color1);
  --jp-notebook-multiselected-color: var(--md-blue-50);

  /* The scroll padding is calculated to fill enough space at the bottom of the
  notebook to show one single-line cell (with appropriate padding) at the top
  when the notebook is scrolled all the way to the bottom. We also subtract one
  pixel so that no scrollbar appears if we have just one single-line cell in the
  notebook. This padding is to enable a 'scroll past end' feature in a notebook.
  */
  --jp-notebook-scroll-padding: calc(
    100% - var(--jp-code-font-size) * var(--jp-code-line-height) -
      var(--jp-code-padding) - var(--jp-cell-padding) - 1px
  );

  /* Rendermime styles */

  --jp-rendermime-error-background: #fdd;
  --jp-rendermime-table-row-background: var(--md-grey-100);
  --jp-rendermime-table-row-hover-background: var(--md-light-blue-50);

  /* Dialog specific styles */

  --jp-dialog-background: rgba(0, 0, 0, 0.25);

  /* Console specific styles */

  --jp-console-padding: 10px;

  /* Toolbar specific styles */

  --jp-toolbar-border-color: var(--jp-border-color1);
  --jp-toolbar-micro-height: 8px;
  --jp-toolbar-background: var(--jp-layout-color1);
  --jp-toolbar-box-shadow: 0 0 2px 0 rgba(0, 0, 0, 0.24);
  --jp-toolbar-header-margin: 4px 4px 0 4px;
  --jp-toolbar-active-background: var(--md-grey-300);

  /* Statusbar specific styles */

  --jp-statusbar-height: 24px;

  /* Input field styles */

  --jp-input-box-shadow: inset 0 0 2px var(--md-blue-300);
  --jp-input-active-background: var(--jp-layout-color1);
  --jp-input-hover-background: var(--jp-layout-color1);
  --jp-input-background: var(--md-grey-100);
  --jp-input-border-color: var(--jp-inverse-border-color);
  --jp-input-active-border-color: var(--jp-brand-color1);
  --jp-input-active-box-shadow-color: rgba(19, 124, 189, 0.3);

  /* General editor styles */

  --jp-editor-selected-background: #d9d9d9;
  --jp-editor-selected-focused-background: #d7d4f0;
  --jp-editor-cursor-color: var(--jp-ui-font-color0);

  /* Code mirror specific styles */

  --jp-mirror-editor-keyword-color: #008000;
  --jp-mirror-editor-atom-color: #88f;
  --jp-mirror-editor-number-color: #080;
  --jp-mirror-editor-def-color: #00f;
  --jp-mirror-editor-variable-color: var(--md-grey-900);
  --jp-mirror-editor-variable-2-color: rgb(0, 54, 109);
  --jp-mirror-editor-variable-3-color: #085;
  --jp-mirror-editor-punctuation-color: #05a;
  --jp-mirror-editor-property-color: #05a;
  --jp-mirror-editor-operator-color: #a2f;
  --jp-mirror-editor-comment-color: #408080;
  --jp-mirror-editor-string-color: #ba2121;
  --jp-mirror-editor-string-2-color: #708;
  --jp-mirror-editor-meta-color: #a2f;
  --jp-mirror-editor-qualifier-color: #555;
  --jp-mirror-editor-builtin-color: #008000;
  --jp-mirror-editor-bracket-color: #997;
  --jp-mirror-editor-tag-color: #170;
  --jp-mirror-editor-attribute-color: #00c;
  --jp-mirror-editor-header-color: blue;
  --jp-mirror-editor-quote-color: #090;
  --jp-mirror-editor-link-color: #00c;
  --jp-mirror-editor-error-color: #f00;
  --jp-mirror-editor-hr-color: #999;

  /*
    RTC user specific colors.
    These colors are used for the cursor, username in the editor,
    and the icon of the user.
  */

  --jp-collaborator-color1: #ffad8e;
  --jp-collaborator-color2: #dac83d;
  --jp-collaborator-color3: #72dd76;
  --jp-collaborator-color4: #00e4d0;
  --jp-collaborator-color5: #45d4ff;
  --jp-collaborator-color6: #e2b1ff;
  --jp-collaborator-color7: #ff9de6;

  /* Vega extension styles */

  --jp-vega-background: white;

  /* Sidebar-related styles */

  --jp-sidebar-min-width: 250px;

  /* Search-related styles */

  --jp-search-toggle-off-opacity: 0.5;
  --jp-search-toggle-hover-opacity: 0.8;
  --jp-search-toggle-on-opacity: 1;
  --jp-search-selected-match-background-color: rgb(245, 200, 0);
  --jp-search-selected-match-color: black;
  --jp-search-unselected-match-background-color: var(
    --jp-inverse-layout-color0
  );
  --jp-search-unselected-match-color: var(--jp-ui-inverse-font-color0);

  /* Icon colors that work well with light or dark backgrounds */
  --jp-icon-contrast-color0: var(--md-purple-600);
  --jp-icon-contrast-color1: var(--md-green-600);
  --jp-icon-contrast-color2: var(--md-pink-600);
  --jp-icon-contrast-color3: var(--md-blue-600);

  /* Button colors */
  --jp-accept-color-normal: var(--md-blue-700);
  --jp-accept-color-hover: var(--md-blue-800);
  --jp-accept-color-active: var(--md-blue-900);
  --jp-warn-color-normal: var(--md-red-700);
  --jp-warn-color-hover: var(--md-red-800);
  --jp-warn-color-active: var(--md-red-900);
  --jp-reject-color-normal: var(--md-grey-600);
  --jp-reject-color-hover: var(--md-grey-700);
  --jp-reject-color-active: var(--md-grey-800);

  /* File or activity icons and switch semantic variables */
  --jp-jupyter-icon-color: #f37626;
  --jp-notebook-icon-color: #f37626;
  --jp-json-icon-color: var(--md-orange-700);
  --jp-console-icon-background-color: var(--md-blue-700);
  --jp-console-icon-color: white;
  --jp-terminal-icon-background-color: var(--md-grey-800);
  --jp-terminal-icon-color: var(--md-grey-200);
  --jp-text-editor-icon-color: var(--md-grey-700);
  --jp-inspector-icon-color: var(--md-grey-700);
  --jp-switch-color: var(--md-grey-400);
  --jp-switch-true-position-color: var(--md-orange-900);
}
</style>
<style type="text/css">
/* Force rendering true colors when outputing to pdf */
* {
  -webkit-print-color-adjust: exact;
}

/* Misc */
a.anchor-link {
  display: none;
}

/* Input area styling */
.jp-InputArea {
  overflow: hidden;
}

.jp-InputArea-editor {
  overflow: hidden;
}

.cm-editor.cm-s-jupyter .highlight pre {
/* weird, but --jp-code-padding defined to be 5px but 4px horizontal padding is hardcoded for pre.cm-line */
  padding: var(--jp-code-padding) 4px;
  margin: 0;

  font-family: inherit;
  font-size: inherit;
  line-height: inherit;
  color: inherit;

}

.jp-OutputArea-output pre {
  line-height: inherit;
  font-family: inherit;
}

.jp-RenderedText pre {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-code-font-size);
}

/* Hiding the collapser by default */
.jp-Collapser {
  display: none;
}

@page {
    margin: 0.5in; /* Margin for each printed piece of paper */
}

@media print {
  .jp-Cell-inputWrapper,
  .jp-Cell-outputWrapper {
    display: block;
  }
}
</style>
<!-- Load mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS_CHTML-full,Safe"> </script>
<!-- MathJax configuration -->
<script type="text/x-mathjax-config">
    init_mathjax = function() {
        if (window.MathJax) {
        // MathJax loaded
            MathJax.Hub.Config({
                TeX: {
                    equationNumbers: {
                    autoNumber: "AMS",
                    useLabelIds: true
                    }
                },
                tex2jax: {
                    inlineMath: [ ['$','$'], ["\\(","\\)"] ],
                    displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
                    processEscapes: true,
                    processEnvironments: true
                },
                displayAlign: 'center',
                messageStyle: 'none',
                CommonHTML: {
                    linebreaks: {
                    automatic: true
                    }
                }
            });

            MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
        }
    }
    init_mathjax();
    </script>
<!-- End of mathjax configuration --><script type="module">
  document.addEventListener("DOMContentLoaded", async () => {
    const diagrams = document.querySelectorAll(".jp-Mermaid > pre.mermaid");
    // do not load mermaidjs if not needed
    if (!diagrams.length) {
      return;
    }
    const mermaid = (await import("https://cdnjs.cloudflare.com/ajax/libs/mermaid/10.7.0/mermaid.esm.min.mjs")).default;
    const parser = new DOMParser();

    mermaid.initialize({
      maxTextSize: 100000,
      maxEdges: 100000,
      startOnLoad: false,
      fontFamily: window
        .getComputedStyle(document.body)
        .getPropertyValue("--jp-ui-font-family"),
      theme: document.querySelector("body[data-jp-theme-light='true']")
        ? "default"
        : "dark",
    });

    let _nextMermaidId = 0;

    function makeMermaidImage(svg) {
      const img = document.createElement("img");
      const doc = parser.parseFromString(svg, "image/svg+xml");
      const svgEl = doc.querySelector("svg");
      const { maxWidth } = svgEl?.style || {};
      const firstTitle = doc.querySelector("title");
      const firstDesc = doc.querySelector("desc");

      img.setAttribute("src", `data:image/svg+xml,${encodeURIComponent(svg)}`);
      if (maxWidth) {
        img.width = parseInt(maxWidth);
      }
      if (firstTitle) {
        img.setAttribute("alt", firstTitle.textContent);
      }
      if (firstDesc) {
        const caption = document.createElement("figcaption");
        caption.className = "sr-only";
        caption.textContent = firstDesc.textContent;
        return [img, caption];
      }
      return [img];
    }

    async function makeMermaidError(text) {
      let errorMessage = "";
      try {
        await mermaid.parse(text);
      } catch (err) {
        errorMessage = `${err}`;
      }

      const result = document.createElement("details");
      result.className = 'jp-RenderedMermaid-Details';
      const summary = document.createElement("summary");
      summary.className = 'jp-RenderedMermaid-Summary';
      const pre = document.createElement("pre");
      const code = document.createElement("code");
      code.innerText = text;
      pre.appendChild(code);
      summary.appendChild(pre);
      result.appendChild(summary);

      const warning = document.createElement("pre");
      warning.innerText = errorMessage;
      result.appendChild(warning);
      return [result];
    }

    async function renderOneMarmaid(src) {
      const id = `jp-mermaid-${_nextMermaidId++}`;
      const parent = src.parentNode;
      let raw = src.textContent.trim();
      const el = document.createElement("div");
      el.style.visibility = "hidden";
      document.body.appendChild(el);
      let results = null;
      let output = null;
      try {
        let { svg } = await mermaid.render(id, raw, el);
        svg = cleanMermaidSvg(svg);
        results = makeMermaidImage(svg);
        output = document.createElement("figure");
        results.map(output.appendChild, output);
      } catch (err) {
        parent.classList.add("jp-mod-warning");
        results = await makeMermaidError(raw);
        output = results[0];
      } finally {
        el.remove();
      }
      parent.classList.add("jp-RenderedMermaid");
      parent.appendChild(output);
    }


    /**
     * Post-process to ensure mermaid diagrams contain only valid SVG and XHTML.
     */
    function cleanMermaidSvg(svg) {
      return svg.replace(RE_VOID_ELEMENT, replaceVoidElement);
    }


    /**
     * A regular expression for all void elements, which may include attributes and
     * a slash.
     *
     * @see https://developer.mozilla.org/en-US/docs/Glossary/Void_element
     *
     * Of these, only `<br>` is generated by Mermaid in place of `\n`,
     * but _any_ "malformed" tag will break the SVG rendering entirely.
     */
    const RE_VOID_ELEMENT =
      /<\s*(area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)\s*([^>]*?)\s*>/gi;

    /**
     * Ensure a void element is closed with a slash, preserving any attributes.
     */
    function replaceVoidElement(match, tag, rest) {
      rest = rest.trim();
      if (!rest.endsWith('/')) {
        rest = `${rest} /`;
      }
      return `<${tag} ${rest}>`;
    }

    void Promise.all([...diagrams].map(renderOneMarmaid));
  });
</script>
<style>
  .jp-Mermaid:not(.jp-RenderedMermaid) {
    display: none;
  }

  .jp-RenderedMermaid {
    overflow: auto;
    display: flex;
  }

  .jp-RenderedMermaid.jp-mod-warning {
    width: auto;
    padding: 0.5em;
    margin-top: 0.5em;
    border: var(--jp-border-width) solid var(--jp-warn-color2);
    border-radius: var(--jp-border-radius);
    color: var(--jp-ui-font-color1);
    font-size: var(--jp-ui-font-size1);
    white-space: pre-wrap;
    word-wrap: break-word;
  }

  .jp-RenderedMermaid figure {
    margin: 0;
    overflow: auto;
    max-width: 100%;
  }

  .jp-RenderedMermaid img {
    max-width: 100%;
  }

  .jp-RenderedMermaid-Details > pre {
    margin-top: 1em;
  }

  .jp-RenderedMermaid-Summary {
    color: var(--jp-warn-color2);
  }

  .jp-RenderedMermaid:not(.jp-mod-warning) pre {
    display: none;
  }

  .jp-RenderedMermaid-Summary > pre {
    display: inline-block;
    white-space: normal;
  }
</style>
<!-- End of mermaid configuration --></head>
<body class="jp-Notebook" data-jp-theme-light="true" data-jp-theme-name="JupyterLab Light">
<main><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=e393c336">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[1]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="kn">import</span><span class="w"> </span><span class="nn">prody</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">pr</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">py3Dmol</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">tempfile</span>
<span class="c1"># --- notebook parameters (non-interactive) ---</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">os</span><span class="o">,</span><span class="w"> </span><span class="nn">sys</span><span class="o">,</span><span class="w"> </span><span class="nn">asyncio</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">pathlib</span><span class="w"> </span><span class="kn">import</span> <span class="n">Path</span>

<span class="kn">import</span><span class="w"> </span><span class="nn">pandas</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">pd</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Analysis.PlotUtils</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">scipy.ndimage</span><span class="w"> </span><span class="kn">import</span> <span class="n">binary_dilation</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">matplotlib.patches</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">mpatches</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">IPython.display</span><span class="w"> </span><span class="kn">import</span> <span class="n">display</span><span class="p">,</span> <span class="n">HTML</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDB</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">pairwise2</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio.PDB</span><span class="w"> </span><span class="kn">import</span> <span class="n">Superimposer</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">py3Dmol</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">ipywidgets</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">widgets</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio.PDB</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDBParser</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">pairwise2</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">py3Dmol</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDB</span><span class="p">,</span> <span class="n">Align</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">Bio.PDB</span><span class="w"> </span><span class="kn">import</span> <span class="n">PDBParser</span><span class="p">,</span> <span class="n">PDBIO</span>
<span class="c1"># from Bio.PDB.Polypeptide import protein_letters_3to1</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">io</span><span class="w"> </span><span class="kn">import</span> <span class="n">StringIO</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">numpy</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">np</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">matplotlib.pyplot</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">plt</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">matplotlib.colors</span><span class="w"> </span><span class="kn">import</span> <span class="n">ListedColormap</span><span class="p">,</span> <span class="n">BoundaryNorm</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">matplotlib.patches</span><span class="w"> </span><span class="kn">import</span> <span class="n">Patch</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">config</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">utils.utils</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">utils.protein_utils</span><span class="w"> </span><span class="kn">import</span> <span class="n">extract_protein_sequence</span>
<span class="kn">from</span><span class="w"> </span><span class="nn">utils.protein_plot_utils</span><span class="w"> </span><span class="kn">import</span> <span class="o">*</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>CONFIG FOLD-SWITCH PIEPLINE WITH USER: orzuk  ENVIRONMENT: Linux
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>PyRosetta not installed; skipping energy/G panels
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=8069b923">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[2]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="c1"># Silence Windows asyncio warning from zmq/tornado</span>
<span class="k">if</span> <span class="n">sys</span><span class="o">.</span><span class="n">platform</span><span class="o">.</span><span class="n">startswith</span><span class="p">(</span><span class="s2">"win"</span><span class="p">):</span>
    <span class="n">asyncio</span><span class="o">.</span><span class="n">set_event_loop_policy</span><span class="p">(</span><span class="n">asyncio</span><span class="o">.</span><span class="n">WindowsSelectorEventLoopPolicy</span><span class="p">())</span>

<span class="c1"># Get pair from environment (set by generate_notebooks.py)</span>
<span class="n">fold_pair</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">getenv</span><span class="p">(</span><span class="s2">"PAIR_ID"</span><span class="p">,</span> <span class="s2">""</span><span class="p">)</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span>

<span class="k">if</span> <span class="ow">not</span> <span class="n">fold_pair</span><span class="p">:</span>
    <span class="c1"># We are likely running interactively in Jupyter UI (not nbconvert).</span>
    <span class="c1"># Fall back to input() ONLY when stdin is available.</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">fold_pair</span> <span class="o">=</span> <span class="nb">input</span><span class="p">(</span><span class="s2">"Enter fold-switch pair (e.g., 3hdfA_3hdeA): "</span><span class="p">)</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span>
    <span class="k">except</span> <span class="ne">Exception</span><span class="p">:</span>
        <span class="k">raise</span> <span class="ne">RuntimeError</span><span class="p">(</span>
            <span class="s2">"PAIR_ID not provided and no stdin available. "</span>
            <span class="s2">"When running via generate_notebooks.py, PAIR_ID is set automatically."</span>
        <span class="p">)</span>

<span class="c1"># Normalize and split into the two PDB+chain ids</span>
<span class="k">if</span> <span class="s2">"_"</span> <span class="ow">not</span> <span class="ow">in</span> <span class="n">fold_pair</span><span class="p">:</span>
    <span class="k">raise</span> <span class="ne">ValueError</span><span class="p">(</span><span class="sa">f</span><span class="s2">"PAIR_ID must look like 'A_B', got: </span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="n">pdb1</span><span class="p">,</span> <span class="n">pdb2</span> <span class="o">=</span> <span class="n">fold_pair</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s2">"_"</span><span class="p">,</span> <span class="mi">1</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"Using pair:"</span><span class="p">,</span> <span class="n">pdb1</span><span class="p">,</span> <span class="n">pdb2</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>Using pair: 1jfkA 2nxqB
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=cf01df3c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[3]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">plot_tool</span> <span class="o">=</span> <span class="n">PlotTool</span><span class="p">(</span><span class="n">folder</span><span class="o">=</span><span class="n">DATA_DIR</span><span class="p">,</span> <span class="n">fold_pair</span><span class="o">=</span><span class="n">fold_pair</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 2096 atoms and 1 coordinate set(s) were parsed in 0.02s.
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 1068 atoms and 1 coordinate set(s) were parsed in 0.01s.
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=80d1165b">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[4]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">VIZ_FOLDER</span>    <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">DATA_DIR</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_cmap_esm/VizCmaps'</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"Fold-pair,"</span><span class="p">,</span> <span class="n">fold_pair</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">chains</span> <span class="o">=</span> <span class="p">(</span><span class="n">pdb1</span><span class="p">[</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="o">.</span><span class="n">upper</span><span class="p">(),</span> <span class="n">pdb2</span><span class="p">[</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="o">.</span><span class="n">upper</span><span class="p">())</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"chains,"</span><span class="p">,</span> <span class="n">chains</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="n">pdb_path1</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">DATA_DIR</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">pdb1</span><span class="p">[:</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="si">}</span><span class="s2">.pdb"</span>
<span class="n">pdb_path2</span> <span class="o">=</span> <span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">DATA_DIR</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">pdb2</span><span class="p">[:</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span><span class="si">}</span><span class="s2">.pdb"</span>
<span class="c1">#fold_pair_subdir = f"{fold_pair[0]}_{fold_pair[1]}"</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>Fold-pair, 1jfkA_2nxqB
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>chains, ('A', 'B')
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=d913aca8">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[5]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">seq_fold1</span> <span class="o">=</span> <span class="n">extract_protein_sequence</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span> <span class="n">chain</span><span class="o">=</span><span class="n">chains</span><span class="p">[</span><span class="mi">0</span><span class="p">],</span> <span class="n">ca_only</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
<span class="n">seq_fold2</span> <span class="o">=</span> <span class="n">extract_protein_sequence</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">,</span> <span class="n">chain</span><span class="o">=</span><span class="n">chains</span><span class="p">[</span><span class="mi">1</span><span class="p">],</span> <span class="n">ca_only</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=7f244dee">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h1 id="Visualizations-and-results">Visualizations and results<a class="anchor-link" href="#Visualizations-and-results"></a></h1>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=0dae88ab">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="Original-Structure-Visualization">Original Structure Visualization<a class="anchor-link" href="#Original-Structure-Visualization"></a></h2>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=e241ff25">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[6]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">pdb_file1</span> <span class="o">=</span> <span class="n">read_pdb_file</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">)</span>
<span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_single_fold</span><span class="p">(</span><span class="n">pdb_file1</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">fold1</span><span class="p">)</span>

<span class="n">pdb_file2</span> <span class="o">=</span> <span class="n">read_pdb_file</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">)</span>
<span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_single_fold</span><span class="p">(</span><span class="n">pdb_file2</span><span class="p">,</span><span class="n">label</span><span class="o">=</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">fold2</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div id="3dmolviewer_17581787013555787" style="position: relative; width: 400px; height: 300px;">
<p id="3dmolwarning_17581787013555787" style="background-color:#ffcccc;color:black">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br/></p>
</div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');
}

var viewer_17581787013555787 = null;
var warn = document.getElementById("3dmolwarning_17581787013555787");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_17581787013555787 = $3Dmol.createViewer(document.getElementById("3dmolviewer_17581787013555787"),{backgroundColor:"white"});
viewer_17581787013555787.zoomTo();
	viewer_17581787013555787.addModel("HEADER    METAL BINDING PROTEIN                   21-JUN-01   1JFK              \nTITLE     MINIMUM ENERGY REPRESENTATIVE STRUCTURE OF A CALCIUM BOUND EF-HAND    \nTITLE    2 PROTEIN FROM ENTAMOEBA HISTOLYTICA                                   \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: CALCIUM-BINDING PROTEIN;                                   \nCOMPND   3 CHAIN: A;                                                            \nCOMPND   4 SYNONYM: EHCABP;                                                     \nCOMPND   5 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 ORGANISM_SCIENTIFIC: ENTAMOEBA HISTOLYTICA;                          \nSOURCE   3 ORGANISM_TAXID: 5759;                                                \nSOURCE   4 EXPRESSION_SYSTEM: ESCHERICHIA COLI BL21(DE3);                       \nSOURCE   5 EXPRESSION_SYSTEM_TAXID: 469008;                                     \nSOURCE   6 EXPRESSION_SYSTEM_STRAIN: BL21(DE3);                                 \nSOURCE   7 EXPRESSION_SYSTEM_VECTOR_TYPE: PLASMID;                              \nSOURCE   8 EXPRESSION_SYSTEM_PLASMID: PET-3C                                    \nKEYWDS    EF-HAND, HELIX-LOOP-HELIX, CALCIUM BINDING PROTEIN, ENTAMOEBA         \nKEYWDS   2 HISTOLYTICA, METAL BINDING PROTEIN                                   \nEXPDTA    SOLUTION NMR                                                          \nAUTHOR    H.S.ATREYA,S.C.SAHU,A.BHATTACHARYA,K.V.R.CHARY,G.GOVIL                \nREVDAT   6   29-MAY-24 1JFK    1       REMARK                                   \nREVDAT   5   23-FEB-22 1JFK    1       REMARK SEQADV LINK                       \nREVDAT   4   24-FEB-09 1JFK    1       VERSN                                    \nREVDAT   3   01-APR-03 1JFK    1       JRNL                                     \nREVDAT   2   25-DEC-02 1JFK    1       REMARK                                   \nREVDAT   1   19-DEC-01 1JFK    0                                                \nJRNL        AUTH   H.S.ATREYA,S.C.SAHU,A.BHATTACHARYA,K.V.CHARY,G.GOVIL         \nJRNL        TITL   NMR DERIVED SOLUTION STRUCTURE OF AN EF-HAND CALCIUM-BINDING \nJRNL        TITL 2 PROTEIN FROM ENTAMOEBA HISTOLYTICA.                          \nJRNL        REF    BIOCHEMISTRY                  V.  40 14392 2001              \nJRNL        REFN                   ISSN 0006-2960                               \nJRNL        PMID   11724551                                                     \nJRNL        DOI    10.1021/BI0114978                                            \nREMARK   2                                                                      \nREMARK   2 RESOLUTION. NOT APPLICABLE.                                          \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : DYANA 1.5, DYANA 1.5                                 \nREMARK   3   AUTHORS     : GUENTERT, MUMENTHALER AND WUTHRICH (DYANA),          \nREMARK   3                 GUENTERT, MUMENTHALER AND WUTHRICH (DYANA)           \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: 1265 NOE-DERIVED DISTANCE CONSTRAINTS     \nREMARK   3  AND 200 DIHEDRAL ANGLE CONSTRAINTS                                  \nREMARK   4                                                                      \nREMARK   4 1JFK COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY PDBJ ON 26-JUN-01.                  \nREMARK 100 THE DEPOSITION ID IS D_1000013711.                                   \nREMARK 210                                                                      \nREMARK 210 EXPERIMENTAL DETAILS                                                 \nREMARK 210  EXPERIMENT TYPE                : NMR                                \nREMARK 210  TEMPERATURE           (KELVIN) : 308                                \nREMARK 210  PH                             : 6.0                                \nREMARK 210  IONIC STRENGTH                 : 30MM CACL2                         \nREMARK 210  PRESSURE                       : 1 ATM                              \nREMARK 210  SAMPLE CONTENTS                : 3MM EHCABP, 30MM DEUTERATED TRIS   \nREMARK 210                                   BUFFER, 30MM CACL2; 3MM EHCABP,    \nREMARK 210                                   30MM DEUTERATED TRIS BUFFER,       \nREMARK 210                                   30MM CACL2                         \nREMARK 210                                                                      \nREMARK 210  NMR EXPERIMENTS CONDUCTED      : 3D_15N-SEPARATED_NOESY; HNHA; 2D   \nREMARK 210                                   NOESY; 3D_13C-SEPARATED_NOESY      \nREMARK 210  SPECTROMETER FIELD STRENGTH    : 600 MHZ                            \nREMARK 210  SPECTROMETER MODEL             : UNITYPLUS                          \nREMARK 210  SPECTROMETER MANUFACTURER      : VARIAN                             \nREMARK 210                                                                      \nREMARK 210  STRUCTURE DETERMINATION.                                            \nREMARK 210   SOFTWARE USED                 : NULL                               \nREMARK 210   METHOD USED                   : TORSION ANGLE DYNAMICS             \nREMARK 210                                                                      \nREMARK 210 CONFORMERS, NUMBER CALCULATED   : NULL                               \nREMARK 210 CONFORMERS, NUMBER SUBMITTED    : 1                                  \nREMARK 210 CONFORMERS, SELECTION CRITERIA  : NULL                               \nREMARK 210                                                                      \nREMARK 210 BEST REPRESENTATIVE CONFORMER IN THIS ENSEMBLE : 1                   \nREMARK 210                                                                      \nREMARK 210 REMARK: THE STRUCTURE WAS DETERMINED USING TRIPLE RESONANCE NMR      \nREMARK 210  SPECTROSCOPY                                                        \nREMARK 215                                                                      \nREMARK 215 NMR STUDY                                                            \nREMARK 215 THE COORDINATES IN THIS ENTRY WERE GENERATED FROM SOLUTION           \nREMARK 215 NMR DATA.  PROTEIN DATA BANK CONVENTIONS REQUIRE THAT                \nREMARK 215 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON              \nREMARK 215 THESE RECORDS ARE MEANINGLESS.                                       \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1                                                       \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \nREMARK 500                                                                      \nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \nREMARK 500                                                                      \nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \nREMARK 500   O    PHE A    42     H    ASP A    46              1.49            \nREMARK 500   O    GLU A   107     H    ALA A   110              1.55            \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    LYS A  29      -79.65    -78.07                                   \nREMARK 500    ALA A  31       54.13   -141.02                                   \nREMARK 500    ILE A  32       52.95    -94.87                                   \nREMARK 500    SER A  64      -68.53   -178.13                                   \nREMARK 500    ILE A  65      -77.74   -144.11                                   \nREMARK 500    LEU A  70       93.33   -163.44                                   \nREMARK 500    ASP A  85       93.63    -53.76                                   \nREMARK 500    ALA A 118       49.22    -89.82                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 620                                                                      \nREMARK 620 METAL COORDINATION                                                   \nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 147  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A  10   OD1                                                    \nREMARK 620 2 ASN A  12   OD1  72.5                                              \nREMARK 620 3 ASP A  14   OD2  64.8  80.7                                        \nREMARK 620 4 ALA A  16   O   102.8 174.7  95.1                                  \nREMARK 620 5 GLU A  21   OE1 123.5 108.1 169.0  76.4                            \nREMARK 620 6 GLU A  21   OE2  88.9  66.6 143.2 116.3  47.6                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 159  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A  46   OD1                                                    \nREMARK 620 2 ASP A  48   OD2  71.4                                              \nREMARK 620 3 ASP A  48   OD1  50.3  39.5                                        \nREMARK 620 4 ASN A  50   ND2 103.3  86.0  65.9                                  \nREMARK 620 5 ASN A  50   OD1  63.5  86.5  48.5  42.1                            \nREMARK 620 6 GLU A  52   O   104.1 134.9 102.2  50.5  54.6                      \nREMARK 620 7 GLU A  57   OE1 136.0 110.3 149.7 120.8 156.8 103.1                \nREMARK 620 8 GLU A  57   OE2  88.9  97.6 123.5 167.9 149.3 127.4  47.1          \nREMARK 620 N                    1     2     3     4     5     6     7           \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 171  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 LYS A  91   O                                                      \nREMARK 620 2 GLU A  96   OE2 121.9                                              \nREMARK 620 3 GLU A  96   OE1  93.2  47.7                                        \nREMARK 620 N                    1     2                                         \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 183  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A 117   OD2                                                    \nREMARK 620 2 ASN A 119   OD1  75.8                                              \nREMARK 620 3 ASP A 121   OD2  72.4  94.6                                        \nREMARK 620 4 TYR A 123   O    57.8 127.6  51.2                                  \nREMARK 620 5 GLU A 128   OE1 143.9 129.6 121.8 102.9                            \nREMARK 620 6 GLU A 128   OE2 132.7  83.5 152.5 145.9  47.7                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 800                                                                      \nREMARK 800 SITE                                                                 \nREMARK 800 SITE_IDENTIFIER: AC1                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 147                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC2                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 159                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC3                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 171                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC4                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 183                  \nREMARK 900                                                                      \nREMARK 900 RELATED ENTRIES                                                      \nREMARK 900 RELATED ID: 1JFJ   RELATED DB: PDB                                   \nREMARK 900 1JFJ CONTAINS SAME HOLO PROTEIN (THE CALCIUM IONS ARE NOT SHOWN      \nREMARK 900 EXPLICILTY IN THE STRUCTURES).                                       \nREMARK 900 RELATED ID: 4271   RELATED DB: BMRB                                  \nREMARK 900 4271 CONTAINS ASSIGNED CHEMICAL SHIFT LISTS                          \nDBREF  1JFK A    1   134  UNP    P38505   CALBP_ENTHI      1    134             \nSEQADV 1JFK GLU A  131  UNP  P38505    UNK   131 CONFLICT                       \nSEQRES   1 A  134  MET ALA GLU ALA LEU PHE LYS GLU ILE ASP VAL ASN GLY          \nSEQRES   2 A  134  ASP GLY ALA VAL SER TYR GLU GLU VAL LYS ALA PHE VAL          \nSEQRES   3 A  134  SER LYS LYS ARG ALA ILE LYS ASN GLU GLN LEU LEU GLN          \nSEQRES   4 A  134  LEU ILE PHE LYS SER ILE ASP ALA ASP GLY ASN GLY GLU          \nSEQRES   5 A  134  ILE ASP GLN ASN GLU PHE ALA LYS PHE TYR GLY SER ILE          \nSEQRES   6 A  134  GLN GLY GLN ASP LEU SER ASP ASP LYS ILE GLY LEU LYS          \nSEQRES   7 A  134  VAL LEU TYR LYS LEU MET ASP VAL ASP GLY ASP GLY LYS          \nSEQRES   8 A  134  LEU THR LYS GLU GLU VAL THR SER PHE PHE LYS LYS HIS          \nSEQRES   9 A  134  GLY ILE GLU LYS VAL ALA GLU GLN VAL MET LYS ALA ASP          \nSEQRES  10 A  134  ALA ASN GLY ASP GLY TYR ILE THR LEU GLU GLU PHE LEU          \nSEQRES  11 A  134  GLU PHE SER LEU                                              \nHET     CA  A 147       1                                                       \nHET     CA  A 159       1                                                       \nHET     CA  A 171       1                                                       \nHET     CA  A 183       1                                                       \nHETNAM      CA CALCIUM ION                                                      \nFORMUL   2   CA    4(CA 2+)                                                     \nHELIX    1   1 MET A    1  ASP A   10  1                                  10    \nHELIX    2   2 TYR A   19  LYS A   28  1                                  10    \nHELIX    3   3 LYS A   33  ASP A   46  1                                  14    \nHELIX    4   4 GLN A   55  SER A   64  1                                  10    \nHELIX    5   5 LEU A   70  ASP A   85  1                                  16    \nHELIX    6   6 LYS A   94  LYS A  102  1                                   9    \nHELIX    7   7 GLU A  107  ASP A  117  1                                  11    \nHELIX    8   8 LEU A  126  LEU A  134  1                                   9    \nSHEET    1   A 2 ALA A  16  SER A  18  0                                        \nSHEET    2   A 2 GLU A  52  ASP A  54 -1  N  ILE A  53   O  VAL A  17           \nSHEET    1   B 2 LYS A  91  THR A  93  0                                        \nSHEET    2   B 2 TYR A 123  THR A 125 -1  N  ILE A 124   O  LEU A  92           \nLINK         OD1 ASP A  10                CA    CA A 147     1555   1555  2.78  \nLINK         OD1 ASN A  12                CA    CA A 147     1555   1555  2.80  \nLINK         OD2 ASP A  14                CA    CA A 147     1555   1555  3.04  \nLINK         O   ALA A  16                CA    CA A 147     1555   1555  2.79  \nLINK         OE1 GLU A  21                CA    CA A 147     1555   1555  2.74  \nLINK         OE2 GLU A  21                CA    CA A 147     1555   1555  2.78  \nLINK         OD1 ASP A  46                CA    CA A 159     1555   1555  2.69  \nLINK         OD2 ASP A  48                CA    CA A 159     1555   1555  3.40  \nLINK         OD1 ASP A  48                CA    CA A 159     1555   1555  3.16  \nLINK         ND2 ASN A  50                CA    CA A 159     1555   1555  3.32  \nLINK         OD1 ASN A  50                CA    CA A 159     1555   1555  2.81  \nLINK         O   GLU A  52                CA    CA A 159     1555   1555  2.81  \nLINK         OE1 GLU A  57                CA    CA A 159     1555   1555  2.77  \nLINK         OE2 GLU A  57                CA    CA A 159     1555   1555  2.81  \nLINK         O   LYS A  91                CA    CA A 171     1555   1555  2.79  \nLINK         OE2 GLU A  96                CA    CA A 171     1555   1555  2.80  \nLINK         OE1 GLU A  96                CA    CA A 171     1555   1555  2.70  \nLINK         OD2 ASP A 117                CA    CA A 183     1555   1555  3.40  \nLINK         OD1 ASN A 119                CA    CA A 183     1555   1555  2.67  \nLINK         OD2 ASP A 121                CA    CA A 183     1555   1555  3.15  \nLINK         O   TYR A 123                CA    CA A 183     1555   1555  2.69  \nLINK         OE1 GLU A 128                CA    CA A 183     1555   1555  2.72  \nLINK         OE2 GLU A 128                CA    CA A 183     1555   1555  2.79  \nSITE     1 AC1  5 ASP A  10  ASN A  12  ASP A  14  ALA A  16                    \nSITE     2 AC1  5 GLU A  21                                                     \nSITE     1 AC2  6 ASP A  46  ASP A  48  ASN A  50  GLU A  52                    \nSITE     2 AC2  6 ILE A  53  GLU A  57                                          \nSITE     1 AC3  4 ASP A  87  ASP A  89  LYS A  91  GLU A  96                    \nSITE     1 AC4  6 ASP A 117  ASN A 119  ASP A 121  TYR A 123                    \nSITE     2 AC4  6 ILE A 124  GLU A 128                                          \nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      1.000000  0.000000  0.000000        0.00000                         \nSCALE2      0.000000  1.000000  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  1.000000        0.00000                         \nATOM      1  N   MET A   1       1.325   0.000   0.000  1.00  0.00           N  \nATOM      2  CA  MET A   1       2.073   0.000  -1.245  1.00  0.00           C  \nATOM      3  C   MET A   1       3.328  -0.868  -1.131  1.00  0.00           C  \nATOM      4  O   MET A   1       4.435  -0.404  -1.399  1.00  0.00           O  \nATOM      5  CB  MET A   1       2.475   1.433  -1.600  1.00  0.00           C  \nATOM      6  CG  MET A   1       3.179   2.112  -0.424  1.00  0.00           C  \nATOM      7  SD  MET A   1       1.986   2.965   0.593  1.00  0.00           S  \nATOM      8  CE  MET A   1       3.081   3.982   1.569  1.00  0.00           C  \nATOM      9  H   MET A   1       1.884   0.000   0.829  1.00  0.00           H  \nATOM     10  HA  MET A   1       1.399  -0.420  -1.992  1.00  0.00           H  \nATOM     11  HB2 MET A   1       3.133   1.426  -2.469  1.00  0.00           H  \nATOM     12  HB3 MET A   1       1.589   2.005  -1.877  1.00  0.00           H  \nATOM     13  HG2 MET A   1       3.711   1.369   0.170  1.00  0.00           H  \nATOM     14  HG3 MET A   1       3.924   2.817  -0.794  1.00  0.00           H  \nATOM     15  HE1 MET A   1       3.658   3.351   2.244  1.00  0.00           H  \nATOM     16  HE2 MET A   1       3.759   4.523   0.908  1.00  0.00           H  \nATOM     17  HE3 MET A   1       2.494   4.695   2.149  1.00  0.00           H  \nATOM     18  N   ALA A   2       3.113  -2.114  -0.734  1.00  0.00           N  \nATOM     19  CA  ALA A   2       4.212  -3.051  -0.581  1.00  0.00           C  \nATOM     20  C   ALA A   2       4.758  -3.420  -1.962  1.00  0.00           C  \nATOM     21  O   ALA A   2       5.947  -3.251  -2.229  1.00  0.00           O  \nATOM     22  CB  ALA A   2       3.735  -4.275   0.203  1.00  0.00           C  \nATOM     23  H   ALA A   2       2.209  -2.484  -0.518  1.00  0.00           H  \nATOM     24  HA  ALA A   2       4.996  -2.553  -0.011  1.00  0.00           H  \nATOM     25  HB1 ALA A   2       2.650  -4.350   0.134  1.00  0.00           H  \nATOM     26  HB2 ALA A   2       4.189  -5.174  -0.214  1.00  0.00           H  \nATOM     27  HB3 ALA A   2       4.026  -4.173   1.249  1.00  0.00           H  \nATOM     28  N   GLU A   3       3.863  -3.917  -2.803  1.00  0.00           N  \nATOM     29  CA  GLU A   3       4.240  -4.311  -4.150  1.00  0.00           C  \nATOM     30  C   GLU A   3       4.710  -3.093  -4.948  1.00  0.00           C  \nATOM     31  O   GLU A   3       5.740  -3.145  -5.617  1.00  0.00           O  \nATOM     32  CB  GLU A   3       3.082  -5.018  -4.858  1.00  0.00           C  \nATOM     33  CG  GLU A   3       3.229  -6.538  -4.766  1.00  0.00           C  \nATOM     34  CD  GLU A   3       2.743  -7.055  -3.411  1.00  0.00           C  \nATOM     35  OE1 GLU A   3       3.425  -6.750  -2.409  1.00  0.00           O  \nATOM     36  OE2 GLU A   3       1.699  -7.743  -3.407  1.00  0.00           O  \nATOM     37  H   GLU A   3       2.898  -4.051  -2.578  1.00  0.00           H  \nATOM     38  HA  GLU A   3       5.065  -5.013  -4.026  1.00  0.00           H  \nATOM     39  HB2 GLU A   3       2.137  -4.713  -4.411  1.00  0.00           H  \nATOM     40  HB3 GLU A   3       3.052  -4.714  -5.905  1.00  0.00           H  \nATOM     41  HG2 GLU A   3       2.659  -7.011  -5.566  1.00  0.00           H  \nATOM     42  HG3 GLU A   3       4.273  -6.816  -4.912  1.00  0.00           H  \nATOM     43  N   ALA A   4       3.931  -2.026  -4.850  1.00  0.00           N  \nATOM     44  CA  ALA A   4       4.254  -0.797  -5.555  1.00  0.00           C  \nATOM     45  C   ALA A   4       5.717  -0.434  -5.293  1.00  0.00           C  \nATOM     46  O   ALA A   4       6.373   0.168  -6.142  1.00  0.00           O  \nATOM     47  CB  ALA A   4       3.292   0.310  -5.119  1.00  0.00           C  \nATOM     48  H   ALA A   4       3.094  -1.992  -4.304  1.00  0.00           H  \nATOM     49  HA  ALA A   4       4.119  -0.980  -6.621  1.00  0.00           H  \nATOM     50  HB1 ALA A   4       2.979   0.883  -5.991  1.00  0.00           H  \nATOM     51  HB2 ALA A   4       2.418  -0.135  -4.643  1.00  0.00           H  \nATOM     52  HB3 ALA A   4       3.794   0.970  -4.411  1.00  0.00           H  \nATOM     53  N   LEU A   5       6.187  -0.816  -4.114  1.00  0.00           N  \nATOM     54  CA  LEU A   5       7.560  -0.539  -3.730  1.00  0.00           C  \nATOM     55  C   LEU A   5       8.498  -1.478  -4.491  1.00  0.00           C  \nATOM     56  O   LEU A   5       9.494  -1.038  -5.064  1.00  0.00           O  \nATOM     57  CB  LEU A   5       7.717  -0.612  -2.210  1.00  0.00           C  \nATOM     58  CG  LEU A   5       7.831   0.729  -1.482  1.00  0.00           C  \nATOM     59  CD1 LEU A   5       6.478   1.441  -1.431  1.00  0.00           C  \nATOM     60  CD2 LEU A   5       8.435   0.547  -0.089  1.00  0.00           C  \nATOM     61  H   LEU A   5       5.647  -1.306  -3.430  1.00  0.00           H  \nATOM     62  HA  LEU A   5       7.782   0.486  -4.028  1.00  0.00           H  \nATOM     63  HB2 LEU A   5       6.863  -1.153  -1.802  1.00  0.00           H  \nATOM     64  HB3 LEU A   5       8.605  -1.202  -1.984  1.00  0.00           H  \nATOM     65  HG  LEU A   5       8.510   1.368  -2.047  1.00  0.00           H  \nATOM     66 HD11 LEU A   5       5.900   1.188  -2.320  1.00  0.00           H  \nATOM     67 HD12 LEU A   5       5.933   1.124  -0.541  1.00  0.00           H  \nATOM     68 HD13 LEU A   5       6.636   2.519  -1.395  1.00  0.00           H  \nATOM     69 HD21 LEU A   5       7.734   0.911   0.662  1.00  0.00           H  \nATOM     70 HD22 LEU A   5       8.636  -0.511   0.085  1.00  0.00           H  \nATOM     71 HD23 LEU A   5       9.366   1.110  -0.020  1.00  0.00           H  \nATOM     72  N   PHE A   6       8.146  -2.756  -4.473  1.00  0.00           N  \nATOM     73  CA  PHE A   6       8.944  -3.761  -5.154  1.00  0.00           C  \nATOM     74  C   PHE A   6       9.099  -3.425  -6.638  1.00  0.00           C  \nATOM     75  O   PHE A   6      10.201  -3.492  -7.181  1.00  0.00           O  \nATOM     76  CB  PHE A   6       8.198  -5.090  -5.019  1.00  0.00           C  \nATOM     77  CG  PHE A   6       8.466  -6.073  -6.161  1.00  0.00           C  \nATOM     78  CD1 PHE A   6       7.721  -6.014  -7.298  1.00  0.00           C  \nATOM     79  CD2 PHE A   6       9.449  -7.005  -6.040  1.00  0.00           C  \nATOM     80  CE1 PHE A   6       7.970  -6.925  -8.358  1.00  0.00           C  \nATOM     81  CE2 PHE A   6       9.698  -7.916  -7.100  1.00  0.00           C  \nATOM     82  CZ  PHE A   6       8.953  -7.857  -8.236  1.00  0.00           C  \nATOM     83  H   PHE A   6       7.335  -3.106  -4.005  1.00  0.00           H  \nATOM     84  HA  PHE A   6       9.926  -3.770  -4.680  1.00  0.00           H  \nATOM     85  HB2 PHE A   6       8.479  -5.560  -4.077  1.00  0.00           H  \nATOM     86  HB3 PHE A   6       7.128  -4.891  -4.969  1.00  0.00           H  \nATOM     87  HD1 PHE A   6       6.933  -5.267  -7.395  1.00  0.00           H  \nATOM     88  HD2 PHE A   6      10.046  -7.052  -5.129  1.00  0.00           H  \nATOM     89  HE1 PHE A   6       7.373  -6.878  -9.269  1.00  0.00           H  \nATOM     90  HE2 PHE A   6      10.485  -8.663  -7.003  1.00  0.00           H  \nATOM     91  HZ  PHE A   6       9.144  -8.557  -9.050  1.00  0.00           H  \nATOM     92  N   LYS A   7       7.980  -3.071  -7.252  1.00  0.00           N  \nATOM     93  CA  LYS A   7       7.978  -2.725  -8.663  1.00  0.00           C  \nATOM     94  C   LYS A   7       8.676  -1.376  -8.854  1.00  0.00           C  \nATOM     95  O   LYS A   7       9.418  -1.188  -9.816  1.00  0.00           O  \nATOM     96  CB  LYS A   7       6.554  -2.764  -9.223  1.00  0.00           C  \nATOM     97  CG  LYS A   7       5.666  -1.725  -8.535  1.00  0.00           C  \nATOM     98  CD  LYS A   7       5.567  -0.449  -9.372  1.00  0.00           C  \nATOM     99  CE  LYS A   7       4.376   0.405  -8.933  1.00  0.00           C  \nATOM    100  NZ  LYS A   7       4.344   1.678  -9.686  1.00  0.00           N  \nATOM    101  H   LYS A   7       7.088  -3.020  -6.803  1.00  0.00           H  \nATOM    102  HA  LYS A   7       8.552  -3.488  -9.188  1.00  0.00           H  \nATOM    103  HB2 LYS A   7       6.576  -2.577 -10.296  1.00  0.00           H  \nATOM    104  HB3 LYS A   7       6.132  -3.759  -9.082  1.00  0.00           H  \nATOM    105  HG2 LYS A   7       4.670  -2.140  -8.379  1.00  0.00           H  \nATOM    106  HG3 LYS A   7       6.071  -1.489  -7.551  1.00  0.00           H  \nATOM    107  HD2 LYS A   7       6.487   0.127  -9.274  1.00  0.00           H  \nATOM    108  HD3 LYS A   7       5.464  -0.707 -10.426  1.00  0.00           H  \nATOM    109  HE2 LYS A   7       3.448  -0.144  -9.095  1.00  0.00           H  \nATOM    110  HE3 LYS A   7       4.442   0.609  -7.864  1.00  0.00           H  \nATOM    111  HZ1 LYS A   7       5.280   1.985  -9.861  1.00  0.00           H  \nATOM    112  HZ2 LYS A   7       3.871   1.539 -10.556  1.00  0.00           H  \nATOM    113  HZ3 LYS A   7       3.861   2.370  -9.150  1.00  0.00           H  \nATOM    114  N   GLU A   8       8.413  -0.473  -7.921  1.00  0.00           N  \nATOM    115  CA  GLU A   8       9.006   0.852  -7.974  1.00  0.00           C  \nATOM    116  C   GLU A   8      10.531   0.748  -8.052  1.00  0.00           C  \nATOM    117  O   GLU A   8      11.143   1.247  -8.995  1.00  0.00           O  \nATOM    118  CB  GLU A   8       8.575   1.695  -6.772  1.00  0.00           C  \nATOM    119  CG  GLU A   8       9.500   2.900  -6.587  1.00  0.00           C  \nATOM    120  CD  GLU A   8       8.823   3.987  -5.751  1.00  0.00           C  \nATOM    121  OE1 GLU A   8       8.014   4.736  -6.340  1.00  0.00           O  \nATOM    122  OE2 GLU A   8       9.129   4.045  -4.540  1.00  0.00           O  \nATOM    123  H   GLU A   8       7.808  -0.634  -7.141  1.00  0.00           H  \nATOM    124  HA  GLU A   8       8.619   1.307  -8.886  1.00  0.00           H  \nATOM    125  HB2 GLU A   8       7.550   2.037  -6.912  1.00  0.00           H  \nATOM    126  HB3 GLU A   8       8.586   1.082  -5.871  1.00  0.00           H  \nATOM    127  HG2 GLU A   8      10.423   2.583  -6.101  1.00  0.00           H  \nATOM    128  HG3 GLU A   8       9.776   3.304  -7.561  1.00  0.00           H  \nATOM    129  N   ILE A   9      11.100   0.097  -7.048  1.00  0.00           N  \nATOM    130  CA  ILE A   9      12.541  -0.079  -6.992  1.00  0.00           C  \nATOM    131  C   ILE A   9      13.032  -0.659  -8.320  1.00  0.00           C  \nATOM    132  O   ILE A   9      14.110  -0.304  -8.795  1.00  0.00           O  \nATOM    133  CB  ILE A   9      12.931  -0.918  -5.773  1.00  0.00           C  \nATOM    134  CG1 ILE A   9      14.167  -0.340  -5.082  1.00  0.00           C  \nATOM    135  CG2 ILE A   9      13.123  -2.387  -6.157  1.00  0.00           C  \nATOM    136  CD1 ILE A   9      15.311  -0.144  -6.078  1.00  0.00           C  \nATOM    137  H   ILE A   9      10.595  -0.306  -6.285  1.00  0.00           H  \nATOM    138  HA  ILE A   9      12.986   0.907  -6.860  1.00  0.00           H  \nATOM    139  HB  ILE A   9      12.112  -0.878  -5.055  1.00  0.00           H  \nATOM    140 HG12 ILE A   9      13.916   0.615  -4.619  1.00  0.00           H  \nATOM    141 HG13 ILE A   9      14.487  -1.007  -4.282  1.00  0.00           H  \nATOM    142 HG21 ILE A   9      13.469  -2.947  -5.288  1.00  0.00           H  \nATOM    143 HG22 ILE A   9      12.175  -2.799  -6.501  1.00  0.00           H  \nATOM    144 HG23 ILE A   9      13.863  -2.460  -6.954  1.00  0.00           H  \nATOM    145 HD11 ILE A   9      15.194  -0.842  -6.908  1.00  0.00           H  \nATOM    146 HD12 ILE A   9      15.293   0.878  -6.457  1.00  0.00           H  \nATOM    147 HD13 ILE A   9      16.263  -0.330  -5.580  1.00  0.00           H  \nATOM    148  N   ASP A  10      12.219  -1.541  -8.881  1.00  0.00           N  \nATOM    149  CA  ASP A  10      12.557  -2.173 -10.145  1.00  0.00           C  \nATOM    150  C   ASP A  10      12.399  -1.157 -11.277  1.00  0.00           C  \nATOM    151  O   ASP A  10      11.449  -1.233 -12.056  1.00  0.00           O  \nATOM    152  CB  ASP A  10      11.629  -3.354 -10.436  1.00  0.00           C  \nATOM    153  CG  ASP A  10      11.740  -3.931 -11.849  1.00  0.00           C  \nATOM    154  OD1 ASP A  10      12.621  -3.447 -12.591  1.00  0.00           O  \nATOM    155  OD2 ASP A  10      10.941  -4.843 -12.154  1.00  0.00           O  \nATOM    156  H   ASP A  10      11.344  -1.825  -8.488  1.00  0.00           H  \nATOM    157  HA  ASP A  10      13.587  -2.512 -10.029  1.00  0.00           H  \nATOM    158  HB2 ASP A  10      11.839  -4.147  -9.718  1.00  0.00           H  \nATOM    159  HB3 ASP A  10      10.600  -3.038 -10.269  1.00  0.00           H  \nATOM    160  N   VAL A  11      13.343  -0.229 -11.333  1.00  0.00           N  \nATOM    161  CA  VAL A  11      13.320   0.801 -12.357  1.00  0.00           C  \nATOM    162  C   VAL A  11      13.134   0.148 -13.728  1.00  0.00           C  \nATOM    163  O   VAL A  11      12.421   0.676 -14.580  1.00  0.00           O  \nATOM    164  CB  VAL A  11      14.586   1.656 -12.269  1.00  0.00           C  \nATOM    165  CG1 VAL A  11      14.718   2.568 -13.489  1.00  0.00           C  \nATOM    166  CG2 VAL A  11      14.611   2.467 -10.972  1.00  0.00           C  \nATOM    167  H   VAL A  11      14.112  -0.174 -10.696  1.00  0.00           H  \nATOM    168  HA  VAL A  11      12.464   1.445 -12.157  1.00  0.00           H  \nATOM    169  HB  VAL A  11      15.444   0.983 -12.259  1.00  0.00           H  \nATOM    170 HG11 VAL A  11      13.763   3.056 -13.683  1.00  0.00           H  \nATOM    171 HG12 VAL A  11      15.480   3.324 -13.297  1.00  0.00           H  \nATOM    172 HG13 VAL A  11      15.005   1.975 -14.357  1.00  0.00           H  \nATOM    173 HG21 VAL A  11      15.560   2.997 -10.893  1.00  0.00           H  \nATOM    174 HG22 VAL A  11      13.792   3.187 -10.978  1.00  0.00           H  \nATOM    175 HG23 VAL A  11      14.497   1.795 -10.121  1.00  0.00           H  \nATOM    176  N   ASN A  12      13.788  -0.992 -13.898  1.00  0.00           N  \nATOM    177  CA  ASN A  12      13.704  -1.723 -15.151  1.00  0.00           C  \nATOM    178  C   ASN A  12      12.233  -1.920 -15.524  1.00  0.00           C  \nATOM    179  O   ASN A  12      11.872  -1.847 -16.697  1.00  0.00           O  \nATOM    180  CB  ASN A  12      14.349  -3.105 -15.026  1.00  0.00           C  \nATOM    181  CG  ASN A  12      15.686  -3.022 -14.287  1.00  0.00           C  \nATOM    182  OD1 ASN A  12      15.859  -3.553 -13.203  1.00  0.00           O  \nATOM    183  ND2 ASN A  12      16.618  -2.326 -14.932  1.00  0.00           N  \nATOM    184  H   ASN A  12      14.366  -1.415 -13.200  1.00  0.00           H  \nATOM    185  HA  ASN A  12      14.240  -1.111 -15.876  1.00  0.00           H  \nATOM    186  HB2 ASN A  12      13.676  -3.778 -14.494  1.00  0.00           H  \nATOM    187  HB3 ASN A  12      14.504  -3.529 -16.019  1.00  0.00           H  \nATOM    188 HD21 ASN A  12      16.411  -1.915 -15.820  1.00  0.00           H  \nATOM    189 HD22 ASN A  12      17.527  -2.215 -14.530  1.00  0.00           H  \nATOM    190  N   GLY A  13      11.425  -2.165 -14.503  1.00  0.00           N  \nATOM    191  CA  GLY A  13      10.001  -2.372 -14.709  1.00  0.00           C  \nATOM    192  C   GLY A  13       9.740  -3.686 -15.449  1.00  0.00           C  \nATOM    193  O   GLY A  13       9.189  -3.685 -16.548  1.00  0.00           O  \nATOM    194  H   GLY A  13      11.727  -2.222 -13.552  1.00  0.00           H  \nATOM    195  HA2 GLY A  13       9.489  -2.384 -13.748  1.00  0.00           H  \nATOM    196  HA3 GLY A  13       9.588  -1.541 -15.281  1.00  0.00           H  \nATOM    197  N   ASP A  14      10.149  -4.776 -14.815  1.00  0.00           N  \nATOM    198  CA  ASP A  14       9.966  -6.094 -15.399  1.00  0.00           C  \nATOM    199  C   ASP A  14       9.014  -6.908 -14.520  1.00  0.00           C  \nATOM    200  O   ASP A  14       8.137  -7.605 -15.029  1.00  0.00           O  \nATOM    201  CB  ASP A  14      11.294  -6.848 -15.483  1.00  0.00           C  \nATOM    202  CG  ASP A  14      12.189  -6.720 -14.249  1.00  0.00           C  \nATOM    203  OD1 ASP A  14      11.861  -7.379 -13.238  1.00  0.00           O  \nATOM    204  OD2 ASP A  14      13.182  -5.967 -14.344  1.00  0.00           O  \nATOM    205  H   ASP A  14      10.596  -4.768 -13.921  1.00  0.00           H  \nATOM    206  HA  ASP A  14       9.562  -5.912 -16.395  1.00  0.00           H  \nATOM    207  HB2 ASP A  14      11.085  -7.904 -15.653  1.00  0.00           H  \nATOM    208  HB3 ASP A  14      11.845  -6.488 -16.352  1.00  0.00           H  \nATOM    209  N   GLY A  15       9.219  -6.794 -13.217  1.00  0.00           N  \nATOM    210  CA  GLY A  15       8.391  -7.511 -12.263  1.00  0.00           C  \nATOM    211  C   GLY A  15       9.215  -8.537 -11.482  1.00  0.00           C  \nATOM    212  O   GLY A  15       8.729  -9.623 -11.171  1.00  0.00           O  \nATOM    213  H   GLY A  15       9.935  -6.225 -12.811  1.00  0.00           H  \nATOM    214  HA2 GLY A  15       7.932  -6.805 -11.571  1.00  0.00           H  \nATOM    215  HA3 GLY A  15       7.579  -8.015 -12.787  1.00  0.00           H  \nATOM    216  N   ALA A  16      10.450  -8.155 -11.188  1.00  0.00           N  \nATOM    217  CA  ALA A  16      11.346  -9.028 -10.449  1.00  0.00           C  \nATOM    218  C   ALA A  16      12.419  -8.183  -9.758  1.00  0.00           C  \nATOM    219  O   ALA A  16      12.739  -7.087 -10.214  1.00  0.00           O  \nATOM    220  CB  ALA A  16      11.945 -10.067 -11.399  1.00  0.00           C  \nATOM    221  H   ALA A  16      10.838  -7.270 -11.444  1.00  0.00           H  \nATOM    222  HA  ALA A  16      10.757  -9.544  -9.691  1.00  0.00           H  \nATOM    223  HB1 ALA A  16      12.871  -9.680 -11.824  1.00  0.00           H  \nATOM    224  HB2 ALA A  16      12.154 -10.985 -10.849  1.00  0.00           H  \nATOM    225  HB3 ALA A  16      11.237 -10.277 -12.201  1.00  0.00           H  \nATOM    226  N   VAL A  17      12.944  -8.727  -8.670  1.00  0.00           N  \nATOM    227  CA  VAL A  17      13.974  -8.037  -7.912  1.00  0.00           C  \nATOM    228  C   VAL A  17      15.240  -8.895  -7.881  1.00  0.00           C  \nATOM    229  O   VAL A  17      15.194 -10.087  -8.183  1.00  0.00           O  \nATOM    230  CB  VAL A  17      13.452  -7.690  -6.516  1.00  0.00           C  \nATOM    231  CG1 VAL A  17      14.605  -7.540  -5.521  1.00  0.00           C  \nATOM    232  CG2 VAL A  17      12.590  -6.426  -6.551  1.00  0.00           C  \nATOM    233  H   VAL A  17      12.678  -9.619  -8.306  1.00  0.00           H  \nATOM    234  HA  VAL A  17      14.195  -7.104  -8.431  1.00  0.00           H  \nATOM    235  HB  VAL A  17      12.824  -8.515  -6.179  1.00  0.00           H  \nATOM    236 HG11 VAL A  17      15.207  -8.449  -5.523  1.00  0.00           H  \nATOM    237 HG12 VAL A  17      15.226  -6.692  -5.810  1.00  0.00           H  \nATOM    238 HG13 VAL A  17      14.204  -7.372  -4.522  1.00  0.00           H  \nATOM    239 HG21 VAL A  17      13.137  -5.627  -7.051  1.00  0.00           H  \nATOM    240 HG22 VAL A  17      11.668  -6.631  -7.094  1.00  0.00           H  \nATOM    241 HG23 VAL A  17      12.352  -6.121  -5.532  1.00  0.00           H  \nATOM    242  N   SER A  18      16.341  -8.256  -7.512  1.00  0.00           N  \nATOM    243  CA  SER A  18      17.617  -8.947  -7.438  1.00  0.00           C  \nATOM    244  C   SER A  18      18.254  -8.723  -6.065  1.00  0.00           C  \nATOM    245  O   SER A  18      17.662  -8.080  -5.199  1.00  0.00           O  \nATOM    246  CB  SER A  18      18.563  -8.476  -8.544  1.00  0.00           C  \nATOM    247  OG  SER A  18      19.159  -7.218  -8.238  1.00  0.00           O  \nATOM    248  H   SER A  18      16.370  -7.287  -7.268  1.00  0.00           H  \nATOM    249  HA  SER A  18      17.384 -10.001  -7.585  1.00  0.00           H  \nATOM    250  HB2 SER A  18      19.345  -9.221  -8.693  1.00  0.00           H  \nATOM    251  HB3 SER A  18      18.014  -8.399  -9.482  1.00  0.00           H  \nATOM    252  HG  SER A  18      18.655  -6.482  -8.690  1.00  0.00           H  \nATOM    253  N   TYR A  19      19.451  -9.267  -5.907  1.00  0.00           N  \nATOM    254  CA  TYR A  19      20.175  -9.136  -4.654  1.00  0.00           C  \nATOM    255  C   TYR A  19      20.605  -7.686  -4.419  1.00  0.00           C  \nATOM    256  O   TYR A  19      20.572  -7.200  -3.290  1.00  0.00           O  \nATOM    257  CB  TYR A  19      21.423 -10.010  -4.793  1.00  0.00           C  \nATOM    258  CG  TYR A  19      22.467  -9.781  -3.699  1.00  0.00           C  \nATOM    259  CD1 TYR A  19      22.139  -9.997  -2.376  1.00  0.00           C  \nATOM    260  CD2 TYR A  19      23.738  -9.360  -4.035  1.00  0.00           C  \nATOM    261  CE1 TYR A  19      23.123  -9.782  -1.346  1.00  0.00           C  \nATOM    262  CE2 TYR A  19      24.721  -9.145  -3.005  1.00  0.00           C  \nATOM    263  CZ  TYR A  19      24.365  -9.367  -1.712  1.00  0.00           C  \nATOM    264  OH  TYR A  19      25.294  -9.164  -0.739  1.00  0.00           O  \nATOM    265  H   TYR A  19      19.926  -9.789  -6.616  1.00  0.00           H  \nATOM    266  HA  TYR A  19      19.509  -9.444  -3.848  1.00  0.00           H  \nATOM    267  HB2 TYR A  19      21.123 -11.058  -4.782  1.00  0.00           H  \nATOM    268  HB3 TYR A  19      21.880  -9.819  -5.764  1.00  0.00           H  \nATOM    269  HD1 TYR A  19      21.136 -10.330  -2.111  1.00  0.00           H  \nATOM    270  HD2 TYR A  19      23.997  -9.190  -5.080  1.00  0.00           H  \nATOM    271  HE1 TYR A  19      22.877  -9.949  -0.297  1.00  0.00           H  \nATOM    272  HE2 TYR A  19      25.728  -8.812  -3.257  1.00  0.00           H  \nATOM    273  HH  TYR A  19      26.207  -9.113  -1.143  1.00  0.00           H  \nATOM    274  N   GLU A  20      20.999  -7.036  -5.504  1.00  0.00           N  \nATOM    275  CA  GLU A  20      21.436  -5.652  -5.431  1.00  0.00           C  \nATOM    276  C   GLU A  20      20.231  -4.725  -5.258  1.00  0.00           C  \nATOM    277  O   GLU A  20      20.279  -3.779  -4.473  1.00  0.00           O  \nATOM    278  CB  GLU A  20      22.251  -5.268  -6.667  1.00  0.00           C  \nATOM    279  CG  GLU A  20      23.413  -4.345  -6.293  1.00  0.00           C  \nATOM    280  CD  GLU A  20      24.240  -3.977  -7.526  1.00  0.00           C  \nATOM    281  OE1 GLU A  20      25.042  -4.838  -7.949  1.00  0.00           O  \nATOM    282  OE2 GLU A  20      24.053  -2.844  -8.017  1.00  0.00           O  \nATOM    283  H   GLU A  20      21.023  -7.439  -6.420  1.00  0.00           H  \nATOM    284  HA  GLU A  20      22.075  -5.597  -4.550  1.00  0.00           H  \nATOM    285  HB2 GLU A  20      22.637  -6.168  -7.146  1.00  0.00           H  \nATOM    286  HB3 GLU A  20      21.607  -4.772  -7.393  1.00  0.00           H  \nATOM    287  HG2 GLU A  20      23.026  -3.439  -5.826  1.00  0.00           H  \nATOM    288  HG3 GLU A  20      24.049  -4.836  -5.556  1.00  0.00           H  \nATOM    289  N   GLU A  21      19.180  -5.027  -6.006  1.00  0.00           N  \nATOM    290  CA  GLU A  21      17.965  -4.232  -5.945  1.00  0.00           C  \nATOM    291  C   GLU A  21      17.362  -4.290  -4.540  1.00  0.00           C  \nATOM    292  O   GLU A  21      16.946  -3.268  -3.997  1.00  0.00           O  \nATOM    293  CB  GLU A  21      16.954  -4.694  -6.996  1.00  0.00           C  \nATOM    294  CG  GLU A  21      16.722  -3.609  -8.049  1.00  0.00           C  \nATOM    295  CD  GLU A  21      15.862  -4.137  -9.200  1.00  0.00           C  \nATOM    296  OE1 GLU A  21      15.436  -5.308  -9.099  1.00  0.00           O  \nATOM    297  OE2 GLU A  21      15.650  -3.357 -10.153  1.00  0.00           O  \nATOM    298  H   GLU A  21      19.149  -5.797  -6.642  1.00  0.00           H  \nATOM    299  HA  GLU A  21      18.275  -3.212  -6.173  1.00  0.00           H  \nATOM    300  HB2 GLU A  21      17.315  -5.603  -7.478  1.00  0.00           H  \nATOM    301  HB3 GLU A  21      16.010  -4.944  -6.512  1.00  0.00           H  \nATOM    302  HG2 GLU A  21      16.234  -2.750  -7.590  1.00  0.00           H  \nATOM    303  HG3 GLU A  21      17.680  -3.262  -8.436  1.00  0.00           H  \nATOM    304  N   VAL A  22      17.334  -5.496  -3.993  1.00  0.00           N  \nATOM    305  CA  VAL A  22      16.788  -5.701  -2.662  1.00  0.00           C  \nATOM    306  C   VAL A  22      17.695  -5.021  -1.634  1.00  0.00           C  \nATOM    307  O   VAL A  22      17.211  -4.387  -0.697  1.00  0.00           O  \nATOM    308  CB  VAL A  22      16.601  -7.196  -2.397  1.00  0.00           C  \nATOM    309  CG1 VAL A  22      17.951  -7.897  -2.231  1.00  0.00           C  \nATOM    310  CG2 VAL A  22      15.707  -7.430  -1.178  1.00  0.00           C  \nATOM    311  H   VAL A  22      17.674  -6.322  -4.442  1.00  0.00           H  \nATOM    312  HA  VAL A  22      15.807  -5.227  -2.632  1.00  0.00           H  \nATOM    313  HB  VAL A  22      16.103  -7.630  -3.265  1.00  0.00           H  \nATOM    314 HG11 VAL A  22      18.532  -7.388  -1.461  1.00  0.00           H  \nATOM    315 HG12 VAL A  22      17.788  -8.934  -1.937  1.00  0.00           H  \nATOM    316 HG13 VAL A  22      18.494  -7.868  -3.175  1.00  0.00           H  \nATOM    317 HG21 VAL A  22      15.796  -6.584  -0.496  1.00  0.00           H  \nATOM    318 HG22 VAL A  22      14.671  -7.530  -1.500  1.00  0.00           H  \nATOM    319 HG23 VAL A  22      16.019  -8.341  -0.668  1.00  0.00           H  \nATOM    320  N   LYS A  23      18.994  -5.177  -1.843  1.00  0.00           N  \nATOM    321  CA  LYS A  23      19.972  -4.586  -0.946  1.00  0.00           C  \nATOM    322  C   LYS A  23      19.723  -3.080  -0.847  1.00  0.00           C  \nATOM    323  O   LYS A  23      19.768  -2.509   0.242  1.00  0.00           O  \nATOM    324  CB  LYS A  23      21.392  -4.947  -1.388  1.00  0.00           C  \nATOM    325  CG  LYS A  23      21.857  -6.248  -0.732  1.00  0.00           C  \nATOM    326  CD  LYS A  23      23.384  -6.311  -0.660  1.00  0.00           C  \nATOM    327  CE  LYS A  23      24.004  -6.187  -2.053  1.00  0.00           C  \nATOM    328  NZ  LYS A  23      24.512  -4.815  -2.274  1.00  0.00           N  \nATOM    329  H   LYS A  23      19.379  -5.694  -2.607  1.00  0.00           H  \nATOM    330  HA  LYS A  23      19.820  -5.027   0.040  1.00  0.00           H  \nATOM    331  HB2 LYS A  23      21.423  -5.051  -2.473  1.00  0.00           H  \nATOM    332  HB3 LYS A  23      22.075  -4.139  -1.127  1.00  0.00           H  \nATOM    333  HG2 LYS A  23      21.438  -6.323   0.272  1.00  0.00           H  \nATOM    334  HG3 LYS A  23      21.481  -7.100  -1.298  1.00  0.00           H  \nATOM    335  HD2 LYS A  23      23.754  -5.510  -0.020  1.00  0.00           H  \nATOM    336  HD3 LYS A  23      23.692  -7.251  -0.203  1.00  0.00           H  \nATOM    337  HE2 LYS A  23      24.818  -6.904  -2.160  1.00  0.00           H  \nATOM    338  HE3 LYS A  23      23.261  -6.432  -2.811  1.00  0.00           H  \nATOM    339  HZ1 LYS A  23      24.480  -4.600  -3.250  1.00  0.00           H  \nATOM    340  HZ2 LYS A  23      23.943  -4.164  -1.771  1.00  0.00           H  \nATOM    341  HZ3 LYS A  23      25.456  -4.752  -1.950  1.00  0.00           H  \nATOM    342  N   ALA A  24      19.466  -2.478  -1.999  1.00  0.00           N  \nATOM    343  CA  ALA A  24      19.209  -1.049  -2.056  1.00  0.00           C  \nATOM    344  C   ALA A  24      17.877  -0.746  -1.367  1.00  0.00           C  \nATOM    345  O   ALA A  24      17.767   0.224  -0.620  1.00  0.00           O  \nATOM    346  CB  ALA A  24      19.231  -0.584  -3.514  1.00  0.00           C  \nATOM    347  H   ALA A  24      19.431  -2.949  -2.880  1.00  0.00           H  \nATOM    348  HA  ALA A  24      20.011  -0.546  -1.515  1.00  0.00           H  \nATOM    349  HB1 ALA A  24      19.621   0.432  -3.565  1.00  0.00           H  \nATOM    350  HB2 ALA A  24      19.868  -1.248  -4.097  1.00  0.00           H  \nATOM    351  HB3 ALA A  24      18.218  -0.605  -3.917  1.00  0.00           H  \nATOM    352  N   PHE A  25      16.898  -1.595  -1.644  1.00  0.00           N  \nATOM    353  CA  PHE A  25      15.578  -1.431  -1.060  1.00  0.00           C  \nATOM    354  C   PHE A  25      15.664  -1.288   0.461  1.00  0.00           C  \nATOM    355  O   PHE A  25      15.250  -0.273   1.018  1.00  0.00           O  \nATOM    356  CB  PHE A  25      14.779  -2.691  -1.398  1.00  0.00           C  \nATOM    357  CG  PHE A  25      13.276  -2.562  -1.143  1.00  0.00           C  \nATOM    358  CD1 PHE A  25      12.772  -2.822   0.093  1.00  0.00           C  \nATOM    359  CD2 PHE A  25      12.444  -2.189  -2.152  1.00  0.00           C  \nATOM    360  CE1 PHE A  25      11.377  -2.702   0.330  1.00  0.00           C  \nATOM    361  CE2 PHE A  25      11.049  -2.069  -1.915  1.00  0.00           C  \nATOM    362  CZ  PHE A  25      10.545  -2.329  -0.679  1.00  0.00           C  \nATOM    363  H   PHE A  25      16.996  -2.382  -2.253  1.00  0.00           H  \nATOM    364  HA  PHE A  25      15.147  -0.523  -1.484  1.00  0.00           H  \nATOM    365  HB2 PHE A  25      14.939  -2.940  -2.447  1.00  0.00           H  \nATOM    366  HB3 PHE A  25      15.167  -3.523  -0.810  1.00  0.00           H  \nATOM    367  HD1 PHE A  25      13.438  -3.121   0.902  1.00  0.00           H  \nATOM    368  HD2 PHE A  25      12.849  -1.981  -3.143  1.00  0.00           H  \nATOM    369  HE1 PHE A  25      10.972  -2.910   1.321  1.00  0.00           H  \nATOM    370  HE2 PHE A  25      10.383  -1.770  -2.724  1.00  0.00           H  \nATOM    371  HZ  PHE A  25       9.474  -2.237  -0.497  1.00  0.00           H  \nATOM    372  N   VAL A  26      16.206  -2.321   1.090  1.00  0.00           N  \nATOM    373  CA  VAL A  26      16.352  -2.324   2.535  1.00  0.00           C  \nATOM    374  C   VAL A  26      17.288  -1.187   2.951  1.00  0.00           C  \nATOM    375  O   VAL A  26      17.037  -0.503   3.942  1.00  0.00           O  \nATOM    376  CB  VAL A  26      16.832  -3.698   3.010  1.00  0.00           C  \nATOM    377  CG1 VAL A  26      18.356  -3.806   2.921  1.00  0.00           C  \nATOM    378  CG2 VAL A  26      16.343  -3.989   4.430  1.00  0.00           C  \nATOM    379  H   VAL A  26      16.541  -3.144   0.629  1.00  0.00           H  \nATOM    380  HA  VAL A  26      15.367  -2.144   2.966  1.00  0.00           H  \nATOM    381  HB  VAL A  26      16.404  -4.450   2.347  1.00  0.00           H  \nATOM    382 HG11 VAL A  26      18.684  -3.505   1.927  1.00  0.00           H  \nATOM    383 HG12 VAL A  26      18.811  -3.154   3.667  1.00  0.00           H  \nATOM    384 HG13 VAL A  26      18.658  -4.837   3.107  1.00  0.00           H  \nATOM    385 HG21 VAL A  26      15.590  -4.777   4.400  1.00  0.00           H  \nATOM    386 HG22 VAL A  26      17.184  -4.312   5.044  1.00  0.00           H  \nATOM    387 HG23 VAL A  26      15.907  -3.085   4.857  1.00  0.00           H  \nATOM    388  N   SER A  27      18.347  -1.021   2.173  1.00  0.00           N  \nATOM    389  CA  SER A  27      19.322   0.021   2.449  1.00  0.00           C  \nATOM    390  C   SER A  27      18.611   1.359   2.659  1.00  0.00           C  \nATOM    391  O   SER A  27      18.971   2.124   3.553  1.00  0.00           O  \nATOM    392  CB  SER A  27      20.342   0.134   1.314  1.00  0.00           C  \nATOM    393  OG  SER A  27      21.305   1.155   1.563  1.00  0.00           O  \nATOM    394  H   SER A  27      18.544  -1.582   1.369  1.00  0.00           H  \nATOM    395  HA  SER A  27      19.828  -0.291   3.362  1.00  0.00           H  \nATOM    396  HB2 SER A  27      20.851  -0.821   1.188  1.00  0.00           H  \nATOM    397  HB3 SER A  27      19.823   0.345   0.379  1.00  0.00           H  \nATOM    398  HG  SER A  27      21.569   1.593   0.703  1.00  0.00           H  \nATOM    399  N   LYS A  28      17.615   1.602   1.820  1.00  0.00           N  \nATOM    400  CA  LYS A  28      16.850   2.835   1.903  1.00  0.00           C  \nATOM    401  C   LYS A  28      16.434   3.076   3.355  1.00  0.00           C  \nATOM    402  O   LYS A  28      16.290   4.221   3.782  1.00  0.00           O  \nATOM    403  CB  LYS A  28      15.675   2.804   0.924  1.00  0.00           C  \nATOM    404  CG  LYS A  28      15.792   3.926  -0.110  1.00  0.00           C  \nATOM    405  CD  LYS A  28      14.798   3.722  -1.255  1.00  0.00           C  \nATOM    406  CE  LYS A  28      13.832   4.904  -1.360  1.00  0.00           C  \nATOM    407  NZ  LYS A  28      12.893   4.708  -2.487  1.00  0.00           N  \nATOM    408  H   LYS A  28      17.328   0.975   1.095  1.00  0.00           H  \nATOM    409  HA  LYS A  28      17.507   3.648   1.592  1.00  0.00           H  \nATOM    410  HB2 LYS A  28      15.644   1.839   0.417  1.00  0.00           H  \nATOM    411  HB3 LYS A  28      14.738   2.905   1.471  1.00  0.00           H  \nATOM    412  HG2 LYS A  28      15.607   4.887   0.370  1.00  0.00           H  \nATOM    413  HG3 LYS A  28      16.807   3.957  -0.505  1.00  0.00           H  \nATOM    414  HD2 LYS A  28      15.339   3.606  -2.194  1.00  0.00           H  \nATOM    415  HD3 LYS A  28      14.237   2.802  -1.095  1.00  0.00           H  \nATOM    416  HE2 LYS A  28      13.274   5.009  -0.429  1.00  0.00           H  \nATOM    417  HE3 LYS A  28      14.392   5.828  -1.502  1.00  0.00           H  \nATOM    418  HZ1 LYS A  28      12.507   5.590  -2.756  1.00  0.00           H  \nATOM    419  HZ2 LYS A  28      13.383   4.308  -3.262  1.00  0.00           H  \nATOM    420  HZ3 LYS A  28      12.157   4.093  -2.205  1.00  0.00           H  \nATOM    421  N   LYS A  29      16.253   1.978   4.075  1.00  0.00           N  \nATOM    422  CA  LYS A  29      15.857   2.056   5.471  1.00  0.00           C  \nATOM    423  C   LYS A  29      17.077   2.409   6.324  1.00  0.00           C  \nATOM    424  O   LYS A  29      17.254   3.562   6.713  1.00  0.00           O  \nATOM    425  CB  LYS A  29      15.158   0.765   5.902  1.00  0.00           C  \nATOM    426  CG  LYS A  29      13.873   1.068   6.674  1.00  0.00           C  \nATOM    427  CD  LYS A  29      12.683   0.316   6.076  1.00  0.00           C  \nATOM    428  CE  LYS A  29      11.969  -0.518   7.142  1.00  0.00           C  \nATOM    429  NZ  LYS A  29      12.823  -1.646   7.576  1.00  0.00           N  \nATOM    430  H   LYS A  29      16.373   1.051   3.720  1.00  0.00           H  \nATOM    431  HA  LYS A  29      15.129   2.862   5.559  1.00  0.00           H  \nATOM    432  HB2 LYS A  29      14.925   0.162   5.024  1.00  0.00           H  \nATOM    433  HB3 LYS A  29      15.830   0.174   6.524  1.00  0.00           H  \nATOM    434  HG2 LYS A  29      13.997   0.787   7.719  1.00  0.00           H  \nATOM    435  HG3 LYS A  29      13.677   2.140   6.653  1.00  0.00           H  \nATOM    436  HD2 LYS A  29      11.983   1.026   5.635  1.00  0.00           H  \nATOM    437  HD3 LYS A  29      13.026  -0.334   5.271  1.00  0.00           H  \nATOM    438  HE2 LYS A  29      11.723   0.110   7.998  1.00  0.00           H  \nATOM    439  HE3 LYS A  29      11.028  -0.898   6.745  1.00  0.00           H  \nATOM    440  HZ1 LYS A  29      13.780  -1.436   7.375  1.00  0.00           H  \nATOM    441  HZ2 LYS A  29      12.713  -1.788   8.560  1.00  0.00           H  \nATOM    442  HZ3 LYS A  29      12.554  -2.475   7.087  1.00  0.00           H  \nATOM    443  N   ARG A  30      17.886   1.394   6.591  1.00  0.00           N  \nATOM    444  CA  ARG A  30      19.084   1.583   7.391  1.00  0.00           C  \nATOM    445  C   ARG A  30      20.279   0.893   6.729  1.00  0.00           C  \nATOM    446  O   ARG A  30      20.179   0.421   5.598  1.00  0.00           O  \nATOM    447  CB  ARG A  30      18.898   1.023   8.802  1.00  0.00           C  \nATOM    448  CG  ARG A  30      17.586   1.513   9.418  1.00  0.00           C  \nATOM    449  CD  ARG A  30      17.499   1.133  10.898  1.00  0.00           C  \nATOM    450  NE  ARG A  30      17.661   2.339  11.740  1.00  0.00           N  \nATOM    451  CZ  ARG A  30      17.265   2.422  13.018  1.00  0.00           C  \nATOM    452  NH1 ARG A  30      16.682   1.369  13.608  1.00  0.00           N  \nATOM    453  NH2 ARG A  30      17.453   3.556  13.705  1.00  0.00           N  \nATOM    454  H   ARG A  30      17.735   0.459   6.271  1.00  0.00           H  \nATOM    455  HA  ARG A  30      19.227   2.663   7.429  1.00  0.00           H  \nATOM    456  HB2 ARG A  30      18.903  -0.067   8.769  1.00  0.00           H  \nATOM    457  HB3 ARG A  30      19.735   1.326   9.431  1.00  0.00           H  \nATOM    458  HG2 ARG A  30      17.512   2.596   9.312  1.00  0.00           H  \nATOM    459  HG3 ARG A  30      16.743   1.082   8.878  1.00  0.00           H  \nATOM    460  HD2 ARG A  30      16.538   0.661  11.104  1.00  0.00           H  \nATOM    461  HD3 ARG A  30      18.271   0.403  11.140  1.00  0.00           H  \nATOM    462  HE  ARG A  30      18.093   3.143  11.330  1.00  0.00           H  \nATOM    463 HH11 ARG A  30      16.542   0.523  13.094  1.00  0.00           H  \nATOM    464 HH12 ARG A  30      16.386   1.431  14.561  1.00  0.00           H  \nATOM    465 HH21 ARG A  30      17.889   4.342  13.265  1.00  0.00           H  \nATOM    466 HH22 ARG A  30      17.158   3.618  14.659  1.00  0.00           H  \nATOM    467  N   ALA A  31      21.381   0.856   7.463  1.00  0.00           N  \nATOM    468  CA  ALA A  31      22.594   0.231   6.962  1.00  0.00           C  \nATOM    469  C   ALA A  31      23.280  -0.527   8.100  1.00  0.00           C  \nATOM    470  O   ALA A  31      24.458  -0.305   8.377  1.00  0.00           O  \nATOM    471  CB  ALA A  31      23.499   1.297   6.342  1.00  0.00           C  \nATOM    472  H   ALA A  31      21.454   1.242   8.383  1.00  0.00           H  \nATOM    473  HA  ALA A  31      22.305  -0.479   6.186  1.00  0.00           H  \nATOM    474  HB1 ALA A  31      22.888   2.028   5.811  1.00  0.00           H  \nATOM    475  HB2 ALA A  31      24.062   1.798   7.129  1.00  0.00           H  \nATOM    476  HB3 ALA A  31      24.190   0.826   5.644  1.00  0.00           H  \nATOM    477  N   ILE A  32      22.514  -1.406   8.730  1.00  0.00           N  \nATOM    478  CA  ILE A  32      23.034  -2.197   9.832  1.00  0.00           C  \nATOM    479  C   ILE A  32      23.516  -3.549   9.300  1.00  0.00           C  \nATOM    480  O   ILE A  32      23.115  -4.597   9.804  1.00  0.00           O  \nATOM    481  CB  ILE A  32      21.993  -2.311  10.947  1.00  0.00           C  \nATOM    482  CG1 ILE A  32      20.760  -3.080  10.470  1.00  0.00           C  \nATOM    483  CG2 ILE A  32      21.631  -0.932  11.501  1.00  0.00           C  \nATOM    484  CD1 ILE A  32      20.366  -4.164  11.476  1.00  0.00           C  \nATOM    485  H   ILE A  32      21.557  -1.580   8.499  1.00  0.00           H  \nATOM    486  HA  ILE A  32      23.890  -1.662  10.243  1.00  0.00           H  \nATOM    487  HB  ILE A  32      22.432  -2.881  11.766  1.00  0.00           H  \nATOM    488 HG12 ILE A  32      19.928  -2.389  10.331  1.00  0.00           H  \nATOM    489 HG13 ILE A  32      20.963  -3.534   9.501  1.00  0.00           H  \nATOM    490 HG21 ILE A  32      22.542  -0.361  11.681  1.00  0.00           H  \nATOM    491 HG22 ILE A  32      21.007  -0.403  10.780  1.00  0.00           H  \nATOM    492 HG23 ILE A  32      21.084  -1.048  12.437  1.00  0.00           H  \nATOM    493 HD11 ILE A  32      20.296  -3.727  12.472  1.00  0.00           H  \nATOM    494 HD12 ILE A  32      19.401  -4.586  11.196  1.00  0.00           H  \nATOM    495 HD13 ILE A  32      21.121  -4.950  11.476  1.00  0.00           H  \nATOM    496  N   LYS A  33      24.370  -3.481   8.289  1.00  0.00           N  \nATOM    497  CA  LYS A  33      24.910  -4.686   7.684  1.00  0.00           C  \nATOM    498  C   LYS A  33      23.802  -5.407   6.915  1.00  0.00           C  \nATOM    499  O   LYS A  33      23.711  -6.633   6.953  1.00  0.00           O  \nATOM    500  CB  LYS A  33      25.592  -5.557   8.741  1.00  0.00           C  \nATOM    501  CG  LYS A  33      27.114  -5.420   8.666  1.00  0.00           C  \nATOM    502  CD  LYS A  33      27.781  -6.037   9.897  1.00  0.00           C  \nATOM    503  CE  LYS A  33      28.047  -4.976  10.967  1.00  0.00           C  \nATOM    504  NZ  LYS A  33      28.335  -5.616  12.270  1.00  0.00           N  \nATOM    505  H   LYS A  33      24.691  -2.624   7.885  1.00  0.00           H  \nATOM    506  HA  LYS A  33      25.679  -4.379   6.975  1.00  0.00           H  \nATOM    507  HB2 LYS A  33      25.246  -5.269   9.734  1.00  0.00           H  \nATOM    508  HB3 LYS A  33      25.309  -6.599   8.596  1.00  0.00           H  \nATOM    509  HG2 LYS A  33      27.483  -5.909   7.765  1.00  0.00           H  \nATOM    510  HG3 LYS A  33      27.385  -4.367   8.591  1.00  0.00           H  \nATOM    511  HD2 LYS A  33      27.143  -6.821  10.307  1.00  0.00           H  \nATOM    512  HD3 LYS A  33      28.720  -6.510   9.608  1.00  0.00           H  \nATOM    513  HE2 LYS A  33      28.888  -4.352  10.667  1.00  0.00           H  \nATOM    514  HE3 LYS A  33      27.181  -4.321  11.060  1.00  0.00           H  \nATOM    515  HZ1 LYS A  33      27.868  -6.499  12.319  1.00  0.00           H  \nATOM    516  HZ2 LYS A  33      29.321  -5.754  12.363  1.00  0.00           H  \nATOM    517  HZ3 LYS A  33      28.010  -5.026  13.010  1.00  0.00           H  \nATOM    518  N   ASN A  34      22.986  -4.615   6.235  1.00  0.00           N  \nATOM    519  CA  ASN A  34      21.887  -5.162   5.458  1.00  0.00           C  \nATOM    520  C   ASN A  34      22.445  -6.084   4.372  1.00  0.00           C  \nATOM    521  O   ASN A  34      21.786  -7.039   3.964  1.00  0.00           O  \nATOM    522  CB  ASN A  34      21.090  -4.051   4.771  1.00  0.00           C  \nATOM    523  CG  ASN A  34      21.909  -3.399   3.655  1.00  0.00           C  \nATOM    524  OD1 ASN A  34      21.874  -3.806   2.506  1.00  0.00           O  \nATOM    525  ND2 ASN A  34      22.644  -2.366   4.057  1.00  0.00           N  \nATOM    526  H   ASN A  34      23.066  -3.618   6.209  1.00  0.00           H  \nATOM    527  HA  ASN A  34      21.265  -5.695   6.177  1.00  0.00           H  \nATOM    528  HB2 ASN A  34      20.168  -4.461   4.359  1.00  0.00           H  \nATOM    529  HB3 ASN A  34      20.804  -3.297   5.505  1.00  0.00           H  \nATOM    530 HD21 ASN A  34      22.627  -2.083   5.015  1.00  0.00           H  \nATOM    531 HD22 ASN A  34      23.215  -1.873   3.400  1.00  0.00           H  \nATOM    532  N   GLU A  35      23.654  -5.765   3.933  1.00  0.00           N  \nATOM    533  CA  GLU A  35      24.308  -6.552   2.902  1.00  0.00           C  \nATOM    534  C   GLU A  35      24.370  -8.024   3.317  1.00  0.00           C  \nATOM    535  O   GLU A  35      23.910  -8.898   2.584  1.00  0.00           O  \nATOM    536  CB  GLU A  35      25.707  -6.008   2.602  1.00  0.00           C  \nATOM    537  CG  GLU A  35      26.563  -5.967   3.869  1.00  0.00           C  \nATOM    538  CD  GLU A  35      27.724  -4.983   3.714  1.00  0.00           C  \nATOM    539  OE1 GLU A  35      28.463  -5.129   2.717  1.00  0.00           O  \nATOM    540  OE2 GLU A  35      27.847  -4.107   4.597  1.00  0.00           O  \nATOM    541  H   GLU A  35      24.183  -4.986   4.270  1.00  0.00           H  \nATOM    542  HA  GLU A  35      23.685  -6.443   2.015  1.00  0.00           H  \nATOM    543  HB2 GLU A  35      26.191  -6.634   1.852  1.00  0.00           H  \nATOM    544  HB3 GLU A  35      25.628  -5.007   2.179  1.00  0.00           H  \nATOM    545  HG2 GLU A  35      25.946  -5.676   4.720  1.00  0.00           H  \nATOM    546  HG3 GLU A  35      26.951  -6.962   4.083  1.00  0.00           H  \nATOM    547  N   GLN A  36      24.941  -8.252   4.490  1.00  0.00           N  \nATOM    548  CA  GLN A  36      25.068  -9.602   5.011  1.00  0.00           C  \nATOM    549  C   GLN A  36      23.701 -10.134   5.445  1.00  0.00           C  \nATOM    550  O   GLN A  36      23.225 -11.137   4.916  1.00  0.00           O  \nATOM    551  CB  GLN A  36      26.069  -9.652   6.167  1.00  0.00           C  \nATOM    552  CG  GLN A  36      26.604 -11.071   6.368  1.00  0.00           C  \nATOM    553  CD  GLN A  36      27.751 -11.086   7.381  1.00  0.00           C  \nATOM    554  OE1 GLN A  36      28.605 -10.216   7.404  1.00  0.00           O  \nATOM    555  NE2 GLN A  36      27.722 -12.122   8.215  1.00  0.00           N  \nATOM    556  H   GLN A  36      25.313  -7.535   5.080  1.00  0.00           H  \nATOM    557  HA  GLN A  36      25.451 -10.197   4.182  1.00  0.00           H  \nATOM    558  HB2 GLN A  36      26.897  -8.973   5.965  1.00  0.00           H  \nATOM    559  HB3 GLN A  36      25.590  -9.307   7.083  1.00  0.00           H  \nATOM    560  HG2 GLN A  36      25.800 -11.721   6.714  1.00  0.00           H  \nATOM    561  HG3 GLN A  36      26.950 -11.472   5.416  1.00  0.00           H  \nATOM    562 HE21 GLN A  36      26.993 -12.802   8.143  1.00  0.00           H  \nATOM    563 HE22 GLN A  36      28.431 -12.221   8.914  1.00  0.00           H  \nATOM    564  N   LEU A  37      23.109  -9.439   6.405  1.00  0.00           N  \nATOM    565  CA  LEU A  37      21.806  -9.829   6.917  1.00  0.00           C  \nATOM    566  C   LEU A  37      20.920 -10.280   5.754  1.00  0.00           C  \nATOM    567  O   LEU A  37      20.404 -11.396   5.759  1.00  0.00           O  \nATOM    568  CB  LEU A  37      21.199  -8.700   7.751  1.00  0.00           C  \nATOM    569  CG  LEU A  37      20.576  -9.113   9.087  1.00  0.00           C  \nATOM    570  CD1 LEU A  37      19.222  -9.793   8.873  1.00  0.00           C  \nATOM    571  CD2 LEU A  37      21.537  -9.990   9.892  1.00  0.00           C  \nATOM    572  H   LEU A  37      23.504  -8.625   6.831  1.00  0.00           H  \nATOM    573  HA  LEU A  37      21.958 -10.677   7.585  1.00  0.00           H  \nATOM    574  HB2 LEU A  37      21.976  -7.962   7.949  1.00  0.00           H  \nATOM    575  HB3 LEU A  37      20.433  -8.205   7.154  1.00  0.00           H  \nATOM    576  HG  LEU A  37      20.394  -8.212   9.672  1.00  0.00           H  \nATOM    577 HD11 LEU A  37      18.487  -9.050   8.562  1.00  0.00           H  \nATOM    578 HD12 LEU A  37      19.317 -10.556   8.100  1.00  0.00           H  \nATOM    579 HD13 LEU A  37      18.898 -10.258   9.804  1.00  0.00           H  \nATOM    580 HD21 LEU A  37      21.572  -9.638  10.923  1.00  0.00           H  \nATOM    581 HD22 LEU A  37      21.190 -11.023   9.872  1.00  0.00           H  \nATOM    582 HD23 LEU A  37      22.534  -9.933   9.455  1.00  0.00           H  \nATOM    583  N   LEU A  38      20.772  -9.388   4.785  1.00  0.00           N  \nATOM    584  CA  LEU A  38      19.957  -9.681   3.618  1.00  0.00           C  \nATOM    585  C   LEU A  38      20.465 -10.963   2.955  1.00  0.00           C  \nATOM    586  O   LEU A  38      19.710 -11.919   2.787  1.00  0.00           O  \nATOM    587  CB  LEU A  38      19.919  -8.475   2.676  1.00  0.00           C  \nATOM    588  CG  LEU A  38      19.051  -8.629   1.425  1.00  0.00           C  \nATOM    589  CD1 LEU A  38      18.265  -7.348   1.142  1.00  0.00           C  \nATOM    590  CD2 LEU A  38      19.894  -9.061   0.223  1.00  0.00           C  \nATOM    591  H   LEU A  38      21.195  -8.483   4.789  1.00  0.00           H  \nATOM    592  HA  LEU A  38      18.938  -9.851   3.965  1.00  0.00           H  \nATOM    593  HB2 LEU A  38      19.562  -7.612   3.238  1.00  0.00           H  \nATOM    594  HB3 LEU A  38      20.938  -8.252   2.362  1.00  0.00           H  \nATOM    595  HG  LEU A  38      18.324  -9.420   1.609  1.00  0.00           H  \nATOM    596 HD11 LEU A  38      17.329  -7.599   0.643  1.00  0.00           H  \nATOM    597 HD12 LEU A  38      18.050  -6.838   2.082  1.00  0.00           H  \nATOM    598 HD13 LEU A  38      18.855  -6.693   0.501  1.00  0.00           H  \nATOM    599 HD21 LEU A  38      20.866  -9.414   0.569  1.00  0.00           H  \nATOM    600 HD22 LEU A  38      19.384  -9.865  -0.308  1.00  0.00           H  \nATOM    601 HD23 LEU A  38      20.033  -8.213  -0.447  1.00  0.00           H  \nATOM    602  N   GLN A  39      21.740 -10.941   2.597  1.00  0.00           N  \nATOM    603  CA  GLN A  39      22.357 -12.090   1.956  1.00  0.00           C  \nATOM    604  C   GLN A  39      21.838 -13.388   2.579  1.00  0.00           C  \nATOM    605  O   GLN A  39      21.562 -14.354   1.870  1.00  0.00           O  \nATOM    606  CB  GLN A  39      23.882 -12.013   2.043  1.00  0.00           C  \nATOM    607  CG  GLN A  39      24.538 -12.880   0.966  1.00  0.00           C  \nATOM    608  CD  GLN A  39      24.978 -14.228   1.541  1.00  0.00           C  \nATOM    609  OE1 GLN A  39      24.194 -15.150   1.700  1.00  0.00           O  \nATOM    610  NE2 GLN A  39      26.271 -14.291   1.842  1.00  0.00           N  \nATOM    611  H   GLN A  39      22.347 -10.159   2.737  1.00  0.00           H  \nATOM    612  HA  GLN A  39      22.055 -12.035   0.910  1.00  0.00           H  \nATOM    613  HB2 GLN A  39      24.205 -10.978   1.928  1.00  0.00           H  \nATOM    614  HB3 GLN A  39      24.212 -12.341   3.029  1.00  0.00           H  \nATOM    615  HG2 GLN A  39      23.836 -13.041   0.148  1.00  0.00           H  \nATOM    616  HG3 GLN A  39      25.400 -12.359   0.550  1.00  0.00           H  \nATOM    617 HE21 GLN A  39      26.860 -13.498   1.687  1.00  0.00           H  \nATOM    618 HE22 GLN A  39      26.655 -15.132   2.224  1.00  0.00           H  \nATOM    619  N   LEU A  40      21.720 -13.367   3.899  1.00  0.00           N  \nATOM    620  CA  LEU A  40      21.239 -14.530   4.625  1.00  0.00           C  \nATOM    621  C   LEU A  40      19.768 -14.769   4.279  1.00  0.00           C  \nATOM    622  O   LEU A  40      19.404 -15.848   3.814  1.00  0.00           O  \nATOM    623  CB  LEU A  40      21.500 -14.371   6.125  1.00  0.00           C  \nATOM    624  CG  LEU A  40      20.853 -15.419   7.033  1.00  0.00           C  \nATOM    625  CD1 LEU A  40      21.701 -16.690   7.094  1.00  0.00           C  \nATOM    626  CD2 LEU A  40      20.580 -14.844   8.425  1.00  0.00           C  \nATOM    627  H   LEU A  40      21.947 -12.577   4.469  1.00  0.00           H  \nATOM    628  HA  LEU A  40      21.818 -15.389   4.287  1.00  0.00           H  \nATOM    629  HB2 LEU A  40      22.577 -14.391   6.290  1.00  0.00           H  \nATOM    630  HB3 LEU A  40      21.149 -13.386   6.432  1.00  0.00           H  \nATOM    631  HG  LEU A  40      19.889 -15.694   6.604  1.00  0.00           H  \nATOM    632 HD11 LEU A  40      21.057 -17.562   6.974  1.00  0.00           H  \nATOM    633 HD12 LEU A  40      22.440 -16.673   6.293  1.00  0.00           H  \nATOM    634 HD13 LEU A  40      22.209 -16.743   8.057  1.00  0.00           H  \nATOM    635 HD21 LEU A  40      20.069 -13.886   8.329  1.00  0.00           H  \nATOM    636 HD22 LEU A  40      19.953 -15.536   8.987  1.00  0.00           H  \nATOM    637 HD23 LEU A  40      21.524 -14.701   8.950  1.00  0.00           H  \nATOM    638  N   ILE A  41      18.962 -13.745   4.520  1.00  0.00           N  \nATOM    639  CA  ILE A  41      17.539 -13.831   4.239  1.00  0.00           C  \nATOM    640  C   ILE A  41      17.335 -14.294   2.795  1.00  0.00           C  \nATOM    641  O   ILE A  41      16.740 -15.343   2.554  1.00  0.00           O  \nATOM    642  CB  ILE A  41      16.849 -12.504   4.562  1.00  0.00           C  \nATOM    643  CG1 ILE A  41      16.821 -12.254   6.072  1.00  0.00           C  \nATOM    644  CG2 ILE A  41      15.449 -12.450   3.946  1.00  0.00           C  \nATOM    645  CD1 ILE A  41      16.535 -10.783   6.378  1.00  0.00           C  \nATOM    646  H   ILE A  41      19.267 -12.871   4.898  1.00  0.00           H  \nATOM    647  HA  ILE A  41      17.119 -14.584   4.906  1.00  0.00           H  \nATOM    648  HB  ILE A  41      17.429 -11.699   4.113  1.00  0.00           H  \nATOM    649 HG12 ILE A  41      16.058 -12.881   6.533  1.00  0.00           H  \nATOM    650 HG13 ILE A  41      17.777 -12.542   6.509  1.00  0.00           H  \nATOM    651 HG21 ILE A  41      15.521 -12.593   2.868  1.00  0.00           H  \nATOM    652 HG22 ILE A  41      14.832 -13.239   4.377  1.00  0.00           H  \nATOM    653 HG23 ILE A  41      14.998 -11.481   4.155  1.00  0.00           H  \nATOM    654 HD11 ILE A  41      15.590 -10.493   5.919  1.00  0.00           H  \nATOM    655 HD12 ILE A  41      16.473 -10.642   7.457  1.00  0.00           H  \nATOM    656 HD13 ILE A  41      17.339 -10.165   5.977  1.00  0.00           H  \nATOM    657  N   PHE A  42      17.841 -13.490   1.872  1.00  0.00           N  \nATOM    658  CA  PHE A  42      17.722 -13.804   0.458  1.00  0.00           C  \nATOM    659  C   PHE A  42      17.999 -15.287   0.201  1.00  0.00           C  \nATOM    660  O   PHE A  42      17.144 -16.000  -0.322  1.00  0.00           O  \nATOM    661  CB  PHE A  42      18.771 -12.967  -0.276  1.00  0.00           C  \nATOM    662  CG  PHE A  42      18.556 -12.886  -1.788  1.00  0.00           C  \nATOM    663  CD1 PHE A  42      19.063 -13.852  -2.600  1.00  0.00           C  \nATOM    664  CD2 PHE A  42      17.856 -11.849  -2.321  1.00  0.00           C  \nATOM    665  CE1 PHE A  42      18.863 -13.777  -4.004  1.00  0.00           C  \nATOM    666  CE2 PHE A  42      17.656 -11.774  -3.725  1.00  0.00           C  \nATOM    667  CZ  PHE A  42      18.163 -12.739  -4.537  1.00  0.00           C  \nATOM    668  H   PHE A  42      18.324 -12.638   2.076  1.00  0.00           H  \nATOM    669  HA  PHE A  42      16.700 -13.570   0.160  1.00  0.00           H  \nATOM    670  HB2 PHE A  42      18.769 -11.958   0.136  1.00  0.00           H  \nATOM    671  HB3 PHE A  42      19.758 -13.388  -0.082  1.00  0.00           H  \nATOM    672  HD1 PHE A  42      19.624 -14.683  -2.173  1.00  0.00           H  \nATOM    673  HD2 PHE A  42      17.449 -11.075  -1.670  1.00  0.00           H  \nATOM    674  HE1 PHE A  42      19.270 -14.551  -4.655  1.00  0.00           H  \nATOM    675  HE2 PHE A  42      17.095 -10.942  -4.152  1.00  0.00           H  \nATOM    676  HZ  PHE A  42      18.010 -12.682  -5.614  1.00  0.00           H  \nATOM    677  N   LYS A  43      19.197 -15.707   0.581  1.00  0.00           N  \nATOM    678  CA  LYS A  43      19.597 -17.092   0.398  1.00  0.00           C  \nATOM    679  C   LYS A  43      18.562 -18.008   1.055  1.00  0.00           C  \nATOM    680  O   LYS A  43      18.356 -19.136   0.611  1.00  0.00           O  \nATOM    681  CB  LYS A  43      21.023 -17.309   0.908  1.00  0.00           C  \nATOM    682  CG  LYS A  43      21.798 -18.251  -0.017  1.00  0.00           C  \nATOM    683  CD  LYS A  43      22.675 -19.211   0.789  1.00  0.00           C  \nATOM    684  CE  LYS A  43      21.989 -20.569   0.958  1.00  0.00           C  \nATOM    685  NZ  LYS A  43      21.297 -20.641   2.264  1.00  0.00           N  \nATOM    686  H   LYS A  43      19.886 -15.120   1.006  1.00  0.00           H  \nATOM    687  HA  LYS A  43      19.603 -17.291  -0.673  1.00  0.00           H  \nATOM    688  HB2 LYS A  43      21.540 -16.352   0.974  1.00  0.00           H  \nATOM    689  HB3 LYS A  43      20.994 -17.725   1.915  1.00  0.00           H  \nATOM    690  HG2 LYS A  43      21.099 -18.819  -0.631  1.00  0.00           H  \nATOM    691  HG3 LYS A  43      22.420 -17.668  -0.696  1.00  0.00           H  \nATOM    692  HD2 LYS A  43      23.633 -19.344   0.286  1.00  0.00           H  \nATOM    693  HD3 LYS A  43      22.886 -18.782   1.768  1.00  0.00           H  \nATOM    694  HE2 LYS A  43      21.273 -20.724   0.151  1.00  0.00           H  \nATOM    695  HE3 LYS A  43      22.728 -21.367   0.886  1.00  0.00           H  \nATOM    696  HZ1 LYS A  43      20.440 -20.128   2.216  1.00  0.00           H  \nATOM    697  HZ2 LYS A  43      21.102 -21.596   2.485  1.00  0.00           H  \nATOM    698  HZ3 LYS A  43      21.882 -20.247   2.973  1.00  0.00           H  \nATOM    699  N   SER A  44      17.940 -17.488   2.103  1.00  0.00           N  \nATOM    700  CA  SER A  44      16.932 -18.245   2.826  1.00  0.00           C  \nATOM    701  C   SER A  44      15.714 -18.483   1.931  1.00  0.00           C  \nATOM    702  O   SER A  44      15.289 -19.622   1.745  1.00  0.00           O  \nATOM    703  CB  SER A  44      16.515 -17.523   4.109  1.00  0.00           C  \nATOM    704  OG  SER A  44      16.261 -18.433   5.175  1.00  0.00           O  \nATOM    705  H   SER A  44      18.113 -16.570   2.458  1.00  0.00           H  \nATOM    706  HA  SER A  44      17.410 -19.191   3.081  1.00  0.00           H  \nATOM    707  HB2 SER A  44      17.301 -16.828   4.405  1.00  0.00           H  \nATOM    708  HB3 SER A  44      15.621 -16.930   3.917  1.00  0.00           H  \nATOM    709  HG  SER A  44      16.246 -17.943   6.047  1.00  0.00           H  \nATOM    710  N   ILE A  45      15.187 -17.390   1.401  1.00  0.00           N  \nATOM    711  CA  ILE A  45      14.026 -17.465   0.530  1.00  0.00           C  \nATOM    712  C   ILE A  45      14.433 -18.099  -0.801  1.00  0.00           C  \nATOM    713  O   ILE A  45      13.704 -18.926  -1.348  1.00  0.00           O  \nATOM    714  CB  ILE A  45      13.377 -16.087   0.382  1.00  0.00           C  \nATOM    715  CG1 ILE A  45      11.921 -16.213  -0.072  1.00  0.00           C  \nATOM    716  CG2 ILE A  45      14.194 -15.194  -0.554  1.00  0.00           C  \nATOM    717  CD1 ILE A  45      10.986 -16.379   1.127  1.00  0.00           C  \nATOM    718  H   ILE A  45      15.538 -16.467   1.558  1.00  0.00           H  \nATOM    719  HA  ILE A  45      13.296 -18.114   1.013  1.00  0.00           H  \nATOM    720  HB  ILE A  45      13.369 -15.606   1.360  1.00  0.00           H  \nATOM    721 HG12 ILE A  45      11.636 -15.328  -0.641  1.00  0.00           H  \nATOM    722 HG13 ILE A  45      11.817 -17.068  -0.740  1.00  0.00           H  \nATOM    723 HG21 ILE A  45      13.626 -14.293  -0.785  1.00  0.00           H  \nATOM    724 HG22 ILE A  45      15.130 -14.919  -0.068  1.00  0.00           H  \nATOM    725 HG23 ILE A  45      14.408 -15.734  -1.476  1.00  0.00           H  \nATOM    726 HD11 ILE A  45      11.350 -17.185   1.763  1.00  0.00           H  \nATOM    727 HD12 ILE A  45      10.958 -15.450   1.698  1.00  0.00           H  \nATOM    728 HD13 ILE A  45       9.982 -16.618   0.775  1.00  0.00           H  \nATOM    729  N   ASP A  46      15.596 -17.688  -1.285  1.00  0.00           N  \nATOM    730  CA  ASP A  46      16.109 -18.206  -2.542  1.00  0.00           C  \nATOM    731  C   ASP A  46      16.507 -19.672  -2.360  1.00  0.00           C  \nATOM    732  O   ASP A  46      17.520 -19.970  -1.730  1.00  0.00           O  \nATOM    733  CB  ASP A  46      17.349 -17.432  -2.992  1.00  0.00           C  \nATOM    734  CG  ASP A  46      17.632 -17.480  -4.495  1.00  0.00           C  \nATOM    735  OD1 ASP A  46      16.830 -18.126  -5.203  1.00  0.00           O  \nATOM    736  OD2 ASP A  46      18.645 -16.871  -4.901  1.00  0.00           O  \nATOM    737  H   ASP A  46      16.183 -17.015  -0.835  1.00  0.00           H  \nATOM    738  HA  ASP A  46      15.295 -18.080  -3.257  1.00  0.00           H  \nATOM    739  HB2 ASP A  46      17.237 -16.390  -2.692  1.00  0.00           H  \nATOM    740  HB3 ASP A  46      18.217 -17.825  -2.461  1.00  0.00           H  \nATOM    741  N   ALA A  47      15.688 -20.549  -2.924  1.00  0.00           N  \nATOM    742  CA  ALA A  47      15.942 -21.976  -2.832  1.00  0.00           C  \nATOM    743  C   ALA A  47      16.901 -22.393  -3.949  1.00  0.00           C  \nATOM    744  O   ALA A  47      17.834 -23.159  -3.718  1.00  0.00           O  \nATOM    745  CB  ALA A  47      14.615 -22.736  -2.889  1.00  0.00           C  \nATOM    746  H   ALA A  47      14.866 -20.298  -3.435  1.00  0.00           H  \nATOM    747  HA  ALA A  47      16.415 -22.168  -1.869  1.00  0.00           H  \nATOM    748  HB1 ALA A  47      14.730 -23.704  -2.401  1.00  0.00           H  \nATOM    749  HB2 ALA A  47      13.845 -22.160  -2.377  1.00  0.00           H  \nATOM    750  HB3 ALA A  47      14.326 -22.885  -3.929  1.00  0.00           H  \nATOM    751  N   ASP A  48      16.637 -21.868  -5.137  1.00  0.00           N  \nATOM    752  CA  ASP A  48      17.465 -22.176  -6.291  1.00  0.00           C  \nATOM    753  C   ASP A  48      18.838 -21.523  -6.118  1.00  0.00           C  \nATOM    754  O   ASP A  48      19.853 -22.213  -6.050  1.00  0.00           O  \nATOM    755  CB  ASP A  48      16.841 -21.630  -7.577  1.00  0.00           C  \nATOM    756  CG  ASP A  48      16.124 -20.287  -7.429  1.00  0.00           C  \nATOM    757  OD1 ASP A  48      16.835 -19.259  -7.458  1.00  0.00           O  \nATOM    758  OD2 ASP A  48      14.883 -20.318  -7.291  1.00  0.00           O  \nATOM    759  H   ASP A  48      15.876 -21.245  -5.317  1.00  0.00           H  \nATOM    760  HA  ASP A  48      17.521 -23.264  -6.321  1.00  0.00           H  \nATOM    761  HB2 ASP A  48      17.625 -21.526  -8.328  1.00  0.00           H  \nATOM    762  HB3 ASP A  48      16.131 -22.364  -7.959  1.00  0.00           H  \nATOM    763  N   GLY A  49      18.824 -20.200  -6.050  1.00  0.00           N  \nATOM    764  CA  GLY A  49      20.055 -19.446  -5.885  1.00  0.00           C  \nATOM    765  C   GLY A  49      20.440 -18.734  -7.184  1.00  0.00           C  \nATOM    766  O   GLY A  49      21.619 -18.651  -7.525  1.00  0.00           O  \nATOM    767  H   GLY A  49      17.993 -19.646  -6.106  1.00  0.00           H  \nATOM    768  HA2 GLY A  49      19.933 -18.714  -5.088  1.00  0.00           H  \nATOM    769  HA3 GLY A  49      20.859 -20.117  -5.582  1.00  0.00           H  \nATOM    770  N   ASN A  50      19.423 -18.239  -7.874  1.00  0.00           N  \nATOM    771  CA  ASN A  50      19.640 -17.537  -9.128  1.00  0.00           C  \nATOM    772  C   ASN A  50      20.207 -16.146  -8.837  1.00  0.00           C  \nATOM    773  O   ASN A  50      20.646 -15.448  -9.749  1.00  0.00           O  \nATOM    774  CB  ASN A  50      18.328 -17.363  -9.895  1.00  0.00           C  \nATOM    775  CG  ASN A  50      17.206 -16.899  -8.965  1.00  0.00           C  \nATOM    776  OD1 ASN A  50      17.346 -16.858  -7.754  1.00  0.00           O  \nATOM    777  ND2 ASN A  50      16.088 -16.553  -9.596  1.00  0.00           N  \nATOM    778  H   ASN A  50      18.467 -18.311  -7.590  1.00  0.00           H  \nATOM    779  HA  ASN A  50      20.335 -18.162  -9.689  1.00  0.00           H  \nATOM    780  HB2 ASN A  50      18.465 -16.636 -10.696  1.00  0.00           H  \nATOM    781  HB3 ASN A  50      18.049 -18.306 -10.365  1.00  0.00           H  \nATOM    782 HD21 ASN A  50      16.039 -16.610 -10.594  1.00  0.00           H  \nATOM    783 HD22 ASN A  50      15.296 -16.236  -9.074  1.00  0.00           H  \nATOM    784  N   GLY A  51      20.178 -15.785  -7.562  1.00  0.00           N  \nATOM    785  CA  GLY A  51      20.683 -14.490  -7.140  1.00  0.00           C  \nATOM    786  C   GLY A  51      19.612 -13.407  -7.292  1.00  0.00           C  \nATOM    787  O   GLY A  51      19.791 -12.282  -6.829  1.00  0.00           O  \nATOM    788  H   GLY A  51      19.818 -16.359  -6.827  1.00  0.00           H  \nATOM    789  HA2 GLY A  51      21.006 -14.543  -6.101  1.00  0.00           H  \nATOM    790  HA3 GLY A  51      21.558 -14.226  -7.733  1.00  0.00           H  \nATOM    791  N   GLU A  52      18.522 -13.786  -7.943  1.00  0.00           N  \nATOM    792  CA  GLU A  52      17.422 -12.862  -8.162  1.00  0.00           C  \nATOM    793  C   GLU A  52      16.107 -13.479  -7.682  1.00  0.00           C  \nATOM    794  O   GLU A  52      15.945 -14.699  -7.703  1.00  0.00           O  \nATOM    795  CB  GLU A  52      17.332 -12.455  -9.634  1.00  0.00           C  \nATOM    796  CG  GLU A  52      18.567 -11.657 -10.058  1.00  0.00           C  \nATOM    797  CD  GLU A  52      19.340 -12.387 -11.158  1.00  0.00           C  \nATOM    798  OE1 GLU A  52      18.679 -12.807 -12.133  1.00  0.00           O  \nATOM    799  OE2 GLU A  52      20.574 -12.508 -11.001  1.00  0.00           O  \nATOM    800  H   GLU A  52      18.384 -14.703  -8.316  1.00  0.00           H  \nATOM    801  HA  GLU A  52      17.658 -11.983  -7.562  1.00  0.00           H  \nATOM    802  HB2 GLU A  52      17.239 -13.345 -10.256  1.00  0.00           H  \nATOM    803  HB3 GLU A  52      16.435 -11.857  -9.796  1.00  0.00           H  \nATOM    804  HG2 GLU A  52      18.264 -10.672 -10.413  1.00  0.00           H  \nATOM    805  HG3 GLU A  52      19.216 -11.499  -9.196  1.00  0.00           H  \nATOM    806  N   ILE A  53      15.200 -12.609  -7.263  1.00  0.00           N  \nATOM    807  CA  ILE A  53      13.903 -13.054  -6.780  1.00  0.00           C  \nATOM    808  C   ILE A  53      12.802 -12.255  -7.479  1.00  0.00           C  \nATOM    809  O   ILE A  53      13.087 -11.386  -8.301  1.00  0.00           O  \nATOM    810  CB  ILE A  53      13.845 -12.975  -5.253  1.00  0.00           C  \nATOM    811  CG1 ILE A  53      14.541 -11.712  -4.742  1.00  0.00           C  \nATOM    812  CG2 ILE A  53      14.419 -14.243  -4.616  1.00  0.00           C  \nATOM    813  CD1 ILE A  53      13.730 -10.462  -5.086  1.00  0.00           C  \nATOM    814  H   ILE A  53      15.339 -11.619  -7.250  1.00  0.00           H  \nATOM    815  HA  ILE A  53      13.795 -14.103  -7.053  1.00  0.00           H  \nATOM    816  HB  ILE A  53      12.799 -12.910  -4.954  1.00  0.00           H  \nATOM    817 HG12 ILE A  53      14.674 -11.778  -3.662  1.00  0.00           H  \nATOM    818 HG13 ILE A  53      15.536 -11.637  -5.181  1.00  0.00           H  \nATOM    819 HG21 ILE A  53      15.501 -14.146  -4.526  1.00  0.00           H  \nATOM    820 HG22 ILE A  53      13.982 -14.383  -3.627  1.00  0.00           H  \nATOM    821 HG23 ILE A  53      14.182 -15.103  -5.243  1.00  0.00           H  \nATOM    822 HD11 ILE A  53      12.666 -10.684  -5.000  1.00  0.00           H  \nATOM    823 HD12 ILE A  53      13.990  -9.658  -4.397  1.00  0.00           H  \nATOM    824 HD13 ILE A  53      13.955 -10.152  -6.107  1.00  0.00           H  \nATOM    825  N   ASP A  54      11.566 -12.577  -7.126  1.00  0.00           N  \nATOM    826  CA  ASP A  54      10.420 -11.900  -7.709  1.00  0.00           C  \nATOM    827  C   ASP A  54       9.589 -11.261  -6.595  1.00  0.00           C  \nATOM    828  O   ASP A  54       9.969 -11.311  -5.426  1.00  0.00           O  \nATOM    829  CB  ASP A  54       9.524 -12.885  -8.463  1.00  0.00           C  \nATOM    830  CG  ASP A  54       9.208 -14.179  -7.710  1.00  0.00           C  \nATOM    831  OD1 ASP A  54       9.383 -14.172  -6.472  1.00  0.00           O  \nATOM    832  OD2 ASP A  54       8.799 -15.145  -8.389  1.00  0.00           O  \nATOM    833  H   ASP A  54      11.343 -13.285  -6.456  1.00  0.00           H  \nATOM    834  HA  ASP A  54      10.841 -11.161  -8.391  1.00  0.00           H  \nATOM    835  HB2 ASP A  54       8.586 -12.385  -8.706  1.00  0.00           H  \nATOM    836  HB3 ASP A  54      10.004 -13.139  -9.408  1.00  0.00           H  \nATOM    837  N   GLN A  55       8.470 -10.677  -6.996  1.00  0.00           N  \nATOM    838  CA  GLN A  55       7.582 -10.029  -6.046  1.00  0.00           C  \nATOM    839  C   GLN A  55       7.135 -11.024  -4.973  1.00  0.00           C  \nATOM    840  O   GLN A  55       6.968 -10.657  -3.811  1.00  0.00           O  \nATOM    841  CB  GLN A  55       6.376  -9.411  -6.756  1.00  0.00           C  \nATOM    842  CG  GLN A  55       5.485 -10.494  -7.367  1.00  0.00           C  \nATOM    843  CD  GLN A  55       4.606  -9.918  -8.479  1.00  0.00           C  \nATOM    844  OE1 GLN A  55       4.846 -10.115  -9.659  1.00  0.00           O  \nATOM    845  NE2 GLN A  55       3.579  -9.197  -8.039  1.00  0.00           N  \nATOM    846  H   GLN A  55       8.168 -10.641  -7.949  1.00  0.00           H  \nATOM    847  HA  GLN A  55       8.174  -9.235  -5.590  1.00  0.00           H  \nATOM    848  HB2 GLN A  55       5.799  -8.815  -6.048  1.00  0.00           H  \nATOM    849  HB3 GLN A  55       6.718  -8.733  -7.538  1.00  0.00           H  \nATOM    850  HG2 GLN A  55       6.104 -11.297  -7.766  1.00  0.00           H  \nATOM    851  HG3 GLN A  55       4.855 -10.932  -6.592  1.00  0.00           H  \nATOM    852 HE21 GLN A  55       3.439  -9.073  -7.057  1.00  0.00           H  \nATOM    853 HE22 GLN A  55       2.947  -8.778  -8.691  1.00  0.00           H  \nATOM    854  N   ASN A  56       6.954 -12.265  -5.401  1.00  0.00           N  \nATOM    855  CA  ASN A  56       6.530 -13.316  -4.492  1.00  0.00           C  \nATOM    856  C   ASN A  56       7.597 -13.512  -3.413  1.00  0.00           C  \nATOM    857  O   ASN A  56       7.322 -13.346  -2.226  1.00  0.00           O  \nATOM    858  CB  ASN A  56       6.354 -14.645  -5.230  1.00  0.00           C  \nATOM    859  CG  ASN A  56       4.947 -14.762  -5.820  1.00  0.00           C  \nATOM    860  OD1 ASN A  56       3.946 -14.621  -5.138  1.00  0.00           O  \nATOM    861  ND2 ASN A  56       4.928 -15.027  -7.123  1.00  0.00           N  \nATOM    862  H   ASN A  56       7.092 -12.556  -6.348  1.00  0.00           H  \nATOM    863  HA  ASN A  56       5.580 -12.974  -4.082  1.00  0.00           H  \nATOM    864  HB2 ASN A  56       7.094 -14.724  -6.026  1.00  0.00           H  \nATOM    865  HB3 ASN A  56       6.534 -15.473  -4.544  1.00  0.00           H  \nATOM    866 HD21 ASN A  56       5.787 -15.131  -7.625  1.00  0.00           H  \nATOM    867 HD22 ASN A  56       4.055 -15.123  -7.601  1.00  0.00           H  \nATOM    868  N   GLU A  57       8.792 -13.863  -3.865  1.00  0.00           N  \nATOM    869  CA  GLU A  57       9.902 -14.084  -2.953  1.00  0.00           C  \nATOM    870  C   GLU A  57      10.189 -12.814  -2.149  1.00  0.00           C  \nATOM    871  O   GLU A  57      10.641 -12.887  -1.007  1.00  0.00           O  \nATOM    872  CB  GLU A  57      11.149 -14.548  -3.708  1.00  0.00           C  \nATOM    873  CG  GLU A  57      11.152 -16.068  -3.880  1.00  0.00           C  \nATOM    874  CD  GLU A  57      12.298 -16.515  -4.790  1.00  0.00           C  \nATOM    875  OE1 GLU A  57      12.218 -16.202  -5.998  1.00  0.00           O  \nATOM    876  OE2 GLU A  57      13.228 -17.159  -4.257  1.00  0.00           O  \nATOM    877  H   GLU A  57       9.008 -13.996  -4.832  1.00  0.00           H  \nATOM    878  HA  GLU A  57       9.573 -14.879  -2.284  1.00  0.00           H  \nATOM    879  HB2 GLU A  57      11.185 -14.068  -4.686  1.00  0.00           H  \nATOM    880  HB3 GLU A  57      12.043 -14.237  -3.167  1.00  0.00           H  \nATOM    881  HG2 GLU A  57      11.248 -16.547  -2.906  1.00  0.00           H  \nATOM    882  HG3 GLU A  57      10.200 -16.392  -4.302  1.00  0.00           H  \nATOM    883  N   PHE A  58       9.913 -11.680  -2.776  1.00  0.00           N  \nATOM    884  CA  PHE A  58      10.136 -10.396  -2.133  1.00  0.00           C  \nATOM    885  C   PHE A  58       9.124 -10.162  -1.010  1.00  0.00           C  \nATOM    886  O   PHE A  58       9.477  -9.651   0.052  1.00  0.00           O  \nATOM    887  CB  PHE A  58       9.948  -9.321  -3.205  1.00  0.00           C  \nATOM    888  CG  PHE A  58      10.437  -7.932  -2.787  1.00  0.00           C  \nATOM    889  CD1 PHE A  58       9.621  -7.110  -2.074  1.00  0.00           C  \nATOM    890  CD2 PHE A  58      11.687  -7.521  -3.128  1.00  0.00           C  \nATOM    891  CE1 PHE A  58      10.075  -5.822  -1.686  1.00  0.00           C  \nATOM    892  CE2 PHE A  58      12.141  -6.232  -2.739  1.00  0.00           C  \nATOM    893  CZ  PHE A  58      11.325  -5.410  -2.027  1.00  0.00           C  \nATOM    894  H   PHE A  58       9.545 -11.630  -3.705  1.00  0.00           H  \nATOM    895  HA  PHE A  58      11.142 -10.414  -1.715  1.00  0.00           H  \nATOM    896  HB2 PHE A  58      10.478  -9.625  -4.108  1.00  0.00           H  \nATOM    897  HB3 PHE A  58       8.890  -9.260  -3.462  1.00  0.00           H  \nATOM    898  HD1 PHE A  58       8.618  -7.440  -1.801  1.00  0.00           H  \nATOM    899  HD2 PHE A  58      12.341  -8.180  -3.699  1.00  0.00           H  \nATOM    900  HE1 PHE A  58       9.421  -5.163  -1.114  1.00  0.00           H  \nATOM    901  HE2 PHE A  58      13.143  -5.903  -3.013  1.00  0.00           H  \nATOM    902  HZ  PHE A  58      11.674  -4.421  -1.729  1.00  0.00           H  \nATOM    903  N   ALA A  59       7.886 -10.548  -1.282  1.00  0.00           N  \nATOM    904  CA  ALA A  59       6.821 -10.387  -0.308  1.00  0.00           C  \nATOM    905  C   ALA A  59       7.112 -11.266   0.910  1.00  0.00           C  \nATOM    906  O   ALA A  59       7.009 -10.811   2.048  1.00  0.00           O  \nATOM    907  CB  ALA A  59       5.477 -10.721  -0.960  1.00  0.00           C  \nATOM    908  H   ALA A  59       7.607 -10.963  -2.148  1.00  0.00           H  \nATOM    909  HA  ALA A  59       6.810  -9.342   0.003  1.00  0.00           H  \nATOM    910  HB1 ALA A  59       5.174  -9.899  -1.608  1.00  0.00           H  \nATOM    911  HB2 ALA A  59       5.576 -11.632  -1.550  1.00  0.00           H  \nATOM    912  HB3 ALA A  59       4.725 -10.870  -0.185  1.00  0.00           H  \nATOM    913  N   LYS A  60       7.470 -12.510   0.630  1.00  0.00           N  \nATOM    914  CA  LYS A  60       7.777 -13.457   1.688  1.00  0.00           C  \nATOM    915  C   LYS A  60       9.091 -13.055   2.361  1.00  0.00           C  \nATOM    916  O   LYS A  60       9.287 -13.308   3.549  1.00  0.00           O  \nATOM    917  CB  LYS A  60       7.778 -14.887   1.144  1.00  0.00           C  \nATOM    918  CG  LYS A  60       6.382 -15.508   1.229  1.00  0.00           C  \nATOM    919  CD  LYS A  60       6.448 -16.933   1.781  1.00  0.00           C  \nATOM    920  CE  LYS A  60       5.191 -17.268   2.585  1.00  0.00           C  \nATOM    921  NZ  LYS A  60       4.729 -18.640   2.277  1.00  0.00           N  \nATOM    922  H   LYS A  60       7.552 -12.873  -0.299  1.00  0.00           H  \nATOM    923  HA  LYS A  60       6.977 -13.394   2.426  1.00  0.00           H  \nATOM    924  HB2 LYS A  60       8.117 -14.886   0.108  1.00  0.00           H  \nATOM    925  HB3 LYS A  60       8.485 -15.495   1.709  1.00  0.00           H  \nATOM    926  HG2 LYS A  60       5.747 -14.895   1.869  1.00  0.00           H  \nATOM    927  HG3 LYS A  60       5.924 -15.518   0.240  1.00  0.00           H  \nATOM    928  HD2 LYS A  60       6.558 -17.640   0.959  1.00  0.00           H  \nATOM    929  HD3 LYS A  60       7.329 -17.041   2.415  1.00  0.00           H  \nATOM    930  HE2 LYS A  60       5.399 -17.179   3.652  1.00  0.00           H  \nATOM    931  HE3 LYS A  60       4.402 -16.552   2.354  1.00  0.00           H  \nATOM    932  HZ1 LYS A  60       3.915 -18.848   2.820  1.00  0.00           H  \nATOM    933  HZ2 LYS A  60       4.507 -18.707   1.305  1.00  0.00           H  \nATOM    934  HZ3 LYS A  60       5.453 -19.293   2.500  1.00  0.00           H  \nATOM    935  N   PHE A  61       9.957 -12.435   1.574  1.00  0.00           N  \nATOM    936  CA  PHE A  61      11.246 -11.995   2.079  1.00  0.00           C  \nATOM    937  C   PHE A  61      11.084 -10.821   3.047  1.00  0.00           C  \nATOM    938  O   PHE A  61      11.816 -10.717   4.030  1.00  0.00           O  \nATOM    939  CB  PHE A  61      12.067 -11.535   0.872  1.00  0.00           C  \nATOM    940  CG  PHE A  61      13.270 -10.662   1.233  1.00  0.00           C  \nATOM    941  CD1 PHE A  61      13.092  -9.347   1.530  1.00  0.00           C  \nATOM    942  CD2 PHE A  61      14.518 -11.201   1.257  1.00  0.00           C  \nATOM    943  CE1 PHE A  61      14.208  -8.537   1.866  1.00  0.00           C  \nATOM    944  CE2 PHE A  61      15.635 -10.391   1.592  1.00  0.00           C  \nATOM    945  CZ  PHE A  61      15.457  -9.076   1.890  1.00  0.00           C  \nATOM    946  H   PHE A  61       9.790 -12.233   0.609  1.00  0.00           H  \nATOM    947  HA  PHE A  61      11.693 -12.839   2.605  1.00  0.00           H  \nATOM    948  HB2 PHE A  61      12.417 -12.412   0.328  1.00  0.00           H  \nATOM    949  HB3 PHE A  61      11.418 -10.979   0.195  1.00  0.00           H  \nATOM    950  HD1 PHE A  61      12.091  -8.915   1.511  1.00  0.00           H  \nATOM    951  HD2 PHE A  61      14.661 -12.255   1.019  1.00  0.00           H  \nATOM    952  HE1 PHE A  61      14.065  -7.483   2.104  1.00  0.00           H  \nATOM    953  HE2 PHE A  61      16.636 -10.823   1.612  1.00  0.00           H  \nATOM    954  HZ  PHE A  61      16.314  -8.454   2.147  1.00  0.00           H  \nATOM    955  N   TYR A  62      10.121  -9.967   2.735  1.00  0.00           N  \nATOM    956  CA  TYR A  62       9.853  -8.805   3.565  1.00  0.00           C  \nATOM    957  C   TYR A  62       9.609  -9.214   5.019  1.00  0.00           C  \nATOM    958  O   TYR A  62      10.095  -8.562   5.942  1.00  0.00           O  \nATOM    959  CB  TYR A  62       8.579  -8.171   3.005  1.00  0.00           C  \nATOM    960  CG  TYR A  62       8.508  -6.653   3.185  1.00  0.00           C  \nATOM    961  CD1 TYR A  62       8.370  -6.114   4.448  1.00  0.00           C  \nATOM    962  CD2 TYR A  62       8.581  -5.823   2.084  1.00  0.00           C  \nATOM    963  CE1 TYR A  62       8.304  -4.685   4.617  1.00  0.00           C  \nATOM    964  CE2 TYR A  62       8.514  -4.395   2.254  1.00  0.00           C  \nATOM    965  CZ  TYR A  62       8.378  -3.896   3.512  1.00  0.00           C  \nATOM    966  OH  TYR A  62       8.315  -2.547   3.672  1.00  0.00           O  \nATOM    967  H   TYR A  62       9.530 -10.059   1.934  1.00  0.00           H  \nATOM    968  HA  TYR A  62      10.725  -8.152   3.521  1.00  0.00           H  \nATOM    969  HB2 TYR A  62       8.505  -8.404   1.943  1.00  0.00           H  \nATOM    970  HB3 TYR A  62       7.715  -8.625   3.491  1.00  0.00           H  \nATOM    971  HD1 TYR A  62       8.313  -6.769   5.317  1.00  0.00           H  \nATOM    972  HD2 TYR A  62       8.689  -6.250   1.087  1.00  0.00           H  \nATOM    973  HE1 TYR A  62       8.195  -4.246   5.609  1.00  0.00           H  \nATOM    974  HE2 TYR A  62       8.570  -3.729   1.393  1.00  0.00           H  \nATOM    975  HH  TYR A  62       7.445  -2.202   3.321  1.00  0.00           H  \nATOM    976  N   GLY A  63       8.855 -10.292   5.178  1.00  0.00           N  \nATOM    977  CA  GLY A  63       8.540 -10.796   6.504  1.00  0.00           C  \nATOM    978  C   GLY A  63       9.774 -11.419   7.161  1.00  0.00           C  \nATOM    979  O   GLY A  63      10.390 -10.811   8.035  1.00  0.00           O  \nATOM    980  H   GLY A  63       8.464 -10.817   4.422  1.00  0.00           H  \nATOM    981  HA2 GLY A  63       8.164  -9.984   7.126  1.00  0.00           H  \nATOM    982  HA3 GLY A  63       7.746 -11.539   6.435  1.00  0.00           H  \nATOM    983  N   SER A  64      10.097 -12.624   6.716  1.00  0.00           N  \nATOM    984  CA  SER A  64      11.246 -13.336   7.249  1.00  0.00           C  \nATOM    985  C   SER A  64      11.413 -14.675   6.528  1.00  0.00           C  \nATOM    986  O   SER A  64      12.372 -14.866   5.781  1.00  0.00           O  \nATOM    987  CB  SER A  64      11.104 -13.560   8.756  1.00  0.00           C  \nATOM    988  OG  SER A  64      12.078 -12.831   9.498  1.00  0.00           O  \nATOM    989  H   SER A  64       9.591 -13.112   6.005  1.00  0.00           H  \nATOM    990  HA  SER A  64      12.101 -12.688   7.058  1.00  0.00           H  \nATOM    991  HB2 SER A  64      10.106 -13.258   9.075  1.00  0.00           H  \nATOM    992  HB3 SER A  64      11.201 -14.623   8.976  1.00  0.00           H  \nATOM    993  HG  SER A  64      12.035 -11.861   9.259  1.00  0.00           H  \nATOM    994  N   ILE A  65      10.466 -15.567   6.776  1.00  0.00           N  \nATOM    995  CA  ILE A  65      10.497 -16.882   6.159  1.00  0.00           C  \nATOM    996  C   ILE A  65       9.067 -17.327   5.845  1.00  0.00           C  \nATOM    997  O   ILE A  65       8.631 -17.261   4.696  1.00  0.00           O  \nATOM    998  CB  ILE A  65      11.268 -17.869   7.038  1.00  0.00           C  \nATOM    999  CG1 ILE A  65      12.777 -17.721   6.834  1.00  0.00           C  \nATOM   1000  CG2 ILE A  65      10.795 -19.304   6.798  1.00  0.00           C  \nATOM   1001  CD1 ILE A  65      13.202 -18.270   5.470  1.00  0.00           C  \nATOM   1002  H   ILE A  65       9.690 -15.404   7.384  1.00  0.00           H  \nATOM   1003  HA  ILE A  65      11.044 -16.790   5.221  1.00  0.00           H  \nATOM   1004  HB  ILE A  65      11.060 -17.631   8.081  1.00  0.00           H  \nATOM   1005 HG12 ILE A  65      13.057 -16.670   6.910  1.00  0.00           H  \nATOM   1006 HG13 ILE A  65      13.309 -18.250   7.625  1.00  0.00           H  \nATOM   1007 HG21 ILE A  65      11.574 -20.000   7.109  1.00  0.00           H  \nATOM   1008 HG22 ILE A  65       9.890 -19.489   7.376  1.00  0.00           H  \nATOM   1009 HG23 ILE A  65      10.584 -19.444   5.738  1.00  0.00           H  \nATOM   1010 HD11 ILE A  65      13.445 -17.442   4.805  1.00  0.00           H  \nATOM   1011 HD12 ILE A  65      14.078 -18.907   5.592  1.00  0.00           H  \nATOM   1012 HD13 ILE A  65      12.386 -18.852   5.043  1.00  0.00           H  \nATOM   1013  N   GLN A  66       8.377 -17.770   6.885  1.00  0.00           N  \nATOM   1014  CA  GLN A  66       7.005 -18.225   6.734  1.00  0.00           C  \nATOM   1015  C   GLN A  66       6.196 -17.898   7.991  1.00  0.00           C  \nATOM   1016  O   GLN A  66       5.830 -18.795   8.749  1.00  0.00           O  \nATOM   1017  CB  GLN A  66       6.956 -19.723   6.425  1.00  0.00           C  \nATOM   1018  CG  GLN A  66       5.527 -20.171   6.110  1.00  0.00           C  \nATOM   1019  CD  GLN A  66       5.517 -21.556   5.460  1.00  0.00           C  \nATOM   1020  OE1 GLN A  66       6.254 -22.452   5.837  1.00  0.00           O  \nATOM   1021  NE2 GLN A  66       4.643 -21.680   4.465  1.00  0.00           N  \nATOM   1022  H   GLN A  66       8.739 -17.820   7.816  1.00  0.00           H  \nATOM   1023  HA  GLN A  66       6.608 -17.672   5.884  1.00  0.00           H  \nATOM   1024  HB2 GLN A  66       7.605 -19.945   5.578  1.00  0.00           H  \nATOM   1025  HB3 GLN A  66       7.338 -20.287   7.276  1.00  0.00           H  \nATOM   1026  HG2 GLN A  66       4.938 -20.191   7.027  1.00  0.00           H  \nATOM   1027  HG3 GLN A  66       5.054 -19.449   5.444  1.00  0.00           H  \nATOM   1028 HE21 GLN A  66       4.067 -20.905   4.206  1.00  0.00           H  \nATOM   1029 HE22 GLN A  66       4.562 -22.549   3.977  1.00  0.00           H  \nATOM   1030  N   GLY A  67       5.941 -16.611   8.173  1.00  0.00           N  \nATOM   1031  CA  GLY A  67       5.182 -16.154   9.325  1.00  0.00           C  \nATOM   1032  C   GLY A  67       5.374 -14.652   9.547  1.00  0.00           C  \nATOM   1033  O   GLY A  67       6.492 -14.192   9.771  1.00  0.00           O  \nATOM   1034  H   GLY A  67       6.243 -15.888   7.552  1.00  0.00           H  \nATOM   1035  HA2 GLY A  67       4.124 -16.371   9.177  1.00  0.00           H  \nATOM   1036  HA3 GLY A  67       5.499 -16.700  10.213  1.00  0.00           H  \nATOM   1037  N   GLN A  68       4.265 -13.930   9.478  1.00  0.00           N  \nATOM   1038  CA  GLN A  68       4.297 -12.490   9.669  1.00  0.00           C  \nATOM   1039  C   GLN A  68       2.920 -11.981  10.100  1.00  0.00           C  \nATOM   1040  O   GLN A  68       1.941 -12.135   9.372  1.00  0.00           O  \nATOM   1041  CB  GLN A  68       4.772 -11.779   8.400  1.00  0.00           C  \nATOM   1042  CG  GLN A  68       4.063 -12.334   7.163  1.00  0.00           C  \nATOM   1043  CD  GLN A  68       3.270 -11.238   6.448  1.00  0.00           C  \nATOM   1044  OE1 GLN A  68       3.700 -10.102   6.328  1.00  0.00           O  \nATOM   1045  NE2 GLN A  68       2.092 -11.640   5.981  1.00  0.00           N  \nATOM   1046  H   GLN A  68       3.359 -14.312   9.296  1.00  0.00           H  \nATOM   1047  HA  GLN A  68       5.021 -12.320  10.466  1.00  0.00           H  \nATOM   1048  HB2 GLN A  68       4.580 -10.709   8.485  1.00  0.00           H  \nATOM   1049  HB3 GLN A  68       5.850 -11.901   8.292  1.00  0.00           H  \nATOM   1050  HG2 GLN A  68       4.797 -12.761   6.480  1.00  0.00           H  \nATOM   1051  HG3 GLN A  68       3.392 -13.141   7.456  1.00  0.00           H  \nATOM   1052 HE21 GLN A  68       1.798 -12.587   6.113  1.00  0.00           H  \nATOM   1053 HE22 GLN A  68       1.499 -10.996   5.498  1.00  0.00           H  \nATOM   1054  N   ASP A  69       2.889 -11.385  11.283  1.00  0.00           N  \nATOM   1055  CA  ASP A  69       1.648 -10.853  11.820  1.00  0.00           C  \nATOM   1056  C   ASP A  69       1.451  -9.422  11.314  1.00  0.00           C  \nATOM   1057  O   ASP A  69       0.467  -9.130  10.636  1.00  0.00           O  \nATOM   1058  CB  ASP A  69       1.682 -10.813  13.349  1.00  0.00           C  \nATOM   1059  CG  ASP A  69       0.427 -11.352  14.038  1.00  0.00           C  \nATOM   1060  OD1 ASP A  69       0.177 -12.568  13.891  1.00  0.00           O  \nATOM   1061  OD2 ASP A  69      -0.254 -10.537  14.696  1.00  0.00           O  \nATOM   1062  H   ASP A  69       3.690 -11.264  11.870  1.00  0.00           H  \nATOM   1063  HA  ASP A  69       0.870 -11.530  11.469  1.00  0.00           H  \nATOM   1064  HB2 ASP A  69       2.542 -11.386  13.694  1.00  0.00           H  \nATOM   1065  HB3 ASP A  69       1.838  -9.782  13.667  1.00  0.00           H  \nATOM   1066  N   LEU A  70       2.402  -8.568  11.663  1.00  0.00           N  \nATOM   1067  CA  LEU A  70       2.345  -7.176  11.253  1.00  0.00           C  \nATOM   1068  C   LEU A  70       3.726  -6.540  11.428  1.00  0.00           C  \nATOM   1069  O   LEU A  70       4.057  -6.054  12.508  1.00  0.00           O  \nATOM   1070  CB  LEU A  70       1.231  -6.442  12.002  1.00  0.00           C  \nATOM   1071  CG  LEU A  70      -0.136  -6.416  11.315  1.00  0.00           C  \nATOM   1072  CD1 LEU A  70      -1.205  -7.063  12.197  1.00  0.00           C  \nATOM   1073  CD2 LEU A  70      -0.517  -4.993  10.904  1.00  0.00           C  \nATOM   1074  H   LEU A  70       3.199  -8.814  12.215  1.00  0.00           H  \nATOM   1075  HA  LEU A  70       2.087  -7.157  10.194  1.00  0.00           H  \nATOM   1076  HB2 LEU A  70       1.114  -6.903  12.983  1.00  0.00           H  \nATOM   1077  HB3 LEU A  70       1.550  -5.413  12.170  1.00  0.00           H  \nATOM   1078  HG  LEU A  70      -0.070  -7.008  10.402  1.00  0.00           H  \nATOM   1079 HD11 LEU A  70      -1.042  -8.140  12.235  1.00  0.00           H  \nATOM   1080 HD12 LEU A  70      -1.144  -6.651  13.204  1.00  0.00           H  \nATOM   1081 HD13 LEU A  70      -2.192  -6.860  11.781  1.00  0.00           H  \nATOM   1082 HD21 LEU A  70       0.388  -4.401  10.764  1.00  0.00           H  \nATOM   1083 HD22 LEU A  70      -1.079  -5.022   9.970  1.00  0.00           H  \nATOM   1084 HD23 LEU A  70      -1.130  -4.541  11.683  1.00  0.00           H  \nATOM   1085  N   SER A  71       4.494  -6.565  10.348  1.00  0.00           N  \nATOM   1086  CA  SER A  71       5.831  -5.998  10.369  1.00  0.00           C  \nATOM   1087  C   SER A  71       5.760  -4.502  10.685  1.00  0.00           C  \nATOM   1088  O   SER A  71       6.632  -3.967  11.368  1.00  0.00           O  \nATOM   1089  CB  SER A  71       6.546  -6.224   9.035  1.00  0.00           C  \nATOM   1090  OG  SER A  71       6.526  -7.595   8.645  1.00  0.00           O  \nATOM   1091  H   SER A  71       4.217  -6.963   9.474  1.00  0.00           H  \nATOM   1092  HA  SER A  71       6.359  -6.531  11.159  1.00  0.00           H  \nATOM   1093  HB2 SER A  71       6.071  -5.620   8.262  1.00  0.00           H  \nATOM   1094  HB3 SER A  71       7.579  -5.885   9.115  1.00  0.00           H  \nATOM   1095  HG  SER A  71       6.540  -7.667   7.648  1.00  0.00           H  \nATOM   1096  N   ASP A  72       4.714  -3.871  10.174  1.00  0.00           N  \nATOM   1097  CA  ASP A  72       4.518  -2.448  10.394  1.00  0.00           C  \nATOM   1098  C   ASP A  72       4.426  -2.176  11.896  1.00  0.00           C  \nATOM   1099  O   ASP A  72       4.812  -1.105  12.362  1.00  0.00           O  \nATOM   1100  CB  ASP A  72       3.220  -1.964   9.745  1.00  0.00           C  \nATOM   1101  CG  ASP A  72       3.068  -0.444   9.657  1.00  0.00           C  \nATOM   1102  OD1 ASP A  72       3.809   0.244  10.392  1.00  0.00           O  \nATOM   1103  OD2 ASP A  72       2.214  -0.004   8.857  1.00  0.00           O  \nATOM   1104  H   ASP A  72       4.009  -4.314   9.620  1.00  0.00           H  \nATOM   1105  HA  ASP A  72       5.381  -1.966   9.935  1.00  0.00           H  \nATOM   1106  HB2 ASP A  72       3.157  -2.379   8.739  1.00  0.00           H  \nATOM   1107  HB3 ASP A  72       2.378  -2.366  10.309  1.00  0.00           H  \nATOM   1108  N   ASP A  73       3.911  -3.164  12.613  1.00  0.00           N  \nATOM   1109  CA  ASP A  73       3.763  -3.045  14.054  1.00  0.00           C  \nATOM   1110  C   ASP A  73       5.139  -3.154  14.714  1.00  0.00           C  \nATOM   1111  O   ASP A  73       5.370  -2.575  15.774  1.00  0.00           O  \nATOM   1112  CB  ASP A  73       2.881  -4.165  14.611  1.00  0.00           C  \nATOM   1113  CG  ASP A  73       2.059  -3.786  15.845  1.00  0.00           C  \nATOM   1114  OD1 ASP A  73       1.620  -2.617  15.898  1.00  0.00           O  \nATOM   1115  OD2 ASP A  73       1.889  -4.675  16.708  1.00  0.00           O  \nATOM   1116  H   ASP A  73       3.599  -4.032  12.227  1.00  0.00           H  \nATOM   1117  HA  ASP A  73       3.299  -2.072  14.215  1.00  0.00           H  \nATOM   1118  HB2 ASP A  73       2.200  -4.495  13.827  1.00  0.00           H  \nATOM   1119  HB3 ASP A  73       3.515  -5.015  14.862  1.00  0.00           H  \nATOM   1120  N   LYS A  74       6.017  -3.901  14.061  1.00  0.00           N  \nATOM   1121  CA  LYS A  74       7.363  -4.094  14.571  1.00  0.00           C  \nATOM   1122  C   LYS A  74       8.167  -2.807  14.374  1.00  0.00           C  \nATOM   1123  O   LYS A  74       8.706  -2.255  15.332  1.00  0.00           O  \nATOM   1124  CB  LYS A  74       8.008  -5.325  13.932  1.00  0.00           C  \nATOM   1125  CG  LYS A  74       7.579  -6.605  14.654  1.00  0.00           C  \nATOM   1126  CD  LYS A  74       6.757  -7.506  13.729  1.00  0.00           C  \nATOM   1127  CE  LYS A  74       6.012  -8.577  14.529  1.00  0.00           C  \nATOM   1128  NZ  LYS A  74       4.885  -7.975  15.276  1.00  0.00           N  \nATOM   1129  H   LYS A  74       5.820  -4.369  13.199  1.00  0.00           H  \nATOM   1130  HA  LYS A  74       7.282  -4.290  15.640  1.00  0.00           H  \nATOM   1131  HB2 LYS A  74       7.726  -5.384  12.881  1.00  0.00           H  \nATOM   1132  HB3 LYS A  74       9.093  -5.231  13.966  1.00  0.00           H  \nATOM   1133  HG2 LYS A  74       8.460  -7.143  15.003  1.00  0.00           H  \nATOM   1134  HG3 LYS A  74       6.991  -6.350  15.535  1.00  0.00           H  \nATOM   1135  HD2 LYS A  74       6.043  -6.903  13.168  1.00  0.00           H  \nATOM   1136  HD3 LYS A  74       7.414  -7.982  13.002  1.00  0.00           H  \nATOM   1137  HE2 LYS A  74       5.638  -9.348  13.855  1.00  0.00           H  \nATOM   1138  HE3 LYS A  74       6.697  -9.064  15.222  1.00  0.00           H  \nATOM   1139  HZ1 LYS A  74       5.173  -7.100  15.666  1.00  0.00           H  \nATOM   1140  HZ2 LYS A  74       4.115  -7.825  14.655  1.00  0.00           H  \nATOM   1141  HZ3 LYS A  74       4.605  -8.592  16.011  1.00  0.00           H  \nATOM   1142  N   ILE A  75       8.223  -2.367  13.125  1.00  0.00           N  \nATOM   1143  CA  ILE A  75       8.952  -1.156  12.790  1.00  0.00           C  \nATOM   1144  C   ILE A  75       8.378   0.017  13.588  1.00  0.00           C  \nATOM   1145  O   ILE A  75       9.117   0.906  14.009  1.00  0.00           O  \nATOM   1146  CB  ILE A  75       8.949  -0.930  11.277  1.00  0.00           C  \nATOM   1147  CG1 ILE A  75      10.096  -0.008  10.858  1.00  0.00           C  \nATOM   1148  CG2 ILE A  75       7.591  -0.407  10.804  1.00  0.00           C  \nATOM   1149  CD1 ILE A  75       9.673   1.461  10.930  1.00  0.00           C  \nATOM   1150  H   ILE A  75       7.781  -2.822  12.352  1.00  0.00           H  \nATOM   1151  HA  ILE A  75       9.989  -1.305  13.092  1.00  0.00           H  \nATOM   1152  HB  ILE A  75       9.112  -1.890  10.788  1.00  0.00           H  \nATOM   1153 HG12 ILE A  75      10.956  -0.174  11.506  1.00  0.00           H  \nATOM   1154 HG13 ILE A  75      10.410  -0.250   9.843  1.00  0.00           H  \nATOM   1155 HG21 ILE A  75       6.813  -1.121  11.073  1.00  0.00           H  \nATOM   1156 HG22 ILE A  75       7.384   0.551  11.280  1.00  0.00           H  \nATOM   1157 HG23 ILE A  75       7.608  -0.279   9.721  1.00  0.00           H  \nATOM   1158 HD11 ILE A  75       9.360   1.700  11.946  1.00  0.00           H  \nATOM   1159 HD12 ILE A  75      10.514   2.095  10.650  1.00  0.00           H  \nATOM   1160 HD13 ILE A  75       8.843   1.634  10.244  1.00  0.00           H  \nATOM   1161  N   GLY A  76       7.067  -0.020  13.773  1.00  0.00           N  \nATOM   1162  CA  GLY A  76       6.386   1.029  14.513  1.00  0.00           C  \nATOM   1163  C   GLY A  76       6.695   0.934  16.009  1.00  0.00           C  \nATOM   1164  O   GLY A  76       6.726   1.947  16.705  1.00  0.00           O  \nATOM   1165  H   GLY A  76       6.474  -0.747  13.428  1.00  0.00           H  \nATOM   1166  HA2 GLY A  76       6.694   2.004  14.136  1.00  0.00           H  \nATOM   1167  HA3 GLY A  76       5.310   0.951  14.355  1.00  0.00           H  \nATOM   1168  N   LEU A  77       6.916  -0.293  16.458  1.00  0.00           N  \nATOM   1169  CA  LEU A  77       7.222  -0.533  17.858  1.00  0.00           C  \nATOM   1170  C   LEU A  77       8.593   0.062  18.185  1.00  0.00           C  \nATOM   1171  O   LEU A  77       8.749   0.755  19.189  1.00  0.00           O  \nATOM   1172  CB  LEU A  77       7.104  -2.023  18.184  1.00  0.00           C  \nATOM   1173  CG  LEU A  77       5.885  -2.435  19.011  1.00  0.00           C  \nATOM   1174  CD1 LEU A  77       5.721  -3.956  19.026  1.00  0.00           C  \nATOM   1175  CD2 LEU A  77       5.958  -1.852  20.423  1.00  0.00           C  \nATOM   1176  H   LEU A  77       6.889  -1.111  15.884  1.00  0.00           H  \nATOM   1177  HA  LEU A  77       6.470  -0.013  18.451  1.00  0.00           H  \nATOM   1178  HB2 LEU A  77       7.088  -2.580  17.246  1.00  0.00           H  \nATOM   1179  HB3 LEU A  77       8.003  -2.330  18.720  1.00  0.00           H  \nATOM   1180  HG  LEU A  77       4.995  -2.020  18.537  1.00  0.00           H  \nATOM   1181 HD11 LEU A  77       6.684  -4.423  19.235  1.00  0.00           H  \nATOM   1182 HD12 LEU A  77       5.006  -4.236  19.800  1.00  0.00           H  \nATOM   1183 HD13 LEU A  77       5.356  -4.292  18.056  1.00  0.00           H  \nATOM   1184 HD21 LEU A  77       4.994  -1.974  20.917  1.00  0.00           H  \nATOM   1185 HD22 LEU A  77       6.727  -2.374  20.993  1.00  0.00           H  \nATOM   1186 HD23 LEU A  77       6.206  -0.792  20.366  1.00  0.00           H  \nATOM   1187  N   LYS A  78       9.552  -0.232  17.319  1.00  0.00           N  \nATOM   1188  CA  LYS A  78      10.905   0.265  17.504  1.00  0.00           C  \nATOM   1189  C   LYS A  78      10.911   1.787  17.343  1.00  0.00           C  \nATOM   1190  O   LYS A  78      11.618   2.488  18.065  1.00  0.00           O  \nATOM   1191  CB  LYS A  78      11.874  -0.456  16.565  1.00  0.00           C  \nATOM   1192  CG  LYS A  78      11.502  -0.212  15.101  1.00  0.00           C  \nATOM   1193  CD  LYS A  78      12.534  -0.839  14.161  1.00  0.00           C  \nATOM   1194  CE  LYS A  78      13.161   0.219  13.251  1.00  0.00           C  \nATOM   1195  NZ  LYS A  78      14.086  -0.412  12.284  1.00  0.00           N  \nATOM   1196  H   LYS A  78       9.417  -0.796  16.505  1.00  0.00           H  \nATOM   1197  HA  LYS A  78      11.205   0.024  18.524  1.00  0.00           H  \nATOM   1198  HB2 LYS A  78      12.891  -0.109  16.748  1.00  0.00           H  \nATOM   1199  HB3 LYS A  78      11.860  -1.526  16.774  1.00  0.00           H  \nATOM   1200  HG2 LYS A  78      10.517  -0.633  14.897  1.00  0.00           H  \nATOM   1201  HG3 LYS A  78      11.436   0.859  14.912  1.00  0.00           H  \nATOM   1202  HD2 LYS A  78      13.313  -1.329  14.746  1.00  0.00           H  \nATOM   1203  HD3 LYS A  78      12.058  -1.610  13.555  1.00  0.00           H  \nATOM   1204  HE2 LYS A  78      12.378   0.757  12.717  1.00  0.00           H  \nATOM   1205  HE3 LYS A  78      13.698   0.952  13.853  1.00  0.00           H  \nATOM   1206  HZ1 LYS A  78      14.852  -0.826  12.776  1.00  0.00           H  \nATOM   1207  HZ2 LYS A  78      13.600  -1.118  11.769  1.00  0.00           H  \nATOM   1208  HZ3 LYS A  78      14.432   0.282  11.652  1.00  0.00           H  \nATOM   1209  N   VAL A  79      10.115   2.253  16.392  1.00  0.00           N  \nATOM   1210  CA  VAL A  79      10.020   3.678  16.128  1.00  0.00           C  \nATOM   1211  C   VAL A  79       9.535   4.395  17.389  1.00  0.00           C  \nATOM   1212  O   VAL A  79      10.164   5.349  17.846  1.00  0.00           O  \nATOM   1213  CB  VAL A  79       9.120   3.927  14.915  1.00  0.00           C  \nATOM   1214  CG1 VAL A  79       8.059   4.984  15.228  1.00  0.00           C  \nATOM   1215  CG2 VAL A  79       9.946   4.324  13.691  1.00  0.00           C  \nATOM   1216  H   VAL A  79       9.543   1.676  15.809  1.00  0.00           H  \nATOM   1217  HA  VAL A  79      11.020   4.035  15.884  1.00  0.00           H  \nATOM   1218  HB  VAL A  79       8.605   2.994  14.684  1.00  0.00           H  \nATOM   1219 HG11 VAL A  79       7.330   4.573  15.926  1.00  0.00           H  \nATOM   1220 HG12 VAL A  79       8.536   5.857  15.673  1.00  0.00           H  \nATOM   1221 HG13 VAL A  79       7.555   5.276  14.306  1.00  0.00           H  \nATOM   1222 HG21 VAL A  79       9.877   5.401  13.540  1.00  0.00           H  \nATOM   1223 HG22 VAL A  79      10.988   4.045  13.850  1.00  0.00           H  \nATOM   1224 HG23 VAL A  79       9.563   3.808  12.811  1.00  0.00           H  \nATOM   1225  N   LEU A  80       8.420   3.910  17.916  1.00  0.00           N  \nATOM   1226  CA  LEU A  80       7.844   4.492  19.116  1.00  0.00           C  \nATOM   1227  C   LEU A  80       8.873   4.443  20.247  1.00  0.00           C  \nATOM   1228  O   LEU A  80       9.229   5.475  20.813  1.00  0.00           O  \nATOM   1229  CB  LEU A  80       6.519   3.809  19.460  1.00  0.00           C  \nATOM   1230  CG  LEU A  80       5.257   4.648  19.249  1.00  0.00           C  \nATOM   1231  CD1 LEU A  80       4.170   3.836  18.544  1.00  0.00           C  \nATOM   1232  CD2 LEU A  80       4.764   5.239  20.572  1.00  0.00           C  \nATOM   1233  H   LEU A  80       7.915   3.134  17.538  1.00  0.00           H  \nATOM   1234  HA  LEU A  80       7.621   5.537  18.897  1.00  0.00           H  \nATOM   1235  HB2 LEU A  80       6.433   2.903  18.860  1.00  0.00           H  \nATOM   1236  HB3 LEU A  80       6.554   3.498  20.504  1.00  0.00           H  \nATOM   1237  HG  LEU A  80       5.508   5.485  18.597  1.00  0.00           H  \nATOM   1238 HD11 LEU A  80       4.497   3.590  17.534  1.00  0.00           H  \nATOM   1239 HD12 LEU A  80       3.985   2.916  19.100  1.00  0.00           H  \nATOM   1240 HD13 LEU A  80       3.252   4.422  18.496  1.00  0.00           H  \nATOM   1241 HD21 LEU A  80       5.449   6.022  20.898  1.00  0.00           H  \nATOM   1242 HD22 LEU A  80       3.769   5.662  20.433  1.00  0.00           H  \nATOM   1243 HD23 LEU A  80       4.723   4.455  21.327  1.00  0.00           H  \nATOM   1244  N   TYR A  81       9.323   3.232  20.542  1.00  0.00           N  \nATOM   1245  CA  TYR A  81      10.304   3.034  21.595  1.00  0.00           C  \nATOM   1246  C   TYR A  81      11.506   3.963  21.407  1.00  0.00           C  \nATOM   1247  O   TYR A  81      12.120   4.395  22.381  1.00  0.00           O  \nATOM   1248  CB  TYR A  81      10.773   1.583  21.469  1.00  0.00           C  \nATOM   1249  CG  TYR A  81      12.143   1.315  22.094  1.00  0.00           C  \nATOM   1250  CD1 TYR A  81      12.499   1.942  23.271  1.00  0.00           C  \nATOM   1251  CD2 TYR A  81      13.022   0.445  21.482  1.00  0.00           C  \nATOM   1252  CE1 TYR A  81      13.789   1.689  23.859  1.00  0.00           C  \nATOM   1253  CE2 TYR A  81      14.312   0.192  22.070  1.00  0.00           C  \nATOM   1254  CZ  TYR A  81      14.632   0.827  23.230  1.00  0.00           C  \nATOM   1255  OH  TYR A  81      15.850   0.588  23.786  1.00  0.00           O  \nATOM   1256  H   TYR A  81       9.029   2.397  20.077  1.00  0.00           H  \nATOM   1257  HA  TYR A  81       9.826   3.260  22.548  1.00  0.00           H  \nATOM   1258  HB2 TYR A  81      10.036   0.932  21.939  1.00  0.00           H  \nATOM   1259  HB3 TYR A  81      10.807   1.314  20.413  1.00  0.00           H  \nATOM   1260  HD1 TYR A  81      11.805   2.629  23.754  1.00  0.00           H  \nATOM   1261  HD2 TYR A  81      12.741  -0.050  20.552  1.00  0.00           H  \nATOM   1262  HE1 TYR A  81      14.083   2.178  24.788  1.00  0.00           H  \nATOM   1263  HE2 TYR A  81      15.016  -0.492  21.597  1.00  0.00           H  \nATOM   1264  HH  TYR A  81      16.431   0.097  23.137  1.00  0.00           H  \nATOM   1265  N   LYS A  82      11.804   4.244  20.147  1.00  0.00           N  \nATOM   1266  CA  LYS A  82      12.921   5.114  19.818  1.00  0.00           C  \nATOM   1267  C   LYS A  82      12.535   6.565  20.113  1.00  0.00           C  \nATOM   1268  O   LYS A  82      13.383   7.371  20.493  1.00  0.00           O  \nATOM   1269  CB  LYS A  82      13.377   4.879  18.377  1.00  0.00           C  \nATOM   1270  CG  LYS A  82      14.186   6.069  17.857  1.00  0.00           C  \nATOM   1271  CD  LYS A  82      13.267   7.230  17.470  1.00  0.00           C  \nATOM   1272  CE  LYS A  82      13.718   7.872  16.157  1.00  0.00           C  \nATOM   1273  NZ  LYS A  82      12.987   9.138  15.921  1.00  0.00           N  \nATOM   1274  H   LYS A  82      11.300   3.888  19.360  1.00  0.00           H  \nATOM   1275  HA  LYS A  82      13.752   4.838  20.468  1.00  0.00           H  \nATOM   1276  HB2 LYS A  82      13.981   3.973  18.325  1.00  0.00           H  \nATOM   1277  HB3 LYS A  82      12.508   4.719  17.738  1.00  0.00           H  \nATOM   1278  HG2 LYS A  82      14.889   6.397  18.622  1.00  0.00           H  \nATOM   1279  HG3 LYS A  82      14.775   5.763  16.992  1.00  0.00           H  \nATOM   1280  HD2 LYS A  82      12.243   6.870  17.371  1.00  0.00           H  \nATOM   1281  HD3 LYS A  82      13.266   7.977  18.264  1.00  0.00           H  \nATOM   1282  HE2 LYS A  82      14.790   8.065  16.189  1.00  0.00           H  \nATOM   1283  HE3 LYS A  82      13.542   7.185  15.330  1.00  0.00           H  \nATOM   1284  HZ1 LYS A  82      12.704   9.527  16.798  1.00  0.00           H  \nATOM   1285  HZ2 LYS A  82      13.585   9.784  15.447  1.00  0.00           H  \nATOM   1286  HZ3 LYS A  82      12.180   8.957  15.360  1.00  0.00           H  \nATOM   1287  N   LEU A  83      11.256   6.853  19.927  1.00  0.00           N  \nATOM   1288  CA  LEU A  83      10.747   8.192  20.168  1.00  0.00           C  \nATOM   1289  C   LEU A  83      10.738   8.466  21.673  1.00  0.00           C  \nATOM   1290  O   LEU A  83      11.235   9.497  22.123  1.00  0.00           O  \nATOM   1291  CB  LEU A  83       9.383   8.376  19.501  1.00  0.00           C  \nATOM   1292  CG  LEU A  83       8.888   9.818  19.372  1.00  0.00           C  \nATOM   1293  CD1 LEU A  83       8.533  10.148  17.920  1.00  0.00           C  \nATOM   1294  CD2 LEU A  83       7.719  10.084  20.322  1.00  0.00           C  \nATOM   1295  H   LEU A  83      10.573   6.191  19.618  1.00  0.00           H  \nATOM   1296  HA  LEU A  83      11.435   8.893  19.694  1.00  0.00           H  \nATOM   1297  HB2 LEU A  83       9.426   7.937  18.504  1.00  0.00           H  \nATOM   1298  HB3 LEU A  83       8.644   7.809  20.067  1.00  0.00           H  \nATOM   1299  HG  LEU A  83       9.699  10.485  19.664  1.00  0.00           H  \nATOM   1300 HD11 LEU A  83       9.436  10.124  17.311  1.00  0.00           H  \nATOM   1301 HD12 LEU A  83       7.821   9.413  17.545  1.00  0.00           H  \nATOM   1302 HD13 LEU A  83       8.088  11.142  17.872  1.00  0.00           H  \nATOM   1303 HD21 LEU A  83       8.052   9.954  21.352  1.00  0.00           H  \nATOM   1304 HD22 LEU A  83       7.362  11.104  20.182  1.00  0.00           H  \nATOM   1305 HD23 LEU A  83       6.912   9.383  20.110  1.00  0.00           H  \nATOM   1306  N   MET A  84      10.166   7.525  22.410  1.00  0.00           N  \nATOM   1307  CA  MET A  84      10.085   7.652  23.855  1.00  0.00           C  \nATOM   1308  C   MET A  84      11.479   7.767  24.474  1.00  0.00           C  \nATOM   1309  O   MET A  84      11.793   8.762  25.127  1.00  0.00           O  \nATOM   1310  CB  MET A  84       9.368   6.431  24.435  1.00  0.00           C  \nATOM   1311  CG  MET A  84       7.852   6.638  24.442  1.00  0.00           C  \nATOM   1312  SD  MET A  84       7.369   7.569  25.887  1.00  0.00           S  \nATOM   1313  CE  MET A  84       6.201   8.701  25.150  1.00  0.00           C  \nATOM   1314  H   MET A  84       9.764   6.689  22.036  1.00  0.00           H  \nATOM   1315  HA  MET A  84       9.522   8.567  24.037  1.00  0.00           H  \nATOM   1316  HB2 MET A  84       9.615   5.547  23.849  1.00  0.00           H  \nATOM   1317  HB3 MET A  84       9.718   6.248  25.451  1.00  0.00           H  \nATOM   1318  HG2 MET A  84       7.545   7.167  23.539  1.00  0.00           H  \nATOM   1319  HG3 MET A  84       7.345   5.673  24.434  1.00  0.00           H  \nATOM   1320  HE1 MET A  84       5.297   8.736  25.758  1.00  0.00           H  \nATOM   1321  HE2 MET A  84       6.643   9.696  25.098  1.00  0.00           H  \nATOM   1322  HE3 MET A  84       5.951   8.361  24.145  1.00  0.00           H  \nATOM   1323  N   ASP A  85      12.280   6.735  24.248  1.00  0.00           N  \nATOM   1324  CA  ASP A  85      13.633   6.708  24.776  1.00  0.00           C  \nATOM   1325  C   ASP A  85      14.369   7.979  24.346  1.00  0.00           C  \nATOM   1326  O   ASP A  85      14.946   8.029  23.261  1.00  0.00           O  \nATOM   1327  CB  ASP A  85      14.411   5.506  24.236  1.00  0.00           C  \nATOM   1328  CG  ASP A  85      15.892   5.469  24.619  1.00  0.00           C  \nATOM   1329  OD1 ASP A  85      16.366   6.496  25.151  1.00  0.00           O  \nATOM   1330  OD2 ASP A  85      16.516   4.415  24.371  1.00  0.00           O  \nATOM   1331  H   ASP A  85      12.017   5.930  23.717  1.00  0.00           H  \nATOM   1332  HA  ASP A  85      13.518   6.639  25.858  1.00  0.00           H  \nATOM   1333  HB2 ASP A  85      13.935   4.594  24.596  1.00  0.00           H  \nATOM   1334  HB3 ASP A  85      14.332   5.500  23.149  1.00  0.00           H  \nATOM   1335  N   VAL A  86      14.324   8.975  25.219  1.00  0.00           N  \nATOM   1336  CA  VAL A  86      14.979  10.242  24.943  1.00  0.00           C  \nATOM   1337  C   VAL A  86      16.266  10.336  25.767  1.00  0.00           C  \nATOM   1338  O   VAL A  86      17.007  11.311  25.657  1.00  0.00           O  \nATOM   1339  CB  VAL A  86      14.014  11.399  25.209  1.00  0.00           C  \nATOM   1340  CG1 VAL A  86      13.729  11.543  26.705  1.00  0.00           C  \nATOM   1341  CG2 VAL A  86      14.552  12.707  24.624  1.00  0.00           C  \nATOM   1342  H   VAL A  86      13.852   8.926  26.099  1.00  0.00           H  \nATOM   1343  HA  VAL A  86      15.239  10.255  23.885  1.00  0.00           H  \nATOM   1344  HB  VAL A  86      13.072  11.171  24.709  1.00  0.00           H  \nATOM   1345 HG11 VAL A  86      14.438  12.246  27.144  1.00  0.00           H  \nATOM   1346 HG12 VAL A  86      12.714  11.914  26.848  1.00  0.00           H  \nATOM   1347 HG13 VAL A  86      13.833  10.572  27.190  1.00  0.00           H  \nATOM   1348 HG21 VAL A  86      14.994  12.513  23.647  1.00  0.00           H  \nATOM   1349 HG22 VAL A  86      13.734  13.420  24.518  1.00  0.00           H  \nATOM   1350 HG23 VAL A  86      15.309  13.118  25.291  1.00  0.00           H  \nATOM   1351  N   ASP A  87      16.491   9.308  26.572  1.00  0.00           N  \nATOM   1352  CA  ASP A  87      17.675   9.263  27.414  1.00  0.00           C  \nATOM   1353  C   ASP A  87      18.885   8.870  26.563  1.00  0.00           C  \nATOM   1354  O   ASP A  87      20.011   9.268  26.858  1.00  0.00           O  \nATOM   1355  CB  ASP A  87      17.521   8.223  28.525  1.00  0.00           C  \nATOM   1356  CG  ASP A  87      17.030   6.850  28.062  1.00  0.00           C  \nATOM   1357  OD1 ASP A  87      17.894   6.050  27.642  1.00  0.00           O  \nATOM   1358  OD2 ASP A  87      15.802   6.630  28.140  1.00  0.00           O  \nATOM   1359  H   ASP A  87      15.883   8.519  26.655  1.00  0.00           H  \nATOM   1360  HA  ASP A  87      17.767  10.265  27.833  1.00  0.00           H  \nATOM   1361  HB2 ASP A  87      18.483   8.099  29.022  1.00  0.00           H  \nATOM   1362  HB3 ASP A  87      16.825   8.609  29.270  1.00  0.00           H  \nATOM   1363  N   GLY A  88      18.611   8.095  25.525  1.00  0.00           N  \nATOM   1364  CA  GLY A  88      19.663   7.644  24.629  1.00  0.00           C  \nATOM   1365  C   GLY A  88      20.444   6.480  25.243  1.00  0.00           C  \nATOM   1366  O   GLY A  88      21.605   6.636  25.619  1.00  0.00           O  \nATOM   1367  H   GLY A  88      17.693   7.775  25.292  1.00  0.00           H  \nATOM   1368  HA2 GLY A  88      19.229   7.335  23.679  1.00  0.00           H  \nATOM   1369  HA3 GLY A  88      20.342   8.470  24.416  1.00  0.00           H  \nATOM   1370  N   ASP A  89      19.775   5.339  25.325  1.00  0.00           N  \nATOM   1371  CA  ASP A  89      20.392   4.149  25.886  1.00  0.00           C  \nATOM   1372  C   ASP A  89      20.516   3.082  24.797  1.00  0.00           C  \nATOM   1373  O   ASP A  89      21.527   2.386  24.718  1.00  0.00           O  \nATOM   1374  CB  ASP A  89      19.542   3.571  27.019  1.00  0.00           C  \nATOM   1375  CG  ASP A  89      19.672   4.295  28.360  1.00  0.00           C  \nATOM   1376  OD1 ASP A  89      20.584   5.144  28.460  1.00  0.00           O  \nATOM   1377  OD2 ASP A  89      18.857   3.984  29.255  1.00  0.00           O  \nATOM   1378  H   ASP A  89      18.831   5.221  25.017  1.00  0.00           H  \nATOM   1379  HA  ASP A  89      21.361   4.478  26.261  1.00  0.00           H  \nATOM   1380  HB2 ASP A  89      18.495   3.589  26.714  1.00  0.00           H  \nATOM   1381  HB3 ASP A  89      19.814   2.525  27.161  1.00  0.00           H  \nATOM   1382  N   GLY A  90      19.474   2.988  23.984  1.00  0.00           N  \nATOM   1383  CA  GLY A  90      19.454   2.017  22.902  1.00  0.00           C  \nATOM   1384  C   GLY A  90      18.929   0.665  23.389  1.00  0.00           C  \nATOM   1385  O   GLY A  90      18.578  -0.195  22.582  1.00  0.00           O  \nATOM   1386  H   GLY A  90      18.656   3.558  24.055  1.00  0.00           H  \nATOM   1387  HA2 GLY A  90      18.826   2.385  22.091  1.00  0.00           H  \nATOM   1388  HA3 GLY A  90      20.459   1.897  22.498  1.00  0.00           H  \nATOM   1389  N   LYS A  91      18.891   0.520  24.705  1.00  0.00           N  \nATOM   1390  CA  LYS A  91      18.415  -0.713  25.308  1.00  0.00           C  \nATOM   1391  C   LYS A  91      17.259  -0.397  26.259  1.00  0.00           C  \nATOM   1392  O   LYS A  91      17.078   0.750  26.662  1.00  0.00           O  \nATOM   1393  CB  LYS A  91      19.569  -1.466  25.973  1.00  0.00           C  \nATOM   1394  CG  LYS A  91      20.846  -1.363  25.135  1.00  0.00           C  \nATOM   1395  CD  LYS A  91      21.909  -0.531  25.856  1.00  0.00           C  \nATOM   1396  CE  LYS A  91      22.994  -1.428  26.454  1.00  0.00           C  \nATOM   1397  NZ  LYS A  91      24.011  -0.614  27.156  1.00  0.00           N  \nATOM   1398  H   LYS A  91      19.178   1.224  25.354  1.00  0.00           H  \nATOM   1399  HA  LYS A  91      18.038  -1.346  24.504  1.00  0.00           H  \nATOM   1400  HB2 LYS A  91      19.749  -1.059  26.968  1.00  0.00           H  \nATOM   1401  HB3 LYS A  91      19.299  -2.514  26.102  1.00  0.00           H  \nATOM   1402  HG2 LYS A  91      21.234  -2.361  24.934  1.00  0.00           H  \nATOM   1403  HG3 LYS A  91      20.616  -0.909  24.171  1.00  0.00           H  \nATOM   1404  HD2 LYS A  91      22.359   0.174  25.157  1.00  0.00           H  \nATOM   1405  HD3 LYS A  91      21.442   0.057  26.646  1.00  0.00           H  \nATOM   1406  HE2 LYS A  91      22.546  -2.138  27.148  1.00  0.00           H  \nATOM   1407  HE3 LYS A  91      23.468  -2.011  25.664  1.00  0.00           H  \nATOM   1408  HZ1 LYS A  91      24.907  -0.767  26.739  1.00  0.00           H  \nATOM   1409  HZ2 LYS A  91      23.771   0.354  27.083  1.00  0.00           H  \nATOM   1410  HZ3 LYS A  91      24.042  -0.878  28.120  1.00  0.00           H  \nATOM   1411  N   LEU A  92      16.505  -1.436  26.589  1.00  0.00           N  \nATOM   1412  CA  LEU A  92      15.371  -1.284  27.485  1.00  0.00           C  \nATOM   1413  C   LEU A  92      15.246  -2.533  28.359  1.00  0.00           C  \nATOM   1414  O   LEU A  92      15.826  -3.572  28.049  1.00  0.00           O  \nATOM   1415  CB  LEU A  92      14.103  -0.957  26.693  1.00  0.00           C  \nATOM   1416  CG  LEU A  92      13.441  -2.133  25.973  1.00  0.00           C  \nATOM   1417  CD1 LEU A  92      12.364  -2.776  26.849  1.00  0.00           C  \nATOM   1418  CD2 LEU A  92      12.889  -1.702  24.612  1.00  0.00           C  \nATOM   1419  H   LEU A  92      16.659  -2.367  26.256  1.00  0.00           H  \nATOM   1420  HA  LEU A  92      15.577  -0.430  28.130  1.00  0.00           H  \nATOM   1421  HB2 LEU A  92      13.376  -0.518  27.376  1.00  0.00           H  \nATOM   1422  HB3 LEU A  92      14.347  -0.194  25.954  1.00  0.00           H  \nATOM   1423  HG  LEU A  92      14.200  -2.892  25.787  1.00  0.00           H  \nATOM   1424 HD11 LEU A  92      12.809  -3.103  27.789  1.00  0.00           H  \nATOM   1425 HD12 LEU A  92      11.578  -2.049  27.053  1.00  0.00           H  \nATOM   1426 HD13 LEU A  92      11.939  -3.635  26.330  1.00  0.00           H  \nATOM   1427 HD21 LEU A  92      12.516  -2.576  24.078  1.00  0.00           H  \nATOM   1428 HD22 LEU A  92      12.077  -0.990  24.758  1.00  0.00           H  \nATOM   1429 HD23 LEU A  92      13.683  -1.232  24.031  1.00  0.00           H  \nATOM   1430  N   THR A  93      14.483  -2.391  29.433  1.00  0.00           N  \nATOM   1431  CA  THR A  93      14.274  -3.495  30.354  1.00  0.00           C  \nATOM   1432  C   THR A  93      12.784  -3.824  30.462  1.00  0.00           C  \nATOM   1433  O   THR A  93      11.952  -3.164  29.841  1.00  0.00           O  \nATOM   1434  CB  THR A  93      14.912  -3.121  31.693  1.00  0.00           C  \nATOM   1435  OG1 THR A  93      14.004  -2.181  32.260  1.00  0.00           O  \nATOM   1436  CG2 THR A  93      16.210  -2.329  31.523  1.00  0.00           C  \nATOM   1437  H   THR A  93      14.014  -1.542  29.677  1.00  0.00           H  \nATOM   1438  HA  THR A  93      14.768  -4.378  29.949  1.00  0.00           H  \nATOM   1439  HB  THR A  93      15.074  -4.007  32.308  1.00  0.00           H  \nATOM   1440  HG1 THR A  93      13.874  -1.411  31.635  1.00  0.00           H  \nATOM   1441 HG21 THR A  93      16.814  -2.423  32.425  1.00  0.00           H  \nATOM   1442 HG22 THR A  93      16.766  -2.722  30.671  1.00  0.00           H  \nATOM   1443 HG23 THR A  93      15.975  -1.279  31.350  1.00  0.00           H  \nATOM   1444  N   LYS A  94      12.492  -4.845  31.254  1.00  0.00           N  \nATOM   1445  CA  LYS A  94      11.116  -5.270  31.451  1.00  0.00           C  \nATOM   1446  C   LYS A  94      10.403  -4.269  32.362  1.00  0.00           C  \nATOM   1447  O   LYS A  94       9.194  -4.073  32.249  1.00  0.00           O  \nATOM   1448  CB  LYS A  94      11.069  -6.711  31.965  1.00  0.00           C  \nATOM   1449  CG  LYS A  94      11.551  -6.792  33.415  1.00  0.00           C  \nATOM   1450  CD  LYS A  94      11.986  -8.215  33.769  1.00  0.00           C  \nATOM   1451  CE  LYS A  94      11.722  -8.518  35.245  1.00  0.00           C  \nATOM   1452  NZ  LYS A  94      10.535  -9.390  35.390  1.00  0.00           N  \nATOM   1453  H   LYS A  94      13.174  -5.377  31.755  1.00  0.00           H  \nATOM   1454  HA  LYS A  94      10.628  -5.259  30.477  1.00  0.00           H  \nATOM   1455  HB2 LYS A  94      10.051  -7.093  31.895  1.00  0.00           H  \nATOM   1456  HB3 LYS A  94      11.691  -7.346  31.335  1.00  0.00           H  \nATOM   1457  HG2 LYS A  94      12.385  -6.105  33.563  1.00  0.00           H  \nATOM   1458  HG3 LYS A  94      10.753  -6.474  34.086  1.00  0.00           H  \nATOM   1459  HD2 LYS A  94      11.448  -8.930  33.145  1.00  0.00           H  \nATOM   1460  HD3 LYS A  94      13.047  -8.340  33.553  1.00  0.00           H  \nATOM   1461  HE2 LYS A  94      12.593  -9.003  35.685  1.00  0.00           H  \nATOM   1462  HE3 LYS A  94      11.566  -7.588  35.791  1.00  0.00           H  \nATOM   1463  HZ1 LYS A  94      10.828 -10.312  35.647  1.00  0.00           H  \nATOM   1464  HZ2 LYS A  94       9.935  -9.021  36.100  1.00  0.00           H  \nATOM   1465  HZ3 LYS A  94      10.040  -9.428  34.522  1.00  0.00           H  \nATOM   1466  N   GLU A  95      11.183  -3.661  33.244  1.00  0.00           N  \nATOM   1467  CA  GLU A  95      10.641  -2.684  34.174  1.00  0.00           C  \nATOM   1468  C   GLU A  95      10.345  -1.369  33.450  1.00  0.00           C  \nATOM   1469  O   GLU A  95       9.364  -0.694  33.758  1.00  0.00           O  \nATOM   1470  CB  GLU A  95      11.593  -2.460  35.351  1.00  0.00           C  \nATOM   1471  CG  GLU A  95      10.855  -1.856  36.546  1.00  0.00           C  \nATOM   1472  CD  GLU A  95      10.606  -2.909  37.627  1.00  0.00           C  \nATOM   1473  OE1 GLU A  95      10.484  -4.094  37.248  1.00  0.00           O  \nATOM   1474  OE2 GLU A  95      10.545  -2.507  38.809  1.00  0.00           O  \nATOM   1475  H   GLU A  95      12.166  -3.826  33.330  1.00  0.00           H  \nATOM   1476  HA  GLU A  95       9.713  -3.120  34.544  1.00  0.00           H  \nATOM   1477  HB2 GLU A  95      12.048  -3.408  35.641  1.00  0.00           H  \nATOM   1478  HB3 GLU A  95      12.404  -1.798  35.046  1.00  0.00           H  \nATOM   1479  HG2 GLU A  95      11.438  -1.034  36.961  1.00  0.00           H  \nATOM   1480  HG3 GLU A  95       9.904  -1.436  36.217  1.00  0.00           H  \nATOM   1481  N   GLU A  96      11.211  -1.045  32.502  1.00  0.00           N  \nATOM   1482  CA  GLU A  96      11.055   0.177  31.731  1.00  0.00           C  \nATOM   1483  C   GLU A  96       9.918   0.025  30.719  1.00  0.00           C  \nATOM   1484  O   GLU A  96       9.126   0.946  30.526  1.00  0.00           O  \nATOM   1485  CB  GLU A  96      12.363   0.556  31.033  1.00  0.00           C  \nATOM   1486  CG  GLU A  96      12.213   1.871  30.265  1.00  0.00           C  \nATOM   1487  CD  GLU A  96      13.495   2.209  29.501  1.00  0.00           C  \nATOM   1488  OE1 GLU A  96      13.597   1.762  28.338  1.00  0.00           O  \nATOM   1489  OE2 GLU A  96      14.343   2.906  30.098  1.00  0.00           O  \nATOM   1490  H   GLU A  96      12.007  -1.600  32.257  1.00  0.00           H  \nATOM   1491  HA  GLU A  96      10.802   0.948  32.459  1.00  0.00           H  \nATOM   1492  HB2 GLU A  96      13.159   0.652  31.772  1.00  0.00           H  \nATOM   1493  HB3 GLU A  96      12.657  -0.238  30.348  1.00  0.00           H  \nATOM   1494  HG2 GLU A  96      11.379   1.795  29.567  1.00  0.00           H  \nATOM   1495  HG3 GLU A  96      11.975   2.677  30.959  1.00  0.00           H  \nATOM   1496  N   VAL A  97       9.872  -1.146  30.100  1.00  0.00           N  \nATOM   1497  CA  VAL A  97       8.845  -1.430  29.113  1.00  0.00           C  \nATOM   1498  C   VAL A  97       7.477  -1.455  29.799  1.00  0.00           C  \nATOM   1499  O   VAL A  97       6.514  -0.881  29.292  1.00  0.00           O  \nATOM   1500  CB  VAL A  97       9.170  -2.734  28.380  1.00  0.00           C  \nATOM   1501  CG1 VAL A  97       8.489  -3.926  29.054  1.00  0.00           C  \nATOM   1502  CG2 VAL A  97       8.782  -2.643  26.903  1.00  0.00           C  \nATOM   1503  H   VAL A  97      10.519  -1.890  30.263  1.00  0.00           H  \nATOM   1504  HA  VAL A  97       8.856  -0.621  28.383  1.00  0.00           H  \nATOM   1505  HB  VAL A  97      10.248  -2.888  28.434  1.00  0.00           H  \nATOM   1506 HG11 VAL A  97       8.711  -4.835  28.495  1.00  0.00           H  \nATOM   1507 HG12 VAL A  97       8.860  -4.028  30.074  1.00  0.00           H  \nATOM   1508 HG13 VAL A  97       7.411  -3.766  29.074  1.00  0.00           H  \nATOM   1509 HG21 VAL A  97       7.795  -2.190  26.813  1.00  0.00           H  \nATOM   1510 HG22 VAL A  97       9.512  -2.032  26.372  1.00  0.00           H  \nATOM   1511 HG23 VAL A  97       8.763  -3.644  26.470  1.00  0.00           H  \nATOM   1512  N   THR A  98       7.436  -2.127  30.940  1.00  0.00           N  \nATOM   1513  CA  THR A  98       6.202  -2.234  31.700  1.00  0.00           C  \nATOM   1514  C   THR A  98       5.853  -0.891  32.344  1.00  0.00           C  \nATOM   1515  O   THR A  98       4.680  -0.581  32.545  1.00  0.00           O  \nATOM   1516  CB  THR A  98       6.369  -3.368  32.714  1.00  0.00           C  \nATOM   1517  OG1 THR A  98       5.033  -3.753  33.028  1.00  0.00           O  \nATOM   1518  CG2 THR A  98       6.940  -2.882  34.048  1.00  0.00           C  \nATOM   1519  H   THR A  98       8.223  -2.591  31.345  1.00  0.00           H  \nATOM   1520  HA  THR A  98       5.394  -2.478  31.011  1.00  0.00           H  \nATOM   1521  HB  THR A  98       6.976  -4.173  32.301  1.00  0.00           H  \nATOM   1522  HG1 THR A  98       4.590  -3.038  33.569  1.00  0.00           H  \nATOM   1523 HG21 THR A  98       6.279  -2.127  34.472  1.00  0.00           H  \nATOM   1524 HG22 THR A  98       7.020  -3.723  34.737  1.00  0.00           H  \nATOM   1525 HG23 THR A  98       7.927  -2.451  33.885  1.00  0.00           H  \nATOM   1526  N   SER A  99       6.894  -0.129  32.649  1.00  0.00           N  \nATOM   1527  CA  SER A  99       6.712   1.174  33.265  1.00  0.00           C  \nATOM   1528  C   SER A  99       6.005   2.119  32.292  1.00  0.00           C  \nATOM   1529  O   SER A  99       5.088   2.842  32.678  1.00  0.00           O  \nATOM   1530  CB  SER A  99       8.053   1.768  33.703  1.00  0.00           C  \nATOM   1531  OG  SER A  99       8.075   3.187  33.575  1.00  0.00           O  \nATOM   1532  H   SER A  99       7.845  -0.389  32.482  1.00  0.00           H  \nATOM   1533  HA  SER A  99       6.092   0.994  34.143  1.00  0.00           H  \nATOM   1534  HB2 SER A  99       8.250   1.494  34.739  1.00  0.00           H  \nATOM   1535  HB3 SER A  99       8.854   1.337  33.102  1.00  0.00           H  \nATOM   1536  HG  SER A  99       8.037   3.612  34.479  1.00  0.00           H  \nATOM   1537  N   PHE A 100       6.459   2.084  31.047  1.00  0.00           N  \nATOM   1538  CA  PHE A 100       5.882   2.929  30.016  1.00  0.00           C  \nATOM   1539  C   PHE A 100       4.520   2.393  29.567  1.00  0.00           C  \nATOM   1540  O   PHE A 100       3.551   3.146  29.480  1.00  0.00           O  \nATOM   1541  CB  PHE A 100       6.843   2.903  28.826  1.00  0.00           C  \nATOM   1542  CG  PHE A 100       6.186   3.240  27.487  1.00  0.00           C  \nATOM   1543  CD1 PHE A 100       5.366   4.320  27.384  1.00  0.00           C  \nATOM   1544  CD2 PHE A 100       6.420   2.459  26.398  1.00  0.00           C  \nATOM   1545  CE1 PHE A 100       4.755   4.633  26.141  1.00  0.00           C  \nATOM   1546  CE2 PHE A 100       5.810   2.772  25.155  1.00  0.00           C  \nATOM   1547  CZ  PHE A 100       4.990   3.852  25.053  1.00  0.00           C  \nATOM   1548  H   PHE A 100       7.206   1.493  30.741  1.00  0.00           H  \nATOM   1549  HA  PHE A 100       5.756   3.923  30.444  1.00  0.00           H  \nATOM   1550  HB2 PHE A 100       7.652   3.610  29.010  1.00  0.00           H  \nATOM   1551  HB3 PHE A 100       7.295   1.913  28.758  1.00  0.00           H  \nATOM   1552  HD1 PHE A 100       5.178   4.946  28.257  1.00  0.00           H  \nATOM   1553  HD2 PHE A 100       7.077   1.593  26.480  1.00  0.00           H  \nATOM   1554  HE1 PHE A 100       4.098   5.499  26.059  1.00  0.00           H  \nATOM   1555  HE2 PHE A 100       5.998   2.146  24.282  1.00  0.00           H  \nATOM   1556  HZ  PHE A 100       4.521   4.092  24.098  1.00  0.00           H  \nATOM   1557  N   PHE A 101       4.490   1.097  29.295  1.00  0.00           N  \nATOM   1558  CA  PHE A 101       3.264   0.452  28.858  1.00  0.00           C  \nATOM   1559  C   PHE A 101       2.169   0.580  29.919  1.00  0.00           C  \nATOM   1560  O   PHE A 101       1.007   0.815  29.592  1.00  0.00           O  \nATOM   1561  CB  PHE A 101       3.586  -1.029  28.651  1.00  0.00           C  \nATOM   1562  CG  PHE A 101       4.041  -1.374  27.232  1.00  0.00           C  \nATOM   1563  CD1 PHE A 101       3.358  -0.887  26.161  1.00  0.00           C  \nATOM   1564  CD2 PHE A 101       5.128  -2.169  27.040  1.00  0.00           C  \nATOM   1565  CE1 PHE A 101       3.780  -1.208  24.844  1.00  0.00           C  \nATOM   1566  CE2 PHE A 101       5.550  -2.489  25.723  1.00  0.00           C  \nATOM   1567  CZ  PHE A 101       4.867  -2.002  24.652  1.00  0.00           C  \nATOM   1568  H   PHE A 101       5.283   0.492  29.368  1.00  0.00           H  \nATOM   1569  HA  PHE A 101       2.943   0.953  27.945  1.00  0.00           H  \nATOM   1570  HB2 PHE A 101       4.367  -1.322  29.353  1.00  0.00           H  \nATOM   1571  HB3 PHE A 101       2.702  -1.620  28.892  1.00  0.00           H  \nATOM   1572  HD1 PHE A 101       2.486  -0.251  26.315  1.00  0.00           H  \nATOM   1573  HD2 PHE A 101       5.676  -2.559  27.898  1.00  0.00           H  \nATOM   1574  HE1 PHE A 101       3.232  -0.818  23.986  1.00  0.00           H  \nATOM   1575  HE2 PHE A 101       6.422  -3.126  25.570  1.00  0.00           H  \nATOM   1576  HZ  PHE A 101       5.191  -2.248  23.641  1.00  0.00           H  \nATOM   1577  N   LYS A 102       2.579   0.421  31.169  1.00  0.00           N  \nATOM   1578  CA  LYS A 102       1.647   0.516  32.280  1.00  0.00           C  \nATOM   1579  C   LYS A 102       0.742   1.733  32.078  1.00  0.00           C  \nATOM   1580  O   LYS A 102      -0.429   1.709  32.454  1.00  0.00           O  \nATOM   1581  CB  LYS A 102       2.401   0.524  33.611  1.00  0.00           C  \nATOM   1582  CG  LYS A 102       1.474   0.918  34.763  1.00  0.00           C  \nATOM   1583  CD  LYS A 102       1.679   2.383  35.156  1.00  0.00           C  \nATOM   1584  CE  LYS A 102       2.197   2.498  36.591  1.00  0.00           C  \nATOM   1585  NZ  LYS A 102       3.088   3.672  36.729  1.00  0.00           N  \nATOM   1586  H   LYS A 102       3.526   0.231  31.426  1.00  0.00           H  \nATOM   1587  HA  LYS A 102       1.029  -0.381  32.265  1.00  0.00           H  \nATOM   1588  HB2 LYS A 102       2.823  -0.462  33.801  1.00  0.00           H  \nATOM   1589  HB3 LYS A 102       3.236   1.223  33.557  1.00  0.00           H  \nATOM   1590  HG2 LYS A 102       0.436   0.759  34.471  1.00  0.00           H  \nATOM   1591  HG3 LYS A 102       1.665   0.277  35.624  1.00  0.00           H  \nATOM   1592  HD2 LYS A 102       2.387   2.851  34.472  1.00  0.00           H  \nATOM   1593  HD3 LYS A 102       0.738   2.924  35.060  1.00  0.00           H  \nATOM   1594  HE2 LYS A 102       1.357   2.588  37.281  1.00  0.00           H  \nATOM   1595  HE3 LYS A 102       2.736   1.591  36.864  1.00  0.00           H  \nATOM   1596  HZ1 LYS A 102       3.790   3.479  37.414  1.00  0.00           H  \nATOM   1597  HZ2 LYS A 102       3.523   3.864  35.849  1.00  0.00           H  \nATOM   1598  HZ3 LYS A 102       2.551   4.466  37.016  1.00  0.00           H  \nATOM   1599  N   LYS A 103       1.319   2.768  31.485  1.00  0.00           N  \nATOM   1600  CA  LYS A 103       0.578   3.991  31.229  1.00  0.00           C  \nATOM   1601  C   LYS A 103      -0.722   3.652  30.498  1.00  0.00           C  \nATOM   1602  O   LYS A 103      -1.766   4.242  30.773  1.00  0.00           O  \nATOM   1603  CB  LYS A 103       1.455   5.003  30.487  1.00  0.00           C  \nATOM   1604  CG  LYS A 103       0.801   6.386  30.469  1.00  0.00           C  \nATOM   1605  CD  LYS A 103      -0.153   6.527  29.282  1.00  0.00           C  \nATOM   1606  CE  LYS A 103       0.118   7.820  28.511  1.00  0.00           C  \nATOM   1607  NZ  LYS A 103      -1.155   8.476  28.135  1.00  0.00           N  \nATOM   1608  H   LYS A 103       2.272   2.779  31.182  1.00  0.00           H  \nATOM   1609  HA  LYS A 103       0.328   4.430  32.194  1.00  0.00           H  \nATOM   1610  HB2 LYS A 103       2.432   5.064  30.966  1.00  0.00           H  \nATOM   1611  HB3 LYS A 103       1.623   4.663  29.465  1.00  0.00           H  \nATOM   1612  HG2 LYS A 103       0.256   6.546  31.399  1.00  0.00           H  \nATOM   1613  HG3 LYS A 103       1.571   7.155  30.415  1.00  0.00           H  \nATOM   1614  HD2 LYS A 103      -0.039   5.672  28.615  1.00  0.00           H  \nATOM   1615  HD3 LYS A 103      -1.184   6.519  29.636  1.00  0.00           H  \nATOM   1616  HE2 LYS A 103       0.715   8.497  29.122  1.00  0.00           H  \nATOM   1617  HE3 LYS A 103       0.700   7.602  27.615  1.00  0.00           H  \nATOM   1618  HZ1 LYS A 103      -1.835   7.778  27.910  1.00  0.00           H  \nATOM   1619  HZ2 LYS A 103      -1.484   9.028  28.901  1.00  0.00           H  \nATOM   1620  HZ3 LYS A 103      -1.005   9.064  27.340  1.00  0.00           H  \nATOM   1621  N   HIS A 104      -0.618   2.702  29.580  1.00  0.00           N  \nATOM   1622  CA  HIS A 104      -1.772   2.277  28.808  1.00  0.00           C  \nATOM   1623  C   HIS A 104      -2.737   1.505  29.710  1.00  0.00           C  \nATOM   1624  O   HIS A 104      -3.920   1.836  29.787  1.00  0.00           O  \nATOM   1625  CB  HIS A 104      -1.339   1.475  27.579  1.00  0.00           C  \nATOM   1626  CG  HIS A 104      -1.686   2.130  26.264  1.00  0.00           C  \nATOM   1627  ND1 HIS A 104      -1.773   1.426  25.075  1.00  0.00           N  \nATOM   1628  CD2 HIS A 104      -1.968   3.430  25.963  1.00  0.00           C  \nATOM   1629  CE1 HIS A 104      -2.092   2.275  24.109  1.00  0.00           C  \nATOM   1630  NE2 HIS A 104      -2.212   3.516  24.661  1.00  0.00           N  \nATOM   1631  H   HIS A 104       0.235   2.227  29.362  1.00  0.00           H  \nATOM   1632  HA  HIS A 104      -2.264   3.184  28.456  1.00  0.00           H  \nATOM   1633  HB2 HIS A 104      -0.261   1.318  27.622  1.00  0.00           H  \nATOM   1634  HB3 HIS A 104      -1.806   0.491  27.617  1.00  0.00           H  \nATOM   1635  HD1 HIS A 104      -1.620   0.444  24.964  1.00  0.00           H  \nATOM   1636  HD2 HIS A 104      -1.989   4.258  26.672  1.00  0.00           H  \nATOM   1637  HE1 HIS A 104      -2.234   2.025  23.058  1.00  0.00           H  \nATOM   1638  N   GLY A 105      -2.197   0.491  30.369  1.00  0.00           N  \nATOM   1639  CA  GLY A 105      -2.995  -0.331  31.263  1.00  0.00           C  \nATOM   1640  C   GLY A 105      -2.900  -1.809  30.880  1.00  0.00           C  \nATOM   1641  O   GLY A 105      -3.523  -2.659  31.514  1.00  0.00           O  \nATOM   1642  H   GLY A 105      -1.235   0.229  30.301  1.00  0.00           H  \nATOM   1643  HA2 GLY A 105      -2.655  -0.194  32.289  1.00  0.00           H  \nATOM   1644  HA3 GLY A 105      -4.036  -0.009  31.227  1.00  0.00           H  \nATOM   1645  N   ILE A 106      -2.115  -2.070  29.846  1.00  0.00           N  \nATOM   1646  CA  ILE A 106      -1.930  -3.431  29.371  1.00  0.00           C  \nATOM   1647  C   ILE A 106      -0.696  -4.037  30.044  1.00  0.00           C  \nATOM   1648  O   ILE A 106       0.410  -3.949  29.515  1.00  0.00           O  \nATOM   1649  CB  ILE A 106      -1.875  -3.463  27.842  1.00  0.00           C  \nATOM   1650  CG1 ILE A 106      -3.277  -3.597  27.246  1.00  0.00           C  \nATOM   1651  CG2 ILE A 106      -0.934  -4.565  27.350  1.00  0.00           C  \nATOM   1652  CD1 ILE A 106      -3.577  -2.444  26.286  1.00  0.00           C  \nATOM   1653  H   ILE A 106      -1.612  -1.373  29.336  1.00  0.00           H  \nATOM   1654  HA  ILE A 106      -2.806  -4.004  29.674  1.00  0.00           H  \nATOM   1655  HB  ILE A 106      -1.467  -2.513  27.496  1.00  0.00           H  \nATOM   1656 HG12 ILE A 106      -3.362  -4.546  26.717  1.00  0.00           H  \nATOM   1657 HG13 ILE A 106      -4.017  -3.610  28.046  1.00  0.00           H  \nATOM   1658 HG21 ILE A 106      -0.899  -5.368  28.087  1.00  0.00           H  \nATOM   1659 HG22 ILE A 106      -1.300  -4.958  26.402  1.00  0.00           H  \nATOM   1660 HG23 ILE A 106       0.066  -4.154  27.212  1.00  0.00           H  \nATOM   1661 HD11 ILE A 106      -4.289  -2.776  25.530  1.00  0.00           H  \nATOM   1662 HD12 ILE A 106      -4.002  -1.609  26.843  1.00  0.00           H  \nATOM   1663 HD13 ILE A 106      -2.655  -2.125  25.801  1.00  0.00           H  \nATOM   1664  N   GLU A 107      -0.929  -4.641  31.200  1.00  0.00           N  \nATOM   1665  CA  GLU A 107       0.149  -5.262  31.950  1.00  0.00           C  \nATOM   1666  C   GLU A 107       0.580  -6.567  31.278  1.00  0.00           C  \nATOM   1667  O   GLU A 107       1.732  -6.981  31.400  1.00  0.00           O  \nATOM   1668  CB  GLU A 107      -0.263  -5.504  33.404  1.00  0.00           C  \nATOM   1669  CG  GLU A 107      -1.339  -6.587  33.496  1.00  0.00           C  \nATOM   1670  CD  GLU A 107      -1.588  -6.990  34.951  1.00  0.00           C  \nATOM   1671  OE1 GLU A 107      -0.597  -7.368  35.613  1.00  0.00           O  \nATOM   1672  OE2 GLU A 107      -2.764  -6.912  35.368  1.00  0.00           O  \nATOM   1673  H   GLU A 107      -1.832  -4.709  31.624  1.00  0.00           H  \nATOM   1674  HA  GLU A 107       0.970  -4.545  31.926  1.00  0.00           H  \nATOM   1675  HB2 GLU A 107       0.608  -5.801  33.988  1.00  0.00           H  \nATOM   1676  HB3 GLU A 107      -0.637  -4.577  33.839  1.00  0.00           H  \nATOM   1677  HG2 GLU A 107      -2.265  -6.224  33.052  1.00  0.00           H  \nATOM   1678  HG3 GLU A 107      -1.031  -7.461  32.921  1.00  0.00           H  \nATOM   1679  N   LYS A 108      -0.368  -7.179  30.584  1.00  0.00           N  \nATOM   1680  CA  LYS A 108      -0.101  -8.429  29.892  1.00  0.00           C  \nATOM   1681  C   LYS A 108       1.226  -8.316  29.138  1.00  0.00           C  \nATOM   1682  O   LYS A 108       1.917  -9.313  28.938  1.00  0.00           O  \nATOM   1683  CB  LYS A 108      -1.284  -8.812  29.001  1.00  0.00           C  \nATOM   1684  CG  LYS A 108      -1.528  -7.750  27.927  1.00  0.00           C  \nATOM   1685  CD  LYS A 108      -2.208  -8.358  26.699  1.00  0.00           C  \nATOM   1686  CE  LYS A 108      -1.177  -8.964  25.745  1.00  0.00           C  \nATOM   1687  NZ  LYS A 108      -1.843  -9.511  24.542  1.00  0.00           N  \nATOM   1688  H   LYS A 108      -1.302  -6.836  30.490  1.00  0.00           H  \nATOM   1689  HA  LYS A 108      -0.002  -9.207  30.650  1.00  0.00           H  \nATOM   1690  HB2 LYS A 108      -1.092  -9.775  28.528  1.00  0.00           H  \nATOM   1691  HB3 LYS A 108      -2.180  -8.930  29.611  1.00  0.00           H  \nATOM   1692  HG2 LYS A 108      -2.150  -6.952  28.334  1.00  0.00           H  \nATOM   1693  HG3 LYS A 108      -0.580  -7.298  27.636  1.00  0.00           H  \nATOM   1694  HD2 LYS A 108      -2.914  -9.127  27.014  1.00  0.00           H  \nATOM   1695  HD3 LYS A 108      -2.783  -7.591  26.180  1.00  0.00           H  \nATOM   1696  HE2 LYS A 108      -0.453  -8.204  25.453  1.00  0.00           H  \nATOM   1697  HE3 LYS A 108      -0.624  -9.754  26.252  1.00  0.00           H  \nATOM   1698  HZ1 LYS A 108      -1.670  -8.909  23.762  1.00  0.00           H  \nATOM   1699  HZ2 LYS A 108      -1.482 -10.422  24.346  1.00  0.00           H  \nATOM   1700  HZ3 LYS A 108      -2.828  -9.571  24.704  1.00  0.00           H  \nATOM   1701  N   VAL A 109       1.541  -7.092  28.740  1.00  0.00           N  \nATOM   1702  CA  VAL A 109       2.772  -6.836  28.012  1.00  0.00           C  \nATOM   1703  C   VAL A 109       3.963  -7.302  28.853  1.00  0.00           C  \nATOM   1704  O   VAL A 109       4.950  -7.800  28.314  1.00  0.00           O  \nATOM   1705  CB  VAL A 109       2.855  -5.357  27.628  1.00  0.00           C  \nATOM   1706  CG1 VAL A 109       3.553  -4.545  28.720  1.00  0.00           C  \nATOM   1707  CG2 VAL A 109       3.556  -5.179  26.280  1.00  0.00           C  \nATOM   1708  H   VAL A 109       0.973  -6.286  28.907  1.00  0.00           H  \nATOM   1709  HA  VAL A 109       2.740  -7.422  27.094  1.00  0.00           H  \nATOM   1710  HB  VAL A 109       1.837  -4.979  27.528  1.00  0.00           H  \nATOM   1711 HG11 VAL A 109       4.590  -4.869  28.807  1.00  0.00           H  \nATOM   1712 HG12 VAL A 109       3.524  -3.486  28.461  1.00  0.00           H  \nATOM   1713 HG13 VAL A 109       3.043  -4.700  29.671  1.00  0.00           H  \nATOM   1714 HG21 VAL A 109       4.274  -4.363  26.349  1.00  0.00           H  \nATOM   1715 HG22 VAL A 109       4.077  -6.100  26.017  1.00  0.00           H  \nATOM   1716 HG23 VAL A 109       2.816  -4.949  25.513  1.00  0.00           H  \nATOM   1717  N   ALA A 110       3.830  -7.125  30.159  1.00  0.00           N  \nATOM   1718  CA  ALA A 110       4.883  -7.521  31.079  1.00  0.00           C  \nATOM   1719  C   ALA A 110       4.983  -9.048  31.105  1.00  0.00           C  \nATOM   1720  O   ALA A 110       6.072  -9.599  31.251  1.00  0.00           O  \nATOM   1721  CB  ALA A 110       4.601  -6.931  32.462  1.00  0.00           C  \nATOM   1722  H   ALA A 110       3.024  -6.719  30.589  1.00  0.00           H  \nATOM   1723  HA  ALA A 110       5.821  -7.110  30.706  1.00  0.00           H  \nATOM   1724  HB1 ALA A 110       3.806  -7.500  32.944  1.00  0.00           H  \nATOM   1725  HB2 ALA A 110       5.504  -6.983  33.070  1.00  0.00           H  \nATOM   1726  HB3 ALA A 110       4.292  -5.892  32.358  1.00  0.00           H  \nATOM   1727  N   GLU A 111       3.831  -9.687  30.961  1.00  0.00           N  \nATOM   1728  CA  GLU A 111       3.775 -11.139  30.966  1.00  0.00           C  \nATOM   1729  C   GLU A 111       4.396 -11.698  29.684  1.00  0.00           C  \nATOM   1730  O   GLU A 111       5.189 -12.637  29.733  1.00  0.00           O  \nATOM   1731  CB  GLU A 111       2.338 -11.632  31.140  1.00  0.00           C  \nATOM   1732  CG  GLU A 111       2.307 -13.013  31.796  1.00  0.00           C  \nATOM   1733  CD  GLU A 111       2.044 -12.900  33.300  1.00  0.00           C  \nATOM   1734  OE1 GLU A 111       2.821 -12.177  33.959  1.00  0.00           O  \nATOM   1735  OE2 GLU A 111       1.072 -13.540  33.755  1.00  0.00           O  \nATOM   1736  H   GLU A 111       2.949  -9.230  30.842  1.00  0.00           H  \nATOM   1737  HA  GLU A 111       4.366 -11.448  31.828  1.00  0.00           H  \nATOM   1738  HB2 GLU A 111       1.778 -10.923  31.750  1.00  0.00           H  \nATOM   1739  HB3 GLU A 111       1.844 -11.676  30.169  1.00  0.00           H  \nATOM   1740  HG2 GLU A 111       1.532 -13.623  31.333  1.00  0.00           H  \nATOM   1741  HG3 GLU A 111       3.256 -13.521  31.627  1.00  0.00           H  \nATOM   1742  N   GLN A 112       4.010 -11.099  28.567  1.00  0.00           N  \nATOM   1743  CA  GLN A 112       4.518 -11.526  27.275  1.00  0.00           C  \nATOM   1744  C   GLN A 112       5.990 -11.134  27.128  1.00  0.00           C  \nATOM   1745  O   GLN A 112       6.758 -11.831  26.466  1.00  0.00           O  \nATOM   1746  CB  GLN A 112       3.680 -10.942  26.136  1.00  0.00           C  \nATOM   1747  CG  GLN A 112       4.222  -9.581  25.697  1.00  0.00           C  \nATOM   1748  CD  GLN A 112       3.324  -8.951  24.630  1.00  0.00           C  \nATOM   1749  OE1 GLN A 112       3.760  -8.590  23.549  1.00  0.00           O  \nATOM   1750  NE2 GLN A 112       2.049  -8.840  24.992  1.00  0.00           N  \nATOM   1751  H   GLN A 112       3.364 -10.336  28.537  1.00  0.00           H  \nATOM   1752  HA  GLN A 112       4.422 -12.611  27.270  1.00  0.00           H  \nATOM   1753  HB2 GLN A 112       3.682 -11.629  25.289  1.00  0.00           H  \nATOM   1754  HB3 GLN A 112       2.644 -10.838  26.459  1.00  0.00           H  \nATOM   1755  HG2 GLN A 112       4.289  -8.917  26.559  1.00  0.00           H  \nATOM   1756  HG3 GLN A 112       5.232  -9.697  25.305  1.00  0.00           H  \nATOM   1757 HE21 GLN A 112       1.757  -9.156  25.895  1.00  0.00           H  \nATOM   1758 HE22 GLN A 112       1.384  -8.440  24.362  1.00  0.00           H  \nATOM   1759  N   VAL A 113       6.339 -10.021  27.755  1.00  0.00           N  \nATOM   1760  CA  VAL A 113       7.705  -9.528  27.703  1.00  0.00           C  \nATOM   1761  C   VAL A 113       8.595 -10.411  28.580  1.00  0.00           C  \nATOM   1762  O   VAL A 113       9.745 -10.678  28.234  1.00  0.00           O  \nATOM   1763  CB  VAL A 113       7.745  -8.053  28.104  1.00  0.00           C  \nATOM   1764  CG1 VAL A 113       9.177  -7.607  28.410  1.00  0.00           C  \nATOM   1765  CG2 VAL A 113       7.116  -7.173  27.022  1.00  0.00           C  \nATOM   1766  H   VAL A 113       5.708  -9.460  28.292  1.00  0.00           H  \nATOM   1767  HA  VAL A 113       8.043  -9.607  26.669  1.00  0.00           H  \nATOM   1768  HB  VAL A 113       7.157  -7.937  29.014  1.00  0.00           H  \nATOM   1769 HG11 VAL A 113       9.868  -8.415  28.172  1.00  0.00           H  \nATOM   1770 HG12 VAL A 113       9.421  -6.731  27.809  1.00  0.00           H  \nATOM   1771 HG13 VAL A 113       9.261  -7.356  29.468  1.00  0.00           H  \nATOM   1772 HG21 VAL A 113       7.864  -6.939  26.264  1.00  0.00           H  \nATOM   1773 HG22 VAL A 113       6.285  -7.704  26.560  1.00  0.00           H  \nATOM   1774 HG23 VAL A 113       6.752  -6.249  27.471  1.00  0.00           H  \nATOM   1775  N   MET A 114       8.029 -10.840  29.699  1.00  0.00           N  \nATOM   1776  CA  MET A 114       8.757 -11.687  30.628  1.00  0.00           C  \nATOM   1777  C   MET A 114       8.939 -13.096  30.060  1.00  0.00           C  \nATOM   1778  O   MET A 114       9.972 -13.728  30.275  1.00  0.00           O  \nATOM   1779  CB  MET A 114       7.994 -11.765  31.952  1.00  0.00           C  \nATOM   1780  CG  MET A 114       8.863 -12.384  33.050  1.00  0.00           C  \nATOM   1781  SD  MET A 114       8.451 -14.108  33.255  1.00  0.00           S  \nATOM   1782  CE  MET A 114       7.087 -13.968  34.398  1.00  0.00           C  \nATOM   1783  H   MET A 114       7.094 -10.618  29.973  1.00  0.00           H  \nATOM   1784  HA  MET A 114       9.729 -11.212  30.758  1.00  0.00           H  \nATOM   1785  HB2 MET A 114       7.678 -10.767  32.254  1.00  0.00           H  \nATOM   1786  HB3 MET A 114       7.090 -12.360  31.821  1.00  0.00           H  \nATOM   1787  HG2 MET A 114       9.917 -12.281  32.792  1.00  0.00           H  \nATOM   1788  HG3 MET A 114       8.711 -11.851  33.989  1.00  0.00           H  \nATOM   1789  HE1 MET A 114       6.682 -12.957  34.357  1.00  0.00           H  \nATOM   1790  HE2 MET A 114       6.310 -14.682  34.126  1.00  0.00           H  \nATOM   1791  HE3 MET A 114       7.437 -14.179  35.408  1.00  0.00           H  \nATOM   1792  N   LYS A 115       7.919 -13.547  29.344  1.00  0.00           N  \nATOM   1793  CA  LYS A 115       7.953 -14.869  28.743  1.00  0.00           C  \nATOM   1794  C   LYS A 115       8.872 -14.845  27.520  1.00  0.00           C  \nATOM   1795  O   LYS A 115       9.538 -15.834  27.220  1.00  0.00           O  \nATOM   1796  CB  LYS A 115       6.535 -15.355  28.437  1.00  0.00           C  \nATOM   1797  CG  LYS A 115       6.059 -16.356  29.492  1.00  0.00           C  \nATOM   1798  CD  LYS A 115       6.880 -17.646  29.434  1.00  0.00           C  \nATOM   1799  CE  LYS A 115       6.070 -18.785  28.812  1.00  0.00           C  \nATOM   1800  NZ  LYS A 115       5.301 -19.503  29.853  1.00  0.00           N  \nATOM   1801  H   LYS A 115       7.082 -13.026  29.174  1.00  0.00           H  \nATOM   1802  HA  LYS A 115       8.376 -15.552  29.480  1.00  0.00           H  \nATOM   1803  HB2 LYS A 115       5.853 -14.505  28.404  1.00  0.00           H  \nATOM   1804  HB3 LYS A 115       6.510 -15.821  27.452  1.00  0.00           H  \nATOM   1805  HG2 LYS A 115       6.142 -15.912  30.484  1.00  0.00           H  \nATOM   1806  HG3 LYS A 115       5.005 -16.584  29.332  1.00  0.00           H  \nATOM   1807  HD2 LYS A 115       7.786 -17.478  28.851  1.00  0.00           H  \nATOM   1808  HD3 LYS A 115       7.195 -17.926  30.439  1.00  0.00           H  \nATOM   1809  HE2 LYS A 115       5.389 -18.386  28.060  1.00  0.00           H  \nATOM   1810  HE3 LYS A 115       6.738 -19.478  28.301  1.00  0.00           H  \nATOM   1811  HZ1 LYS A 115       5.922 -20.074  30.390  1.00  0.00           H  \nATOM   1812  HZ2 LYS A 115       4.854 -18.841  30.454  1.00  0.00           H  \nATOM   1813  HZ3 LYS A 115       4.610 -20.080  29.418  1.00  0.00           H  \nATOM   1814  N   ALA A 116       8.878 -13.704  26.847  1.00  0.00           N  \nATOM   1815  CA  ALA A 116       9.704 -13.537  25.663  1.00  0.00           C  \nATOM   1816  C   ALA A 116      11.180 -13.593  26.065  1.00  0.00           C  \nATOM   1817  O   ALA A 116      11.990 -14.215  25.380  1.00  0.00           O  \nATOM   1818  CB  ALA A 116       9.339 -12.226  24.965  1.00  0.00           C  \nATOM   1819  H   ALA A 116       8.333 -12.903  27.098  1.00  0.00           H  \nATOM   1820  HA  ALA A 116       9.488 -14.366  24.989  1.00  0.00           H  \nATOM   1821  HB1 ALA A 116       9.000 -11.502  25.706  1.00  0.00           H  \nATOM   1822  HB2 ALA A 116      10.214 -11.833  24.448  1.00  0.00           H  \nATOM   1823  HB3 ALA A 116       8.542 -12.408  24.244  1.00  0.00           H  \nATOM   1824  N   ASP A 117      11.483 -12.934  27.173  1.00  0.00           N  \nATOM   1825  CA  ASP A 117      12.847 -12.900  27.674  1.00  0.00           C  \nATOM   1826  C   ASP A 117      13.097 -14.133  28.545  1.00  0.00           C  \nATOM   1827  O   ASP A 117      13.003 -14.062  29.769  1.00  0.00           O  \nATOM   1828  CB  ASP A 117      13.088 -11.658  28.534  1.00  0.00           C  \nATOM   1829  CG  ASP A 117      14.530 -11.468  29.007  1.00  0.00           C  \nATOM   1830  OD1 ASP A 117      15.160 -12.497  29.335  1.00  0.00           O  \nATOM   1831  OD2 ASP A 117      14.971 -10.299  29.031  1.00  0.00           O  \nATOM   1832  H   ASP A 117      10.818 -12.430  27.725  1.00  0.00           H  \nATOM   1833  HA  ASP A 117      13.478 -12.882  26.785  1.00  0.00           H  \nATOM   1834  HB2 ASP A 117      12.789 -10.777  27.965  1.00  0.00           H  \nATOM   1835  HB3 ASP A 117      12.438 -11.708  29.408  1.00  0.00           H  \nATOM   1836  N   ALA A 118      13.409 -15.235  27.878  1.00  0.00           N  \nATOM   1837  CA  ALA A 118      13.673 -16.482  28.576  1.00  0.00           C  \nATOM   1838  C   ALA A 118      15.164 -16.570  28.909  1.00  0.00           C  \nATOM   1839  O   ALA A 118      15.804 -17.585  28.642  1.00  0.00           O  \nATOM   1840  CB  ALA A 118      13.196 -17.657  27.720  1.00  0.00           C  \nATOM   1841  H   ALA A 118      13.483 -15.284  26.882  1.00  0.00           H  \nATOM   1842  HA  ALA A 118      13.102 -16.470  29.505  1.00  0.00           H  \nATOM   1843  HB1 ALA A 118      12.254 -18.037  28.116  1.00  0.00           H  \nATOM   1844  HB2 ALA A 118      13.051 -17.323  26.693  1.00  0.00           H  \nATOM   1845  HB3 ALA A 118      13.945 -18.449  27.741  1.00  0.00           H  \nATOM   1846  N   ASN A 119      15.673 -15.492  29.487  1.00  0.00           N  \nATOM   1847  CA  ASN A 119      17.076 -15.435  29.859  1.00  0.00           C  \nATOM   1848  C   ASN A 119      17.192 -15.308  31.379  1.00  0.00           C  \nATOM   1849  O   ASN A 119      17.955 -16.038  32.010  1.00  0.00           O  \nATOM   1850  CB  ASN A 119      17.764 -14.221  29.230  1.00  0.00           C  \nATOM   1851  CG  ASN A 119      17.350 -14.054  27.767  1.00  0.00           C  \nATOM   1852  OD1 ASN A 119      16.990 -12.979  27.315  1.00  0.00           O  \nATOM   1853  ND2 ASN A 119      17.422 -15.174  27.053  1.00  0.00           N  \nATOM   1854  H   ASN A 119      15.145 -14.670  29.701  1.00  0.00           H  \nATOM   1855  HA  ASN A 119      17.510 -16.361  29.483  1.00  0.00           H  \nATOM   1856  HB2 ASN A 119      17.505 -13.322  29.790  1.00  0.00           H  \nATOM   1857  HB3 ASN A 119      18.845 -14.337  29.295  1.00  0.00           H  \nATOM   1858 HD21 ASN A 119      17.725 -16.023  27.485  1.00  0.00           H  \nATOM   1859 HD22 ASN A 119      17.172 -15.167  26.085  1.00  0.00           H  \nATOM   1860  N   GLY A 120      16.422 -14.377  31.924  1.00  0.00           N  \nATOM   1861  CA  GLY A 120      16.429 -14.146  33.358  1.00  0.00           C  \nATOM   1862  C   GLY A 120      17.203 -12.872  33.705  1.00  0.00           C  \nATOM   1863  O   GLY A 120      17.195 -12.429  34.853  1.00  0.00           O  \nATOM   1864  H   GLY A 120      15.804 -13.788  31.404  1.00  0.00           H  \nATOM   1865  HA2 GLY A 120      15.404 -14.063  33.722  1.00  0.00           H  \nATOM   1866  HA3 GLY A 120      16.879 -14.999  33.866  1.00  0.00           H  \nATOM   1867  N   ASP A 121      17.854 -12.319  32.692  1.00  0.00           N  \nATOM   1868  CA  ASP A 121      18.631 -11.105  32.875  1.00  0.00           C  \nATOM   1869  C   ASP A 121      17.701  -9.968  33.300  1.00  0.00           C  \nATOM   1870  O   ASP A 121      17.988  -9.251  34.258  1.00  0.00           O  \nATOM   1871  CB  ASP A 121      19.320 -10.691  31.573  1.00  0.00           C  \nATOM   1872  CG  ASP A 121      18.383 -10.499  30.379  1.00  0.00           C  \nATOM   1873  OD1 ASP A 121      17.300 -11.122  30.401  1.00  0.00           O  \nATOM   1874  OD2 ASP A 121      18.772  -9.734  29.470  1.00  0.00           O  \nATOM   1875  H   ASP A 121      17.855 -12.686  31.761  1.00  0.00           H  \nATOM   1876  HA  ASP A 121      19.368 -11.350  33.640  1.00  0.00           H  \nATOM   1877  HB2 ASP A 121      19.860  -9.760  31.746  1.00  0.00           H  \nATOM   1878  HB3 ASP A 121      20.062 -11.446  31.316  1.00  0.00           H  \nATOM   1879  N   GLY A 122      16.604  -9.838  32.568  1.00  0.00           N  \nATOM   1880  CA  GLY A 122      15.629  -8.800  32.858  1.00  0.00           C  \nATOM   1881  C   GLY A 122      15.914  -7.537  32.043  1.00  0.00           C  \nATOM   1882  O   GLY A 122      15.570  -6.433  32.461  1.00  0.00           O  \nATOM   1883  H   GLY A 122      16.377 -10.426  31.792  1.00  0.00           H  \nATOM   1884  HA2 GLY A 122      14.627  -9.164  32.633  1.00  0.00           H  \nATOM   1885  HA3 GLY A 122      15.651  -8.563  33.922  1.00  0.00           H  \nATOM   1886  N   TYR A 123      16.541  -7.743  30.893  1.00  0.00           N  \nATOM   1887  CA  TYR A 123      16.876  -6.635  30.015  1.00  0.00           C  \nATOM   1888  C   TYR A 123      16.911  -7.084  28.553  1.00  0.00           C  \nATOM   1889  O   TYR A 123      17.053  -8.273  28.269  1.00  0.00           O  \nATOM   1890  CB  TYR A 123      18.277  -6.178  30.429  1.00  0.00           C  \nATOM   1891  CG  TYR A 123      18.365  -5.671  31.870  1.00  0.00           C  \nATOM   1892  CD1 TYR A 123      18.352  -6.568  32.919  1.00  0.00           C  \nATOM   1893  CD2 TYR A 123      18.459  -4.317  32.121  1.00  0.00           C  \nATOM   1894  CE1 TYR A 123      18.435  -6.091  34.275  1.00  0.00           C  \nATOM   1895  CE2 TYR A 123      18.542  -3.840  33.477  1.00  0.00           C  \nATOM   1896  CZ  TYR A 123      18.526  -4.750  34.487  1.00  0.00           C  \nATOM   1897  OH  TYR A 123      18.605  -4.300  35.768  1.00  0.00           O  \nATOM   1898  H   TYR A 123      16.818  -8.644  30.560  1.00  0.00           H  \nATOM   1899  HA  TYR A 123      16.111  -5.868  30.134  1.00  0.00           H  \nATOM   1900  HB2 TYR A 123      18.971  -7.009  30.304  1.00  0.00           H  \nATOM   1901  HB3 TYR A 123      18.603  -5.385  29.756  1.00  0.00           H  \nATOM   1902  HD1 TYR A 123      18.278  -7.637  32.721  1.00  0.00           H  \nATOM   1903  HD2 TYR A 123      18.470  -3.609  31.292  1.00  0.00           H  \nATOM   1904  HE1 TYR A 123      18.425  -6.788  35.113  1.00  0.00           H  \nATOM   1905  HE2 TYR A 123      18.617  -2.773  33.689  1.00  0.00           H  \nATOM   1906  HH  TYR A 123      18.852  -3.331  35.775  1.00  0.00           H  \nATOM   1907  N   ILE A 124      16.778  -6.111  27.665  1.00  0.00           N  \nATOM   1908  CA  ILE A 124      16.793  -6.391  26.239  1.00  0.00           C  \nATOM   1909  C   ILE A 124      17.305  -5.162  25.487  1.00  0.00           C  \nATOM   1910  O   ILE A 124      17.363  -4.067  26.045  1.00  0.00           O  \nATOM   1911  CB  ILE A 124      15.416  -6.868  25.773  1.00  0.00           C  \nATOM   1912  CG1 ILE A 124      14.430  -5.701  25.690  1.00  0.00           C  \nATOM   1913  CG2 ILE A 124      14.895  -7.996  26.666  1.00  0.00           C  \nATOM   1914  CD1 ILE A 124      13.218  -6.067  24.831  1.00  0.00           C  \nATOM   1915  H   ILE A 124      16.663  -5.147  27.904  1.00  0.00           H  \nATOM   1916  HA  ILE A 124      17.491  -7.212  26.075  1.00  0.00           H  \nATOM   1917  HB  ILE A 124      15.518  -7.276  24.767  1.00  0.00           H  \nATOM   1918 HG12 ILE A 124      14.101  -5.427  26.693  1.00  0.00           H  \nATOM   1919 HG13 ILE A 124      14.929  -4.828  25.269  1.00  0.00           H  \nATOM   1920 HG21 ILE A 124      14.743  -7.618  27.677  1.00  0.00           H  \nATOM   1921 HG22 ILE A 124      13.949  -8.365  26.271  1.00  0.00           H  \nATOM   1922 HG23 ILE A 124      15.622  -8.808  26.687  1.00  0.00           H  \nATOM   1923 HD11 ILE A 124      13.149  -7.151  24.742  1.00  0.00           H  \nATOM   1924 HD12 ILE A 124      12.312  -5.682  25.300  1.00  0.00           H  \nATOM   1925 HD13 ILE A 124      13.329  -5.627  23.840  1.00  0.00           H  \nATOM   1926  N   THR A 125      17.663  -5.383  24.230  1.00  0.00           N  \nATOM   1927  CA  THR A 125      18.168  -4.306  23.395  1.00  0.00           C  \nATOM   1928  C   THR A 125      17.297  -4.146  22.148  1.00  0.00           C  \nATOM   1929  O   THR A 125      16.341  -4.894  21.953  1.00  0.00           O  \nATOM   1930  CB  THR A 125      19.635  -4.602  23.079  1.00  0.00           C  \nATOM   1931  OG1 THR A 125      19.577  -5.632  22.095  1.00  0.00           O  \nATOM   1932  CG2 THR A 125      20.369  -5.251  24.254  1.00  0.00           C  \nATOM   1933  H   THR A 125      17.612  -6.276  23.783  1.00  0.00           H  \nATOM   1934  HA  THR A 125      18.098  -3.374  23.957  1.00  0.00           H  \nATOM   1935  HB  THR A 125      20.151  -3.701  22.747  1.00  0.00           H  \nATOM   1936  HG1 THR A 125      20.275  -5.477  21.396  1.00  0.00           H  \nATOM   1937 HG21 THR A 125      21.402  -4.905  24.273  1.00  0.00           H  \nATOM   1938 HG22 THR A 125      19.877  -4.974  25.187  1.00  0.00           H  \nATOM   1939 HG23 THR A 125      20.350  -6.335  24.141  1.00  0.00           H  \nATOM   1940  N   LEU A 126      17.660  -3.164  21.334  1.00  0.00           N  \nATOM   1941  CA  LEU A 126      16.924  -2.896  20.111  1.00  0.00           C  \nATOM   1942  C   LEU A 126      17.177  -4.025  19.110  1.00  0.00           C  \nATOM   1943  O   LEU A 126      16.264  -4.451  18.405  1.00  0.00           O  \nATOM   1944  CB  LEU A 126      17.271  -1.507  19.571  1.00  0.00           C  \nATOM   1945  CG  LEU A 126      16.395  -0.995  18.425  1.00  0.00           C  \nATOM   1946  CD1 LEU A 126      15.307  -0.054  18.945  1.00  0.00           C  \nATOM   1947  CD2 LEU A 126      17.246  -0.341  17.335  1.00  0.00           C  \nATOM   1948  H   LEU A 126      18.439  -2.560  21.500  1.00  0.00           H  \nATOM   1949  HA  LEU A 126      15.864  -2.889  20.362  1.00  0.00           H  \nATOM   1950  HB2 LEU A 126      17.211  -0.794  20.393  1.00  0.00           H  \nATOM   1951  HB3 LEU A 126      18.307  -1.518  19.233  1.00  0.00           H  \nATOM   1952  HG  LEU A 126      15.893  -1.849  17.971  1.00  0.00           H  \nATOM   1953 HD11 LEU A 126      14.460  -0.640  19.303  1.00  0.00           H  \nATOM   1954 HD12 LEU A 126      15.705   0.546  19.763  1.00  0.00           H  \nATOM   1955 HD13 LEU A 126      14.979   0.602  18.139  1.00  0.00           H  \nATOM   1956 HD21 LEU A 126      18.251  -0.762  17.356  1.00  0.00           H  \nATOM   1957 HD22 LEU A 126      16.795  -0.527  16.361  1.00  0.00           H  \nATOM   1958 HD23 LEU A 126      17.299   0.734  17.512  1.00  0.00           H  \nATOM   1959  N   GLU A 127      18.422  -4.478  19.080  1.00  0.00           N  \nATOM   1960  CA  GLU A 127      18.807  -5.550  18.178  1.00  0.00           C  \nATOM   1961  C   GLU A 127      18.101  -6.851  18.567  1.00  0.00           C  \nATOM   1962  O   GLU A 127      17.634  -7.590  17.702  1.00  0.00           O  \nATOM   1963  CB  GLU A 127      20.326  -5.734  18.161  1.00  0.00           C  \nATOM   1964  CG  GLU A 127      20.935  -5.150  16.885  1.00  0.00           C  \nATOM   1965  CD  GLU A 127      22.292  -4.505  17.171  1.00  0.00           C  \nATOM   1966  OE1 GLU A 127      22.314  -3.575  18.007  1.00  0.00           O  \nATOM   1967  OE2 GLU A 127      23.278  -4.955  16.548  1.00  0.00           O  \nATOM   1968  H   GLU A 127      19.159  -4.126  19.657  1.00  0.00           H  \nATOM   1969  HA  GLU A 127      18.475  -5.230  17.190  1.00  0.00           H  \nATOM   1970  HB2 GLU A 127      20.764  -5.248  19.033  1.00  0.00           H  \nATOM   1971  HB3 GLU A 127      20.568  -6.794  18.232  1.00  0.00           H  \nATOM   1972  HG2 GLU A 127      21.052  -5.937  16.140  1.00  0.00           H  \nATOM   1973  HG3 GLU A 127      20.258  -4.408  16.461  1.00  0.00           H  \nATOM   1974  N   GLU A 128      18.046  -7.090  19.869  1.00  0.00           N  \nATOM   1975  CA  GLU A 128      17.406  -8.289  20.383  1.00  0.00           C  \nATOM   1976  C   GLU A 128      15.884  -8.160  20.286  1.00  0.00           C  \nATOM   1977  O   GLU A 128      15.185  -9.153  20.092  1.00  0.00           O  \nATOM   1978  CB  GLU A 128      17.842  -8.569  21.822  1.00  0.00           C  \nATOM   1979  CG  GLU A 128      17.115  -9.791  22.387  1.00  0.00           C  \nATOM   1980  CD  GLU A 128      17.706 -10.206  23.736  1.00  0.00           C  \nATOM   1981  OE1 GLU A 128      18.324  -9.329  24.379  1.00  0.00           O  \nATOM   1982  OE2 GLU A 128      17.528 -11.390  24.094  1.00  0.00           O  \nATOM   1983  H   GLU A 128      18.429  -6.483  20.566  1.00  0.00           H  \nATOM   1984  HA  GLU A 128      17.750  -9.099  19.741  1.00  0.00           H  \nATOM   1985  HB2 GLU A 128      18.918  -8.735  21.854  1.00  0.00           H  \nATOM   1986  HB3 GLU A 128      17.635  -7.698  22.445  1.00  0.00           H  \nATOM   1987  HG2 GLU A 128      16.055  -9.566  22.505  1.00  0.00           H  \nATOM   1988  HG3 GLU A 128      17.189 -10.620  21.684  1.00  0.00           H  \nATOM   1989  N   PHE A 129      15.416  -6.929  20.426  1.00  0.00           N  \nATOM   1990  CA  PHE A 129      13.990  -6.658  20.357  1.00  0.00           C  \nATOM   1991  C   PHE A 129      13.454  -6.901  18.944  1.00  0.00           C  \nATOM   1992  O   PHE A 129      12.351  -7.418  18.776  1.00  0.00           O  \nATOM   1993  CB  PHE A 129      13.797  -5.183  20.716  1.00  0.00           C  \nATOM   1994  CG  PHE A 129      12.355  -4.692  20.577  1.00  0.00           C  \nATOM   1995  CD1 PHE A 129      11.805  -4.537  19.343  1.00  0.00           C  \nATOM   1996  CD2 PHE A 129      11.622  -4.412  21.688  1.00  0.00           C  \nATOM   1997  CE1 PHE A 129      10.466  -4.081  19.214  1.00  0.00           C  \nATOM   1998  CE2 PHE A 129      10.283  -3.957  21.559  1.00  0.00           C  \nATOM   1999  CZ  PHE A 129       9.733  -3.801  20.325  1.00  0.00           C  \nATOM   2000  H   PHE A 129      15.991  -6.126  20.584  1.00  0.00           H  \nATOM   2001  HA  PHE A 129      13.497  -7.337  21.052  1.00  0.00           H  \nATOM   2002  HB2 PHE A 129      14.127  -5.023  21.742  1.00  0.00           H  \nATOM   2003  HB3 PHE A 129      14.439  -4.577  20.076  1.00  0.00           H  \nATOM   2004  HD1 PHE A 129      12.392  -4.761  18.452  1.00  0.00           H  \nATOM   2005  HD2 PHE A 129      12.062  -4.537  22.677  1.00  0.00           H  \nATOM   2006  HE1 PHE A 129      10.025  -3.956  18.225  1.00  0.00           H  \nATOM   2007  HE2 PHE A 129       9.696  -3.732  22.450  1.00  0.00           H  \nATOM   2008  HZ  PHE A 129       8.706  -3.452  20.226  1.00  0.00           H  \nATOM   2009  N   LEU A 130      14.259  -6.516  17.966  1.00  0.00           N  \nATOM   2010  CA  LEU A 130      13.880  -6.686  16.573  1.00  0.00           C  \nATOM   2011  C   LEU A 130      14.018  -8.160  16.188  1.00  0.00           C  \nATOM   2012  O   LEU A 130      13.059  -8.777  15.726  1.00  0.00           O  \nATOM   2013  CB  LEU A 130      14.682  -5.738  15.680  1.00  0.00           C  \nATOM   2014  CG  LEU A 130      14.244  -5.665  14.216  1.00  0.00           C  \nATOM   2015  CD1 LEU A 130      14.118  -4.211  13.754  1.00  0.00           C  \nATOM   2016  CD2 LEU A 130      15.186  -6.471  13.320  1.00  0.00           C  \nATOM   2017  H   LEU A 130      15.155  -6.096  18.111  1.00  0.00           H  \nATOM   2018  HA  LEU A 130      12.831  -6.402  16.484  1.00  0.00           H  \nATOM   2019  HB2 LEU A 130      14.629  -4.736  16.105  1.00  0.00           H  \nATOM   2020  HB3 LEU A 130      15.729  -6.042  15.710  1.00  0.00           H  \nATOM   2021  HG  LEU A 130      13.255  -6.116  14.132  1.00  0.00           H  \nATOM   2022 HD11 LEU A 130      15.072  -3.703  13.891  1.00  0.00           H  \nATOM   2023 HD12 LEU A 130      13.842  -4.188  12.699  1.00  0.00           H  \nATOM   2024 HD13 LEU A 130      13.349  -3.709  14.341  1.00  0.00           H  \nATOM   2025 HD21 LEU A 130      14.632  -7.284  12.849  1.00  0.00           H  \nATOM   2026 HD22 LEU A 130      15.601  -5.821  12.550  1.00  0.00           H  \nATOM   2027 HD23 LEU A 130      15.995  -6.885  13.922  1.00  0.00           H  \nATOM   2028  N   GLU A 131      15.218  -8.682  16.391  1.00  0.00           N  \nATOM   2029  CA  GLU A 131      15.494 -10.072  16.071  1.00  0.00           C  \nATOM   2030  C   GLU A 131      14.508 -10.990  16.795  1.00  0.00           C  \nATOM   2031  O   GLU A 131      14.181 -12.068  16.303  1.00  0.00           O  \nATOM   2032  CB  GLU A 131      16.939 -10.438  16.416  1.00  0.00           C  \nATOM   2033  CG  GLU A 131      17.527 -11.393  15.375  1.00  0.00           C  \nATOM   2034  CD  GLU A 131      17.653 -12.809  15.941  1.00  0.00           C  \nATOM   2035  OE1 GLU A 131      18.256 -12.935  17.028  1.00  0.00           O  \nATOM   2036  OE2 GLU A 131      17.143 -13.735  15.272  1.00  0.00           O  \nATOM   2037  H   GLU A 131      15.993  -8.173  16.767  1.00  0.00           H  \nATOM   2038  HA  GLU A 131      15.353 -10.154  14.993  1.00  0.00           H  \nATOM   2039  HB2 GLU A 131      17.545  -9.533  16.467  1.00  0.00           H  \nATOM   2040  HB3 GLU A 131      16.975 -10.901  17.402  1.00  0.00           H  \nATOM   2041  HG2 GLU A 131      16.892 -11.407  14.489  1.00  0.00           H  \nATOM   2042  HG3 GLU A 131      18.507 -11.035  15.060  1.00  0.00           H  \nATOM   2043  N   PHE A 132      14.062 -10.528  17.954  1.00  0.00           N  \nATOM   2044  CA  PHE A 132      13.119 -11.294  18.752  1.00  0.00           C  \nATOM   2045  C   PHE A 132      11.831 -11.562  17.971  1.00  0.00           C  \nATOM   2046  O   PHE A 132      11.234 -12.630  18.099  1.00  0.00           O  \nATOM   2047  CB  PHE A 132      12.788 -10.451  19.985  1.00  0.00           C  \nATOM   2048  CG  PHE A 132      11.448 -10.800  20.636  1.00  0.00           C  \nATOM   2049  CD1 PHE A 132      11.192 -12.078  21.022  1.00  0.00           C  \nATOM   2050  CD2 PHE A 132      10.513  -9.831  20.828  1.00  0.00           C  \nATOM   2051  CE1 PHE A 132       9.948 -12.402  21.626  1.00  0.00           C  \nATOM   2052  CE2 PHE A 132       9.269 -10.155  21.431  1.00  0.00           C  \nATOM   2053  CZ  PHE A 132       9.013 -11.433  21.818  1.00  0.00           C  \nATOM   2054  H   PHE A 132      14.333  -9.650  18.348  1.00  0.00           H  \nATOM   2055  HA  PHE A 132      13.598 -12.242  18.996  1.00  0.00           H  \nATOM   2056  HB2 PHE A 132      13.581 -10.576  20.722  1.00  0.00           H  \nATOM   2057  HB3 PHE A 132      12.780  -9.399  19.702  1.00  0.00           H  \nATOM   2058  HD1 PHE A 132      11.941 -12.855  20.868  1.00  0.00           H  \nATOM   2059  HD2 PHE A 132      10.718  -8.807  20.518  1.00  0.00           H  \nATOM   2060  HE1 PHE A 132       9.743 -13.427  21.936  1.00  0.00           H  \nATOM   2061  HE2 PHE A 132       8.519  -9.379  21.585  1.00  0.00           H  \nATOM   2062  HZ  PHE A 132       8.058 -11.682  22.281  1.00  0.00           H  \nATOM   2063  N   SER A 133      11.440 -10.574  17.180  1.00  0.00           N  \nATOM   2064  CA  SER A 133      10.234 -10.690  16.378  1.00  0.00           C  \nATOM   2065  C   SER A 133      10.480 -11.627  15.194  1.00  0.00           C  \nATOM   2066  O   SER A 133       9.562 -12.298  14.727  1.00  0.00           O  \nATOM   2067  CB  SER A 133       9.767  -9.320  15.884  1.00  0.00           C  \nATOM   2068  OG  SER A 133       9.186  -9.390  14.584  1.00  0.00           O  \nATOM   2069  H   SER A 133      11.931  -9.709  17.081  1.00  0.00           H  \nATOM   2070  HA  SER A 133       9.482 -11.110  17.047  1.00  0.00           H  \nATOM   2071  HB2 SER A 133       9.040  -8.910  16.584  1.00  0.00           H  \nATOM   2072  HB3 SER A 133      10.614  -8.634  15.866  1.00  0.00           H  \nATOM   2073  HG  SER A 133       8.617 -10.209  14.508  1.00  0.00           H  \nATOM   2074  N   LEU A 134      11.725 -11.641  14.741  1.00  0.00           N  \nATOM   2075  CA  LEU A 134      12.104 -12.484  13.620  1.00  0.00           C  \nATOM   2076  C   LEU A 134      12.194 -13.938  14.087  1.00  0.00           C  \nATOM   2077  O   LEU A 134      11.608 -14.828  13.473  1.00  0.00           O  \nATOM   2078  CB  LEU A 134      13.390 -11.966  12.971  1.00  0.00           C  \nATOM   2079  CG  LEU A 134      13.212 -11.141  11.695  1.00  0.00           C  \nATOM   2080  CD1 LEU A 134      13.059  -9.654  12.021  1.00  0.00           C  \nATOM   2081  CD2 LEU A 134      14.357 -11.397  10.713  1.00  0.00           C  \nATOM   2082  H   LEU A 134      12.467 -11.092  15.126  1.00  0.00           H  \nATOM   2083  HA  LEU A 134      11.314 -12.409  12.873  1.00  0.00           H  \nATOM   2084  HB2 LEU A 134      13.923 -11.358  13.702  1.00  0.00           H  \nATOM   2085  HB3 LEU A 134      14.027 -12.820  12.742  1.00  0.00           H  \nATOM   2086  HG  LEU A 134      12.292 -11.460  11.207  1.00  0.00           H  \nATOM   2087 HD11 LEU A 134      13.594  -9.427  12.943  1.00  0.00           H  \nATOM   2088 HD12 LEU A 134      13.470  -9.059  11.205  1.00  0.00           H  \nATOM   2089 HD13 LEU A 134      12.002  -9.417  12.146  1.00  0.00           H  \nATOM   2090 HD21 LEU A 134      14.514 -12.471  10.610  1.00  0.00           H  \nATOM   2091 HD22 LEU A 134      14.104 -10.972   9.741  1.00  0.00           H  \nATOM   2092 HD23 LEU A 134      15.268 -10.931  11.087  1.00  0.00           H  \nTER    2093      LEU A 134                                                      \nHETATM 2094 CA    CA A 147      14.398  -5.351 -11.632  1.00  0.00          CA  \nHETATM 2095 CA    CA A 159      14.806 -17.000  -6.571  1.00  0.00          CA  \nHETATM 2096 CA    CA A 171      15.920   3.031  27.784  1.00  0.00          CA  \nHETATM 2097 CA    CA A 183      17.441 -10.424  26.705  1.00  0.00          CA  \nCONECT  154 2094                                                                \nCONECT  182 2094                                                                \nCONECT  204 2094                                                                \nCONECT  219 2094                                                                \nCONECT  296 2094                                                                \nCONECT  297 2094                                                                \nCONECT  735 2095                                                                \nCONECT  757 2095                                                                \nCONECT  758 2095                                                                \nCONECT  776 2095                                                                \nCONECT  777 2095                                                                \nCONECT  794 2095                                                                \nCONECT  875 2095                                                                \nCONECT  876 2095                                                                \nCONECT 1392 2096                                                                \nCONECT 1488 2096                                                                \nCONECT 1489 2096                                                                \nCONECT 1831 2097                                                                \nCONECT 1852 2097                                                                \nCONECT 1874 2097                                                                \nCONECT 1889 2097                                                                \nCONECT 1981 2097                                                                \nCONECT 1982 2097                                                                \nCONECT 2094  154  182  204  219                                                 \nCONECT 2094  296  297                                                           \nCONECT 2095  735  757  758  776                                                 \nCONECT 2095  777  794  875  876                                                 \nCONECT 2096 1392 1488 1489                                                      \nCONECT 2097 1831 1852 1874 1889                                                 \nCONECT 2097 1981 1982                                                           \nMASTER      177    0    4    8    4    0    7    6 1056    1   30   11          \nEND                                                                             \n","pdb");
	viewer_17581787013555787.setStyle({"model": 0},{"stick": {}});
	viewer_17581787013555787.setStyle({"model": 0},{"cartoon": {"color": "blue"}});
	viewer_17581787013555787.addLabel("1jfkA",{"fontOpacity": 0.3},{"resi": 158});
	viewer_17581787013555787.zoomTo();
	viewer_17581787013555787.zoom(1.2);
	viewer_17581787013555787.setBackgroundColor("white");
viewer_17581787013555787.render();
});
</script>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div id="3dmolviewer_1758178701358389" style="position: relative; width: 400px; height: 300px;">
<p id="3dmolwarning_1758178701358389" style="background-color:#ffcccc;color:black">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br/></p>
</div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');
}

var viewer_1758178701358389 = null;
var warn = document.getElementById("3dmolwarning_1758178701358389");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_1758178701358389 = $3Dmol.createViewer(document.getElementById("3dmolviewer_1758178701358389"),{backgroundColor:"white"});
viewer_1758178701358389.zoomTo();
	viewer_1758178701358389.addModel("HEADER    METAL BINDING PROTEIN                   18-NOV-06   2NXQ              \nTITLE     CRYSTAL STRUCTURE OF CALCIUM BINDING PROTEIN 1 FROM ENTAMOEBA         \nTITLE    2 HISTOLYTICA: A NOVEL ARRANGEMENT OF EF HAND MOTIFS                   \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: CALCIUM-BINDING PROTEIN;                                   \nCOMPND   3 CHAIN: A, B;                                                         \nCOMPND   4 SYNONYM: CABP;                                                       \nCOMPND   5 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 ORGANISM_SCIENTIFIC: ENTAMOEBA HISTOLYTICA;                          \nSOURCE   3 ORGANISM_TAXID: 294381;                                              \nSOURCE   4 STRAIN: HM-1:IMSS;                                                   \nSOURCE   5 EXPRESSION_SYSTEM: ESCHERICHIA COLI BL21;                            \nSOURCE   6 EXPRESSION_SYSTEM_TAXID: 511693;                                     \nSOURCE   7 EXPRESSION_SYSTEM_STRAIN: BL21;                                      \nSOURCE   8 EXPRESSION_SYSTEM_VECTOR_TYPE: PLASMID;                              \nSOURCE   9 EXPRESSION_SYSTEM_PLASMID: PET-3C                                    \nKEYWDS    EF HAND MOTIFS, CALCIUM BINDING, METAL BINDING PROTEIN                \nEXPDTA    X-RAY DIFFRACTION                                                     \nAUTHOR    S.KUMAR,N.PADHAN,N.ALAM,S.GOURINATH                                   \nREVDAT   5   30-AUG-23 2NXQ    1       REMARK LINK                              \nREVDAT   4   18-OCT-17 2NXQ    1       REMARK                                   \nREVDAT   3   13-JUL-11 2NXQ    1       VERSN                                    \nREVDAT   2   24-FEB-09 2NXQ    1       VERSN                                    \nREVDAT   1   21-AUG-07 2NXQ    0                                                \nJRNL        AUTH   S.KUMAR,N.PADHAN,N.ALAM,S.GOURINATH                          \nJRNL        TITL   CRYSTAL STRUCTURE OF CALCIUM BINDING PROTEIN-1 FROM          \nJRNL        TITL 2 ENTAMOEBA HISTOLYTICA: A NOVEL ARRANGEMENT OF EF HAND        \nJRNL        TITL 3 MOTIFS.                                                      \nJRNL        REF    PROTEINS                      V.  68   990 2007              \nJRNL        REFN                   ISSN 0887-3585                               \nJRNL        PMID   17554780                                                     \nJRNL        DOI    10.1002/PROT.21455                                           \nREMARK   2                                                                      \nREMARK   2 RESOLUTION.    2.40 ANGSTROMS.                                       \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : CNS                                                  \nREMARK   3   AUTHORS     : BRUNGER,ADAMS,CLORE,DELANO,GROS,GROSSE-              \nREMARK   3               : KUNSTLEVE,JIANG,KUSZEWSKI,NILGES,PANNU,              \nREMARK   3               : READ,RICE,SIMONSON,WARREN                            \nREMARK   3                                                                      \nREMARK   3  REFINEMENT TARGET : ENGH & HUBER                                    \nREMARK   3                                                                      \nREMARK   3  DATA USED IN REFINEMENT.                                            \nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.40                           \nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 50.00                          \nREMARK   3   DATA CUTOFF            (SIGMA(F)) : 0.000                          \nREMARK   3   DATA CUTOFF HIGH         (ABS(F)) : NULL                           \nREMARK   3   DATA CUTOFF LOW          (ABS(F)) : NULL                           \nREMARK   3   COMPLETENESS (WORKING+TEST)   (%) : 86.1                           \nREMARK   3   NUMBER OF REFLECTIONS             : 11409                          \nREMARK   3                                                                      \nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \nREMARK   3   CROSS-VALIDATION METHOD          : THROUGHOUT                      \nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \nREMARK   3   R VALUE            (WORKING SET) : 0.258                           \nREMARK   3   FREE R VALUE                     : 0.282                           \nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 8.800                           \nREMARK   3   FREE R VALUE TEST SET COUNT      : 1161                            \nREMARK   3   ESTIMATED ERROR OF FREE R VALUE  : NULL                            \nREMARK   3                                                                      \nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \nREMARK   3   TOTAL NUMBER OF BINS USED           : NULL                         \nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : 2.40                         \nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : 2.55                         \nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : 66.60                        \nREMARK   3   REFLECTIONS IN BIN    (WORKING SET) : NULL                         \nREMARK   3   BIN R VALUE           (WORKING SET) : 0.3470                       \nREMARK   3   BIN FREE R VALUE                    : 0.3680                       \nREMARK   3   BIN FREE R VALUE TEST SET SIZE  (%) : NULL                         \nREMARK   3   BIN FREE R VALUE TEST SET COUNT     : 146                          \nREMARK   3   ESTIMATED ERROR OF BIN FREE R VALUE : 0.030                        \nREMARK   3                                                                      \nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \nREMARK   3   PROTEIN ATOMS            : 1013                                    \nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \nREMARK   3   HETEROGEN ATOMS          : 24                                      \nREMARK   3   SOLVENT ATOMS            : 31                                      \nREMARK   3                                                                      \nREMARK   3  B VALUES.                                                           \nREMARK   3   FROM WILSON PLOT           (A**2) : 43.30                          \nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 66.48                          \nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \nREMARK   3    B11 (A**2) : 8.42300                                              \nREMARK   3    B22 (A**2) : 8.42300                                              \nREMARK   3    B33 (A**2) : -16.84700                                            \nREMARK   3    B12 (A**2) : 0.45400                                              \nREMARK   3    B13 (A**2) : 0.00000                                              \nREMARK   3    B23 (A**2) : 0.00000                                              \nREMARK   3                                                                      \nREMARK   3  ESTIMATED COORDINATE ERROR.                                         \nREMARK   3   ESD FROM LUZZATI PLOT        (A) : 0.38                            \nREMARK   3   ESD FROM SIGMAA              (A) : NULL                            \nREMARK   3   LOW RESOLUTION CUTOFF        (A) : 5.00                            \nREMARK   3                                                                      \nREMARK   3  CROSS-VALIDATED ESTIMATED COORDINATE ERROR.                         \nREMARK   3   ESD FROM C-V LUZZATI PLOT    (A) : 0.43                            \nREMARK   3   ESD FROM C-V SIGMAA          (A) : NULL                            \nREMARK   3                                                                      \nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES.                                   \nREMARK   3   BOND LENGTHS                 (A) : NULL                            \nREMARK   3   BOND ANGLES            (DEGREES) : NULL                            \nREMARK   3   DIHEDRAL ANGLES        (DEGREES) : NULL                            \nREMARK   3   IMPROPER ANGLES        (DEGREES) : NULL                            \nREMARK   3                                                                      \nREMARK   3  ISOTROPIC THERMAL MODEL : NULL                                      \nREMARK   3                                                                      \nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.    RMS    SIGMA                \nREMARK   3   MAIN-CHAIN BOND              (A**2) : 1.464 ; 1.500                \nREMARK   3   MAIN-CHAIN ANGLE             (A**2) : 2.545 ; 2.000                \nREMARK   3   SIDE-CHAIN BOND              (A**2) : 2.372 ; 2.000                \nREMARK   3   SIDE-CHAIN ANGLE             (A**2) : 3.763 ; 2.500                \nREMARK   3                                                                      \nREMARK   3  BULK SOLVENT MODELING.                                              \nREMARK   3   METHOD USED : NULL                                                 \nREMARK   3   KSOL        : NULL                                                 \nREMARK   3   BSOL        : 61.48                                                \nREMARK   3                                                                      \nREMARK   3  NCS MODEL : NULL                                                    \nREMARK   3                                                                      \nREMARK   3  NCS RESTRAINTS.                         RMS   SIGMA/WEIGHT          \nREMARK   3   GROUP  1  POSITIONAL            (A) : NULL  ; NULL                 \nREMARK   3   GROUP  1  B-FACTOR           (A**2) : NULL  ; NULL                 \nREMARK   3                                                                      \nREMARK   3  PARAMETER FILE  1  : PROTEIN_REP.PARAM                              \nREMARK   3  PARAMETER FILE  2  : ION.PARAM                                      \nREMARK   3  PARAMETER FILE  3  : ACT.PARAM                                      \nREMARK   3  PARAMETER FILE  4  : MPD.PARAM                                      \nREMARK   3  PARAMETER FILE  5  : WATER.PARAM                                    \nREMARK   3  PARAMETER FILE  6  : NULL                                           \nREMARK   3  TOPOLOGY FILE  1   : NULL                                           \nREMARK   3  TOPOLOGY FILE  2   : NULL                                           \nREMARK   3  TOPOLOGY FILE  3   : NULL                                           \nREMARK   3  TOPOLOGY FILE  4   : NULL                                           \nREMARK   3  TOPOLOGY FILE  5   : NULL                                           \nREMARK   3  TOPOLOGY FILE  6   : NULL                                           \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \nREMARK   4                                                                      \nREMARK   4 2NXQ COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 22-NOV-06.                  \nREMARK 100 THE DEPOSITION ID IS D_1000040432.                                   \nREMARK 200                                                                      \nREMARK 200 EXPERIMENTAL DETAILS                                                 \nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \nREMARK 200  DATE OF DATA COLLECTION        : 23-MAY-06                          \nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \nREMARK 200  PH                             : 4.4                                \nREMARK 200  NUMBER OF CRYSTALS USED        : 2                                  \nREMARK 200                                                                      \nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \nREMARK 200  RADIATION SOURCE               : CHESS                              \nREMARK 200  BEAMLINE                       : A1                                 \nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.977                              \nREMARK 200  MONOCHROMATOR                  : SI 111                             \nREMARK 200  OPTICS                         : CYLINDRICALLY BENT RHODIUM         \nREMARK 200                                   -COATED SILICON MIRRORS            \nREMARK 200                                                                      \nREMARK 200  DETECTOR TYPE                  : CCD                                \nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 4                     \nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : DENZO, HKL-2000                    \nREMARK 200  DATA SCALING SOFTWARE          : SCALEPACK                          \nREMARK 200                                                                      \nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 12484                              \nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.370                              \nREMARK 200  RESOLUTION RANGE LOW       (A) : 50.000                             \nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \nREMARK 200                                                                      \nREMARK 200 OVERALL.                                                             \nREMARK 200  COMPLETENESS FOR RANGE     (%) : 91.2                               \nREMARK 200  DATA REDUNDANCY                : 4.500                              \nREMARK 200  R MERGE                    (I) : 0.07100                            \nREMARK 200  R SYM                      (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 15.5000                            \nREMARK 200                                                                      \nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 2.37                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 2.45                     \nREMARK 200  COMPLETENESS FOR SHELL     (%) : 73.9                               \nREMARK 200  DATA REDUNDANCY IN SHELL       : 1.70                               \nREMARK 200  R MERGE FOR SHELL          (I) : 0.38600                            \nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \nREMARK 200                                                                      \nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MOLECULAR REPLACEMENT        \nREMARK 200 SOFTWARE USED: PHASER                                                \nREMARK 200 STARTING MODEL: PDB CODE 1CLL: N-TERMINAL DOMAIN                     \nREMARK 200                                                                      \nREMARK 200 REMARK: NULL                                                         \nREMARK 280                                                                      \nREMARK 280 CRYSTAL                                                              \nREMARK 280 SOLVENT CONTENT, VS   (%): 57.20                                     \nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.90                     \nREMARK 280                                                                      \nREMARK 280 CRYSTALLIZATION CONDITIONS: 60% MPD, 5MM CACL2, 50MM SODIUM          \nREMARK 280  ACETATE, PH 4.4, VAPOR DIFFUSION, HANGING DROP, TEMPERATURE 289K    \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 63                             \nREMARK 290                                                                      \nREMARK 290      SYMOP   SYMMETRY                                                \nREMARK 290     NNNMMM   OPERATOR                                                \nREMARK 290       1555   X,Y,Z                                                   \nREMARK 290       2555   -Y,X-Y,Z                                                \nREMARK 290       3555   -X+Y,-X,Z                                               \nREMARK 290       4555   -X,-Y,Z+1/2                                             \nREMARK 290       5555   Y,-X+Y,Z+1/2                                            \nREMARK 290       6555   X-Y,X,Z+1/2                                             \nREMARK 290                                                                      \nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \nREMARK 290           MMM -> TRANSLATION VECTOR                                  \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \nREMARK 290 RELATED MOLECULES.                                                   \nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   2 -0.500000 -0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   2  0.866025 -0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   3 -0.500000  0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   3 -0.866025 -0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   4 -1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       32.49800            \nREMARK 290   SMTRY1   5  0.500000  0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   5 -0.866025  0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   5  0.000000  0.000000  1.000000       32.49800            \nREMARK 290   SMTRY1   6  0.500000 -0.866025  0.000000        0.00000            \nREMARK 290   SMTRY2   6  0.866025  0.500000  0.000000        0.00000            \nREMARK 290   SMTRY3   6  0.000000  0.000000  1.000000       32.49800            \nREMARK 290                                                                      \nREMARK 290 REMARK: NULL                                                         \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1, 2, 3, 4, 5                                           \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TRIMERIC                          \nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: TRIMERIC                   \nREMARK 350 SOFTWARE USED: PISA                                                  \nREMARK 350 TOTAL BURIED SURFACE AREA: 5270 ANGSTROM**2                          \nREMARK 350 SURFACE AREA OF THE COMPLEX: 13460 ANGSTROM**2                       \nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -114.0 KCAL/MOL                       \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000       47.62500            \nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000      -82.48892            \nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000       95.25000            \nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 2                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TRIMERIC                          \nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: TRIMERIC                   \nREMARK 350 SOFTWARE USED: PISA                                                  \nREMARK 350 TOTAL BURIED SURFACE AREA: 5020 ANGSTROM**2                          \nREMARK 350 SURFACE AREA OF THE COMPLEX: 13080 ANGSTROM**2                       \nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -112.0 KCAL/MOL                       \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000       47.62500            \nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000      -82.48892            \nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000       95.25000            \nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 3                                                       \nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: HEXAMERIC                  \nREMARK 350 SOFTWARE USED: PISA                                                  \nREMARK 350 TOTAL BURIED SURFACE AREA: 14090 ANGSTROM**2                         \nREMARK 350 SURFACE AREA OF THE COMPLEX: 22740 ANGSTROM**2                       \nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -250.0 KCAL/MOL                       \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A, B                                  \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   2 -0.500000 -0.866025  0.000000       47.62500            \nREMARK 350   BIOMT2   2  0.866025 -0.500000  0.000000      -82.48892            \nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   3 -0.500000  0.866025  0.000000       95.25000            \nREMARK 350   BIOMT2   3 -0.866025 -0.500000  0.000000        0.00000            \nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000        0.00000            \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 4                                                       \nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \nREMARK 350 SOFTWARE USED: PQS                                                   \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 5                                                       \nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: MONOMERIC                  \nREMARK 350 SOFTWARE USED: PQS                                                   \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: B                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 465                                                                      \nREMARK 465 MISSING RESIDUES                                                     \nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \nREMARK 465                                                                      \nREMARK 465   M RES C SSSEQI                                                     \nREMARK 465     MET A     1                                                      \nREMARK 465     GLN A    68                                                      \nREMARK 465     ASP A    69                                                      \nREMARK 465     LEU A    70                                                      \nREMARK 465     SER A    71                                                      \nREMARK 465     ASP A    72                                                      \nREMARK 465     ASP A    73                                                      \nREMARK 465     LYS A    74                                                      \nREMARK 465     ILE A    75                                                      \nREMARK 465     GLY A    76                                                      \nREMARK 465     LEU A    77                                                      \nREMARK 465     LYS A    78                                                      \nREMARK 465     VAL A    79                                                      \nREMARK 465     LEU A    80                                                      \nREMARK 465     TYR A    81                                                      \nREMARK 465     LYS A    82                                                      \nREMARK 465     LEU A    83                                                      \nREMARK 465     MET A    84                                                      \nREMARK 465     ASP A    85                                                      \nREMARK 465     VAL A    86                                                      \nREMARK 465     ASP A    87                                                      \nREMARK 465     GLY A    88                                                      \nREMARK 465     ASP A    89                                                      \nREMARK 465     GLY A    90                                                      \nREMARK 465     LYS A    91                                                      \nREMARK 465     LEU A    92                                                      \nREMARK 465     THR A    93                                                      \nREMARK 465     LYS A    94                                                      \nREMARK 465     GLU A    95                                                      \nREMARK 465     GLU A    96                                                      \nREMARK 465     VAL A    97                                                      \nREMARK 465     THR A    98                                                      \nREMARK 465     SER A    99                                                      \nREMARK 465     PHE A   100                                                      \nREMARK 465     PHE A   101                                                      \nREMARK 465     LYS A   102                                                      \nREMARK 465     LYS A   103                                                      \nREMARK 465     HIS A   104                                                      \nREMARK 465     GLY A   105                                                      \nREMARK 465     ILE A   106                                                      \nREMARK 465     GLU A   107                                                      \nREMARK 465     LYS A   108                                                      \nREMARK 465     VAL A   109                                                      \nREMARK 465     ALA A   110                                                      \nREMARK 465     GLU A   111                                                      \nREMARK 465     GLN A   112                                                      \nREMARK 465     VAL A   113                                                      \nREMARK 465     MET A   114                                                      \nREMARK 465     LYS A   115                                                      \nREMARK 465     ALA A   116                                                      \nREMARK 465     ASP A   117                                                      \nREMARK 465     ALA A   118                                                      \nREMARK 465     ASN A   119                                                      \nREMARK 465     GLY A   120                                                      \nREMARK 465     ASP A   121                                                      \nREMARK 465     GLY A   122                                                      \nREMARK 465     TYR A   123                                                      \nREMARK 465     ILE A   124                                                      \nREMARK 465     THR A   125                                                      \nREMARK 465     LEU A   126                                                      \nREMARK 465     GLU A   127                                                      \nREMARK 465     GLU A   128                                                      \nREMARK 465     PHE A   129                                                      \nREMARK 465     LEU A   130                                                      \nREMARK 465     GLU A   131                                                      \nREMARK 465     PHE A   132                                                      \nREMARK 465     SER A   133                                                      \nREMARK 465     LEU A   134                                                      \nREMARK 465     MET B     1                                                      \nREMARK 465     GLN B    66                                                      \nREMARK 465     GLY B    67                                                      \nREMARK 465     GLN B    68                                                      \nREMARK 465     ASP B    69                                                      \nREMARK 465     LEU B    70                                                      \nREMARK 465     SER B    71                                                      \nREMARK 465     ASP B    72                                                      \nREMARK 465     ASP B    73                                                      \nREMARK 465     LYS B    74                                                      \nREMARK 465     ILE B    75                                                      \nREMARK 465     GLY B    76                                                      \nREMARK 465     LEU B    77                                                      \nREMARK 465     LYS B    78                                                      \nREMARK 465     VAL B    79                                                      \nREMARK 465     LEU B    80                                                      \nREMARK 465     TYR B    81                                                      \nREMARK 465     LYS B    82                                                      \nREMARK 465     LEU B    83                                                      \nREMARK 465     MET B    84                                                      \nREMARK 465     ASP B    85                                                      \nREMARK 465     VAL B    86                                                      \nREMARK 465     ASP B    87                                                      \nREMARK 465     GLY B    88                                                      \nREMARK 465     ASP B    89                                                      \nREMARK 465     GLY B    90                                                      \nREMARK 465     LYS B    91                                                      \nREMARK 465     LEU B    92                                                      \nREMARK 465     THR B    93                                                      \nREMARK 465     LYS B    94                                                      \nREMARK 465     GLU B    95                                                      \nREMARK 465     GLU B    96                                                      \nREMARK 465     VAL B    97                                                      \nREMARK 465     THR B    98                                                      \nREMARK 465     SER B    99                                                      \nREMARK 465     PHE B   100                                                      \nREMARK 465     PHE B   101                                                      \nREMARK 465     LYS B   102                                                      \nREMARK 465     LYS B   103                                                      \nREMARK 465     HIS B   104                                                      \nREMARK 465     GLY B   105                                                      \nREMARK 465     ILE B   106                                                      \nREMARK 465     GLU B   107                                                      \nREMARK 465     LYS B   108                                                      \nREMARK 465     VAL B   109                                                      \nREMARK 465     ALA B   110                                                      \nREMARK 465     GLU B   111                                                      \nREMARK 465     GLN B   112                                                      \nREMARK 465     VAL B   113                                                      \nREMARK 465     MET B   114                                                      \nREMARK 465     LYS B   115                                                      \nREMARK 465     ALA B   116                                                      \nREMARK 465     ASP B   117                                                      \nREMARK 465     ALA B   118                                                      \nREMARK 465     ASN B   119                                                      \nREMARK 465     GLY B   120                                                      \nREMARK 465     ASP B   121                                                      \nREMARK 465     GLY B   122                                                      \nREMARK 465     TYR B   123                                                      \nREMARK 465     ILE B   124                                                      \nREMARK 465     THR B   125                                                      \nREMARK 465     LEU B   126                                                      \nREMARK 465     GLU B   127                                                      \nREMARK 465     GLU B   128                                                      \nREMARK 465     PHE B   129                                                      \nREMARK 465     LEU B   130                                                      \nREMARK 465     GLU B   131                                                      \nREMARK 465     PHE B   132                                                      \nREMARK 465     SER B   133                                                      \nREMARK 465     LEU B   134                                                      \nREMARK 470                                                                      \nREMARK 470 MISSING ATOM                                                         \nREMARK 470 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \nREMARK 470 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \nREMARK 470 I=INSERTION CODE):                                                   \nREMARK 470   M RES CSSEQI  ATOMS                                                \nREMARK 470     GLN A  66    CG   CD   OE1  NE2                                  \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \nREMARK 500    SER A  64   N   -  CA  -  C   ANGL. DEV. = -16.4 DEGREES          \nREMARK 500    SER B  64   N   -  CA  -  C   ANGL. DEV. = -16.5 DEGREES          \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    ILE A  65      101.02     23.58                                   \nREMARK 500    GLU B   3      -75.39    -52.05                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 620                                                                      \nREMARK 620 METAL COORDINATION                                                   \nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 149  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A  10   OD1                                                    \nREMARK 620 2 ASN A  12   OD1  86.3                                              \nREMARK 620 3 ASP A  14   OD1  87.7  91.0                                        \nREMARK 620 4 ALA A  16   O    89.0 163.4  72.9                                  \nREMARK 620 5 GLU A  21   OE1  98.0 117.4 151.2  78.9                            \nREMARK 620 6 GLU A  21   OE2 100.7  72.0 160.3 124.5  45.8                      \nREMARK 620 7 HOH A 215   O   176.4  91.3  89.7  92.5  85.5  81.1                \nREMARK 620 N                    1     2     3     4     5     6                 \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 150  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A  46   OD1                                                    \nREMARK 620 2 ASP A  48   OD1  77.5                                              \nREMARK 620 3 ASN A  50   OD1  86.7  71.2                                        \nREMARK 620 4 GLU A  52   O    92.6 145.3  75.1                                  \nREMARK 620 5 GLU A  57   OE1 105.5 141.4 146.5  73.3                            \nREMARK 620 6 GLU A  57   OE2  90.0  91.7 162.8 121.9  50.4                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA B 151  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP B  10   OD1                                                    \nREMARK 620 2 ASN B  12   OD1  79.7                                              \nREMARK 620 3 ASP B  14   OD1  83.3  86.2                                        \nREMARK 620 4 ALA B  16   O    87.3 156.0  72.2                                  \nREMARK 620 5 GLU B  21   OE1  98.2 121.7 151.9  79.8                            \nREMARK 620 6 GLU B  21   OE2  99.3  74.7 159.8 127.8  47.9                      \nREMARK 620 7 HOH B 222   O   171.5  94.3  90.4  96.2  90.0  84.7                \nREMARK 620 N                    1     2     3     4     5     6                 \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA B 152  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP B  46   OD1                                                    \nREMARK 620 2 ASP B  48   OD1  84.0                                              \nREMARK 620 3 ASN B  50   OD1  93.9  72.3                                        \nREMARK 620 4 GLU B  52   O    86.8 143.1  72.8                                  \nREMARK 620 5 GLU B  57   OE1  97.3 142.3 144.5  74.3                            \nREMARK 620 6 GLU B  57   OE2  90.3  92.6 163.7 123.1  49.8                      \nREMARK 620 7 HOH B 223   O   172.2  89.4  80.1  96.1  90.4  94.1                \nREMARK 620 N                    1     2     3     4     5     6                 \nREMARK 800                                                                      \nREMARK 800 SITE                                                                 \nREMARK 800 SITE_IDENTIFIER: AC1                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 149                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC2                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 150                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC3                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ACT A 201                 \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC4                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ACT A 203                 \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC5                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA B 151                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC6                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA B 152                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC7                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ACT B 202                 \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC8                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE ACT B 205                 \nDBREF  2NXQ A    1   134  UNP    P38505   CALBP_ENTHI      1    134             \nDBREF  2NXQ B    1   134  UNP    P38505   CALBP_ENTHI      1    134             \nSEQRES   1 A  134  MET ALA GLU ALA LEU PHE LYS GLU ILE ASP VAL ASN GLY          \nSEQRES   2 A  134  ASP GLY ALA VAL SER TYR GLU GLU VAL LYS ALA PHE VAL          \nSEQRES   3 A  134  SER LYS LYS ARG ALA ILE LYS ASN GLU GLN LEU LEU GLN          \nSEQRES   4 A  134  LEU ILE PHE LYS SER ILE ASP ALA ASP GLY ASN GLY GLU          \nSEQRES   5 A  134  ILE ASP GLN ASN GLU PHE ALA LYS PHE TYR GLY SER ILE          \nSEQRES   6 A  134  GLN GLY GLN ASP LEU SER ASP ASP LYS ILE GLY LEU LYS          \nSEQRES   7 A  134  VAL LEU TYR LYS LEU MET ASP VAL ASP GLY ASP GLY LYS          \nSEQRES   8 A  134  LEU THR LYS GLU GLU VAL THR SER PHE PHE LYS LYS HIS          \nSEQRES   9 A  134  GLY ILE GLU LYS VAL ALA GLU GLN VAL MET LYS ALA ASP          \nSEQRES  10 A  134  ALA ASN GLY ASP GLY TYR ILE THR LEU GLU GLU PHE LEU          \nSEQRES  11 A  134  GLU PHE SER LEU                                              \nSEQRES   1 B  134  MET ALA GLU ALA LEU PHE LYS GLU ILE ASP VAL ASN GLY          \nSEQRES   2 B  134  ASP GLY ALA VAL SER TYR GLU GLU VAL LYS ALA PHE VAL          \nSEQRES   3 B  134  SER LYS LYS ARG ALA ILE LYS ASN GLU GLN LEU LEU GLN          \nSEQRES   4 B  134  LEU ILE PHE LYS SER ILE ASP ALA ASP GLY ASN GLY GLU          \nSEQRES   5 B  134  ILE ASP GLN ASN GLU PHE ALA LYS PHE TYR GLY SER ILE          \nSEQRES   6 B  134  GLN GLY GLN ASP LEU SER ASP ASP LYS ILE GLY LEU LYS          \nSEQRES   7 B  134  VAL LEU TYR LYS LEU MET ASP VAL ASP GLY ASP GLY LYS          \nSEQRES   8 B  134  LEU THR LYS GLU GLU VAL THR SER PHE PHE LYS LYS HIS          \nSEQRES   9 B  134  GLY ILE GLU LYS VAL ALA GLU GLN VAL MET LYS ALA ASP          \nSEQRES  10 B  134  ALA ASN GLY ASP GLY TYR ILE THR LEU GLU GLU PHE LEU          \nSEQRES  11 B  134  GLU PHE SER LEU                                              \nHET     CA  A 149       1                                                       \nHET     CA  A 150       1                                                       \nHET    ACT  A 201       4                                                       \nHET    ACT  A 203       4                                                       \nHET     CA  B 151       1                                                       \nHET     CA  B 152       1                                                       \nHET    ACT  B 202       4                                                       \nHET    ACT  B 204       4                                                       \nHET    ACT  B 205       4                                                       \nHETNAM      CA CALCIUM ION                                                      \nHETNAM     ACT ACETATE ION                                                      \nFORMUL   3   CA    4(CA 2+)                                                     \nFORMUL   5  ACT    5(C2 H3 O2 1-)                                               \nFORMUL  12  HOH   *31(H2 O)                                                     \nHELIX    1   1 ALA A    2  ASP A   10  1                                   9    \nHELIX    2   2 SER A   18  ASP A   46  1                                  29    \nHELIX    3   3 ASP A   54  TYR A   62  1                                   9    \nHELIX    4   4 ALA B    2  ASP B   10  1                                   9    \nHELIX    5   5 SER B   18  ASP B   46  1                                  29    \nHELIX    6   6 ASP B   54  TYR B   62  1                                   9    \nLINK         OD1 ASP A  10                CA    CA A 149     1555   1555  2.53  \nLINK         OD1 ASN A  12                CA    CA A 149     1555   1555  2.45  \nLINK         OD1 ASP A  14                CA    CA A 149     1555   1555  2.85  \nLINK         O   ALA A  16                CA    CA A 149     1555   1555  2.43  \nLINK         OE1 GLU A  21                CA    CA A 149     1555   1555  2.72  \nLINK         OE2 GLU A  21                CA    CA A 149     1555   1555  2.91  \nLINK         OD1 ASP A  46                CA    CA A 150     1555   1555  2.64  \nLINK         OD1 ASP A  48                CA    CA A 150     1555   1555  2.84  \nLINK         OD1 ASN A  50                CA    CA A 150     1555   1555  2.49  \nLINK         O   GLU A  52                CA    CA A 150     1555   1555  2.56  \nLINK         OE1 GLU A  57                CA    CA A 150     1555   1555  2.68  \nLINK         OE2 GLU A  57                CA    CA A 150     1555   1555  2.44  \nLINK        CA    CA A 149                 O   HOH A 215     1555   1555  2.80  \nLINK         OD1 ASP B  10                CA    CA B 151     1555   1555  2.57  \nLINK         OD1 ASN B  12                CA    CA B 151     1555   1555  2.50  \nLINK         OD1 ASP B  14                CA    CA B 151     1555   1555  2.81  \nLINK         O   ALA B  16                CA    CA B 151     1555   1555  2.39  \nLINK         OE1 GLU B  21                CA    CA B 151     1555   1555  2.73  \nLINK         OE2 GLU B  21                CA    CA B 151     1555   1555  2.72  \nLINK         OD1 ASP B  46                CA    CA B 152     1555   1555  2.48  \nLINK         OD1 ASP B  48                CA    CA B 152     1555   1555  2.79  \nLINK         OD1 ASN B  50                CA    CA B 152     1555   1555  2.57  \nLINK         O   GLU B  52                CA    CA B 152     1555   1555  2.50  \nLINK         OE1 GLU B  57                CA    CA B 152     1555   1555  2.69  \nLINK         OE2 GLU B  57                CA    CA B 152     1555   1555  2.43  \nLINK        CA    CA B 151                 O   HOH B 222     1555   1555  2.74  \nLINK        CA    CA B 152                 O   HOH B 223     1555   1555  3.00  \nSITE     1 AC1  6 ASP A  10  ASN A  12  ASP A  14  ALA A  16                    \nSITE     2 AC1  6 GLU A  21  HOH A 215                                          \nSITE     1 AC2  6 ASP A  46  ASP A  48  ASN A  50  GLU A  52                    \nSITE     2 AC2  6 GLU A  57  HOH A 216                                          \nSITE     1 AC3  1 TYR A  62                                                     \nSITE     1 AC4  1 LYS A  33                                                     \nSITE     1 AC5  6 ASP B  10  ASN B  12  ASP B  14  ALA B  16                    \nSITE     2 AC5  6 GLU B  21  HOH B 222                                          \nSITE     1 AC6  6 ASP B  46  ASP B  48  ASN B  50  GLU B  52                    \nSITE     2 AC6  6 GLU B  57  HOH B 223                                          \nSITE     1 AC7  1 TYR B  62                                                     \nSITE     1 AC8  1 VAL B  26                                                     \nCRYST1   95.250   95.250   64.996  90.00  90.00 120.00 P 63         12          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      0.010499  0.006061  0.000000        0.00000                         \nSCALE2      0.000000  0.012123  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  0.015386        0.00000                         \nATOM      1  N   ALA A   2      28.970 -48.661  11.361  1.00 95.39           N  \nATOM      2  CA  ALA A   2      28.428 -47.416  10.712  1.00 96.52           C  \nATOM      3  C   ALA A   2      28.119 -47.679   9.243  1.00 96.31           C  \nATOM      4  O   ALA A   2      29.022 -48.004   8.467  1.00 95.73           O  \nATOM      5  CB  ALA A   2      29.434 -46.288  10.828  1.00 96.78           C  \nATOM      6  N   GLU A   3      26.860 -47.481   8.855  1.00 95.81           N  \nATOM      7  CA  GLU A   3      26.418 -47.776   7.497  1.00 93.90           C  \nATOM      8  C   GLU A   3      26.517 -46.774   6.356  1.00 89.90           C  \nATOM      9  O   GLU A   3      27.175 -47.064   5.357  1.00 89.17           O  \nATOM     10  CB  GLU A   3      24.995 -48.318   7.555  1.00 96.38           C  \nATOM     11  CG  GLU A   3      24.937 -49.798   7.802  1.00 99.93           C  \nATOM     12  CD  GLU A   3      25.393 -50.601   6.598  1.00102.13           C  \nATOM     13  OE1 GLU A   3      24.731 -51.616   6.301  1.00102.20           O  \nATOM     14  OE2 GLU A   3      26.404 -50.233   5.956  1.00102.58           O  \nATOM     15  N   ALA A   4      25.852 -45.625   6.444  1.00 85.45           N  \nATOM     16  CA  ALA A   4      25.952 -44.689   5.327  1.00 80.75           C  \nATOM     17  C   ALA A   4      27.378 -44.175   5.337  1.00 77.21           C  \nATOM     18  O   ALA A   4      27.930 -43.762   4.310  1.00 78.15           O  \nATOM     19  CB  ALA A   4      24.964 -43.541   5.472  1.00 80.33           C  \nATOM     20  N   LEU A   5      27.978 -44.228   6.519  1.00 71.68           N  \nATOM     21  CA  LEU A   5      29.343 -43.789   6.680  1.00 67.53           C  \nATOM     22  C   LEU A   5      30.291 -44.692   5.904  1.00 64.81           C  \nATOM     23  O   LEU A   5      31.190 -44.200   5.218  1.00 64.07           O  \nATOM     24  CB  LEU A   5      29.739 -43.777   8.154  1.00 66.53           C  \nATOM     25  CG  LEU A   5      31.150 -43.233   8.393  1.00 68.94           C  \nATOM     26  CD1 LEU A   5      31.216 -41.807   7.878  1.00 67.40           C  \nATOM     27  CD2 LEU A   5      31.512 -43.294   9.869  1.00 69.05           C  \nATOM     28  N   PHE A   6      30.106 -46.008   5.995  1.00 61.66           N  \nATOM     29  CA  PHE A   6      31.003 -46.905   5.268  1.00 58.61           C  \nATOM     30  C   PHE A   6      30.792 -46.714   3.772  1.00 57.93           C  \nATOM     31  O   PHE A   6      31.752 -46.700   3.001  1.00 56.67           O  \nATOM     32  CB  PHE A   6      30.773 -48.365   5.690  1.00 55.83           C  \nATOM     33  CG  PHE A   6      31.688 -49.350   5.012  1.00 55.01           C  \nATOM     34  CD1 PHE A   6      31.403 -49.804   3.729  1.00 52.99           C  \nATOM     35  CD2 PHE A   6      32.845 -49.804   5.647  1.00 52.99           C  \nATOM     36  CE1 PHE A   6      32.257 -50.682   3.081  1.00 52.31           C  \nATOM     37  CE2 PHE A   6      33.717 -50.688   5.009  1.00 52.82           C  \nATOM     38  CZ  PHE A   6      33.423 -51.130   3.726  1.00 53.88           C  \nATOM     39  N   LYS A   7      29.534 -46.505   3.385  1.00 58.68           N  \nATOM     40  CA  LYS A   7      29.164 -46.304   1.988  1.00 58.58           C  \nATOM     41  C   LYS A   7      29.702 -44.980   1.494  1.00 57.50           C  \nATOM     42  O   LYS A   7      30.096 -44.858   0.342  1.00 57.37           O  \nATOM     43  CB  LYS A   7      27.642 -46.323   1.827  1.00 60.46           C  \nATOM     44  CG  LYS A   7      27.018 -47.722   1.787  1.00 61.53           C  \nATOM     45  CD  LYS A   7      25.487 -47.619   1.695  1.00 63.08           C  \nATOM     46  CE  LYS A   7      24.910 -48.438   0.529  1.00 62.65           C  \nATOM     47  NZ  LYS A   7      25.277 -47.920  -0.826  1.00 60.21           N  \nATOM     48  N   GLU A   8      29.718 -43.985   2.371  1.00 57.83           N  \nATOM     49  CA  GLU A   8      30.224 -42.670   2.000  1.00 57.05           C  \nATOM     50  C   GLU A   8      31.732 -42.750   1.726  1.00 56.68           C  \nATOM     51  O   GLU A   8      32.235 -42.158   0.770  1.00 56.70           O  \nATOM     52  CB  GLU A   8      29.960 -41.667   3.127  1.00 61.41           C  \nATOM     53  CG  GLU A   8      30.002 -40.198   2.701  1.00 67.90           C  \nATOM     54  CD  GLU A   8      30.126 -39.235   3.885  1.00 72.64           C  \nATOM     55  OE1 GLU A   8      29.667 -39.564   5.008  1.00 74.58           O  \nATOM     56  OE2 GLU A   8      30.679 -38.132   3.685  1.00 73.67           O  \nATOM     57  N   ILE A   9      32.453 -43.492   2.561  1.00 54.12           N  \nATOM     58  CA  ILE A   9      33.893 -43.622   2.388  1.00 55.61           C  \nATOM     59  C   ILE A   9      34.170 -44.529   1.196  1.00 56.75           C  \nATOM     60  O   ILE A   9      35.053 -44.235   0.392  1.00 56.04           O  \nATOM     61  CB  ILE A   9      34.566 -44.217   3.661  1.00 55.68           C  \nATOM     62  CG1 ILE A   9      34.283 -43.322   4.869  1.00 55.66           C  \nATOM     63  CG2 ILE A   9      36.066 -44.328   3.473  1.00 51.15           C  \nATOM     64  CD1 ILE A   9      34.728 -43.909   6.196  1.00 55.80           C  \nATOM     65  N   ASP A  10      33.403 -45.619   1.084  1.00 56.97           N  \nATOM     66  CA  ASP A  10      33.548 -46.593  -0.005  1.00 56.57           C  \nATOM     67  C   ASP A  10      33.006 -45.950  -1.278  1.00 57.43           C  \nATOM     68  O   ASP A  10      31.898 -46.232  -1.715  1.00 58.33           O  \nATOM     69  CB  ASP A  10      32.772 -47.883   0.338  1.00 55.67           C  \nATOM     70  CG  ASP A  10      32.767 -48.909  -0.805  1.00 55.75           C  \nATOM     71  OD1 ASP A  10      33.669 -48.871  -1.657  1.00 54.64           O  \nATOM     72  OD2 ASP A  10      31.870 -49.779  -0.849  1.00 54.83           O  \nATOM     73  N   VAL A  11      33.820 -45.097  -1.883  1.00 58.57           N  \nATOM     74  CA  VAL A  11      33.428 -44.364  -3.080  1.00 59.29           C  \nATOM     75  C   VAL A  11      33.185 -45.111  -4.400  1.00 60.79           C  \nATOM     76  O   VAL A  11      32.339 -44.676  -5.176  1.00 60.37           O  \nATOM     77  CB  VAL A  11      34.422 -43.185  -3.337  1.00 58.68           C  \nATOM     78  CG1 VAL A  11      34.449 -42.812  -4.805  1.00 57.54           C  \nATOM     79  CG2 VAL A  11      34.003 -41.977  -2.516  1.00 56.98           C  \nATOM     80  N   ASN A  12      33.916 -46.185  -4.700  1.00 60.56           N  \nATOM     81  CA  ASN A  12      33.649 -46.887  -5.960  1.00 59.21           C  \nATOM     82  C   ASN A  12      32.650 -48.030  -5.710  1.00 59.37           C  \nATOM     83  O   ASN A  12      32.406 -48.875  -6.573  1.00 59.63           O  \nATOM     84  CB  ASN A  12      34.954 -47.403  -6.600  1.00 58.22           C  \nATOM     85  CG  ASN A  12      35.569 -48.511  -5.829  1.00 59.80           C  \nATOM     86  OD1 ASN A  12      35.250 -48.682  -4.669  1.00 60.92           O  \nATOM     87  ND2 ASN A  12      36.466 -49.274  -6.456  1.00 60.17           N  \nATOM     88  N   GLY A  13      32.084 -48.041  -4.507  1.00 59.98           N  \nATOM     89  CA  GLY A  13      31.096 -49.033  -4.130  1.00 58.82           C  \nATOM     90  C   GLY A  13      31.417 -50.513  -4.170  1.00 59.27           C  \nATOM     91  O   GLY A  13      30.496 -51.302  -4.330  1.00 59.63           O  \nATOM     92  N   ASP A  14      32.665 -50.934  -3.980  1.00 59.65           N  \nATOM     93  CA  ASP A  14      32.945 -52.378  -4.047  1.00 57.53           C  \nATOM     94  C   ASP A  14      32.990 -53.157  -2.733  1.00 56.57           C  \nATOM     95  O   ASP A  14      33.463 -54.300  -2.707  1.00 56.39           O  \nATOM     96  CB  ASP A  14      34.247 -52.646  -4.782  1.00 58.39           C  \nATOM     97  CG  ASP A  14      35.446 -52.088  -4.051  1.00 62.20           C  \nATOM     98  OD1 ASP A  14      35.389 -51.980  -2.797  1.00 61.50           O  \nATOM     99  OD2 ASP A  14      36.444 -51.769  -4.735  1.00 62.18           O  \nATOM    100  N   GLY A  15      32.499 -52.551  -1.654  1.00 56.09           N  \nATOM    101  CA  GLY A  15      32.480 -53.204  -0.352  1.00 54.47           C  \nATOM    102  C   GLY A  15      33.752 -53.098   0.480  1.00 56.37           C  \nATOM    103  O   GLY A  15      33.759 -53.428   1.659  1.00 57.86           O  \nATOM    104  N   ALA A  16      34.840 -52.638  -0.120  1.00 56.22           N  \nATOM    105  CA  ALA A  16      36.091 -52.526   0.613  1.00 56.67           C  \nATOM    106  C   ALA A  16      36.552 -51.076   0.675  1.00 56.43           C  \nATOM    107  O   ALA A  16      36.502 -50.374  -0.319  1.00 55.22           O  \nATOM    108  CB  ALA A  16      37.164 -53.386  -0.062  1.00 56.75           C  \nATOM    109  N   VAL A  17      36.979 -50.616   1.846  1.00 54.67           N  \nATOM    110  CA  VAL A  17      37.484 -49.243   1.955  1.00 52.70           C  \nATOM    111  C   VAL A  17      38.997 -49.337   1.767  1.00 50.54           C  \nATOM    112  O   VAL A  17      39.690 -49.960   2.571  1.00 49.71           O  \nATOM    113  CB  VAL A  17      37.152 -48.619   3.335  1.00 54.42           C  \nATOM    114  CG1 VAL A  17      38.010 -47.394   3.578  1.00 55.59           C  \nATOM    115  CG2 VAL A  17      35.686 -48.222   3.380  1.00 53.15           C  \nATOM    116  N   SER A  18      39.499 -48.757   0.680  1.00 48.41           N  \nATOM    117  CA  SER A  18      40.929 -48.786   0.385  1.00 49.60           C  \nATOM    118  C   SER A  18      41.690 -47.647   1.077  1.00 51.34           C  \nATOM    119  O   SER A  18      41.083 -46.780   1.722  1.00 50.44           O  \nATOM    120  CB  SER A  18      41.165 -48.703  -1.126  1.00 48.21           C  \nATOM    121  OG  SER A  18      40.839 -47.421  -1.639  1.00 53.15           O  \nATOM    122  N   TYR A  19      43.017 -47.656   0.936  1.00 52.54           N  \nATOM    123  CA  TYR A  19      43.849 -46.624   1.538  1.00 53.84           C  \nATOM    124  C   TYR A  19      43.572 -45.288   0.861  1.00 54.56           C  \nATOM    125  O   TYR A  19      43.403 -44.277   1.528  1.00 54.99           O  \nATOM    126  CB  TYR A  19      45.338 -46.979   1.450  1.00 53.86           C  \nATOM    127  CG  TYR A  19      46.206 -45.926   2.087  1.00 51.04           C  \nATOM    128  CD1 TYR A  19      46.154 -45.690   3.461  1.00 49.34           C  \nATOM    129  CD2 TYR A  19      47.040 -45.125   1.310  1.00 50.86           C  \nATOM    130  CE1 TYR A  19      46.906 -44.676   4.043  1.00 51.73           C  \nATOM    131  CE2 TYR A  19      47.799 -44.105   1.882  1.00 50.34           C  \nATOM    132  CZ  TYR A  19      47.729 -43.886   3.245  1.00 52.15           C  \nATOM    133  OH  TYR A  19      48.463 -42.867   3.800  1.00 51.74           O  \nATOM    134  N   GLU A  20      43.496 -45.288  -0.461  1.00 56.85           N  \nATOM    135  CA  GLU A  20      43.177 -44.073  -1.204  1.00 59.12           C  \nATOM    136  C   GLU A  20      41.886 -43.442  -0.696  1.00 58.52           C  \nATOM    137  O   GLU A  20      41.804 -42.235  -0.493  1.00 59.05           O  \nATOM    138  CB  GLU A  20      42.979 -44.404  -2.677  1.00 63.73           C  \nATOM    139  CG  GLU A  20      44.100 -43.959  -3.533  1.00 71.85           C  \nATOM    140  CD  GLU A  20      45.420 -44.265  -2.886  1.00 78.06           C  \nATOM    141  OE1 GLU A  20      45.702 -45.464  -2.650  1.00 80.29           O  \nATOM    142  OE2 GLU A  20      46.173 -43.306  -2.604  1.00 79.85           O  \nATOM    143  N   GLU A  21      40.872 -44.277  -0.512  1.00 57.11           N  \nATOM    144  CA  GLU A  21      39.581 -43.803  -0.069  1.00 55.00           C  \nATOM    145  C   GLU A  21      39.586 -43.251   1.344  1.00 52.76           C  \nATOM    146  O   GLU A  21      38.871 -42.291   1.641  1.00 51.69           O  \nATOM    147  CB  GLU A  21      38.561 -44.927  -0.232  1.00 54.54           C  \nATOM    148  CG  GLU A  21      38.445 -45.346  -1.683  1.00 58.17           C  \nATOM    149  CD  GLU A  21      37.424 -46.435  -1.905  1.00 58.92           C  \nATOM    150  OE1 GLU A  21      37.214 -47.237  -0.975  1.00 58.47           O  \nATOM    151  OE2 GLU A  21      36.852 -46.499  -3.013  1.00 59.55           O  \nATOM    152  N   VAL A  22      40.377 -43.852   2.226  1.00 52.09           N  \nATOM    153  CA  VAL A  22      40.442 -43.344   3.592  1.00 52.74           C  \nATOM    154  C   VAL A  22      41.114 -41.976   3.503  1.00 52.42           C  \nATOM    155  O   VAL A  22      40.688 -41.012   4.144  1.00 52.25           O  \nATOM    156  CB  VAL A  22      41.255 -44.291   4.543  1.00 52.10           C  \nATOM    157  CG1 VAL A  22      41.982 -43.487   5.608  1.00 51.45           C  \nATOM    158  CG2 VAL A  22      40.308 -45.257   5.244  1.00 49.84           C  \nATOM    159  N   LYS A  23      42.146 -41.900   2.670  1.00 53.11           N  \nATOM    160  CA  LYS A  23      42.903 -40.671   2.472  1.00 53.96           C  \nATOM    161  C   LYS A  23      42.096 -39.561   1.805  1.00 52.88           C  \nATOM    162  O   LYS A  23      42.242 -38.400   2.160  1.00 54.26           O  \nATOM    163  CB  LYS A  23      44.157 -40.966   1.651  1.00 57.80           C  \nATOM    164  CG  LYS A  23      45.099 -39.782   1.491  1.00 61.23           C  \nATOM    165  CD  LYS A  23      46.344 -40.214   0.752  1.00 62.97           C  \nATOM    166  CE  LYS A  23      47.269 -39.041   0.493  1.00 66.67           C  \nATOM    167  NZ  LYS A  23      48.322 -39.379  -0.510  1.00 67.64           N  \nATOM    168  N   ALA A  24      41.248 -39.912   0.842  1.00 51.34           N  \nATOM    169  CA  ALA A  24      40.422 -38.916   0.165  1.00 49.06           C  \nATOM    170  C   ALA A  24      39.337 -38.434   1.126  1.00 48.67           C  \nATOM    171  O   ALA A  24      38.942 -37.263   1.102  1.00 50.72           O  \nATOM    172  CB  ALA A  24      39.795 -39.512  -1.096  1.00 46.07           C  \nATOM    173  N   PHE A  25      38.886 -39.337   1.992  1.00 46.74           N  \nATOM    174  CA  PHE A  25      37.844 -39.047   2.968  1.00 45.55           C  \nATOM    175  C   PHE A  25      38.295 -38.131   4.091  1.00 47.15           C  \nATOM    176  O   PHE A  25      37.609 -37.168   4.421  1.00 47.06           O  \nATOM    177  CB  PHE A  25      37.305 -40.346   3.572  1.00 44.11           C  \nATOM    178  CG  PHE A  25      36.208 -40.129   4.565  1.00 44.40           C  \nATOM    179  CD1 PHE A  25      36.448 -40.245   5.932  1.00 44.48           C  \nATOM    180  CD2 PHE A  25      34.943 -39.747   4.136  1.00 46.53           C  \nATOM    181  CE1 PHE A  25      35.446 -39.984   6.854  1.00 44.83           C  \nATOM    182  CE2 PHE A  25      33.923 -39.479   5.056  1.00 46.80           C  \nATOM    183  CZ  PHE A  25      34.181 -39.598   6.420  1.00 45.86           C  \nATOM    184  N   VAL A  26      39.426 -38.462   4.705  1.00 48.49           N  \nATOM    185  CA  VAL A  26      39.983 -37.653   5.778  1.00 51.01           C  \nATOM    186  C   VAL A  26      40.248 -36.259   5.179  1.00 52.16           C  \nATOM    187  O   VAL A  26      40.006 -35.230   5.812  1.00 51.87           O  \nATOM    188  CB  VAL A  26      41.318 -38.271   6.292  1.00 53.20           C  \nATOM    189  CG1 VAL A  26      42.098 -37.248   7.103  1.00 49.44           C  \nATOM    190  CG2 VAL A  26      41.031 -39.542   7.127  1.00 51.33           C  \nATOM    191  N   SER A  27      40.720 -36.239   3.939  1.00 52.43           N  \nATOM    192  CA  SER A  27      41.016 -34.987   3.259  1.00 53.05           C  \nATOM    193  C   SER A  27      39.789 -34.080   3.062  1.00 54.74           C  \nATOM    194  O   SER A  27      39.830 -32.891   3.392  1.00 54.90           O  \nATOM    195  CB  SER A  27      41.675 -35.276   1.912  1.00 52.61           C  \nATOM    196  OG  SER A  27      42.026 -34.077   1.246  1.00 58.91           O  \nATOM    197  N   LYS A  28      38.697 -34.630   2.535  1.00 56.89           N  \nATOM    198  CA  LYS A  28      37.502 -33.826   2.322  1.00 58.10           C  \nATOM    199  C   LYS A  28      36.769 -33.529   3.627  1.00 57.99           C  \nATOM    200  O   LYS A  28      36.057 -32.533   3.736  1.00 60.20           O  \nATOM    201  CB  LYS A  28      36.569 -34.500   1.309  1.00 60.18           C  \nATOM    202  CG  LYS A  28      35.579 -35.535   1.831  1.00 65.20           C  \nATOM    203  CD  LYS A  28      34.211 -35.219   1.209  1.00 67.90           C  \nATOM    204  CE  LYS A  28      33.259 -36.412   1.050  1.00 71.12           C  \nATOM    205  NZ  LYS A  28      32.541 -36.839   2.285  1.00 72.35           N  \nATOM    206  N   LYS A  29      36.948 -34.376   4.631  1.00 57.79           N  \nATOM    207  CA  LYS A  29      36.302 -34.140   5.919  1.00 59.04           C  \nATOM    208  C   LYS A  29      36.903 -32.874   6.508  1.00 57.75           C  \nATOM    209  O   LYS A  29      36.200 -32.017   7.029  1.00 56.90           O  \nATOM    210  CB  LYS A  29      36.539 -35.308   6.882  1.00 60.66           C  \nATOM    211  CG  LYS A  29      36.012 -35.068   8.294  1.00 66.56           C  \nATOM    212  CD  LYS A  29      35.630 -36.376   8.991  1.00 72.57           C  \nATOM    213  CE  LYS A  29      36.744 -36.944   9.867  1.00 74.66           C  \nATOM    214  NZ  LYS A  29      36.676 -36.411  11.262  1.00 77.39           N  \nATOM    215  N   ARG A  30      38.221 -32.773   6.409  1.00 57.71           N  \nATOM    216  CA  ARG A  30      38.958 -31.622   6.910  1.00 57.99           C  \nATOM    217  C   ARG A  30      38.557 -30.343   6.149  1.00 55.90           C  \nATOM    218  O   ARG A  30      38.333 -29.295   6.753  1.00 55.11           O  \nATOM    219  CB  ARG A  30      40.458 -31.895   6.760  1.00 60.54           C  \nATOM    220  CG  ARG A  30      41.367 -30.755   7.163  1.00 68.07           C  \nATOM    221  CD  ARG A  30      42.823 -31.092   6.825  1.00 73.66           C  \nATOM    222  NE  ARG A  30      43.231 -32.365   7.422  1.00 79.67           N  \nATOM    223  CZ  ARG A  30      43.422 -32.565   8.726  1.00 82.24           C  \nATOM    224  NH1 ARG A  30      43.263 -31.572   9.598  1.00 81.63           N  \nATOM    225  NH2 ARG A  30      43.736 -33.780   9.162  1.00 82.68           N  \nATOM    226  N   ALA A  31      38.465 -30.439   4.823  1.00 53.57           N  \nATOM    227  CA  ALA A  31      38.091 -29.297   3.996  1.00 52.08           C  \nATOM    228  C   ALA A  31      36.705 -28.788   4.372  1.00 51.01           C  \nATOM    229  O   ALA A  31      36.482 -27.585   4.458  1.00 53.20           O  \nATOM    230  CB  ALA A  31      38.132 -29.676   2.514  1.00 50.78           C  \nATOM    231  N   ILE A  32      35.778 -29.707   4.607  1.00 51.29           N  \nATOM    232  CA  ILE A  32      34.412 -29.350   4.974  1.00 51.44           C  \nATOM    233  C   ILE A  32      34.354 -28.663   6.326  1.00 51.40           C  \nATOM    234  O   ILE A  32      33.644 -27.681   6.504  1.00 52.30           O  \nATOM    235  CB  ILE A  32      33.513 -30.609   4.975  1.00 52.68           C  \nATOM    236  CG1 ILE A  32      33.324 -31.075   3.529  1.00 51.57           C  \nATOM    237  CG2 ILE A  32      32.172 -30.331   5.687  1.00 52.78           C  \nATOM    238  CD1 ILE A  32      32.674 -32.415   3.407  1.00 51.38           C  \nATOM    239  N   LYS A  33      35.121 -29.178   7.274  1.00 53.16           N  \nATOM    240  CA  LYS A  33      35.155 -28.617   8.615  1.00 55.48           C  \nATOM    241  C   LYS A  33      35.713 -27.194   8.616  1.00 54.58           C  \nATOM    242  O   LYS A  33      35.300 -26.345   9.406  1.00 53.88           O  \nATOM    243  CB  LYS A  33      36.007 -29.498   9.526  1.00 60.19           C  \nATOM    244  CG  LYS A  33      35.403 -29.682  10.897  1.00 67.78           C  \nATOM    245  CD  LYS A  33      34.114 -30.489  10.804  1.00 74.45           C  \nATOM    246  CE  LYS A  33      32.983 -29.856  11.618  1.00 78.13           C  \nATOM    247  NZ  LYS A  33      32.631 -28.473  11.159  1.00 81.27           N  \nATOM    248  N   ASN A  34      36.674 -26.949   7.738  1.00 54.04           N  \nATOM    249  CA  ASN A  34      37.274 -25.637   7.621  1.00 54.76           C  \nATOM    250  C   ASN A  34      36.265 -24.657   7.041  1.00 53.60           C  \nATOM    251  O   ASN A  34      36.132 -23.552   7.541  1.00 51.50           O  \nATOM    252  CB  ASN A  34      38.523 -25.730   6.759  1.00 56.88           C  \nATOM    253  CG  ASN A  34      39.691 -26.327   7.516  1.00 60.49           C  \nATOM    254  OD1 ASN A  34      40.624 -26.860   6.921  1.00 62.60           O  \nATOM    255  ND2 ASN A  34      39.647 -26.230   8.845  1.00 62.27           N  \nATOM    256  N   GLU A  35      35.542 -25.078   6.008  1.00 54.52           N  \nATOM    257  CA  GLU A  35      34.519 -24.238   5.385  1.00 56.92           C  \nATOM    258  C   GLU A  35      33.455 -23.837   6.400  1.00 55.87           C  \nATOM    259  O   GLU A  35      33.123 -22.664   6.537  1.00 56.00           O  \nATOM    260  CB  GLU A  35      33.812 -24.986   4.247  1.00 60.06           C  \nATOM    261  CG  GLU A  35      34.567 -25.093   2.936  1.00 66.47           C  \nATOM    262  CD  GLU A  35      34.745 -23.749   2.254  1.00 70.12           C  \nATOM    263  OE1 GLU A  35      33.819 -22.909   2.344  1.00 68.84           O  \nATOM    264  OE2 GLU A  35      35.807 -23.544   1.618  1.00 72.11           O  \nATOM    265  N   GLN A  36      32.919 -24.828   7.107  1.00 56.17           N  \nATOM    266  CA  GLN A  36      31.853 -24.585   8.076  1.00 55.32           C  \nATOM    267  C   GLN A  36      32.245 -23.592   9.145  1.00 54.36           C  \nATOM    268  O   GLN A  36      31.433 -22.767   9.547  1.00 54.88           O  \nATOM    269  CB  GLN A  36      31.396 -25.900   8.725  1.00 56.58           C  \nATOM    270  CG  GLN A  36      30.815 -26.899   7.730  1.00 56.61           C  \nATOM    271  CD  GLN A  36      30.448 -28.238   8.370  1.00 58.00           C  \nATOM    272  OE1 GLN A  36      30.988 -28.621   9.411  1.00 56.56           O  \nATOM    273  NE2 GLN A  36      29.538 -28.961   7.732  1.00 54.90           N  \nATOM    274  N   LEU A  37      33.486 -23.658   9.609  1.00 53.55           N  \nATOM    275  CA  LEU A  37      33.922 -22.723  10.627  1.00 53.56           C  \nATOM    276  C   LEU A  37      34.032 -21.315  10.033  1.00 56.30           C  \nATOM    277  O   LEU A  37      33.573 -20.348  10.655  1.00 56.23           O  \nATOM    278  CB  LEU A  37      35.263 -23.160  11.224  1.00 51.15           C  \nATOM    279  CG  LEU A  37      35.858 -22.244  12.308  1.00 51.45           C  \nATOM    280  CD1 LEU A  37      34.872 -22.063  13.451  1.00 48.91           C  \nATOM    281  CD2 LEU A  37      37.166 -22.833  12.821  1.00 48.05           C  \nATOM    282  N   LEU A  38      34.617 -21.203   8.834  1.00 56.10           N  \nATOM    283  CA  LEU A  38      34.785 -19.907   8.176  1.00 56.72           C  \nATOM    284  C   LEU A  38      33.434 -19.281   7.909  1.00 56.43           C  \nATOM    285  O   LEU A  38      33.221 -18.111   8.199  1.00 56.32           O  \nATOM    286  CB  LEU A  38      35.522 -20.051   6.847  1.00 58.11           C  \nATOM    287  CG  LEU A  38      36.792 -19.214   6.658  1.00 59.65           C  \nATOM    288  CD1 LEU A  38      36.763 -18.636   5.262  1.00 61.09           C  \nATOM    289  CD2 LEU A  38      36.899 -18.089   7.682  1.00 59.80           C  \nATOM    290  N   GLN A  39      32.524 -20.065   7.344  1.00 56.65           N  \nATOM    291  CA  GLN A  39      31.185 -19.575   7.051  1.00 57.09           C  \nATOM    292  C   GLN A  39      30.482 -19.105   8.309  1.00 56.68           C  \nATOM    293  O   GLN A  39      29.807 -18.086   8.305  1.00 57.20           O  \nATOM    294  CB  GLN A  39      30.335 -20.656   6.399  1.00 55.45           C  \nATOM    295  CG  GLN A  39      30.854 -21.107   5.067  1.00 58.33           C  \nATOM    296  CD  GLN A  39      29.840 -21.913   4.280  1.00 60.68           C  \nATOM    297  OE1 GLN A  39      30.139 -22.426   3.195  1.00 60.37           O  \nATOM    298  NE2 GLN A  39      28.630 -22.019   4.815  1.00 61.06           N  \nATOM    299  N   LEU A  40      30.626 -19.861   9.382  1.00 56.60           N  \nATOM    300  CA  LEU A  40      29.984 -19.488  10.623  1.00 55.70           C  \nATOM    301  C   LEU A  40      30.577 -18.163  11.117  1.00 55.34           C  \nATOM    302  O   LEU A  40      29.865 -17.315  11.646  1.00 56.24           O  \nATOM    303  CB  LEU A  40      30.166 -20.608  11.650  1.00 54.19           C  \nATOM    304  CG  LEU A  40      29.532 -20.381  13.015  1.00 55.55           C  \nATOM    305  CD1 LEU A  40      29.356 -21.712  13.715  1.00 55.83           C  \nATOM    306  CD2 LEU A  40      30.405 -19.438  13.837  1.00 56.64           C  \nATOM    307  N   ILE A  41      31.879 -17.983  10.937  1.00 54.35           N  \nATOM    308  CA  ILE A  41      32.523 -16.750  11.364  1.00 54.54           C  \nATOM    309  C   ILE A  41      32.048 -15.595  10.477  1.00 57.02           C  \nATOM    310  O   ILE A  41      31.819 -14.474  10.955  1.00 56.48           O  \nATOM    311  CB  ILE A  41      34.049 -16.849  11.274  1.00 53.68           C  \nATOM    312  CG1 ILE A  41      34.562 -17.812  12.344  1.00 52.49           C  \nATOM    313  CG2 ILE A  41      34.666 -15.465  11.442  1.00 53.06           C  \nATOM    314  CD1 ILE A  41      36.051 -18.036  12.310  1.00 49.33           C  \nATOM    315  N   PHE A  42      31.902 -15.876   9.187  1.00 56.17           N  \nATOM    316  CA  PHE A  42      31.428 -14.873   8.250  1.00 58.04           C  \nATOM    317  C   PHE A  42      30.005 -14.464   8.661  1.00 59.07           C  \nATOM    318  O   PHE A  42      29.744 -13.293   8.958  1.00 58.91           O  \nATOM    319  CB  PHE A  42      31.412 -15.459   6.840  1.00 58.04           C  \nATOM    320  CG  PHE A  42      30.981 -14.490   5.788  1.00 60.21           C  \nATOM    321  CD1 PHE A  42      31.878 -13.566   5.273  1.00 59.94           C  \nATOM    322  CD2 PHE A  42      29.671 -14.488   5.316  1.00 58.76           C  \nATOM    323  CE1 PHE A  42      31.470 -12.652   4.297  1.00 62.30           C  \nATOM    324  CE2 PHE A  42      29.264 -13.575   4.344  1.00 61.79           C  \nATOM    325  CZ  PHE A  42      30.162 -12.663   3.838  1.00 63.22           C  \nATOM    326  N   LYS A  43      29.096 -15.440   8.672  1.00 59.76           N  \nATOM    327  CA  LYS A  43      27.708 -15.209   9.040  1.00 60.13           C  \nATOM    328  C   LYS A  43      27.550 -14.390  10.308  1.00 60.67           C  \nATOM    329  O   LYS A  43      26.612 -13.617  10.427  1.00 61.23           O  \nATOM    330  CB  LYS A  43      26.981 -16.531   9.243  1.00 62.08           C  \nATOM    331  CG  LYS A  43      26.472 -17.206   7.992  1.00 65.54           C  \nATOM    332  CD  LYS A  43      25.485 -18.290   8.414  1.00 70.32           C  \nATOM    333  CE  LYS A  43      25.695 -19.584   7.663  1.00 72.14           C  \nATOM    334  NZ  LYS A  43      25.363 -19.415   6.227  1.00 76.10           N  \nATOM    335  N   SER A  44      28.455 -14.568  11.260  1.00 61.05           N  \nATOM    336  CA  SER A  44      28.377 -13.852  12.521  1.00 63.99           C  \nATOM    337  C   SER A  44      28.811 -12.388  12.405  1.00 65.75           C  \nATOM    338  O   SER A  44      28.383 -11.544  13.196  1.00 67.65           O  \nATOM    339  CB  SER A  44      29.220 -14.588  13.571  1.00 65.71           C  \nATOM    340  OG  SER A  44      30.104 -13.707  14.244  1.00 70.19           O  \nATOM    341  N   ILE A  45      29.668 -12.088  11.432  1.00 65.18           N  \nATOM    342  CA  ILE A  45      30.122 -10.719  11.235  1.00 63.17           C  \nATOM    343  C   ILE A  45      29.077  -9.998  10.389  1.00 63.97           C  \nATOM    344  O   ILE A  45      28.797  -8.818  10.610  1.00 63.70           O  \nATOM    345  CB  ILE A  45      31.493 -10.664  10.506  1.00 62.12           C  \nATOM    346  CG1 ILE A  45      32.560 -11.376  11.348  1.00 61.94           C  \nATOM    347  CG2 ILE A  45      31.890  -9.213  10.256  1.00 61.46           C  \nATOM    348  CD1 ILE A  45      33.967 -11.430  10.736  1.00 59.04           C  \nATOM    349  N   ASP A  46      28.513 -10.721   9.420  1.00 63.87           N  \nATOM    350  CA  ASP A  46      27.488 -10.202   8.517  1.00 65.90           C  \nATOM    351  C   ASP A  46      26.182 -10.077   9.296  1.00 68.39           C  \nATOM    352  O   ASP A  46      25.229 -10.823   9.060  1.00 68.79           O  \nATOM    353  CB  ASP A  46      27.299 -11.163   7.333  1.00 64.85           C  \nATOM    354  CG  ASP A  46      26.160 -10.747   6.408  1.00 65.15           C  \nATOM    355  OD1 ASP A  46      25.455  -9.795   6.770  1.00 65.82           O  \nATOM    356  OD2 ASP A  46      25.958 -11.367   5.334  1.00 63.25           O  \nATOM    357  N   ALA A  47      26.140  -9.113  10.211  1.00 70.88           N  \nATOM    358  CA  ALA A  47      24.978  -8.896  11.063  1.00 72.87           C  \nATOM    359  C   ALA A  47      23.610  -8.766  10.399  1.00 74.65           C  \nATOM    360  O   ALA A  47      22.614  -9.097  11.029  1.00 75.62           O  \nATOM    361  CB  ALA A  47      25.230  -7.709  11.985  1.00 71.10           C  \nATOM    362  N   ASP A  48      23.524  -8.318   9.150  1.00 76.10           N  \nATOM    363  CA  ASP A  48      22.201  -8.207   8.537  1.00 76.91           C  \nATOM    364  C   ASP A  48      21.901  -9.310   7.524  1.00 76.46           C  \nATOM    365  O   ASP A  48      20.868  -9.278   6.853  1.00 78.51           O  \nATOM    366  CB  ASP A  48      21.997  -6.834   7.878  1.00 79.13           C  \nATOM    367  CG  ASP A  48      22.900  -6.608   6.677  1.00 82.23           C  \nATOM    368  OD1 ASP A  48      23.043  -7.533   5.844  1.00 83.07           O  \nATOM    369  OD2 ASP A  48      23.455  -5.494   6.561  1.00 82.40           O  \nATOM    370  N   GLY A  49      22.808 -10.274   7.402  1.00 74.73           N  \nATOM    371  CA  GLY A  49      22.608 -11.388   6.490  1.00 72.08           C  \nATOM    372  C   GLY A  49      22.449 -11.132   5.001  1.00 72.06           C  \nATOM    373  O   GLY A  49      21.905 -11.978   4.296  1.00 71.52           O  \nATOM    374  N   ASN A  50      22.904  -9.989   4.500  1.00 71.61           N  \nATOM    375  CA  ASN A  50      22.774  -9.735   3.067  1.00 71.61           C  \nATOM    376  C   ASN A  50      23.892 -10.456   2.281  1.00 71.10           C  \nATOM    377  O   ASN A  50      23.914 -10.410   1.046  1.00 70.57           O  \nATOM    378  CB  ASN A  50      22.813  -8.231   2.789  1.00 71.33           C  \nATOM    379  CG  ASN A  50      24.166  -7.650   3.060  1.00 71.84           C  \nATOM    380  OD1 ASN A  50      25.033  -8.346   3.585  1.00 70.61           O  \nATOM    381  ND2 ASN A  50      24.372  -6.383   2.709  1.00 69.84           N  \nATOM    382  N   GLY A  51      24.829 -11.094   2.991  1.00 70.23           N  \nATOM    383  CA  GLY A  51      25.893 -11.837   2.322  1.00 70.86           C  \nATOM    384  C   GLY A  51      27.182 -11.105   1.992  1.00 71.12           C  \nATOM    385  O   GLY A  51      27.943 -11.538   1.124  1.00 71.96           O  \nATOM    386  N   GLU A  52      27.443 -10.011   2.692  1.00 70.02           N  \nATOM    387  CA  GLU A  52      28.634  -9.217   2.451  1.00 70.16           C  \nATOM    388  C   GLU A  52      28.900  -8.465   3.749  1.00 68.33           C  \nATOM    389  O   GLU A  52      27.973  -8.188   4.489  1.00 67.83           O  \nATOM    390  CB  GLU A  52      28.345  -8.206   1.348  1.00 72.94           C  \nATOM    391  CG  GLU A  52      27.366  -7.137   1.827  1.00 76.86           C  \nATOM    392  CD  GLU A  52      27.178  -6.006   0.839  1.00 79.96           C  \nATOM    393  OE1 GLU A  52      26.727  -4.909   1.255  1.00 77.35           O  \nATOM    394  OE2 GLU A  52      27.469  -6.223  -0.358  1.00 81.92           O  \nATOM    395  N   ILE A  53      30.147  -8.130   4.043  1.00 66.11           N  \nATOM    396  CA  ILE A  53      30.420  -7.386   5.267  1.00 65.03           C  \nATOM    397  C   ILE A  53      30.761  -5.944   4.905  1.00 66.40           C  \nATOM    398  O   ILE A  53      31.716  -5.692   4.161  1.00 67.68           O  \nATOM    399  CB  ILE A  53      31.577  -8.037   6.060  1.00 62.51           C  \nATOM    400  CG1 ILE A  53      31.092  -9.375   6.624  1.00 63.80           C  \nATOM    401  CG2 ILE A  53      32.059  -7.116   7.176  1.00 60.38           C  \nATOM    402  CD1 ILE A  53      32.196 -10.303   7.065  1.00 61.98           C  \nATOM    403  N   ASP A  54      29.959  -4.996   5.385  1.00 66.39           N  \nATOM    404  CA  ASP A  54      30.242  -3.594   5.095  1.00 67.11           C  \nATOM    405  C   ASP A  54      31.174  -3.099   6.185  1.00 67.02           C  \nATOM    406  O   ASP A  54      31.385  -3.800   7.173  1.00 67.62           O  \nATOM    407  CB  ASP A  54      28.959  -2.758   5.022  1.00 66.36           C  \nATOM    408  CG  ASP A  54      28.129  -2.836   6.279  1.00 66.75           C  \nATOM    409  OD1 ASP A  54      28.659  -2.544   7.375  1.00 66.63           O  \nATOM    410  OD2 ASP A  54      26.935  -3.180   6.167  1.00 65.12           O  \nATOM    411  N   GLN A  55      31.744  -1.913   6.008  1.00 68.14           N  \nATOM    412  CA  GLN A  55      32.699  -1.392   6.977  1.00 68.15           C  \nATOM    413  C   GLN A  55      32.158  -1.268   8.377  1.00 68.99           C  \nATOM    414  O   GLN A  55      32.932  -1.220   9.334  1.00 69.56           O  \nATOM    415  CB  GLN A  55      33.229  -0.021   6.550  1.00 67.63           C  \nATOM    416  CG  GLN A  55      33.384   0.163   5.064  1.00 67.89           C  \nATOM    417  CD  GLN A  55      34.320   1.297   4.722  1.00 68.76           C  \nATOM    418  OE1 GLN A  55      34.404   2.288   5.447  1.00 70.13           O  \nATOM    419  NE2 GLN A  55      35.027   1.165   3.608  1.00 67.87           N  \nATOM    420  N   ASN A  56      30.839  -1.204   8.515  1.00 70.66           N  \nATOM    421  CA  ASN A  56      30.259  -1.050   9.841  1.00 71.83           C  \nATOM    422  C   ASN A  56      30.244  -2.391  10.550  1.00 70.10           C  \nATOM    423  O   ASN A  56      30.712  -2.513  11.685  1.00 70.18           O  \nATOM    424  CB  ASN A  56      28.850  -0.466   9.742  1.00 76.17           C  \nATOM    425  CG  ASN A  56      28.413   0.189  11.027  1.00 80.22           C  \nATOM    426  OD1 ASN A  56      27.495  -0.289  11.696  1.00 80.13           O  \nATOM    427  ND2 ASN A  56      29.077   1.291  11.389  1.00 81.73           N  \nATOM    428  N   GLU A  57      29.727  -3.407   9.874  1.00 68.32           N  \nATOM    429  CA  GLU A  57      29.711  -4.732  10.471  1.00 66.02           C  \nATOM    430  C   GLU A  57      31.132  -5.109  10.813  1.00 66.48           C  \nATOM    431  O   GLU A  57      31.386  -5.692  11.868  1.00 66.68           O  \nATOM    432  CB  GLU A  57      29.144  -5.738   9.499  1.00 62.73           C  \nATOM    433  CG  GLU A  57      27.733  -5.383   9.178  1.00 57.94           C  \nATOM    434  CD  GLU A  57      27.199  -6.141   7.997  1.00 58.67           C  \nATOM    435  OE1 GLU A  57      27.984  -6.412   7.067  1.00 57.68           O  \nATOM    436  OE2 GLU A  57      25.993  -6.455   7.978  1.00 55.58           O  \nATOM    437  N   PHE A  58      32.052  -4.753   9.918  1.00 65.19           N  \nATOM    438  CA  PHE A  58      33.467  -5.047  10.099  1.00 66.29           C  \nATOM    439  C   PHE A  58      34.034  -4.301  11.295  1.00 66.77           C  \nATOM    440  O   PHE A  58      34.887  -4.821  12.006  1.00 66.23           O  \nATOM    441  CB  PHE A  58      34.257  -4.652   8.842  1.00 65.26           C  \nATOM    442  CG  PHE A  58      35.739  -4.965   8.918  1.00 64.30           C  \nATOM    443  CD1 PHE A  58      36.248  -6.182   8.444  1.00 64.45           C  \nATOM    444  CD2 PHE A  58      36.632  -4.038   9.455  1.00 62.83           C  \nATOM    445  CE1 PHE A  58      37.619  -6.461   8.509  1.00 63.56           C  \nATOM    446  CE2 PHE A  58      38.001  -4.314   9.523  1.00 61.37           C  \nATOM    447  CZ  PHE A  58      38.490  -5.529   9.046  1.00 62.84           C  \nATOM    448  N   ALA A  59      33.570  -3.073  11.495  1.00 69.76           N  \nATOM    449  CA  ALA A  59      34.044  -2.229  12.588  1.00 71.11           C  \nATOM    450  C   ALA A  59      33.662  -2.757  13.968  1.00 71.82           C  \nATOM    451  O   ALA A  59      34.494  -2.768  14.882  1.00 70.47           O  \nATOM    452  CB  ALA A  59      33.518  -0.819  12.408  1.00 70.48           C  \nATOM    453  N   LYS A  60      32.408  -3.176  14.134  1.00 73.21           N  \nATOM    454  CA  LYS A  60      31.973  -3.702  15.429  1.00 77.01           C  \nATOM    455  C   LYS A  60      32.858  -4.879  15.808  1.00 78.78           C  \nATOM    456  O   LYS A  60      33.469  -4.895  16.881  1.00 78.94           O  \nATOM    457  CB  LYS A  60      30.512  -4.150  15.376  1.00 79.88           C  \nATOM    458  CG  LYS A  60      29.511  -3.003  15.337  1.00 84.50           C  \nATOM    459  CD  LYS A  60      28.184  -3.495  14.782  1.00 86.60           C  \nATOM    460  CE  LYS A  60      27.230  -2.354  14.462  1.00 87.76           C  \nATOM    461  NZ  LYS A  60      26.189  -2.144  15.513  1.00 88.30           N  \nATOM    462  N   PHE A  61      32.922  -5.853  14.906  1.00 79.85           N  \nATOM    463  CA  PHE A  61      33.729  -7.045  15.082  1.00 81.17           C  \nATOM    464  C   PHE A  61      35.115  -6.716  15.625  1.00 83.52           C  \nATOM    465  O   PHE A  61      35.602  -7.397  16.523  1.00 84.19           O  \nATOM    466  CB  PHE A  61      33.856  -7.778  13.746  1.00 80.08           C  \nATOM    467  CG  PHE A  61      34.697  -9.017  13.809  1.00 78.61           C  \nATOM    468  CD1 PHE A  61      34.201 -10.192  14.372  1.00 77.96           C  \nATOM    469  CD2 PHE A  61      35.993  -9.012  13.314  1.00 77.34           C  \nATOM    470  CE1 PHE A  61      34.986 -11.337  14.442  1.00 76.92           C  \nATOM    471  CE2 PHE A  61      36.785 -10.154  13.383  1.00 78.40           C  \nATOM    472  CZ  PHE A  61      36.278 -11.320  13.948  1.00 77.68           C  \nATOM    473  N   TYR A  62      35.770  -5.695  15.083  1.00 86.85           N  \nATOM    474  CA  TYR A  62      37.091  -5.354  15.595  1.00 91.54           C  \nATOM    475  C   TYR A  62      37.040  -4.449  16.824  1.00 95.16           C  \nATOM    476  O   TYR A  62      35.976  -3.944  17.208  1.00 95.30           O  \nATOM    477  CB  TYR A  62      37.970  -4.731  14.504  1.00 91.83           C  \nATOM    478  CG  TYR A  62      38.700  -5.772  13.690  1.00 93.17           C  \nATOM    479  CD1 TYR A  62      38.097  -6.372  12.587  1.00 94.03           C  \nATOM    480  CD2 TYR A  62      39.959  -6.225  14.077  1.00 93.75           C  \nATOM    481  CE1 TYR A  62      38.741  -7.404  11.883  1.00 94.66           C  \nATOM    482  CE2 TYR A  62      40.605  -7.250  13.388  1.00 94.04           C  \nATOM    483  CZ  TYR A  62      39.991  -7.838  12.296  1.00 94.49           C  \nATOM    484  OH  TYR A  62      40.632  -8.853  11.628  1.00 94.92           O  \nATOM    485  N   GLY A  63      38.199  -4.277  17.450  1.00 98.54           N  \nATOM    486  CA  GLY A  63      38.305  -3.455  18.641  1.00102.52           C  \nATOM    487  C   GLY A  63      39.067  -4.309  19.627  1.00105.41           C  \nATOM    488  O   GLY A  63      38.624  -4.532  20.760  1.00106.17           O  \nATOM    489  N   SER A  64      40.248  -4.748  19.190  1.00107.73           N  \nATOM    490  CA  SER A  64      41.068  -5.658  19.978  1.00109.81           C  \nATOM    491  C   SER A  64      39.974  -6.727  20.109  1.00110.88           C  \nATOM    492  O   SER A  64      39.711  -7.279  21.184  1.00111.30           O  \nATOM    493  CB  SER A  64      41.470  -5.009  21.323  1.00109.62           C  \nATOM    494  OG  SER A  64      40.526  -5.219  22.367  1.00109.17           O  \nATOM    495  N   ILE A  65      39.355  -6.963  18.944  1.00112.22           N  \nATOM    496  CA  ILE A  65      38.218  -7.846  18.692  1.00112.85           C  \nATOM    497  C   ILE A  65      37.298  -8.173  19.872  1.00112.86           C  \nATOM    498  O   ILE A  65      37.591  -8.962  20.773  1.00113.51           O  \nATOM    499  CB  ILE A  65      38.643  -9.117  17.841  1.00112.76           C  \nATOM    500  CG1 ILE A  65      37.999 -10.393  18.391  1.00112.63           C  \nATOM    501  CG2 ILE A  65      40.158  -9.142  17.648  1.00112.92           C  \nATOM    502  CD1 ILE A  65      36.538 -10.593  17.973  1.00112.22           C  \nATOM    503  N   GLN A  66      36.183  -7.460  19.852  1.00112.67           N  \nATOM    504  CA  GLN A  66      35.121  -7.572  20.827  1.00111.53           C  \nATOM    505  C   GLN A  66      34.018  -6.963  19.983  1.00110.94           C  \nATOM    506  O   GLN A  66      34.101  -5.791  19.608  1.00110.41           O  \nATOM    507  CB  GLN A  66      35.417  -6.704  22.053  1.00113.59           C  \nATOM    508  N   GLY A  67      33.018  -7.758  19.629  1.00110.66           N  \nATOM    509  CA  GLY A  67      31.951  -7.234  18.800  1.00110.60           C  \nATOM    510  C   GLY A  67      31.123  -8.366  18.229  1.00111.34           C  \nATOM    511  O   GLY A  67      31.372  -8.800  17.080  1.00109.76           O  \nTER     512      GLY A  67                                                      \nATOM    513  N   ALA B   2      20.076 -32.134  18.404  1.00 95.47           N  \nATOM    514  CA  ALA B   2      20.129 -33.613  18.635  1.00 95.82           C  \nATOM    515  C   ALA B   2      19.992 -34.020  20.119  1.00 95.77           C  \nATOM    516  O   ALA B   2      20.571 -33.391  21.006  1.00 95.11           O  \nATOM    517  CB  ALA B   2      21.425 -34.169  18.046  1.00 96.01           C  \nATOM    518  N   GLU B   3      19.249 -35.106  20.348  1.00 95.42           N  \nATOM    519  CA  GLU B   3      18.909 -35.683  21.664  1.00 93.30           C  \nATOM    520  C   GLU B   3      19.913 -36.043  22.783  1.00 90.15           C  \nATOM    521  O   GLU B   3      19.979 -35.353  23.803  1.00 90.58           O  \nATOM    522  CB  GLU B   3      18.032 -36.901  21.397  1.00 95.00           C  \nATOM    523  CG  GLU B   3      17.372 -36.824  20.043  1.00 97.61           C  \nATOM    524  CD  GLU B   3      16.612 -38.085  19.698  1.00100.23           C  \nATOM    525  OE1 GLU B   3      16.614 -38.449  18.503  1.00100.56           O  \nATOM    526  OE2 GLU B   3      16.012 -38.711  20.607  1.00 99.87           O  \nATOM    527  N   ALA B   4      20.650 -37.141  22.637  1.00 85.59           N  \nATOM    528  CA  ALA B   4      21.578 -37.545  23.697  1.00 80.40           C  \nATOM    529  C   ALA B   4      22.774 -36.624  23.801  1.00 76.71           C  \nATOM    530  O   ALA B   4      23.317 -36.398  24.883  1.00 76.72           O  \nATOM    531  CB  ALA B   4      22.046 -38.978  23.485  1.00 80.62           C  \nATOM    532  N   LEU B   5      23.181 -36.092  22.662  1.00 72.37           N  \nATOM    533  CA  LEU B   5      24.305 -35.186  22.623  1.00 68.42           C  \nATOM    534  C   LEU B   5      24.010 -33.937  23.453  1.00 65.54           C  \nATOM    535  O   LEU B   5      24.872 -33.460  24.191  1.00 64.25           O  \nATOM    536  CB  LEU B   5      24.606 -34.800  21.177  1.00 69.45           C  \nATOM    537  CG  LEU B   5      25.833 -33.912  20.967  1.00 71.43           C  \nATOM    538  CD1 LEU B   5      27.050 -34.530  21.645  1.00 71.65           C  \nATOM    539  CD2 LEU B   5      26.073 -33.733  19.481  1.00 71.25           C  \nATOM    540  N   PHE B   6      22.796 -33.408  23.353  1.00 61.71           N  \nATOM    541  CA  PHE B   6      22.465 -32.210  24.117  1.00 59.19           C  \nATOM    542  C   PHE B   6      22.516 -32.496  25.620  1.00 57.98           C  \nATOM    543  O   PHE B   6      22.944 -31.646  26.408  1.00 56.49           O  \nATOM    544  CB  PHE B   6      21.083 -31.680  23.709  1.00 56.79           C  \nATOM    545  CG  PHE B   6      20.706 -30.386  24.377  1.00 56.41           C  \nATOM    546  CD1 PHE B   6      20.222 -30.374  25.688  1.00 54.17           C  \nATOM    547  CD2 PHE B   6      20.865 -29.173  23.712  1.00 53.81           C  \nATOM    548  CE1 PHE B   6      19.914 -29.179  26.323  1.00 52.39           C  \nATOM    549  CE2 PHE B   6      20.559 -27.972  24.343  1.00 53.56           C  \nATOM    550  CZ  PHE B   6      20.083 -27.977  25.651  1.00 52.29           C  \nATOM    551  N   LYS B   7      22.090 -33.698  26.006  1.00 57.51           N  \nATOM    552  CA  LYS B   7      22.091 -34.095  27.409  1.00 56.99           C  \nATOM    553  C   LYS B   7      23.516 -34.266  27.884  1.00 56.23           C  \nATOM    554  O   LYS B   7      23.836 -33.967  29.028  1.00 55.52           O  \nATOM    555  CB  LYS B   7      21.328 -35.412  27.602  1.00 58.83           C  \nATOM    556  CG  LYS B   7      19.811 -35.266  27.704  1.00 57.62           C  \nATOM    557  CD  LYS B   7      19.126 -36.635  27.760  1.00 59.19           C  \nATOM    558  CE  LYS B   7      18.175 -36.770  28.953  1.00 58.59           C  \nATOM    559  NZ  LYS B   7      18.877 -37.020  30.248  1.00 58.58           N  \nATOM    560  N   GLU B   8      24.378 -34.750  27.001  1.00 57.34           N  \nATOM    561  CA  GLU B   8      25.769 -34.949  27.367  1.00 57.77           C  \nATOM    562  C   GLU B   8      26.479 -33.621  27.629  1.00 57.06           C  \nATOM    563  O   GLU B   8      27.293 -33.512  28.546  1.00 58.29           O  \nATOM    564  CB  GLU B   8      26.507 -35.729  26.271  1.00 62.14           C  \nATOM    565  CG  GLU B   8      27.956 -36.050  26.622  1.00 68.05           C  \nATOM    566  CD  GLU B   8      28.726 -36.692  25.469  1.00 72.75           C  \nATOM    567  OE1 GLU B   8      28.128 -36.919  24.387  1.00 73.97           O  \nATOM    568  OE2 GLU B   8      29.935 -36.970  25.645  1.00 74.25           O  \nATOM    569  N   ILE B   9      26.184 -32.608  26.824  1.00 55.30           N  \nATOM    570  CA  ILE B   9      26.817 -31.312  27.019  1.00 55.05           C  \nATOM    571  C   ILE B   9      26.155 -30.623  28.211  1.00 56.32           C  \nATOM    572  O   ILE B   9      26.841 -30.032  29.038  1.00 55.44           O  \nATOM    573  CB  ILE B   9      26.672 -30.418  25.764  1.00 54.92           C  \nATOM    574  CG1 ILE B   9      27.363 -31.093  24.580  1.00 53.41           C  \nATOM    575  CG2 ILE B   9      27.234 -29.015  26.030  1.00 52.19           C  \nATOM    576  CD1 ILE B   9      27.138 -30.410  23.250  1.00 55.03           C  \nATOM    577  N   ASP B  10      24.821 -30.704  28.291  1.00 56.81           N  \nATOM    578  CA  ASP B  10      24.060 -30.090  29.388  1.00 56.37           C  \nATOM    579  C   ASP B  10      24.323 -30.866  30.672  1.00 56.94           C  \nATOM    580  O   ASP B  10      23.472 -31.584  31.177  1.00 57.14           O  \nATOM    581  CB  ASP B  10      22.557 -30.085  29.064  1.00 55.76           C  \nATOM    582  CG  ASP B  10      21.691 -29.620  30.245  1.00 55.92           C  \nATOM    583  OD1 ASP B  10      22.192 -28.885  31.128  1.00 53.72           O  \nATOM    584  OD2 ASP B  10      20.496 -29.989  30.285  1.00 54.83           O  \nATOM    585  N   VAL B  11      25.520 -30.693  31.206  1.00 57.72           N  \nATOM    586  CA  VAL B  11      25.954 -31.399  32.408  1.00 58.49           C  \nATOM    587  C   VAL B  11      25.179 -31.249  33.732  1.00 59.51           C  \nATOM    588  O   VAL B  11      25.120 -32.201  34.511  1.00 59.54           O  \nATOM    589  CB  VAL B  11      27.460 -31.105  32.644  1.00 57.09           C  \nATOM    590  CG1 VAL B  11      27.847 -31.326  34.096  1.00 56.42           C  \nATOM    591  CG2 VAL B  11      28.275 -31.977  31.731  1.00 55.23           C  \nATOM    592  N   ASN B  12      24.607 -30.080  34.015  1.00 59.39           N  \nATOM    593  CA  ASN B  12      23.867 -29.932  35.268  1.00 58.33           C  \nATOM    594  C   ASN B  12      22.370 -30.185  35.057  1.00 58.79           C  \nATOM    595  O   ASN B  12      21.556 -30.005  35.969  1.00 58.20           O  \nATOM    596  CB  ASN B  12      24.105 -28.549  35.896  1.00 57.74           C  \nATOM    597  CG  ASN B  12      23.417 -27.460  35.154  1.00 58.79           C  \nATOM    598  OD1 ASN B  12      23.016 -27.657  34.021  1.00 60.25           O  \nATOM    599  ND2 ASN B  12      23.275 -26.295  35.780  1.00 59.19           N  \nATOM    600  N   GLY B  13      22.024 -30.583  33.835  1.00 59.41           N  \nATOM    601  CA  GLY B  13      20.658 -30.927  33.481  1.00 57.34           C  \nATOM    602  C   GLY B  13      19.521 -29.940  33.533  1.00 57.47           C  \nATOM    603  O   GLY B  13      18.378 -30.364  33.619  1.00 56.77           O  \nATOM    604  N   ASP B  14      19.791 -28.644  33.454  1.00 58.36           N  \nATOM    605  CA  ASP B  14      18.700 -27.674  33.515  1.00 56.69           C  \nATOM    606  C   ASP B  14      18.049 -27.313  32.198  1.00 56.22           C  \nATOM    607  O   ASP B  14      17.279 -26.351  32.147  1.00 57.62           O  \nATOM    608  CB  ASP B  14      19.154 -26.377  34.164  1.00 57.93           C  \nATOM    609  CG  ASP B  14      20.224 -25.667  33.363  1.00 61.27           C  \nATOM    610  OD1 ASP B  14      20.393 -25.969  32.155  1.00 60.89           O  \nATOM    611  OD2 ASP B  14      20.888 -24.791  33.954  1.00 60.93           O  \nATOM    612  N   GLY B  15      18.363 -28.050  31.136  1.00 55.54           N  \nATOM    613  CA  GLY B  15      17.776 -27.770  29.834  1.00 54.34           C  \nATOM    614  C   GLY B  15      18.488 -26.720  28.995  1.00 55.32           C  \nATOM    615  O   GLY B  15      18.154 -26.525  27.840  1.00 56.93           O  \nATOM    616  N   ALA B  16      19.469 -26.033  29.559  1.00 54.89           N  \nATOM    617  CA  ALA B  16      20.181 -25.012  28.797  1.00 56.39           C  \nATOM    618  C   ALA B  16      21.661 -25.346  28.713  1.00 55.83           C  \nATOM    619  O   ALA B  16      22.234 -25.777  29.688  1.00 55.14           O  \nATOM    620  CB  ALA B  16      20.001 -23.645  29.462  1.00 55.14           C  \nATOM    621  N   VAL B  17      22.285 -25.179  27.552  1.00 54.73           N  \nATOM    622  CA  VAL B  17      23.726 -25.441  27.450  1.00 53.18           C  \nATOM    623  C   VAL B  17      24.403 -24.092  27.677  1.00 51.38           C  \nATOM    624  O   VAL B  17      24.192 -23.150  26.910  1.00 50.05           O  \nATOM    625  CB  VAL B  17      24.086 -26.015  26.057  1.00 55.38           C  \nATOM    626  CG1 VAL B  17      25.571 -25.871  25.777  1.00 56.58           C  \nATOM    627  CG2 VAL B  17      23.689 -27.487  25.997  1.00 54.50           C  \nATOM    628  N   SER B  18      25.161 -23.975  28.764  1.00 50.36           N  \nATOM    629  CA  SER B  18      25.834 -22.712  29.072  1.00 50.97           C  \nATOM    630  C   SER B  18      27.199 -22.581  28.384  1.00 52.81           C  \nATOM    631  O   SER B  18      27.715 -23.548  27.814  1.00 51.44           O  \nATOM    632  CB  SER B  18      26.015 -22.556  30.587  1.00 49.79           C  \nATOM    633  OG  SER B  18      27.037 -23.402  31.086  1.00 53.03           O  \nATOM    634  N   TYR B  19      27.781 -21.383  28.443  1.00 54.93           N  \nATOM    635  CA  TYR B  19      29.082 -21.153  27.829  1.00 55.62           C  \nATOM    636  C   TYR B  19      30.148 -22.027  28.468  1.00 55.99           C  \nATOM    637  O   TYR B  19      30.909 -22.693  27.772  1.00 56.26           O  \nATOM    638  CB  TYR B  19      29.510 -19.685  27.932  1.00 55.25           C  \nATOM    639  CG  TYR B  19      30.871 -19.479  27.335  1.00 52.62           C  \nATOM    640  CD1 TYR B  19      31.086 -19.702  25.975  1.00 51.39           C  \nATOM    641  CD2 TYR B  19      31.958 -19.147  28.135  1.00 53.37           C  \nATOM    642  CE1 TYR B  19      32.357 -19.599  25.423  1.00 54.31           C  \nATOM    643  CE2 TYR B  19      33.239 -19.036  27.593  1.00 53.93           C  \nATOM    644  CZ  TYR B  19      33.431 -19.269  26.243  1.00 54.61           C  \nATOM    645  OH  TYR B  19      34.699 -19.185  25.729  1.00 55.25           O  \nATOM    646  N   GLU B  20      30.199 -22.030  29.793  1.00 57.18           N  \nATOM    647  CA  GLU B  20      31.171 -22.846  30.499  1.00 60.17           C  \nATOM    648  C   GLU B  20      30.972 -24.346  30.229  1.00 58.63           C  \nATOM    649  O   GLU B  20      31.898 -25.140  30.380  1.00 59.80           O  \nATOM    650  CB  GLU B  20      31.120 -22.551  32.006  1.00 64.98           C  \nATOM    651  CG  GLU B  20      29.715 -22.342  32.575  1.00 75.73           C  \nATOM    652  CD  GLU B  20      29.295 -20.867  32.626  1.00 81.43           C  \nATOM    653  OE1 GLU B  20      29.536 -20.209  33.664  1.00 83.27           O  \nATOM    654  OE2 GLU B  20      28.729 -20.361  31.627  1.00 82.64           O  \nATOM    655  N   GLU B  21      29.772 -24.738  29.814  1.00 56.81           N  \nATOM    656  CA  GLU B  21      29.511 -26.146  29.520  1.00 53.63           C  \nATOM    657  C   GLU B  21      29.987 -26.505  28.121  1.00 52.09           C  \nATOM    658  O   GLU B  21      30.416 -27.635  27.874  1.00 50.79           O  \nATOM    659  CB  GLU B  21      28.023 -26.466  29.660  1.00 53.23           C  \nATOM    660  CG  GLU B  21      27.581 -26.573  31.107  1.00 57.45           C  \nATOM    661  CD  GLU B  21      26.102 -26.842  31.242  1.00 58.49           C  \nATOM    662  OE1 GLU B  21      25.370 -26.617  30.247  1.00 58.46           O  \nATOM    663  OE2 GLU B  21      25.678 -27.262  32.342  1.00 59.49           O  \nATOM    664  N   VAL B  22      29.890 -25.546  27.203  1.00 50.79           N  \nATOM    665  CA  VAL B  22      30.347 -25.756  25.832  1.00 51.61           C  \nATOM    666  C   VAL B  22      31.869 -25.868  25.931  1.00 50.48           C  \nATOM    667  O   VAL B  22      32.485 -26.731  25.305  1.00 48.12           O  \nATOM    668  CB  VAL B  22      29.943 -24.559  24.896  1.00 51.69           C  \nATOM    669  CG1 VAL B  22      30.867 -24.471  23.692  1.00 52.55           C  \nATOM    670  CG2 VAL B  22      28.521 -24.757  24.384  1.00 51.90           C  \nATOM    671  N   LYS B  23      32.454 -25.008  26.761  1.00 51.15           N  \nATOM    672  CA  LYS B  23      33.891 -24.978  26.973  1.00 52.73           C  \nATOM    673  C   LYS B  23      34.422 -26.282  27.569  1.00 51.50           C  \nATOM    674  O   LYS B  23      35.446 -26.797  27.128  1.00 52.63           O  \nATOM    675  CB  LYS B  23      34.247 -23.795  27.873  1.00 56.44           C  \nATOM    676  CG  LYS B  23      35.726 -23.490  27.930  1.00 60.90           C  \nATOM    677  CD  LYS B  23      35.976 -22.155  28.599  1.00 63.82           C  \nATOM    678  CE  LYS B  23      36.893 -22.333  29.800  1.00 67.19           C  \nATOM    679  NZ  LYS B  23      37.705 -21.121  30.130  1.00 69.13           N  \nATOM    680  N   ALA B  24      33.727 -26.825  28.562  1.00 50.23           N  \nATOM    681  CA  ALA B  24      34.164 -28.070  29.175  1.00 47.47           C  \nATOM    682  C   ALA B  24      34.019 -29.225  28.197  1.00 47.89           C  \nATOM    683  O   ALA B  24      34.818 -30.165  28.206  1.00 48.13           O  \nATOM    684  CB  ALA B  24      33.357 -28.345  30.407  1.00 44.03           C  \nATOM    685  N   PHE B  25      32.999 -29.144  27.347  1.00 46.65           N  \nATOM    686  CA  PHE B  25      32.728 -30.198  26.379  1.00 46.20           C  \nATOM    687  C   PHE B  25      33.762 -30.258  25.268  1.00 48.36           C  \nATOM    688  O   PHE B  25      34.266 -31.333  24.928  1.00 48.16           O  \nATOM    689  CB  PHE B  25      31.324 -30.030  25.776  1.00 44.92           C  \nATOM    690  CG  PHE B  25      30.962 -31.111  24.803  1.00 46.42           C  \nATOM    691  CD1 PHE B  25      30.988 -30.869  23.437  1.00 47.67           C  \nATOM    692  CD2 PHE B  25      30.659 -32.397  25.253  1.00 49.02           C  \nATOM    693  CE1 PHE B  25      30.732 -31.886  22.526  1.00 49.24           C  \nATOM    694  CE2 PHE B  25      30.402 -33.428  24.352  1.00 49.86           C  \nATOM    695  CZ  PHE B  25      30.437 -33.172  22.980  1.00 49.93           C  \nATOM    696  N   VAL B  26      34.061 -29.102  24.691  1.00 48.36           N  \nATOM    697  CA  VAL B  26      35.038 -29.023  23.625  1.00 50.83           C  \nATOM    698  C   VAL B  26      36.385 -29.508  24.181  1.00 51.61           C  \nATOM    699  O   VAL B  26      37.107 -30.277  23.545  1.00 51.51           O  \nATOM    700  CB  VAL B  26      35.144 -27.569  23.124  1.00 52.19           C  \nATOM    701  CG1 VAL B  26      36.507 -27.319  22.511  1.00 52.04           C  \nATOM    702  CG2 VAL B  26      34.015 -27.286  22.106  1.00 51.15           C  \nATOM    703  N   SER B  27      36.704 -29.064  25.387  1.00 51.99           N  \nATOM    704  CA  SER B  27      37.941 -29.451  26.045  1.00 53.35           C  \nATOM    705  C   SER B  27      38.015 -30.975  26.265  1.00 54.05           C  \nATOM    706  O   SER B  27      39.003 -31.629  25.910  1.00 53.39           O  \nATOM    707  CB  SER B  27      38.047 -28.721  27.378  1.00 54.91           C  \nATOM    708  OG  SER B  27      39.054 -29.314  28.175  1.00 64.57           O  \nATOM    709  N   LYS B  28      36.949 -31.523  26.841  1.00 55.10           N  \nATOM    710  CA  LYS B  28      36.825 -32.950  27.115  1.00 55.50           C  \nATOM    711  C   LYS B  28      36.894 -33.767  25.825  1.00 55.10           C  \nATOM    712  O   LYS B  28      37.495 -34.831  25.783  1.00 56.39           O  \nATOM    713  CB  LYS B  28      35.485 -33.222  27.801  1.00 56.98           C  \nATOM    714  CG  LYS B  28      35.193 -34.705  27.980  1.00 60.37           C  \nATOM    715  CD  LYS B  28      33.779 -35.019  28.483  1.00 60.75           C  \nATOM    716  CE  LYS B  28      32.800 -35.288  27.335  1.00 63.32           C  \nATOM    717  NZ  LYS B  28      31.514 -35.943  27.773  1.00 64.17           N  \nATOM    718  N   LYS B  29      36.261 -33.263  24.772  1.00 55.86           N  \nATOM    719  CA  LYS B  29      36.238 -33.959  23.493  1.00 55.69           C  \nATOM    720  C   LYS B  29      37.636 -34.072  22.916  1.00 55.51           C  \nATOM    721  O   LYS B  29      38.018 -35.103  22.359  1.00 56.03           O  \nATOM    722  CB  LYS B  29      35.318 -33.229  22.505  1.00 57.61           C  \nATOM    723  CG  LYS B  29      35.324 -33.816  21.089  1.00 62.10           C  \nATOM    724  CD  LYS B  29      34.128 -33.308  20.268  1.00 67.12           C  \nATOM    725  CE  LYS B  29      34.482 -32.201  19.268  1.00 69.04           C  \nATOM    726  NZ  LYS B  29      34.531 -32.707  17.855  1.00 69.56           N  \nATOM    727  N   ARG B  30      38.385 -32.987  23.057  1.00 55.86           N  \nATOM    728  CA  ARG B  30      39.754 -32.899  22.580  1.00 56.05           C  \nATOM    729  C   ARG B  30      40.626 -33.956  23.285  1.00 53.47           C  \nATOM    730  O   ARG B  30      41.330 -34.718  22.634  1.00 53.79           O  \nATOM    731  CB  ARG B  30      40.288 -31.489  22.870  1.00 59.14           C  \nATOM    732  CG  ARG B  30      41.514 -31.077  22.060  1.00 65.48           C  \nATOM    733  CD  ARG B  30      42.400 -30.060  22.811  1.00 71.24           C  \nATOM    734  NE  ARG B  30      41.671 -28.928  23.397  1.00 76.38           N  \nATOM    735  CZ  ARG B  30      40.968 -28.026  22.712  1.00 78.47           C  \nATOM    736  NH1 ARG B  30      40.880 -28.109  21.390  1.00 78.07           N  \nATOM    737  NH2 ARG B  30      40.365 -27.025  23.349  1.00 78.49           N  \nATOM    738  N   ALA B  31      40.570 -33.986  24.616  1.00 51.84           N  \nATOM    739  CA  ALA B  31      41.335 -34.930  25.429  1.00 51.56           C  \nATOM    740  C   ALA B  31      41.078 -36.364  25.011  1.00 51.25           C  \nATOM    741  O   ALA B  31      42.010 -37.144  24.830  1.00 52.66           O  \nATOM    742  CB  ALA B  31      40.983 -34.760  26.890  1.00 49.83           C  \nATOM    743  N   ILE B  32      39.807 -36.703  24.855  1.00 50.88           N  \nATOM    744  CA  ILE B  32      39.415 -38.047  24.460  1.00 51.68           C  \nATOM    745  C   ILE B  32      39.982 -38.440  23.109  1.00 51.92           C  \nATOM    746  O   ILE B  32      40.498 -39.541  22.955  1.00 51.03           O  \nATOM    747  CB  ILE B  32      37.873 -38.176  24.444  1.00 52.67           C  \nATOM    748  CG1 ILE B  32      37.359 -38.142  25.887  1.00 51.94           C  \nATOM    749  CG2 ILE B  32      37.440 -39.452  23.715  1.00 51.65           C  \nATOM    750  CD1 ILE B  32      35.870 -37.900  26.008  1.00 53.78           C  \nATOM    751  N   LYS B  33      39.902 -37.528  22.142  1.00 54.32           N  \nATOM    752  CA  LYS B  33      40.400 -37.779  20.790  1.00 56.02           C  \nATOM    753  C   LYS B  33      41.911 -37.978  20.776  1.00 55.33           C  \nATOM    754  O   LYS B  33      42.441 -38.768  19.988  1.00 54.86           O  \nATOM    755  CB  LYS B  33      40.033 -36.624  19.855  1.00 59.75           C  \nATOM    756  CG  LYS B  33      39.341 -37.079  18.572  1.00 66.26           C  \nATOM    757  CD  LYS B  33      37.822 -37.135  18.750  1.00 73.32           C  \nATOM    758  CE  LYS B  33      37.090 -37.682  17.512  1.00 76.87           C  \nATOM    759  NZ  LYS B  33      37.291 -39.146  17.265  1.00 78.45           N  \nATOM    760  N   ASN B  34      42.602 -37.249  21.645  1.00 55.38           N  \nATOM    761  CA  ASN B  34      44.043 -37.373  21.755  1.00 54.76           C  \nATOM    762  C   ASN B  34      44.401 -38.721  22.374  1.00 54.18           C  \nATOM    763  O   ASN B  34      45.317 -39.387  21.911  1.00 52.49           O  \nATOM    764  CB  ASN B  34      44.588 -36.221  22.584  1.00 56.84           C  \nATOM    765  CG  ASN B  34      44.664 -34.934  21.786  1.00 60.81           C  \nATOM    766  OD1 ASN B  34      44.787 -33.845  22.346  1.00 62.79           O  \nATOM    767  ND2 ASN B  34      44.604 -35.058  20.458  1.00 60.83           N  \nATOM    768  N   GLU B  35      43.670 -39.127  23.408  1.00 54.61           N  \nATOM    769  CA  GLU B  35      43.900 -40.420  24.055  1.00 57.20           C  \nATOM    770  C   GLU B  35      43.739 -41.539  23.037  1.00 56.57           C  \nATOM    771  O   GLU B  35      44.628 -42.371  22.854  1.00 57.34           O  \nATOM    772  CB  GLU B  35      42.872 -40.686  25.165  1.00 59.86           C  \nATOM    773  CG  GLU B  35      43.149 -40.067  26.520  1.00 65.79           C  \nATOM    774  CD  GLU B  35      44.409 -40.609  27.182  1.00 69.94           C  \nATOM    775  OE1 GLU B  35      44.687 -41.820  27.022  1.00 68.64           O  \nATOM    776  OE2 GLU B  35      45.104 -39.823  27.876  1.00 72.52           O  \nATOM    777  N   GLN B  36      42.583 -41.555  22.380  1.00 56.19           N  \nATOM    778  CA  GLN B  36      42.272 -42.599  21.414  1.00 54.83           C  \nATOM    779  C   GLN B  36      43.321 -42.751  20.331  1.00 53.25           C  \nATOM    780  O   GLN B  36      43.600 -43.857  19.885  1.00 52.42           O  \nATOM    781  CB  GLN B  36      40.905 -42.349  20.777  1.00 55.13           C  \nATOM    782  CG  GLN B  36      39.747 -42.383  21.767  1.00 55.95           C  \nATOM    783  CD  GLN B  36      38.398 -42.064  21.109  1.00 58.62           C  \nATOM    784  OE1 GLN B  36      38.331 -41.380  20.077  1.00 56.92           O  \nATOM    785  NE2 GLN B  36      37.321 -42.544  21.718  1.00 57.52           N  \nATOM    786  N   LEU B  37      43.899 -41.643  19.898  1.00 51.73           N  \nATOM    787  CA  LEU B  37      44.913 -41.726  18.870  1.00 52.55           C  \nATOM    788  C   LEU B  37      46.192 -42.337  19.461  1.00 54.46           C  \nATOM    789  O   LEU B  37      46.791 -43.232  18.849  1.00 53.20           O  \nATOM    790  CB  LEU B  37      45.195 -40.339  18.256  1.00 50.02           C  \nATOM    791  CG  LEU B  37      46.215 -40.370  17.104  1.00 51.08           C  \nATOM    792  CD1 LEU B  37      45.783 -41.417  16.095  1.00 49.26           C  \nATOM    793  CD2 LEU B  37      46.350 -38.999  16.438  1.00 49.37           C  \nATOM    794  N   LEU B  38      46.599 -41.870  20.647  1.00 53.85           N  \nATOM    795  CA  LEU B  38      47.801 -42.388  21.297  1.00 54.34           C  \nATOM    796  C   LEU B  38      47.651 -43.878  21.542  1.00 54.03           C  \nATOM    797  O   LEU B  38      48.549 -44.650  21.248  1.00 54.02           O  \nATOM    798  CB  LEU B  38      48.057 -41.698  22.638  1.00 55.11           C  \nATOM    799  CG  LEU B  38      49.452 -41.092  22.863  1.00 58.71           C  \nATOM    800  CD1 LEU B  38      49.744 -41.154  24.358  1.00 59.75           C  \nATOM    801  CD2 LEU B  38      50.548 -41.836  22.097  1.00 58.46           C  \nATOM    802  N   GLN B  39      46.508 -44.274  22.088  1.00 54.63           N  \nATOM    803  CA  GLN B  39      46.247 -45.679  22.369  1.00 56.09           C  \nATOM    804  C   GLN B  39      46.319 -46.524  21.110  1.00 56.52           C  \nATOM    805  O   GLN B  39      46.907 -47.601  21.103  1.00 57.15           O  \nATOM    806  CB  GLN B  39      44.875 -45.858  23.004  1.00 55.05           C  \nATOM    807  CG  GLN B  39      44.727 -45.133  24.300  1.00 59.26           C  \nATOM    808  CD  GLN B  39      43.564 -45.637  25.128  1.00 63.26           C  \nATOM    809  OE1 GLN B  39      43.289 -45.109  26.210  1.00 64.37           O  \nATOM    810  NE2 GLN B  39      42.882 -46.671  24.638  1.00 64.00           N  \nATOM    811  N   LEU B  40      45.711 -46.037  20.042  1.00 57.03           N  \nATOM    812  CA  LEU B  40      45.720 -46.768  18.789  1.00 57.08           C  \nATOM    813  C   LEU B  40      47.169 -46.938  18.322  1.00 57.00           C  \nATOM    814  O   LEU B  40      47.562 -48.013  17.895  1.00 58.58           O  \nATOM    815  CB  LEU B  40      44.885 -46.026  17.746  1.00 55.60           C  \nATOM    816  CG  LEU B  40      44.707 -46.766  16.416  1.00 57.00           C  \nATOM    817  CD1 LEU B  40      43.423 -46.302  15.762  1.00 57.76           C  \nATOM    818  CD2 LEU B  40      45.903 -46.533  15.495  1.00 57.07           C  \nATOM    819  N   ILE B  41      47.960 -45.875  18.407  1.00 57.00           N  \nATOM    820  CA  ILE B  41      49.367 -45.932  18.017  1.00 57.62           C  \nATOM    821  C   ILE B  41      50.114 -46.934  18.912  1.00 59.23           C  \nATOM    822  O   ILE B  41      50.957 -47.709  18.434  1.00 57.97           O  \nATOM    823  CB  ILE B  41      50.032 -44.547  18.143  1.00 56.94           C  \nATOM    824  CG1 ILE B  41      49.618 -43.692  16.953  1.00 56.82           C  \nATOM    825  CG2 ILE B  41      51.559 -44.676  18.207  1.00 55.56           C  \nATOM    826  CD1 ILE B  41      49.867 -42.228  17.141  1.00 56.84           C  \nATOM    827  N   PHE B  42      49.801 -46.914  20.205  1.00 59.00           N  \nATOM    828  CA  PHE B  42      50.429 -47.823  21.149  1.00 60.08           C  \nATOM    829  C   PHE B  42      50.112 -49.273  20.793  1.00 61.49           C  \nATOM    830  O   PHE B  42      51.009 -50.096  20.605  1.00 60.83           O  \nATOM    831  CB  PHE B  42      49.921 -47.548  22.556  1.00 60.01           C  \nATOM    832  CG  PHE B  42      50.587 -48.383  23.601  1.00 61.05           C  \nATOM    833  CD1 PHE B  42      51.846 -48.033  24.076  1.00 61.05           C  \nATOM    834  CD2 PHE B  42      49.977 -49.534  24.088  1.00 59.79           C  \nATOM    835  CE1 PHE B  42      52.492 -48.813  25.017  1.00 62.04           C  \nATOM    836  CE2 PHE B  42      50.625 -50.331  25.042  1.00 60.84           C  \nATOM    837  CZ  PHE B  42      51.884 -49.963  25.502  1.00 61.47           C  \nATOM    838  N   LYS B  43      48.822 -49.583  20.723  1.00 62.91           N  \nATOM    839  CA  LYS B  43      48.384 -50.932  20.397  1.00 63.72           C  \nATOM    840  C   LYS B  43      48.977 -51.468  19.100  1.00 63.46           C  \nATOM    841  O   LYS B  43      49.152 -52.672  18.953  1.00 62.93           O  \nATOM    842  CB  LYS B  43      46.865 -50.995  20.294  1.00 65.12           C  \nATOM    843  CG  LYS B  43      46.142 -50.945  21.611  1.00 69.50           C  \nATOM    844  CD  LYS B  43      44.661 -51.174  21.371  1.00 72.15           C  \nATOM    845  CE  LYS B  43      43.825 -50.668  22.528  1.00 74.11           C  \nATOM    846  NZ  LYS B  43      42.417 -50.435  22.088  1.00 74.80           N  \nATOM    847  N   SER B  44      49.292 -50.593  18.155  1.00 63.71           N  \nATOM    848  CA  SER B  44      49.831 -51.074  16.894  1.00 65.96           C  \nATOM    849  C   SER B  44      51.308 -51.428  17.008  1.00 67.57           C  \nATOM    850  O   SER B  44      51.799 -52.309  16.305  1.00 70.05           O  \nATOM    851  CB  SER B  44      49.602 -50.044  15.787  1.00 65.66           C  \nATOM    852  OG  SER B  44      50.791 -49.360  15.462  1.00 71.25           O  \nATOM    853  N   ILE B  45      52.021 -50.747  17.896  1.00 67.66           N  \nATOM    854  CA  ILE B  45      53.434 -51.044  18.081  1.00 67.20           C  \nATOM    855  C   ILE B  45      53.536 -52.328  18.908  1.00 67.45           C  \nATOM    856  O   ILE B  45      54.393 -53.188  18.651  1.00 67.32           O  \nATOM    857  CB  ILE B  45      54.167 -49.916  18.837  1.00 66.49           C  \nATOM    858  CG1 ILE B  45      54.004 -48.585  18.101  1.00 65.28           C  \nATOM    859  CG2 ILE B  45      55.645 -50.261  18.952  1.00 66.79           C  \nATOM    860  CD1 ILE B  45      54.562 -47.395  18.852  1.00 62.95           C  \nATOM    861  N   ASP B  46      52.630 -52.438  19.882  1.00 67.39           N  \nATOM    862  CA  ASP B  46      52.529 -53.571  20.802  1.00 67.77           C  \nATOM    863  C   ASP B  46      51.963 -54.808  20.110  1.00 69.95           C  \nATOM    864  O   ASP B  46      50.881 -55.289  20.463  1.00 70.19           O  \nATOM    865  CB  ASP B  46      51.640 -53.175  21.998  1.00 67.43           C  \nATOM    866  CG  ASP B  46      51.406 -54.318  22.968  1.00 68.79           C  \nATOM    867  OD1 ASP B  46      52.064 -55.347  22.789  1.00 69.26           O  \nATOM    868  OD2 ASP B  46      50.576 -54.194  23.901  1.00 68.63           O  \nATOM    869  N   ALA B  47      52.711 -55.334  19.143  1.00 71.51           N  \nATOM    870  CA  ALA B  47      52.283 -56.503  18.385  1.00 73.22           C  \nATOM    871  C   ALA B  47      51.718 -57.686  19.178  1.00 74.80           C  \nATOM    872  O   ALA B  47      50.654 -58.196  18.827  1.00 75.62           O  \nATOM    873  CB  ALA B  47      53.410 -56.968  17.495  1.00 70.79           C  \nATOM    874  N   ASP B  48      52.386 -58.125  20.244  1.00 76.38           N  \nATOM    875  CA  ASP B  48      51.860 -59.274  20.984  1.00 77.53           C  \nATOM    876  C   ASP B  48      50.719 -58.925  21.934  1.00 77.20           C  \nATOM    877  O   ASP B  48      50.255 -59.775  22.705  1.00 79.34           O  \nATOM    878  CB  ASP B  48      52.974 -60.015  21.746  1.00 79.55           C  \nATOM    879  CG  ASP B  48      53.397 -59.320  23.033  1.00 82.18           C  \nATOM    880  OD1 ASP B  48      52.534 -58.721  23.719  1.00 81.91           O  \nATOM    881  OD2 ASP B  48      54.600 -59.403  23.376  1.00 82.63           O  \nATOM    882  N   GLY B  49      50.272 -57.675  21.878  1.00 74.95           N  \nATOM    883  CA  GLY B  49      49.176 -57.226  22.715  1.00 72.66           C  \nATOM    884  C   GLY B  49      49.218 -57.506  24.206  1.00 72.81           C  \nATOM    885  O   GLY B  49      48.168 -57.662  24.826  1.00 72.19           O  \nATOM    886  N   ASN B  50      50.403 -57.534  24.806  1.00 72.26           N  \nATOM    887  CA  ASN B  50      50.480 -57.802  26.236  1.00 72.09           C  \nATOM    888  C   ASN B  50      50.484 -56.522  27.085  1.00 71.52           C  \nATOM    889  O   ASN B  50      50.623 -56.600  28.306  1.00 71.51           O  \nATOM    890  CB  ASN B  50      51.715 -58.650  26.554  1.00 72.35           C  \nATOM    891  CG  ASN B  50      52.976 -57.840  26.547  1.00 73.15           C  \nATOM    892  OD1 ASN B  50      52.972 -56.690  26.109  1.00 73.49           O  \nATOM    893  ND2 ASN B  50      54.064 -58.420  27.033  1.00 73.37           N  \nATOM    894  N   GLY B  51      50.354 -55.350  26.452  1.00 70.35           N  \nATOM    895  CA  GLY B  51      50.305 -54.098  27.204  1.00 68.97           C  \nATOM    896  C   GLY B  51      51.596 -53.334  27.463  1.00 68.94           C  \nATOM    897  O   GLY B  51      51.588 -52.288  28.124  1.00 68.76           O  \nATOM    898  N   GLU B  52      52.704 -53.833  26.932  1.00 68.03           N  \nATOM    899  CA  GLU B  52      53.995 -53.191  27.117  1.00 69.35           C  \nATOM    900  C   GLU B  52      54.672 -53.318  25.765  1.00 68.43           C  \nATOM    901  O   GLU B  52      54.391 -54.256  25.049  1.00 68.10           O  \nATOM    902  CB  GLU B  52      54.811 -53.979  28.142  1.00 72.30           C  \nATOM    903  CG  GLU B  52      55.535 -55.162  27.491  1.00 76.21           C  \nATOM    904  CD  GLU B  52      56.199 -56.109  28.475  1.00 78.68           C  \nATOM    905  OE1 GLU B  52      56.922 -57.021  28.006  1.00 75.56           O  \nATOM    906  OE2 GLU B  52      55.998 -55.952  29.704  1.00 80.23           O  \nATOM    907  N   ILE B  53      55.547 -52.403  25.378  1.00 67.23           N  \nATOM    908  CA  ILE B  53      56.202 -52.623  24.104  1.00 66.14           C  \nATOM    909  C   ILE B  53      57.677 -52.917  24.435  1.00 66.42           C  \nATOM    910  O   ILE B  53      58.347 -52.138  25.117  1.00 66.40           O  \nATOM    911  CB  ILE B  53      55.996 -51.417  23.107  1.00 64.91           C  \nATOM    912  CG1 ILE B  53      57.257 -50.590  22.988  1.00 63.04           C  \nATOM    913  CG2 ILE B  53      54.775 -50.589  23.492  1.00 65.64           C  \nATOM    914  CD1 ILE B  53      58.137 -51.164  21.951  1.00 62.03           C  \nATOM    915  N   ASP B  54      58.156 -54.084  24.003  1.00 66.45           N  \nATOM    916  CA  ASP B  54      59.535 -54.499  24.270  1.00 67.20           C  \nATOM    917  C   ASP B  54      60.451 -53.952  23.195  1.00 67.26           C  \nATOM    918  O   ASP B  54      59.969 -53.449  22.179  1.00 68.06           O  \nATOM    919  CB  ASP B  54      59.637 -56.037  24.361  1.00 65.29           C  \nATOM    920  CG  ASP B  54      59.089 -56.747  23.135  1.00 65.48           C  \nATOM    921  OD1 ASP B  54      59.620 -56.522  22.023  1.00 63.45           O  \nATOM    922  OD2 ASP B  54      58.129 -57.536  23.291  1.00 64.55           O  \nATOM    923  N   GLN B  55      61.760 -54.029  23.422  1.00 67.85           N  \nATOM    924  CA  GLN B  55      62.727 -53.501  22.464  1.00 68.49           C  \nATOM    925  C   GLN B  55      62.537 -54.066  21.070  1.00 68.61           C  \nATOM    926  O   GLN B  55      62.926 -53.436  20.081  1.00 68.25           O  \nATOM    927  CB  GLN B  55      64.159 -53.796  22.908  1.00 68.45           C  \nATOM    928  CG  GLN B  55      64.412 -53.559  24.367  1.00 68.94           C  \nATOM    929  CD  GLN B  55      65.874 -53.405  24.676  1.00 69.44           C  \nATOM    930  OE1 GLN B  55      66.739 -53.786  23.888  1.00 71.16           O  \nATOM    931  NE2 GLN B  55      66.165 -52.852  25.832  1.00 69.06           N  \nATOM    932  N   ASN B  56      61.963 -55.261  20.985  1.00 69.67           N  \nATOM    933  CA  ASN B  56      61.750 -55.882  19.684  1.00 71.82           C  \nATOM    934  C   ASN B  56      60.597 -55.201  18.956  1.00 70.10           C  \nATOM    935  O   ASN B  56      60.739 -54.777  17.806  1.00 69.82           O  \nATOM    936  CB  ASN B  56      61.477 -57.377  19.837  1.00 76.43           C  \nATOM    937  CG  ASN B  56      61.832 -58.150  18.585  1.00 80.47           C  \nATOM    938  OD1 ASN B  56      60.951 -58.610  17.854  1.00 81.33           O  \nATOM    939  ND2 ASN B  56      63.136 -58.278  18.316  1.00 80.76           N  \nATOM    940  N   GLU B  57      59.459 -55.090  19.628  1.00 68.33           N  \nATOM    941  CA  GLU B  57      58.316 -54.412  19.036  1.00 65.56           C  \nATOM    942  C   GLU B  57      58.729 -53.006  18.662  1.00 65.96           C  \nATOM    943  O   GLU B  57      58.368 -52.518  17.601  1.00 66.55           O  \nATOM    944  CB  GLU B  57      57.176 -54.364  20.028  1.00 62.01           C  \nATOM    945  CG  GLU B  57      56.679 -55.754  20.286  1.00 58.21           C  \nATOM    946  CD  GLU B  57      55.801 -55.857  21.506  1.00 60.93           C  \nATOM    947  OE1 GLU B  57      55.920 -55.015  22.413  1.00 62.81           O  \nATOM    948  OE2 GLU B  57      54.994 -56.797  21.588  1.00 59.27           O  \nATOM    949  N   PHE B  58      59.512 -52.374  19.531  1.00 65.66           N  \nATOM    950  CA  PHE B  58      59.980 -51.009  19.308  1.00 67.39           C  \nATOM    951  C   PHE B  58      60.891 -50.889  18.107  1.00 68.93           C  \nATOM    952  O   PHE B  58      60.823 -49.910  17.370  1.00 69.50           O  \nATOM    953  CB  PHE B  58      60.741 -50.498  20.541  1.00 66.40           C  \nATOM    954  CG  PHE B  58      61.227 -49.067  20.426  1.00 65.68           C  \nATOM    955  CD1 PHE B  58      60.480 -48.013  20.949  1.00 65.30           C  \nATOM    956  CD2 PHE B  58      62.457 -48.772  19.835  1.00 63.56           C  \nATOM    957  CE1 PHE B  58      60.955 -46.701  20.887  1.00 63.64           C  \nATOM    958  CE2 PHE B  58      62.931 -47.444  19.774  1.00 61.96           C  \nATOM    959  CZ  PHE B  58      62.182 -46.425  20.300  1.00 61.74           C  \nATOM    960  N   ALA B  59      61.767 -51.870  17.929  1.00 72.18           N  \nATOM    961  CA  ALA B  59      62.727 -51.854  16.834  1.00 73.76           C  \nATOM    962  C   ALA B  59      62.063 -51.994  15.470  1.00 74.89           C  \nATOM    963  O   ALA B  59      62.431 -51.301  14.518  1.00 73.44           O  \nATOM    964  CB  ALA B  59      63.742 -52.954  17.037  1.00 73.68           C  \nATOM    965  N   LYS B  60      61.097 -52.902  15.366  1.00 76.67           N  \nATOM    966  CA  LYS B  60      60.393 -53.086  14.103  1.00 79.47           C  \nATOM    967  C   LYS B  60      59.835 -51.729  13.694  1.00 81.18           C  \nATOM    968  O   LYS B  60      60.111 -51.221  12.602  1.00 81.09           O  \nATOM    969  CB  LYS B  60      59.247 -54.081  14.272  1.00 81.66           C  \nATOM    970  CG  LYS B  60      59.703 -55.529  14.374  1.00 85.25           C  \nATOM    971  CD  LYS B  60      58.569 -56.428  14.851  1.00 87.92           C  \nATOM    972  CE  LYS B  60      58.677 -57.826  14.256  1.00 89.39           C  \nATOM    973  NZ  LYS B  60      60.045 -58.403  14.390  1.00 89.82           N  \nATOM    974  N   PHE B  61      59.065 -51.141  14.604  1.00 82.47           N  \nATOM    975  CA  PHE B  61      58.443 -49.844  14.392  1.00 83.42           C  \nATOM    976  C   PHE B  61      59.376 -48.802  13.791  1.00 84.97           C  \nATOM    977  O   PHE B  61      59.062 -48.217  12.762  1.00 85.61           O  \nATOM    978  CB  PHE B  61      57.872 -49.339  15.717  1.00 82.64           C  \nATOM    979  CG  PHE B  61      57.173 -48.016  15.618  1.00 82.58           C  \nATOM    980  CD1 PHE B  61      55.924 -47.903  15.002  1.00 82.41           C  \nATOM    981  CD2 PHE B  61      57.753 -46.877  16.170  1.00 81.76           C  \nATOM    982  CE1 PHE B  61      55.274 -46.670  14.937  1.00 81.08           C  \nATOM    983  CE2 PHE B  61      57.106 -45.644  16.104  1.00 81.98           C  \nATOM    984  CZ  PHE B  61      55.865 -45.546  15.491  1.00 81.41           C  \nATOM    985  N   TYR B  62      60.525 -48.562  14.410  1.00 87.26           N  \nATOM    986  CA  TYR B  62      61.416 -47.554  13.861  1.00 90.70           C  \nATOM    987  C   TYR B  62      62.146 -47.951  12.580  1.00 93.31           C  \nATOM    988  O   TYR B  62      62.203 -49.125  12.193  1.00 93.24           O  \nATOM    989  CB  TYR B  62      62.411 -47.070  14.922  1.00 91.91           C  \nATOM    990  CG  TYR B  62      61.838 -45.973  15.796  1.00 94.28           C  \nATOM    991  CD1 TYR B  62      61.161 -46.276  16.975  1.00 94.86           C  \nATOM    992  CD2 TYR B  62      61.904 -44.634  15.405  1.00 95.39           C  \nATOM    993  CE1 TYR B  62      60.564 -45.274  17.747  1.00 95.25           C  \nATOM    994  CE2 TYR B  62      61.304 -43.621  16.173  1.00 95.65           C  \nATOM    995  CZ  TYR B  62      60.636 -43.951  17.339  1.00 95.27           C  \nATOM    996  OH  TYR B  62      60.047 -42.955  18.083  1.00 95.09           O  \nATOM    997  N   GLY B  63      62.677 -46.940  11.910  1.00 95.58           N  \nATOM    998  CA  GLY B  63      63.396 -47.140  10.666  1.00 98.12           C  \nATOM    999  C   GLY B  63      63.167 -45.786  10.055  1.00 99.57           C  \nATOM   1000  O   GLY B  63      63.401 -44.775  10.728  1.00100.42           O  \nATOM   1001  N   SER B  64      62.748 -45.734   8.794  1.00100.22           N  \nATOM   1002  CA  SER B  64      62.397 -44.442   8.222  1.00100.36           C  \nATOM   1003  C   SER B  64      61.071 -44.334   8.974  1.00 99.60           C  \nATOM   1004  O   SER B  64      60.523 -43.249   9.183  1.00 97.73           O  \nATOM   1005  CB  SER B  64      62.116 -44.524   6.712  1.00101.67           C  \nATOM   1006  OG  SER B  64      63.291 -44.714   5.943  1.00102.08           O  \nATOM   1007  N   ILE B  65      60.618 -45.522   9.393  1.00100.22           N  \nATOM   1008  CA  ILE B  65      59.391 -45.810  10.127  1.00 99.78           C  \nATOM   1009  C   ILE B  65      58.394 -46.341   9.116  1.00100.54           C  \nATOM   1010  O   ILE B  65      58.840 -47.035   8.179  1.00100.36           O  \nATOM   1011  CB  ILE B  65      58.836 -44.589  10.919  1.00 99.33           C  \nATOM   1012  CG1 ILE B  65      59.758 -44.312  12.111  1.00 98.72           C  \nATOM   1013  CG2 ILE B  65      57.466 -44.909  11.520  1.00 97.47           C  \nATOM   1014  CD1 ILE B  65      59.162 -43.387  13.162  1.00100.37           C  \nTER    1015      ILE B  65                                                      \nHETATM 1016 CA    CA A 149      36.051 -49.233  -2.416  1.00 65.06          CA  \nHETATM 1017 CA    CA A 150      25.879  -7.406   5.735  1.00 64.64          CA  \nHETATM 1018  C   ACT A 201      38.990 -11.905   9.521  1.00 87.25           C  \nHETATM 1019  O   ACT A 201      38.407 -11.169   8.564  1.00 86.82           O  \nHETATM 1020  OXT ACT A 201      38.588 -13.031   9.758  1.00 86.12           O  \nHETATM 1021  CH3 ACT A 201      40.150 -11.335  10.300  1.00 86.21           C  \nHETATM 1022  C   ACT A 203      32.096 -26.174  13.836  1.00 87.40           C  \nHETATM 1023  O   ACT A 203      31.345 -27.267  13.951  1.00 86.37           O  \nHETATM 1024  OXT ACT A 203      33.289 -26.285  13.699  1.00 87.07           O  \nHETATM 1025  CH3 ACT A 203      31.461 -24.807  13.877  1.00 87.30           C  \nHETATM 1026 CA    CA B 151      23.108 -26.574  31.768  1.00 62.37          CA  \nHETATM 1027 CA    CA B 152      54.071 -56.397  23.802  1.00 66.61          CA  \nHETATM 1028  C   ACT B 202      56.579 -42.671  19.825  1.00 89.57           C  \nHETATM 1029  O   ACT B 202      55.261 -42.687  19.630  1.00 88.87           O  \nHETATM 1030  OXT ACT B 202      57.070 -43.323  20.743  1.00 89.04           O  \nHETATM 1031  CH3 ACT B 202      57.439 -41.834  18.901  1.00 88.76           C  \nHETATM 1032  C   ACT B 204      40.979 -41.637  15.598  1.00 85.82           C  \nHETATM 1033  O   ACT B 204      39.678 -41.786  15.437  1.00 85.24           O  \nHETATM 1034  OXT ACT B 204      41.400 -40.573  15.980  1.00 86.18           O  \nHETATM 1035  CH3 ACT B 204      41.911 -42.791  15.307  1.00 85.54           C  \nHETATM 1036  C   ACT B 205      37.353 -24.645  18.793  1.00 98.62           C  \nHETATM 1037  O   ACT B 205      37.260 -25.492  19.782  1.00 98.54           O  \nHETATM 1038  OXT ACT B 205      37.899 -23.578  19.008  1.00 97.84           O  \nHETATM 1039  CH3 ACT B 205      36.777 -25.030  17.429  1.00 97.56           C  \nHETATM 1040  O   HOH A 204      29.876 -49.594   0.516  1.00 45.91           O  \nHETATM 1041  O   HOH A 205      45.834 -36.457   3.111  1.00 85.65           O  \nHETATM 1042  O   HOH A 206      26.586 -44.315   8.875  1.00 93.71           O  \nHETATM 1043  O   HOH A 207      43.838 -48.225  -5.291  1.00 91.41           O  \nHETATM 1044  O   HOH A 208      38.026 -26.033   2.594  1.00 56.76           O  \nHETATM 1045  O   HOH A 209      32.655 -54.899   3.161  1.00 72.45           O  \nHETATM 1046  O   HOH A 210      36.422 -33.187  12.398  1.00 83.35           O  \nHETATM 1047  O   HOH A 211      38.814 -33.276  10.483  1.00 63.96           O  \nHETATM 1048  O   HOH A 212      24.177 -12.740  10.983  1.00 68.50           O  \nHETATM 1049  O   HOH A 213      23.753 -15.977  10.852  1.00 81.71           O  \nHETATM 1050  O   HOH A 214      22.126 -49.275   7.543  1.00 82.73           O  \nHETATM 1051  O   HOH A 215      38.625 -49.736  -3.385  1.00 68.29           O  \nHETATM 1052  O   HOH A 216      25.551  -4.655   4.580  1.00 79.45           O  \nHETATM 1053  O   HOH B 206      16.186 -26.765  26.266  1.00 79.89           O  \nHETATM 1054  O   HOH B 207      27.045 -27.002  34.575  1.00 58.46           O  \nHETATM 1055  O   HOH B 208      37.105 -18.861  34.209  1.00 85.93           O  \nHETATM 1056  O   HOH B 209      47.291 -52.834  15.236  1.00 57.96           O  \nHETATM 1057  O   HOH B 210      23.710 -38.163  19.820  1.00 85.15           O  \nHETATM 1058  O   HOH B 211      58.159 -49.447   9.934  1.00 81.22           O  \nHETATM 1059  O   HOH B 212      48.243 -54.772  20.435  1.00 71.27           O  \nHETATM 1060  O   HOH B 213      46.368 -41.846  29.869  1.00 81.64           O  \nHETATM 1061  O   HOH B 214      36.147 -30.929  30.474  1.00 61.55           O  \nHETATM 1062  O   HOH B 215      26.196 -34.947  30.952  1.00 65.46           O  \nHETATM 1063  O   HOH B 216      21.431 -36.159  20.185  1.00 88.02           O  \nHETATM 1064  O   HOH B 217      39.613 -22.785  24.139  1.00 92.63           O  \nHETATM 1065  O   HOH B 218      41.805 -29.392  19.283  1.00 95.35           O  \nHETATM 1066  O   HOH B 219      47.791 -28.358  18.363  0.33 92.30           O  \nHETATM 1067  O   HOH B 220      46.371 -54.201  17.801  1.00 71.13           O  \nHETATM 1068  O   HOH B 221      49.314 -55.492  17.415  1.00 73.72           O  \nHETATM 1069  O   HOH B 222      23.815 -24.118  32.751  1.00 65.43           O  \nHETATM 1070  O   HOH B 223      56.235 -57.917  25.216  1.00 71.15           O  \nCONECT   71 1016                                                                \nCONECT   86 1016                                                                \nCONECT   98 1016                                                                \nCONECT  107 1016                                                                \nCONECT  150 1016                                                                \nCONECT  151 1016                                                                \nCONECT  355 1017                                                                \nCONECT  368 1017                                                                \nCONECT  380 1017                                                                \nCONECT  389 1017                                                                \nCONECT  435 1017                                                                \nCONECT  436 1017                                                                \nCONECT  583 1026                                                                \nCONECT  598 1026                                                                \nCONECT  610 1026                                                                \nCONECT  619 1026                                                                \nCONECT  662 1026                                                                \nCONECT  663 1026                                                                \nCONECT  867 1027                                                                \nCONECT  880 1027                                                                \nCONECT  892 1027                                                                \nCONECT  901 1027                                                                \nCONECT  947 1027                                                                \nCONECT  948 1027                                                                \nCONECT 1016   71   86   98  107                                                 \nCONECT 1016  150  151 1051                                                      \nCONECT 1017  355  368  380  389                                                 \nCONECT 1017  435  436                                                           \nCONECT 1018 1019 1020 1021                                                      \nCONECT 1019 1018                                                                \nCONECT 1020 1018                                                                \nCONECT 1021 1018                                                                \nCONECT 1022 1023 1024 1025                                                      \nCONECT 1023 1022                                                                \nCONECT 1024 1022                                                                \nCONECT 1025 1022                                                                \nCONECT 1026  583  598  610  619                                                 \nCONECT 1026  662  663 1069                                                      \nCONECT 1027  867  880  892  901                                                 \nCONECT 1027  947  948 1070                                                      \nCONECT 1028 1029 1030 1031                                                      \nCONECT 1029 1028                                                                \nCONECT 1030 1028                                                                \nCONECT 1031 1028                                                                \nCONECT 1032 1033 1034 1035                                                      \nCONECT 1033 1032                                                                \nCONECT 1034 1032                                                                \nCONECT 1035 1032                                                                \nCONECT 1036 1037 1038 1039                                                      \nCONECT 1037 1036                                                                \nCONECT 1038 1036                                                                \nCONECT 1039 1036                                                                \nCONECT 1051 1016                                                                \nCONECT 1069 1026                                                                \nCONECT 1070 1027                                                                \nMASTER      564    0    9    6    0    0   12    6 1068    2   55   22          \nEND                                                                             \n","pdb");
	viewer_1758178701358389.setStyle({"model": 0},{"stick": {}});
	viewer_1758178701358389.setStyle({"model": 0},{"cartoon": {"color": "blue"}});
	viewer_1758178701358389.addLabel("2nxqB",{"fontOpacity": 0.3},{"resi": 158});
	viewer_1758178701358389.zoomTo();
	viewer_1758178701358389.zoom(1.2);
	viewer_1758178701358389.setBackgroundColor("white");
viewer_1758178701358389.render();
});
</script>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=258fa5c0">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[7]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">try</span><span class="p">:</span>
    <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span><span class="n">pdb_path1</span><span class="p">)</span>
<span class="k">except</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s1">'Structural Alignment Plot fail !'</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 2096 atoms and 1 coordinate set(s) were parsed in 0.02s.
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr" tabindex="0">
<pre>@&gt; 2096 atoms and 1 coordinate set(s) were parsed in 0.02s.
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>1jfk_TEMP
Blue:1jfk
Red:1jfk
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div id="3dmolviewer_17581787016276908" style="position: relative; width: 800px; height: 600px;">
<p id="3dmolwarning_17581787016276908" style="background-color:#ffcccc;color:black">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br/></p>
</div>
<script>

var loadScriptAsync = function(uri){
  return new Promise((resolve, reject) => {
    //this is to ignore the existence of requirejs amd
    var savedexports, savedmodule;
    if (typeof exports !== 'undefined') savedexports = exports;
    else exports = {}
    if (typeof module !== 'undefined') savedmodule = module;
    else module = {}

    var tag = document.createElement('script');
    tag.src = uri;
    tag.async = true;
    tag.onload = () => {
        exports = savedexports;
        module = savedmodule;
        resolve();
    };
  var firstScriptTag = document.getElementsByTagName('script')[0];
  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
});
};

if(typeof $3Dmolpromise === 'undefined') {
$3Dmolpromise = null;
  $3Dmolpromise = loadScriptAsync('https://cdn.jsdelivr.net/npm/3dmol@2.5.2/build/3Dmol-min.js');
}

var viewer_17581787016276908 = null;
var warn = document.getElementById("3dmolwarning_17581787016276908");
if(warn) {
    warn.parentNode.removeChild(warn);
}
$3Dmolpromise.then(function() {
viewer_17581787016276908 = $3Dmol.createViewer(document.getElementById("3dmolviewer_17581787016276908"),{backgroundColor:"white"});
viewer_17581787016276908.zoomTo();
	viewer_17581787016276908.addModel("HEADER    METAL BINDING PROTEIN                   21-JUN-01   1JFK              \nTITLE     MINIMUM ENERGY REPRESENTATIVE STRUCTURE OF A CALCIUM BOUND EF-HAND    \nTITLE    2 PROTEIN FROM ENTAMOEBA HISTOLYTICA                                   \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: CALCIUM-BINDING PROTEIN;                                   \nCOMPND   3 CHAIN: A;                                                            \nCOMPND   4 SYNONYM: EHCABP;                                                     \nCOMPND   5 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 ORGANISM_SCIENTIFIC: ENTAMOEBA HISTOLYTICA;                          \nSOURCE   3 ORGANISM_TAXID: 5759;                                                \nSOURCE   4 EXPRESSION_SYSTEM: ESCHERICHIA COLI BL21(DE3);                       \nSOURCE   5 EXPRESSION_SYSTEM_TAXID: 469008;                                     \nSOURCE   6 EXPRESSION_SYSTEM_STRAIN: BL21(DE3);                                 \nSOURCE   7 EXPRESSION_SYSTEM_VECTOR_TYPE: PLASMID;                              \nSOURCE   8 EXPRESSION_SYSTEM_PLASMID: PET-3C                                    \nKEYWDS    EF-HAND, HELIX-LOOP-HELIX, CALCIUM BINDING PROTEIN, ENTAMOEBA         \nKEYWDS   2 HISTOLYTICA, METAL BINDING PROTEIN                                   \nEXPDTA    SOLUTION NMR                                                          \nAUTHOR    H.S.ATREYA,S.C.SAHU,A.BHATTACHARYA,K.V.R.CHARY,G.GOVIL                \nREVDAT   6   29-MAY-24 1JFK    1       REMARK                                   \nREVDAT   5   23-FEB-22 1JFK    1       REMARK SEQADV LINK                       \nREVDAT   4   24-FEB-09 1JFK    1       VERSN                                    \nREVDAT   3   01-APR-03 1JFK    1       JRNL                                     \nREVDAT   2   25-DEC-02 1JFK    1       REMARK                                   \nREVDAT   1   19-DEC-01 1JFK    0                                                \nJRNL        AUTH   H.S.ATREYA,S.C.SAHU,A.BHATTACHARYA,K.V.CHARY,G.GOVIL         \nJRNL        TITL   NMR DERIVED SOLUTION STRUCTURE OF AN EF-HAND CALCIUM-BINDING \nJRNL        TITL 2 PROTEIN FROM ENTAMOEBA HISTOLYTICA.                          \nJRNL        REF    BIOCHEMISTRY                  V.  40 14392 2001              \nJRNL        REFN                   ISSN 0006-2960                               \nJRNL        PMID   11724551                                                     \nJRNL        DOI    10.1021/BI0114978                                            \nREMARK   2                                                                      \nREMARK   2 RESOLUTION. NOT APPLICABLE.                                          \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : DYANA 1.5, DYANA 1.5                                 \nREMARK   3   AUTHORS     : GUENTERT, MUMENTHALER AND WUTHRICH (DYANA),          \nREMARK   3                 GUENTERT, MUMENTHALER AND WUTHRICH (DYANA)           \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: 1265 NOE-DERIVED DISTANCE CONSTRAINTS     \nREMARK   3  AND 200 DIHEDRAL ANGLE CONSTRAINTS                                  \nREMARK   4                                                                      \nREMARK   4 1JFK COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY PDBJ ON 26-JUN-01.                  \nREMARK 100 THE DEPOSITION ID IS D_1000013711.                                   \nREMARK 210                                                                      \nREMARK 210 EXPERIMENTAL DETAILS                                                 \nREMARK 210  EXPERIMENT TYPE                : NMR                                \nREMARK 210  TEMPERATURE           (KELVIN) : 308                                \nREMARK 210  PH                             : 6.0                                \nREMARK 210  IONIC STRENGTH                 : 30MM CACL2                         \nREMARK 210  PRESSURE                       : 1 ATM                              \nREMARK 210  SAMPLE CONTENTS                : 3MM EHCABP, 30MM DEUTERATED TRIS   \nREMARK 210                                   BUFFER, 30MM CACL2; 3MM EHCABP,    \nREMARK 210                                   30MM DEUTERATED TRIS BUFFER,       \nREMARK 210                                   30MM CACL2                         \nREMARK 210                                                                      \nREMARK 210  NMR EXPERIMENTS CONDUCTED      : 3D_15N-SEPARATED_NOESY; HNHA; 2D   \nREMARK 210                                   NOESY; 3D_13C-SEPARATED_NOESY      \nREMARK 210  SPECTROMETER FIELD STRENGTH    : 600 MHZ                            \nREMARK 210  SPECTROMETER MODEL             : UNITYPLUS                          \nREMARK 210  SPECTROMETER MANUFACTURER      : VARIAN                             \nREMARK 210                                                                      \nREMARK 210  STRUCTURE DETERMINATION.                                            \nREMARK 210   SOFTWARE USED                 : NULL                               \nREMARK 210   METHOD USED                   : TORSION ANGLE DYNAMICS             \nREMARK 210                                                                      \nREMARK 210 CONFORMERS, NUMBER CALCULATED   : NULL                               \nREMARK 210 CONFORMERS, NUMBER SUBMITTED    : 1                                  \nREMARK 210 CONFORMERS, SELECTION CRITERIA  : NULL                               \nREMARK 210                                                                      \nREMARK 210 BEST REPRESENTATIVE CONFORMER IN THIS ENSEMBLE : 1                   \nREMARK 210                                                                      \nREMARK 210 REMARK: THE STRUCTURE WAS DETERMINED USING TRIPLE RESONANCE NMR      \nREMARK 210  SPECTROSCOPY                                                        \nREMARK 215                                                                      \nREMARK 215 NMR STUDY                                                            \nREMARK 215 THE COORDINATES IN THIS ENTRY WERE GENERATED FROM SOLUTION           \nREMARK 215 NMR DATA.  PROTEIN DATA BANK CONVENTIONS REQUIRE THAT                \nREMARK 215 CRYST1 AND SCALE RECORDS BE INCLUDED, BUT THE VALUES ON              \nREMARK 215 THESE RECORDS ARE MEANINGLESS.                                       \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1                                                       \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: MONOMERIC                         \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \nREMARK 500                                                                      \nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \nREMARK 500                                                                      \nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \nREMARK 500   O    PHE A    42     H    ASP A    46              1.49            \nREMARK 500   O    GLU A   107     H    ALA A   110              1.55            \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    LYS A  29      -79.65    -78.07                                   \nREMARK 500    ALA A  31       54.13   -141.02                                   \nREMARK 500    ILE A  32       52.95    -94.87                                   \nREMARK 500    SER A  64      -68.53   -178.13                                   \nREMARK 500    ILE A  65      -77.74   -144.11                                   \nREMARK 500    LEU A  70       93.33   -163.44                                   \nREMARK 500    ASP A  85       93.63    -53.76                                   \nREMARK 500    ALA A 118       49.22    -89.82                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 620                                                                      \nREMARK 620 METAL COORDINATION                                                   \nREMARK 620 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 620 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE):                             \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 147  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A  10   OD1                                                    \nREMARK 620 2 ASN A  12   OD1  72.5                                              \nREMARK 620 3 ASP A  14   OD2  64.8  80.7                                        \nREMARK 620 4 ALA A  16   O   102.8 174.7  95.1                                  \nREMARK 620 5 GLU A  21   OE1 123.5 108.1 169.0  76.4                            \nREMARK 620 6 GLU A  21   OE2  88.9  66.6 143.2 116.3  47.6                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 159  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A  46   OD1                                                    \nREMARK 620 2 ASP A  48   OD2  71.4                                              \nREMARK 620 3 ASP A  48   OD1  50.3  39.5                                        \nREMARK 620 4 ASN A  50   ND2 103.3  86.0  65.9                                  \nREMARK 620 5 ASN A  50   OD1  63.5  86.5  48.5  42.1                            \nREMARK 620 6 GLU A  52   O   104.1 134.9 102.2  50.5  54.6                      \nREMARK 620 7 GLU A  57   OE1 136.0 110.3 149.7 120.8 156.8 103.1                \nREMARK 620 8 GLU A  57   OE2  88.9  97.6 123.5 167.9 149.3 127.4  47.1          \nREMARK 620 N                    1     2     3     4     5     6     7           \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 171  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 LYS A  91   O                                                      \nREMARK 620 2 GLU A  96   OE2 121.9                                              \nREMARK 620 3 GLU A  96   OE1  93.2  47.7                                        \nREMARK 620 N                    1     2                                         \nREMARK 620                                                                      \nREMARK 620 COORDINATION ANGLES FOR:  M RES CSSEQI METAL                         \nREMARK 620                              CA A 183  CA                            \nREMARK 620 N RES CSSEQI ATOM                                                    \nREMARK 620 1 ASP A 117   OD2                                                    \nREMARK 620 2 ASN A 119   OD1  75.8                                              \nREMARK 620 3 ASP A 121   OD2  72.4  94.6                                        \nREMARK 620 4 TYR A 123   O    57.8 127.6  51.2                                  \nREMARK 620 5 GLU A 128   OE1 143.9 129.6 121.8 102.9                            \nREMARK 620 6 GLU A 128   OE2 132.7  83.5 152.5 145.9  47.7                      \nREMARK 620 N                    1     2     3     4     5                       \nREMARK 800                                                                      \nREMARK 800 SITE                                                                 \nREMARK 800 SITE_IDENTIFIER: AC1                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 147                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC2                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 159                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC3                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 171                  \nREMARK 800                                                                      \nREMARK 800 SITE_IDENTIFIER: AC4                                                 \nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CA A 183                  \nREMARK 900                                                                      \nREMARK 900 RELATED ENTRIES                                                      \nREMARK 900 RELATED ID: 1JFJ   RELATED DB: PDB                                   \nREMARK 900 1JFJ CONTAINS SAME HOLO PROTEIN (THE CALCIUM IONS ARE NOT SHOWN      \nREMARK 900 EXPLICILTY IN THE STRUCTURES).                                       \nREMARK 900 RELATED ID: 4271   RELATED DB: BMRB                                  \nREMARK 900 4271 CONTAINS ASSIGNED CHEMICAL SHIFT LISTS                          \nDBREF  1JFK A    1   134  UNP    P38505   CALBP_ENTHI      1    134             \nSEQADV 1JFK GLU A  131  UNP  P38505    UNK   131 CONFLICT                       \nSEQRES   1 A  134  MET ALA GLU ALA LEU PHE LYS GLU ILE ASP VAL ASN GLY          \nSEQRES   2 A  134  ASP GLY ALA VAL SER TYR GLU GLU VAL LYS ALA PHE VAL          \nSEQRES   3 A  134  SER LYS LYS ARG ALA ILE LYS ASN GLU GLN LEU LEU GLN          \nSEQRES   4 A  134  LEU ILE PHE LYS SER ILE ASP ALA ASP GLY ASN GLY GLU          \nSEQRES   5 A  134  ILE ASP GLN ASN GLU PHE ALA LYS PHE TYR GLY SER ILE          \nSEQRES   6 A  134  GLN GLY GLN ASP LEU SER ASP ASP LYS ILE GLY LEU LYS          \nSEQRES   7 A  134  VAL LEU TYR LYS LEU MET ASP VAL ASP GLY ASP GLY LYS          \nSEQRES   8 A  134  LEU THR LYS GLU GLU VAL THR SER PHE PHE LYS LYS HIS          \nSEQRES   9 A  134  GLY ILE GLU LYS VAL ALA GLU GLN VAL MET LYS ALA ASP          \nSEQRES  10 A  134  ALA ASN GLY ASP GLY TYR ILE THR LEU GLU GLU PHE LEU          \nSEQRES  11 A  134  GLU PHE SER LEU                                              \nHET     CA  A 147       1                                                       \nHET     CA  A 159       1                                                       \nHET     CA  A 171       1                                                       \nHET     CA  A 183       1                                                       \nHETNAM      CA CALCIUM ION                                                      \nFORMUL   2   CA    4(CA 2+)                                                     \nHELIX    1   1 MET A    1  ASP A   10  1                                  10    \nHELIX    2   2 TYR A   19  LYS A   28  1                                  10    \nHELIX    3   3 LYS A   33  ASP A   46  1                                  14    \nHELIX    4   4 GLN A   55  SER A   64  1                                  10    \nHELIX    5   5 LEU A   70  ASP A   85  1                                  16    \nHELIX    6   6 LYS A   94  LYS A  102  1                                   9    \nHELIX    7   7 GLU A  107  ASP A  117  1                                  11    \nHELIX    8   8 LEU A  126  LEU A  134  1                                   9    \nSHEET    1   A 2 ALA A  16  SER A  18  0                                        \nSHEET    2   A 2 GLU A  52  ASP A  54 -1  N  ILE A  53   O  VAL A  17           \nSHEET    1   B 2 LYS A  91  THR A  93  0                                        \nSHEET    2   B 2 TYR A 123  THR A 125 -1  N  ILE A 124   O  LEU A  92           \nLINK         OD1 ASP A  10                CA    CA A 147     1555   1555  2.78  \nLINK         OD1 ASN A  12                CA    CA A 147     1555   1555  2.80  \nLINK         OD2 ASP A  14                CA    CA A 147     1555   1555  3.04  \nLINK         O   ALA A  16                CA    CA A 147     1555   1555  2.79  \nLINK         OE1 GLU A  21                CA    CA A 147     1555   1555  2.74  \nLINK         OE2 GLU A  21                CA    CA A 147     1555   1555  2.78  \nLINK         OD1 ASP A  46                CA    CA A 159     1555   1555  2.69  \nLINK         OD2 ASP A  48                CA    CA A 159     1555   1555  3.40  \nLINK         OD1 ASP A  48                CA    CA A 159     1555   1555  3.16  \nLINK         ND2 ASN A  50                CA    CA A 159     1555   1555  3.32  \nLINK         OD1 ASN A  50                CA    CA A 159     1555   1555  2.81  \nLINK         O   GLU A  52                CA    CA A 159     1555   1555  2.81  \nLINK         OE1 GLU A  57                CA    CA A 159     1555   1555  2.77  \nLINK         OE2 GLU A  57                CA    CA A 159     1555   1555  2.81  \nLINK         O   LYS A  91                CA    CA A 171     1555   1555  2.79  \nLINK         OE2 GLU A  96                CA    CA A 171     1555   1555  2.80  \nLINK         OE1 GLU A  96                CA    CA A 171     1555   1555  2.70  \nLINK         OD2 ASP A 117                CA    CA A 183     1555   1555  3.40  \nLINK         OD1 ASN A 119                CA    CA A 183     1555   1555  2.67  \nLINK         OD2 ASP A 121                CA    CA A 183     1555   1555  3.15  \nLINK         O   TYR A 123                CA    CA A 183     1555   1555  2.69  \nLINK         OE1 GLU A 128                CA    CA A 183     1555   1555  2.72  \nLINK         OE2 GLU A 128                CA    CA A 183     1555   1555  2.79  \nSITE     1 AC1  5 ASP A  10  ASN A  12  ASP A  14  ALA A  16                    \nSITE     2 AC1  5 GLU A  21                                                     \nSITE     1 AC2  6 ASP A  46  ASP A  48  ASN A  50  GLU A  52                    \nSITE     2 AC2  6 ILE A  53  GLU A  57                                          \nSITE     1 AC3  4 ASP A  87  ASP A  89  LYS A  91  GLU A  96                    \nSITE     1 AC4  6 ASP A 117  ASN A 119  ASP A 121  TYR A 123                    \nSITE     2 AC4  6 ILE A 124  GLU A 128                                          \nCRYST1    1.000    1.000    1.000  90.00  90.00  90.00 P 1           1          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      1.000000  0.000000  0.000000        0.00000                         \nSCALE2      0.000000  1.000000  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  1.000000        0.00000                         \nATOM      1  N   MET A   1       1.325   0.000   0.000  1.00  0.00           N  \nATOM      2  CA  MET A   1       2.073   0.000  -1.245  1.00  0.00           C  \nATOM      3  C   MET A   1       3.328  -0.868  -1.131  1.00  0.00           C  \nATOM      4  O   MET A   1       4.435  -0.404  -1.399  1.00  0.00           O  \nATOM      5  CB  MET A   1       2.475   1.433  -1.600  1.00  0.00           C  \nATOM      6  CG  MET A   1       3.179   2.112  -0.424  1.00  0.00           C  \nATOM      7  SD  MET A   1       1.986   2.965   0.593  1.00  0.00           S  \nATOM      8  CE  MET A   1       3.081   3.982   1.569  1.00  0.00           C  \nATOM      9  H   MET A   1       1.884   0.000   0.829  1.00  0.00           H  \nATOM     10  HA  MET A   1       1.399  -0.420  -1.992  1.00  0.00           H  \nATOM     11  HB2 MET A   1       3.133   1.426  -2.469  1.00  0.00           H  \nATOM     12  HB3 MET A   1       1.589   2.005  -1.877  1.00  0.00           H  \nATOM     13  HG2 MET A   1       3.711   1.369   0.170  1.00  0.00           H  \nATOM     14  HG3 MET A   1       3.924   2.817  -0.794  1.00  0.00           H  \nATOM     15  HE1 MET A   1       3.658   3.351   2.244  1.00  0.00           H  \nATOM     16  HE2 MET A   1       3.759   4.523   0.908  1.00  0.00           H  \nATOM     17  HE3 MET A   1       2.494   4.695   2.149  1.00  0.00           H  \nATOM     18  N   ALA A   2       3.113  -2.114  -0.734  1.00  0.00           N  \nATOM     19  CA  ALA A   2       4.212  -3.051  -0.581  1.00  0.00           C  \nATOM     20  C   ALA A   2       4.758  -3.420  -1.962  1.00  0.00           C  \nATOM     21  O   ALA A   2       5.947  -3.251  -2.229  1.00  0.00           O  \nATOM     22  CB  ALA A   2       3.735  -4.275   0.203  1.00  0.00           C  \nATOM     23  H   ALA A   2       2.209  -2.484  -0.518  1.00  0.00           H  \nATOM     24  HA  ALA A   2       4.996  -2.553  -0.011  1.00  0.00           H  \nATOM     25  HB1 ALA A   2       2.650  -4.350   0.134  1.00  0.00           H  \nATOM     26  HB2 ALA A   2       4.189  -5.174  -0.214  1.00  0.00           H  \nATOM     27  HB3 ALA A   2       4.026  -4.173   1.249  1.00  0.00           H  \nATOM     28  N   GLU A   3       3.863  -3.917  -2.803  1.00  0.00           N  \nATOM     29  CA  GLU A   3       4.240  -4.311  -4.150  1.00  0.00           C  \nATOM     30  C   GLU A   3       4.710  -3.093  -4.948  1.00  0.00           C  \nATOM     31  O   GLU A   3       5.740  -3.145  -5.617  1.00  0.00           O  \nATOM     32  CB  GLU A   3       3.082  -5.018  -4.858  1.00  0.00           C  \nATOM     33  CG  GLU A   3       3.229  -6.538  -4.766  1.00  0.00           C  \nATOM     34  CD  GLU A   3       2.743  -7.055  -3.411  1.00  0.00           C  \nATOM     35  OE1 GLU A   3       3.425  -6.750  -2.409  1.00  0.00           O  \nATOM     36  OE2 GLU A   3       1.699  -7.743  -3.407  1.00  0.00           O  \nATOM     37  H   GLU A   3       2.898  -4.051  -2.578  1.00  0.00           H  \nATOM     38  HA  GLU A   3       5.065  -5.013  -4.026  1.00  0.00           H  \nATOM     39  HB2 GLU A   3       2.137  -4.713  -4.411  1.00  0.00           H  \nATOM     40  HB3 GLU A   3       3.052  -4.714  -5.905  1.00  0.00           H  \nATOM     41  HG2 GLU A   3       2.659  -7.011  -5.566  1.00  0.00           H  \nATOM     42  HG3 GLU A   3       4.273  -6.816  -4.912  1.00  0.00           H  \nATOM     43  N   ALA A   4       3.931  -2.026  -4.850  1.00  0.00           N  \nATOM     44  CA  ALA A   4       4.254  -0.797  -5.555  1.00  0.00           C  \nATOM     45  C   ALA A   4       5.717  -0.434  -5.293  1.00  0.00           C  \nATOM     46  O   ALA A   4       6.373   0.168  -6.142  1.00  0.00           O  \nATOM     47  CB  ALA A   4       3.292   0.310  -5.119  1.00  0.00           C  \nATOM     48  H   ALA A   4       3.094  -1.992  -4.304  1.00  0.00           H  \nATOM     49  HA  ALA A   4       4.119  -0.980  -6.621  1.00  0.00           H  \nATOM     50  HB1 ALA A   4       2.979   0.883  -5.991  1.00  0.00           H  \nATOM     51  HB2 ALA A   4       2.418  -0.135  -4.643  1.00  0.00           H  \nATOM     52  HB3 ALA A   4       3.794   0.970  -4.411  1.00  0.00           H  \nATOM     53  N   LEU A   5       6.187  -0.816  -4.114  1.00  0.00           N  \nATOM     54  CA  LEU A   5       7.560  -0.539  -3.730  1.00  0.00           C  \nATOM     55  C   LEU A   5       8.498  -1.478  -4.491  1.00  0.00           C  \nATOM     56  O   LEU A   5       9.494  -1.038  -5.064  1.00  0.00           O  \nATOM     57  CB  LEU A   5       7.717  -0.612  -2.210  1.00  0.00           C  \nATOM     58  CG  LEU A   5       7.831   0.729  -1.482  1.00  0.00           C  \nATOM     59  CD1 LEU A   5       6.478   1.441  -1.431  1.00  0.00           C  \nATOM     60  CD2 LEU A   5       8.435   0.547  -0.089  1.00  0.00           C  \nATOM     61  H   LEU A   5       5.647  -1.306  -3.430  1.00  0.00           H  \nATOM     62  HA  LEU A   5       7.782   0.486  -4.028  1.00  0.00           H  \nATOM     63  HB2 LEU A   5       6.863  -1.153  -1.802  1.00  0.00           H  \nATOM     64  HB3 LEU A   5       8.605  -1.202  -1.984  1.00  0.00           H  \nATOM     65  HG  LEU A   5       8.510   1.368  -2.047  1.00  0.00           H  \nATOM     66 HD11 LEU A   5       5.900   1.188  -2.320  1.00  0.00           H  \nATOM     67 HD12 LEU A   5       5.933   1.124  -0.541  1.00  0.00           H  \nATOM     68 HD13 LEU A   5       6.636   2.519  -1.395  1.00  0.00           H  \nATOM     69 HD21 LEU A   5       7.734   0.911   0.662  1.00  0.00           H  \nATOM     70 HD22 LEU A   5       8.636  -0.511   0.085  1.00  0.00           H  \nATOM     71 HD23 LEU A   5       9.366   1.110  -0.020  1.00  0.00           H  \nATOM     72  N   PHE A   6       8.146  -2.756  -4.473  1.00  0.00           N  \nATOM     73  CA  PHE A   6       8.944  -3.761  -5.154  1.00  0.00           C  \nATOM     74  C   PHE A   6       9.099  -3.425  -6.638  1.00  0.00           C  \nATOM     75  O   PHE A   6      10.201  -3.492  -7.181  1.00  0.00           O  \nATOM     76  CB  PHE A   6       8.198  -5.090  -5.019  1.00  0.00           C  \nATOM     77  CG  PHE A   6       8.466  -6.073  -6.161  1.00  0.00           C  \nATOM     78  CD1 PHE A   6       7.721  -6.014  -7.298  1.00  0.00           C  \nATOM     79  CD2 PHE A   6       9.449  -7.005  -6.040  1.00  0.00           C  \nATOM     80  CE1 PHE A   6       7.970  -6.925  -8.358  1.00  0.00           C  \nATOM     81  CE2 PHE A   6       9.698  -7.916  -7.100  1.00  0.00           C  \nATOM     82  CZ  PHE A   6       8.953  -7.857  -8.236  1.00  0.00           C  \nATOM     83  H   PHE A   6       7.335  -3.106  -4.005  1.00  0.00           H  \nATOM     84  HA  PHE A   6       9.926  -3.770  -4.680  1.00  0.00           H  \nATOM     85  HB2 PHE A   6       8.479  -5.560  -4.077  1.00  0.00           H  \nATOM     86  HB3 PHE A   6       7.128  -4.891  -4.969  1.00  0.00           H  \nATOM     87  HD1 PHE A   6       6.933  -5.267  -7.395  1.00  0.00           H  \nATOM     88  HD2 PHE A   6      10.046  -7.052  -5.129  1.00  0.00           H  \nATOM     89  HE1 PHE A   6       7.373  -6.878  -9.269  1.00  0.00           H  \nATOM     90  HE2 PHE A   6      10.485  -8.663  -7.003  1.00  0.00           H  \nATOM     91  HZ  PHE A   6       9.144  -8.557  -9.050  1.00  0.00           H  \nATOM     92  N   LYS A   7       7.980  -3.071  -7.252  1.00  0.00           N  \nATOM     93  CA  LYS A   7       7.978  -2.725  -8.663  1.00  0.00           C  \nATOM     94  C   LYS A   7       8.676  -1.376  -8.854  1.00  0.00           C  \nATOM     95  O   LYS A   7       9.418  -1.188  -9.816  1.00  0.00           O  \nATOM     96  CB  LYS A   7       6.554  -2.764  -9.223  1.00  0.00           C  \nATOM     97  CG  LYS A   7       5.666  -1.725  -8.535  1.00  0.00           C  \nATOM     98  CD  LYS A   7       5.567  -0.449  -9.372  1.00  0.00           C  \nATOM     99  CE  LYS A   7       4.376   0.405  -8.933  1.00  0.00           C  \nATOM    100  NZ  LYS A   7       4.344   1.678  -9.686  1.00  0.00           N  \nATOM    101  H   LYS A   7       7.088  -3.020  -6.803  1.00  0.00           H  \nATOM    102  HA  LYS A   7       8.552  -3.488  -9.188  1.00  0.00           H  \nATOM    103  HB2 LYS A   7       6.576  -2.577 -10.296  1.00  0.00           H  \nATOM    104  HB3 LYS A   7       6.132  -3.759  -9.082  1.00  0.00           H  \nATOM    105  HG2 LYS A   7       4.670  -2.140  -8.379  1.00  0.00           H  \nATOM    106  HG3 LYS A   7       6.071  -1.489  -7.551  1.00  0.00           H  \nATOM    107  HD2 LYS A   7       6.487   0.127  -9.274  1.00  0.00           H  \nATOM    108  HD3 LYS A   7       5.464  -0.707 -10.426  1.00  0.00           H  \nATOM    109  HE2 LYS A   7       3.448  -0.144  -9.095  1.00  0.00           H  \nATOM    110  HE3 LYS A   7       4.442   0.609  -7.864  1.00  0.00           H  \nATOM    111  HZ1 LYS A   7       5.280   1.985  -9.861  1.00  0.00           H  \nATOM    112  HZ2 LYS A   7       3.871   1.539 -10.556  1.00  0.00           H  \nATOM    113  HZ3 LYS A   7       3.861   2.370  -9.150  1.00  0.00           H  \nATOM    114  N   GLU A   8       8.413  -0.473  -7.921  1.00  0.00           N  \nATOM    115  CA  GLU A   8       9.006   0.852  -7.974  1.00  0.00           C  \nATOM    116  C   GLU A   8      10.531   0.748  -8.052  1.00  0.00           C  \nATOM    117  O   GLU A   8      11.143   1.247  -8.995  1.00  0.00           O  \nATOM    118  CB  GLU A   8       8.575   1.695  -6.772  1.00  0.00           C  \nATOM    119  CG  GLU A   8       9.500   2.900  -6.587  1.00  0.00           C  \nATOM    120  CD  GLU A   8       8.823   3.987  -5.751  1.00  0.00           C  \nATOM    121  OE1 GLU A   8       8.014   4.736  -6.340  1.00  0.00           O  \nATOM    122  OE2 GLU A   8       9.129   4.045  -4.540  1.00  0.00           O  \nATOM    123  H   GLU A   8       7.808  -0.634  -7.141  1.00  0.00           H  \nATOM    124  HA  GLU A   8       8.619   1.307  -8.886  1.00  0.00           H  \nATOM    125  HB2 GLU A   8       7.550   2.037  -6.912  1.00  0.00           H  \nATOM    126  HB3 GLU A   8       8.586   1.082  -5.871  1.00  0.00           H  \nATOM    127  HG2 GLU A   8      10.423   2.583  -6.101  1.00  0.00           H  \nATOM    128  HG3 GLU A   8       9.776   3.304  -7.561  1.00  0.00           H  \nATOM    129  N   ILE A   9      11.100   0.097  -7.048  1.00  0.00           N  \nATOM    130  CA  ILE A   9      12.541  -0.079  -6.992  1.00  0.00           C  \nATOM    131  C   ILE A   9      13.032  -0.659  -8.320  1.00  0.00           C  \nATOM    132  O   ILE A   9      14.110  -0.304  -8.795  1.00  0.00           O  \nATOM    133  CB  ILE A   9      12.931  -0.918  -5.773  1.00  0.00           C  \nATOM    134  CG1 ILE A   9      14.167  -0.340  -5.082  1.00  0.00           C  \nATOM    135  CG2 ILE A   9      13.123  -2.387  -6.157  1.00  0.00           C  \nATOM    136  CD1 ILE A   9      15.311  -0.144  -6.078  1.00  0.00           C  \nATOM    137  H   ILE A   9      10.595  -0.306  -6.285  1.00  0.00           H  \nATOM    138  HA  ILE A   9      12.986   0.907  -6.860  1.00  0.00           H  \nATOM    139  HB  ILE A   9      12.112  -0.878  -5.055  1.00  0.00           H  \nATOM    140 HG12 ILE A   9      13.916   0.615  -4.619  1.00  0.00           H  \nATOM    141 HG13 ILE A   9      14.487  -1.007  -4.282  1.00  0.00           H  \nATOM    142 HG21 ILE A   9      13.469  -2.947  -5.288  1.00  0.00           H  \nATOM    143 HG22 ILE A   9      12.175  -2.799  -6.501  1.00  0.00           H  \nATOM    144 HG23 ILE A   9      13.863  -2.460  -6.954  1.00  0.00           H  \nATOM    145 HD11 ILE A   9      15.194  -0.842  -6.908  1.00  0.00           H  \nATOM    146 HD12 ILE A   9      15.293   0.878  -6.457  1.00  0.00           H  \nATOM    147 HD13 ILE A   9      16.263  -0.330  -5.580  1.00  0.00           H  \nATOM    148  N   ASP A  10      12.219  -1.541  -8.881  1.00  0.00           N  \nATOM    149  CA  ASP A  10      12.557  -2.173 -10.145  1.00  0.00           C  \nATOM    150  C   ASP A  10      12.399  -1.157 -11.277  1.00  0.00           C  \nATOM    151  O   ASP A  10      11.449  -1.233 -12.056  1.00  0.00           O  \nATOM    152  CB  ASP A  10      11.629  -3.354 -10.436  1.00  0.00           C  \nATOM    153  CG  ASP A  10      11.740  -3.931 -11.849  1.00  0.00           C  \nATOM    154  OD1 ASP A  10      12.621  -3.447 -12.591  1.00  0.00           O  \nATOM    155  OD2 ASP A  10      10.941  -4.843 -12.154  1.00  0.00           O  \nATOM    156  H   ASP A  10      11.344  -1.825  -8.488  1.00  0.00           H  \nATOM    157  HA  ASP A  10      13.587  -2.512 -10.029  1.00  0.00           H  \nATOM    158  HB2 ASP A  10      11.839  -4.147  -9.718  1.00  0.00           H  \nATOM    159  HB3 ASP A  10      10.600  -3.038 -10.269  1.00  0.00           H  \nATOM    160  N   VAL A  11      13.343  -0.229 -11.333  1.00  0.00           N  \nATOM    161  CA  VAL A  11      13.320   0.801 -12.357  1.00  0.00           C  \nATOM    162  C   VAL A  11      13.134   0.148 -13.728  1.00  0.00           C  \nATOM    163  O   VAL A  11      12.421   0.676 -14.580  1.00  0.00           O  \nATOM    164  CB  VAL A  11      14.586   1.656 -12.269  1.00  0.00           C  \nATOM    165  CG1 VAL A  11      14.718   2.568 -13.489  1.00  0.00           C  \nATOM    166  CG2 VAL A  11      14.611   2.467 -10.972  1.00  0.00           C  \nATOM    167  H   VAL A  11      14.112  -0.174 -10.696  1.00  0.00           H  \nATOM    168  HA  VAL A  11      12.464   1.445 -12.157  1.00  0.00           H  \nATOM    169  HB  VAL A  11      15.444   0.983 -12.259  1.00  0.00           H  \nATOM    170 HG11 VAL A  11      13.763   3.056 -13.683  1.00  0.00           H  \nATOM    171 HG12 VAL A  11      15.480   3.324 -13.297  1.00  0.00           H  \nATOM    172 HG13 VAL A  11      15.005   1.975 -14.357  1.00  0.00           H  \nATOM    173 HG21 VAL A  11      15.560   2.997 -10.893  1.00  0.00           H  \nATOM    174 HG22 VAL A  11      13.792   3.187 -10.978  1.00  0.00           H  \nATOM    175 HG23 VAL A  11      14.497   1.795 -10.121  1.00  0.00           H  \nATOM    176  N   ASN A  12      13.788  -0.992 -13.898  1.00  0.00           N  \nATOM    177  CA  ASN A  12      13.704  -1.723 -15.151  1.00  0.00           C  \nATOM    178  C   ASN A  12      12.233  -1.920 -15.524  1.00  0.00           C  \nATOM    179  O   ASN A  12      11.872  -1.847 -16.697  1.00  0.00           O  \nATOM    180  CB  ASN A  12      14.349  -3.105 -15.026  1.00  0.00           C  \nATOM    181  CG  ASN A  12      15.686  -3.022 -14.287  1.00  0.00           C  \nATOM    182  OD1 ASN A  12      15.859  -3.553 -13.203  1.00  0.00           O  \nATOM    183  ND2 ASN A  12      16.618  -2.326 -14.932  1.00  0.00           N  \nATOM    184  H   ASN A  12      14.366  -1.415 -13.200  1.00  0.00           H  \nATOM    185  HA  ASN A  12      14.240  -1.111 -15.876  1.00  0.00           H  \nATOM    186  HB2 ASN A  12      13.676  -3.778 -14.494  1.00  0.00           H  \nATOM    187  HB3 ASN A  12      14.504  -3.529 -16.019  1.00  0.00           H  \nATOM    188 HD21 ASN A  12      16.411  -1.915 -15.820  1.00  0.00           H  \nATOM    189 HD22 ASN A  12      17.527  -2.215 -14.530  1.00  0.00           H  \nATOM    190  N   GLY A  13      11.425  -2.165 -14.503  1.00  0.00           N  \nATOM    191  CA  GLY A  13      10.001  -2.372 -14.709  1.00  0.00           C  \nATOM    192  C   GLY A  13       9.740  -3.686 -15.449  1.00  0.00           C  \nATOM    193  O   GLY A  13       9.189  -3.685 -16.548  1.00  0.00           O  \nATOM    194  H   GLY A  13      11.727  -2.222 -13.552  1.00  0.00           H  \nATOM    195  HA2 GLY A  13       9.489  -2.384 -13.748  1.00  0.00           H  \nATOM    196  HA3 GLY A  13       9.588  -1.541 -15.281  1.00  0.00           H  \nATOM    197  N   ASP A  14      10.149  -4.776 -14.815  1.00  0.00           N  \nATOM    198  CA  ASP A  14       9.966  -6.094 -15.399  1.00  0.00           C  \nATOM    199  C   ASP A  14       9.014  -6.908 -14.520  1.00  0.00           C  \nATOM    200  O   ASP A  14       8.137  -7.605 -15.029  1.00  0.00           O  \nATOM    201  CB  ASP A  14      11.294  -6.848 -15.483  1.00  0.00           C  \nATOM    202  CG  ASP A  14      12.189  -6.720 -14.249  1.00  0.00           C  \nATOM    203  OD1 ASP A  14      11.861  -7.379 -13.238  1.00  0.00           O  \nATOM    204  OD2 ASP A  14      13.182  -5.967 -14.344  1.00  0.00           O  \nATOM    205  H   ASP A  14      10.596  -4.768 -13.921  1.00  0.00           H  \nATOM    206  HA  ASP A  14       9.562  -5.912 -16.395  1.00  0.00           H  \nATOM    207  HB2 ASP A  14      11.085  -7.904 -15.653  1.00  0.00           H  \nATOM    208  HB3 ASP A  14      11.845  -6.488 -16.352  1.00  0.00           H  \nATOM    209  N   GLY A  15       9.219  -6.794 -13.217  1.00  0.00           N  \nATOM    210  CA  GLY A  15       8.391  -7.511 -12.263  1.00  0.00           C  \nATOM    211  C   GLY A  15       9.215  -8.537 -11.482  1.00  0.00           C  \nATOM    212  O   GLY A  15       8.729  -9.623 -11.171  1.00  0.00           O  \nATOM    213  H   GLY A  15       9.935  -6.225 -12.811  1.00  0.00           H  \nATOM    214  HA2 GLY A  15       7.932  -6.805 -11.571  1.00  0.00           H  \nATOM    215  HA3 GLY A  15       7.579  -8.015 -12.787  1.00  0.00           H  \nATOM    216  N   ALA A  16      10.450  -8.155 -11.188  1.00  0.00           N  \nATOM    217  CA  ALA A  16      11.346  -9.028 -10.449  1.00  0.00           C  \nATOM    218  C   ALA A  16      12.419  -8.183  -9.758  1.00  0.00           C  \nATOM    219  O   ALA A  16      12.739  -7.087 -10.214  1.00  0.00           O  \nATOM    220  CB  ALA A  16      11.945 -10.067 -11.399  1.00  0.00           C  \nATOM    221  H   ALA A  16      10.838  -7.270 -11.444  1.00  0.00           H  \nATOM    222  HA  ALA A  16      10.757  -9.544  -9.691  1.00  0.00           H  \nATOM    223  HB1 ALA A  16      12.871  -9.680 -11.824  1.00  0.00           H  \nATOM    224  HB2 ALA A  16      12.154 -10.985 -10.849  1.00  0.00           H  \nATOM    225  HB3 ALA A  16      11.237 -10.277 -12.201  1.00  0.00           H  \nATOM    226  N   VAL A  17      12.944  -8.727  -8.670  1.00  0.00           N  \nATOM    227  CA  VAL A  17      13.974  -8.037  -7.912  1.00  0.00           C  \nATOM    228  C   VAL A  17      15.240  -8.895  -7.881  1.00  0.00           C  \nATOM    229  O   VAL A  17      15.194 -10.087  -8.183  1.00  0.00           O  \nATOM    230  CB  VAL A  17      13.452  -7.690  -6.516  1.00  0.00           C  \nATOM    231  CG1 VAL A  17      14.605  -7.540  -5.521  1.00  0.00           C  \nATOM    232  CG2 VAL A  17      12.590  -6.426  -6.551  1.00  0.00           C  \nATOM    233  H   VAL A  17      12.678  -9.619  -8.306  1.00  0.00           H  \nATOM    234  HA  VAL A  17      14.195  -7.104  -8.431  1.00  0.00           H  \nATOM    235  HB  VAL A  17      12.824  -8.515  -6.179  1.00  0.00           H  \nATOM    236 HG11 VAL A  17      15.207  -8.449  -5.523  1.00  0.00           H  \nATOM    237 HG12 VAL A  17      15.226  -6.692  -5.810  1.00  0.00           H  \nATOM    238 HG13 VAL A  17      14.204  -7.372  -4.522  1.00  0.00           H  \nATOM    239 HG21 VAL A  17      13.137  -5.627  -7.051  1.00  0.00           H  \nATOM    240 HG22 VAL A  17      11.668  -6.631  -7.094  1.00  0.00           H  \nATOM    241 HG23 VAL A  17      12.352  -6.121  -5.532  1.00  0.00           H  \nATOM    242  N   SER A  18      16.341  -8.256  -7.512  1.00  0.00           N  \nATOM    243  CA  SER A  18      17.617  -8.947  -7.438  1.00  0.00           C  \nATOM    244  C   SER A  18      18.254  -8.723  -6.065  1.00  0.00           C  \nATOM    245  O   SER A  18      17.662  -8.080  -5.199  1.00  0.00           O  \nATOM    246  CB  SER A  18      18.563  -8.476  -8.544  1.00  0.00           C  \nATOM    247  OG  SER A  18      19.159  -7.218  -8.238  1.00  0.00           O  \nATOM    248  H   SER A  18      16.370  -7.287  -7.268  1.00  0.00           H  \nATOM    249  HA  SER A  18      17.384 -10.001  -7.585  1.00  0.00           H  \nATOM    250  HB2 SER A  18      19.345  -9.221  -8.693  1.00  0.00           H  \nATOM    251  HB3 SER A  18      18.014  -8.399  -9.482  1.00  0.00           H  \nATOM    252  HG  SER A  18      18.655  -6.482  -8.690  1.00  0.00           H  \nATOM    253  N   TYR A  19      19.451  -9.267  -5.907  1.00  0.00           N  \nATOM    254  CA  TYR A  19      20.175  -9.136  -4.654  1.00  0.00           C  \nATOM    255  C   TYR A  19      20.605  -7.686  -4.419  1.00  0.00           C  \nATOM    256  O   TYR A  19      20.572  -7.200  -3.290  1.00  0.00           O  \nATOM    257  CB  TYR A  19      21.423 -10.010  -4.793  1.00  0.00           C  \nATOM    258  CG  TYR A  19      22.467  -9.781  -3.699  1.00  0.00           C  \nATOM    259  CD1 TYR A  19      22.139  -9.997  -2.376  1.00  0.00           C  \nATOM    260  CD2 TYR A  19      23.738  -9.360  -4.035  1.00  0.00           C  \nATOM    261  CE1 TYR A  19      23.123  -9.782  -1.346  1.00  0.00           C  \nATOM    262  CE2 TYR A  19      24.721  -9.145  -3.005  1.00  0.00           C  \nATOM    263  CZ  TYR A  19      24.365  -9.367  -1.712  1.00  0.00           C  \nATOM    264  OH  TYR A  19      25.294  -9.164  -0.739  1.00  0.00           O  \nATOM    265  H   TYR A  19      19.926  -9.789  -6.616  1.00  0.00           H  \nATOM    266  HA  TYR A  19      19.509  -9.444  -3.848  1.00  0.00           H  \nATOM    267  HB2 TYR A  19      21.123 -11.058  -4.782  1.00  0.00           H  \nATOM    268  HB3 TYR A  19      21.880  -9.819  -5.764  1.00  0.00           H  \nATOM    269  HD1 TYR A  19      21.136 -10.330  -2.111  1.00  0.00           H  \nATOM    270  HD2 TYR A  19      23.997  -9.190  -5.080  1.00  0.00           H  \nATOM    271  HE1 TYR A  19      22.877  -9.949  -0.297  1.00  0.00           H  \nATOM    272  HE2 TYR A  19      25.728  -8.812  -3.257  1.00  0.00           H  \nATOM    273  HH  TYR A  19      26.207  -9.113  -1.143  1.00  0.00           H  \nATOM    274  N   GLU A  20      20.999  -7.036  -5.504  1.00  0.00           N  \nATOM    275  CA  GLU A  20      21.436  -5.652  -5.431  1.00  0.00           C  \nATOM    276  C   GLU A  20      20.231  -4.725  -5.258  1.00  0.00           C  \nATOM    277  O   GLU A  20      20.279  -3.779  -4.473  1.00  0.00           O  \nATOM    278  CB  GLU A  20      22.251  -5.268  -6.667  1.00  0.00           C  \nATOM    279  CG  GLU A  20      23.413  -4.345  -6.293  1.00  0.00           C  \nATOM    280  CD  GLU A  20      24.240  -3.977  -7.526  1.00  0.00           C  \nATOM    281  OE1 GLU A  20      25.042  -4.838  -7.949  1.00  0.00           O  \nATOM    282  OE2 GLU A  20      24.053  -2.844  -8.017  1.00  0.00           O  \nATOM    283  H   GLU A  20      21.023  -7.439  -6.420  1.00  0.00           H  \nATOM    284  HA  GLU A  20      22.075  -5.597  -4.550  1.00  0.00           H  \nATOM    285  HB2 GLU A  20      22.637  -6.168  -7.146  1.00  0.00           H  \nATOM    286  HB3 GLU A  20      21.607  -4.772  -7.393  1.00  0.00           H  \nATOM    287  HG2 GLU A  20      23.026  -3.439  -5.826  1.00  0.00           H  \nATOM    288  HG3 GLU A  20      24.049  -4.836  -5.556  1.00  0.00           H  \nATOM    289  N   GLU A  21      19.180  -5.027  -6.006  1.00  0.00           N  \nATOM    290  CA  GLU A  21      17.965  -4.232  -5.945  1.00  0.00           C  \nATOM    291  C   GLU A  21      17.362  -4.290  -4.540  1.00  0.00           C  \nATOM    292  O   GLU A  21      16.946  -3.268  -3.997  1.00  0.00           O  \nATOM    293  CB  GLU A  21      16.954  -4.694  -6.996  1.00  0.00           C  \nATOM    294  CG  GLU A  21      16.722  -3.609  -8.049  1.00  0.00           C  \nATOM    295  CD  GLU A  21      15.862  -4.137  -9.200  1.00  0.00           C  \nATOM    296  OE1 GLU A  21      15.436  -5.308  -9.099  1.00  0.00           O  \nATOM    297  OE2 GLU A  21      15.650  -3.357 -10.153  1.00  0.00           O  \nATOM    298  H   GLU A  21      19.149  -5.797  -6.642  1.00  0.00           H  \nATOM    299  HA  GLU A  21      18.275  -3.212  -6.173  1.00  0.00           H  \nATOM    300  HB2 GLU A  21      17.315  -5.603  -7.478  1.00  0.00           H  \nATOM    301  HB3 GLU A  21      16.010  -4.944  -6.512  1.00  0.00           H  \nATOM    302  HG2 GLU A  21      16.234  -2.750  -7.590  1.00  0.00           H  \nATOM    303  HG3 GLU A  21      17.680  -3.262  -8.436  1.00  0.00           H  \nATOM    304  N   VAL A  22      17.334  -5.496  -3.993  1.00  0.00           N  \nATOM    305  CA  VAL A  22      16.788  -5.701  -2.662  1.00  0.00           C  \nATOM    306  C   VAL A  22      17.695  -5.021  -1.634  1.00  0.00           C  \nATOM    307  O   VAL A  22      17.211  -4.387  -0.697  1.00  0.00           O  \nATOM    308  CB  VAL A  22      16.601  -7.196  -2.397  1.00  0.00           C  \nATOM    309  CG1 VAL A  22      17.951  -7.897  -2.231  1.00  0.00           C  \nATOM    310  CG2 VAL A  22      15.707  -7.430  -1.178  1.00  0.00           C  \nATOM    311  H   VAL A  22      17.674  -6.322  -4.442  1.00  0.00           H  \nATOM    312  HA  VAL A  22      15.807  -5.227  -2.632  1.00  0.00           H  \nATOM    313  HB  VAL A  22      16.103  -7.630  -3.265  1.00  0.00           H  \nATOM    314 HG11 VAL A  22      18.532  -7.388  -1.461  1.00  0.00           H  \nATOM    315 HG12 VAL A  22      17.788  -8.934  -1.937  1.00  0.00           H  \nATOM    316 HG13 VAL A  22      18.494  -7.868  -3.175  1.00  0.00           H  \nATOM    317 HG21 VAL A  22      15.796  -6.584  -0.496  1.00  0.00           H  \nATOM    318 HG22 VAL A  22      14.671  -7.530  -1.500  1.00  0.00           H  \nATOM    319 HG23 VAL A  22      16.019  -8.341  -0.668  1.00  0.00           H  \nATOM    320  N   LYS A  23      18.994  -5.177  -1.843  1.00  0.00           N  \nATOM    321  CA  LYS A  23      19.972  -4.586  -0.946  1.00  0.00           C  \nATOM    322  C   LYS A  23      19.723  -3.080  -0.847  1.00  0.00           C  \nATOM    323  O   LYS A  23      19.768  -2.509   0.242  1.00  0.00           O  \nATOM    324  CB  LYS A  23      21.392  -4.947  -1.388  1.00  0.00           C  \nATOM    325  CG  LYS A  23      21.857  -6.248  -0.732  1.00  0.00           C  \nATOM    326  CD  LYS A  23      23.384  -6.311  -0.660  1.00  0.00           C  \nATOM    327  CE  LYS A  23      24.004  -6.187  -2.053  1.00  0.00           C  \nATOM    328  NZ  LYS A  23      24.512  -4.815  -2.274  1.00  0.00           N  \nATOM    329  H   LYS A  23      19.379  -5.694  -2.607  1.00  0.00           H  \nATOM    330  HA  LYS A  23      19.820  -5.027   0.040  1.00  0.00           H  \nATOM    331  HB2 LYS A  23      21.423  -5.051  -2.473  1.00  0.00           H  \nATOM    332  HB3 LYS A  23      22.075  -4.139  -1.127  1.00  0.00           H  \nATOM    333  HG2 LYS A  23      21.438  -6.323   0.272  1.00  0.00           H  \nATOM    334  HG3 LYS A  23      21.481  -7.100  -1.298  1.00  0.00           H  \nATOM    335  HD2 LYS A  23      23.754  -5.510  -0.020  1.00  0.00           H  \nATOM    336  HD3 LYS A  23      23.692  -7.251  -0.203  1.00  0.00           H  \nATOM    337  HE2 LYS A  23      24.818  -6.904  -2.160  1.00  0.00           H  \nATOM    338  HE3 LYS A  23      23.261  -6.432  -2.811  1.00  0.00           H  \nATOM    339  HZ1 LYS A  23      24.480  -4.600  -3.250  1.00  0.00           H  \nATOM    340  HZ2 LYS A  23      23.943  -4.164  -1.771  1.00  0.00           H  \nATOM    341  HZ3 LYS A  23      25.456  -4.752  -1.950  1.00  0.00           H  \nATOM    342  N   ALA A  24      19.466  -2.478  -1.999  1.00  0.00           N  \nATOM    343  CA  ALA A  24      19.209  -1.049  -2.056  1.00  0.00           C  \nATOM    344  C   ALA A  24      17.877  -0.746  -1.367  1.00  0.00           C  \nATOM    345  O   ALA A  24      17.767   0.224  -0.620  1.00  0.00           O  \nATOM    346  CB  ALA A  24      19.231  -0.584  -3.514  1.00  0.00           C  \nATOM    347  H   ALA A  24      19.431  -2.949  -2.880  1.00  0.00           H  \nATOM    348  HA  ALA A  24      20.011  -0.546  -1.515  1.00  0.00           H  \nATOM    349  HB1 ALA A  24      19.621   0.432  -3.565  1.00  0.00           H  \nATOM    350  HB2 ALA A  24      19.868  -1.248  -4.097  1.00  0.00           H  \nATOM    351  HB3 ALA A  24      18.218  -0.605  -3.917  1.00  0.00           H  \nATOM    352  N   PHE A  25      16.898  -1.595  -1.644  1.00  0.00           N  \nATOM    353  CA  PHE A  25      15.578  -1.431  -1.060  1.00  0.00           C  \nATOM    354  C   PHE A  25      15.664  -1.288   0.461  1.00  0.00           C  \nATOM    355  O   PHE A  25      15.250  -0.273   1.018  1.00  0.00           O  \nATOM    356  CB  PHE A  25      14.779  -2.691  -1.398  1.00  0.00           C  \nATOM    357  CG  PHE A  25      13.276  -2.562  -1.143  1.00  0.00           C  \nATOM    358  CD1 PHE A  25      12.772  -2.822   0.093  1.00  0.00           C  \nATOM    359  CD2 PHE A  25      12.444  -2.189  -2.152  1.00  0.00           C  \nATOM    360  CE1 PHE A  25      11.377  -2.702   0.330  1.00  0.00           C  \nATOM    361  CE2 PHE A  25      11.049  -2.069  -1.915  1.00  0.00           C  \nATOM    362  CZ  PHE A  25      10.545  -2.329  -0.679  1.00  0.00           C  \nATOM    363  H   PHE A  25      16.996  -2.382  -2.253  1.00  0.00           H  \nATOM    364  HA  PHE A  25      15.147  -0.523  -1.484  1.00  0.00           H  \nATOM    365  HB2 PHE A  25      14.939  -2.940  -2.447  1.00  0.00           H  \nATOM    366  HB3 PHE A  25      15.167  -3.523  -0.810  1.00  0.00           H  \nATOM    367  HD1 PHE A  25      13.438  -3.121   0.902  1.00  0.00           H  \nATOM    368  HD2 PHE A  25      12.849  -1.981  -3.143  1.00  0.00           H  \nATOM    369  HE1 PHE A  25      10.972  -2.910   1.321  1.00  0.00           H  \nATOM    370  HE2 PHE A  25      10.383  -1.770  -2.724  1.00  0.00           H  \nATOM    371  HZ  PHE A  25       9.474  -2.237  -0.497  1.00  0.00           H  \nATOM    372  N   VAL A  26      16.206  -2.321   1.090  1.00  0.00           N  \nATOM    373  CA  VAL A  26      16.352  -2.324   2.535  1.00  0.00           C  \nATOM    374  C   VAL A  26      17.288  -1.187   2.951  1.00  0.00           C  \nATOM    375  O   VAL A  26      17.037  -0.503   3.942  1.00  0.00           O  \nATOM    376  CB  VAL A  26      16.832  -3.698   3.010  1.00  0.00           C  \nATOM    377  CG1 VAL A  26      18.356  -3.806   2.921  1.00  0.00           C  \nATOM    378  CG2 VAL A  26      16.343  -3.989   4.430  1.00  0.00           C  \nATOM    379  H   VAL A  26      16.541  -3.144   0.629  1.00  0.00           H  \nATOM    380  HA  VAL A  26      15.367  -2.144   2.966  1.00  0.00           H  \nATOM    381  HB  VAL A  26      16.404  -4.450   2.347  1.00  0.00           H  \nATOM    382 HG11 VAL A  26      18.684  -3.505   1.927  1.00  0.00           H  \nATOM    383 HG12 VAL A  26      18.811  -3.154   3.667  1.00  0.00           H  \nATOM    384 HG13 VAL A  26      18.658  -4.837   3.107  1.00  0.00           H  \nATOM    385 HG21 VAL A  26      15.590  -4.777   4.400  1.00  0.00           H  \nATOM    386 HG22 VAL A  26      17.184  -4.312   5.044  1.00  0.00           H  \nATOM    387 HG23 VAL A  26      15.907  -3.085   4.857  1.00  0.00           H  \nATOM    388  N   SER A  27      18.347  -1.021   2.173  1.00  0.00           N  \nATOM    389  CA  SER A  27      19.322   0.021   2.449  1.00  0.00           C  \nATOM    390  C   SER A  27      18.611   1.359   2.659  1.00  0.00           C  \nATOM    391  O   SER A  27      18.971   2.124   3.553  1.00  0.00           O  \nATOM    392  CB  SER A  27      20.342   0.134   1.314  1.00  0.00           C  \nATOM    393  OG  SER A  27      21.305   1.155   1.563  1.00  0.00           O  \nATOM    394  H   SER A  27      18.544  -1.582   1.369  1.00  0.00           H  \nATOM    395  HA  SER A  27      19.828  -0.291   3.362  1.00  0.00           H  \nATOM    396  HB2 SER A  27      20.851  -0.821   1.188  1.00  0.00           H  \nATOM    397  HB3 SER A  27      19.823   0.345   0.379  1.00  0.00           H  \nATOM    398  HG  SER A  27      21.569   1.593   0.703  1.00  0.00           H  \nATOM    399  N   LYS A  28      17.615   1.602   1.820  1.00  0.00           N  \nATOM    400  CA  LYS A  28      16.850   2.835   1.903  1.00  0.00           C  \nATOM    401  C   LYS A  28      16.434   3.076   3.355  1.00  0.00           C  \nATOM    402  O   LYS A  28      16.290   4.221   3.782  1.00  0.00           O  \nATOM    403  CB  LYS A  28      15.675   2.804   0.924  1.00  0.00           C  \nATOM    404  CG  LYS A  28      15.792   3.926  -0.110  1.00  0.00           C  \nATOM    405  CD  LYS A  28      14.798   3.722  -1.255  1.00  0.00           C  \nATOM    406  CE  LYS A  28      13.832   4.904  -1.360  1.00  0.00           C  \nATOM    407  NZ  LYS A  28      12.893   4.708  -2.487  1.00  0.00           N  \nATOM    408  H   LYS A  28      17.328   0.975   1.095  1.00  0.00           H  \nATOM    409  HA  LYS A  28      17.507   3.648   1.592  1.00  0.00           H  \nATOM    410  HB2 LYS A  28      15.644   1.839   0.417  1.00  0.00           H  \nATOM    411  HB3 LYS A  28      14.738   2.905   1.471  1.00  0.00           H  \nATOM    412  HG2 LYS A  28      15.607   4.887   0.370  1.00  0.00           H  \nATOM    413  HG3 LYS A  28      16.807   3.957  -0.505  1.00  0.00           H  \nATOM    414  HD2 LYS A  28      15.339   3.606  -2.194  1.00  0.00           H  \nATOM    415  HD3 LYS A  28      14.237   2.802  -1.095  1.00  0.00           H  \nATOM    416  HE2 LYS A  28      13.274   5.009  -0.429  1.00  0.00           H  \nATOM    417  HE3 LYS A  28      14.392   5.828  -1.502  1.00  0.00           H  \nATOM    418  HZ1 LYS A  28      12.507   5.590  -2.756  1.00  0.00           H  \nATOM    419  HZ2 LYS A  28      13.383   4.308  -3.262  1.00  0.00           H  \nATOM    420  HZ3 LYS A  28      12.157   4.093  -2.205  1.00  0.00           H  \nATOM    421  N   LYS A  29      16.253   1.978   4.075  1.00  0.00           N  \nATOM    422  CA  LYS A  29      15.857   2.056   5.471  1.00  0.00           C  \nATOM    423  C   LYS A  29      17.077   2.409   6.324  1.00  0.00           C  \nATOM    424  O   LYS A  29      17.254   3.562   6.713  1.00  0.00           O  \nATOM    425  CB  LYS A  29      15.158   0.765   5.902  1.00  0.00           C  \nATOM    426  CG  LYS A  29      13.873   1.068   6.674  1.00  0.00           C  \nATOM    427  CD  LYS A  29      12.683   0.316   6.076  1.00  0.00           C  \nATOM    428  CE  LYS A  29      11.969  -0.518   7.142  1.00  0.00           C  \nATOM    429  NZ  LYS A  29      12.823  -1.646   7.576  1.00  0.00           N  \nATOM    430  H   LYS A  29      16.373   1.051   3.720  1.00  0.00           H  \nATOM    431  HA  LYS A  29      15.129   2.862   5.559  1.00  0.00           H  \nATOM    432  HB2 LYS A  29      14.925   0.162   5.024  1.00  0.00           H  \nATOM    433  HB3 LYS A  29      15.830   0.174   6.524  1.00  0.00           H  \nATOM    434  HG2 LYS A  29      13.997   0.787   7.719  1.00  0.00           H  \nATOM    435  HG3 LYS A  29      13.677   2.140   6.653  1.00  0.00           H  \nATOM    436  HD2 LYS A  29      11.983   1.026   5.635  1.00  0.00           H  \nATOM    437  HD3 LYS A  29      13.026  -0.334   5.271  1.00  0.00           H  \nATOM    438  HE2 LYS A  29      11.723   0.110   7.998  1.00  0.00           H  \nATOM    439  HE3 LYS A  29      11.028  -0.898   6.745  1.00  0.00           H  \nATOM    440  HZ1 LYS A  29      13.780  -1.436   7.375  1.00  0.00           H  \nATOM    441  HZ2 LYS A  29      12.713  -1.788   8.560  1.00  0.00           H  \nATOM    442  HZ3 LYS A  29      12.554  -2.475   7.087  1.00  0.00           H  \nATOM    443  N   ARG A  30      17.886   1.394   6.591  1.00  0.00           N  \nATOM    444  CA  ARG A  30      19.084   1.583   7.391  1.00  0.00           C  \nATOM    445  C   ARG A  30      20.279   0.893   6.729  1.00  0.00           C  \nATOM    446  O   ARG A  30      20.179   0.421   5.598  1.00  0.00           O  \nATOM    447  CB  ARG A  30      18.898   1.023   8.802  1.00  0.00           C  \nATOM    448  CG  ARG A  30      17.586   1.513   9.418  1.00  0.00           C  \nATOM    449  CD  ARG A  30      17.499   1.133  10.898  1.00  0.00           C  \nATOM    450  NE  ARG A  30      17.661   2.339  11.740  1.00  0.00           N  \nATOM    451  CZ  ARG A  30      17.265   2.422  13.018  1.00  0.00           C  \nATOM    452  NH1 ARG A  30      16.682   1.369  13.608  1.00  0.00           N  \nATOM    453  NH2 ARG A  30      17.453   3.556  13.705  1.00  0.00           N  \nATOM    454  H   ARG A  30      17.735   0.459   6.271  1.00  0.00           H  \nATOM    455  HA  ARG A  30      19.227   2.663   7.429  1.00  0.00           H  \nATOM    456  HB2 ARG A  30      18.903  -0.067   8.769  1.00  0.00           H  \nATOM    457  HB3 ARG A  30      19.735   1.326   9.431  1.00  0.00           H  \nATOM    458  HG2 ARG A  30      17.512   2.596   9.312  1.00  0.00           H  \nATOM    459  HG3 ARG A  30      16.743   1.082   8.878  1.00  0.00           H  \nATOM    460  HD2 ARG A  30      16.538   0.661  11.104  1.00  0.00           H  \nATOM    461  HD3 ARG A  30      18.271   0.403  11.140  1.00  0.00           H  \nATOM    462  HE  ARG A  30      18.093   3.143  11.330  1.00  0.00           H  \nATOM    463 HH11 ARG A  30      16.542   0.523  13.094  1.00  0.00           H  \nATOM    464 HH12 ARG A  30      16.386   1.431  14.561  1.00  0.00           H  \nATOM    465 HH21 ARG A  30      17.889   4.342  13.265  1.00  0.00           H  \nATOM    466 HH22 ARG A  30      17.158   3.618  14.659  1.00  0.00           H  \nATOM    467  N   ALA A  31      21.381   0.856   7.463  1.00  0.00           N  \nATOM    468  CA  ALA A  31      22.594   0.231   6.962  1.00  0.00           C  \nATOM    469  C   ALA A  31      23.280  -0.527   8.100  1.00  0.00           C  \nATOM    470  O   ALA A  31      24.458  -0.305   8.377  1.00  0.00           O  \nATOM    471  CB  ALA A  31      23.499   1.297   6.342  1.00  0.00           C  \nATOM    472  H   ALA A  31      21.454   1.242   8.383  1.00  0.00           H  \nATOM    473  HA  ALA A  31      22.305  -0.479   6.186  1.00  0.00           H  \nATOM    474  HB1 ALA A  31      22.888   2.028   5.811  1.00  0.00           H  \nATOM    475  HB2 ALA A  31      24.062   1.798   7.129  1.00  0.00           H  \nATOM    476  HB3 ALA A  31      24.190   0.826   5.644  1.00  0.00           H  \nATOM    477  N   ILE A  32      22.514  -1.406   8.730  1.00  0.00           N  \nATOM    478  CA  ILE A  32      23.034  -2.197   9.832  1.00  0.00           C  \nATOM    479  C   ILE A  32      23.516  -3.549   9.300  1.00  0.00           C  \nATOM    480  O   ILE A  32      23.115  -4.597   9.804  1.00  0.00           O  \nATOM    481  CB  ILE A  32      21.993  -2.311  10.947  1.00  0.00           C  \nATOM    482  CG1 ILE A  32      20.760  -3.080  10.470  1.00  0.00           C  \nATOM    483  CG2 ILE A  32      21.631  -0.932  11.501  1.00  0.00           C  \nATOM    484  CD1 ILE A  32      20.366  -4.164  11.476  1.00  0.00           C  \nATOM    485  H   ILE A  32      21.557  -1.580   8.499  1.00  0.00           H  \nATOM    486  HA  ILE A  32      23.890  -1.662  10.243  1.00  0.00           H  \nATOM    487  HB  ILE A  32      22.432  -2.881  11.766  1.00  0.00           H  \nATOM    488 HG12 ILE A  32      19.928  -2.389  10.331  1.00  0.00           H  \nATOM    489 HG13 ILE A  32      20.963  -3.534   9.501  1.00  0.00           H  \nATOM    490 HG21 ILE A  32      22.542  -0.361  11.681  1.00  0.00           H  \nATOM    491 HG22 ILE A  32      21.007  -0.403  10.780  1.00  0.00           H  \nATOM    492 HG23 ILE A  32      21.084  -1.048  12.437  1.00  0.00           H  \nATOM    493 HD11 ILE A  32      20.296  -3.727  12.472  1.00  0.00           H  \nATOM    494 HD12 ILE A  32      19.401  -4.586  11.196  1.00  0.00           H  \nATOM    495 HD13 ILE A  32      21.121  -4.950  11.476  1.00  0.00           H  \nATOM    496  N   LYS A  33      24.370  -3.481   8.289  1.00  0.00           N  \nATOM    497  CA  LYS A  33      24.910  -4.686   7.684  1.00  0.00           C  \nATOM    498  C   LYS A  33      23.802  -5.407   6.915  1.00  0.00           C  \nATOM    499  O   LYS A  33      23.711  -6.633   6.953  1.00  0.00           O  \nATOM    500  CB  LYS A  33      25.592  -5.557   8.741  1.00  0.00           C  \nATOM    501  CG  LYS A  33      27.114  -5.420   8.666  1.00  0.00           C  \nATOM    502  CD  LYS A  33      27.781  -6.037   9.897  1.00  0.00           C  \nATOM    503  CE  LYS A  33      28.047  -4.976  10.967  1.00  0.00           C  \nATOM    504  NZ  LYS A  33      28.335  -5.616  12.270  1.00  0.00           N  \nATOM    505  H   LYS A  33      24.691  -2.624   7.885  1.00  0.00           H  \nATOM    506  HA  LYS A  33      25.679  -4.379   6.975  1.00  0.00           H  \nATOM    507  HB2 LYS A  33      25.246  -5.269   9.734  1.00  0.00           H  \nATOM    508  HB3 LYS A  33      25.309  -6.599   8.596  1.00  0.00           H  \nATOM    509  HG2 LYS A  33      27.483  -5.909   7.765  1.00  0.00           H  \nATOM    510  HG3 LYS A  33      27.385  -4.367   8.591  1.00  0.00           H  \nATOM    511  HD2 LYS A  33      27.143  -6.821  10.307  1.00  0.00           H  \nATOM    512  HD3 LYS A  33      28.720  -6.510   9.608  1.00  0.00           H  \nATOM    513  HE2 LYS A  33      28.888  -4.352  10.667  1.00  0.00           H  \nATOM    514  HE3 LYS A  33      27.181  -4.321  11.060  1.00  0.00           H  \nATOM    515  HZ1 LYS A  33      27.868  -6.499  12.319  1.00  0.00           H  \nATOM    516  HZ2 LYS A  33      29.321  -5.754  12.363  1.00  0.00           H  \nATOM    517  HZ3 LYS A  33      28.010  -5.026  13.010  1.00  0.00           H  \nATOM    518  N   ASN A  34      22.986  -4.615   6.235  1.00  0.00           N  \nATOM    519  CA  ASN A  34      21.887  -5.162   5.458  1.00  0.00           C  \nATOM    520  C   ASN A  34      22.445  -6.084   4.372  1.00  0.00           C  \nATOM    521  O   ASN A  34      21.786  -7.039   3.964  1.00  0.00           O  \nATOM    522  CB  ASN A  34      21.090  -4.051   4.771  1.00  0.00           C  \nATOM    523  CG  ASN A  34      21.909  -3.399   3.655  1.00  0.00           C  \nATOM    524  OD1 ASN A  34      21.874  -3.806   2.506  1.00  0.00           O  \nATOM    525  ND2 ASN A  34      22.644  -2.366   4.057  1.00  0.00           N  \nATOM    526  H   ASN A  34      23.066  -3.618   6.209  1.00  0.00           H  \nATOM    527  HA  ASN A  34      21.265  -5.695   6.177  1.00  0.00           H  \nATOM    528  HB2 ASN A  34      20.168  -4.461   4.359  1.00  0.00           H  \nATOM    529  HB3 ASN A  34      20.804  -3.297   5.505  1.00  0.00           H  \nATOM    530 HD21 ASN A  34      22.627  -2.083   5.015  1.00  0.00           H  \nATOM    531 HD22 ASN A  34      23.215  -1.873   3.400  1.00  0.00           H  \nATOM    532  N   GLU A  35      23.654  -5.765   3.933  1.00  0.00           N  \nATOM    533  CA  GLU A  35      24.308  -6.552   2.902  1.00  0.00           C  \nATOM    534  C   GLU A  35      24.370  -8.024   3.317  1.00  0.00           C  \nATOM    535  O   GLU A  35      23.910  -8.898   2.584  1.00  0.00           O  \nATOM    536  CB  GLU A  35      25.707  -6.008   2.602  1.00  0.00           C  \nATOM    537  CG  GLU A  35      26.563  -5.967   3.869  1.00  0.00           C  \nATOM    538  CD  GLU A  35      27.724  -4.983   3.714  1.00  0.00           C  \nATOM    539  OE1 GLU A  35      28.463  -5.129   2.717  1.00  0.00           O  \nATOM    540  OE2 GLU A  35      27.847  -4.107   4.597  1.00  0.00           O  \nATOM    541  H   GLU A  35      24.183  -4.986   4.270  1.00  0.00           H  \nATOM    542  HA  GLU A  35      23.685  -6.443   2.015  1.00  0.00           H  \nATOM    543  HB2 GLU A  35      26.191  -6.634   1.852  1.00  0.00           H  \nATOM    544  HB3 GLU A  35      25.628  -5.007   2.179  1.00  0.00           H  \nATOM    545  HG2 GLU A  35      25.946  -5.676   4.720  1.00  0.00           H  \nATOM    546  HG3 GLU A  35      26.951  -6.962   4.083  1.00  0.00           H  \nATOM    547  N   GLN A  36      24.941  -8.252   4.490  1.00  0.00           N  \nATOM    548  CA  GLN A  36      25.068  -9.602   5.011  1.00  0.00           C  \nATOM    549  C   GLN A  36      23.701 -10.134   5.445  1.00  0.00           C  \nATOM    550  O   GLN A  36      23.225 -11.137   4.916  1.00  0.00           O  \nATOM    551  CB  GLN A  36      26.069  -9.652   6.167  1.00  0.00           C  \nATOM    552  CG  GLN A  36      26.604 -11.071   6.368  1.00  0.00           C  \nATOM    553  CD  GLN A  36      27.751 -11.086   7.381  1.00  0.00           C  \nATOM    554  OE1 GLN A  36      28.605 -10.216   7.404  1.00  0.00           O  \nATOM    555  NE2 GLN A  36      27.722 -12.122   8.215  1.00  0.00           N  \nATOM    556  H   GLN A  36      25.313  -7.535   5.080  1.00  0.00           H  \nATOM    557  HA  GLN A  36      25.451 -10.197   4.182  1.00  0.00           H  \nATOM    558  HB2 GLN A  36      26.897  -8.973   5.965  1.00  0.00           H  \nATOM    559  HB3 GLN A  36      25.590  -9.307   7.083  1.00  0.00           H  \nATOM    560  HG2 GLN A  36      25.800 -11.721   6.714  1.00  0.00           H  \nATOM    561  HG3 GLN A  36      26.950 -11.472   5.416  1.00  0.00           H  \nATOM    562 HE21 GLN A  36      26.993 -12.802   8.143  1.00  0.00           H  \nATOM    563 HE22 GLN A  36      28.431 -12.221   8.914  1.00  0.00           H  \nATOM    564  N   LEU A  37      23.109  -9.439   6.405  1.00  0.00           N  \nATOM    565  CA  LEU A  37      21.806  -9.829   6.917  1.00  0.00           C  \nATOM    566  C   LEU A  37      20.920 -10.280   5.754  1.00  0.00           C  \nATOM    567  O   LEU A  37      20.404 -11.396   5.759  1.00  0.00           O  \nATOM    568  CB  LEU A  37      21.199  -8.700   7.751  1.00  0.00           C  \nATOM    569  CG  LEU A  37      20.576  -9.113   9.087  1.00  0.00           C  \nATOM    570  CD1 LEU A  37      19.222  -9.793   8.873  1.00  0.00           C  \nATOM    571  CD2 LEU A  37      21.537  -9.990   9.892  1.00  0.00           C  \nATOM    572  H   LEU A  37      23.504  -8.625   6.831  1.00  0.00           H  \nATOM    573  HA  LEU A  37      21.958 -10.677   7.585  1.00  0.00           H  \nATOM    574  HB2 LEU A  37      21.976  -7.962   7.949  1.00  0.00           H  \nATOM    575  HB3 LEU A  37      20.433  -8.205   7.154  1.00  0.00           H  \nATOM    576  HG  LEU A  37      20.394  -8.212   9.672  1.00  0.00           H  \nATOM    577 HD11 LEU A  37      18.487  -9.050   8.562  1.00  0.00           H  \nATOM    578 HD12 LEU A  37      19.317 -10.556   8.100  1.00  0.00           H  \nATOM    579 HD13 LEU A  37      18.898 -10.258   9.804  1.00  0.00           H  \nATOM    580 HD21 LEU A  37      21.572  -9.638  10.923  1.00  0.00           H  \nATOM    581 HD22 LEU A  37      21.190 -11.023   9.872  1.00  0.00           H  \nATOM    582 HD23 LEU A  37      22.534  -9.933   9.455  1.00  0.00           H  \nATOM    583  N   LEU A  38      20.772  -9.388   4.785  1.00  0.00           N  \nATOM    584  CA  LEU A  38      19.957  -9.681   3.618  1.00  0.00           C  \nATOM    585  C   LEU A  38      20.465 -10.963   2.955  1.00  0.00           C  \nATOM    586  O   LEU A  38      19.710 -11.919   2.787  1.00  0.00           O  \nATOM    587  CB  LEU A  38      19.919  -8.475   2.676  1.00  0.00           C  \nATOM    588  CG  LEU A  38      19.051  -8.629   1.425  1.00  0.00           C  \nATOM    589  CD1 LEU A  38      18.265  -7.348   1.142  1.00  0.00           C  \nATOM    590  CD2 LEU A  38      19.894  -9.061   0.223  1.00  0.00           C  \nATOM    591  H   LEU A  38      21.195  -8.483   4.789  1.00  0.00           H  \nATOM    592  HA  LEU A  38      18.938  -9.851   3.965  1.00  0.00           H  \nATOM    593  HB2 LEU A  38      19.562  -7.612   3.238  1.00  0.00           H  \nATOM    594  HB3 LEU A  38      20.938  -8.252   2.362  1.00  0.00           H  \nATOM    595  HG  LEU A  38      18.324  -9.420   1.609  1.00  0.00           H  \nATOM    596 HD11 LEU A  38      17.329  -7.599   0.643  1.00  0.00           H  \nATOM    597 HD12 LEU A  38      18.050  -6.838   2.082  1.00  0.00           H  \nATOM    598 HD13 LEU A  38      18.855  -6.693   0.501  1.00  0.00           H  \nATOM    599 HD21 LEU A  38      20.866  -9.414   0.569  1.00  0.00           H  \nATOM    600 HD22 LEU A  38      19.384  -9.865  -0.308  1.00  0.00           H  \nATOM    601 HD23 LEU A  38      20.033  -8.213  -0.447  1.00  0.00           H  \nATOM    602  N   GLN A  39      21.740 -10.941   2.597  1.00  0.00           N  \nATOM    603  CA  GLN A  39      22.357 -12.090   1.956  1.00  0.00           C  \nATOM    604  C   GLN A  39      21.838 -13.388   2.579  1.00  0.00           C  \nATOM    605  O   GLN A  39      21.562 -14.354   1.870  1.00  0.00           O  \nATOM    606  CB  GLN A  39      23.882 -12.013   2.043  1.00  0.00           C  \nATOM    607  CG  GLN A  39      24.538 -12.880   0.966  1.00  0.00           C  \nATOM    608  CD  GLN A  39      24.978 -14.228   1.541  1.00  0.00           C  \nATOM    609  OE1 GLN A  39      24.194 -15.150   1.700  1.00  0.00           O  \nATOM    610  NE2 GLN A  39      26.271 -14.291   1.842  1.00  0.00           N  \nATOM    611  H   GLN A  39      22.347 -10.159   2.737  1.00  0.00           H  \nATOM    612  HA  GLN A  39      22.055 -12.035   0.910  1.00  0.00           H  \nATOM    613  HB2 GLN A  39      24.205 -10.978   1.928  1.00  0.00           H  \nATOM    614  HB3 GLN A  39      24.212 -12.341   3.029  1.00  0.00           H  \nATOM    615  HG2 GLN A  39      23.836 -13.041   0.148  1.00  0.00           H  \nATOM    616  HG3 GLN A  39      25.400 -12.359   0.550  1.00  0.00           H  \nATOM    617 HE21 GLN A  39      26.860 -13.498   1.687  1.00  0.00           H  \nATOM    618 HE22 GLN A  39      26.655 -15.132   2.224  1.00  0.00           H  \nATOM    619  N   LEU A  40      21.720 -13.367   3.899  1.00  0.00           N  \nATOM    620  CA  LEU A  40      21.239 -14.530   4.625  1.00  0.00           C  \nATOM    621  C   LEU A  40      19.768 -14.769   4.279  1.00  0.00           C  \nATOM    622  O   LEU A  40      19.404 -15.848   3.814  1.00  0.00           O  \nATOM    623  CB  LEU A  40      21.500 -14.371   6.125  1.00  0.00           C  \nATOM    624  CG  LEU A  40      20.853 -15.419   7.033  1.00  0.00           C  \nATOM    625  CD1 LEU A  40      21.701 -16.690   7.094  1.00  0.00           C  \nATOM    626  CD2 LEU A  40      20.580 -14.844   8.425  1.00  0.00           C  \nATOM    627  H   LEU A  40      21.947 -12.577   4.469  1.00  0.00           H  \nATOM    628  HA  LEU A  40      21.818 -15.389   4.287  1.00  0.00           H  \nATOM    629  HB2 LEU A  40      22.577 -14.391   6.290  1.00  0.00           H  \nATOM    630  HB3 LEU A  40      21.149 -13.386   6.432  1.00  0.00           H  \nATOM    631  HG  LEU A  40      19.889 -15.694   6.604  1.00  0.00           H  \nATOM    632 HD11 LEU A  40      21.057 -17.562   6.974  1.00  0.00           H  \nATOM    633 HD12 LEU A  40      22.440 -16.673   6.293  1.00  0.00           H  \nATOM    634 HD13 LEU A  40      22.209 -16.743   8.057  1.00  0.00           H  \nATOM    635 HD21 LEU A  40      20.069 -13.886   8.329  1.00  0.00           H  \nATOM    636 HD22 LEU A  40      19.953 -15.536   8.987  1.00  0.00           H  \nATOM    637 HD23 LEU A  40      21.524 -14.701   8.950  1.00  0.00           H  \nATOM    638  N   ILE A  41      18.962 -13.745   4.520  1.00  0.00           N  \nATOM    639  CA  ILE A  41      17.539 -13.831   4.239  1.00  0.00           C  \nATOM    640  C   ILE A  41      17.335 -14.294   2.795  1.00  0.00           C  \nATOM    641  O   ILE A  41      16.740 -15.343   2.554  1.00  0.00           O  \nATOM    642  CB  ILE A  41      16.849 -12.504   4.562  1.00  0.00           C  \nATOM    643  CG1 ILE A  41      16.821 -12.254   6.072  1.00  0.00           C  \nATOM    644  CG2 ILE A  41      15.449 -12.450   3.946  1.00  0.00           C  \nATOM    645  CD1 ILE A  41      16.535 -10.783   6.378  1.00  0.00           C  \nATOM    646  H   ILE A  41      19.267 -12.871   4.898  1.00  0.00           H  \nATOM    647  HA  ILE A  41      17.119 -14.584   4.906  1.00  0.00           H  \nATOM    648  HB  ILE A  41      17.429 -11.699   4.113  1.00  0.00           H  \nATOM    649 HG12 ILE A  41      16.058 -12.881   6.533  1.00  0.00           H  \nATOM    650 HG13 ILE A  41      17.777 -12.542   6.509  1.00  0.00           H  \nATOM    651 HG21 ILE A  41      15.521 -12.593   2.868  1.00  0.00           H  \nATOM    652 HG22 ILE A  41      14.832 -13.239   4.377  1.00  0.00           H  \nATOM    653 HG23 ILE A  41      14.998 -11.481   4.155  1.00  0.00           H  \nATOM    654 HD11 ILE A  41      15.590 -10.493   5.919  1.00  0.00           H  \nATOM    655 HD12 ILE A  41      16.473 -10.642   7.457  1.00  0.00           H  \nATOM    656 HD13 ILE A  41      17.339 -10.165   5.977  1.00  0.00           H  \nATOM    657  N   PHE A  42      17.841 -13.490   1.872  1.00  0.00           N  \nATOM    658  CA  PHE A  42      17.722 -13.804   0.458  1.00  0.00           C  \nATOM    659  C   PHE A  42      17.999 -15.287   0.201  1.00  0.00           C  \nATOM    660  O   PHE A  42      17.144 -16.000  -0.322  1.00  0.00           O  \nATOM    661  CB  PHE A  42      18.771 -12.967  -0.276  1.00  0.00           C  \nATOM    662  CG  PHE A  42      18.556 -12.886  -1.788  1.00  0.00           C  \nATOM    663  CD1 PHE A  42      19.063 -13.852  -2.600  1.00  0.00           C  \nATOM    664  CD2 PHE A  42      17.856 -11.849  -2.321  1.00  0.00           C  \nATOM    665  CE1 PHE A  42      18.863 -13.777  -4.004  1.00  0.00           C  \nATOM    666  CE2 PHE A  42      17.656 -11.774  -3.725  1.00  0.00           C  \nATOM    667  CZ  PHE A  42      18.163 -12.739  -4.537  1.00  0.00           C  \nATOM    668  H   PHE A  42      18.324 -12.638   2.076  1.00  0.00           H  \nATOM    669  HA  PHE A  42      16.700 -13.570   0.160  1.00  0.00           H  \nATOM    670  HB2 PHE A  42      18.769 -11.958   0.136  1.00  0.00           H  \nATOM    671  HB3 PHE A  42      19.758 -13.388  -0.082  1.00  0.00           H  \nATOM    672  HD1 PHE A  42      19.624 -14.683  -2.173  1.00  0.00           H  \nATOM    673  HD2 PHE A  42      17.449 -11.075  -1.670  1.00  0.00           H  \nATOM    674  HE1 PHE A  42      19.270 -14.551  -4.655  1.00  0.00           H  \nATOM    675  HE2 PHE A  42      17.095 -10.942  -4.152  1.00  0.00           H  \nATOM    676  HZ  PHE A  42      18.010 -12.682  -5.614  1.00  0.00           H  \nATOM    677  N   LYS A  43      19.197 -15.707   0.581  1.00  0.00           N  \nATOM    678  CA  LYS A  43      19.597 -17.092   0.398  1.00  0.00           C  \nATOM    679  C   LYS A  43      18.562 -18.008   1.055  1.00  0.00           C  \nATOM    680  O   LYS A  43      18.356 -19.136   0.611  1.00  0.00           O  \nATOM    681  CB  LYS A  43      21.023 -17.309   0.908  1.00  0.00           C  \nATOM    682  CG  LYS A  43      21.798 -18.251  -0.017  1.00  0.00           C  \nATOM    683  CD  LYS A  43      22.675 -19.211   0.789  1.00  0.00           C  \nATOM    684  CE  LYS A  43      21.989 -20.569   0.958  1.00  0.00           C  \nATOM    685  NZ  LYS A  43      21.297 -20.641   2.264  1.00  0.00           N  \nATOM    686  H   LYS A  43      19.886 -15.120   1.006  1.00  0.00           H  \nATOM    687  HA  LYS A  43      19.603 -17.291  -0.673  1.00  0.00           H  \nATOM    688  HB2 LYS A  43      21.540 -16.352   0.974  1.00  0.00           H  \nATOM    689  HB3 LYS A  43      20.994 -17.725   1.915  1.00  0.00           H  \nATOM    690  HG2 LYS A  43      21.099 -18.819  -0.631  1.00  0.00           H  \nATOM    691  HG3 LYS A  43      22.420 -17.668  -0.696  1.00  0.00           H  \nATOM    692  HD2 LYS A  43      23.633 -19.344   0.286  1.00  0.00           H  \nATOM    693  HD3 LYS A  43      22.886 -18.782   1.768  1.00  0.00           H  \nATOM    694  HE2 LYS A  43      21.273 -20.724   0.151  1.00  0.00           H  \nATOM    695  HE3 LYS A  43      22.728 -21.367   0.886  1.00  0.00           H  \nATOM    696  HZ1 LYS A  43      20.440 -20.128   2.216  1.00  0.00           H  \nATOM    697  HZ2 LYS A  43      21.102 -21.596   2.485  1.00  0.00           H  \nATOM    698  HZ3 LYS A  43      21.882 -20.247   2.973  1.00  0.00           H  \nATOM    699  N   SER A  44      17.940 -17.488   2.103  1.00  0.00           N  \nATOM    700  CA  SER A  44      16.932 -18.245   2.826  1.00  0.00           C  \nATOM    701  C   SER A  44      15.714 -18.483   1.931  1.00  0.00           C  \nATOM    702  O   SER A  44      15.289 -19.622   1.745  1.00  0.00           O  \nATOM    703  CB  SER A  44      16.515 -17.523   4.109  1.00  0.00           C  \nATOM    704  OG  SER A  44      16.261 -18.433   5.175  1.00  0.00           O  \nATOM    705  H   SER A  44      18.113 -16.570   2.458  1.00  0.00           H  \nATOM    706  HA  SER A  44      17.410 -19.191   3.081  1.00  0.00           H  \nATOM    707  HB2 SER A  44      17.301 -16.828   4.405  1.00  0.00           H  \nATOM    708  HB3 SER A  44      15.621 -16.930   3.917  1.00  0.00           H  \nATOM    709  HG  SER A  44      16.246 -17.943   6.047  1.00  0.00           H  \nATOM    710  N   ILE A  45      15.187 -17.390   1.401  1.00  0.00           N  \nATOM    711  CA  ILE A  45      14.026 -17.465   0.530  1.00  0.00           C  \nATOM    712  C   ILE A  45      14.433 -18.099  -0.801  1.00  0.00           C  \nATOM    713  O   ILE A  45      13.704 -18.926  -1.348  1.00  0.00           O  \nATOM    714  CB  ILE A  45      13.377 -16.087   0.382  1.00  0.00           C  \nATOM    715  CG1 ILE A  45      11.921 -16.213  -0.072  1.00  0.00           C  \nATOM    716  CG2 ILE A  45      14.194 -15.194  -0.554  1.00  0.00           C  \nATOM    717  CD1 ILE A  45      10.986 -16.379   1.127  1.00  0.00           C  \nATOM    718  H   ILE A  45      15.538 -16.467   1.558  1.00  0.00           H  \nATOM    719  HA  ILE A  45      13.296 -18.114   1.013  1.00  0.00           H  \nATOM    720  HB  ILE A  45      13.369 -15.606   1.360  1.00  0.00           H  \nATOM    721 HG12 ILE A  45      11.636 -15.328  -0.641  1.00  0.00           H  \nATOM    722 HG13 ILE A  45      11.817 -17.068  -0.740  1.00  0.00           H  \nATOM    723 HG21 ILE A  45      13.626 -14.293  -0.785  1.00  0.00           H  \nATOM    724 HG22 ILE A  45      15.130 -14.919  -0.068  1.00  0.00           H  \nATOM    725 HG23 ILE A  45      14.408 -15.734  -1.476  1.00  0.00           H  \nATOM    726 HD11 ILE A  45      11.350 -17.185   1.763  1.00  0.00           H  \nATOM    727 HD12 ILE A  45      10.958 -15.450   1.698  1.00  0.00           H  \nATOM    728 HD13 ILE A  45       9.982 -16.618   0.775  1.00  0.00           H  \nATOM    729  N   ASP A  46      15.596 -17.688  -1.285  1.00  0.00           N  \nATOM    730  CA  ASP A  46      16.109 -18.206  -2.542  1.00  0.00           C  \nATOM    731  C   ASP A  46      16.507 -19.672  -2.360  1.00  0.00           C  \nATOM    732  O   ASP A  46      17.520 -19.970  -1.730  1.00  0.00           O  \nATOM    733  CB  ASP A  46      17.349 -17.432  -2.992  1.00  0.00           C  \nATOM    734  CG  ASP A  46      17.632 -17.480  -4.495  1.00  0.00           C  \nATOM    735  OD1 ASP A  46      16.830 -18.126  -5.203  1.00  0.00           O  \nATOM    736  OD2 ASP A  46      18.645 -16.871  -4.901  1.00  0.00           O  \nATOM    737  H   ASP A  46      16.183 -17.015  -0.835  1.00  0.00           H  \nATOM    738  HA  ASP A  46      15.295 -18.080  -3.257  1.00  0.00           H  \nATOM    739  HB2 ASP A  46      17.237 -16.390  -2.692  1.00  0.00           H  \nATOM    740  HB3 ASP A  46      18.217 -17.825  -2.461  1.00  0.00           H  \nATOM    741  N   ALA A  47      15.688 -20.549  -2.924  1.00  0.00           N  \nATOM    742  CA  ALA A  47      15.942 -21.976  -2.832  1.00  0.00           C  \nATOM    743  C   ALA A  47      16.901 -22.393  -3.949  1.00  0.00           C  \nATOM    744  O   ALA A  47      17.834 -23.159  -3.718  1.00  0.00           O  \nATOM    745  CB  ALA A  47      14.615 -22.736  -2.889  1.00  0.00           C  \nATOM    746  H   ALA A  47      14.866 -20.298  -3.435  1.00  0.00           H  \nATOM    747  HA  ALA A  47      16.415 -22.168  -1.869  1.00  0.00           H  \nATOM    748  HB1 ALA A  47      14.730 -23.704  -2.401  1.00  0.00           H  \nATOM    749  HB2 ALA A  47      13.845 -22.160  -2.377  1.00  0.00           H  \nATOM    750  HB3 ALA A  47      14.326 -22.885  -3.929  1.00  0.00           H  \nATOM    751  N   ASP A  48      16.637 -21.868  -5.137  1.00  0.00           N  \nATOM    752  CA  ASP A  48      17.465 -22.176  -6.291  1.00  0.00           C  \nATOM    753  C   ASP A  48      18.838 -21.523  -6.118  1.00  0.00           C  \nATOM    754  O   ASP A  48      19.853 -22.213  -6.050  1.00  0.00           O  \nATOM    755  CB  ASP A  48      16.841 -21.630  -7.577  1.00  0.00           C  \nATOM    756  CG  ASP A  48      16.124 -20.287  -7.429  1.00  0.00           C  \nATOM    757  OD1 ASP A  48      16.835 -19.259  -7.458  1.00  0.00           O  \nATOM    758  OD2 ASP A  48      14.883 -20.318  -7.291  1.00  0.00           O  \nATOM    759  H   ASP A  48      15.876 -21.245  -5.317  1.00  0.00           H  \nATOM    760  HA  ASP A  48      17.521 -23.264  -6.321  1.00  0.00           H  \nATOM    761  HB2 ASP A  48      17.625 -21.526  -8.328  1.00  0.00           H  \nATOM    762  HB3 ASP A  48      16.131 -22.364  -7.959  1.00  0.00           H  \nATOM    763  N   GLY A  49      18.824 -20.200  -6.050  1.00  0.00           N  \nATOM    764  CA  GLY A  49      20.055 -19.446  -5.885  1.00  0.00           C  \nATOM    765  C   GLY A  49      20.440 -18.734  -7.184  1.00  0.00           C  \nATOM    766  O   GLY A  49      21.619 -18.651  -7.525  1.00  0.00           O  \nATOM    767  H   GLY A  49      17.993 -19.646  -6.106  1.00  0.00           H  \nATOM    768  HA2 GLY A  49      19.933 -18.714  -5.088  1.00  0.00           H  \nATOM    769  HA3 GLY A  49      20.859 -20.117  -5.582  1.00  0.00           H  \nATOM    770  N   ASN A  50      19.423 -18.239  -7.874  1.00  0.00           N  \nATOM    771  CA  ASN A  50      19.640 -17.537  -9.128  1.00  0.00           C  \nATOM    772  C   ASN A  50      20.207 -16.146  -8.837  1.00  0.00           C  \nATOM    773  O   ASN A  50      20.646 -15.448  -9.749  1.00  0.00           O  \nATOM    774  CB  ASN A  50      18.328 -17.363  -9.895  1.00  0.00           C  \nATOM    775  CG  ASN A  50      17.206 -16.899  -8.965  1.00  0.00           C  \nATOM    776  OD1 ASN A  50      17.346 -16.858  -7.754  1.00  0.00           O  \nATOM    777  ND2 ASN A  50      16.088 -16.553  -9.596  1.00  0.00           N  \nATOM    778  H   ASN A  50      18.467 -18.311  -7.590  1.00  0.00           H  \nATOM    779  HA  ASN A  50      20.335 -18.162  -9.689  1.00  0.00           H  \nATOM    780  HB2 ASN A  50      18.465 -16.636 -10.696  1.00  0.00           H  \nATOM    781  HB3 ASN A  50      18.049 -18.306 -10.365  1.00  0.00           H  \nATOM    782 HD21 ASN A  50      16.039 -16.610 -10.594  1.00  0.00           H  \nATOM    783 HD22 ASN A  50      15.296 -16.236  -9.074  1.00  0.00           H  \nATOM    784  N   GLY A  51      20.178 -15.785  -7.562  1.00  0.00           N  \nATOM    785  CA  GLY A  51      20.683 -14.490  -7.140  1.00  0.00           C  \nATOM    786  C   GLY A  51      19.612 -13.407  -7.292  1.00  0.00           C  \nATOM    787  O   GLY A  51      19.791 -12.282  -6.829  1.00  0.00           O  \nATOM    788  H   GLY A  51      19.818 -16.359  -6.827  1.00  0.00           H  \nATOM    789  HA2 GLY A  51      21.006 -14.543  -6.101  1.00  0.00           H  \nATOM    790  HA3 GLY A  51      21.558 -14.226  -7.733  1.00  0.00           H  \nATOM    791  N   GLU A  52      18.522 -13.786  -7.943  1.00  0.00           N  \nATOM    792  CA  GLU A  52      17.422 -12.862  -8.162  1.00  0.00           C  \nATOM    793  C   GLU A  52      16.107 -13.479  -7.682  1.00  0.00           C  \nATOM    794  O   GLU A  52      15.945 -14.699  -7.703  1.00  0.00           O  \nATOM    795  CB  GLU A  52      17.332 -12.455  -9.634  1.00  0.00           C  \nATOM    796  CG  GLU A  52      18.567 -11.657 -10.058  1.00  0.00           C  \nATOM    797  CD  GLU A  52      19.340 -12.387 -11.158  1.00  0.00           C  \nATOM    798  OE1 GLU A  52      18.679 -12.807 -12.133  1.00  0.00           O  \nATOM    799  OE2 GLU A  52      20.574 -12.508 -11.001  1.00  0.00           O  \nATOM    800  H   GLU A  52      18.384 -14.703  -8.316  1.00  0.00           H  \nATOM    801  HA  GLU A  52      17.658 -11.983  -7.562  1.00  0.00           H  \nATOM    802  HB2 GLU A  52      17.239 -13.345 -10.256  1.00  0.00           H  \nATOM    803  HB3 GLU A  52      16.435 -11.857  -9.796  1.00  0.00           H  \nATOM    804  HG2 GLU A  52      18.264 -10.672 -10.413  1.00  0.00           H  \nATOM    805  HG3 GLU A  52      19.216 -11.499  -9.196  1.00  0.00           H  \nATOM    806  N   ILE A  53      15.200 -12.609  -7.263  1.00  0.00           N  \nATOM    807  CA  ILE A  53      13.903 -13.054  -6.780  1.00  0.00           C  \nATOM    808  C   ILE A  53      12.802 -12.255  -7.479  1.00  0.00           C  \nATOM    809  O   ILE A  53      13.087 -11.386  -8.301  1.00  0.00           O  \nATOM    810  CB  ILE A  53      13.845 -12.975  -5.253  1.00  0.00           C  \nATOM    811  CG1 ILE A  53      14.541 -11.712  -4.742  1.00  0.00           C  \nATOM    812  CG2 ILE A  53      14.419 -14.243  -4.616  1.00  0.00           C  \nATOM    813  CD1 ILE A  53      13.730 -10.462  -5.086  1.00  0.00           C  \nATOM    814  H   ILE A  53      15.339 -11.619  -7.250  1.00  0.00           H  \nATOM    815  HA  ILE A  53      13.795 -14.103  -7.053  1.00  0.00           H  \nATOM    816  HB  ILE A  53      12.799 -12.910  -4.954  1.00  0.00           H  \nATOM    817 HG12 ILE A  53      14.674 -11.778  -3.662  1.00  0.00           H  \nATOM    818 HG13 ILE A  53      15.536 -11.637  -5.181  1.00  0.00           H  \nATOM    819 HG21 ILE A  53      15.501 -14.146  -4.526  1.00  0.00           H  \nATOM    820 HG22 ILE A  53      13.982 -14.383  -3.627  1.00  0.00           H  \nATOM    821 HG23 ILE A  53      14.182 -15.103  -5.243  1.00  0.00           H  \nATOM    822 HD11 ILE A  53      12.666 -10.684  -5.000  1.00  0.00           H  \nATOM    823 HD12 ILE A  53      13.990  -9.658  -4.397  1.00  0.00           H  \nATOM    824 HD13 ILE A  53      13.955 -10.152  -6.107  1.00  0.00           H  \nATOM    825  N   ASP A  54      11.566 -12.577  -7.126  1.00  0.00           N  \nATOM    826  CA  ASP A  54      10.420 -11.900  -7.709  1.00  0.00           C  \nATOM    827  C   ASP A  54       9.589 -11.261  -6.595  1.00  0.00           C  \nATOM    828  O   ASP A  54       9.969 -11.311  -5.426  1.00  0.00           O  \nATOM    829  CB  ASP A  54       9.524 -12.885  -8.463  1.00  0.00           C  \nATOM    830  CG  ASP A  54       9.208 -14.179  -7.710  1.00  0.00           C  \nATOM    831  OD1 ASP A  54       9.383 -14.172  -6.472  1.00  0.00           O  \nATOM    832  OD2 ASP A  54       8.799 -15.145  -8.389  1.00  0.00           O  \nATOM    833  H   ASP A  54      11.343 -13.285  -6.456  1.00  0.00           H  \nATOM    834  HA  ASP A  54      10.841 -11.161  -8.391  1.00  0.00           H  \nATOM    835  HB2 ASP A  54       8.586 -12.385  -8.706  1.00  0.00           H  \nATOM    836  HB3 ASP A  54      10.004 -13.139  -9.408  1.00  0.00           H  \nATOM    837  N   GLN A  55       8.470 -10.677  -6.996  1.00  0.00           N  \nATOM    838  CA  GLN A  55       7.582 -10.029  -6.046  1.00  0.00           C  \nATOM    839  C   GLN A  55       7.135 -11.024  -4.973  1.00  0.00           C  \nATOM    840  O   GLN A  55       6.968 -10.657  -3.811  1.00  0.00           O  \nATOM    841  CB  GLN A  55       6.376  -9.411  -6.756  1.00  0.00           C  \nATOM    842  CG  GLN A  55       5.485 -10.494  -7.367  1.00  0.00           C  \nATOM    843  CD  GLN A  55       4.606  -9.918  -8.479  1.00  0.00           C  \nATOM    844  OE1 GLN A  55       4.846 -10.115  -9.659  1.00  0.00           O  \nATOM    845  NE2 GLN A  55       3.579  -9.197  -8.039  1.00  0.00           N  \nATOM    846  H   GLN A  55       8.168 -10.641  -7.949  1.00  0.00           H  \nATOM    847  HA  GLN A  55       8.174  -9.235  -5.590  1.00  0.00           H  \nATOM    848  HB2 GLN A  55       5.799  -8.815  -6.048  1.00  0.00           H  \nATOM    849  HB3 GLN A  55       6.718  -8.733  -7.538  1.00  0.00           H  \nATOM    850  HG2 GLN A  55       6.104 -11.297  -7.766  1.00  0.00           H  \nATOM    851  HG3 GLN A  55       4.855 -10.932  -6.592  1.00  0.00           H  \nATOM    852 HE21 GLN A  55       3.439  -9.073  -7.057  1.00  0.00           H  \nATOM    853 HE22 GLN A  55       2.947  -8.778  -8.691  1.00  0.00           H  \nATOM    854  N   ASN A  56       6.954 -12.265  -5.401  1.00  0.00           N  \nATOM    855  CA  ASN A  56       6.530 -13.316  -4.492  1.00  0.00           C  \nATOM    856  C   ASN A  56       7.597 -13.512  -3.413  1.00  0.00           C  \nATOM    857  O   ASN A  56       7.322 -13.346  -2.226  1.00  0.00           O  \nATOM    858  CB  ASN A  56       6.354 -14.645  -5.230  1.00  0.00           C  \nATOM    859  CG  ASN A  56       4.947 -14.762  -5.820  1.00  0.00           C  \nATOM    860  OD1 ASN A  56       3.946 -14.621  -5.138  1.00  0.00           O  \nATOM    861  ND2 ASN A  56       4.928 -15.027  -7.123  1.00  0.00           N  \nATOM    862  H   ASN A  56       7.092 -12.556  -6.348  1.00  0.00           H  \nATOM    863  HA  ASN A  56       5.580 -12.974  -4.082  1.00  0.00           H  \nATOM    864  HB2 ASN A  56       7.094 -14.724  -6.026  1.00  0.00           H  \nATOM    865  HB3 ASN A  56       6.534 -15.473  -4.544  1.00  0.00           H  \nATOM    866 HD21 ASN A  56       5.787 -15.131  -7.625  1.00  0.00           H  \nATOM    867 HD22 ASN A  56       4.055 -15.123  -7.601  1.00  0.00           H  \nATOM    868  N   GLU A  57       8.792 -13.863  -3.865  1.00  0.00           N  \nATOM    869  CA  GLU A  57       9.902 -14.084  -2.953  1.00  0.00           C  \nATOM    870  C   GLU A  57      10.189 -12.814  -2.149  1.00  0.00           C  \nATOM    871  O   GLU A  57      10.641 -12.887  -1.007  1.00  0.00           O  \nATOM    872  CB  GLU A  57      11.149 -14.548  -3.708  1.00  0.00           C  \nATOM    873  CG  GLU A  57      11.152 -16.068  -3.880  1.00  0.00           C  \nATOM    874  CD  GLU A  57      12.298 -16.515  -4.790  1.00  0.00           C  \nATOM    875  OE1 GLU A  57      12.218 -16.202  -5.998  1.00  0.00           O  \nATOM    876  OE2 GLU A  57      13.228 -17.159  -4.257  1.00  0.00           O  \nATOM    877  H   GLU A  57       9.008 -13.996  -4.832  1.00  0.00           H  \nATOM    878  HA  GLU A  57       9.573 -14.879  -2.284  1.00  0.00           H  \nATOM    879  HB2 GLU A  57      11.185 -14.068  -4.686  1.00  0.00           H  \nATOM    880  HB3 GLU A  57      12.043 -14.237  -3.167  1.00  0.00           H  \nATOM    881  HG2 GLU A  57      11.248 -16.547  -2.906  1.00  0.00           H  \nATOM    882  HG3 GLU A  57      10.200 -16.392  -4.302  1.00  0.00           H  \nATOM    883  N   PHE A  58       9.913 -11.680  -2.776  1.00  0.00           N  \nATOM    884  CA  PHE A  58      10.136 -10.396  -2.133  1.00  0.00           C  \nATOM    885  C   PHE A  58       9.124 -10.162  -1.010  1.00  0.00           C  \nATOM    886  O   PHE A  58       9.477  -9.651   0.052  1.00  0.00           O  \nATOM    887  CB  PHE A  58       9.948  -9.321  -3.205  1.00  0.00           C  \nATOM    888  CG  PHE A  58      10.437  -7.932  -2.787  1.00  0.00           C  \nATOM    889  CD1 PHE A  58       9.621  -7.110  -2.074  1.00  0.00           C  \nATOM    890  CD2 PHE A  58      11.687  -7.521  -3.128  1.00  0.00           C  \nATOM    891  CE1 PHE A  58      10.075  -5.822  -1.686  1.00  0.00           C  \nATOM    892  CE2 PHE A  58      12.141  -6.232  -2.739  1.00  0.00           C  \nATOM    893  CZ  PHE A  58      11.325  -5.410  -2.027  1.00  0.00           C  \nATOM    894  H   PHE A  58       9.545 -11.630  -3.705  1.00  0.00           H  \nATOM    895  HA  PHE A  58      11.142 -10.414  -1.715  1.00  0.00           H  \nATOM    896  HB2 PHE A  58      10.478  -9.625  -4.108  1.00  0.00           H  \nATOM    897  HB3 PHE A  58       8.890  -9.260  -3.462  1.00  0.00           H  \nATOM    898  HD1 PHE A  58       8.618  -7.440  -1.801  1.00  0.00           H  \nATOM    899  HD2 PHE A  58      12.341  -8.180  -3.699  1.00  0.00           H  \nATOM    900  HE1 PHE A  58       9.421  -5.163  -1.114  1.00  0.00           H  \nATOM    901  HE2 PHE A  58      13.143  -5.903  -3.013  1.00  0.00           H  \nATOM    902  HZ  PHE A  58      11.674  -4.421  -1.729  1.00  0.00           H  \nATOM    903  N   ALA A  59       7.886 -10.548  -1.282  1.00  0.00           N  \nATOM    904  CA  ALA A  59       6.821 -10.387  -0.308  1.00  0.00           C  \nATOM    905  C   ALA A  59       7.112 -11.266   0.910  1.00  0.00           C  \nATOM    906  O   ALA A  59       7.009 -10.811   2.048  1.00  0.00           O  \nATOM    907  CB  ALA A  59       5.477 -10.721  -0.960  1.00  0.00           C  \nATOM    908  H   ALA A  59       7.607 -10.963  -2.148  1.00  0.00           H  \nATOM    909  HA  ALA A  59       6.810  -9.342   0.003  1.00  0.00           H  \nATOM    910  HB1 ALA A  59       5.174  -9.899  -1.608  1.00  0.00           H  \nATOM    911  HB2 ALA A  59       5.576 -11.632  -1.550  1.00  0.00           H  \nATOM    912  HB3 ALA A  59       4.725 -10.870  -0.185  1.00  0.00           H  \nATOM    913  N   LYS A  60       7.470 -12.510   0.630  1.00  0.00           N  \nATOM    914  CA  LYS A  60       7.777 -13.457   1.688  1.00  0.00           C  \nATOM    915  C   LYS A  60       9.091 -13.055   2.361  1.00  0.00           C  \nATOM    916  O   LYS A  60       9.287 -13.308   3.549  1.00  0.00           O  \nATOM    917  CB  LYS A  60       7.778 -14.887   1.144  1.00  0.00           C  \nATOM    918  CG  LYS A  60       6.382 -15.508   1.229  1.00  0.00           C  \nATOM    919  CD  LYS A  60       6.448 -16.933   1.781  1.00  0.00           C  \nATOM    920  CE  LYS A  60       5.191 -17.268   2.585  1.00  0.00           C  \nATOM    921  NZ  LYS A  60       4.729 -18.640   2.277  1.00  0.00           N  \nATOM    922  H   LYS A  60       7.552 -12.873  -0.299  1.00  0.00           H  \nATOM    923  HA  LYS A  60       6.977 -13.394   2.426  1.00  0.00           H  \nATOM    924  HB2 LYS A  60       8.117 -14.886   0.108  1.00  0.00           H  \nATOM    925  HB3 LYS A  60       8.485 -15.495   1.709  1.00  0.00           H  \nATOM    926  HG2 LYS A  60       5.747 -14.895   1.869  1.00  0.00           H  \nATOM    927  HG3 LYS A  60       5.924 -15.518   0.240  1.00  0.00           H  \nATOM    928  HD2 LYS A  60       6.558 -17.640   0.959  1.00  0.00           H  \nATOM    929  HD3 LYS A  60       7.329 -17.041   2.415  1.00  0.00           H  \nATOM    930  HE2 LYS A  60       5.399 -17.179   3.652  1.00  0.00           H  \nATOM    931  HE3 LYS A  60       4.402 -16.552   2.354  1.00  0.00           H  \nATOM    932  HZ1 LYS A  60       3.915 -18.848   2.820  1.00  0.00           H  \nATOM    933  HZ2 LYS A  60       4.507 -18.707   1.305  1.00  0.00           H  \nATOM    934  HZ3 LYS A  60       5.453 -19.293   2.500  1.00  0.00           H  \nATOM    935  N   PHE A  61       9.957 -12.435   1.574  1.00  0.00           N  \nATOM    936  CA  PHE A  61      11.246 -11.995   2.079  1.00  0.00           C  \nATOM    937  C   PHE A  61      11.084 -10.821   3.047  1.00  0.00           C  \nATOM    938  O   PHE A  61      11.816 -10.717   4.030  1.00  0.00           O  \nATOM    939  CB  PHE A  61      12.067 -11.535   0.872  1.00  0.00           C  \nATOM    940  CG  PHE A  61      13.270 -10.662   1.233  1.00  0.00           C  \nATOM    941  CD1 PHE A  61      13.092  -9.347   1.530  1.00  0.00           C  \nATOM    942  CD2 PHE A  61      14.518 -11.201   1.257  1.00  0.00           C  \nATOM    943  CE1 PHE A  61      14.208  -8.537   1.866  1.00  0.00           C  \nATOM    944  CE2 PHE A  61      15.635 -10.391   1.592  1.00  0.00           C  \nATOM    945  CZ  PHE A  61      15.457  -9.076   1.890  1.00  0.00           C  \nATOM    946  H   PHE A  61       9.790 -12.233   0.609  1.00  0.00           H  \nATOM    947  HA  PHE A  61      11.693 -12.839   2.605  1.00  0.00           H  \nATOM    948  HB2 PHE A  61      12.417 -12.412   0.328  1.00  0.00           H  \nATOM    949  HB3 PHE A  61      11.418 -10.979   0.195  1.00  0.00           H  \nATOM    950  HD1 PHE A  61      12.091  -8.915   1.511  1.00  0.00           H  \nATOM    951  HD2 PHE A  61      14.661 -12.255   1.019  1.00  0.00           H  \nATOM    952  HE1 PHE A  61      14.065  -7.483   2.104  1.00  0.00           H  \nATOM    953  HE2 PHE A  61      16.636 -10.823   1.612  1.00  0.00           H  \nATOM    954  HZ  PHE A  61      16.314  -8.454   2.147  1.00  0.00           H  \nATOM    955  N   TYR A  62      10.121  -9.967   2.735  1.00  0.00           N  \nATOM    956  CA  TYR A  62       9.853  -8.805   3.565  1.00  0.00           C  \nATOM    957  C   TYR A  62       9.609  -9.214   5.019  1.00  0.00           C  \nATOM    958  O   TYR A  62      10.095  -8.562   5.942  1.00  0.00           O  \nATOM    959  CB  TYR A  62       8.579  -8.171   3.005  1.00  0.00           C  \nATOM    960  CG  TYR A  62       8.508  -6.653   3.185  1.00  0.00           C  \nATOM    961  CD1 TYR A  62       8.370  -6.114   4.448  1.00  0.00           C  \nATOM    962  CD2 TYR A  62       8.581  -5.823   2.084  1.00  0.00           C  \nATOM    963  CE1 TYR A  62       8.304  -4.685   4.617  1.00  0.00           C  \nATOM    964  CE2 TYR A  62       8.514  -4.395   2.254  1.00  0.00           C  \nATOM    965  CZ  TYR A  62       8.378  -3.896   3.512  1.00  0.00           C  \nATOM    966  OH  TYR A  62       8.315  -2.547   3.672  1.00  0.00           O  \nATOM    967  H   TYR A  62       9.530 -10.059   1.934  1.00  0.00           H  \nATOM    968  HA  TYR A  62      10.725  -8.152   3.521  1.00  0.00           H  \nATOM    969  HB2 TYR A  62       8.505  -8.404   1.943  1.00  0.00           H  \nATOM    970  HB3 TYR A  62       7.715  -8.625   3.491  1.00  0.00           H  \nATOM    971  HD1 TYR A  62       8.313  -6.769   5.317  1.00  0.00           H  \nATOM    972  HD2 TYR A  62       8.689  -6.250   1.087  1.00  0.00           H  \nATOM    973  HE1 TYR A  62       8.195  -4.246   5.609  1.00  0.00           H  \nATOM    974  HE2 TYR A  62       8.570  -3.729   1.393  1.00  0.00           H  \nATOM    975  HH  TYR A  62       7.445  -2.202   3.321  1.00  0.00           H  \nATOM    976  N   GLY A  63       8.855 -10.292   5.178  1.00  0.00           N  \nATOM    977  CA  GLY A  63       8.540 -10.796   6.504  1.00  0.00           C  \nATOM    978  C   GLY A  63       9.774 -11.419   7.161  1.00  0.00           C  \nATOM    979  O   GLY A  63      10.390 -10.811   8.035  1.00  0.00           O  \nATOM    980  H   GLY A  63       8.464 -10.817   4.422  1.00  0.00           H  \nATOM    981  HA2 GLY A  63       8.164  -9.984   7.126  1.00  0.00           H  \nATOM    982  HA3 GLY A  63       7.746 -11.539   6.435  1.00  0.00           H  \nATOM    983  N   SER A  64      10.097 -12.624   6.716  1.00  0.00           N  \nATOM    984  CA  SER A  64      11.246 -13.336   7.249  1.00  0.00           C  \nATOM    985  C   SER A  64      11.413 -14.675   6.528  1.00  0.00           C  \nATOM    986  O   SER A  64      12.372 -14.866   5.781  1.00  0.00           O  \nATOM    987  CB  SER A  64      11.104 -13.560   8.756  1.00  0.00           C  \nATOM    988  OG  SER A  64      12.078 -12.831   9.498  1.00  0.00           O  \nATOM    989  H   SER A  64       9.591 -13.112   6.005  1.00  0.00           H  \nATOM    990  HA  SER A  64      12.101 -12.688   7.058  1.00  0.00           H  \nATOM    991  HB2 SER A  64      10.106 -13.258   9.075  1.00  0.00           H  \nATOM    992  HB3 SER A  64      11.201 -14.623   8.976  1.00  0.00           H  \nATOM    993  HG  SER A  64      12.035 -11.861   9.259  1.00  0.00           H  \nATOM    994  N   ILE A  65      10.466 -15.567   6.776  1.00  0.00           N  \nATOM    995  CA  ILE A  65      10.497 -16.882   6.159  1.00  0.00           C  \nATOM    996  C   ILE A  65       9.067 -17.327   5.845  1.00  0.00           C  \nATOM    997  O   ILE A  65       8.631 -17.261   4.696  1.00  0.00           O  \nATOM    998  CB  ILE A  65      11.268 -17.869   7.038  1.00  0.00           C  \nATOM    999  CG1 ILE A  65      12.777 -17.721   6.834  1.00  0.00           C  \nATOM   1000  CG2 ILE A  65      10.795 -19.304   6.798  1.00  0.00           C  \nATOM   1001  CD1 ILE A  65      13.202 -18.270   5.470  1.00  0.00           C  \nATOM   1002  H   ILE A  65       9.690 -15.404   7.384  1.00  0.00           H  \nATOM   1003  HA  ILE A  65      11.044 -16.790   5.221  1.00  0.00           H  \nATOM   1004  HB  ILE A  65      11.060 -17.631   8.081  1.00  0.00           H  \nATOM   1005 HG12 ILE A  65      13.057 -16.670   6.910  1.00  0.00           H  \nATOM   1006 HG13 ILE A  65      13.309 -18.250   7.625  1.00  0.00           H  \nATOM   1007 HG21 ILE A  65      11.574 -20.000   7.109  1.00  0.00           H  \nATOM   1008 HG22 ILE A  65       9.890 -19.489   7.376  1.00  0.00           H  \nATOM   1009 HG23 ILE A  65      10.584 -19.444   5.738  1.00  0.00           H  \nATOM   1010 HD11 ILE A  65      13.445 -17.442   4.805  1.00  0.00           H  \nATOM   1011 HD12 ILE A  65      14.078 -18.907   5.592  1.00  0.00           H  \nATOM   1012 HD13 ILE A  65      12.386 -18.852   5.043  1.00  0.00           H  \nATOM   1013  N   GLN A  66       8.377 -17.770   6.885  1.00  0.00           N  \nATOM   1014  CA  GLN A  66       7.005 -18.225   6.734  1.00  0.00           C  \nATOM   1015  C   GLN A  66       6.196 -17.898   7.991  1.00  0.00           C  \nATOM   1016  O   GLN A  66       5.830 -18.795   8.749  1.00  0.00           O  \nATOM   1017  CB  GLN A  66       6.956 -19.723   6.425  1.00  0.00           C  \nATOM   1018  CG  GLN A  66       5.527 -20.171   6.110  1.00  0.00           C  \nATOM   1019  CD  GLN A  66       5.517 -21.556   5.460  1.00  0.00           C  \nATOM   1020  OE1 GLN A  66       6.254 -22.452   5.837  1.00  0.00           O  \nATOM   1021  NE2 GLN A  66       4.643 -21.680   4.465  1.00  0.00           N  \nATOM   1022  H   GLN A  66       8.739 -17.820   7.816  1.00  0.00           H  \nATOM   1023  HA  GLN A  66       6.608 -17.672   5.884  1.00  0.00           H  \nATOM   1024  HB2 GLN A  66       7.605 -19.945   5.578  1.00  0.00           H  \nATOM   1025  HB3 GLN A  66       7.338 -20.287   7.276  1.00  0.00           H  \nATOM   1026  HG2 GLN A  66       4.938 -20.191   7.027  1.00  0.00           H  \nATOM   1027  HG3 GLN A  66       5.054 -19.449   5.444  1.00  0.00           H  \nATOM   1028 HE21 GLN A  66       4.067 -20.905   4.206  1.00  0.00           H  \nATOM   1029 HE22 GLN A  66       4.562 -22.549   3.977  1.00  0.00           H  \nATOM   1030  N   GLY A  67       5.941 -16.611   8.173  1.00  0.00           N  \nATOM   1031  CA  GLY A  67       5.182 -16.154   9.325  1.00  0.00           C  \nATOM   1032  C   GLY A  67       5.374 -14.652   9.547  1.00  0.00           C  \nATOM   1033  O   GLY A  67       6.492 -14.192   9.771  1.00  0.00           O  \nATOM   1034  H   GLY A  67       6.243 -15.888   7.552  1.00  0.00           H  \nATOM   1035  HA2 GLY A  67       4.124 -16.371   9.177  1.00  0.00           H  \nATOM   1036  HA3 GLY A  67       5.499 -16.700  10.213  1.00  0.00           H  \nATOM   1037  N   GLN A  68       4.265 -13.930   9.478  1.00  0.00           N  \nATOM   1038  CA  GLN A  68       4.297 -12.490   9.669  1.00  0.00           C  \nATOM   1039  C   GLN A  68       2.920 -11.981  10.100  1.00  0.00           C  \nATOM   1040  O   GLN A  68       1.941 -12.135   9.372  1.00  0.00           O  \nATOM   1041  CB  GLN A  68       4.772 -11.779   8.400  1.00  0.00           C  \nATOM   1042  CG  GLN A  68       4.063 -12.334   7.163  1.00  0.00           C  \nATOM   1043  CD  GLN A  68       3.270 -11.238   6.448  1.00  0.00           C  \nATOM   1044  OE1 GLN A  68       3.700 -10.102   6.328  1.00  0.00           O  \nATOM   1045  NE2 GLN A  68       2.092 -11.640   5.981  1.00  0.00           N  \nATOM   1046  H   GLN A  68       3.359 -14.312   9.296  1.00  0.00           H  \nATOM   1047  HA  GLN A  68       5.021 -12.320  10.466  1.00  0.00           H  \nATOM   1048  HB2 GLN A  68       4.580 -10.709   8.485  1.00  0.00           H  \nATOM   1049  HB3 GLN A  68       5.850 -11.901   8.292  1.00  0.00           H  \nATOM   1050  HG2 GLN A  68       4.797 -12.761   6.480  1.00  0.00           H  \nATOM   1051  HG3 GLN A  68       3.392 -13.141   7.456  1.00  0.00           H  \nATOM   1052 HE21 GLN A  68       1.798 -12.587   6.113  1.00  0.00           H  \nATOM   1053 HE22 GLN A  68       1.499 -10.996   5.498  1.00  0.00           H  \nATOM   1054  N   ASP A  69       2.889 -11.385  11.283  1.00  0.00           N  \nATOM   1055  CA  ASP A  69       1.648 -10.853  11.820  1.00  0.00           C  \nATOM   1056  C   ASP A  69       1.451  -9.422  11.314  1.00  0.00           C  \nATOM   1057  O   ASP A  69       0.467  -9.130  10.636  1.00  0.00           O  \nATOM   1058  CB  ASP A  69       1.682 -10.813  13.349  1.00  0.00           C  \nATOM   1059  CG  ASP A  69       0.427 -11.352  14.038  1.00  0.00           C  \nATOM   1060  OD1 ASP A  69       0.177 -12.568  13.891  1.00  0.00           O  \nATOM   1061  OD2 ASP A  69      -0.254 -10.537  14.696  1.00  0.00           O  \nATOM   1062  H   ASP A  69       3.690 -11.264  11.870  1.00  0.00           H  \nATOM   1063  HA  ASP A  69       0.870 -11.530  11.469  1.00  0.00           H  \nATOM   1064  HB2 ASP A  69       2.542 -11.386  13.694  1.00  0.00           H  \nATOM   1065  HB3 ASP A  69       1.838  -9.782  13.667  1.00  0.00           H  \nATOM   1066  N   LEU A  70       2.402  -8.568  11.663  1.00  0.00           N  \nATOM   1067  CA  LEU A  70       2.345  -7.176  11.253  1.00  0.00           C  \nATOM   1068  C   LEU A  70       3.726  -6.540  11.428  1.00  0.00           C  \nATOM   1069  O   LEU A  70       4.057  -6.054  12.508  1.00  0.00           O  \nATOM   1070  CB  LEU A  70       1.231  -6.442  12.002  1.00  0.00           C  \nATOM   1071  CG  LEU A  70      -0.136  -6.416  11.315  1.00  0.00           C  \nATOM   1072  CD1 LEU A  70      -1.205  -7.063  12.197  1.00  0.00           C  \nATOM   1073  CD2 LEU A  70      -0.517  -4.993  10.904  1.00  0.00           C  \nATOM   1074  H   LEU A  70       3.199  -8.814  12.215  1.00  0.00           H  \nATOM   1075  HA  LEU A  70       2.087  -7.157  10.194  1.00  0.00           H  \nATOM   1076  HB2 LEU A  70       1.114  -6.903  12.983  1.00  0.00           H  \nATOM   1077  HB3 LEU A  70       1.550  -5.413  12.170  1.00  0.00           H  \nATOM   1078  HG  LEU A  70      -0.070  -7.008  10.402  1.00  0.00           H  \nATOM   1079 HD11 LEU A  70      -1.042  -8.140  12.235  1.00  0.00           H  \nATOM   1080 HD12 LEU A  70      -1.144  -6.651  13.204  1.00  0.00           H  \nATOM   1081 HD13 LEU A  70      -2.192  -6.860  11.781  1.00  0.00           H  \nATOM   1082 HD21 LEU A  70       0.388  -4.401  10.764  1.00  0.00           H  \nATOM   1083 HD22 LEU A  70      -1.079  -5.022   9.970  1.00  0.00           H  \nATOM   1084 HD23 LEU A  70      -1.130  -4.541  11.683  1.00  0.00           H  \nATOM   1085  N   SER A  71       4.494  -6.565  10.348  1.00  0.00           N  \nATOM   1086  CA  SER A  71       5.831  -5.998  10.369  1.00  0.00           C  \nATOM   1087  C   SER A  71       5.760  -4.502  10.685  1.00  0.00           C  \nATOM   1088  O   SER A  71       6.632  -3.967  11.368  1.00  0.00           O  \nATOM   1089  CB  SER A  71       6.546  -6.224   9.035  1.00  0.00           C  \nATOM   1090  OG  SER A  71       6.526  -7.595   8.645  1.00  0.00           O  \nATOM   1091  H   SER A  71       4.217  -6.963   9.474  1.00  0.00           H  \nATOM   1092  HA  SER A  71       6.359  -6.531  11.159  1.00  0.00           H  \nATOM   1093  HB2 SER A  71       6.071  -5.620   8.262  1.00  0.00           H  \nATOM   1094  HB3 SER A  71       7.579  -5.885   9.115  1.00  0.00           H  \nATOM   1095  HG  SER A  71       6.540  -7.667   7.648  1.00  0.00           H  \nATOM   1096  N   ASP A  72       4.714  -3.871  10.174  1.00  0.00           N  \nATOM   1097  CA  ASP A  72       4.518  -2.448  10.394  1.00  0.00           C  \nATOM   1098  C   ASP A  72       4.426  -2.176  11.896  1.00  0.00           C  \nATOM   1099  O   ASP A  72       4.812  -1.105  12.362  1.00  0.00           O  \nATOM   1100  CB  ASP A  72       3.220  -1.964   9.745  1.00  0.00           C  \nATOM   1101  CG  ASP A  72       3.068  -0.444   9.657  1.00  0.00           C  \nATOM   1102  OD1 ASP A  72       3.809   0.244  10.392  1.00  0.00           O  \nATOM   1103  OD2 ASP A  72       2.214  -0.004   8.857  1.00  0.00           O  \nATOM   1104  H   ASP A  72       4.009  -4.314   9.620  1.00  0.00           H  \nATOM   1105  HA  ASP A  72       5.381  -1.966   9.935  1.00  0.00           H  \nATOM   1106  HB2 ASP A  72       3.157  -2.379   8.739  1.00  0.00           H  \nATOM   1107  HB3 ASP A  72       2.378  -2.366  10.309  1.00  0.00           H  \nATOM   1108  N   ASP A  73       3.911  -3.164  12.613  1.00  0.00           N  \nATOM   1109  CA  ASP A  73       3.763  -3.045  14.054  1.00  0.00           C  \nATOM   1110  C   ASP A  73       5.139  -3.154  14.714  1.00  0.00           C  \nATOM   1111  O   ASP A  73       5.370  -2.575  15.774  1.00  0.00           O  \nATOM   1112  CB  ASP A  73       2.881  -4.165  14.611  1.00  0.00           C  \nATOM   1113  CG  ASP A  73       2.059  -3.786  15.845  1.00  0.00           C  \nATOM   1114  OD1 ASP A  73       1.620  -2.617  15.898  1.00  0.00           O  \nATOM   1115  OD2 ASP A  73       1.889  -4.675  16.708  1.00  0.00           O  \nATOM   1116  H   ASP A  73       3.599  -4.032  12.227  1.00  0.00           H  \nATOM   1117  HA  ASP A  73       3.299  -2.072  14.215  1.00  0.00           H  \nATOM   1118  HB2 ASP A  73       2.200  -4.495  13.827  1.00  0.00           H  \nATOM   1119  HB3 ASP A  73       3.515  -5.015  14.862  1.00  0.00           H  \nATOM   1120  N   LYS A  74       6.017  -3.901  14.061  1.00  0.00           N  \nATOM   1121  CA  LYS A  74       7.363  -4.094  14.571  1.00  0.00           C  \nATOM   1122  C   LYS A  74       8.167  -2.807  14.374  1.00  0.00           C  \nATOM   1123  O   LYS A  74       8.706  -2.255  15.332  1.00  0.00           O  \nATOM   1124  CB  LYS A  74       8.008  -5.325  13.932  1.00  0.00           C  \nATOM   1125  CG  LYS A  74       7.579  -6.605  14.654  1.00  0.00           C  \nATOM   1126  CD  LYS A  74       6.757  -7.506  13.729  1.00  0.00           C  \nATOM   1127  CE  LYS A  74       6.012  -8.577  14.529  1.00  0.00           C  \nATOM   1128  NZ  LYS A  74       4.885  -7.975  15.276  1.00  0.00           N  \nATOM   1129  H   LYS A  74       5.820  -4.369  13.199  1.00  0.00           H  \nATOM   1130  HA  LYS A  74       7.282  -4.290  15.640  1.00  0.00           H  \nATOM   1131  HB2 LYS A  74       7.726  -5.384  12.881  1.00  0.00           H  \nATOM   1132  HB3 LYS A  74       9.093  -5.231  13.966  1.00  0.00           H  \nATOM   1133  HG2 LYS A  74       8.460  -7.143  15.003  1.00  0.00           H  \nATOM   1134  HG3 LYS A  74       6.991  -6.350  15.535  1.00  0.00           H  \nATOM   1135  HD2 LYS A  74       6.043  -6.903  13.168  1.00  0.00           H  \nATOM   1136  HD3 LYS A  74       7.414  -7.982  13.002  1.00  0.00           H  \nATOM   1137  HE2 LYS A  74       5.638  -9.348  13.855  1.00  0.00           H  \nATOM   1138  HE3 LYS A  74       6.697  -9.064  15.222  1.00  0.00           H  \nATOM   1139  HZ1 LYS A  74       5.173  -7.100  15.666  1.00  0.00           H  \nATOM   1140  HZ2 LYS A  74       4.115  -7.825  14.655  1.00  0.00           H  \nATOM   1141  HZ3 LYS A  74       4.605  -8.592  16.011  1.00  0.00           H  \nATOM   1142  N   ILE A  75       8.223  -2.367  13.125  1.00  0.00           N  \nATOM   1143  CA  ILE A  75       8.952  -1.156  12.790  1.00  0.00           C  \nATOM   1144  C   ILE A  75       8.378   0.017  13.588  1.00  0.00           C  \nATOM   1145  O   ILE A  75       9.117   0.906  14.009  1.00  0.00           O  \nATOM   1146  CB  ILE A  75       8.949  -0.930  11.277  1.00  0.00           C  \nATOM   1147  CG1 ILE A  75      10.096  -0.008  10.858  1.00  0.00           C  \nATOM   1148  CG2 ILE A  75       7.591  -0.407  10.804  1.00  0.00           C  \nATOM   1149  CD1 ILE A  75       9.673   1.461  10.930  1.00  0.00           C  \nATOM   1150  H   ILE A  75       7.781  -2.822  12.352  1.00  0.00           H  \nATOM   1151  HA  ILE A  75       9.989  -1.305  13.092  1.00  0.00           H  \nATOM   1152  HB  ILE A  75       9.112  -1.890  10.788  1.00  0.00           H  \nATOM   1153 HG12 ILE A  75      10.956  -0.174  11.506  1.00  0.00           H  \nATOM   1154 HG13 ILE A  75      10.410  -0.250   9.843  1.00  0.00           H  \nATOM   1155 HG21 ILE A  75       6.813  -1.121  11.073  1.00  0.00           H  \nATOM   1156 HG22 ILE A  75       7.384   0.551  11.280  1.00  0.00           H  \nATOM   1157 HG23 ILE A  75       7.608  -0.279   9.721  1.00  0.00           H  \nATOM   1158 HD11 ILE A  75       9.360   1.700  11.946  1.00  0.00           H  \nATOM   1159 HD12 ILE A  75      10.514   2.095  10.650  1.00  0.00           H  \nATOM   1160 HD13 ILE A  75       8.843   1.634  10.244  1.00  0.00           H  \nATOM   1161  N   GLY A  76       7.067  -0.020  13.773  1.00  0.00           N  \nATOM   1162  CA  GLY A  76       6.386   1.029  14.513  1.00  0.00           C  \nATOM   1163  C   GLY A  76       6.695   0.934  16.009  1.00  0.00           C  \nATOM   1164  O   GLY A  76       6.726   1.947  16.705  1.00  0.00           O  \nATOM   1165  H   GLY A  76       6.474  -0.747  13.428  1.00  0.00           H  \nATOM   1166  HA2 GLY A  76       6.694   2.004  14.136  1.00  0.00           H  \nATOM   1167  HA3 GLY A  76       5.310   0.951  14.355  1.00  0.00           H  \nATOM   1168  N   LEU A  77       6.916  -0.293  16.458  1.00  0.00           N  \nATOM   1169  CA  LEU A  77       7.222  -0.533  17.858  1.00  0.00           C  \nATOM   1170  C   LEU A  77       8.593   0.062  18.185  1.00  0.00           C  \nATOM   1171  O   LEU A  77       8.749   0.755  19.189  1.00  0.00           O  \nATOM   1172  CB  LEU A  77       7.104  -2.023  18.184  1.00  0.00           C  \nATOM   1173  CG  LEU A  77       5.885  -2.435  19.011  1.00  0.00           C  \nATOM   1174  CD1 LEU A  77       5.721  -3.956  19.026  1.00  0.00           C  \nATOM   1175  CD2 LEU A  77       5.958  -1.852  20.423  1.00  0.00           C  \nATOM   1176  H   LEU A  77       6.889  -1.111  15.884  1.00  0.00           H  \nATOM   1177  HA  LEU A  77       6.470  -0.013  18.451  1.00  0.00           H  \nATOM   1178  HB2 LEU A  77       7.088  -2.580  17.246  1.00  0.00           H  \nATOM   1179  HB3 LEU A  77       8.003  -2.330  18.720  1.00  0.00           H  \nATOM   1180  HG  LEU A  77       4.995  -2.020  18.537  1.00  0.00           H  \nATOM   1181 HD11 LEU A  77       6.684  -4.423  19.235  1.00  0.00           H  \nATOM   1182 HD12 LEU A  77       5.006  -4.236  19.800  1.00  0.00           H  \nATOM   1183 HD13 LEU A  77       5.356  -4.292  18.056  1.00  0.00           H  \nATOM   1184 HD21 LEU A  77       4.994  -1.974  20.917  1.00  0.00           H  \nATOM   1185 HD22 LEU A  77       6.727  -2.374  20.993  1.00  0.00           H  \nATOM   1186 HD23 LEU A  77       6.206  -0.792  20.366  1.00  0.00           H  \nATOM   1187  N   LYS A  78       9.552  -0.232  17.319  1.00  0.00           N  \nATOM   1188  CA  LYS A  78      10.905   0.265  17.504  1.00  0.00           C  \nATOM   1189  C   LYS A  78      10.911   1.787  17.343  1.00  0.00           C  \nATOM   1190  O   LYS A  78      11.618   2.488  18.065  1.00  0.00           O  \nATOM   1191  CB  LYS A  78      11.874  -0.456  16.565  1.00  0.00           C  \nATOM   1192  CG  LYS A  78      11.502  -0.212  15.101  1.00  0.00           C  \nATOM   1193  CD  LYS A  78      12.534  -0.839  14.161  1.00  0.00           C  \nATOM   1194  CE  LYS A  78      13.161   0.219  13.251  1.00  0.00           C  \nATOM   1195  NZ  LYS A  78      14.086  -0.412  12.284  1.00  0.00           N  \nATOM   1196  H   LYS A  78       9.417  -0.796  16.505  1.00  0.00           H  \nATOM   1197  HA  LYS A  78      11.205   0.024  18.524  1.00  0.00           H  \nATOM   1198  HB2 LYS A  78      12.891  -0.109  16.748  1.00  0.00           H  \nATOM   1199  HB3 LYS A  78      11.860  -1.526  16.774  1.00  0.00           H  \nATOM   1200  HG2 LYS A  78      10.517  -0.633  14.897  1.00  0.00           H  \nATOM   1201  HG3 LYS A  78      11.436   0.859  14.912  1.00  0.00           H  \nATOM   1202  HD2 LYS A  78      13.313  -1.329  14.746  1.00  0.00           H  \nATOM   1203  HD3 LYS A  78      12.058  -1.610  13.555  1.00  0.00           H  \nATOM   1204  HE2 LYS A  78      12.378   0.757  12.717  1.00  0.00           H  \nATOM   1205  HE3 LYS A  78      13.698   0.952  13.853  1.00  0.00           H  \nATOM   1206  HZ1 LYS A  78      14.852  -0.826  12.776  1.00  0.00           H  \nATOM   1207  HZ2 LYS A  78      13.600  -1.118  11.769  1.00  0.00           H  \nATOM   1208  HZ3 LYS A  78      14.432   0.282  11.652  1.00  0.00           H  \nATOM   1209  N   VAL A  79      10.115   2.253  16.392  1.00  0.00           N  \nATOM   1210  CA  VAL A  79      10.020   3.678  16.128  1.00  0.00           C  \nATOM   1211  C   VAL A  79       9.535   4.395  17.389  1.00  0.00           C  \nATOM   1212  O   VAL A  79      10.164   5.349  17.846  1.00  0.00           O  \nATOM   1213  CB  VAL A  79       9.120   3.927  14.915  1.00  0.00           C  \nATOM   1214  CG1 VAL A  79       8.059   4.984  15.228  1.00  0.00           C  \nATOM   1215  CG2 VAL A  79       9.946   4.324  13.691  1.00  0.00           C  \nATOM   1216  H   VAL A  79       9.543   1.676  15.809  1.00  0.00           H  \nATOM   1217  HA  VAL A  79      11.020   4.035  15.884  1.00  0.00           H  \nATOM   1218  HB  VAL A  79       8.605   2.994  14.684  1.00  0.00           H  \nATOM   1219 HG11 VAL A  79       7.330   4.573  15.926  1.00  0.00           H  \nATOM   1220 HG12 VAL A  79       8.536   5.857  15.673  1.00  0.00           H  \nATOM   1221 HG13 VAL A  79       7.555   5.276  14.306  1.00  0.00           H  \nATOM   1222 HG21 VAL A  79       9.877   5.401  13.540  1.00  0.00           H  \nATOM   1223 HG22 VAL A  79      10.988   4.045  13.850  1.00  0.00           H  \nATOM   1224 HG23 VAL A  79       9.563   3.808  12.811  1.00  0.00           H  \nATOM   1225  N   LEU A  80       8.420   3.910  17.916  1.00  0.00           N  \nATOM   1226  CA  LEU A  80       7.844   4.492  19.116  1.00  0.00           C  \nATOM   1227  C   LEU A  80       8.873   4.443  20.247  1.00  0.00           C  \nATOM   1228  O   LEU A  80       9.229   5.475  20.813  1.00  0.00           O  \nATOM   1229  CB  LEU A  80       6.519   3.809  19.460  1.00  0.00           C  \nATOM   1230  CG  LEU A  80       5.257   4.648  19.249  1.00  0.00           C  \nATOM   1231  CD1 LEU A  80       4.170   3.836  18.544  1.00  0.00           C  \nATOM   1232  CD2 LEU A  80       4.764   5.239  20.572  1.00  0.00           C  \nATOM   1233  H   LEU A  80       7.915   3.134  17.538  1.00  0.00           H  \nATOM   1234  HA  LEU A  80       7.621   5.537  18.897  1.00  0.00           H  \nATOM   1235  HB2 LEU A  80       6.433   2.903  18.860  1.00  0.00           H  \nATOM   1236  HB3 LEU A  80       6.554   3.498  20.504  1.00  0.00           H  \nATOM   1237  HG  LEU A  80       5.508   5.485  18.597  1.00  0.00           H  \nATOM   1238 HD11 LEU A  80       4.497   3.590  17.534  1.00  0.00           H  \nATOM   1239 HD12 LEU A  80       3.985   2.916  19.100  1.00  0.00           H  \nATOM   1240 HD13 LEU A  80       3.252   4.422  18.496  1.00  0.00           H  \nATOM   1241 HD21 LEU A  80       5.449   6.022  20.898  1.00  0.00           H  \nATOM   1242 HD22 LEU A  80       3.769   5.662  20.433  1.00  0.00           H  \nATOM   1243 HD23 LEU A  80       4.723   4.455  21.327  1.00  0.00           H  \nATOM   1244  N   TYR A  81       9.323   3.232  20.542  1.00  0.00           N  \nATOM   1245  CA  TYR A  81      10.304   3.034  21.595  1.00  0.00           C  \nATOM   1246  C   TYR A  81      11.506   3.963  21.407  1.00  0.00           C  \nATOM   1247  O   TYR A  81      12.120   4.395  22.381  1.00  0.00           O  \nATOM   1248  CB  TYR A  81      10.773   1.583  21.469  1.00  0.00           C  \nATOM   1249  CG  TYR A  81      12.143   1.315  22.094  1.00  0.00           C  \nATOM   1250  CD1 TYR A  81      12.499   1.942  23.271  1.00  0.00           C  \nATOM   1251  CD2 TYR A  81      13.022   0.445  21.482  1.00  0.00           C  \nATOM   1252  CE1 TYR A  81      13.789   1.689  23.859  1.00  0.00           C  \nATOM   1253  CE2 TYR A  81      14.312   0.192  22.070  1.00  0.00           C  \nATOM   1254  CZ  TYR A  81      14.632   0.827  23.230  1.00  0.00           C  \nATOM   1255  OH  TYR A  81      15.850   0.588  23.786  1.00  0.00           O  \nATOM   1256  H   TYR A  81       9.029   2.397  20.077  1.00  0.00           H  \nATOM   1257  HA  TYR A  81       9.826   3.260  22.548  1.00  0.00           H  \nATOM   1258  HB2 TYR A  81      10.036   0.932  21.939  1.00  0.00           H  \nATOM   1259  HB3 TYR A  81      10.807   1.314  20.413  1.00  0.00           H  \nATOM   1260  HD1 TYR A  81      11.805   2.629  23.754  1.00  0.00           H  \nATOM   1261  HD2 TYR A  81      12.741  -0.050  20.552  1.00  0.00           H  \nATOM   1262  HE1 TYR A  81      14.083   2.178  24.788  1.00  0.00           H  \nATOM   1263  HE2 TYR A  81      15.016  -0.492  21.597  1.00  0.00           H  \nATOM   1264  HH  TYR A  81      16.431   0.097  23.137  1.00  0.00           H  \nATOM   1265  N   LYS A  82      11.804   4.244  20.147  1.00  0.00           N  \nATOM   1266  CA  LYS A  82      12.921   5.114  19.818  1.00  0.00           C  \nATOM   1267  C   LYS A  82      12.535   6.565  20.113  1.00  0.00           C  \nATOM   1268  O   LYS A  82      13.383   7.371  20.493  1.00  0.00           O  \nATOM   1269  CB  LYS A  82      13.377   4.879  18.377  1.00  0.00           C  \nATOM   1270  CG  LYS A  82      14.186   6.069  17.857  1.00  0.00           C  \nATOM   1271  CD  LYS A  82      13.267   7.230  17.470  1.00  0.00           C  \nATOM   1272  CE  LYS A  82      13.718   7.872  16.157  1.00  0.00           C  \nATOM   1273  NZ  LYS A  82      12.987   9.138  15.921  1.00  0.00           N  \nATOM   1274  H   LYS A  82      11.300   3.888  19.360  1.00  0.00           H  \nATOM   1275  HA  LYS A  82      13.752   4.838  20.468  1.00  0.00           H  \nATOM   1276  HB2 LYS A  82      13.981   3.973  18.325  1.00  0.00           H  \nATOM   1277  HB3 LYS A  82      12.508   4.719  17.738  1.00  0.00           H  \nATOM   1278  HG2 LYS A  82      14.889   6.397  18.622  1.00  0.00           H  \nATOM   1279  HG3 LYS A  82      14.775   5.763  16.992  1.00  0.00           H  \nATOM   1280  HD2 LYS A  82      12.243   6.870  17.371  1.00  0.00           H  \nATOM   1281  HD3 LYS A  82      13.266   7.977  18.264  1.00  0.00           H  \nATOM   1282  HE2 LYS A  82      14.790   8.065  16.189  1.00  0.00           H  \nATOM   1283  HE3 LYS A  82      13.542   7.185  15.330  1.00  0.00           H  \nATOM   1284  HZ1 LYS A  82      12.704   9.527  16.798  1.00  0.00           H  \nATOM   1285  HZ2 LYS A  82      13.585   9.784  15.447  1.00  0.00           H  \nATOM   1286  HZ3 LYS A  82      12.180   8.957  15.360  1.00  0.00           H  \nATOM   1287  N   LEU A  83      11.256   6.853  19.927  1.00  0.00           N  \nATOM   1288  CA  LEU A  83      10.747   8.192  20.168  1.00  0.00           C  \nATOM   1289  C   LEU A  83      10.738   8.466  21.673  1.00  0.00           C  \nATOM   1290  O   LEU A  83      11.235   9.497  22.123  1.00  0.00           O  \nATOM   1291  CB  LEU A  83       9.383   8.376  19.501  1.00  0.00           C  \nATOM   1292  CG  LEU A  83       8.888   9.818  19.372  1.00  0.00           C  \nATOM   1293  CD1 LEU A  83       8.533  10.148  17.920  1.00  0.00           C  \nATOM   1294  CD2 LEU A  83       7.719  10.084  20.322  1.00  0.00           C  \nATOM   1295  H   LEU A  83      10.573   6.191  19.618  1.00  0.00           H  \nATOM   1296  HA  LEU A  83      11.435   8.893  19.694  1.00  0.00           H  \nATOM   1297  HB2 LEU A  83       9.426   7.937  18.504  1.00  0.00           H  \nATOM   1298  HB3 LEU A  83       8.644   7.809  20.067  1.00  0.00           H  \nATOM   1299  HG  LEU A  83       9.699  10.485  19.664  1.00  0.00           H  \nATOM   1300 HD11 LEU A  83       9.436  10.124  17.311  1.00  0.00           H  \nATOM   1301 HD12 LEU A  83       7.821   9.413  17.545  1.00  0.00           H  \nATOM   1302 HD13 LEU A  83       8.088  11.142  17.872  1.00  0.00           H  \nATOM   1303 HD21 LEU A  83       8.052   9.954  21.352  1.00  0.00           H  \nATOM   1304 HD22 LEU A  83       7.362  11.104  20.182  1.00  0.00           H  \nATOM   1305 HD23 LEU A  83       6.912   9.383  20.110  1.00  0.00           H  \nATOM   1306  N   MET A  84      10.166   7.525  22.410  1.00  0.00           N  \nATOM   1307  CA  MET A  84      10.085   7.652  23.855  1.00  0.00           C  \nATOM   1308  C   MET A  84      11.479   7.767  24.474  1.00  0.00           C  \nATOM   1309  O   MET A  84      11.793   8.762  25.127  1.00  0.00           O  \nATOM   1310  CB  MET A  84       9.368   6.431  24.435  1.00  0.00           C  \nATOM   1311  CG  MET A  84       7.852   6.638  24.442  1.00  0.00           C  \nATOM   1312  SD  MET A  84       7.369   7.569  25.887  1.00  0.00           S  \nATOM   1313  CE  MET A  84       6.201   8.701  25.150  1.00  0.00           C  \nATOM   1314  H   MET A  84       9.764   6.689  22.036  1.00  0.00           H  \nATOM   1315  HA  MET A  84       9.522   8.567  24.037  1.00  0.00           H  \nATOM   1316  HB2 MET A  84       9.615   5.547  23.849  1.00  0.00           H  \nATOM   1317  HB3 MET A  84       9.718   6.248  25.451  1.00  0.00           H  \nATOM   1318  HG2 MET A  84       7.545   7.167  23.539  1.00  0.00           H  \nATOM   1319  HG3 MET A  84       7.345   5.673  24.434  1.00  0.00           H  \nATOM   1320  HE1 MET A  84       5.297   8.736  25.758  1.00  0.00           H  \nATOM   1321  HE2 MET A  84       6.643   9.696  25.098  1.00  0.00           H  \nATOM   1322  HE3 MET A  84       5.951   8.361  24.145  1.00  0.00           H  \nATOM   1323  N   ASP A  85      12.280   6.735  24.248  1.00  0.00           N  \nATOM   1324  CA  ASP A  85      13.633   6.708  24.776  1.00  0.00           C  \nATOM   1325  C   ASP A  85      14.369   7.979  24.346  1.00  0.00           C  \nATOM   1326  O   ASP A  85      14.946   8.029  23.261  1.00  0.00           O  \nATOM   1327  CB  ASP A  85      14.411   5.506  24.236  1.00  0.00           C  \nATOM   1328  CG  ASP A  85      15.892   5.469  24.619  1.00  0.00           C  \nATOM   1329  OD1 ASP A  85      16.366   6.496  25.151  1.00  0.00           O  \nATOM   1330  OD2 ASP A  85      16.516   4.415  24.371  1.00  0.00           O  \nATOM   1331  H   ASP A  85      12.017   5.930  23.717  1.00  0.00           H  \nATOM   1332  HA  ASP A  85      13.518   6.639  25.858  1.00  0.00           H  \nATOM   1333  HB2 ASP A  85      13.935   4.594  24.596  1.00  0.00           H  \nATOM   1334  HB3 ASP A  85      14.332   5.500  23.149  1.00  0.00           H  \nATOM   1335  N   VAL A  86      14.324   8.975  25.219  1.00  0.00           N  \nATOM   1336  CA  VAL A  86      14.979  10.242  24.943  1.00  0.00           C  \nATOM   1337  C   VAL A  86      16.266  10.336  25.767  1.00  0.00           C  \nATOM   1338  O   VAL A  86      17.007  11.311  25.657  1.00  0.00           O  \nATOM   1339  CB  VAL A  86      14.014  11.399  25.209  1.00  0.00           C  \nATOM   1340  CG1 VAL A  86      13.729  11.543  26.705  1.00  0.00           C  \nATOM   1341  CG2 VAL A  86      14.552  12.707  24.624  1.00  0.00           C  \nATOM   1342  H   VAL A  86      13.852   8.926  26.099  1.00  0.00           H  \nATOM   1343  HA  VAL A  86      15.239  10.255  23.885  1.00  0.00           H  \nATOM   1344  HB  VAL A  86      13.072  11.171  24.709  1.00  0.00           H  \nATOM   1345 HG11 VAL A  86      14.438  12.246  27.144  1.00  0.00           H  \nATOM   1346 HG12 VAL A  86      12.714  11.914  26.848  1.00  0.00           H  \nATOM   1347 HG13 VAL A  86      13.833  10.572  27.190  1.00  0.00           H  \nATOM   1348 HG21 VAL A  86      14.994  12.513  23.647  1.00  0.00           H  \nATOM   1349 HG22 VAL A  86      13.734  13.420  24.518  1.00  0.00           H  \nATOM   1350 HG23 VAL A  86      15.309  13.118  25.291  1.00  0.00           H  \nATOM   1351  N   ASP A  87      16.491   9.308  26.572  1.00  0.00           N  \nATOM   1352  CA  ASP A  87      17.675   9.263  27.414  1.00  0.00           C  \nATOM   1353  C   ASP A  87      18.885   8.870  26.563  1.00  0.00           C  \nATOM   1354  O   ASP A  87      20.011   9.268  26.858  1.00  0.00           O  \nATOM   1355  CB  ASP A  87      17.521   8.223  28.525  1.00  0.00           C  \nATOM   1356  CG  ASP A  87      17.030   6.850  28.062  1.00  0.00           C  \nATOM   1357  OD1 ASP A  87      17.894   6.050  27.642  1.00  0.00           O  \nATOM   1358  OD2 ASP A  87      15.802   6.630  28.140  1.00  0.00           O  \nATOM   1359  H   ASP A  87      15.883   8.519  26.655  1.00  0.00           H  \nATOM   1360  HA  ASP A  87      17.767  10.265  27.833  1.00  0.00           H  \nATOM   1361  HB2 ASP A  87      18.483   8.099  29.022  1.00  0.00           H  \nATOM   1362  HB3 ASP A  87      16.825   8.609  29.270  1.00  0.00           H  \nATOM   1363  N   GLY A  88      18.611   8.095  25.525  1.00  0.00           N  \nATOM   1364  CA  GLY A  88      19.663   7.644  24.629  1.00  0.00           C  \nATOM   1365  C   GLY A  88      20.444   6.480  25.243  1.00  0.00           C  \nATOM   1366  O   GLY A  88      21.605   6.636  25.619  1.00  0.00           O  \nATOM   1367  H   GLY A  88      17.693   7.775  25.292  1.00  0.00           H  \nATOM   1368  HA2 GLY A  88      19.229   7.335  23.679  1.00  0.00           H  \nATOM   1369  HA3 GLY A  88      20.342   8.470  24.416  1.00  0.00           H  \nATOM   1370  N   ASP A  89      19.775   5.339  25.325  1.00  0.00           N  \nATOM   1371  CA  ASP A  89      20.392   4.149  25.886  1.00  0.00           C  \nATOM   1372  C   ASP A  89      20.516   3.082  24.797  1.00  0.00           C  \nATOM   1373  O   ASP A  89      21.527   2.386  24.718  1.00  0.00           O  \nATOM   1374  CB  ASP A  89      19.542   3.571  27.019  1.00  0.00           C  \nATOM   1375  CG  ASP A  89      19.672   4.295  28.360  1.00  0.00           C  \nATOM   1376  OD1 ASP A  89      20.584   5.144  28.460  1.00  0.00           O  \nATOM   1377  OD2 ASP A  89      18.857   3.984  29.255  1.00  0.00           O  \nATOM   1378  H   ASP A  89      18.831   5.221  25.017  1.00  0.00           H  \nATOM   1379  HA  ASP A  89      21.361   4.478  26.261  1.00  0.00           H  \nATOM   1380  HB2 ASP A  89      18.495   3.589  26.714  1.00  0.00           H  \nATOM   1381  HB3 ASP A  89      19.814   2.525  27.161  1.00  0.00           H  \nATOM   1382  N   GLY A  90      19.474   2.988  23.984  1.00  0.00           N  \nATOM   1383  CA  GLY A  90      19.454   2.017  22.902  1.00  0.00           C  \nATOM   1384  C   GLY A  90      18.929   0.665  23.389  1.00  0.00           C  \nATOM   1385  O   GLY A  90      18.578  -0.195  22.582  1.00  0.00           O  \nATOM   1386  H   GLY A  90      18.656   3.558  24.055  1.00  0.00           H  \nATOM   1387  HA2 GLY A  90      18.826   2.385  22.091  1.00  0.00           H  \nATOM   1388  HA3 GLY A  90      20.459   1.897  22.498  1.00  0.00           H  \nATOM   1389  N   LYS A  91      18.891   0.520  24.705  1.00  0.00           N  \nATOM   1390  CA  LYS A  91      18.415  -0.713  25.308  1.00  0.00           C  \nATOM   1391  C   LYS A  91      17.259  -0.397  26.259  1.00  0.00           C  \nATOM   1392  O   LYS A  91      17.078   0.750  26.662  1.00  0.00           O  \nATOM   1393  CB  LYS A  91      19.569  -1.466  25.973  1.00  0.00           C  \nATOM   1394  CG  LYS A  91      20.846  -1.363  25.135  1.00  0.00           C  \nATOM   1395  CD  LYS A  91      21.909  -0.531  25.856  1.00  0.00           C  \nATOM   1396  CE  LYS A  91      22.994  -1.428  26.454  1.00  0.00           C  \nATOM   1397  NZ  LYS A  91      24.011  -0.614  27.156  1.00  0.00           N  \nATOM   1398  H   LYS A  91      19.178   1.224  25.354  1.00  0.00           H  \nATOM   1399  HA  LYS A  91      18.038  -1.346  24.504  1.00  0.00           H  \nATOM   1400  HB2 LYS A  91      19.749  -1.059  26.968  1.00  0.00           H  \nATOM   1401  HB3 LYS A  91      19.299  -2.514  26.102  1.00  0.00           H  \nATOM   1402  HG2 LYS A  91      21.234  -2.361  24.934  1.00  0.00           H  \nATOM   1403  HG3 LYS A  91      20.616  -0.909  24.171  1.00  0.00           H  \nATOM   1404  HD2 LYS A  91      22.359   0.174  25.157  1.00  0.00           H  \nATOM   1405  HD3 LYS A  91      21.442   0.057  26.646  1.00  0.00           H  \nATOM   1406  HE2 LYS A  91      22.546  -2.138  27.148  1.00  0.00           H  \nATOM   1407  HE3 LYS A  91      23.468  -2.011  25.664  1.00  0.00           H  \nATOM   1408  HZ1 LYS A  91      24.907  -0.767  26.739  1.00  0.00           H  \nATOM   1409  HZ2 LYS A  91      23.771   0.354  27.083  1.00  0.00           H  \nATOM   1410  HZ3 LYS A  91      24.042  -0.878  28.120  1.00  0.00           H  \nATOM   1411  N   LEU A  92      16.505  -1.436  26.589  1.00  0.00           N  \nATOM   1412  CA  LEU A  92      15.371  -1.284  27.485  1.00  0.00           C  \nATOM   1413  C   LEU A  92      15.246  -2.533  28.359  1.00  0.00           C  \nATOM   1414  O   LEU A  92      15.826  -3.572  28.049  1.00  0.00           O  \nATOM   1415  CB  LEU A  92      14.103  -0.957  26.693  1.00  0.00           C  \nATOM   1416  CG  LEU A  92      13.441  -2.133  25.973  1.00  0.00           C  \nATOM   1417  CD1 LEU A  92      12.364  -2.776  26.849  1.00  0.00           C  \nATOM   1418  CD2 LEU A  92      12.889  -1.702  24.612  1.00  0.00           C  \nATOM   1419  H   LEU A  92      16.659  -2.367  26.256  1.00  0.00           H  \nATOM   1420  HA  LEU A  92      15.577  -0.430  28.130  1.00  0.00           H  \nATOM   1421  HB2 LEU A  92      13.376  -0.518  27.376  1.00  0.00           H  \nATOM   1422  HB3 LEU A  92      14.347  -0.194  25.954  1.00  0.00           H  \nATOM   1423  HG  LEU A  92      14.200  -2.892  25.787  1.00  0.00           H  \nATOM   1424 HD11 LEU A  92      12.809  -3.103  27.789  1.00  0.00           H  \nATOM   1425 HD12 LEU A  92      11.578  -2.049  27.053  1.00  0.00           H  \nATOM   1426 HD13 LEU A  92      11.939  -3.635  26.330  1.00  0.00           H  \nATOM   1427 HD21 LEU A  92      12.516  -2.576  24.078  1.00  0.00           H  \nATOM   1428 HD22 LEU A  92      12.077  -0.990  24.758  1.00  0.00           H  \nATOM   1429 HD23 LEU A  92      13.683  -1.232  24.031  1.00  0.00           H  \nATOM   1430  N   THR A  93      14.483  -2.391  29.433  1.00  0.00           N  \nATOM   1431  CA  THR A  93      14.274  -3.495  30.354  1.00  0.00           C  \nATOM   1432  C   THR A  93      12.784  -3.824  30.462  1.00  0.00           C  \nATOM   1433  O   THR A  93      11.952  -3.164  29.841  1.00  0.00           O  \nATOM   1434  CB  THR A  93      14.912  -3.121  31.693  1.00  0.00           C  \nATOM   1435  OG1 THR A  93      14.004  -2.181  32.260  1.00  0.00           O  \nATOM   1436  CG2 THR A  93      16.210  -2.329  31.523  1.00  0.00           C  \nATOM   1437  H   THR A  93      14.014  -1.542  29.677  1.00  0.00           H  \nATOM   1438  HA  THR A  93      14.768  -4.378  29.949  1.00  0.00           H  \nATOM   1439  HB  THR A  93      15.074  -4.007  32.308  1.00  0.00           H  \nATOM   1440  HG1 THR A  93      13.874  -1.411  31.635  1.00  0.00           H  \nATOM   1441 HG21 THR A  93      16.814  -2.423  32.425  1.00  0.00           H  \nATOM   1442 HG22 THR A  93      16.766  -2.722  30.671  1.00  0.00           H  \nATOM   1443 HG23 THR A  93      15.975  -1.279  31.350  1.00  0.00           H  \nATOM   1444  N   LYS A  94      12.492  -4.845  31.254  1.00  0.00           N  \nATOM   1445  CA  LYS A  94      11.116  -5.270  31.451  1.00  0.00           C  \nATOM   1446  C   LYS A  94      10.403  -4.269  32.362  1.00  0.00           C  \nATOM   1447  O   LYS A  94       9.194  -4.073  32.249  1.00  0.00           O  \nATOM   1448  CB  LYS A  94      11.069  -6.711  31.965  1.00  0.00           C  \nATOM   1449  CG  LYS A  94      11.551  -6.792  33.415  1.00  0.00           C  \nATOM   1450  CD  LYS A  94      11.986  -8.215  33.769  1.00  0.00           C  \nATOM   1451  CE  LYS A  94      11.722  -8.518  35.245  1.00  0.00           C  \nATOM   1452  NZ  LYS A  94      10.535  -9.390  35.390  1.00  0.00           N  \nATOM   1453  H   LYS A  94      13.174  -5.377  31.755  1.00  0.00           H  \nATOM   1454  HA  LYS A  94      10.628  -5.259  30.477  1.00  0.00           H  \nATOM   1455  HB2 LYS A  94      10.051  -7.093  31.895  1.00  0.00           H  \nATOM   1456  HB3 LYS A  94      11.691  -7.346  31.335  1.00  0.00           H  \nATOM   1457  HG2 LYS A  94      12.385  -6.105  33.563  1.00  0.00           H  \nATOM   1458  HG3 LYS A  94      10.753  -6.474  34.086  1.00  0.00           H  \nATOM   1459  HD2 LYS A  94      11.448  -8.930  33.145  1.00  0.00           H  \nATOM   1460  HD3 LYS A  94      13.047  -8.340  33.553  1.00  0.00           H  \nATOM   1461  HE2 LYS A  94      12.593  -9.003  35.685  1.00  0.00           H  \nATOM   1462  HE3 LYS A  94      11.566  -7.588  35.791  1.00  0.00           H  \nATOM   1463  HZ1 LYS A  94      10.828 -10.312  35.647  1.00  0.00           H  \nATOM   1464  HZ2 LYS A  94       9.935  -9.021  36.100  1.00  0.00           H  \nATOM   1465  HZ3 LYS A  94      10.040  -9.428  34.522  1.00  0.00           H  \nATOM   1466  N   GLU A  95      11.183  -3.661  33.244  1.00  0.00           N  \nATOM   1467  CA  GLU A  95      10.641  -2.684  34.174  1.00  0.00           C  \nATOM   1468  C   GLU A  95      10.345  -1.369  33.450  1.00  0.00           C  \nATOM   1469  O   GLU A  95       9.364  -0.694  33.758  1.00  0.00           O  \nATOM   1470  CB  GLU A  95      11.593  -2.460  35.351  1.00  0.00           C  \nATOM   1471  CG  GLU A  95      10.855  -1.856  36.546  1.00  0.00           C  \nATOM   1472  CD  GLU A  95      10.606  -2.909  37.627  1.00  0.00           C  \nATOM   1473  OE1 GLU A  95      10.484  -4.094  37.248  1.00  0.00           O  \nATOM   1474  OE2 GLU A  95      10.545  -2.507  38.809  1.00  0.00           O  \nATOM   1475  H   GLU A  95      12.166  -3.826  33.330  1.00  0.00           H  \nATOM   1476  HA  GLU A  95       9.713  -3.120  34.544  1.00  0.00           H  \nATOM   1477  HB2 GLU A  95      12.048  -3.408  35.641  1.00  0.00           H  \nATOM   1478  HB3 GLU A  95      12.404  -1.798  35.046  1.00  0.00           H  \nATOM   1479  HG2 GLU A  95      11.438  -1.034  36.961  1.00  0.00           H  \nATOM   1480  HG3 GLU A  95       9.904  -1.436  36.217  1.00  0.00           H  \nATOM   1481  N   GLU A  96      11.211  -1.045  32.502  1.00  0.00           N  \nATOM   1482  CA  GLU A  96      11.055   0.177  31.731  1.00  0.00           C  \nATOM   1483  C   GLU A  96       9.918   0.025  30.719  1.00  0.00           C  \nATOM   1484  O   GLU A  96       9.126   0.946  30.526  1.00  0.00           O  \nATOM   1485  CB  GLU A  96      12.363   0.556  31.033  1.00  0.00           C  \nATOM   1486  CG  GLU A  96      12.213   1.871  30.265  1.00  0.00           C  \nATOM   1487  CD  GLU A  96      13.495   2.209  29.501  1.00  0.00           C  \nATOM   1488  OE1 GLU A  96      13.597   1.762  28.338  1.00  0.00           O  \nATOM   1489  OE2 GLU A  96      14.343   2.906  30.098  1.00  0.00           O  \nATOM   1490  H   GLU A  96      12.007  -1.600  32.257  1.00  0.00           H  \nATOM   1491  HA  GLU A  96      10.802   0.948  32.459  1.00  0.00           H  \nATOM   1492  HB2 GLU A  96      13.159   0.652  31.772  1.00  0.00           H  \nATOM   1493  HB3 GLU A  96      12.657  -0.238  30.348  1.00  0.00           H  \nATOM   1494  HG2 GLU A  96      11.379   1.795  29.567  1.00  0.00           H  \nATOM   1495  HG3 GLU A  96      11.975   2.677  30.959  1.00  0.00           H  \nATOM   1496  N   VAL A  97       9.872  -1.146  30.100  1.00  0.00           N  \nATOM   1497  CA  VAL A  97       8.845  -1.430  29.113  1.00  0.00           C  \nATOM   1498  C   VAL A  97       7.477  -1.455  29.799  1.00  0.00           C  \nATOM   1499  O   VAL A  97       6.514  -0.881  29.292  1.00  0.00           O  \nATOM   1500  CB  VAL A  97       9.170  -2.734  28.380  1.00  0.00           C  \nATOM   1501  CG1 VAL A  97       8.489  -3.926  29.054  1.00  0.00           C  \nATOM   1502  CG2 VAL A  97       8.782  -2.643  26.903  1.00  0.00           C  \nATOM   1503  H   VAL A  97      10.519  -1.890  30.263  1.00  0.00           H  \nATOM   1504  HA  VAL A  97       8.856  -0.621  28.383  1.00  0.00           H  \nATOM   1505  HB  VAL A  97      10.248  -2.888  28.434  1.00  0.00           H  \nATOM   1506 HG11 VAL A  97       8.711  -4.835  28.495  1.00  0.00           H  \nATOM   1507 HG12 VAL A  97       8.860  -4.028  30.074  1.00  0.00           H  \nATOM   1508 HG13 VAL A  97       7.411  -3.766  29.074  1.00  0.00           H  \nATOM   1509 HG21 VAL A  97       7.795  -2.190  26.813  1.00  0.00           H  \nATOM   1510 HG22 VAL A  97       9.512  -2.032  26.372  1.00  0.00           H  \nATOM   1511 HG23 VAL A  97       8.763  -3.644  26.470  1.00  0.00           H  \nATOM   1512  N   THR A  98       7.436  -2.127  30.940  1.00  0.00           N  \nATOM   1513  CA  THR A  98       6.202  -2.234  31.700  1.00  0.00           C  \nATOM   1514  C   THR A  98       5.853  -0.891  32.344  1.00  0.00           C  \nATOM   1515  O   THR A  98       4.680  -0.581  32.545  1.00  0.00           O  \nATOM   1516  CB  THR A  98       6.369  -3.368  32.714  1.00  0.00           C  \nATOM   1517  OG1 THR A  98       5.033  -3.753  33.028  1.00  0.00           O  \nATOM   1518  CG2 THR A  98       6.940  -2.882  34.048  1.00  0.00           C  \nATOM   1519  H   THR A  98       8.223  -2.591  31.345  1.00  0.00           H  \nATOM   1520  HA  THR A  98       5.394  -2.478  31.011  1.00  0.00           H  \nATOM   1521  HB  THR A  98       6.976  -4.173  32.301  1.00  0.00           H  \nATOM   1522  HG1 THR A  98       4.590  -3.038  33.569  1.00  0.00           H  \nATOM   1523 HG21 THR A  98       6.279  -2.127  34.472  1.00  0.00           H  \nATOM   1524 HG22 THR A  98       7.020  -3.723  34.737  1.00  0.00           H  \nATOM   1525 HG23 THR A  98       7.927  -2.451  33.885  1.00  0.00           H  \nATOM   1526  N   SER A  99       6.894  -0.129  32.649  1.00  0.00           N  \nATOM   1527  CA  SER A  99       6.712   1.174  33.265  1.00  0.00           C  \nATOM   1528  C   SER A  99       6.005   2.119  32.292  1.00  0.00           C  \nATOM   1529  O   SER A  99       5.088   2.842  32.678  1.00  0.00           O  \nATOM   1530  CB  SER A  99       8.053   1.768  33.703  1.00  0.00           C  \nATOM   1531  OG  SER A  99       8.075   3.187  33.575  1.00  0.00           O  \nATOM   1532  H   SER A  99       7.845  -0.389  32.482  1.00  0.00           H  \nATOM   1533  HA  SER A  99       6.092   0.994  34.143  1.00  0.00           H  \nATOM   1534  HB2 SER A  99       8.250   1.494  34.739  1.00  0.00           H  \nATOM   1535  HB3 SER A  99       8.854   1.337  33.102  1.00  0.00           H  \nATOM   1536  HG  SER A  99       8.037   3.612  34.479  1.00  0.00           H  \nATOM   1537  N   PHE A 100       6.459   2.084  31.047  1.00  0.00           N  \nATOM   1538  CA  PHE A 100       5.882   2.929  30.016  1.00  0.00           C  \nATOM   1539  C   PHE A 100       4.520   2.393  29.567  1.00  0.00           C  \nATOM   1540  O   PHE A 100       3.551   3.146  29.480  1.00  0.00           O  \nATOM   1541  CB  PHE A 100       6.843   2.903  28.826  1.00  0.00           C  \nATOM   1542  CG  PHE A 100       6.186   3.240  27.487  1.00  0.00           C  \nATOM   1543  CD1 PHE A 100       5.366   4.320  27.384  1.00  0.00           C  \nATOM   1544  CD2 PHE A 100       6.420   2.459  26.398  1.00  0.00           C  \nATOM   1545  CE1 PHE A 100       4.755   4.633  26.141  1.00  0.00           C  \nATOM   1546  CE2 PHE A 100       5.810   2.772  25.155  1.00  0.00           C  \nATOM   1547  CZ  PHE A 100       4.990   3.852  25.053  1.00  0.00           C  \nATOM   1548  H   PHE A 100       7.206   1.493  30.741  1.00  0.00           H  \nATOM   1549  HA  PHE A 100       5.756   3.923  30.444  1.00  0.00           H  \nATOM   1550  HB2 PHE A 100       7.652   3.610  29.010  1.00  0.00           H  \nATOM   1551  HB3 PHE A 100       7.295   1.913  28.758  1.00  0.00           H  \nATOM   1552  HD1 PHE A 100       5.178   4.946  28.257  1.00  0.00           H  \nATOM   1553  HD2 PHE A 100       7.077   1.593  26.480  1.00  0.00           H  \nATOM   1554  HE1 PHE A 100       4.098   5.499  26.059  1.00  0.00           H  \nATOM   1555  HE2 PHE A 100       5.998   2.146  24.282  1.00  0.00           H  \nATOM   1556  HZ  PHE A 100       4.521   4.092  24.098  1.00  0.00           H  \nATOM   1557  N   PHE A 101       4.490   1.097  29.295  1.00  0.00           N  \nATOM   1558  CA  PHE A 101       3.264   0.452  28.858  1.00  0.00           C  \nATOM   1559  C   PHE A 101       2.169   0.580  29.919  1.00  0.00           C  \nATOM   1560  O   PHE A 101       1.007   0.815  29.592  1.00  0.00           O  \nATOM   1561  CB  PHE A 101       3.586  -1.029  28.651  1.00  0.00           C  \nATOM   1562  CG  PHE A 101       4.041  -1.374  27.232  1.00  0.00           C  \nATOM   1563  CD1 PHE A 101       3.358  -0.887  26.161  1.00  0.00           C  \nATOM   1564  CD2 PHE A 101       5.128  -2.169  27.040  1.00  0.00           C  \nATOM   1565  CE1 PHE A 101       3.780  -1.208  24.844  1.00  0.00           C  \nATOM   1566  CE2 PHE A 101       5.550  -2.489  25.723  1.00  0.00           C  \nATOM   1567  CZ  PHE A 101       4.867  -2.002  24.652  1.00  0.00           C  \nATOM   1568  H   PHE A 101       5.283   0.492  29.368  1.00  0.00           H  \nATOM   1569  HA  PHE A 101       2.943   0.953  27.945  1.00  0.00           H  \nATOM   1570  HB2 PHE A 101       4.367  -1.322  29.353  1.00  0.00           H  \nATOM   1571  HB3 PHE A 101       2.702  -1.620  28.892  1.00  0.00           H  \nATOM   1572  HD1 PHE A 101       2.486  -0.251  26.315  1.00  0.00           H  \nATOM   1573  HD2 PHE A 101       5.676  -2.559  27.898  1.00  0.00           H  \nATOM   1574  HE1 PHE A 101       3.232  -0.818  23.986  1.00  0.00           H  \nATOM   1575  HE2 PHE A 101       6.422  -3.126  25.570  1.00  0.00           H  \nATOM   1576  HZ  PHE A 101       5.191  -2.248  23.641  1.00  0.00           H  \nATOM   1577  N   LYS A 102       2.579   0.421  31.169  1.00  0.00           N  \nATOM   1578  CA  LYS A 102       1.647   0.516  32.280  1.00  0.00           C  \nATOM   1579  C   LYS A 102       0.742   1.733  32.078  1.00  0.00           C  \nATOM   1580  O   LYS A 102      -0.429   1.709  32.454  1.00  0.00           O  \nATOM   1581  CB  LYS A 102       2.401   0.524  33.611  1.00  0.00           C  \nATOM   1582  CG  LYS A 102       1.474   0.918  34.763  1.00  0.00           C  \nATOM   1583  CD  LYS A 102       1.679   2.383  35.156  1.00  0.00           C  \nATOM   1584  CE  LYS A 102       2.197   2.498  36.591  1.00  0.00           C  \nATOM   1585  NZ  LYS A 102       3.088   3.672  36.729  1.00  0.00           N  \nATOM   1586  H   LYS A 102       3.526   0.231  31.426  1.00  0.00           H  \nATOM   1587  HA  LYS A 102       1.029  -0.381  32.265  1.00  0.00           H  \nATOM   1588  HB2 LYS A 102       2.823  -0.462  33.801  1.00  0.00           H  \nATOM   1589  HB3 LYS A 102       3.236   1.223  33.557  1.00  0.00           H  \nATOM   1590  HG2 LYS A 102       0.436   0.759  34.471  1.00  0.00           H  \nATOM   1591  HG3 LYS A 102       1.665   0.277  35.624  1.00  0.00           H  \nATOM   1592  HD2 LYS A 102       2.387   2.851  34.472  1.00  0.00           H  \nATOM   1593  HD3 LYS A 102       0.738   2.924  35.060  1.00  0.00           H  \nATOM   1594  HE2 LYS A 102       1.357   2.588  37.281  1.00  0.00           H  \nATOM   1595  HE3 LYS A 102       2.736   1.591  36.864  1.00  0.00           H  \nATOM   1596  HZ1 LYS A 102       3.790   3.479  37.414  1.00  0.00           H  \nATOM   1597  HZ2 LYS A 102       3.523   3.864  35.849  1.00  0.00           H  \nATOM   1598  HZ3 LYS A 102       2.551   4.466  37.016  1.00  0.00           H  \nATOM   1599  N   LYS A 103       1.319   2.768  31.485  1.00  0.00           N  \nATOM   1600  CA  LYS A 103       0.578   3.991  31.229  1.00  0.00           C  \nATOM   1601  C   LYS A 103      -0.722   3.652  30.498  1.00  0.00           C  \nATOM   1602  O   LYS A 103      -1.766   4.242  30.773  1.00  0.00           O  \nATOM   1603  CB  LYS A 103       1.455   5.003  30.487  1.00  0.00           C  \nATOM   1604  CG  LYS A 103       0.801   6.386  30.469  1.00  0.00           C  \nATOM   1605  CD  LYS A 103      -0.153   6.527  29.282  1.00  0.00           C  \nATOM   1606  CE  LYS A 103       0.118   7.820  28.511  1.00  0.00           C  \nATOM   1607  NZ  LYS A 103      -1.155   8.476  28.135  1.00  0.00           N  \nATOM   1608  H   LYS A 103       2.272   2.779  31.182  1.00  0.00           H  \nATOM   1609  HA  LYS A 103       0.328   4.430  32.194  1.00  0.00           H  \nATOM   1610  HB2 LYS A 103       2.432   5.064  30.966  1.00  0.00           H  \nATOM   1611  HB3 LYS A 103       1.623   4.663  29.465  1.00  0.00           H  \nATOM   1612  HG2 LYS A 103       0.256   6.546  31.399  1.00  0.00           H  \nATOM   1613  HG3 LYS A 103       1.571   7.155  30.415  1.00  0.00           H  \nATOM   1614  HD2 LYS A 103      -0.039   5.672  28.615  1.00  0.00           H  \nATOM   1615  HD3 LYS A 103      -1.184   6.519  29.636  1.00  0.00           H  \nATOM   1616  HE2 LYS A 103       0.715   8.497  29.122  1.00  0.00           H  \nATOM   1617  HE3 LYS A 103       0.700   7.602  27.615  1.00  0.00           H  \nATOM   1618  HZ1 LYS A 103      -1.835   7.778  27.910  1.00  0.00           H  \nATOM   1619  HZ2 LYS A 103      -1.484   9.028  28.901  1.00  0.00           H  \nATOM   1620  HZ3 LYS A 103      -1.005   9.064  27.340  1.00  0.00           H  \nATOM   1621  N   HIS A 104      -0.618   2.702  29.580  1.00  0.00           N  \nATOM   1622  CA  HIS A 104      -1.772   2.277  28.808  1.00  0.00           C  \nATOM   1623  C   HIS A 104      -2.737   1.505  29.710  1.00  0.00           C  \nATOM   1624  O   HIS A 104      -3.920   1.836  29.787  1.00  0.00           O  \nATOM   1625  CB  HIS A 104      -1.339   1.475  27.579  1.00  0.00           C  \nATOM   1626  CG  HIS A 104      -1.686   2.130  26.264  1.00  0.00           C  \nATOM   1627  ND1 HIS A 104      -1.773   1.426  25.075  1.00  0.00           N  \nATOM   1628  CD2 HIS A 104      -1.968   3.430  25.963  1.00  0.00           C  \nATOM   1629  CE1 HIS A 104      -2.092   2.275  24.109  1.00  0.00           C  \nATOM   1630  NE2 HIS A 104      -2.212   3.516  24.661  1.00  0.00           N  \nATOM   1631  H   HIS A 104       0.235   2.227  29.362  1.00  0.00           H  \nATOM   1632  HA  HIS A 104      -2.264   3.184  28.456  1.00  0.00           H  \nATOM   1633  HB2 HIS A 104      -0.261   1.318  27.622  1.00  0.00           H  \nATOM   1634  HB3 HIS A 104      -1.806   0.491  27.617  1.00  0.00           H  \nATOM   1635  HD1 HIS A 104      -1.620   0.444  24.964  1.00  0.00           H  \nATOM   1636  HD2 HIS A 104      -1.989   4.258  26.672  1.00  0.00           H  \nATOM   1637  HE1 HIS A 104      -2.234   2.025  23.058  1.00  0.00           H  \nATOM   1638  N   GLY A 105      -2.197   0.491  30.369  1.00  0.00           N  \nATOM   1639  CA  GLY A 105      -2.995  -0.331  31.263  1.00  0.00           C  \nATOM   1640  C   GLY A 105      -2.900  -1.809  30.880  1.00  0.00           C  \nATOM   1641  O   GLY A 105      -3.523  -2.659  31.514  1.00  0.00           O  \nATOM   1642  H   GLY A 105      -1.235   0.229  30.301  1.00  0.00           H  \nATOM   1643  HA2 GLY A 105      -2.655  -0.194  32.289  1.00  0.00           H  \nATOM   1644  HA3 GLY A 105      -4.036  -0.009  31.227  1.00  0.00           H  \nATOM   1645  N   ILE A 106      -2.115  -2.070  29.846  1.00  0.00           N  \nATOM   1646  CA  ILE A 106      -1.930  -3.431  29.371  1.00  0.00           C  \nATOM   1647  C   ILE A 106      -0.696  -4.037  30.044  1.00  0.00           C  \nATOM   1648  O   ILE A 106       0.410  -3.949  29.515  1.00  0.00           O  \nATOM   1649  CB  ILE A 106      -1.875  -3.463  27.842  1.00  0.00           C  \nATOM   1650  CG1 ILE A 106      -3.277  -3.597  27.246  1.00  0.00           C  \nATOM   1651  CG2 ILE A 106      -0.934  -4.565  27.350  1.00  0.00           C  \nATOM   1652  CD1 ILE A 106      -3.577  -2.444  26.286  1.00  0.00           C  \nATOM   1653  H   ILE A 106      -1.612  -1.373  29.336  1.00  0.00           H  \nATOM   1654  HA  ILE A 106      -2.806  -4.004  29.674  1.00  0.00           H  \nATOM   1655  HB  ILE A 106      -1.467  -2.513  27.496  1.00  0.00           H  \nATOM   1656 HG12 ILE A 106      -3.362  -4.546  26.717  1.00  0.00           H  \nATOM   1657 HG13 ILE A 106      -4.017  -3.610  28.046  1.00  0.00           H  \nATOM   1658 HG21 ILE A 106      -0.899  -5.368  28.087  1.00  0.00           H  \nATOM   1659 HG22 ILE A 106      -1.300  -4.958  26.402  1.00  0.00           H  \nATOM   1660 HG23 ILE A 106       0.066  -4.154  27.212  1.00  0.00           H  \nATOM   1661 HD11 ILE A 106      -4.289  -2.776  25.530  1.00  0.00           H  \nATOM   1662 HD12 ILE A 106      -4.002  -1.609  26.843  1.00  0.00           H  \nATOM   1663 HD13 ILE A 106      -2.655  -2.125  25.801  1.00  0.00           H  \nATOM   1664  N   GLU A 107      -0.929  -4.641  31.200  1.00  0.00           N  \nATOM   1665  CA  GLU A 107       0.149  -5.262  31.950  1.00  0.00           C  \nATOM   1666  C   GLU A 107       0.580  -6.567  31.278  1.00  0.00           C  \nATOM   1667  O   GLU A 107       1.732  -6.981  31.400  1.00  0.00           O  \nATOM   1668  CB  GLU A 107      -0.263  -5.504  33.404  1.00  0.00           C  \nATOM   1669  CG  GLU A 107      -1.339  -6.587  33.496  1.00  0.00           C  \nATOM   1670  CD  GLU A 107      -1.588  -6.990  34.951  1.00  0.00           C  \nATOM   1671  OE1 GLU A 107      -0.597  -7.368  35.613  1.00  0.00           O  \nATOM   1672  OE2 GLU A 107      -2.764  -6.912  35.368  1.00  0.00           O  \nATOM   1673  H   GLU A 107      -1.832  -4.709  31.624  1.00  0.00           H  \nATOM   1674  HA  GLU A 107       0.970  -4.545  31.926  1.00  0.00           H  \nATOM   1675  HB2 GLU A 107       0.608  -5.801  33.988  1.00  0.00           H  \nATOM   1676  HB3 GLU A 107      -0.637  -4.577  33.839  1.00  0.00           H  \nATOM   1677  HG2 GLU A 107      -2.265  -6.224  33.052  1.00  0.00           H  \nATOM   1678  HG3 GLU A 107      -1.031  -7.461  32.921  1.00  0.00           H  \nATOM   1679  N   LYS A 108      -0.368  -7.179  30.584  1.00  0.00           N  \nATOM   1680  CA  LYS A 108      -0.101  -8.429  29.892  1.00  0.00           C  \nATOM   1681  C   LYS A 108       1.226  -8.316  29.138  1.00  0.00           C  \nATOM   1682  O   LYS A 108       1.917  -9.313  28.938  1.00  0.00           O  \nATOM   1683  CB  LYS A 108      -1.284  -8.812  29.001  1.00  0.00           C  \nATOM   1684  CG  LYS A 108      -1.528  -7.750  27.927  1.00  0.00           C  \nATOM   1685  CD  LYS A 108      -2.208  -8.358  26.699  1.00  0.00           C  \nATOM   1686  CE  LYS A 108      -1.177  -8.964  25.745  1.00  0.00           C  \nATOM   1687  NZ  LYS A 108      -1.843  -9.511  24.542  1.00  0.00           N  \nATOM   1688  H   LYS A 108      -1.302  -6.836  30.490  1.00  0.00           H  \nATOM   1689  HA  LYS A 108      -0.002  -9.207  30.650  1.00  0.00           H  \nATOM   1690  HB2 LYS A 108      -1.092  -9.775  28.528  1.00  0.00           H  \nATOM   1691  HB3 LYS A 108      -2.180  -8.930  29.611  1.00  0.00           H  \nATOM   1692  HG2 LYS A 108      -2.150  -6.952  28.334  1.00  0.00           H  \nATOM   1693  HG3 LYS A 108      -0.580  -7.298  27.636  1.00  0.00           H  \nATOM   1694  HD2 LYS A 108      -2.914  -9.127  27.014  1.00  0.00           H  \nATOM   1695  HD3 LYS A 108      -2.783  -7.591  26.180  1.00  0.00           H  \nATOM   1696  HE2 LYS A 108      -0.453  -8.204  25.453  1.00  0.00           H  \nATOM   1697  HE3 LYS A 108      -0.624  -9.754  26.252  1.00  0.00           H  \nATOM   1698  HZ1 LYS A 108      -1.670  -8.909  23.762  1.00  0.00           H  \nATOM   1699  HZ2 LYS A 108      -1.482 -10.422  24.346  1.00  0.00           H  \nATOM   1700  HZ3 LYS A 108      -2.828  -9.571  24.704  1.00  0.00           H  \nATOM   1701  N   VAL A 109       1.541  -7.092  28.740  1.00  0.00           N  \nATOM   1702  CA  VAL A 109       2.772  -6.836  28.012  1.00  0.00           C  \nATOM   1703  C   VAL A 109       3.963  -7.302  28.853  1.00  0.00           C  \nATOM   1704  O   VAL A 109       4.950  -7.800  28.314  1.00  0.00           O  \nATOM   1705  CB  VAL A 109       2.855  -5.357  27.628  1.00  0.00           C  \nATOM   1706  CG1 VAL A 109       3.553  -4.545  28.720  1.00  0.00           C  \nATOM   1707  CG2 VAL A 109       3.556  -5.179  26.280  1.00  0.00           C  \nATOM   1708  H   VAL A 109       0.973  -6.286  28.907  1.00  0.00           H  \nATOM   1709  HA  VAL A 109       2.740  -7.422  27.094  1.00  0.00           H  \nATOM   1710  HB  VAL A 109       1.837  -4.979  27.528  1.00  0.00           H  \nATOM   1711 HG11 VAL A 109       4.590  -4.869  28.807  1.00  0.00           H  \nATOM   1712 HG12 VAL A 109       3.524  -3.486  28.461  1.00  0.00           H  \nATOM   1713 HG13 VAL A 109       3.043  -4.700  29.671  1.00  0.00           H  \nATOM   1714 HG21 VAL A 109       4.274  -4.363  26.349  1.00  0.00           H  \nATOM   1715 HG22 VAL A 109       4.077  -6.100  26.017  1.00  0.00           H  \nATOM   1716 HG23 VAL A 109       2.816  -4.949  25.513  1.00  0.00           H  \nATOM   1717  N   ALA A 110       3.830  -7.125  30.159  1.00  0.00           N  \nATOM   1718  CA  ALA A 110       4.883  -7.521  31.079  1.00  0.00           C  \nATOM   1719  C   ALA A 110       4.983  -9.048  31.105  1.00  0.00           C  \nATOM   1720  O   ALA A 110       6.072  -9.599  31.251  1.00  0.00           O  \nATOM   1721  CB  ALA A 110       4.601  -6.931  32.462  1.00  0.00           C  \nATOM   1722  H   ALA A 110       3.024  -6.719  30.589  1.00  0.00           H  \nATOM   1723  HA  ALA A 110       5.821  -7.110  30.706  1.00  0.00           H  \nATOM   1724  HB1 ALA A 110       3.806  -7.500  32.944  1.00  0.00           H  \nATOM   1725  HB2 ALA A 110       5.504  -6.983  33.070  1.00  0.00           H  \nATOM   1726  HB3 ALA A 110       4.292  -5.892  32.358  1.00  0.00           H  \nATOM   1727  N   GLU A 111       3.831  -9.687  30.961  1.00  0.00           N  \nATOM   1728  CA  GLU A 111       3.775 -11.139  30.966  1.00  0.00           C  \nATOM   1729  C   GLU A 111       4.396 -11.698  29.684  1.00  0.00           C  \nATOM   1730  O   GLU A 111       5.189 -12.637  29.733  1.00  0.00           O  \nATOM   1731  CB  GLU A 111       2.338 -11.632  31.140  1.00  0.00           C  \nATOM   1732  CG  GLU A 111       2.307 -13.013  31.796  1.00  0.00           C  \nATOM   1733  CD  GLU A 111       2.044 -12.900  33.300  1.00  0.00           C  \nATOM   1734  OE1 GLU A 111       2.821 -12.177  33.959  1.00  0.00           O  \nATOM   1735  OE2 GLU A 111       1.072 -13.540  33.755  1.00  0.00           O  \nATOM   1736  H   GLU A 111       2.949  -9.230  30.842  1.00  0.00           H  \nATOM   1737  HA  GLU A 111       4.366 -11.448  31.828  1.00  0.00           H  \nATOM   1738  HB2 GLU A 111       1.778 -10.923  31.750  1.00  0.00           H  \nATOM   1739  HB3 GLU A 111       1.844 -11.676  30.169  1.00  0.00           H  \nATOM   1740  HG2 GLU A 111       1.532 -13.623  31.333  1.00  0.00           H  \nATOM   1741  HG3 GLU A 111       3.256 -13.521  31.627  1.00  0.00           H  \nATOM   1742  N   GLN A 112       4.010 -11.099  28.567  1.00  0.00           N  \nATOM   1743  CA  GLN A 112       4.518 -11.526  27.275  1.00  0.00           C  \nATOM   1744  C   GLN A 112       5.990 -11.134  27.128  1.00  0.00           C  \nATOM   1745  O   GLN A 112       6.758 -11.831  26.466  1.00  0.00           O  \nATOM   1746  CB  GLN A 112       3.680 -10.942  26.136  1.00  0.00           C  \nATOM   1747  CG  GLN A 112       4.222  -9.581  25.697  1.00  0.00           C  \nATOM   1748  CD  GLN A 112       3.324  -8.951  24.630  1.00  0.00           C  \nATOM   1749  OE1 GLN A 112       3.760  -8.590  23.549  1.00  0.00           O  \nATOM   1750  NE2 GLN A 112       2.049  -8.840  24.992  1.00  0.00           N  \nATOM   1751  H   GLN A 112       3.364 -10.336  28.537  1.00  0.00           H  \nATOM   1752  HA  GLN A 112       4.422 -12.611  27.270  1.00  0.00           H  \nATOM   1753  HB2 GLN A 112       3.682 -11.629  25.289  1.00  0.00           H  \nATOM   1754  HB3 GLN A 112       2.644 -10.838  26.459  1.00  0.00           H  \nATOM   1755  HG2 GLN A 112       4.289  -8.917  26.559  1.00  0.00           H  \nATOM   1756  HG3 GLN A 112       5.232  -9.697  25.305  1.00  0.00           H  \nATOM   1757 HE21 GLN A 112       1.757  -9.156  25.895  1.00  0.00           H  \nATOM   1758 HE22 GLN A 112       1.384  -8.440  24.362  1.00  0.00           H  \nATOM   1759  N   VAL A 113       6.339 -10.021  27.755  1.00  0.00           N  \nATOM   1760  CA  VAL A 113       7.705  -9.528  27.703  1.00  0.00           C  \nATOM   1761  C   VAL A 113       8.595 -10.411  28.580  1.00  0.00           C  \nATOM   1762  O   VAL A 113       9.745 -10.678  28.234  1.00  0.00           O  \nATOM   1763  CB  VAL A 113       7.745  -8.053  28.104  1.00  0.00           C  \nATOM   1764  CG1 VAL A 113       9.177  -7.607  28.410  1.00  0.00           C  \nATOM   1765  CG2 VAL A 113       7.116  -7.173  27.022  1.00  0.00           C  \nATOM   1766  H   VAL A 113       5.708  -9.460  28.292  1.00  0.00           H  \nATOM   1767  HA  VAL A 113       8.043  -9.607  26.669  1.00  0.00           H  \nATOM   1768  HB  VAL A 113       7.157  -7.937  29.014  1.00  0.00           H  \nATOM   1769 HG11 VAL A 113       9.868  -8.415  28.172  1.00  0.00           H  \nATOM   1770 HG12 VAL A 113       9.421  -6.731  27.809  1.00  0.00           H  \nATOM   1771 HG13 VAL A 113       9.261  -7.356  29.468  1.00  0.00           H  \nATOM   1772 HG21 VAL A 113       7.864  -6.939  26.264  1.00  0.00           H  \nATOM   1773 HG22 VAL A 113       6.285  -7.704  26.560  1.00  0.00           H  \nATOM   1774 HG23 VAL A 113       6.752  -6.249  27.471  1.00  0.00           H  \nATOM   1775  N   MET A 114       8.029 -10.840  29.699  1.00  0.00           N  \nATOM   1776  CA  MET A 114       8.757 -11.687  30.628  1.00  0.00           C  \nATOM   1777  C   MET A 114       8.939 -13.096  30.060  1.00  0.00           C  \nATOM   1778  O   MET A 114       9.972 -13.728  30.275  1.00  0.00           O  \nATOM   1779  CB  MET A 114       7.994 -11.765  31.952  1.00  0.00           C  \nATOM   1780  CG  MET A 114       8.863 -12.384  33.050  1.00  0.00           C  \nATOM   1781  SD  MET A 114       8.451 -14.108  33.255  1.00  0.00           S  \nATOM   1782  CE  MET A 114       7.087 -13.968  34.398  1.00  0.00           C  \nATOM   1783  H   MET A 114       7.094 -10.618  29.973  1.00  0.00           H  \nATOM   1784  HA  MET A 114       9.729 -11.212  30.758  1.00  0.00           H  \nATOM   1785  HB2 MET A 114       7.678 -10.767  32.254  1.00  0.00           H  \nATOM   1786  HB3 MET A 114       7.090 -12.360  31.821  1.00  0.00           H  \nATOM   1787  HG2 MET A 114       9.917 -12.281  32.792  1.00  0.00           H  \nATOM   1788  HG3 MET A 114       8.711 -11.851  33.989  1.00  0.00           H  \nATOM   1789  HE1 MET A 114       6.682 -12.957  34.357  1.00  0.00           H  \nATOM   1790  HE2 MET A 114       6.310 -14.682  34.126  1.00  0.00           H  \nATOM   1791  HE3 MET A 114       7.437 -14.179  35.408  1.00  0.00           H  \nATOM   1792  N   LYS A 115       7.919 -13.547  29.344  1.00  0.00           N  \nATOM   1793  CA  LYS A 115       7.953 -14.869  28.743  1.00  0.00           C  \nATOM   1794  C   LYS A 115       8.872 -14.845  27.520  1.00  0.00           C  \nATOM   1795  O   LYS A 115       9.538 -15.834  27.220  1.00  0.00           O  \nATOM   1796  CB  LYS A 115       6.535 -15.355  28.437  1.00  0.00           C  \nATOM   1797  CG  LYS A 115       6.059 -16.356  29.492  1.00  0.00           C  \nATOM   1798  CD  LYS A 115       6.880 -17.646  29.434  1.00  0.00           C  \nATOM   1799  CE  LYS A 115       6.070 -18.785  28.812  1.00  0.00           C  \nATOM   1800  NZ  LYS A 115       5.301 -19.503  29.853  1.00  0.00           N  \nATOM   1801  H   LYS A 115       7.082 -13.026  29.174  1.00  0.00           H  \nATOM   1802  HA  LYS A 115       8.376 -15.552  29.480  1.00  0.00           H  \nATOM   1803  HB2 LYS A 115       5.853 -14.505  28.404  1.00  0.00           H  \nATOM   1804  HB3 LYS A 115       6.510 -15.821  27.452  1.00  0.00           H  \nATOM   1805  HG2 LYS A 115       6.142 -15.912  30.484  1.00  0.00           H  \nATOM   1806  HG3 LYS A 115       5.005 -16.584  29.332  1.00  0.00           H  \nATOM   1807  HD2 LYS A 115       7.786 -17.478  28.851  1.00  0.00           H  \nATOM   1808  HD3 LYS A 115       7.195 -17.926  30.439  1.00  0.00           H  \nATOM   1809  HE2 LYS A 115       5.389 -18.386  28.060  1.00  0.00           H  \nATOM   1810  HE3 LYS A 115       6.738 -19.478  28.301  1.00  0.00           H  \nATOM   1811  HZ1 LYS A 115       5.922 -20.074  30.390  1.00  0.00           H  \nATOM   1812  HZ2 LYS A 115       4.854 -18.841  30.454  1.00  0.00           H  \nATOM   1813  HZ3 LYS A 115       4.610 -20.080  29.418  1.00  0.00           H  \nATOM   1814  N   ALA A 116       8.878 -13.704  26.847  1.00  0.00           N  \nATOM   1815  CA  ALA A 116       9.704 -13.537  25.663  1.00  0.00           C  \nATOM   1816  C   ALA A 116      11.180 -13.593  26.065  1.00  0.00           C  \nATOM   1817  O   ALA A 116      11.990 -14.215  25.380  1.00  0.00           O  \nATOM   1818  CB  ALA A 116       9.339 -12.226  24.965  1.00  0.00           C  \nATOM   1819  H   ALA A 116       8.333 -12.903  27.098  1.00  0.00           H  \nATOM   1820  HA  ALA A 116       9.488 -14.366  24.989  1.00  0.00           H  \nATOM   1821  HB1 ALA A 116       9.000 -11.502  25.706  1.00  0.00           H  \nATOM   1822  HB2 ALA A 116      10.214 -11.833  24.448  1.00  0.00           H  \nATOM   1823  HB3 ALA A 116       8.542 -12.408  24.244  1.00  0.00           H  \nATOM   1824  N   ASP A 117      11.483 -12.934  27.173  1.00  0.00           N  \nATOM   1825  CA  ASP A 117      12.847 -12.900  27.674  1.00  0.00           C  \nATOM   1826  C   ASP A 117      13.097 -14.133  28.545  1.00  0.00           C  \nATOM   1827  O   ASP A 117      13.003 -14.062  29.769  1.00  0.00           O  \nATOM   1828  CB  ASP A 117      13.088 -11.658  28.534  1.00  0.00           C  \nATOM   1829  CG  ASP A 117      14.530 -11.468  29.007  1.00  0.00           C  \nATOM   1830  OD1 ASP A 117      15.160 -12.497  29.335  1.00  0.00           O  \nATOM   1831  OD2 ASP A 117      14.971 -10.299  29.031  1.00  0.00           O  \nATOM   1832  H   ASP A 117      10.818 -12.430  27.725  1.00  0.00           H  \nATOM   1833  HA  ASP A 117      13.478 -12.882  26.785  1.00  0.00           H  \nATOM   1834  HB2 ASP A 117      12.789 -10.777  27.965  1.00  0.00           H  \nATOM   1835  HB3 ASP A 117      12.438 -11.708  29.408  1.00  0.00           H  \nATOM   1836  N   ALA A 118      13.409 -15.235  27.878  1.00  0.00           N  \nATOM   1837  CA  ALA A 118      13.673 -16.482  28.576  1.00  0.00           C  \nATOM   1838  C   ALA A 118      15.164 -16.570  28.909  1.00  0.00           C  \nATOM   1839  O   ALA A 118      15.804 -17.585  28.642  1.00  0.00           O  \nATOM   1840  CB  ALA A 118      13.196 -17.657  27.720  1.00  0.00           C  \nATOM   1841  H   ALA A 118      13.483 -15.284  26.882  1.00  0.00           H  \nATOM   1842  HA  ALA A 118      13.102 -16.470  29.505  1.00  0.00           H  \nATOM   1843  HB1 ALA A 118      12.254 -18.037  28.116  1.00  0.00           H  \nATOM   1844  HB2 ALA A 118      13.051 -17.323  26.693  1.00  0.00           H  \nATOM   1845  HB3 ALA A 118      13.945 -18.449  27.741  1.00  0.00           H  \nATOM   1846  N   ASN A 119      15.673 -15.492  29.487  1.00  0.00           N  \nATOM   1847  CA  ASN A 119      17.076 -15.435  29.859  1.00  0.00           C  \nATOM   1848  C   ASN A 119      17.192 -15.308  31.379  1.00  0.00           C  \nATOM   1849  O   ASN A 119      17.955 -16.038  32.010  1.00  0.00           O  \nATOM   1850  CB  ASN A 119      17.764 -14.221  29.230  1.00  0.00           C  \nATOM   1851  CG  ASN A 119      17.350 -14.054  27.767  1.00  0.00           C  \nATOM   1852  OD1 ASN A 119      16.990 -12.979  27.315  1.00  0.00           O  \nATOM   1853  ND2 ASN A 119      17.422 -15.174  27.053  1.00  0.00           N  \nATOM   1854  H   ASN A 119      15.145 -14.670  29.701  1.00  0.00           H  \nATOM   1855  HA  ASN A 119      17.510 -16.361  29.483  1.00  0.00           H  \nATOM   1856  HB2 ASN A 119      17.505 -13.322  29.790  1.00  0.00           H  \nATOM   1857  HB3 ASN A 119      18.845 -14.337  29.295  1.00  0.00           H  \nATOM   1858 HD21 ASN A 119      17.725 -16.023  27.485  1.00  0.00           H  \nATOM   1859 HD22 ASN A 119      17.172 -15.167  26.085  1.00  0.00           H  \nATOM   1860  N   GLY A 120      16.422 -14.377  31.924  1.00  0.00           N  \nATOM   1861  CA  GLY A 120      16.429 -14.146  33.358  1.00  0.00           C  \nATOM   1862  C   GLY A 120      17.203 -12.872  33.705  1.00  0.00           C  \nATOM   1863  O   GLY A 120      17.195 -12.429  34.853  1.00  0.00           O  \nATOM   1864  H   GLY A 120      15.804 -13.788  31.404  1.00  0.00           H  \nATOM   1865  HA2 GLY A 120      15.404 -14.063  33.722  1.00  0.00           H  \nATOM   1866  HA3 GLY A 120      16.879 -14.999  33.866  1.00  0.00           H  \nATOM   1867  N   ASP A 121      17.854 -12.319  32.692  1.00  0.00           N  \nATOM   1868  CA  ASP A 121      18.631 -11.105  32.875  1.00  0.00           C  \nATOM   1869  C   ASP A 121      17.701  -9.968  33.300  1.00  0.00           C  \nATOM   1870  O   ASP A 121      17.988  -9.251  34.258  1.00  0.00           O  \nATOM   1871  CB  ASP A 121      19.320 -10.691  31.573  1.00  0.00           C  \nATOM   1872  CG  ASP A 121      18.383 -10.499  30.379  1.00  0.00           C  \nATOM   1873  OD1 ASP A 121      17.300 -11.122  30.401  1.00  0.00           O  \nATOM   1874  OD2 ASP A 121      18.772  -9.734  29.470  1.00  0.00           O  \nATOM   1875  H   ASP A 121      17.855 -12.686  31.761  1.00  0.00           H  \nATOM   1876  HA  ASP A 121      19.368 -11.350  33.640  1.00  0.00           H  \nATOM   1877  HB2 ASP A 121      19.860  -9.760  31.746  1.00  0.00           H  \nATOM   1878  HB3 ASP A 121      20.062 -11.446  31.316  1.00  0.00           H  \nATOM   1879  N   GLY A 122      16.604  -9.838  32.568  1.00  0.00           N  \nATOM   1880  CA  GLY A 122      15.629  -8.800  32.858  1.00  0.00           C  \nATOM   1881  C   GLY A 122      15.914  -7.537  32.043  1.00  0.00           C  \nATOM   1882  O   GLY A 122      15.570  -6.433  32.461  1.00  0.00           O  \nATOM   1883  H   GLY A 122      16.377 -10.426  31.792  1.00  0.00           H  \nATOM   1884  HA2 GLY A 122      14.627  -9.164  32.633  1.00  0.00           H  \nATOM   1885  HA3 GLY A 122      15.651  -8.563  33.922  1.00  0.00           H  \nATOM   1886  N   TYR A 123      16.541  -7.743  30.893  1.00  0.00           N  \nATOM   1887  CA  TYR A 123      16.876  -6.635  30.015  1.00  0.00           C  \nATOM   1888  C   TYR A 123      16.911  -7.084  28.553  1.00  0.00           C  \nATOM   1889  O   TYR A 123      17.053  -8.273  28.269  1.00  0.00           O  \nATOM   1890  CB  TYR A 123      18.277  -6.178  30.429  1.00  0.00           C  \nATOM   1891  CG  TYR A 123      18.365  -5.671  31.870  1.00  0.00           C  \nATOM   1892  CD1 TYR A 123      18.352  -6.568  32.919  1.00  0.00           C  \nATOM   1893  CD2 TYR A 123      18.459  -4.317  32.121  1.00  0.00           C  \nATOM   1894  CE1 TYR A 123      18.435  -6.091  34.275  1.00  0.00           C  \nATOM   1895  CE2 TYR A 123      18.542  -3.840  33.477  1.00  0.00           C  \nATOM   1896  CZ  TYR A 123      18.526  -4.750  34.487  1.00  0.00           C  \nATOM   1897  OH  TYR A 123      18.605  -4.300  35.768  1.00  0.00           O  \nATOM   1898  H   TYR A 123      16.818  -8.644  30.560  1.00  0.00           H  \nATOM   1899  HA  TYR A 123      16.111  -5.868  30.134  1.00  0.00           H  \nATOM   1900  HB2 TYR A 123      18.971  -7.009  30.304  1.00  0.00           H  \nATOM   1901  HB3 TYR A 123      18.603  -5.385  29.756  1.00  0.00           H  \nATOM   1902  HD1 TYR A 123      18.278  -7.637  32.721  1.00  0.00           H  \nATOM   1903  HD2 TYR A 123      18.470  -3.609  31.292  1.00  0.00           H  \nATOM   1904  HE1 TYR A 123      18.425  -6.788  35.113  1.00  0.00           H  \nATOM   1905  HE2 TYR A 123      18.617  -2.773  33.689  1.00  0.00           H  \nATOM   1906  HH  TYR A 123      18.852  -3.331  35.775  1.00  0.00           H  \nATOM   1907  N   ILE A 124      16.778  -6.111  27.665  1.00  0.00           N  \nATOM   1908  CA  ILE A 124      16.793  -6.391  26.239  1.00  0.00           C  \nATOM   1909  C   ILE A 124      17.305  -5.162  25.487  1.00  0.00           C  \nATOM   1910  O   ILE A 124      17.363  -4.067  26.045  1.00  0.00           O  \nATOM   1911  CB  ILE A 124      15.416  -6.868  25.773  1.00  0.00           C  \nATOM   1912  CG1 ILE A 124      14.430  -5.701  25.690  1.00  0.00           C  \nATOM   1913  CG2 ILE A 124      14.895  -7.996  26.666  1.00  0.00           C  \nATOM   1914  CD1 ILE A 124      13.218  -6.067  24.831  1.00  0.00           C  \nATOM   1915  H   ILE A 124      16.663  -5.147  27.904  1.00  0.00           H  \nATOM   1916  HA  ILE A 124      17.491  -7.212  26.075  1.00  0.00           H  \nATOM   1917  HB  ILE A 124      15.518  -7.276  24.767  1.00  0.00           H  \nATOM   1918 HG12 ILE A 124      14.101  -5.427  26.693  1.00  0.00           H  \nATOM   1919 HG13 ILE A 124      14.929  -4.828  25.269  1.00  0.00           H  \nATOM   1920 HG21 ILE A 124      14.743  -7.618  27.677  1.00  0.00           H  \nATOM   1921 HG22 ILE A 124      13.949  -8.365  26.271  1.00  0.00           H  \nATOM   1922 HG23 ILE A 124      15.622  -8.808  26.687  1.00  0.00           H  \nATOM   1923 HD11 ILE A 124      13.149  -7.151  24.742  1.00  0.00           H  \nATOM   1924 HD12 ILE A 124      12.312  -5.682  25.300  1.00  0.00           H  \nATOM   1925 HD13 ILE A 124      13.329  -5.627  23.840  1.00  0.00           H  \nATOM   1926  N   THR A 125      17.663  -5.383  24.230  1.00  0.00           N  \nATOM   1927  CA  THR A 125      18.168  -4.306  23.395  1.00  0.00           C  \nATOM   1928  C   THR A 125      17.297  -4.146  22.148  1.00  0.00           C  \nATOM   1929  O   THR A 125      16.341  -4.894  21.953  1.00  0.00           O  \nATOM   1930  CB  THR A 125      19.635  -4.602  23.079  1.00  0.00           C  \nATOM   1931  OG1 THR A 125      19.577  -5.632  22.095  1.00  0.00           O  \nATOM   1932  CG2 THR A 125      20.369  -5.251  24.254  1.00  0.00           C  \nATOM   1933  H   THR A 125      17.612  -6.276  23.783  1.00  0.00           H  \nATOM   1934  HA  THR A 125      18.098  -3.374  23.957  1.00  0.00           H  \nATOM   1935  HB  THR A 125      20.151  -3.701  22.747  1.00  0.00           H  \nATOM   1936  HG1 THR A 125      20.275  -5.477  21.396  1.00  0.00           H  \nATOM   1937 HG21 THR A 125      21.402  -4.905  24.273  1.00  0.00           H  \nATOM   1938 HG22 THR A 125      19.877  -4.974  25.187  1.00  0.00           H  \nATOM   1939 HG23 THR A 125      20.350  -6.335  24.141  1.00  0.00           H  \nATOM   1940  N   LEU A 126      17.660  -3.164  21.334  1.00  0.00           N  \nATOM   1941  CA  LEU A 126      16.924  -2.896  20.111  1.00  0.00           C  \nATOM   1942  C   LEU A 126      17.177  -4.025  19.110  1.00  0.00           C  \nATOM   1943  O   LEU A 126      16.264  -4.451  18.405  1.00  0.00           O  \nATOM   1944  CB  LEU A 126      17.271  -1.507  19.571  1.00  0.00           C  \nATOM   1945  CG  LEU A 126      16.395  -0.995  18.425  1.00  0.00           C  \nATOM   1946  CD1 LEU A 126      15.307  -0.054  18.945  1.00  0.00           C  \nATOM   1947  CD2 LEU A 126      17.246  -0.341  17.335  1.00  0.00           C  \nATOM   1948  H   LEU A 126      18.439  -2.560  21.500  1.00  0.00           H  \nATOM   1949  HA  LEU A 126      15.864  -2.889  20.362  1.00  0.00           H  \nATOM   1950  HB2 LEU A 126      17.211  -0.794  20.393  1.00  0.00           H  \nATOM   1951  HB3 LEU A 126      18.307  -1.518  19.233  1.00  0.00           H  \nATOM   1952  HG  LEU A 126      15.893  -1.849  17.971  1.00  0.00           H  \nATOM   1953 HD11 LEU A 126      14.460  -0.640  19.303  1.00  0.00           H  \nATOM   1954 HD12 LEU A 126      15.705   0.546  19.763  1.00  0.00           H  \nATOM   1955 HD13 LEU A 126      14.979   0.602  18.139  1.00  0.00           H  \nATOM   1956 HD21 LEU A 126      18.251  -0.762  17.356  1.00  0.00           H  \nATOM   1957 HD22 LEU A 126      16.795  -0.527  16.361  1.00  0.00           H  \nATOM   1958 HD23 LEU A 126      17.299   0.734  17.512  1.00  0.00           H  \nATOM   1959  N   GLU A 127      18.422  -4.478  19.080  1.00  0.00           N  \nATOM   1960  CA  GLU A 127      18.807  -5.550  18.178  1.00  0.00           C  \nATOM   1961  C   GLU A 127      18.101  -6.851  18.567  1.00  0.00           C  \nATOM   1962  O   GLU A 127      17.634  -7.590  17.702  1.00  0.00           O  \nATOM   1963  CB  GLU A 127      20.326  -5.734  18.161  1.00  0.00           C  \nATOM   1964  CG  GLU A 127      20.935  -5.150  16.885  1.00  0.00           C  \nATOM   1965  CD  GLU A 127      22.292  -4.505  17.171  1.00  0.00           C  \nATOM   1966  OE1 GLU A 127      22.314  -3.575  18.007  1.00  0.00           O  \nATOM   1967  OE2 GLU A 127      23.278  -4.955  16.548  1.00  0.00           O  \nATOM   1968  H   GLU A 127      19.159  -4.126  19.657  1.00  0.00           H  \nATOM   1969  HA  GLU A 127      18.475  -5.230  17.190  1.00  0.00           H  \nATOM   1970  HB2 GLU A 127      20.764  -5.248  19.033  1.00  0.00           H  \nATOM   1971  HB3 GLU A 127      20.568  -6.794  18.232  1.00  0.00           H  \nATOM   1972  HG2 GLU A 127      21.052  -5.937  16.140  1.00  0.00           H  \nATOM   1973  HG3 GLU A 127      20.258  -4.408  16.461  1.00  0.00           H  \nATOM   1974  N   GLU A 128      18.046  -7.090  19.869  1.00  0.00           N  \nATOM   1975  CA  GLU A 128      17.406  -8.289  20.383  1.00  0.00           C  \nATOM   1976  C   GLU A 128      15.884  -8.160  20.286  1.00  0.00           C  \nATOM   1977  O   GLU A 128      15.185  -9.153  20.092  1.00  0.00           O  \nATOM   1978  CB  GLU A 128      17.842  -8.569  21.822  1.00  0.00           C  \nATOM   1979  CG  GLU A 128      17.115  -9.791  22.387  1.00  0.00           C  \nATOM   1980  CD  GLU A 128      17.706 -10.206  23.736  1.00  0.00           C  \nATOM   1981  OE1 GLU A 128      18.324  -9.329  24.379  1.00  0.00           O  \nATOM   1982  OE2 GLU A 128      17.528 -11.390  24.094  1.00  0.00           O  \nATOM   1983  H   GLU A 128      18.429  -6.483  20.566  1.00  0.00           H  \nATOM   1984  HA  GLU A 128      17.750  -9.099  19.741  1.00  0.00           H  \nATOM   1985  HB2 GLU A 128      18.918  -8.735  21.854  1.00  0.00           H  \nATOM   1986  HB3 GLU A 128      17.635  -7.698  22.445  1.00  0.00           H  \nATOM   1987  HG2 GLU A 128      16.055  -9.566  22.505  1.00  0.00           H  \nATOM   1988  HG3 GLU A 128      17.189 -10.620  21.684  1.00  0.00           H  \nATOM   1989  N   PHE A 129      15.416  -6.929  20.426  1.00  0.00           N  \nATOM   1990  CA  PHE A 129      13.990  -6.658  20.357  1.00  0.00           C  \nATOM   1991  C   PHE A 129      13.454  -6.901  18.944  1.00  0.00           C  \nATOM   1992  O   PHE A 129      12.351  -7.418  18.776  1.00  0.00           O  \nATOM   1993  CB  PHE A 129      13.797  -5.183  20.716  1.00  0.00           C  \nATOM   1994  CG  PHE A 129      12.355  -4.692  20.577  1.00  0.00           C  \nATOM   1995  CD1 PHE A 129      11.805  -4.537  19.343  1.00  0.00           C  \nATOM   1996  CD2 PHE A 129      11.622  -4.412  21.688  1.00  0.00           C  \nATOM   1997  CE1 PHE A 129      10.466  -4.081  19.214  1.00  0.00           C  \nATOM   1998  CE2 PHE A 129      10.283  -3.957  21.559  1.00  0.00           C  \nATOM   1999  CZ  PHE A 129       9.733  -3.801  20.325  1.00  0.00           C  \nATOM   2000  H   PHE A 129      15.991  -6.126  20.584  1.00  0.00           H  \nATOM   2001  HA  PHE A 129      13.497  -7.337  21.052  1.00  0.00           H  \nATOM   2002  HB2 PHE A 129      14.127  -5.023  21.742  1.00  0.00           H  \nATOM   2003  HB3 PHE A 129      14.439  -4.577  20.076  1.00  0.00           H  \nATOM   2004  HD1 PHE A 129      12.392  -4.761  18.452  1.00  0.00           H  \nATOM   2005  HD2 PHE A 129      12.062  -4.537  22.677  1.00  0.00           H  \nATOM   2006  HE1 PHE A 129      10.025  -3.956  18.225  1.00  0.00           H  \nATOM   2007  HE2 PHE A 129       9.696  -3.732  22.450  1.00  0.00           H  \nATOM   2008  HZ  PHE A 129       8.706  -3.452  20.226  1.00  0.00           H  \nATOM   2009  N   LEU A 130      14.259  -6.516  17.966  1.00  0.00           N  \nATOM   2010  CA  LEU A 130      13.880  -6.686  16.573  1.00  0.00           C  \nATOM   2011  C   LEU A 130      14.018  -8.160  16.188  1.00  0.00           C  \nATOM   2012  O   LEU A 130      13.059  -8.777  15.726  1.00  0.00           O  \nATOM   2013  CB  LEU A 130      14.682  -5.738  15.680  1.00  0.00           C  \nATOM   2014  CG  LEU A 130      14.244  -5.665  14.216  1.00  0.00           C  \nATOM   2015  CD1 LEU A 130      14.118  -4.211  13.754  1.00  0.00           C  \nATOM   2016  CD2 LEU A 130      15.186  -6.471  13.320  1.00  0.00           C  \nATOM   2017  H   LEU A 130      15.155  -6.096  18.111  1.00  0.00           H  \nATOM   2018  HA  LEU A 130      12.831  -6.402  16.484  1.00  0.00           H  \nATOM   2019  HB2 LEU A 130      14.629  -4.736  16.105  1.00  0.00           H  \nATOM   2020  HB3 LEU A 130      15.729  -6.042  15.710  1.00  0.00           H  \nATOM   2021  HG  LEU A 130      13.255  -6.116  14.132  1.00  0.00           H  \nATOM   2022 HD11 LEU A 130      15.072  -3.703  13.891  1.00  0.00           H  \nATOM   2023 HD12 LEU A 130      13.842  -4.188  12.699  1.00  0.00           H  \nATOM   2024 HD13 LEU A 130      13.349  -3.709  14.341  1.00  0.00           H  \nATOM   2025 HD21 LEU A 130      14.632  -7.284  12.849  1.00  0.00           H  \nATOM   2026 HD22 LEU A 130      15.601  -5.821  12.550  1.00  0.00           H  \nATOM   2027 HD23 LEU A 130      15.995  -6.885  13.922  1.00  0.00           H  \nATOM   2028  N   GLU A 131      15.218  -8.682  16.391  1.00  0.00           N  \nATOM   2029  CA  GLU A 131      15.494 -10.072  16.071  1.00  0.00           C  \nATOM   2030  C   GLU A 131      14.508 -10.990  16.795  1.00  0.00           C  \nATOM   2031  O   GLU A 131      14.181 -12.068  16.303  1.00  0.00           O  \nATOM   2032  CB  GLU A 131      16.939 -10.438  16.416  1.00  0.00           C  \nATOM   2033  CG  GLU A 131      17.527 -11.393  15.375  1.00  0.00           C  \nATOM   2034  CD  GLU A 131      17.653 -12.809  15.941  1.00  0.00           C  \nATOM   2035  OE1 GLU A 131      18.256 -12.935  17.028  1.00  0.00           O  \nATOM   2036  OE2 GLU A 131      17.143 -13.735  15.272  1.00  0.00           O  \nATOM   2037  H   GLU A 131      15.993  -8.173  16.767  1.00  0.00           H  \nATOM   2038  HA  GLU A 131      15.353 -10.154  14.993  1.00  0.00           H  \nATOM   2039  HB2 GLU A 131      17.545  -9.533  16.467  1.00  0.00           H  \nATOM   2040  HB3 GLU A 131      16.975 -10.901  17.402  1.00  0.00           H  \nATOM   2041  HG2 GLU A 131      16.892 -11.407  14.489  1.00  0.00           H  \nATOM   2042  HG3 GLU A 131      18.507 -11.035  15.060  1.00  0.00           H  \nATOM   2043  N   PHE A 132      14.062 -10.528  17.954  1.00  0.00           N  \nATOM   2044  CA  PHE A 132      13.119 -11.294  18.752  1.00  0.00           C  \nATOM   2045  C   PHE A 132      11.831 -11.562  17.971  1.00  0.00           C  \nATOM   2046  O   PHE A 132      11.234 -12.630  18.099  1.00  0.00           O  \nATOM   2047  CB  PHE A 132      12.788 -10.451  19.985  1.00  0.00           C  \nATOM   2048  CG  PHE A 132      11.448 -10.800  20.636  1.00  0.00           C  \nATOM   2049  CD1 PHE A 132      11.192 -12.078  21.022  1.00  0.00           C  \nATOM   2050  CD2 PHE A 132      10.513  -9.831  20.828  1.00  0.00           C  \nATOM   2051  CE1 PHE A 132       9.948 -12.402  21.626  1.00  0.00           C  \nATOM   2052  CE2 PHE A 132       9.269 -10.155  21.431  1.00  0.00           C  \nATOM   2053  CZ  PHE A 132       9.013 -11.433  21.818  1.00  0.00           C  \nATOM   2054  H   PHE A 132      14.333  -9.650  18.348  1.00  0.00           H  \nATOM   2055  HA  PHE A 132      13.598 -12.242  18.996  1.00  0.00           H  \nATOM   2056  HB2 PHE A 132      13.581 -10.576  20.722  1.00  0.00           H  \nATOM   2057  HB3 PHE A 132      12.780  -9.399  19.702  1.00  0.00           H  \nATOM   2058  HD1 PHE A 132      11.941 -12.855  20.868  1.00  0.00           H  \nATOM   2059  HD2 PHE A 132      10.718  -8.807  20.518  1.00  0.00           H  \nATOM   2060  HE1 PHE A 132       9.743 -13.427  21.936  1.00  0.00           H  \nATOM   2061  HE2 PHE A 132       8.519  -9.379  21.585  1.00  0.00           H  \nATOM   2062  HZ  PHE A 132       8.058 -11.682  22.281  1.00  0.00           H  \nATOM   2063  N   SER A 133      11.440 -10.574  17.180  1.00  0.00           N  \nATOM   2064  CA  SER A 133      10.234 -10.690  16.378  1.00  0.00           C  \nATOM   2065  C   SER A 133      10.480 -11.627  15.194  1.00  0.00           C  \nATOM   2066  O   SER A 133       9.562 -12.298  14.727  1.00  0.00           O  \nATOM   2067  CB  SER A 133       9.767  -9.320  15.884  1.00  0.00           C  \nATOM   2068  OG  SER A 133       9.186  -9.390  14.584  1.00  0.00           O  \nATOM   2069  H   SER A 133      11.931  -9.709  17.081  1.00  0.00           H  \nATOM   2070  HA  SER A 133       9.482 -11.110  17.047  1.00  0.00           H  \nATOM   2071  HB2 SER A 133       9.040  -8.910  16.584  1.00  0.00           H  \nATOM   2072  HB3 SER A 133      10.614  -8.634  15.866  1.00  0.00           H  \nATOM   2073  HG  SER A 133       8.617 -10.209  14.508  1.00  0.00           H  \nATOM   2074  N   LEU A 134      11.725 -11.641  14.741  1.00  0.00           N  \nATOM   2075  CA  LEU A 134      12.104 -12.484  13.620  1.00  0.00           C  \nATOM   2076  C   LEU A 134      12.194 -13.938  14.087  1.00  0.00           C  \nATOM   2077  O   LEU A 134      11.608 -14.828  13.473  1.00  0.00           O  \nATOM   2078  CB  LEU A 134      13.390 -11.966  12.971  1.00  0.00           C  \nATOM   2079  CG  LEU A 134      13.212 -11.141  11.695  1.00  0.00           C  \nATOM   2080  CD1 LEU A 134      13.059  -9.654  12.021  1.00  0.00           C  \nATOM   2081  CD2 LEU A 134      14.357 -11.397  10.713  1.00  0.00           C  \nATOM   2082  H   LEU A 134      12.467 -11.092  15.126  1.00  0.00           H  \nATOM   2083  HA  LEU A 134      11.314 -12.409  12.873  1.00  0.00           H  \nATOM   2084  HB2 LEU A 134      13.923 -11.358  13.702  1.00  0.00           H  \nATOM   2085  HB3 LEU A 134      14.027 -12.820  12.742  1.00  0.00           H  \nATOM   2086  HG  LEU A 134      12.292 -11.460  11.207  1.00  0.00           H  \nATOM   2087 HD11 LEU A 134      13.594  -9.427  12.943  1.00  0.00           H  \nATOM   2088 HD12 LEU A 134      13.470  -9.059  11.205  1.00  0.00           H  \nATOM   2089 HD13 LEU A 134      12.002  -9.417  12.146  1.00  0.00           H  \nATOM   2090 HD21 LEU A 134      14.514 -12.471  10.610  1.00  0.00           H  \nATOM   2091 HD22 LEU A 134      14.104 -10.972   9.741  1.00  0.00           H  \nATOM   2092 HD23 LEU A 134      15.268 -10.931  11.087  1.00  0.00           H  \nTER    2093      LEU A 134                                                      \nHETATM 2094 CA    CA A 147      14.398  -5.351 -11.632  1.00  0.00          CA  \nHETATM 2095 CA    CA A 159      14.806 -17.000  -6.571  1.00  0.00          CA  \nHETATM 2096 CA    CA A 171      15.920   3.031  27.784  1.00  0.00          CA  \nHETATM 2097 CA    CA A 183      17.441 -10.424  26.705  1.00  0.00          CA  \nCONECT  154 2094                                                                \nCONECT  182 2094                                                                \nCONECT  204 2094                                                                \nCONECT  219 2094                                                                \nCONECT  296 2094                                                                \nCONECT  297 2094                                                                \nCONECT  735 2095                                                                \nCONECT  757 2095                                                                \nCONECT  758 2095                                                                \nCONECT  776 2095                                                                \nCONECT  777 2095                                                                \nCONECT  794 2095                                                                \nCONECT  875 2095                                                                \nCONECT  876 2095                                                                \nCONECT 1392 2096                                                                \nCONECT 1488 2096                                                                \nCONECT 1489 2096                                                                \nCONECT 1831 2097                                                                \nCONECT 1852 2097                                                                \nCONECT 1874 2097                                                                \nCONECT 1889 2097                                                                \nCONECT 1981 2097                                                                \nCONECT 1982 2097                                                                \nCONECT 2094  154  182  204  219                                                 \nCONECT 2094  296  297                                                           \nCONECT 2095  735  757  758  776                                                 \nCONECT 2095  777  794  875  876                                                 \nCONECT 2096 1392 1488 1489                                                      \nCONECT 2097 1831 1852 1874 1889                                                 \nCONECT 2097 1981 1982                                                           \nMASTER      177    0    4    8    4    0    7    6 1056    1   30   11          \nEND                                                                             \n","pdb");
	viewer_17581787016276908.setStyle({"model": 0},{"cartoon": {"color": "blue"}});
	viewer_17581787016276908.addModel("REMARK AtomGroup 2nxq\nATOM      1  N   ALA A   2      28.970 -48.661  11.361  1.00 95.39           N  \nATOM      2  CA  ALA A   2      28.428 -47.416  10.712  1.00 96.52           C  \nATOM      3  C   ALA A   2      28.119 -47.679   9.243  1.00 96.31           C  \nATOM      4  O   ALA A   2      29.022 -48.004   8.467  1.00 95.73           O  \nATOM      5  CB  ALA A   2      29.434 -46.288  10.828  1.00 96.78           C  \nATOM      6  N   GLU A   3      26.860 -47.481   8.855  1.00 95.81           N  \nATOM      7  CA  GLU A   3      26.418 -47.776   7.497  1.00 93.90           C  \nATOM      8  C   GLU A   3      26.517 -46.774   6.356  1.00 89.90           C  \nATOM      9  O   GLU A   3      27.175 -47.064   5.357  1.00 89.17           O  \nATOM     10  CB  GLU A   3      24.995 -48.318   7.555  1.00 96.38           C  \nATOM     11  CG  GLU A   3      24.937 -49.798   7.802  1.00 99.93           C  \nATOM     12  CD  GLU A   3      25.393 -50.601   6.598  1.00102.13           C  \nATOM     13  OE1 GLU A   3      24.731 -51.616   6.301  1.00102.20           O  \nATOM     14  OE2 GLU A   3      26.404 -50.233   5.956  1.00102.58           O  \nATOM     15  N   ALA A   4      25.852 -45.625   6.444  1.00 85.45           N  \nATOM     16  CA  ALA A   4      25.952 -44.689   5.327  1.00 80.75           C  \nATOM     17  C   ALA A   4      27.378 -44.175   5.337  1.00 77.21           C  \nATOM     18  O   ALA A   4      27.930 -43.762   4.310  1.00 78.15           O  \nATOM     19  CB  ALA A   4      24.964 -43.541   5.472  1.00 80.33           C  \nATOM     20  N   LEU A   5      27.978 -44.228   6.519  1.00 71.68           N  \nATOM     21  CA  LEU A   5      29.343 -43.789   6.680  1.00 67.53           C  \nATOM     22  C   LEU A   5      30.291 -44.692   5.904  1.00 64.81           C  \nATOM     23  O   LEU A   5      31.190 -44.200   5.218  1.00 64.07           O  \nATOM     24  CB  LEU A   5      29.739 -43.777   8.154  1.00 66.53           C  \nATOM     25  CG  LEU A   5      31.150 -43.233   8.393  1.00 68.94           C  \nATOM     26  CD1 LEU A   5      31.216 -41.807   7.878  1.00 67.40           C  \nATOM     27  CD2 LEU A   5      31.512 -43.294   9.869  1.00 69.05           C  \nATOM     28  N   PHE A   6      30.106 -46.008   5.995  1.00 61.66           N  \nATOM     29  CA  PHE A   6      31.003 -46.905   5.268  1.00 58.61           C  \nATOM     30  C   PHE A   6      30.792 -46.714   3.772  1.00 57.93           C  \nATOM     31  O   PHE A   6      31.752 -46.700   3.001  1.00 56.67           O  \nATOM     32  CB  PHE A   6      30.773 -48.365   5.690  1.00 55.83           C  \nATOM     33  CG  PHE A   6      31.688 -49.350   5.012  1.00 55.01           C  \nATOM     34  CD1 PHE A   6      31.403 -49.804   3.729  1.00 52.99           C  \nATOM     35  CD2 PHE A   6      32.845 -49.804   5.647  1.00 52.99           C  \nATOM     36  CE1 PHE A   6      32.257 -50.682   3.081  1.00 52.31           C  \nATOM     37  CE2 PHE A   6      33.717 -50.688   5.009  1.00 52.82           C  \nATOM     38  CZ  PHE A   6      33.423 -51.130   3.726  1.00 53.88           C  \nATOM     39  N   LYS A   7      29.534 -46.505   3.385  1.00 58.68           N  \nATOM     40  CA  LYS A   7      29.164 -46.304   1.988  1.00 58.58           C  \nATOM     41  C   LYS A   7      29.702 -44.980   1.494  1.00 57.50           C  \nATOM     42  O   LYS A   7      30.096 -44.858   0.342  1.00 57.37           O  \nATOM     43  CB  LYS A   7      27.642 -46.323   1.827  1.00 60.46           C  \nATOM     44  CG  LYS A   7      27.018 -47.722   1.787  1.00 61.53           C  \nATOM     45  CD  LYS A   7      25.487 -47.619   1.695  1.00 63.08           C  \nATOM     46  CE  LYS A   7      24.910 -48.438   0.529  1.00 62.65           C  \nATOM     47  NZ  LYS A   7      25.277 -47.920  -0.826  1.00 60.21           N  \nATOM     48  N   GLU A   8      29.718 -43.985   2.371  1.00 57.83           N  \nATOM     49  CA  GLU A   8      30.224 -42.670   2.000  1.00 57.05           C  \nATOM     50  C   GLU A   8      31.732 -42.750   1.726  1.00 56.68           C  \nATOM     51  O   GLU A   8      32.235 -42.158   0.770  1.00 56.70           O  \nATOM     52  CB  GLU A   8      29.960 -41.667   3.127  1.00 61.41           C  \nATOM     53  CG  GLU A   8      30.002 -40.198   2.701  1.00 67.90           C  \nATOM     54  CD  GLU A   8      30.126 -39.235   3.885  1.00 72.64           C  \nATOM     55  OE1 GLU A   8      29.667 -39.564   5.008  1.00 74.58           O  \nATOM     56  OE2 GLU A   8      30.679 -38.132   3.685  1.00 73.67           O  \nATOM     57  N   ILE A   9      32.453 -43.492   2.561  1.00 54.12           N  \nATOM     58  CA  ILE A   9      33.893 -43.622   2.388  1.00 55.61           C  \nATOM     59  C   ILE A   9      34.170 -44.529   1.196  1.00 56.75           C  \nATOM     60  O   ILE A   9      35.053 -44.235   0.392  1.00 56.04           O  \nATOM     61  CB  ILE A   9      34.566 -44.217   3.661  1.00 55.68           C  \nATOM     62  CG1 ILE A   9      34.283 -43.322   4.869  1.00 55.66           C  \nATOM     63  CG2 ILE A   9      36.066 -44.328   3.473  1.00 51.15           C  \nATOM     64  CD1 ILE A   9      34.728 -43.909   6.196  1.00 55.80           C  \nATOM     65  N   ASP A  10      33.403 -45.619   1.084  1.00 56.97           N  \nATOM     66  CA  ASP A  10      33.548 -46.593  -0.005  1.00 56.57           C  \nATOM     67  C   ASP A  10      33.006 -45.950  -1.278  1.00 57.43           C  \nATOM     68  O   ASP A  10      31.898 -46.232  -1.715  1.00 58.33           O  \nATOM     69  CB  ASP A  10      32.772 -47.883   0.338  1.00 55.67           C  \nATOM     70  CG  ASP A  10      32.767 -48.909  -0.805  1.00 55.75           C  \nATOM     71  OD1 ASP A  10      33.669 -48.871  -1.657  1.00 54.64           O  \nATOM     72  OD2 ASP A  10      31.870 -49.779  -0.849  1.00 54.83           O  \nATOM     73  N   VAL A  11      33.820 -45.097  -1.883  1.00 58.57           N  \nATOM     74  CA  VAL A  11      33.428 -44.364  -3.080  1.00 59.29           C  \nATOM     75  C   VAL A  11      33.185 -45.111  -4.400  1.00 60.79           C  \nATOM     76  O   VAL A  11      32.339 -44.676  -5.176  1.00 60.37           O  \nATOM     77  CB  VAL A  11      34.422 -43.185  -3.337  1.00 58.68           C  \nATOM     78  CG1 VAL A  11      34.449 -42.812  -4.805  1.00 57.54           C  \nATOM     79  CG2 VAL A  11      34.003 -41.977  -2.516  1.00 56.98           C  \nATOM     80  N   ASN A  12      33.916 -46.185  -4.700  1.00 60.56           N  \nATOM     81  CA  ASN A  12      33.649 -46.887  -5.960  1.00 59.21           C  \nATOM     82  C   ASN A  12      32.650 -48.030  -5.710  1.00 59.37           C  \nATOM     83  O   ASN A  12      32.406 -48.875  -6.573  1.00 59.63           O  \nATOM     84  CB  ASN A  12      34.954 -47.403  -6.600  1.00 58.22           C  \nATOM     85  CG  ASN A  12      35.569 -48.511  -5.829  1.00 59.80           C  \nATOM     86  OD1 ASN A  12      35.250 -48.682  -4.669  1.00 60.92           O  \nATOM     87  ND2 ASN A  12      36.466 -49.274  -6.456  1.00 60.17           N  \nATOM     88  N   GLY A  13      32.084 -48.041  -4.507  1.00 59.98           N  \nATOM     89  CA  GLY A  13      31.096 -49.033  -4.130  1.00 58.82           C  \nATOM     90  C   GLY A  13      31.417 -50.513  -4.170  1.00 59.27           C  \nATOM     91  O   GLY A  13      30.496 -51.302  -4.330  1.00 59.63           O  \nATOM     92  N   ASP A  14      32.665 -50.934  -3.980  1.00 59.65           N  \nATOM     93  CA  ASP A  14      32.945 -52.378  -4.047  1.00 57.53           C  \nATOM     94  C   ASP A  14      32.990 -53.157  -2.733  1.00 56.57           C  \nATOM     95  O   ASP A  14      33.463 -54.300  -2.707  1.00 56.39           O  \nATOM     96  CB  ASP A  14      34.247 -52.646  -4.782  1.00 58.39           C  \nATOM     97  CG  ASP A  14      35.446 -52.088  -4.051  1.00 62.20           C  \nATOM     98  OD1 ASP A  14      35.389 -51.980  -2.797  1.00 61.50           O  \nATOM     99  OD2 ASP A  14      36.444 -51.769  -4.735  1.00 62.18           O  \nATOM    100  N   GLY A  15      32.499 -52.551  -1.654  1.00 56.09           N  \nATOM    101  CA  GLY A  15      32.480 -53.204  -0.352  1.00 54.47           C  \nATOM    102  C   GLY A  15      33.752 -53.098   0.480  1.00 56.37           C  \nATOM    103  O   GLY A  15      33.759 -53.428   1.659  1.00 57.86           O  \nATOM    104  N   ALA A  16      34.840 -52.638  -0.120  1.00 56.22           N  \nATOM    105  CA  ALA A  16      36.091 -52.526   0.613  1.00 56.67           C  \nATOM    106  C   ALA A  16      36.552 -51.076   0.675  1.00 56.43           C  \nATOM    107  O   ALA A  16      36.502 -50.374  -0.319  1.00 55.22           O  \nATOM    108  CB  ALA A  16      37.164 -53.386  -0.062  1.00 56.75           C  \nATOM    109  N   VAL A  17      36.979 -50.616   1.846  1.00 54.67           N  \nATOM    110  CA  VAL A  17      37.484 -49.243   1.955  1.00 52.70           C  \nATOM    111  C   VAL A  17      38.997 -49.337   1.767  1.00 50.54           C  \nATOM    112  O   VAL A  17      39.690 -49.960   2.571  1.00 49.71           O  \nATOM    113  CB  VAL A  17      37.152 -48.619   3.335  1.00 54.42           C  \nATOM    114  CG1 VAL A  17      38.010 -47.394   3.578  1.00 55.59           C  \nATOM    115  CG2 VAL A  17      35.686 -48.222   3.380  1.00 53.15           C  \nATOM    116  N   SER A  18      39.499 -48.757   0.680  1.00 48.41           N  \nATOM    117  CA  SER A  18      40.929 -48.786   0.385  1.00 49.60           C  \nATOM    118  C   SER A  18      41.690 -47.647   1.077  1.00 51.34           C  \nATOM    119  O   SER A  18      41.083 -46.780   1.722  1.00 50.44           O  \nATOM    120  CB  SER A  18      41.165 -48.703  -1.126  1.00 48.21           C  \nATOM    121  OG  SER A  18      40.839 -47.421  -1.639  1.00 53.15           O  \nATOM    122  N   TYR A  19      43.017 -47.656   0.936  1.00 52.54           N  \nATOM    123  CA  TYR A  19      43.849 -46.624   1.538  1.00 53.84           C  \nATOM    124  C   TYR A  19      43.572 -45.288   0.861  1.00 54.56           C  \nATOM    125  O   TYR A  19      43.403 -44.277   1.528  1.00 54.99           O  \nATOM    126  CB  TYR A  19      45.338 -46.979   1.450  1.00 53.86           C  \nATOM    127  CG  TYR A  19      46.206 -45.926   2.087  1.00 51.04           C  \nATOM    128  CD1 TYR A  19      46.154 -45.690   3.461  1.00 49.34           C  \nATOM    129  CD2 TYR A  19      47.040 -45.125   1.310  1.00 50.86           C  \nATOM    130  CE1 TYR A  19      46.906 -44.676   4.043  1.00 51.73           C  \nATOM    131  CE2 TYR A  19      47.799 -44.105   1.882  1.00 50.34           C  \nATOM    132  CZ  TYR A  19      47.729 -43.886   3.245  1.00 52.15           C  \nATOM    133  OH  TYR A  19      48.463 -42.867   3.800  1.00 51.74           O  \nATOM    134  N   GLU A  20      43.496 -45.288  -0.461  1.00 56.85           N  \nATOM    135  CA  GLU A  20      43.177 -44.073  -1.204  1.00 59.12           C  \nATOM    136  C   GLU A  20      41.886 -43.442  -0.696  1.00 58.52           C  \nATOM    137  O   GLU A  20      41.804 -42.235  -0.493  1.00 59.05           O  \nATOM    138  CB  GLU A  20      42.979 -44.404  -2.677  1.00 63.73           C  \nATOM    139  CG  GLU A  20      44.100 -43.959  -3.533  1.00 71.85           C  \nATOM    140  CD  GLU A  20      45.420 -44.265  -2.886  1.00 78.06           C  \nATOM    141  OE1 GLU A  20      45.702 -45.464  -2.650  1.00 80.29           O  \nATOM    142  OE2 GLU A  20      46.173 -43.306  -2.604  1.00 79.85           O  \nATOM    143  N   GLU A  21      40.872 -44.277  -0.512  1.00 57.11           N  \nATOM    144  CA  GLU A  21      39.581 -43.803  -0.069  1.00 55.00           C  \nATOM    145  C   GLU A  21      39.586 -43.251   1.344  1.00 52.76           C  \nATOM    146  O   GLU A  21      38.871 -42.291   1.641  1.00 51.69           O  \nATOM    147  CB  GLU A  21      38.561 -44.927  -0.232  1.00 54.54           C  \nATOM    148  CG  GLU A  21      38.445 -45.346  -1.683  1.00 58.17           C  \nATOM    149  CD  GLU A  21      37.424 -46.435  -1.905  1.00 58.92           C  \nATOM    150  OE1 GLU A  21      37.214 -47.237  -0.975  1.00 58.47           O  \nATOM    151  OE2 GLU A  21      36.852 -46.499  -3.013  1.00 59.55           O  \nATOM    152  N   VAL A  22      40.377 -43.852   2.226  1.00 52.09           N  \nATOM    153  CA  VAL A  22      40.442 -43.344   3.592  1.00 52.74           C  \nATOM    154  C   VAL A  22      41.114 -41.976   3.503  1.00 52.42           C  \nATOM    155  O   VAL A  22      40.688 -41.012   4.144  1.00 52.25           O  \nATOM    156  CB  VAL A  22      41.255 -44.291   4.543  1.00 52.10           C  \nATOM    157  CG1 VAL A  22      41.982 -43.487   5.608  1.00 51.45           C  \nATOM    158  CG2 VAL A  22      40.308 -45.257   5.244  1.00 49.84           C  \nATOM    159  N   LYS A  23      42.146 -41.900   2.670  1.00 53.11           N  \nATOM    160  CA  LYS A  23      42.903 -40.671   2.472  1.00 53.96           C  \nATOM    161  C   LYS A  23      42.096 -39.561   1.805  1.00 52.88           C  \nATOM    162  O   LYS A  23      42.242 -38.400   2.160  1.00 54.26           O  \nATOM    163  CB  LYS A  23      44.157 -40.966   1.651  1.00 57.80           C  \nATOM    164  CG  LYS A  23      45.099 -39.782   1.491  1.00 61.23           C  \nATOM    165  CD  LYS A  23      46.344 -40.214   0.752  1.00 62.97           C  \nATOM    166  CE  LYS A  23      47.269 -39.041   0.493  1.00 66.67           C  \nATOM    167  NZ  LYS A  23      48.322 -39.379  -0.510  1.00 67.64           N  \nATOM    168  N   ALA A  24      41.248 -39.912   0.842  1.00 51.34           N  \nATOM    169  CA  ALA A  24      40.422 -38.916   0.165  1.00 49.06           C  \nATOM    170  C   ALA A  24      39.337 -38.434   1.126  1.00 48.67           C  \nATOM    171  O   ALA A  24      38.942 -37.263   1.102  1.00 50.72           O  \nATOM    172  CB  ALA A  24      39.795 -39.512  -1.096  1.00 46.07           C  \nATOM    173  N   PHE A  25      38.886 -39.337   1.992  1.00 46.74           N  \nATOM    174  CA  PHE A  25      37.844 -39.047   2.968  1.00 45.55           C  \nATOM    175  C   PHE A  25      38.295 -38.131   4.091  1.00 47.15           C  \nATOM    176  O   PHE A  25      37.609 -37.168   4.421  1.00 47.06           O  \nATOM    177  CB  PHE A  25      37.305 -40.346   3.572  1.00 44.11           C  \nATOM    178  CG  PHE A  25      36.208 -40.129   4.565  1.00 44.40           C  \nATOM    179  CD1 PHE A  25      36.448 -40.245   5.932  1.00 44.48           C  \nATOM    180  CD2 PHE A  25      34.943 -39.747   4.136  1.00 46.53           C  \nATOM    181  CE1 PHE A  25      35.446 -39.984   6.854  1.00 44.83           C  \nATOM    182  CE2 PHE A  25      33.923 -39.479   5.056  1.00 46.80           C  \nATOM    183  CZ  PHE A  25      34.181 -39.598   6.420  1.00 45.86           C  \nATOM    184  N   VAL A  26      39.426 -38.462   4.705  1.00 48.49           N  \nATOM    185  CA  VAL A  26      39.983 -37.653   5.778  1.00 51.01           C  \nATOM    186  C   VAL A  26      40.248 -36.259   5.179  1.00 52.16           C  \nATOM    187  O   VAL A  26      40.006 -35.230   5.812  1.00 51.87           O  \nATOM    188  CB  VAL A  26      41.318 -38.271   6.292  1.00 53.20           C  \nATOM    189  CG1 VAL A  26      42.098 -37.248   7.103  1.00 49.44           C  \nATOM    190  CG2 VAL A  26      41.031 -39.542   7.127  1.00 51.33           C  \nATOM    191  N   SER A  27      40.720 -36.239   3.939  1.00 52.43           N  \nATOM    192  CA  SER A  27      41.016 -34.987   3.259  1.00 53.05           C  \nATOM    193  C   SER A  27      39.789 -34.080   3.062  1.00 54.74           C  \nATOM    194  O   SER A  27      39.830 -32.891   3.392  1.00 54.90           O  \nATOM    195  CB  SER A  27      41.675 -35.276   1.912  1.00 52.61           C  \nATOM    196  OG  SER A  27      42.026 -34.077   1.246  1.00 58.91           O  \nATOM    197  N   LYS A  28      38.697 -34.630   2.535  1.00 56.89           N  \nATOM    198  CA  LYS A  28      37.502 -33.826   2.322  1.00 58.10           C  \nATOM    199  C   LYS A  28      36.769 -33.529   3.627  1.00 57.99           C  \nATOM    200  O   LYS A  28      36.057 -32.533   3.736  1.00 60.20           O  \nATOM    201  CB  LYS A  28      36.569 -34.500   1.309  1.00 60.18           C  \nATOM    202  CG  LYS A  28      35.579 -35.535   1.831  1.00 65.20           C  \nATOM    203  CD  LYS A  28      34.211 -35.219   1.209  1.00 67.90           C  \nATOM    204  CE  LYS A  28      33.259 -36.412   1.050  1.00 71.12           C  \nATOM    205  NZ  LYS A  28      32.541 -36.839   2.285  1.00 72.35           N  \nATOM    206  N   LYS A  29      36.948 -34.376   4.631  1.00 57.79           N  \nATOM    207  CA  LYS A  29      36.302 -34.140   5.919  1.00 59.04           C  \nATOM    208  C   LYS A  29      36.903 -32.874   6.508  1.00 57.75           C  \nATOM    209  O   LYS A  29      36.200 -32.017   7.029  1.00 56.90           O  \nATOM    210  CB  LYS A  29      36.539 -35.308   6.882  1.00 60.66           C  \nATOM    211  CG  LYS A  29      36.012 -35.068   8.294  1.00 66.56           C  \nATOM    212  CD  LYS A  29      35.630 -36.376   8.991  1.00 72.57           C  \nATOM    213  CE  LYS A  29      36.744 -36.944   9.867  1.00 74.66           C  \nATOM    214  NZ  LYS A  29      36.676 -36.411  11.262  1.00 77.39           N  \nATOM    215  N   ARG A  30      38.221 -32.773   6.409  1.00 57.71           N  \nATOM    216  CA  ARG A  30      38.958 -31.622   6.910  1.00 57.99           C  \nATOM    217  C   ARG A  30      38.557 -30.343   6.149  1.00 55.90           C  \nATOM    218  O   ARG A  30      38.333 -29.295   6.753  1.00 55.11           O  \nATOM    219  CB  ARG A  30      40.458 -31.895   6.760  1.00 60.54           C  \nATOM    220  CG  ARG A  30      41.367 -30.755   7.163  1.00 68.07           C  \nATOM    221  CD  ARG A  30      42.823 -31.092   6.825  1.00 73.66           C  \nATOM    222  NE  ARG A  30      43.231 -32.365   7.422  1.00 79.67           N  \nATOM    223  CZ  ARG A  30      43.422 -32.565   8.726  1.00 82.24           C  \nATOM    224  NH1 ARG A  30      43.263 -31.572   9.598  1.00 81.63           N  \nATOM    225  NH2 ARG A  30      43.736 -33.780   9.162  1.00 82.68           N  \nATOM    226  N   ALA A  31      38.465 -30.439   4.823  1.00 53.57           N  \nATOM    227  CA  ALA A  31      38.091 -29.297   3.996  1.00 52.08           C  \nATOM    228  C   ALA A  31      36.705 -28.788   4.372  1.00 51.01           C  \nATOM    229  O   ALA A  31      36.482 -27.585   4.458  1.00 53.20           O  \nATOM    230  CB  ALA A  31      38.132 -29.676   2.514  1.00 50.78           C  \nATOM    231  N   ILE A  32      35.778 -29.707   4.607  1.00 51.29           N  \nATOM    232  CA  ILE A  32      34.412 -29.350   4.974  1.00 51.44           C  \nATOM    233  C   ILE A  32      34.354 -28.663   6.326  1.00 51.40           C  \nATOM    234  O   ILE A  32      33.644 -27.681   6.504  1.00 52.30           O  \nATOM    235  CB  ILE A  32      33.513 -30.609   4.975  1.00 52.68           C  \nATOM    236  CG1 ILE A  32      33.324 -31.075   3.529  1.00 51.57           C  \nATOM    237  CG2 ILE A  32      32.172 -30.331   5.687  1.00 52.78           C  \nATOM    238  CD1 ILE A  32      32.674 -32.415   3.407  1.00 51.38           C  \nATOM    239  N   LYS A  33      35.121 -29.178   7.274  1.00 53.16           N  \nATOM    240  CA  LYS A  33      35.155 -28.617   8.615  1.00 55.48           C  \nATOM    241  C   LYS A  33      35.713 -27.194   8.616  1.00 54.58           C  \nATOM    242  O   LYS A  33      35.300 -26.345   9.406  1.00 53.88           O  \nATOM    243  CB  LYS A  33      36.007 -29.498   9.526  1.00 60.19           C  \nATOM    244  CG  LYS A  33      35.403 -29.682  10.897  1.00 67.78           C  \nATOM    245  CD  LYS A  33      34.114 -30.489  10.804  1.00 74.45           C  \nATOM    246  CE  LYS A  33      32.983 -29.856  11.618  1.00 78.13           C  \nATOM    247  NZ  LYS A  33      32.631 -28.473  11.159  1.00 81.27           N  \nATOM    248  N   ASN A  34      36.674 -26.949   7.738  1.00 54.04           N  \nATOM    249  CA  ASN A  34      37.274 -25.637   7.621  1.00 54.76           C  \nATOM    250  C   ASN A  34      36.265 -24.657   7.041  1.00 53.60           C  \nATOM    251  O   ASN A  34      36.132 -23.552   7.541  1.00 51.50           O  \nATOM    252  CB  ASN A  34      38.523 -25.730   6.759  1.00 56.88           C  \nATOM    253  CG  ASN A  34      39.691 -26.327   7.516  1.00 60.49           C  \nATOM    254  OD1 ASN A  34      40.624 -26.860   6.921  1.00 62.60           O  \nATOM    255  ND2 ASN A  34      39.647 -26.230   8.845  1.00 62.27           N  \nATOM    256  N   GLU A  35      35.542 -25.078   6.008  1.00 54.52           N  \nATOM    257  CA  GLU A  35      34.519 -24.238   5.385  1.00 56.92           C  \nATOM    258  C   GLU A  35      33.455 -23.837   6.400  1.00 55.87           C  \nATOM    259  O   GLU A  35      33.123 -22.664   6.537  1.00 56.00           O  \nATOM    260  CB  GLU A  35      33.812 -24.986   4.247  1.00 60.06           C  \nATOM    261  CG  GLU A  35      34.567 -25.093   2.936  1.00 66.47           C  \nATOM    262  CD  GLU A  35      34.745 -23.749   2.254  1.00 70.12           C  \nATOM    263  OE1 GLU A  35      33.819 -22.909   2.344  1.00 68.84           O  \nATOM    264  OE2 GLU A  35      35.807 -23.544   1.618  1.00 72.11           O  \nATOM    265  N   GLN A  36      32.919 -24.828   7.107  1.00 56.17           N  \nATOM    266  CA  GLN A  36      31.853 -24.585   8.076  1.00 55.32           C  \nATOM    267  C   GLN A  36      32.245 -23.592   9.145  1.00 54.36           C  \nATOM    268  O   GLN A  36      31.433 -22.767   9.547  1.00 54.88           O  \nATOM    269  CB  GLN A  36      31.396 -25.900   8.725  1.00 56.58           C  \nATOM    270  CG  GLN A  36      30.815 -26.899   7.730  1.00 56.61           C  \nATOM    271  CD  GLN A  36      30.448 -28.238   8.370  1.00 58.00           C  \nATOM    272  OE1 GLN A  36      30.988 -28.621   9.411  1.00 56.56           O  \nATOM    273  NE2 GLN A  36      29.538 -28.961   7.732  1.00 54.90           N  \nATOM    274  N   LEU A  37      33.486 -23.658   9.609  1.00 53.55           N  \nATOM    275  CA  LEU A  37      33.922 -22.723  10.627  1.00 53.56           C  \nATOM    276  C   LEU A  37      34.032 -21.315  10.033  1.00 56.30           C  \nATOM    277  O   LEU A  37      33.573 -20.348  10.655  1.00 56.23           O  \nATOM    278  CB  LEU A  37      35.263 -23.160  11.224  1.00 51.15           C  \nATOM    279  CG  LEU A  37      35.858 -22.244  12.308  1.00 51.45           C  \nATOM    280  CD1 LEU A  37      34.872 -22.063  13.451  1.00 48.91           C  \nATOM    281  CD2 LEU A  37      37.166 -22.833  12.821  1.00 48.05           C  \nATOM    282  N   LEU A  38      34.617 -21.203   8.834  1.00 56.10           N  \nATOM    283  CA  LEU A  38      34.785 -19.907   8.176  1.00 56.72           C  \nATOM    284  C   LEU A  38      33.434 -19.281   7.909  1.00 56.43           C  \nATOM    285  O   LEU A  38      33.221 -18.111   8.199  1.00 56.32           O  \nATOM    286  CB  LEU A  38      35.522 -20.051   6.847  1.00 58.11           C  \nATOM    287  CG  LEU A  38      36.792 -19.214   6.658  1.00 59.65           C  \nATOM    288  CD1 LEU A  38      36.763 -18.636   5.262  1.00 61.09           C  \nATOM    289  CD2 LEU A  38      36.899 -18.089   7.682  1.00 59.80           C  \nATOM    290  N   GLN A  39      32.524 -20.065   7.344  1.00 56.65           N  \nATOM    291  CA  GLN A  39      31.185 -19.575   7.051  1.00 57.09           C  \nATOM    292  C   GLN A  39      30.482 -19.105   8.309  1.00 56.68           C  \nATOM    293  O   GLN A  39      29.807 -18.086   8.305  1.00 57.20           O  \nATOM    294  CB  GLN A  39      30.335 -20.656   6.399  1.00 55.45           C  \nATOM    295  CG  GLN A  39      30.854 -21.107   5.067  1.00 58.33           C  \nATOM    296  CD  GLN A  39      29.840 -21.913   4.280  1.00 60.68           C  \nATOM    297  OE1 GLN A  39      30.139 -22.426   3.195  1.00 60.37           O  \nATOM    298  NE2 GLN A  39      28.630 -22.019   4.815  1.00 61.06           N  \nATOM    299  N   LEU A  40      30.626 -19.861   9.382  1.00 56.60           N  \nATOM    300  CA  LEU A  40      29.984 -19.488  10.623  1.00 55.70           C  \nATOM    301  C   LEU A  40      30.577 -18.163  11.117  1.00 55.34           C  \nATOM    302  O   LEU A  40      29.865 -17.315  11.646  1.00 56.24           O  \nATOM    303  CB  LEU A  40      30.166 -20.608  11.650  1.00 54.19           C  \nATOM    304  CG  LEU A  40      29.532 -20.381  13.015  1.00 55.55           C  \nATOM    305  CD1 LEU A  40      29.356 -21.712  13.715  1.00 55.83           C  \nATOM    306  CD2 LEU A  40      30.405 -19.438  13.837  1.00 56.64           C  \nATOM    307  N   ILE A  41      31.879 -17.983  10.937  1.00 54.35           N  \nATOM    308  CA  ILE A  41      32.523 -16.750  11.364  1.00 54.54           C  \nATOM    309  C   ILE A  41      32.048 -15.595  10.477  1.00 57.02           C  \nATOM    310  O   ILE A  41      31.819 -14.474  10.955  1.00 56.48           O  \nATOM    311  CB  ILE A  41      34.049 -16.849  11.274  1.00 53.68           C  \nATOM    312  CG1 ILE A  41      34.562 -17.812  12.344  1.00 52.49           C  \nATOM    313  CG2 ILE A  41      34.666 -15.465  11.442  1.00 53.06           C  \nATOM    314  CD1 ILE A  41      36.051 -18.036  12.310  1.00 49.33           C  \nATOM    315  N   PHE A  42      31.902 -15.876   9.187  1.00 56.17           N  \nATOM    316  CA  PHE A  42      31.428 -14.873   8.250  1.00 58.04           C  \nATOM    317  C   PHE A  42      30.005 -14.464   8.661  1.00 59.07           C  \nATOM    318  O   PHE A  42      29.744 -13.293   8.958  1.00 58.91           O  \nATOM    319  CB  PHE A  42      31.412 -15.459   6.840  1.00 58.04           C  \nATOM    320  CG  PHE A  42      30.981 -14.490   5.788  1.00 60.21           C  \nATOM    321  CD1 PHE A  42      31.878 -13.566   5.273  1.00 59.94           C  \nATOM    322  CD2 PHE A  42      29.671 -14.488   5.316  1.00 58.76           C  \nATOM    323  CE1 PHE A  42      31.470 -12.652   4.297  1.00 62.30           C  \nATOM    324  CE2 PHE A  42      29.264 -13.575   4.344  1.00 61.79           C  \nATOM    325  CZ  PHE A  42      30.162 -12.663   3.838  1.00 63.22           C  \nATOM    326  N   LYS A  43      29.096 -15.440   8.672  1.00 59.76           N  \nATOM    327  CA  LYS A  43      27.708 -15.209   9.040  1.00 60.13           C  \nATOM    328  C   LYS A  43      27.550 -14.390  10.308  1.00 60.67           C  \nATOM    329  O   LYS A  43      26.612 -13.617  10.427  1.00 61.23           O  \nATOM    330  CB  LYS A  43      26.981 -16.531   9.243  1.00 62.08           C  \nATOM    331  CG  LYS A  43      26.472 -17.206   7.992  1.00 65.54           C  \nATOM    332  CD  LYS A  43      25.485 -18.290   8.414  1.00 70.32           C  \nATOM    333  CE  LYS A  43      25.695 -19.584   7.663  1.00 72.14           C  \nATOM    334  NZ  LYS A  43      25.363 -19.415   6.227  1.00 76.10           N  \nATOM    335  N   SER A  44      28.455 -14.568  11.260  1.00 61.05           N  \nATOM    336  CA  SER A  44      28.377 -13.852  12.521  1.00 63.99           C  \nATOM    337  C   SER A  44      28.811 -12.388  12.405  1.00 65.75           C  \nATOM    338  O   SER A  44      28.383 -11.544  13.196  1.00 67.65           O  \nATOM    339  CB  SER A  44      29.220 -14.588  13.571  1.00 65.71           C  \nATOM    340  OG  SER A  44      30.104 -13.707  14.244  1.00 70.19           O  \nATOM    341  N   ILE A  45      29.668 -12.088  11.432  1.00 65.18           N  \nATOM    342  CA  ILE A  45      30.122 -10.719  11.235  1.00 63.17           C  \nATOM    343  C   ILE A  45      29.077  -9.998  10.389  1.00 63.97           C  \nATOM    344  O   ILE A  45      28.797  -8.818  10.610  1.00 63.70           O  \nATOM    345  CB  ILE A  45      31.493 -10.664  10.506  1.00 62.12           C  \nATOM    346  CG1 ILE A  45      32.560 -11.376  11.348  1.00 61.94           C  \nATOM    347  CG2 ILE A  45      31.890  -9.213  10.256  1.00 61.46           C  \nATOM    348  CD1 ILE A  45      33.967 -11.430  10.736  1.00 59.04           C  \nATOM    349  N   ASP A  46      28.513 -10.721   9.420  1.00 63.87           N  \nATOM    350  CA  ASP A  46      27.488 -10.202   8.517  1.00 65.90           C  \nATOM    351  C   ASP A  46      26.182 -10.077   9.296  1.00 68.39           C  \nATOM    352  O   ASP A  46      25.229 -10.823   9.060  1.00 68.79           O  \nATOM    353  CB  ASP A  46      27.299 -11.163   7.333  1.00 64.85           C  \nATOM    354  CG  ASP A  46      26.160 -10.747   6.408  1.00 65.15           C  \nATOM    355  OD1 ASP A  46      25.455  -9.795   6.770  1.00 65.82           O  \nATOM    356  OD2 ASP A  46      25.958 -11.367   5.334  1.00 63.25           O  \nATOM    357  N   ALA A  47      26.140  -9.113  10.211  1.00 70.88           N  \nATOM    358  CA  ALA A  47      24.978  -8.896  11.063  1.00 72.87           C  \nATOM    359  C   ALA A  47      23.610  -8.766  10.399  1.00 74.65           C  \nATOM    360  O   ALA A  47      22.614  -9.097  11.029  1.00 75.62           O  \nATOM    361  CB  ALA A  47      25.230  -7.709  11.985  1.00 71.10           C  \nATOM    362  N   ASP A  48      23.524  -8.318   9.150  1.00 76.10           N  \nATOM    363  CA  ASP A  48      22.201  -8.207   8.537  1.00 76.91           C  \nATOM    364  C   ASP A  48      21.901  -9.310   7.524  1.00 76.46           C  \nATOM    365  O   ASP A  48      20.868  -9.278   6.853  1.00 78.51           O  \nATOM    366  CB  ASP A  48      21.997  -6.834   7.878  1.00 79.13           C  \nATOM    367  CG  ASP A  48      22.900  -6.608   6.677  1.00 82.23           C  \nATOM    368  OD1 ASP A  48      23.043  -7.533   5.844  1.00 83.07           O  \nATOM    369  OD2 ASP A  48      23.455  -5.494   6.561  1.00 82.40           O  \nATOM    370  N   GLY A  49      22.808 -10.274   7.402  1.00 74.73           N  \nATOM    371  CA  GLY A  49      22.608 -11.388   6.490  1.00 72.08           C  \nATOM    372  C   GLY A  49      22.449 -11.132   5.001  1.00 72.06           C  \nATOM    373  O   GLY A  49      21.905 -11.978   4.296  1.00 71.52           O  \nATOM    374  N   ASN A  50      22.904  -9.989   4.500  1.00 71.61           N  \nATOM    375  CA  ASN A  50      22.774  -9.735   3.067  1.00 71.61           C  \nATOM    376  C   ASN A  50      23.892 -10.456   2.281  1.00 71.10           C  \nATOM    377  O   ASN A  50      23.914 -10.410   1.046  1.00 70.57           O  \nATOM    378  CB  ASN A  50      22.813  -8.231   2.789  1.00 71.33           C  \nATOM    379  CG  ASN A  50      24.166  -7.650   3.060  1.00 71.84           C  \nATOM    380  OD1 ASN A  50      25.033  -8.346   3.585  1.00 70.61           O  \nATOM    381  ND2 ASN A  50      24.372  -6.383   2.709  1.00 69.84           N  \nATOM    382  N   GLY A  51      24.829 -11.094   2.991  1.00 70.23           N  \nATOM    383  CA  GLY A  51      25.893 -11.837   2.322  1.00 70.86           C  \nATOM    384  C   GLY A  51      27.182 -11.105   1.992  1.00 71.12           C  \nATOM    385  O   GLY A  51      27.943 -11.538   1.124  1.00 71.96           O  \nATOM    386  N   GLU A  52      27.443 -10.011   2.692  1.00 70.02           N  \nATOM    387  CA  GLU A  52      28.634  -9.217   2.451  1.00 70.16           C  \nATOM    388  C   GLU A  52      28.900  -8.465   3.749  1.00 68.33           C  \nATOM    389  O   GLU A  52      27.973  -8.188   4.489  1.00 67.83           O  \nATOM    390  CB  GLU A  52      28.345  -8.206   1.348  1.00 72.94           C  \nATOM    391  CG  GLU A  52      27.366  -7.137   1.827  1.00 76.86           C  \nATOM    392  CD  GLU A  52      27.178  -6.006   0.839  1.00 79.96           C  \nATOM    393  OE1 GLU A  52      26.727  -4.909   1.255  1.00 77.35           O  \nATOM    394  OE2 GLU A  52      27.469  -6.223  -0.358  1.00 81.92           O  \nATOM    395  N   ILE A  53      30.147  -8.130   4.043  1.00 66.11           N  \nATOM    396  CA  ILE A  53      30.420  -7.386   5.267  1.00 65.03           C  \nATOM    397  C   ILE A  53      30.761  -5.944   4.905  1.00 66.40           C  \nATOM    398  O   ILE A  53      31.716  -5.692   4.161  1.00 67.68           O  \nATOM    399  CB  ILE A  53      31.577  -8.037   6.060  1.00 62.51           C  \nATOM    400  CG1 ILE A  53      31.092  -9.375   6.624  1.00 63.80           C  \nATOM    401  CG2 ILE A  53      32.059  -7.116   7.176  1.00 60.38           C  \nATOM    402  CD1 ILE A  53      32.196 -10.303   7.065  1.00 61.98           C  \nATOM    403  N   ASP A  54      29.959  -4.996   5.385  1.00 66.39           N  \nATOM    404  CA  ASP A  54      30.242  -3.594   5.095  1.00 67.11           C  \nATOM    405  C   ASP A  54      31.174  -3.099   6.185  1.00 67.02           C  \nATOM    406  O   ASP A  54      31.385  -3.800   7.173  1.00 67.62           O  \nATOM    407  CB  ASP A  54      28.959  -2.758   5.022  1.00 66.36           C  \nATOM    408  CG  ASP A  54      28.129  -2.836   6.279  1.00 66.75           C  \nATOM    409  OD1 ASP A  54      28.659  -2.544   7.375  1.00 66.63           O  \nATOM    410  OD2 ASP A  54      26.935  -3.180   6.167  1.00 65.12           O  \nATOM    411  N   GLN A  55      31.744  -1.913   6.008  1.00 68.14           N  \nATOM    412  CA  GLN A  55      32.699  -1.392   6.977  1.00 68.15           C  \nATOM    413  C   GLN A  55      32.158  -1.268   8.377  1.00 68.99           C  \nATOM    414  O   GLN A  55      32.932  -1.220   9.334  1.00 69.56           O  \nATOM    415  CB  GLN A  55      33.229  -0.021   6.550  1.00 67.63           C  \nATOM    416  CG  GLN A  55      33.384   0.163   5.064  1.00 67.89           C  \nATOM    417  CD  GLN A  55      34.320   1.297   4.722  1.00 68.76           C  \nATOM    418  OE1 GLN A  55      34.404   2.288   5.447  1.00 70.13           O  \nATOM    419  NE2 GLN A  55      35.027   1.165   3.608  1.00 67.87           N  \nATOM    420  N   ASN A  56      30.839  -1.204   8.515  1.00 70.66           N  \nATOM    421  CA  ASN A  56      30.259  -1.050   9.841  1.00 71.83           C  \nATOM    422  C   ASN A  56      30.244  -2.391  10.550  1.00 70.10           C  \nATOM    423  O   ASN A  56      30.712  -2.513  11.685  1.00 70.18           O  \nATOM    424  CB  ASN A  56      28.850  -0.466   9.742  1.00 76.17           C  \nATOM    425  CG  ASN A  56      28.413   0.189  11.027  1.00 80.22           C  \nATOM    426  OD1 ASN A  56      27.495  -0.289  11.696  1.00 80.13           O  \nATOM    427  ND2 ASN A  56      29.077   1.291  11.389  1.00 81.73           N  \nATOM    428  N   GLU A  57      29.727  -3.407   9.874  1.00 68.32           N  \nATOM    429  CA  GLU A  57      29.711  -4.732  10.471  1.00 66.02           C  \nATOM    430  C   GLU A  57      31.132  -5.109  10.813  1.00 66.48           C  \nATOM    431  O   GLU A  57      31.386  -5.692  11.868  1.00 66.68           O  \nATOM    432  CB  GLU A  57      29.144  -5.738   9.499  1.00 62.73           C  \nATOM    433  CG  GLU A  57      27.733  -5.383   9.178  1.00 57.94           C  \nATOM    434  CD  GLU A  57      27.199  -6.141   7.997  1.00 58.67           C  \nATOM    435  OE1 GLU A  57      27.984  -6.412   7.067  1.00 57.68           O  \nATOM    436  OE2 GLU A  57      25.993  -6.455   7.978  1.00 55.58           O  \nATOM    437  N   PHE A  58      32.052  -4.753   9.918  1.00 65.19           N  \nATOM    438  CA  PHE A  58      33.467  -5.047  10.099  1.00 66.29           C  \nATOM    439  C   PHE A  58      34.034  -4.301  11.295  1.00 66.77           C  \nATOM    440  O   PHE A  58      34.887  -4.821  12.006  1.00 66.23           O  \nATOM    441  CB  PHE A  58      34.257  -4.652   8.842  1.00 65.26           C  \nATOM    442  CG  PHE A  58      35.739  -4.965   8.918  1.00 64.30           C  \nATOM    443  CD1 PHE A  58      36.248  -6.182   8.444  1.00 64.45           C  \nATOM    444  CD2 PHE A  58      36.632  -4.038   9.455  1.00 62.83           C  \nATOM    445  CE1 PHE A  58      37.619  -6.461   8.509  1.00 63.56           C  \nATOM    446  CE2 PHE A  58      38.001  -4.314   9.523  1.00 61.37           C  \nATOM    447  CZ  PHE A  58      38.490  -5.529   9.046  1.00 62.84           C  \nATOM    448  N   ALA A  59      33.570  -3.073  11.495  1.00 69.76           N  \nATOM    449  CA  ALA A  59      34.044  -2.229  12.588  1.00 71.11           C  \nATOM    450  C   ALA A  59      33.662  -2.757  13.968  1.00 71.82           C  \nATOM    451  O   ALA A  59      34.494  -2.768  14.882  1.00 70.47           O  \nATOM    452  CB  ALA A  59      33.518  -0.819  12.408  1.00 70.48           C  \nATOM    453  N   LYS A  60      32.408  -3.176  14.134  1.00 73.21           N  \nATOM    454  CA  LYS A  60      31.973  -3.702  15.429  1.00 77.01           C  \nATOM    455  C   LYS A  60      32.858  -4.879  15.808  1.00 78.78           C  \nATOM    456  O   LYS A  60      33.469  -4.895  16.881  1.00 78.94           O  \nATOM    457  CB  LYS A  60      30.512  -4.150  15.376  1.00 79.88           C  \nATOM    458  CG  LYS A  60      29.511  -3.003  15.337  1.00 84.50           C  \nATOM    459  CD  LYS A  60      28.184  -3.495  14.782  1.00 86.60           C  \nATOM    460  CE  LYS A  60      27.230  -2.354  14.462  1.00 87.76           C  \nATOM    461  NZ  LYS A  60      26.189  -2.144  15.513  1.00 88.30           N  \nATOM    462  N   PHE A  61      32.922  -5.853  14.906  1.00 79.85           N  \nATOM    463  CA  PHE A  61      33.729  -7.045  15.082  1.00 81.17           C  \nATOM    464  C   PHE A  61      35.115  -6.716  15.625  1.00 83.52           C  \nATOM    465  O   PHE A  61      35.602  -7.397  16.523  1.00 84.19           O  \nATOM    466  CB  PHE A  61      33.856  -7.778  13.746  1.00 80.08           C  \nATOM    467  CG  PHE A  61      34.697  -9.017  13.809  1.00 78.61           C  \nATOM    468  CD1 PHE A  61      34.201 -10.192  14.372  1.00 77.96           C  \nATOM    469  CD2 PHE A  61      35.993  -9.012  13.314  1.00 77.34           C  \nATOM    470  CE1 PHE A  61      34.986 -11.337  14.442  1.00 76.92           C  \nATOM    471  CE2 PHE A  61      36.785 -10.154  13.383  1.00 78.40           C  \nATOM    472  CZ  PHE A  61      36.278 -11.320  13.948  1.00 77.68           C  \nATOM    473  N   TYR A  62      35.770  -5.695  15.083  1.00 86.85           N  \nATOM    474  CA  TYR A  62      37.091  -5.354  15.595  1.00 91.54           C  \nATOM    475  C   TYR A  62      37.040  -4.449  16.824  1.00 95.16           C  \nATOM    476  O   TYR A  62      35.976  -3.944  17.208  1.00 95.30           O  \nATOM    477  CB  TYR A  62      37.970  -4.731  14.504  1.00 91.83           C  \nATOM    478  CG  TYR A  62      38.700  -5.772  13.690  1.00 93.17           C  \nATOM    479  CD1 TYR A  62      38.097  -6.372  12.587  1.00 94.03           C  \nATOM    480  CD2 TYR A  62      39.959  -6.225  14.077  1.00 93.75           C  \nATOM    481  CE1 TYR A  62      38.741  -7.404  11.883  1.00 94.66           C  \nATOM    482  CE2 TYR A  62      40.605  -7.250  13.388  1.00 94.04           C  \nATOM    483  CZ  TYR A  62      39.991  -7.838  12.296  1.00 94.49           C  \nATOM    484  OH  TYR A  62      40.632  -8.853  11.628  1.00 94.92           O  \nATOM    485  N   GLY A  63      38.199  -4.277  17.450  1.00 98.54           N  \nATOM    486  CA  GLY A  63      38.305  -3.455  18.641  1.00102.52           C  \nATOM    487  C   GLY A  63      39.067  -4.309  19.627  1.00105.41           C  \nATOM    488  O   GLY A  63      38.624  -4.532  20.760  1.00106.17           O  \nATOM    489  N   SER A  64      40.248  -4.748  19.190  1.00107.73           N  \nATOM    490  CA  SER A  64      41.068  -5.658  19.978  1.00109.81           C  \nATOM    491  C   SER A  64      39.974  -6.727  20.109  1.00110.88           C  \nATOM    492  O   SER A  64      39.711  -7.279  21.184  1.00111.30           O  \nATOM    493  CB  SER A  64      41.470  -5.009  21.323  1.00109.62           C  \nATOM    494  OG  SER A  64      40.526  -5.219  22.367  1.00109.17           O  \nATOM    495  N   ILE A  65      39.355  -6.963  18.944  1.00112.22           N  \nATOM    496  CA  ILE A  65      38.218  -7.846  18.692  1.00112.85           C  \nATOM    497  C   ILE A  65      37.298  -8.173  19.872  1.00112.86           C  \nATOM    498  O   ILE A  65      37.591  -8.962  20.773  1.00113.51           O  \nATOM    499  CB  ILE A  65      38.643  -9.117  17.841  1.00112.76           C  \nATOM    500  CG1 ILE A  65      37.999 -10.393  18.391  1.00112.63           C  \nATOM    501  CG2 ILE A  65      40.158  -9.142  17.648  1.00112.92           C  \nATOM    502  CD1 ILE A  65      36.538 -10.593  17.973  1.00112.22           C  \nATOM    503  N   GLN A  66      36.183  -7.460  19.852  1.00112.67           N  \nATOM    504  CA  GLN A  66      35.121  -7.572  20.827  1.00111.53           C  \nATOM    505  C   GLN A  66      34.018  -6.963  19.983  1.00110.94           C  \nATOM    506  O   GLN A  66      34.101  -5.791  19.608  1.00110.41           O  \nATOM    507  CB  GLN A  66      35.417  -6.704  22.053  1.00113.59           C  \nATOM    508  N   GLY A  67      33.018  -7.758  19.629  1.00110.66           N  \nATOM    509  CA  GLY A  67      31.951  -7.234  18.800  1.00110.60           C  \nATOM    510  C   GLY A  67      31.123  -8.366  18.229  1.00111.34           C  \nATOM    511  O   GLY A  67      31.372  -8.800  17.080  1.00109.76           O  \nTER     512      GLY A  67                                                      \nATOM    513  N   ALA B   2      20.076 -32.134  18.404  1.00 95.47           N  \nATOM    514  CA  ALA B   2      20.129 -33.613  18.635  1.00 95.82           C  \nATOM    515  C   ALA B   2      19.992 -34.020  20.119  1.00 95.77           C  \nATOM    516  O   ALA B   2      20.571 -33.391  21.006  1.00 95.11           O  \nATOM    517  CB  ALA B   2      21.425 -34.169  18.046  1.00 96.01           C  \nATOM    518  N   GLU B   3      19.249 -35.106  20.348  1.00 95.42           N  \nATOM    519  CA  GLU B   3      18.909 -35.683  21.664  1.00 93.30           C  \nATOM    520  C   GLU B   3      19.913 -36.043  22.783  1.00 90.15           C  \nATOM    521  O   GLU B   3      19.979 -35.353  23.803  1.00 90.58           O  \nATOM    522  CB  GLU B   3      18.032 -36.901  21.397  1.00 95.00           C  \nATOM    523  CG  GLU B   3      17.372 -36.824  20.043  1.00 97.61           C  \nATOM    524  CD  GLU B   3      16.612 -38.085  19.698  1.00100.23           C  \nATOM    525  OE1 GLU B   3      16.614 -38.449  18.503  1.00100.56           O  \nATOM    526  OE2 GLU B   3      16.012 -38.711  20.607  1.00 99.87           O  \nATOM    527  N   ALA B   4      20.650 -37.141  22.637  1.00 85.59           N  \nATOM    528  CA  ALA B   4      21.578 -37.545  23.697  1.00 80.40           C  \nATOM    529  C   ALA B   4      22.774 -36.624  23.801  1.00 76.71           C  \nATOM    530  O   ALA B   4      23.317 -36.398  24.883  1.00 76.72           O  \nATOM    531  CB  ALA B   4      22.046 -38.978  23.485  1.00 80.62           C  \nATOM    532  N   LEU B   5      23.181 -36.092  22.662  1.00 72.37           N  \nATOM    533  CA  LEU B   5      24.305 -35.186  22.623  1.00 68.42           C  \nATOM    534  C   LEU B   5      24.010 -33.937  23.453  1.00 65.54           C  \nATOM    535  O   LEU B   5      24.872 -33.460  24.191  1.00 64.25           O  \nATOM    536  CB  LEU B   5      24.606 -34.800  21.177  1.00 69.45           C  \nATOM    537  CG  LEU B   5      25.833 -33.912  20.967  1.00 71.43           C  \nATOM    538  CD1 LEU B   5      27.050 -34.530  21.645  1.00 71.65           C  \nATOM    539  CD2 LEU B   5      26.073 -33.733  19.481  1.00 71.25           C  \nATOM    540  N   PHE B   6      22.796 -33.408  23.353  1.00 61.71           N  \nATOM    541  CA  PHE B   6      22.465 -32.210  24.117  1.00 59.19           C  \nATOM    542  C   PHE B   6      22.516 -32.496  25.620  1.00 57.98           C  \nATOM    543  O   PHE B   6      22.944 -31.646  26.408  1.00 56.49           O  \nATOM    544  CB  PHE B   6      21.083 -31.680  23.709  1.00 56.79           C  \nATOM    545  CG  PHE B   6      20.706 -30.386  24.377  1.00 56.41           C  \nATOM    546  CD1 PHE B   6      20.222 -30.374  25.688  1.00 54.17           C  \nATOM    547  CD2 PHE B   6      20.865 -29.173  23.712  1.00 53.81           C  \nATOM    548  CE1 PHE B   6      19.914 -29.179  26.323  1.00 52.39           C  \nATOM    549  CE2 PHE B   6      20.559 -27.972  24.343  1.00 53.56           C  \nATOM    550  CZ  PHE B   6      20.083 -27.977  25.651  1.00 52.29           C  \nATOM    551  N   LYS B   7      22.090 -33.698  26.006  1.00 57.51           N  \nATOM    552  CA  LYS B   7      22.091 -34.095  27.409  1.00 56.99           C  \nATOM    553  C   LYS B   7      23.516 -34.266  27.884  1.00 56.23           C  \nATOM    554  O   LYS B   7      23.836 -33.967  29.028  1.00 55.52           O  \nATOM    555  CB  LYS B   7      21.328 -35.412  27.602  1.00 58.83           C  \nATOM    556  CG  LYS B   7      19.811 -35.266  27.704  1.00 57.62           C  \nATOM    557  CD  LYS B   7      19.126 -36.635  27.760  1.00 59.19           C  \nATOM    558  CE  LYS B   7      18.175 -36.770  28.953  1.00 58.59           C  \nATOM    559  NZ  LYS B   7      18.877 -37.020  30.248  1.00 58.58           N  \nATOM    560  N   GLU B   8      24.378 -34.750  27.001  1.00 57.34           N  \nATOM    561  CA  GLU B   8      25.769 -34.949  27.367  1.00 57.77           C  \nATOM    562  C   GLU B   8      26.479 -33.621  27.629  1.00 57.06           C  \nATOM    563  O   GLU B   8      27.293 -33.512  28.546  1.00 58.29           O  \nATOM    564  CB  GLU B   8      26.507 -35.729  26.271  1.00 62.14           C  \nATOM    565  CG  GLU B   8      27.956 -36.050  26.622  1.00 68.05           C  \nATOM    566  CD  GLU B   8      28.726 -36.692  25.469  1.00 72.75           C  \nATOM    567  OE1 GLU B   8      28.128 -36.919  24.387  1.00 73.97           O  \nATOM    568  OE2 GLU B   8      29.935 -36.970  25.645  1.00 74.25           O  \nATOM    569  N   ILE B   9      26.184 -32.608  26.824  1.00 55.30           N  \nATOM    570  CA  ILE B   9      26.817 -31.312  27.019  1.00 55.05           C  \nATOM    571  C   ILE B   9      26.155 -30.623  28.211  1.00 56.32           C  \nATOM    572  O   ILE B   9      26.841 -30.032  29.038  1.00 55.44           O  \nATOM    573  CB  ILE B   9      26.672 -30.418  25.764  1.00 54.92           C  \nATOM    574  CG1 ILE B   9      27.363 -31.093  24.580  1.00 53.41           C  \nATOM    575  CG2 ILE B   9      27.234 -29.015  26.030  1.00 52.19           C  \nATOM    576  CD1 ILE B   9      27.138 -30.410  23.250  1.00 55.03           C  \nATOM    577  N   ASP B  10      24.821 -30.704  28.291  1.00 56.81           N  \nATOM    578  CA  ASP B  10      24.060 -30.090  29.388  1.00 56.37           C  \nATOM    579  C   ASP B  10      24.323 -30.866  30.672  1.00 56.94           C  \nATOM    580  O   ASP B  10      23.472 -31.584  31.177  1.00 57.14           O  \nATOM    581  CB  ASP B  10      22.557 -30.085  29.064  1.00 55.76           C  \nATOM    582  CG  ASP B  10      21.691 -29.620  30.245  1.00 55.92           C  \nATOM    583  OD1 ASP B  10      22.192 -28.885  31.128  1.00 53.72           O  \nATOM    584  OD2 ASP B  10      20.496 -29.989  30.285  1.00 54.83           O  \nATOM    585  N   VAL B  11      25.520 -30.693  31.206  1.00 57.72           N  \nATOM    586  CA  VAL B  11      25.954 -31.399  32.408  1.00 58.49           C  \nATOM    587  C   VAL B  11      25.179 -31.249  33.732  1.00 59.51           C  \nATOM    588  O   VAL B  11      25.120 -32.201  34.511  1.00 59.54           O  \nATOM    589  CB  VAL B  11      27.460 -31.105  32.644  1.00 57.09           C  \nATOM    590  CG1 VAL B  11      27.847 -31.326  34.096  1.00 56.42           C  \nATOM    591  CG2 VAL B  11      28.275 -31.977  31.731  1.00 55.23           C  \nATOM    592  N   ASN B  12      24.607 -30.080  34.015  1.00 59.39           N  \nATOM    593  CA  ASN B  12      23.867 -29.932  35.268  1.00 58.33           C  \nATOM    594  C   ASN B  12      22.370 -30.185  35.057  1.00 58.79           C  \nATOM    595  O   ASN B  12      21.556 -30.005  35.969  1.00 58.20           O  \nATOM    596  CB  ASN B  12      24.105 -28.549  35.896  1.00 57.74           C  \nATOM    597  CG  ASN B  12      23.417 -27.460  35.154  1.00 58.79           C  \nATOM    598  OD1 ASN B  12      23.016 -27.657  34.021  1.00 60.25           O  \nATOM    599  ND2 ASN B  12      23.275 -26.295  35.780  1.00 59.19           N  \nATOM    600  N   GLY B  13      22.024 -30.583  33.835  1.00 59.41           N  \nATOM    601  CA  GLY B  13      20.658 -30.927  33.481  1.00 57.34           C  \nATOM    602  C   GLY B  13      19.521 -29.940  33.533  1.00 57.47           C  \nATOM    603  O   GLY B  13      18.378 -30.364  33.619  1.00 56.77           O  \nATOM    604  N   ASP B  14      19.791 -28.644  33.454  1.00 58.36           N  \nATOM    605  CA  ASP B  14      18.700 -27.674  33.515  1.00 56.69           C  \nATOM    606  C   ASP B  14      18.049 -27.313  32.198  1.00 56.22           C  \nATOM    607  O   ASP B  14      17.279 -26.351  32.147  1.00 57.62           O  \nATOM    608  CB  ASP B  14      19.154 -26.377  34.164  1.00 57.93           C  \nATOM    609  CG  ASP B  14      20.224 -25.667  33.363  1.00 61.27           C  \nATOM    610  OD1 ASP B  14      20.393 -25.969  32.155  1.00 60.89           O  \nATOM    611  OD2 ASP B  14      20.888 -24.791  33.954  1.00 60.93           O  \nATOM    612  N   GLY B  15      18.363 -28.050  31.136  1.00 55.54           N  \nATOM    613  CA  GLY B  15      17.776 -27.770  29.834  1.00 54.34           C  \nATOM    614  C   GLY B  15      18.488 -26.720  28.995  1.00 55.32           C  \nATOM    615  O   GLY B  15      18.154 -26.525  27.840  1.00 56.93           O  \nATOM    616  N   ALA B  16      19.469 -26.033  29.559  1.00 54.89           N  \nATOM    617  CA  ALA B  16      20.181 -25.012  28.797  1.00 56.39           C  \nATOM    618  C   ALA B  16      21.661 -25.346  28.713  1.00 55.83           C  \nATOM    619  O   ALA B  16      22.234 -25.777  29.688  1.00 55.14           O  \nATOM    620  CB  ALA B  16      20.001 -23.645  29.462  1.00 55.14           C  \nATOM    621  N   VAL B  17      22.285 -25.179  27.552  1.00 54.73           N  \nATOM    622  CA  VAL B  17      23.726 -25.441  27.450  1.00 53.18           C  \nATOM    623  C   VAL B  17      24.403 -24.092  27.677  1.00 51.38           C  \nATOM    624  O   VAL B  17      24.192 -23.150  26.910  1.00 50.05           O  \nATOM    625  CB  VAL B  17      24.086 -26.015  26.057  1.00 55.38           C  \nATOM    626  CG1 VAL B  17      25.571 -25.871  25.777  1.00 56.58           C  \nATOM    627  CG2 VAL B  17      23.689 -27.487  25.997  1.00 54.50           C  \nATOM    628  N   SER B  18      25.161 -23.975  28.764  1.00 50.36           N  \nATOM    629  CA  SER B  18      25.834 -22.712  29.072  1.00 50.97           C  \nATOM    630  C   SER B  18      27.199 -22.581  28.384  1.00 52.81           C  \nATOM    631  O   SER B  18      27.715 -23.548  27.814  1.00 51.44           O  \nATOM    632  CB  SER B  18      26.015 -22.556  30.587  1.00 49.79           C  \nATOM    633  OG  SER B  18      27.037 -23.402  31.086  1.00 53.03           O  \nATOM    634  N   TYR B  19      27.781 -21.383  28.443  1.00 54.93           N  \nATOM    635  CA  TYR B  19      29.082 -21.153  27.829  1.00 55.62           C  \nATOM    636  C   TYR B  19      30.148 -22.027  28.468  1.00 55.99           C  \nATOM    637  O   TYR B  19      30.909 -22.693  27.772  1.00 56.26           O  \nATOM    638  CB  TYR B  19      29.510 -19.685  27.932  1.00 55.25           C  \nATOM    639  CG  TYR B  19      30.871 -19.479  27.335  1.00 52.62           C  \nATOM    640  CD1 TYR B  19      31.086 -19.702  25.975  1.00 51.39           C  \nATOM    641  CD2 TYR B  19      31.958 -19.147  28.135  1.00 53.37           C  \nATOM    642  CE1 TYR B  19      32.357 -19.599  25.423  1.00 54.31           C  \nATOM    643  CE2 TYR B  19      33.239 -19.036  27.593  1.00 53.93           C  \nATOM    644  CZ  TYR B  19      33.431 -19.269  26.243  1.00 54.61           C  \nATOM    645  OH  TYR B  19      34.699 -19.185  25.729  1.00 55.25           O  \nATOM    646  N   GLU B  20      30.199 -22.030  29.793  1.00 57.18           N  \nATOM    647  CA  GLU B  20      31.171 -22.846  30.499  1.00 60.17           C  \nATOM    648  C   GLU B  20      30.972 -24.346  30.229  1.00 58.63           C  \nATOM    649  O   GLU B  20      31.898 -25.140  30.380  1.00 59.80           O  \nATOM    650  CB  GLU B  20      31.120 -22.551  32.006  1.00 64.98           C  \nATOM    651  CG  GLU B  20      29.715 -22.342  32.575  1.00 75.73           C  \nATOM    652  CD  GLU B  20      29.295 -20.867  32.626  1.00 81.43           C  \nATOM    653  OE1 GLU B  20      29.536 -20.209  33.664  1.00 83.27           O  \nATOM    654  OE2 GLU B  20      28.729 -20.361  31.627  1.00 82.64           O  \nATOM    655  N   GLU B  21      29.772 -24.738  29.814  1.00 56.81           N  \nATOM    656  CA  GLU B  21      29.511 -26.146  29.520  1.00 53.63           C  \nATOM    657  C   GLU B  21      29.987 -26.505  28.121  1.00 52.09           C  \nATOM    658  O   GLU B  21      30.416 -27.635  27.874  1.00 50.79           O  \nATOM    659  CB  GLU B  21      28.023 -26.466  29.660  1.00 53.23           C  \nATOM    660  CG  GLU B  21      27.581 -26.573  31.107  1.00 57.45           C  \nATOM    661  CD  GLU B  21      26.102 -26.842  31.242  1.00 58.49           C  \nATOM    662  OE1 GLU B  21      25.370 -26.617  30.247  1.00 58.46           O  \nATOM    663  OE2 GLU B  21      25.678 -27.262  32.342  1.00 59.49           O  \nATOM    664  N   VAL B  22      29.890 -25.546  27.203  1.00 50.79           N  \nATOM    665  CA  VAL B  22      30.347 -25.756  25.832  1.00 51.61           C  \nATOM    666  C   VAL B  22      31.869 -25.868  25.931  1.00 50.48           C  \nATOM    667  O   VAL B  22      32.485 -26.731  25.305  1.00 48.12           O  \nATOM    668  CB  VAL B  22      29.943 -24.559  24.896  1.00 51.69           C  \nATOM    669  CG1 VAL B  22      30.867 -24.471  23.692  1.00 52.55           C  \nATOM    670  CG2 VAL B  22      28.521 -24.757  24.384  1.00 51.90           C  \nATOM    671  N   LYS B  23      32.454 -25.008  26.761  1.00 51.15           N  \nATOM    672  CA  LYS B  23      33.891 -24.978  26.973  1.00 52.73           C  \nATOM    673  C   LYS B  23      34.422 -26.282  27.569  1.00 51.50           C  \nATOM    674  O   LYS B  23      35.446 -26.797  27.128  1.00 52.63           O  \nATOM    675  CB  LYS B  23      34.247 -23.795  27.873  1.00 56.44           C  \nATOM    676  CG  LYS B  23      35.726 -23.490  27.930  1.00 60.90           C  \nATOM    677  CD  LYS B  23      35.976 -22.155  28.599  1.00 63.82           C  \nATOM    678  CE  LYS B  23      36.893 -22.333  29.800  1.00 67.19           C  \nATOM    679  NZ  LYS B  23      37.705 -21.121  30.130  1.00 69.13           N  \nATOM    680  N   ALA B  24      33.727 -26.825  28.562  1.00 50.23           N  \nATOM    681  CA  ALA B  24      34.164 -28.070  29.175  1.00 47.47           C  \nATOM    682  C   ALA B  24      34.019 -29.225  28.197  1.00 47.89           C  \nATOM    683  O   ALA B  24      34.818 -30.165  28.206  1.00 48.13           O  \nATOM    684  CB  ALA B  24      33.357 -28.345  30.407  1.00 44.03           C  \nATOM    685  N   PHE B  25      32.999 -29.144  27.347  1.00 46.65           N  \nATOM    686  CA  PHE B  25      32.728 -30.198  26.379  1.00 46.20           C  \nATOM    687  C   PHE B  25      33.762 -30.258  25.268  1.00 48.36           C  \nATOM    688  O   PHE B  25      34.266 -31.333  24.928  1.00 48.16           O  \nATOM    689  CB  PHE B  25      31.324 -30.030  25.776  1.00 44.92           C  \nATOM    690  CG  PHE B  25      30.962 -31.111  24.803  1.00 46.42           C  \nATOM    691  CD1 PHE B  25      30.988 -30.869  23.437  1.00 47.67           C  \nATOM    692  CD2 PHE B  25      30.659 -32.397  25.253  1.00 49.02           C  \nATOM    693  CE1 PHE B  25      30.732 -31.886  22.526  1.00 49.24           C  \nATOM    694  CE2 PHE B  25      30.402 -33.428  24.352  1.00 49.86           C  \nATOM    695  CZ  PHE B  25      30.437 -33.172  22.980  1.00 49.93           C  \nATOM    696  N   VAL B  26      34.061 -29.102  24.691  1.00 48.36           N  \nATOM    697  CA  VAL B  26      35.038 -29.023  23.625  1.00 50.83           C  \nATOM    698  C   VAL B  26      36.385 -29.508  24.181  1.00 51.61           C  \nATOM    699  O   VAL B  26      37.107 -30.277  23.545  1.00 51.51           O  \nATOM    700  CB  VAL B  26      35.144 -27.569  23.124  1.00 52.19           C  \nATOM    701  CG1 VAL B  26      36.507 -27.319  22.511  1.00 52.04           C  \nATOM    702  CG2 VAL B  26      34.015 -27.286  22.106  1.00 51.15           C  \nATOM    703  N   SER B  27      36.704 -29.064  25.387  1.00 51.99           N  \nATOM    704  CA  SER B  27      37.941 -29.451  26.045  1.00 53.35           C  \nATOM    705  C   SER B  27      38.015 -30.975  26.265  1.00 54.05           C  \nATOM    706  O   SER B  27      39.003 -31.629  25.910  1.00 53.39           O  \nATOM    707  CB  SER B  27      38.047 -28.721  27.378  1.00 54.91           C  \nATOM    708  OG  SER B  27      39.054 -29.314  28.175  1.00 64.57           O  \nATOM    709  N   LYS B  28      36.949 -31.523  26.841  1.00 55.10           N  \nATOM    710  CA  LYS B  28      36.825 -32.950  27.115  1.00 55.50           C  \nATOM    711  C   LYS B  28      36.894 -33.767  25.825  1.00 55.10           C  \nATOM    712  O   LYS B  28      37.495 -34.831  25.783  1.00 56.39           O  \nATOM    713  CB  LYS B  28      35.485 -33.222  27.801  1.00 56.98           C  \nATOM    714  CG  LYS B  28      35.193 -34.705  27.980  1.00 60.37           C  \nATOM    715  CD  LYS B  28      33.779 -35.019  28.483  1.00 60.75           C  \nATOM    716  CE  LYS B  28      32.800 -35.288  27.335  1.00 63.32           C  \nATOM    717  NZ  LYS B  28      31.514 -35.943  27.773  1.00 64.17           N  \nATOM    718  N   LYS B  29      36.261 -33.263  24.772  1.00 55.86           N  \nATOM    719  CA  LYS B  29      36.238 -33.959  23.493  1.00 55.69           C  \nATOM    720  C   LYS B  29      37.636 -34.072  22.916  1.00 55.51           C  \nATOM    721  O   LYS B  29      38.018 -35.103  22.359  1.00 56.03           O  \nATOM    722  CB  LYS B  29      35.318 -33.229  22.505  1.00 57.61           C  \nATOM    723  CG  LYS B  29      35.324 -33.816  21.089  1.00 62.10           C  \nATOM    724  CD  LYS B  29      34.128 -33.308  20.268  1.00 67.12           C  \nATOM    725  CE  LYS B  29      34.482 -32.201  19.268  1.00 69.04           C  \nATOM    726  NZ  LYS B  29      34.531 -32.707  17.855  1.00 69.56           N  \nATOM    727  N   ARG B  30      38.385 -32.987  23.057  1.00 55.86           N  \nATOM    728  CA  ARG B  30      39.754 -32.899  22.580  1.00 56.05           C  \nATOM    729  C   ARG B  30      40.626 -33.956  23.285  1.00 53.47           C  \nATOM    730  O   ARG B  30      41.330 -34.718  22.634  1.00 53.79           O  \nATOM    731  CB  ARG B  30      40.288 -31.489  22.870  1.00 59.14           C  \nATOM    732  CG  ARG B  30      41.514 -31.077  22.060  1.00 65.48           C  \nATOM    733  CD  ARG B  30      42.400 -30.060  22.811  1.00 71.24           C  \nATOM    734  NE  ARG B  30      41.671 -28.928  23.397  1.00 76.38           N  \nATOM    735  CZ  ARG B  30      40.968 -28.026  22.712  1.00 78.47           C  \nATOM    736  NH1 ARG B  30      40.880 -28.109  21.390  1.00 78.07           N  \nATOM    737  NH2 ARG B  30      40.365 -27.025  23.349  1.00 78.49           N  \nATOM    738  N   ALA B  31      40.570 -33.986  24.616  1.00 51.84           N  \nATOM    739  CA  ALA B  31      41.335 -34.930  25.429  1.00 51.56           C  \nATOM    740  C   ALA B  31      41.078 -36.364  25.011  1.00 51.25           C  \nATOM    741  O   ALA B  31      42.010 -37.144  24.830  1.00 52.66           O  \nATOM    742  CB  ALA B  31      40.983 -34.760  26.890  1.00 49.83           C  \nATOM    743  N   ILE B  32      39.807 -36.703  24.855  1.00 50.88           N  \nATOM    744  CA  ILE B  32      39.415 -38.047  24.460  1.00 51.68           C  \nATOM    745  C   ILE B  32      39.982 -38.440  23.109  1.00 51.92           C  \nATOM    746  O   ILE B  32      40.498 -39.541  22.955  1.00 51.03           O  \nATOM    747  CB  ILE B  32      37.873 -38.176  24.444  1.00 52.67           C  \nATOM    748  CG1 ILE B  32      37.359 -38.142  25.887  1.00 51.94           C  \nATOM    749  CG2 ILE B  32      37.440 -39.452  23.715  1.00 51.65           C  \nATOM    750  CD1 ILE B  32      35.870 -37.900  26.008  1.00 53.78           C  \nATOM    751  N   LYS B  33      39.902 -37.528  22.142  1.00 54.32           N  \nATOM    752  CA  LYS B  33      40.400 -37.779  20.790  1.00 56.02           C  \nATOM    753  C   LYS B  33      41.911 -37.978  20.776  1.00 55.33           C  \nATOM    754  O   LYS B  33      42.441 -38.768  19.988  1.00 54.86           O  \nATOM    755  CB  LYS B  33      40.033 -36.624  19.855  1.00 59.75           C  \nATOM    756  CG  LYS B  33      39.341 -37.079  18.572  1.00 66.26           C  \nATOM    757  CD  LYS B  33      37.822 -37.135  18.750  1.00 73.32           C  \nATOM    758  CE  LYS B  33      37.090 -37.682  17.512  1.00 76.87           C  \nATOM    759  NZ  LYS B  33      37.291 -39.146  17.265  1.00 78.45           N  \nATOM    760  N   ASN B  34      42.602 -37.249  21.645  1.00 55.38           N  \nATOM    761  CA  ASN B  34      44.043 -37.373  21.755  1.00 54.76           C  \nATOM    762  C   ASN B  34      44.401 -38.721  22.374  1.00 54.18           C  \nATOM    763  O   ASN B  34      45.317 -39.387  21.911  1.00 52.49           O  \nATOM    764  CB  ASN B  34      44.588 -36.221  22.584  1.00 56.84           C  \nATOM    765  CG  ASN B  34      44.664 -34.934  21.786  1.00 60.81           C  \nATOM    766  OD1 ASN B  34      44.787 -33.845  22.346  1.00 62.79           O  \nATOM    767  ND2 ASN B  34      44.604 -35.058  20.458  1.00 60.83           N  \nATOM    768  N   GLU B  35      43.670 -39.127  23.408  1.00 54.61           N  \nATOM    769  CA  GLU B  35      43.900 -40.420  24.055  1.00 57.20           C  \nATOM    770  C   GLU B  35      43.739 -41.539  23.037  1.00 56.57           C  \nATOM    771  O   GLU B  35      44.628 -42.371  22.854  1.00 57.34           O  \nATOM    772  CB  GLU B  35      42.872 -40.686  25.165  1.00 59.86           C  \nATOM    773  CG  GLU B  35      43.149 -40.067  26.520  1.00 65.79           C  \nATOM    774  CD  GLU B  35      44.409 -40.609  27.182  1.00 69.94           C  \nATOM    775  OE1 GLU B  35      44.687 -41.820  27.022  1.00 68.64           O  \nATOM    776  OE2 GLU B  35      45.104 -39.823  27.876  1.00 72.52           O  \nATOM    777  N   GLN B  36      42.583 -41.555  22.380  1.00 56.19           N  \nATOM    778  CA  GLN B  36      42.272 -42.599  21.414  1.00 54.83           C  \nATOM    779  C   GLN B  36      43.321 -42.751  20.331  1.00 53.25           C  \nATOM    780  O   GLN B  36      43.600 -43.857  19.885  1.00 52.42           O  \nATOM    781  CB  GLN B  36      40.905 -42.349  20.777  1.00 55.13           C  \nATOM    782  CG  GLN B  36      39.747 -42.383  21.767  1.00 55.95           C  \nATOM    783  CD  GLN B  36      38.398 -42.064  21.109  1.00 58.62           C  \nATOM    784  OE1 GLN B  36      38.331 -41.380  20.077  1.00 56.92           O  \nATOM    785  NE2 GLN B  36      37.321 -42.544  21.718  1.00 57.52           N  \nATOM    786  N   LEU B  37      43.899 -41.643  19.898  1.00 51.73           N  \nATOM    787  CA  LEU B  37      44.913 -41.726  18.870  1.00 52.55           C  \nATOM    788  C   LEU B  37      46.192 -42.337  19.461  1.00 54.46           C  \nATOM    789  O   LEU B  37      46.791 -43.232  18.849  1.00 53.20           O  \nATOM    790  CB  LEU B  37      45.195 -40.339  18.256  1.00 50.02           C  \nATOM    791  CG  LEU B  37      46.215 -40.370  17.104  1.00 51.08           C  \nATOM    792  CD1 LEU B  37      45.783 -41.417  16.095  1.00 49.26           C  \nATOM    793  CD2 LEU B  37      46.350 -38.999  16.438  1.00 49.37           C  \nATOM    794  N   LEU B  38      46.599 -41.870  20.647  1.00 53.85           N  \nATOM    795  CA  LEU B  38      47.801 -42.388  21.297  1.00 54.34           C  \nATOM    796  C   LEU B  38      47.651 -43.878  21.542  1.00 54.03           C  \nATOM    797  O   LEU B  38      48.549 -44.650  21.248  1.00 54.02           O  \nATOM    798  CB  LEU B  38      48.057 -41.698  22.638  1.00 55.11           C  \nATOM    799  CG  LEU B  38      49.452 -41.092  22.863  1.00 58.71           C  \nATOM    800  CD1 LEU B  38      49.744 -41.154  24.358  1.00 59.75           C  \nATOM    801  CD2 LEU B  38      50.548 -41.836  22.097  1.00 58.46           C  \nATOM    802  N   GLN B  39      46.508 -44.274  22.088  1.00 54.63           N  \nATOM    803  CA  GLN B  39      46.247 -45.679  22.369  1.00 56.09           C  \nATOM    804  C   GLN B  39      46.319 -46.524  21.110  1.00 56.52           C  \nATOM    805  O   GLN B  39      46.907 -47.601  21.103  1.00 57.15           O  \nATOM    806  CB  GLN B  39      44.875 -45.858  23.004  1.00 55.05           C  \nATOM    807  CG  GLN B  39      44.727 -45.133  24.300  1.00 59.26           C  \nATOM    808  CD  GLN B  39      43.564 -45.637  25.128  1.00 63.26           C  \nATOM    809  OE1 GLN B  39      43.289 -45.109  26.210  1.00 64.37           O  \nATOM    810  NE2 GLN B  39      42.882 -46.671  24.638  1.00 64.00           N  \nATOM    811  N   LEU B  40      45.711 -46.037  20.042  1.00 57.03           N  \nATOM    812  CA  LEU B  40      45.720 -46.768  18.789  1.00 57.08           C  \nATOM    813  C   LEU B  40      47.169 -46.938  18.322  1.00 57.00           C  \nATOM    814  O   LEU B  40      47.562 -48.013  17.895  1.00 58.58           O  \nATOM    815  CB  LEU B  40      44.885 -46.026  17.746  1.00 55.60           C  \nATOM    816  CG  LEU B  40      44.707 -46.766  16.416  1.00 57.00           C  \nATOM    817  CD1 LEU B  40      43.423 -46.302  15.762  1.00 57.76           C  \nATOM    818  CD2 LEU B  40      45.903 -46.533  15.495  1.00 57.07           C  \nATOM    819  N   ILE B  41      47.960 -45.875  18.407  1.00 57.00           N  \nATOM    820  CA  ILE B  41      49.367 -45.932  18.017  1.00 57.62           C  \nATOM    821  C   ILE B  41      50.114 -46.934  18.912  1.00 59.23           C  \nATOM    822  O   ILE B  41      50.957 -47.709  18.434  1.00 57.97           O  \nATOM    823  CB  ILE B  41      50.032 -44.547  18.143  1.00 56.94           C  \nATOM    824  CG1 ILE B  41      49.618 -43.692  16.953  1.00 56.82           C  \nATOM    825  CG2 ILE B  41      51.559 -44.676  18.207  1.00 55.56           C  \nATOM    826  CD1 ILE B  41      49.867 -42.228  17.141  1.00 56.84           C  \nATOM    827  N   PHE B  42      49.801 -46.914  20.205  1.00 59.00           N  \nATOM    828  CA  PHE B  42      50.429 -47.823  21.149  1.00 60.08           C  \nATOM    829  C   PHE B  42      50.112 -49.273  20.793  1.00 61.49           C  \nATOM    830  O   PHE B  42      51.009 -50.096  20.605  1.00 60.83           O  \nATOM    831  CB  PHE B  42      49.921 -47.548  22.556  1.00 60.01           C  \nATOM    832  CG  PHE B  42      50.587 -48.383  23.601  1.00 61.05           C  \nATOM    833  CD1 PHE B  42      51.846 -48.033  24.076  1.00 61.05           C  \nATOM    834  CD2 PHE B  42      49.977 -49.534  24.088  1.00 59.79           C  \nATOM    835  CE1 PHE B  42      52.492 -48.813  25.017  1.00 62.04           C  \nATOM    836  CE2 PHE B  42      50.625 -50.331  25.042  1.00 60.84           C  \nATOM    837  CZ  PHE B  42      51.884 -49.963  25.502  1.00 61.47           C  \nATOM    838  N   LYS B  43      48.822 -49.583  20.723  1.00 62.91           N  \nATOM    839  CA  LYS B  43      48.384 -50.932  20.397  1.00 63.72           C  \nATOM    840  C   LYS B  43      48.977 -51.468  19.100  1.00 63.46           C  \nATOM    841  O   LYS B  43      49.152 -52.672  18.953  1.00 62.93           O  \nATOM    842  CB  LYS B  43      46.865 -50.995  20.294  1.00 65.12           C  \nATOM    843  CG  LYS B  43      46.142 -50.945  21.611  1.00 69.50           C  \nATOM    844  CD  LYS B  43      44.661 -51.174  21.371  1.00 72.15           C  \nATOM    845  CE  LYS B  43      43.825 -50.668  22.528  1.00 74.11           C  \nATOM    846  NZ  LYS B  43      42.417 -50.435  22.088  1.00 74.80           N  \nATOM    847  N   SER B  44      49.292 -50.593  18.155  1.00 63.71           N  \nATOM    848  CA  SER B  44      49.831 -51.074  16.894  1.00 65.96           C  \nATOM    849  C   SER B  44      51.308 -51.428  17.008  1.00 67.57           C  \nATOM    850  O   SER B  44      51.799 -52.309  16.305  1.00 70.05           O  \nATOM    851  CB  SER B  44      49.602 -50.044  15.787  1.00 65.66           C  \nATOM    852  OG  SER B  44      50.791 -49.360  15.462  1.00 71.25           O  \nATOM    853  N   ILE B  45      52.021 -50.747  17.896  1.00 67.66           N  \nATOM    854  CA  ILE B  45      53.434 -51.044  18.081  1.00 67.20           C  \nATOM    855  C   ILE B  45      53.536 -52.328  18.908  1.00 67.45           C  \nATOM    856  O   ILE B  45      54.393 -53.188  18.651  1.00 67.32           O  \nATOM    857  CB  ILE B  45      54.167 -49.916  18.837  1.00 66.49           C  \nATOM    858  CG1 ILE B  45      54.004 -48.585  18.101  1.00 65.28           C  \nATOM    859  CG2 ILE B  45      55.645 -50.261  18.952  1.00 66.79           C  \nATOM    860  CD1 ILE B  45      54.562 -47.395  18.852  1.00 62.95           C  \nATOM    861  N   ASP B  46      52.630 -52.438  19.882  1.00 67.39           N  \nATOM    862  CA  ASP B  46      52.529 -53.571  20.802  1.00 67.77           C  \nATOM    863  C   ASP B  46      51.963 -54.808  20.110  1.00 69.95           C  \nATOM    864  O   ASP B  46      50.881 -55.289  20.463  1.00 70.19           O  \nATOM    865  CB  ASP B  46      51.640 -53.175  21.998  1.00 67.43           C  \nATOM    866  CG  ASP B  46      51.406 -54.318  22.968  1.00 68.79           C  \nATOM    867  OD1 ASP B  46      52.064 -55.347  22.789  1.00 69.26           O  \nATOM    868  OD2 ASP B  46      50.576 -54.194  23.901  1.00 68.63           O  \nATOM    869  N   ALA B  47      52.711 -55.334  19.143  1.00 71.51           N  \nATOM    870  CA  ALA B  47      52.283 -56.503  18.385  1.00 73.22           C  \nATOM    871  C   ALA B  47      51.718 -57.686  19.178  1.00 74.80           C  \nATOM    872  O   ALA B  47      50.654 -58.196  18.827  1.00 75.62           O  \nATOM    873  CB  ALA B  47      53.410 -56.968  17.495  1.00 70.79           C  \nATOM    874  N   ASP B  48      52.386 -58.125  20.244  1.00 76.38           N  \nATOM    875  CA  ASP B  48      51.860 -59.274  20.984  1.00 77.53           C  \nATOM    876  C   ASP B  48      50.719 -58.925  21.934  1.00 77.20           C  \nATOM    877  O   ASP B  48      50.255 -59.775  22.705  1.00 79.34           O  \nATOM    878  CB  ASP B  48      52.974 -60.015  21.746  1.00 79.55           C  \nATOM    879  CG  ASP B  48      53.397 -59.320  23.033  1.00 82.18           C  \nATOM    880  OD1 ASP B  48      52.534 -58.721  23.719  1.00 81.91           O  \nATOM    881  OD2 ASP B  48      54.600 -59.403  23.376  1.00 82.63           O  \nATOM    882  N   GLY B  49      50.272 -57.675  21.878  1.00 74.95           N  \nATOM    883  CA  GLY B  49      49.176 -57.226  22.715  1.00 72.66           C  \nATOM    884  C   GLY B  49      49.218 -57.506  24.206  1.00 72.81           C  \nATOM    885  O   GLY B  49      48.168 -57.662  24.826  1.00 72.19           O  \nATOM    886  N   ASN B  50      50.403 -57.534  24.806  1.00 72.26           N  \nATOM    887  CA  ASN B  50      50.480 -57.802  26.236  1.00 72.09           C  \nATOM    888  C   ASN B  50      50.484 -56.522  27.085  1.00 71.52           C  \nATOM    889  O   ASN B  50      50.623 -56.600  28.306  1.00 71.51           O  \nATOM    890  CB  ASN B  50      51.715 -58.650  26.554  1.00 72.35           C  \nATOM    891  CG  ASN B  50      52.976 -57.840  26.547  1.00 73.15           C  \nATOM    892  OD1 ASN B  50      52.972 -56.690  26.109  1.00 73.49           O  \nATOM    893  ND2 ASN B  50      54.064 -58.420  27.033  1.00 73.37           N  \nATOM    894  N   GLY B  51      50.354 -55.350  26.452  1.00 70.35           N  \nATOM    895  CA  GLY B  51      50.305 -54.098  27.204  1.00 68.97           C  \nATOM    896  C   GLY B  51      51.596 -53.334  27.463  1.00 68.94           C  \nATOM    897  O   GLY B  51      51.588 -52.288  28.124  1.00 68.76           O  \nATOM    898  N   GLU B  52      52.704 -53.833  26.932  1.00 68.03           N  \nATOM    899  CA  GLU B  52      53.995 -53.191  27.117  1.00 69.35           C  \nATOM    900  C   GLU B  52      54.672 -53.318  25.765  1.00 68.43           C  \nATOM    901  O   GLU B  52      54.391 -54.256  25.049  1.00 68.10           O  \nATOM    902  CB  GLU B  52      54.811 -53.979  28.142  1.00 72.30           C  \nATOM    903  CG  GLU B  52      55.535 -55.162  27.491  1.00 76.21           C  \nATOM    904  CD  GLU B  52      56.199 -56.109  28.475  1.00 78.68           C  \nATOM    905  OE1 GLU B  52      56.922 -57.021  28.006  1.00 75.56           O  \nATOM    906  OE2 GLU B  52      55.998 -55.952  29.704  1.00 80.23           O  \nATOM    907  N   ILE B  53      55.547 -52.403  25.378  1.00 67.23           N  \nATOM    908  CA  ILE B  53      56.202 -52.623  24.104  1.00 66.14           C  \nATOM    909  C   ILE B  53      57.677 -52.917  24.435  1.00 66.42           C  \nATOM    910  O   ILE B  53      58.347 -52.138  25.117  1.00 66.40           O  \nATOM    911  CB  ILE B  53      55.996 -51.417  23.107  1.00 64.91           C  \nATOM    912  CG1 ILE B  53      57.257 -50.590  22.988  1.00 63.04           C  \nATOM    913  CG2 ILE B  53      54.775 -50.589  23.492  1.00 65.64           C  \nATOM    914  CD1 ILE B  53      58.137 -51.164  21.951  1.00 62.03           C  \nATOM    915  N   ASP B  54      58.156 -54.084  24.003  1.00 66.45           N  \nATOM    916  CA  ASP B  54      59.535 -54.499  24.270  1.00 67.20           C  \nATOM    917  C   ASP B  54      60.451 -53.952  23.195  1.00 67.26           C  \nATOM    918  O   ASP B  54      59.969 -53.449  22.179  1.00 68.06           O  \nATOM    919  CB  ASP B  54      59.637 -56.037  24.361  1.00 65.29           C  \nATOM    920  CG  ASP B  54      59.089 -56.747  23.135  1.00 65.48           C  \nATOM    921  OD1 ASP B  54      59.620 -56.522  22.023  1.00 63.45           O  \nATOM    922  OD2 ASP B  54      58.129 -57.536  23.291  1.00 64.55           O  \nATOM    923  N   GLN B  55      61.760 -54.029  23.422  1.00 67.85           N  \nATOM    924  CA  GLN B  55      62.727 -53.501  22.464  1.00 68.49           C  \nATOM    925  C   GLN B  55      62.537 -54.066  21.070  1.00 68.61           C  \nATOM    926  O   GLN B  55      62.926 -53.436  20.081  1.00 68.25           O  \nATOM    927  CB  GLN B  55      64.159 -53.796  22.908  1.00 68.45           C  \nATOM    928  CG  GLN B  55      64.412 -53.559  24.367  1.00 68.94           C  \nATOM    929  CD  GLN B  55      65.874 -53.405  24.676  1.00 69.44           C  \nATOM    930  OE1 GLN B  55      66.739 -53.786  23.888  1.00 71.16           O  \nATOM    931  NE2 GLN B  55      66.165 -52.852  25.832  1.00 69.06           N  \nATOM    932  N   ASN B  56      61.963 -55.261  20.985  1.00 69.67           N  \nATOM    933  CA  ASN B  56      61.750 -55.882  19.684  1.00 71.82           C  \nATOM    934  C   ASN B  56      60.597 -55.201  18.956  1.00 70.10           C  \nATOM    935  O   ASN B  56      60.739 -54.777  17.806  1.00 69.82           O  \nATOM    936  CB  ASN B  56      61.477 -57.377  19.837  1.00 76.43           C  \nATOM    937  CG  ASN B  56      61.832 -58.150  18.585  1.00 80.47           C  \nATOM    938  OD1 ASN B  56      60.951 -58.610  17.854  1.00 81.33           O  \nATOM    939  ND2 ASN B  56      63.136 -58.278  18.316  1.00 80.76           N  \nATOM    940  N   GLU B  57      59.459 -55.090  19.628  1.00 68.33           N  \nATOM    941  CA  GLU B  57      58.316 -54.412  19.036  1.00 65.56           C  \nATOM    942  C   GLU B  57      58.729 -53.006  18.662  1.00 65.96           C  \nATOM    943  O   GLU B  57      58.368 -52.518  17.601  1.00 66.55           O  \nATOM    944  CB  GLU B  57      57.176 -54.364  20.028  1.00 62.01           C  \nATOM    945  CG  GLU B  57      56.679 -55.754  20.286  1.00 58.21           C  \nATOM    946  CD  GLU B  57      55.801 -55.857  21.506  1.00 60.93           C  \nATOM    947  OE1 GLU B  57      55.920 -55.015  22.413  1.00 62.81           O  \nATOM    948  OE2 GLU B  57      54.994 -56.797  21.588  1.00 59.27           O  \nATOM    949  N   PHE B  58      59.512 -52.374  19.531  1.00 65.66           N  \nATOM    950  CA  PHE B  58      59.980 -51.009  19.308  1.00 67.39           C  \nATOM    951  C   PHE B  58      60.891 -50.889  18.107  1.00 68.93           C  \nATOM    952  O   PHE B  58      60.823 -49.910  17.370  1.00 69.50           O  \nATOM    953  CB  PHE B  58      60.741 -50.498  20.541  1.00 66.40           C  \nATOM    954  CG  PHE B  58      61.227 -49.067  20.426  1.00 65.68           C  \nATOM    955  CD1 PHE B  58      60.480 -48.013  20.949  1.00 65.30           C  \nATOM    956  CD2 PHE B  58      62.457 -48.772  19.835  1.00 63.56           C  \nATOM    957  CE1 PHE B  58      60.955 -46.701  20.887  1.00 63.64           C  \nATOM    958  CE2 PHE B  58      62.931 -47.444  19.774  1.00 61.96           C  \nATOM    959  CZ  PHE B  58      62.182 -46.425  20.300  1.00 61.74           C  \nATOM    960  N   ALA B  59      61.767 -51.870  17.929  1.00 72.18           N  \nATOM    961  CA  ALA B  59      62.727 -51.854  16.834  1.00 73.76           C  \nATOM    962  C   ALA B  59      62.063 -51.994  15.470  1.00 74.89           C  \nATOM    963  O   ALA B  59      62.431 -51.301  14.518  1.00 73.44           O  \nATOM    964  CB  ALA B  59      63.742 -52.954  17.037  1.00 73.68           C  \nATOM    965  N   LYS B  60      61.097 -52.902  15.366  1.00 76.67           N  \nATOM    966  CA  LYS B  60      60.393 -53.086  14.103  1.00 79.47           C  \nATOM    967  C   LYS B  60      59.835 -51.729  13.694  1.00 81.18           C  \nATOM    968  O   LYS B  60      60.111 -51.221  12.602  1.00 81.09           O  \nATOM    969  CB  LYS B  60      59.247 -54.081  14.272  1.00 81.66           C  \nATOM    970  CG  LYS B  60      59.703 -55.529  14.374  1.00 85.25           C  \nATOM    971  CD  LYS B  60      58.569 -56.428  14.851  1.00 87.92           C  \nATOM    972  CE  LYS B  60      58.677 -57.826  14.256  1.00 89.39           C  \nATOM    973  NZ  LYS B  60      60.045 -58.403  14.390  1.00 89.82           N  \nATOM    974  N   PHE B  61      59.065 -51.141  14.604  1.00 82.47           N  \nATOM    975  CA  PHE B  61      58.443 -49.844  14.392  1.00 83.42           C  \nATOM    976  C   PHE B  61      59.376 -48.802  13.791  1.00 84.97           C  \nATOM    977  O   PHE B  61      59.062 -48.217  12.762  1.00 85.61           O  \nATOM    978  CB  PHE B  61      57.872 -49.339  15.717  1.00 82.64           C  \nATOM    979  CG  PHE B  61      57.173 -48.016  15.618  1.00 82.58           C  \nATOM    980  CD1 PHE B  61      55.924 -47.903  15.002  1.00 82.41           C  \nATOM    981  CD2 PHE B  61      57.753 -46.877  16.170  1.00 81.76           C  \nATOM    982  CE1 PHE B  61      55.274 -46.670  14.937  1.00 81.08           C  \nATOM    983  CE2 PHE B  61      57.106 -45.644  16.104  1.00 81.98           C  \nATOM    984  CZ  PHE B  61      55.865 -45.546  15.491  1.00 81.41           C  \nATOM    985  N   TYR B  62      60.525 -48.562  14.410  1.00 87.26           N  \nATOM    986  CA  TYR B  62      61.416 -47.554  13.861  1.00 90.70           C  \nATOM    987  C   TYR B  62      62.146 -47.951  12.580  1.00 93.31           C  \nATOM    988  O   TYR B  62      62.203 -49.125  12.193  1.00 93.24           O  \nATOM    989  CB  TYR B  62      62.411 -47.070  14.922  1.00 91.91           C  \nATOM    990  CG  TYR B  62      61.838 -45.973  15.796  1.00 94.28           C  \nATOM    991  CD1 TYR B  62      61.161 -46.276  16.975  1.00 94.86           C  \nATOM    992  CD2 TYR B  62      61.904 -44.634  15.405  1.00 95.39           C  \nATOM    993  CE1 TYR B  62      60.564 -45.274  17.747  1.00 95.25           C  \nATOM    994  CE2 TYR B  62      61.304 -43.621  16.173  1.00 95.65           C  \nATOM    995  CZ  TYR B  62      60.636 -43.951  17.339  1.00 95.27           C  \nATOM    996  OH  TYR B  62      60.047 -42.955  18.083  1.00 95.09           O  \nATOM    997  N   GLY B  63      62.677 -46.940  11.910  1.00 95.58           N  \nATOM    998  CA  GLY B  63      63.396 -47.140  10.666  1.00 98.12           C  \nATOM    999  C   GLY B  63      63.167 -45.786  10.055  1.00 99.57           C  \nATOM   1000  O   GLY B  63      63.401 -44.775  10.728  1.00100.42           O  \nATOM   1001  N   SER B  64      62.748 -45.734   8.794  1.00100.22           N  \nATOM   1002  CA  SER B  64      62.397 -44.442   8.222  1.00100.36           C  \nATOM   1003  C   SER B  64      61.071 -44.334   8.974  1.00 99.60           C  \nATOM   1004  O   SER B  64      60.523 -43.249   9.183  1.00 97.73           O  \nATOM   1005  CB  SER B  64      62.116 -44.524   6.712  1.00101.67           C  \nATOM   1006  OG  SER B  64      63.291 -44.714   5.943  1.00102.08           O  \nATOM   1007  N   ILE B  65      60.618 -45.522   9.393  1.00100.22           N  \nATOM   1008  CA  ILE B  65      59.391 -45.810  10.127  1.00 99.78           C  \nATOM   1009  C   ILE B  65      58.394 -46.341   9.116  1.00100.54           C  \nATOM   1010  O   ILE B  65      58.840 -47.035   8.179  1.00100.36           O  \nATOM   1011  CB  ILE B  65      58.836 -44.589  10.919  1.00 99.33           C  \nATOM   1012  CG1 ILE B  65      59.758 -44.312  12.111  1.00 98.72           C  \nATOM   1013  CG2 ILE B  65      57.466 -44.909  11.520  1.00 97.47           C  \nATOM   1014  CD1 ILE B  65      59.162 -43.387  13.162  1.00100.37           C  \nTER    1015      ILE B  65                                                      \nHETATM 1016  CA  CA  A 149      36.051 -49.233  -2.416  1.00 65.06          CA  \nHETATM 1017  CA  CA  A 150      25.879  -7.406   5.735  1.00 64.64          CA  \nHETATM 1018  C   ACT A 201      38.990 -11.905   9.521  1.00 87.25           C  \nHETATM 1019  O   ACT A 201      38.407 -11.169   8.564  1.00 86.82           O  \nHETATM 1020  OXT ACT A 201      38.588 -13.031   9.758  1.00 86.12           O  \nHETATM 1021  CH3 ACT A 201      40.150 -11.335  10.300  1.00 86.21           C  \nHETATM 1022  C   ACT A 203      32.096 -26.174  13.836  1.00 87.40           C  \nHETATM 1023  O   ACT A 203      31.345 -27.267  13.951  1.00 86.37           O  \nHETATM 1024  OXT ACT A 203      33.289 -26.285  13.699  1.00 87.07           O  \nHETATM 1025  CH3 ACT A 203      31.461 -24.807  13.877  1.00 87.30           C  \nHETATM 1026  CA  CA  B 151      23.108 -26.574  31.768  1.00 62.37          CA  \nHETATM 1027  CA  CA  B 152      54.071 -56.397  23.802  1.00 66.61          CA  \nHETATM 1028  C   ACT B 202      56.579 -42.671  19.825  1.00 89.57           C  \nHETATM 1029  O   ACT B 202      55.261 -42.687  19.630  1.00 88.87           O  \nHETATM 1030  OXT ACT B 202      57.070 -43.323  20.743  1.00 89.04           O  \nHETATM 1031  CH3 ACT B 202      57.439 -41.834  18.901  1.00 88.76           C  \nHETATM 1032  C   ACT B 204      40.979 -41.637  15.598  1.00 85.82           C  \nHETATM 1033  O   ACT B 204      39.678 -41.786  15.437  1.00 85.24           O  \nHETATM 1034  OXT ACT B 204      41.400 -40.573  15.980  1.00 86.18           O  \nHETATM 1035  CH3 ACT B 204      41.911 -42.791  15.307  1.00 85.54           C  \nHETATM 1036  C   ACT B 205      37.353 -24.645  18.793  1.00 98.62           C  \nHETATM 1037  O   ACT B 205      37.260 -25.492  19.782  1.00 98.54           O  \nHETATM 1038  OXT ACT B 205      37.899 -23.578  19.008  1.00 97.84           O  \nHETATM 1039  CH3 ACT B 205      36.777 -25.030  17.429  1.00 97.56           C  \nHETATM 1040  O   HOH A 204      29.876 -49.594   0.516  1.00 45.91           O  \nHETATM 1041  O   HOH A 205      45.834 -36.457   3.111  1.00 85.65           O  \nHETATM 1042  O   HOH A 206      26.586 -44.315   8.875  1.00 93.71           O  \nHETATM 1043  O   HOH A 207      43.838 -48.225  -5.291  1.00 91.41           O  \nHETATM 1044  O   HOH A 208      38.026 -26.033   2.594  1.00 56.76           O  \nHETATM 1045  O   HOH A 209      32.655 -54.899   3.161  1.00 72.45           O  \nHETATM 1046  O   HOH A 210      36.422 -33.187  12.398  1.00 83.35           O  \nHETATM 1047  O   HOH A 211      38.814 -33.276  10.483  1.00 63.96           O  \nHETATM 1048  O   HOH A 212      24.177 -12.740  10.983  1.00 68.50           O  \nHETATM 1049  O   HOH A 213      23.753 -15.977  10.852  1.00 81.71           O  \nHETATM 1050  O   HOH A 214      22.126 -49.275   7.543  1.00 82.73           O  \nHETATM 1051  O   HOH A 215      38.625 -49.736  -3.385  1.00 68.29           O  \nHETATM 1052  O   HOH A 216      25.551  -4.655   4.580  1.00 79.45           O  \nHETATM 1053  O   HOH B 206      16.186 -26.765  26.266  1.00 79.89           O  \nHETATM 1054  O   HOH B 207      27.045 -27.002  34.575  1.00 58.46           O  \nHETATM 1055  O   HOH B 208      37.105 -18.861  34.209  1.00 85.93           O  \nHETATM 1056  O   HOH B 209      47.291 -52.834  15.236  1.00 57.96           O  \nHETATM 1057  O   HOH B 210      23.710 -38.163  19.820  1.00 85.15           O  \nHETATM 1058  O   HOH B 211      58.159 -49.447   9.934  1.00 81.22           O  \nHETATM 1059  O   HOH B 212      48.243 -54.772  20.435  1.00 71.27           O  \nHETATM 1060  O   HOH B 213      46.368 -41.846  29.869  1.00 81.64           O  \nHETATM 1061  O   HOH B 214      36.147 -30.929  30.474  1.00 61.55           O  \nHETATM 1062  O   HOH B 215      26.196 -34.947  30.952  1.00 65.46           O  \nHETATM 1063  O   HOH B 216      21.431 -36.159  20.185  1.00 88.02           O  \nHETATM 1064  O   HOH B 217      39.613 -22.785  24.139  1.00 92.63           O  \nHETATM 1065  O   HOH B 218      41.805 -29.392  19.283  1.00 95.35           O  \nHETATM 1066  O   HOH B 219      47.791 -28.358  18.363  0.33 92.30           O  \nHETATM 1067  O   HOH B 220      46.371 -54.201  17.801  1.00 71.13           O  \nHETATM 1068  O   HOH B 221      49.314 -55.492  17.415  1.00 73.72           O  \nHETATM 1069  O   HOH B 222      23.815 -24.118  32.751  1.00 65.43           O  \nHETATM 1070  O   HOH B 223      56.235 -57.917  25.216  1.00 71.15           O  \nEND                                                                             \n","pdb");
	viewer_17581787016276908.setStyle({"model": 1},{"cartoon": {"color": "red"}});
	viewer_17581787016276908.zoomTo();
	viewer_17581787016276908.zoom(1.2);
	viewer_17581787016276908.setBackgroundColor("white");
viewer_17581787016276908.render();
});
</script>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=0d6dc04a">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="AlphaFold-predictions">AlphaFold predictions<a class="anchor-link" href="#AlphaFold-predictions"></a></h2>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=c8628fb7">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-AF-prediction-to-fold-1">Best AF prediction to fold 1<a class="anchor-link" href="#Best-AF-prediction-to-fold-1"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=e7c9b423">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[8]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="kn">from</span><span class="w"> </span><span class="nn">pathlib</span><span class="w"> </span><span class="kn">import</span> <span class="n">Path</span>

<span class="n">df_af</span> <span class="o">=</span> <span class="n">load_csv_or_none</span><span class="p">(</span><span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s2">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s2">/Analysis/df_af.csv"</span><span class="p">)</span>
<span class="k">if</span> <span class="n">df_af</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
    <span class="c1"># keep your existing filters</span>
    <span class="n">df_af</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">cluster_num</span> <span class="o">!=</span> <span class="s1">'Query'</span><span class="p">]</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span> <span class="mi">1</span><span class="p">:</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span>
    <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[run] AlphaFold df ready: </span><span class="si">{</span><span class="nb">len</span><span class="p">(</span><span class="n">df_af</span><span class="p">)</span><span class="si">}</span><span class="s2"> rows"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] AlphaFold section (df_af.csv not available)"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[ok] loaded: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1jfkA_2nxqB/Analysis/df_af.csv  rows=176
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[run] AlphaFold df ready: 176 rows
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=03ee9989">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[9]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">MAX_AF_1</span><span class="p">,</span> <span class="n">max_af_pdb1</span> <span class="o">=</span> <span class="s1">''</span><span class="p">,</span> <span class="s1">''</span>
<span class="k">if</span> <span class="n">df_af</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_af</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">max_af_pdb1</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&gt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb1'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">pdb_file</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">max_af_pdb1</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/AF_preds/</span><span class="si">{</span><span class="n">max_af_pdb1</span><span class="si">}</span><span class="s1">'</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">MAX_AF_1</span> <span class="o">=</span> <span class="n">max_af_pdb1</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s1">'/'</span><span class="p">)[</span><span class="o">-</span><span class="mi">1</span><span class="p">][</span><span class="mi">11</span><span class="p">:</span><span class="mi">14</span><span class="p">]</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="n">e</span><span class="p">)</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&gt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb1'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">score_pdb1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">max_af_pdb1</span><span class="p">,</span><span class="n">pdb_path1</span><span class="p">,</span><span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 alignment: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">max_af_pdb1</span><span class="p">,</span><span class="n">pdb_path1</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e3</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 fallback visualization: </span><span class="si">{</span><span class="n">e3</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] AF fold1: df_af not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] AF fold1 alignment: 'DataFrame' object has no attribute 'score_pdb1'
[skip] AF fold1 fallback visualization:  is not a valid filename or a valid PDB identifier.
</pre>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=5584ce3b">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-AF-prediction-to-fold-2">Best AF prediction to fold 2<a class="anchor-link" href="#Best-AF-prediction-to-fold-2"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=5473c6cd">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[10]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">MAX_AF_2</span><span class="p">,</span> <span class="n">max_af_pdb2</span> <span class="o">=</span> <span class="s1">''</span><span class="p">,</span> <span class="s1">''</span>
<span class="k">if</span> <span class="n">df_af</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_af</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">max_af_pdb2</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&lt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb2'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">pdb_file</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">max_af_pdb2</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/AF_preds/</span><span class="si">{</span><span class="n">max_af_pdb2</span><span class="si">}</span><span class="s1">'</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">MAX_AF_2</span> <span class="o">=</span> <span class="n">max_af_pdb2</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s1">'/'</span><span class="p">)[</span><span class="o">-</span><span class="mi">1</span><span class="p">][</span><span class="mi">11</span><span class="p">:</span><span class="mi">14</span><span class="p">]</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="n">e</span><span class="p">)</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_af</span><span class="p">[</span><span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb1</span> <span class="o">&lt;</span> <span class="n">df_af</span><span class="o">.</span><span class="n">score_pdb2</span><span class="p">]</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'score_pdb2'</span><span class="p">,</span><span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">score_pdb2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">max_af_pdb2</span><span class="p">,</span><span class="n">pdb_path2</span><span class="p">,</span><span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 alignment: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">max_af_pdb2</span><span class="p">,</span><span class="n">pdb_path2</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e3</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] AF fold1 fallback visualization: </span><span class="si">{</span><span class="n">e3</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] AF fold1: df_af not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] AF fold1 alignment: 'DataFrame' object has no attribute 'score_pdb1'
[skip] AF fold1 fallback visualization:  is not a valid filename or a valid PDB identifier.
</pre>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=a817e741">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="EsmFold-Prediction">EsmFold Prediction<a class="anchor-link" href="#EsmFold-Prediction"></a></h2>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=d914376c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[11]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="kn">from</span><span class="w"> </span><span class="nn">pathlib</span><span class="w"> </span><span class="kn">import</span> <span class="n">Path</span>
<span class="kn">import</span><span class="w"> </span><span class="nn">os</span><span class="o">,</span><span class="w"> </span><span class="nn">pandas</span><span class="w"> </span><span class="k">as</span><span class="w"> </span><span class="nn">pd</span>

<span class="c1"># Inputs from the generator (below patch passes these env vars)</span>
<span class="n">ROOT</span> <span class="o">=</span> <span class="n">Path</span><span class="p">(</span><span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s2">"MSACLUSTER_ROOT"</span><span class="p">,</span> <span class="s2">"."</span><span class="p">))</span><span class="o">.</span><span class="n">resolve</span><span class="p">()</span>
<span class="n">PAIR_ID</span> <span class="o">=</span> <span class="n">os</span><span class="o">.</span><span class="n">environ</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s2">"PAIR_ID"</span><span class="p">,</span> <span class="kc">None</span><span class="p">)</span> <span class="ow">or</span> <span class="n">fold_pair</span>  <span class="c1"># keep using fold_pair if already defined</span>

<span class="n">PAIR_ANALYSIS</span> <span class="o">=</span> <span class="n">ROOT</span> <span class="o">/</span> <span class="s2">"Pipeline"</span> <span class="o">/</span> <span class="n">PAIR_ID</span> <span class="o">/</span> <span class="s2">"Analysis"</span>
<span class="n">GLOBAL_RES</span>    <span class="o">=</span> <span class="n">ROOT</span> <span class="o">/</span> <span class="s2">"Pipeline_res"</span>

<span class="k">def</span><span class="w"> </span><span class="nf">_read_csv_or_none</span><span class="p">(</span><span class="n">p</span><span class="p">:</span> <span class="n">Path</span><span class="p">):</span>
    <span class="k">return</span> <span class="n">pd</span><span class="o">.</span><span class="n">read_csv</span><span class="p">(</span><span class="n">p</span><span class="p">)</span> <span class="k">if</span> <span class="n">p</span><span class="o">.</span><span class="n">exists</span><span class="p">()</span> <span class="ow">and</span> <span class="n">p</span><span class="o">.</span><span class="n">stat</span><span class="p">()</span><span class="o">.</span><span class="n">st_size</span> <span class="o">&gt;</span> <span class="mi">1</span> <span class="k">else</span> <span class="kc">None</span>

<span class="c1"># Prefer per-pair CSV; only fall back to global CSV if really needed</span>
<span class="n">df_esmfold_analysis</span> <span class="o">=</span> <span class="n">_read_csv_or_none</span><span class="p">(</span><span class="n">PAIR_ANALYSIS</span> <span class="o">/</span> <span class="s2">"df_esm.csv"</span><span class="p">)</span>
<span class="k">if</span> <span class="n">df_esmfold_analysis</span> <span class="ow">is</span> <span class="kc">None</span><span class="p">:</span>
    <span class="n">df_esmfold_analysis</span> <span class="o">=</span> <span class="n">_read_csv_or_none</span><span class="p">(</span><span class="n">GLOBAL_RES</span> <span class="o">/</span> <span class="s2">"df_esmfold_analysis.csv"</span><span class="p">)</span>

<span class="n">df_esmfold</span> <span class="o">=</span> <span class="kc">None</span>
<span class="k">if</span> <span class="n">df_esmfold_analysis</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
    <span class="c1"># Your original filtering logic</span>
    <span class="n">df_esmfold</span> <span class="o">=</span> <span class="n">df_esmfold_analysis</span><span class="p">[</span><span class="n">df_esmfold_analysis</span><span class="o">.</span><span class="n">get</span><span class="p">(</span><span class="s2">"fold_pair"</span><span class="p">,</span> <span class="n">PAIR_ID</span><span class="p">)</span> <span class="o">==</span> <span class="n">PAIR_ID</span><span class="p">]</span>
    <span class="k">if</span> <span class="s2">"fold"</span> <span class="ow">in</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">columns</span><span class="p">:</span>
        <span class="n">df_esmfold</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="p">[</span><span class="n">df_esmfold</span><span class="p">[</span><span class="s2">"fold"</span><span class="p">]</span><span class="o">.</span><span class="n">astype</span><span class="p">(</span><span class="nb">str</span><span class="p">)</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="s2">"ShallowMsa"</span><span class="p">)]</span>
    <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[run] ESMFold df ready: </span><span class="si">{</span><span class="nb">len</span><span class="p">(</span><span class="n">df_esmfold</span><span class="p">)</span><span class="si">}</span><span class="s2"> rows"</span><span class="p">)</span>
    <span class="n">display</span><span class="p">(</span><span class="n">df_esmfold</span><span class="o">.</span><span class="n">head</span><span class="p">())</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESMFold section (no per-pair df_esm.csv and no global df_esmfold_analysis.csv)"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[run] ESMFold df ready: 860 rows
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output" data-mime-type="text/html" tabindex="0">
<div>
<style scoped="">
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
<thead>
<tr style="text-align: right;">
<th></th>
<th>fold_pair</th>
<th>model</th>
<th>cluster_num</th>
<th>name</th>
<th>TMscore_fold1</th>
<th>TMscore_fold2</th>
<th>TMdiff</th>
</tr>
</thead>
<tbody>
<tr>
<th>0</th>
<td>1jfkA_2nxqB</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_001</td>
<td>0.368917</td>
<td>0.567255</td>
<td>-0.198338</td>
</tr>
<tr>
<th>1</th>
<td>1jfkA_2nxqB</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_002</td>
<td>0.482794</td>
<td>0.296676</td>
<td>0.186118</td>
</tr>
<tr>
<th>2</th>
<td>1jfkA_2nxqB</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_003</td>
<td>0.545994</td>
<td>0.302425</td>
<td>0.243569</td>
</tr>
<tr>
<th>3</th>
<td>1jfkA_2nxqB</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_004</td>
<td>0.545994</td>
<td>0.302425</td>
<td>0.243569</td>
</tr>
<tr>
<th>4</th>
<td>1jfkA_2nxqB</td>
<td>esm2</td>
<td>ShallowMsa_000</td>
<td>ShallowMsa_000__sample_005</td>
<td>0.494658</td>
<td>0.326380</td>
<td>0.168278</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=2ad0b9e2">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-EsmFold-prediction-to-fold-1">Best EsmFold prediction to fold 1<a class="anchor-link" href="#Best-EsmFold-prediction-to-fold-1"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=c9be45fc">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[12]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">if</span> <span class="n">df_esmfold</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="c1"># keep your original selection logic here (fixed snippets below are placeholders)</span>
        <span class="n">max_esm_pdb1</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold1'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">fold</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold1'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">TMscore_fold1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb1</span><span class="si">}</span><span class="s1">'</span><span class="p">,</span> <span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold1 align/plot: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">pdb_path1</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb1</span><span class="si">}</span><span class="s1">'</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold1 fallback visualization: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESM fold1: df_esmfold not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] ESM fold1 align/plot: 'DataFrame' object has no attribute 'fold'
[skip] ESM fold1 fallback visualization: name 'max_esm_pdb1' is not defined
</pre>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=1c4279db">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h3 id="Best-EsmFold-prediction-to-fold-2">Best EsmFold prediction to fold 2<a class="anchor-link" href="#Best-EsmFold-prediction-to-fold-2"></a></h3>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=3f4fda59">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[13]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">if</span> <span class="n">df_esmfold</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">max_esm_pdb2</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold2'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">fold</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">score</span> <span class="o">=</span> <span class="n">df_esmfold</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'TMscore_fold2'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">TMscore_fold2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_tool</span><span class="o">.</span><span class="n">align_and_visualize_pdb</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb2</span><span class="si">}</span><span class="s1">'</span><span class="p">,</span> <span class="n">score</span><span class="p">)</span>
        <span class="nb">print</span><span class="p">(</span><span class="n">score</span><span class="p">)</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold2 align/plot: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
        <span class="k">try</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="n">score</span><span class="p">)</span>
            <span class="n">plot_tool</span><span class="o">.</span><span class="n">plot_fold_alignement_</span><span class="p">(</span><span class="n">pdb_path2</span><span class="p">,</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">plot_tool</span><span class="o">.</span><span class="n">folder</span><span class="si">}</span><span class="s1">/</span><span class="si">{</span><span class="n">fold_pair</span><span class="si">}</span><span class="s1">/output_esm_fold/</span><span class="si">{</span><span class="n">max_esm_pdb2</span><span class="si">}</span><span class="s1">'</span><span class="p">)</span>
        <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e2</span><span class="p">:</span>
            <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] ESM fold2 fallback visualization: </span><span class="si">{</span><span class="n">e2</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESM fold2: df_esmfold not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] ESM fold2 align/plot: 'DataFrame' object has no attribute 'fold'
[skip] ESM fold2 fallback visualization: name 'score' is not defined
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=4daa9929">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[14]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="k">if</span> <span class="n">df_esmfold</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span><span class="p">:</span>
    <span class="n">df_esmfold</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] ESM df display (not available)"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
</div>
<div class="jp-Cell jp-MarkdownCell jp-Notebook-cell" id="cell-id=caed9007">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
<h2 id="Contact-maps-Msa-Transformers">Contact maps Msa Transformers<a class="anchor-link" href="#Contact-maps-Msa-Transformers"></a></h2>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=56fbbee9">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[15]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="c1"># Robust load of CMap CSV referenced by CMAP_RES_PATH (if defined)</span>
<span class="n">df_cmap</span> <span class="o">=</span> <span class="kc">None</span>
<span class="k">try</span><span class="p">:</span>
    <span class="n">CMAP_RES_PATH</span>  <span class="c1"># just to see if it exists</span>
    <span class="n">df_cmap</span> <span class="o">=</span> <span class="n">load_csv_or_none</span><span class="p">(</span><span class="n">CMAP_RES_PATH</span><span class="p">)</span>
<span class="k">except</span> <span class="ne">NameError</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] CMAP_RES_PATH is not defined in this notebook"</span><span class="p">)</span>

<span class="n">MAX_RECALL_1</span> <span class="o">=</span> <span class="n">MAX_RECALL_2</span> <span class="o">=</span> <span class="kc">None</span>
<span class="n">MAX_CLUSTER_RECALL_1</span> <span class="o">=</span> <span class="n">MAX_CLUSTER_RECALL_2</span> <span class="o">=</span> <span class="s1">''</span>

<span class="k">if</span> <span class="n">df_cmap</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="n">df_cmap</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">fold_pair</span> <span class="o">==</span> <span class="n">fold_pair</span><span class="p">]</span>
    <span class="k">try</span><span class="p">:</span>
        <span class="n">MAX_RECALL_1</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold1</span> <span class="o">&gt;</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="p">]</span> \
            <span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'recall_only_fold1'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">MAX_CLUSTER_RECALL_1</span> <span class="o">=</span> <span class="n">MAX_RECALL_1</span><span class="o">.</span><span class="n">File</span><span class="p">[</span><span class="o">-</span><span class="mi">7</span><span class="p">:</span><span class="o">-</span><span class="mi">4</span><span class="p">]</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] MAX_RECALL_1: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>

    <span class="k">try</span><span class="p">:</span>
        <span class="n">MAX_RECALL_2</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold1</span> <span class="o">&lt;</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="p">]</span> \
            <span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="s1">'recall_only_fold2'</span><span class="p">,</span> <span class="n">ascending</span><span class="o">=</span><span class="kc">False</span><span class="p">)</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">MAX_CLUSTER_RECALL_2</span> <span class="o">=</span> <span class="n">MAX_RECALL_2</span><span class="o">.</span><span class="n">File</span><span class="p">[</span><span class="o">-</span><span class="mi">7</span><span class="p">:</span><span class="o">-</span><span class="mi">4</span><span class="p">]</span>
    <span class="k">except</span> <span class="ne">Exception</span> <span class="k">as</span> <span class="n">e</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="sa">f</span><span class="s2">"[skip] MAX_RECALL_2: </span><span class="si">{</span><span class="n">e</span><span class="si">}</span><span class="s2">"</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] df_cmap not available"</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] missing CSV: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/data/df_cmap_all.csv
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] df_cmap not available
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=d06b9bea">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[16]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">viz_folder</span> <span class="o">=</span> <span class="n">VIZ_FOLDER</span>
<span class="k">if</span> <span class="n">MAX_CLUSTER_RECALL_1</span> <span class="o">!=</span> <span class="s1">''</span><span class="p">:</span>
    <span class="n">file_max_recall_1</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">viz_folder</span><span class="si">}</span><span class="s1">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_CLUSTER_RECALL_1</span><span class="si">}</span><span class="s1">_visualization_map_1_tol_0.npy'</span>
<span class="k">else</span><span class="p">:</span>
    <span class="n">file_max_recall_1</span> <span class="o">=</span> <span class="s1">''</span>

<span class="k">if</span> <span class="n">MAX_CLUSTER_RECALL_2</span> <span class="o">!=</span> <span class="s1">''</span><span class="p">:</span>
    <span class="n">file_max_recall_2</span> <span class="o">=</span> <span class="sa">f</span><span class="s1">'</span><span class="si">{</span><span class="n">viz_folder</span><span class="si">}</span><span class="s1">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_CLUSTER_RECALL_2</span><span class="si">}</span><span class="s1">_visualization_map_2_tol_0.npy'</span>
<span class="k">else</span><span class="p">:</span>
    <span class="n">file_max_recall_2</span> <span class="o">=</span> <span class="s1">''</span>
</pre></div>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=4c68d72c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span> 
</pre></div>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=69a04a8c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[17]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="c1"># build AF npy path explicitly and debug-print</span>
<span class="n">af1_npy</span> <span class="o">=</span> <span class="p">(</span><span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">VIZ_FOLDER</span><span class="si">}</span><span class="s2">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_AF_1</span><span class="si">}</span><span class="s2">_visualization_map_1_tol_0.npy"</span>
    <span class="k">if</span> <span class="n">MAX_AF_1</span> <span class="k">else</span> <span class="s2">""</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"VIZ_FOLDER:"</span><span class="p">,</span> <span class="n">VIZ_FOLDER</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"file_max_recall_1:"</span><span class="p">,</span> <span class="n">file_max_recall_1</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">),</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"af1_npy:"</span><span class="p">,</span> <span class="n">af1_npy</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">)</span> <span class="k">if</span> <span class="n">af1_npy</span> <span class="k">else</span> <span class="kc">None</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"fold_pair:"</span><span class="p">,</span> <span class="n">fold_pair</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"viz_folder:"</span><span class="p">,</span> <span class="n">viz_folder</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"MAX_CLUSTER_RECALL_1:"</span><span class="p">,</span> <span class="n">MAX_CLUSTER_RECALL_1</span><span class="p">,</span> <span class="s2">"MAX_CLUSTER_RECALL_2:"</span><span class="p">,</span> <span class="n">MAX_CLUSTER_RECALL_2</span><span class="p">)</span>


<span class="k">if</span> <span class="n">df_cmap</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">file_max_recall_1</span> <span class="ow">and</span> <span class="n">af1_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">)</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_1</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap2</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">,</span> <span class="n">af1_npy</span><span class="p">,</span>
                       <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 1 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_1</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold1"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">) '</span>
                                   <span class="sa">f</span><span class="s1">'Vs Best AF 1 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">file_max_recall_1</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">):</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">file_max_recall_1</span><span class="p">,</span>
                      <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 1 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_1</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold1"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">af1_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_1</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold1</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">af1_npy</span><span class="p">,</span> <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best AF 1 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">else</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] fold1 viz: missing both maps"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] cmap fold1 viz: df_cmap not available"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>VIZ_FOLDER: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1jfkA_2nxqB/output_cmap_esm/VizCmaps
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>file_max_recall_1:  exists? False
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>af1_npy:  exists? None
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>fold_pair: 1jfkA_2nxqB
viz_folder: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1jfkA_2nxqB/output_cmap_esm/VizCmaps
MAX_CLUSTER_RECALL_1:  MAX_CLUSTER_RECALL_2: 
[skip] cmap fold1 viz: df_cmap not available
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell" id="cell-id=a4dd4c44">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[18]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span><span class="n">af2_npy</span> <span class="o">=</span> <span class="p">(</span><span class="sa">f</span><span class="s2">"</span><span class="si">{</span><span class="n">VIZ_FOLDER</span><span class="si">}</span><span class="s2">/msa_t__ShallowMsa_</span><span class="si">{</span><span class="n">MAX_AF_2</span><span class="si">}</span><span class="s2">_visualization_map_2_tol_0.npy"</span>
    <span class="k">if</span> <span class="n">MAX_AF_2</span> <span class="k">else</span> <span class="s2">""</span><span class="p">)</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">"VIZ_FOLDER:"</span><span class="p">,</span> <span class="n">VIZ_FOLDER</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"file_max_recall_2:"</span><span class="p">,</span> <span class="n">file_max_recall_2</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">),</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">"af2_npy:"</span><span class="p">,</span> <span class="n">af2_npy</span><span class="p">,</span> <span class="s2">"exists?"</span><span class="p">,</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">)</span> <span class="k">if</span> <span class="n">af2_npy</span> <span class="k">else</span> <span class="kc">None</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>

<span class="k">if</span> <span class="n">df_cmap</span> <span class="ow">is</span> <span class="ow">not</span> <span class="kc">None</span> <span class="ow">and</span> <span class="ow">not</span> <span class="n">df_cmap</span><span class="o">.</span><span class="n">empty</span><span class="p">:</span>
    <span class="k">if</span> <span class="n">file_max_recall_2</span> <span class="ow">and</span> <span class="n">af2_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">)</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_2</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap2</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">,</span> <span class="n">af2_npy</span><span class="p">,</span>
                       <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 2 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_2</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold2"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">) '</span>
                                   <span class="sa">f</span><span class="s1">'Vs Best AF 2 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">file_max_recall_2</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">):</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">file_max_recall_2</span><span class="p">,</span>
                      <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best Recall Fold 2 (</span><span class="si">{</span><span class="nb">getattr</span><span class="p">(</span><span class="n">MAX_RECALL_2</span><span class="p">,</span><span class="w"> </span><span class="s2">"recall_only_fold2"</span><span class="p">,</span><span class="w"> </span><span class="s2">"NA"</span><span class="p">)</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">elif</span> <span class="n">af2_npy</span> <span class="ow">and</span> <span class="n">os</span><span class="o">.</span><span class="n">path</span><span class="o">.</span><span class="n">exists</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">):</span>
        <span class="n">af_cmap_recall</span> <span class="o">=</span> <span class="n">df_cmap</span><span class="p">[</span><span class="n">df_cmap</span><span class="o">.</span><span class="n">File</span><span class="o">.</span><span class="n">str</span><span class="o">.</span><span class="n">contains</span><span class="p">(</span><span class="n">MAX_AF_2</span><span class="p">)]</span><span class="o">.</span><span class="n">recall_only_fold2</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
        <span class="n">plot_viz_cmap</span><span class="p">(</span><span class="n">af2_npy</span><span class="p">,</span> <span class="n">legend_plot</span><span class="o">=</span><span class="sa">f</span><span class="s1">'Best AF 2 (</span><span class="si">{</span><span class="n">af_cmap_recall</span><span class="si">}</span><span class="s1">)'</span><span class="p">)</span>
    <span class="k">else</span><span class="p">:</span>
        <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] fold2 viz: missing both maps"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
<span class="k">else</span><span class="p">:</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">"[skip] cmap fold2 viz: df_cmap not available"</span><span class="p">,</span> <span class="n">flush</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
</div>
<div class="jp-Cell-outputWrapper">
<div class="jp-Collapser jp-OutputCollapser jp-Cell-outputCollapser">
</div>
<div class="jp-OutputArea jp-Cell-outputArea">
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>VIZ_FOLDER: /sci/labs/orzuk/orzuk/github/MsaCluster/Pipeline/1jfkA_2nxqB/output_cmap_esm/VizCmaps
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>file_max_recall_2:  exists? False
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>af2_npy:  exists? None
</pre>
</div>
</div>
<div class="jp-OutputArea-child">
<div class="jp-OutputPrompt jp-OutputArea-prompt"></div>
<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain" tabindex="0">
<pre>[skip] cmap fold2 viz: df_cmap not available
</pre>
</div>
</div>
</div>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs" id="cell-id=b0112a5c">
<div class="jp-Cell-inputWrapper" tabindex="0">
<div class="jp-Collapser jp-InputCollapser jp-Cell-inputCollapser">
</div>
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In[]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
<div class="cm-editor cm-s-jupyter">
<div class="highlight hl-ipython3"><pre><span></span> 
</pre></div>
</div>
</div>
</div>
</div>
</div>
</main>
</body>
</html>
